<!doctype html>
<html>
<head>
<base target="_blank" />
<table border=1 style="table-layout:fixed;">
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
	<META NAME="Author" CONTENT="zhouyunlai@novogene.cn">
    <META NAME="Version" CONTENT="20171103" "Migratory Bird ^_^">
<title></title><link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,700,400&subset=latin,latin-ext' rel='stylesheet' type='text/css' /><style type='text/css'>html {overflow-x: initial !important;}.CodeMirror { height: auto; }
.CodeMirror-scroll { overflow-y: hidden; overflow-x: auto; }
.CodeMirror-lines { padding: 4px 0px; }
.CodeMirror pre { }
.CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler { background-color: white; }
.CodeMirror-gutters { border-right: 1px solid rgb(221, 221, 221); background-color: rgb(247, 247, 247); white-space: nowrap; }
.CodeMirror-linenumbers { }
.CodeMirror-linenumber { padding: 0px 3px 0px 5px; text-align: right; color: rgb(153, 153, 153); }
.CodeMirror div.CodeMirror-cursor { border-left: 1px solid black; z-index: 3; }
.CodeMirror div.CodeMirror-secondarycursor { border-left: 1px solid silver; }
.CodeMirror.cm-keymap-fat-cursor div.CodeMirror-cursor { width: auto; border: 0px; background: rgb(119, 238, 119); z-index: 1; }
.CodeMirror div.CodeMirror-cursor.CodeMirror-overwrite { }
.cm-tab { display: inline-block; }
.cm-s-typora-default .cm-header, .cm-s-typora-default .cm-property { color: rgb(217, 79, 138); }
.cm-s-typora-default pre.cm-header1:not(.cm-atom) :not(.cm-overlay) { font-size: 2rem; line-height: 2rem; }
.cm-s-typora-default pre.cm-header2:not(.cm-atom) :not(.cm-overlay) { font-size: 1.4rem; line-height: 1.4rem; }
.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number { color: rgb(149, 132, 134); }
.cm-s-typora-default .cm-table-row, .cm-s-typora-default .cm-block-start { font-family: monospace; }
.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code { color: rgb(74, 90, 159); font-family: monospace; }
.cm-s-typora-default .cm-tag { color: rgb(169, 68, 66); }
.cm-s-typora-default .cm-string { color: rgb(126, 134, 169); }
.cm-s-typora-default .cm-link { color: rgb(196, 122, 15); text-decoration: underline; }
.cm-s-typora-default .cm-variable-2, .cm-s-typora-default .cm-variable-1 { color: inherit; }
.cm-s-typora-default .cm-overlay { font-family: monospace; }
.CodeMirror.cm-s-typora-default div.CodeMirror-cursor { border-left: 3px solid rgb(228, 98, 154); }
.cm-s-typora-default .CodeMirror-activeline-background { left: -60px; right: -30px; background: rgba(204, 204, 204, 0.2); }
.cm-s-typora-default .CodeMirror-gutters { border-right: none; background-color: inherit; }
.cm-s-typora-default .cm-trailing-space-new-line::after, .cm-startspace::after, .cm-starttab .cm-tab::after { content: "•"; position: absolute; left: 0px; opacity: 0; font-family: LetterGothicStd, monospace; }
.os-windows .cm-startspace::after, .os-windows .cm-starttab .cm-tab::after { left: -0.1em; }
.cm-starttab .cm-tab::after { content: " "; }
.cm-startspace, .cm-tab, .cm-starttab, .cm-trailing-space-a, .cm-trailing-space-b, .cm-trailing-space-new-line { font-family: monospace; position: relative; }
.cm-s-typora-default .cm-trailing-space-new-line::after { content: "↓"; opacity: 0.3; }
.cm-s-inner .cm-keyword { color: rgb(119, 0, 136); }
.cm-s-inner .cm-atom, .cm-s-inner.cm-atom { color: rgb(34, 17, 153); }
.cm-s-inner .cm-number { color: rgb(17, 102, 68); }
.cm-s-inner .cm-def { color: rgb(0, 0, 255); }
.cm-s-inner .cm-variable { color: black; }
.cm-s-inner .cm-variable-2 { color: rgb(0, 85, 170); }
.cm-s-inner .cm-variable-3 { color: rgb(0, 136, 85); }
.cm-s-inner .cm-property { color: black; }
.cm-s-inner .cm-operator { color: rgb(152, 26, 26); }
.cm-s-inner .cm-comment, .cm-s-inner.cm-comment { color: rgb(170, 85, 0); }
.cm-s-inner .cm-string { color: rgb(170, 17, 17); }
.cm-s-inner .cm-string-2 { color: rgb(255, 85, 0); }
.cm-s-inner .cm-meta { color: rgb(85, 85, 85); }
.cm-s-inner .cm-qualifier { color: rgb(85, 85, 85); }
.cm-s-inner .cm-builtin { color: rgb(51, 0, 170); }
.cm-s-inner .cm-bracket { color: rgb(153, 153, 119); }
.cm-s-inner .cm-tag { color: rgb(17, 119, 0); }
.cm-s-inner .cm-attribute { color: rgb(0, 0, 204); }
.cm-s-inner .cm-header, .cm-s-inner.cm-header { color: blue; }
.cm-s-inner .cm-quote, .cm-s-inner.cm-quote { color: rgb(0, 153, 0); }
.cm-s-inner .cm-hr, .cm-s-inner.cm-hr { color: rgb(153, 153, 153); }
.cm-s-inner .cm-link, .cm-s-inner.cm-link { color: rgb(0, 0, 204); }
.cm-negative { color: rgb(221, 68, 68); }
.cm-positive { color: rgb(34, 153, 34); }
.cm-header, .cm-strong { font-weight: bold; }
.cm-del { text-decoration: line-through; }
.cm-em { font-style: italic; }
.cm-link { text-decoration: underline; }
.cm-error { color: rgb(255, 0, 0); }
.cm-invalidchar { color: rgb(255, 0, 0); }
.cm-constant { color: rgb(38, 139, 210); }
.cm-defined { color: rgb(181, 137, 0); }
div.CodeMirror span.CodeMirror-matchingbracket { color: rgb(0, 255, 0); }
div.CodeMirror span.CodeMirror-nonmatchingbracket { color: rgb(255, 34, 34); }
.cm-s-inner .CodeMirror-activeline-background { background: inherit; }
.CodeMirror { position: relative; overflow: hidden; }
.CodeMirror-scroll { margin-bottom: -30px; margin-right: -30px; padding-bottom: 30px; padding-right: 30px; height: 100%; outline: none; position: relative; box-sizing: content-box; }
.CodeMirror-sizer { position: relative; }
.CodeMirror-vscrollbar, .CodeMirror-hscrollbar, .CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler { position: absolute; z-index: 6; display: none; }
.CodeMirror-vscrollbar { right: 0px; top: 0px; overflow-x: hidden; overflow-y: scroll; }
.CodeMirror-hscrollbar { bottom: 0px; left: 0px; overflow-y: hidden; overflow-x: scroll; }
.CodeMirror-scrollbar-filler { right: 0px; bottom: 0px; }
.CodeMirror-gutter-filler { left: 0px; bottom: 0px; }
.CodeMirror-gutters { position: absolute; left: 0px; top: 0px; padding-bottom: 30px; z-index: 3; }
.CodeMirror-gutter { white-space: normal; height: 100%; box-sizing: content-box; padding-bottom: 30px; margin-bottom: -32px; display: inline-block; }
.CodeMirror-gutter-wrapper { position: absolute; z-index: 4; background: none !important; border: none !important; }
.CodeMirror-gutter-background { position: absolute; top: 0px; bottom: 0px; z-index: 4; }
.CodeMirror-gutter-elt { position: absolute; cursor: default; z-index: 4; }
.CodeMirror-lines { cursor: text; }
.CodeMirror pre { border-radius: 0px; border-width: 0px; background: transparent; font-family: inherit; font-size: inherit; margin: 0px; white-space: pre; word-wrap: normal; color: inherit; z-index: 2; position: relative; overflow: visible; }
.CodeMirror-wrap pre { word-wrap: break-word; white-space: pre-wrap; word-break: normal; }
.CodeMirror-code pre { border-right: 30px solid transparent; width: fit-content; }
.CodeMirror-wrap .CodeMirror-code pre { border-right: none; width: auto; }
.CodeMirror-linebackground { position: absolute; left: 0px; right: 0px; top: 0px; bottom: 0px; z-index: 0; }
.CodeMirror-linewidget { position: relative; z-index: 2; overflow: auto; }
.CodeMirror-widget { }
.CodeMirror-wrap .CodeMirror-scroll { overflow-x: hidden; }
.CodeMirror-measure { position: absolute; width: 100%; height: 0px; overflow: hidden; visibility: hidden; }
.CodeMirror-measure pre { position: static; }
.CodeMirror div.CodeMirror-cursor { position: absolute; visibility: hidden; border-right: none; width: 0px; }
.CodeMirror div.CodeMirror-cursor { visibility: hidden; }
.CodeMirror-focused div.CodeMirror-cursor { visibility: inherit; }
.CodeMirror-selected { background: rgb(217, 217, 217); }
.CodeMirror-focused .CodeMirror-selected { background: rgb(215, 212, 240); }
.cm-searching { background: rgba(255, 255, 0, 0.4); }
.CodeMirror span { }
@media print {
  .CodeMirror div.CodeMirror-cursor { visibility: hidden; }
}
.CodeMirror-lint-markers { width: 16px; }
.CodeMirror-lint-tooltip { background-color: infobackground; border: 1px solid black; border-radius: 4px; color: infotext; font-family: monospace; overflow: hidden; padding: 2px 5px; position: fixed; white-space: pre-wrap; z-index: 10000; max-width: 600px; opacity: 0; transition: opacity 0.4s; font-size: 0.8em; }
.CodeMirror-lint-mark-error, .CodeMirror-lint-mark-warning { background-position: left bottom; background-repeat: repeat-x; }
.CodeMirror-lint-mark-error { background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAADCAYAAAC09K7GAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9sJDw4cOCW1/KIAAAAZdEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIEdJTVBXgQ4XAAAAHElEQVQI12NggIL/DAz/GdA5/xkY/qPKMDAwAADLZwf5rvm+LQAAAABJRU5ErkJggg=="); }
.CodeMirror-lint-marker-error, .CodeMirror-lint-marker-warning { background-position: center center; background-repeat: no-repeat; cursor: pointer; display: inline-block; height: 16px; width: 16px; vertical-align: middle; position: relative; }
.CodeMirror-lint-message-error, .CodeMirror-lint-message-warning { padding-left: 18px; background-position: left top; background-repeat: no-repeat; }
.CodeMirror-lint-marker-error, .CodeMirror-lint-message-error { background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAHlBMVEW7AAC7AACxAAC7AAC7AAAAAAC4AAC5AAD///+7AAAUdclpAAAABnRSTlMXnORSiwCK0ZKSAAAATUlEQVR42mWPOQ7AQAgDuQLx/z8csYRmPRIFIwRGnosRrpamvkKi0FTIiMASR3hhKW+hAN6/tIWhu9PDWiTGNEkTtIOucA5Oyr9ckPgAWm0GPBog6v4AAAAASUVORK5CYII="); }
.CodeMirror-lint-marker-warning, .CodeMirror-lint-message-warning { background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAANlBMVEX/uwDvrwD/uwD/uwD/uwD/uwD/uwD/uwD/uwD6twD/uwAAAADurwD2tQD7uAD+ugAAAAD/uwDhmeTRAAAADHRSTlMJ8mN1EYcbmiixgACm7WbuAAAAVklEQVR42n3PUQqAIBBFUU1LLc3u/jdbOJoW1P08DA9Gba8+YWJ6gNJoNYIBzAA2chBth5kLmG9YUoG0NHAUwFXwO9LuBQL1giCQb8gC9Oro2vp5rncCIY8L8uEx5ZkAAAAASUVORK5CYII="); }
.CodeMirror-lint-marker-multiple { background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAcAAAAHCAMAAADzjKfhAAAACVBMVEUAAAAAAAC/v7914kyHAAAAAXRSTlMAQObYZgAAACNJREFUeNo1ioEJAAAIwmz/H90iFFSGJgFMe3gaLZ0od+9/AQZ0ADosbYraAAAAAElFTkSuQmCC"); background-repeat: no-repeat; background-position: right bottom; width: 100%; height: 100%; }


:root { --bg-color: #ffffff; --text-color: #333333; --code-block-bg-color: inherit; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; bottom: 0px; top: 0px; left: 0px; right: 0px; font-size: 1rem; line-height: 1.42857; overflow-x: hidden; background: inherit; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { background: rgb(181, 214, 252); text-shadow: none; }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; padding-bottom: 70px; white-space: pre-wrap; overflow-x: visible; contain: layout paint; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
@media screen and (max-width: 500px) {
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
.typora-export #write { margin: 0px auto; }
#write > p:first-child, #write > ul:first-child, #write > ol:first-child, #write > pre:first-child, #write > blockquote:first-child, #write > div:first-child, #write > table:first-child { margin-top: 30px; }
#write li > table:first-child { margin-top: -20px; }
img { max-width: 100%; vertical-align: middle; }
input, button, select, textarea { color: inherit; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: inherit; line-height: inherit; font-family: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
::before, ::after, * { box-sizing: border-box; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write div, #write pre { width: inherit; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6 { position: relative; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
p { -webkit-margin-before: 1rem; -webkit-margin-after: 1rem; -webkit-margin-start: 0px; -webkit-margin-end: 0px; }
.typora-export p { white-space: normal; }
.mathjax-block { margin-top: 0px; margin-bottom: 0px; -webkit-margin-before: 0rem; -webkit-margin-after: 0rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: bold; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-radius: 4px; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; margin: 4px 0px 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 80px; }
.CodeMirror-gutters { border-right: 0px; background-color: inherit; }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
pre { white-space: pre-wrap; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background: var(--code-block-bg-color); position: relative !important; }
.md-diagram-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
.md-fences .CodeMirror.CodeMirror-wrap { top: -1.6em; margin-bottom: -1.6em; }
.md-fences.mock-cm { white-space: pre-wrap; }
.show-fences-line-number .md-fences { padding-left: 0px; }
.show-fences-line-number .md-fences.mock-cm { padding-left: 40px; }
.footnotes { opacity: 0.8; font-size: 0.9rem; padding-top: 1em; padding-bottom: 1em; }
.footnotes + .footnotes { margin-top: -1em; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; background: transparent; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; -webkit-tap-highlight-color: transparent; line-height: normal; font-weight: normal; text-align: left; box-sizing: content-box; direction: ltr; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li p, li .mathjax-block { margin: 0.5rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; }
@media print {
  html, body { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  h1, h2, h3, h4, h5, h6 { break-after: avoid-page; orphans: 2; }
  p { orphans: 4; }
  html.blink-to-pdf { font-size: 13px; }
  .typora-export #write { padding-left: 1cm; padding-right: 1cm; padding-bottom: 0px; break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  @page { margin: 20mm 0mm; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 2.86rem; white-space: pre-wrap; background: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p .md-image:only-child { display: inline-block; width: 100%; text-align: center; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.mathjax-block { white-space: pre; overflow: hidden; width: 100%; }
p + .mathjax-block { margin-top: -1.143rem; }
.mathjax-block:not(:empty)::after { display: none; }
[contenteditable="true"]:active, [contenteditable="true"]:focus { outline: none; box-shadow: none; }
.task-list { list-style-type: none; }
.task-list-item { position: relative; padding-left: 1em; }
.task-list-item input { position: absolute; top: 0px; left: 0px; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc::after, .md-toc-content::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: bold; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) {
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
.md-tag { opacity: 0.5; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: monospace; }
code { text-align: left; }
h1 .md-tag, h2 .md-tag, h3 .md-tag, h4 .md-tag, h5 .md-tag, h6 .md-tag { font-weight: initial; opacity: 0.35; }
a.md-print-anchor { border-width: initial !important; border-style: none !important; border-color: initial !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: none !important; background: transparent !important; text-decoration: initial !important; text-shadow: initial !important; }
.md-inline-math .MathJax_SVG .noError { display: none !important; }
.mathjax-block .MathJax_SVG_Display { text-align: center; margin: 1em 0em; position: relative; text-indent: 0px; max-width: none; max-height: none; min-height: 0px; min-width: 100%; width: auto; display: block !important; }
.MathJax_SVG_Display, .md-inline-math .MathJax_SVG_Display { width: auto; margin: inherit; display: inline-block !important; }
.MathJax_SVG .MJX-monospace { font-family: monospace; }
.MathJax_SVG .MJX-sans-serif { font-family: sans-serif; }
.MathJax_SVG { display: inline; font-style: normal; font-weight: normal; line-height: normal; zoom: 90%; text-indent: 0px; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0px; min-height: 0px; border: 0px; padding: 0px; margin: 0px; }
.MathJax_SVG * { transition: none; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; }


:root { --side-bar-bg-color: #fafafa; --control-text-color: #777; }
@font-face { font-family: "Open Sans"; font-style: normal; font-weight: normal; src: local("Open Sans Regular"), url("./github/400.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: normal; src: local("Open Sans Italic"), url("./github/400i.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: normal; font-weight: bold; src: local("Open Sans Bold"), url("./github/700.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: bold; src: local("Open Sans Bold Italic"), url("./github/700i.woff") format("woff"); }
html { font-size: 16px; }
body { font-family: "Open Sans", "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif; color: rgb(51, 51, 51); line-height: 1.6; }
#write { max-width: 860px; margin: 0px auto; padding: 20px 30px 100px; }
#write > ul:first-child, #write > ol:first-child { margin-top: 30px; }
body > :first-child { margin-top: 0px !important; }
body > :last-child { margin-bottom: 0px !important; }
a { color: rgb(65, 131, 196); }
h1, h2, h3, h4, h5, h6 { position: relative; margin-top: 1rem; margin-bottom: 1rem; font-weight: bold; line-height: 1.4; cursor: text; }
h1:hover a.anchor, h2:hover a.anchor, h3:hover a.anchor, h4:hover a.anchor, h5:hover a.anchor, h6:hover a.anchor { text-decoration: none; }
h1 tt, h1 code { font-size: inherit; }
h2 tt, h2 code { font-size: inherit; }
h3 tt, h3 code { font-size: inherit; }
h4 tt, h4 code { font-size: inherit; }
h5 tt, h5 code { font-size: inherit; }
h6 tt, h6 code { font-size: inherit; }
h1 { padding-bottom: 0.3em; font-size: 2.25em; line-height: 1.2; border-bottom: 1px solid rgb(238, 238, 238); }
h2 { padding-bottom: 0.3em; font-size: 1.75em; line-height: 1.225; border-bottom: 1px solid rgb(238, 238, 238); }
h3 { font-size: 1.5em; line-height: 1.43; }
h4 { font-size: 1.25em; }
h5 { font-size: 1em; }
h6 { font-size: 1em; color: rgb(119, 119, 119); }
p, blockquote, ul, ol, dl, table { margin: 0.8em 0px; }
li > ol, li > ul { margin: 0px; }
hr { height: 4px; padding: 0px; margin: 16px 0px; background-color: rgb(231, 231, 231); border-width: 0px 0px 1px; border-style: none none solid; border-top-color: initial; border-right-color: initial; border-left-color: initial; border-image: initial; overflow: hidden; box-sizing: content-box; border-bottom-color: rgb(221, 221, 221); }
body > h2:first-child { margin-top: 0px; padding-top: 0px; }
body > h1:first-child { margin-top: 0px; padding-top: 0px; }
body > h1:first-child + h2 { margin-top: 0px; padding-top: 0px; }
body > h3:first-child, body > h4:first-child, body > h5:first-child, body > h6:first-child { margin-top: 0px; padding-top: 0px; }
a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 { margin-top: 0px; padding-top: 0px; }
h1 p, h2 p, h3 p, h4 p, h5 p, h6 p { margin-top: 0px; }
li p.first { display: inline-block; }
ul, ol { padding-left: 30px; }
ul:first-child, ol:first-child { margin-top: 0px; }
ul:last-child, ol:last-child { margin-bottom: 0px; }
blockquote { border-left: 4px solid rgb(221, 221, 221); padding: 0px 15px; color: rgb(119, 119, 119); }
blockquote blockquote { padding-right: 0px; }
table { padding: 0px; word-break: initial; }
table tr { border-top: 1px solid rgb(204, 204, 204); margin: 0px; padding: 0px; }
table tr:nth-child(2n) { background-color: rgb(248, 248, 248); }
table tr th { font-weight: bold; border: 1px solid rgb(204, 204, 204); text-align: left; margin: 0px; padding: 6px 13px; }
table tr td { border: 1px solid rgb(204, 204, 204); text-align: left; margin: 0px; padding: 6px 13px; }
table tr th:first-child, table tr td:first-child { margin-top: 0px; }
table tr th:last-child, table tr td:last-child { margin-bottom: 0px; }
.CodeMirror-gutters { border-right: 1px solid rgb(221, 221, 221); }
.md-fences, code, tt { border: 1px solid rgb(221, 221, 221); background-color: rgb(248, 248, 248); border-radius: 3px; font-family: Consolas, "Liberation Mono", Courier, monospace; padding: 2px 4px 0px; font-size: 0.9em; }
.md-fences { margin-bottom: 15px; margin-top: 15px; padding: 8px 1em 6px; }
.task-list { padding-left: 0px; }
.task-list-item { padding-left: 32px; }
.task-list-item input { top: 3px; left: 8px; }
@media screen and (min-width: 914px) {
}
@media print {
  html { font-size: 13px; }
  table, pre { break-inside: avoid; }
  pre { word-wrap: break-word; }
}
.md-fences { background-color: rgb(248, 248, 248); }
#write pre.md-meta-block { padding: 1rem; font-size: 85%; line-height: 1.45; background-color: rgb(247, 247, 247); border: 0px; border-radius: 3px; color: rgb(119, 119, 119); margin-top: 0px !important; }
.mathjax-block > .code-tooltip { bottom: 0.375rem; }
#write > h3.md-focus::before { left: -1.5625rem; top: 0.375rem; }
#write > h4.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h5.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h6.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
.md-image > .md-meta { border: 1px solid rgb(221, 221, 221); border-radius: 3px; font-family: Consolas, "Liberation Mono", Courier, monospace; padding: 2px 4px 0px; font-size: 0.9em; color: inherit; }
.md-tag { color: inherit; }
.md-toc { margin-top: 20px; padding-bottom: 20px; }
.sidebar-tabs { border-bottom: none; }
#typora-quick-open { border: 1px solid rgb(221, 221, 221); background-color: rgb(248, 248, 248); }
#typora-quick-open-item { background-color: rgb(250, 250, 250); border-color: rgb(254, 254, 254) rgb(229, 229, 229) rgb(229, 229, 229) rgb(238, 238, 238); border-style: solid; border-width: 1px; }
#md-notification::before { top: 10px; }
.on-focus-mode blockquote { border-left-color: rgba(85, 85, 85, 0.12); }
header, .context-menu, .megamenu-content, footer { font-family: "Segoe UI", Arial, sans-serif; }
.file-node-content:hover .file-node-icon, .file-node-content:hover .file-node-open-state { visibility: visible; }
.mac-seamless-mode #typora-sidebar { background-color: var(--side-bar-bg-color); }
.md-lang { color: rgb(180, 101, 77); }

</style>
</head>
<body class='typora-export' >
<div  id='write'  class = 'is-node'><table><thead><tr><th>result</th><th></th></tr></thead><tbody><tr><td>
|-- <a href='../../../../00.CleanData'>00.CleanData</a>【若00.CleanData为压缩包，解压后链接生效】</td><td>【质控后可用于后续分析的有效数据】</td></tr><tr><td>
|   |-- <a href='../../../../00.CleanData/data_table.xls'>data_table.xls</a></td><td>【数据预处理统计及质控信息表】</td></tr><tr><td>
|   |-- Sample_Name</td><td>【每个样品的质控结果】</td></tr><tr><td>
|   |   |-- *.effective.fastq.gz</td><td>【去除嵌合体之后的fastq序列】</td></tr><tr><td>
|   |   |-- *.fna.gz</td><td>【去除嵌合体之后的fasta序列】</td></tr><tr><td>
|-- <a href='../../../../01.RawData'>01.RawData</a>【若01.RawData为压缩包，解压后链接生效】</td><td>【原始下机数据和原始拼接后数据】</td></tr><tr><td>
|   |-- Sample_Name</td><td>【每个样品对应的原始下机数据和原始拼接后数据】</td></tr><tr><td>
|   |   |-- *.extendedFrags.fastq</td><td>【原始下机的 reads 拼接后的序列】</td></tr><tr><td>
|   |   |-- *_1.fastq.gz</td><td>【reads1 去除 baraode 和 primer 后得到的序列】</td></tr><tr><td>
|   |   |-- *_2.fastq.gz</td><td>【reads2 去除 barcode 和 primer 后得到的序列】</td></tr><tr><td>
|   |   |-- *.raw_1.fastq.gz</td><td>【reads1 未去除 baraode 和 primer 的原始序列】</td></tr><tr><td>
|   |   |-- *.raw_2.fastq.gz</td><td>【reads2 未去除 baraode 和 primer 的原始序列】</td></tr><tr><td>
|   |-- <a href='../../../../01.RawData/data_table.xls'>data_table.xls</a></td><td>【数据预处理统计及质控信息表】</td></tr><tr><td>
|-- <a href='../../../result/02.FeatureAnalysis'>02.FeatureAnalysis</a></td><td>【特征序列和特征表】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/aligned-rep-seqs.qza'>aligned-rep-seqs.qza</a></td><td>【多序列比对结果】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/denoisingTable_qza'>denoisingTable_qza</a></td><td>【去噪丰度qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/denoisingTable.qza'>denoisingTable.qza</a></td><td>【去噪丰度qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/denoisingTable_qzv'>denoisingTable_qzv</a></td><td>【去噪丰度qzv导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/denoisingTable.qzv'>denoisingTable.qzv</a></td><td>【去噪丰度qzv文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/feature.fasta'>feature.fasta</a></td><td>【代表序列】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureSeqs_qza'>featureSeqs_qza</a></td><td>【代表序列qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureSeqs.qza'>ffeatureSeqs.qza</a></td><td>【代表序列qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureSeqs_qzv'>featureSeqs_qzv</a></td><td>【代表序列qzv导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureSeqs.qzv'>featureSeqs.qzv</a></td><td>【代表序列qzv文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureTable_qza'>featureTable_qza</a></td><td>【特征表丰度qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureTable.qza'>featureTable.qza</a></td><td>【特征表丰度qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureTable_qzv'>featureTable_qzv</a></td><td>【特征表丰度qzv导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/featureTable.qzv'>featureTable.qzv</a></td><td>【特征表丰度qzv文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/feature.tax_assignments.txt'>feature.tax_assignments.txt</a></td><td>【物种注释结果】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/masked-aligned-rep-seqs.qza'>masked-aligned-rep-seqs.qza</a></td><td>【过滤去除高变区后的多序列比对结果】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/rooted-tree_qza'>rooted-tree_qza</a></td><td>【有根树qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/rooted-tree.qza'>rooted-tree.qza</a></td><td>【有根树qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/unrooted-tree_qza'>unrooted-tree_qza</a></td><td>【无根树qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/unrooted-tree.qza'>unrooted-tree.qza</a></td><td>【无根树qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/seq_taxonomy_qza'>seq_taxonomy_qza</a></td><td>【物种注释qza导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/seq_taxonomy.qza'>seq_taxonomy.qza</a></td><td>【物种注释qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/seq_taxonomy_qzv'>seq_taxonomy_qzv</a></td><td>【物种注释qzv导出文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/seq_taxonomy.qzv'>seq_taxonomy.qzv</a></td><td>【物种注释qzv文件】</td></tr><tr><td>
|   |-- <a href='../../../result/02.FeatureAnalysis/tables'>tables</a></td><td>【特征表】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组的特征表】</td></tr><tr><td>
|   |   |   |-- featureTable_Absolute</td><td>【各（样本,组）绝对丰度特征表】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.*.absolute.txt</td><td>【各（样本,组）总界门纲目科属种（total,k,p,c,o,f,g,s）水平上的绝对丰度】</td></tr><tr><td>
|   |   |   |-- featureTable_Relative</td><td>【各（样本,组）相对丰度特征表】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.*.relative.xls</td><td>【各（样本,组）总界门纲目科属种（total,k,p,c,o,f,g,s）水平上的相对丰度】</td></tr><tr><td>
|   |   |   |-- featureTable_Even</td><td>【各（样本,组）均一化处理后的绝对丰度特征表（均一化处理使每个样品的各个 OTU/ASV 丰度之和相等）】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.total.even.txt</td><td>【各（样本,组）均一化处理后的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.*.absolute.xls</td><td>【各（样本,组）界门纲目科属种（k,p,c,o,f,g,s）水平上均一化处理后的绝对丰度】</td></tr><tr><td>
|   |   |   |-- featureTable_Biom</td><td>【各（样本,组）biom文件】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.total.absolute.txt.biom</td><td>【各（样本,组）的绝对丰度 biom表】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.total.even.txt.biom</td><td>【各（样本,组）均一化处理后的绝对丰度 biom表】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.total.relative.txt.biom</td><td>【各（样本,组）的相对丰度 biom表】</td></tr><tr><td>
|   |   |   |-- featureTable_tableStat</td><td>【各（样本,组）注释结果统计】</td></tr><tr><td>
|   |   |   |   |-- featureTable.briefStatInfo.txt</td><td>【各（样本,组）简版注释统计结果】</td></tr><tr><td>
|   |   |   |   |-- featureTable.relativeStat.*.xls</td><td>【各（样本,组）各层级注释数量统计】</td></tr><tr><td>
|   |   |   |   |-- classifiedStat.sample.absolute.{xls,svg,png}</td><td>【各（样本,组）各层级注释统计结果及可视化展示】</td></tr><tr><td>
|   |   |   |   |-- featureTable.*.Tags_stat.xls</td><td>【各（样本,组）Tags统计结果】</td></tr><tr><td> 
|   |   |   |   |-- featureTable.*.TagsDistribution.{svg,,png}}</td><td>【各（样本,组）Tags统计结果可视化展示】</td></tr><tr><td>
|-- <a href='../../../result/03.TaxonVisual'>03.TaxonVisual</a></td><td>【注释结果可视化】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/barplot'>barplot<div id="barplot"></div></a></td><td>【物种相对丰度柱形图展示】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组的物种相对丰度柱形图】</td></tr><tr><td>
|   |   |   |-- top10.*.*.barplot.{xls,png,svg}</td><td>【各（样本,组）门纲目科属种（p,c,o,f,g,s）水平上的 top10 物种相对丰度柱形图】</td></tr><tr><td>
|   |   |   |-- top30.*.g.barplot.{xls,png,svg}</td><td>【各（样本,组）属(g)水平的top30物种相对丰度柱形图】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/heatmap'>heatmap<div id="heatmap"></div></a></td><td>【物种注释聚类热图】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组的物种注释聚类热图】</td></tr><tr><td>
|   |   |   |-- heatmap.*.*.{pdf,png,txt}</td><td>【各（样本,组）门纲目科属种（p,c,o,f,g,s）水平上的物种聚类热图】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/tree100'>tree100</a></td><td>【属水平top100物种进化树圈图】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组的属水平top100物种进化树圈图】</td></tr><tr><td>
|   |   |   |-- genus_100.tree</td><td>【属水平top100物种进化树】</td></tr><tr><td>
|   |   |   |-- genus_100.tree.{png,svg}</td><td>【属水平top100物种进化树圈图】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/venn'>venn<div id="venn"></div></a></td><td>【基于 OTU/ASV 的Venn图】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组基于 OTU/ASV 的Venn图】</td></tr><tr><td>
|   |   |   |-- *</td><td>【进行venn图分析的（样本,组）名字，以-连接，例A-B】</td></tr><tr><td>
|   |   |   |   |-- *_*.txt</td><td>【（样本,组） 特有的OTU/ASV,例 A-B_A.txt(A特有)】</td></tr><tr><td>
|   |   |   |   |-- *_*:*.txt</td><td>【（样本,组） 共有的OTU/ASV,例 A-B_A：B.txt(A,B共有)】</td></tr><tr><td>
|   |   |   |   |-- *.{png,svg}</td><td>【（样本,组）Venn图展示】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/flower'>flower<div id="flower"></div></a></td><td>【基于 OTU/ASV 的花瓣图】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组基于 OTU/ASV 的花瓣图】</td></tr><tr><td>
|   |   |   |-- *</td><td>【进行花瓣图分析的（样本,组）名字，以-连接，例A-B-C-D-E-F】</td></tr><tr><td>
|   |   |   |   |-- *_all-core.xls</td><td>【（样本,组） 共有的OTU/ASV,例 A-B-C-D-E-F_all-core.xls(共有OTU/ASV)】</td></tr><tr><td>
|   |   |   |   |-- *_*.xls</td><td>【每个（样本,组） 特有的OTU/ASV,例 A-B-C-D-E-F_A.xls(A特有)】</td></tr><tr><td>
|   |   |   |   |-- *.{xls,png,svg}</td><td>【（样本,组）花瓣图作图数据及可视化展示】</td></tr><tr><td>
|   |-- <a href='../../../result/03.TaxonVisual/ternary'>ternary</a></td><td>【三元相图分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及分组三元相图分析结果】</td></tr><tr><td>
|   |   |   |-- */*.ternary.{jpeg,pdf}</td><td>【各（样本,组）门纲目科属种（p,c,o,f,g,s）水平上的三元相图可视化展示】</td></tr><tr><td>
|-- <a href='../../../result/04.AlphaDiversity'>04.AlphaDiversity</a></td><td>【alpha 多样性分析结果】</td></tr><tr><td>
|   |-- <a href='../../../result/04.AlphaDiversity/alpha_index_table'>alpha_index_table</a></td><td>【alpha 多样性指数】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 alpha 多样性指数】</td></tr><tr><td>
|   |   |   |-- alpha_diversity_index.{sample,group}.txt</td><td>【 alpha 多样性指数】</td></tr><tr><td>
|   |-- <a href='../../../result/04.AlphaDiversity/visual_specaccum'>visual_specaccum</a></td><td>【物种累积曲线】</td></tr><tr><td>
|   |   |-- <a href='../../../result/04.AlphaDiversity/visual_specaccum/sample'>sample</a></td><td>【样本的物种累积曲线】</td></tr><tr><td>
|   |   |   |-- specaccum_test.{pdf,png}</td><td>【（样本,组）物种累积曲线图】</td></tr><tr><td>
|   |-- <a href='../../../result/04.AlphaDiversity/visual_rankAbundance'>visual_rankAbundance</a></td><td>【Rank Abundance 曲线】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Rank Abundance 曲线】</td></tr><tr><td>
|   |   |   |-- rank_abundance.{sample,group}.{pdf,png}</td><td>【 Rank Abundance 曲线展示】</td></tr><tr><td>
|   |-- <a href='../../../result/04.AlphaDiversity/visual_rarefaction'>visual_rarefaction</a></td><td>【稀疏性曲线】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 稀疏性曲线】</td></tr><tr><td>
|   |   |   |-- */*.{sample,group}.{pdf,png,xls}</td><td>【(chao1,goods_coverage,observed_species,PD_whole_tree,shannon,simpson)稀疏性曲线展示】</td></tr><tr><td>
|   |-- <a href='../../../result/04.AlphaDiversity/visual_alphaDiversity'>visual_alphaDiversity</a></td><td>【alpha 多样性组间比较箱型图】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间比较箱型图】</td></tr><tr><td>
|   |   |   |-- *</td><td>【组间 (ACE,chao1,goods_coverage,observed_species,PD_whole_tree,shannon,simpson) 比较箱型图】</td></tr><tr><td>
|   |   |   |   |-- *.{pdf,png}</td><td>【指数箱型图】</td></tr><tr><td>
|   |   |   |   |-- *_kruskalWallis.{txt,pdf,png}</td><td>【多组间非参数 kruskalWallis 检验及可视化（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- *_Tukey.{txt,pdf,png}</td><td>【多组间方差 Tukey 分析及可视化（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- *_test.{txt,pdf,png}</td><td>【组间 T-test 检验及可视化（组内样本>2,组=2）】</td></tr><tr><td>
|   |   |   |   |-- *_wilcox.{txt,pdf,png}</td><td>【组间 wilcox秩和检验及可视化（组内样本>2,组=2）】</td></tr><tr><td>
|-- <a href='../../../result/05.BetaDiversity'>05.BetaDiversity</a></td><td>【Beta 多样性分析结果】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_distance'>beta_distance</a></td><td>【Beta 距离分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta 距离分析结果】</td></tr><tr><td>
|   |   |   |-- *_distance_matrix_qza</td><td>【(bray_curtis,jaccard,(un)weighted_unifrac) 距离qza导出文件】</td></tr><tr><td>
|   |   |   |-- *_distance_matrix.qza</td><td>【(bray_curtis,jaccard,(un)weighted_unifrac) 距离qza文件】</td></tr><tr><td>
|   |   |   |-- *_pcoa_results_qza</td><td>【(bray_curtis,jaccard,(un)weighted_unifrac) PCoA 分析qza导出文件】</td></tr><tr><td>
|   |   |   |-- *_pcoa_results.qza</td><td>【(bray_curtis,jaccard,(un)weighted_unifrac) PCoA 分析qza文件】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_upgma'>beta_upgma</a></td><td>【Beta UPGMA 分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta UPGMA 分析结果】</td></tr><tr><td>
|   |   |   |-- (un)weighted_unifrac</td><td>【基于(非)加权 unifrac 距离 Beta UPGMA 分析结果】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac.{sample,group}_upgma.tre</td><td>【基于(非)加权 unifrac 距离的 UPGMA 聚类树文件，MEGA 软件可以打开】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac.{sample,group}.{pdf,png}</td><td>【基于(非)加权 unifrac 距离的 UPGMA 聚类树图】</td></tr><tr><td>
|   |   |   |   |-- upgam.(un)weighted_unifrac.{sample,group}.tree.{svg,png}</td><td>【基于(非)加权 unifrac 距离的 UPGMA 聚类树合并图】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_pca'>beta_pca<div id="pca"></div></a></td><td>【Beta PCA 分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta PCA 分析结果】</td></tr><tr><td>
|   |   |   |-- PCA.{xls,pdf,png}</td><td>【 PCA 作图数据及未标样品名称的 PCA 图】</td></tr><tr><td>
|   |   |   |-- PCA_lable.{pdf,png}</td><td>【标有样品名称的 PCA 图】</td></tr><tr><td>
|   |   |   |-- PCA_circle.{pdf,png}</td><td>【未标样本名带有置信椭圆的 PCA 图】</td></tr><tr><td>
|   |   |   |-- PCA_circleLable.{pdf,png}</td><td>【标有样本名带有置信椭圆的 PCA 图】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_pcoa'>beta_pcoa</a></td><td>【Beta PCoA 分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta PCoA 分析结果】</td></tr><tr><td>
|   |   |   |-- PCoA_display.{pdf,png}</td><td>【 基于(非)加权 unifrac 距离的 PCoA 合并展示图】</td></tr><tr><td>
|   |   |   |-- *</td><td>【 基于（(un)weighted_unifrac,binary_jaccard,bray_curtis）PCoA 分析结果】</td></tr><tr><td>
|   |   |   |   |-- PCoA.{xls,pdf,png}</td><td>【 PCoA 作图数据及未标样品名称的 PCoA 图】</td></tr><tr><td>
|   |   |   |   |-- PCoA_label.{pdf,png}</td><td>【标有样品名称的 PCoA 图】</td></tr><tr><td>
|   |   |   |   |-- PCoA_circle.{pdf,png}</td><td>【未标样本名带有置信椭圆的 PCoA 图】</td></tr><tr><td>
|   |   |   |   |-- PCoA_circleLabel.{pdf,png}</td><td>【标有样本名带有置信椭圆的 PCoA 图】</td></tr><tr><td>
|   |   |   |   |-- emperor_pcoa_plots</td><td>【三维 PCoA 网页版展示】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_nmds'>beta_nmds</a></td><td>【Beta NMDS 分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta NMDS 分析结果】</td></tr><tr><td>
|   |   |   |-- NMDS_display.{pdf,png}</td><td>【 基于(非)加权 unifrac 距离的 NMDS 合并展示图】</td></tr><tr><td>
|   |   |   |-- *</td><td>【 基于（(un)weighted_unifrac,binary_jaccard,bray_curtis）NMDS 分析结果】</td></tr><tr><td>
|   |   |   |   |-- *.NMDS.{xls,pdf,png}</td><td>【 NMDS 作图数据及未标样品名称的 NMDS 图】</td></tr><tr><td>
|   |   |   |   |-- *.NMDS_lable.{pdf,png}</td><td>【标有样品名称的 NMDS 图】</td></tr><tr><td>
|   |   |   |   |-- *.NMDS_circle.{pdf,png}</td><td>【未标样本名带有置信椭圆的 NMDS 图】</td></tr><tr><td>
|   |   |   |   |-- *.NMDS_circleLable.{pdf,png}</td><td>【标有样本名带有置信椭圆的 NMDS 图】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_dca'>beta_dca</a></td><td>【Beta DCA 分析结果】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta DCA 分析结果】</td></tr><tr><td>
|   |   |   |-- DCA.{xls,pdf,png}</td><td>【 DCA 作图数据及 DCA 图】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beat_unifracHeatmap'>beat_unifracHeatmap</a></td><td>【Beta unifrac 距离热图】</td></tr><tr><td>
|   |   |-- sample/group1/...</td><td>【样本及组 Beta unifrac 距离热图】</td></tr><tr><td>
|   |   |   |-- beta_diversity.heatmap.{svg,png}</td><td>【包含(un)Weighted 两种距离的热图】</td></tr><tr><td>
|   |   |   |-- beta_diversity.heatmap.(un)Weighted.{svg,png}</td><td>【(非)加权 unifrac 距离热图】</td></tr><tr><td>
|   |-- <a href='../../../result/05.BetaDiversity/beta_div'>beta_div</a></td><td>【组间 Beta 多样性比较箱型图】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间 Beta 多样性比较箱型图】</td></tr><tr><td>
|   |   |   |-- (un)weighted_unifrac</td><td>【基于(非)加权 unifrac 距离的组间 Beta 多样性比较箱型图】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac.{pdf,png}</td><td>【基于(非)加权 unifrac 距离的 Beta 多样性箱型图】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_kruskalWallis_all.txt</td><td>【多组间非参数 kruskalWallis 检验（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_kruskalWallis.{pdf,png}</td><td>【多组间非参数 kruskalWallis 检验可视化展示（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_TukeyHSD_all.txt</td><td>【多组间方差 TukeyHSD 分析（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_TukeyHSD.{pdf,png}</td><td>【多组间方差 TukeyHSD 分析可视化展示（组内样本>2,组>2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_tTest_all.txt</td><td>【组间 T-test 检验（组内样本>2,组=2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_tTest.{pdf,png}</td><td>【组间 T-test 检验可视化展示（组内样本>2,组=2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_twoWilcox_all.txt</td><td>【组间 twowilcox 秩和检验（组内样本>2,组=2）】</td></tr><tr><td>
|   |   |   |   |-- (un)weighted_unifrac_twoWilcox.{pdf,png}</td><td>【组间 twowilcox 秩和检验可视化展示（组内样本>2,组=2）】</td></tr><tr><td>
|-- <a href='../../../result/06.StatisticalTest'>06.StatisticalTest</a></td><td>【统计检验结果】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/adonis'>adonis</a></td><td>【 Adonis 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【分组 Adonis 分析结果】</td></tr><tr><td>
|   |   |   |-- bray_adonis.txt</td><td>【 Adonis 分析结果】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/anosim'>anosim</a></td><td>【 Anosim 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【分组 Anosim 分析结果】</td></tr><tr><td>
|   |   |   |-- stat_anosim.txt</td><td>【 Anosim 分析结果】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/mrpp'>mrpp</a></td><td>【 MRPP 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【分组 MRPP 分析结果】</td></tr><tr><td>
|   |   |   |-- stat_mrpp.txt</td><td>【 MRPP 分析结果】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/metagenomeseq'>metagenomeseq</a></td><td>【 Metagenomeseq 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【分组下Metagenomeseq 分析结果】</td></tr><tr><td>
|   |   |   |-- phylum/...</td><td>【门纲目科属种（p,c,o,f,g,s）水平上的Metagenomeseq 分析结果】</td></tr><tr><td>
|   |   |   |   |-- diffHeatmap</td><td>【 top10 丰度物种的 heatmap 热图分析结果】</td></tr><tr><td>
|   |   |   |   |   |-- annotation.heatmap.{pdf,png}</td><td>【具有显著性差异物种的 top10 丰度的 heatmap 热图分析结果】</td></tr><tr><td>
|   |   |   |   |-- *-*</td><td>【组间 Metagenomeseq 分析结果】</td></tr><tr><td>
|   |   |   |   |   |-- *.psig.xls</td><td>【从 Metagenomeseq 分析结果中，筛选出的 pvalues<=0.05 的信息】</td></tr><tr><td>
|   |   |   |   |   |-- *.qsig.xls</td><td>【从 *.psig.xls 分析结果中，筛选出的 adjPvalues<=0.05 的信息】</td></tr><tr><td>
|   |   |   |   |   |-- boxplot</td><td>【具有显著性差异物种的箱图结果】</td></tr><tr><td>
|   |   |   |   |   |   |-- *.{pdf,png}</td><td>【显著性差异物种的箱图结果展示】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/simper'>simper</a></td><td>【物种对样本差异贡献度分析（simper分析）】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【分组下物种对样本差异贡献度分析（simper分析）】</td></tr><tr><td>
|   |   |   |-- Rplots.pdf</td><td>【种（s）水平上物种对样本差异贡献度分析合并图】</td></tr><tr><td>
|   |   |   |-- phylum/...</td><td>【门纲目科属种（p,c,o,f,g,s）水平上物种对样本差异贡献度分析】</td></tr><tr><td>
|   |   |   |   |-- *-*</td><td>【组间 top10 物种对样本差异贡献度分析】</td></tr><tr><td>
|   |   |   |   |   |-- *-*.{xls,pdf,png}</td><td>【组间 top10 物种对样本差异贡献度分析结果及可视化】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/ttest'>ttest<div id="ttest"></div></a></td><td>【组间差异显著的物种分析】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间差异显著的物种分析】</td></tr><tr><td>
|   |   |   |-- phylum/...</td><td>【门纲目科属种（p,c,o,f,g,s）水平上组间差异显著的物种分析】</td></tr><tr><td>
|   |   |   |   |-- *-*</td><td>【组间差异显著的物种分析】</td></tr><tr><td>
|   |   |   |   |   |-- *.psig.xls</td><td>【两组之间的t-test数据结果，筛选出的 p value<=0.05 的信息】</td></tr><tr><td>
|   |   |   |   |   |-- *.qsig.xls</td><td>【两组之间的t-test数据结果，筛选出的 q value<=0.05 的信息】</td></tr><tr><td>
|   |   |   |   |   |-- *.Volcano.{pdf,png}</td><td>【具有显著性差异物种未标样品名称的火山图】</td></tr><tr><td>
|   |   |   |   |   |-- *.Volcano..label.{pdf,png}</td><td>【具有显著性差异物种标有样品名称的火山图】</td></tr><tr><td>
|   |   |   |   |   |-- boxplot</td><td>【具有显著性差异物种的箱图结果】</td></tr><tr><td>
|   |   |   |   |   |   |-- *.{pdf,png}</td><td>【显著性差异物种的箱图结果展示】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/lefse'>lefse</a></td><td>【LEfSe 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间 LEfSe 分析结果】</td></tr><tr><td>
|   |   |   |-- *-*</td><td>【组间 LEfSe 分析结果】</td></tr><tr><td>
|   |   |   |   |-- LDA.*.all.res</td><td>【 LEfSe 统计结果】</td></tr><tr><td>
|   |   |   |   |-- LDA.*.draw.res</td><td>【 LEfSe 作图数据】</td></tr><tr><td>
|   |   |   |   |-- LDA.*.{svg,png}</td><td>【 LDA 值分布柱状图】</td></tr><tr><td>
|   |   |   |   |-- LDA.*.tree.{svg,png}</td><td>【 LEfSe 进化分支图】</td></tr><tr><td>
|   |-- <a href='../../../result/06.StatisticalTest/randomForest'>randomForest</a></td><td>【随机森林结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间随机森林结果】</td></tr><tr><td>
|   |   |   |-- class/...</td><td>【纲目科属种（c,o,f,g,s）水平上随机森林结果】</td></tr><tr><td>
|   |   |   |   |-- trainset_group_max_roc.{pdf,png}</td><td>【各变量训练集最大AUC值,绘制ROC曲线组合图】</td></tr><tr><td>
|   |   |   |   |-- *-*</td><td>【组间随机森林结果】</td></tr><tr><td>
|   |   |   |   |   |-- imptop10MD.{pdf,png}</td><td>【 top10 变量平均精度下降图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- imptop10MG.{pdf,png}</td><td>【 top10 变量平均 Gini 指数下降图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- testset_auc.{pdf,png}</td><td>【各重要变量测试集 ROC 曲线组合图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- testset.point_auc.{pdf,png}</td><td>【各重要变量测试集 AUC 点图图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- trainset_auc.{pdf,png}</td><td>【各重要变量训练集 ROC 曲线组合图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- trainset.point_auc.{pdf,png}</td><td>【各重要变量训练集 AUC 点图图形展示】</td></tr><tr><td>
|   |   |   |   |   |-- 3/5/...</td><td>【选取 num（3，5，10，50，70，100，150，200）个最重要变量的数目构建随机森林模型】</td></tr><tr><td>
|   |   |   |   |   |   |-- cross_validation.*testset.marker.predict.xls</td><td>【交叉验证对随机森林模型测试集样本的预测概率，预测指标为AUC】</td></tr><tr><td>
|   |   |   |   |   |   |-- cross_validation.*trainset.marker.predict.xls</td><td>【交叉验证对随机森林模型训练集样本的预测概率，预测指标为AUC】</td></tr><tr><td>
|   |   |   |   |   |   |-- cross_validation_pick_*.im.var.txt</td><td>【通过交叉验证得到的重要变量特征】</td></tr><tr><td>
|   |   |   |   |   |   |-- cverrof.{pdf,png}</td><td>【变量筛选数目与错误率图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- impplot_MeanDecreaseAccuracy*.{pdf,png}</td><td>【平均精度下降图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- impplot_MeanDecreaseGin*.{pdf,png}</td><td>【平均 Gini 指数下降图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- testset.probabilty.xls</td><td>【随机森林模型对测试集样本的预测概率】</td></tr><tr><td>
|   |   |   |   |   |   |-- testset.ROC.{pdf,png}</td><td>【测试集 ROC 曲线图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- testset.roc.xls</td><td>【测试集 AUC 值 1 代表平均值 2 代表最小最 3 代表最大值】</td></tr><tr><td>
|   |   |   |   |   |   |-- top_*testset.var_predict_box_plot.{pdf,png}</td><td>【构建随机森林模型时,测试集预测的准确率图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- top_*trainset.var_predict_box_plot.{pdf,png}</td><td>【构建随机森林模型时,训练集预测的准确率图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- top_*var_imp.xls</td><td>【选取的n个重要变量分别对样本的贡献度及对模型构建的影响程度】</td></tr><tr><td>
|   |   |   |   |   |   |-- trainset.probabilty.xls</td><td>【随机森林模型对训练集样本的预测概率】</td></tr><tr><td>
|   |   |   |   |   |   |-- trainset.ROC.{pdf,png}</td><td>【训练集 ROC 曲线图形展示】</td></tr><tr><td>
|   |   |   |   |   |   |-- trainset.roc.xls</td><td>【训练集 AUC 值 1 代表平均值 2 代表最小最 3 代表最大值】</td></tr><tr><td>
|-- <a href='../../../result/07.CorrelationAnalysis'>07.CorrelationAnalysis</a></td><td>【关联分析】</td></tr><tr><td>
|   |-- <a href='../../../result/07.CorrelationAnalysis/network'>network</a></td><td>【 network 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间 network 分析结果】</td></tr><tr><td>
|   |   |   |-- */dot/genus.{png,svg}</td><td>【network结果图形展示】</td></tr><tr><td>
|   |   |   |-- */dot/genus.spearman.index.list</td><td>【network 网页图绘制边文件】</td></tr><tr><td>
|   |   |   |-- */dot/node.xls</td><td>【network 网页图绘制节点文件】</td></tr><tr><td>
|   |   |   |-- */dot/igraph.calculate.txt</td><td>【network图中的关键参数展示，如网络图直径（ND），平均连接度（AD）等】</td></tr><tr><td>
|   |   |-- sample/...</td><td>【样本间 network 分析结果】</td></tr><tr><td>
|   |   |   |-- dot/genus.{png,svg}</td><td>【network结果图形展示】</td></tr><tr><td>
|   |   |   |-- dot/genus.spearman.index.list</td><td>【network 网页图绘制边文件】</td></tr><tr><td>
|   |   |   |-- dot/node.xls</td><td>【network 网页图绘制节点文件】</td></tr><tr><td>
|   |   |   |-- dot/igraph.calculate.txt</td><td>【network图中的关键参数展示，如网络图直径（ND），平均连接度（AD）等】</td></tr><tr><td>
|   |-- <a href='../../../result/07.CorrelationAnalysis/network3D'>network3D</a></td><td>【 network3D 分析结果】</td></tr><tr><td>
|   |   |-- group1/...</td><td>【组间 network 分析结果】</td></tr><tr><td>
|   |   |   |-- */cytoplot.csv</td><td>【network3D 本地化Cytoscape绘制网络图数据】</td></tr><tr><td>
|   |   |   |-- */edge.xls</td><td>【network3D 网页图绘制边文件】</td></tr><tr><td>
|   |   |   |-- */node.xls</td><td>【network3D 网页图绘制节点文件】</td></tr><tr><td>
|   |   |   |-- <a href='../../../result/07.CorrelationAnalysis/network3D/group1/*/netfig'>netfig</a></td><td>【 network3D 网页展示文件】</td></tr><tr><td>
|   |   |-- sample/...</td><td>【样本间 network 分析结果】</td></tr><tr><td>
|   |   |   |-- cytoplot.csv</td><td>【network3D 本地化Cytoscape绘制网络图数据】</td></tr><tr><td>
|   |   |   |-- edge.xls</td><td>【network3D 网页图绘制边文件】</td></tr><tr><td>
|   |   |   |-- node.xls</td><td>【network3D 网页图绘制节点文件】</td></tr><tr><td>
|   |   |   |-- <a href='../../../result/07.CorrelationAnalysis/network3D/sample/netfig'>netfig</a></td><td>【 network3D 网页展示文件】</td></tr><tr><td>

|-- <a href='../../../result/08.FunctionPrediction'>08.FunctionPrediction</a></td><td>【功能预测结果】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/picrust2'>picrust2</a></td><td>【 picrust2 预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/predicted'>predicted</td><td>【 picrust2（COG,EC,KO,PFAM,TIGRFAM,pathway）数据库预测结果】</td></tr><tr><td>
|   |   |   |-- *_metagenome_descrip.tsv</td><td>【（COG,EC,KO,PFAM,TIGRFAM,pathway）数据库预测结果】</td></tr><tr><td>
|   |   |   |-- *_seqtab_norm.tsv.gz</td><td>【基于预测 16S 拷贝数校正的特征表, pathways_out 
无此结果】</td></tr><tr><td>
|   |   |   |-- *_weighted_nsti.tsv.gz</td><td>【每个样本的 NSTI 权重, pathways_out 无此结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/tables'>tables</td><td>【预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |-- sample/group1/...</td><td>【样本及分组 预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |   |-- picrust2.{sample,group}.*.absolute.xls</td><td>【(样本,分组)各数据库预测结果的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- picrust2.{sample,group}.*.relative.xls</td><td>【(样本,分组)各数据库预测结果的相对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/barplot'>barplot</a></td><td>【<a href="#barplot">功能相对丰度柱形图，展示同物种barplot</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/heatmap'>heatmap</a></td><td>【<a href="#heatmap">功能注释聚类热图，展示同物种heatmap</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/venn'>venn</a></td><td>【<a href="#venn">基于功能的venn结果，展示同物种venn</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/flower'>flower</a></td><td>【<a href="#flower">基于功能的花瓣图，展示同物种flower</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/pca'>pca</a></td><td>【<a href="#pca">各功能 PCA 分析结果，展示同beta_pca</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust2/ttest'>ttest</a></td><td>【<a href="#ttest">组间差异显著的功能分析各，展示同物种ttest</a>】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/picrust'>picrust</a></td><td>【 picrust 预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/predicted'>predicted</td><td>【 KEGG 预测结果】</td></tr><tr><td>
|   |   |   |-- KEGG_metagenome_predictions.{biom,txt}</td><td>【 KEGG 预测结果】</td></tr><tr><td>
|   |   |   |-- predicted_metagenomes.KEGG_L*.txt</td><td>【 KEGG（1,2,3）层级预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/tables'>tables</td><td>【预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |-- sample/group1/...</td><td>【样本及分组 预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |   |-- picrust.{sample,group}.*.absolute.xls</td><td>【(样本,分组)（1,2,3,k）层级预测结果的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- picrust.{sample,group}.*.relative.xls</td><td>【(样本,分组)（1,2,3,k）层级预测结果的相对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/barplot'>barplot</a></td><td>【<a href="#barplot">功能相对丰度柱形图，展示同物种barplot</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/heatmap'>heatmap</a></td><td>【<a href="#heatmap">功能注释聚类热图，展示同物种heatmap</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/venn'>venn</a></td><td>【<a href="#venn">基于功能的venn结果，展示同物种venn</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/flower'>flower</a></td><td>【<a href="#flower">基于功能的花瓣图，展示同物种flower</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/pca'>pca</a></td><td>【<a href="#pca">各功能 PCA 分析结果，展示同beta_pca</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/picrust/ttest'>ttest</a></td><td>【<a href="#ttest">组间差异显著的功能分析各，展示同物种ttest</a>】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/tax4fun'>tax4fun</a></td><td>【 tax4fun 预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/predicted'>predicted</td><td>【 KEGG 预测结果】</td></tr><tr><td>
|   |   |   |-- KEGG_metagenome_predictions.txt</td><td>【 KEGG 预测结果】</td></tr><tr><td>
|   |   |   |-- predicted_metagenomes.KEGG_L*.txt</td><td>【 KEGG（1,2,3）层级预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/tables'>tables</td><td>【预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |-- sample/group1/...</td><td>【样本及分组 预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |   |-- tax4fun.{sample,group}.*.absolute.xls</td><td>【(样本,分组)（1,2,3,k）层级预测结果的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- tax4fun.{sample,group}.*.relative.xls</td><td>【(样本,分组)（1,2,3,k）层级预测结果的相对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/barplot'>barplot</a></td><td>【<a href="#barplot">功能相对丰度柱形图，展示同物种barplot</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/heatmap'>heatmap</a></td><td>【<a href="#heatmap">功能注释聚类热图，展示同物种heatmap</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/venn'>venn</a></td><td>【<a href="#venn">基于功能的venn结果，展示同物种venn</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/flower'>flower</a></td><td>【<a href="#flower">基于功能的花瓣图，展示同物种flower</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/pca'>pca</a></td><td>【<a href="#pca">各功能 PCA 分析结果，展示同beta_pca</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/tax4fun/ttest'>ttest</a></td><td>【<a href="#ttest">组间差异显著的功能分析各，展示同物种ttest</a>】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/bugbase'>bugbase</a></td><td>【 bugbase 预测结果】</td></tr><tr><td>
|   |   |   |-- normalized_otus</td><td>【标准化 OTU 表】</td></tr><tr><td>
|   |   |   |-- otu_contributions</td><td>【表型预测结果堆叠柱形图】</td></tr><tr><td>
|   |   |   |   |-- *.{pdf,png}</td><td>【表型预测结果堆叠柱形图】</td></tr><tr><td>
|   |   |   |   |-- contributing_otus.txt</td><td>【表型预测结果表】</td></tr><tr><td>
|   |   |   |   |-- otu_table.relative.xls</td><td>【表型预测结果相对丰度表】</td></tr><tr><td>
|   |   |   |-- predicted_phenotypes</td><td>【表型预测结果比较箱线图】</td></tr><tr><td>
|   |   |   |   |-- *.{pdf,png}</td><td>【表型预测结果比较箱线图】</td></tr><tr><td>
|   |   |   |   |-- *_stats.txt</td><td>【表型预测结果箱型图绘制参数，如平均数，中位数，标准差】</td></tr><tr><td>
|   |   |   |   |-- predictions.{txt,xls}</td><td>【样本表型丰度表】</td></tr><tr><td>
|   |   |   |-- thresholds</td><td>【阈值选择表】</td></tr><tr><td>
|   |   |   |   |-- variances.txt</td><td>【不同阈值下的结果】</td></tr><tr><td>
|   |   |   |   |-- thresholds_used.txt</td><td>【表型预测分析使用的阈值】</td></tr><tr><td>
|   |   |   |   |-- *.{pdf,png}</td><td>【9表型在不同阈值下相对丰度变化】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/funguild'>funguild</a></td><td>【 funguild 预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/predicted'>predicted</td><td>【 funguild 预测结果】</td></tr><tr><td>
|   |   |   |-- funguild.sample.total.absolute.xls</td><td>【 funguild 预测结果】</td></tr><tr><td>
|   |   |   |-- funguild.sample.total.absolute.{guild, mode}</td><td>【 funguild（guild,mode）预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/tables'>tables</td><td>【预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |-- sample/group1/...</td><td>【样本及分组 预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |   |-- funguild.{sample,group}.*.absolute.xls</td><td>【(样本,分组)（total,guild,mode）预测结果的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- funguild.{sample,group}.*.relative.xls</td><td>【(样本,分组)（total,guild,mode）预测结果的相对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/barplot'>barplot</a></td><td>【<a href="#barplot">功能相对丰度柱形图，展示同物种barplot</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/heatmap'>heatmap</a></td><td>【<a href="#heatmap">功能注释聚类热图，展示同物种heatmap</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/venn'>venn</a></td><td>【<a href="#venn">基于功能的venn结果，展示同物种venn</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/flower'>flower</a></td><td>【<a href="#flower">基于功能的花瓣图，展示同物种flower</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/pca'>pca</a></td><td>【<a href="#pca">各功能 PCA 分析结果，展示同beta_pca</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/funguild/ttest'>ttest</a></td><td>【<a href="#ttest">组间差异显著的功能分析各，展示同物种ttest</a>】</td></tr><tr><td>
|   |-- <a href='../../../result/08.FunctionPrediction/faprotax'>faprotax</a></td><td>【 faprotax 预测结果】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/predicted'>predicted</td><td>【 funguild 预测结果】</td></tr><tr><td>
|   |   |   |-- faprotax.report.txt</td><td>【 faprotax 预测结果】</td></tr><tr><td>
|   |   |   |-- faprotax.sample.anno.absolute.xls</td><td>【 faprotax 预测结果绝对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/tables'>tables</td><td>【预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |-- sample/group1/...</td><td>【样本及分组 预测结果的相对（绝对）丰度】</td></tr><tr><td>
|   |   |   |   |-- faprotax.{sample,group}.anno.absolute.xls</td><td>【(样本,分组)预测结果的绝对丰度】</td></tr><tr><td>
|   |   |   |   |-- faprotax.{sample,group}.anno.relative.xls</td><td>【(样本,分组)预测结果的相对丰度】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/barplot'>barplot</a></td><td>【<a href="#barplot">功能相对丰度柱形图，展示同物种barplot</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/heatmap'>heatmap</a></td><td>【<a href="#heatmap">功能注释聚类热图，展示同物种heatmap</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/venn'>venn</a></td><td>【<a href="#venn">基于功能的venn结果，展示同物种venn</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/flower'>flower</a></td><td>【<a href="#flower">基于功能的花瓣图，展示同物种flower</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/pca'>pca</a></td><td>【<a href="#pca">各功能 PCA 分析结果，展示同beta_pca</a>】</td></tr><tr><td>
|   |   |-- <a href='../../../result/08.FunctionPrediction/faprotax/ttest'>ttest</a></td><td>【<a href="#ttest">组间差异显著的功能分析各，展示同物种ttest</a>】</td></tr><tr><td>

|-- <a href='../../../result/09.EnvironmentFactor'>09.EnvironmentFactor</a></td><td>【环境因子分析】</td></tr><tr><td>
|   |--<a href='../../../result/09.EnvironmentFactor/CCA'>CCA</a></td><td>【CCA 分析结果】</td></tr><tr><td>
|   |   |--<a href='../../../result/09.EnvironmentFactor/CCA/CCA_standard'>CCA_standard</a></td><td>【包含所有环境因子的CCA和RDA】</td></tr><tr><td>
|   |   |--<a href='../../../result/09.EnvironmentFactor/CCA/CCA_FitENV'>CCA_FitENV</a></td><td>【经envfit筛选后环境因子的CCA和RDA】</td></tr><tr><td>
|   |   |--<a href='../../../result/09.EnvironmentFactor/CCA/CCA_VIF'>CCA_VIF</a></td><td>【经VIF筛选后环境因子的CCA和RDA】</td></tr><tr><td>
|   |   |--<a href='../../../result/09.EnvironmentFactor/CCA/CCA_BioENV'>CCA_BioENV</a></td><td>【经BioENV筛选后环境因子的CCA和RDA】</td></tr><tr><td>
|   |   |--<a href='BioENV.txt'>BioENV.txt</a></td><td>【经Bioenv筛选后得到的环境因子组合】</td></tr><tr><td>
|   |   |--<a href='VIF.txt'>VIF.txt</a></td><td>【经VIF筛选后得到的环境因子组合】</td></tr><tr><td>
|   |   |--<a href='selected_FitENV.list'>selected_FitENV.list</a></td><td>【经FitENV筛选后得到的环境因子组合】</td></tr><tr><td>
|   |--<a href='../../../result/09.EnvironmentFactor/mantelTest'>mantelTest</a></td><td>【门纲目科属种以及otu（p,c,o,f,g,s,otu）水平上的mantel_test 分析结果】</td></tr><tr><td>
|   |--<a href='../../../result/09.EnvironmentFactor/spearman'>spearman</a></td><td>【门纲目科属种和otu以及alpha（p,c,o,f,g,s,otu,alpha）水平上的 spearman 分析结果,属水平目录下含有环境因子与丰度的玫瑰图展示】</td></tr><tr><td>
|   |--<a href='../../../result/09.EnvironmentFactor/vpa'>vpa</a></td><td>【VPA 分析结果】</td></tr><tr><td>
|   |   |--VPA1.{png,svg}</td><td>【VPA分组环境因子解释率图】</td></tr><tr><td>

|-- <a href='./ReadMe.html'>ReadMe.html</a></td><td></td></tr></tbody></table><p>  <code>
Tips:</code></p><ul class='task-list'><li class='task-list-item task-list-done' ><input type='checkbox' disabled='disabled' checked></input><p>Metagenomeseq、Adonis、MRPP、Anosim、t.test和LEfSe等组间差异分析，每个分组不少于三个生物学重复时才能做相应的统计检验（若组间无差异物种，则相应的检验结果不展示）。</p></li><li class='task-list-item task-list-done' ><input type='checkbox' disabled='disabled' checked></input><p>随机森林要求每个分组不少于15个生物学重复，否则无此分析。</p></li><li class='task-list-item task-list-done' ><input type='checkbox' disabled='disabled' checked></input><p>功能预测和环境因子部分需在搜集表中填写，若未填写相关分析则对应结果不展示。</p></li><li class='task-list-item task-list-done' ><input type='checkbox' disabled='disabled' checked></input><p>network分析，2D默认每个分组不少于五个生物学重复时方可出图；3D默认样本数不少于六个时方可出图。</p><p>​</p></li></ul><hr /><pre class="md-fences md-end-block md-focus" lang=""> <div class="CodeMirror cm-s-inner CodeMirror-wrap CodeMirror-focused"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 73.3333px;"></div><div class="CodeMirror-scrollbar-filler" cm-not-content="true"></div><div class="CodeMirror-gutter-filler" cm-not-content="true"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; margin-bottom: 0px; border-right-width: 30px; padding-right: 0px; padding-bottom: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines" role="presentation"><div role="presentation" style="position: relative; outline: none;"><div class="CodeMirror-measure"><pre><span>xxxxxxxxxx</span></pre></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" role="presentation"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><div class="CodeMirror-gutter-background CodeMirror-activeline-gutter" style="left: 0px; width: 0px;"></div><pre class=" CodeMirror-line " role="presentation"><span role="presentation" style="padding-right: 0.1px;"><span class="cm-tab" role="presentation" cm-text="	">    </span><span class="cm-tab" role="presentation" cm-text="	">         </span>              如有疑问请咨询项目销售或运营，我们将竭诚为您服务。感谢您对Novogene的信任！</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; border-bottom: 0px solid transparent; top: 46px;"></div><div class="CodeMirror-gutters" style="display: none; height: 76px;"></div></div></div></pre><p></p></div>
</body>
</html>
