
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>

<head>



<title> TESTXXXXXXXXXX Project Standard Analysis Report </title>



	


	<!-- add 20200312 -->
	<link rel="stylesheet" href="src/css/jquery.tocify.css" />
	<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
	<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

	<!--<script src="src/js/jquery-1.4.2.min.js"></script>-->
	<link rel="stylesheet" type="text/css" href="src/css/base_new.css">
	<link rel="StyleSheet" href="/src/" type="text/css">
	<link rel="stylesheet" type="text/css" href="src/js/fancybox/jquery.fancybox-1.3.4.css" media="screen" />
	<link rel="stylesheet" href="src/css/style.css" />
	<script src="src/js/scrollTop.js"></script>
	<script src="src/js/common.js"></script>
	<script src="src/js/jquery.albumSlider.min.js"></script>
	<script src="src/js/tree/tree.js"></script>
	<script src="src/js/fancybox/jquery.mousewheel-3.0.4.pack.js"></script>
	<script src="src/js/fancybox/jquery.fancybox-1.3.4.pack.js"></script>
	<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>



	<style>
		.longtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		.sublongtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		#menu_list li {
			height: 45px;
			line-height: 45px;
		}
	</style>


	<script>
		$(document).ready(function () {
			/*
			*   Examples - images
			*/

			$("a#example1").fancybox();

			$("a#example2").fancybox({
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'elastic'
			});
			$("a#example3").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("a#example4").fancybox({
				'opacity': true,
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'none'
			});

			$("a#example5").fancybox();

			$("a#example6").fancybox({
				'titlePosition': 'outside',
				'overlayColor': '#000',
				'overlayOpacity': 0.9
			});

			$("a#example7").fancybox({
				'titlePosition': 'inside'
			});

			$("a#example8").fancybox({
				'titlePosition': 'over'
			});

			$("a[rel=example_group]").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none',
				'titlePosition': 'over',
				'titleFormat': function (title, currentArray, currentIndex, currentOpts) {
					return '<span id="fancybox-title-over">Image ' + (currentIndex + 1) + ' / ' + currentArray.length + (title.length ? ' &nbsp; ' + title : '') + '</span>';
				}
			});

			/*
			*   Examples - various
			*/

			$("#various1").fancybox({
				'titlePosition': 'inside',
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("#various2").fancybox();

			$("#various3").fancybox({
				'width': '75%',
				'height': '75%',
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none',
				'type': 'iframe'
			});

			$("#various4").fancybox({
				'padding': 0,
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none'
			});
		});
	</script>

	<script type="text/javascript">

		function trans (img_div_id) {

			img_div = $("#" + img_div_id)

			img = $("#" + img_div_id + " img").first();

			if (img_div.hasClass("fullscreen-div")) {

				img_div.removeClass("fullscreen-div");

				img.removeClass("fullscreen-img");

			}

			else {

				img_div.addClass("fullscreen-div");

				img.addClass("fullscreen-img");

			}

		}



		function filter_display () {

			return $(this).css("display") != "none";

		}



		function show_default () {

			var list_group_id = "#" + $(this).attr("id");
			if ($(list_group_id + " a.active").css("display") == "none") {

				var list_tab = $(list_group_id + " a.active");

				var pan_id = list_tab.attr("href");

				$(pan_id).removeClass("active show");

				list_tab.removeClass("active");

			}

			if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

				$(list_group_id + " a").filter(filter_display).first().tab("show");

			}

		}

		$.fn.show_default = show_default



		function next (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

			while (present_tab.next().length == 1) {

				if (present_tab.next().css("display") != "none") {

					present_tab.next().tab("show");


					break;

				}

				else {

					present_tab = present_tab.next();

				}

			}



		}

		function prev (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

			while (present_tab.prev().length == 1) {

				if (present_tab.prev().css("display") != "none") {

					present_tab.prev().tab("show");

					break;

				}

				else {

					present_tab = present_tab.prev();

				}

			}





		}



		$(document).ready(function () {

			$(".form-control").on("keyup", function () {

				var value = $(this).val().toLowerCase();

				var input_box_id = "#" + $(this).attr("id");

				var list_group_id = input_box_id.replace("_InputList", "_list-tab");

				$(list_group_id + " a").filter(function () {

					$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

				});

				$(list_group_id).show_default();

			});
			$(".list-group").each(show_default);

		});
	</script>

	<style media="print">
		.noprint {
			DISPLAY: none;
		}
	</style>

	<div class="noprint">
		<div style="display: block;" id="goTopBtn">
			<a class="backtotop" title="回顶部"><img src="src/images/goTop.jpg" width="30" height="30" class="back-tip" /></a>
		</div>
	</div>

	<script>
		function displaySubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "block";
		}
		function hideSubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "none";
		} 
	</script>

	</script>
	<script type="text/javascript">
		/* show or hide element*/
		function hiddenShow (btnObj) {
			ctObj = btnObj.nextElementSibling;
			if (ctObj.style.display == 'inline') {
				ctObj.style.display = 'none';
				btnObj.innerHTML = "Show Annotation";
			} else {
				ctObj.style.display = 'inline';
				btnObj.innerHTML = "Hide Annotation";
			};
			return 0;
		}
	</script>

	<script>
		var Tree = new Array;
	// nodeId | parentNodeId | nodeName | nodeUrl
< {jtree} >
	</script>

	<script>
			$(function () {
				//纵向,默认,移动间隔2
				$('div.albumSlider').albumSlider();
				//横向设置
				$('div.albumSlider-h').albumSlider({ direction: 'h', step: 3 });
			});
	</script>


	<!-- 20200314 -->
	<script type="text/javascript">
		$(function () {
			var options = {
				selectors: "h2,h3,h4,h5",
				theme: "bootstrap",
				scrollTo: "60"
			};
			var toc = $("#toc").tocify(options).data("toc-tocify");
		});
	</script>

	<!------------------------------------>



</head>

<body id="body">

	<!------------------------------------顶部图片logo---------------------------------------->
	<div id="toc" class="tocify">
	</div>


	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="60px" border="0" cellpadding="5" cellspacing="0"
			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
            
			<tr height="0%">

				<td width='0%'></td>

				<td><img height="60px" src="src/images/logo_new.png"></td>

				<td width="0%"></td>

				<td width="40%"></td>

				<td width="20%" style="text-align:right;padding:0px;margin:0px;color:black;font-size:8px;font-weight:50;">

					<a>
				</td>

				<td width="1%"></td>

			</tr>
            
		</table>
	</div>



	<!-- 不可删，删后是满页，不靠右  -->
	<div id="main">
		<!---------------------------------------menu overdd-------------------------------------------->


		<!---------------------------------------- 报告标题 ---------------------------------------------->


		<div id="page">
			<p><br /><br />

		
				<h1> <font size="6"> Human WGS Standard Analysis Report </font></h1>
		



			<span style="display: block;text-align: center;font-size: 4em;font-family: Arial;color: #003f76;"></span>
			<br /><br /></p>
			
		</div>


		<div id="page">
			<br />
			<h2>1 Introduction</h2>
			<h3>1.1 Background</h3>
			

			
			<p class="paragraph">Human whole genome sequencing enables researchers to catalog the genetic constitution of individuals and capture all the variants present in a single assay. It is applied to the study of cancer and a variety of diseases, pharmacogenomics as well as human population evolution studies.</p>
			<p class="paragraph">Our company is one of the first few companies in the world employing the powerful  platform, capable of sequencing up to 280,000 human genomes per year at the lowest cost per genome. We have extensive experience providing whole genome sequencing service on this powerful system, having successfully sequenced thousands of genomes with high quality results. With the throughput and capacity of the   Illumina  platform, our deep experience with the system, and our advanced bioinformatics capabilities, our company is able to expertly meet customer needs for executing large projects with timely turn-around and the highest quality results.</p>
			<br />
			<p class="paragraph">Workflow is shown as follows:</p>
			<p class="center"> <img  class="middle" src="src/images/hWGS_Project_Workflow.png" /></p>
			<br>
			
			<p class="name_fig">Figure 1 Project workflow </p>
			
			
			<br>	



		<!----------------------------------------样品信息----------------------------------------->
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Sample information"></a>
				<hr />
			</p><br />
			<h3>1.2 Sample Information</h3>
			<p class="center">
			<p class="name_fig">Table 1 Sample information</p>
			<div id="tb">
				<table class="gy">
				
					
					<tr>
						
							<th>PatientID</th>
						
							<th>SampleID</th>
						
							<th>ID</th>
						
							<th>Type</th>
						</tr>
					
				
					
					<tr>
						
							<td>P1</td>
						
							<td>tian</td>
						
							<td>FKDO230350531-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P1</td>
						
							<td>songshu</td>
						
							<td>FKDO230350530-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P1</td>
						
							<td>tianshu</td>
						
							<td>FKDO230350529-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P1</td>
						
							<td>tianzi</td>
						
							<td>FKDO230350528-1A</td>
						
							<td>P</td>
						</tr>
					
				
				</table>
			</div>
			</p>
		

			<p class="tremark" align="left">Type: sample type (N: normal; T: tumor; P: patient; U: unknown)</p>

			
		</div>



		<!----------------------------------------实验流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Experimental procedure"></a>
				<hr />
			</p><br />
			<h2>2 Library Construction and Sequencing</h2>
			<a name="样品检测"></a>
			<br />
			<h3>2.1 Sample Quality Control</h3>
			<p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;Methods of sample quality control refer to QC report.</p>
			<br />

			<a name="建库捕获"></a>

			<h3>2.2 Library Construction, Quality Control and Sequencing</h3>



			

			

			

			
			<p class="paragraph">The genomic DNA was randomly sheared into shorter fragments. The obtained fragments were then end-repaired, A-tailed, and further ligated with  Illumina   adapters. The resulting fragments with adapters were size selected, PCR amplified unless otherwise specified as PCR-free, before proceeding for purification.</p>
			<p class="paragraph">The experimental procedures of DNA library preparation are shown in Figure 2.</p>
			<br>
			<p class="center">
				<img style="width: 200px;" src="src/images/WGS_library_construction.png" />
			</p>
			<p class="name_fig">Figure 2 Workflow of Library construction</p>
			<p class="paragraph">The library was quantified through Qubit and qPCR, and size distribution detected with fragment analyzer. Quantified libraries were pooled and sequenced on  Illumina  platforms according to the effective library concentration and required data amount.</p>
			


			
			
			<p class="paragraph">Method details involved in the project are available at <a href="src/method/WGS_method.pdf" title="Click to open" target="_blank">method.</a></p>
			
			



			<a name="上机测序"></a>
		</div>

		<!----------------------------------------生物信息分析流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<br />
				<a name="Bioinformatics analysis pipeline"></a>
				<hr />
			</p><br />

			<h2>3 Bioinformatics Analysis Pipeline</h2>

			<p class="paragraph">Bioinformatics analysis includes:</p>
			<p class="paragraph">1) Data quality control: filtering out reads containing adapters or with low quality;</p>
			<p class="paragraph">2) Alignment with reference genome, statistics of sequencing depth and coverage;</p>

			
			
			<p class="paragraph">3) SNP/InDel/SV/CNV calling, annotation and statistics;</p>
			<p class="paragraph">4) Somatic SNP/InDdel/SV/CNV calling, annotation and statistics (tumor-normal paired samples).</p>
			
			

			
			
			<p class="paragraph">The flowchart below depicts the pipeline of data QC and bioinformatics analysis we
				used. Somatic analysis will be performed only when tumor-normal paired samples are provided. </p>
			<p class="center">
				
				<img style="width: 500px;" src="src/images/primary_workflow_for_WGS_demo.png" />
				
			</p>
			
			



			<p class="name_fig">Figure 3 Bioinformatics analysis pipeline</p>

		
			
			
			<p class="paragraph">Method details involved in the project are available at <a href="Method/WGS_method.pdf"
                                title="Click to open" target="_blank">method.</a></p>
			
			

		</div>
		<!----------------------------------------------- 原始序列数据 --------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Analysis Result"></a>
				<hr />
			</p><br />

			<h2>4 Analysis Result</h2>
			<a name="Raw data"></a>
			<br />
			<h3>4.1 Raw Data</h3>

			<p class="paragraph">The original fluorescence image files obtained from sequencing platform are transformed to
				short reads (Raw data) by base calling. These short reads are recorded in FASTQ (.fq) format files, which
				contain sequence information (reads) and corresponding base quality.</p>

			<p class="paragraph">Each read in FASTQ format is stored in four lines as follows: </p>
			<p class="seq">
				
				@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG <br />
				GCTCTTTGCCCTTCTCGTCGAAAATTGTCTCCTCATTCGAAACTTCTCTGT <br />
				+ <br />
				@@CFFFDEHHHHFIJJJ@FHGIIIEHIIJBHHHIJJEGIIJJIGHIGHCCF </p>
				<p class="paragraph">Line 1 begins with a '@' character which is followed by a sequence identifier and an
				optional description. </p>
				<p class="paragraph">Line 2 shows the sequenced bases. </p>
				<p class="paragraph">Line 3 begins with a '+' character and is optionally followed by the same sequence
				identifier. </p>
				<p class="paragraph">Line 4 encodes the sequencing quality for each base in line 2, and contains the same number
				of characters as bases in line 2. </p>
				<p class="paragraph">Illumina sequence identifier details: </p>
				<div id="tb">
					<p class="center">
					<table class="gy">
						<tr>
							<td>EAS139</td>
							<td>Unique instrument name</td>
						</tr>
						<tr>
							<td>136</td>
							<td>Run ID</td>
						</tr>
						<tr>
							<td>FC706VJ</td>
							<td>Flowcell ID</td>
						</tr>
						<tr>
							<td>2</td>
							<td>Flowcell lane</td>
						</tr>
						<tr>
							<td>2104</td>
							<td>Tile number within the flowcell lane</td>
						</tr>
						<tr>
							<td>15343</td>
							<td>'x'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>197393</td>
							<td>'y'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>1</td>
							<td>Member of a pair, 1 or 2 (paired-end or mate-pair reads only)</td>
						</tr>
						<tr>
							<td>Y</td>
							<td>Y if the read fails filter (read is bad), otherwise N</td>
						</tr>
						<tr>
							<td>18</td>
							<td>0 when none of the control bits are on, Otherwise it is an even number</td>
						</tr>
						<tr>
							<td>ATCACG</td>
							<td>Index sequence</td>
						</tr>
					</table>
					</p>
				</div>
				

			<p class="paragraph">The ASCII value of each character at the fourth line minus 33 equals to the Phred-scaled
				quality value of the corresponding sequenced base in the second line. The relationship between sequencing
				error rate (e) and base quality value (Qphred) can be expressed by the following equation: </p>
			<p class="center">Qphred = -10log10(e)</p>
			<p class="paragraph">The table below shows examples of corresponding values among sequencing error rate (e),
				base quality value (Qphred) and characters. </p>
			<div id="tb">
				<p class="center">
				<table class="gy">
					<tr>
						<th>Sequencing error rate</th>
						<th>Sequencing quality value</th>
						<th>Corresponding character</th>
					</tr>
					<tr>
						<td>5%</td>
						<td>13</td>
						<td>.</td>
					</tr>
					<tr>
						<td>1%</td>
						<td>20</td>
						<td>5</td>
					</tr>
					<tr>
						<td>0.1%</td>
						<td>30</td>
						<td>?</td>
					</tr>
					<tr>
						<td>0.01%</td>
						<td>40</td>
						<td>I</td>
					</tr>
				</table>
				</p>
			</div>

				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Quality control"></a>
					<hr />
				</p><br />
				<h3>4.2 Quality Control</h3>
				<a name="Sequencing data filtration"></a>
				<br />
				<h4>4.2.1 Sequencing Data Filtration</h4>
				<p class="paragraph">Sequencing artifacts, including reads containing adapter contamination, low-quality
					nucleotides and unrecognizable nucleotides (N), apparently brings ambiguity for the subsequent
					bioinformatics analysis. Hence quality control is an essential step and applied to guarantee the
					meaningful analysis. All of downstream analysis will be based on clean reads that pass quality control.
				</p>
				<p class="paragraph">Our company performed quality control according to following procedures: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1) Discard paired-end reads if either one
					read contains adapter contamination (>10 nucleotides aligned to the adapter, allowing ≤ 10% mismatches);
					<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2) Discard paired-end reads if more than 10% of bases are
					uncertain (read as N) in either one read; <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3) Discard paired-end reads if the proportion of
					low-quality (Phred quality &lt; 5) bases is over 50% in either one read. </p>
				<p class="paragraph">Sequencing Adapter: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCT<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P7 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_class_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_class_nav-tabContent">
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/tian_FKDO230350531-1A_AHWN3FDSX5-new_L1.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/tian_FKDO230350531-1A_BHWC2NDSX5-new_L2.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2.pie3d.png" /></div>
							
						</div>

					</div>

					<div class="col-1">
						<div id="gallery_figure_class_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_class_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_class_list-tab" role="tablist">

								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_class_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1" role="tab"
								aria-controls="tian_FKDO230350531-1A_AHWN3FDSX5-new_L1">tian_FKDO230350531-1A_AHWN3FDSX5-new_L1</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2" data-toggle="list"
								href="#gallery_figure_class_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2" role="tab"
								aria-controls="tian_FKDO230350531-1A_BHWC2NDSX5-new_L2">tian_FKDO230350531-1A_BHWC2NDSX5-new_L2</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_class_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1" role="tab"
								aria-controls="songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1">songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_class_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1" role="tab"
								aria-controls="tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1">tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2" data-toggle="list"
								href="#gallery_figure_class_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2" role="tab"
								aria-controls="tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2">tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2</a>
							
							


						</div>
					</div>
				</div>
			   </p>


				<p class="name_fig">Figure 4.1 Raw data filtration result</p>
				<br />

				</script>
				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
					<p class="tremark">
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Clean Reads: read pairs that passed quality
						control; <br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Containing N: read pairs in either one read of
						which more than 10% of bases are uncertain;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Low Quality: read pairs in either one read of
						which the proportion of low-quality bases is over 50%;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Adapter Related: read pairs that contain adapter
						contamination in either one read.</p>
				</div>
				<br />
				<br />

				<a name="Sequencing error rate examination"></a>
				<br />


				<h4>4.2.2 Examination of Sequencing Error Rate </h4>
				<p class="paragraph">Sequencing error rate and base quality can be affected by various factors such as
					sequencing platform, chemical reagent and sample quality. Due to the consumption of chemical reagents,
					error rate is increasing with the read extension, which is a common feature of the high throughput
					sequencing platform.</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_error_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_error_nav-tabContent">

							

							<div class="tab-pane fade" id="gallery_figure_error_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/tian_FKDO230350531-1A_AHWN3FDSX5-new_L1.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/tian_FKDO230350531-1A_BHWC2NDSX5-new_L2.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2.Error.png" /></div>
							


						</div>
					</div>
					<div class="col-1">
						<div id="gallery_figure_error_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_error_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_error_list-tab" role="tablist">
							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_error_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1" role="tab"
								aria-controls="tian_FKDO230350531-1A_AHWN3FDSX5-new_L1">tian_FKDO230350531-1A_AHWN3FDSX5-new_L1</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2" data-toggle="list"
								href="#gallery_figure_error_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2" role="tab"
								aria-controls="tian_FKDO230350531-1A_BHWC2NDSX5-new_L2">tian_FKDO230350531-1A_BHWC2NDSX5-new_L2</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_error_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1" role="tab"
								aria-controls="songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1">songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1" data-toggle="list"
								href="#gallery_figure_error_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1" role="tab"
								aria-controls="tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1">tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2" data-toggle="list"
								href="#gallery_figure_error_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2" role="tab"
								aria-controls="tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2">tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2</a>

							

						</div>
					</div>
				</div>
			   </p>

				<p class="name_fig">Figure 4.2 Sequencing error rate distribution</p>
				<p class="premark">The x-axis represents position in reads, and the y-axis represents the average error rate
					of bases of all reads at a position. </p>


					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="GC content distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.3 GC Content Distribution</h4>

					<p class="paragraph">Evaluation of GC content distribution aims to check the potential of AT/GC
						separation phenomenon, which maybe resulted from sample contamination, sequencing bias or errors in
						library preparation. </p>
					<p class="paragraph">In theory, GC or AT content should be constant along read positions. But it is
						common to see that the first 6 to 7 bases in both reads (read1 & read2) fluctuate in GC content, due
						to primer amplification bias and some other reasons. </p>

					<p class="center">
					<div class="row gallery-row">
						<div class="col-1">
							<div id="gallery_figure_gc_button_prev" class="button_prev" onclick="prev(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_prev">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-6">
							<div class="tab-content" id="gallery_figure_gc_nav-tabContent">

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/tian_FKDO230350531-1A_AHWN3FDSX5-new_L1.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/tian_FKDO230350531-1A_BHWC2NDSX5-new_L2.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2.GC.png" /></div>

								


							</div>
						</div>
						<div class="col-1">
							<div id="gallery_figure_gc_button_next" class="button_next" onclick="next(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_next">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-4">
							<input class="form-control" id="gallery_figure_gc_InputList" type="text" placeholder="Search..">
							<div class="list-group" id="gallery_figure_gc_list-tab" role="tablist">

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
									data-toggle="list"
									href="#gallery_figure_gc_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
									role="tab"
									aria-controls="tian_FKDO230350531-1A_AHWN3FDSX5-new_L1">tian_FKDO230350531-1A_AHWN3FDSX5-new_L1</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
									data-toggle="list"
									href="#gallery_figure_gc_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
									role="tab"
									aria-controls="tian_FKDO230350531-1A_BHWC2NDSX5-new_L2">tian_FKDO230350531-1A_BHWC2NDSX5-new_L2</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
									data-toggle="list"
									href="#gallery_figure_gc_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
									role="tab"
									aria-controls="songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1">songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
									data-toggle="list"
									href="#gallery_figure_gc_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
									role="tab"
									aria-controls="tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1">tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
									data-toggle="list"
									href="#gallery_figure_gc_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
									role="tab"
									aria-controls="tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2">tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2</a>

								

							</div>
						</div>
					</div>
				   </p>
					<p class="name_fig">Figure 4.3 GC content distribution</p>

					<p class="premark">The x-axis represents the position in reads, and the y-axis represents the percentage
						of each type of bases (A, T, G, C); different bases can be distinguished by different colors.</p>

					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequencing quality distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.4 Sequencing Quality Distribution</h4>

					<p class="paragraph">The Phred-scaled quality scores of most bases should be greater than 20, it is also
						required for downstream analysis. Base quality decreases along reads is common, which is an inherent
						circumstance of next generation sequencing. </p>
					<p class="center">
					

						<div class="row gallery-row">
							<div class="col-1">
								<div id="gallery_figure_qm_button_prev" class="button_prev" onclick="prev(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_prev">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-6">
								<div class="tab-content" id="gallery_figure_qm_nav-tabContent">
	
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/tian_FKDO230350531-1A_AHWN3FDSX5-new_L1.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/tian_FKDO230350531-1A_BHWC2NDSX5-new_L2.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2.QM.png"></div>
									
									
									
								</div>
							</div>
							<div class="col-1">
								<div id="gallery_figure_qm_button_next" class="button_next" onclick="next(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_next">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-4">
								<input class="form-control" id="gallery_figure_qm_InputList" type="text" placeholder="Search..">
								<div class="list-group" id="gallery_figure_qm_list-tab" role="tablist">
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
										data-toggle="list"
										href="#gallery_figure_qm_pane_tian_FKDO230350531-1A_AHWN3FDSX5-new_L1"
										role="tab"
										aria-controls="tian_FKDO230350531-1A_AHWN3FDSX5-new_L1">tian_FKDO230350531-1A_AHWN3FDSX5-new_L1</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
										data-toggle="list"
										href="#gallery_figure_qm_pane_tian_FKDO230350531-1A_BHWC2NDSX5-new_L2"
										role="tab"
										aria-controls="tian_FKDO230350531-1A_BHWC2NDSX5-new_L2">tian_FKDO230350531-1A_BHWC2NDSX5-new_L2</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
										data-toggle="list"
										href="#gallery_figure_qm_pane_songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1"
										role="tab"
										aria-controls="songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1">songshu_FKDO230350530-1A_BHWC2NDSX5-new_L1</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
										data-toggle="list"
										href="#gallery_figure_qm_pane_tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1"
										role="tab"
										aria-controls="tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1">tianshu_FKDO230350529-1A_BHWC2NDSX5-new_L1</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
										data-toggle="list"
										href="#gallery_figure_qm_pane_tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2"
										role="tab"
										aria-controls="tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2">tianzi_FKDO230350528-1A_BHWC2NDSX5-new_L2</a>
	
									
	
						
								</div>
							</div>
						</div>
					   </p>	

				<p class="name_fig">Figure 4.4 Sequencing quality distribution</p>
				<p class="premark">The x-axis represents the position in reads, and the y-axis represents the average
					quality score of bases of all reads at a position.</p>

				<p class="premark">&nbsp</p>
				</p>


					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Statistics of sequencing quality"></a>
						<hr />
					</p><br />
					<h4>4.2.5 Statistics of Sequencing Quality</h4>

					<P class="paragraph">According to the sequencing feature of sequencing platforms, our company requires the
						value of Q30 (the percentage of bases with Phred-scaled quality scores is greater than 30) should be
						above 85% for paired-end sequencing data.</p>

					<p class="name_fig">Table 4.1 Overview of data production quality</p>
					<div id="tb">
						<table class="gy">
							
							<tr>
								<th>Sample name</th>
								<th>NovoID</th>
								<th>Flowcell/Lane</th>
								<th>Raw reads</th>
								<th>Raw data(G)</th>
								<th>Effective(%)</th>
								<th>Error(%)</th>
								<th>Q20(%)</th>
								<th>Q30(%)</th>
								<th>GC(%)</th>
							</tr>
							

						
							
								
									<tr>
										<td style="text-align:left">tian</td>
										<td>FKDO230350531-1A</td>
										<td>AHWN3FDSX5-new_L1</td>
										<td>31606068</td>
										<td rowspan=2>93.1</td>
										<td>99.16</td>
										<td>0.03</td>
										<td>97.19</td>
										<td>92.45</td>
										<td>40.27</td>
									</tr>
								
							
								
									<tr>
										<td style="text-align:left">tian</td>
										<td>FKDO230350531-1A</td>
										<td>BHWC2NDSX5-new_L2</td>
										<td>278872927</td>
										<td>99.21</td>
										<td>0.03</td>
										<td>97.44</td>
										<td>93.00</td>
										<td>40.37</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">songshu</td>
										<td>FKDO230350530-1A</td>
										<td>BHWC2NDSX5-new_L1</td>
										<td>333681575</td>
										<td rowspan=1>100.1</td>
										<td>99.30</td>
										<td>0.03</td>
										<td>97.70</td>
										<td>93.59</td>
										<td>40.58</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">tianshu</td>
										<td>FKDO230350529-1A</td>
										<td>BHWC2NDSX5-new_L1</td>
										<td>309579461</td>
										<td rowspan=1>92.9</td>
										<td>99.27</td>
										<td>0.03</td>
										<td>97.77</td>
										<td>93.72</td>
										<td>40.83</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">tianzi</td>
										<td>FKDO230350528-1A</td>
										<td>BHWC2NDSX5-new_L2</td>
										<td>332879560</td>
										<td rowspan=1>99.9</td>
										<td>99.25</td>
										<td>0.03</td>
										<td>97.42</td>
										<td>93.02</td>
										<td>41.02</td>
									</tr>
								
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">

							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) NovoID: NovoID for the sample.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Flowcell/Lane: The flowcell ID and lane
							number.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Raw reads: The number of sequencing reads
							pairs; four lines would be considered as one unit according to the format of FASTQ.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Raw data: The original sequencing
							data.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) Effective: The percentage of clean reads
							in all raw reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) Error: The average error rate of all
							bases on read1 and read2; the error rate of a base is obtained from equation 1.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) Q20: The percent of bases with
							phred-scaled quality scores greater than 20.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) Q30: The percent of bases with
							phred-scaled quality scores greater than 30.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) GC content: The percentage of G and C in
							the all bases.<br /></p>
					</div>
					<p class="paragraph"><a href="../../../01.Rawdata" title="Click to open" target="_blank">Result: 01.Rawdata</a></p>


				<!----------------------------------------- 测序深度,覆盖度统计 -------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequence alignment"></a>
						<hr />
					</p><br />
					<h3>4.3 Sequence Alignment</h3>
					<P class="paragraph">Burrows-Wheeler Aligner (BWA) (Li <I><I>et al.</I></I>, 2018) was utilized to map the paired-end
						clean reads to the human reference genome. The original mapping result can be obtained in BAM format.
						Sambamba (Tarasov <I>et al.</I>, 2015) is used for sorting the BAM files and mark duplicate reads.</p>
					<P class="paragraph">Final BAM files can be obtained after these steps. Then the coverage and depth will
						be computed based on BAM.</p>
					<br />
					<h4>4.3.1 Sequencing Depth & Coverage Distribution</h4>
					<a name="测序深度,覆盖度分布"></a>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/depth.png" />
					<p class="name_fig">Figure 4.5 Sequencing depth</p>
					<!-- <p class="name_fig">Figure 3.1 Sequencing depth</p>-->
					<p class="premark">The left figure shows sequencing depth distribution of all bases in each sample. The
						x-axis represents the sequencing depth, and the y-axis represents the fraction of bases with the given
						sequencing depth. <br />
						The right figure shows accumulative sequencing depth distribution of all bases in each sample. The
						x-axis represents the accumulative sequencing depth, and the y-axis represents the fraction of bases
						above the given sequencing depth.
					</p>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/covByChr0.png" />

					<p class="name_fig">Figure 4.6 Average sequencing depth (bar plot) and coverage (dot-line plot) in each
						chromosome</p>
					<!--<p class="name_fig">Figure 3.2　Average sequencing depth (bar plot) and coverage (dot-line plot) in each chromosome</p>-->
					<p class="premark">The x-axis represents the chromosome; the left y-axis represents the average depth;
						the right y-axis represents the coverage (proportion of covered bases). </p>
					<p class="premark">&nbsp</p>
				</div>

				<div id="page">
					<h4>4.3.2 Statistics of Mapping, Coverage and Depth</h4>
					<a name="覆盖度统计结果"></a>
					<p class="center">
					<p class="name_fig">Table 4.2 Statistics of mapping, coverage and depth in each sample</p>
					<!--<p class="name_fig">Table 3　Statistics of mapping, coverage and depth in each sample</p>-->
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
									
									<th>Sample name</th>
									
								
									
									<th>tian</th>
									
								
									
									<th>songshu</th>
									
								
									
									<th>tianshu</th>
									
								
									
									<th>tianzi</th>
									
								
							</tr>
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Total</td>
										
									
										
										<td>616035864 (100%)</td>
										
									
										
										<td>662707358 (100%)</td>
										
									
										
										<td>614609896 (100%)</td>
										
									
										
										<td>660777708 (100%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Duplicate</td>
										
									
										
										<td>150007046 (24.39%)</td>
										
									
										
										<td>170383356 (25.75%)</td>
										
									
										
										<td>144944719 (23.62%)</td>
										
									
										
										<td>160279691 (24.30%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Mapped</td>
										
									
										
										<td>615082144 (99.85%)</td>
										
									
										
										<td>661719049 (99.85%)</td>
										
									
										
										<td>613696421 (99.85%)</td>
										
									
										
										<td>659582928 (99.82%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Properly mapped</td>
										
									
										
										<td>596719822 (96.86%)</td>
										
									
										
										<td>646150804 (97.50%)</td>
										
									
										
										<td>598514944 (97.38%)</td>
										
									
										
										<td>642694460 (97.26%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">PE mapped</td>
										
									
										
										<td>614670810 (99.78%)</td>
										
									
										
										<td>661346570 (99.79%)</td>
										
									
										
										<td>613376232 (99.80%)</td>
										
									
										
										<td>659171532 (99.76%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">SE mapped</td>
										
									
										
										<td>822668 (0.13%)</td>
										
									
										
										<td>744958 (0.11%)</td>
										
									
										
										<td>640378 (0.10%)</td>
										
									
										
										<td>822792 (0.12%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr</td>
										
									
										
										<td>14853268 (2.41%)</td>
										
									
										
										<td>12077240 (1.82%)</td>
										
									
										
										<td>11983926 (1.95%)</td>
										
									
										
										<td>13409950 (2.03%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr ((mapQ>=5))</td>
										
									
										
										<td>12155108 (1.97%)</td>
										
									
										
										<td>9465551 (1.43%)</td>
										
									
										
										<td>9170668 (1.49%)</td>
										
									
										
										<td>10628560 (1.61%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Average_sequencing_depth</td>
										
									
										
										<td>31.21</td>
										
									
										
										<td>33.60</td>
										
									
										
										<td>31.13</td>
										
									
										
										<td>33.47</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage</td>
										
									
										
										<td>99.75%</td>
										
									
										
										<td>99.01%</td>
										
									
										
										<td>99.72%</td>
										
									
										
										<td>99.03%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage_at_least_4X</td>
										
									
										
										<td>99.45%</td>
										
									
										
										<td>98.74%</td>
										
									
										
										<td>99.43%</td>
										
									
										
										<td>98.76%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage_at_least_10X</td>
										
									
										
										<td>98.12%</td>
										
									
										
										<td>98.20%</td>
										
									
										
										<td>98.15%</td>
										
									
										
										<td>98.27%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage_at_least_20X</td>
										
									
										
										<td>87.91%</td>
										
									
										
										<td>93.36%</td>
										
									
										
										<td>88.34%</td>
										
									
										
										<td>93.71%</td>
										
									
									</tr>
								
							
						



						</table>
					</div>
					</p>

					
					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of clean reads.
							The below indexes are calculated based on the clean reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Duplicate: The number of duplicated reads
							(percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Mapped: The number of reads that mapped
							to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Properly mapped: The number of reads that
							mapped to the reference genome and within the expected insert size (percentage). <br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) PE mapped: The number of pair-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) SE mapped: The number of single-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) With mate mapped to a different chr: The
							number of reads with mate reads mapped to different chromosomes (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) With mate mapped to a different chr (mapQ
							>= 5): The number of reads with mate reads mapped to different chromosomes and the MAQ > 5.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) Average_sequencing_depth: The average sequencing depth in the whole genome.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) Coverage: The coverage in the whole genome.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) Coverage_at_least_4X: The coverage in the whole genome when only bases with depth > 4X are considered.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Coverage_at_least_10X: The coverage in the whole genome when only bases with depth > 10X are considered.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(14) Coverage_at_least_20X: The coverage in the whole genome when only bases with depth > 20X are considered.<br />
						</p>
					</div>


					
					<p class="paragraph"><a href="../../../02.Bam" title="Click to open" target="_blank">Result: 02.Bam</a></p>
				</div>
				
				<!---------------------------------------------- snp/InDel结果 ------------------------------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Variant detection"></a>
						<hr />
					</p><br />
					<h3>4.4 Germline Mutation Detection</h3>
					<br />
					<h4>4.4.1 SNP Detection Result</h4>
					<p class="paragraph"> Single nucleotide polymorphisms (SNPs), also known as single nucleotide variants
						(SNVs), constitute the largest class of genetic variants in genome. A typical human genome has about
						3.6 million SNPs (DePristo <I>et al.</I>, 2011). Statistics of detected SNPs are shown below.</p>
					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.3 The number of SNPs in various genomic regions </p>
						<!--<p class="name_fig">Table 4.1 The number of SNPs in various genomic regions </p>-->
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>tian</th>
									
								
									
										<th>songshu</th>
									
								
									
										<th>tianshu</th>
									
								
									
										<th>tianzi</th>
									
								
							</tr>

							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>24299</td>
										
									
										
											<td>24285</td>
										
									
										
											<td>24383</td>
										
									
										
											<td>24791</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;synonymous_SNP</td>
										
									
										
											<td>12171</td>
										
									
										
											<td>12170</td>
										
									
										
											<td>12212</td>
										
									
										
											<td>12414</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;missense_SNP</td>
										
									
										
											<td>11697</td>
										
									
										
											<td>11690</td>
										
									
										
											<td>11762</td>
										
									
										
											<td>11907</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>109</td>
										
									
										
											<td>110</td>
										
									
										
											<td>98</td>
										
									
										
											<td>113</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>12</td>
										
									
										
											<td>10</td>
										
									
										
											<td>11</td>
										
									
										
											<td>12</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>22</td>
										
									
										
											<td>28</td>
										
									
										
											<td>20</td>
										
									
										
											<td>20</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>282</td>
										
									
										
											<td>262</td>
										
									
										
											<td>268</td>
										
									
										
											<td>310</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>1309559</td>
										
									
										
											<td>1315162</td>
										
									
										
											<td>1320346</td>
										
									
										
											<td>1323267</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>34172</td>
										
									
										
											<td>34467</td>
										
									
										
											<td>34671</td>
										
									
										
											<td>35235</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>6479</td>
										
									
										
											<td>6554</td>
										
									
										
											<td>6625</td>
										
									
										
											<td>6612</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>611</td>
										
									
										
											<td>655</td>
										
									
										
											<td>642</td>
										
									
										
											<td>653</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>15209</td>
										
									
										
											<td>15548</td>
										
									
										
											<td>15393</td>
										
									
										
											<td>15656</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>244582</td>
										
									
										
											<td>247634</td>
										
									
										
											<td>245081</td>
										
									
										
											<td>248581</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>182</td>
										
									
										
											<td>183</td>
										
									
										
											<td>180</td>
										
									
										
											<td>185</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>24796</td>
										
									
										
											<td>25081</td>
										
									
										
											<td>25059</td>
										
									
										
											<td>25597</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>25904</td>
										
									
										
											<td>26375</td>
										
									
										
											<td>26169</td>
										
									
										
											<td>26686</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>2114574</td>
										
									
										
											<td>2133913</td>
										
									
										
											<td>2124609</td>
										
									
										
											<td>2138355</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>3799175</td>
										
									
										
											<td>3828692</td>
										
									
										
											<td>3821973</td>
										
									
										
											<td>3844446</td>
										
									
								</tr>
								
							
						
	
						</table>
						</p>
					</div>


					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of SNPs in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							synonymous_SNP: A single nucleotide change that does not cause an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							missense_SNP: A single nucleotide change that causes an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stopgain: A nonsynonymous SNP that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stoploss: A nonsynonymous SNP that leads to the elimination of stop codon at the variant
							site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of SNPs in intronic
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of SNPs in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of SNPs in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of SNPs within 2-bp
							of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of SNPs in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of SNPs in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of SNPs within
							2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of SNPs within 1kb
							away from the transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of SNPs within
							1kb away from the transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of SNPs in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of SNPs.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src='' /></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/SNP/tian.snp.png'><img
												src='src/pictures/SNP/tian.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/songshu.snp.png'><img
												src='src/pictures/SNP/songshu.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/tianshu.snp.png'><img
												src='src/pictures/SNP/tianshu.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/tianzi.snp.png'><img
												src='src/pictures/SNP/tianzi.snp.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>
					<p class="name_fig">Figure 4.7 Number of SNPs in various genomic regions (left pie plot); number of
						different types of SNPs in coding region (right pie plot)</p>
					<p class="premark">&nbsp</p>

					<p class="center">
					<p class="name_fig">Table 4.4 Feature of SNPs</p>
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
											
										<th>Sample name</th>
									
								
									
										<th>tian</th>
									
								
									
										<th>songshu</th>
									
								
									
										<th>tianshu</th>
									
								
									
										<th>tianzi</th>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>3799175</td>
									
								
									
										<td>3828692</td>
									
								
									
										<td>3821973</td>
									
								
									
										<td>3844446</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>2377981</td>
									
								
									
										<td>2444618</td>
									
								
									
										<td>2410511</td>
									
								
									
										<td>2456841</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>1421194</td>
									
								
									
										<td>1384074</td>
									
								
									
										<td>1411462</td>
									
								
									
										<td>1387605</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts</td>
									
								
									
										<td>2534420</td>
									
								
									
										<td>2553719</td>
									
								
									
										<td>2550648</td>
									
								
									
										<td>2563413</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">tv</td>
									
								
									
										<td>1264755</td>
									
								
									
										<td>1274973</td>
									
								
									
										<td>1271325</td>
									
								
									
										<td>1281033</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts/tv</td>
									
								
									
										<td>2.00</td>
									
								
									
										<td>2.00</td>
									
								
									
										<td>2.01</td>
									
								
									
										<td>2.00</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>3678965(96.84%)</td>
									
								
									
										<td>3708978(96.87%)</td>
									
								
									
										<td>3701024(96.84%)</td>
									
								
									
										<td>3721862(96.81%)</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>120210</td>
									
								
									
										<td>119714</td>
									
								
									
										<td>120949</td>
									
								
									
										<td>122584</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts</td>
									
								
									
										<td>62460</td>
									
								
									
										<td>61861</td>
									
								
									
										<td>62902</td>
									
								
									
										<td>62341</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_tv</td>
									
								
									
										<td>57750</td>
									
								
									
										<td>57853</td>
									
								
									
										<td>58047</td>
									
								
									
										<td>60243</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts/tv</td>
									
								
									
										<td>1.08</td>
									
								
									
										<td>1.07</td>
									
								
									
										<td>1.08</td>
									
								
									
										<td>1.03</td>
									
								
							</tr>
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) transition (ts) : The number of
							transitions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) transversion (tv) : The number of
							transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ts/tv: The number of transitions divided
							by the number of transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) dbSNP percentage: The number of SNPs that
							have been reported in dbSNP database divided by the total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) novel: The number of SNPs that have not been reported in
							dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) novel ts: The number of transition SNPs
							that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) novel tv: The number of transversion
							SNPs that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) novel ts/tv: Novel ts divided by novel
							tv.</p>
						<p class="tremark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/SNP" title="Click to open" target="_blank">Result: 03.Result/result/Mutation/SNP</a></p>
				</div>
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="InDel检测结果"></a>
						<hr />
					</p><br />
					<h4>4.4.2 InDel Detection Result</h4>
					<p class="paragraph">Small insertions and deletions (InDels) that are less than 50 bp in length
						constitute another class of genetic variants in human genome. </p>
					<p class="paragraph">The InDels occurred in coding region or splicing sites may cause changes in
						transcripts and proteins. If the number of inserted or deleted nucleotides is not in frame of three or
						multiples of three, the entire reading frame would be altered. The statistics of InDels (DePristo <I>et al.</I>, 2011) called in the
						samples are listed below: </p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.5 The number of InDels in various genomic regions</p>
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>tian</th>
									
								
									
										<th>songshu</th>
									
								
									
										<th>tianshu</th>
									
								
									
										<th>tianzi</th>
									
								
							</tr>
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>803</td>
										
									
										
											<td>778</td>
										
									
										
											<td>796</td>
										
									
										
											<td>919</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift deletion</td>
										
									
										
											<td>175</td>
										
									
										
											<td>162</td>
										
									
										
											<td>168</td>
										
									
										
											<td>158</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift insertion</td>
										
									
										
											<td>127</td>
										
									
										
											<td>112</td>
										
									
										
											<td>121</td>
										
									
										
											<td>195</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift deletion</td>
										
									
										
											<td>262</td>
										
									
										
											<td>263</td>
										
									
										
											<td>268</td>
										
									
										
											<td>255</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift insertion</td>
										
									
										
											<td>193</td>
										
									
										
											<td>195</td>
										
									
										
											<td>191</td>
										
									
										
											<td>250</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>20</td>
										
									
										
											<td>18</td>
										
									
										
											<td>20</td>
										
									
										
											<td>23</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>3</td>
										
									
										
											<td>3</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
										
											<td>4</td>
										
									
										
											<td>5</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>21</td>
										
									
										
											<td>21</td>
										
									
										
											<td>19</td>
										
									
										
											<td>29</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>408763</td>
										
									
										
											<td>416407</td>
										
									
										
											<td>415218</td>
										
									
										
											<td>411336</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>11322</td>
										
									
										
											<td>11465</td>
										
									
										
											<td>11501</td>
										
									
										
											<td>11470</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>1223</td>
										
									
										
											<td>1237</td>
										
									
										
											<td>1268</td>
										
									
										
											<td>1285</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>317</td>
										
									
										
											<td>327</td>
										
									
										
											<td>333</td>
										
									
										
											<td>327</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>2511</td>
										
									
										
											<td>2512</td>
										
									
										
											<td>2556</td>
										
									
										
											<td>2604</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>67178</td>
										
									
										
											<td>68511</td>
										
									
										
											<td>67517</td>
										
									
										
											<td>67759</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>102</td>
										
									
										
											<td>92</td>
										
									
										
											<td>91</td>
										
									
										
											<td>86</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>7195</td>
										
									
										
											<td>7408</td>
										
									
										
											<td>7417</td>
										
									
										
											<td>7565</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>8452</td>
										
									
										
											<td>8613</td>
										
									
										
											<td>8745</td>
										
									
										
											<td>8632</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>540961</td>
										
									
										
											<td>551706</td>
										
									
										
											<td>546200</td>
										
									
										
											<td>547780</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>1048546</td>
										
									
										
											<td>1068753</td>
										
									
										
											<td>1061333</td>
										
									
										
											<td>1059459</td>
										
									
								</tr>
								
							
						

						</table>
						</p>
					</div>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of InDels in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_deletion:
							A deletion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_insertion:
							An insertion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_deletion:
							A deletion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_insertion:
							An insertion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stopgain: An
							insertion or a deletion that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stoploss: An
							insertion or a deletion that leads to the elimination of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of InDels in
							intronic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of InDels in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of InDels in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of InDels within
							2-bp of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of InDels in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of InDels in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of InDels
							within 2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of InDels within
							1kb away from transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of InDels within
							1kb away from transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of InDels in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of InDels.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src=/></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/INDEL/tian.indel.png'><img
												src='src/pictures/INDEL/tian.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/songshu.indel.png'><img
												src='src/pictures/INDEL/songshu.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/tianshu.indel.png'><img
												src='src/pictures/INDEL/tianshu.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/tianzi.indel.png'><img
												src='src/pictures/INDEL/tianzi.indel.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>

					<p class="name_fig">Figure 4.8 Number of InDels in various genomic regions (left); number ofdifferent
						types of InDels in coding region (right)</p>
					<p class="premark">&nbsp</p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.6 Feature of InDels in genome</p>

						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>tian</th>
									
								
									
										<th>songshu</th>
									
								
									
										<th>tianshu</th>
									
								
									
										<th>tianzi</th>
									
								
							</tr>

							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>1048546</td>
									
								
									
										<td>1068753</td>
									
								
									
										<td>1061333</td>
									
								
									
										<td>1059459</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>842072</td>
									
								
									
										<td>866789</td>
									
								
									
										<td>854870</td>
									
								
									
										<td>849071</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>206474</td>
									
								
									
										<td>201964</td>
									
								
									
										<td>206463</td>
									
								
									
										<td>210388</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>835990(79.73%)</td>
									
								
									
										<td>849792(79.51%)</td>
									
								
									
										<td>843227(79.45%)</td>
									
								
									
										<td>847356(79.98%)</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>212556</td>
									
								
									
										<td>218961</td>
									
								
									
										<td>218106</td>
									
								
									
										<td>212103</td>
									
								
							</tr>			
							
						

						</table>
						</p>
					</div>

					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) dbSNP percentage: The number of InDels
							that have been reported in dbSNP database divided by the total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) novel: The number of InDels that have not
							been reported in dbSNP database.<br />
						<p class="premark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/INDEL" title="Click to open" target="_blank">Result: 03.Result/result/Mutation/INDEL</a></p>
				</div>
				
				<div id="page">
					<p class="head"><a href="#home" title = "返回首页"></a>
						<a name="SV结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.3 SV Detection Result</h4>
					<p class="paragraph">Structural variants (SVs) are genetic variants with relatively large size (>50 bp), including deletions, duplications, insertions, inversions and translocations. SVs may form the underlying genetic basis of individual differences, and have potential effect on disease and cancer susceptibility. The statistics of SV (Rausch et al., 2012) are shown below: </p>
					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.7 SV detection result</p>
							
							<table class="gy">
							
								
								<tr>
									
										
											<th>Sample name</th>
										
									
										
											<th>DUP</th>
										
									
										
											<th>INV</th>
										
									
										
											<th>INS</th>
										
									
										
											<th>DEL</th>
										
									
										
											<th>BND</th>
										
									
								</tr>
								
							
								
								<tr>
									
										<td>tian</td>
									
										<td>119</td>
									
										<td>129</td>
									
										<td>84</td>
									
										<td>2505</td>
									
										<td>81</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>songshu</td>
									
										<td>113</td>
									
										<td>124</td>
									
										<td>82</td>
									
										<td>2541</td>
									
										<td>81</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>tianshu</td>
									
										<td>126</td>
									
										<td>102</td>
									
										<td>88</td>
									
										<td>2569</td>
									
										<td>58</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>tianzi</td>
									
										<td>132</td>
									
										<td>129</td>
									
										<td>87</td>
									
										<td>2596</td>
									
										<td>85</td>
									
								</tr>
								
							

							</table>
							</p>
					</div>

					

					

					
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /> 
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) DUP: (Tandem duplication) The number of tandem duplications.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) INV: (Inversion) The number of inversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) INS: (Insertion) The number of Insertions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) DEL: (Deletion) The number of deletions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) BND: (Translocation) The number of translocations.<br />
						<p class="premark">&nbsp</p>
					</div>
					

					

					
				
					<p class="center">
						<img height="300px" src="src/pictures/SV/sv_type_stat.png" />
						<p class="name_fig">Figure 4.9 Number of different types of SV in each sample </p>
						<p class="premark">The x-axis represents samples, and the y-axis represents the number of each type of SV </p>
					<p class="premark">&nbsp</p>
					</p>

					<p class="paragraph"><a href="../../result/Mutation/SV" title="Click to open" target="_blank">Result: 03.Result/result/Mutation/SV</a></p>
				</div>
				

				
<!----------------------------------------- CNV结果  --------------------------------------------------->
				<div id="page">
					<p class="head"><a href="#home" title = "返回首页"></a>
					<a name="CNV结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.4 CNV Detection Result</h4>
					<p class="paragraph">Copy number variants (CNVs) are genetic variants that lead to variation in copy number of relatively large fragment (longer than 50 bp) among individuals. There are two types of CNVs, i.e. gains and losses of copies. CNVs may form the underlying genetic basis of individual differences and cancer. The statistics of detected CNVs (Boeva et al., 2012) are listed below: </p>  

					<p class="center">
						<p class="name_fig">Table 4.8 CNV detection result</p>
					</p>

					<div id="tb">
						<table class="gy">
							<tr>
								<th>Sample name</th>
								<th>gain_count</th>
								<th>gain_size</th>
								<th>loss_count</th>
								<th>loss_size</th>
								<th>total_count</th>
								<th>total_size</th>
							</tr>
							
								
								<tr>
									
										<td>tian</td>
									
										<td>236</td>
									
										<td>3810236</td>
									
										<td>148</td>
									
										<td>1842148</td>
									
										<td>384</td>
									
										<td>5652384</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>songshu</td>
									
										<td>176</td>
									
										<td>5817176</td>
									
										<td>168</td>
									
										<td>2079168</td>
									
										<td>344</td>
									
										<td>7896344</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>tianshu</td>
									
										<td>242</td>
									
										<td>5301242</td>
									
										<td>139</td>
									
										<td>1577139</td>
									
										<td>381</td>
									
										<td>6878381</td>
									
								</tr>
								
							
								
								<tr>
									
										<td>tianzi</td>
									
										<td>191</td>
									
										<td>5618191</td>
									
										<td>161</td>
									
										<td>2597161</td>
									
										<td>352</td>
									
										<td>8215352</td>
									
								</tr>
								
								
						</table>
					</div>
					</p>
				
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
					<p class="tremark">
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) gain_count: The number of gains.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) gain_size: The total size of gains.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) loss_count: The number of losses.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) loss_size: The total size of losses.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) total_count: The total number of CNVs.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) total_size: The total size of CNVs.</p>
						<p class="premark">&nbsp</p>		
					</div>

					<p class="center">
						<img height="300px" src="src/pictures/SV/cnv_type_stat.png" />
						<p class="name_fig">Figure 4.10 The size of genomic regions affected by CNVs in each sample</p>
						<p class="premark">The x-axis represents samples name, and the y-axis represents the total size of genomic regions affected by gains or losses (Mb)</p>
					</p>

					<p class="paragraph"><a href="../../result/Mutation/CNV" title="Click to open" target="_blank">Result: 03.Result/result/Mutation/CNV</a></p>
				</div>
				
			
				
				
				<div id="page">
					<p class="head"><a href="#home" title = "返回首页"></a>
						<a name="Circos结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.5 Circos</h4>
					<p class="paragraph">The Circos for visualizing the genetic variants of whole genome are shown below:</p>
					<p class="center">
					<div class="albumSlider">
						<div class="fullview"><img src="src/pictures/Circos/tian.png"/></div>
						<div class="slider">
							<div class="button movebackward" title="Up"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								
									<li><a id="example2" href="src/pictures/Circos/tian.png" ><img src="src/pictures/Circos/tian.JPEG"/></a></li>
								
									<li><a id="example2" href="src/pictures/Circos/songshu.png" ><img src="src/pictures/Circos/songshu.JPEG"/></a></li>
								
									<li><a id="example2" href="src/pictures/Circos/tianshu.png" ><img src="src/pictures/Circos/tianshu.JPEG"/></a></li>
								
									<li><a id="example2" href="src/pictures/Circos/tianzi.png" ><img src="src/pictures/Circos/tianzi.JPEG"/></a></li>
								
								</ul>
							</div>
							<div class="button moveforward" title="Down"></div>
						</div>
					</div>
				
					<p class="name_fig">Figure 4.11 Circos </p>


					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We shows Circos only when CNV analysis was carried out. The figure consists seven rings from outer to inner.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) The outer circle (the first circle) is Chromosomal	 information.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) The second ring represents the read coverage in histogram style. A histogram is the average coverage of a 0.5Mbp region.<br />
							
								&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) The third ring represents InDel density in scatter style. A black dot is calculated as InDel number in a range of 1Mbp.<br />
								&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) The fourth ring represents SNP density in scatter style. A green dot is calculated as SNP number in a range of 1Mbp.<br />
								&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) The fifth ring represents the proportion of homozygous SNP (orange) and heterozygous SNP (grey) in histogram style.A histogram is calculated from a 1Mbp region.<br />
								&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) The sixth ring represents the CNV inference. Red means gain, and green means loss.<br />
							
							
							
							
							
								&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) The most central ring represents the SV inference in exonic and splicing regions. BND (brown), INS (green), DEL (grey), DUP (red) and INV (blue).<br />
							
							
		                    
							
						
				</div>	
				</div>	
				


				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="SNP/InDel结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.6 Variant Annotation </h4>
					<p class="paragraph">Following genomic variant detection, annotation of variants with the tool ANNOVAR (Wang et
						al., 2010) in multiple aspects are performed, including protein coding changes, genomic regions affected by
						the variants, allele frequency, deleteriousness prediction, etc. The main databases used are as
						follows:<br />
					<ul>
						<li style="font-size: 0.6em">RefSeq (O'Leary <I>et al.</I>, 2016) and Gencode (Frankish <I>et al.</I>, 2021) databases
							were used to find genomic regions affected by the variant and possible changes in protein. </li>
						<li style="font-size: 0.6em">We annotated the features of the genomic regions affected by the variants, such
							as cytoband, small RNA, conserved mammalian microRNA regulatory target sites, conservative regions of
							vertebrates, transcription factor binding sites, repeats, etc. </li>
						<li style="font-size: 0.6em">SIFT (Ng <I>et al.</I>, 2003), PolyPhen (Adzhubei <I>et al.</I>, 2013), MutationAssessor
							(Reva <I>et al.</I>, 2011), LRT (Chun <I>et al.</I>, 2009) and CADD (Rentzsch <I>et al.</I>, 2019) scores were used to predict
							the deleteriousness of mutations. GERP++ (Huber <I>et al.</I>, 2020) scores were used to access the conservation
							of mutations.</li>
						<li style="font-size: 0.6em">The established databases, such as 1000 Human Genome (Abecasis <I>et al.</I>, 2012),
							Exome Aggregation Consortium (ExAC) (Kobayashi <I>et al.</I>, 2017), Genome Aggregation Database (gnomAD) (Pio
							<I>et al.</I>, 2021) and exome sequencing project (ESP), were used to find alternative allele frequencies in
							populations that are reported.</li>
						<li style="font-size: 0.6em">Databases dbSNP (Sherry <I>et al.</I>, 2001), COSMIC (Tate <I>et al.</I>, 2019), OMIM (Hamosh
							<I>et al.</I>, 2005), GWAS Catalog (Buniello <I>et al.</I>, 2019) and HGMD (Stenson <I>et al.</I>, 2020) were used to find
							reported information of the variant, such as top SNPs in GWAS and cancer/disease associations. </li>
						<li style="font-size: 0.6em">Databases including Gene Ontology (Lee <I>et al.</I>, 2004), KEGG (Kanehisa <I>et al.</I>,
							2000), Reactome (Jassal <I>et al.</I>, 2020), Biocarta and PID (Schaefer <I>et al.</I>, 2009) were applied to provide
							functional or pathway annotation. </li>
					</ul>
					</p>
		
					<p class="center">
					
					<p class="name_fig">Table 4.9 Example of annotation result</p>
					
					<div id="tb">
						<table class="gy">
							
								
								<tr>
									
										<th>Priority</th>
									
										<th>CHROM</th>
									
										<th>POS</th>
									
										<th>ID</th>
									
										<th>REF</th>
									
										<th>ALT</th>
									
										<th>QUAL</th>
									
										<th>FILTER</th>
									
										<th>GeneName</th>
									
										<th>Description</th>
									
										<th>Func</th>
									
										<th>Gene</th>
									
										<th>GeneDetail</th>
									
										<th>ExonicFunc</th>
									
										<th>AAChange</th>
									
										<th>Gencode</th>
									
										<th>cpgIslandExt</th>
									
										<th>cytoBand</th>
									
										<th>wgRna</th>
									
										<th>genomicSuperDups</th>
									
										<th>Repeat</th>
									
										<th>avsnp</th>
									
										<th>CLNALLELEID</th>
									
										<th>CLNDN</th>
									
										<th>CLNDISDB</th>
									
										<th>CLNREVSTAT</th>
									
										<th>CLNSIG</th>
									
										<th>cosmic</th>
									
										<th>gwasCatalog</th>
									
										<th>1000g_SAS</th>
									
										<th>1000g_EUR</th>
									
										<th>1000g_AFR</th>
									
										<th>1000g_AMR</th>
									
										<th>1000g_EAS</th>
									
										<th>1000g_ALL</th>
									
										<th>esp6500siv2_all</th>
									
										<th>gnomad_exome_AF</th>
									
										<th>gnomad_exome_AF_raw</th>
									
										<th>gnomad_exome_AF_afr</th>
									
										<th>gnomad_exome_AF_sas</th>
									
										<th>gnomad_exome_AF_amr</th>
									
										<th>gnomad_exome_AF_eas</th>
									
										<th>gnomad_exome_AF_nfe</th>
									
										<th>gnomad_exome_AF_fin</th>
									
										<th>gnomad_exome_AF_asj</th>
									
										<th>gnomad_exome_AF_oth</th>
									
										<th>gnomad_genome_AF</th>
									
										<th>gnomad_genome_AF_raw</th>
									
										<th>gnomad_genome_AF_afr</th>
									
										<th>gnomad_genome_AF_sas</th>
									
										<th>gnomad_genome_AF_amr</th>
									
										<th>gnomad_genome_AF_eas</th>
									
										<th>gnomad_genome_AF_nfe</th>
									
										<th>gnomad_genome_AF_fin</th>
									
										<th>gnomad_genome_AF_asj</th>
									
										<th>gnomad_genome_AF_oth</th>
									
										<th>ExAC_ALL</th>
									
										<th>ExAC_AFR</th>
									
										<th>ExAC_AMR</th>
									
										<th>ExAC_EAS</th>
									
										<th>ExAC_FIN</th>
									
										<th>ExAC_NFE</th>
									
										<th>ExAC_OTH</th>
									
										<th>ExAC_SAS</th>
									
										<th>dbscSNV_ADA_SCORE</th>
									
										<th>dbscSNV_RF_SCORE</th>
									
										<th>Interpro_domain</th>
									
										<th>SIFT</th>
									
										<th>Polyphen2_HDIV</th>
									
										<th>Polyphen2_HVAR</th>
									
										<th>LRT</th>
									
										<th>MutationTaster</th>
									
										<th>MutationAssessor</th>
									
										<th>FATHMM</th>
									
										<th>phyloP100way_vertebrate</th>
									
										<th>phyloP30way_mammalian</th>
									
										<th>SiPhy_29way_logOdds</th>
									
										<th>CADD</th>
									
										<th>gerp++gt2</th>
									
										<th>REVEL</th>
									
										<th>INFO</th>
									
										<th>FORMAT</th>
									
										<th>tianzi</th>
									
										<th>Ori_REF</th>
									
										<th>Ori_ALT</th>
									
										<th>shared_hom</th>
									
										<th>shared_het</th>
									
										<th>OMIM</th>
									
										<th>GWAS_Pubmed_pValue</th>
									
										<th>HGMD_Disease_ID</th>
									
										<th>HGMD_Mutation_URL</th>
									
										<th>PubMedID</th>
									
										<th>GO_BP</th>
									
										<th>GO_CC</th>
									
										<th>GO_MF</th>
									
										<th>KEGG_PATHWAY</th>
									
										<th>PID_PATHWAY</th>
									
										<th>BIOCARTA_PATHWAY</th>
									
										<th>REACTOME_PATHWAY</th>
									
										<th>Expression_summary</th>
									
										<th>RNA tissue specificity</th>
									
										<th>RNA tissue distribution</th>
									
										<th>Subcellular_Location</th>
									
										<th>RNA tissue specific nTPM</th>
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>16298</td>
										
									
										
											<td>rs2747966</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>337.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>WASH7P</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_intronic</td>
										
									
										
											<td>NR_024540</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000488147.1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.993729;Name=chr9:10843</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs2747966</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ID=COSN19621909;OCCURENCE=2(haematopoietic_and_lymphoid_tissue)</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.5598</td>
										
									
										
											<td>0.5583</td>
										
									
										
											<td>0.5017</td>
										
									
										
											<td>0.5486</td>
										
									
										
											<td>0.5826</td>
										
									
										
											<td>0.4950</td>
										
									
										
											<td>0.5810</td>
										
									
										
											<td>0.5966</td>
										
									
										
											<td>0.6096</td>
										
									
										
											<td>0.5455</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=0.067;DP=43;ExcessHet=0;FS=30.377;MLEAC=1;MLEAF=0.5;MQ=45.76;MQRankSum=-5.269;QD=8.24;ReadPosRankSum=0.688;SOR=2.943</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:27,14:41:99:345,0,896</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>17571</td>
										
									
										
											<td>rs557947346</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>104.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>WASH7P</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_intronic</td>
										
									
										
											<td>NR_024540</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000488147.1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.993729;Name=chr9:10843</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs557947346</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.001</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.001</td>
										
									
										
											<td>0.000399361</td>
										
									
										
											<td>.</td>
										
									
										
											<td>2.469e-05</td>
										
									
										
											<td>4.034e-05</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>5.155e-05</td>
										
									
										
											<td>0.0001</td>
										
									
										
											<td>2.11e-05</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>5.893e-05</td>
										
									
										
											<td>0</td>
										
									
										
											<td>4.595e-05</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=0.937;DP=48;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=45.51;MQRankSum=0.767;QD=2.27;ReadPosRankSum=-0.441;SOR=0.941</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:40,6:46:99:112,0,1448</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>49298</td>
										
									
										
											<td>rs10399793</td>
										
									
										
											<td>T</td>
										
									
										
											<td>C</td>
										
									
										
											<td>915.06</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>intergenic</td>
										
									
										
											<td>NR_026818,NM_001005484</td>
										
									
										
											<td>dist=13217;dist=19793</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.994828;Name=chr19:60000</td>
										
									
										
											<td>Score=5956;Name="61:L1PA14(LINE)"</td>
										
									
										
											<td>rs10399793</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.8855</td>
										
									
										
											<td>0.8936</td>
										
									
										
											<td>0.5204</td>
										
									
										
											<td>0.7104</td>
										
									
										
											<td>0.9633</td>
										
									
										
											<td>0.782149</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.5963</td>
										
									
										
											<td>0.5177</td>
										
									
										
											<td>0.3468</td>
										
									
										
											<td>0.7700</td>
										
									
										
											<td>0.5384</td>
										
									
										
											<td>0.8390</td>
										
									
										
											<td>0.7633</td>
										
									
										
											<td>0.7434</td>
										
									
										
											<td>0.7889</td>
										
									
										
											<td>0.6205</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=2;AF=1;AN=2;DP=26;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=41.31;QD=25.36;SOR=1.609</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>1/1:0,25:25:75:929,75,0</td>
										
									
										
											<td>T</td>
										
									
										
											<td>C</td>
										
									
										
											<td>1</td>
										
									
										
											<td>0</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>51803</td>
										
									
										
											<td>rs62637812</td>
										
									
										
											<td>T</td>
										
									
										
											<td>C</td>
										
									
										
											<td>612.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>intergenic</td>
										
									
										
											<td>NR_026818,NM_001005484</td>
										
									
										
											<td>dist=15722;dist=17288</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.994828;Name=chr19:60000</td>
										
									
										
											<td>Score=2373;Name="65:AluYj4(SINE)"</td>
										
									
										
											<td>rs62637812</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.3829</td>
										
									
										
											<td>0.3431</td>
										
									
										
											<td>0.2757</td>
										
									
										
											<td>0.4704</td>
										
									
										
											<td>0.3834</td>
										
									
										
											<td>0.5733</td>
										
									
										
											<td>0.4277</td>
										
									
										
											<td>0.4279</td>
										
									
										
											<td>0.3873</td>
										
									
										
											<td>0.4017</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=1.27;DP=38;ExcessHet=0;FS=13.443;MLEAC=1;MLEAF=0.5;MQ=42.19;MQRankSum=-1.091;QD=16.56;ReadPosRankSum=0.475;SOR=0.943</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:16,21:37:99:620,0,465</td>
										
									
										
											<td>T</td>
										
									
										
											<td>C</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
						</table>
					</div>
					<br />
		
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="premark">&nbsp;</p>
                        

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>Note: </b>Annotation information includes seven parts: Priority information (1), Chromosome regions and gene structures related to this variation (2-21), Database (frequency) annotation (22-64), Conservation (Deleterious) prediction (65-80), Basic information on the variation (81-87), Gene function and pathway annotation (88-99), and Gene tissue-specific expression annotation (100-104).<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The first item is Priority information.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(1) Priority: </b>H: High: indicates that the mutation is not in the genomic repeat region but in the exonic or splicing region (that is, there is no information for genomicSuperDups and Repeat annotation). The frequency of it in the 1000 genome database is less than 0.01. At least one of SIFT, Polyphen, MutationTaster and CADD predicted that the site is deleterious. M: Medium: indicates that this site is not in the genomic repeat region but in the exonic or splicing region (that is, the mutation cannot be annotated by genomicSuperDups and Repeat). The frequency of it in the 1000-genome database is less than 0.01. L: Low, indicating that the site does not meet the H or M condition of all other mutations. The International Mouse Phenotyping Consortium (IMPC) performs large-scale gene knockout and phenotypic analysis on mice. The symbols (*, +, -) after the priority (H, M, L) mean: *: the gene is lethal (complete lethal gene) in the analysis of large-scale gene knockout phenotype of mice; +: subviable gene (possible lethal gene: < 12.5% embryo survival) in the analysis of large-scale gene knockout phenotype of mice; -: In a large-scale gene knockout phenotype analysis of mice, the gene was viable (viable gene).<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The second item is chrosomal regions and gene structures related to this variation--The Gene where the variation sites located may be associated with certain diseases. We annotates chrosomal regions and gene structures related to this variation, to help teachers understand gene structure and areas information.</b><br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(2) CHROM: </b>Chromosome ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(3) POS: </b>The position of the variant on chromosomes. The value refers to the position of the first base in the REF sting.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(4) ID: </b>The rs number of the variant in dbSNP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(5) REF: </b>Reference base(s).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(6) ALT: </b>Alternate base(s). Comma separated list of alternate non-reference alleles detected in at least one of the samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(7) QUAL: </b>Quality value for the variant. Phred-scaled quality score for the assertion made in ALT. i.e. -10log10 prob (call in ALT is wrong).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(8) FILTER: </b>Filter status, PASS if the position has passed all filters.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(9) GeneName: </b>Names of genes in which this variant is located according to the refGene annotations.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(10) Description: </b>Gene description.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(11) Func: </b>This field tells whether the variant hits exons or hits intergenic regions, or hits introns, or hits non-coding RNA genes. The value of this field takes the following precedence: exonic = splicing > ncRNA > UTR5/UTR3 > intronic > upstream/downstream > intergenic. Notes: 1. When a variant hits different genes or transcripts, the variant may fit multiple functional categories, and then the precedence mentioned above is used to decide what function to print out; 2. The 'exonic' here refers only to coding exonic portion, but not UTR portion, as there are two keywords (UTR5, UTR3) that are specifically reserved for UTR annotations; 3. If a variant is located in both 5'UTR and 3'UTR region (possibly for two different genes), then the 'UTR5,UTR3' will be printed as the output; 4. 'splicing' in ANNOVAR is defined as variant that is within 2bp away from an exon/intron boundary by default; 5. 'splicing' in ANNOVAR only refers to the 2bp in the intron that is close to an exon; 6. The term 'upstream' and 'downstream' is defined as 1kb away from transcription start site or transcription end site, respectively, taking in account of the strand of the mRNA. If a variant is located in both downstream and upstream region (possibly for 2 different genes), then the 'upstream, downstream' will be printed as the output.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(12) Gene: </b>The transcript name(s). If a variant has 'intergenic' in 'Func' field, this field will give two neighboring transcripts. If a variant hits multiple transcripts with different functional categories, only transcript names in accordance with the value of 'Func' field will be output. For example, rs333970 hits the exonic, splicing, intronic, exonic of the four transcripts of gene CSF1, the 'Func' value will be 'exonic; splicing' and the 'Gene' value will be 'NM_000757, NM_172210, NM_172212' (NM_172211 will be ignored).<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(13) GeneDetail: </b>Description of the sequence change in UTR, splicing, ncRNA_splicing or intergenic region. If 'Func' is 'exonic; splicing' or 'splicing', this field gives the sequence change in splicing region(s); for example, NM_172210:exon6:c.1090+5C>A, NM_172210 is the transcript identifier; exon6:c.1090+5C>A is the sequence change and means that this C>A substitution is at the fifth base downstream from the 6th exon (1090 is the end position of the 6th exon of the cDNA). If 'Func' is 'intergenic', this field gives the distance to the neighboring transcripts, such as 'dist=1366; dist=22344'. If 'Func' is 'UTR*', this field gives the sequence change in UTR; for example, NM_198576:c.*19C>T means that this C>T substitution is at the 19th base downstream from stop codon on NM_198576.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(14) ExonicFunc: </b>This field tells the functional consequences of the variant (possible values included: missense SNV, synonymous SNV, frameshift insertion, frameshift deletion, nonframeshift insertion, nonframeshift deletion, frameshift block substitution, nonframshift block substitution, stopgain, stoploss, unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(15) AAChange: </b>This field tells the amino acid changes as a result of the exonic variant. Only exonic variants have information in this field, i.e. when 'Func' is 'exonic' or 'exonic; splicing', this field gives the amino acid change in each related transcript. For example, AIM1L:NM_001039775:exon2:c.C2768T:p.P923L, AIM1L is gene name; NM_001039775 is the transcript identifier; exon2 means this variant is on the second exon of NM_001039775; c.C2768T is the sequence change and means that this C>T substitution is at the 2,768 position on the cDNA; p.P923L is the amino acid change and means that the 923 amino acid on protein is changed from Pro to Leu due to this variant. Another example, NADK:NM_001198995:exon10:c.1240_1241insAGG:p.G414delinsEG, c.1240_1241insAGG is the sequence change and means that there is a 3bp insertion between position 1,240 and 1,241 on the cDNA; p.G414delinsEG is the amono acid change and means that Gly at the 414th amino acid on protein is changed to Glu-Gly.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(16) Gencode: </b>The transcript name(s) in which this variant is located according to Gencode gene definitions.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(17) cpgIslandExt: </b>CpG island prediction. CpG:116 means that there are 116 CG dinucleotides in this CpG island.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(18) cytoband: </b>This field gives the Giemsa-stained chromosomes bands. When a variant spans multiple bands, they will be connected by a dash (for example, 1q21.1-q23.3).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(19) wgRna: </b>Gene names of snoRNAs and microRNAs based on the miRBase Release and snoRNABase.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(20) genomicSuperDups: </b>This field tells whether the variant hits segmental duplications in reference genome. Variants that mapped to segmental duplications are most likely sequence alignment errors and should be treated with extreme caution. The 'Score' field in output is the sequence identity ranging from 0 to 1 between two genomic segments. The 'Name' field represents the other "matching" segments in genome. For example, 'Score=0.994828;Name=chr19:60000' means that the fragment at the position of chr19:60000 is homologous to the fragment containing this variant, and the sequence identity is 0.994828. Note, for a region to be included in the segmental duplications, at least 1 Kb of the total sequence (containing at least 500bp of non-RepeatMasked sequence) had to align and a sequence identity of at least 90% was required.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(21) Repeat: </b>This field tells whether the variant hits interspersed repeats and low complexity DNA sequences output by RepeatMasker program, such as SINE, LINE and Simple repeats. For example, 'Score=180;Name=1385:(CACCC) n(Simple_repeat)', 180 is the score of the repeat, (CACCC) n is the name of the repeat, 'Simple_repeat' is type of repeat. Note, variants mapped to repeats are likely to be false and should be treated with extreme caution.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The third item is database annotation—There are a great number of common polymorphism sites in human population, while many deleterious variants are rare or low-frequency. This part gives the allele frequency and clinical information for each variant.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(22) avsnp: </b>The RS number of the variant in dbSNP database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(23) CLNALLELEID: </b>The ClinVar Allele ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(24) CLNDN: </b>ClinVar's preferred disease name for the concept specified by disease identifiers in CLNDISDB.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(25) CLNDISDB: </b>Tag-value pairs of disease database name and identifier, e.g. OMIM: NNNNNN.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(26) CLNREVSTAT: </b>ClinVar review status for the Variation ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(27) CLNSIG: </b>Clinical significance for this single variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(28) cosmic: </b>The Cosmic id in Catalogue Of Somatic Mutations In Cancer (COSMIC) database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(29) gwasCatalog: </b>This field tells whether this variant was previously reported to be associated with diseases or traits in genome-wide association studies. It lists the disease names related to this variation. "." means this variation has not been reported by published GWAS study.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(30) 1000g_SAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(31) 1000g_EUR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(32) 1000g_AFR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(33) 1000g_AMR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(34) 1000g_EAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(35) 1000g_ALL: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in ALL population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(36) esp6500siv2_all: </b>The ESP is a NHLBI funded exome sequencing project aiming to identify genetic variants in exonic regions from over 6000 individuals, including healthy ones as well as subjects with different diseases. This field gives alternative allele frequency for the variant in ESP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(37) gnomAD_exome_AF: </b>The Genome Aggregation Database (gnomAD), is a coalition of investigators seeking to aggregate and harmonize exome and genome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. This release comprises two callsets: exome sequence data from 730,947 individuals and whole genome sequencing from 76,215 individuals. GenomAD v4.0.0 was used for annotation analysis of the mutation sites. gnomAD_exome_AF gives allele frequency for the variation in all exome  samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(38) gnomAD_exome_AF_raw: </b>The allele frequency for the variation in gnomAD_exome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(39) gnomAD_exome_AF_afr: </b>The allele frequency for the variation in gnomAD_exome for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(40) gnomAD_exome_AF_sas: </b>The allele frequency for the variation in gnomAD_exome for South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(41) gnomAD_exome_AF_amr: </b>The allele frequency for the variation in gnomAD_exome for Latino/Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(42) gnomAD_exome_AF_eas: </b>The allele frequency for the variation in gnomAD_exome for East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(43) gnomAD_exome_AF_nfe: </b>The allele frequency for the variation in gnomAD_exome for Non-Finnish European population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(44) gnomAD_exome_AF_fin: </b>The allele frequency for the variation in gnomAD_exome for  Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(45) gnomAD_exome_AF_asj: </b>The allele frequency for the variation in gnomAD_exome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(46) gnomAD_exome_AF_oth: </b>The allele frequency for the variation in gnomAD_exome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(47) gnomAD_genome_AF: </b>gnomAD_genome_AF gives allele frequency for the variation in all genome samples of gnomAD. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(48) gnomAD_genome_AF_raw: </b>The allele frequency for the variation in gnomAD_genome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(49) gnomAD_genome_AF_afr: </b>The allele frequency for the variation in gnomAD_genome for African population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(50) gnomAD_genome_AF_sas: </b>The allele frequency for the variation in gnomAD_genome for  South Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(51) gnomAD_genome_AF_amr: </b>The allele frequency for the variation in gnomAD_genome for  Latino/Admixed American population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(52) gnomAD_genome_AF_eas: </b>The allele frequency for the variation in gnomAD_genome for  East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(53) gnomAD_genome_AF_nfe: </b>The allele frequency for the variation in gnomAD_genome for  Non-Finnish European population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(54) gnomAD_genome_AF_fin: </b>The allele frequency for the variation in gnomAD_genome for  Finnish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(55) gnomAD_genome_AF_asj: </b>The allele frequency for the variation in gnomAD_genome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(56) gnomAD_genome_AF_oth: </b>The allele frequency for the variation in gnomAD_genome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(57) ExAC_ALL: </b>ExAC is short for Exome Aggregation Consortium. The data set spans 60,706 unrelated individuals and should serve as a useful reference set of allele frequencies for severe disease studies. Currently supported population groups include ALL, AFR (African), AMR (Admixed American), EAS (East Asian), FIN (Finnish), NFE (Non-finnish European), OTH (other) and SAS (South Asian). ExAC_ALL gives alternative allele frequency for the variation in ALL ExAC samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(58) ExAC_AFR: </b>The alternative allele frequency for the variation in ExAC for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(59) ExAC_AMR: </b>The alternative allele frequency for the variation in ExAC for Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(60) ExAC_EAS: </b>The alternative allele frequency for the variation in ExAC for East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(61) ExAC_FIN: </b>The alternative allele frequency for the variation in ExAC for Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(62) ExAC_NFE: </b>The alternative allele frequency for the variation in ExAC for Non-Finnish European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(63) ExAC_OTH: </b>The alternative allele frequency for the variation in ExAC for other population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(64) ExAC_SAS: </b>The alternative allele frequency for the variation in ExAC for South Asian population.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fourth item is functional prediction—These annotations can help to evaluate deleteriousness of a variation. Note 1: SIFT, Polyphen2, MutationTaster, LRT, MutationAssessor and FATHMM are similar and all predict whether an amino acid substitution affects protein function; only coding variants have these annotations. Note 2: phyloP, SiPhy, gerp++ and CADD are similar and predict the conservation level of the site; these types of 'conservation scores' only consider conservation level at the current base, and they do not care about the actual nucleotide identity, so synonymous and non-synonymous variants at the same site will be scored as the same; these scores are used for finding functionally important sites, so variants that confer increased susceptibility may be scored well.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(65) dbscSNV_ADA_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on adaptive boosting. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(66) dbscSNV_RF_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on Random Forest. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(67) Interpro_domain: </b>The domain of the protein where the mutation is annotated in the Interpro database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(68) SIFT: </b>SIFT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction; the scores and predictions are separated by comma. There are two possible predictions: D (Deleterious, score&lt=0.05)); T (Tolerated, score&gt0.05).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(69) Polyphen2_HDIV: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanDiv database. This annotation should be used when evaluating rare alleles at loci potentially involved in complex phenotypes, dense mapping of regions identified by genome-wide association studies, and analysis of natural selection from sequence data. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score&gt=0.957), P (possibly damaging, 0.453&lt=score&lt=0.956), B (benign, score&lt=0.452).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(70) Polyphen2_HVAR: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanVar database. This annotation should be used for diagnostics of Mendelian diseases. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score>=0.909), P (possibly damaging, 0.447&lt=score&lt=0.909), B (benign, score&lt=0.446).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(71) LRT: </b>LRT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are three possible predictions: D (Deleterious), N (Neutral), U (Unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(72) MutationTaster: </b>MutationTaster annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: 'A' (Disease_causing_automatic), 'D (Disease_causing), 'N' (Polymorphism), 'P' (Polymorphism_automatic'). D and N are categorized by only score, while A and P are categorized by score and other information (if nonsynonymous SNV leads to stop-gain, the variation will be predicted an 'A'; if all three genotypes of nonsynonymous SNV has frequency information in HapMap, the variation will be predicted a 'P'). So, both A and D should be considered deleterious.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(73) MutationAssessor: </b>MutationAssessor annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: H (high), M (medium), L (low), N (neutral). H/M means functional and L/N means non-functional.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(74) FATHMM: </b>FATHMM annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are two possible predictions: D (Deleterious, score&lt=-1.5)); T (Tolerated, score&gt-1.5).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(75) phyloP100way_vertebrate: </b>PhyloP score (dbNSFP version 3.3a) based on the whole genome alignment of 100 vertebrates. Generally, the higher the score, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(76) phyloP30way_mammalian: </b>PhyloP score (dbNSFP) based on the whole genome alignment of 30 mammals.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(77) SiPhy_29way_logOdds: </b>SiPhy score (dbNSFP version 3.3a) based on the whole genome alignment of 29 mammals genomes. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(78) CADD: </b>CADD (Combined Annotation Dependent Depletion) is a score that is based on SVM on multiple other scores. It assigns a score to each possible mutation in the human genome including non-coding and coding variants. In the output, the comma-delimited values are raw scores and phred-scaled scores. "." stands for CADD score&lt10. For phred-scaled scores, 10 means 10% percentile highest scores, 20 means 1% percentile highest scores, and 30% means 0.1% percentile highest scores. CADD official website suggests 15 as a cutoff; in published studies, 10 or 15 is used as a cutoff.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(79) gerp++gt2: </b>GERP++ scores for all mutations with GERP++>2 in human genome, as this threshold is typically regarded as evolutionarily conserved and potentially functional. Variants with '.' in this field should be considered not conserved. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(80) REVEL: </b>A method for predicting the pathogenicity of rare missense mutations by integrating multiple software prediction scores. The score of REVEL ranges from 0 to 1, and the larger the score is, the more deleterious it will be. In the literature, when the score is 0.5, the sensitivity and specificity of detecting harmful mutations are 0.754 and 0.891 respectively.<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fifth item is basic information on the variation  —This part shows the detail information of variation sites, including allelic depths , types of DNA base before and after mutation, and genotype information, et al. these information play a critical role in pedigree analysis.</b> <br />

<p class="premark">&nbsp;</p>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(81) INFO: </b>Information about this variation from variant calling software. INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: &ltkey&gt=&ltdata&gt[,data].<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(82) FORMAT: </b>The FORMAT field specifies the data types and order (colon-separated alphanumeric String). This is followed by one field per sample, with the colon-separated data corresponding to the types specified in the FORMAT.<br />


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GT: genotype, encoded as allele values separated by either of / or |. The allele values are 0 for the reference allele (what is in the Ori_REF field), 1 for the first allele listed in Ori_ALT, 2 for the second allele list in Ori_ALT and so on. 0/0 and 1/1 represent homozygous. 0/1 represents heterozygous. ‘.’ means that a call cannot be made for a sample at a given locus.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AD: Allelic depths for the ref and alt alleles in the order listed (Allelic depths).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DP: Approximate read depth (reads with MQ=255 or with bad mates are filtered).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GQ: Genotype Quality.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PL: Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(83) SampleID: </b>The colon-separated data in this sample corresponding to the types specified in the FORMAT.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(84) Ori_REF: </b>The reference allele (what is in the REF field) in VCF file. For InDel, the allele in "REF" field in this file may be different from (usually shorter than) the "REF" in VCF file.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(85) Ori_ALT: </b>The alternative allele(s) (what is in the ALT field) in VCF file. In this file, the allele in "ALT" field corresponds to one alelle in "Ori_ALT" field; for InDel, the allele in "ALT" field may be different from (usually shorter than) the corresponding allele in the "Ori_ALT" field.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(86) shared_hom: </b>The number of samples with homozygous mutations at the current site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(87) shared_het: </b>The number of samples with a heterozygous mutation at the current site.<br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The sixth item is gene function and pathway annotation: These annotations are for genes containing this variation.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(88) OMIM: </b>Annotation from Online Mendelian Inheritance in Man (OMIM).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(89) GWAS_Pubmed_pValue: </b>Annotation from the NHGRI-EBI GWAS Catalog. The value is like 'pubmedID(p-value);pubmedID(p-value)'. 'pubmedID' is PubMed ID of publication of the study which reported the association between the variation and disease. 'p-value' is the corresponding p-value in the publication.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(90) HGMD_Disease_ID: </b>Annotation from the Human Gene Mutation Database (HGMD®). The value is like 'ID_HGMD(Disease_name);ID_HGMD(Disease_name)'. ID_HGMD is HGMD internal identifier. Disease_name is the name for the disease or condition associated with the mutation.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(91) HGMD_mutation_URL: </b> Annotation from the Human Gene Mutation Database (HGMD®). The value is infomation about this variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(92) PubMedID: </b>Given PubMed documents associated with the mutations or gene number, which from the HGMD and OMIM database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(93-95) GO_BP, GO_CC, GO_MF:</b>Annotation from Gene Ontology. BP is Biological Process; CC is cellular component; MF is molecular function.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(96) KEGG_PATHWAY: </b>Annotation from KEGG PATHWAY Database.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(97) PID_PATHWAY: </b>Annotation from PID (Pathway Interaction Database).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(98) BIOCARTA_PATHWAY: </b>Annotation from BioCarta.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(99) REACTOME_PATHWAY: </b>Annotation from Reactome Pathway Database.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The seventh item is gene tissue-specific expression. This section presents the Human Protein Atlas (HPA) database recorded protein expression of protein-encoding genes in 44 tissues of normal human and detection results of RNA transcription levels in 32 tissues, as well as links to corresponding visual results on HPA website.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(100) Expression_summary: </b>An overview of protein expression, combining immunohistochemical results, RNA results and subcellular distribution results, to describe the distribution of gene expression in different tissues and at the subcellular level.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(101) RNA tissue specificity: </b>Elevated expression includes three subcategory types of elevated expression:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enriched: At least four-fold higher mRNA level in a particular tissue compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Group enriched: At least four-fold higher average mRNA level in a group of 2-5 tissues compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enhanced: At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In addition, there is another form of low tissue specificity: Low expression in various tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(102) RNA tissue distribution: </b>Visualization of gene expression in tissues. All elevated genes are categorized as:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in single: Detected in a single tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in some: Detected in more than one but less than one third of tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in many: Detected in at least a third but not all tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in all: Detected in all tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(103) Subcellular_Location: </b>subcellular distribution positions of genes, such as "nucleus", "mitochondria", etc.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(104) RNA tissue specific nTPM: </b>Consensus transcript expression levels summarized per gene in 50 tissues based on transcriptomics data from HPA and GTEx. The consensus normalized expression ("nTPM") value is calculated as the maximum nTPM value for each gene in the two data sources. If the "RNA tissue specificity" is "Low tissue specificity", this column is empty.<br />



		
						</ul>
					</div>
	


			<p class="paragraph"><a href="../../result/Mutation/SNP/Annotation" title="Click to open" target="_blank">Result:03.Result/result/Mutation/SNP/Annotation</a></p>
				</div>
				

			</div>

				<!-------------------------------------------- 参考文献 ------------------------------------------->

			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="References"></a>
					<hr />
				</p><br />

				<h2>5 Appendix</h2>
				<a name="Appendix"></a>
				<br />
				<h3>5.1 Softwares</h3>

				<p class="center">
				<p class="name_fig">Main softwares used in the analysis </p>

				<div id="tb">
					<table class="gy">

						<tr>
							<th>Analytical content</th>
							<th>Software</th>
							<th>Version</th>
							<th>Parameter</th>
							<th>Remarks</th>
						</tr>
                        
						<tr>
							<td style="text-align:center">Quality Control</td>
							<td>fastp</td>
							<td>0.23.1</td>
							<td>Default</td>
							<td>Quality Control</td>
						</tr>
						
						<tr>
							<td rowspan=3 style="text-align:center">Alignment</td>
							<td>BWA</td>
							<td>v0.7.17</td>
							<td>Default</td>
							<td>Reads alignment</td>
						</tr>
						
						<tr>
							<td>Sambamba</td>
							<td>v1.0.0</td>
							<td>Default</td>
							<td>Sort the bam file and mark the duplicated reads</td>
						</tr>
						<tr>
							<td>Picard</td>
							<td>v2.18.9</td>
							<td>ASSUME_SORTED=true,CREATE_INDEX=true</td>
							<td>Merge all bam files</td>
						</tr>

						<!--<tr><td>GATK</td><td>local realignment of reads around indels; base quality score recalibration</td><td>v3.1</td></tr>-->


						
							
							<tr>
								<td style="text-align:center">SNPs/InDels detection</td>
								<td>GATK</td>
								<td>v4.3.0</td>
								<td>Default</td>
								<td>Detect and filter SNPs/InDels</td>
							</tr>
							

						

						
						
					
						

						
						<tr>
							<td style="text-align:center">SVs detection</td>
							<td>DELLY</td>
							<td>v1.1.5</td>
							<td>Default</td>
							<td>Detect SVs</td>
						</tr>
						

						
                        

						

						
						<tr>
							<td style="text-align:center">CNVs detection</td>
							<td>Control-FREEC</td>
							<td>v11.4</td>
							<td>Default</td>
							<td>Detect CNVs</td>
						</tr>
						

						
						

					</table>
				</div>
				</p><br />


				<h3>5.2 Method</h3>

				<p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
				
				
				<p class="paragraph">File catalog: <a href="src/method/WGS_method.pdf" title="Click to open"
                                        target="_blank">Method</a></p><br />
				
				

				<h3>5.3 Result File Decompression Method and Format Description</h3>

				<p class="center">
				<div id="tb">
					<table class="gy">

						<tr>
							<th>File Type</th>
							<th>File Description</th>
							<th>Way to Open</th>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.tar files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: tar -xvf *.tar</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software: WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.gz files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: gzip -d *.gz</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software:WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.zip files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: unzip *.zip</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software:WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>file.fq/fastq</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: less *.fq.gz or more</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Software:Editplus/Notepad++ <I>et al.</I></td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>file.bam</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: samtools view/tview *.bam or more</td>
						</tr>
					</table>
				</div>
			</div>

			<div id="page">
			<br />
			<br />

				<h2>6 References</h2>


				<p class="ref">Abecasis GR, Auton A, <I>et al.</I> An integrated map of genetic variation from 1,092 human genomes.
					Nature. 2012;491(7422):56-65. doi:10.1038/nature11632 (1000 Genomes)</br></br>

					Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using
					PolyPhen-2. Curr Protoc Hum Genet. 2013;Chapter 7:Unit7.20. doi:10.1002/0471142905.hg0720s76
					(PolyPhen)</br></br>

					Boeva V, Popova T, Bleakley K, <I>et al.</I> Control-FREEC: a tool for assessing copy number and allelic content
					using next-generation sequencing data. Bioinformatics. 2012;28(3):423-425.
					doi:10.1093/bioinformatics/btr670 (Control-FREEC)</br></br>

					Buniello A, MacArthur JAL, Cerezo M, <I>et al.</I> The NHGRI-EBI GWAS Catalog of published genome-wide
					association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res.
					2019;47(D1):D1005-D1012. doi:10.1093/nar/gky1120 (GWAS Catalog)</br></br>

					Chun S, Fay JC. Identification of deleterious mutations within three human genomes. Genome Res.
					2009;19(9):1553-1561. doi:10.1101/gr.092619.109 (LRT)</br></br>

					Cibulskis K, Lawrence MS, Carter SL, <I>et al.</I> Sensitive detection of somatic point mutations in impure and
					heterogeneous cancer samples. Nat Biotechnol. 2013;31(3):213-219. doi:10.1038/nbt.2514 (muTect)</br></br>


					DePristo MA, Banks E, Poplin R, <I>et al.</I> A framework for variation discovery and genotyping using
					next-generation DNA sequencing data. Nat Genet. 2011;43(5):491-498. doi:10.1038/ng.806 (GATK)</br></br>

					Frankish A, Diekhans M, Jungreis I, <I>et al.</I> GENCODE 2021. Nucleic Acids Res. 2021;49(D1):D916-D923.
					doi:10.1093/nar/gkaa1087 (GENCODE)</br></br>

					Garber M, Guttman M, Clamp M, Zody MC, Friedman N, Xie X. Identifying novel constrained elements by
					exploiting biased substitution patterns. Bioinformatics. 2009;25(12):i54-i62.
					doi:10.1093/bioinformatics/btp190 (SiPhy)</br> </br>

					Hamosh A, Scott AF, Amberger JS, Bocchini CA, McKusick VA. Online Mendelian Inheritance in Man (OMIM), a
					knowledgebase of human genes and genetic disorders. Nucleic Acids Res. 2005;33(Database issue):D514-D517.
					doi:10.1093/nar/gki033 (OMIM)</br></br>

					Harris MA, Clark J, Ireland A, <I>et al.</I> The Gene Ontology (GO) database and informatics resource. Nucleic
					Acids Res. 2004;32(Database issue):D258-D261. doi:10.1093/nar/gkh036 (GO)</br></br>

					Huber CD, Kim BY, Lohmueller KE. Population genetic models of GERP scores suggest pervasive turnover of
					constrained sites across mammalian evolution. PLoS Genet. 2020;16(5):e1008827. Published 2020 May 29.
					doi:10.1371/journal.pgen.1008827 (GERP)</br></br>

					Jassal B, Matthews L, Viteri G, <I>et al.</I> The reactome pathway knowledgebase. Nucleic Acids Res.
					2020;48(D1):D498-D503. doi:10.1093/nar/gkz1031 (Reactome)</br></br>

					Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27-30.
					doi:10.1093/nar/28.1.27 (KEGG PATHWAY)</br></br>

					Kent WJ, Sugnet CW, Furey TS, <I>et al.</I> The human genome browser at UCSC. Genome Res. 2002;12(6):996-1006.
					doi:10.1101/gr.229102 (UCSC)</br></br>

					Kobayashi Y, Yang S, Nykamp K, Garcia J, Lincoln SE, Topper SE. Pathogenic variant burden in the ExAC
					database: an empirical approach to evaluating population data for clinical variant interpretation. Genome
					Med. 2017;9(1):13. Published 2017 Feb 6. doi:10.1186/s13073-017-0403-7 (ExAc)</br></br>

					Li H, Durbin R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics.
					2010;26(5):589-595. doi:10.1093/bioinformatics/btp698 (BWA_MEM)</br></br>

					Ng PC, Henikoff S. SIFT: Predicting amino acid changes that affect protein function. Nucleic Acids Res.
					2003;31(13):3812-3814. doi:10.1093/nar/gkg509 (SIFT)</br></br>

					O'Leary NA, Wright MW, Brister JR, <I>et al.</I> Reference sequence (RefSeq) database at NCBI: current status,
					taxonomic expansion, and functional annotation. Nucleic Acids Res. 2016;44(D1):D733-D745.
					doi:10.1093/nar/gkv1189 (RefSeq)</br></br>

					Pio MG, Siffo S, Scheps KG, <I>et al.</I> Curating the gnomAD database: Report of novel variants in the
					thyrogobulin gene using in silico bioinformatics algorithms. Mol Cell Endocrinol. 2021;534:111359.
					doi:10.1016/j.mce.2021.111359 (gnomAD)</br></br>

					Pollard KS, Hubisz MJ, Rosenbloom KR, Siepel A. Detection of nonneutral substitution rates on mammalian
					phylogenies. Genome Res. 2010;20(1):110-121. doi:10.1101/gr.097857.109 (phyloP)</br></br>

					Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. CADD: predicting the deleteriousness of variants
					throughout the human genome. Nucleic Acids Res. 2019;47(D1):D886-D894. doi:10.1093/nar/gky1016
					(CADD)</br></br>

					Reva B, Antipin Y, Sander C. Predicting the functional impact of protein mutations: application to cancer
					genomics. Nucleic Acids Res. 2011;39(17):e118. doi:10.1093/nar/gkr407 (MutationAssessor)</br></br>

					Schaefer CF, Anthony K, Krupa S, <I>et al.</I> PID: the Pathway Interaction Database. Nucleic Acids Res.
					2009;37(Database issue):D674-D679. doi:10.1093/nar/gkn653 (PID)</br></br>

					Rausch T, Zichner T, Schlattl A, Stutz AM, Benes V, Korbel JO. DELLY: structural variant discovery by
					integrated paired-end and split-read analysis. Bioinformatics. 2012;28(18):i333-i339.
					doi:10.1093/bioinformatics/bts378 (DELLY)</br></br>


					Sherry ST, Ward MH, Kholodov M, <I>et al.</I> dbSNP: the NCBI database of genetic variation. Nucleic Acids Res.
					2001;29(1):308-311. doi:10.1093/nar/29.1.308 (dbSNP)</br></br>

					Shihab HA, Gough J, Cooper DN, <I>et al.</I> Predicting the functional, molecular, and phenotypic consequences
					of amino acid substitutions using hidden Markov models. Hum Mutat. 2013;34(1):57-65.
					doi:10.1002/humu.22225 (FATHMM)</br></br>

					Steinhaus R, Proft S, Schuelke M, Cooper DN, Schwarz JM, Seelow D. MutationTaster2021. Nucleic Acids Res.
					2021;49(W1):W446-W451. doi:10.1093/nar/gkab266 (MutationTaster)</br></br>

					Stenson PD, Mort M, Ball EV, <I>et al.</I> The Human Gene Mutation Database (HGMD): optimizing its use in a
					clinical diagnostic or research setting. Hum Genet. 2020;139(10):1197-1207.
					doi:10.1007/s00439-020-02199-3 (HGMD)</br></br>

					Tarasov A, Vilella AJ, Cuppen E, Nijman IJ, Prins P. Sambamba: fast processing of NGS alignment formats.
					Bioinformatics. 2015;31(12):2032-2034. doi:10.1093/bioinformatics/btv098 (Sambamba)</br></br>

					Tate JG, Bamford S, Jubb HC, <I>et al.</I> COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids
					Res. 2019;47(D1):D941-D947. doi:10.1093/nar/gky1015 (COSMIC)</br></br>

					Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput
					sequencing data. Nucleic Acids Res. 2010;38(16):e164. doi:10.1093/nar/gkq603 (ANNOVAR)</br></br>

				</p>
			</div>

			<!-- shan diao 3 ge div-->
		</div>
</body>

</html>
