<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >

<html>

<head>

<TITLE>Reduced Representation Bisulfite Sequencing Report</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />



<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>

<script type="text/javascript">

        $(function () {

                var options = {

                        selectors: "h2,h3,h4,h5",

                        theme: "bootstrap",

                        scrollTo: "60"

                };

                var toc = $("#toc").tocify(options).data("toc-tocify");

        });

</script>





<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");
if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");


break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});
$(".list-group").each(show_default);

});


</script>

    <script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
  </script>

  <script type="text/javascript">
    /* show or hide element*/
    function hiddenQA(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Q&A";
      };
      return 0;
    }
  </script>


</head>



<body>



<div id="toc" class="tocify">

</div>





	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

			<tr height="86%">

				<td width='1%'></td>

				<td><img src="src/images/logo_slogan.png"></td>

				<td>

					<div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>

				</td>
				<td width="20%"></td>

				<td width="15%"></td>

				<td width="25%"

					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">

					<br><a href="mailto:op-rna@novogene.com" target="black"

						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">

						<a></td>

				<td width="1%"></td>

			</tr>

		</table>

	</div>

<div id="main">
<div id="page">
</br><h1>Reduced Representation Bisulfite Sequencing Report</h1></br>
<table class= 'gy' style="text-align:left">
<tbody>
<tr><td>Contract ID</td><td style="text-align:left">XXXXXXXX</td></tr>
<tr><td>Contract Name</td><td style="text-align:left">XX</td></tr>
<tr><td>Batch ID</td><td style="text-align:left">XXXXXXXX-XXX-XXXX</td></tr>
<tr><td>Reference Genome and Version</td><td style="text-align:left">species_verison</td></tr>
<tr><td>Report Time</td><td style="text-align:left">2023-02-09</td></tr>
<tr><td>Warm Tips</td><td style="text-align:left">Partial results are presented in this report, while full results will be delivered in data release.<br>Hyperlink of results in this report will be only valid in data release, after statement confirmation.</td></tr>
</tbody></table>
</p></div>

<!--------------------------------------------Introduction------------------------------------------->
<div id="page">
<h2><a id="Introduction">1 Introduction</a></h2>
<p class="paragraph">DNA methylation is a process by which methyl groups are added to DNA C5 position mediated by DNA methyltransferase (DNMT)(Goldberg et al., 2007). It results in the conversion of the cytosine to 5-methylcytosine (5mC). As an important kind of epigenetic modifications, DNA methylation is widely studied, and it works with histone modifications to play significant roles in regulation of gene expression, chromatin conformation and so on. In vertebrates, DNA methylation typically occurs at CpG dinucleotides. However, there is a large proportion of non-CG methylation (CHH, CHG, H represents A, C, T) in plants.</p>
<p class="paragraph">In general, high DNA methylation could repress gene expression, and demethylation would recover gene expression. DNA methylation participates in many cell activities, including cell differentiation, tissue-specific expression, genomic imprinting, X-chromosome inactivation. Abnormal DNA methylation can cause developmental abnormalities, tumors and other diseases. Thus DNA methylation plays a significant role in understanding gene expression, individual development and the mechanisms of diseases occurrence and development.</p>
<p class="paragraph">RRBS is an accurate, efficient and economical method for DNA methylation research. Enrichment of promoter and CpG island regions by enzymatic cleavage (Msp I), combined with Bisulfite sequencing, provides high resolution DNA methylation detection and high utilization of sequencing data. As a cost-effective methylation research method, RRBS has broad application prospects in large-scale clinical research.</p>
<p class="paragraph">From DNA samples to final data report, each step of sample QC, library preparation and sequencing may have an impact on the quality and quantity of data output, and the quality of data will directly affect the results of subsequent information analysis. Therefore, obtaining high-quality data is the premise to ensure that bioinformatics analysis is correct, comprehensive and credible. In order to ensure the accuracy and reliability of sequencing data from the beginning, Novogene strictly checks every experimental step and fundamentally ensures the output of high-quality data. Workflow is shown as follows:</p>
<p class="center">
<img class="pip2" src="src/images/workflow_RRBS.png" width=700 height=400 />
<p class="name_fig">Figure 1 Project workflow</p>
</div>


<!----------------------------Library Construction and Sequencing---------------------------------->
<div id="page">
<h2><a id="Introduction">2 Library Construction and Sequencing</a></h2>
<h3 id="id_11"><a id="part1.1">2.1 Sample Quality Control</a></h3>
<p class="paragraph">Please refer to Novogene’s sample QC report.</p>

<h3 id="id_12"><a id="part1.2">2.2 Library Construction, Quality Control and Sequencing</a></h3>
<p class="paragraph">Firstly, the genomic DNA spiked with lambda DNA were digested using methylation-insensitive restriction enzyme MspI, followed by end repair, and dA-tailing. The adapter ligation was performed by using adapters with all cytosines being methylated. After size selection, DNA fragments were bisulfite treated. After the bisulfite conversion, unmethylated cytosines were converted to U (to T after PCR amplification), while methylated cytosines remained unchanged. The DNA library was ready after PCR amplification. The experimental procedures of RRBS library preparation are shown in Figure 2.</p>
<p class="center">
<img class="pip2" src="src/images/library_pipeline_RRBS.png" width=150 height=300 />
<p class="name_fig">Figure 2 Library construction workflow</p>
<p class="paragraph">The library was checked with Qubit and real-time PCR for quantification and bioanalyzer for size distribution detection. Quantified libraries will be pooled and sequenced on Illumina platforms, according to effective library concentration and data amount required.</p>
</p>
</div>
<br />

<!---------------------------------Bioinformatic Analysis Workflow---------------------------------->
<div id="page">


<h2 id= "id_order_analyses"><a id="Bioinformatic Analysis Workflow">3 Bioinformatics Analysis Pipeline</a></h2>
<p class="paragraph">The bioinformatic analysis workflow of raw sequencing data is shown as follows：</p>
</br>
<p class="center">
<img width="800"  src="src/images/analysis_pipeline.png" />
<p class="name_fig">Figure 3 Bioinformatic analysis workflow</p>
<p class="paragraph">Method details involved in the project are available at <a href="./src/Method-RRBS.pdf" style="text-decoration: none" title="点击打开" target="_blank">method</a></p>
</p>
</div>


<!-----------------   III. Results------ ---------->

<div id="page">
<h2 id="Results" ><a id="Results" >4 Analysis Results</a></h2>

<!---------------1. Raw Data Quality Assessment -------------------------------------->
<!----------------------------------------- FastQC of raw reads-------------------------------------->
<h3 id="4.1 Data Quality Control"><a id ="4.1 Data Quality Control">4.1 Data Quality Control</a></h3>
<h4>4.1.1 Raw Data</h4>
<p class="paragraph">The original images obtained from high-throughput sequencing platform(eg.illumina) are transformed into sequenced reads by CASAVA base calling, we call these reads as raw data or raw reads which are saved in fastq format. It contains reads information and the corresponding sequencing quality information. These files contain reads information and the corresponding sequencing quality information. Each read has four lines described in details as follows:</p>
<p class="seq">
@HWI-ST1276:71:C1162ACXX:1:1101:1208:2458 1:N:0:CGATGT&nbsp;&nbsp;&nbsp;<br />
NAAGAACACGTTCGGTCACCTCAGCACACTTGTGAATGTCATGGGATCCAT<br />
+<br />
#55???BBBBB?BA@DEEFFCFFHHFFCFFHHHHHHHFAE0ECFFD/AEHH<br />
</p>
<table class="gy">
	<tr>

			<th>Identifier</th>

			<th>Meaning</th>


</tr>
	<tr>

			<td>HWI-ST1276</td>

			<td>Instrument – unique identifier of the sequencer</td>
</tr>
	<tr>
<td>71</td>
<td>Run number – Run number on instrument</td>
</tr>
	<tr>
<td>C1162ACXX</td>
<td>Flowcell ID - ID of flowcell</td>
</tr>
	<tr>
<td>1</td>
<td>LaneNumber - positive integer</td>
</tr>
	<tr>
<td>1101</td>
<td>TileNumber - positive integer</td>
</tr>
	<tr>
<td>1208</td>
<td>X - x coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>2458</td>
<td>Y - y coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>1</td>
<td>ReadNumber - 1 for single reads, 1 or 2 for paired ends</td>
</tr>
	<tr>
<td>N</td>
<td>whether it is filtered - NB: Y if the read is filtered out, not in the delivered fastq files, otherwise N</td>
</tr>
	<tr>
<td>0</td>
<td>Control number - 0 when none of the control bits are on. Otherwise it is an even number</td>
</tr>
	<tr>
<td>CGATGT</td>
<td>Illumina index sequences</td>
</tr>
</table>
<p class="paragraph">Line 1 begins with a '@' character and is followed by the Illumina Sequence Identifiers and an optional description. 
</p>
<p class="paragraph">
Line 2 is the raw sequence read.</p>
<p class="paragraph">
Line 3 begins with a '+' character and is optionally followed by the same sequence identifiers and descriptions as in Line 1.</p>
<p class="paragraph">
Line 4 encodes the quality values for the sequence in Line 2 and must contain the same number of characters as the bases in the sequence.</p>
<p class="paragraph">Sequencing process itself has the possibility of machine errors. Sequencing error rate distribution can reflect the quality of sequencing data. The quality value of each base in sequence information is stored in the FASTQ file. Quality value of the read represented by Qphred and the “e” represents the sequence error rate, Qphred=-10log<sub>10</sub><sup>(e)</sup>. The relationship between sequencing error rate (e) and sequencing base quality value (Qphred) is as following(Base Quality and Phred score relationship with the Illumina CASAVA v1.8 software).</p>
<table class="gy"  >
<tr><th>Phred Score</th><th>Error Rate</th><th>Correct Rate</th><th>Q-score</th></tr>
<tr><td>10</td><td>1/10</td><td>90%</td><td>Q10</td></tr>
<tr><td>20</td><td>1/100</td><td>99%</td><td>Q20</td></tr>
<tr><td>30</td><td>1/1000</td><td>99.9%</td><td>Q30</td></tr>
<tr><td>40</td><td>1/10000</td><td>99.99%</td><td>Q40</td></tr>
</table>
<p class="paragraph">(1) Due to the gradual consumption of reagents during sequencing, the sequencing error rate increases with the length of reads, which is a common feature of the Illumina high-throughput sequencing platform.</p>
<p class="paragraph">(2) For the normal methylation library, the sequencing of read1 and read2 will exhibit a directional characteristic: T content is higher in read 1, and A content is higher in read 2.</p>

<h4 id="FastQC（Raw_reads）"><a id="FastQC（Raw_reads）1">4.1.2 FastQC of Raw_reads</a></h4>
<p class="paragraph" >The massive raw data high-through sequencing technology must be qualified to guarantee the downstream bioinformatics analysis. Fastqc is used to generate a QC report to evaluate the raw data quality.  Novogene uses FastQC to get a basic statistics of the raw reads, shown in the follows:</p>

<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure1_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_1_pane_A1_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A1_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A1_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_A1_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A1_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A1_2.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_A2_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A2_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A2_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_A2_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A2_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A2_2.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_A3_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A3_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A3_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_A3_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_A3_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_A3_2.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B1_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B1_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B1_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B1_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B1_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B1_2.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B2_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B2_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B2_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B2_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B2_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B2_2.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B3_1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B3_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B3_1.raw.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_B3_2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_B3_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/1.FastQC_raw/FastQC_B3_2.raw.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure1_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A1_1"
data-toggle="list" href="#gallery_figure1_1_pane_A1_1" role="tab" aria-controls="A1_1">A1_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A1_2"
data-toggle="list" href="#gallery_figure1_1_pane_A1_2" role="tab" aria-controls="A1_2">A1_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A2_1"
data-toggle="list" href="#gallery_figure1_1_pane_A2_1" role="tab" aria-controls="A2_1">A2_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A2_2"
data-toggle="list" href="#gallery_figure1_1_pane_A2_2" role="tab" aria-controls="A2_2">A2_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A3_1"
data-toggle="list" href="#gallery_figure1_1_pane_A3_1" role="tab" aria-controls="A3_1">A3_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_A3_2"
data-toggle="list" href="#gallery_figure1_1_pane_A3_2" role="tab" aria-controls="A3_2">A3_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B1_1"
data-toggle="list" href="#gallery_figure1_1_pane_B1_1" role="tab" aria-controls="B1_1">B1_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B1_2"
data-toggle="list" href="#gallery_figure1_1_pane_B1_2" role="tab" aria-controls="B1_2">B1_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B2_1"
data-toggle="list" href="#gallery_figure1_1_pane_B2_1" role="tab" aria-controls="B2_1">B2_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B2_2"
data-toggle="list" href="#gallery_figure1_1_pane_B2_2" role="tab" aria-controls="B2_2">B2_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B3_1"
data-toggle="list" href="#gallery_figure1_1_pane_B3_1" role="tab" aria-controls="B3_1">B3_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_B3_2"
data-toggle="list" href="#gallery_figure1_1_pane_B3_2" role="tab" aria-controls="B3_2">B3_2</a>

</div></div></div>

<p class="name_fig"> Figure 4.1 FastQC of raw reads</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Top-left: horizontal axis represents position in read (bp), vertical axis represents quality scores. Different colors represent different quality range;</br>
(2) Top-middle: horizontal axis represents position in read (bp), vertical axis represents base percentage; </br>
(3) Top-right: the distribution of read length;</br>
(4) Bottom-left: GC distribution over all sequences. Red line is the actual GC content distribution, blue line is the theoretical distribution; </br>
(5) Bottom-middle: N-ratio across all bases; </br>
(6) Bottom-right: x-axis is the sequence duplication level. y-axis is the percent of the deduplicated reads.</br>
</ul>
</div>
</p>


<!----------------------------------------- 质控 -------------------------------------->
<h4 id="Raw Data Trimming"><a id="Raw Data Trimming">4.1.3 Trimming of raw data</a></h4>
<p class="paragraph" >To obtain the qualified clean data, we trim the raw reads to filter out the contaminated adapter sequence and low-quality reads. Fastp is used for data trimming and the selection of trimming steps and their associated parameters are shown in the follows (Bolger et al., 2014):</p>
<p class="paragraph2">(1) SLIDINGWINDOW:4:15: It perform a 4-base sliding window trimming, cutting once the average quality within the window falls below a threshold of 15;</p>
<p class="paragraph2">(2) LEADING:3 and TRAILING:3: Cut bases off the start and end of a read, if below a threshold quality of 3 or containing N；</p>
<p class="paragraph2">(3) ILLUMINACLIP:2:30:10: Cut adapter and other illumina-specific sequences from the read. Modes: 1. a total 10-base alignment to adapter sequence with 2 mismatches allowed. 2.Palindrome Clip cut the non-overlapped region when the alignment score of overlap part of the read pairs rises above than 30;</p>
<p class="paragraph2">(4) MINLEN:36: The minimum read length to be kept is specified as 36 nt;</p>
<p class="paragraph2">(5) The unpaired reads after trimming is discarded;</p>
<p class="paragraph2">(6) HEADCROP:10: For single cell methylation and low initial methylation libraries</p>

<p class="name_table" align="center">Table 4.1 Raw data summary</p>
<p class="center">
<div style="overflow-x: scroll"> 	
<table class="gy">
<tr><th>LibraryID</th><th>Sample name</th><th>Raw_reads</th><th>Raw_bases(G)</th><th>clean_reads</th><th>clean_bases(G)</th><th>Clean_ratio(%)</th><th>Q20(%)</th><th>Q30(%)</th><th>GC(%)</th><th>BS conversion rate(%)</th></tr>

<tr><td>FDBS220485026-1a</td><td>A1</td><td>41118007</td><td>12.34</td><td>40181630</td><td>10.77</td><td>87.28</td><td>96.4</td><td>89.75</td><td>29.02</td><td>99.586</td></tr>

<tr><td>FDBS220485027-1a</td><td>A2</td><td>49703411</td><td>14.91</td><td>48565058</td><td>12.97</td><td>86.99</td><td>96.39</td><td>89.78</td><td>29.06</td><td>99.562</td></tr>

<tr><td>FDBS220485028-1a</td><td>A3</td><td>34835862</td><td>10.45</td><td>34084764</td><td>9.1</td><td>87.08</td><td>96.84</td><td>90.84</td><td>29.12</td><td>99.541</td></tr>

<tr><td>FDBS220485030-1a</td><td>B1</td><td>37255966</td><td>11.18</td><td>36427059</td><td>9.69</td><td>86.67</td><td>96.84</td><td>90.82</td><td>29.39</td><td>99.54</td></tr>

<tr><td>FDBS220485031-1a</td><td>B2</td><td>46115122</td><td>13.83</td><td>45195612</td><td>12.06</td><td>87.2</td><td>96.7</td><td>90.36</td><td>28.63</td><td>99.546</td></tr>

<tr><td>FDBS220485032-1a</td><td>B3</td><td>42257020</td><td>12.68</td><td>41293781</td><td>10.95</td><td>86.36</td><td>96.92</td><td>91.06</td><td>29.22</td><td>99.568</td></tr>

</table>
</div>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment" display="none">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) LibraryID: name of library<br />
(2) Sample name: sample name<br />
(3) Raw Reads: the read counts of raw data<br />
(4) Raw Bases(G): read counts of raw data multiplied by read length, saved in G unit<br />
(5) Clean reads: the sequencing read counts after trimming filtering<br />
(6) Clean bases(G): clean read counts multiplied by read length, saved in G unit<br />
(7) Clean ratio(%): the proportion of clean bases in raw bases<br />
(8) Clean_Q20(%): the proportion of bases with quality score more than 20 <br />
(9) Clean_Q30(%): the proportion of bases with quality score more than 20<br />
(10) GC Content(%): the proportion of G and C in all the bases<br />
(11) BS conversion rate(%): the Bisulfite conversion rate for C converting to T<br />	
</ul>
</div>


<!----------------------------------------- FastQC of clean reads-------------------------------------->

</p><br />

<h4 id="id_order_qc3"><a id="part3.1.3">4.1.4 FastQC of Clean_reads</a></h4>
<p class="paragraph" >
FastQC is also performed on the clean data obtained from trimming, the results as shown:
</p>

<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure1_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_2_pane_A1_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A1_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A1_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_A1_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A1_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A1_2.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_A2_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A2_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A2_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_A2_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A2_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A2_2.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_A3_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A3_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A3_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_A3_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_A3_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_A3_2.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B1_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B1_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B1_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B1_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B1_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B1_2.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B2_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B2_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B2_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B2_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B2_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B2_2.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B3_1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B3_1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B3_1.clean.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_B3_2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_B3_2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/2.FastQC_clean/FastQC_B3_2.clean.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure1_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A1_1"
data-toggle="list" href="#gallery_figure1_2_pane_A1_1" role="tab" aria-controls="A1_1">A1_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A1_2"
data-toggle="list" href="#gallery_figure1_2_pane_A1_2" role="tab" aria-controls="A1_2">A1_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A2_1"
data-toggle="list" href="#gallery_figure1_2_pane_A2_1" role="tab" aria-controls="A2_1">A2_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A2_2"
data-toggle="list" href="#gallery_figure1_2_pane_A2_2" role="tab" aria-controls="A2_2">A2_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A3_1"
data-toggle="list" href="#gallery_figure1_2_pane_A3_1" role="tab" aria-controls="A3_1">A3_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_A3_2"
data-toggle="list" href="#gallery_figure1_2_pane_A3_2" role="tab" aria-controls="A3_2">A3_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B1_1"
data-toggle="list" href="#gallery_figure1_2_pane_B1_1" role="tab" aria-controls="B1_1">B1_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B1_2"
data-toggle="list" href="#gallery_figure1_2_pane_B1_2" role="tab" aria-controls="B1_2">B1_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B2_1"
data-toggle="list" href="#gallery_figure1_2_pane_B2_1" role="tab" aria-controls="B2_1">B2_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B2_2"
data-toggle="list" href="#gallery_figure1_2_pane_B2_2" role="tab" aria-controls="B2_2">B2_2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B3_1"
data-toggle="list" href="#gallery_figure1_2_pane_B3_1" role="tab" aria-controls="B3_1">B3_1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_B3_2"
data-toggle="list" href="#gallery_figure1_2_pane_B3_2" role="tab" aria-controls="B3_2">B3_2</a>

</div></div></div>

<p class="name_fig">Figure 4.2 FastQC of clean reads</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Top-left: horizontal axis represents position in read (bp), vertical axis represents quality scores. Different colors represent different quality range;</br>
(2) Top-middle: horizontal axis represents position in read (bp), vertical axis represents base percentage; </br>
(3) Top-right: the distribution of read length;</br>
(4) Bottom-left: GC distribution over all sequences. Red line is the actual GC content distribution, blue line is the theoretical distribution; </br>
(5) Bottom-middle: N-ratio across all bases; </br>
(6) Bottom-right: x-axis is the sequence duplication level. y-axis is the percent of the deduplicated reads.</br>
</ul>
</div>
</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/1.QualityControl" title="点击打开" target="_blank">QualityControl</a></p>



<!---------------2. Alignment to reference genome ------------>


</p><br />
<h3 id="id_order_bismark"><a id="Alignment to reference genome">4.2 Alignment to reference genome</a></h3>
<p class="paragraph">Bismark software (Krueger et al., 2011) is used to perform alignments of bisulfite-treated reads to a reference genome(Langmead et al., 2012). C is transformed to T and G is transformed to A (reverse complementary) in sequence reads and reference genome via Bismark, and then these sequencing reads are aligned to similarly converted versions of the genome separately.</p>
<p class="paragraph">The process of Bismark performing alignment is as follows:</p>
<p class="paragraph">(1) Perform C-to-T and G-to-A conversions for both sequencing reads and a reference genome;</p>
<p class="paragraph">(2) Align converted reads to the converted genome;</p>
<p class="paragraph">(3) Choose the best alignment from the parallel four alignment processes and consider it as the final result.</p>

</p>

<p class="center">
<img class="small3" src="src/images/Bismark.png" />
<p class="name_fig">Figure 4.3 Bismark Alignment Strategy</p>
<p class="premark">Genomic fragment: shown in italics; Sequence after bisulfite treatment: shown in Roman letters</p>
</p></br>

<!---------------------------------------Overview of Mapping Status---------------------------------------------->

</p><br />
<h4 id="id_order_bismark1"><a id="part3.2.1">4.2.1 Overview of Mapping Status</a></h4>
<p class="paragraph">The summary of mapping result as follows:</p>
<p class="name_table" align="center">Table 4.2 Overview of mapping status</p>
<p class="center">
<table class="gy">
<tr><th>Samples</th><th>Total reads</th><th>Mapped reads</th><th>Mapping rate(%)</th><th>Mspl digestion efficiency</th></tr>

<tr><td>A1</td><td>40181630</td><td>26724802</td><td>66.51</td><td>99.586</td></tr>

<tr><td>A2</td><td>48565058</td><td>32145211</td><td>66.19</td><td>99.562</td></tr>

<tr><td>A3</td><td>34084764</td><td>23041300</td><td>67.60</td><td>99.541</td></tr>

<tr><td>B1</td><td>36427059</td><td>24661118</td><td>67.70</td><td>99.540</td></tr>

<tr><td>B2</td><td>45195612</td><td>20048773</td><td>44.36</td><td>99.546</td></tr>

<tr><td>B3</td><td>41293781</td><td>28323404</td><td>68.59</td><td>99.568</td></tr>

</table>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Samples: Sample name<br />
(2) Total reads: The filtered clean reads used for mapping<br />
(3) Mapped reads: The uniquely mapped reads<br />
(4) Mapping rate (%): The precentage of uniquely mapped reads in the filtered clean reads used for mapping <br />
(5) Mspl digestion efficiency: The precentage of Mspl digestion <br />
</ul>
</div>
</br>


<!-------------------------------------- Read Coverage and Sequencing Depth------------------------------------------------>
</br>
<div id="page">
<h4 id="id_order_bismark2"><a id="part3.2.2">4.2.2 Read Coverage and Sequencing Depth</a></h4>
<h5>4.2.2.1 Distribution of coverd genomic feature</h5>
<br />
<p class="paragraph">The summary of reads-coverd genomic feature as follows:</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_1_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A1.features.coverage.count.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A2.features.coverage.count.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A3.features.coverage.count.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B1.features.coverage.count.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B2.features.coverage.count.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B3.features.coverage.count.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_A1"
data-toggle="list" href="#gallery_figure2_1_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_A2"
data-toggle="list" href="#gallery_figure2_1_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_A3"
data-toggle="list" href="#gallery_figure2_1_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_B1"
data-toggle="list" href="#gallery_figure2_1_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_B2"
data-toggle="list" href="#gallery_figure2_1_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_B3"
data-toggle="list" href="#gallery_figure2_1_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.4 The number of reads on each genomic feature</p>
<p class="premark">The x-axis denotes the genomic feature; the y-axis denotes the number of genomic feature covered by at least 1 reads. </p>
</p>

</div>
<!---------------------------------------------------------->
<div id="page">

</p><br />
<h5>4.2.2.2 Distribution of genomic feature depth and coverage</h5>
<br />
<p class="paragraph">The summary of genomic feature depth and coverage as follows:</p>

<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_2_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A1.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A2.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A3.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B1.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B2.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B3.region_RCP.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_A1"
data-toggle="list" href="#gallery_figure2_2_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_A2"
data-toggle="list" href="#gallery_figure2_2_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_A3"
data-toggle="list" href="#gallery_figure2_2_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_B1"
data-toggle="list" href="#gallery_figure2_2_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_B2"
data-toggle="list" href="#gallery_figure2_2_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_B3"
data-toggle="list" href="#gallery_figure2_2_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.5 The distribution of reads on each genomic feature</p>
<p class="premark">The x-axis denotes the genomic feature; the left y-axis denotes the average squencing depth of each genomic feature, the right y-axis denotes the read coverage percentage of each genomic feature.</p>
</p>

</div>

<!---------------------------------------------------------->
<div id="page">

</p><br />
<h5>4.2.2.3 Distribution of genome coverage</h5>
<br />
<p class="paragraph">To fully determine the Genome-wide methylation status, a minimum 30x coverage is recommended by NIH Roadmap Epigenomics Project (<a href="http://www.roadmapepigenomics.org/protocols" target=_blank>http://www.roadmapepigenomics.org/protocols</a>). And biological replicates influence more than the coverage. So when there exist biological replicates, a minimum 20x coverage for each replicate is enough. Besides, the Cytosine coverage is also another good indicator for methylation analysis. Genome coverage and Cytosine coverage are summarized in the follows:</p>
<p class="paragraph"> Single-base read coverage (the total read amount which covers this specified base) of the genome is calculated to get the genome coverage distribution and cumulative genome coverage distribution of read coverage.</p>
<p class="name_table" align="center">Table 4.3 genome coverage summary</p>
<p class="center">
<table class="gy">
<tr><th>Samples</th><th>sites_num</th><th>sites_covgMean</th><th>sites_numCovg1</th><th>sites_numCovg5</th><th>sites_numCovg10</th></tr>

<tr><td>A1</td><td>132843642</td><td>2.60</td><td>4.87</td><td>1.56</td><td>1.33</td></tr>

<tr><td>A2</td><td>116335885</td><td>3.12</td><td>4.26</td><td>1.60</td><td>1.37</td></tr>

<tr><td>A3</td><td>115530711</td><td>2.23</td><td>4.23</td><td>1.50</td><td>1.27</td></tr>

<tr><td>B1</td><td>82344707</td><td>2.36</td><td>3.02</td><td>1.44</td><td>1.31</td></tr>

<tr><td>B2</td><td>95748897</td><td>1.95</td><td>3.51</td><td>1.49</td><td>1.34</td></tr>

<tr><td>B3</td><td>113659851</td><td>2.72</td><td>4.17</td><td>1.51</td><td>1.34</td></tr>

</table>
</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Samples: sample name<br />
(2) sites_num: the total detected genome base amount<br />
(3) sites_covgMean: the average base coverage of the genome<br />
(4) sites_numCovg1: the percentage of bases with a minimum 1x coverage of the genome<br />
(5) sites_numCovg5: the percentage of bases with a minimum 5x coverage of the genome<br />
(6) sites_numCovg10: the percentage of bases with a minimum 10x coverage of the genome<br />
</ul>
</div>
<br />
<br />
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_3_pane_distribution" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_distribution"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/all.base_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_accumulative_distribution" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_accumulative_distribution"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/all.base_accumulative.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_distribution"
data-toggle="list" href="#gallery_figure2_3_pane_distribution" role="tab" aria-controls="distribution">distribution</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_accumulative_distribution"
data-toggle="list" href="#gallery_figure2_3_pane_accumulative_distribution" role="tab" aria-controls="accumulative_distribution">accumulative_distribution</a>

</div></div></div>
<p class="name_fig">Figure 4.6 Distribution of genome coverage of all samples</p>
<p class="premark">Top: x-axis shows the coverage depth, y-axis shows the fraction of bases with the corresponding coverge depth in the genome.<br />
Bottom: x-axis shows the coverage depth, y-axis shows the accumulative fraction of bases above corresponding depth in the genome.<br />
Different colors denote different samples.</p>
<br /><br />
</p>

</div>

<!--------------------------Distribution of chromosome depth and coverage-------------------------------->
<div id="page">

</p><br />
<h5>4.2.2.4 Distribution of chromosome depth and coverage</h5>
<br />
<p class="paragraph">The read amount mapped on each chromosome is calculated to get the mean sequencing depth and coverage depth of each chromosome.</p>

<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_4_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A1.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_4_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A2.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_4_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A3.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_4_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B1.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_4_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B2.region_RCP.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_4_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure2_4_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B3.region_RCP.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_A1"
data-toggle="list" href="#gallery_figure2_4_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_A2"
data-toggle="list" href="#gallery_figure2_4_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_A3"
data-toggle="list" href="#gallery_figure2_4_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_B1"
data-toggle="list" href="#gallery_figure2_4_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_B2"
data-toggle="list" href="#gallery_figure2_4_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_4_list_B3"
data-toggle="list" href="#gallery_figure2_4_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>

<p class="name_fig">Figure 4.7 The distribution of reads on each chromosome</p>
<p class="premark">The x-axis denotes the chromosome; the left y-axis denotes the average squencing depth of each chromosome, the right y-axis denotes the read coverage percentage of each chromosome.</p>
</p>

</div>



<!-----------------------------------------Coverage depth of cytosine sites---------------------------------------------->
<div id="page">

</p><br />
<h4 id="id_order_bismark3"><a id="part3.2.3">4.2.3 Coverage depth of cytosine sites</a></h4>
<p class="paragraph">The coverage depth of Cytosine is a standard for sequencing depth. The coverage depth of each CX context(CpG, CHH, CHG, H represents A, C, T) is shown as follows.</p>
<p class="paragraph">The coverage of Cytosine in each CX context is calculated (the read amount supporting the context) to get the accumulative coverage distribution diagrams.</p>

<p class="name_table" align="center">Table 4.4 The coverage statistics of cytosine in different context</p>
<p class="center">
<div style="overflow-x: scroll"> 	
<table class="gy">
<tr><th>sample</th><th>C_covgMean</th><th>C(Mb)</th><th>CG(Mb)</th><th>CHG(Mb)</th><th>CHH(Mb)</th><th>MeanC(%)</th><th>MeanCG(%)</th><th>MeanCHG(%)</th><th>MeanCHH(%)</th></tr>

<tr><td>A1</td><td>24.7</td><td>1084.5</td><td>157.7</td><td>274.8</td><td>652.0</td><td>6.02</td><td>38.69</td><td>0.52</td><td>0.43</td></tr>

<tr><td>A2</td><td>29.6</td><td>1296.8</td><td>190.3</td><td>329.4</td><td>777.2</td><td>6.02</td><td>38.20</td><td>0.55</td><td>0.46</td></tr>

<tr><td>A3</td><td>21.0</td><td>918.9</td><td>134.7</td><td>232.6</td><td>551.6</td><td>5.98</td><td>37.89</td><td>0.56</td><td>0.47</td></tr>

<tr><td>B1</td><td>22.6</td><td>991.1</td><td>151.1</td><td>250.3</td><td>589.6</td><td>5.99</td><td>36.34</td><td>0.59</td><td>0.50</td></tr>

<tr><td>B2</td><td>18.7</td><td>818.6</td><td>120.5</td><td>207.6</td><td>490.6</td><td>5.83</td><td>36.81</td><td>0.55</td><td>0.46</td></tr>

<tr><td>B3</td><td>25.8</td><td>1132.4</td><td>169.1</td><td>286.3</td><td>677.0</td><td>5.83</td><td>36.29</td><td>0.54</td><td>0.45</td></tr>

</table>
</div>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) sample: sample name<br />
(2) C_covgMean: The mean coverage of all the cytosine sites<br />
(3) C(Mb): The base number mapped onto the genome cytosine sites<br />
(5) CG(Mb): The base number mapped onto the genome CG-context cytosine sites<br />
(6) CHG(Mb): The base number mapped onto the genome CHG-context cytosine sites<br />
(7) CHH(Mb): The base number mapped onto the genome CHH-context cytosine sites<br />
(8) MeanC: The average methylation level of all the genome cytosine sites<br />
(9) MeanCG: The average methylation level of all the genome CG-context cytosine sites<br />
(10) MeanCHG: The average methylation level of all the genome CHG-context cytosine sites<br />
(11) MeanCHH: The average methylation level of all the genome CHH-context cytosine sites<br />
</ul>
</div>
<br />




<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_5_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_5_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_5_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A1.CX_accumulative.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_5_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A2.CX_accumulative.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_5_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/A3.CX_accumulative.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_5_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B1.CX_accumulative.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_5_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B2.CX_accumulative.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_5_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure2_5_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/3.cov_Dep/B3.CX_accumulative.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_5_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_5_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_5_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_A1"
data-toggle="list" href="#gallery_figure2_5_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_A2"
data-toggle="list" href="#gallery_figure2_5_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_A3"
data-toggle="list" href="#gallery_figure2_5_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_B1"
data-toggle="list" href="#gallery_figure2_5_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_B2"
data-toggle="list" href="#gallery_figure2_5_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_5_list_B3"
data-toggle="list" href="#gallery_figure2_5_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.8 The accumulative coverage distribution of cytosines </p>
<p class="premark">The x-axis shows the coverage depth of Cytosine sites, y-axis shows the accumulative fraction of cytosine sites above corresponding coverage depth.
</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/2.Mapping" title="点击打开" target="_blank">Mapping</a></p>
<br />
<br />




<!--------- 3. Identification of Methylated Cytosines -------->

<!------------------------3.1-------------------->
<div id="page">

</p><br />
<h3 id="id_order_MCstat"><a id="Identification of Methylated Cytosines">4.3 Identification of Methylated Cytosines</a></h3>
<p class="paragraph">After the alignment, site calling, and evaluation of genomic coverage/depth and C site coverage/depth of methylation data, C-site analysis of methylation is processed as follows:</p>

<h4 id="id_order_MCstat1"><a id="part3.3.1">4.3.1. Detection of methylated sites</a></h3>
<p class="paragraph">After alignment, Bismark (Krueger et al., 2011) is used to identify methyalted sites. Firstly, PCR duplicates must be removed and then methylation state of cytosines can be identified by comparison of reads base and the reference genome base at the same position. If the read base is C, it suggests that this cytosine is methylated; otherwise, it is unmethylated. Algorithm of Bismark identifying methylation sites is as follows:</p>
<p class="center">
<img class="small3" src="src/images/Bismark_B.png" />
</p>
<p class="paragraph">The PE reads overlap portion may cause some bias in the mC site identification results (Gao et al., 2015). Hence, we select no_overlap in bismark_methylation_extractor, and remain read1 information when dealing with PE reads overlap area (Wang et al., 2014; Krueger et al., 2011).</p>
<p class="paragraph">Besides, the bisulfite conversion rate may also influence the accuracy of methylation identification. By adding Lambda genome DNA in the library as the negative control, we calculated the BS conversion rate to evaluate the reliability of the methylation level. In next generation sequencing, the sequencing depth of each site is different, which cause the different ratio of methylated and unmethylated sites. However, the probability of each methylated site should be subjected to the binomial distribution. Binomial distribution is the discrete probability distribution of the number of successes in a sequence of n independent experiments, is a widely used probability distribution of discrete random variables. In analysis process, in order to identify the true methylated sites, methylated and unmethylated counts at each site from Bismark output is tested by binomial distribution. Assuming the number of methylated cytosine is x in a certain site, where the read coverage is n and BS un-conversion rate is p, so the reliability of x methylated cytosine needs to be tested in above conditions. A set of thresholds are set in analysis process to find accurate methylated sites (Habibi et al., 2013; Gifford et al., 2013): (1) the sequencing depth is greater than or equal to five; (2 ) q-value less than or equal to 0.01.</p>
<p class="paragraph">For the methylated sites, the methylation level is calculated using the following formula: ML=mC/(mC+umC). ML represents the methylation level, and mC and umC represent the number of mythylaed and unmethylated cytosines, respectively. The results of methylation level of cytosine are shown in follows:</p>
<p class="name_table" align="center">Table 4.5 Methylation Level of Cytosines</p>
<p class="center">
<table class="gy">
<tr><th>Chromosome</th><th>Position</th><th>Strand</th><th>Methylation Level</th><th>mC count</th><th>umC count</th><th>Pvalue</th><th>Corrected pvalue</th><th>Context</th></tr>

<tr><td>1</td><td>3091100</td><td>+</td><td>1.000</td><td>6</td><td>0</td><td>4.9802e-15</td><td>8.3235e-15</td><td>CG</td></tr>

<tr><td>1</td><td>3091114</td><td>+</td><td>1.000</td><td>6</td><td>0</td><td>4.9802e-15</td><td>8.3235e-15</td><td>CG</td></tr>

<tr><td>1</td><td>3091115</td><td>-</td><td>0.833</td><td>5</td><td>1</td><td>7.2060e-12</td><td>1.1474e-11</td><td>CG</td></tr>

<tr><td>1</td><td>3091168</td><td>+</td><td>0.588</td><td>10</td><td>7</td><td>2.7511e-20</td><td>4.9324e-20</td><td>CG</td></tr>

<tr><td>1</td><td>3091169</td><td>-</td><td>0.308</td><td>4</td><td>9</td><td>2.0240e-07</td><td>3.0103e-07</td><td>CG</td></tr>

<tr><td>1</td><td>3091194</td><td>+</td><td>0.545</td><td>6</td><td>5</td><td>2.2604e-12</td><td>3.6307e-12</td><td>CG</td></tr>

</table>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Chromosome: chromosome ID<br />
(2) Position: cytosine coordinate at genome<br />
(3) Strand: strand information of the cytosine<br />
(4) Methylation level: ML=mC/(mC+umC)<br />
(5) mC count: the number of methylated cytosines<br />
(6) umC count: the number of unmethylated cytosines<br />
(7) Pvalue: p value of binomial distribution test<br />
(8) Corrected pvalue: adjusted p value<br />
(9) Context: contexts in which the cytosine is located<br />
</ul>
</div>

</p><br />
<!--------- 3.4 Mehtylation analysis of C sites -------->

<div id="page">
<h4 id="id_order_MCstat4"><a id="part3.3.4">4.3.2 Methylation analysis of C sites</a></h4>
<p class="paragraph">For the identified methylated sites, the methylation level is calculate as: ML=mC/(mC+umC), where ML is the methylation level. The distribution of methylation level in each context is shown in follows.</p>
<br />
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure3_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure3_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure3_4_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/A1.mCX_methlevel_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_4_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/A2.mCX_methlevel_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_4_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/A3.mCX_methlevel_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_4_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/B1.mCX_methlevel_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_4_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/B2.mCX_methlevel_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_4_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure3_4_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/5.stat/B3.mCX_methlevel_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure3_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure3_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure3_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_A1"
data-toggle="list" href="#gallery_figure3_4_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_A2"
data-toggle="list" href="#gallery_figure3_4_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_A3"
data-toggle="list" href="#gallery_figure3_4_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_B1"
data-toggle="list" href="#gallery_figure3_4_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_B2"
data-toggle="list" href="#gallery_figure3_4_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_B3"
data-toggle="list" href="#gallery_figure3_4_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.9 The distribution of methylation level in each context</p>
<p class="premark">The x-axis is methylation level, the y-axis is the percentage of methylated sites in all sits. Colors donates different context (count methylation level as 0% if < 5%, count as 10% if > 5% and < 15%, and so on. Count as 100% if > 95%). </p>
</p><br />
	
</div>

<!--------------------------------------Visualization-------------------------------------------->

<div id="page">
<h4 id="id_order_MCstat6"><a id="part3.3.6">4.3.3 Visualization of mapping results</a></h4>
<p class="paragraph">The bw format files containing mapping results, corresponding reference genome and gene annotation files are provided. IGV (Integrative Genomics Viewer) software is recommended for visualization of bw file (Robinson et al., 2011; Thorvaldsdóttir et al., 2013). The advantages of IGV include:</p>
<p class="paragraph">(1) display the positions of single or multiple reads in the reference genome, as well as reads distribution in annotated exons, introns or intergenic regions in adjustable scale;</p>
<p class="paragraph">(2) display reads abundance in different regions to surrogate the methylation level of different regions in adjustable scale;</p>
<p class="paragraph">(3) provide annotation information of both genes and splicing isoforms and other related annotation information;</p>
<p class="paragraph">(4) display annotation downloaded from remote server or imported from local machines or both.</p>
<p class="center">
<img class="small3" src="src/images/IGV.png" />
</p>
<p class="name_fig">Figure 4.10 IGV Visualization</p>
<br />
<p class="center">
<img class="small3" src="src/images/IGV11.png" />
</p>
<p class="name_fig">Figure 4.11 The IGV visualization can be saved as SVG format and edited using AI</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/3.MethylationStat" title="点击打开" target="_blank">MethylationStat</a></p>
</div>




<!-----------------Methylation analysis-------------------------->

<div id="page">

</p><br />
<h3 id="id_order_sample"><a id="Methylation analysis">4.4. Methylation analysis</a></h3>
<p class="paragraph">The methylation analysis for single sample includes: overall methylation density, methylation density on chromosome, methylation distribution on functional genetic region, methylation distribution on up/downstream 2kb and gene body. These analysis can be used to construct a species methylation profile. The results are in follows:</p>

<!-------- Methylation Level of Genomic Features---------------------------------------->
<div id="page">
<h4 id="id_order_sample4"><a id="part3.4.4">4.4.1 Methylation Level of Genomic Features</a></h4>
<p class="paragraph">The average methylated level in different cytosine contexts is calculated in different functional genomic regions such as promoter, exon, intron, CGI, CGI shore, repeat(Promoter region is the 2kb region above TTS, 5’UTR, Exon and Intron are obtained from the Ensembl structure annotation files), the average methylation level distribution of all samples in genomic functional regions are shown in follows:</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure4_5_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure4_5_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure4_5_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A1.mC_genomic_feature_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_5_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A2.mC_genomic_feature_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_5_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A3.mC_genomic_feature_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_5_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B1.mC_genomic_feature_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_5_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B2.mC_genomic_feature_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_5_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure4_5_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B3.mC_genomic_feature_level.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure4_5_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure4_5_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure4_5_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_A1"
data-toggle="list" href="#gallery_figure4_5_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_A2"
data-toggle="list" href="#gallery_figure4_5_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_A3"
data-toggle="list" href="#gallery_figure4_5_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_B1"
data-toggle="list" href="#gallery_figure4_5_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_B2"
data-toggle="list" href="#gallery_figure4_5_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_5_list_B3"
data-toggle="list" href="#gallery_figure4_5_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.15 Methylation Level Distribution of All samples in Genomic Features</p>
<p class="premark">The x-axis represents different genomic elements, and the y-axis represents methylation level. The various functional areas of each gene are divided into 20 bins, and average methylation level within each bin region is calculated. Different colours stand for CpG context.</p>
</div>

<div id="page">
<h4 id="id_order_sample4"><a id="part3.4.4">4.4.2 The distribution of different methylation level CpG site number in genomic frature</a></h4>
<p class="paragraph">The cytosine sites in different methylated level is calculated in different functional genomic regions such as promoter, exon, intron, CGI, CGI shore, repeat(Promoter region is the 2kb region above TTS, 5’UTR, Exon and Intron are obtained from the Ensembl structure annotation files), the cytosine sites distribution of all samples in genomic functional regions are shown in follows:</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure4_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure4_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure4_2_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A1.genomic_feature_level_count.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_2_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A2.genomic_feature_level_count.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_2_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A3.genomic_feature_level_count.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_2_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B1.genomic_feature_level_count.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_2_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B2.genomic_feature_level_count.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_2_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure4_2_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B3.genomic_feature_level_count.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure4_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure4_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure4_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_A1"
data-toggle="list" href="#gallery_figure4_2_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_A2"
data-toggle="list" href="#gallery_figure4_2_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_A3"
data-toggle="list" href="#gallery_figure4_2_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_B1"
data-toggle="list" href="#gallery_figure4_2_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_B2"
data-toggle="list" href="#gallery_figure4_2_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_2_list_B3"
data-toggle="list" href="#gallery_figure4_2_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.16 The feature number of different methylated level in each samples</p>
<p class="premark">The x-axis is methylation level, the left y-axis is each genomic feature numbers in different methylated level. The overall methylation level of each functional region of each gene is the sum of the methylation levels of cpG sites within that region.</p>
</div>

<!-------- Methylation Level of up/downstream---------------------------------------->

<div id="page">

<h4 id="id_order_sample5"><a id="part3.4.5">4.4.3 Methylation Level of up/downstream</a></h4>
<p class="paragraph">The methylation level from upstream 2kb to downstream 2kb and the genebody(from TSS to TES) of each C context is summarized. The distribution of methylation level from TSS to TES is shown as follows :</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure4_6_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure4_6_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure4_6_pane_A1" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_A1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A1.gene_updown_2K_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_6_pane_A2" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_A2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A2.gene_updown_2K_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_6_pane_A3" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_A3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/A3.gene_updown_2K_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_6_pane_B1" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_B1"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B1.gene_updown_2K_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_6_pane_B2" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_B2"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B2.gene_updown_2K_level.png" /></div>

<div class="tab-pane fade" id="gallery_figure4_6_pane_B3" 
role="tabpanel" aria-labelledby="gallery_figure4_6_list_B3"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/7.sample_met/B3.gene_updown_2K_level.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure4_6_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure4_6_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure4_6_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_A1"
data-toggle="list" href="#gallery_figure4_6_pane_A1" role="tab" aria-controls="A1">A1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_A2"
data-toggle="list" href="#gallery_figure4_6_pane_A2" role="tab" aria-controls="A2">A2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_A3"
data-toggle="list" href="#gallery_figure4_6_pane_A3" role="tab" aria-controls="A3">A3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_B1"
data-toggle="list" href="#gallery_figure4_6_pane_B1" role="tab" aria-controls="B1">B1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_B2"
data-toggle="list" href="#gallery_figure4_6_pane_B2" role="tab" aria-controls="B2">B2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure4_6_list_B3"
data-toggle="list" href="#gallery_figure4_6_pane_B3" role="tab" aria-controls="B3">B3</a>

</div></div></div>
<p class="name_fig">Figure 4.17a methylation level from upstream 2kb to downstream 2kb</p>
<p class="premark">The x-axis is each bin of upstream 2kb, genebody, and downstream 2kb. The y-axis is the methylation level.</p>
</div>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/4.sample_methylation_Analysis" title="点击打开" target="_blank">Sample Methylation Analysis</a></p>



<!--------- 5. Correlation and cluster heatmap-------------------------------------------->
<!-------6 相关性分析------------------------------------>
<div id="page">

</p><br />
<h3 id="id_order_corheatmap"><a id="Correlation and cluster heatmap">4.5. Correlation and cluster heatmap</a></h3>
<p class="paragraph">In order to ensure the reproducibility of the results, it is often necessary to design biological duplication in the experiment. NIH Roadmap Epigenomics Project suggested that two or more biological replicates should be used in the design of the methylation experiment. There are two main uses for biological duplication: one is to prove that the biological experiments involved are repeatable and not very variable, and the other is to ensure that differential genetic analysis have more reliable results.</p>

<h4 id="id_order_cor"><a id="part3.5.1">4.5.1 Correlation and PCA analysis</a></h4>
<p class="paragraph">Correlation and principal component analysis (PCA) will be conducted to ensure the reproducibility of the analytical results. Correlation analysis was performed when the sample number is greater than 2, and PCA was performed when the sample number is greater than 6. (The PCA results will not be displayed when the number of samples is less than 6)</p>
<br />
<h5>4.5.1.1 Correlation analysis</h5>
<br />
<p class="paragraph">The correlation of methylation levels between samples is an important indicator to test the reliability of the experiment and whether the sample selection is reasonable. The closer the correlation coefficient is to 1, the higher the similarity in the methylation pattern between samples.</p>
<p class="paragraph">The whole genome is split by 2kb sub-bin. We provide the methylation level for each bin and pearson correlation analysis (Smallwood et al., 2014).</p>
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure5_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_1_pane_CG_pearson_correlation" 
role="tabpanel" aria-labelledby="gallery_figure5_1_list_CG_pearson_correlation"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/8.cor_pheatmap/CG.cor_pearson.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_1_list_CG_pearson_correlation"
data-toggle="list" href="#gallery_figure5_1_pane_CG_pearson_correlation" role="tab" aria-controls="CG_pearson_correlation">CG_pearson_correlation</a>

</div></div></div>

<p class="name_fig">Figure 4.18 Correlation analysis</p>
<p class="premark">Heatmap: heatmap represents the correlation between samples. R2:the square of pearson correlation, which is calculate by CG context.</p>
</div>
<!-------PCA分析------------------------------------>

<div id="page">
<h5>4.5.1.2 PCA</h5>
<br />
<p class="paragraph">
Principal component analysis (PCA) is a statistical procedure that uses an orthogonal transformation to convert a set of observations of possibly correlated variables (entities each of which takes on various numerical values) into a set of values of linearly uncorrelated variables called principal components.</p>
<p class="paragraph">
The core idea of PCA is to reduce the dimension of the data under the premise of keeping the data as much as possible.It is conceivable that the data set is a group of points in the multi-dimensional space. When the relative spatial position of the group is kept unchanged, the data is rotated to a new coordinate system (the axes are the PCs), so that the points are new. The coordinate (projection) on the coordinate axis has the largest variance, and the axis with the largest projection variance is PC1, followed by PC2.</p>
<p class="paragraph">The result of PCA analysis using correlation analysis data is shown in follows: </p>
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure5_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_3_pane_Dendorgram_CG" 
role="tabpanel" aria-labelledby="gallery_figure5_3_list_Dendorgram_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/8.cor_pheatmap/Dendorgram_CG.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_3_pane_PCA12_CG" 
role="tabpanel" aria-labelledby="gallery_figure5_3_list_PCA12_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/8.cor_pheatmap/PCA12_CG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_Dendorgram_CG"
data-toggle="list" href="#gallery_figure5_3_pane_Dendorgram_CG" role="tab" aria-controls="Dendorgram_CG">Dendorgram_CG</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_PCA12_CG"
data-toggle="list" href="#gallery_figure5_3_pane_PCA12_CG" role="tab" aria-controls="PCA12_CG">PCA12_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.19 PCA analysis of methylation between samples </p>
<p class="premark">Hierarchical clustering plot and PCA plot for Principal component 1 and Principal component 2 for CG context</p>
</div>

<!-----------------Heatmap analysis of functional regions-------------------------->
<div id="page">

</p><br />
<h4 id="id_order_heatmap"><a id="part3.5.2">4.5.2 Heatmap analysis of functional regions</a></h4>
<p class="paragraph">The heatmap analysis of the methylation level of functional regions between samples can reflect the degree of agreement of the methylation level distribution of each sample in the gene function region. </p>
<p class="paragraph">Data of this analysis comes from Methylation Level of Genomic Features for single sample.</p>
<br />
<div class='all-img2'>
<img src=src/pictures/8.cor_pheatmap/gene_region_mCG_heatmap.png />
<br/><br/>
<p class="name_fig">Figure 4.20 Heatmap analysis of gene functional region methylation levels in different sequence environments of different samples</p>
</div>
<br/>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/4.sample_methylation_Analysis" title="点击打开" target="_blank">Sample Methylation Analysis</a></p>
</div>

<!------- Methylation level in comparison group------------------------------------>
<div id="page">

</p><br />
<h3 id="id_order_compare"><a id="Methylation level in comparison group">4.6. Methylation level in comparison group</a></h3>
<p class="paragraph">For most studies, we focus on the differential methylation analysis between samples. The methylation analysis for comparison groups includes: overall methylation density, methylation density on chromosome, methylation distribution on functional genetic region, methylation distribution on up/downstream 2kb and gene body. (combined the information if there are biological replicates (Song et al., 2013).</p>
<br />
<h4 id="id_order_compare2"><a id="part3.6.2">4.6.1 Methylation level of functional gene region</a></h4>
<p class="paragraph">The methylated level in different cytosine contexts is calculated in different functional genomic regions such as promoter, exon, intron, CGI, CGI shore, repeat (Promoter region is the 2kb region above TTS, 5’UTR, Exon and Intron are obtained from the Ensembl structure annotation files), the average methylation level distribution of comparison groups in genomic functional regions are shown in follows:</p>
<p class="paragraph">Methylation level distribution at functional genetic elements in CG context</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure6_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure6_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure6_3_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure6_3_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/9.compare_met/B_vs_A.region_mCG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure6_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure6_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure6_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure6_3_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure6_3_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.21 methylation level distribution at functional genetic elements in CG context</p>
<p class="premark">The x-axis is the functional genetic elements，the y-axis is the methylation level. After combining the samples with biological repeats (if have), each functional region is divided into 20 bins and methylation level is calculated in each bin. Color donates group</p>
</div>

<!------- Methylation level of up/downstream 2kb and gene body.------------------------------------>
<div id="page">

</p><br />
<h4 id="id_order_compare3"><a id="part3.6.3">4.6.2 Methylation level of up/downstream 2kb and gene body.</a></h4>
<p class="paragraph">We analyze the methylation level from upstream 2kb to downstream 2kb and the gene body (from TSS to TES) in C context and each C context for each compare group.</p>
<p class="paragraph">Methylation level distribution at up/downstream 2kb and gene body in CG context</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure6_5_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure6_5_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure6_5_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure6_5_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/9.compare_met/B_vs_A.geneupdownstream_mCG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure6_5_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure6_5_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure6_5_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure6_5_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure6_5_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.22 An overview of methylation level distribution at up/downstream 2kb and gene body in CG context</p>
<p class="premark">The x-axis is the functional genetic elements，the y-axis is the methylation level. After combining the samples with biological repeats (if have), each region is divided into 50 bins and methylation level is calculate in each bin. Color donates group.</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/5.compare_methylation_Analysis" title="点击打开" target="_blank">Compare Methylation Analysis</a></p>
</div>

<!--------- 7 DMR------------------------------------>
<div id="page">
</p><br />
<h3 id="id_order_dmr"><a id="Differential methylation analysis">4.7. Differential methylation analysis</a></h3>
<p class="paragraph">The methylation status may vary among the genomes of various samples (tissue, cells, individuals, etc.). The differentially methylated regions involved could be participated in the transcriptional regulation of genes. Identification of differential methylation can be used to examine differences in epigenetics in different tissues. It is well known that DNA methylation is involved in cell differentiation and proliferation, and many DMRs have been found in embryo reprogramming (R-DMRs) and developmental stages (D-DMRs) (Reik et al., 2001; Meissner et al., 2008; Doi et al., 2009). Cancer also exhibits overall methylation loss and hypermethylation of localized areas such as CpG islands in the promoter region of the tumor suppressor gene compared to normal samples  (Irizarry et al., 2009).</p>
<!--------- DMR------------------------------------>
<h4 id="id_order_dmr1"><a id="part3.7.1">4.7.1 DMR analysis</a></h4>
<p class="paragraph">DSS-single(DSS) is used to analyze differentially methylated regions (DMR) in the pipeline. Considering the spatial correlation of DMR, smoothing is applied when conducting the analysis. (Feng et al., 2014; Wu et al., 2015; Park et al., 2016).</p>
<p class="paragraph">The factors considered in DSS analysis: </p>
<p class="paragraph">1. The spatial correlation of the methylation site: appropriate use of information on nearby methylation sites can improve the assessment of methylation levels at each site and the identification of DMR; Use a smooth method to identify relatively long DMRs.</p>
<p class="paragraph">2. The read depth of the sites: the sequencing depth of the site can improve the accuracy and the statistical values of DMR identification.</p>
<p class="paragraph">3. The variance among biological replicates: Ignoring differences in biological duplication may lead to false positives. DSS software is based on beta binomial distribution, taking into account heterogeneity between biological replicates. When studying samples without replicated, DSS uses smoothing to nearby sites as biological repeat sites for analysis (DSS analysis pipeline and parameters can refer to README.txt in results and references.</p>
<p class="paragraph">DMR analysis results using DSS is shown in follows:</p>
<p class="name_table" align="center">Table 4.6 DMR analysis results</p>
<p class="center">
<div style="overflow-x: scroll"> 	
<table class="gy">
<tr><th>chr</th><th>start</th><th>end</th><th>DMR_length</th><th>C_number</th><th>group1_meanMethy</th><th>group2_meanMethy</th><th>diff.Methy</th><th>areaStat</th><th>C_context</th></tr>

<tr><td>1</td><td>39398798</td><td>39398849</td><td>52</td><td>4</td><td>0.610458260542504</td><td>0.39808924766286</td><td>0.212369012879644</td><td>10.1188681985479</td><td>CG</td></tr>

<tr><td>1</td><td>58842130</td><td>58842243</td><td>114</td><td>4</td><td>0.904282426841227</td><td>0.0378306196651317</td><td>0.866451807176095</td><td>24.3606569867689</td><td>CG</td></tr>

<tr><td>1</td><td>87585708</td><td>87585758</td><td>51</td><td>5</td><td>0.251698192595411</td><td>0.563677076072109</td><td>-0.311978883476699</td><td>-22.955613043566</td><td>CG</td></tr>

<tr><td>1</td><td>90262906</td><td>90262960</td><td>55</td><td>5</td><td>0.335120032613093</td><td>0.678767282383275</td><td>-0.343647249770182</td><td>-25.1652079107759</td><td>CG</td></tr>

</table>
</p>
</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
（1）chr: chromosome ID<br />
（2）start: start position of DMR<br />
（3）end: end position of DMR<br />
（4）DMR_length: length of DMR<br />
（5）C_number: number of sites inside DMR<br />
（6）group1_meanMethy: average methylation level in case group<br />
（7）group2_meanMethy: average methylation level in control group<br />
（8）diff.Methy: difference of methylation level between two groups<br />
（9）areaStat: statistical value to perform the feneral difference between two group<br />
（10) C_context: sequence context <br />
<br />
 PS: DMR.all.result.xls: all DMR identified by DSS software. If the number of DMR in all context is higher than 10000, the top 10000 DMR are selected for further analysis.<br />
</ul>
</div>
<br />
</div>
<!---------------------------DMR results annotation------------------------------------>
<div id="page">

</p><br />
<h4 id="id_order_dmr2"><a id="part3.7.2">4.7.2 DMR results annotation</a></h4>
<p class="paragraph">Annotation to DMR regions such as promoter，exon，intron，CGI，CGI shore，repeat，TSS，TES; and add information of gene name if DMR is in the gene region.</p>
<p class="name_table" align="center">Table 4.7 DMR annotation</p>
<p class="center">
<div style="overflow-x: scroll"> 	
<table class="gy">
<tr><th>chr</th><th>start</th><th>end</th><th>DMR_length</th><th>C_number</th><th>group1_meanMethy</th><th>group2_meanMethy</th><th>diff.Methy</th><th>areaStat</th><th>C_context</th><th>regionID</th><th>region</th><th>Gene name</th><th>Gene description</th></tr>

<tr><td>1</td><td>39398798</td><td>39398849</td><td>52</td><td>4</td><td>0.610458260542504</td><td>0.39808924766286</td><td>0.212369012879644</td><td>10.1188681985479</td><td>CG</td><td>ENSMUSG00000026077</td><td>exon</td><td>Npas2</td><td>- &amp;&amp; sp|P97460|NPAS2_MOUSE Neuronal PAS domain-containing protein 2 OS=Mus musculus OX=10090 GN=Npas2 PE=1 SV=1 &amp;&amp; PF00989:PAS fold|PF00010:Helix-loop-helix DNA-binding domain|PF14598:PAS domain</td></tr>

<tr><td>1</td><td>58842130</td><td>58842243</td><td>114</td><td>4</td><td>0.904282426841227</td><td>0.0378306196651317</td><td>0.866451807176095</td><td>24.3606569867689</td><td>CG</td><td>ENSMUSG00000026029</td><td>intron</td><td>Casp8</td><td>- &amp;&amp; sp|O89110|CASP8_MOUSE Caspase-8 OS=Mus musculus OX=10090 GN=Casp8 PE=1 SV=1 &amp;&amp; PF00656:Caspase domain|PF01335:Death effector domain</td></tr>

<tr><td>1</td><td>87585708</td><td>87585758</td><td>51</td><td>5</td><td>0.251698192595411</td><td>0.563677076072109</td><td>-0.311978883476699</td><td>-22.955613043566</td><td>CG</td><td>ENSMUSG00000026288</td><td>intron</td><td>Inpp5d</td><td>- &amp;&amp; sp|Q9ES52|SHIP1_MOUSE Phosphatidylinositol 3,4,5-trisphosphate 5-phosphatase 1 OS=Mus musculus OX=10090 GN=Inpp5d PE=1 SV=2 &amp;&amp; PF03372:Endonuclease/Exonuclease/phosphatase family|PF00017:SH2 domain</td></tr>

<tr><td>1</td><td>93139454</td><td>93139516</td><td>63</td><td>10</td><td>0.244801898040076</td><td>0.68846551739176</td><td>-0.443663619351685</td><td>-54.5474530722981</td><td>CG</td><td>ENSMUSG00000084989</td><td>intron</td><td>Crocc2</td><td>- &amp;&amp; sp|F6XLV1|CRCC2_MOUSE Ciliary rootlet coiled-coil protein 2 OS=Mus musculus OX=10090 GN=Crocc2 PE=3 SV=2 &amp;&amp; PF15035:Ciliary rootlet component, centrosome cohesion</td></tr>

<tr><td>13</td><td>56696277</td><td>56696368</td><td>92</td><td>9</td><td>0.0991525950671427</td><td>0.402350738981562</td><td>-0.303198143914419</td><td>-47.8039075284936</td><td>CG</td><td>ENSMUSG00000021539</td><td>exon</td><td>Lect2</td><td>- &amp;&amp; sp|O88803|LECT2_MOUSE Leukocyte cell-derived chemotaxin-2 OS=Mus musculus OX=10090 GN=Lect2 PE=1 SV=1 &amp;&amp; -</td></tr>

</table>
</p>
</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
（1）chr: chromosome ID<br />
（2）start: start position of DMR<br />
（3）end: end position of DMR<br />
（4）DMR_length: length of DMR<br />
（5）C_number: number of sites inside DMR<br />
（6）group1_meanMethy: average methylation level in case group<br />
（7）group2_neanMethy: average methylation level in control group<br />
（8）diff.Methy: difference of methylation level between two groups<br />
（9）areaStat: statistical value to perform the feneral difference between two groups<br />
（10) C_context: sequence context <br />
（11) regionID: region ID（could be gene ID，repeat ID，CGI position）<br />
（12) region: region context<br />
（13) Gene name: gene name or related description if gene ID related region<br />
（14) Gene description: gene description<br />
</ul>
</div>
<p><br />
</div>
<!--------- DMR length ------------------------------------>
<div id="page">

<h4 id="id_order_dmr3"><a id="part3.7.3">4.7.3 DMR length distribution</a></h4>
<p class="paragraph">The following plot shows DMR length distribution. Previous study indicates that DMR lengths range from hundreds to thousands bp, which follows a normal distribution model (Sun et al., 2014; Wen et al., 2016).</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure7_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_1_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure7_1_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/10.DM/B_vs_A.CG.DMRs_length.distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure7_1_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.23 DMR length distribution in CG context</p>
<p class="premark">The x-axis represents DMR Length, and y-axis represents DMR counts with corresponding length</p>
</div>
<!--------- DMR violin_boxplot ------------------------------------>
<div id="page">
<h4 id="id_order_dmr4"><a id="part3.7.4">4.7.4 Violin plot for DMR methylation level</a></h4>
<br />
<p class="paragraph"> For DMR methylation levels, the DSS software gives the average methylation level of the differentially methylated region to draw the methylation level distribution violin boxplot.</p>
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure7_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_2_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/10.DM/B_vs_A.DMR.boxplot.CG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure7_2_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.24 Violin plot for DMR methylation level in CG context</p>
<p class="premark">The x-axis is the comparison group name, the y-axis is the methylation level.</p>
<br />
<br />
</div>
<!--------- DMR pheatmap ------------------------------------>
<div id="page">

<h4 id="id_order_dmr5"><a id="part3.7.5">4.7.5 Cluster heatmap for DMR methylation level</a></h4>
<br />
<p class="paragraph">For DMR methylation levels, the DSS software gives the average methylation level of the differentially methylated region to draw the heatmap to show the methylation and differences between the combinations.</p>
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure7_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_3_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/10.DM/B_vs_A.DMR.heatmap.CG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure7_3_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.25 DMR methylation level cluster heatmap in CG context</p>
<p class="premark">The x-axis is the comparison group name, the y-axis is the methylation level and cluster results.</p>
<br />

</div>
<!--------- DMR annot_region_distribution ------------------------------------>
<div id="page">
<h4 id="id_order_dmr6"><a id="part3.7.6">4.7.6 The distribution of Hyper/Hypo DMR methylation level</a></h4>
<br />
<p class="paragraph">For DMR target regions (such as promoter, exon, intron, CGI, CGI shore, repeat, TSS, TES, etc.), we provide statistics between hyper (high methylation DMR) and hypo (low methylation DMR).</p>
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure7_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_4_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure7_4_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/10.DM/B_vs_A.CG.DMRs_generegion.distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_4_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure7_4_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.26 DMR distribution plot in CG context in different functional region</p>
<p class="premark">The x-axis is functional regions, the y-axis is number of hyper/hypo DMR in each region.</p>
<br />
</div>
<!--------- DMR  circos------------------------------------>
<div id="page">
<h4 id="id_order_dmr7"><a id="part3.7.7">4.7.7 Circos plot for DMR significance</a></h4>
<br />
<p class="paragraph"> DMR distribution on the genome and significant analysis, reflected by circos diagram (Krzywinski et al., 2009).</p>
<p class="paragraph"> After statistical testing, there will be a statistical value called areaStat to indicate a significant difference in the DMR region. The significance of DMR on a specific chromosome and the distribution of DMR on the chromosome are reflected by circus plot.</p>
<br />
<p class="paragraph">1. DMR condition in CG context</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure7_6_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_6_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_6_pane_B_vs_A_CG" 
role="tabpanel" aria-labelledby="gallery_figure7_6_list_B_vs_A_CG"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/10.DM/B_vs_A_CG.DMR_circos.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_6_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_6_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_6_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_6_list_B_vs_A_CG"
data-toggle="list" href="#gallery_figure7_6_pane_B_vs_A_CG" role="tab" aria-controls="B_vs_A_CG">B_vs_A_CG</a>

</div></div></div>
<p class="name_fig">Figure 4.27 Circos plot for DMR condition in CG context</p>
<p class="tremark"><br />
The circos plot represents (from outside to inside): 1. Hyper DMR statistical value: log5 (|areaStat|). The higher and bigger the point, the larger differences between two groups 2. TE, the heatmap of percentage of repeat element (if repeats are provided) 3. heatmap of gene density 4. Hypo DMR statistical value: log5 (|areaStat|). The higher and bigger the point, the larger differences between two groups.
<br />
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/6.compare_DM_Analysis" title="点击打开" target="_blank">Compare DM Analysis</a></p>
<br />
</div>



<!---------GO enrichment analysis for DMR related gene-------------------------------->
<div id="page">

<h3 id="id_order_dmr_geneGO"><a id="GO enrichment analysis for DMR related gene">4.8 GO enrichment analysis for DMR related gene</a></h3>
<p class="center">
<p class="paragraph">Gene enrichment analysis are applied based on the distribution of DMR in the genome in gene body region (from TSS to TES) and overlapped gene with DMR in CG context.</p>
<p class="paragraph">Gene Ontology (GO,<a href="http://www.geneontology.org/" target=_blank>http://www.geneontology.org/</a>）is an international standard classification system for gene function. Performing Go enrichment analysis of DMR-related genes contributes to dig out the biological processes of biological problems. GO enrichment analysis is used by GOseq (Young et al., 2010), which is based on Wallenius non-central hyper-geometric distribution. Its characteristics are: the probability of drawing an individual from a certain category is different from that of drawing it from outsides of the category, and this difference is obtained from the estimation of the preference of gene length.</p>
<p class="name_table" align="center">Table 4.8 GO enrichment for DMR related gene</p>
<p class="center">
<table class="gy">
<tr><th>GO accession</th><th>Description</th><th>Term type</th><th>Over represented p-value</th><th>Corrected p-value</th><th>DMR genes item</th><th>DMR genes list</th></tr>

<tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>0.030071</td><td>1</td><td>8</td><td>16</td></tr>

<tr><td>GO:0051056</td><td>regulation of small GTPase mediated signal transduction</td><td>biological_process</td><td>0.041573</td><td>1</td><td>1</td><td>16</td></tr>

<tr><td>GO:1902531</td><td>regulation of intracellular signal transduction</td><td>biological_process</td><td>0.047479</td><td>1</td><td>1</td><td>16</td></tr>

<tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>0.051219</td><td>1</td><td>3</td><td>16</td></tr>

<tr><td>GO:0005096</td><td>GTPase activator activity</td><td>molecular_function</td><td>0.061916</td><td>1</td><td>1</td><td>16</td></tr>

</table>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) GO_accession: Gene Ontology entry<br />
(2) Description: Detail description of Gene Ontology<br />
(3) Term_type: GO types,including cellular_component,biological_process,molecular_functio<br />
(4) Over_represented_pValue: P-value in hypergenometric test<br />
(5) Corrected_pValue: GO with Corrected P-value < 0.05 are significantly enriched genes<br />
(6) DMR_gene_item: Number of DMR reference genes  with GO anntation<br />
(7) DMR_gene_list: Number of all DMR related genes with GO annotation</p>
</ul>
</div>
</div>
<!----------------------GO富集柱状图-------------------------------------------------->
<div id="page">

<p class="paragraph">GO enrichment histogram of DMR related gene (in different context) can reflect the distribution of DMR related gene in biological process, cellular component, and molecular function.</p>
<p class="paragraph">Figures in this chapter contains: 1. Classification charts for GO Enrichment contains, 2. Annotation for significant enrichment term, 3. Comparison of significant enrichment term and background gene.</p>
<p class="paragraph">GO enrichment results are show in follows. (results files include ALL/Hyper/Hypo DMR target gene and different sequence context, in pdf format.</p>
<br />
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure8_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_1_pane_B_vs_A_CG_ALL_bar" 
role="tabpanel" aria-labelledby="gallery_figure8_1_list_B_vs_A_CG_ALL_bar"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/12.goenrichdir/B_vs_A.CG.ALL.DMR_genes.Enriched_GO.bar_graph.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_pane_B_vs_A_CG_Hyper_bar" 
role="tabpanel" aria-labelledby="gallery_figure8_1_list_B_vs_A_CG_Hyper_bar"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/12.goenrichdir/B_vs_A.CG.Hyper.DMR_genes.Enriched_GO.bar_graph.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_pane_B_vs_A_CG_Hypo_bar" 
role="tabpanel" aria-labelledby="gallery_figure8_1_list_B_vs_A_CG_Hypo_bar"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/12.goenrichdir/B_vs_A.CG.Hypo.DMR_genes.Enriched_GO.bar_graph.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_list_B_vs_A_CG_ALL_bar"
data-toggle="list" href="#gallery_figure8_1_pane_B_vs_A_CG_ALL_bar" role="tab" aria-controls="B_vs_A_CG_ALL_bar">B_vs_A_CG_ALL_bar</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_list_B_vs_A_CG_Hyper_bar"
data-toggle="list" href="#gallery_figure8_1_pane_B_vs_A_CG_Hyper_bar" role="tab" aria-controls="B_vs_A_CG_Hyper_bar">B_vs_A_CG_Hyper_bar</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_list_B_vs_A_CG_Hypo_bar"
data-toggle="list" href="#gallery_figure8_1_pane_B_vs_A_CG_Hypo_bar" role="tab" aria-controls="B_vs_A_CG_Hypo_bar">B_vs_A_CG_Hypo_bar</a>

</div></div></div>
<p class="name_fig">Figure 4.28 GO enrichment bar chart</p>
<p class="premark">1. Classification chart for GO enrichment: The y-axis is the enriched GO term, The x-axis is the number of DMR related genes, colors donates three GO term type: biological process, cellular component and molecular function. 2. Annotation for significant enrichment term, the x-axis is the enriched GO term, The y-axis is the percentage of DMR related genes. 3. Comparison of significant enrichment term and background gene, the x-axis is the enriched GO term, The y-axis is the percentage of DMR related genes. Light blue indicates the DMR related gene and light green indicates the background gene </p>
</div>
<!-------------------------------------------------GO DAG 图------------------------------------------------>



<!----------------------------------------------------DMR gene KEGG -------------------------------->
<div id="page">
<h3 id="id_order_dmr_geneKEGG"><a id="KEGG enrichment analysis for DMR related gene">4.9. KEGG enrichment analysis for DMR related gene</a></h3>
<hr />
<p class="center">
<p class="paragraph"> Based on the distribution of DMR on the genome, KEGG enrichment analysis was performed on genes with overlapping body bodies (from TSS to TES) and DMR (sequence context).</p>
<p class="paragraph"> Different genes coordinate with each other to achieve biological functions. Pathway enrichment can help quickly understand the main signaling pathways involved in DMR-related genes. KEGG (Kyoto Encyclopedia of Genes and Genomes,http://www.kegg.jp/)is an important public database of Pathway (Kanehisa et al., 2008).</p>
<p class="paragraph"> Pathway Significant Enrichment Analysis using the KEGG Pathway, a hypergeometric test was used to find Pathway that was significantly enriched in DMR-related genes compared to the entire genomic background. The results of the KEGG metabolic pathway enrichment are as follows (detailed result can refers to results file).</p>
<br />
<p class="name_table" align="center">Table 4.9 KEGG enrichment results for all DMR related genes</p>
<p class="center">
<div id="tb">
<table class="gy">
<tr><th>Term</th><th>Database</th><th>ID</th><th>DMR genes number</th><th>Background number</th><th>P-value</th><th>Corrected p-value</th></tr>

<tr><td>Non-alcoholic fatty liver disease (NAFLD)</td><td>KEGG PATHWAY</td><td>mmu04932</td><td>2</td><td>151</td><td>0.00377921301924</td><td>0.0992784930101</td></tr>

<tr><td>Hepatitis C</td><td>KEGG PATHWAY</td><td>mmu05160</td><td>2</td><td>160</td><td>0.00422461672383</td><td>0.0992784930101</td></tr>

<tr><td>Circadian rhythm</td><td>KEGG PATHWAY</td><td>mmu04710</td><td>1</td><td>30</td><td>0.0184146837035</td><td>0.126444067888</td></tr>

<tr><td>Apoptosis - multiple species</td><td>KEGG PATHWAY</td><td>mmu04215</td><td>1</td><td>32</td><td>0.0195915217065</td><td>0.126444067888</td></tr>

<tr><td>Hedgehog signaling pathway</td><td>KEGG PATHWAY</td><td>mmu04340</td><td>1</td><td>44</td><td>0.0266242586283</td><td>0.126444067888</td></tr>

</table>
</div>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Term: The description of KEGG pathways<br />
(2) Database: The database of pathway<br />
(3) ID: KEGG ID in KEGG database<br />
(4) Input number: Number of DMR related genes with pathway annotation<br />
(5) Background number: Number of all reference genes with pathway annotation <br />
(6) P-value: significant value for enrichment analysis<br />
(7) Corrected P-value: adjusted p-value. The function with corrected P-value < 0.05 is regarded as enrichment<br />
</ul>
</div>
<br />
<br />
<!------------------------------------------------------------------------------------------------------------>
</div>
<div id="page">
<p class="paragraph">The scatter plot is a graphical representation of the results of the KEGG enrichment analysis. In this figure, the degree of KEGG enrichment is measured by the number of genes, the Qvalue, and the number of genes enriched in this pathway. Where Rich factor refers to the ratio of the number of DMR-related genes enriched in the pathway to the number of annotated genes. The larger the Rich factor, the greater the degree of enrichment. Qvalue is the Pvalue after multiple hypothesis test correction. The value of Qvalue is [0,1]. The enrichment will be more significant if the q-value is close to 0.</p>
<p class="paragraph">The most significant enriched 20 pathways are presented in the scatter plot. If there are less than 20 pathways, all the pathways will be presented.
In addition, we also provide a web interface that can display DMR-related genes through KEGG's metabolic pathway map. (The result file contains detailed results on different context, distinguishing the DMR anchoring gene ALL/Hyper/Hypo, enrichment mapping pdf format results)</p>

<br />
<p class="paragraph">1. Scatter plot of KEGG metabolic pathway enrichment</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure9_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure9_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure9_1_pane_B_vs_A_CG_ALL" 
role="tabpanel" aria-labelledby="gallery_figure9_1_list_B_vs_A_CG_ALL"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/B_vs_A.CG.ALL.DMR_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_1_pane_B_vs_A_CG_Hyper" 
role="tabpanel" aria-labelledby="gallery_figure9_1_list_B_vs_A_CG_Hyper"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/B_vs_A.CG.Hyper.DMR_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_1_pane_B_vs_A_CG_Hypo" 
role="tabpanel" aria-labelledby="gallery_figure9_1_list_B_vs_A_CG_Hypo"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/B_vs_A.CG.Hypo.DMR_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure9_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure9_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure9_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure9_1_list_B_vs_A_CG_ALL"
data-toggle="list" href="#gallery_figure9_1_pane_B_vs_A_CG_ALL" role="tab" aria-controls="B_vs_A_CG_ALL">B_vs_A_CG_ALL</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_1_list_B_vs_A_CG_Hyper"
data-toggle="list" href="#gallery_figure9_1_pane_B_vs_A_CG_Hyper" role="tab" aria-controls="B_vs_A_CG_Hyper">B_vs_A_CG_Hyper</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_1_list_B_vs_A_CG_Hypo"
data-toggle="list" href="#gallery_figure9_1_pane_B_vs_A_CG_Hypo" role="tab" aria-controls="B_vs_A_CG_Hypo">B_vs_A_CG_Hypo</a>

</div></div></div>
<p class="name_fig">Figure 4.30 KEGG enrichment plot DMR related pathway</p>
<p class="premark">The x-axis represents Rich factor, and the y-axis represents pathway name. The size of points stand for DMRs related genes counts and the colours stand for different Qvalues range.</p>
<br />
<p class="paragraph">2. The metabolic pathway map of KEGG</p>
<div class="row">
<div class="col-1">
<div id="gallery_figure9_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure9_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure9_2_pane_mmu04932" 
role="tabpanel" aria-labelledby="gallery_figure9_2_list_mmu04932"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/mmu04932.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_2_pane_mmu05160" 
role="tabpanel" aria-labelledby="gallery_figure9_2_list_mmu05160"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/mmu05160.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_2_pane_mmu04710" 
role="tabpanel" aria-labelledby="gallery_figure9_2_list_mmu04710"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/mmu04710.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_2_pane_mmu04215" 
role="tabpanel" aria-labelledby="gallery_figure9_2_list_mmu04215"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/mmu04215.png" /></div>

<div class="tab-pane fade" id="gallery_figure9_2_pane_mmu04340" 
role="tabpanel" aria-labelledby="gallery_figure9_2_list_mmu04340"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/13.keggenrichdir/mmu04340.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure9_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure9_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure9_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure9_2_list_mmu04932"
data-toggle="list" href="#gallery_figure9_2_pane_mmu04932" role="tab" aria-controls="mmu04932">mmu04932</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_2_list_mmu05160"
data-toggle="list" href="#gallery_figure9_2_pane_mmu05160" role="tab" aria-controls="mmu05160">mmu05160</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_2_list_mmu04710"
data-toggle="list" href="#gallery_figure9_2_pane_mmu04710" role="tab" aria-controls="mmu04710">mmu04710</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_2_list_mmu04215"
data-toggle="list" href="#gallery_figure9_2_pane_mmu04215" role="tab" aria-controls="mmu04215">mmu04215</a>

<a class="list-group-item list-group-item-action" id="gallery_figure9_2_list_mmu04340"
data-toggle="list" href="#gallery_figure9_2_pane_mmu04340" role="tab" aria-controls="mmu04340">mmu04340</a>

</div></div></div>
<p class="name_fig">Figure 4.31 The metabolic pathway map of KEGG</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/8.KEGG_Enrichment_Methy" title="点击打开" target="_blank">KEGG Enrichment Methy</a></p>
<br />
</div>

<!----------------------------------------------------DMR gene KEGG  over-------------------------------->

<!---------- 10 GO enrichment analysis for DMR target promoter gene------------------------------------->
<div id="page">
<h3 id="id_order_dmr_promoter_geneGO"><a id="GO enrichment analysis for DMR target promoter gene">4.10. GO enrichment analysis for DMR target promoter gene</a></h3>
<p class="center">
<p class="paragraph">Gene enrichment analysis are applied based on the distribution of DMR in the genome in overlapped gene with DMR (TSS upstream 2kb) in CG context. </p>
<p class="paragraph">Gene Ontology (GO, <a href="http://www.geneontology.org/" target=_blank>http://www.geneontology.org/</a>）is an international standard classification system for gene function. Performing Go enrichment analysis of DMR target promotor genes contributes to dig out the biological processes of biological problems. GO enrichment analysis is used by GOseq (Young et al, 2010), which is based on Wallenius non-central hyper-geometric distribution. Its characteristics are: the probability of drawing an individual from a certain category is different from that of drawing it from outsides of the category, and this difference is obtained from the estimation of the preference of gene length.</p>
<p class="paragraph">Shown below are the results of gene GO enrichment of the DMR anchor gene promoter region. (There are detailed sequenced environments in the results file, distinguishing the DMR anchoring gene ALL/Hyper/Hypo, enrichment analysis results)</p>
<p class="paragraph">The concept of the promoter area: This is only a rough selection of this area; clients can use other techniques to further determine the promoter area of the gene for their own research purposes</p>
<p class="name_table" align="center">Table 4.10 GO enrichment for DMR target promoter gene</p>
<p class="center">
<div id="tb">
<table class="gy">
<tr><th>GO accession</th><th>Description</th><th>Term type</th><th>Over represented p-value</th><th>Corrected p-value</th><th>DMR genes item</th><th>DMR genes list</th></tr>

<tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>0.030071</td><td>1</td><td>8</td><td>16</td></tr>

<tr><td>GO:0051056</td><td>regulation of small GTPase mediated signal transduction</td><td>biological_process</td><td>0.041573</td><td>1</td><td>1</td><td>16</td></tr>

<tr><td>GO:1902531</td><td>regulation of intracellular signal transduction</td><td>biological_process</td><td>0.047479</td><td>1</td><td>1</td><td>16</td></tr>

<tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>0.051219</td><td>1</td><td>3</td><td>16</td></tr>

<tr><td>GO:0005096</td><td>GTPase activator activity</td><td>molecular_function</td><td>0.061916</td><td>1</td><td>1</td><td>16</td></tr>

</table>
</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) GO_accession: Gene Ontology entry<br />
(2) Description: Detail description of Gene Ontology<br />
(3) Term_type: GO types,including cellular_component,biological_process,molecular_functio<br />
(4) Over_represented_pValue: P-value in hypergenometric test<br />
(5) Corrected_pValue: GO with Corrected P-value < 0.05 are significantly enriched genes<br />
(6) DMR_gene_item: Number of DMR reference genes  with GO anntation<br />
(7) DMR_gene_list: Number of all DMR related genes with GO annotation<br />
</ul>
</div>
</p>
</div>
<!----------------------GO富集柱状图-------------------------------------------------->
<div id="page">
<p class="paragraph">GO enrichment histogram of DMR target promoter gene  (in different context) can reflect the distribution of DMR target promoter gene  in biological process, cellular component, and molecular function.</p>
<p class="paragraph">Figures in this chapter contains: 1. Classification charts for GO Enrichment contains, 2. Annotation for significant enrichment term, 3. Comparison of significant enrichment term and background gene.</p>
<p class="paragraph">GO enrichment results are show in follows. (results files include ALL/Hyper/Hypo DMR target promoter gene and different sequence context, in pdf format.</p>
<br />
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure10_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure10_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure10_1_pane_B_vs_A_CG_ALL_bar" 
role="tabpanel" aria-labelledby="gallery_figure10_1_list_B_vs_A_CG_ALL_bar"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/14.promoter_godir/B_vs_A.CG.ALL.DMR_promoter_genes.Enriched_GO.bar_graph.png" /></div>

<div class="tab-pane fade" id="gallery_figure10_1_pane_B_vs_A_CG_Hypo_bar" 
role="tabpanel" aria-labelledby="gallery_figure10_1_list_B_vs_A_CG_Hypo_bar"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/14.promoter_godir/B_vs_A.CG.Hypo.DMR_promoter_genes.Enriched_GO.bar_graph.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure10_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure10_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure10_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure10_1_list_B_vs_A_CG_ALL_bar"
data-toggle="list" href="#gallery_figure10_1_pane_B_vs_A_CG_ALL_bar" role="tab" aria-controls="B_vs_A_CG_ALL_bar">B_vs_A_CG_ALL_bar</a>

<a class="list-group-item list-group-item-action" id="gallery_figure10_1_list_B_vs_A_CG_Hypo_bar"
data-toggle="list" href="#gallery_figure10_1_pane_B_vs_A_CG_Hypo_bar" role="tab" aria-controls="B_vs_A_CG_Hypo_bar">B_vs_A_CG_Hypo_bar</a>

</div></div></div>
<p class="name_fig">Figure 4.32 GO enrichment bar chart</p>
<p class="premark">1. Classification chart for GO enrichment: The y-axis is the enriched GO term, The x-axis is the number of DMR related genes, colors donates three GO term type: biological process, cellular component and molecular function. 2. Annotation for significant enrichment term, the x-axis is the enriched GO term, The y-axis is the percentage of DMR related genes. 3. Comparison of significant enrichment term and background gene, the x-axis is the enriched GO term, The y-axis is the percentage of DMR related genes. Light blue indicates the DMR related gene and light green indicates the background gene. </p>
<p class="premark">PS：GO classification annotation of all DMR target promoter gene and figures are included in the result file.</p>
</div>
<!-------------------------------------------------GO DAG 图------------------------------------------------>


<!---------- 10 dmr_promoter_geneGO over --------------------------------------------------------------------------->

<!-------------11 KEGG enrichment analysis of DMR anchoring promoter region related genes--------------------------->

</p><br />
<h3 id="iKEGG enrichment analysis of DMR anchoring promoter region related genes"><a id="KEGG enrichment analysis of DMR anchoring promoter region related genes">4.11. KEGG enrichment analysis of DMR anchoring promoter region related genes</a></h3>
<p class="center">
<p class="paragraph">
Functional enrichment analysis based on the distribution of DMR on the genome and overlapping genes in the gene promoter region (2K upstream of the transcription start site).</p>
<p class="paragraph">
Different genes coordinate with each other to achieve biological functions. Pathway enrichment can help quickly understand the main signaling pathways involved in DMR-related genes. KEGG(Kyoto Encyclopedia of Genes and Genomes,http://www.kegg.jp/)is an important public database of Pathway.</p>
<p class="paragraph">
The concept of the promoter area: This is only a rough selection of this area; clients can use other techniques to further determine the promoter area of the gene for their own research purposes.</p>
<br />
<p class="name_table" align="center">Table 4.11 DMR anchoring promoter region related gene KEGG enrichment list</p>
<p class="center">
<div id="tb">
<table class="gy">
<tr><th>Term</th><th>Database</th><th>ID</th><th>DMR genes number</th><th>Background number</th><th>P-value</th><th>Corrected p-value</th></tr>

<tr><td>Non-alcoholic fatty liver disease (NAFLD)</td><td>KEGG PATHWAY</td><td>mmu04932</td><td>2</td><td>151</td><td>0.00377921301924</td><td>0.0992784930101</td></tr>

<tr><td>Hepatitis C</td><td>KEGG PATHWAY</td><td>mmu05160</td><td>2</td><td>160</td><td>0.00422461672383</td><td>0.0992784930101</td></tr>

<tr><td>Circadian rhythm</td><td>KEGG PATHWAY</td><td>mmu04710</td><td>1</td><td>30</td><td>0.0184146837035</td><td>0.126444067888</td></tr>

<tr><td>Apoptosis - multiple species</td><td>KEGG PATHWAY</td><td>mmu04215</td><td>1</td><td>32</td><td>0.0195915217065</td><td>0.126444067888</td></tr>

<tr><td>Hedgehog signaling pathway</td><td>KEGG PATHWAY</td><td>mmu04340</td><td>1</td><td>44</td><td>0.0266242586283</td><td>0.126444067888</td></tr>

</table>
</div>
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Term: The description of KEGG pathways<br />
(2) Database: The database of pathway<br />
(3) ID: KEGG ID in database<br />
(4) Input number: Number of DMR related genes with pathway annotation<br />
(5) Background number: Number of all reference genes with pathway annotation<br />
(6) P-value: P-value in hypergenometric test<br />
(7) Corrected P-value: adjusted p-value. The function with corrected P-value < 0.05 is regarded as enrichment<br /></ul>
</div>
<br />
<!------------------------------------------------------------------------------------------------------------>
<p class="paragraph">
The scatter plot is a graphical representation of the results of the KEGG enrichment analysis. In this figure, the degree of KEGG enrichment is measured by the number of genes, the Qvalue, and the number of genes enriched in this pathway. Where Rich factor refers to the ratio of the number of DMR-related genes enriched in the pathway to the number of annotated genes. The larger the Rich factor, the greater the degree of enrichment. Qvalue is the Pvalue after multiple hypothesis test correction. The value of Qvalue is [0,1]. The closer the q-value to 0, the more significant the enrichment.</p>
<p class="paragraph">
The most significant enriched 20 pathways are presented in the scatter plot. If there are less than 20 pathways, all the pathways will be presented.</p>
<p class="paragraph">
In addition, we also provide a web interface that can display DMR-related genes through KEGG's metabolic pathway map. (The result file contains detailed results on different context, distinguishing the DMR anchoring gene ALL/Hyper/Hypo, enrichment mapping pdf format results)</p>
<br />
<br />
<div class="row">
<div class="col-1">
<div id="gallery_figure11_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure11_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure11_1_pane_B_vs_A_CG_ALL" 
role="tabpanel" aria-labelledby="gallery_figure11_1_list_B_vs_A_CG_ALL"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/15.promoter_keggdir/B_vs_A.CG.ALL.DMR_promoter_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

<div class="tab-pane fade" id="gallery_figure11_1_pane_B_vs_A_CG_Hyper" 
role="tabpanel" aria-labelledby="gallery_figure11_1_list_B_vs_A_CG_Hyper"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/15.promoter_keggdir/B_vs_A.CG.Hyper.DMR_promoter_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

<div class="tab-pane fade" id="gallery_figure11_1_pane_B_vs_A_CG_Hypo" 
role="tabpanel" aria-labelledby="gallery_figure11_1_list_B_vs_A_CG_Hypo"
onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/15.promoter_keggdir/B_vs_A.CG.Hypo.DMR_promoter_genes.enriched_KEGG_pathway_scatterplot.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure11_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure11_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure11_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure11_1_list_B_vs_A_CG_ALL"
data-toggle="list" href="#gallery_figure11_1_pane_B_vs_A_CG_ALL" role="tab" aria-controls="B_vs_A_CG_ALL">B_vs_A_CG_ALL</a>

<a class="list-group-item list-group-item-action" id="gallery_figure11_1_list_B_vs_A_CG_Hyper"
data-toggle="list" href="#gallery_figure11_1_pane_B_vs_A_CG_Hyper" role="tab" aria-controls="B_vs_A_CG_Hyper">B_vs_A_CG_Hyper</a>

<a class="list-group-item list-group-item-action" id="gallery_figure11_1_list_B_vs_A_CG_Hypo"
data-toggle="list" href="#gallery_figure11_1_pane_B_vs_A_CG_Hypo" role="tab" aria-controls="B_vs_A_CG_Hypo">B_vs_A_CG_Hypo</a>

</div></div></div>
<p class="name_fig">Figure 4.34 KEGG enrichment plot DMR related pathway</p>
<p class="premark">The x-axis represents Rich factor, and the y-axis represents pathway name. The size of points stand for DMRs related genes counts and the colours stand for different Qvalues range.</p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXX-XXX-XXXX-17_species/10.KEGG_Enrichment_promoter" title="点击打开" target="_blank">KEGG Enrichment Promoter</a></p>
<br />
<br />
</div></div></div>

<!----------------- Appendix -------------------------------------------->
<div id="pageIV">
</p><br />
<h2 id="Appendix" ><a id="Appendix"> 5. Appendix</a></h2>
<h3>5.1 Software</h3>
		<table class= 'gy' style="text-align:left">
        <tr><th>Analysis</th><th>Software</th><th>Version</th><th>Parameters</th><th>Remarks</th></tr>
		<tr><td>CGI</td><td>cpgIslandExt</td><td>-</td><td>-</td><td>UCSC</td></tr>
		<tr><td>repeat</td><td>RepeatMasker</td><td>-</td><td>-</td><td>-</td></tr>
		
		<tr><td>Trim</td><td>fastp</td><td>0.23.1</td><td>--cut_front_window_size=1 --cut_front_mean_quality=3</br>--cut_tail_window_size=1 --cut_tail_mean_quality=3</br>--cut_right_window_size=4 --cut_right_mean_quality=15 --length_required=36</td><td>-</td></tr>

		<tr><td>QC</td><td>fastqc</td><td>0.11.8</td><td>-</td><td>-</td></tr>
	
		<tr><td>mapping</td><td>Bismark</td><td>0.24.0</td><td>--score_min L, 0, -0.2, -X 700 --dovetail</td><td>bowtie2 (2.4.1) as aligner engine</td></tr>

		<tr><td>deduplicate</td><td>Bismark(deduplicate_bismark)</td><td>-</td><td>--paired --samtools_path</td><td></td></tr>

		<tr><td>mC calling</td><td>Bismark(bismark_methylation_extractor)</td><td>-</td><td>--multicore 4 --paired-end --no_overlap -ignore 5  --ignore_r2 5</td><td>-</td></tr>
		
		<tr><td>DMR analysis</td><td>DSS</td><td>2.12.0</td><td>smoothing.span=200, delta=0, p.threshold=1e-05,</br>minlen=50, minCG=3, dis.merge=100, pct.sig=0.5</td><td>smoothing=TRUE,sequencing depth > 1</td></tr>
	
		<tr><td>GO enrichment </td><td>Goseq/topGO</br>Bioconductor</td><td>2.13</td><td>corrected p-value<0.05</td><td>-</td></tr>
			
	
		<tr><td>KEGG enrichment </td><td>KOBAS</td><td>3.0</td><td>corrected p-value<0.05</td><td>-</td></tr>
			
	    <tr><td>Circos plot</td><td>circos</td><td>0.62-1</td><td>-</td><td>-</td></tr>
		
	</table>
	<div style="height:20px;"></div>


<h3 id="Methods"><a id="Methods" >5.2 Method</a></h3>
<p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
<p class="paragraph">File catalog: <a href="./src/Method-RRBS.pdf" title="点击打开" target="_blank">methods</a></p>

<h3 id="id_order_original" ><a name="original_data">5.3 Result File Decompression Method and Format Description</a></h3>
<table class= 'gy' style="text-align:left">
<tr><th>Compressed File Format</th><th>Operating System</th><th>File Decompression</th></tr>
<tr><td rowspan="2">*.gz</td><td>Unix/Linux/Mac</td><td>Command line: gzip –d *.gz</td></tr>
<tr><td>Windows</td><td>Software: such as WinRAR, 7-Zip et al.</td></tr>
<tr><td rowspan="2">*.zip</td><td>Unix/Linux/Mac</td><td>Command line: unzip *.zip</td></tr>
<tr><td>Windows</td><td>Software: such as WinRAR, 7-Zip et al.</td></tr>
</table>
</br>
<table class= 'gy' style="text-align:left">
<tr><th>File Type</th><th>File Description</th><th>Way to Open</th></tr>
<tr><td rowspan="2">file.fa</td><td rowspan="2">Fasta format is a text-based format for representing sequences</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td rowspan="2">file.gtf</td><td rowspan="2">GFF/GTF is general feature format.</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td rowspan="2">file.fq</td><td rowspan="2">Text file that contains sequence data delivered from a sequencer</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td>file.bw</td><td>BigWig file. Indexed binary format which is useful for dense, continuous<br>data that will be dispalyed in the Genome Browser as a graph.</td><td>It can be visualized with the IGV observation or upload to UCSC browser.</td></tr>
<tr><td>file.bam</td><td>BAM is compressed format for SAM file which contains sequence<br>alignment information.</td><td>Unix/Linux/Mac user use samtools software.</td></tr>
<tr><td>file.bam.bai</td><td>Index file for bam</td><td>No need to open, put it in the same directory with bam file when<br>manipulate bam.</td></tr>
</table>
</div>

<!---------- Reference ------------------------------------------>
<div id="pageV">

</p><br />
<h2 id="order5_ref" ><a  id="Reference">6. References</a></h2>
<p class="ref">Bock C. Analysing and interpreting DNA methylation data. Nat Rev Genet. 2012;13(10):705-719. doi:10.1038/nrg3273 </p>
<p class="ref">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30(15):2114-2120. doi:10.1093/bioinformatics/btu170</p>
<p class="ref">Doi A, Park IH, Wen B, et al. Differential methylation of tissue- and cancer-specific CpG island shores distinguishes human induced pluripotent stem cells, embryonic stem cells and fibroblasts. Nat Genet. 2009;41(12):1350-1353. doi:10.1038/ng.471</p>
<p class="ref">Feng H, Conneely KN, Wu H. A Bayesian hierarchical model to detect differentially methylated loci from single nucleotide resolution sequencing data. Nucleic Acids Res. 2014;42(8):e69. doi:10.1093/nar/gku154</p>
<p class="ref">Gao S, Zou D, Mao L, et al. SMAP: a streamlined methylation analysis pipeline for bisulfite sequencing. Gigascience. 2015;4:29. Published 2015 Jul 1. doi:10.1186/s13742-015-0070-9</p>
<p class="ref">Gifford CA, Ziller MJ, Gu H, et al. Transcriptional and epigenetic dynamics during specification of human embryonic stem cells. Cell. 2013;153(5):1149-1163. doi:10.1016/j.cell.2013.04.037</p>
<p class="ref">Goldberg AD, Allis CD, Bernstein E. Epigenetics: a landscape takes shape. Cell. 2007;128(4):635-638. doi:10.1016/j.cell.2007.02.006</p>
<p class="ref">Habibi E, Brinkman AB, Arand J, et al. Whole-genome bisulfite sequencing of two distinct interconvertible DNA methylomes of mouse embryonic stem cells. Cell Stem Cell. 2013;13(3):360-369. doi:10.1016/j.stem.2013.06.002</p>
<p class="ref">Irizarry RA, Ladd-Acosta C, Wen B, et al. The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. Nat Genet. 2009;41(2):178-186. doi:10.1038/ng.298</p>
<p class="ref">Kanehisa M, Araki M, Goto S, et al. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 2008;36(Database issue):D480-D484. doi:10.1093/nar/gkm882</p>
<p class="ref">Krueger F, Andrews SR. Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics. 2011;27(11):1571-1572. doi:10.1093/bioinformatics/btr167</p>
<p class="ref">Krzywinski M, Schein J, Birol I, et al. Circos: an information aesthetic for comparative genomics. Genome Res. 2009;19(9):1639-1645. doi:10.1101/gr.092759.109</p>
<p class="ref">Kulis M, Heath S, Bibikova M, et al. Epigenomic analysis detects widespread gene-body DNA hypomethylation in chronic lymphocytic leukemia. Nat Genet. 2012;44(11):1236-1242. doi:10.1038/ng.2443</p>
<p class="ref">Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357-359. Published 2012 Mar 4. doi:10.1038/nmeth.1923</p>
<p class="ref">Lister R, Pelizzola M, Dowen RH, et al. Human DNA methylomes at base resolution show widespread epigenomic differences. Nature. 2009;462(7271):315-322. doi:10.1038/nature08514</p>
<p class="ref">Meissner A, Mikkelsen TS, Gu H, et al. Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature. 2008;454(7205):766-770. doi:10.1038/nature07107</p>
<p class="ref">Park Y, Wu H. Differential methylation analysis for BS-seq data under general experimental design. Bioinformatics. 2016;32(10):1446-1453. doi:10.1093/bioinformatics/btw026</p>
<p class="ref">Reik W, Dean W, Walter J. Epigenetic reprogramming in mammalian development. Science. 2001;293(5532):1089-1093. doi:10.1126/science.1063443</p>
<p class="ref">Robinson JT, Thorvaldsdóttir H, Winckler W, et al. Integrative genomics viewer. Nat Biotechnol. 2011;29(1):24-26. doi:10.1038/nbt.1754</p>
<p class="ref">Smallwood SA, Lee HJ, Angermueller C, et al. Single-cell genome-wide bisulfite sequencing for assessing epigenetic heterogeneity. Nat Methods. 2014;11(8):817-820. doi:10.1038/nmeth.3035</p>
<p class="ref">Song Q, Decato B, Hong EE, et al. A reference methylome database and analysis pipeline to facilitate integrative and comparative epigenomics. PLoS One. 2013;8(12):e81148. Published 2013 Dec 6. doi:10.1371/journal.pone.0081148</p>
<p class="ref">Sun D, Xi Y, Rodriguez B, et al. MOABS: model based analysis of bisulfite sequencing data. Genome Biol. 2014;15(2):R38. Published 2014 Feb 24. doi:10.1186/gb-2014-15-2-r38</p>
<p class="ref">Thorvaldsdóttir H, Robinson JT, Mesirov JP. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 2013;14(2):178-192. doi:10.1093/bib/bbs017</p>
<p class="ref">Wang L, Zhang J, Duan J, et al. Programming and inheritance of parental DNA methylomes in mammals [published correction appears in Cell. 2014 Jun 19;157(7):1735]. Cell. 2014;157(4):979-991. doi:10.1016/j.cell.2014.04.017</p>
<p class="ref">Wen Y, Chen F, Zhang Q, Zhuang Y, Li Z. Detection of differentially methylated regions in whole genome bisulfite sequencing data using local Getis-Ord statistics. Bioinformatics. 2016;32(22):3396-3404. doi:10.1093/bioinformatics/btw497</p>
<p class="ref">Wu H, Xu T, Feng H, et al. Detection of differentially methylated regions from whole-genome bisulfite sequencing data without replicates. Nucleic Acids Res. 2015;43(21):e141. doi:10.1093/nar/gkv715</p>
<p class="ref">Young MD, Wakefield MJ, Smyth GK, Oshlack A. Gene ontology analysis for RNA-seq: accounting for selection bias. Genome Biol. 2010;11(2):R14. doi:10.1186/gb-2010-11-2-r14 </p>
<p class="ref">Zhong S, Fei Z, Chen YR, et al. Single-base resolution methylomes of tomato fruit development reveal epigenome modifications associated with ripening. Nat Biotechnol. 2013;31(2):154-159. doi:10.1038/nbt.2462</p>
<br />
</div>
</div>
</div>
</body>
</html>
