<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
  <META NAME="Author" CONTENT="liuruolin@novogene.cn">

    <META NAME="Version" CONTENT="20171224" "Silent Night ^_^">
    <title>Noref3+2 ReadMe</title>
  <!--tree/start-->
  <script src="https://cdn.bootcss.com/jquery-treetable/3.2.0/jquery.treetable.min.js"></script>
  <link href="https://cdn.bootcss.com/jquery-treetable/3.2.0/css/jquery.treetable.min.css" rel="stylesheet">
  <link href="https://cdn.bootcss.com/jquery-treetable/3.2.0/css/jquery.treetable.theme.default.css" rel="stylesheet">
  <!--tree/end-->
  <style type='text/css'>
  .task-list { list-style-type: none; }
    .task-list-item { position: relative; padding-left: 2em; }
    .task-list-item input { position: absolute; top: 0px; left: 0px; }
   </style>
  </head>
  <body>
    <div id="main">
      <p><h3 align="center"  style="color:#696969" >Noref3+2 Readme</h3></p>
      <table id="example-advanced">
        <caption>
          <a href="#" onclick="jQuery('#example-advanced').treetable('expandAll'); return false;">Expand all</a>
          <a href="#" onclick="jQuery('#example-advanced').treetable('collapseAll'); return false;">Collapse all</a>
        </caption>
        <thead>
          <tr>
            <th>Result</th>
            <th>Contents</th>
      <th>Type</th>
          </tr>
        </thead>
      <tbody>
  <tr data-tt-id='2'><td><span class='folder'><a href='./01.SMRT'>01.SMRT</span></td><td>【Iso-seq result file】</td><td>folder</td></tr>   
  <tr data-tt-id='2-1' data-tt-parent-id='2'><td><span class='file'><a href='./01.SMRT/01.SMRT--ReadMe.pdf'>01.SMRT--ReadMe.pdf</span></td><td>【SMRT Delivery Results Catalog Instructions】</td><td>file</td></tr>
  <tr data-tt-id='2-4' data-tt-parent-id='2'><td><span class='folder'>Sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='2-4-1' data-tt-parent-id='2-4'><td><span class='folder'>ccs</span></td><td>【Circular Consensus Sequence Catalog】</td><td>folder</td></tr> 
  <tr data-tt-id='2-4-1-1' data-tt-parent-id='2-4-1'><td><span class='file'>Sample.ccs.fastq.zip</span></td><td>【Circular consensus sequence, fastq format】</td><td>file</td></tr> 
    <tr data-tt-id='2-4-2' data-tt-parent-id='2-4'><td><span class='folder'>classify</span></td><td>【transcript taxonomy】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-2-1' data-tt-parent-id='2-4-2'><td><span class='file'>Sample.isoseq_flnc.fasta</span></td><td>【Full-length non-chimeric sequence file, fasta format】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-3' data-tt-parent-id='2-4'><td><span class='folder'>cluster</span></td><td>【Transcript Clustering Results Catalog】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-3-1' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_hq.fasta</span></td><td>【Modified high-quality transcript sequences in fasta format】</td><td>file</td></tr>
    <tr data-tt-id='2-4-3-2' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_lq.fast</span></td><td>【Modified low-quality transcript sequences, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='2-4-3-3' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_consensus.fasta</span></td><td>【Modified consensus transcript sequence, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4' data-tt-parent-id='2-4'><td><span class='folder'>stat</span></td><td>【Table of Transcript Statistical Results】</td><td>folder</td></tr>  
  <tr data-tt-id='2-4-4-1' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_distri.xls</span></td><td>【Polymerase_reads length distribution statistical results】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-2' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_distri.{png|pdf}</span></td><td>【Polymerase_reads length distribution graph】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-4' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_stat.xls</span></td><td>【polymerase_reads statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-5' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_stat.xls</span></td><td>【subreads statistical results】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-6' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_distri.xls</span></td><td>【Statistical results of subreads length distribution】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-7' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_distri.{png|pdf}</span></td><td>【subreads length distribution chart】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-9' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_stat.xls</span></td><td>【The statistical results of the length distribution of the circular consensus sequence】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-10' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_distri.xls</span></td><td>【Circular Consensus Sequence Statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-11' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_distri.{png|pdf}</span></td><td>【Circular Consensus Sequence Length Distribution】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-13' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_stat.xls</span></td><td>【Statistical results of full-length non-chimeric sequences】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-14' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_distri.xls</span></td><td>【Statistical results of length distribution of full-length non-chimeric sequences】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-15' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_distri.{png|pdf}</span></td><td>【Full-length non-chimeric sequence length distribution map】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-18' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_stat.xls</span></td><td>【Consensus sequence statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-19' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_distri.xls</span></td><td>【Statistical results of consensus sequence length distribution】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-20' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_distri.{png|pdf}</span></td><td>【Consensus sequence length distribution graph】</td><td>file</td></tr>

  <tr data-tt-id='1'><td><span class='folder'><a href='./02.Correct'>02.Correct</td><td>【Second-generation data correction catalog】</td><td>folder</td></tr>
  <tr data-tt-id='1-2' data-tt-parent-id='1'><td><span class='file'><a href='./02.Correct/02.Correct--ReadMe.pdf'>02.Correct--ReadMe.pdf</td><td>【Correct Deliverables Catalog Description】</td><td>file</td></tr>
  <tr data-tt-id='1-1' data-tt-parent-id='1'><td><span class='folder'><a href='./02.Correct'>sample</td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='1-1-1' data-tt-parent-id='1-1'><td><span class='file'>Sample.FL_corrected.fasta</td><td>【The corrected consensus sequence of the second-generation data, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='1-1-1' data-tt-parent-id='1-1'><td><span class='file'>Sample.correct_stat.xls</td><td>【consensus Statistical results of length distribution before and after correction】</td><td>file</td></tr>


    <tr data-tt-id='3'><td><span class='folder'><a href='./03.CD-HIT'>03.CD-HIT</td><td>【Transcript deredundancy directory】</td><td>folder</td></tr>
    <tr data-tt-id='3-1' data-tt-parent-id='3'><td><span class='file'><a href ='./03.CD-HIT/03.CD-HIT--ReadMe.pdf'>03.CD-HIT--ReadMe.pdf</a></span></td><td>【Transcript deredundancy deliverables catalog description】</td><td>file</td></tr>
    <tr data-tt-id='3-3' data-tt-parent-id='3'><td><span class='file'>transcript_Sample1_share.xls</a></span></td><td>【Transcript IDs unique to Sample2】</td><td>file</td></tr>
    <tr data-tt-id='3-3' data-tt-parent-id='3'><td><span class='file'>transcript_Sample1_share.xls</a></span></td><td>【Transcript IDs unique to Sample1】</td><td>file</td></tr>
    <tr data-tt-id='3-4' data-tt-parent-id='3'><td><span class='file'>transcript_Sample1_Sample2_share.xls </a></span></td><td>【Two or more samples present: Transcript ID shared by Sample1 and Sample2】</td><td>file</td></tr>
    <tr data-tt-id='3-5' data-tt-parent-id='3'><td><span class='file'>transcript_compare.{pdf|png}</a></span></td><td>【Existence of two samples or more: Venn diagram of transcriptome comparison, pdf|png format】</td><td>file</td></tr>
    <tr data-tt-id='3-6' data-tt-parent-id='3'><td><span class='file'>all.FL_NR_NR.fasta</a></span></td><td>【A collection of two samples of fasta】</td><td>file</td></tr>
    <tr data-tt-id='3-2' data-tt-parent-id='3'><td><span class='folder'>Sample</a></span></td><td>【Sample catalog】</td><td>table</td></tr>

    <tr data-tt-id='3-2-1' data-tt-parent-id='3-2'><td><span class='file'>Sample.cdhit_stat.xls</td><td>【Statistical table of sequence length distribution after redundancy removal】</td><td>file</td></tr>
    <tr data-tt-id='3-2-2' data-tt-parent-id='3-2'><td><span class='file'>Sample.FL_NR.fasta</td><td>【unigene sequence, fasta format】</td><td>file</td></tr>
    <tr data-tt-id='3-2-3' data-tt-parent-id='3-2'><td><span class='file'>Sample.interval_stat.xls</td><td>【Statistical results of length frequency distribution of transcripts before and after de-redundancy】</td><td>file</td></tr>
    <tr data-tt-id='3-2-4' data-tt-parent-id='3-2'><td><span class='file'>Sample.unigene_distri.xls </td><td>【Statistical results of unigene length distribution】</td><td>file</td></tr>
    <tr data-tt-id='3-2-5' data-tt-parent-id='3-2'><td><span class='file'>Sample.unigene_distri.{png|pdf}</td><td>【unigene length distribution chart, png format】</td><td>file</td></tr>
    <tr data-tt-id='3-2-7' data-tt-parent-id='3-2'><td><span class='file'>Sample.cluster_stat.xls</td><td>【Statistical results of transcript cluster analysis】</td><td>file</td></tr>
    <tr data-tt-id='3-2-9' data-tt-parent-id='3-2'><td><span class='file'>Sample.cluster_distri.{png|pdf}</td><td>【Transcript cluster distribution map, pdf|png format】</td><td>file</td></tr>
    <tr data-tt-id='3-2-10' data-tt-parent-id='3-2'><td><span class='file'>Sample.cluster_unigene_detail.xls</td><td>【Transcript clustering detailed statistical results】</td><td>file</td></tr>



   <tr data-tt-id='4'><td><span class='folder'><a href='./04.SSR_CDS_TF'>04.SSR_CDS_TF</td><td>【SSR, CDS and TF prediction results】</td><td>folder</td></tr>  
   <tr data-tt-id='4-2' data-tt-parent-id='4'><td><span class='folder'>Sample</a></span></td><td>【sample name】</td><td>folder</td></tr>

    <tr data-tt-id='4-1' data-tt-parent-id='4'><td><span class='file'><a href='./04.SSR_CDS_TF/04.SSR_CDS_TF--ReadMe.pdf'>04.SSR_CDS_TF--ReadMe.pdf</a></span></td><td>【SSR_CDS_TF, delivery result directory description】</td><td>file</td></tr>
   <tr data-tt-id='4-2-1' data-tt-parent-id='4-2'><td><span class='folder'>CDS</a></span></td><td>【CDS Forecast Results】</td><td>table</td></tr>
   <tr data-tt-id='4-2-1-1' data-tt-parent-id='4-2-1'><td><span class='file'>Sample.ANGEL.cds</a></span></td><td>【CDS final prediction results, respectively cds, pep and utr sequences, fasta format】</td><td>file</td></tr>
   <tr data-tt-id='4-2-1-3' data-tt-parent-id='4-2-1'><td><span class='file'>Sample.cds_len.{png|svg}</a></span></td><td>【CDS Length Distribution Plot】</td><td>file</td></tr>
   <tr data-tt-id='4-2-1-4' data-tt-parent-id='4-2-1'><td><span class='file'>Sample.ANGEL.pep</a></span></td><td>【CDS final prediction results, respectively cds, pep and utr sequences, fasta format】</td><td>file</td></tr>
   <tr data-tt-id='4-2-1-5' data-tt-parent-id='4-2-1'><td><span class='file'>Sample.ANGEL.utr</a></span></td><td>【CDS final prediction results, respectively cds, pep and utr sequences, fasta format】</td><td>file</td></tr>

  <tr data-tt-id='4-2-2' data-tt-parent-id='4-2'><td><span class='folder'>SSR</a></span></td><td>【SSR prediction results】</td><td>table</td></tr>
  <tr data-tt-id='4-2-2-1' data-tt-parent-id='4-2-2'><td><span class='file'>Sample.polished_consensus.fasta.misa.xls</a></span></td><td>【SSR Analysis Results File】</td><td>file</td></tr>
  <tr data-tt-id='4-2-2-2' data-tt-parent-id='4-2-2'><td><span class='file'>Sample.SSR_motifs_distribution.{png|pdf}</a></span></td><td>【SSR distribution map】</td><td>file</td></tr>
  <tr data-tt-id='4-2-2-3' data-tt-parent-id='4-2-2'><td><span class='file'>Sample.summary_plot.xls</a></span></td><td>【SSR distribution map statistics file】</td><td>file</td></tr>


  <tr data-tt-id='4-2-3' data-tt-parent-id='4-2'><td><span class='folder'>TF</a></span></td><td>【Transcription factor identification results】</td><td>table</td></tr>
  <tr data-tt-id='4-2-3-1' data-tt-parent-id='4-2-3'><td><span class='file'>Sample.TF.{png|svg}</a></span></td><td>【Transcription factor frequency statistics】</td><td>file</td></tr>
  <tr data-tt-id='4-2-3-3' data-tt-parent-id='4-2-3'><td><span class='file'>Sample.transcription_factor.txt</a></span></td><td>【Transcription Factor Results (Plant) | Animal】</td><td>file</td></tr>
  <tr data-tt-id='4-2-3-4' data-tt-parent-id='4-2-3'><td><span class='file'>Sample.tf_stat.xls</a></span></td><td>【Statistical results of the top 30 transcription factor families annotated with the largest number of transcripts】</td><td>file</td></tr>



   <tr data-tt-id='5'><td><span class='folder'><a href='./05.LncRNA'>05.LncRNA</td><td>【LncRNA Analysis Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-1' data-tt-parent-id='5'><td><span class='folder'>Sample</a></span></td><td>【sample name】</td><td>table</td></tr>
  <tr data-tt-id='5-2' data-tt-parent-id='5'><td><span class='file'><a href='./05.LncRNA/05.LncRNA--ReadMe.pdf'>05.LncRNA--ReadMe.pdf</a></span></td><td>【LncRNA Delivery Results Catalog Instructions】</td><td>file</td></tr>
  <tr data-tt-id='5-1-2' data-tt-parent-id='5-1'><td><span class='file'>Sample.cnci.xls</a></span></td><td>【CNCI Software Coding Potential Prediction Results】</td><td>file</td></tr>
  <tr data-tt-id='5-1-3' data-tt-parent-id='5-1'><td><span class='file'>Sample.cpc.xls</a></span></td><td>【CPC Software Coding Potential Prediction Results】</td><td>file</td></tr>
  <tr data-tt-id='5-1-4' data-tt-parent-id='5-1'><td><span class='file'>Sample.pfam.xls</a></span></td><td>【pfam software coding potential prediction results】</td><td>file</td></tr>
  <tr data-tt-id='5-1-5' data-tt-parent-id='5-1'><td><span class='file'>Sample.plek.xls</a></span></td><td>【plek software coding potential prediction results】</td><td>file</td></tr>
  <tr data-tt-id='5-1-6' data-tt-parent-id='5-1'><td><span class='file'>Sample.lnc.xls</a></span></td><td>【Each coding potential prediction software predicts the results of LncRNA】</td><td>file</td></tr>
  <tr data-tt-id='5-1-1' data-tt-parent-id='5-1'><td><span class='file'>Sample.venn.{png|pdf}</a></span></td><td>【The Venn diagram of LncRNA predicted by each coding potential prediction software in pdf|png format】</td><td>file</td></tr>
  <tr data-tt-id='5-1-8' data-tt-parent-id='5-1'><td><span class='file'>Sample.length_stat.xls</a></span></td><td>【LncRNA and mRNA length statistics】</td><td>file</td></tr>
  <tr data-tt-id='5-1-9' data-tt-parent-id='5-1'><td><span class='file'>Sample.LncRNA_mRNA_density.{png|pdf}</a></span></td><td>【LncRNA and mRNA length density distribution map in pdf|png format】</td><td>file</td></tr>

  <tr data-tt-id='5-1-11' data-tt-parent-id='5-1'><td><span class='file'>Sample.lncRNA.fasta</a></span></td><td>【LncRNA sequence file, fasta format】</td><td>file</td></tr>


   <tr data-tt-id='6'><td><span class='folder'><a href='./06.Annotation'>06.Annotation</td><td>【Function Notes Directory】</td><td>folder</td></tr>
  <tr data-tt-id='6-1' data-tt-parent-id='6'><td><span class='folder'>Sample</a></span></td><td>【sample name】</td><td>table</td></tr>
  <tr data-tt-id='6-2' data-tt-parent-id='6'><td><span class='file'><a href='./06.Annotation/06.Annotation--ReadMe.pdf'>06.Annotation--ReadMe.pdf</a></span></td><td>【Annotation Delivery result catalog description】</td><td>file</td></tr>
  <tr data-tt-id='6-1-2' data-tt-parent-id='6-1'><td><span class='file'>Sample.anno.{png|svg}</a></span></td><td>【Statistical chart of annotation results of each database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-5' data-tt-parent-id='6-1'><td><span class='file'>Sample.anno_stat.xls</a></span></td><td>【Each database function comment statistics file】</td><td>file</td></tr>
  <tr data-tt-id='6-1-6' data-tt-parent-id='6-1'><td><span class='file'>Sample.anno.xls</a></span></td><td>【Functional annotation results of each transcript in each database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-7' data-tt-parent-id='6-1'><td><span class='file'>Sample.anno.venn.{png|pdf}</a></span></td><td>【Gene function annotation Venn diagram】</td><td>file</td></tr>

  <tr data-tt-id='6-1-8' data-tt-parent-id='6-1'><td><span class='folder'>NT</a></span></td><td>【NT Database Feature Annotation Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-8-1' data-tt-parent-id='6-1-8'><td><span class='file'>Sample.NT.filter.m8.xls</a></span></td><td>【The alignment with the highest score for the consensus sequence alignment】</td><td>file</td></tr>
  <tr data-tt-id='6-1-8-2' data-tt-parent-id='6-1-8'><td><span class='file'>Sample.filter.m8.anno.xls</a></span></td><td>【Consensus sequence and NT database comparison results Species information annotation results】</td><td>fiel</td></tr>


  <tr data-tt-id='6-1-9' data-tt-parent-id='6-1'><td><span class='folder'>NR</a></span></td><td>【NR Database Feature Annotation Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-9-1' data-tt-parent-id='6-1-9'><td><span class='file'>Sample.filter.m8.xls</a></span></td><td>【The highest-scoring alignment between the consensus sequence and the NR database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-9-2' data-tt-parent-id='6-1-9'><td><span class='file'>Sample.filter.m8.anno.xls</a></span></td><td>【Consensus sequence and NR database comparison result species information annotation】</td><td>file</td></tr>
  <tr data-tt-id='6-1-9-3' data-tt-parent-id='6-1-9'><td><span class='file'>Sample.NR.nr.species.xls</a></span></td><td>【NR database comparison annotation results species distribution list】</td><td>file</td></tr>
  <tr data-tt-id='6-1-9-4' data-tt-parent-id='6-1-9'><td><span class='file'>Sample.nr.anno.{png|pdf}</a></span></td><td>【Species distribution map of NR database comparison annotation results.pdfTOP20】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10' data-tt-parent-id='6-1'><td><span class='folder'>KEGG</a></span></td><td>【KEGG Database Functional Annotation Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-10-1' data-tt-parent-id='6-1-10'><td><span class='file'>Sample.KEGG.filter.m8.xls</a></span></td><td>【A list of the highest scoring results for consensus sequences compared to the KEGG database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10-2' data-tt-parent-id='6-1-10'><td><span class='file'>Sample.KEGG.filter.m8.anno.xls</a></span></td><td>【Consensus sequence and KEGG database alignment annotation results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10-5' data-tt-parent-id='6-1-10'><td><span class='file'>Sample.consensus.kegg.num.{png|pdf}</a></span></td><td>【KEGG pathway annotation gene number result picture pdf|png format】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10-6' data-tt-parent-id='6-1-10'><td><span class='file'>Sample.kegg.anno.stat.xls</a></span></td><td>【KEGG database annotation results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10-7' data-tt-parent-id='6-1-10'><td><span class='file'>Annotation_KO.html</a></span></td><td>【Annotate KO web page results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-10-8' data-tt-parent-id='6-1-10'><td><span class='folder'>src</a></span></td><td>【Web Directory Support Files】</td><td>file</td></tr>


  <tr data-tt-id='6-1-11' data-tt-parent-id='6-1'><td><span class='folder'>PFAM</a></span></td><td>【PFAM Database Functional Annotation Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-11-1' data-tt-parent-id='6-1-11'><td><span class='file'>Sample.pfam.anno.xls</a></span></td><td>【Protein annotation results of pfam database】</td><td>folder</td></tr>


  <tr data-tt-id='6-1-12' data-tt-parent-id='6-1'><td><span class='folder'>GO</a></span></td><td>【GO Database Function Annotation Results Directory】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-12-1' data-tt-parent-id='6-1-12'><td><span class='file'>Sample.GO.anno.xls</a></span></td><td>【List of GO annotation information】</td><td>file</td></tr>
  <tr data-tt-id='6-1-12-2' data-tt-parent-id='6-1-12'><td><span class='file'>Sample.GO_classification_count.xls</a></span></td><td>【GO database annotation statistical results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-12-3' data-tt-parent-id='6-1-12'><td><span class='file'>Sample.GO_classification_bar.{png|pdf}</a></span></td><td>【GO Annotated Taxonomy Map】</td><td>file</td></tr>

  <tr data-tt-id='6-1-13' data-tt-parent-id='6-1'><td><span class='folder'>Swiss-Prot</a></span></td><td>【Swiss-Prot database functional annotation results folder】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-13-1' data-tt-parent-id='6-1-13'><td><span class='file'>Sample.Swissprot.filter.m8.xls</a></span></td><td>【Consensus sequence alignment with the highest score in the SwissProt database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-13-2' data-tt-parent-id='6-1-13'><td><span class='file'>Sample.Swissprot.filter.m8.anno.xls</a></span></td><td>【Annotation information of SwissProt database blast comparison results】</td><td>file</td></tr>


  <tr data-tt-id='6-1-14' data-tt-parent-id='6-1'><td><span class='folder'>KOG</a></span></td><td>【KOG Database Functional Annotation Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-14-1' data-tt-parent-id='6-1-14'><td><span class='file'>Sample.KOG.filter.m8.xls</a></span></td><td>【The highest-scoring alignment between the consensus sequence and the KOG database】</td><td>file</td></tr>
  <tr data-tt-id='6-1-14-2' data-tt-parent-id='6-1-14'><td><span class='file'>Sample.KOG.filter.m8.anno.xls</a></span></td><td>【Protein classification information annotation of consensus sequence and KOG database comparison results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-14-3' data-tt-parent-id='6-1-14'><td><span class='file'>Sample.KOG.level.xls</a></span></td><td>【KOG database feature annotation Annotate information by KOG number】</td><td>file</td></tr>
  <tr data-tt-id='6-1-14-5' data-tt-parent-id='6-1-14'><td><span class='file'>Sample.KOG.class.{png|pdf}</a></span></td><td>【KOG database functional annotation protein classification map.pdf|png】</td><td>file</td></tr>
  <tr data-tt-id='6-1-14-5' data-tt-parent-id='6-1-14'><td><span class='file'>Sample.KOG.class.xls</a></span></td><td>【KOG database functional annotation protein classification 26 types of annotation information】</td><td>file</td></tr>


   <tr data-tt-id='7'><td><span class='folder'><a href='./07.Expression'>07.Expression</td><td>【Second-generation data quantification, difference and enrichment analysis results】</td><td>folder</td></tr>
    <tr data-tt-id='7-5' data-tt-parent-id='7'><td><span class='file'><a href='./07.Expression/07.Expression--ReadMe.pdf'>07.Expression--ReadMe.pdf</span></td><td>【07.Expression delivery result catalog description】</td><td>file</td></tr>
  <tr data-tt-id='7-1' data-tt-parent-id='7'><td><span class='folder'>Quantification</span></td><td>【Quantitative results】</td><td>folder</td></tr>
  <tr data-tt-id='7-1-1' data-tt-parent-id='7-1'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-1-1-1' data-tt-parent-id='7-1-1'><td><span class='file'>Sample.Readcount_FPKM.xls</span></td><td>【Statistics of gene expression levels for each sample】</td><td>file</td></tr>
  <tr data-tt-id='7-1-1-2' data-tt-parent-id='7-1-1'><td><span class='file'>Sample.FPKM_density_distribution.{png|pdf}</span></td><td>【FPKM density profile for each sample】</td><td>file</td></tr>
  <tr data-tt-id='7-1-2' data-tt-parent-id='7-1'><td><span class='file'>Mapping_rate.xls</span></td><td>【Statistics on the mapping (comparison) rate of the second-generation data of each sample】</td><td>file</td></tr>
  <tr data-tt-id='7-1-3' data-tt-parent-id='7-1'><td><span class='file'>FPKM_interval.xls</span></td><td>【Statistical table of the number of genes in different expression level intervals】</td><td>file</td></tr>
  <tr data-tt-id='7-1-4' data-tt-parent-id='7-1'><td><span class='file'>FPKM_interval.bar.{png|pdf}</span></td><td>【Histogram of FPKM interval distribution for different groups (samples)】</td><td>file</td></tr>
  <tr data-tt-id='7-1-5' data-tt-parent-id='7-1'><td><span class='file'>merged.readcount.xls</span></td><td>【Gene readcount value statistics】</td><td>file</td></tr>
  <tr data-tt-id='7-1-6' data-tt-parent-id='7-1'><td><span class='file'>merged.fpkm.xls</span></td><td>【Gene fpkm value statistics】</td><td>file</td></tr>
  <tr data-tt-id='7-1-7' data-tt-parent-id='7-1'><td><span class='file'>rowmeans_fpkm.xls</span></td><td>【Gene fpkm value statistics among sample groups】</td><td>file</td></tr>
  <tr data-tt-id='7-1-8' data-tt-parent-id='7-1'><td><span class='file'>FPKM_density_distribution.{png|pdf}</span></td><td>【FPKM density distribution plot of different groups (samples)】</td><td>file</td></tr>
  <tr data-tt-id='7-1-9' data-tt-parent-id='7-1'><td><span class='file'>FPKM_boxplot.{png|pdf}</span></td><td>【FPKM boxplot for different groupings (samples)】</td><td>file</td></tr>
  <tr data-tt-id='7-1-10' data-tt-parent-id='7-1'><td><span class='file'>Readcount_FPKM_Anno.xls</span></td><td>【Gene expression level and annotation statistics table】</td><td>file</td></tr>

  <tr data-tt-id='7-2' data-tt-parent-id='7'><td><span class='folder'>DiffExprAnalysis</span></td><td>【Difference Analysis Results】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-1' data-tt-parent-id='7-2'><td><span class='folder'>Correlation</span></td><td>【Correlation analysis results between second-generation samples】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-1-1' data-tt-parent-id='7-2-1'><td><span class='file'>cor_pearson.{png|pdf}</span></td><td>【Correlation coefficient heat map for all samples】</td><td>file</td></tr>
  <tr data-tt-id='7-2-1-2' data-tt-parent-id='7-2-1'><td><span class='file'>cor_pearson.xls</span></td><td>【Correlation coefficient matrix for all samples】</td><td>file</td></tr>
  <tr data-tt-id='7-2-1-3' data-tt-parent-id='7-2-1'><td><span class='file'>Sample1_vs_Sample2.scatter.{png|pdf}</span></td><td>【The scatter plot of the correlation between the gene expression of Sample1 and Sample2】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2' data-tt-parent-id='7-2'><td><span class='folder'>DiffExprAnalysis</span></td><td>【Difference Analysis Results】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-2-1' data-tt-parent-id='7-2-2'><td><span class='folder'>compare</span></td><td>【Each comparative group item】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-2-1-2' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.Differential_analysis_results.xls</span></td><td>【Differential analysis results for all genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-3' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEG.xls</span></td><td>【Differential Gene Results】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-4' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare. DEG_anno.xls</span></td><td>【Annotation results of differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-5' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEGlist.txt</span></td><td>【Differential gene list (differential gene id)】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-6' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEG_up.xls</span></td><td>【Results of up-regulated differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-7' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEGlist_up.txt</span></td><td>【List of up-regulated differential genes (up-regulated differential gene id)】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-8' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEG_down.xls</span></td><td>【Results of down-regulated differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-9' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.DEGlist_down.txt</span></td><td>【List of down-regulated differential genes (down-regulated differential gene id)】</td><td>file</td></tr>
  <tr data-tt-id='7-2-2-1-10' data-tt-parent-id='7-2-2-1'><td><span class='file'>compare.Volcanoplot.{png|pdf}</span></td><td>【Gene Differential Expression Analysis Volcano Plot】</td><td>file</td></tr>


  <tr data-tt-id='7-2-3' data-tt-parent-id='7-2'><td><span class='folder'>DEGcluster</span></td><td>【Differential gene cluster analysis results】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-3-1' data-tt-parent-id='7-2-3'><td><span class='file'>DEG_union_for_cluster.xls</span></td><td>【Union of all combined differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-2' data-tt-parent-id='7-2-3'><td><span class='file'>Hcluster_heatmap.{png|pdf}</span></td><td>【Hierarchical clustering heat map of differential gene expression】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-3' data-tt-parent-id='7-2-3'><td><span class='file'>Hcluster_heatmap.detail.pdf</span></td><td>【Differential gene expression hierarchical clustering heatmap (with differential gene geneid)】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-4' data-tt-parent-id='7-2-3'><td><span class='folder'>H_cluster</span></td><td>【Hierarchical cluster analysis results directory】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-3-4-1' data-tt-parent-id='7-2-3-4'><td><span class='file'>H_cluster.{png|pdf}</span></td><td>【hierarchical clustering plot】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-4-2' data-tt-parent-id='7-2-3-4'><td><span class='file'>subcluster_n</span></td><td>【text file, nth subclustering result file】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-5' data-tt-parent-id='7-2-3'><td><span class='folder'>K_means_cluster</span></td><td>【K-means cluster analysis results directory】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-3-5-1' data-tt-parent-id='7-2-3-5'><td><span class='file'>K_means_cluster.{png|pdf}</span></td><td>【K-means cluster plot】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-5-2' data-tt-parent-id='7-2-3-5'><td><span class='file'>subcluster_n</span></td><td>【text file, nth subclustering result file】</td><td>file</td></tr>
  <tr data-tt-id='7-2-3-6' data-tt-parent-id='7-2-3'><td><span class='folder'>SOM_cluster</span></td><td>【SOM Cluster Analysis Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-3-6-1' data-tt-parent-id='7-2-3-6'><td><span class='file'>SOM_cluster.{png|pdf}</span></td><td>【SOM cluster plot】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-3-6-2' data-tt-parent-id='7-2-3-6'><td><span class='file'>subcluster_n</span></td><td>【text file, nth subclustering result file】</td><td>folder</td></tr>

  <tr data-tt-id='7-2-5' data-tt-parent-id='7-2'><td><span class='folder'>VennDiagram</span></td><td>【venn graph results】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-5-1' data-tt-parent-id='7-2-5'><td><span class='folder'>venn_compare1_compare2_ compare3</span></td><td>【Venn Diagram Drawing Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-2-5-1-1' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3.DEG_Venn_diagram.{png|pdf}</span></td><td>【compare1, compare2, compare3 Venn diagram of differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-2-5-1-2' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3.DEG_up_Venn_diagram.{png|pdf}</span></td><td>【Venn diagram of compare1, compare2, compare3 up-regulated differential genes】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-3' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3.DEG_down_Venn_diagram.{png|pdf}</span></td><td>【Venn diagram of compare1, compare2, compare3 down-regulated differential genes】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-4' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_up-no-compare2_up-yes-compare3_up-no.txt</span></td><td>【Indicates up-regulated differential genes in compare2, non-up-regulated differential genes in compare1 and compare3, and so on for other files】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-5' data-tt-parent-id='7-2-5-1'><td><span class='file'>all.txt</span></td><td>【Differential gene ids in all venn plots】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-6' data-tt-parent-id='7-2-5-1'><td><span class='file'>all_up.txt</span></td><td>【Upregulated differential gene ids in all venn plots】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-7' data-tt-parent-id='7-2-5-1'><td><span class='file'>all_down.txt</span></td><td>【Downregulated differential gene ids in all venn plots】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-8' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3.Hcluster_heatmap.{png|pdf}</span></td><td>【Compare1, compare2, compare3 hierarchical clustering diagram of differential gene expression】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-9' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3_up.Hcluster_heatmap.{png|pdf}</span></td><td>【Hierarchical clustering diagram of compare1, compare2, compare3 up-regulated differential gene expression】</td><td>file</td></tr>
<tr data-tt-id='7-2-5-1-10' data-tt-parent-id='7-2-5-1'><td><span class='file'>compare1_compare2_compare3_down.Hcluster_heatmap.{png|pdf}</span></td><td>【Hierarchical clustering diagram of compare1, compare2, compare3 down-regulated differential gene expression】</td><td>file</td></tr>




  <tr data-tt-id='7-4' data-tt-parent-id='7'><td><span class='folder'>DEG_GOenrichment</span></td><td>【GO enrichment results of differential genes】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-1' data-tt-parent-id='7-4'><td><span class='folder'>ALL</span></td><td>【GO enrichment list of all differential genes】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-1-1' data-tt-parent-id='7-4-1'><td><span class='folder'>compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-1-1-1' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.DEG_GO_enrichment_result.xls</span></td><td>【GO enrichment results】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-2' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.GO_enrichment_result_up_down.xls</span></td><td>【GO enrichment list of differential genes】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-3' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.bar_graph_updown.{png|pdf} </span></td><td>【Categorical histogram of up- and down-regulated genes associated with enriched GOs】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-4' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.DEG_Enriched_GO_bp_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graphs of GO Enriched in Biological Processes】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-6' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.DEG_Enriched_GO_mf_DAG.{png|pdf} </span></td><td>【Directed Acyclic Graph Enriched in Molecular Functions for GO】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-7' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.DEG_Enriched_GO_classification_gene_count.txt</span></td><td>【Plot data for classification plots】</td><td>file</td></tr>
  <tr data-tt-id='7-4-1-1-8' data-tt-parent-id='7-4-1-1'><td><span class='file'>Compare.DEG_Enriched_GO_classification.{png|pdf} </span></td><td>【Classification plot of genes associated with enriched GOs】</td><td>file</td></tr>

  <tr data-tt-id='7-4-2' data-tt-parent-id='7-4'><td><span class='folder'>UP</span></td><td>【Upregulated Gene GO Enrichment Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-2-1' data-tt-parent-id='7-4-2'><td><span class='folder'>Compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-2-1-1' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare.DEG_GO_enrichment_result.xls</span></td><td>【GO enrichment results】</td><td>file</td></tr>
  <tr data-tt-id='7-4-2-1-4' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare.DEG_Enriched_GO_bp_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graphs of GO Enriched in Biological Processes】</td><td>file</td></tr>
  <tr data-tt-id='7-4-2-1-5' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare.DEG_Enriched_GO_cc_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graph of GO Enriched on Cellular Components】</td><td>file</td></tr>
  <tr data-tt-id='7-4-2-1-6' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare.DEG_Enriched_GO_mf_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graph Enriched in Molecular Functions for GO】</td><td>file</td></tr>
  <tr data-tt-id='7-4-2-1-7' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare..DEG_Enriched_GO_classification_gene_count.txt</span></td><td>【Data Enriched to GO】</td><td>file</td></tr>
  <tr data-tt-id='7-4-2-1-8' data-tt-parent-id='7-4-2-1'><td><span class='file'>Compare..DEG_Enriched_GO_classification.{png|pdf}</span></td><td>【Classification statistics of genes related to GO】</td><td>file</td></tr>


  <tr data-tt-id='7-4-3' data-tt-parent-id='7-4'><td><span class='folder'>DOWN</span></td><td>【Downregulated Gene GO Enrichment Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-3-1' data-tt-parent-id='7-4-3'><td><span class='folder'>Compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-4-3-1-1' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare.DEG_GO_enrichment_result.xls</span></td><td>【GO enrichment results】</td><td>file</td></tr>

  <tr data-tt-id='7-4-3-1-4' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare.DEG_Enriched_GO_bp_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graphs of GO Enriched in Biological Processes】</td><td>file</td></tr>
  <tr data-tt-id='7-4-3-1-5' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare.DEG_Enriched_GO_cc_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graph of GO Enriched on Cellular Components】</td><td>file</td></tr>
  <tr data-tt-id='7-4-3-1-6' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare.DEG_Enriched_GO_mf_DAG.{png|pdf}</span></td><td>【Directed Acyclic Graph Enriched in Molecular Functions for GO】</td><td>file</td></tr>
  <tr data-tt-id='7-4-3-1-7' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare..DEG_Enriched_GO_classification_gene_count.txt</span></td><td>【Data Enriched to GO】</td><td>file</td></tr>
  <tr data-tt-id='7-4-3-1-8' data-tt-parent-id='7-4-3-1'><td><span class='file'>Compare..DEG_Enriched_GO_classification.{png|pdf}</span></td><td>【Classification statistics of genes related to GO】</td><td>file</td></tr>

  <tr data-tt-id='7-5' data-tt-parent-id='7'><td><span class='folder'>DEG_KEGGenrichment</span></td><td>【Differential gene KEGG enrichment results】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-1' data-tt-parent-id='7-5'><td><span class='folder'>ALL</span></td><td>【KEGG enrichment directory of all differential genes】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-1-1' data-tt-parent-id='7-5-1'><td><span class='folder'>compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-1-1-1' data-tt-parent-id='7-5-1-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_result.xls</span></td><td>【KEGG PATHWAY enrichment results】</td><td>file</td></tr>
  <tr data-tt-id='7-5-1-1-2' data-tt-parent-id='7-5-1-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_top20.xls</span></td><td>【PATHWAY enrichment top 20 results file】</td><td>file</td></tr>
  <tr data-tt-id='7-5-1-1-3' data-tt-parent-id='7-5-1-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_scatterplot.{png|pdf}</span></td><td>【Scatter plot of enriched PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-1-1-5' data-tt-parent-id='7-5-1-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_add.xls_rendered_html_detail.html</span></td><td>【Enriched Pages for PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-1-1-6' data-tt-parent-id='7-5-1-1'><td><span class='folder'>src</span></td><td>【Supporting data directory for pages enriched with PATHWAY】</td><td>file</td></tr>

  <tr data-tt-id='7-5-2' data-tt-parent-id='7-5'><td><span class='folder'>UP</span></td><td>【Up-regulated gene KEGG enrichment catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-2-1' data-tt-parent-id='7-5-2'><td><span class='folder'>compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-2-1-1' data-tt-parent-id='7-5-2-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_result.xls</span></td><td>【KEGG PATHWAY enrichment results】</td><td>file</td></tr>
  <tr data-tt-id='7-5-2-1-2' data-tt-parent-id='7-5-2-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_top20.xls</span></td><td>【PATHWAY enrichment top 20 results file】</td><td>file</td></tr>
  <tr data-tt-id='7-5-2-1-3' data-tt-parent-id='7-5-2-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_scatterplot.{png|pdf}</span></td><td>【Scatter plot of enriched PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-2-1-5' data-tt-parent-id='7-5-2-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_add.xls_rendered_html_detail.html</span></td><td>【Enriched Pages for PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-2-1-6' data-tt-parent-id='7-5-2-1'><td><span class='folder'>src</span></td><td>【Supporting data directory for pages enriched with PATHWAY】</td><td>folder</td></tr>

  <tr data-tt-id='7-5-3' data-tt-parent-id='7-5'><td><span class='folder'>DOWN</span></td><td>【Downregulated Gene KEGG Enrichment Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-3-1' data-tt-parent-id='7-5-3'><td><span class='folder'>compare</span></td><td>【Compare Portfolio Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-5-3-1-1' data-tt-parent-id='7-5-3-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_result.xls</span></td><td>【KEGG PATHWAY enrichment results】</td><td>file</td></tr>
  <tr data-tt-id='7-5-3-1-2' data-tt-parent-id='7-5-3-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_top20.xls</span></td><td>【PATHWAY enrichment top 20 results file】</td><td>file</td></tr>
  <tr data-tt-id='7-5-3-1-3' data-tt-parent-id='7-5-3-1'><td><span class='file'>Compare.DEG_enriched_KEGG_pathway_scatterplot.{png|pdf}</span></td><td>【Scatter plot of enriched PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-3-1-5' data-tt-parent-id='7-5-3-1'><td><span class='file'>Compare.DEG_KEGG_pathway_enrichment_add.xls_rendered_html_detail.html</span></td><td>【Enriched Pages for PATHWAY】</td><td>file</td></tr>
  <tr data-tt-id='7-5-3-1-6' data-tt-parent-id='7-5-3-1'><td><span class='folder'>src</span></td><td>【Supporting data directory for pages enriched with PATHWAY】</td><td>folder</td></tr>


     </tbody>
      </table>    
    </div>
  
    <script src="http://www.jq22.com/jquery/jquery-1.10.2.js"></script>
    <script src="https://cdn.bootcss.com/jquery-treetable/3.2.0/jquery.treetable.js"></script>
    <script>
      $("#example-basic").treetable({ expandable: true });

      $("#example-basic-static").treetable();

      $("#example-basic-expandable").treetable({ expandable: true });

      $("#example-advanced").treetable({ expandable: true });

      // Highlight selected row
      $("#example-advanced tbody").on("mousedown", "tr", function() {
        $(".selected").not(this).removeClass("selected");
        $(this).toggleClass("selected");
      });

      // Drag & Drop Example Code
      $("#example-advanced .file, #example-advanced .folder").draggable({
        helper: "clone",
        opacity: .75,
        refreshPositions: true, // Performance?
        revert: "invalid",
        revertDuration: 300,
        scroll: true
      });

      $("#example-advanced .folder").each(function() {
        $(this).parents("#example-advanced tr").droppable({
          accept: ".file, .folder",
          drop: function(e, ui) {
            var droppedEl = ui.draggable.parents("tr");
            $("#example-advanced").treetable("move", droppedEl.data("ttId"), $(this).data("ttId"));
          },
          hoverClass: "accept",
          over: function(e, ui) {
            var droppedEl = ui.draggable.parents("tr");
            if(this != droppedEl[0] && !$(this).is(".expanded")) {
              $("#example-advanced").treetable("expandNode", $(this).data("ttId"));
            }
          }
        });
      });

      $("form#reveal").submit(function() {
        var nodeId = $("#revealNodeId").val()

        try {
          $("#example-advanced").treetable("reveal", nodeId);
        }
        catch(error) {
          alert(error.message);
        }

        return false;
      });
    </script>
  
 <form action="">
  <fieldset>
  <legend><p><button type="button" class="btn btn-default btn-sm">
          <span class="glyphicon glyphicon-exclamation-sign"></span>Tip:</button></p></legend>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Full-length transcripts: CCS sequences containing 5'-primer, 3'-primer and poly-A.</p>
    </li>
  </ul>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Transcript de-redundancy:Two or more third-generation sample data were combined and de-redundant, and then were transcribed using CD-HIT software (parameters: -c 0.95 -T 6 -G 0 -aL 0.90 -AL 100 -aS 0.99 -AS 30).For this cluster analysis, the comparative analysis of transcripts here was based on sequence similarity (>95%) and was not related to transcript expression levels.</p>
    </li>
  </ul>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Shared transcripts:When there are less than 2 third-generation samples, it's not possible to do the analysis of shared transcripts and the corresponding venn diagram.</p>
    </li>
  </ul>
    <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>CDS prediction: Starting with the three coding reading boxes in the forward direction, there are three possibilities, that is, reading and translating from the first to the third base in the forward direction (labeled 0,1,2). According to the prediction results, multiple CDS sequences (amino acid length ≥50) may be output in the end, and the CDS sequence with the longest transcript is selected as the best prediction result.</p>
    </li>
  </ul>
   <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Difference analysis: When the q value(P value adjust) was less than 0.05, the difference between the groups was considered significant. If there were no different genes between the groups, the corresponding GO and KEGG enrichment results were not shown.</p>
    </li>
  </ul>
 
  </fieldset>
  </form>
  <hr style="height:5px;border:none;border-top:5px ridge grey;" />
  <table align="center"  height="100%" style="font-size:12px font-weight:bold color:#F5F5F5" >
  <td "align="center" valign="top">If you have any questions, please consult project sales or operations, we will be happy to serve you. Sincere thanks for your trust in Novogene!</td>
  </body>
</html>

      
