
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>

<head>




<title> TESTXXXXXXXXXX Project Advance Analysis Report </title>

	
<meta http-equiv="content-type" content="text/html; charset=utf-8">
	<link rel="stylesheet" href="src/css/jquery.tocify.css" />
	<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
    <script src="src/js/jquery-1.8.3.min.js" type="text/javascript"></script>
    <script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
	<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>
	<link rel="stylesheet" type="text/css" href="src/css/base.css">
	<link rel="stylesheet" type="text/css" href="src/css/base_new.css">
	<link rel="StyleSheet" href="/src/" type="text/css">
	<link rel="stylesheet" type="text/css" href="src/js/fancybox/jquery.fancybox-1.3.4.css" media="screen" />
	<link rel="stylesheet" href="src/css/style.css" />
	<script src="src/js/scrollTop.js"></script>
	<script src="src/js/common.js"></script>
	<script src="src/js/test.js"></script>
	<script src="src/js/tree/tree.js"></script>
	<script src="src/js/fancybox/jquery.mousewheel-3.0.4.pack.js"></script>
	<script src="src/js/fancybox/jquery.fancybox-1.3.4.pack.js"></script>
	<link rel="stylesheet" href="src/css/jquery-ui.css" />
	<script src="src/js/dataTables.rowsGroup.js" type="text/javascript"></script>
  <script src="src/js/bootstrap.min.js" type="text/javascript"></script>
  <link rel="stylesheet" href="src/css/memu_style.css" media="screen" type="text/css" />
  <link href="src/css/cloth.css" rel="stylesheet" type="text/css" media="screen" />
  <script type="text/javascript" src="src/js/tablecloth.js"></script>

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />
<link rel="stylesheet" href="src/css/jquery.tocify.css" />
<link rel="stylesheet" href="src/css/style.css"/>
<link rel="stylesheet" href="src/css/base.css"/>
<link rel="stylesheet" href="src/css/gallery.css"/>
<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>
<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>
<script src="src/js/scrollTop.js" type="text/javascript"></script>

<script src="src/js/echarts.min.js" type="text/javascript"></script>
<script src="src/js/dagre-d3.min.js" type="text/javascript"></script>
<script src="src/js/d3.v3.min.js" type="text/javascript"></script>
<script src="src/js/jquery.flexslider.js" type="text/javascript"></script>
<script src="src/js/jquery.dataTables.js" type="text/javascript"></script>
<script src="src/js/material-scrollTop.js" type="text/javascript"></script>
<script src="src/js/bootstrap-select.min.js" type="text/javascript"></script>
<script src="src/js/bootstrap-switch.js" type="text/javascript"></script>
<script src="src/js/network.js" type="text/javascript"></script>
<script src="src/js/tableData.disease.js" type="text/javascript"></script>
<link href="css/base_new.css" type="text/css" rel="stylesheet">
<link href="src/css/flexslider.css" type="text/css" rel="stylesheet">
<script src="src/js/jquery.flexslider.js" type="text/javascript"></script>
<script src="src/js/jquery.dataTables.js" type="text/javascript"></script>
<link href="src/css/jquery.dataTables.css" type="text/css" rel="stylesheet">
<script src="src/js/cytoscape.min.js" type="text/javascript"></script>
<script src="src/js/jquery.cytoscape.js-panzoom.js" type="text/javascript"></script>
<!-- 增加导出excel功能 -->
<link rel="stylesheet" href="src/css/buttons.dataTables.min.css">
<script src="src/js/dataTables.buttons.min.js" type="text/javascript"></script>
<script src="src/js/jszip.min.js" type="text/javascript"></script>
<script src="src/js/buttons.html5.min.js" type="text/javascript"></script>
<link href="src/css/bootstrap-switch.css" rel="stylesheet" />
<link rel="stylesheet" type="text/css" href="src/css/font-awesome.min.css">
<link rel="stylesheet" type="text/css" href="src/css/buttons.css">
<script src="src/js/jquery.albumSlider.min.js"></script>




	<style>
		.longtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		.sublongtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		#menu_list li {
			height: 45px;
			line-height: 45px;
		}
	</style>


	<script>
		$(document).ready(function () {
			/*
			*   Examples - images
			*/

			$("a#example1").fancybox();

			$("a#example2").fancybox({
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'elastic'
			});
			$("a#example3").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("a#example4").fancybox({
				'opacity': true,
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'none'
			});

			$("a#example5").fancybox();

			$("a#example6").fancybox({
				'titlePosition': 'outside',
				'overlayColor': '#000',
				'overlayOpacity': 0.9
			});

			$("a#example7").fancybox({
				'titlePosition': 'inside'
			});

			$("a#example8").fancybox({
				'titlePosition': 'over'
			});

			$("a[rel=example_group]").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none',
				'titlePosition': 'over',
				'titleFormat': function (title, currentArray, currentIndex, currentOpts) {
					return '<span id="fancybox-title-over">Image ' + (currentIndex + 1) + ' / ' + currentArray.length + (title.length ? ' &nbsp; ' + title : '') + '</span>';
				}
			});

			/*
			*   Examples - various
			*/

			$("#various1").fancybox({
				'titlePosition': 'inside',
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("#various2").fancybox();

			$("#various3").fancybox({
				'width': '75%',
				'height': '75%',
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none',
				'type': 'iframe'
			});

			$("#various4").fancybox({
				'padding': 0,
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none'
			});
		});
	</script>

	<script type="text/javascript">

		function trans (img_div_id) {

			img_div = $("#" + img_div_id)

			img = $("#" + img_div_id + " img").first();

			if (img_div.hasClass("fullscreen-div")) {

				img_div.removeClass("fullscreen-div");

				img.removeClass("fullscreen-img");

			}

			else {

				img_div.addClass("fullscreen-div");

				img.addClass("fullscreen-img");

			}

		}



		function filter_display () {

			return $(this).css("display") != "none";

		}



		function show_default () {

			var list_group_id = "#" + $(this).attr("id");
			if ($(list_group_id + " a.active").css("display") == "none") {

				var list_tab = $(list_group_id + " a.active");

				var pan_id = list_tab.attr("href");

				$(pan_id).removeClass("active show");

				list_tab.removeClass("active");

			}

			if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

				$(list_group_id + " a").filter(filter_display).first().tab("show");

			}

		}

		$.fn.show_default = show_default



		function next (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

			while (present_tab.next().length == 1) {

				if (present_tab.next().css("display") != "none") {

					present_tab.next().tab("show");


					break;

				}

				else {

					present_tab = present_tab.next();

				}

			}



		}

		function prev (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

			while (present_tab.prev().length == 1) {

				if (present_tab.prev().css("display") != "none") {

					present_tab.prev().tab("show");

					break;

				}

				else {

					present_tab = present_tab.prev();

				}

			}





		}



		$(document).ready(function () {

			$(".form-control").on("keyup", function () {

				var value = $(this).val().toLowerCase();

				var input_box_id = "#" + $(this).attr("id");

				var list_group_id = input_box_id.replace("_InputList", "_list-tab");

				$(list_group_id + " a").filter(function () {

					$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

				});

				$(list_group_id).show_default();

			});
			$(".list-group").each(show_default);

		});
	</script>

	<style media="print">
		.noprint {
			DISPLAY: none;
		}
	</style>

	<div class="noprint">
		<div style="display: block;" id="goTopBtn">
			<a class="backtotop" title="回顶部"><img src="src/images/goTop.jpg" width="30" height="30" class="back-tip" /></a>
		</div>
	</div>

	<script>
		function displaySubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "block";
		}
		function hideSubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "none";
		} 
	</script>

	</script>
	<script type="text/javascript">
		/* show or hide element*/
		function hiddenShow (btnObj) {
			ctObj = btnObj.nextElementSibling;
			if (ctObj.style.display == 'inline') {
				ctObj.style.display = 'none';
				btnObj.innerHTML = "Show Annotation";
			} else {
				ctObj.style.display = 'inline';
				btnObj.innerHTML = "Hide Annotation";
			};
			return 0;
		}
	</script>

    <script type="text/javascript">
		/* show or hide element*/
		function hiddenQA (btnObj) {
			ctObj = btnObj.nextElementSibling;
			if (ctObj.style.display == 'inline') {
				ctObj.style.display = 'none';
				btnObj.innerHTML = "Show Q&A";
			} else {
				ctObj.style.display = 'inline';
				btnObj.innerHTML = "Hide Q&A";
			};
			return 0;
		}
	</script>


	<script>
		var Tree = new Array;
	// nodeId | parentNodeId | nodeName | nodeUrl
< {jtree} >
	</script>

	<script>
			$(function () {
				//纵向,默认,移动间隔2
				$('div.albumSlider').albumSlider();
				//横向设置
				$('div.albumSlider-h').albumSlider({ direction: 'h', step: 3 });
			});
	</script>


	<!-- 20200314 -->
	<script type="text/javascript">
		$(function () {
			var options = {
				selectors: "h2,h3,h4,h5",
				theme: "bootstrap",
				scrollTo: "60"
			};
			var toc = $("#toc").tocify(options).data("toc-tocify");
		});
	</script>

	<script type="text/javascript">
  let index = 0;
  function previousFun(e) {
    // a li ul -> div ul
    let transformStr = $(e).parent().parent().prev().children().css('transform');
    let transformMatrix = transformStr.slice(7, transformStr.length - 1).split(', ');
    const x = transformMatrix[transformMatrix.length - 2];
    console.log('x = ' + x);
    let liElements = $(e).parent().parent().prev().children().children();
    const liNum = liElements.length;
    index = index - 1;
    let temp = liElements.eq(index);
    const liWidth = temp.width();
    console.log('y = ' + liWidth);
    const y = parseInt(x) + parseInt(liWidth);
    if (y <= 20) {
      $(e).parent().parent().prev().children().css('transform', 'translate3d(' + y + 'px, 0px, 0px)');
    } else {
      index = index + 1;
    }
  }
  function nextFun(e) {
    // a li ul -> div ul
    let transformStr = $(e).parent().parent().prev().children().css('transform');
    let transformMatrix = transformStr.slice(7, transformStr.length - 1).split(', ');
    const x = transformMatrix[transformMatrix.length - 2];
    console.log('x = ' + x);                                
    let liElements = $(e).parent().parent().prev().children().children();
    const liNum = liElements.length;
    index = index + 1;
    let temp = liElements.eq(index);
    const liWidth = temp.width();
    console.log('y = ' + liWidth);
    const y = parseInt(x) - parseInt(liWidth);
    if (y >= parseInt(liNum) * parseInt(liWidth) * parseInt('-1')) {
      $(e).parent().parent().prev().children().css('transform', 'translate3d(' + y + 'px, 0px, 0px)');
    } else {
      index = index - 1;
    }
  }
  </script>
  
  <!--<script src='js/jquery-1.8.3.min.js'></script>-->
  <script src='src/js/jquery.flexslider.js'></script>
  <script src='src/js/jquery.dataTables.js'></script>
  <script src="src/js/material-scrollTop.js"></script>
  <script src="src/js/bootstrap.min.js"></script>
  <script src="src/js/bootstrap-select.min.js"></script>
  <script src="src/js/bootstrap-switch.js"></script>
  <script src="src/js/network.js"></script>
  <script src="src/js/tableData.disease.js"></script>
  <script src="src/js/d3.v3.min.js"></script>
  <script src="src/js/dagre-d3.min.js"></script>
  
  <script src="src/js/cytoscape.min.js"></script>
  <script src="src/js/jquery.cytoscape.js-panzoom.js"></script>
  <script src="src/js/final.js"></script>
  
  <script src="src/js/echarts.min.js"></script>
  <!-- 20220528 -->
  <script src="src/js/echarts-gl.min.js"></script>
  <script src="src/js/esl.js"></script>
  

	<!------------------------------------>




</head>

<body id="body">

	<!------------------------------------顶部图片logo---------------------------------------->
	<div id="toc" class="tocify">
	</div>


	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="60px" border="0" cellpadding="5" cellspacing="0"
			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
            
			<tr height="0%">

				<td width='0%'></td>

				<td><img height="60px" src="src/images/logo_new.png"></td>

				<td width="0%"></td>

				<td width="40%"></td>

				<td width="20%" style="text-align:right;padding:0px;margin:0px;color:black;font-size:8px;font-weight:50;">

					<a>
				</td>

				<td width="1%"></td>

			</tr>
            
		</table>
	</div>



	<!-- 不可删,删后是满页,不靠右  -->
	<div id="main">
		<!---------------------------------------menu overdd-------------------------------------------->


		<!---------------------------------------- 报告标题 ---------------------------------------------->


		<div id="page">
			<p><br /><br />

		
				<h1> <font size="6"> Human WES Standard Analysis Report </font></h1>
		



			<span style="display: block;text-align: center;font-size: 4em;font-family: Arial;color: #003f76;"></span>
			<br /><br /></p>
			<table class="gy">
			
				<tr>
					
						<td>Contract ID</td>
					
						<td>H20XXXXX</td>
					
				</tr>
			
				<tr>
					
						<td>Contract Name</td>
					
						<td>TESTXXXXXXXXXX</td>
					
				</tr>
			
				<tr>
					
						<td>Batch ID</td>
					
						<td>X202XXXXXX-Z01-F001</td>
					
				</tr>
			
				<tr>
					
						<td>Reference Genome and Version</td>
					
						<td>hg38</td>
					
				</tr>
			
				<tr>
					
						<td>Report Time</td>
					
						<td>20250106</td>
					
				</tr>
			
				<tr>
					
						<td>Reminder</td>
					
						<td>Partial results are presented in this report, while full results will be delivered in data release.Hyperlink of results in this report will be only valid in data release, after statement confirmation.</td>
					
				</tr>
			
				<br /><br /></p>
			</table>
		</div>


		<div id="page">
			<br />
			<h2>1 Introduction</h2>
			<h3>1.1 Background</h3>
			
			<p class="paragraph">Exome sequencing provides a cost-effective alternative to whole genome sequencing as it
				targets only coding region of the human genome responsible for a majority of known diseases and their
				related variants. Whether you are conducting studies in rare Mendelian disorders, complex diseases, cancer
				research, or human population studies, our company's comprehensive human whole exome sequencing provides a
				high-quality, affordable and convenient solution.</p>
			<p class="paragraph">Our company's bioinformatics analysis includes data QC, mapping with reference genome,
				SNP/InDel, somatic SNP/InDel/CNV Calling, statistics and annotation. Our company utilizes prevalent software in
				bioinformatics analysis, e.g. BWA, Sambamba, GATK, etc.</p>
			<p class="paragraph">In particular, the bioinformatics pipeline applies the exome aggregation consortium
				(ExAC). ExAC dataset spans 60,706 unrelated individuals representing various specific diseases, for
				tremendous genetic studies across human population. The scalable database greatly facilitates the research
				of disease pathogenesis.</p>
			<br />
			<p class="paragraph">Workflow is shown as follows:</p>
			<p class="center"> <img class="middle" src="src/images/hWES_Project_Workflow.png" /> </p>
			<br>
			
			<p class="name_fig">Figure 1 Project workflow </p>
			
			

			
			<br>	



		<!----------------------------------------样品信息----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Sample information"></a>
				<hr />
			</p><br />
			<h3>1.2 Sample Information</h3>
			<p class="center">
			<p class="name_fig">Table 1 Sample information</p>
			<div id="tb">
				<table class="gy">
				
					
					<tr>
						
							<th>PatientID</th>
						
							<th>SampleID</th>
						
							<th>ID</th>
						
							<th>Type</th>
						</tr>
					
				
					
					<tr>
						
							<td>p1</td>
						
							<td>YXHF</td>
						
							<td>FKDO230317194-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>p1</td>
						
							<td>YXHM</td>
						
							<td>FKDO230317193-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>p1</td>
						
							<td>YXHB</td>
						
							<td>FKDO230317195-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>p1</td>
						
							<td>YXHP</td>
						
							<td>FKDO230317192-1A</td>
						
							<td>P</td>
						</tr>
					
				
				</table>
			</div>
			</p>
		

			<p class="tremark" align="left">Type: sample type (N: normal; T: tumor; P: patient; U: unknown)</p>

			
		</div>
	</div>



		<!----------------------------------------实验流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Experimental procedure"></a>
				<hr />
			</p><br />
			<h2>2 Library Construction and Sequencing</h2>
			<a name="样品检测"></a>
			<br />
			<h3>2.1 Sample Quality Control</h3>
			<p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;Methods of sample quality control refer to QC report.</p>
			<br />

			<a name="建库捕获"></a>

			<h3>2.2 Library Construction, Quality Control and Sequencing</h3>



			
			<p class="paragraph">The genomic DNA was randomly sheared into short fragments with the size of 180-280 bp. The
				obtained fragments were end repaired, A-tailed, and further ligated with Illumina adapters. The fragments
				with adapters were PCR amplified, size selected, and purified. Hybridization capture of libraries was
				proceeded according to the following procedures. Briefly, the prepped libraries were hybridized in the
				buffer with biotin-labeled probes, and magnetic beads with streptavidin were used to capture the exons of
				genes. Subsequently, non-hybridized fragments were washed out and probes were digested. The captured
				libraries were enriched by PCR amplification.</p>
			<p class="paragraph">The experimental procedures of DNA library preparation are shown in Figure 2.</p>
			<br>
			<p class="center">
				<img style="width: 200px;" src="src/images/WES_library_construction.png" />
			</p>
			<p class="name_fig">Figure 2 Workflow of library construction</p>
			<p class="paragraph">The library was checked with Qubit and real-time PCR for quantification and bioanalyzer
				for size distribution detection. Quantified libraries were pooled and sequenced on Illumina platforms
				with PE150 strategy, according to the effective library concentration and data amount required.</p>
			

			

			

			


			
			<p class="paragraph">Method details involved in the project are available at <a	href="src/method/WES_method.pdf" title="Click to open" target="_blank">method.</a></p>
			



			<a name="上机测序"></a>
		</div>

		<!----------------------------------------生物信息分析流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<br />
				<a name="Bioinformatics analysis pipeline"></a>
				<hr />
			</p><br />

			<h2>3 Bioinformatics Analysis Pipeline</h2>

			<p class="paragraph">Bioinformatics analysis includes:</p>
			<p class="paragraph">1) Data quality control: filtering out reads containing adapters or with low quality;</p>
			<p class="paragraph">2) Alignment with reference genome, statistics of sequencing depth and coverage;</p>

			
			<p class="paragraph">3) SNP/InDel calling, annotation and statistics;</p>
			
			
			

			
			<p class="paragraph">The flowchart below depicts the pipeline of data QC and bioinformatics analysis we
				used. </p>
			<p class="center">
				<img style="width: 500px;" src="src/images/primary_workflow_for_WES_demo.png" />
			</p>
			
			
			



			<p class="name_fig">Figure 3 Bioinformatics analysis pipeline</p>

		
			
			<p class="paragraph">Method details involved in the project are available at <a href="Method/WES_method.pdf"
				title="Click to open" target="_blank">method.</a></p>
			

		</div>
		<!----------------------------------------------- 原始序列数据 --------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Analysis Result"></a>
				<hr />
			</p><br />

			<h2>4 Analysis Result</h2>
			<a name="Raw data"></a>
			<br />
			<h3>4.1 Raw Data</h3>

			<p class="paragraph">The original fluorescence image files obtained from sequencing platform are transformed to
				short reads (Raw data) by base calling. These short reads are recorded in FASTQ (.fq) format files, which
				contain sequence information (reads) and corresponding base quality.</p>

			<p class="paragraph">Each read in FASTQ format is stored in four lines as follows: </p>
			<p class="seq">
				
				@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG <br />
				GCTCTTTGCCCTTCTCGTCGAAAATTGTCTCCTCATTCGAAACTTCTCTGT <br />
				+ <br />
				@@CFFFDEHHHHFIJJJ@FHGIIIEHIIJBHHHIJJEGIIJJIGHIGHCCF </p>
				<p class="paragraph">Line 1 begins with a '@' character which is followed by a sequence identifier and an
				optional description. </p>
				<p class="paragraph">Line 2 shows the sequenced bases. </p>
				<p class="paragraph">Line 3 begins with a '+' character and is optionally followed by the same sequence
				identifier. </p>
				<p class="paragraph">Line 4 encodes the sequencing quality for each base in line 2, and contains the same number
				of characters as bases in line 2. </p>
				<p class="paragraph">Illumina sequence identifier details: </p>
				<div id="tb">
					<p class="center">
					<table class="gy">
						<tr>
							<td>EAS139</td>
							<td>Unique instrument name</td>
						</tr>
						<tr>
							<td>136</td>
							<td>Run ID</td>
						</tr>
						<tr>
							<td>FC706VJ</td>
							<td>Flowcell ID</td>
						</tr>
						<tr>
							<td>2</td>
							<td>Flowcell lane</td>
						</tr>
						<tr>
							<td>2104</td>
							<td>Tile number within the flowcell lane</td>
						</tr>
						<tr>
							<td>15343</td>
							<td>'x'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>197393</td>
							<td>'y'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>1</td>
							<td>Member of a pair, 1 or 2 (paired-end or mate-pair reads only)</td>
						</tr>
						<tr>
							<td>Y</td>
							<td>Y if the read fails filter (read is bad), otherwise N</td>
						</tr>
						<tr>
							<td>18</td>
							<td>0 when none of the control bits are on, Otherwise it is an even number</td>
						</tr>
						<tr>
							<td>ATCACG</td>
							<td>Index sequence</td>
						</tr>
					</table>
					</p>
				</div>
				

			<p class="paragraph">The ASCII value of each character at the fourth line minus 33 equals to the Phred-scaled
				quality value of the corresponding sequenced base in the second line. The relationship between sequencing
				error rate (e) and base quality value (Qphred) can be expressed by the following equation: </p>
			<p class="center">Qphred = -10log10(e)</p>
			<p class="paragraph">The table below shows examples of corresponding values among sequencing error rate (e),
				base quality value (Qphred) and characters. </p>
			<div id="tb">
				<p class="center">
				<table class="gy">
					<tr>
						<th>Sequencing error rate</th>
						<th>Sequencing quality value</th>
						<th>Corresponding character</th>
					</tr>
					<tr>
						<td>5%</td>
						<td>13</td>
						<td>.</td>
					</tr>
					<tr>
						<td>1%</td>
						<td>20</td>
						<td>5</td>
					</tr>
					<tr>
						<td>0.1%</td>
						<td>30</td>
						<td>?</td>
					</tr>
					<tr>
						<td>0.01%</td>
						<td>40</td>
						<td>I</td>
					</tr>
				</table>
				</p>
			</div>

			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Quality control"></a>
					<hr />
				</p><br />
				<h3>4.2 Quality Control</h3>
				<a name="Sequencing data filtration"></a>
				<br />
				<h4>4.2.1 Sequencing Data Filtration</h4>
				<p class="paragraph">Sequencing artifacts, including reads containing adapter contamination, low-quality
					nucleotides and unrecognizable nucleotides (N), apparently brings ambiguity for the subsequent
					bioinformatics analysis. Hence quality control is an essential step and applied to guarantee the
					meaningful analysis. All of downstream analysis will be based on clean reads that pass quality control.
				</p>
				<p class="paragraph">Our company performed quality control according to following procedures: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1) Discard paired-end reads if either one
					read contains adapter contamination (>10 nucleotides aligned to the adapter, allowing ≤ 10% mismatches);
					<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2) Discard paired-end reads if more than 10% of bases are
					uncertain (read as N) in either one read; <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3) Discard paired-end reads if the proportion of
					low-quality (Phred quality &lt; 5) bases is over 50% in either one read. </p>
				<p class="paragraph">DNA-Seq Adapter (Adapter, Oligonucleotide sequences for DNA Sample Prep Kits)
					information: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCT<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P7 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_class_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_class_nav-tabContent">
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3.pie3d.png" /></div>
							
						</div>

					</div>

					<div class="col-1">
						<div id="gallery_figure_class_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_class_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_class_list-tab" role="tablist">

								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3">YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3">YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3">YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3">YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3">YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3">YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3">YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3">YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3</a>
							
							


						</div>
					</div>
				</div>
			   </p>


				<p class="name_fig">Figure 4.1 Raw data filtration result</p>
				<br />

				</script>
				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
					<p class="tremark">
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Clean Reads: read pairs that passed quality
						control; <br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Containing N: read pairs in either one read of
						which more than 10% of bases are uncertain;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Low Quality: read pairs in either one read of
						which the proportion of low-quality bases is over 50%;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Adapter Related: read pairs that contain adapter
						contamination in either one read.</p>
				</div>
				<br />
				<br />

				<a name="Sequencing error rate examination"></a>
				<br />


				<h4>4.2.2 Examination of Sequencing Error Rate </h4>
				<p class="paragraph">Sequencing error rate and base quality can be affected by various factors such as
					sequencing platform, chemical reagent and sample quality. Due to the consumption of chemical reagents,
					error rate is increasing with the read extension, which is a common feature of the high throughput
					sequencing platform.</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_error_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_error_nav-tabContent">

							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3.Error.png" /></div>
							


						</div>
					</div>
					<div class="col-1">
						<div id="gallery_figure_error_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_error_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_error_list-tab" role="tablist">
							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3">YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3">YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3">YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3">YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3">YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3">YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3" role="tab"
								aria-controls="YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3">YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3" role="tab"
								aria-controls="YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3">YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3</a>

							

						</div>
					</div>
				</div>
			   </p>

				<p class="name_fig">Figure 4.2 Sequencing error rate distribution</p>
				<p class="premark">The x-axis represents position in reads, and the y-axis represents the average error rate
					of bases of all reads at a position. </p>


				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="GC content distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.3 GC Content Distribution</h4>

					<p class="paragraph">Evaluation of GC content distribution aims to check the potential of AT/GC
						separation phenomenon, which maybe resulted from sample contamination, sequencing bias or errors in
						library preparation. </p>
					<p class="paragraph">In theory, GC or AT content should be constant along read positions. But it is
						common to see that the first 6 to 7 bases in both reads (read1 & read2) fluctuate in GC content, due
						to primer amplification bias and some other reasons. </p>

					<p class="center">
					<div class="row gallery-row">
						<div class="col-1">
							<div id="gallery_figure_gc_button_prev" class="button_prev" onclick="prev(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_prev">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-6">
							<div class="tab-content" id="gallery_figure_gc_nav-tabContent">

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3.GC.png" /></div>

								


							</div>
						</div>
						<div class="col-1">
							<div id="gallery_figure_gc_button_next" class="button_next" onclick="next(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_next">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-4">
							<input class="form-control" id="gallery_figure_gc_InputList" type="text" placeholder="Search..">
							<div class="list-group" id="gallery_figure_gc_list-tab" role="tablist">

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
									role="tab"
									aria-controls="YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3">YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
									role="tab"
									aria-controls="YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3">YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
									role="tab"
									aria-controls="YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3">YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
									role="tab"
									aria-controls="YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3">YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
									role="tab"
									aria-controls="YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3">YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
									role="tab"
									aria-controls="YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3">YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
									role="tab"
									aria-controls="YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3">YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
									role="tab"
									aria-controls="YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3">YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3</a>

								

							</div>
						</div>
					</div>
				   </p>
					<p class="name_fig">Figure 4.3 GC content distribution</p>

					<p class="premark">The x-axis represents the position in reads, and the y-axis represents the percentage
						of each type of bases (A, T, G, C); different bases can be distinguished by different colors.</p>
				</div>

				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequencing quality distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.4 Sequencing Quality Distribution</h4>

					<p class="paragraph">The Phred-scaled quality scores of most bases should be greater than 20, it is also
						required for downstream analysis. Base quality decreases along reads is common, which is an inherent
						circumstance of next generation sequencing. </p>
					<p class="center">
					

						<div class="row gallery-row">
							<div class="col-1">
								<div id="gallery_figure_qm_button_prev" class="button_prev" onclick="prev(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_prev">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-6">
								<div class="tab-content" id="gallery_figure_qm_nav-tabContent">
	
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3.QM.png"></div>
									
									
									
								</div>
							</div>
							<div class="col-1">
								<div id="gallery_figure_qm_button_next" class="button_next" onclick="next(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_next">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-4">
								<input class="form-control" id="gallery_figure_qm_InputList" type="text" placeholder="Search..">
								<div class="list-group" id="gallery_figure_qm_list-tab" role="tablist">
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3"
										role="tab"
										aria-controls="YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3">YXHF_FKDO230317194-1A_AH5LKHDSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3"
										role="tab"
										aria-controls="YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3">YXHF_FKDO230317194-1A_AH5MH2DSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3"
										role="tab"
										aria-controls="YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3">YXHM_FKDO230317193-1A_AH5MH2DSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3"
										role="tab"
										aria-controls="YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3">YXHM_FKDO230317193-1A_AH5LKHDSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3"
										role="tab"
										aria-controls="YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3">YXHB_FKDO230317195-1A_AH5MH2DSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3"
										role="tab"
										aria-controls="YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3">YXHB_FKDO230317195-1A_AH5LKHDSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3"
										role="tab"
										aria-controls="YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3">YXHP_FKDO230317192-1A_AH5MH2DSX7-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3"
										role="tab"
										aria-controls="YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3">YXHP_FKDO230317192-1A_AH5LKHDSX7-new_L3</a>
	
									
	
						
								</div>
							</div>
						</div>
					   </p>	
				</div>

				<p class="name_fig">Figure 4.4 Sequencing quality distribution</p>
				<p class="premark">The x-axis represents the position in reads, and the y-axis represents the average
					quality score of bases of all reads at a position.</p>

				<p class="premark">&nbsp</p>
				</p>


				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Statistics of sequencing quality"></a>
						<hr />
					</p><br />
					<h4>4.2.5 Statistics of Sequencing Quality</h4>

					<P class="paragraph">According to the sequencing feature of sequencing platforms, our company requires the
						value of Q30 (the percentage of bases with Phred-scaled quality scores is greater than 30) should be
						above 85% for paired-end sequencing data.</p>

					<p class="name_fig">Table 4.1 Overview of data production quality</p>
					<div id="tb">
						<table class="gy">
							
							<tr>
								<th>Sample name</th>
								<th>NovoID</th>
								<th>Flowcell/Lane</th>
								<th>Raw reads</th>
								<th>Raw data(G)</th>
								<th>Effective(%)</th>
								<th>Error(%)</th>
								<th>Q20(%)</th>
								<th>Q30(%)</th>
								<th>GC(%)</th>
							</tr>
							

						
							
								
									<tr>
										<td style="text-align:left">YXHF</td>
										<td>FKDO230317194-1A</td>
										<td>AH5LKHDSX7-new_L3</td>
										<td>13730012</td>
										<td rowspan=2>11.4</td>
										<td>99.01</td>
										<td>0.02</td>
										<td>98.09</td>
										<td>94.65</td>
										<td>51.73</td>
									</tr>
								
							
								
									<tr>
										<td style="text-align:left">YXHF</td>
										<td>FKDO230317194-1A</td>
										<td>AH5MH2DSX7-new_L3</td>
										<td>24204995</td>
										<td>99.01</td>
										<td>0.03</td>
										<td>97.36</td>
										<td>92.87</td>
										<td>51.75</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">YXHM</td>
										<td>FKDO230317193-1A</td>
										<td>AH5LKHDSX7-new_L3</td>
										<td>15469255</td>
										<td rowspan=2>12.6</td>
										<td>99.32</td>
										<td>0.03</td>
										<td>97.83</td>
										<td>94.08</td>
										<td>51.78</td>
									</tr>
								
							
								
									<tr>
										<td style="text-align:left">YXHM</td>
										<td>FKDO230317193-1A</td>
										<td>AH5MH2DSX7-new_L3</td>
										<td>26500880</td>
										<td>99.34</td>
										<td>0.03</td>
										<td>97.06</td>
										<td>92.32</td>
										<td>51.71</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">YXHB</td>
										<td>FKDO230317195-1A</td>
										<td>AH5LKHDSX7-new_L3</td>
										<td>16948091</td>
										<td rowspan=2>13.8</td>
										<td>99.17</td>
										<td>0.03</td>
										<td>97.89</td>
										<td>94.22</td>
										<td>51.98</td>
									</tr>
								
							
								
									<tr>
										<td style="text-align:left">YXHB</td>
										<td>FKDO230317195-1A</td>
										<td>AH5MH2DSX7-new_L3</td>
										<td>29033087</td>
										<td>99.19</td>
										<td>0.03</td>
										<td>97.12</td>
										<td>92.47</td>
										<td>51.99</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">YXHP</td>
										<td>FKDO230317192-1A</td>
										<td>AH5MH2DSX7-new_L3</td>
										<td>27629338</td>
										<td rowspan=2>13.1</td>
										<td>98.79</td>
										<td>0.03</td>
										<td>97.02</td>
										<td>92.26</td>
										<td>51.80</td>
									</tr>
								
							
								
									<tr>
										<td style="text-align:left">YXHP</td>
										<td>FKDO230317192-1A</td>
										<td>AH5LKHDSX7-new_L3</td>
										<td>16139292</td>
										<td>98.76</td>
										<td>0.03</td>
										<td>97.73</td>
										<td>93.87</td>
										<td>51.84</td>
									</tr>
								
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">

							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) NovoID: NovoID for the sample.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Flowcell/Lane: The flowcell ID and lane
							number.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Raw reads: The number of sequencing reads
							pairs; four lines would be considered as one unit according to the format of FASTQ.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Raw data: The original sequencing
							data.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) Effective: The percentage of clean reads
							in all raw reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) Error: The average error rate of all
							bases on read1 and read2; the error rate of a base is obtained from equation 1.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) Q20: The percent of bases with
							phred-scaled quality scores greater than 20.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) Q30: The percent of bases with
							phred-scaled quality scores greater than 30.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) GC content: The percentage of G and C in
							the all bases.<br /></p>
					</div>
				
				</div>
				<!--
	</div>
	</div>
	-->


				<!----------------------------------------- 测序深度,覆盖度统计 -------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequence alignment"></a>
						<hr />
					</p><br />
					<h3>4.3 Sequence Alignment</h3>
					<P class="paragraph">Burrows-Wheeler Aligner (BWA) (Li <I><I>et al.</I></I>, 2018) was utilized to map the paired-end
						clean reads to the human reference genome. The original mapping result can be obtained in BAM format.
						Sambamba (Tarasov <I>et al.</I>, 2015) is used for sorting the BAM files, and Picard (The Quick Start is
						available online: https://broadinstitute.github.io/picard/) is utilized to mark duplicate reads.</p>
					<P class="paragraph">Final BAM files can be obtained after these steps. Then the coverage and depth will
						be computed based on BAM.</p>
					<br />
					<h4>4.3.1 Sequencing Depth & Coverage Distribution</h4>
					<a name="测序深度,覆盖度分布"></a>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/depth.png" />
					<p class="name_fig">Figure 4.5 Sequencing depth</p>
					<!-- <p class="name_fig">Figure 3.1 Sequencing depth</p>-->
					<p class="premark">The left figure shows sequencing depth distribution of all bases in each sample. The
						x-axis represents the sequencing depth, and the y-axis represents the fraction of bases with the given
						sequencing depth. <br />
						The right figure shows accumulative sequencing depth distribution of all bases in each sample. The
						x-axis represents the accumulative sequencing depth, and the y-axis represents the fraction of bases
						above the given sequencing depth.
					</p>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/covByChr0.png" />

					<p class="name_fig">Figure 4.6 Average sequencing depth (bar plot) and coverage (dot-line plot) in each
						chromosome</p>
					<!--<p class="name_fig">Figure 3.2　Average sequencing depth (bar plot) and coverage (dot-line plot) in each chromosome</p>-->
					<p class="premark">The x-axis represents the chromosome; the left y-axis represents the average depth;
						the right y-axis represents the coverage (proportion of covered bases). </p>
					<p class="premark">&nbsp</p>
				</div>

				<div id="page">
					<h4>4.3.2 Statistics of Mapping, Coverage and Depth</h4>
					<a name="覆盖度统计结果"></a>
					<p class="center">
					<p class="name_fig">Table 4.2 Statistics of mapping, coverage and depth in each sample</p>
					<!--<p class="name_fig">Table 3　Statistics of mapping, coverage and depth in each sample</p>-->
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
									
									<th>Sample name</th>
									
								
									
									<th>YXHF</th>
									
								
									
									<th>YXHM</th>
									
								
									
									<th>YXHB</th>
									
								
									
									<th>YXHP</th>
									
								
							</tr>
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Total</td>
										
									
										
										<td>75117166 (100%)</td>
										
									
										
										<td>83379130 (100%)</td>
										
									
										
										<td>91208202 (100%)</td>
										
									
										
										<td>86468072 (100%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Duplicate</td>
										
									
										
										<td>12603157 (16.79%)</td>
										
									
										
										<td>14238608 (17.09%)</td>
										
									
										
										<td>17425030 (19.12%)</td>
										
									
										
										<td>16695122 (19.32%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Mapped</td>
										
									
										
										<td>75064562 (99.93%)</td>
										
									
										
										<td>83303349 (99.91%)</td>
										
									
										
										<td>91122763 (99.91%)</td>
										
									
										
										<td>86391770 (99.91%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Properly mapped</td>
										
									
										
										<td>74577914 (99.28%)</td>
										
									
										
										<td>82793532 (99.30%)</td>
										
									
										
										<td>90518948 (99.24%)</td>
										
									
										
										<td>85844730 (99.28%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">PE mapped</td>
										
									
										
										<td>75021104 (99.87%)</td>
										
									
										
										<td>83245290 (99.84%)</td>
										
									
										
										<td>91057834 (99.84%)</td>
										
									
										
										<td>86331616 (99.84%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">SE mapped</td>
										
									
										
										<td>86916 (0.12%)</td>
										
									
										
										<td>116118 (0.14%)</td>
										
									
										
										<td>129858 (0.14%)</td>
										
									
										
										<td>120308 (0.14%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr</td>
										
									
										
										<td>302504 (0.40%)</td>
										
									
										
										<td>285474 (0.34%)</td>
										
									
										
										<td>357050 (0.39%)</td>
										
									
										
										<td>342464 (0.40%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr ((mapQ>=5))</td>
										
									
										
										<td>256605 (0.34%)</td>
										
									
										
										<td>240879 (0.29%)</td>
										
									
										
										<td>303003 (0.33%)</td>
										
									
										
										<td>295809 (0.34%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Initial_bases_on_target</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Initial_bases_on_or_near_target</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Total_effective_yield(Mb)</td>
										
									
										
										<td>11220.79</td>
										
									
										
										<td>12448.90</td>
										
									
										
										<td>13617.07</td>
										
									
										
										<td>12909.63</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Effective_yield_on_target(Mb)</td>
										
									
										
										<td>7934.30</td>
										
									
										
										<td>8483.89</td>
										
									
										
										<td>9475.07</td>
										
									
										
										<td>9295.33</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_effective_bases_on_target</td>
										
									
										
										<td>70.71%</td>
										
									
										
										<td>68.15%</td>
										
									
										
										<td>69.58%</td>
										
									
										
										<td>72.00%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_effective_bases_on_or_near_target</td>
										
									
										
										<td>91.67%</td>
										
									
										
										<td>91.23%</td>
										
									
										
										<td>91.64%</td>
										
									
										
										<td>92.18%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Average_sequencing_depth_on_target</td>
										
									
										
										<td>131.13</td>
										
									
										
										<td>140.21</td>
										
									
										
										<td>156.59</td>
										
									
										
										<td>153.62</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Bases_covered_on_target</td>
										
									
										
										<td>60292701</td>
										
									
										
										<td>60170337</td>
										
									
										
										<td>60320486</td>
										
									
										
										<td>60308717</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage_of_target_region</td>
										
									
										
										<td>99.64%</td>
										
									
										
										<td>99.44%</td>
										
									
										
										<td>99.69%</td>
										
									
										
										<td>99.67%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_100x</td>
										
									
										
										<td>52.38%</td>
										
									
										
										<td>55.30%</td>
										
									
										
										<td>59.81%</td>
										
									
										
										<td>59.38%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_50x</td>
										
									
										
										<td>78.70%</td>
										
									
										
										<td>80.44%</td>
										
									
										
										<td>82.74%</td>
										
									
										
										<td>82.36%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_20x</td>
										
									
										
										<td>93.39%</td>
										
									
										
										<td>93.85%</td>
										
									
										
										<td>94.72%</td>
										
									
										
										<td>94.52%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_10x</td>
										
									
										
										<td>97.15%</td>
										
									
										
										<td>97.21%</td>
										
									
										
										<td>97.71%</td>
										
									
										
										<td>97.62%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_4x</td>
										
									
										
										<td>98.96%</td>
										
									
										
										<td>98.82%</td>
										
									
										
										<td>99.14%</td>
										
									
										
										<td>99.10%</td>
										
									
									</tr>
								
							
						



						</table>
					</div>
					</p>

					
					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of clean reads.
							The below indexes are calculated based on the clean reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Duplicate: The number of duplicated reads
							(percentage: duplicated reads/clean reads).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Mapped: The number of reads that mapped
							to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Properly mapped: The number of reads that
							mapped to the reference genome and within the expected insert size (percentage). <br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) PE mapped: The number of pair-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) SE mapped: The number of single-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) With mate mapped to a different chr: The
							number of reads with mate reads mapped to different chromosomes (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) With mate mapped to a different chr (mapQ
							>= 5): The number of reads with mate reads mapped to different chromosomes and the MAQ > 5.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) Initial_bases_on_target: The number of
							bases in the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) Initial_bases_on_or_near_target: The
							number of bases in the target region or flanking region of the target.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) Total_effective_yield (Mb) : The data
							size of the effective reads mapped to the reference genome (MB as a unit).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Effective_yield_on_target (Mb) : The
							data size of the reads mapped to the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(14) Fraction_of_effective_bases_on_target:
							The percentage of bases mapped to the target region in all bases mapped to the reference genome
							<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Effective_sequences_on_target/Total_effective_yield).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(15)
							Fraction_of_effective_bases_on_or_near_target: The percentage of bases mapped to the target or
							flanking region in all bases mapped to the reference genome.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(16) Average_sequencing_depth_on_target: The
							average sequencing depth in the target region (Effective_sequences_on_target * 1
							million/Initial_bases_on_target).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(17) Bases_covered_on_target: The number of
							the bases covered in the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(18) Coverage_of_target_region: The coverage
							percentage of target regions (Base_covered_on_target/Initial_bases_on_target).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(19)
							Fraction_of_target_covered_with_at_least_100x: The percentage of bases with depth >100X in target
							regions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(20)
							Fraction_of_target_covered_with_at_least_50x: The percentage of bases with depth >50X in target
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(21)
							Fraction_of_target_covered_with_at_least_20x: The percentage of bases with depth > 20X in the
							target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(22)
							Fraction_of_target_covered_with_at_least_10x: The percentage of bases with depth > 10X in the
							target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(23)
							Fraction_of_target_covered_with_at_least_4x: The percentage of bases with depth > 4X in the target
							region.<br />
						</p>
					</div>
					
					<p class="paragraph"><a href="../../../02.Bam" title="Click to open" target="_blank">Result: 02.Bam</a></p>
				</div>
				
				<!---------------------------------------------- snp/InDel结果 ------------------------------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Variant detection"></a>
						<hr />
					</p><br />
					<h3>4.4 Germline Mutation Detection</h3>
					<br />
					<h4>4.4.1 SNP Detection Result</h4>
					<p class="paragraph"> Single nucleotide polymorphisms (SNPs), also known as single nucleotide variants
						(SNVs), constitute the largest class of genetic variants in genome. A typical human genome has about
						3.6 million SNPs (DePristo <I>et al.</I>, 2011). Statistics of detected SNPs are shown below.</p>
					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.3 The number of SNPs in various genomic regions </p>
						<!--<p class="name_fig">Table 4.1 The number of SNPs in various genomic regions </p>-->
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>YXHF</th>
									
								
									
										<th>YXHM</th>
									
								
									
										<th>YXHB</th>
									
								
									
										<th>YXHP</th>
									
								
							</tr>

							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>22549</td>
										
									
										
											<td>22828</td>
										
									
										
											<td>22995</td>
										
									
										
											<td>22494</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;synonymous_SNP</td>
										
									
										
											<td>11540</td>
										
									
										
											<td>11632</td>
										
									
										
											<td>11678</td>
										
									
										
											<td>11540</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;missense_SNP</td>
										
									
										
											<td>10625</td>
										
									
										
											<td>10857</td>
										
									
										
											<td>10944</td>
										
									
										
											<td>10598</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>89</td>
										
									
										
											<td>86</td>
										
									
										
											<td>90</td>
										
									
										
											<td>83</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>14</td>
										
									
										
											<td>11</td>
										
									
										
											<td>15</td>
										
									
										
											<td>15</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>29</td>
										
									
										
											<td>20</td>
										
									
										
											<td>23</td>
										
									
										
											<td>24</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>230</td>
										
									
										
											<td>213</td>
										
									
										
											<td>236</td>
										
									
										
											<td>223</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>72902</td>
										
									
										
											<td>84278</td>
										
									
										
											<td>82375</td>
										
									
										
											<td>73098</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>4493</td>
										
									
										
											<td>5222</td>
										
									
										
											<td>5064</td>
										
									
										
											<td>4505</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>2529</td>
										
									
										
											<td>2729</td>
										
									
										
											<td>2734</td>
										
									
										
											<td>2554</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>532</td>
										
									
										
											<td>529</td>
										
									
										
											<td>536</td>
										
									
										
											<td>525</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>2888</td>
										
									
										
											<td>3227</td>
										
									
										
											<td>3158</td>
										
									
										
											<td>2922</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>5258</td>
										
									
										
											<td>6208</td>
										
									
										
											<td>6182</td>
										
									
										
											<td>5199</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>40</td>
										
									
										
											<td>39</td>
										
									
										
											<td>38</td>
										
									
										
											<td>37</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>2536</td>
										
									
										
											<td>3055</td>
										
									
										
											<td>3023</td>
										
									
										
											<td>2564</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>1004</td>
										
									
										
											<td>1140</td>
										
									
										
											<td>1133</td>
										
									
										
											<td>985</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>13302</td>
										
									
										
											<td>14183</td>
										
									
										
											<td>15566</td>
										
									
										
											<td>14288</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>127838</td>
										
									
										
											<td>143218</td>
										
									
										
											<td>142581</td>
										
									
										
											<td>128966</td>
										
									
								</tr>
								
							
						
	
						</table>
						</p>
					</div>


					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of SNPs in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							synonymous_SNP: A single nucleotide change that does not cause an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							missense_SNP: A single nucleotide change that causes an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stopgain: A nonsynonymous SNP that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stoploss: A nonsynonymous SNP that leads to the elimination of stop codon at the variant
							site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of SNPs in intronic
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of SNPs in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of SNPs in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of SNPs within 2-bp
							of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of SNPs in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of SNPs in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of SNPs within
							2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of SNPs within 1kb
							away from the transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of SNPs within
							1kb away from the transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of SNPs in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of SNPs.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src='' /></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/SNP/YXHF.snp.png'><img
												src='src/pictures/SNP/YXHF.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/YXHM.snp.png'><img
												src='src/pictures/SNP/YXHM.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/YXHB.snp.png'><img
												src='src/pictures/SNP/YXHB.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/YXHP.snp.png'><img
												src='src/pictures/SNP/YXHP.snp.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>
					<p class="name_fig">Figure 4.7 Number of SNPs in various genomic regions (left pie plot); number of
						different types of SNPs in coding region (right pie plot)</p>
					<p class="premark">&nbsp</p>

					<p class="center">
					<p class="name_fig">Table 4.4 Feature of SNPs</p>
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
											
										<th>Sample name</th>
									
								
									
										<th>YXHF</th>
									
								
									
										<th>YXHM</th>
									
								
									
										<th>YXHB</th>
									
								
									
										<th>YXHP</th>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>127838</td>
									
								
									
										<td>143218</td>
									
								
									
										<td>142581</td>
									
								
									
										<td>128966</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>79693</td>
									
								
									
										<td>89766</td>
									
								
									
										<td>88349</td>
									
								
									
										<td>80505</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>48145</td>
									
								
									
										<td>53452</td>
									
								
									
										<td>54232</td>
									
								
									
										<td>48461</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts</td>
									
								
									
										<td>88675</td>
									
								
									
										<td>99049</td>
									
								
									
										<td>98550</td>
									
								
									
										<td>89324</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">tv</td>
									
								
									
										<td>39163</td>
									
								
									
										<td>44169</td>
									
								
									
										<td>44031</td>
									
								
									
										<td>39642</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts/tv</td>
									
								
									
										<td>2.26</td>
									
								
									
										<td>2.24</td>
									
								
									
										<td>2.24</td>
									
								
									
										<td>2.25</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>124781(97.61%)</td>
									
								
									
										<td>139989(97.75%)</td>
									
								
									
										<td>139157(97.60%)</td>
									
								
									
										<td>125974(97.68%)</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>3057</td>
									
								
									
										<td>3229</td>
									
								
									
										<td>3424</td>
									
								
									
										<td>2992</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts</td>
									
								
									
										<td>1727</td>
									
								
									
										<td>1764</td>
									
								
									
										<td>1923</td>
									
								
									
										<td>1635</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_tv</td>
									
								
									
										<td>1330</td>
									
								
									
										<td>1465</td>
									
								
									
										<td>1501</td>
									
								
									
										<td>1357</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts/tv</td>
									
								
									
										<td>1.30</td>
									
								
									
										<td>1.20</td>
									
								
									
										<td>1.28</td>
									
								
									
										<td>1.20</td>
									
								
							</tr>
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) transition (ts) : The number of
							transitions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) transversion (tv) : The number of
							transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ts/tv: The number of transitions divided
							by the number of transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) dbSNP percentage: The number of SNPs that
							have been reported in dbSNP database divided by the total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) novel: The number of SNPs that have not been reported in
							dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) novel ts: The number of transition SNPs
							that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) novel tv: The number of transversion
							SNPs that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) novel ts/tv: Novel ts divided by novel
							tv.</p>
						<p class="tremark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/SNP" title="Click to open" target="_blank">Result: 03.Result_Homo_sapiens/result/Mutation/SNP</a></p>
				</div>

				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="InDel检测结果"></a>
						<hr />
					</p><br />
					<h4>4.4.2 InDel Detection Result</h4>
					<p class="paragraph">Small insertions and deletions (InDels) that are less than 50 bp in length
						constitute another class of genetic variants in human genome. </p>
					<p class="paragraph">The InDels occurred in coding region or splicing sites may cause changes in
						transcripts and proteins. If the number of inserted or deleted nucleotides is not in frame of three or
						multiples of three, the entire reading frame would be altered. The statistics of InDels (DePristo <I>et al.</I>, 2011) called in the
						samples are listed below: </p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.5 The number of InDels in various genomic regions</p>
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>YXHF</th>
									
								
									
										<th>YXHM</th>
									
								
									
										<th>YXHB</th>
									
								
									
										<th>YXHP</th>
									
								
							</tr>
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>661</td>
										
									
										
											<td>690</td>
										
									
										
											<td>698</td>
										
									
										
											<td>659</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift deletion</td>
										
									
										
											<td>130</td>
										
									
										
											<td>136</td>
										
									
										
											<td>143</td>
										
									
										
											<td>124</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift insertion</td>
										
									
										
											<td>85</td>
										
									
										
											<td>95</td>
										
									
										
											<td>103</td>
										
									
										
											<td>81</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift deletion</td>
										
									
										
											<td>233</td>
										
									
										
											<td>238</td>
										
									
										
											<td>227</td>
										
									
										
											<td>234</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift insertion</td>
										
									
										
											<td>176</td>
										
									
										
											<td>185</td>
										
									
										
											<td>189</td>
										
									
										
											<td>179</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>17</td>
										
									
										
											<td>19</td>
										
									
										
											<td>17</td>
										
									
										
											<td>15</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
										
											<td>1</td>
										
									
										
											<td>4</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>3</td>
										
									
										
											<td>3</td>
										
									
										
											<td>4</td>
										
									
										
											<td>4</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>12</td>
										
									
										
											<td>10</td>
										
									
										
											<td>10</td>
										
									
										
											<td>15</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>15947</td>
										
									
										
											<td>18586</td>
										
									
										
											<td>18016</td>
										
									
										
											<td>15981</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>994</td>
										
									
										
											<td>1168</td>
										
									
										
											<td>1117</td>
										
									
										
											<td>984</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>426</td>
										
									
										
											<td>455</td>
										
									
										
											<td>463</td>
										
									
										
											<td>400</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>225</td>
										
									
										
											<td>233</td>
										
									
										
											<td>236</td>
										
									
										
											<td>228</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>354</td>
										
									
										
											<td>376</td>
										
									
										
											<td>395</td>
										
									
										
											<td>344</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>1018</td>
										
									
										
											<td>1187</td>
										
									
										
											<td>1185</td>
										
									
										
											<td>1022</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>7</td>
										
									
										
											<td>5</td>
										
									
										
											<td>9</td>
										
									
										
											<td>7</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>449</td>
										
									
										
											<td>501</td>
										
									
										
											<td>530</td>
										
									
										
											<td>448</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>180</td>
										
									
										
											<td>217</td>
										
									
										
											<td>223</td>
										
									
										
											<td>180</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>1711</td>
										
									
										
											<td>1783</td>
										
									
										
											<td>1963</td>
										
									
										
											<td>1774</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>21935</td>
										
									
										
											<td>25162</td>
										
									
										
											<td>24783</td>
										
									
										
											<td>21976</td>
										
									
								</tr>
								
							
						

						</table>
						</p>
					</div>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of InDels in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_deletion:
							A deletion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_insertion:
							An insertion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_deletion:
							A deletion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_insertion:
							An insertion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stopgain: An
							insertion or a deletion that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stoploss: An
							insertion or a deletion that leads to the elimination of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of InDels in
							intronic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of InDels in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of InDels in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of InDels within
							2-bp of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of InDels in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of InDels in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of InDels
							within 2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of InDels within
							1kb away from transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of InDels within
							1kb away from transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of InDels in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of InDels.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src=/></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/INDEL/YXHF.indel.png'><img
												src='src/pictures/INDEL/YXHF.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/YXHM.indel.png'><img
												src='src/pictures/INDEL/YXHM.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/YXHB.indel.png'><img
												src='src/pictures/INDEL/YXHB.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/YXHP.indel.png'><img
												src='src/pictures/INDEL/YXHP.indel.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>

					<p class="name_fig">Figure 4.8 Number of InDels in various genomic regions (left); number ofdifferent
						types of InDels in coding region (right)</p>
					<p class="premark">&nbsp</p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.6 Feature of InDels in genome</p>

						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>YXHF</th>
									
								
									
										<th>YXHM</th>
									
								
									
										<th>YXHB</th>
									
								
									
										<th>YXHP</th>
									
								
							</tr>

							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>21935</td>
									
								
									
										<td>25162</td>
									
								
									
										<td>24783</td>
									
								
									
										<td>21976</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>16493</td>
									
								
									
										<td>18847</td>
									
								
									
										<td>18505</td>
									
								
									
										<td>16460</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>5442</td>
									
								
									
										<td>6315</td>
									
								
									
										<td>6278</td>
									
								
									
										<td>5516</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>19202(87.54%)</td>
									
								
									
										<td>21958(87.27%)</td>
									
								
									
										<td>21627(87.27%)</td>
									
								
									
										<td>19284(87.75%)</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>2733</td>
									
								
									
										<td>3204</td>
									
								
									
										<td>3156</td>
									
								
									
										<td>2692</td>
									
								
							</tr>			
							
						

						</table>
						</p>
					</div>

					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) dbSNP percentage: The number of InDels
							that have been reported in dbSNP database divided by the total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) novel: The number of InDels that have not
							been reported in dbSNP database.<br />
						<p class="premark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/INDEL" title="Click to open" target="_blank">Result: 03.Result_Homo_sapiens/result/Mutation/INDEL</a></p>
				</div>

				

				
			
				
				


				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="SNP/InDel结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.3 Variant Annotation </h4>
					<p class="paragraph">Following genomic variant detection, annotation of variants with the tool ANNOVAR (Wang et
						al., 2010) in multiple aspects are performed, including protein coding changes, genomic regions affected by
						the variants, allele frequency, deleteriousness prediction, etc. The main databases used are as
						follows:<br />
					<ul>
						<li style="font-size: 0.6em">RefSeq (O'Leary <I>et al.</I>, 2016) and Gencode (Frankish <I>et al.</I>, 2021) databases
							were used to find genomic regions affected by the variant and possible changes in protein. </li>
						<li style="font-size: 0.6em">We annotated the features of the genomic regions affected by the variants, such
							as cytoband, small RNA, conserved mammalian microRNA regulatory target sites, conservative regions of
							vertebrates, transcription factor binding sites, repeats, etc. </li>
						<li style="font-size: 0.6em">SIFT (Ng <I>et al.</I>, 2003), PolyPhen (Adzhubei <I>et al.</I>, 2013), MutationAssessor
							(Reva <I>et al.</I>, 2011), LRT (Chun <I>et al.</I>, 2009) and CADD (Rentzsch <I>et al.</I>, 2019) scores were used to predict
							the deleteriousness of mutations. GERP++ (Huber <I>et al.</I>, 2020) scores were used to access the conservation
							of mutations.</li>
						<li style="font-size: 0.6em">The established databases, such as 1000 Human Genome (Abecasis <I>et al.</I>, 2012),
							Exome Aggregation Consortium (ExAC) (Kobayashi <I>et al.</I>, 2017), Genome Aggregation Database (gnomAD) (Pio
							<I>et al.</I>, 2021) and exome sequencing project (ESP), were used to find alternative allele frequencies in
							populations that are reported.</li>
						<li style="font-size: 0.6em">Databases dbSNP (Sherry <I>et al.</I>, 2001), COSMIC (Tate <I>et al.</I>, 2019), OMIM (Hamosh
							<I>et al.</I>, 2005), GWAS Catalog (Buniello <I>et al.</I>, 2019) and HGMD (Stenson <I>et al.</I>, 2020) were used to find
							reported information of the variant, such as top SNPs in GWAS and cancer/disease associations. </li>
						<li style="font-size: 0.6em">Databases including Gene Ontology (Lee <I>et al.</I>, 2004), KEGG (Kanehisa <I>et al.</I>,
							2000), Reactome (Jassal <I>et al.</I>, 2020), Biocarta and PID (Schaefer <I>et al.</I>, 2009) were applied to provide
							functional or pathway annotation. </li>
					</ul>
					</p>
		
					<p class="center">
					
					<p class="name_fig">Table 4.7 Example of annotation result</p>
					
					<div id="tb">
						<table class="gy">
							
								
								<tr>
									
										<th>Priority</th>
									
										<th>CHROM</th>
									
										<th>POS</th>
									
										<th>ID</th>
									
										<th>REF</th>
									
										<th>ALT</th>
									
										<th>QUAL</th>
									
										<th>FILTER</th>
									
										<th>GeneName</th>
									
										<th>Description</th>
									
										<th>Func</th>
									
										<th>Gene</th>
									
										<th>GeneDetail</th>
									
										<th>ExonicFunc</th>
									
										<th>AAChange</th>
									
										<th>Gencode</th>
									
										<th>cpgIslandExt</th>
									
										<th>cytoBand</th>
									
										<th>wgRna</th>
									
										<th>genomicSuperDups</th>
									
										<th>Repeat</th>
									
										<th>avsnp</th>
									
										<th>CLNALLELEID</th>
									
										<th>CLNDN</th>
									
										<th>CLNDISDB</th>
									
										<th>CLNREVSTAT</th>
									
										<th>CLNSIG</th>
									
										<th>cosmic</th>
									
										<th>gwasCatalog</th>
									
										<th>1000g_SAS</th>
									
										<th>1000g_EUR</th>
									
										<th>1000g_AFR</th>
									
										<th>1000g_AMR</th>
									
										<th>1000g_EAS</th>
									
										<th>1000g_ALL</th>
									
										<th>esp6500siv2_all</th>
									
										<th>gnomad_exome_AF</th>
									
										<th>gnomad_exome_AF_raw</th>
									
										<th>gnomad_exome_AF_afr</th>
									
										<th>gnomad_exome_AF_sas</th>
									
										<th>gnomad_exome_AF_amr</th>
									
										<th>gnomad_exome_AF_eas</th>
									
										<th>gnomad_exome_AF_nfe</th>
									
										<th>gnomad_exome_AF_fin</th>
									
										<th>gnomad_exome_AF_asj</th>
									
										<th>gnomad_exome_AF_oth</th>
									
										<th>gnomad_genome_AF</th>
									
										<th>gnomad_genome_AF_raw</th>
									
										<th>gnomad_genome_AF_afr</th>
									
										<th>gnomad_genome_AF_sas</th>
									
										<th>gnomad_genome_AF_amr</th>
									
										<th>gnomad_genome_AF_eas</th>
									
										<th>gnomad_genome_AF_nfe</th>
									
										<th>gnomad_genome_AF_fin</th>
									
										<th>gnomad_genome_AF_asj</th>
									
										<th>gnomad_genome_AF_oth</th>
									
										<th>ExAC_ALL</th>
									
										<th>ExAC_AFR</th>
									
										<th>ExAC_AMR</th>
									
										<th>ExAC_EAS</th>
									
										<th>ExAC_FIN</th>
									
										<th>ExAC_NFE</th>
									
										<th>ExAC_OTH</th>
									
										<th>ExAC_SAS</th>
									
										<th>dbscSNV_ADA_SCORE</th>
									
										<th>dbscSNV_RF_SCORE</th>
									
										<th>Interpro_domain</th>
									
										<th>SIFT</th>
									
										<th>Polyphen2_HDIV</th>
									
										<th>Polyphen2_HVAR</th>
									
										<th>LRT</th>
									
										<th>MutationTaster</th>
									
										<th>MutationAssessor</th>
									
										<th>FATHMM</th>
									
										<th>phyloP100way_vertebrate</th>
									
										<th>phyloP30way_mammalian</th>
									
										<th>SiPhy_29way_logOdds</th>
									
										<th>CADD</th>
									
										<th>gerp++gt2</th>
									
										<th>REVEL</th>
									
										<th>INFO</th>
									
										<th>FORMAT</th>
									
										<th>YXHP</th>
									
										<th>Ori_REF</th>
									
										<th>Ori_ALT</th>
									
										<th>shared_hom</th>
									
										<th>shared_het</th>
									
										<th>OMIM</th>
									
										<th>GWAS_Pubmed_pValue</th>
									
										<th>HGMD_Disease_ID</th>
									
										<th>HGMD_Mutation_URL</th>
									
										<th>PubMedID</th>
									
										<th>GO_BP</th>
									
										<th>GO_CC</th>
									
										<th>GO_MF</th>
									
										<th>KEGG_PATHWAY</th>
									
										<th>PID_PATHWAY</th>
									
										<th>BIOCARTA_PATHWAY</th>
									
										<th>REACTOME_PATHWAY</th>
									
										<th>Expression_summary</th>
									
										<th>RNA tissue specificity</th>
									
										<th>RNA tissue distribution</th>
									
										<th>Subcellular_Location</th>
									
										<th>RNA tissue specific nTPM</th>
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>17398</td>
										
									
										
											<td>rs200784459</td>
										
									
										
											<td>C</td>
										
									
										
											<td>A</td>
										
									
										
											<td>138.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>MIR6859-3,MIR6859-1,MIR6859-4,MIR6859-2</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_exonic</td>
										
									
										
											<td>NR_106918,NR_107062,NR_107063,NR_128720</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000619216.1,ENST00000488147.1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>hsa-mir-6859-1</td>
										
									
										
											<td>Score=0.993729;Name=chr9:10843</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs200784459</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.0007</td>
										
									
										
											<td>0.0022</td>
										
									
										
											<td>0.0107</td>
										
									
										
											<td>0.0004</td>
										
									
										
											<td>0.0030</td>
										
									
										
											<td>0.0031</td>
										
									
										
											<td>0.0002</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0.0004</td>
										
									
										
											<td>0.0010</td>
										
									
										
											<td>0.0054</td>
										
									
										
											<td>0.0179</td>
										
									
										
											<td>0.0173</td>
										
									
										
											<td>0.0012</td>
										
									
										
											<td>0.0058</td>
										
									
										
											<td>0.0022</td>
										
									
										
											<td>0.0003</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0.0003</td>
										
									
										
											<td>0.0054</td>
										
									
										
											<td>0.0193</td>
										
									
										
											<td>0.0714</td>
										
									
										
											<td>0.0180</td>
										
									
										
											<td>0.0271</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0.0058</td>
										
									
										
											<td>0</td>
										
									
										
											<td>0.0025</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.881754,10.24</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=-0.987;DP=12;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=41.36;MQRankSum=-1.663;QD=11.55;ReadPosRankSum=0.618;SOR=0.693</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:6,6:12:99:146,0,180</td>
										
									
										
											<td>C</td>
										
									
										
											<td>A</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>69511</td>
										
									
										
											<td>rs2691305</td>
										
									
										
											<td>A</td>
										
									
										
											<td>G</td>
										
									
										
											<td>8792.06</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>OR4F5</td>
										
									
										
											<td>.</td>
										
									
										
											<td>exonic</td>
										
									
										
											<td>NM_001005484</td>
										
									
										
											<td>.</td>
										
									
										
											<td>missense SNV</td>
										
									
										
											<td>OR4F5:NM_001005484:exon1:c.A421G:p.T141A</td>
										
									
										
											<td>ENST00000641515.2</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.994828;Name=chr19:60000</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs2691305</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ID=COSM4144171;OCCURENCE=1(thyroid)</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.7598</td>
										
									
										
											<td>0.9616</td>
										
									
										
											<td>0.7768</td>
										
									
										
											<td>0.6019</td>
										
									
										
											<td>0.9847</td>
										
									
										
											<td>0.9484</td>
										
									
										
											<td>0.9999</td>
										
									
										
											<td>0.9707</td>
										
									
										
											<td>0.9900</td>
										
									
										
											<td>0.9766</td>
										
									
										
											<td>0.9418</td>
										
									
										
											<td>0.8457</td>
										
									
										
											<td>0.8612</td>
										
									
										
											<td>0.5948</td>
										
									
										
											<td>0.9776</td>
										
									
										
											<td>0.8953</td>
										
									
										
											<td>0.9998</td>
										
									
										
											<td>0.9674</td>
										
									
										
											<td>0.9907</td>
										
									
										
											<td>0.9784</td>
										
									
										
											<td>0.8655</td>
										
									
										
											<td>0.9394</td>
										
									
										
											<td>0.5942</td>
										
									
										
											<td>0.9507</td>
										
									
										
											<td>0.9994</td>
										
									
										
											<td>0.9907</td>
										
									
										
											<td>0.9716</td>
										
									
										
											<td>0.9597</td>
										
									
										
											<td>0.9832</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GPCR, rhodopsin-like, 7TM;GPCR, rhodopsin-like, 7TM</td>
										
									
										
											<td>0.652,T</td>
										
									
										
											<td>0.0,B</td>
										
									
										
											<td>0.0,B</td>
										
									
										
											<td>0.000,N</td>
										
									
										
											<td>1.000,P</td>
										
									
										
											<td>-0.855,N</td>
										
									
										
											<td>1.26,T</td>
										
									
										
											<td>-0.684</td>
										
									
										
											<td>-0.327</td>
										
									
										
											<td>4.198</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.053</td>
										
									
										
											<td>AC=2;AF=1;AN=2;DP=295;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=46.47;QD=30.96;SOR=0.892</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>1/1:0,284:284:99:8806,850,0</td>
										
									
										
											<td>A</td>
										
									
										
											<td>G</td>
										
									
										
											<td>1</td>
										
									
										
											<td>0</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GOBP_G_PROTEIN_COUPLED_RECEPTOR_SIGNALING_PATHWAY;GOBP_SENSORY_PERCEPTION;GOBP_SENSORY_PERCEPTION_OF_CHEMICAL_STIMULUS;GOBP_SENSORY_PERCEPTION_OF_SMELL;GOBP_DETECTION_OF_CHEMICAL_STIMULUS;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_DETECTION_OF_STIMULUS_INVOLVED_IN_SENSORY_PERCEPTION;GOBP_DETECTION_OF_STIMULUS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GOMF_G_PROTEIN_COUPLED_RECEPTOR_ACTIVITY;GOMF_MOLECULAR_TRANSDUCER_ACTIVITY</td>
										
									
										
											<td>KEGG_OLFACTORY_TRANSDUCTION</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>REACTOME_OLFACTORY_SIGNALING_PATHWAY;REACTOME_SENSORY_PERCEPTION;REACTOME_GPCR_DOWNSTREAM_SIGNALING</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>88169</td>
										
									
										
											<td>rs940550</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>68.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>intergenic</td>
										
									
										
											<td>NM_001005484,NR_039983</td>
										
									
										
											<td>dist=18161;dist=46604</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.992727;Name=chr1:265774</td>
										
									
										
											<td>Score=811;Name="118:L2(LINE)"</td>
										
									
										
											<td>rs940550</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.2342</td>
										
									
										
											<td>0.1571</td>
										
									
										
											<td>0.4085</td>
										
									
										
											<td>0.3919</td>
										
									
										
											<td>0.2976</td>
										
									
										
											<td>0.299321</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.2674</td>
										
									
										
											<td>0.2730</td>
										
									
										
											<td>0.4092</td>
										
									
										
											<td>0.2671</td>
										
									
										
											<td>0.4123</td>
										
									
										
											<td>0.3670</td>
										
									
										
											<td>0.1335</td>
										
									
										
											<td>0.1449</td>
										
									
										
											<td>0.1312</td>
										
									
										
											<td>0.2768</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=0.524;DP=5;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=50.85;MQRankSum=0.253;QD=13.73;ReadPosRankSum=1.65;SOR=1.179</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:2,3:5:34:76,0,34</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>183598</td>
										
									
										
											<td>.</td>
										
									
										
											<td>C</td>
										
									
										
											<td>G</td>
										
									
										
											<td>239.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>DDX11L17</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_intronic</td>
										
									
										
											<td>NR_148357</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000624431.2</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.988896;Name=chr1:10169</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.0715</td>
										
									
										
											<td>0.0980</td>
										
									
										
											<td>0.2889</td>
										
									
										
											<td>0.0826</td>
										
									
										
											<td>0.0922</td>
										
									
										
											<td>0.3868</td>
										
									
										
											<td>0.0429</td>
										
									
										
											<td>0.0860</td>
										
									
										
											<td>0.0305</td>
										
									
										
											<td>0.0834</td>
										
									
										
											<td>0.2182</td>
										
									
										
											<td>0.3027</td>
										
									
										
											<td>0.4216</td>
										
									
										
											<td>0.2056</td>
										
									
										
											<td>0.2243</td>
										
									
										
											<td>0.3983</td>
										
									
										
											<td>0.0863</td>
										
									
										
											<td>0.2185</td>
										
									
										
											<td>0.0664</td>
										
									
										
											<td>0.1652</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=1.11;DP=15;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=41.72;MQRankSum=-1.458;QD=15.98;ReadPosRankSum=0.112;SOR=1.179</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:8,7:15:99:247,0,231</td>
										
									
										
											<td>C</td>
										
									
										
											<td>G</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
						</table>
					</div>
					<br />
		
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="premark">&nbsp;</p>
                        

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>Note: </b>Annotation information includes seven parts: Priority information (1), Chromosome regions and gene structures related to this variation (2-21), Database (frequency) annotation (22-64), Conservation (Deleterious) prediction (65-80), Basic information on the variation (81-87), Gene function and pathway annotation (88-99), and Gene tissue-specific expression annotation (100-104).<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The first item is Priority information.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(1) Priority: </b>H: High: indicates that the mutation is not in the genomic repeat region but in the exonic or splicing region (that is, there is no information for genomicSuperDups and Repeat annotation). The frequency of it in the 1000 genome database is less than 0.01. At least one of SIFT, Polyphen, MutationTaster and CADD predicted that the site is deleterious. M: Medium: indicates that this site is not in the genomic repeat region but in the exonic or splicing region (that is, the mutation cannot be annotated by genomicSuperDups and Repeat). The frequency of it in the 1000-genome database is less than 0.01. L: Low, indicating that the site does not meet the H or M condition of all other mutations. The International Mouse Phenotyping Consortium (IMPC) performs large-scale gene knockout and phenotypic analysis on mice. The symbols (*, +, -) after the priority (H, M, L) mean: *: the gene is lethal (complete lethal gene) in the analysis of large-scale gene knockout phenotype of mice; +: subviable gene (possible lethal gene: < 12.5% embryo survival) in the analysis of large-scale gene knockout phenotype of mice; -: In a large-scale gene knockout phenotype analysis of mice, the gene was viable (viable gene).<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The second item is chrosomal regions and gene structures related to this variation--The Gene where the variation sites located may be associated with certain diseases. We annotates chrosomal regions and gene structures related to this variation, to help teachers understand gene structure and areas information.</b><br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(2) CHROM: </b>Chromosome ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(3) POS: </b>The position of the variant on chromosomes. The value refers to the position of the first base in the REF sting.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(4) ID: </b>The rs number of the variant in dbSNP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(5) REF: </b>Reference base(s).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(6) ALT: </b>Alternate base(s). Comma separated list of alternate non-reference alleles detected in at least one of the samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(7) QUAL: </b>Quality value for the variant. Phred-scaled quality score for the assertion made in ALT. i.e. -10log10 prob (call in ALT is wrong).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(8) FILTER: </b>Filter status, PASS if the position has passed all filters.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(9) GeneName: </b>Names of genes in which this variant is located according to the refGene annotations.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(10) Description: </b>Gene description.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(11) Func: </b>This field tells whether the variant hits exons or hits intergenic regions, or hits introns, or hits non-coding RNA genes. The value of this field takes the following precedence: exonic = splicing > ncRNA > UTR5/UTR3 > intronic > upstream/downstream > intergenic. Notes: 1. When a variant hits different genes or transcripts, the variant may fit multiple functional categories, and then the precedence mentioned above is used to decide what function to print out; 2. The 'exonic' here refers only to coding exonic portion, but not UTR portion, as there are two keywords (UTR5, UTR3) that are specifically reserved for UTR annotations; 3. If a variant is located in both 5'UTR and 3'UTR region (possibly for two different genes), then the 'UTR5,UTR3' will be printed as the output; 4. 'splicing' in ANNOVAR is defined as variant that is within 2bp away from an exon/intron boundary by default; 5. 'splicing' in ANNOVAR only refers to the 2bp in the intron that is close to an exon; 6. The term 'upstream' and 'downstream' is defined as 1kb away from transcription start site or transcription end site, respectively, taking in account of the strand of the mRNA. If a variant is located in both downstream and upstream region (possibly for 2 different genes), then the 'upstream, downstream' will be printed as the output.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(12) Gene: </b>The transcript name(s). If a variant has 'intergenic' in 'Func' field, this field will give two neighboring transcripts. If a variant hits multiple transcripts with different functional categories, only transcript names in accordance with the value of 'Func' field will be output. For example, rs333970 hits the exonic, splicing, intronic, exonic of the four transcripts of gene CSF1, the 'Func' value will be 'exonic; splicing' and the 'Gene' value will be 'NM_000757, NM_172210, NM_172212' (NM_172211 will be ignored).<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(13) GeneDetail: </b>Description of the sequence change in UTR, splicing, ncRNA_splicing or intergenic region. If 'Func' is 'exonic; splicing' or 'splicing', this field gives the sequence change in splicing region(s); for example, NM_172210:exon6:c.1090+5C>A, NM_172210 is the transcript identifier; exon6:c.1090+5C>A is the sequence change and means that this C>A substitution is at the fifth base downstream from the 6th exon (1090 is the end position of the 6th exon of the cDNA). If 'Func' is 'intergenic', this field gives the distance to the neighboring transcripts, such as 'dist=1366; dist=22344'. If 'Func' is 'UTR*', this field gives the sequence change in UTR; for example, NM_198576:c.*19C>T means that this C>T substitution is at the 19th base downstream from stop codon on NM_198576.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(14) ExonicFunc: </b>This field tells the functional consequences of the variant (possible values included: missense SNV, synonymous SNV, frameshift insertion, frameshift deletion, nonframeshift insertion, nonframeshift deletion, frameshift block substitution, nonframshift block substitution, stopgain, stoploss, unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(15) AAChange: </b>This field tells the amino acid changes as a result of the exonic variant. Only exonic variants have information in this field, i.e. when 'Func' is 'exonic' or 'exonic; splicing', this field gives the amino acid change in each related transcript. For example, AIM1L:NM_001039775:exon2:c.C2768T:p.P923L, AIM1L is gene name; NM_001039775 is the transcript identifier; exon2 means this variant is on the second exon of NM_001039775; c.C2768T is the sequence change and means that this C>T substitution is at the 2,768 position on the cDNA; p.P923L is the amino acid change and means that the 923 amino acid on protein is changed from Pro to Leu due to this variant. Another example, NADK:NM_001198995:exon10:c.1240_1241insAGG:p.G414delinsEG, c.1240_1241insAGG is the sequence change and means that there is a 3bp insertion between position 1,240 and 1,241 on the cDNA; p.G414delinsEG is the amono acid change and means that Gly at the 414th amino acid on protein is changed to Glu-Gly.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(16) Gencode: </b>The transcript name(s) in which this variant is located according to Gencode gene definitions.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(17) cpgIslandExt: </b>CpG island prediction. CpG:116 means that there are 116 CG dinucleotides in this CpG island.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(18) cytoband: </b>This field gives the Giemsa-stained chromosomes bands. When a variant spans multiple bands, they will be connected by a dash (for example, 1q21.1-q23.3).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(19) wgRna: </b>Gene names of snoRNAs and microRNAs based on the miRBase Release and snoRNABase.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(20) genomicSuperDups: </b>This field tells whether the variant hits segmental duplications in reference genome. Variants that mapped to segmental duplications are most likely sequence alignment errors and should be treated with extreme caution. The 'Score' field in output is the sequence identity ranging from 0 to 1 between two genomic segments. The 'Name' field represents the other "matching" segments in genome. For example, 'Score=0.994828;Name=chr19:60000' means that the fragment at the position of chr19:60000 is homologous to the fragment containing this variant, and the sequence identity is 0.994828. Note, for a region to be included in the segmental duplications, at least 1 Kb of the total sequence (containing at least 500bp of non-RepeatMasked sequence) had to align and a sequence identity of at least 90% was required.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(21) Repeat: </b>This field tells whether the variant hits interspersed repeats and low complexity DNA sequences output by RepeatMasker program, such as SINE, LINE and Simple repeats. For example, 'Score=180;Name=1385:(CACCC) n(Simple_repeat)', 180 is the score of the repeat, (CACCC) n is the name of the repeat, 'Simple_repeat' is type of repeat. Note, variants mapped to repeats are likely to be false and should be treated with extreme caution.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The third item is database annotation—There are a great number of common polymorphism sites in human population, while many deleterious variants are rare or low-frequency. This part gives the allele frequency and clinical information for each variant.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(22) avsnp: </b>The RS number of the variant in dbSNP database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(23) CLNALLELEID: </b>The ClinVar Allele ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(24) CLNDN: </b>ClinVar's preferred disease name for the concept specified by disease identifiers in CLNDISDB.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(25) CLNDISDB: </b>Tag-value pairs of disease database name and identifier, e.g. OMIM: NNNNNN.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(26) CLNREVSTAT: </b>ClinVar review status for the Variation ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(27) CLNSIG: </b>Clinical significance for this single variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(28) cosmic: </b>The Cosmic id in Catalogue Of Somatic Mutations In Cancer (COSMIC) database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(29) gwasCatalog: </b>This field tells whether this variant was previously reported to be associated with diseases or traits in genome-wide association studies. It lists the disease names related to this variation. "." means this variation has not been reported by published GWAS study.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(30) 1000g_SAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(31) 1000g_EUR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(32) 1000g_AFR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(33) 1000g_AMR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(34) 1000g_EAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(35) 1000g_ALL: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in ALL population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(36) esp6500siv2_all: </b>The ESP is a NHLBI funded exome sequencing project aiming to identify genetic variants in exonic regions from over 6000 individuals, including healthy ones as well as subjects with different diseases. This field gives alternative allele frequency for the variant in ESP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(37) gnomAD_exome_AF: </b>The Genome Aggregation Database (gnomAD), is a coalition of investigators seeking to aggregate and harmonize exome and genome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. This release comprises two callsets: exome sequence data from 123,136 individuals and whole genome sequencing from 15,496 individuals. GenomAD 2.1.1 was used for annotation analysis of the mutation sites. gnomAD_exome_AF gives allele frequency for the variation in all exome  samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(38) gnomAD_exome_AF_raw: </b>The allele frequency for the variation in gnomAD_exome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(39) gnomAD_exome_AF_afr: </b>The allele frequency for the variation in gnomAD_exome for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(40) gnomAD_exome_AF_sas: </b>The allele frequency for the variation in gnomAD_exome for South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(41) gnomAD_exome_AF_amr: </b>The allele frequency for the variation in gnomAD_exome for Latino/Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(42) gnomAD_exome_AF_eas: </b>The allele frequency for the variation in gnomAD_exome for East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(43) gnomAD_exome_AF_nfe: </b>The allele frequency for the variation in gnomAD_exome for Non-Finnish European population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(44) gnomAD_exome_AF_fin: </b>The allele frequency for the variation in gnomAD_exome for  Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(45) gnomAD_exome_AF_asj: </b>The allele frequency for the variation in gnomAD_exome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(46) gnomAD_exome_AF_oth: </b>The allele frequency for the variation in gnomAD_exome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(47) gnomAD_genome_AF: </b>gnomAD_genome_AF gives allele frequency for the variation in all genome samples of gnomAD. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(48) gnomAD_genome_AF_raw: </b>The allele frequency for the variation in gnomAD_genome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(49) gnomAD_genome_AF_afr: </b>The allele frequency for the variation in gnomAD_genome for African population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(50) gnomAD_genome_AF_sas: </b>The allele frequency for the variation in gnomAD_genome for  South Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(51) gnomAD_genome_AF_amr: </b>The allele frequency for the variation in gnomAD_genome for  Latino/Admixed American population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(52) gnomAD_genome_AF_eas: </b>The allele frequency for the variation in gnomAD_genome for  East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(53) gnomAD_genome_AF_nfe: </b>The allele frequency for the variation in gnomAD_genome for  Non-Finnish European population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(54) gnomAD_genome_AF_fin: </b>The allele frequency for the variation in gnomAD_genome for  Finnish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(55) gnomAD_genome_AF_asj: </b>The allele frequency for the variation in gnomAD_genome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(56) gnomAD_genome_AF_oth: </b>The allele frequency for the variation in gnomAD_genome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(57) ExAC_ALL: </b>ExAC is short for Exome Aggregation Consortium. The data set spans 60,706 unrelated individuals and should serve as a useful reference set of allele frequencies for severe disease studies. Currently supported population groups include ALL, AFR (African), AMR (Admixed American), EAS (East Asian), FIN (Finnish), NFE (Non-finnish European), OTH (other) and SAS (South Asian). ExAC_ALL gives alternative allele frequency for the variation in ALL ExAC samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(58) ExAC_AFR: </b>The alternative allele frequency for the variation in ExAC for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(59) ExAC_AMR: </b>The alternative allele frequency for the variation in ExAC for Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(60) ExAC_EAS: </b>The alternative allele frequency for the variation in ExAC for East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(61) ExAC_FIN: </b>The alternative allele frequency for the variation in ExAC for Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(62) ExAC_NFE: </b>The alternative allele frequency for the variation in ExAC for Non-Finnish European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(63) ExAC_OTH: </b>The alternative allele frequency for the variation in ExAC for other population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(64) ExAC_SAS: </b>The alternative allele frequency for the variation in ExAC for South Asian population.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fourth item is functional prediction—These annotations can help to evaluate deleteriousness of a variation. Note 1: SIFT, Polyphen2, MutationTaster, LRT, MutationAssessor and FATHMM are similar and all predict whether an amino acid substitution affects protein function; only coding variants have these annotations. Note 2: phyloP, SiPhy, gerp++ and CADD are similar and predict the conservation level of the site; these types of 'conservation scores' only consider conservation level at the current base, and they do not care about the actual nucleotide identity, so synonymous and non-synonymous variants at the same site will be scored as the same; these scores are used for finding functionally important sites, so variants that confer increased susceptibility may be scored well.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(65) dbscSNV_ADA_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on adaptive boosting. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(66) dbscSNV_RF_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on Random Forest. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(67) Interpro_domain: </b>The domain of the protein where the mutation is annotated in the Interpro database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(68) SIFT: </b>SIFT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction; the scores and predictions are separated by comma. There are two possible predictions: D (Deleterious, score&lt=0.05)); T (Tolerated, score&gt0.05).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(69) Polyphen2_HDIV: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanDiv database. This annotation should be used when evaluating rare alleles at loci potentially involved in complex phenotypes, dense mapping of regions identified by genome-wide association studies, and analysis of natural selection from sequence data. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score&gt=0.957), P (possibly damaging, 0.453&lt=score&lt=0.956), B (benign, score&lt=0.452).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(70) Polyphen2_HVAR: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanVar database. This annotation should be used for diagnostics of Mendelian diseases. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score>=0.909), P (possibly damaging, 0.447&lt=score&lt=0.909), B (benign, score&lt=0.446).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(71) LRT: </b>LRT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are three possible predictions: D (Deleterious), N (Neutral), U (Unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(72) MutationTaster: </b>MutationTaster annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: 'A' (Disease_causing_automatic), 'D (Disease_causing), 'N' (Polymorphism), 'P' (Polymorphism_automatic'). D and N are categorized by only score, while A and P are categorized by score and other information (if nonsynonymous SNV leads to stop-gain, the variation will be predicted an 'A'; if all three genotypes of nonsynonymous SNV has frequency information in HapMap, the variation will be predicted a 'P'). So, both A and D should be considered deleterious.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(73) MutationAssessor: </b>MutationAssessor annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: H (high), M (medium), L (low), N (neutral). H/M means functional and L/N means non-functional.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(74) FATHMM: </b>FATHMM annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are two possible predictions: D (Deleterious, score&lt=-1.5)); T (Tolerated, score&gt-1.5).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(75) phyloP100way_vertebrate: </b>PhyloP score (dbNSFP version 3.3a) based on the whole genome alignment of 100 vertebrates. Generally, the higher the score, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(76) phyloP30way_mammalian: </b>PhyloP score (dbNSFP) based on the whole genome alignment of 30 mammals.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(77) SiPhy_29way_logOdds: </b>SiPhy score (dbNSFP version 3.3a) based on the whole genome alignment of 29 mammals genomes. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(78) CADD: </b>CADD (Combined Annotation Dependent Depletion) is a score that is based on SVM on multiple other scores. It assigns a score to each possible mutation in the human genome including non-coding and coding variants. In the output, the comma-delimited values are raw scores and phred-scaled scores. "." stands for CADD score&lt10. For phred-scaled scores, 10 means 10% percentile highest scores, 20 means 1% percentile highest scores, and 30% means 0.1% percentile highest scores. CADD official website suggests 15 as a cutoff; in published studies, 10 or 15 is used as a cutoff.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(79) gerp++gt2: </b>GERP++ scores for all mutations with GERP++>2 in human genome, as this threshold is typically regarded as evolutionarily conserved and potentially functional. Variants with '.' in this field should be considered not conserved. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(80) REVEL: </b>A method for predicting the pathogenicity of rare missense mutations by integrating multiple software prediction scores. The score of REVEL ranges from 0 to 1, and the larger the score is, the more deleterious it will be. In the literature, when the score is 0.5, the sensitivity and specificity of detecting harmful mutations are 0.754 and 0.891 respectively.<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fifth item is basic information on the variation  —This part shows the detail information of variation sites, including allelic depths , types of DNA base before and after mutation, and genotype information, et al. these information play a critical role in pedigree analysis.</b> <br />

<p class="premark">&nbsp;</p>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(81) INFO: </b>Information about this variation from variant calling software. INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: &ltkey&gt=&ltdata&gt[,data].<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(82) FORMAT: </b>The FORMAT field specifies the data types and order (colon-separated alphanumeric String). This is followed by one field per sample, with the colon-separated data corresponding to the types specified in the FORMAT.<br />


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GT: genotype, encoded as allele values separated by either of / or |. The allele values are 0 for the reference allele (what is in the Ori_REF field), 1 for the first allele listed in Ori_ALT, 2 for the second allele list in Ori_ALT and so on. 0/0 and 1/1 represent homozygous. 0/1 represents heterozygous. ‘.’ means that a call cannot be made for a sample at a given locus.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AD: Allelic depths for the ref and alt alleles in the order listed (Allelic depths).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DP: Approximate read depth (reads with MQ=255 or with bad mates are filtered).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GQ: Genotype Quality.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PL: Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(83) SampleID: </b>The colon-separated data in this sample corresponding to the types specified in the FORMAT.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(84) Ori_REF: </b>The reference allele (what is in the REF field) in VCF file. For InDel, the allele in "REF" field in this file may be different from (usually shorter than) the "REF" in VCF file.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(85) Ori_ALT: </b>The alternative allele(s) (what is in the ALT field) in VCF file. In this file, the allele in "ALT" field corresponds to one alelle in "Ori_ALT" field; for InDel, the allele in "ALT" field may be different from (usually shorter than) the corresponding allele in the "Ori_ALT" field.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(86) shared_hom: </b>The number of samples with homozygous mutations at the current site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(87) shared_het: </b>The number of samples with a heterozygous mutation at the current site.<br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The sixth item is gene function and pathway annotation: These annotations are for genes containing this variation.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(88) OMIM: </b>Annotation from Online Mendelian Inheritance in Man (OMIM).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(89) GWAS_Pubmed_pValue: </b>Annotation from the NHGRI-EBI GWAS Catalog. The value is like 'pubmedID(p-value);pubmedID(p-value)'. 'pubmedID' is PubMed ID of publication of the study which reported the association between the variation and disease. 'p-value' is the corresponding p-value in the publication.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(90) HGMD_Disease_ID: </b>Annotation from the Human Gene Mutation Database (HGMD®). The value is like 'ID_HGMD(Disease_name);ID_HGMD(Disease_name)'. ID_HGMD is HGMD internal identifier. Disease_name is the name for the disease or condition associated with the mutation.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(91) HGMD_mutation_URL: </b> Annotation from the Human Gene Mutation Database (HGMD®). The value is infomation about this variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(92) PubMedID: </b>Given PubMed documents associated with the mutations or gene number, which from the HGMD and OMIM database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(93-95) GO_BP, GO_CC, GO_MF:</b>Annotation from Gene Ontology. BP is Biological Process; CC is cellular component; MF is molecular function.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(96) KEGG_PATHWAY: </b>Annotation from KEGG PATHWAY Database.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(97) PID_PATHWAY: </b>Annotation from PID (Pathway Interaction Database).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(98) BIOCARTA_PATHWAY: </b>Annotation from BioCarta.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(99) REACTOME_PATHWAY: </b>Annotation from Reactome Pathway Database.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The seventh item is gene tissue-specific expression. This section presents the Human Protein Atlas (HPA) database recorded protein expression of protein-encoding genes in 44 tissues of normal human and detection results of RNA transcription levels in 32 tissues, as well as links to corresponding visual results on HPA website.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(100) Expression_summary: </b>An overview of protein expression, combining immunohistochemical results, RNA results and subcellular distribution results, to describe the distribution of gene expression in different tissues and at the subcellular level.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(101) RNA tissue specificity: </b>Elevated expression includes three subcategory types of elevated expression:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enriched: At least four-fold higher mRNA level in a particular tissue compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Group enriched: At least four-fold higher average mRNA level in a group of 2-5 tissues compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enhanced: At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In addition, there is another form of low tissue specificity: Low expression in various tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(102) RNA tissue distribution: </b>Visualization of gene expression in tissues. All elevated genes are categorized as:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in single: Detected in a single tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in some: Detected in more than one but less than one third of tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in many: Detected in at least a third but not all tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in all: Detected in all tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(103) Subcellular_Location: </b>subcellular distribution positions of genes, such as "nucleus", "mitochondria", etc.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(104) RNA tissue specific nTPM: </b>Consensus transcript expression levels summarized per gene in 50 tissues based on transcriptomics data from HPA and GTEx. The consensus normalized expression ("nTPM") value is calculated as the maximum nTPM value for each gene in the two data sources. If the "RNA tissue specificity" is "Low tissue specificity", this column is empty.<br />



		
						</ul>
					</div>
				</div>
		</div>
	


			<p class="paragraph"><a href="../../result/Mutation/SNP/Annotation" title="Click to open" target="_blank">Result:03.Result_Homo_sapiens/result/Mutation/SNP/Annotation</a></p>
				


                                
                                <!----------------------------------------------- 高级分析结果 --------------------------------------->
                                <div id="page">
                                <p class="head"><a href="#home" title = "返回首页"></a>
                                <a name="Advance Analysis Result"></a>
                                <hr />
                                </p><br />
                                
                                <h3>4.5 Advance Analysis Result</h3>
                                
                                
                                <p class="paragraph">There are so many mutations detected from our samples, but only few of them are related to the specific disease. In order to identify mutations that are truly associated with the disease being studied, we need to use existing databases, software and other tools, combined with sample information, to screen the mutations from detection results step by step, and get the final candidate disease-causing mutations.The specific analysis method is as follows:</p>
                                  (1)SNP and InDel filtering and deleterious classification;</p>
                                  (2)Screening mutations according to dominant/recessive genetic pattern or screening common mutations in sporadic samples;</p>
                                  (3)Denovo mutation detection for trio samples with normal parents and sick children;</p>
                                  (4)Linkage analysis for families with at least three generations;</p>
                                  (5)Taken pathogenic and likely pathogenic genes obtained from step (1) as candidate genes to do GO and KEGG pathway analysis, gene-disease phenotypic association analysis and PPI analysis.</p>
                                
                                        <style id="css">
                                            g.type-TK>rect {
                                                fill: #00ffd0;
                                            }
                                
                                            text {
                                                font-weight: 150;
                                                font-family: "Arial";
                                                font-size: 8px;
                                            }
                                
                                            .node rect {
                                                stroke: #999;
                                                fill: #fff;
                                                stroke-width: 1.2px;
                                            }
                                
                                            .edgePath path {
                                                stroke: #333;
                                                stroke-width: 1.2px;
                                            }
                                        </style>
                                
                                        <script>
                                            $(function () {
                                                var g = new dagreD3.graphlib.Graph()
                                                    .setGraph({})
                                                    .setDefaultEdgeLabel(function () { return {}; });
                                
                                                // Here we"re setting nodeclass, which is used by our custom drawNodes function
                                                // below.
                                                
                                                g.setNode(0, { labelType: "html", label: "All SNPs/InDels", class: "type-TOP" });
                                                
                                                g.setNode(1, { labelType: "html", label: "<a href='#filter-db'>Variant filtering</a>", class: "type-TOP" });
                                                g.setNode(2, { labelType: "html", label: "Variant filtering based on frequency", class: "type-TOP" });
                                                g.setNode(3, { labelType: "html", label: "Variant filtering based on functional region", class: "type-TOP" });
                                                g.setNode(4, { labelType: "html", label: "Variant filtering based on coding region", class: "type-TOP" });
                                                g.setNode(5, { labelType: "html", label: "Variant filtering based on deleterious score", class: "type-TOP" });
                                                
                                                g.setNode(22, { labelType: "html", label: "<a href='#pathway-ana'>Pathway analysis</a>", class: "type-TOP" });
                                                g.setNode(24, { labelType: "html", label: "GO Pathway", class: "type-TOP" });
                                                g.setNode(25, { labelType: "html", label: "KEGG Pathway", class: "type-TOP" });
                                                
                                                g.setNode(26, { labelType: "html", label: "Candidate genes", class: "type-TOP" });
                                                
                                                
                                
                                                
                                                g.setNode(6, { labelType: "html", label: "<a href='#filter-acmg'>Deleterious classification</a>", class: "type-TOP" });
                                                g.setNode(7, { labelType: "html", label: "Pathogenic", class: "type-TOP" });
                                                g.setNode(8, { labelType: "html", label: "Likely pathogenic", class: "type-TOP" });
                                                g.setNode(9, { labelType: "html", label: "Uncertain significance", class: "type-TOP" });
                                                g.setNode(10, { labelType: "html", label: "Likely benign", class: "type-TOP" });
                                                g.setNode(11, { labelType: "html", label: "Benign", class: "type-TOP" });
                                                
                                
                                                
                                
                                                
                                                g.setNode(20, { labelType: "html", label: "<a href='#filter-model'>Variant filtering according to dominant/recessive inheritance pattern</a>", class: "type-TOP" });
                                                
                                                g.setNode(15, { labelType: "html", label: "<a href='#显性模式筛选'>Variant filtering according to dominant pattern", class: "type-TOP" });
                                                
                                                
                                                g.setNode(16, { labelType: "html", label: "<a href='#隐性模式筛选'>Variant filtering according to recessive pattern", class: "type-TOP" });
                                                
                                                
                                                g.setNode(17, { labelType: "html", label: "<a href='#隐性模式筛选'>Compound heterozygous variant filtering", class: "type-TOP" });
                                                
                                                
                                
                                                
                                                g.setNode(18, { labelType: "html", label: "<a href='#denovo-mutation'>De novo mutation analysis</a>", class: "type-TOP" });
                                                
                                
                                                
                                                g.setNode(19, { labelType: "html", label: "<a href='#ROH-ana'>ROH analysis</a>", class: "type-TOP" });
                                                


                                                
                                                g.setNode(31, { labelType: "html", label: "<a href='#HLA-ana'>HLA analysis</a>", class: "type-TOP" });
                                                

                                
                                                
                                                g.setNode(21, { labelType: "html", label: "<a href='#linkage-ana'>Linkage analysis</a>", class: "type-TOP" });
                                                
                                
                                                
                                                g.setNode(30, { labelType: "html", label: "<a href='#disease-network'>Gene-disease phenotypic association analysis</a>", class: "type-TOP" });
                                                
                                
                                                
                                
                                                
                                                g.setNode(23, { labelType: "html", label: "<a href='#pip-ana'>PPI analysis</a>", class: "type-TOP" });
                                                
                                
                                                g.nodes().forEach(function (v) {
                                                    var node = g.node(v);
                                                    // Round the corners of the nodes
                                                    node.rx = node.ry = 5;
                                                });
                                
                                                
                                                g.setEdge(0, 1);
                                                g.setEdge(1, 2);
                                                g.setEdge(2, 3);
                                                g.setEdge(3, 4);
                                                g.setEdge(4, 5);
                                                
                                
                                                
                                
                                
                                                
                                
                                                
                                                g.setEdge(5, 20);
                                                
                                                g.setEdge(20, 15);
                                                
                                                
                                                g.setEdge(20, 16);
                                                
                                                
                                                g.setEdge(20, 17);
                                                
                                                
                                
                                                
                                                g.setEdge(0, 18);
                                                g.setEdge(18, 26);
                                                
                                
                                                
                                                g.setEdge(0, 19);
                                                
                                
                                                
                                                g.setEdge(0, 21);
                                                
                                
                                                
                                                g.setEdge(0, 6);
                                                g.setEdge(6, 7);
                                                g.setEdge(6, 8);
                                                g.setEdge(6, 9);
                                                g.setEdge(6, 10);
                                                g.setEdge(6, 11);
                                                g.setEdge(7, 26);
                                                g.setEdge(8, 26);
                                                
                                
                                                
                                                g.setEdge(22, 24);
                                                g.setEdge(22, 25);
                                                
                                
                                                
                                                
                                                g.setEdge(15, 26);
                                                
                                                
                                                g.setEdge(16, 26);
                                                
                                                
                                                g.setEdge(17, 26);
                                                
                                                
                                
                                                
                                
                                                
                                                g.setEdge(26, 22);
                                                
                                
                                                
                                                g.setEdge(26, 30);
                                                
                                
                                                
                                                g.setEdge(26, 23);
                                                

                                                
                                                g.setEdge(0, 31);
                                                


                                
                                                
                                                // Create the renderer
                                                var render = new dagreD3.render();
                                
                                                // Set up an SVG group so that we can translate the final graph.
                                                var svg = d3.select("#svg1"),
                                                    svgGroup = svg.append("g");
                                
                                                // Run the renderer. This is what draws the final graph.
                                                render(d3.select("svg g"), g);
                                
                                                // Center the graph
                                                var xCenterOffset = (svg.attr("width") - g.graph().width) / 2;
                                                svgGroup.attr("transform", "translate(" + xCenterOffset + ", 20)");
                                                svg.attr("height", g.graph().height + 40);
                                            })
                                        </script>
                                        <svg id='svg1' width='100%' height='600'></svg>
                                        <p class="name_fig">Figure 4.5.3 Screening process for candidate pathogenic mutations</p>
                                        <div id="flow_mz"></div>
                                
                                </div>
                                <a name="Comprehensive Quality Control"></a>
                                <br />
                                
                                <h4>4.5.1 Comprehensive Quality Control of Sequencing Data</h4>
                                <h5>4.5.1.1 Comprehensive Evaluation of Sequencing Data Quality</h5>
                                
                                
                                <p class="paragraph">According to the summary results of sample sequencing quality, statistical results of sequencing depth and coverage, and statistical results of variation detection, the comprehensive judgment shows that the sample data meets the analysis requirements, and subsequent analysis can be conducted.</p>
                                
                                <a name="Sample Gender Judgment"></a>
                                <br />
                                
                                <h5>4.5.1.2 Sample Gender Judgment</h5>
                                
                                <p class="paragraph">The gender identified by the analysis process of the sequencing sample was consistent with the gender filled in the information collection form (BIF), which could be used for subsequent analysis.</p>
                                
                                <a name="IBD Analysis"></a>
                                <br />
                                
                                <h5>4.5.1.3 IBD Analysis (Sample kinship judgment)</h5>
                                
                                
                                <p class="paragraph">The IBD analysis results of the samples were consistent with the family information of the samples, and subsequent analysis could be carried out.</p>
                                
                                
                                
                                
                                <a name="Variant filtering"></a>
                                <br />
                                
                                <h4>4.5.2 Variant filtering based on deleteriousness</h4>
                                <h5>4.5.2.1 Variant filtering using known databases</h5>
                                 
                                <p class="paragraph">The specific filtering SNP/InDel process in the standard analysis is as follows:
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Removing mutations with the allele frequency >1% in at least one of the 1000 Genomes Project (1000g_all), ESP6500 database (esp6500si_all), and gnomAD database (gnomAD_ALL and gnomAD_EAS), in order to remove the diversity mutations between individuals to obtain rare mutations that may cause disease;<br />
                                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Retaining mutations in the coding region (exonic) or splicing site region (up and down 10bp);<br />
                                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Removing synonymous SNP mutations that are not located in the highly conserved region and are not predicted by the software to affect splicing (the sites with GERP++ score >2 are in the highly conserved region; It is considered that mutation affects splicing if at least one of the two predicted dbscSNV scores is greater than 0.6). Remove small fragments (&lt;10bp) in the Repeat region and non-frameshift InDels;<br />
                                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4)Retaining mutations that meet one of the following conditions: a) Mutations are predicted to be deleterious: Mutations are screened according to the scores of SIFT, Polyphen, MutationTaster, and CADD. It is required that at least half of the softwares with scores in these four softwares supports that the mutation may be deleterious, and then the site is retained;<br />
                                                    b) Mutations are predicted to affect splicing: mutations in splicing site regions that are not more than 2bp (±1~2bp) away from exons are retained; Retaining mutations predicted by dbscSNV to affect splicing (there is at least one of the two predicted scores of dbscSNV greater than 0.6). (e.g., the prediction results of one site were 'SIFT=.', 'Polyphen=.', 'MutationTaster=1.000,N', 'CADD=.', 'dbscSNV_score=0.5589,0.636', and 4 deleterious prediction softwares did not support the deleterious results of this site. But one of the dbscSNV scores was higher than 0.6, meaning the software predicted that the mutation would affect splicing, and that site was also preserved.)</p>
                                
                                <p class="paragraph">The mutations obtained through the above four steps are listed in Table 4.5.6:</p>
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.6 Statistics of mutations filtering result</p>
                                <div id="tb">
                                    <table class="gy tb2">
                                        
                                        
                                        <tr class="tb2"><th class="tb2">Total</th><th class="tb2">Freq</th><th class="tb2">Func</th><th class="tb2">ExonicFunc</th><th class="tb2">Deleterious</th></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">219136</td><td class="tb2">14947</td><td class="tb2">2734</td><td class="tb2">1706</td><td class="tb2">1008</td></tr>
                                        
                                        
                                    </table>
                                </div>
                                <!--fil-->
                                </p>
                                <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                <p class="tremark">Notes:</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1)Total: The total number of variants before filtering.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2)Freq: The number of variants after step(1) filtering.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3)Func: The number of variants after step(2) filtering.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4)ExonicFunc: The number of variants after step(3) filtering.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5)Deleterious: The number of variants after step(4) filtering.</p>
                                </div>
                                
                                <p class="paragraph">The annotation of variations obtained after screening through the above four steps is shown in Table 4.5.7 (only the first few lines are shown):</p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.7 The annotation result of mutations filtering</p>
                                <div id="tb">
                                    <table class="gy tb2">
                                        
                                        
                                        <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cytoBand</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">OMIM</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">974054</td><td class="tb2">rs544341319</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">321.75</td><td class="tb2">PLEKHN1</td><td class="tb2">.</td><td class="tb2">splicing</td><td class="tb2">NM_001160184:exon13:c.1548+3G>A;NM_001367552:exon14:c.1689+3G>A;NM_032129:exon14:c.1653+3G>A</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000379410.8,ENST00000379409.6,ENST00000491024.1,ENST00000379407.7</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:37,0:37:99:0,101,1254</td><td class="tb2">0/0:26,0:26:75:0,75,957</td><td class="tb2">0/1:9,11:20:99:331,0,270</td><td class="tb2">0/0:18,0:18:51:0,51,765</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">0.000199681</td><td class="tb2">.</td><td class="tb2">1.334e-05</td><td class="tb2">1.368e-05</td><td class="tb2">0</td><td class="tb2">4.819e-05</td><td class="tb2">2.498e-05</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">3.393e-05</td><td class="tb2">3.282e-05</td><td class="tb2">3.281e-05</td><td class="tb2">0</td><td class="tb2">0.0006</td><td class="tb2">0</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">2.657e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.946700,10.96</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_RNA_CATABOLIC_PROCESS;GOBP_MRNA_CATABOLIC_PROCESS;GOBP_PEPTIDE_METABOLIC_PROCESS;GOBP_APOPTOTIC_PROCESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CATABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CATABOLIC_PROCESS;GOBP_POST_TRANSCRIPTIONAL_REGULATION_OF_GENE_EXPRESSION;GOBP_NEGATIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_POSITIVE_REGULATION_OF_CELL_DEATH;GOBP_MRNA_METABOLIC_PROCESS;GOBP_REGULATION_OF_CELLULAR_CATABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CELLULAR_CATABOLIC_PROCESS;GOBP_REGULATION_OF_AMIDE_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_AMIDE_METABOLIC_PROCESS;GOBP_CELLULAR_MACROMOLECULE_BIOSYNTHETIC_PROCESS;GOBP_PEPTIDE_BIOSYNTHETIC_PROCESS;GOBP_AMIDE_METABOLIC_PROCESS;GOBP_AMIDE_BIOSYNTHETIC_PROCESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_HETEROCYCLE_CATABOLIC_PROCESS;GOBP_RNA_DESTABILIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_RNA_METABOLIC_PROCESS;GOBP_REGULATION_OF_MRNA_CATABOLIC_PROCESS;GOBP_3_UTR_MEDIATED_MRNA_DESTABILIZATION;GOBP_RESPONSE_TO_OXYGEN_LEVELS;GOBP_ORGANIC_CYCLIC_COMPOUND_CATABOLIC_PROCESS;GOBP_ORGANONITROGEN_COMPOUND_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_MRNA_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_MRNA_METABOLIC_PROCESS;GOBP_REGULATION_OF_CELLULAR_MACROMOLECULE_BIOSYNTHETIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION;GOCC_MITOCHONDRIAL_ENVELOPE;GOCC_ENVELOPE</td><td class="tb2">GOMF_PHOSPHATIDYLSERINE_BINDING;GOMF_PHOSPHOLIPID_BINDING;GOMF_LIPID_BINDING;GOMF_PHOSPHATIDYLINOSITOL_BINDING;GOMF_PHOSPHATIDIC_ACID_BINDING;GOMF_MODIFIED_AMINO_ACID_BINDING;GOMF_PHOSPHATIDYLGLYCEROL_BINDING;GOMF_CARDIOLIPIN_BINDING;GOMF_PHOSPHATIDYLINOSITOL_PHOSPHATE_BINDING</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Nuclear expression in several tissues.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">esophagus: 54.9;skin 1: 46.0;vagina: 24.2</td></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">1974098</td><td class="tb2">rs779021655</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">4054.11</td><td class="tb2">CFAP74</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">CFAP74:NM_001304360:exon7:c.C601T:p.R201C</td><td class="tb2">ENST00000468610.5,ENST00000270720.11,ENST00000493964.5,ENST00000682832.2</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:37,45:82:99:1449,0,1219</td><td class="tb2">0/0:34,0:34:99:0,99,1485</td><td class="tb2">0/1:34,52:86:99:1730,0,942</td><td class="tb2">0/1:42,31:73:99:887,0,1180</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.726e-05</td><td class="tb2">4.857e-05</td><td class="tb2">0.0001</td><td class="tb2">6.97e-05</td><td class="tb2">0.0004</td><td class="tb2">2.524e-05</td><td class="tb2">3.15e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">6.631e-05</td><td class="tb2">5.263e-05</td><td class="tb2">5.254e-05</td><td class="tb2">7.251e-05</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">8.578e-05</td><td class="tb2">0.0001</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">4.648e-05</td><td class="tb2">0</td><td class="tb2">6.215e-05</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.998,D</td><td class="tb2">0.736,P</td><td class="tb2">2.562482,22.6</td><td class="tb2">3.4</td><td class="tb2">0.160</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BUNDLE_FORMATION;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_CELL_PROJECTION_ASSEMBLY;GOBP_AXONEME_ASSEMBLY;GOBP_CILIUM_ORGANIZATION;GOBP_ORGANELLE_ASSEMBLY</td><td class="tb2">GOCC_CILIUM;GOCC_AXONEME;GOCC_CYTOPLASMIC_REGION</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Expression in ciliated cells in fallopian tube and respiratory epithelia. Additional expression in testis.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in more than one but less than one third of tissues.</td><td class="tb2">.</td><td class="tb2">fallopian tube: 7.1;testis: 5.2</td></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">2502334</td><td class="tb2">rs181993963</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">1210</td><td class="tb2">PLCH2</td><td class="tb2">PHOSPHOLIPASE C, ETA-2; PLCH2</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">PLCH2:NM_001303012:exon21:c.G2803A:p.D935N,PLCH2:NM_001303013:exon21:c.G2944A:p.D982N,PLCH2:NM_014638:exon21:c.G2884A:p.D962N</td><td class="tb2">ENST00000278878.6,ENST00000343889.8,ENST00000419816.6,ENST00000462379.1,ENST00000473964.5,ENST00000378486.8,ENST00000449969.5</td><td class="tb2">1p36.32</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1294</td><td class="tb2">0/1:22,19:41:99:637,0,615</td><td class="tb2">0/0:40,0:40:99:0,108,1344</td><td class="tb2">0/1:14,20:34:99:585,0,414</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">0.00179712</td><td class="tb2">.</td><td class="tb2">0.0010</td><td class="tb2">0.0010</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">2.947e-05</td><td class="tb2">0.0348</td><td class="tb2">3.818e-05</td><td class="tb2">0.0020</td><td class="tb2">0</td><td class="tb2">0.0006</td><td class="tb2">0.0006</td><td class="tb2">0.0006</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">0</td><td class="tb2">0.0118</td><td class="tb2">0.0001</td><td class="tb2">0.0023</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0052</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.003,D</td><td class="tb2">0.01,B</td><td class="tb2">0.002,B</td><td class="tb2">1.449541,15.41</td><td class="tb2">.</td><td class="tb2">0.078</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">PHOSPHOLIPASE C, ETA-2; PLCH2, 612836</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">15899900| 15583837| 16107206</td><td class="tb2">GOBP_LIPID_METABOLIC_PROCESS;GOBP_PHOSPHOLIPID_METABOLIC_PROCESS;GOBP_GLYCEROPHOSPHOLIPID_METABOLIC_PROCESS;GOBP_MONOATOMIC_ION_TRANSPORT;GOBP_MONOATOMIC_CATION_TRANSPORT;GOBP_CALCIUM_ION_TRANSPORT;GOBP_INTRACELLULAR_MONOATOMIC_ION_HOMEOSTASIS;GOBP_LIPID_CATABOLIC_PROCESS;GOBP_ORGANOPHOSPHATE_METABOLIC_PROCESS;GOBP_CELLULAR_HOMEOSTASIS;GOBP_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_HOMEOSTATIC_PROCESS;GOBP_CELLULAR_LIPID_METABOLIC_PROCESS;GOBP_GLYCEROLIPID_METABOLIC_PROCESS;GOBP_PHOSPHATIDYLINOSITOL_METABOLIC_PROCESS;GOBP_INOSITOL_LIPID_MEDIATED_SIGNALING;GOBP_CHEMICAL_HOMEOSTASIS;GOBP_MONOATOMIC_ION_HOMEOSTASIS;GOBP_SEQUESTERING_OF_CALCIUM_ION;GOBP_MAINTENANCE_OF_LOCATION;GOBP_MAINTENANCE_OF_LOCATION_IN_CELL;GOBP_CALCIUM_ION_HOMEOSTASIS;GOBP_TRANSMEMBRANE_TRANSPORT;GOBP_CALCIUM_ION_TRANSMEMBRANE_TRANSPORT;GOBP_CALCIUM_ION_TRANSMEMBRANE_IMPORT_INTO_CYTOSOL;GOBP_MONOATOMIC_CATION_TRANSMEMBRANE_TRANSPORT;GOBP_INORGANIC_ION_HOMEOSTASIS</td><td class="tb2">.</td><td class="tb2">GOMF_PHOSPHOLIPASE_ACTIVITY;GOMF_PHOSPHOLIPASE_C_ACTIVITY;GOMF_CALCIUM_ION_BINDING;GOMF_PHOSPHORIC_DIESTER_HYDROLASE_ACTIVITY;GOMF_LIPASE_ACTIVITY;GOMF_HYDROLASE_ACTIVITY_ACTING_ON_ESTER_BONDS;GOMF_PHOSPHORIC_ESTER_HYDROLASE_ACTIVITY</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_INOSITOL_PHOSPHATE_METABOLISM;REACTOME_SYNTHESIS_OF_IP3_AND_IP4_IN_THE_CYTOSOL</td><td class="tb2">Cytoplasmic expression in neuronal cells, spermatogonia and in secretory cells of sweat glands.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">.</td><td class="tb2">retina: 44.8;skin 1: 59.6</td></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">6213171</td><td class="tb2">rs372508886</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">12224.1</td><td class="tb2">RNF207</td><td class="tb2">RING FINGER PROTEIN 207; RNF207</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">RNF207:NM_207396:exon16:c.G1640A:p.R547Q</td><td class="tb2">ENST00000496676.5,ENST00000377939.5,ENST00000483336.1</td><td class="tb2">1p36.31</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:147,175:322:99:4820,0,3884</td><td class="tb2">0/0:38,0:38:99:0,99,1363</td><td class="tb2">0/1:109,127:236:99:3508,0,3038</td><td class="tb2">0/1:167,152:319:99:3908,0,4125</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">0.000199681</td><td class="tb2">8.2e-05</td><td class="tb2">1.165e-05</td><td class="tb2">1.231e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">9.908e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">1.658e-05</td><td class="tb2">1.315e-05</td><td class="tb2">1.313e-05</td><td class="tb2">4.818e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">1.688e-05</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.104,T</td><td class="tb2">0.987,D</td><td class="tb2">0.531,P</td><td class="tb2">3.573441,24.9</td><td class="tb2">3.88</td><td class="tb2">0.159</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">RING FINGER PROTEIN 207; RNF207, 616923</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">25281747</td><td class="tb2">GOBP_ACTION_POTENTIAL;GOBP_MORPHOGENESIS_OF_AN_EPITHELIUM;GOBP_HEART_MORPHOGENESIS;GOBP_MUSCLE_SYSTEM_PROCESS;GOBP_CIRCULATORY_SYSTEM_PROCESS;GOBP_HEART_PROCESS;GOBP_EMBRYONIC_HEART_TUBE_MORPHOGENESIS;GOBP_MONOATOMIC_ION_TRANSPORT;GOBP_MONOATOMIC_CATION_TRANSPORT;GOBP_POTASSIUM_ION_TRANSPORT;GOBP_MUSCLE_CONTRACTION;GOBP_REGULATION_OF_MUSCLE_CONTRACTION;GOBP_STRIATED_MUSCLE_CONTRACTION;GOBP_REGULATION_OF_STRIATED_MUSCLE_CONTRACTION;GOBP_CELL_CELL_SIGNALING;GOBP_PATTERN_SPECIFICATION_PROCESS;GOBP_HEART_DEVELOPMENT;GOBP_REGULATION_OF_HEART_CONTRACTION;GOBP_EMBRYO_DEVELOPMENT;GOBP_SPECIFICATION_OF_SYMMETRY;GOBP_ANIMAL_ORGAN_MORPHOGENESIS;GOBP_POSITIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_REGULATION_OF_METAL_ION_TRANSPORT;GOBP_REGULATION_OF_ANATOMICAL_STRUCTURE_MORPHOGENESIS;GOBP_ACTIN_FILAMENT_BASED_PROCESS;GOBP_ACTIN_FILAMENT_BASED_MOVEMENT;GOBP_REGULATION_OF_TRANSPORTER_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_TRANSPORTER_ACTIVITY;GOBP_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_TRANSMEMBRANE_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_EMBRYONIC_HEART_TUBE_DEVELOPMENT;GOBP_TUBE_MORPHOGENESIS;GOBP_TUBE_DEVELOPMENT;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_REGULATION_OF_POTASSIUM_ION_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_POTASSIUM_ION_TRANSPORT;GOBP_REGULATION_OF_MONOATOMIC_ION_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_MONOATOMIC_ION_TRANSPORT;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EMBRYONIC_ORGAN_MORPHOGENESIS;GOBP_EMBRYONIC_ORGAN_DEVELOPMENT;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_TISSUE_MORPHOGENESIS;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_CARDIAC_MUSCLE_CONTRACTION;GOBP_CARDIAC_MUSCLE_CONTRACTION;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_VENTRICULAR_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_EPITHELIUM_DEVELOPMENT;GOBP_EPITHELIAL_TUBE_MORPHOGENESIS;GOBP_CARDIAC_CONDUCTION;GOBP_ACTIN_MEDIATED_CELL_CONTRACTION;GOBP_CIRCULATORY_SYSTEM_DEVELOPMENT;GOBP_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL_INVOLVED_IN_CONTRACTION;GOBP_CARDIAC_MUSCLE_CELL_CONTRACTION;GOBP_VENTRICULAR_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_MEMBRANE_REPOLARIZATION;GOBP_MEMBRANE_REPOLARIZATION_DURING_ACTION_POTENTIAL;GOBP_MEMBRANE_REPOLARIZATION_DURING_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_CELL_CELL_SIGNALING_INVOLVED_IN_CARDIAC_CONDUCTION;GOBP_CELL_COMMUNICATION_INVOLVED_IN_CARDIAC_CONDUCTION;GOBP_REGULATION_OF_MUSCLE_SYSTEM_PROCESS;GOBP_MONOATOMIC_CATION_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_ACTION_POTENTIAL;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION_DURING_ACTION_POTENTIAL;GOBP_MEMBRANE_REPOLARIZATION_DURING_VENTRICULAR_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_VENTRICULAR_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_POTASSIUM_ION_TRANSMEMBRANE_TRANSPORTER_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_POTASSIUM_ION_TRANSMEMBRANE_TRANSPORTER_ACTIVITY;GOBP_REGULATION_OF_DELAYED_RECTIFIER_POTASSIUM_CHANNEL_ACTIVITY;GOBP_REGULATION_OF_BLOOD_CIRCULATION;GOBP_REGULATION_OF_VOLTAGE_GATED_POTASSIUM_CHANNEL_ACTIVITY_INVOLVED_IN_VENTRICULAR_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL_REPOLARIZATION;GOBP_POSITIVE_REGULATION_OF_VOLTAGE_GATED_POTASSIUM_CHANNEL_ACTIVITY;GOBP_REGULATION_OF_MONOATOMIC_CATION_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION_DURING_VENTRICULAR_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION_DURING_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_REGULATION_OF_MORPHOGENESIS_OF_AN_EPITHELIUM;GOBP_REGULATION_OF_CATION_CHANNEL_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_CATION_CHANNEL_ACTIVITY</td><td class="tb2">GOCC_PERINUCLEAR_REGION_OF_CYTOPLASM</td><td class="tb2">GOMF_ZINC_ION_BINDING;GOMF_HSP70_PROTEIN_BINDING;GOMF_HEAT_SHOCK_PROTEIN_BINDING;GOMF_TRANSMEMBRANE_TRANSPORTER_BINDING;GOMF_TRANSITION_METAL_ION_BINDING;GOMF_CHAPERONE_BINDING</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Estimation of protein expression could not be performed. View primary data.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Cytosol</td><td class="tb2">heart muscle: 34.7</td></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">6451996</td><td class="tb2">rs754878916</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">832.75</td><td class="tb2">ESPN</td><td class="tb2">ESPIN; ESPN</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">ESPN:NM_031475:exon10:c.C2225T:p.T742M,ESPN:NM_001367473:exon11:c.C2135T:p.T712M,ESPN:NM_001367474:exon12:c.C2162T:p.T721M</td><td class="tb2">ENST00000434576.2,ENST00000633239.1,ENST00000636644.1,ENST00000461727.6,ENST00000636330.1,ENST00000477679.2,ENST00000645284.1,ENST00000475228.6</td><td class="tb2">1p36.31</td><td class="tb2">Score=0.952198;Name=chr1:16689054</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:36,0:36:99:0,102,1376</td><td class="tb2">0/0:33,0:33:99:0,99,1358</td><td class="tb2">0/1:33,29:62:99:842,0,1059</td><td class="tb2">0/0:33,0:33:99:0,99,1272</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">5.018e-05</td><td class="tb2">5.062e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">4.51e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">3.328e-05</td><td class="tb2">2.628e-05</td><td class="tb2">2.627e-05</td><td class="tb2">2.413e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">4.41e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">4.106e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.024,D</td><td class="tb2">1.0,D</td><td class="tb2">0.997,D</td><td class="tb2">3.366850,24.3</td><td class="tb2">4.23</td><td class="tb2">0.339</td><td class="tb2">1295842</td><td class="tb2">not_provided</td><td class="tb2">MedGen:CN517202</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Uncertain_significance</td><td class="tb2">?Usher syndrome, type 1M, 618632 (3), AR;Deafness, autosomal recessive 36, 609006 (3), AR;Deafness, neurosensory, without vestibular involvement, autosomal dominant, 609006 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">29572253| 8799813| 9763424| 18973245| 10588661| 15930085| 15286153| 10980526| 10975527</td><td class="tb2">GOBP_CYTOSKELETON_ORGANIZATION;GOBP_ACTIN_FILAMENT_ORGANIZATION;GOBP_SENSORY_PERCEPTION;GOBP_ACTIN_FILAMENT_BASED_PROCESS;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_CELL_PROJECTION_ASSEMBLY;GOBP_MICROVILLUS_ASSEMBLY;GOBP_PARALLEL_ACTIN_FILAMENT_BUNDLE_ASSEMBLY;GOBP_MICROVILLUS_ORGANIZATION;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_SENSORY_PERCEPTION_OF_MECHANICAL_STIMULUS;GOBP_ACTIN_FILAMENT_BUNDLE_ORGANIZATION;GOBP_SUPRAMOLECULAR_FIBER_ORGANIZATION</td><td class="tb2">GOCC_ACTIN_FILAMENT;GOCC_MICROVILLUS;GOCC_BRUSH_BORDER;GOCC_ACTIN_CYTOSKELETON;GOCC_FILAMENTOUS_ACTIN;GOCC_STEREOCILIUM_BUNDLE;GOCC_STEREOCILIUM_TIP;GOCC_NEURON_PROJECTION;GOCC_ACTIN_BASED_CELL_PROJECTION;GOCC_CLUSTER_OF_ACTIN_BASED_CELL_PROJECTIONS;GOCC_SUPRAMOLECULAR_COMPLEX;GOCC_SUPRAMOLECULAR_POLYMER;GOCC_POLYMERIC_CYTOSKELETAL_FIBER</td><td class="tb2">GOMF_ACTIN_BINDING;GOMF_CYTOSKELETAL_PROTEIN_BINDING;GOMF_SH3_DOMAIN_BINDING;GOMF_PROTEIN_DOMAIN_SPECIFIC_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_ACTIN_FILAMENT_BINDING</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_SENSORY_PROCESSING_OF_SOUND;REACTOME_SENSORY_PROCESSING_OF_SOUND_BY_OUTER_HAIR_CELLS_OF_THE_COCHLEA;REACTOME_SENSORY_PERCEPTION</td><td class="tb2">Distinct microvilli expression in several tissues including gastrointestinal tract, kidney and epididymal stereocilia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Vesicles</td><td class="tb2">liver: 129.1;retina: 117.8;testis: 145.3</td></tr>
                                        
                                        
                                    </table>
                                </div>
                                <!--fil-->
                                </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                <p class="paragraph">Result: <a href="../../../result/Pro/FilterDB" title="Click to open" target="_blank">/result/Pro/FilterDB</a></p>
                                
                                
                                <div id="page">
                                <p class="head"><a href="#home" title = "返回首页"></a>
                                <a name="Mutation Deleterious Classification"></a>
                                <hr />
                                
                                <h5>4.5.2.2 Mutation Deleterious Classification</h5>
                                
                                <p class="paragraph">In 2015, the American Society for Medical Genetics and Genomics (ACMG) developed standards and guidelines for the interpretation of sequence variation, which became the gold standard for the interpretation of high-throughput sequenced data (Sue Richards et al,2015). ACMG developed a variation classification system and recommended the use of specific standard terms. The system classifies variations as pathogenic, likely pathogenic, uncertain significance, likely benign, benign describes mutations found in the genes that cause Mendelian diseases. ACMG's variation classification system has a total of 28 evidence categories, and the harmfulness of variations is classified according to the combination form of 28 evidence. The specific description is shown in Table 4.5.8.</p>
                                <p class="name_fig">Table 4.5.8 ACMG evidence and classification</p>
                                <table style='text-align:center;width:90%;margin:auto' class='gy tb2' rules=rows border="1">
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='8'>Pathogenic</font></td>
                                        <td class="tb2" rowSpan='4'>PVS1+</td>
                                        <td class="tb2">>=1 PS(PS1 -PS4) or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2">>=2 PM(PM1 -PM6) or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2">1PM(PM1 -PM6)+1PP(PP1 -PP5)or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2">>=2 PP(PP1 -PP5) or</font></td>
                                    </tr>
                                     <tr class="tb2">
                                        <td class="tb2" colSpan='2'>>=2 PS(PS1 -PS4) or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='3'>1 PS(PS1 -PS4) +</td>
                                        <td class="tb2">>=3 PM(PM1 -PM6) or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2">2 PM(PM1 -PM6) + >=2 PP or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2">1PM(PM1 -PM6) + >=4PP (PP1 -PP5)</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='5'style="background-color: #f1f5fb">Likely pathogenic</td>
                                        <td class="tb2" colSpan='2'>1 PVS + 1 PM(PM1 -PM6) or</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>1 PS(PS1 -PS4) + 1 -2 PM(PM1 -PM6) or</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>>=3 PM(PM1 -PM6) or</td>
                                    </tr>
                                    <tr>
                                        <td class="tb2" colSpan='2'>2 PM(PM1 -PM6) + >=2PP(PP1 -PP5)or</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>1 PM(PM1 -PM6) + >=4PP(PP1 -PP5)</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='2' style="background-color: #e3eff8">Benign</td>
                                        <td class="tb2" colSpan='2'>1 BA or</td>
                                    </font></tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>>=2 BS(BS1 -BS4) or</font></td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='2' style="background-color: #f1f5fb">Likely benign</td>
                                        <td class="tb2" colSpan='2'>1 BS(BS1 -BS4)+ 1 BP(BP1 -BP7) or</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>>=2 BP(BP1 -BP7)</td>
                                    </tr>
                                    <tr class="tb2">
                                        <td class="tb2" rowSpan='2'style="background-color: #e3eff8">Uncertain significance</td>
                                        <td class="tb2" colSpan='2'>Not fit any of the above categories or</td>
                                    </font></tr>
                                    <tr class="tb2">
                                        <td class="tb2" colSpan='2'>The pathogenic and the benign contradict each other</font></td>
                                    </tr></table>
                                <p class="tremark">Note: Among the 28 evidences in the table, deleterious categories of variation can be obtained according to different combination forms. The relationship between different columns is directly "or", while the relationship between different columns is "and". However, there are some evidence that cannot be confirmed by biological information alone, which is detailed in Appendix D.</p>
                                <p class='paragraph'>Based on the existing information analysis methods, we classified the deleteriousness of mutations with reference to the evidence of ACMG. Taking p1 families as an example, the number of mutations of each deleterious category obtained according to the results of biological information analysis is shown in the table 4.5.9. Deleteriousness annotation for each variation are shown in the table 4.5.10.</p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.9 The result of deleterious classification screening</p>
                                <div id="tb">
                                    <table class="gy tb2">
                                        
                                        
                                        <tr class="tb2"><th class="tb2">Total</th><th class="tb2">Pathogenic</th><th class="tb2">LikelyPathogenic</th><th class="tb2">VUS</th><th class="tb2">LikelyBenign</th><th class="tb2">Benign</th></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">25151</td><td class="tb2">2</td><td class="tb2">12</td><td class="tb2">3389</td><td class="tb2">348</td><td class="tb2">21400</td></tr>
                                        
                                        
                                    </table>
                                </div>
                                <!--dam-->
                                </p>
                                <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1)Total: The total number of mutations.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2)Pathogenic: The number of pathogenic mutations.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3)Likely Pathogenic: The number of likely pathogenic mutations.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4)VUS: he number of deleterious for uncertain mutations.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5)Likely Benign: The number of likely benign mutations.</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6)Benign: The number of benign mutations.</p>
                                </div>
                                
                                
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.10 Annotation of pathogenic mutations</p>
                                <div id="tb">
                                    <table class="gy tb2">
                                        
                                        
                                        <tr class="tb2"><th class="tb2">ACMG</th><th class="tb2">Heredity</th><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">FILTER</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">Gene</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cpgIslandExt</th><th class="tb2">cytoBand</th><th class="tb2">wgRna</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">PVS1, PM2, PP3, PP5</td><td class="tb2">.</td><td class="tb2">H</td><td class="tb2">chr3</td><td class="tb2">48592773</td><td class="tb2">.</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">12647.1</td><td class="tb2">.</td><td class="tb2">COL7A1</td><td class="tb2">COLLAGEN, TYPE VII, ALPHA-1; COL7A1</td><td class="tb2">splicing</td><td class="tb2">NM_000094</td><td class="tb2">NM_000094:exon7:c.846+2T>C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000681320.1,ENST00000328333.12</td><td class="tb2">.</td><td class="tb2">3p21.31</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">939945</td><td class="tb2">Epidermolysis_bullosa_dystrophica|not_provided</td><td class="tb2">MONDO:MONDO:0006543,MedGen:C0079294,Orphanet:ORPHA303,SNOMED_CT:254185007|MedGen:CN517202</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Likely_pathogenic</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.9832</td><td class="tb2">0.602</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.527</td><td class="tb2">1.307</td><td class="tb2">12.773</td><td class="tb2">4.705513,32</td><td class="tb2">4.31</td><td class="tb2">.</td><td class="tb2">AC=3;AF=0.375;AN=8;BaseQRankSum=-0.994;DP=900;ExcessHet=2.4304;FS=0.523;MLEAC=3;MLEAF=0.375;MQ=60;MQRankSum=0;QD=15.24;ReadPosRankSum=0.914;SOR=0.772</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:148,161:309:99:4952,0,4702</td><td class="tb2">0/0:33,0:33:99:0,99,1206</td><td class="tb2">0/1:145,135:280:99:4235,0,4504</td><td class="tb2">0/1:128,113:241:99:3472,0,4187</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0</td><td class="tb2">3</td><td class="tb2">EBD inversa, 226600 (3), AR;EBD, Bart type, 132000 (3), AD;EBD, localisata variant,  (3), ;Epidermolysis bullosa dystrophica, AD, 131750 (3), AD;Epidermolysis bullosa dystrophica, AR, 226600 (3), AR;Epidermolysis bullosa pruriginosa, 604129 (3), AD, AR;Epidermolysis bullosa, pretibial, 131850 (3), AD, AR;Toenail dystrophy, isolated, 607523 (3), AD;Transient bullous of the newborn, 131705 (3), AD, AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">12874109| 5910871| 6574478| 10583163| 3868959| 8088783| 8644730| 8618021| 9406826| 8513326| 8051117| 8088784| 8541842| 8644729| 8170945| 7883979| 9881948| 16965329| 17434045| 16225626| 2653224| 8345225| 18382769| 8755915| 12485454| 8499916| 8422754| 9856844| 9668111| 7577595| 8037207| 9326325| 9375848| 12787275| 9347800| 9740253| 8288900| 7691888| 9182828| 8325648| 11874498| 10084325| 10383749| 9892921| 11167698| 15774758| 10886525| 1871109| 15365990| 9856843| 1729268| 1680286| 9718359| 11843659| 16557343| 10469344| 1567409| 9804332| 18030675| 21113014| 16971478| 9215684</td><td class="tb2">GOBP_FORMATION_OF_PRIMARY_GERM_LAYER;GOBP_ENDODERM_FORMATION;GOBP_PROTEOLYSIS;GOBP_CELL_ADHESION;GOBP_GASTRULATION;GOBP_ENDODERM_DEVELOPMENT;GOBP_EPIDERMIS_DEVELOPMENT;GOBP_EMBRYO_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_PEPTIDASE_ACTIVITY;GOBP_REGULATION_OF_PROTEOLYSIS;GOBP_ENDODERMAL_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EXTERNAL_ENCAPSULATING_STRUCTURE_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEOLYSIS;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_PEPTIDASE_ACTIVITY</td><td class="tb2">GOCC_COLLAGEN_TRIMER;GOCC_BASEMENT_MEMBRANE;GOCC_ENDOPLASMIC_RETICULUM_LUMEN;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT;GOCC_COPII_COATED_ER_TO_GOLGI_TRANSPORT_VESICLE;GOCC_COATED_VESICLE;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT_MEMBRANE;GOCC_COLLAGEN_CONTAINING_EXTRACELLULAR_MATRIX;GOCC_COMPLEX_OF_COLLAGEN_TRIMERS;GOCC_SUPRAMOLECULAR_COMPLEX</td><td class="tb2">GOMF_ENZYME_INHIBITOR_ACTIVITY;GOMF_SERINE_TYPE_ENDOPEPTIDASE_INHIBITOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT_CONFERRING_TENSILE_STRENGTH;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_PEPTIDASE_REGULATOR_ACTIVITY;GOMF_ENDOPEPTIDASE_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">PID_INTEGRIN1_PATHWAY;PID_AVB3_INTEGRIN_PATHWAY;PID_SYNDECAN_1_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_COLLAGEN_DEGRADATION;REACTOME_DEGRADATION_OF_THE_EXTRACELLULAR_MATRIX;REACTOME_EXTRACELLULAR_MATRIX_ORGANIZATION;REACTOME_COLLAGEN_FORMATION;REACTOME_COLLAGEN_BIOSYNTHESIS_AND_MODIFYING_ENZYMES;REACTOME_ER_TO_GOLGI_ANTEROGRADE_TRANSPORT;REACTOME_MEMBRANE_TRAFFICKING;REACTOME_ASSEMBLY_OF_COLLAGEN_FIBRILS_AND_OTHER_MULTIMERIC_STRUCTURES;REACTOME_COPII_MEDIATED_VESICLE_TRANSPORT;REACTOME_INTEGRIN_CELL_SURFACE_INTERACTIONS;REACTOME_ANCHORING_FIBRIL_FORMATION;REACTOME_LAMININ_INTERACTIONS;REACTOME_ASPARAGINE_N_LINKED_GLYCOSYLATION;REACTOME_VESICLE_MEDIATED_TRANSPORT;REACTOME_CARGO_CONCENTRATION_IN_THE_ER;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_COLLAGEN_CHAIN_TRIMERIZATION;REACTOME_TRANSPORT_TO_THE_GOLGI_AND_SUBSEQUENT_MODIFICATION</td><td class="tb2">Expression in basal membranes of squamous epithelia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Endoplasmic reticulum,Vesicles</td><td class="tb2">skin 1: 98.2</td><td class="tb2">Epidermolysis_bullosa_dystrophica|not_provided</td><td class="tb2">MONDO:MONDO:0006543,MedGen:C0079294,Orphanet:ORPHA303,SNOMED_CT:254185007|MedGen:CN517202</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Likely_pathogenic</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.9832</td><td class="tb2">0.602</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.527</td><td class="tb2">1.307</td><td class="tb2">12.773</td><td class="tb2">4.705513,32</td><td class="tb2">4.31</td><td class="tb2">.</td><td class="tb2">AC=3;AF=0.375;AN=8;BaseQRankSum=-0.994;DP=900;ExcessHet=2.4304;FS=0.523;MLEAC=3;MLEAF=0.375;MQ=60;MQRankSum=0;QD=15.24;ReadPosRankSum=0.914;SOR=0.772</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:148,161:309:99:4952,0,4702</td><td class="tb2">0/0:33,0:33:99:0,99,1206</td><td class="tb2">0/1:145,135:280:99:4235,0,4504</td><td class="tb2">0/1:128,113:241:99:3472,0,4187</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0</td><td class="tb2">3</td><td class="tb2">EBD inversa, 226600 (3), AR;EBD, Bart type, 132000 (3), AD;EBD, localisata variant,  (3), ;Epidermolysis bullosa dystrophica, AD, 131750 (3), AD;Epidermolysis bullosa dystrophica, AR, 226600 (3), AR;Epidermolysis bullosa pruriginosa, 604129 (3), AD, AR;Epidermolysis bullosa, pretibial, 131850 (3), AD, AR;Toenail dystrophy, isolated, 607523 (3), AD;Transient bullous of the newborn, 131705 (3), AD, AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">12874109| 5910871| 6574478| 10583163| 3868959| 8088783| 8644730| 8618021| 9406826| 8513326| 8051117| 8088784| 8541842| 8644729| 8170945| 7883979| 9881948| 16965329| 17434045| 16225626| 2653224| 8345225| 18382769| 8755915| 12485454| 8499916| 8422754| 9856844| 9668111| 7577595| 8037207| 9326325| 9375848| 12787275| 9347800| 9740253| 8288900| 7691888| 9182828| 8325648| 11874498| 10084325| 10383749| 9892921| 11167698| 15774758| 10886525| 1871109| 15365990| 9856843| 1729268| 1680286| 9718359| 11843659| 16557343| 10469344| 1567409| 9804332| 18030675| 21113014| 16971478| 9215684</td><td class="tb2">GOBP_FORMATION_OF_PRIMARY_GERM_LAYER;GOBP_ENDODERM_FORMATION;GOBP_PROTEOLYSIS;GOBP_CELL_ADHESION;GOBP_GASTRULATION;GOBP_ENDODERM_DEVELOPMENT;GOBP_EPIDERMIS_DEVELOPMENT;GOBP_EMBRYO_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_PEPTIDASE_ACTIVITY;GOBP_REGULATION_OF_PROTEOLYSIS;GOBP_ENDODERMAL_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EXTERNAL_ENCAPSULATING_STRUCTURE_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEOLYSIS;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_PEPTIDASE_ACTIVITY</td><td class="tb2">GOCC_COLLAGEN_TRIMER;GOCC_BASEMENT_MEMBRANE;GOCC_ENDOPLASMIC_RETICULUM_LUMEN;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT;GOCC_COPII_COATED_ER_TO_GOLGI_TRANSPORT_VESICLE;GOCC_COATED_VESICLE;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT_MEMBRANE;GOCC_COLLAGEN_CONTAINING_EXTRACELLULAR_MATRIX;GOCC_COMPLEX_OF_COLLAGEN_TRIMERS;GOCC_SUPRAMOLECULAR_COMPLEX</td><td class="tb2">GOMF_ENZYME_INHIBITOR_ACTIVITY;GOMF_SERINE_TYPE_ENDOPEPTIDASE_INHIBITOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT_CONFERRING_TENSILE_STRENGTH;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_PEPTIDASE_REGULATOR_ACTIVITY;GOMF_ENDOPEPTIDASE_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">PID_INTEGRIN1_PATHWAY;PID_AVB3_INTEGRIN_PATHWAY;PID_SYNDECAN_1_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_COLLAGEN_DEGRADATION;REACTOME_DEGRADATION_OF_THE_EXTRACELLULAR_MATRIX;REACTOME_EXTRACELLULAR_MATRIX_ORGANIZATION;REACTOME_COLLAGEN_FORMATION;REACTOME_COLLAGEN_BIOSYNTHESIS_AND_MODIFYING_ENZYMES;REACTOME_ER_TO_GOLGI_ANTEROGRADE_TRANSPORT;REACTOME_MEMBRANE_TRAFFICKING;REACTOME_ASSEMBLY_OF_COLLAGEN_FIBRILS_AND_OTHER_MULTIMERIC_STRUCTURES;REACTOME_COPII_MEDIATED_VESICLE_TRANSPORT;REACTOME_INTEGRIN_CELL_SURFACE_INTERACTIONS;REACTOME_ANCHORING_FIBRIL_FORMATION;REACTOME_LAMININ_INTERACTIONS;REACTOME_ASPARAGINE_N_LINKED_GLYCOSYLATION;REACTOME_VESICLE_MEDIATED_TRANSPORT;REACTOME_CARGO_CONCENTRATION_IN_THE_ER;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_COLLAGEN_CHAIN_TRIMERIZATION;REACTOME_TRANSPORT_TO_THE_GOLGI_AND_SUBSEQUENT_MODIFICATION</td><td class="tb2">Expression in basal membranes of squamous epithelia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Endoplasmic reticulum,Vesicles</td><td class="tb2">skin 1: 98.2</td><td class="tb2">MONDO:MONDO:0006543,MedGen:C0079294,Orphanet:ORPHA303,SNOMED_CT:254185007|MedGen:CN517202</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Likely_pathogenic</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.9832</td><td class="tb2">0.602</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.527</td><td class="tb2">1.307</td><td class="tb2">12.773</td><td class="tb2">4.705513,32</td><td class="tb2">4.31</td><td class="tb2">.</td><td class="tb2">AC=3;AF=0.375;AN=8;BaseQRankSum=-0.994;DP=900;ExcessHet=2.4304;FS=0.523;MLEAC=3;MLEAF=0.375;MQ=60;MQRankSum=0;QD=15.24;ReadPosRankSum=0.914;SOR=0.772</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:148,161:309:99:4952,0,4702</td><td class="tb2">0/0:33,0:33:99:0,99,1206</td><td class="tb2">0/1:145,135:280:99:4235,0,4504</td><td class="tb2">0/1:128,113:241:99:3472,0,4187</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0</td><td class="tb2">3</td><td class="tb2">EBD inversa, 226600 (3), AR;EBD, Bart type, 132000 (3), AD;EBD, localisata variant,  (3), ;Epidermolysis bullosa dystrophica, AD, 131750 (3), AD;Epidermolysis bullosa dystrophica, AR, 226600 (3), AR;Epidermolysis bullosa pruriginosa, 604129 (3), AD, AR;Epidermolysis bullosa, pretibial, 131850 (3), AD, AR;Toenail dystrophy, isolated, 607523 (3), AD;Transient bullous of the newborn, 131705 (3), AD, AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">12874109| 5910871| 6574478| 10583163| 3868959| 8088783| 8644730| 8618021| 9406826| 8513326| 8051117| 8088784| 8541842| 8644729| 8170945| 7883979| 9881948| 16965329| 17434045| 16225626| 2653224| 8345225| 18382769| 8755915| 12485454| 8499916| 8422754| 9856844| 9668111| 7577595| 8037207| 9326325| 9375848| 12787275| 9347800| 9740253| 8288900| 7691888| 9182828| 8325648| 11874498| 10084325| 10383749| 9892921| 11167698| 15774758| 10886525| 1871109| 15365990| 9856843| 1729268| 1680286| 9718359| 11843659| 16557343| 10469344| 1567409| 9804332| 18030675| 21113014| 16971478| 9215684</td><td class="tb2">GOBP_FORMATION_OF_PRIMARY_GERM_LAYER;GOBP_ENDODERM_FORMATION;GOBP_PROTEOLYSIS;GOBP_CELL_ADHESION;GOBP_GASTRULATION;GOBP_ENDODERM_DEVELOPMENT;GOBP_EPIDERMIS_DEVELOPMENT;GOBP_EMBRYO_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_PEPTIDASE_ACTIVITY;GOBP_REGULATION_OF_PROTEOLYSIS;GOBP_ENDODERMAL_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EXTERNAL_ENCAPSULATING_STRUCTURE_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEOLYSIS;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_PEPTIDASE_ACTIVITY</td><td class="tb2">GOCC_COLLAGEN_TRIMER;GOCC_BASEMENT_MEMBRANE;GOCC_ENDOPLASMIC_RETICULUM_LUMEN;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT;GOCC_COPII_COATED_ER_TO_GOLGI_TRANSPORT_VESICLE;GOCC_COATED_VESICLE;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT_MEMBRANE;GOCC_COLLAGEN_CONTAINING_EXTRACELLULAR_MATRIX;GOCC_COMPLEX_OF_COLLAGEN_TRIMERS;GOCC_SUPRAMOLECULAR_COMPLEX</td><td class="tb2">GOMF_ENZYME_INHIBITOR_ACTIVITY;GOMF_SERINE_TYPE_ENDOPEPTIDASE_INHIBITOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT_CONFERRING_TENSILE_STRENGTH;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_PEPTIDASE_REGULATOR_ACTIVITY;GOMF_ENDOPEPTIDASE_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">PID_INTEGRIN1_PATHWAY;PID_AVB3_INTEGRIN_PATHWAY;PID_SYNDECAN_1_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_COLLAGEN_DEGRADATION;REACTOME_DEGRADATION_OF_THE_EXTRACELLULAR_MATRIX;REACTOME_EXTRACELLULAR_MATRIX_ORGANIZATION;REACTOME_COLLAGEN_FORMATION;REACTOME_COLLAGEN_BIOSYNTHESIS_AND_MODIFYING_ENZYMES;REACTOME_ER_TO_GOLGI_ANTEROGRADE_TRANSPORT;REACTOME_MEMBRANE_TRAFFICKING;REACTOME_ASSEMBLY_OF_COLLAGEN_FIBRILS_AND_OTHER_MULTIMERIC_STRUCTURES;REACTOME_COPII_MEDIATED_VESICLE_TRANSPORT;REACTOME_INTEGRIN_CELL_SURFACE_INTERACTIONS;REACTOME_ANCHORING_FIBRIL_FORMATION;REACTOME_LAMININ_INTERACTIONS;REACTOME_ASPARAGINE_N_LINKED_GLYCOSYLATION;REACTOME_VESICLE_MEDIATED_TRANSPORT;REACTOME_CARGO_CONCENTRATION_IN_THE_ER;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_COLLAGEN_CHAIN_TRIMERIZATION;REACTOME_TRANSPORT_TO_THE_GOLGI_AND_SUBSEQUENT_MODIFICATION</td><td class="tb2">Expression in basal membranes of squamous epithelia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Endoplasmic reticulum,Vesicles</td><td class="tb2">skin 1: 98.2</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Likely_pathogenic</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.9832</td><td class="tb2">0.602</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.527</td><td class="tb2">1.307</td><td class="tb2">12.773</td><td class="tb2">4.705513,32</td><td class="tb2">4.31</td><td class="tb2">.</td><td class="tb2">AC=3;AF=0.375;AN=8;BaseQRankSum=-0.994;DP=900;ExcessHet=2.4304;FS=0.523;MLEAC=3;MLEAF=0.375;MQ=60;MQRankSum=0;QD=15.24;ReadPosRankSum=0.914;SOR=0.772</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:148,161:309:99:4952,0,4702</td><td class="tb2">0/0:33,0:33:99:0,99,1206</td><td class="tb2">0/1:145,135:280:99:4235,0,4504</td><td class="tb2">0/1:128,113:241:99:3472,0,4187</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0</td><td class="tb2">3</td><td class="tb2">EBD inversa, 226600 (3), AR;EBD, Bart type, 132000 (3), AD;EBD, localisata variant,  (3), ;Epidermolysis bullosa dystrophica, AD, 131750 (3), AD;Epidermolysis bullosa dystrophica, AR, 226600 (3), AR;Epidermolysis bullosa pruriginosa, 604129 (3), AD, AR;Epidermolysis bullosa, pretibial, 131850 (3), AD, AR;Toenail dystrophy, isolated, 607523 (3), AD;Transient bullous of the newborn, 131705 (3), AD, AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">12874109| 5910871| 6574478| 10583163| 3868959| 8088783| 8644730| 8618021| 9406826| 8513326| 8051117| 8088784| 8541842| 8644729| 8170945| 7883979| 9881948| 16965329| 17434045| 16225626| 2653224| 8345225| 18382769| 8755915| 12485454| 8499916| 8422754| 9856844| 9668111| 7577595| 8037207| 9326325| 9375848| 12787275| 9347800| 9740253| 8288900| 7691888| 9182828| 8325648| 11874498| 10084325| 10383749| 9892921| 11167698| 15774758| 10886525| 1871109| 15365990| 9856843| 1729268| 1680286| 9718359| 11843659| 16557343| 10469344| 1567409| 9804332| 18030675| 21113014| 16971478| 9215684</td><td class="tb2">GOBP_FORMATION_OF_PRIMARY_GERM_LAYER;GOBP_ENDODERM_FORMATION;GOBP_PROTEOLYSIS;GOBP_CELL_ADHESION;GOBP_GASTRULATION;GOBP_ENDODERM_DEVELOPMENT;GOBP_EPIDERMIS_DEVELOPMENT;GOBP_EMBRYO_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_PEPTIDASE_ACTIVITY;GOBP_REGULATION_OF_PROTEOLYSIS;GOBP_ENDODERMAL_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EXTERNAL_ENCAPSULATING_STRUCTURE_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEOLYSIS;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_PEPTIDASE_ACTIVITY</td><td class="tb2">GOCC_COLLAGEN_TRIMER;GOCC_BASEMENT_MEMBRANE;GOCC_ENDOPLASMIC_RETICULUM_LUMEN;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT;GOCC_COPII_COATED_ER_TO_GOLGI_TRANSPORT_VESICLE;GOCC_COATED_VESICLE;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT_MEMBRANE;GOCC_COLLAGEN_CONTAINING_EXTRACELLULAR_MATRIX;GOCC_COMPLEX_OF_COLLAGEN_TRIMERS;GOCC_SUPRAMOLECULAR_COMPLEX</td><td class="tb2">GOMF_ENZYME_INHIBITOR_ACTIVITY;GOMF_SERINE_TYPE_ENDOPEPTIDASE_INHIBITOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT_CONFERRING_TENSILE_STRENGTH;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_PEPTIDASE_REGULATOR_ACTIVITY;GOMF_ENDOPEPTIDASE_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">PID_INTEGRIN1_PATHWAY;PID_AVB3_INTEGRIN_PATHWAY;PID_SYNDECAN_1_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_COLLAGEN_DEGRADATION;REACTOME_DEGRADATION_OF_THE_EXTRACELLULAR_MATRIX;REACTOME_EXTRACELLULAR_MATRIX_ORGANIZATION;REACTOME_COLLAGEN_FORMATION;REACTOME_COLLAGEN_BIOSYNTHESIS_AND_MODIFYING_ENZYMES;REACTOME_ER_TO_GOLGI_ANTEROGRADE_TRANSPORT;REACTOME_MEMBRANE_TRAFFICKING;REACTOME_ASSEMBLY_OF_COLLAGEN_FIBRILS_AND_OTHER_MULTIMERIC_STRUCTURES;REACTOME_COPII_MEDIATED_VESICLE_TRANSPORT;REACTOME_INTEGRIN_CELL_SURFACE_INTERACTIONS;REACTOME_ANCHORING_FIBRIL_FORMATION;REACTOME_LAMININ_INTERACTIONS;REACTOME_ASPARAGINE_N_LINKED_GLYCOSYLATION;REACTOME_VESICLE_MEDIATED_TRANSPORT;REACTOME_CARGO_CONCENTRATION_IN_THE_ER;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_COLLAGEN_CHAIN_TRIMERIZATION;REACTOME_TRANSPORT_TO_THE_GOLGI_AND_SUBSEQUENT_MODIFICATION</td><td class="tb2">Expression in basal membranes of squamous epithelia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Endoplasmic reticulum,Vesicles</td><td class="tb2">skin 1: 98.2</td><td class="tb2">Likely_pathogenic</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.9832</td><td class="tb2">0.602</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">4.527</td><td class="tb2">1.307</td><td class="tb2">12.773</td><td class="tb2">4.705513,32</td><td class="tb2">4.31</td><td class="tb2">.</td><td class="tb2">AC=3;AF=0.375;AN=8;BaseQRankSum=-0.994;DP=900;ExcessHet=2.4304;FS=0.523;MLEAC=3;MLEAF=0.375;MQ=60;MQRankSum=0;QD=15.24;ReadPosRankSum=0.914;SOR=0.772</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:148,161:309:99:4952,0,4702</td><td class="tb2">0/0:33,0:33:99:0,99,1206</td><td class="tb2">0/1:145,135:280:99:4235,0,4504</td><td class="tb2">0/1:128,113:241:99:3472,0,4187</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0</td><td class="tb2">3</td><td class="tb2">EBD inversa, 226600 (3), AR;EBD, Bart type, 132000 (3), AD;EBD, localisata variant,  (3), ;Epidermolysis bullosa dystrophica, AD, 131750 (3), AD;Epidermolysis bullosa dystrophica, AR, 226600 (3), AR;Epidermolysis bullosa pruriginosa, 604129 (3), AD, AR;Epidermolysis bullosa, pretibial, 131850 (3), AD, AR;Toenail dystrophy, isolated, 607523 (3), AD;Transient bullous of the newborn, 131705 (3), AD, AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">12874109| 5910871| 6574478| 10583163| 3868959| 8088783| 8644730| 8618021| 9406826| 8513326| 8051117| 8088784| 8541842| 8644729| 8170945| 7883979| 9881948| 16965329| 17434045| 16225626| 2653224| 8345225| 18382769| 8755915| 12485454| 8499916| 8422754| 9856844| 9668111| 7577595| 8037207| 9326325| 9375848| 12787275| 9347800| 9740253| 8288900| 7691888| 9182828| 8325648| 11874498| 10084325| 10383749| 9892921| 11167698| 15774758| 10886525| 1871109| 15365990| 9856843| 1729268| 1680286| 9718359| 11843659| 16557343| 10469344| 1567409| 9804332| 18030675| 21113014| 16971478| 9215684</td><td class="tb2">GOBP_FORMATION_OF_PRIMARY_GERM_LAYER;GOBP_ENDODERM_FORMATION;GOBP_PROTEOLYSIS;GOBP_CELL_ADHESION;GOBP_GASTRULATION;GOBP_ENDODERM_DEVELOPMENT;GOBP_EPIDERMIS_DEVELOPMENT;GOBP_EMBRYO_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_PEPTIDASE_ACTIVITY;GOBP_REGULATION_OF_PROTEOLYSIS;GOBP_ENDODERMAL_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_EXTERNAL_ENCAPSULATING_STRUCTURE_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_PROTEOLYSIS;GOBP_EMBRYONIC_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_PEPTIDASE_ACTIVITY</td><td class="tb2">GOCC_COLLAGEN_TRIMER;GOCC_BASEMENT_MEMBRANE;GOCC_ENDOPLASMIC_RETICULUM_LUMEN;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT;GOCC_COPII_COATED_ER_TO_GOLGI_TRANSPORT_VESICLE;GOCC_COATED_VESICLE;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE;GOCC_ENDOPLASMIC_RETICULUM_GOLGI_INTERMEDIATE_COMPARTMENT_MEMBRANE;GOCC_COLLAGEN_CONTAINING_EXTRACELLULAR_MATRIX;GOCC_COMPLEX_OF_COLLAGEN_TRIMERS;GOCC_SUPRAMOLECULAR_COMPLEX</td><td class="tb2">GOMF_ENZYME_INHIBITOR_ACTIVITY;GOMF_SERINE_TYPE_ENDOPEPTIDASE_INHIBITOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT_CONFERRING_TENSILE_STRENGTH;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_PEPTIDASE_REGULATOR_ACTIVITY;GOMF_ENDOPEPTIDASE_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">PID_INTEGRIN1_PATHWAY;PID_AVB3_INTEGRIN_PATHWAY;PID_SYNDECAN_1_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_COLLAGEN_DEGRADATION;REACTOME_DEGRADATION_OF_THE_EXTRACELLULAR_MATRIX;REACTOME_EXTRACELLULAR_MATRIX_ORGANIZATION;REACTOME_COLLAGEN_FORMATION;REACTOME_COLLAGEN_BIOSYNTHESIS_AND_MODIFYING_ENZYMES;REACTOME_ER_TO_GOLGI_ANTEROGRADE_TRANSPORT;REACTOME_MEMBRANE_TRAFFICKING;REACTOME_ASSEMBLY_OF_COLLAGEN_FIBRILS_AND_OTHER_MULTIMERIC_STRUCTURES;REACTOME_COPII_MEDIATED_VESICLE_TRANSPORT;REACTOME_INTEGRIN_CELL_SURFACE_INTERACTIONS;REACTOME_ANCHORING_FIBRIL_FORMATION;REACTOME_LAMININ_INTERACTIONS;REACTOME_ASPARAGINE_N_LINKED_GLYCOSYLATION;REACTOME_VESICLE_MEDIATED_TRANSPORT;REACTOME_CARGO_CONCENTRATION_IN_THE_ER;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_COLLAGEN_CHAIN_TRIMERIZATION;REACTOME_TRANSPORT_TO_THE_GOLGI_AND_SUBSEQUENT_MODIFICATION</td><td class="tb2">Expression in basal membranes of squamous epithelia.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Endoplasmic reticulum,Vesicles</td><td class="tb2">skin 1: 98.2</td></tr>
                                        
                                        
                                        
                                        <tr class="tb2"><td class="tb2">PVS1, PM2, PP3</td><td class="tb2">.</td><td class="tb2">H</td><td class="tb2">chr5</td><td class="tb2">60928960</td><td class="tb2">.</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">501</td><td class="tb2">.</td><td class="tb2">ERCC8</td><td class="tb2">EXCISION REPAIR CROSS-COMPLEMENTING, GROUP 8; ERCC8</td><td class="tb2">splicing</td><td class="tb2">NM_000082,NM_001007233,NM_001007234,NM_001290285</td><td class="tb2">NM_000082:exon2:c.78-1G>A;NM_001007233:exon2:UTR5;NM_001007234:exon2:c.78-1G>A;NM_001290285:exon2:UTR5</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000675378.1,ENST00000643034.1,ENST00000675229.1,ENST00000265038.10,ENST00000682418.1,ENST00000682217.1,ENST00000676185.1,ENST00000675920.1,ENST00000682246.1,ENST00000684453.1,ENST00000439176.6,ENST00000682380.1,ENST00000675042.2,ENST00000682041.1,ENST00000497892.6,ENST00000381118.7,ENST00000643708.1,ENST00000683460.1,ENST00000675452.2,ENST00000683052.1,ENST00000683199.1,ENST00000647431.2,ENST00000684621.1,ENST00000647486.2,ENST00000684394.1,ENST00000682375.1,ENST00000477893.1,ENST00000682874.1</td><td class="tb2">.</td><td class="tb2">5q12.1</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.0000</td><td class="tb2">0.932</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.816</td><td class="tb2">1.026</td><td class="tb2">10.811</td><td class="tb2">5.411997,34</td><td class="tb2">4.06</td><td class="tb2">.</td><td class="tb2">AC=2;AF=0.25;AN=8;BaseQRankSum=1.49;DP=95;ExcessHet=0.6695;FS=0;MLEAC=2;MLEAF=0.25;MQ=60;MQRankSum=0;QD=14.31;ReadPosRankSum=1.68;SOR=0.776</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1129</td><td class="tb2">0/1:10,10:20:99:324,0,258</td><td class="tb2">0/0:26,0:26:72:0,72,1080</td><td class="tb2">0/1:7,8:15:99:189,0,185</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0</td><td class="tb2">2</td><td class="tb2">Cockayne syndrome, type A, 216400 (3), AR;UV-sensitive syndrome 2, 614621 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">16865293| 8876179| 14661080| 10447254| 16916636| 12732143| 7664335| 21108394| 9338586| 19329487| 8090199| 15744458| 1339317| 9250659</td><td class="tb2">GOBP_SINGLE_STRAND_BREAK_REPAIR;GOBP_PROTEIN_POLYUBIQUITINATION;GOBP_DNA_METABOLIC_PROCESS;GOBP_DNA_REPAIR;GOBP_REGULATION_OF_DNA_REPAIR;GOBP_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR;GOBP_NUCLEOTIDE_EXCISION_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_NONHOMOLOGOUS_END_JOINING;GOBP_PROTEOLYSIS;GOBP_CELLULAR_RESPONSE_TO_DNA_DAMAGE_STIMULUS;GOBP_RESPONSE_TO_OXIDATIVE_STRESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_RADIATION;GOBP_RESPONSE_TO_UV;GOBP_RESPONSE_TO_LIGHT_STIMULUS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_RESPONSE_TO_X_RAY;GOBP_RESPONSE_TO_IONIZING_RADIATION;GOBP_PROTEASOMAL_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_REGULATION_OF_DNA_METABOLIC_PROCESS;GOBP_PROTEOLYSIS_INVOLVED_IN_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_AUTOUBIQUITINATION;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION_OR_REMOVAL;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_CLASSICAL_NONHOMOLOGOUS_END_JOINING;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOTIDE_EXCISION_REPAIR;GOBP_REGULATION_OF_RESPONSE_TO_DNA_DAMAGE_STIMULUS</td><td class="tb2">GOCC_NUCLEOTIDE_EXCISION_REPAIR_COMPLEX;GOCC_UBIQUITIN_LIGASE_COMPLEX;GOCC_CULLIN_RING_UBIQUITIN_LIGASE_COMPLEX;GOCC_CUL4A_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PERIPHERY;GOCC_CUL4_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_INTRACELLULAR_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_UBIQUITIN_LIKE_PROTEIN_TRANSFERASE_ACTIVITY;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING</td><td class="tb2">KEGG_NUCLEOTIDE_EXCISION_REPAIR;KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_NUCLEOTIDE_EXCISION_REPAIR;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_FORMATION_OF_TC_NER_PRE_INCISION_COMPLEX;REACTOME_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR_TC_NER;REACTOME_DUAL_INCISION_IN_TC_NER;REACTOME_DNA_REPAIR;REACTOME_NEDDYLATION;REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER;REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX</td><td class="tb2">.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.0000</td><td class="tb2">0.932</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.816</td><td class="tb2">1.026</td><td class="tb2">10.811</td><td class="tb2">5.411997,34</td><td class="tb2">4.06</td><td class="tb2">.</td><td class="tb2">AC=2;AF=0.25;AN=8;BaseQRankSum=1.49;DP=95;ExcessHet=0.6695;FS=0;MLEAC=2;MLEAF=0.25;MQ=60;MQRankSum=0;QD=14.31;ReadPosRankSum=1.68;SOR=0.776</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1129</td><td class="tb2">0/1:10,10:20:99:324,0,258</td><td class="tb2">0/0:26,0:26:72:0,72,1080</td><td class="tb2">0/1:7,8:15:99:189,0,185</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0</td><td class="tb2">2</td><td class="tb2">Cockayne syndrome, type A, 216400 (3), AR;UV-sensitive syndrome 2, 614621 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">16865293| 8876179| 14661080| 10447254| 16916636| 12732143| 7664335| 21108394| 9338586| 19329487| 8090199| 15744458| 1339317| 9250659</td><td class="tb2">GOBP_SINGLE_STRAND_BREAK_REPAIR;GOBP_PROTEIN_POLYUBIQUITINATION;GOBP_DNA_METABOLIC_PROCESS;GOBP_DNA_REPAIR;GOBP_REGULATION_OF_DNA_REPAIR;GOBP_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR;GOBP_NUCLEOTIDE_EXCISION_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_NONHOMOLOGOUS_END_JOINING;GOBP_PROTEOLYSIS;GOBP_CELLULAR_RESPONSE_TO_DNA_DAMAGE_STIMULUS;GOBP_RESPONSE_TO_OXIDATIVE_STRESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_RADIATION;GOBP_RESPONSE_TO_UV;GOBP_RESPONSE_TO_LIGHT_STIMULUS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_RESPONSE_TO_X_RAY;GOBP_RESPONSE_TO_IONIZING_RADIATION;GOBP_PROTEASOMAL_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_REGULATION_OF_DNA_METABOLIC_PROCESS;GOBP_PROTEOLYSIS_INVOLVED_IN_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_AUTOUBIQUITINATION;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION_OR_REMOVAL;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_CLASSICAL_NONHOMOLOGOUS_END_JOINING;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOTIDE_EXCISION_REPAIR;GOBP_REGULATION_OF_RESPONSE_TO_DNA_DAMAGE_STIMULUS</td><td class="tb2">GOCC_NUCLEOTIDE_EXCISION_REPAIR_COMPLEX;GOCC_UBIQUITIN_LIGASE_COMPLEX;GOCC_CULLIN_RING_UBIQUITIN_LIGASE_COMPLEX;GOCC_CUL4A_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PERIPHERY;GOCC_CUL4_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_INTRACELLULAR_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_UBIQUITIN_LIKE_PROTEIN_TRANSFERASE_ACTIVITY;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING</td><td class="tb2">KEGG_NUCLEOTIDE_EXCISION_REPAIR;KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_NUCLEOTIDE_EXCISION_REPAIR;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_FORMATION_OF_TC_NER_PRE_INCISION_COMPLEX;REACTOME_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR_TC_NER;REACTOME_DUAL_INCISION_IN_TC_NER;REACTOME_DNA_REPAIR;REACTOME_NEDDYLATION;REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER;REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX</td><td class="tb2">.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.0000</td><td class="tb2">0.932</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.816</td><td class="tb2">1.026</td><td class="tb2">10.811</td><td class="tb2">5.411997,34</td><td class="tb2">4.06</td><td class="tb2">.</td><td class="tb2">AC=2;AF=0.25;AN=8;BaseQRankSum=1.49;DP=95;ExcessHet=0.6695;FS=0;MLEAC=2;MLEAF=0.25;MQ=60;MQRankSum=0;QD=14.31;ReadPosRankSum=1.68;SOR=0.776</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1129</td><td class="tb2">0/1:10,10:20:99:324,0,258</td><td class="tb2">0/0:26,0:26:72:0,72,1080</td><td class="tb2">0/1:7,8:15:99:189,0,185</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0</td><td class="tb2">2</td><td class="tb2">Cockayne syndrome, type A, 216400 (3), AR;UV-sensitive syndrome 2, 614621 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">16865293| 8876179| 14661080| 10447254| 16916636| 12732143| 7664335| 21108394| 9338586| 19329487| 8090199| 15744458| 1339317| 9250659</td><td class="tb2">GOBP_SINGLE_STRAND_BREAK_REPAIR;GOBP_PROTEIN_POLYUBIQUITINATION;GOBP_DNA_METABOLIC_PROCESS;GOBP_DNA_REPAIR;GOBP_REGULATION_OF_DNA_REPAIR;GOBP_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR;GOBP_NUCLEOTIDE_EXCISION_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_NONHOMOLOGOUS_END_JOINING;GOBP_PROTEOLYSIS;GOBP_CELLULAR_RESPONSE_TO_DNA_DAMAGE_STIMULUS;GOBP_RESPONSE_TO_OXIDATIVE_STRESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_RADIATION;GOBP_RESPONSE_TO_UV;GOBP_RESPONSE_TO_LIGHT_STIMULUS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_RESPONSE_TO_X_RAY;GOBP_RESPONSE_TO_IONIZING_RADIATION;GOBP_PROTEASOMAL_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_REGULATION_OF_DNA_METABOLIC_PROCESS;GOBP_PROTEOLYSIS_INVOLVED_IN_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_AUTOUBIQUITINATION;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION_OR_REMOVAL;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_CLASSICAL_NONHOMOLOGOUS_END_JOINING;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOTIDE_EXCISION_REPAIR;GOBP_REGULATION_OF_RESPONSE_TO_DNA_DAMAGE_STIMULUS</td><td class="tb2">GOCC_NUCLEOTIDE_EXCISION_REPAIR_COMPLEX;GOCC_UBIQUITIN_LIGASE_COMPLEX;GOCC_CULLIN_RING_UBIQUITIN_LIGASE_COMPLEX;GOCC_CUL4A_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PERIPHERY;GOCC_CUL4_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_INTRACELLULAR_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_UBIQUITIN_LIKE_PROTEIN_TRANSFERASE_ACTIVITY;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING</td><td class="tb2">KEGG_NUCLEOTIDE_EXCISION_REPAIR;KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_NUCLEOTIDE_EXCISION_REPAIR;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_FORMATION_OF_TC_NER_PRE_INCISION_COMPLEX;REACTOME_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR_TC_NER;REACTOME_DUAL_INCISION_IN_TC_NER;REACTOME_DNA_REPAIR;REACTOME_NEDDYLATION;REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER;REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX</td><td class="tb2">.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.0000</td><td class="tb2">0.932</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.816</td><td class="tb2">1.026</td><td class="tb2">10.811</td><td class="tb2">5.411997,34</td><td class="tb2">4.06</td><td class="tb2">.</td><td class="tb2">AC=2;AF=0.25;AN=8;BaseQRankSum=1.49;DP=95;ExcessHet=0.6695;FS=0;MLEAC=2;MLEAF=0.25;MQ=60;MQRankSum=0;QD=14.31;ReadPosRankSum=1.68;SOR=0.776</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1129</td><td class="tb2">0/1:10,10:20:99:324,0,258</td><td class="tb2">0/0:26,0:26:72:0,72,1080</td><td class="tb2">0/1:7,8:15:99:189,0,185</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0</td><td class="tb2">2</td><td class="tb2">Cockayne syndrome, type A, 216400 (3), AR;UV-sensitive syndrome 2, 614621 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">16865293| 8876179| 14661080| 10447254| 16916636| 12732143| 7664335| 21108394| 9338586| 19329487| 8090199| 15744458| 1339317| 9250659</td><td class="tb2">GOBP_SINGLE_STRAND_BREAK_REPAIR;GOBP_PROTEIN_POLYUBIQUITINATION;GOBP_DNA_METABOLIC_PROCESS;GOBP_DNA_REPAIR;GOBP_REGULATION_OF_DNA_REPAIR;GOBP_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR;GOBP_NUCLEOTIDE_EXCISION_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_NONHOMOLOGOUS_END_JOINING;GOBP_PROTEOLYSIS;GOBP_CELLULAR_RESPONSE_TO_DNA_DAMAGE_STIMULUS;GOBP_RESPONSE_TO_OXIDATIVE_STRESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_RADIATION;GOBP_RESPONSE_TO_UV;GOBP_RESPONSE_TO_LIGHT_STIMULUS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_RESPONSE_TO_X_RAY;GOBP_RESPONSE_TO_IONIZING_RADIATION;GOBP_PROTEASOMAL_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_REGULATION_OF_DNA_METABOLIC_PROCESS;GOBP_PROTEOLYSIS_INVOLVED_IN_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_AUTOUBIQUITINATION;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION_OR_REMOVAL;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_CLASSICAL_NONHOMOLOGOUS_END_JOINING;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOTIDE_EXCISION_REPAIR;GOBP_REGULATION_OF_RESPONSE_TO_DNA_DAMAGE_STIMULUS</td><td class="tb2">GOCC_NUCLEOTIDE_EXCISION_REPAIR_COMPLEX;GOCC_UBIQUITIN_LIGASE_COMPLEX;GOCC_CULLIN_RING_UBIQUITIN_LIGASE_COMPLEX;GOCC_CUL4A_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PERIPHERY;GOCC_CUL4_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_INTRACELLULAR_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_UBIQUITIN_LIKE_PROTEIN_TRANSFERASE_ACTIVITY;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING</td><td class="tb2">KEGG_NUCLEOTIDE_EXCISION_REPAIR;KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_NUCLEOTIDE_EXCISION_REPAIR;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_FORMATION_OF_TC_NER_PRE_INCISION_COMPLEX;REACTOME_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR_TC_NER;REACTOME_DUAL_INCISION_IN_TC_NER;REACTOME_DNA_REPAIR;REACTOME_NEDDYLATION;REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER;REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX</td><td class="tb2">.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.0000</td><td class="tb2">0.932</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.816</td><td class="tb2">1.026</td><td class="tb2">10.811</td><td class="tb2">5.411997,34</td><td class="tb2">4.06</td><td class="tb2">.</td><td class="tb2">AC=2;AF=0.25;AN=8;BaseQRankSum=1.49;DP=95;ExcessHet=0.6695;FS=0;MLEAC=2;MLEAF=0.25;MQ=60;MQRankSum=0;QD=14.31;ReadPosRankSum=1.68;SOR=0.776</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:33,0:33:99:0,99,1129</td><td class="tb2">0/1:10,10:20:99:324,0,258</td><td class="tb2">0/0:26,0:26:72:0,72,1080</td><td class="tb2">0/1:7,8:15:99:189,0,185</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0</td><td class="tb2">2</td><td class="tb2">Cockayne syndrome, type A, 216400 (3), AR;UV-sensitive syndrome 2, 614621 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">16865293| 8876179| 14661080| 10447254| 16916636| 12732143| 7664335| 21108394| 9338586| 19329487| 8090199| 15744458| 1339317| 9250659</td><td class="tb2">GOBP_SINGLE_STRAND_BREAK_REPAIR;GOBP_PROTEIN_POLYUBIQUITINATION;GOBP_DNA_METABOLIC_PROCESS;GOBP_DNA_REPAIR;GOBP_REGULATION_OF_DNA_REPAIR;GOBP_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR;GOBP_NUCLEOTIDE_EXCISION_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_NONHOMOLOGOUS_END_JOINING;GOBP_PROTEOLYSIS;GOBP_CELLULAR_RESPONSE_TO_DNA_DAMAGE_STIMULUS;GOBP_RESPONSE_TO_OXIDATIVE_STRESS;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_RADIATION;GOBP_RESPONSE_TO_UV;GOBP_RESPONSE_TO_LIGHT_STIMULUS;GOBP_RESPONSE_TO_ABIOTIC_STIMULUS;GOBP_RESPONSE_TO_X_RAY;GOBP_RESPONSE_TO_IONIZING_RADIATION;GOBP_PROTEASOMAL_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_CELLULAR_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_REGULATION_OF_DNA_METABOLIC_PROCESS;GOBP_PROTEOLYSIS_INVOLVED_IN_PROTEIN_CATABOLIC_PROCESS;GOBP_PROTEIN_AUTOUBIQUITINATION;GOBP_PROTEIN_MODIFICATION_BY_SMALL_PROTEIN_CONJUGATION_OR_REMOVAL;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_DOUBLE_STRAND_BREAK_REPAIR_VIA_CLASSICAL_NONHOMOLOGOUS_END_JOINING;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOTIDE_EXCISION_REPAIR;GOBP_REGULATION_OF_RESPONSE_TO_DNA_DAMAGE_STIMULUS</td><td class="tb2">GOCC_NUCLEOTIDE_EXCISION_REPAIR_COMPLEX;GOCC_UBIQUITIN_LIGASE_COMPLEX;GOCC_CULLIN_RING_UBIQUITIN_LIGASE_COMPLEX;GOCC_CUL4A_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PERIPHERY;GOCC_CUL4_RING_E3_UBIQUITIN_LIGASE_COMPLEX;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_INTRACELLULAR_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_UBIQUITIN_LIKE_PROTEIN_TRANSFERASE_ACTIVITY;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING</td><td class="tb2">KEGG_NUCLEOTIDE_EXCISION_REPAIR;KEGG_UBIQUITIN_MEDIATED_PROTEOLYSIS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_NUCLEOTIDE_EXCISION_REPAIR;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_FORMATION_OF_TC_NER_PRE_INCISION_COMPLEX;REACTOME_TRANSCRIPTION_COUPLED_NUCLEOTIDE_EXCISION_REPAIR_TC_NER;REACTOME_DUAL_INCISION_IN_TC_NER;REACTOME_DNA_REPAIR;REACTOME_NEDDYLATION;REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER;REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX</td><td class="tb2">.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Nuclear speckles</td><td class="tb2">.</td></tr>
                                        
                                        
                                    </table>
                                </div>
                                <!--dam-->
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                </p>
                                
                                <!--dam-->
                                
                                <p class="paragraph">Result: <a href="../../../result/Pro/ACMG" title="Click to open" target="_blank">/result/Pro/ACMG</a></p>
                                </div>
                                
                                
                                
                                
                                
                                
                                
                                <!-------------------------------------------------------------------------------- 2 显隐性模式筛选  ---------------------------------------------------------------->
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                <a name="Analysis under dominant/recessive mode"></a>
                                <br />
                                
                                <h4>4.5.3 Variant filtering based on sample information</h4>
                                
                                
                                
                                <h5>4.5.3.1 Variant filtering according to dominant/recessive inheritance pattern (for family samples only)</h5>
                                
                                <p class="paragraph">Mendelian genetic diseases can be divided into dominant genetic diseases and recessive genetic diseases according to the genetic pattern. For dominant inherited diseases, the pathogenic variation comes from the father or mother, which is generally heterozygous, and heterozygous mutations on candidate genes should be considered first (Joshi <i>et al.</i>, 2015). For recessive genetic diseases, the pathophysiology is generally homozygous from both parents (Keller <i>et al.</i>, 2012; Magi <i>et al.</i>, 2014) or compound heterozygous variation. Complex diseases are caused by multiple factors such as susceptibility genes and environment, and the screening of recessive inheritance patterns of these diseases needs family support to be more accurate.
                                Complex diseases may have multiple minor genes that influence each other to cause individual disease. This kind of recessive screening is based on the search of major genes, which is similar to the screening of single gene diseases in local families.
                                This analysis only applicable to family samples. If the genetic pattern is known to be dominant, only the dominant pattern analysis will be performed. And if the genetic pattern is known to be recessive, only the recessive pattern analysis will be performed.</p>
                                
                                <br/>
                                
                                <a name="Analysis under dominant model"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.3.1.1 Variant filtering according to dominant inheritance pattern</p>
                                
                                <p class="paragraph">On the basis of mutations filtering, if a single gene disease is inherited in a dominant pattern in the family, heterozygous mutations in the patients' autosomes (mutations in the allosomes) in the family are retained. And the mutation that normal person in the family who does not have was considered as a candidate mutation.</p>
                                
                                <p class="paragraph">Taking p1 family as an example, we detected 20 candidate genes with SNPs and 12 candidate genes with InDels in this family. The filtering and annotation results are shown in the following table.</p>
                                    <p class="center">
                                    <p class="name_fig">Table 4.5.11 Genetic heterozygous variation screening and annotation result (only the first few lines of information are shown)
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cytoBand</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">OMIM</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">12794207</td><td class="tb2">rs61775051</td><td class="tb2">T</td><td class="tb2">C</td><td class="tb2">55.75</td><td class="tb2">PRAMEF1</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">PRAMEF1:NM_023013:exon3:c.T580C:p.Y194H</td><td class="tb2">ENST00000332296.7</td><td class="tb2">1p36.21</td><td class="tb2">Score=0.967478;Name=chr1:13338551</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:61,0:61:99:0,120,1800</td><td class="tb2">0/0:40,0:40:99:0,120,1470</td><td class="tb2">0/0:66,0:66:43:0,43,2240</td><td class="tb2">0/1:72,9:81:65:65,0,2274</td><td class="tb2">T</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0002</td><td class="tb2">0.0013</td><td class="tb2">5.988e-05</td><td class="tb2">0.0005</td><td class="tb2">0.0002</td><td class="tb2">0.0009</td><td class="tb2">0.0001</td><td class="tb2">0.0002</td><td class="tb2">0.0001</td><td class="tb2">0.0002</td><td class="tb2">9.292e-05</td><td class="tb2">0.0004</td><td class="tb2">7.297e-05</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">9.54e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0067</td><td class="tb2">0.0026</td><td class="tb2">0.0026</td><td class="tb2">0.0156</td><td class="tb2">0.0069</td><td class="tb2">0.0043</td><td class="tb2">0.0101</td><td class="tb2">0.0173</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.02,D</td><td class="tb2">0.004,B</td><td class="tb2">0.006,B</td><td class="tb2">0.917329,10.62</td><td class="tb2">.</td><td class="tb2">0.014</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_APOPTOTIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CELL_DEATH;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_NUCLEOBASE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_NEGATIVE_REGULATION_OF_RNA_BIOSYNTHETIC_PROCESS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Not detected</td><td class="tb2">Not detected</td><td class="tb2">.</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">13322125</td><td class="tb2">.</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">239.75</td><td class="tb2">PRAMEF15</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">PRAMEF15:NM_001098376:exon4:c.T1298G:p.F433C</td><td class="tb2">ENST00000376152.2</td><td class="tb2">1p36.21</td><td class="tb2">Score=0.990208;Name=chr1:12855323</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:37,0:37:99:.:.:0,102,1422:.</td><td class="tb2">0/0:80,0:80:31:.:.:0,31,2297:.</td><td class="tb2">0/0:40,0:40:99:.:.:0,105,1578:.</td><td class="tb2">0|1:1,6:7:18:0|1:13322125_T_G:249,0,18:13322125</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0025</td><td class="tb2">0.0184</td><td class="tb2">0.0002</td><td class="tb2">0.0024</td><td class="tb2">0.0007</td><td class="tb2">0.0470</td><td class="tb2">0.0013</td><td class="tb2">0.0038</td><td class="tb2">0.0007</td><td class="tb2">0.0040</td><td class="tb2">0.0073</td><td class="tb2">0.0209</td><td class="tb2">0.0017</td><td class="tb2">0.0098</td><td class="tb2">0.0046</td><td class="tb2">0.0427</td><td class="tb2">0.0090</td><td class="tb2">0.0087</td><td class="tb2">0.0057</td><td class="tb2">0.0096</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.001,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.723959,17.19</td><td class="tb2">.</td><td class="tb2">0.035</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_APOPTOTIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CELL_DEATH;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_NUCLEOBASE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_NEGATIVE_REGULATION_OF_RNA_BIOSYNTHETIC_PROCESS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Not detected</td><td class="tb2">Not detected</td><td class="tb2">.</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">27882058</td><td class="tb2">.</td><td class="tb2">A</td><td class="tb2">C</td><td class="tb2">243.75</td><td class="tb2">THEMIS2</td><td class="tb2">THYMOCYTE SELECTION-ASSOCIATED PROTEIN FAMILY, MEMBER 2; THEMIS2</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">THEMIS2:NM_001105556:exon4:c.A734C:p.K245T,THEMIS2:NM_001286113:exon4:c.A734C:p.K245T</td><td class="tb2">ENST00000373921.8,ENST00000328928.11,ENST00000467258.1,ENST00000373925.5,ENST00000456990.1,ENST00000466068.1,ENST00000482828.1,ENST00000373927.7,ENST00000442118.5</td><td class="tb2">1p35.3</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:35,0:35:99:0,105,1177</td><td class="tb2">0/0:33,0:33:99:0,99,1096</td><td class="tb2">0/0:50,0:50:99:0,111,1714</td><td class="tb2">0/1:59,23:82:99:253,0,1618</td><td class="tb2">A</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0023</td><td class="tb2">0.0073</td><td class="tb2">0.0019</td><td class="tb2">0.0009</td><td class="tb2">0.0001</td><td class="tb2">0.0005</td><td class="tb2">0.0028</td><td class="tb2">5.645e-05</td><td class="tb2">0.0009</td><td class="tb2">0.0017</td><td class="tb2">1.972e-05</td><td class="tb2">0.0002</td><td class="tb2">2.406e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">2.944e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.005,D</td><td class="tb2">0.225,B</td><td class="tb2">0.151,B</td><td class="tb2">1.400196,15.08</td><td class="tb2">5.38</td><td class="tb2">0.145</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">THYMOCYTE SELECTION-ASSOCIATED PROTEIN FAMILY, MEMBER 2; THEMIS2, 617856</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">21455565| 22732588| 9563507</td><td class="tb2">GOBP_CELL_ACTIVATION;GOBP_ACTIVATION_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_IMMUNE_RESPONSE_REGULATING_SIGNALING_PATHWAY;GOBP_IMMUNE_RESPONSE_REGULATING_CELL_SURFACE_RECEPTOR_SIGNALING_PATHWAY;GOBP_DEFENSE_RESPONSE;GOBP_INFLAMMATORY_RESPONSE;GOBP_IMMUNE_RESPONSE;GOBP_CELL_ADHESION;GOBP_B_CELL_ACTIVATION;GOBP_LYMPHOCYTE_ACTIVATION;GOBP_REGULATION_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_IMMUNE_RESPONSE;GOBP_ANTIGEN_RECEPTOR_MEDIATED_SIGNALING_PATHWAY;GOBP_T_CELL_RECEPTOR_SIGNALING_PATHWAY;GOBP_REGULATION_OF_B_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_REGULATION_OF_LYMPHOCYTE_ACTIVATION</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Cytoplasmic and nuclear expression at variable levels in several different tissues, including immune cells in various tissues.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Nucleoplasm</td><td class="tb2">bone marrow: 215.5;lymphoid tissue: 126.0</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">52238880</td><td class="tb2">.</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">182.75</td><td class="tb2">ZFYVE9</td><td class="tb2">ZINC FINGER FYVE TYPE-CONTAINING PROTEIN 9; ZFYVE9</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">ZFYVE9:NM_004799:exon4:c.T1463G:p.V488G,ZFYVE9:NM_007324:exon4:c.T1463G:p.V488G</td><td class="tb2">ENST00000357206.6,ENST00000371591.2,ENST00000361625.5,ENST00000287727.8</td><td class="tb2">1p32.3</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:93,0:93:99:0,120,1800</td><td class="tb2">0/0:35,0:35:99:0,99,1485</td><td class="tb2">0/0:37,0:37:99:0,99,1485</td><td class="tb2">0/1:110,28:138:99:192,0,3068</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0005</td><td class="tb2">0.0012</td><td class="tb2">0.0006</td><td class="tb2">0.0002</td><td class="tb2">2.238e-05</td><td class="tb2">0.0001</td><td class="tb2">0.0006</td><td class="tb2">5.636e-05</td><td class="tb2">0.0001</td><td class="tb2">0.0005</td><td class="tb2">1.972e-05</td><td class="tb2">3.283e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">2.943e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0,D</td><td class="tb2">0.989,D</td><td class="tb2">0.637,P</td><td class="tb2">3.359995,24.3</td><td class="tb2">3.26</td><td class="tb2">0.191</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ZINC FINGER FYVE TYPE-CONTAINING PROTEIN 9; ZFYVE9, 603755</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">17110576| 15356634| 9865696| 10615055</td><td class="tb2">GOBP_ENDOCYTOSIS;GOBP_ENZYME_LINKED_RECEPTOR_PROTEIN_SIGNALING_PATHWAY;GOBP_TRANSMEMBRANE_RECEPTOR_PROTEIN_SERINE_THREONINE_KINASE_SIGNALING_PATHWAY;GOBP_TRANSFORMING_GROWTH_FACTOR_BETA_RECEPTOR_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_VESICLE_MEDIATED_TRANSPORT;GOBP_ENDOSOMAL_TRANSPORT;GOBP_INTRACELLULAR_TRANSPORT;GOBP_RESPONSE_TO_GROWTH_FACTOR;GOBP_RESPONSE_TO_TRANSFORMING_GROWTH_FACTOR_BETA</td><td class="tb2">GOCC_ENDOSOME;GOCC_EARLY_ENDOSOME;GOCC_ENDOSOME_MEMBRANE;GOCC_VESICLE_MEMBRANE;GOCC_EARLY_ENDOSOME_MEMBRANE</td><td class="tb2">GOMF_PHOSPHOLIPID_BINDING;GOMF_1_PHOSPHATIDYLINOSITOL_BINDING;GOMF_LIPID_BINDING;GOMF_PROTEIN_DOMAIN_SPECIFIC_BINDING;GOMF_PHOSPHATIDYLINOSITOL_BINDING</td><td class="tb2">KEGG_TGF_BETA_SIGNALING_PATHWAY</td><td class="tb2">PID_TGFBR_PATHWAY</td><td class="tb2">BIOCARTA_TGFB_PATHWAY</td><td class="tb2">REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX;REACTOME_DOWNREGULATION_OF_TGF_BETA_RECEPTOR_SIGNALING;REACTOME_TGF_BETA_RECEPTOR_SIGNALING_ACTIVATES_SMADS;REACTOME_LOSS_OF_FUNCTION_OF_SMAD2_3_IN_CANCER;REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX_IN_CANCER;REACTOME_DISEASES_OF_SIGNAL_TRANSDUCTION_BY_GROWTH_FACTOR_RECEPTORS_AND_SECOND_MESSENGERS;REACTOME_SIGNALING_BY_TGFB_FAMILY_MEMBERS</td><td class="tb2">Cytoplasmic expression in neuronal cells and most glandular epithelia.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Vesicles,Cytosol</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">100196426</td><td class="tb2">.</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">58.87</td><td class="tb2">DBT</td><td class="tb2">DIHYDROLIPOAMIDE BRANCHED-CHAIN TRANSACYLASE; DBT</td><td class="tb2">splicing</td><td class="tb2">NM_001918:exon11:c.1282-4C>G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000681780.1,ENST00000681617.1,ENST00000370132.8</td><td class="tb2">1p21.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:15,0:15:15:.:.:0,15,225:.</td><td class="tb2">0/0:25,0:25:43:.:.:0,43,577:.</td><td class="tb2">0/0:31,0:31:0:.:.:0,0,444:.</td><td class="tb2">0|1:8,3:11:67:0|1:100196418_A_C:67,0,265:100196418</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0002</td><td class="tb2">0.0003</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">0.0002</td><td class="tb2">0.0003</td><td class="tb2">0.0004</td><td class="tb2">0.0002</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0001</td><td class="tb2">0.038</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.001385,11.64</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Maple syrup urine disease, type II, 248600 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">18063578| 14508502| 1943690| 9239422| 14742428| 2708389| 8430702| 1847055| 1990841| 3245861| 3417306| 2004553| 2837277| 1429740| 10915611| 3189339| 2932110| 16203741| 2010537| 2742576| 7672509| 9621512| 1889817</td><td class="tb2">GOBP_ORGANIC_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_CATABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_METABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_CATABOLIC_PROCESS;GOBP_ORGANIC_ACID_CATABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_CATABOLIC_PROCESS;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION;GOCC_MITOCHONDRIAL_MATRIX;GOCC_MITOCHONDRIAL_ALPHA_KETOGLUTARATE_DEHYDROGENASE_COMPLEX;GOCC_NUCLEOID;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_MITOCHONDRIAL_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_DIHYDROLIPOYL_DEHYDROGENASE_COMPLEX;GOCC_MITOCHONDRIAL_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_OXIDOREDUCTASE_COMPLEX</td><td class="tb2">GOMF_FATTY_ACID_BINDING;GOMF_LIPID_BINDING;GOMF_ACETYLTRANSFERASE_ACTIVITY;GOMF_ACYLTRANSFERASE_ACTIVITY;GOMF_MONOCARBOXYLIC_ACID_BINDING;GOMF_ORGANIC_ACID_BINDING;GOMF_UBIQUITIN_LIKE_PROTEIN_LIGASE_BINDING;GOMF_SULFUR_COMPOUND_BINDING</td><td class="tb2">KEGG_VALINE_LEUCINE_AND_ISOLEUCINE_DEGRADATION</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_GLYOXYLATE_METABOLISM_AND_GLYCINE_DEGRADATION;REACTOME_BRANCHED_CHAIN_AMINO_ACID_CATABOLISM;REACTOME_METABOLISM_OF_AMINO_ACIDS_AND_DERIVATIVES;REACTOME_RHO_GTPASE_CYCLE;REACTOME_RHOH_GTPASE_CYCLE;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3</td><td class="tb2">Granular cytoplasmic expression in most tissues at variable levels.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Mitochondria</td><td class="tb2">.</td></tr>
                                            
                                            
                                        </table>
                                    </div><!-- fcnv-->
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">Note: The search of heterozygous variation is conducted by comparing the genetic information of the affected individual and their parents. Firstly, the mutations were annotated according to gene structure, and then the mutations was screened according to the gene frenquency in the 1000G, gnomAD and other databases. The "functional importance" of non-synonymous SNPs is scored using SIFT and PolyPhen and other software for functional conservative prediction. Then the mutation annotation was carried out and mutations were screened based on function and pathway.
                                The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                
                                
                                <a name="Analysis under recessive model"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.3.1.2 Variant filtering according to recessive inheritance pattern</p>
                                
                                <p class="paragraph">Recessive genetic disease is manifested as normal phenotype of parents, carrying disease-causing mutations, offspring who inherit the disease-causing homozygous mutations from autosomes of parents appear disease. There are two types of mutations that cause recessive disease: homozygous mutation and compound heterozygous mutation. We also take the X-linkage into account when we perform recessive pattern analysis. Recessive pattern analysis means that if a single gene disease is inherited in a recessive pattern in the family, if a mutation is homozygous in the diseased offspring, heterozygous in the normal parent, or the normal parent does not have the mutation, it can be screened as a candidate site. Compound heterozygous mode screening means that if a single gene disease is inherited in a recessive mode in the family, filtering out homozygous mutaitons of patients and normal person, and then screeing the mutation with a person inherits two alleles, one from each parent, and these alleles are located at different positions within the same gene. And the distribution of mutations on this gene cannot be the same as the distribution of that in any normal person (this gene), nor can it be a subset of the mutations on this gene in any normal person.</p>
                                
                                <p class="paragraph">Taking p1 family as an example, we detected 5 candidate genes with SNPs and 0 candidate genes with InDels in this family. And 19 candidate genes with compound heterozygous mutations.
                                The results of gene homozygous mutation screening and annotation are shown in Table 4.12, and the results of gene compound heterozygous mutation screening and annotation are shown in Table 4.5.13.</p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.12 Genetic homozygous mutation screening and annotation result (only the first few lines of information are shown)
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cytoBand</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">OMIM</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr2</td><td class="tb2">29143761</td><td class="tb2">rs138901368</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">8119.33</td><td class="tb2">CLIP4</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">CLIP4:NM_001287527:exon7:c.C701T:p.P234L,CLIP4:NM_001287528:exon7:c.C701T:p.P234L,CLIP4:NM_024692:exon7:c.C701T:p.P234L</td><td class="tb2">ENST00000686125.1,ENST00000693694.1,ENST00000687915.1,ENST00000687166.1,ENST00000693264.1,ENST00000686848.1,ENST00000401617.6,ENST00000320081.10,ENST00000404424.5,ENST00000687104.1,ENST00000687889.1,ENST00000493551.2,ENST00000689070.1,ENST00000687450.1,ENST00000687506.1,ENST00000691509.1,ENST00000685126.1,ENST00000686470.1,ENST00000688724.1,ENST00000401605.5,ENST00000691585.1,ENST00000689045.1,ENST00000456385.5,ENST00000691796.1,ENST00000690063.1,ENST00000693513.1,ENST00000693041.1,ENST00000415891.5,ENST00000690615.1,ENST00000689017.1,ENST00000691341.1,ENST00000693446.1,ENST00000688337.1,ENST00000689605.1,ENST00000691129.1</td><td class="tb2">2p23.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/1:58,38:96:99:1095,0,1819</td><td class="tb2">0/1:44,45:89:99:1247,0,1250</td><td class="tb2">0/1:57,52:109:99:1556,0,1700</td><td class="tb2">1/1:0,124:124:99:4233,372,0</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0.00119808</td><td class="tb2">0.0002</td><td class="tb2">0.0004</td><td class="tb2">0.0004</td><td class="tb2">0.0002</td><td class="tb2">0.0004</td><td class="tb2">0.0074</td><td class="tb2">0.0030</td><td class="tb2">1.079e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0016</td><td class="tb2">0.0006</td><td class="tb2">0.0006</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0.0037</td><td class="tb2">0.0037</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">0.0011</td><td class="tb2">0.0004</td><td class="tb2">0.0087</td><td class="tb2">0.0034</td><td class="tb2">0</td><td class="tb2">1.5e-05</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Ankyrin repeat-containing domain;Ankyrin repeat-containing domain;Ankyrin repeat-containing domain</td><td class="tb2">0.004,D</td><td class="tb2">1.0,D</td><td class="tb2">0.959,D</td><td class="tb2">3.152850,23.7</td><td class="tb2">5.49</td><td class="tb2">0.460</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_CYTOPLASMIC_MICROTUBULE_ORGANIZATION;GOBP_SUPRAMOLECULAR_FIBER_ORGANIZATION</td><td class="tb2">GOCC_MICROTUBULE;GOCC_CELL_CORTEX;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_MICROTUBULE_PLUS_END;GOCC_SUPRAMOLECULAR_COMPLEX;GOCC_SUPRAMOLECULAR_POLYMER;GOCC_POLYMERIC_CYTOSKELETAL_FIBER;GOCC_MICROTUBULE_END</td><td class="tb2">GOMF_MICROTUBULE_BINDING;GOMF_CYTOSKELETAL_PROTEIN_BINDING;GOMF_TUBULIN_BINDING;GOMF_MICROTUBULE_PLUS_END_BINDING</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Cytoplasmic expression in smooth muscle cells in several tissues.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Vesicles</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr9</td><td class="tb2">34726473</td><td class="tb2">rs145034991</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">15245.3</td><td class="tb2">FAM205A</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">FAM205A:NM_001141917:exon4:c.C767T:p.S256F</td><td class="tb2">ENST00000664167.1,ENST00000378788.4</td><td class="tb2">9p13.3</td><td class="tb2">Score=0.912588;Name=chr9:34832017</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:38,0:38:99:0,99,1506</td><td class="tb2">0/1:183,115:298:99:2768,0,5012</td><td class="tb2">0/1:170,124:294:99:3135,0,4982</td><td class="tb2">1/1:0,309:309:99:9358,926,0</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">0.00459265</td><td class="tb2">.</td><td class="tb2">0.0012</td><td class="tb2">0.0012</td><td class="tb2">6.33e-05</td><td class="tb2">0.0006</td><td class="tb2">5.601e-05</td><td class="tb2">0.0408</td><td class="tb2">6.673e-05</td><td class="tb2">2.027e-05</td><td class="tb2">0.0003</td><td class="tb2">0.0026</td><td class="tb2">0.0014</td><td class="tb2">0.0015</td><td class="tb2">0.0002</td><td class="tb2">0.0006</td><td class="tb2">0.0005</td><td class="tb2">0.0358</td><td class="tb2">7.354e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0052</td><td class="tb2">0.0011</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0257</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">0.0008</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.001,D</td><td class="tb2">0.688,P</td><td class="tb2">0.332,B</td><td class="tb2">1.116191,12.93</td><td class="tb2">.</td><td class="tb2">0.028</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Distinct expression in spermatid acrosomes.</td><td class="tb2">Tissue enriched<At least four-fold higher mRNA level in a particular tissue compared to any other tissue.></td><td class="tb2">Detected in a single tissue.</td><td class="tb2">Nucleoli fibrillar center,Plasma membrane</td><td class="tb2">testis: 24.8</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr19</td><td class="tb2">50723539</td><td class="tb2">rs201461602</td><td class="tb2">G</td><td class="tb2">T</td><td class="tb2">1271.3</td><td class="tb2">CLEC11A</td><td class="tb2">C-TYPE LECTIN DOMAIN FAMILY 11, MEMBER A; CLEC11A</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">CLEC11A:NM_002975:exon1:c.G14T:p.W5L</td><td class="tb2">ENST00000250340.9,ENST00000599973.1</td><td class="tb2">19q13.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:30,0:30:87:0,87,1305</td><td class="tb2">0/1:13,6:19:99:188,0,414</td><td class="tb2">0/1:9,14:23:99:457,0,300</td><td class="tb2">1/1:0,17:17:51:642,51,0</td><td class="tb2">G</td><td class="tb2">T</td><td class="tb2">0.00159744</td><td class="tb2">.</td><td class="tb2">0.0003</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">6.963e-05</td><td class="tb2">0</td><td class="tb2">0.0099</td><td class="tb2">8.993e-07</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">0.0002</td><td class="tb2">0.0002</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0068</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0057</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0,D</td><td class="tb2">0.816,P</td><td class="tb2">0.227,B</td><td class="tb2">3.599607,25.0</td><td class="tb2">2.08</td><td class="tb2">0.196</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">C-TYPE LECTIN DOMAIN FAMILY 11, MEMBER A; CLEC11A, 604713</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">10198175| 9442024| 9207134| 9705843</td><td class="tb2">GOBP_OSSIFICATION;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION</td><td class="tb2">.</td><td class="tb2">GOMF_SIGNALING_RECEPTOR_BINDING;GOMF_GROWTH_FACTOR_ACTIVITY;GOMF_CARBOHYDRATE_BINDING;GOMF_SIGNALING_RECEPTOR_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_ACTIVATOR_ACTIVITY</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Cytoplasmic expression in bone marrow and chondrocytes. Extracellular deposits were positive.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Centrosome</td><td class="tb2">bone marrow: 132.6</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chrX</td><td class="tb2">154310989</td><td class="tb2">rs74546727</td><td class="tb2">C</td><td class="tb2">G</td><td class="tb2">8866.08</td><td class="tb2">TKTL1</td><td class="tb2">TRANSKETOLASE-LIKE 1; TKTL1</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">TKTL1:NM_001145934:exon3:c.C336G:p.H112Q,TKTL1:NM_001145933:exon4:c.C486G:p.H162Q,TKTL1:NM_012253:exon4:c.C504G:p.H168Q</td><td class="tb2">ENST00000369915.8,ENST00000439635.5,ENST00000426203.5,ENST00000426989.5,ENST00000369912.2</td><td class="tb2">Xq28</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:35,0:35:99:0,99,1362</td><td class="tb2">0/0:35,0:35:99:0,105,1359</td><td class="tb2">0/1:151,128:279:99:3663,0,4364</td><td class="tb2">1/1:0,158:158:99:5221,474,0</td><td class="tb2">C</td><td class="tb2">G</td><td class="tb2">0.00688742</td><td class="tb2">.</td><td class="tb2">0.0008</td><td class="tb2">0.0008</td><td class="tb2">0</td><td class="tb2">0.0004</td><td class="tb2">0</td><td class="tb2">0.0261</td><td class="tb2">8.313e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0018</td><td class="tb2">0.0010</td><td class="tb2">0.0010</td><td class="tb2">0</td><td class="tb2">0.0011</td><td class="tb2">0.0002</td><td class="tb2">0.0304</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0027</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0347</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0016</td><td class="tb2">0.0004</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Transketolase, N-terminal;Transketolase, N-terminal</td><td class="tb2">0.011,D</td><td class="tb2">0.98,D</td><td class="tb2">0.878,P</td><td class="tb2">1.462249,15.49</td><td class="tb2">3.38</td><td class="tb2">0.194</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">TRANSKETOLASE-LIKE 1; TKTL1, 300044</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">8838793| 8419340</td><td class="tb2">GOBP_CARBOHYDRATE_METABOLIC_PROCESS;GOBP_MONOSACCHARIDE_METABOLIC_PROCESS;GOBP_GLUCOSE_METABOLIC_PROCESS;GOBP_GLUCOSE_CATABOLIC_PROCESS;GOBP_ALCOHOL_METABOLIC_PROCESS;GOBP_VITAMIN_METABOLIC_PROCESS;GOBP_WATER_SOLUBLE_VITAMIN_METABOLIC_PROCESS;GOBP_SULFUR_COMPOUND_METABOLIC_PROCESS;GOBP_CARBOHYDRATE_CATABOLIC_PROCESS;GOBP_HEXOSE_CATABOLIC_PROCESS;GOBP_PRIMARY_ALCOHOL_METABOLIC_PROCESS;GOBP_THIAMINE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_CATABOLIC_PROCESS;GOBP_MONOSACCHARIDE_CATABOLIC_PROCESS;GOBP_PYRIMIDINE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_ORGANIC_HYDROXY_COMPOUND_METABOLIC_PROCESS</td><td class="tb2">.</td><td class="tb2">GOMF_TRANSKETOLASE_OR_TRANSALDOLASE_ACTIVITY;GOMF_VITAMIN_BINDING;GOMF_THIAMINE_PYROPHOSPHATE_BINDING;GOMF_QUATERNARY_AMMONIUM_GROUP_BINDING;GOMF_SULFUR_COMPOUND_BINDING</td><td class="tb2">KEGG_PENTOSE_PHOSPHATE_PATHWAY</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Selective cytoplasmic expression in spermatogonia.</td><td class="tb2">Tissue enriched<At least four-fold higher mRNA level in a particular tissue compared to any other tissue.></td><td class="tb2">Detected in more than one but less than one third of tissues.</td><td class="tb2">Cytosol</td><td class="tb2">testis: 127.1</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chrX</td><td class="tb2">154365245</td><td class="tb2">rs143873938</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">6168.08</td><td class="tb2">FLNA</td><td class="tb2">FILAMIN A; FLNA</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">FLNA:NM_001110556:exon11:c.G1582A:p.V528M,FLNA:NM_001456:exon11:c.G1582A:p.V528M</td><td class="tb2">ENST00000422373.6,ENST00000360319.9,ENST00000673639.2,ENST00000610817.5,ENST00000369856.8,ENST00000369850.10,ENST00000420627.5,ENST00000676696.1</td><td class="tb2">Xq28</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">0/0:36,0:36:99:0,99,1383</td><td class="tb2">0/0:36,0:36:99:0,99,1398</td><td class="tb2">0/1:93,87:180:99:2653,0,2846</td><td class="tb2">1/1:0,104:104:99:3533,312,0</td><td class="tb2">C</td><td class="tb2">T</td><td class="tb2">0.00847682</td><td class="tb2">.</td><td class="tb2">0.0012</td><td class="tb2">0.0012</td><td class="tb2">3.787e-05</td><td class="tb2">0.0002</td><td class="tb2">2.841e-05</td><td class="tb2">0.0420</td><td class="tb2">2.138e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0008</td><td class="tb2">0.0012</td><td class="tb2">0.0013</td><td class="tb2">0</td><td class="tb2">0.0004</td><td class="tb2">0.0002</td><td class="tb2">0.0362</td><td class="tb2">5.639e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0033</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0427</td><td class="tb2">0</td><td class="tb2">4.202e-05</td><td class="tb2">0.0032</td><td class="tb2">0.0004</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.001,D</td><td class="tb2">0.999,D</td><td class="tb2">0.953,D</td><td class="tb2">3.049226,23.5</td><td class="tb2">4.3</td><td class="tb2">0.447</td><td class="tb2">191008</td><td class="tb2">Heterotopia,_periventricular,_X-linked_dominant|Oto-palato-digital_syndrome,_type_II|Melnick-Needles_syndrome|Frontometaphyseal_dysplasia|History_of_neurodevelopmental_disorder|not_specified|Cardiovascular_phenotype|not_provided</td><td class="tb2">MONDO:MONDO:0010233,MedGen:C1848213,OMIM:300049,SNOMED_CT:448227009|MONDO:MONDO:0010571,MedGen:C1844696,OMIM:304120,Orphanet:ORPHA90652,SNOMED_CT:42432003|MONDO:MONDO:0010650,MedGen:C0025237,OMIM:309350,Orphanet:ORPHA2484,SNOMED_CT:13449007|MONDO:MONDO:0015942,MedGen:C0265293,OMIM:PS305620,Orphanet:ORPHA1826,SNOMED_CT:62803002|MedGen:C2711754|MedGen:CN169374|MedGen:CN230736|MedGen:CN517202</td><td class="tb2">criteria_provided,_multiple_submitters,_no_conflicts</td><td class="tb2">Benign/Likely_benign</td><td class="tb2">?FG syndrome 2, 300321 (3), XL;Cardiac valvular dysplasia, X-linked, 314400 (3), XL;Congenital short bowel syndrome, 300048 (3), XLR;Frontometaphyseal dysplasia 1, 305620 (3), XLR;Heterotopia, periventricular, 1, 300049 (3), XLD;Intestinal pseudoobstruction, neuronal, 300048 (3), XLR;Melnick-Needles syndrome, 309350 (3), XLD;Otopalatodigital syndrome, type I, 311300 (3), XLD;Otopalatodigital syndrome, type II, 304120 (3), XLD;Terminal osseous dysplasia, 300244 (3), XLD</td><td class="tb2">Heterotopia, periventricular(CM023918)</td><td class="tb2">GTG-ATG|Val528Met|c.1582G>A|p.V528M(12410386|20844545|26133172|26133172|28976722|28411558)</td><td class="tb2">12410386|20844545|26133172|26133172|28976722|28411558|22121117| 8644737| 30561107| 1854572| 17152064| 10982965| 11153914| 19773341| 18854860| 240645| 15864382| 9071288| 21926999| 17172441| 9311743| 9279759| 9883725| 17357080| 8088838| 15523633| 15994863| 8406501| 2391361| 15128042| 16299064| 15940695| 9800904| 24760772| 8290091| 25140959| 20014127| 17572668| 2123245| 17264970| 17190868| 9497244| 12682292| 7689010| 2177445| 17431908| 29237676| 11914408| 8230166| 25755106| 8088819| 27739212| 16538226| 12612583| 15917206| 3658675| 11532987| 12393796| 15668422| 12022328| 9107678| 15654694| 20598277| 17235394| 17632775| 12198493| 23037936| 16596676| 14988809</td><td class="tb2">GOBP_MITOTIC_SISTER_CHROMATID_SEGREGATION;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_MITOTIC_CELL_CYCLE;GOBP_SISTER_CHROMATID_SEGREGATION;GOBP_SKELETAL_SYSTEM_DEVELOPMENT;GOBP_ACTION_POTENTIAL;GOBP_CELL_ACTIVATION;GOBP_EPITHELIAL_TO_MESENCHYMAL_TRANSITION;GOBP_VASCULATURE_DEVELOPMENT;GOBP_BLOOD_VESSEL_REMODELING;GOBP_MORPHOGENESIS_OF_AN_EPITHELIUM;GOBP_MORPHOGENESIS_OF_AN_EPITHELIAL_SHEET;GOBP_HEART_MORPHOGENESIS;GOBP_MUSCLE_SYSTEM_PROCESS;GOBP_CIRCULATORY_SYSTEM_PROCESS;GOBP_HEART_PROCESS;GOBP_REGULATION_OF_TRANSCRIPTION_BY_RNA_POLYMERASE_I;GOBP_TRANSCRIPTION_BY_RNA_POLYMERASE_I;GOBP_MONOATOMIC_ION_TRANSPORT;GOBP_MONOATOMIC_CATION_TRANSPORT;GOBP_POTASSIUM_ION_TRANSPORT;GOBP_CALCIUM_ION_TRANSPORT;GOBP_INTRACELLULAR_MONOATOMIC_ION_HOMEOSTASIS;GOBP_INTRACELLULAR_PROTEIN_TRANSPORT;GOBP_APOPTOTIC_PROCESS;GOBP_MUSCLE_CONTRACTION;GOBP_STRIATED_MUSCLE_CONTRACTION;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_ACTIN_FILAMENT_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_CELL_CYCLE;GOBP_SPINDLE_ORGANIZATION;GOBP_MITOTIC_SPINDLE_ORGANIZATION;GOBP_CHROMOSOME_SEGREGATION;GOBP_CELL_ADHESION;GOBP_G_PROTEIN_COUPLED_RECEPTOR_SIGNALING_PATHWAY;GOBP_ADENYLATE_CYCLASE_MODULATING_G_PROTEIN_COUPLED_RECEPTOR_SIGNALING_PATHWAY;GOBP_ADENYLATE_CYCLASE_INHIBITING_G_PROTEIN_COUPLED_RECEPTOR_SIGNALING_PATHWAY;GOBP_ADENYLATE_CYCLASE_INHIBITING_DOPAMINE_RECEPTOR_SIGNALING_PATHWAY;GOBP_DOPAMINE_RECEPTOR_SIGNALING_PATHWAY;GOBP_INTEGRIN_MEDIATED_SIGNALING_PATHWAY;GOBP_I_KAPPAB_KINASE_NF_KAPPAB_SIGNALING;GOBP_CELL_CELL_SIGNALING;GOBP_HEART_DEVELOPMENT;GOBP_BLOOD_COAGULATION_INTRINSIC_PATHWAY;GOBP_HEMOSTASIS;GOBP_REGULATION_OF_HEART_CONTRACTION;GOBP_MACROMOLECULE_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_WOUNDING;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_ANIMAL_ORGAN_MORPHOGENESIS;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CATABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CATABOLIC_PROCESS;GOBP_REGULATION_OF_PLATELET_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_PLATELET_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_COMMUNICATION;GOBP_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_REGULATION_OF_METAL_ION_TRANSPORT;GOBP_REGULATION_OF_NEURON_PROJECTION_DEVELOPMENT;GOBP_POSITIVE_REGULATION_OF_NEURON_PROJECTION_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_NEURON_PROJECTION_DEVELOPMENT;GOBP_RESPONSE_TO_ORGANIC_CYCLIC_COMPOUND;GOBP_VESICLE_MEDIATED_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_TRANSCRIPTION_BY_RNA_POLYMERASE_I;GOBP_CYTOSOLIC_TRANSPORT;GOBP_CELLULAR_HOMEOSTASIS;GOBP_NEUROGENESIS;GOBP_CELL_CYCLE_PROCESS;GOBP_ACTIN_FILAMENT_BASED_PROCESS;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_CELL_PROJECTION_ASSEMBLY;GOBP_ACTIN_FILAMENT_BASED_MOVEMENT;GOBP_HEMOPOIESIS;GOBP_MYELOID_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CELL_ADHESION;GOBP_PROTEIN_CATABOLIC_PROCESS;GOBP_PLATELET_ACTIVATION;GOBP_MEGAKARYOCYTE_DIFFERENTIATION;GOBP_REGENERATION;GOBP_NEURON_PROJECTION_REGENERATION;GOBP_REGULATION_OF_CELL_PROJECTION_ORGANIZATION;GOBP_NEGATIVE_REGULATION_OF_CELL_PROJECTION_ORGANIZATION;GOBP_POSITIVE_REGULATION_OF_CELL_PROJECTION_ORGANIZATION;GOBP_ACTIN_CYTOSKELETON_REORGANIZATION;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_PROTEIN_STABILITY;GOBP_REGULATION_OF_RESPONSE_TO_EXTERNAL_STIMULUS;GOBP_POSITIVE_REGULATION_OF_RESPONSE_TO_EXTERNAL_STIMULUS;GOBP_REGULATION_OF_ACTIN_FILAMENT_BUNDLE_ASSEMBLY;GOBP_REGULATION_OF_INTRACELLULAR_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_TRANSPORT;GOBP_REGULATION_OF_ACTIN_FILAMENT_BASED_PROCESS;GOBP_REGULATION_OF_ORGANELLE_ORGANIZATION;GOBP_REGULATION_OF_INTRACELLULAR_PROTEIN_TRANSPORT;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_CELLULAR_RESPONSE_TO_STRESS;GOBP_HOMOTYPIC_CELL_CELL_ADHESION;GOBP_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_CELL_JUNCTION_ORGANIZATION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_SURFACE;GOBP_SUBSTRATE_ADHESION_DEPENDENT_CELL_SPREADING;GOBP_PROTEIN_LOCALIZATION_TO_NUCLEUS;GOBP_REGULATION_OF_TRANSMEMBRANE_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSMEMBRANE_TRANSPORT;GOBP_REGULATION_OF_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_MONOATOMIC_ION_TRANSMEMBRANE_TRANSPORT;GOBP_TUBE_MORPHOGENESIS;GOBP_TUBE_DEVELOPMENT;GOBP_MEGAKARYOCYTE_DEVELOPMENT;GOBP_LOCOMOTION;GOBP_REGULATION_OF_LOCOMOTION;GOBP_WOUND_HEALING;GOBP_REGULATION_OF_PROTEIN_CATABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_CATABOLIC_PROCESS;GOBP_REGULATION_OF_PROTEIN_IMPORT_INTO_NUCLEUS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_IMPORT_INTO_NUCLEUS;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_HOMEOSTATIC_PROCESS;GOBP_RECEPTOR_CLUSTERING;GOBP_POSITIVE_REGULATION_OF_I_KAPPAB_KINASE_NF_KAPPAB_SIGNALING;GOBP_REGULATION_OF_POTASSIUM_ION_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_POTASSIUM_ION_TRANSPORT;GOBP_REGULATION_OF_MONOATOMIC_ION_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_MONOATOMIC_ION_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_REGULATION_OF_CELLULAR_COMPONENT_BIOGENESIS;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_CILIUM_ORGANIZATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_MAINTENANCE_OF_PROTEIN_LOCATION;GOBP_CELL_CELL_JUNCTION_ORGANIZATION;GOBP_POSITIVE_REGULATION_OF_CELL_ADHESION;GOBP_NEGATIVE_REGULATION_OF_NUCLEOBASE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_REGULATION_OF_NUCLEOCYTOPLASMIC_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_NUCLEOCYTOPLASMIC_TRANSPORT;GOBP_INTRACELLULAR_TRANSPORT;GOBP_ORGANELLE_FISSION;GOBP_BLOOD_VESSEL_MORPHOGENESIS;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_NEURON_DEVELOPMENT;GOBP_RESPONSE_TO_AXON_INJURY;GOBP_GENERATION_OF_NEURONS;GOBP_TISSUE_MORPHOGENESIS;GOBP_MESENCHYMAL_CELL_DIFFERENTIATION;GOBP_TISSUE_REMODELING;GOBP_CELL_MOTILITY;GOBP_CHEMICAL_HOMEOSTASIS;GOBP_MONOATOMIC_ION_HOMEOSTASIS;GOBP_SYNAPSE_ORGANIZATION;GOBP_PROTEIN_STABILIZATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_ACTIVATION;GOBP_REGULATION_OF_BODY_FLUID_LEVELS;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_POSITIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_NUCLEAR_TRANSPORT;GOBP_IMPORT_INTO_NUCLEUS;GOBP_SEQUESTERING_OF_CALCIUM_ION;GOBP_CYTOPLASMIC_SEQUESTERING_OF_PROTEIN;GOBP_SPINDLE_ASSEMBLY;GOBP_MAINTENANCE_OF_LOCATION;GOBP_POSITIVE_REGULATION_OF_MULTICELLULAR_ORGANISMAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_CHROMOSOME_ORGANIZATION;GOBP_REGULATION_OF_CYTOSKELETON_ORGANIZATION;GOBP_PROTEIN_MATURATION;GOBP_MAINTENANCE_OF_LOCATION_IN_CELL;GOBP_LOCALIZATION_WITHIN_MEMBRANE;GOBP_ACTIN_CROSSLINK_FORMATION;GOBP_CALCIUM_ION_HOMEOSTASIS;GOBP_TRANSMEMBRANE_TRANSPORT;GOBP_CARDIAC_MUSCLE_CONTRACTION;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_CELLULAR_LOCALIZATION;GOBP_BONE_DEVELOPMENT;GOBP_REGULATION_OF_ATRIAL_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_EPITHELIUM_DEVELOPMENT;GOBP_MESENCHYME_DEVELOPMENT;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_CARDIAC_CONDUCTION;GOBP_MYELOID_CELL_DEVELOPMENT;GOBP_AXON_DEVELOPMENT;GOBP_ACTIN_FILAMENT_BUNDLE_ORGANIZATION;GOBP_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_ACTIN_MEDIATED_CELL_CONTRACTION;GOBP_PLATELET_AGGREGATION;GOBP_REGULATION_OF_NEURON_PROJECTION_REGENERATION;GOBP_POSITIVE_REGULATION_OF_NEURON_PROJECTION_REGENERATION;GOBP_CALCIUM_ION_TRANSMEMBRANE_TRANSPORT;GOBP_ORGANELLE_ASSEMBLY;GOBP_CELLULAR_RESPONSE_TO_ORGANIC_CYCLIC_COMPOUND;GOBP_SEMAPHORIN_PLEXIN_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_CATECHOLAMINE;GOBP_CIRCULATORY_SYSTEM_DEVELOPMENT;GOBP_PROTEIN_ACTIVATION_CASCADE;GOBP_BLOOD_COAGULATION_FIBRIN_CLOT_FORMATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_PROTEIN_LOCALIZATION_TO_PLASMA_MEMBRANE;GOBP_REGULATION_OF_RESPONSE_TO_STRESS;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_STRESS;GOBP_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL_INVOLVED_IN_CONTRACTION;GOBP_CARDIAC_MUSCLE_CELL_CONTRACTION;GOBP_MEMBRANE_REPOLARIZATION;GOBP_MEMBRANE_REPOLARIZATION_DURING_ACTION_POTENTIAL;GOBP_MEMBRANE_REPOLARIZATION_DURING_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_CELL_CELL_SIGNALING_INVOLVED_IN_CARDIAC_CONDUCTION;GOBP_ATRIAL_CARDIAC_MUSCLE_CELL_TO_AV_NODE_CELL_SIGNALING;GOBP_CELL_COMMUNICATION_INVOLVED_IN_CARDIAC_CONDUCTION;GOBP_TUBULIN_DEACETYLATION;GOBP_MITOTIC_SPINDLE_ASSEMBLY;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_PROTEIN_TRANSPORT;GOBP_EPIBOLY;GOBP_SUPRAMOLECULAR_FIBER_ORGANIZATION;GOBP_CALCIUM_ION_TRANSMEMBRANE_IMPORT_INTO_CYTOSOL;GOBP_CELL_CELL_ADHESION;GOBP_MONOATOMIC_CATION_TRANSMEMBRANE_TRANSPORT;GOBP_MACROMOLECULE_DEACYLATION;GOBP_BONE_CELL_DEVELOPMENT;GOBP_INORGANIC_ION_HOMEOSTASIS;GOBP_NUCLEAR_CHROMOSOME_SEGREGATION;GOBP_REGULATION_OF_ACTION_POTENTIAL;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION_DURING_ACTION_POTENTIAL;GOBP_MEMBRANE_REPOLARIZATION_DURING_ATRIAL_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_VESICLE_MEDIATED_TRANSPORT_BETWEEN_ENDOSOMAL_COMPARTMENTS;GOBP_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_ATRIAL_CARDIAC_MUSCLE_CELL_MEMBRANE_REPOLARIZATION;GOBP_REGULATION_OF_ACTIN_FILAMENT_ORGANIZATION;GOBP_MITOTIC_NUCLEAR_DIVISION;GOBP_NON_MEMBRANE_BOUNDED_ORGANELLE_ASSEMBLY;GOBP_REGULATION_OF_SUBSTRATE_ADHESION_DEPENDENT_CELL_SPREADING;GOBP_POSITIVE_REGULATION_OF_SUBSTRATE_ADHESION_DEPENDENT_CELL_SPREADING;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_NUCLEUS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION_TO_NUCLEUS;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS;GOBP_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_NEGATIVE_REGULATION_OF_RNA_BIOSYNTHETIC_PROCESS;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION_INVOLVED_IN_MITOSIS;GOBP_REGULATION_OF_SUPRAMOLECULAR_FIBER_ORGANIZATION;GOBP_REGULATION_OF_RESPONSE_TO_WOUNDING;GOBP_POSITIVE_REGULATION_OF_RESPONSE_TO_WOUNDING;GOBP_MITOTIC_CELL_CYCLE_PROCESS;GOBP_RESPONSE_TO_DOPAMINE;GOBP_REGULATION_OF_BLOOD_CIRCULATION;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_MONOATOMIC_CATION_TRANSMEMBRANE_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_MEMBRANE_REPOLARIZATION_DURING_CARDIAC_MUSCLE_CELL_ACTION_POTENTIAL;GOBP_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_REGULATION_OF_INTEGRIN_MEDIATED_SIGNALING_PATHWAY;GOBP_POSITIVE_REGULATION_OF_INTEGRIN_MEDIATED_SIGNALING_PATHWAY</td><td class="tb2">GOCC_NUCLEOLUS;GOCC_GOLGI_APPARATUS;GOCC_TRANS_GOLGI_NETWORK;GOCC_BRUSH_BORDER;GOCC_CELL_CELL_JUNCTION;GOCC_CELL_CORTEX;GOCC_ACTIN_CYTOSKELETON;GOCC_CELL_SUBSTRATE_JUNCTION;GOCC_AXON;GOCC_SITE_OF_POLARIZED_GROWTH;GOCC_I_BAND;GOCC_ORGANELLE_SUBCOMPARTMENT;GOCC_ACTIN_FILAMENT_BUNDLE;GOCC_SOMATODENDRITIC_COMPARTMENT;GOCC_NEURON_PROJECTION;GOCC_PERIKARYON;GOCC_RECEPTOR_COMPLEX;GOCC_CONTRACTILE_FIBER;GOCC_AXONAL_GROWTH_CONE;GOCC_CELL_BODY;GOCC_ANCHORING_JUNCTION;GOCC_GLYCOPROTEIN_COMPLEX;GOCC_GOLGI_APPARATUS_SUBCOMPARTMENT;GOCC_MEMBRANE_PROTEIN_COMPLEX;GOCC_PLASMA_MEMBRANE_PROTEIN_COMPLEX;GOCC_PLASMA_MEMBRANE_SIGNALING_RECEPTOR_COMPLEX;GOCC_CLUSTER_OF_ACTIN_BASED_CELL_PROJECTIONS;GOCC_SUPRAMOLECULAR_COMPLEX;GOCC_SUPRAMOLECULAR_POLYMER;GOCC_NUCLEAR_PROTEIN_CONTAINING_COMPLEX;GOCC_DISTAL_AXON;GOCC_GLYCOPROTEIN_IB_IX_V_COMPLEX</td><td class="tb2">GOMF_G_PROTEIN_COUPLED_RECEPTOR_BINDING;GOMF_RNA_BINDING;GOMF_ACTIN_BINDING;GOMF_PROTEIN_KINASE_C_BINDING;GOMF_SIGNALING_RECEPTOR_BINDING;GOMF_CYTOSKELETAL_PROTEIN_BINDING;GOMF_TRANSCRIPTION_FACTOR_BINDING;GOMF_POTASSIUM_CHANNEL_REGULATOR_ACTIVITY;GOMF_CHANNEL_REGULATOR_ACTIVITY;GOMF_KINASE_BINDING;GOMF_IMMUNOGLOBULIN_RECEPTOR_BINDING;GOMF_PROTEIN_HOMODIMERIZATION_ACTIVITY;GOMF_TRANSMEMBRANE_TRANSPORTER_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_CADHERIN_BINDING;GOMF_PROTEIN_DIMERIZATION_ACTIVITY;GOMF_CELL_ADHESION_MOLECULE_BINDING;GOMF_ACTIN_FILAMENT_BINDING;GOMF_GTPASE_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_DNA_BINDING_TRANSCRIPTION_FACTOR_BINDING</td><td class="tb2">KEGG_MAPK_SIGNALING_PATHWAY;KEGG_FOCAL_ADHESION</td><td class="tb2">PID_WNT_NONCANONICAL_PATHWAY;PID_TCR_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_HEMOSTASIS;REACTOME_ANTIVIRAL_MECHANISM_BY_IFN_STIMULATED_GENES;REACTOME_CYTOKINE_SIGNALING_IN_IMMUNE_SYSTEM;REACTOME_CELL_CELL_COMMUNICATION;REACTOME_RHO_GTPASE_EFFECTORS;REACTOME_GP1B_IX_V_ACTIVATION_SIGNALLING;REACTOME_CELL_EXTRACELLULAR_MATRIX_INTERACTIONS;REACTOME_CELL_JUNCTION_ORGANIZATION;REACTOME_RHO_GTPASES_ACTIVATE_PAKS;REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION;REACTOME_RESPONSE_TO_ELEVATED_PLATELET_CYTOSOLIC_CA2;REACTOME_OAS_ANTIVIRAL_RESPONSE;REACTOME_INTERFERON_SIGNALING;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Selective cytoplasmic expression in smooth muscle, myoepithelia, squamous epithelia and subsets of lymphoid cells.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in all tissues.</td><td class="tb2">Plasma membrane,Actin filaments,Cytosol</td><td class="tb2">endometrium 1: 1203.7;intestine: 1462.2</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">Recessive homozygous variation refers to normal parents who carry the same autosomal recessive disease-causing mutation.
                                The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                
                                
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.13 Genetic homozygous mutation screening and annotation result (only the first few lines of information are shown)
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cytoBand</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">FORMAT</th><th class="tb2">YXHB</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">OMIM</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">13318448</td><td class="tb2">.</td><td class="tb2">T</td><td class="tb2">A</td><td class="tb2">157.98</td><td class="tb2">PRAMEF15</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">PRAMEF15:NM_001098376:exon2:c.T41A:p.L14H</td><td class="tb2">ENST00000376152.2</td><td class="tb2">1p36.21</td><td class="tb2">Score=0.990208;Name=chr1:12855323</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:40,0:40:99:.:.:0,107,1488:.</td><td class="tb2">0/1:38,4:42:17:.:.:17,0,1486:.</td><td class="tb2">0/0:34,0:34:99:.:.:0,99,1485:.</td><td class="tb2">0|1:40,7:47:99:0|1:13318448_T_A:153,0,1617:13318448</td><td class="tb2">T</td><td class="tb2">A</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0009</td><td class="tb2">0.0174</td><td class="tb2">6.023e-05</td><td class="tb2">0.0008</td><td class="tb2">0.0002</td><td class="tb2">0.0244</td><td class="tb2">0.0003</td><td class="tb2">0.0023</td><td class="tb2">0.0002</td><td class="tb2">0.0013</td><td class="tb2">0.0066</td><td class="tb2">0.0279</td><td class="tb2">0.0014</td><td class="tb2">0.0137</td><td class="tb2">0.0044</td><td class="tb2">0.0603</td><td class="tb2">0.0061</td><td class="tb2">0.0116</td><td class="tb2">0.0033</td><td class="tb2">0.0082</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.972357,19.05</td><td class="tb2">.</td><td class="tb2">0.084</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_APOPTOTIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CELL_DEATH;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_NUCLEOBASE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_NEGATIVE_REGULATION_OF_RNA_BIOSYNTHETIC_PROCESS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Not detected</td><td class="tb2">Not detected</td><td class="tb2">.</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">13322125</td><td class="tb2">.</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">239.75</td><td class="tb2">PRAMEF15</td><td class="tb2">.</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">PRAMEF15:NM_001098376:exon4:c.T1298G:p.F433C</td><td class="tb2">ENST00000376152.2</td><td class="tb2">1p36.21</td><td class="tb2">Score=0.990208;Name=chr1:12855323</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:37,0:37:99:.:.:0,102,1422:.</td><td class="tb2">0/0:80,0:80:31:.:.:0,31,2297:.</td><td class="tb2">0/0:40,0:40:99:.:.:0,105,1578:.</td><td class="tb2">0|1:1,6:7:18:0|1:13322125_T_G:249,0,18:13322125</td><td class="tb2">T</td><td class="tb2">G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0025</td><td class="tb2">0.0184</td><td class="tb2">0.0002</td><td class="tb2">0.0024</td><td class="tb2">0.0007</td><td class="tb2">0.0470</td><td class="tb2">0.0013</td><td class="tb2">0.0038</td><td class="tb2">0.0007</td><td class="tb2">0.0040</td><td class="tb2">0.0073</td><td class="tb2">0.0209</td><td class="tb2">0.0017</td><td class="tb2">0.0098</td><td class="tb2">0.0046</td><td class="tb2">0.0427</td><td class="tb2">0.0090</td><td class="tb2">0.0087</td><td class="tb2">0.0057</td><td class="tb2">0.0096</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.001,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.723959,17.19</td><td class="tb2">.</td><td class="tb2">0.035</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GOBP_APOPTOTIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_NEGATIVE_REGULATION_OF_BIOSYNTHETIC_PROCESS;GOBP_REGULATION_OF_CELL_DEATH;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_NEGATIVE_REGULATION_OF_NUCLEOBASE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_NEGATIVE_REGULATION_OF_RNA_BIOSYNTHETIC_PROCESS</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Not detected</td><td class="tb2">Not detected</td><td class="tb2">.</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">100196418</td><td class="tb2">.</td><td class="tb2">A</td><td class="tb2">C</td><td class="tb2">54.37</td><td class="tb2">DBT</td><td class="tb2">DIHYDROLIPOAMIDE BRANCHED-CHAIN TRANSACYLASE; DBT</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">DBT:NM_001918:exon11:c.T1286G:p.I429S</td><td class="tb2">ENST00000370132.8,ENST00000681617.1,ENST00000681780.1</td><td class="tb2">1p21.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/1:15,4:19:4:.:.:4,0,424:.</td><td class="tb2">0/0:28,0:28:37:.:.:0,37,693:.</td><td class="tb2">0/0:43,0:43:65:.:.:0,65,985:.</td><td class="tb2">0|1:10,3:13:61:0|1:100196418_A_C:61,0,338:100196418</td><td class="tb2">A</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0093</td><td class="tb2">0.0214</td><td class="tb2">0.0091</td><td class="tb2">0.0019</td><td class="tb2">0.0004</td><td class="tb2">0.0021</td><td class="tb2">0.0121</td><td class="tb2">0.0007</td><td class="tb2">0.0033</td><td class="tb2">0.0080</td><td class="tb2">7.338e-05</td><td class="tb2">0.0003</td><td class="tb2">3.388e-05</td><td class="tb2">0</td><td class="tb2">0.0001</td><td class="tb2">0.0003</td><td class="tb2">5.271e-05</td><td class="tb2">0.0002</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.002,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">3.530535,24.8</td><td class="tb2">5.96</td><td class="tb2">0.200</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Maple syrup urine disease, type II, 248600 (3), AR</td><td class="tb2">Maple syrup urine disease(CM032882)</td><td class="tb2">ATT-AAT|Ile429Asn|c.1286T>A|p.I429N(14517957)</td><td class="tb2">14517957|18063578| 14508502| 1943690| 9239422| 14742428| 2708389| 8430702| 1847055| 1990841| 3245861| 3417306| 2004553| 2837277| 1429740| 10915611| 3189339| 2932110| 16203741| 2010537| 2742576| 7672509| 9621512| 1889817</td><td class="tb2">GOBP_ORGANIC_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_CATABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_METABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_CATABOLIC_PROCESS;GOBP_ORGANIC_ACID_CATABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_CATABOLIC_PROCESS;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION;GOCC_MITOCHONDRIAL_MATRIX;GOCC_MITOCHONDRIAL_ALPHA_KETOGLUTARATE_DEHYDROGENASE_COMPLEX;GOCC_NUCLEOID;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_MITOCHONDRIAL_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_DIHYDROLIPOYL_DEHYDROGENASE_COMPLEX;GOCC_MITOCHONDRIAL_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_OXIDOREDUCTASE_COMPLEX</td><td class="tb2">GOMF_FATTY_ACID_BINDING;GOMF_LIPID_BINDING;GOMF_ACETYLTRANSFERASE_ACTIVITY;GOMF_ACYLTRANSFERASE_ACTIVITY;GOMF_MONOCARBOXYLIC_ACID_BINDING;GOMF_ORGANIC_ACID_BINDING;GOMF_UBIQUITIN_LIKE_PROTEIN_LIGASE_BINDING;GOMF_SULFUR_COMPOUND_BINDING</td><td class="tb2">KEGG_VALINE_LEUCINE_AND_ISOLEUCINE_DEGRADATION</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_GLYOXYLATE_METABOLISM_AND_GLYCINE_DEGRADATION;REACTOME_BRANCHED_CHAIN_AMINO_ACID_CATABOLISM;REACTOME_METABOLISM_OF_AMINO_ACIDS_AND_DERIVATIVES;REACTOME_RHO_GTPASE_CYCLE;REACTOME_RHOH_GTPASE_CYCLE;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3</td><td class="tb2">Granular cytoplasmic expression in most tissues at variable levels.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Mitochondria</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">100196426</td><td class="tb2">.</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">58.87</td><td class="tb2">DBT</td><td class="tb2">DIHYDROLIPOAMIDE BRANCHED-CHAIN TRANSACYLASE; DBT</td><td class="tb2">splicing</td><td class="tb2">NM_001918:exon11:c.1282-4C>G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000681780.1,ENST00000681617.1,ENST00000370132.8</td><td class="tb2">1p21.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:15,0:15:15:.:.:0,15,225:.</td><td class="tb2">0/0:25,0:25:43:.:.:0,43,577:.</td><td class="tb2">0/0:31,0:31:0:.:.:0,0,444:.</td><td class="tb2">0|1:8,3:11:67:0|1:100196418_A_C:67,0,265:100196418</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0002</td><td class="tb2">0.0003</td><td class="tb2">0.0001</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">0.0002</td><td class="tb2">0.0003</td><td class="tb2">0.0004</td><td class="tb2">0.0002</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0001</td><td class="tb2">0.038</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.001385,11.64</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Maple syrup urine disease, type II, 248600 (3), AR</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">18063578| 14508502| 1943690| 9239422| 14742428| 2708389| 8430702| 1847055| 1990841| 3245861| 3417306| 2004553| 2837277| 1429740| 10915611| 3189339| 2932110| 16203741| 2010537| 2742576| 7672509| 9621512| 1889817</td><td class="tb2">GOBP_ORGANIC_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_METABOLIC_PROCESS;GOBP_AMINO_ACID_CATABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_METABOLIC_PROCESS;GOBP_BRANCHED_CHAIN_AMINO_ACID_CATABOLIC_PROCESS;GOBP_ORGANIC_ACID_CATABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_CATABOLIC_PROCESS;GOBP_ORGANONITROGEN_COMPOUND_CATABOLIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION;GOCC_MITOCHONDRIAL_MATRIX;GOCC_MITOCHONDRIAL_ALPHA_KETOGLUTARATE_DEHYDROGENASE_COMPLEX;GOCC_NUCLEOID;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_MITOCHONDRIAL_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_TRICARBOXYLIC_ACID_CYCLE_ENZYME_COMPLEX;GOCC_DIHYDROLIPOYL_DEHYDROGENASE_COMPLEX;GOCC_MITOCHONDRIAL_PROTEIN_CONTAINING_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_OXIDOREDUCTASE_COMPLEX</td><td class="tb2">GOMF_FATTY_ACID_BINDING;GOMF_LIPID_BINDING;GOMF_ACETYLTRANSFERASE_ACTIVITY;GOMF_ACYLTRANSFERASE_ACTIVITY;GOMF_MONOCARBOXYLIC_ACID_BINDING;GOMF_ORGANIC_ACID_BINDING;GOMF_UBIQUITIN_LIKE_PROTEIN_LIGASE_BINDING;GOMF_SULFUR_COMPOUND_BINDING</td><td class="tb2">KEGG_VALINE_LEUCINE_AND_ISOLEUCINE_DEGRADATION</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_GLYOXYLATE_METABOLISM_AND_GLYCINE_DEGRADATION;REACTOME_BRANCHED_CHAIN_AMINO_ACID_CATABOLISM;REACTOME_METABOLISM_OF_AMINO_ACIDS_AND_DERIVATIVES;REACTOME_RHO_GTPASE_CYCLE;REACTOME_RHOH_GTPASE_CYCLE;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3</td><td class="tb2">Granular cytoplasmic expression in most tissues at variable levels.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Mitochondria</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr1</td><td class="tb2">228315978</td><td class="tb2">rs183406337</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">6362</td><td class="tb2">OBSCN</td><td class="tb2">OBSCURIN; OBSCN</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">OBSCN:NM_001098623:exon50:c.A13144G:p.T4382A,OBSCN:NM_052843:exon50:c.A13144G:p.T4382A,OBSCN:NM_001271223:exon61:c.A16015G:p.T5339A</td><td class="tb2">ENST00000422127.5,ENST00000636476.2,ENST00000662438.1,ENST00000680850.1,ENST00000284548.16,ENST00000660857.1,ENST00000570156.7</td><td class="tb2">1q42.13</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">0/0:41,0:41:99:.:.:0,99,1485:.</td><td class="tb2">0/0:62,0:62:99:.:.:0,120,1800:.</td><td class="tb2">0|1:95,81:176:99:0|1:228315976_A_G:2476,0,2690:228315976</td><td class="tb2">0|1:90,101:191:99:0|1:228315976_A_G:3898,0,3464:228315976</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">0.00339457</td><td class="tb2">0.0004</td><td class="tb2">0.0007</td><td class="tb2">0.0007</td><td class="tb2">0.0002</td><td class="tb2">0.0012</td><td class="tb2">0.0001</td><td class="tb2">0.0122</td><td class="tb2">0.0002</td><td class="tb2">0.0022</td><td class="tb2">0.0002</td><td class="tb2">0.0008</td><td class="tb2">0.0010</td><td class="tb2">0.0010</td><td class="tb2">0.0002</td><td class="tb2">0.0012</td><td class="tb2">0.0006</td><td class="tb2">0.0161</td><td class="tb2">0.0004</td><td class="tb2">0.0016</td><td class="tb2">0.0006</td><td class="tb2">0</td><td class="tb2">0.0021</td><td class="tb2">0.0008</td><td class="tb2">0.0002</td><td class="tb2">0.0185</td><td class="tb2">0.0026</td><td class="tb2">0.0007</td><td class="tb2">0</td><td class="tb2">0.0018</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Immunoglobulin subtype;Immunoglobulin subtype;Immunoglobulin subtype;Immunoglobulin subtype</td><td class="tb2">0.137,T</td><td class="tb2">0.99,D</td><td class="tb2">0.829,P</td><td class="tb2">2.828235,23.1</td><td class="tb2">5.44</td><td class="tb2">0.117</td><td class="tb2">746225</td><td class="tb2">not_provided</td><td class="tb2">MedGen:CN517202</td><td class="tb2">criteria_provided,_single_submitter</td><td class="tb2">Benign</td><td class="tb2">OBSCURIN; OBSCN, 608616</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">14550291| 10997877| 11814696| 11448995</td><td class="tb2">GOBP_CYTOSKELETON_ORGANIZATION;GOBP_SMALL_GTPASE_MEDIATED_SIGNAL_TRANSDUCTION;GOBP_CELLULAR_COMPONENT_ASSEMBLY_INVOLVED_IN_MORPHOGENESIS;GOBP_PHOSPHORYLATION;GOBP_ACTIN_FILAMENT_BASED_PROCESS;GOBP_ACTOMYOSIN_STRUCTURE_ORGANIZATION;GOBP_CELLULAR_COMPONENT_MORPHOGENESIS;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_MUSCLE_CELL_DIFFERENTIATION;GOBP_SARCOMERE_ORGANIZATION;GOBP_ANATOMICAL_STRUCTURE_FORMATION_INVOLVED_IN_MORPHOGENESIS;GOBP_REGULATION_OF_SMALL_GTPASE_MEDIATED_SIGNAL_TRANSDUCTION;GOBP_STRIATED_MUSCLE_CELL_DIFFERENTIATION;GOBP_MUSCLE_CELL_DEVELOPMENT;GOBP_STRIATED_MUSCLE_CELL_DEVELOPMENT;GOBP_MUSCLE_STRUCTURE_DEVELOPMENT;GOBP_ORGANELLE_ASSEMBLY;GOBP_SUPRAMOLECULAR_FIBER_ORGANIZATION;GOBP_NON_MEMBRANE_BOUNDED_ORGANELLE_ASSEMBLY;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION</td><td class="tb2">GOCC_NUCLEAR_BODY;GOCC_M_BAND;GOCC_A_BAND;GOCC_I_BAND;GOCC_SARCOLEMMA;GOCC_CONTRACTILE_FIBER;GOCC_SUPRAMOLECULAR_COMPLEX;GOCC_SUPRAMOLECULAR_POLYMER</td><td class="tb2">GOMF_PROTEIN_KINASE_ACTIVITY;GOMF_PROTEIN_SERINE_THREONINE_KINASE_ACTIVITY;GOMF_GUANYL_NUCLEOTIDE_EXCHANGE_FACTOR_ACTIVITY;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_CALMODULIN_BINDING;GOMF_PHOSPHOLIPID_BINDING;GOMF_PHOSPHATIDYLINOSITOL_4_5_BISPHOSPHATE_BINDING;GOMF_PHOSPHATIDYLINOSITOL_3_4_5_TRISPHOSPHATE_BINDING;GOMF_CYTOSKELETAL_PROTEIN_BINDING;GOMF_LIPID_BINDING;GOMF_STRUCTURAL_CONSTITUENT_OF_MUSCLE;GOMF_PHOSPHATIDYLINOSITOL_5_PHOSPHATE_BINDING;GOMF_KINASE_ACTIVITY;GOMF_TRANSFERASE_ACTIVITY_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOMF_PURINE_NUCLEOTIDE_BINDING;GOMF_ENZYME_REGULATOR_ACTIVITY;GOMF_ANKYRIN_BINDING;GOMF_ADENYL_NUCLEOTIDE_BINDING;GOMF_TITIN_BINDING;GOMF_PHOSPHATIDYLINOSITOL_3_PHOSPHATE_BINDING;GOMF_PHOSPHATIDYLINOSITOL_BINDING;GOMF_PHOSPHATIDYLINOSITOL_3_4_BISPHOSPHATE_BINDING;GOMF_NUCLEOSIDE_TRIPHOSPHATASE_REGULATOR_ACTIVITY;GOMF_PHOSPHATIDYLINOSITOL_4_PHOSPHATE_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_PROTEIN_SERINE_KINASE_ACTIVITY;GOMF_MOLECULAR_FUNCTION_ACTIVATOR_ACTIVITY;GOMF_MOLECULAR_FUNCTION_INHIBITOR_ACTIVITY;GOMF_PHOSPHATIDYLINOSITOL_PHOSPHATE_BINDING;GOMF_PHOSPHATIDYLINOSITOL_BISPHOSPHATE_BINDING</td><td class="tb2">.</td><td class="tb2">PID_RHOA_REG_PATHWAY</td><td class="tb2">.</td><td class="tb2">REACTOME_NRAGE_SIGNALS_DEATH_THROUGH_JNK;REACTOME_P75_NTR_RECEPTOR_MEDIATED_SIGNALLING;REACTOME_CELL_DEATH_SIGNALLING_VIA_NRAGE_NRIF_AND_NADE;REACTOME_SIGNALING_BY_GPCR;REACTOME_G_ALPHA_12_13_SIGNALLING_EVENTS;REACTOME_DEATH_RECEPTOR_SIGNALING;REACTOME_RHOA_GTPASE_CYCLE;REACTOME_RHO_GTPASE_CYCLE;REACTOME_RHOQ_GTPASE_CYCLE;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_GPCR_DOWNSTREAM_SIGNALING;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Cytoplasmic expression in striated muscle.</td><td class="tb2">Tissue enriched<At least four-fold higher mRNA level in a particular tissue compared to any other tissue.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Plasma membrane,Cytosol</td><td class="tb2">skeletal muscle: 481.3</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">Compound heterozygous variation means that the parents of the affected offspring have normal phenotypes, and the key genes in the two sets of genetic material from the parents are mutated, the offspring inherits two alleles, one from each parent, and these alleles are located at different positions within the same gene.
                                The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                
                                <p class="paragraph">Result: <a href="../../../result/Pro/Filter_Model" title="Click to open" target="_blank">/result/Pro/Filter_Model</a></p>
                                </div>
                                
                                
                                
                                <!-- -------------------------------------------------------------------------3 新生突变--------------------------------------------------------------------- -->
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                <a name="De novo mutation analysis"></a>
                                <br />
                                
                                <h5>4.5.3.2 De novo mutation analysis (for trios with normal parents and affected child only)</h5>
                                
                                <p class="paragraph">De novo mutation means an alteration in a gene that is present for the first time in one family member as a result of a mutation in a germ cell (egg or sperm) of one of the parents or in the fertilized egg itself. New mutations have long been known to cause genetic disease, but their true contribution to the disease burden can only now be determined using family-based whole-genome or whole-exome sequencing approaches.<br/></p><p class="paragraph">Currently, there are several public softwares developed for de novo mutation detection, including SAMtools, GATK, DeNovoGear, FamSeq, and so on. Besides, de novo mutation can be obtained by simply screening variants that exist in child while not exist in parents.
                                On a genome-wide scale, each newborn contained an average of 60 denovo SNV mutations and was strongly associated with the age of the father at conception. For every 1 year increase in the age of the father, the offspring had an average of about 2 more mutations (Kong et al., 2012). Of these, 10% are deleterious mutations, but most deleterious mutations have only minor effects. In addition, studies have shown that denovo CNVs and SVs are also closely related to the occurrence of complex diseases, such as autism, intellectual disability, schizophrenia, etc. (Muona et al., 2015; Sanders et al., 2012;  Zaidi et al., 2013).
                                For trios with affected children and normal parent, we performed de novo mutation detection with the goal of looking for mutations present in children without parents. In this project, we used two software, samtools and triodenovo, respectively to detect denovo mutations, and then intersected the results, and finally screened the mutations by using the steps in "Variant filtering using known databases" section to get the final candidate sites.</p>
                                <br/>
                                <div id="denovo-stat">
                                <a name="De novo SNPs and InDels screening result"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.3.2.1 The result of de novo SNPs and InDels screening</p>
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.14 The statistic of SNPs number
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Total</th><th class="tb2">Freq</th><th class="tb2">Func</th><th class="tb2">ExonicFunc</th><th class="tb2">Deleterious</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">14</td><td class="tb2">6</td><td class="tb2">6</td><td class="tb2">6</td><td class="tb2">3</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.15 The statistic of InDels number
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Total</th><th class="tb2">Freq</th><th class="tb2">Func</th><th class="tb2">ExonicFunc</th><th class="tb2">Deleterious</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">6</td><td class="tb2">4</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                    <p class="tremark">Notes:</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1)Total: The total number of variants before filtering.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2)Freq: The number of variants after step(1) of "Variant filtering using known databases" filtering.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3)Func: The number of variants after step(2) of "Variant filtering using known databases" filtering.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4)ExonicFunc: The number of variants after step(3) of "Variant filtering using known databases" filtering.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5)Deleterious: The number of variants after step(4) of "Variant filtering using known databases" filtering.</p>
                                    </div>
                                </div>
                                <div id="denovo-rate">
                                <a name="De novo mutation rate"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.3.2.2 De novo mutation rate</p>
                                
                                <p class="paragraph">In most de novo mutation studies, the <i>de novo</i> mutation rate (<i>de novo</i> mutation rate) of selected candidates can effectively evaluate the bias of these mutations in the diseased population. It was verified that the occurrence of the disease was closely related to the occurrence of de novo mutations (Conrad <i>et al.</i>, 2011).
                                The statistical results of SNP and InDel neonatal mutation rates are shown in Table 4.5.16 and table 4.5.17:</p><br/>
                                <p class="center">
                                <p class="name_fig">Table 4.5.16 The statistic of de novo SNP rates
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Family_patient</th><th class="tb2">de novo number</th><th class="tb2">Base</th><th class="tb2">Mutation rate</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">p1_YXHP</td><td class="tb2">14</td><td class="tb2">59965353</td><td class="tb2">1.16734074758e-07</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.17 The statistic of de novo InDel rates
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Family_patient</th><th class="tb2">de novo number</th><th class="tb2">Base</th><th class="tb2">Mutation rate</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">p1_YXHP</td><td class="tb2">6</td><td class="tb2">59965353</td><td class="tb2">5.00288891821e-08</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                    <p class="tremark">Notes:</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Family_patient: The family number and ID of the patient.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) number: The number of mutant bases.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Base: The number of bases with coverage greater than 4X.</p>
                                    <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Mutation rate: The rate of new mutation should theoretically be calculated by the number of <i>de novo</i> mutations verified by experiment. Here, we calculated by the number of <i>de novo</i> mutations detected by the <i>de novo</i> detection software.</p>
                                    </div>
                                </div>
                                <div id="denovo-result">
                                <a name="De novo mutation annotation"></a>
                                <br />
                                
                                <p style="font-size: 16px; font-weight: bold;">&nbsp;&nbsp;&nbsp;4.5.3.2.3 De novo mutation annotation</p>
                                
                                
                                
                                <p class="paragraph">The annotation results of denovo mutations by ANNOVAR are shown in the table below.
                                </p>
                                
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.18 The annotation result of de novo SNPs (only the first few lines of information are shown)
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cytoBand</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">FORMAT</th><th class="tb2">YXHP</th><th class="tb2">YXHF</th><th class="tb2">YXHM</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">OMIM</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr3</td><td class="tb2">25791780</td><td class="tb2">rs773621113</td><td class="tb2">C</td><td class="tb2">G</td><td class="tb2">220</td><td class="tb2">OXSM</td><td class="tb2">3-OXOACYL-ACP SYNTHASE, MITOCHONDRIAL; OXSM</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">OXSM:NM_017897:exon2:c.C760G:p.L254V</td><td class="tb2">ENST00000626005.1,ENST00000448177.1,ENST00000449808.1,ENST00000280701.8,ENST00000420173.2</td><td class="tb2">3p24.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:DQ:DGQ:DP:PL</td><td class="tb2">0/1:24.72:100:160:255,0,255</td><td class="tb2">0/0:24.72:100:132:0,255,255</td><td class="tb2">0/0:24.72:100:175:0,255,255</td><td class="tb2">C</td><td class="tb2">G</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">6.84e-07</td><td class="tb2">6.84e-07</td><td class="tb2">0</td><td class="tb2">1.159e-05</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">8.239e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">6.057e-05</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Beta-ketoacyl synthase, N-terminal|Polyketide synthase, beta-ketoacyl synthase domain</td><td class="tb2">0.029,D</td><td class="tb2">1.0,D</td><td class="tb2">1.0,D</td><td class="tb2">3.084494,23.6</td><td class="tb2">4.39</td><td class="tb2">0.492</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">3-OXOACYL-ACP SYNTHASE, MITOCHONDRIAL; OXSM, 610324</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">15668256</td><td class="tb2">GOBP_ORGANIC_ACID_METABOLIC_PROCESS;GOBP_LIPID_METABOLIC_PROCESS;GOBP_FATTY_ACID_METABOLIC_PROCESS;GOBP_FATTY_ACID_BIOSYNTHETIC_PROCESS;GOBP_SULFUR_COMPOUND_METABOLIC_PROCESS;GOBP_LIPID_BIOSYNTHETIC_PROCESS;GOBP_ORGANIC_ACID_BIOSYNTHETIC_PROCESS;GOBP_ORGANOPHOSPHATE_METABOLIC_PROCESS;GOBP_RIBOSE_PHOSPHATE_METABOLIC_PROCESS;GOBP_MONOCARBOXYLIC_ACID_METABOLIC_PROCESS;GOBP_NUCLEOSIDE_BISPHOSPHATE_METABOLIC_PROCESS;GOBP_THIOESTER_METABOLIC_PROCESS;GOBP_AMIDE_METABOLIC_PROCESS;GOBP_CELLULAR_LIPID_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_BIOSYNTHETIC_PROCESS;GOBP_SHORT_CHAIN_FATTY_ACID_METABOLIC_PROCESS;GOBP_MEDIUM_CHAIN_FATTY_ACID_METABOLIC_PROCESS;GOBP_MEDIUM_CHAIN_FATTY_ACID_BIOSYNTHETIC_PROCESS;GOBP_NUCLEOBASE_CONTAINING_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_MONOCARBOXYLIC_ACID_BIOSYNTHETIC_PROCESS;GOBP_PURINE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_CARBOHYDRATE_DERIVATIVE_METABOLIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION</td><td class="tb2">GOMF_FATTY_ACID_SYNTHASE_ACTIVITY;GOMF_ACYLTRANSFERASE_ACTIVITY</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Cytoplasmic expression with a granular pattern in all tissues.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Mitochondria,Cytosol</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">H</td><td class="tb2">chr3</td><td class="tb2">25791795</td><td class="tb2">.</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">220</td><td class="tb2">OXSM</td><td class="tb2">3-OXOACYL-ACP SYNTHASE, MITOCHONDRIAL; OXSM</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">OXSM:NM_017897:exon2:c.G775C:p.D259H</td><td class="tb2">ENST00000448177.1,ENST00000420173.2,ENST00000280701.8,ENST00000449808.1</td><td class="tb2">3p24.2</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">GT:DQ:DGQ:DP:PL</td><td class="tb2">0/1:24.72:100:164:255,0,255</td><td class="tb2">0/0:24.72:100:131:0,255,255</td><td class="tb2">0/0:24.72:100:177:0,255,255</td><td class="tb2">G</td><td class="tb2">C</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.2e-06</td><td class="tb2">1.2e-06</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Beta-ketoacyl synthase, N-terminal|Polyketide synthase, beta-ketoacyl synthase domain</td><td class="tb2">0.034,D</td><td class="tb2">0.049,B</td><td class="tb2">0.046,B</td><td class="tb2">3.013651,23.4</td><td class="tb2">6.16</td><td class="tb2">0.153</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">3-OXOACYL-ACP SYNTHASE, MITOCHONDRIAL; OXSM, 610324</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">15668256</td><td class="tb2">GOBP_ORGANIC_ACID_METABOLIC_PROCESS;GOBP_LIPID_METABOLIC_PROCESS;GOBP_FATTY_ACID_METABOLIC_PROCESS;GOBP_FATTY_ACID_BIOSYNTHETIC_PROCESS;GOBP_SULFUR_COMPOUND_METABOLIC_PROCESS;GOBP_LIPID_BIOSYNTHETIC_PROCESS;GOBP_ORGANIC_ACID_BIOSYNTHETIC_PROCESS;GOBP_ORGANOPHOSPHATE_METABOLIC_PROCESS;GOBP_RIBOSE_PHOSPHATE_METABOLIC_PROCESS;GOBP_MONOCARBOXYLIC_ACID_METABOLIC_PROCESS;GOBP_NUCLEOSIDE_BISPHOSPHATE_METABOLIC_PROCESS;GOBP_THIOESTER_METABOLIC_PROCESS;GOBP_AMIDE_METABOLIC_PROCESS;GOBP_CELLULAR_LIPID_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_SMALL_MOLECULE_BIOSYNTHETIC_PROCESS;GOBP_SHORT_CHAIN_FATTY_ACID_METABOLIC_PROCESS;GOBP_MEDIUM_CHAIN_FATTY_ACID_METABOLIC_PROCESS;GOBP_MEDIUM_CHAIN_FATTY_ACID_BIOSYNTHETIC_PROCESS;GOBP_NUCLEOBASE_CONTAINING_SMALL_MOLECULE_METABOLIC_PROCESS;GOBP_MONOCARBOXYLIC_ACID_BIOSYNTHETIC_PROCESS;GOBP_PURINE_CONTAINING_COMPOUND_METABOLIC_PROCESS;GOBP_CARBOHYDRATE_DERIVATIVE_METABOLIC_PROCESS</td><td class="tb2">GOCC_MITOCHONDRION</td><td class="tb2">GOMF_FATTY_ACID_SYNTHASE_ACTIVITY;GOMF_ACYLTRANSFERASE_ACTIVITY</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">Cytoplasmic expression with a granular pattern in all tissues.</td><td class="tb2">Low tissue specificity</td><td class="tb2">Detected in all tissues.</td><td class="tb2">Mitochondria,Cytosol</td><td class="tb2">.</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr3</td><td class="tb2">195782032</td><td class="tb2">rs778176783</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">69</td><td class="tb2">MUC4</td><td class="tb2">MUCIN 4, TRACHEOBRONCHIAL; MUC4</td><td class="tb2">exonic</td><td class="tb2">.</td><td class="tb2">missense SNV</td><td class="tb2">MUC4:NM_018406:exon2:c.C9548T:p.T3183M</td><td class="tb2">ENST00000349607.8,ENST00000475231.5,ENST00000479406.5,ENST00000477756.5,ENST00000480843.5,ENST00000477086.5,ENST00000466475.5,ENST00000346145.8,ENST00000478156.5,ENST00000462323.5,ENST00000463781.8,ENST00000470451.5</td><td class="tb2">3q29</td><td class="tb2">Score=0.913924;Name=chr3:195783434</td><td class="tb2">.</td><td class="tb2">GT:DQ:DGQ:DP:PL</td><td class="tb2">0/1:10.22:100:55:104,0,255</td><td class="tb2">0/0:10.22:100:68:0,115,255</td><td class="tb2">0/0:10.22:100:69:0,134,255</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0005</td><td class="tb2">0.0026</td><td class="tb2">0.0007</td><td class="tb2">0.0002</td><td class="tb2">9.427e-05</td><td class="tb2">0.0022</td><td class="tb2">0.0005</td><td class="tb2">0.0011</td><td class="tb2">0.0002</td><td class="tb2">0.0005</td><td class="tb2">0.0066</td><td class="tb2">0.0134</td><td class="tb2">0.0035</td><td class="tb2">0.0112</td><td class="tb2">0.0073</td><td class="tb2">0.0216</td><td class="tb2">0.0066</td><td class="tb2">0.0084</td><td class="tb2">0.0134</td><td class="tb2">0.0132</td><td class="tb2">0.0003</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0</td><td class="tb2">0.0002</td><td class="tb2">0</td><td class="tb2">0.0005</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.02,D</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1.771839,17.53</td><td class="tb2">.</td><td class="tb2">0.043</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">MUCIN 4, TRACHEOBRONCHIAL; MUC4, 158372</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">1351710| 17058067| 1673336</td><td class="tb2">GOBP_TISSUE_HOMEOSTASIS;GOBP_CELL_ADHESION;GOBP_CELL_MATRIX_ADHESION;GOBP_DIGESTION;GOBP_REGULATION_OF_SIGNALING_RECEPTOR_ACTIVITY;GOBP_EPITHELIAL_STRUCTURE_MAINTENANCE;GOBP_DIGESTIVE_SYSTEM_PROCESS;GOBP_MAINTENANCE_OF_GASTROINTESTINAL_EPITHELIUM;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_HOMEOSTATIC_PROCESS;GOBP_MULTICELLULAR_ORGANISMAL_LEVEL_HOMEOSTASIS</td><td class="tb2">GOCC_GOLGI_APPARATUS;GOCC_GOLGI_LUMEN;GOCC_EXTERNAL_ENCAPSULATING_STRUCTURE</td><td class="tb2">GOMF_SIGNALING_RECEPTOR_BINDING;GOMF_STRUCTURAL_MOLECULE_ACTIVITY;GOMF_EXTRACELLULAR_MATRIX_STRUCTURAL_CONSTITUENT</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">REACTOME_INNATE_IMMUNE_SYSTEM;REACTOME_DISEASES_OF_GLYCOSYLATION;REACTOME_DISEASES_ASSOCIATED_WITH_O_GLYCOSYLATION_OF_PROTEINS;REACTOME_DEFECTIVE_GALNT3_CAUSES_HFTC;REACTOME_DEFECTIVE_C1GALT1C1_CAUSES_TNPS;REACTOME_O_LINKED_GLYCOSYLATION;REACTOME_DECTIN_2_FAMILY;REACTOME_C_TYPE_LECTIN_RECEPTORS_CLRS;REACTOME_DISEASES_OF_METABOLISM;REACTOME_POST_TRANSLATIONAL_PROTEIN_MODIFICATION;REACTOME_O_LINKED_GLYCOSYLATION_OF_MUCINS;REACTOME_TERMINATION_OF_O_GLYCAN_BIOSYNTHESIS</td><td class="tb2">Cytoplasmic and membranous expression in respiratory epithelia, glandular cells in the gastrointestinal tract, cervix and prostate.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in more than one but less than one third of tissues.</td><td class="tb2">.</td><td class="tb2">esophagus: 19.2;intestine: 31.1;prostate: 14.8;vagina: 15.0</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                
                                
                                </div>
                                
                                <p class="paragraph">Result: <a href="../../../result/Pro/Denovo" title="Click to open" target="_blank">/result/Pro/Denovo</a></p>
                                </div>
                                
                                 
                                <!-- ------------------------------------------------------------------------ 连锁分析--------------------------------------------------------------------- -->
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                <a name="Linkage analysis"></a>
                                <br />
                                
                                <h5>4.5.3.3 Linkage analysis (for family samples of three or more generations only)</h5>
                                
                                <p class="paragraph">Linkage analysis is to observe whether marker sites and disease-causing gene sites are co-separated in the family based on three or more generations of family materials, and calculate the genetic distance and linkage degree. Several methods have been proposed to do linkage analysis: the U scores, the sib pair test, the likelihood ratios, the lod score method. The lod score method is the one most commonly used method at present.
                                LOD score, or logarithm of odds score, is a statistical test method used in genetic linkage analysis. The LOD score represents the ratio of the probability that two sites are linked to the probability that they are not linked. For Mendelian diseases, the decision thresholds of the test are usually set at -2 and +3, i.e. LOD score greater than 3 suggests linkage, and LOD score less than -2 rejects linkage; for regions with LOD scores between -2 and 3, it is necessary to do futher analysis with more family materials and identify the site in this candidate region. For complex diseases, the threshold corresponding to LOD is low.
                                Linkage candidate regions was obtained by using Merlin and known SNPs as linkage analysis markers, combining high-throughput sequencing data in the pedigree and allele frequencies of the Chinese population (CHB) in the HapMap database. <br /></p>
                                </p>
                                
                                <p class="paragraph">No linkage region was detected in this family. </p>
                                <!-- ------------------------------------------------------------------------连锁分析图 ------------------------------------------------------------- -->
                                
                                </div>
                                <p class="paragraph">Result: <a href="../../../result/Pro/Linkage" title="Click to open" target="_blank">/result/Pro/Linkage</a></p>
                                
                                
                                 <!------------------------------------------------------------------ ROH分析-------------------------------------------------------------------- -->
                                 <div id="page">
                                     <p class="head">
                                         <a href="#home" title="返回首页"></a>
                                         <hr />
                                     </p>
                                <a name="ROH analysis"></a>
                                <br />
                                
                                <h5>4.5.3.4 Regions of homozygosity (ROH) analysis</h5>
                                
                                
                                <p class="paragraph">Regions of homozygosity are chromosomal stretches that in a diploid genome appear in the homozygous state, that is, display identical alleles at multiple contiguous loci.</p>
                                <p class="paragraph">Recent parental relatedness favors the formation of long ROH that occur because of IBD (identity by descent, when the two alleles at a locus match because they originate from the same common ancestor). Homozygosity originating from the occurrence of individual IBD regions owing to parental relatedness is known to possibly contain recessive highly penetrant deleterious disease-causing mutations surrounded by an unusually long homozygous haplotype. This is the principle that inspired homozygosity mapping in the study of rare recessive disorders affecting inbred individuals.ROH analysis is generally used for genetic diseases caused by consanguineous marriage.</p>
                                <p class="paragraph">PLINK was used to detect ROH and the screening and annotation results are shown in the following table:</p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.19 The screening and annotation result of ROH (only the first few lines of information are shown)
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Priority</th><th class="tb2">CHROM</th><th class="tb2">POS</th><th class="tb2">ID</th><th class="tb2">REF</th><th class="tb2">ALT</th><th class="tb2">QUAL</th><th class="tb2">FILTER</th><th class="tb2">GeneName</th><th class="tb2">Description</th><th class="tb2">Func</th><th class="tb2">Gene</th><th class="tb2">GeneDetail</th><th class="tb2">ExonicFunc</th><th class="tb2">AAChange</th><th class="tb2">Gencode</th><th class="tb2">cpgIslandExt</th><th class="tb2">cytoBand</th><th class="tb2">wgRna</th><th class="tb2">genomicSuperDups</th><th class="tb2">Repeat</th><th class="tb2">avsnp</th><th class="tb2">CLNALLELEID</th><th class="tb2">CLNDN</th><th class="tb2">CLNDISDB</th><th class="tb2">CLNREVSTAT</th><th class="tb2">CLNSIG</th><th class="tb2">cosmic</th><th class="tb2">gwasCatalog</th><th class="tb2">1000g_SAS</th><th class="tb2">1000g_EUR</th><th class="tb2">1000g_AFR</th><th class="tb2">1000g_AMR</th><th class="tb2">1000g_EAS</th><th class="tb2">1000g_ALL</th><th class="tb2">esp6500siv2_all</th><th class="tb2">gnomad_exome_AF</th><th class="tb2">gnomad_exome_AF_raw</th><th class="tb2">gnomad_exome_AF_afr</th><th class="tb2">gnomad_exome_AF_sas</th><th class="tb2">gnomad_exome_AF_amr</th><th class="tb2">gnomad_exome_AF_eas</th><th class="tb2">gnomad_exome_AF_nfe</th><th class="tb2">gnomad_exome_AF_fin</th><th class="tb2">gnomad_exome_AF_asj</th><th class="tb2">gnomad_exome_AF_oth</th><th class="tb2">gnomad_genome_AF</th><th class="tb2">gnomad_genome_AF_raw</th><th class="tb2">gnomad_genome_AF_afr</th><th class="tb2">gnomad_genome_AF_sas</th><th class="tb2">gnomad_genome_AF_amr</th><th class="tb2">gnomad_genome_AF_eas</th><th class="tb2">gnomad_genome_AF_nfe</th><th class="tb2">gnomad_genome_AF_fin</th><th class="tb2">gnomad_genome_AF_asj</th><th class="tb2">gnomad_genome_AF_oth</th><th class="tb2">ExAC_ALL</th><th class="tb2">ExAC_AFR</th><th class="tb2">ExAC_AMR</th><th class="tb2">ExAC_EAS</th><th class="tb2">ExAC_FIN</th><th class="tb2">ExAC_NFE</th><th class="tb2">ExAC_OTH</th><th class="tb2">ExAC_SAS</th><th class="tb2">dbscSNV_ADA_SCORE</th><th class="tb2">dbscSNV_RF_SCORE</th><th class="tb2">Interpro_domain</th><th class="tb2">SIFT</th><th class="tb2">Polyphen2_HDIV</th><th class="tb2">Polyphen2_HVAR</th><th class="tb2">LRT</th><th class="tb2">MutationTaster</th><th class="tb2">MutationAssessor</th><th class="tb2">FATHMM</th><th class="tb2">phyloP100way_vertebrate</th><th class="tb2">phyloP30way_mammalian</th><th class="tb2">SiPhy_29way_logOdds</th><th class="tb2">CADD</th><th class="tb2">gerp++gt2</th><th class="tb2">REVEL</th><th class="tb2">INFO</th><th class="tb2">FORMAT</th><th class="tb2">YXHP</th><th class="tb2">Ori_REF</th><th class="tb2">Ori_ALT</th><th class="tb2">shared_hom</th><th class="tb2">shared_het</th><th class="tb2">OMIM</th><th class="tb2">GWAS_Pubmed_pValue</th><th class="tb2">HGMD_Disease_ID</th><th class="tb2">HGMD_Mutation_URL</th><th class="tb2">PubMedID</th><th class="tb2">GO_BP</th><th class="tb2">GO_CC</th><th class="tb2">GO_MF</th><th class="tb2">KEGG_PATHWAY</th><th class="tb2">PID_PATHWAY</th><th class="tb2">BIOCARTA_PATHWAY</th><th class="tb2">REACTOME_PATHWAY</th><th class="tb2">Expression_summary</th><th class="tb2">RNA tissue specificity</th><th class="tb2">RNA tissue distribution</th><th class="tb2">Subcellular_Location</th><th class="tb2">RNA tissue specific nTPM</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">Region</td><td class="tb2">chr1</td><td class="tb2">2055295</td><td class="tb2">2074301</td><td class="tb2">PRKCZ</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td><td class="tb2">nan</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">2055295</td><td class="tb2">rs59687002</td><td class="tb2">G</td><td class="tb2">T</td><td class="tb2">425.06</td><td class="tb2">PASS</td><td class="tb2">PRKCZ</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ</td><td class="tb2">intronic</td><td class="tb2">NM_002744</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000468310.5,ENST00000378567.8,ENST00000503297.1,ENST00000484419.1</td><td class="tb2">.</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">rs59687002</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0194</td><td class="tb2">0.001</td><td class="tb2">0.0038</td><td class="tb2">.</td><td class="tb2">0.2133</td><td class="tb2">0.0479233</td><td class="tb2">.</td><td class="tb2">0.0103</td><td class="tb2">0.0076</td><td class="tb2">0.0019</td><td class="tb2">0.0157</td><td class="tb2">0.0011</td><td class="tb2">0.2272</td><td class="tb2">0.0004</td><td class="tb2">0.0018</td><td class="tb2">0.0008</td><td class="tb2">0.0124</td><td class="tb2">0.0086</td><td class="tb2">0.0092</td><td class="tb2">0.0023</td><td class="tb2">0.0191</td><td class="tb2">0.0022</td><td class="tb2">0.1989</td><td class="tb2">0.0005</td><td class="tb2">0.0013</td><td class="tb2">0</td><td class="tb2">0.0092</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">AC=2;AF=1;AN=2;DP=10;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=60;QD=27.23;SOR=4.804</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">1|1:0,10:10:30:1|1:2055295_G_T:439,30,0:2055295</td><td class="tb2">G</td><td class="tb2">T</td><td class="tb2">1</td><td class="tb2">0</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ, 176982</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">11836310| 8224878| 22174128| 19520910| 12610628| 10477520| 12857744| 23283171| 11684013| 21127255| 11914719| 12145205| 17254974| 16931766| 21791414| 8378304| 15958741| 21385716| 17702943| 11462174| 23283174| 17558396</td><td class="tb2">GOBP_MAPK_CASCADE;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_CELL_MORPHOGENESIS;GOBP_CELL_ACTIVATION;GOBP_CYTOKINE_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_ADAPTIVE_IMMUNE_RESPONSE;GOBP_IMMUNE_EFFECTOR_PROCESS;GOBP_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_LYMPHOCYTE_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_DIFFERENTIATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_CYTOKINE_PRODUCTION;GOBP_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_T_CELL_MEDIATED_IMMUNITY;GOBP_ADAPTIVE_IMMUNE_RESPONSE_BASED_ON_SOMATIC_RECOMBINATION_OF_IMMUNE_RECEPTORS_BUILT_FROM_IMMUNOGLOBULIN_SUPERFAMILY_DOMAINS;GOBP_LEUKOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_T_CELL_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_APOPTOTIC_PROCESS;GOBP_DEFENSE_RESPONSE;GOBP_INFLAMMATORY_RESPONSE;GOBP_IMMUNE_RESPONSE;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_MICROTUBULE_BASED_MOVEMENT;GOBP_CELL_ADHESION;GOBP_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_CELL_MATRIX_ADHESION;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_CELL_POLARITY;GOBP_ENZYME_LINKED_RECEPTOR_PROTEIN_SIGNALING_PATHWAY;GOBP_TRANSMEMBRANE_RECEPTOR_PROTEIN_TYROSINE_KINASE_SIGNALING_PATHWAY;GOBP_CELL_CELL_SIGNALING;GOBP_BEHAVIOR;GOBP_MEMORY;GOBP_LONG_TERM_MEMORY;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_RESPONSE_TO_HORMONE;GOBP_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_POSITIVE_REGULATION_OF_CELL_COMMUNICATION;GOBP_POSITIVE_REGULATION_OF_CELL_DEVELOPMENT;GOBP_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_TRANSPORT_ALONG_MICROTUBULE;GOBP_CELL_GROWTH;GOBP_PHOSPHORYLATION;GOBP_PEPTIDYL_AMINO_ACID_MODIFICATION;GOBP_PEPTIDYL_SERINE_MODIFICATION;GOBP_PEPTIDYL_TYROSINE_MODIFICATION;GOBP_NEUROGENESIS;GOBP_REGULATION_OF_CELL_CELL_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_CELL_ADHESION;GOBP_NEGATIVE_REGULATION_OF_SIGNALING;GOBP_ESTABLISHMENT_OF_CELL_POLARITY;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_HEMOPOIESIS;GOBP_REGULATION_OF_CELL_ADHESION;GOBP_T_CELL_DIFFERENTIATION;GOBP_CYTOSKELETON_DEPENDENT_INTRACELLULAR_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_PROTEIN_CONTAINING_COMPLEX_LOCALIZATION;GOBP_ACTIVATION_OF_PHOSPHOLIPASE_D_ACTIVITY;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_NERVOUS_SYSTEM_PROCESS;GOBP_ACTIVATION_OF_PROTEIN_KINASE_ACTIVITY;GOBP_ACTIVATION_OF_PROTEIN_KINASE_B_ACTIVITY;GOBP_INTERLEUKIN_10_PRODUCTION;GOBP_INTERLEUKIN_13_PRODUCTION;GOBP_INTERLEUKIN_4_PRODUCTION;GOBP_INTERLEUKIN_5_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_10_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_13_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_4_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_5_PRODUCTION;GOBP_RESPONSE_TO_INSULIN;GOBP_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_CELLULAR_RESPONSE_TO_HORMONE_STIMULUS;GOBP_CELLULAR_COMPONENT_MORPHOGENESIS;GOBP_CELL_PART_MORPHOGENESIS;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_POSITIVE_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_CYTOKINE_PRODUCTION;GOBP_T_HELPER_2_CELL_CYTOKINE_PRODUCTION;GOBP_GROWTH;GOBP_TYPE_2_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_POSITIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_POSITIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_MAPK_CASCADE;GOBP_RESPONSE_TO_PEPTIDE_HORMONE;GOBP_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_CONTAINING_COMPLEX_ORGANIZATION;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_REGULATION_OF_CELLULAR_COMPONENT_BIOGENESIS;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_POSITIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_ADHESION;GOBP_REGULATION_OF_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_LYMPHOCYTE_ACTIVATION;GOBP_INTRACELLULAR_TRANSPORT;GOBP_VESICLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_REGULATION_OF_SYNAPTIC_PLASTICITY;GOBP_NEGATIVE_REGULATION_OF_RESPONSE_TO_STIMULUS;GOBP_DEVELOPMENTAL_CELL_GROWTH;GOBP_DEVELOPMENTAL_GROWTH;GOBP_NEURON_DEVELOPMENT;GOBP_GENERATION_OF_NEURONS;GOBP_CELL_MOTILITY;GOBP_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_REGULATION_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_SYNAPTIC_TRANSMISSION;GOBP_REGULATION_OF_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_ACTIVATION;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_COGNITION;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_NF_KAPPAB_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_MULTICELLULAR_ORGANISMAL_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_ORGANELLE_LOCALIZATION;GOBP_VESICLE_LOCALIZATION;GOBP_ESTABLISHMENT_OF_ORGANELLE_LOCALIZATION;GOBP_LOCALIZATION_WITHIN_MEMBRANE;GOBP_MEMBRANE_DEPOLARIZATION;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_POTENTIAL;GOBP_MEMBRANE_HYPERPOLARIZATION;GOBP_REGULATION_OF_LIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_LIPASE_ACTIVITY;GOBP_REGULATION_OF_CELL_DEVELOPMENT;GOBP_LONG_TERM_SYNAPTIC_POTENTIATION;GOBP_REGULATION_OF_CELLULAR_LOCALIZATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_DEVELOPMENTAL_GROWTH_INVOLVED_IN_MORPHOGENESIS;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_BIPOLAR_CELL_POLARITY;GOBP_PROTEIN_LOCALIZATION_TO_POSTSYNAPSE;GOBP_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_ERK1_AND_ERK2_CASCADE;GOBP_POSITIVE_REGULATION_OF_ERK1_AND_ERK2_CASCADE;GOBP_PROTEIN_KINASE_C_SIGNALING;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE_HORMONE_STIMULUS;GOBP_ORGANELLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_PROTEIN_LOCALIZATION_TO_PLASMA_MEMBRANE;GOBP_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_CELL_CELL_ADHESION;GOBP_REGULATION_OF_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_MODULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_NEUROTRANSMITTER_RECEPTOR_LEVELS;GOBP_MICROTUBULE_BASED_TRANSPORT;GOBP_REGULATION_OF_TRANS_SYNAPTIC_SIGNALING;GOBP_VESICLE_CYTOSKELETAL_TRAFFICKING;GOBP_SYNAPTIC_SIGNALING;GOBP_CHEMICAL_SYNAPTIC_TRANSMISSION_POSTSYNAPTIC;GOBP_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_POSITIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_RESPONSE_TO_PEPTIDE;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE;GOBP_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_REGULATION_OF_LEUKOCYTE_DIFFERENTIATION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_JUNCTION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_REGULATION_OF_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_MONONUCLEAR_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_POSITIVE_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_CELL_SURFACE_RECEPTOR_SIGNALING_PATHWAY_INVOLVED_IN_CELL_CELL_SIGNALING;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_MEMBRANE;GOBP_NEURON_PROJECTION_EXTENSION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_REGULATION_OF_MULTICELLULAR_ORGANISMAL_DEVELOPMENT;GOBP_POSITIVE_REGULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_CYTOKINE_PRODUCTION</td><td class="tb2">GOCC_NUCLEAR_ENVELOPE;GOCC_ENDOSOME;GOCC_MICROTUBULE_ORGANIZING_CENTER;GOCC_CELL_CELL_JUNCTION;GOCC_CELL_CORTEX;GOCC_ACTIN_CYTOSKELETON;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_AXON;GOCC_CELL_LEADING_EDGE;GOCC_ENVELOPE;GOCC_ACTIN_FILAMENT_BUNDLE;GOCC_NUCLEAR_PERIPHERY;GOCC_MYELIN_SHEATH_ABAXONAL_REGION;GOCC_SOMATODENDRITIC_COMPARTMENT;GOCC_ACTOMYOSIN;GOCC_NEURON_PROJECTION;GOCC_AXON_HILLOCK;GOCC_MYELIN_SHEATH;GOCC_APICAL_JUNCTION_COMPLEX;GOCC_CELL_BODY;GOCC_APICAL_PART_OF_CELL;GOCC_APICAL_CORTEX;GOCC_SYNAPSE;GOCC_PERINUCLEAR_REGION_OF_CYTOPLASM;GOCC_TRANSFERASE_COMPLEX_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOCC_TIGHT_JUNCTION;GOCC_ANCHORING_JUNCTION;GOCC_PLASMA_MEMBRANE_REGION;GOCC_SCHAFFER_COLLATERAL_CA1_SYNAPSE;GOCC_POSTSYNAPSE;GOCC_MEMBRANE_MICRODOMAIN;GOCC_GLUTAMATERGIC_SYNAPSE;GOCC_NEURON_TO_NEURON_SYNAPSE;GOCC_CYTOPLASMIC_REGION;GOCC_POSTSYNAPTIC_SPECIALIZATION;GOCC_CELL_CORTEX_REGION;GOCC_PAR_POLARITY_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_PROTEIN_KINASE_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_PROTEIN_KINASE_ACTIVITY;GOMF_PROTEIN_SERINE_THREONINE_KINASE_ACTIVITY;GOMF_PROTEIN_KINASE_C_ACTIVITY;GOMF_CALCIUM_DEPENDENT_PROTEIN_KINASE_ACTIVITY;GOMF_POTASSIUM_CHANNEL_REGULATOR_ACTIVITY;GOMF_CHANNEL_REGULATOR_ACTIVITY;GOMF_KINASE_ACTIVITY;GOMF_TRANSFERASE_ACTIVITY_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOMF_PURINE_NUCLEOTIDE_BINDING;GOMF_KINASE_BINDING;GOMF_ADENYL_NUCLEOTIDE_BINDING;GOMF_PHOSPHOLIPASE_BINDING;GOMF_INSULIN_RECEPTOR_SUBSTRATE_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_14_3_3_PROTEIN_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_PROTEIN_SERINE_KINASE_ACTIVITY</td><td class="tb2">KEGG_CHEMOKINE_SIGNALING_PATHWAY;KEGG_ENDOCYTOSIS;KEGG_TIGHT_JUNCTION;KEGG_INSULIN_SIGNALING_PATHWAY;KEGG_TYPE_II_DIABETES_MELLITUS</td><td class="tb2">PID_RHOA_PATHWAY;PID_INSULIN_PATHWAY;PID_WNT_NONCANONICAL_PATHWAY;PID_MET_PATHWAY;PID_CDC42_PATHWAY;PID_TXA2PATHWAY;PID_IL1_PATHWAY;PID_NFAT_3PATHWAY;PID_CXCR4_PATHWAY;PID_IGF1_PATHWAY;PID_TNF_PATHWAY;PID_IL2_PI3K_PATHWAY;PID_CERAMIDE_PATHWAY;PID_P75_NTR_PATHWAY;PID_AMB2_NEUTROPHILS_PATHWAY;PID_ERBB1_DOWNSTREAM_PATHWAY;PID_TRKR_PATHWAY;PID_NEPHRIN_NEPH1_PATHWAY;PID_INSULIN_GLUCOSE_PATHWAY;PID_RAS_PATHWAY</td><td class="tb2">BIOCARTA_PTDINS_PATHWAY</td><td class="tb2">REACTOME_HEMOSTASIS;REACTOME_GPVI_MEDIATED_ACTIVATION_CASCADE;REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX;REACTOME_SIGNALING_BY_VEGF;REACTOME_RHO_GTPASE_EFFECTORS;REACTOME_TGF_BETA_RECEPTOR_SIGNALING_IN_EMT_EPITHELIAL_TO_MESENCHYMAL_TRANSITION;REACTOME_VEGFR2_MEDIATED_CELL_PROLIFERATION;REACTOME_RHO_GTPASES_ACTIVATE_NADPH_OXIDASES;REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION;REACTOME_ESR_MEDIATED_SIGNALING;REACTOME_SIGNALING_BY_NUCLEAR_RECEPTORS;REACTOME_SIGNALING_BY_RECEPTOR_TYROSINE_KINASES;REACTOME_SIGNALING_BY_TGFB_FAMILY_MEMBERS;REACTOME_EXTRA_NUCLEAR_ESTROGEN_SIGNALING;REACTOME_ESTROGEN_STIMULATED_SIGNALING_THROUGH_PRKCZ;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Membranous and cytoplasmic expression in most tissues, including brain, kidney and retina.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Plasma membrane,Cytosol</td><td class="tb2">brain: 58.0</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">2055298</td><td class="tb2">rs58725160</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">425.06</td><td class="tb2">PASS</td><td class="tb2">PRKCZ</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ</td><td class="tb2">intronic</td><td class="tb2">NM_002744</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000503297.1,ENST00000378567.8,ENST00000468310.5,ENST00000484419.1</td><td class="tb2">.</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">rs58725160</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0245</td><td class="tb2">0.0149</td><td class="tb2">0.3245</td><td class="tb2">0.0216</td><td class="tb2">0.2242</td><td class="tb2">0.141573</td><td class="tb2">.</td><td class="tb2">0.0184</td><td class="tb2">0.0177</td><td class="tb2">0.3009</td><td class="tb2">0.0244</td><td class="tb2">0.0161</td><td class="tb2">0.2816</td><td class="tb2">0.0014</td><td class="tb2">0.0023</td><td class="tb2">0.0024</td><td class="tb2">0.0290</td><td class="tb2">0.2274</td><td class="tb2">0.2191</td><td class="tb2">0.4604</td><td class="tb2">0.1592</td><td class="tb2">0.0943</td><td class="tb2">0.4309</td><td class="tb2">0.0959</td><td class="tb2">0.0214</td><td class="tb2">0.0987</td><td class="tb2">0.1784</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">AC=2;AF=1;AN=2;DP=10;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=60;QD=31.92;SOR=4.804</td><td class="tb2">GT:AD:DP:GQ:PGT:PID:PL:PS</td><td class="tb2">1|1:0,10:10:30:1|1:2055295_G_T:439,30,0:2055295</td><td class="tb2">A</td><td class="tb2">G</td><td class="tb2">1</td><td class="tb2">0</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ, 176982</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">11836310| 8224878| 22174128| 19520910| 12610628| 10477520| 12857744| 23283171| 11684013| 21127255| 11914719| 12145205| 17254974| 16931766| 21791414| 8378304| 15958741| 21385716| 17702943| 11462174| 23283174| 17558396</td><td class="tb2">GOBP_MAPK_CASCADE;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_CELL_MORPHOGENESIS;GOBP_CELL_ACTIVATION;GOBP_CYTOKINE_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_ADAPTIVE_IMMUNE_RESPONSE;GOBP_IMMUNE_EFFECTOR_PROCESS;GOBP_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_LYMPHOCYTE_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_DIFFERENTIATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_CYTOKINE_PRODUCTION;GOBP_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_T_CELL_MEDIATED_IMMUNITY;GOBP_ADAPTIVE_IMMUNE_RESPONSE_BASED_ON_SOMATIC_RECOMBINATION_OF_IMMUNE_RECEPTORS_BUILT_FROM_IMMUNOGLOBULIN_SUPERFAMILY_DOMAINS;GOBP_LEUKOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_T_CELL_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_APOPTOTIC_PROCESS;GOBP_DEFENSE_RESPONSE;GOBP_INFLAMMATORY_RESPONSE;GOBP_IMMUNE_RESPONSE;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_MICROTUBULE_BASED_MOVEMENT;GOBP_CELL_ADHESION;GOBP_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_CELL_MATRIX_ADHESION;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_CELL_POLARITY;GOBP_ENZYME_LINKED_RECEPTOR_PROTEIN_SIGNALING_PATHWAY;GOBP_TRANSMEMBRANE_RECEPTOR_PROTEIN_TYROSINE_KINASE_SIGNALING_PATHWAY;GOBP_CELL_CELL_SIGNALING;GOBP_BEHAVIOR;GOBP_MEMORY;GOBP_LONG_TERM_MEMORY;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_RESPONSE_TO_HORMONE;GOBP_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_POSITIVE_REGULATION_OF_CELL_COMMUNICATION;GOBP_POSITIVE_REGULATION_OF_CELL_DEVELOPMENT;GOBP_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_TRANSPORT_ALONG_MICROTUBULE;GOBP_CELL_GROWTH;GOBP_PHOSPHORYLATION;GOBP_PEPTIDYL_AMINO_ACID_MODIFICATION;GOBP_PEPTIDYL_SERINE_MODIFICATION;GOBP_PEPTIDYL_TYROSINE_MODIFICATION;GOBP_NEUROGENESIS;GOBP_REGULATION_OF_CELL_CELL_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_CELL_ADHESION;GOBP_NEGATIVE_REGULATION_OF_SIGNALING;GOBP_ESTABLISHMENT_OF_CELL_POLARITY;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_HEMOPOIESIS;GOBP_REGULATION_OF_CELL_ADHESION;GOBP_T_CELL_DIFFERENTIATION;GOBP_CYTOSKELETON_DEPENDENT_INTRACELLULAR_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_PROTEIN_CONTAINING_COMPLEX_LOCALIZATION;GOBP_ACTIVATION_OF_PHOSPHOLIPASE_D_ACTIVITY;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_NERVOUS_SYSTEM_PROCESS;GOBP_ACTIVATION_OF_PROTEIN_KINASE_ACTIVITY;GOBP_ACTIVATION_OF_PROTEIN_KINASE_B_ACTIVITY;GOBP_INTERLEUKIN_10_PRODUCTION;GOBP_INTERLEUKIN_13_PRODUCTION;GOBP_INTERLEUKIN_4_PRODUCTION;GOBP_INTERLEUKIN_5_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_10_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_13_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_4_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_5_PRODUCTION;GOBP_RESPONSE_TO_INSULIN;GOBP_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_CELLULAR_RESPONSE_TO_HORMONE_STIMULUS;GOBP_CELLULAR_COMPONENT_MORPHOGENESIS;GOBP_CELL_PART_MORPHOGENESIS;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_POSITIVE_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_CYTOKINE_PRODUCTION;GOBP_T_HELPER_2_CELL_CYTOKINE_PRODUCTION;GOBP_GROWTH;GOBP_TYPE_2_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_POSITIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_POSITIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_MAPK_CASCADE;GOBP_RESPONSE_TO_PEPTIDE_HORMONE;GOBP_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_CONTAINING_COMPLEX_ORGANIZATION;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_REGULATION_OF_CELLULAR_COMPONENT_BIOGENESIS;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_POSITIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_ADHESION;GOBP_REGULATION_OF_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_LYMPHOCYTE_ACTIVATION;GOBP_INTRACELLULAR_TRANSPORT;GOBP_VESICLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_REGULATION_OF_SYNAPTIC_PLASTICITY;GOBP_NEGATIVE_REGULATION_OF_RESPONSE_TO_STIMULUS;GOBP_DEVELOPMENTAL_CELL_GROWTH;GOBP_DEVELOPMENTAL_GROWTH;GOBP_NEURON_DEVELOPMENT;GOBP_GENERATION_OF_NEURONS;GOBP_CELL_MOTILITY;GOBP_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_REGULATION_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_SYNAPTIC_TRANSMISSION;GOBP_REGULATION_OF_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_ACTIVATION;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_COGNITION;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_NF_KAPPAB_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_MULTICELLULAR_ORGANISMAL_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_ORGANELLE_LOCALIZATION;GOBP_VESICLE_LOCALIZATION;GOBP_ESTABLISHMENT_OF_ORGANELLE_LOCALIZATION;GOBP_LOCALIZATION_WITHIN_MEMBRANE;GOBP_MEMBRANE_DEPOLARIZATION;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_POTENTIAL;GOBP_MEMBRANE_HYPERPOLARIZATION;GOBP_REGULATION_OF_LIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_LIPASE_ACTIVITY;GOBP_REGULATION_OF_CELL_DEVELOPMENT;GOBP_LONG_TERM_SYNAPTIC_POTENTIATION;GOBP_REGULATION_OF_CELLULAR_LOCALIZATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_DEVELOPMENTAL_GROWTH_INVOLVED_IN_MORPHOGENESIS;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_BIPOLAR_CELL_POLARITY;GOBP_PROTEIN_LOCALIZATION_TO_POSTSYNAPSE;GOBP_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_ERK1_AND_ERK2_CASCADE;GOBP_POSITIVE_REGULATION_OF_ERK1_AND_ERK2_CASCADE;GOBP_PROTEIN_KINASE_C_SIGNALING;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE_HORMONE_STIMULUS;GOBP_ORGANELLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_PROTEIN_LOCALIZATION_TO_PLASMA_MEMBRANE;GOBP_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_CELL_CELL_ADHESION;GOBP_REGULATION_OF_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_MODULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_NEUROTRANSMITTER_RECEPTOR_LEVELS;GOBP_MICROTUBULE_BASED_TRANSPORT;GOBP_REGULATION_OF_TRANS_SYNAPTIC_SIGNALING;GOBP_VESICLE_CYTOSKELETAL_TRAFFICKING;GOBP_SYNAPTIC_SIGNALING;GOBP_CHEMICAL_SYNAPTIC_TRANSMISSION_POSTSYNAPTIC;GOBP_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_POSITIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_RESPONSE_TO_PEPTIDE;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE;GOBP_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_REGULATION_OF_LEUKOCYTE_DIFFERENTIATION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_JUNCTION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_REGULATION_OF_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_MONONUCLEAR_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_POSITIVE_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_CELL_SURFACE_RECEPTOR_SIGNALING_PATHWAY_INVOLVED_IN_CELL_CELL_SIGNALING;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_MEMBRANE;GOBP_NEURON_PROJECTION_EXTENSION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_REGULATION_OF_MULTICELLULAR_ORGANISMAL_DEVELOPMENT;GOBP_POSITIVE_REGULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_CYTOKINE_PRODUCTION</td><td class="tb2">GOCC_NUCLEAR_ENVELOPE;GOCC_ENDOSOME;GOCC_MICROTUBULE_ORGANIZING_CENTER;GOCC_CELL_CELL_JUNCTION;GOCC_CELL_CORTEX;GOCC_ACTIN_CYTOSKELETON;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_AXON;GOCC_CELL_LEADING_EDGE;GOCC_ENVELOPE;GOCC_ACTIN_FILAMENT_BUNDLE;GOCC_NUCLEAR_PERIPHERY;GOCC_MYELIN_SHEATH_ABAXONAL_REGION;GOCC_SOMATODENDRITIC_COMPARTMENT;GOCC_ACTOMYOSIN;GOCC_NEURON_PROJECTION;GOCC_AXON_HILLOCK;GOCC_MYELIN_SHEATH;GOCC_APICAL_JUNCTION_COMPLEX;GOCC_CELL_BODY;GOCC_APICAL_PART_OF_CELL;GOCC_APICAL_CORTEX;GOCC_SYNAPSE;GOCC_PERINUCLEAR_REGION_OF_CYTOPLASM;GOCC_TRANSFERASE_COMPLEX_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOCC_TIGHT_JUNCTION;GOCC_ANCHORING_JUNCTION;GOCC_PLASMA_MEMBRANE_REGION;GOCC_SCHAFFER_COLLATERAL_CA1_SYNAPSE;GOCC_POSTSYNAPSE;GOCC_MEMBRANE_MICRODOMAIN;GOCC_GLUTAMATERGIC_SYNAPSE;GOCC_NEURON_TO_NEURON_SYNAPSE;GOCC_CYTOPLASMIC_REGION;GOCC_POSTSYNAPTIC_SPECIALIZATION;GOCC_CELL_CORTEX_REGION;GOCC_PAR_POLARITY_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_PROTEIN_KINASE_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_PROTEIN_KINASE_ACTIVITY;GOMF_PROTEIN_SERINE_THREONINE_KINASE_ACTIVITY;GOMF_PROTEIN_KINASE_C_ACTIVITY;GOMF_CALCIUM_DEPENDENT_PROTEIN_KINASE_ACTIVITY;GOMF_POTASSIUM_CHANNEL_REGULATOR_ACTIVITY;GOMF_CHANNEL_REGULATOR_ACTIVITY;GOMF_KINASE_ACTIVITY;GOMF_TRANSFERASE_ACTIVITY_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOMF_PURINE_NUCLEOTIDE_BINDING;GOMF_KINASE_BINDING;GOMF_ADENYL_NUCLEOTIDE_BINDING;GOMF_PHOSPHOLIPASE_BINDING;GOMF_INSULIN_RECEPTOR_SUBSTRATE_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_14_3_3_PROTEIN_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_PROTEIN_SERINE_KINASE_ACTIVITY</td><td class="tb2">KEGG_CHEMOKINE_SIGNALING_PATHWAY;KEGG_ENDOCYTOSIS;KEGG_TIGHT_JUNCTION;KEGG_INSULIN_SIGNALING_PATHWAY;KEGG_TYPE_II_DIABETES_MELLITUS</td><td class="tb2">PID_RHOA_PATHWAY;PID_INSULIN_PATHWAY;PID_WNT_NONCANONICAL_PATHWAY;PID_MET_PATHWAY;PID_CDC42_PATHWAY;PID_TXA2PATHWAY;PID_IL1_PATHWAY;PID_NFAT_3PATHWAY;PID_CXCR4_PATHWAY;PID_IGF1_PATHWAY;PID_TNF_PATHWAY;PID_IL2_PI3K_PATHWAY;PID_CERAMIDE_PATHWAY;PID_P75_NTR_PATHWAY;PID_AMB2_NEUTROPHILS_PATHWAY;PID_ERBB1_DOWNSTREAM_PATHWAY;PID_TRKR_PATHWAY;PID_NEPHRIN_NEPH1_PATHWAY;PID_INSULIN_GLUCOSE_PATHWAY;PID_RAS_PATHWAY</td><td class="tb2">BIOCARTA_PTDINS_PATHWAY</td><td class="tb2">REACTOME_HEMOSTASIS;REACTOME_GPVI_MEDIATED_ACTIVATION_CASCADE;REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX;REACTOME_SIGNALING_BY_VEGF;REACTOME_RHO_GTPASE_EFFECTORS;REACTOME_TGF_BETA_RECEPTOR_SIGNALING_IN_EMT_EPITHELIAL_TO_MESENCHYMAL_TRANSITION;REACTOME_VEGFR2_MEDIATED_CELL_PROLIFERATION;REACTOME_RHO_GTPASES_ACTIVATE_NADPH_OXIDASES;REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION;REACTOME_ESR_MEDIATED_SIGNALING;REACTOME_SIGNALING_BY_NUCLEAR_RECEPTORS;REACTOME_SIGNALING_BY_RECEPTOR_TYROSINE_KINASES;REACTOME_SIGNALING_BY_TGFB_FAMILY_MEMBERS;REACTOME_EXTRA_NUCLEAR_ESTROGEN_SIGNALING;REACTOME_ESTROGEN_STIMULATED_SIGNALING_THROUGH_PRKCZ;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Membranous and cytoplasmic expression in most tissues, including brain, kidney and retina.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Plasma membrane,Cytosol</td><td class="tb2">brain: 58.0</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">2055383</td><td class="tb2">rs70937049</td><td class="tb2">T</td><td class="tb2">A</td><td class="tb2">1531.06</td><td class="tb2">PASS</td><td class="tb2">PRKCZ</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ</td><td class="tb2">intronic</td><td class="tb2">NM_002744</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000468310.5,ENST00000503297.1,ENST00000378567.8,ENST00000484419.1</td><td class="tb2">.</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">rs70937049</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ID=COSN5265563;OCCURENCE=1(haematopoietic_and_lymphoid_tissue)</td><td class="tb2">.</td><td class="tb2">0.002</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">0.0784</td><td class="tb2">0.0161741</td><td class="tb2">.</td><td class="tb2">0.0024</td><td class="tb2">0.0024</td><td class="tb2">0</td><td class="tb2">0.0012</td><td class="tb2">0.0001</td><td class="tb2">0.0764</td><td class="tb2">7.023e-05</td><td class="tb2">0.0006</td><td class="tb2">0</td><td class="tb2">0.0044</td><td class="tb2">0.0031</td><td class="tb2">0.0031</td><td class="tb2">2.406e-05</td><td class="tb2">0.0027</td><td class="tb2">0.0015</td><td class="tb2">0.0768</td><td class="tb2">0.0002</td><td class="tb2">0.0007</td><td class="tb2">0</td><td class="tb2">0.0047</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">AC=2;AF=1;AN=2;DP=46;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=60;QD=34.02;SOR=2.059</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">1/1:0,45:45:99:1545,135,0</td><td class="tb2">T</td><td class="tb2">A</td><td class="tb2">1</td><td class="tb2">0</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ, 176982</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">11836310| 8224878| 22174128| 19520910| 12610628| 10477520| 12857744| 23283171| 11684013| 21127255| 11914719| 12145205| 17254974| 16931766| 21791414| 8378304| 15958741| 21385716| 17702943| 11462174| 23283174| 17558396</td><td class="tb2">GOBP_MAPK_CASCADE;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_CELL_MORPHOGENESIS;GOBP_CELL_ACTIVATION;GOBP_CYTOKINE_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_ADAPTIVE_IMMUNE_RESPONSE;GOBP_IMMUNE_EFFECTOR_PROCESS;GOBP_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_LYMPHOCYTE_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_DIFFERENTIATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_CYTOKINE_PRODUCTION;GOBP_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_T_CELL_MEDIATED_IMMUNITY;GOBP_ADAPTIVE_IMMUNE_RESPONSE_BASED_ON_SOMATIC_RECOMBINATION_OF_IMMUNE_RECEPTORS_BUILT_FROM_IMMUNOGLOBULIN_SUPERFAMILY_DOMAINS;GOBP_LEUKOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_T_CELL_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_APOPTOTIC_PROCESS;GOBP_DEFENSE_RESPONSE;GOBP_INFLAMMATORY_RESPONSE;GOBP_IMMUNE_RESPONSE;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_MICROTUBULE_BASED_MOVEMENT;GOBP_CELL_ADHESION;GOBP_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_CELL_MATRIX_ADHESION;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_CELL_POLARITY;GOBP_ENZYME_LINKED_RECEPTOR_PROTEIN_SIGNALING_PATHWAY;GOBP_TRANSMEMBRANE_RECEPTOR_PROTEIN_TYROSINE_KINASE_SIGNALING_PATHWAY;GOBP_CELL_CELL_SIGNALING;GOBP_BEHAVIOR;GOBP_MEMORY;GOBP_LONG_TERM_MEMORY;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_RESPONSE_TO_HORMONE;GOBP_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_POSITIVE_REGULATION_OF_CELL_COMMUNICATION;GOBP_POSITIVE_REGULATION_OF_CELL_DEVELOPMENT;GOBP_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_TRANSPORT_ALONG_MICROTUBULE;GOBP_CELL_GROWTH;GOBP_PHOSPHORYLATION;GOBP_PEPTIDYL_AMINO_ACID_MODIFICATION;GOBP_PEPTIDYL_SERINE_MODIFICATION;GOBP_PEPTIDYL_TYROSINE_MODIFICATION;GOBP_NEUROGENESIS;GOBP_REGULATION_OF_CELL_CELL_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_CELL_ADHESION;GOBP_NEGATIVE_REGULATION_OF_SIGNALING;GOBP_ESTABLISHMENT_OF_CELL_POLARITY;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_HEMOPOIESIS;GOBP_REGULATION_OF_CELL_ADHESION;GOBP_T_CELL_DIFFERENTIATION;GOBP_CYTOSKELETON_DEPENDENT_INTRACELLULAR_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_PROTEIN_CONTAINING_COMPLEX_LOCALIZATION;GOBP_ACTIVATION_OF_PHOSPHOLIPASE_D_ACTIVITY;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_NERVOUS_SYSTEM_PROCESS;GOBP_ACTIVATION_OF_PROTEIN_KINASE_ACTIVITY;GOBP_ACTIVATION_OF_PROTEIN_KINASE_B_ACTIVITY;GOBP_INTERLEUKIN_10_PRODUCTION;GOBP_INTERLEUKIN_13_PRODUCTION;GOBP_INTERLEUKIN_4_PRODUCTION;GOBP_INTERLEUKIN_5_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_10_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_13_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_4_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_5_PRODUCTION;GOBP_RESPONSE_TO_INSULIN;GOBP_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_CELLULAR_RESPONSE_TO_HORMONE_STIMULUS;GOBP_CELLULAR_COMPONENT_MORPHOGENESIS;GOBP_CELL_PART_MORPHOGENESIS;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_POSITIVE_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_CYTOKINE_PRODUCTION;GOBP_T_HELPER_2_CELL_CYTOKINE_PRODUCTION;GOBP_GROWTH;GOBP_TYPE_2_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_POSITIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_POSITIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_MAPK_CASCADE;GOBP_RESPONSE_TO_PEPTIDE_HORMONE;GOBP_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_CONTAINING_COMPLEX_ORGANIZATION;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_REGULATION_OF_CELLULAR_COMPONENT_BIOGENESIS;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_POSITIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_ADHESION;GOBP_REGULATION_OF_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_LYMPHOCYTE_ACTIVATION;GOBP_INTRACELLULAR_TRANSPORT;GOBP_VESICLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_REGULATION_OF_SYNAPTIC_PLASTICITY;GOBP_NEGATIVE_REGULATION_OF_RESPONSE_TO_STIMULUS;GOBP_DEVELOPMENTAL_CELL_GROWTH;GOBP_DEVELOPMENTAL_GROWTH;GOBP_NEURON_DEVELOPMENT;GOBP_GENERATION_OF_NEURONS;GOBP_CELL_MOTILITY;GOBP_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_REGULATION_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_SYNAPTIC_TRANSMISSION;GOBP_REGULATION_OF_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_ACTIVATION;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_COGNITION;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_NF_KAPPAB_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_MULTICELLULAR_ORGANISMAL_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_ORGANELLE_LOCALIZATION;GOBP_VESICLE_LOCALIZATION;GOBP_ESTABLISHMENT_OF_ORGANELLE_LOCALIZATION;GOBP_LOCALIZATION_WITHIN_MEMBRANE;GOBP_MEMBRANE_DEPOLARIZATION;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_POTENTIAL;GOBP_MEMBRANE_HYPERPOLARIZATION;GOBP_REGULATION_OF_LIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_LIPASE_ACTIVITY;GOBP_REGULATION_OF_CELL_DEVELOPMENT;GOBP_LONG_TERM_SYNAPTIC_POTENTIATION;GOBP_REGULATION_OF_CELLULAR_LOCALIZATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_DEVELOPMENTAL_GROWTH_INVOLVED_IN_MORPHOGENESIS;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_BIPOLAR_CELL_POLARITY;GOBP_PROTEIN_LOCALIZATION_TO_POSTSYNAPSE;GOBP_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_ERK1_AND_ERK2_CASCADE;GOBP_POSITIVE_REGULATION_OF_ERK1_AND_ERK2_CASCADE;GOBP_PROTEIN_KINASE_C_SIGNALING;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE_HORMONE_STIMULUS;GOBP_ORGANELLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_PROTEIN_LOCALIZATION_TO_PLASMA_MEMBRANE;GOBP_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_CELL_CELL_ADHESION;GOBP_REGULATION_OF_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_MODULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_NEUROTRANSMITTER_RECEPTOR_LEVELS;GOBP_MICROTUBULE_BASED_TRANSPORT;GOBP_REGULATION_OF_TRANS_SYNAPTIC_SIGNALING;GOBP_VESICLE_CYTOSKELETAL_TRAFFICKING;GOBP_SYNAPTIC_SIGNALING;GOBP_CHEMICAL_SYNAPTIC_TRANSMISSION_POSTSYNAPTIC;GOBP_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_POSITIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_RESPONSE_TO_PEPTIDE;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE;GOBP_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_REGULATION_OF_LEUKOCYTE_DIFFERENTIATION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_JUNCTION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_REGULATION_OF_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_MONONUCLEAR_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_POSITIVE_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_CELL_SURFACE_RECEPTOR_SIGNALING_PATHWAY_INVOLVED_IN_CELL_CELL_SIGNALING;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_MEMBRANE;GOBP_NEURON_PROJECTION_EXTENSION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_REGULATION_OF_MULTICELLULAR_ORGANISMAL_DEVELOPMENT;GOBP_POSITIVE_REGULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_CYTOKINE_PRODUCTION</td><td class="tb2">GOCC_NUCLEAR_ENVELOPE;GOCC_ENDOSOME;GOCC_MICROTUBULE_ORGANIZING_CENTER;GOCC_CELL_CELL_JUNCTION;GOCC_CELL_CORTEX;GOCC_ACTIN_CYTOSKELETON;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_AXON;GOCC_CELL_LEADING_EDGE;GOCC_ENVELOPE;GOCC_ACTIN_FILAMENT_BUNDLE;GOCC_NUCLEAR_PERIPHERY;GOCC_MYELIN_SHEATH_ABAXONAL_REGION;GOCC_SOMATODENDRITIC_COMPARTMENT;GOCC_ACTOMYOSIN;GOCC_NEURON_PROJECTION;GOCC_AXON_HILLOCK;GOCC_MYELIN_SHEATH;GOCC_APICAL_JUNCTION_COMPLEX;GOCC_CELL_BODY;GOCC_APICAL_PART_OF_CELL;GOCC_APICAL_CORTEX;GOCC_SYNAPSE;GOCC_PERINUCLEAR_REGION_OF_CYTOPLASM;GOCC_TRANSFERASE_COMPLEX_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOCC_TIGHT_JUNCTION;GOCC_ANCHORING_JUNCTION;GOCC_PLASMA_MEMBRANE_REGION;GOCC_SCHAFFER_COLLATERAL_CA1_SYNAPSE;GOCC_POSTSYNAPSE;GOCC_MEMBRANE_MICRODOMAIN;GOCC_GLUTAMATERGIC_SYNAPSE;GOCC_NEURON_TO_NEURON_SYNAPSE;GOCC_CYTOPLASMIC_REGION;GOCC_POSTSYNAPTIC_SPECIALIZATION;GOCC_CELL_CORTEX_REGION;GOCC_PAR_POLARITY_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_PROTEIN_KINASE_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_PROTEIN_KINASE_ACTIVITY;GOMF_PROTEIN_SERINE_THREONINE_KINASE_ACTIVITY;GOMF_PROTEIN_KINASE_C_ACTIVITY;GOMF_CALCIUM_DEPENDENT_PROTEIN_KINASE_ACTIVITY;GOMF_POTASSIUM_CHANNEL_REGULATOR_ACTIVITY;GOMF_CHANNEL_REGULATOR_ACTIVITY;GOMF_KINASE_ACTIVITY;GOMF_TRANSFERASE_ACTIVITY_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOMF_PURINE_NUCLEOTIDE_BINDING;GOMF_KINASE_BINDING;GOMF_ADENYL_NUCLEOTIDE_BINDING;GOMF_PHOSPHOLIPASE_BINDING;GOMF_INSULIN_RECEPTOR_SUBSTRATE_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_14_3_3_PROTEIN_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_PROTEIN_SERINE_KINASE_ACTIVITY</td><td class="tb2">KEGG_CHEMOKINE_SIGNALING_PATHWAY;KEGG_ENDOCYTOSIS;KEGG_TIGHT_JUNCTION;KEGG_INSULIN_SIGNALING_PATHWAY;KEGG_TYPE_II_DIABETES_MELLITUS</td><td class="tb2">PID_RHOA_PATHWAY;PID_INSULIN_PATHWAY;PID_WNT_NONCANONICAL_PATHWAY;PID_MET_PATHWAY;PID_CDC42_PATHWAY;PID_TXA2PATHWAY;PID_IL1_PATHWAY;PID_NFAT_3PATHWAY;PID_CXCR4_PATHWAY;PID_IGF1_PATHWAY;PID_TNF_PATHWAY;PID_IL2_PI3K_PATHWAY;PID_CERAMIDE_PATHWAY;PID_P75_NTR_PATHWAY;PID_AMB2_NEUTROPHILS_PATHWAY;PID_ERBB1_DOWNSTREAM_PATHWAY;PID_TRKR_PATHWAY;PID_NEPHRIN_NEPH1_PATHWAY;PID_INSULIN_GLUCOSE_PATHWAY;PID_RAS_PATHWAY</td><td class="tb2">BIOCARTA_PTDINS_PATHWAY</td><td class="tb2">REACTOME_HEMOSTASIS;REACTOME_GPVI_MEDIATED_ACTIVATION_CASCADE;REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX;REACTOME_SIGNALING_BY_VEGF;REACTOME_RHO_GTPASE_EFFECTORS;REACTOME_TGF_BETA_RECEPTOR_SIGNALING_IN_EMT_EPITHELIAL_TO_MESENCHYMAL_TRANSITION;REACTOME_VEGFR2_MEDIATED_CELL_PROLIFERATION;REACTOME_RHO_GTPASES_ACTIVATE_NADPH_OXIDASES;REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION;REACTOME_ESR_MEDIATED_SIGNALING;REACTOME_SIGNALING_BY_NUCLEAR_RECEPTORS;REACTOME_SIGNALING_BY_RECEPTOR_TYROSINE_KINASES;REACTOME_SIGNALING_BY_TGFB_FAMILY_MEMBERS;REACTOME_EXTRA_NUCLEAR_ESTROGEN_SIGNALING;REACTOME_ESTROGEN_STIMULATED_SIGNALING_THROUGH_PRKCZ;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Membranous and cytoplasmic expression in most tissues, including brain, kidney and retina.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Plasma membrane,Cytosol</td><td class="tb2">brain: 58.0</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">L</td><td class="tb2">chr1</td><td class="tb2">2056364</td><td class="tb2">rs2803309</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">820.06</td><td class="tb2">PASS</td><td class="tb2">PRKCZ</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ</td><td class="tb2">intronic</td><td class="tb2">NM_002744</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ENST00000479566.5,ENST00000466651.5,ENST00000503297.1,ENST00000378567.8,ENST00000468310.5</td><td class="tb2">.</td><td class="tb2">1p36.33</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">rs2803309</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">ID=COSN6402106;OCCURENCE=1(large_intestine),1(liver)</td><td class="tb2">.</td><td class="tb2">0.4356</td><td class="tb2">0.1988</td><td class="tb2">0.8328</td><td class="tb2">0.3184</td><td class="tb2">0.8333</td><td class="tb2">0.556709</td><td class="tb2">.</td><td class="tb2">0.2331</td><td class="tb2">0.2195</td><td class="tb2">0.7621</td><td class="tb2">0.3473</td><td class="tb2">0.2871</td><td class="tb2">0.7970</td><td class="tb2">0.1695</td><td class="tb2">0.2375</td><td class="tb2">0.2495</td><td class="tb2">0.2775</td><td class="tb2">0.3858</td><td class="tb2">0.3860</td><td class="tb2">0.7538</td><td class="tb2">0.3933</td><td class="tb2">0.3112</td><td class="tb2">0.8052</td><td class="tb2">0.1784</td><td class="tb2">0.2409</td><td class="tb2">0.2648</td><td class="tb2">0.3548</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">AC=2;AF=1;AN=2;DP=22;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=60;QD=31.19;SOR=4.712</td><td class="tb2">GT:AD:DP:GQ:PL</td><td class="tb2">1/1:0,21:21:63:834,63,0</td><td class="tb2">G</td><td class="tb2">A</td><td class="tb2">1</td><td class="tb2">0</td><td class="tb2">PROTEIN KINASE C, ZETA FORM; PRKCZ, 176982</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">.</td><td class="tb2">11836310| 8224878| 22174128| 19520910| 12610628| 10477520| 12857744| 23283171| 11684013| 21127255| 11914719| 12145205| 17254974| 16931766| 21791414| 8378304| 15958741| 21385716| 17702943| 11462174| 23283174| 17558396</td><td class="tb2">GOBP_MAPK_CASCADE;GOBP_MICROTUBULE_CYTOSKELETON_ORGANIZATION;GOBP_CELL_MORPHOGENESIS;GOBP_CELL_ACTIVATION;GOBP_CYTOKINE_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_MATRIX_ADHESION;GOBP_ADAPTIVE_IMMUNE_RESPONSE;GOBP_IMMUNE_EFFECTOR_PROCESS;GOBP_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_LYMPHOCYTE_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_DIFFERENTIATION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_T_CELL_CYTOKINE_PRODUCTION;GOBP_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_T_CELL_MEDIATED_IMMUNITY;GOBP_ADAPTIVE_IMMUNE_RESPONSE_BASED_ON_SOMATIC_RECOMBINATION_OF_IMMUNE_RECEPTORS_BUILT_FROM_IMMUNOGLOBULIN_SUPERFAMILY_DOMAINS;GOBP_LEUKOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_SYSTEM_PROCESS;GOBP_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_POSITIVE_REGULATION_OF_IMMUNE_EFFECTOR_PROCESS;GOBP_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_PRODUCTION_OF_MOLECULAR_MEDIATOR_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_MEDIATED_IMMUNITY;GOBP_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_T_CELL_MEDIATED_IMMUNITY;GOBP_POSITIVE_REGULATION_OF_CYTOKINE_PRODUCTION_INVOLVED_IN_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_T_CELL_CYTOKINE_PRODUCTION;GOBP_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_ADAPTIVE_IMMUNE_RESPONSE;GOBP_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_TYPE_2_IMMUNE_RESPONSE;GOBP_APOPTOTIC_PROCESS;GOBP_DEFENSE_RESPONSE;GOBP_INFLAMMATORY_RESPONSE;GOBP_IMMUNE_RESPONSE;GOBP_CYTOSKELETON_ORGANIZATION;GOBP_MICROTUBULE_BASED_PROCESS;GOBP_MICROTUBULE_BASED_MOVEMENT;GOBP_CELL_ADHESION;GOBP_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_CELL_MATRIX_ADHESION;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_CELL_POLARITY;GOBP_ENZYME_LINKED_RECEPTOR_PROTEIN_SIGNALING_PATHWAY;GOBP_TRANSMEMBRANE_RECEPTOR_PROTEIN_TYROSINE_KINASE_SIGNALING_PATHWAY;GOBP_CELL_CELL_SIGNALING;GOBP_BEHAVIOR;GOBP_MEMORY;GOBP_LONG_TERM_MEMORY;GOBP_POSITIVE_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_RESPONSE_TO_ENDOGENOUS_STIMULUS;GOBP_RESPONSE_TO_HORMONE;GOBP_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHOLIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_GENE_EXPRESSION;GOBP_POSITIVE_REGULATION_OF_CELL_COMMUNICATION;GOBP_POSITIVE_REGULATION_OF_CELL_DEVELOPMENT;GOBP_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_CELL_DEATH;GOBP_TRANSPORT_ALONG_MICROTUBULE;GOBP_CELL_GROWTH;GOBP_PHOSPHORYLATION;GOBP_PEPTIDYL_AMINO_ACID_MODIFICATION;GOBP_PEPTIDYL_SERINE_MODIFICATION;GOBP_PEPTIDYL_TYROSINE_MODIFICATION;GOBP_NEUROGENESIS;GOBP_REGULATION_OF_CELL_CELL_ADHESION;GOBP_POSITIVE_REGULATION_OF_CELL_CELL_ADHESION;GOBP_NEGATIVE_REGULATION_OF_SIGNALING;GOBP_ESTABLISHMENT_OF_CELL_POLARITY;GOBP_CELL_PROJECTION_ORGANIZATION;GOBP_HEMOPOIESIS;GOBP_REGULATION_OF_CELL_ADHESION;GOBP_T_CELL_DIFFERENTIATION;GOBP_CYTOSKELETON_DEPENDENT_INTRACELLULAR_TRANSPORT;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_MODIFICATION_PROCESS;GOBP_PROTEIN_CONTAINING_COMPLEX_LOCALIZATION;GOBP_ACTIVATION_OF_PHOSPHOLIPASE_D_ACTIVITY;GOBP_CELL_SUBSTRATE_ADHESION;GOBP_REGULATION_OF_NERVOUS_SYSTEM_PROCESS;GOBP_ACTIVATION_OF_PROTEIN_KINASE_ACTIVITY;GOBP_ACTIVATION_OF_PROTEIN_KINASE_B_ACTIVITY;GOBP_INTERLEUKIN_10_PRODUCTION;GOBP_INTERLEUKIN_13_PRODUCTION;GOBP_INTERLEUKIN_4_PRODUCTION;GOBP_INTERLEUKIN_5_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_10_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_13_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_4_PRODUCTION;GOBP_POSITIVE_REGULATION_OF_INTERLEUKIN_5_PRODUCTION;GOBP_RESPONSE_TO_INSULIN;GOBP_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_CELLULAR_RESPONSE_TO_HORMONE_STIMULUS;GOBP_CELLULAR_COMPONENT_MORPHOGENESIS;GOBP_CELL_PART_MORPHOGENESIS;GOBP_PROTEIN_LOCALIZATION_TO_ORGANELLE;GOBP_POSITIVE_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_CYTOKINE_PRODUCTION;GOBP_T_HELPER_2_CELL_CYTOKINE_PRODUCTION;GOBP_GROWTH;GOBP_TYPE_2_IMMUNE_RESPONSE;GOBP_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_POPULATION_PROLIFERATION;GOBP_REGULATION_OF_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_POSITIVE_REGULATION_OF_PHOSPHORYLATION;GOBP_REGULATION_OF_MEMBRANE_POTENTIAL;GOBP_POSITIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_REGULATION_OF_PROTEIN_CONTAINING_COMPLEX_ASSEMBLY;GOBP_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_MAPK_CASCADE;GOBP_RESPONSE_TO_PEPTIDE_HORMONE;GOBP_REGULATION_OF_KINASE_ACTIVITY;GOBP_PROTEIN_CONTAINING_COMPLEX_ORGANIZATION;GOBP_REGULATION_OF_SYSTEM_PROCESS;GOBP_REGULATION_OF_CELLULAR_COMPONENT_BIOGENESIS;GOBP_NEGATIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_POSITIVE_REGULATION_OF_MOLECULAR_FUNCTION;GOBP_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_CELL_ADHESION;GOBP_REGULATION_OF_INSULIN_RECEPTOR_SIGNALING_PATHWAY;GOBP_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_POSITIVE_REGULATION_OF_ALPHA_BETA_T_CELL_DIFFERENTIATION;GOBP_LYMPHOCYTE_ACTIVATION;GOBP_INTRACELLULAR_TRANSPORT;GOBP_VESICLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_REGULATION_OF_SYNAPTIC_PLASTICITY;GOBP_NEGATIVE_REGULATION_OF_RESPONSE_TO_STIMULUS;GOBP_DEVELOPMENTAL_CELL_GROWTH;GOBP_DEVELOPMENTAL_GROWTH;GOBP_NEURON_DEVELOPMENT;GOBP_GENERATION_OF_NEURONS;GOBP_CELL_MOTILITY;GOBP_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_NEGATIVE_REGULATION_OF_PEPTIDYL_TYROSINE_PHOSPHORYLATION;GOBP_REGULATION_OF_IMMUNE_RESPONSE;GOBP_POSITIVE_REGULATION_OF_IMMUNE_RESPONSE;GOBP_REGULATION_OF_CATALYTIC_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_SYNAPTIC_TRANSMISSION;GOBP_REGULATION_OF_T_CELL_ACTIVATION;GOBP_REGULATION_OF_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CELL_ACTIVATION;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_COGNITION;GOBP_REGULATION_OF_TRANSPORT;GOBP_POSITIVE_REGULATION_OF_TRANSPORT;GOBP_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DNA_BINDING_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_NF_KAPPAB_TRANSCRIPTION_FACTOR_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_DEVELOPMENTAL_PROCESS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_COMPONENT_ORGANIZATION;GOBP_REGULATION_OF_PHOSPHORUS_METABOLIC_PROCESS;GOBP_POSITIVE_REGULATION_OF_MULTICELLULAR_ORGANISMAL_PROCESS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_NEGATIVE_REGULATION_OF_PROTEIN_METABOLIC_PROCESS;GOBP_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_LYMPHOCYTE_ACTIVATION;GOBP_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_NEGATIVE_REGULATION_OF_HYDROLASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_TRANSFERASE_ACTIVITY;GOBP_ORGANELLE_LOCALIZATION;GOBP_VESICLE_LOCALIZATION;GOBP_ESTABLISHMENT_OF_ORGANELLE_LOCALIZATION;GOBP_LOCALIZATION_WITHIN_MEMBRANE;GOBP_MEMBRANE_DEPOLARIZATION;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_POTENTIAL;GOBP_MEMBRANE_HYPERPOLARIZATION;GOBP_REGULATION_OF_LIPASE_ACTIVITY;GOBP_POSITIVE_REGULATION_OF_LIPASE_ACTIVITY;GOBP_REGULATION_OF_CELL_DEVELOPMENT;GOBP_LONG_TERM_SYNAPTIC_POTENTIATION;GOBP_REGULATION_OF_CELLULAR_LOCALIZATION;GOBP_NEGATIVE_REGULATION_OF_CELL_DEATH;GOBP_DEVELOPMENTAL_GROWTH_INVOLVED_IN_MORPHOGENESIS;GOBP_ESTABLISHMENT_OR_MAINTENANCE_OF_BIPOLAR_CELL_POLARITY;GOBP_PROTEIN_LOCALIZATION_TO_POSTSYNAPSE;GOBP_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_ERK1_AND_ERK2_CASCADE;GOBP_POSITIVE_REGULATION_OF_ERK1_AND_ERK2_CASCADE;GOBP_PROTEIN_KINASE_C_SIGNALING;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE_HORMONE_STIMULUS;GOBP_ORGANELLE_TRANSPORT_ALONG_MICROTUBULE;GOBP_PROTEIN_LOCALIZATION_TO_PLASMA_MEMBRANE;GOBP_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_CELL_CELL_ADHESION;GOBP_REGULATION_OF_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_MODULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_POSTSYNAPTIC_MEMBRANE_NEUROTRANSMITTER_RECEPTOR_LEVELS;GOBP_MICROTUBULE_BASED_TRANSPORT;GOBP_REGULATION_OF_TRANS_SYNAPTIC_SIGNALING;GOBP_VESICLE_CYTOSKELETAL_TRAFFICKING;GOBP_SYNAPTIC_SIGNALING;GOBP_CHEMICAL_SYNAPTIC_TRANSMISSION_POSTSYNAPTIC;GOBP_NEUROTRANSMITTER_RECEPTOR_LOCALIZATION_TO_POSTSYNAPTIC_SPECIALIZATION_MEMBRANE;GOBP_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_NEGATIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_POSITIVE_REGULATION_OF_CELLULAR_RESPONSE_TO_INSULIN_STIMULUS;GOBP_RESPONSE_TO_PEPTIDE;GOBP_CELLULAR_RESPONSE_TO_PEPTIDE;GOBP_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_NITROGEN_COMPOUND;GOBP_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_CELLULAR_RESPONSE_TO_OXYGEN_CONTAINING_COMPOUND;GOBP_REGULATION_OF_LEUKOCYTE_DIFFERENTIATION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_JUNCTION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_SYNAPSE;GOBP_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_POSITIVE_REGULATION_OF_INTRACELLULAR_SIGNAL_TRANSDUCTION;GOBP_REGULATION_OF_RECEPTOR_LOCALIZATION_TO_SYNAPSE;GOBP_POSITIVE_REGULATION_OF_LEUKOCYTE_CELL_CELL_ADHESION;GOBP_MONONUCLEAR_CELL_DIFFERENTIATION;GOBP_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_HEMOPOIESIS;GOBP_POSITIVE_REGULATION_OF_PROTEIN_LOCALIZATION;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_POSITIVE_REGULATION_OF_ESTABLISHMENT_OF_PROTEIN_LOCALIZATION;GOBP_CELL_SURFACE_RECEPTOR_SIGNALING_PATHWAY_INVOLVED_IN_CELL_CELL_SIGNALING;GOBP_REGULATION_OF_PROTEIN_LOCALIZATION_TO_MEMBRANE;GOBP_NEURON_PROJECTION_EXTENSION;GOBP_PROTEIN_LOCALIZATION_TO_CELL_PERIPHERY;GOBP_REGULATION_OF_MULTICELLULAR_ORGANISMAL_DEVELOPMENT;GOBP_POSITIVE_REGULATION_OF_EXCITATORY_POSTSYNAPTIC_POTENTIAL;GOBP_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_CD4_POSITIVE_ALPHA_BETA_T_CELL_ACTIVATION;GOBP_POSITIVE_REGULATION_OF_T_HELPER_2_CELL_CYTOKINE_PRODUCTION</td><td class="tb2">GOCC_NUCLEAR_ENVELOPE;GOCC_ENDOSOME;GOCC_MICROTUBULE_ORGANIZING_CENTER;GOCC_CELL_CELL_JUNCTION;GOCC_CELL_CORTEX;GOCC_ACTIN_CYTOSKELETON;GOCC_MICROTUBULE_CYTOSKELETON;GOCC_AXON;GOCC_CELL_LEADING_EDGE;GOCC_ENVELOPE;GOCC_ACTIN_FILAMENT_BUNDLE;GOCC_NUCLEAR_PERIPHERY;GOCC_MYELIN_SHEATH_ABAXONAL_REGION;GOCC_SOMATODENDRITIC_COMPARTMENT;GOCC_ACTOMYOSIN;GOCC_NEURON_PROJECTION;GOCC_AXON_HILLOCK;GOCC_MYELIN_SHEATH;GOCC_APICAL_JUNCTION_COMPLEX;GOCC_CELL_BODY;GOCC_APICAL_PART_OF_CELL;GOCC_APICAL_CORTEX;GOCC_SYNAPSE;GOCC_PERINUCLEAR_REGION_OF_CYTOPLASM;GOCC_TRANSFERASE_COMPLEX_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOCC_TIGHT_JUNCTION;GOCC_ANCHORING_JUNCTION;GOCC_PLASMA_MEMBRANE_REGION;GOCC_SCHAFFER_COLLATERAL_CA1_SYNAPSE;GOCC_POSTSYNAPSE;GOCC_MEMBRANE_MICRODOMAIN;GOCC_GLUTAMATERGIC_SYNAPSE;GOCC_NEURON_TO_NEURON_SYNAPSE;GOCC_CYTOPLASMIC_REGION;GOCC_POSTSYNAPTIC_SPECIALIZATION;GOCC_CELL_CORTEX_REGION;GOCC_PAR_POLARITY_COMPLEX;GOCC_CATALYTIC_COMPLEX;GOCC_PROTEIN_KINASE_COMPLEX;GOCC_TRANSFERASE_COMPLEX</td><td class="tb2">GOMF_PROTEIN_KINASE_ACTIVITY;GOMF_PROTEIN_SERINE_THREONINE_KINASE_ACTIVITY;GOMF_PROTEIN_KINASE_C_ACTIVITY;GOMF_CALCIUM_DEPENDENT_PROTEIN_KINASE_ACTIVITY;GOMF_POTASSIUM_CHANNEL_REGULATOR_ACTIVITY;GOMF_CHANNEL_REGULATOR_ACTIVITY;GOMF_KINASE_ACTIVITY;GOMF_TRANSFERASE_ACTIVITY_TRANSFERRING_PHOSPHORUS_CONTAINING_GROUPS;GOMF_PURINE_NUCLEOTIDE_BINDING;GOMF_KINASE_BINDING;GOMF_ADENYL_NUCLEOTIDE_BINDING;GOMF_PHOSPHOLIPASE_BINDING;GOMF_INSULIN_RECEPTOR_SUBSTRATE_BINDING;GOMF_PROTEIN_CONTAINING_COMPLEX_BINDING;GOMF_14_3_3_PROTEIN_BINDING;GOMF_MOLECULAR_FUNCTION_REGULATOR_ACTIVITY;GOMF_PROTEIN_SERINE_KINASE_ACTIVITY</td><td class="tb2">KEGG_CHEMOKINE_SIGNALING_PATHWAY;KEGG_ENDOCYTOSIS;KEGG_TIGHT_JUNCTION;KEGG_INSULIN_SIGNALING_PATHWAY;KEGG_TYPE_II_DIABETES_MELLITUS</td><td class="tb2">PID_RHOA_PATHWAY;PID_INSULIN_PATHWAY;PID_WNT_NONCANONICAL_PATHWAY;PID_MET_PATHWAY;PID_CDC42_PATHWAY;PID_TXA2PATHWAY;PID_IL1_PATHWAY;PID_NFAT_3PATHWAY;PID_CXCR4_PATHWAY;PID_IGF1_PATHWAY;PID_TNF_PATHWAY;PID_IL2_PI3K_PATHWAY;PID_CERAMIDE_PATHWAY;PID_P75_NTR_PATHWAY;PID_AMB2_NEUTROPHILS_PATHWAY;PID_ERBB1_DOWNSTREAM_PATHWAY;PID_TRKR_PATHWAY;PID_NEPHRIN_NEPH1_PATHWAY;PID_INSULIN_GLUCOSE_PATHWAY;PID_RAS_PATHWAY</td><td class="tb2">BIOCARTA_PTDINS_PATHWAY</td><td class="tb2">REACTOME_HEMOSTASIS;REACTOME_GPVI_MEDIATED_ACTIVATION_CASCADE;REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX;REACTOME_SIGNALING_BY_VEGF;REACTOME_RHO_GTPASE_EFFECTORS;REACTOME_TGF_BETA_RECEPTOR_SIGNALING_IN_EMT_EPITHELIAL_TO_MESENCHYMAL_TRANSITION;REACTOME_VEGFR2_MEDIATED_CELL_PROLIFERATION;REACTOME_RHO_GTPASES_ACTIVATE_NADPH_OXIDASES;REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION;REACTOME_ESR_MEDIATED_SIGNALING;REACTOME_SIGNALING_BY_NUCLEAR_RECEPTORS;REACTOME_SIGNALING_BY_RECEPTOR_TYROSINE_KINASES;REACTOME_SIGNALING_BY_TGFB_FAMILY_MEMBERS;REACTOME_EXTRA_NUCLEAR_ESTROGEN_SIGNALING;REACTOME_ESTROGEN_STIMULATED_SIGNALING_THROUGH_PRKCZ;REACTOME_SIGNALING_BY_RHO_GTPASES_MIRO_GTPASES_AND_RHOBTB3;REACTOME_SIGNALING_BY_RHO_GTPASES</td><td class="tb2">Membranous and cytoplasmic expression in most tissues, including brain, kidney and retina.</td><td class="tb2">Tissue enhanced<At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.></td><td class="tb2">Detected in at least a third but not all tissues.</td><td class="tb2">Plasma membrane,Cytosol</td><td class="tb2">brain: 58.0</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                    <p class="tremark">Note: The first line shows homozygous regions and its included genes, the line below shows the SNPs in homozygous regions and their annotation.
                                The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                
                                </div>
                                <p class="paragraph">Result: <a href="../../../result/Pro/ROH" title="Click to open" target="_blank">/result/Pro/ROH</a></p>
                                



                                <!------------------------------------------------------------------ Pathway-------------------------------------------------------------------- -->
                                
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                <a name="Candidate gene set pathway enrichment analysis"></a>
                                <br />
                                
                                <h4>4.5.4 Variant filtering based on candidate genes and their relationship with disease phenotypes</h4>
                                <h5>4.5.4.1 Candidate gene set pathway enrichment analysis</h5>
                                
                                <p class="paragraph">In vivo, different genes perform their biological functions through mutual coordination. Especially for complex diseases, phenotypic differences may be caused by mutations of multiple genes. Through significant enrichment analysis, the most important biochemical metabolic pathways and signal transduction pathways involved in mutant genes are determined.</p>
                                
                                <a name="GO pathway enrichment analysis"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.4.1.1 GO functional enrichment analysis</p>
                                
                                
                                <p class="paragraph">The results of GO functional enrichment analysis are shown below. GO functional enrichment analysis will generate CC (cell component), BP (biological pathway) and MF (molecular function) results respectively.</p>
                                <div id="slider3" class="flexslider" style="margin:auto;width:600px;">
                                  <div class="flex-viewport" style="overflow: hidden; position: relative;">
                                    <ul class="slides" style="transition-duration: 0s;transform: translate3d(0px, 0px, 0px);width: 1600%;">
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_cc_dotplot.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_CC dot chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_go_CC.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_CC bar chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_bp_dotplot.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_BP dot chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_go_BP.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_BP bar chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_mf_dotplot.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_MF dot chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_go_MF.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">GO_MF bar chart</p>
                                        </li>
                                        
                                    </ul>
                                  </div>
                                  <ul class="flex-direction-nav">
                                    <li class="flex-nav-prev">
                                      <a class="flex-prev" onclick="previousFun(this)"></a>
                                    </li>
                                    <li class="flex-nav-next">
                                      <a class="flex-next" onclick="nextFun(this)"></a>
                                    </li>
                                  </ul>
                                </div>
                                <p class="name_fig"> Figure 4.5.6 Scatter plot and bar chart of GO functional enrichment analysis</p>
                                <p class="tremark">Note:
                                As shown in the scatter plot, the horizontal coordinate represents the proportion of genes enriched in the pathway to the total enriched genes, and the vertical coordinate represents the enriched GO term name. The dot size represents the number of genes enriched to the gene term, and the color represents p-value.
                                As shown in the bar chart, the horizontal coordinate represents the number of genes belonging to this term in the gene set, the vertical coordinate represents the enriched GO term name, and the color represents p-value.
                                </p>
                                
                                <div id="page"></div>
                                <a name="KEGG pathway enrichment analysis"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.4.1.2 KEGG functional enrichment analysis</p>
                                
                                
                                <p class="paragraph">The following figures are the scatterplot and bar plot of KEGG pathway enrichment analysis.</p>
                                <div id="slider3" class="flexslider" style="margin:auto;width:600px;">
                                  <div class="flex-viewport" style="overflow: hidden; position: relative;">
                                    <ul class="slides" style="transition-duration: 0s;transform: translate3d(0px, 0px, 0px);width: 1600%;">
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_kegg_dotplot.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">KEGG dot chart</p>
                                        </li>
                                        
                                        <li style="width: 596px; margin-right: 0px; float: left; display: block;">
                                            <img src=src/pictures/Pathway/Pathway_kegg.png style="margin:auto" />
                                            <p class="flex-caption" style="text-align:center">KEGG bar chart</p>
                                        </li>
                                        
                                    </ul>
                                  </div>
                                  <ul class="flex-direction-nav">
                                    <li class="flex-nav-prev">
                                      <a class="flex-prev" onclick="previousFun(this)"></a>
                                    </li>
                                    <li class="flex-nav-next">
                                      <a class="flex-next" onclick="nextFun(this)"></a>
                                    </li>
                                  </ul>
                                </div>
                                <p class="name_fig"> Figure 4.5.7 Scatter plot and bar chart of KEGG functional enrichment analysis</p>
                                <p class="tremark">Note:
                                As shown in the scatter plot, the horizontal coordinate represents the proportion of genes enriched in the pathway to the total enriched genes, and the vertical coordinate represents the enriched GO term name. The dot size represents the number of genes enriched to the gene term, and the color represents p-value.
                                As shown in the bar chart, the horizontal coordinate represents the number of genes belonging to this term in the gene set, the vertical coordinate represents the enriched GO term name, and the color represents p-value.</p>
                                </p>
                                
                                <div class="hide_in_pdf">
                                    <p class="paragraph">Table 4.5.20 shows the detailed information of KEGG PATHWAY enrichment. Click ID to link to the pathway diagram.</p>
                                </div>
                                <!-- 20220601 -->
                                <p class="center"></p>
                                <p class="name_fig">Table 4.5.20KEGG path enrichment details</p>
                                <div id="tb">
                                    <table class="gy tb2">
                                        
                                          
                                            <tr class="tb2"><th class="tb2">ID</th><th class="tb2">Description</th><th class="tb2">GeneRatio</th><th class="tb2">BgRatio</th><th class="tb2">pvalue</th><th class="tb2">padjust</th><th class="tb2">qvalue</th><th class="tb2">geneID</th><th class="tb2">Count</th></tr>
                                          
                                        
                                          
                                            <tr class="tb2">
                                              
                                                
                                                    
                                                      <td class="tb2"><a hreslice:f="src/pictures/Pathway/hsa04974.svg" target="_blank">hsa04974</a></td>
                                                    
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">Protein digestion and absorption</td>
                                                    
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">2|26</td>
                                                    
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">95|8018</td>
                                                    
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">0.0375458760011132</td>
                                                    
                                                
                                              
                                                
                                                  <td class="tb2">1</td>
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">0.427950510105234</td>
                                                    
                                                
                                              
                                                
                                                    
                                                      <td class="tb2">ATP1B2/COL7A1</td>
                                                    
                                                
                                              
                                                
                                                  <td class="tb2">2</td>
                                                
                                              
                                            </tr>
                                          
                                        
                                    </table>
                                </div>
                                <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) ID: ID of GO term/KEGG pathway. The ID link is the PATHWAY diagram, named after the path number, begining with 'hsa' (hsa is an abbreviation of the human species name Homo sapiens), such as 'hsa00010.svg'. The PATHWAY map uses Orthology (homologous genes) as a unit to display the position and interaction of genes in the pathway, so it is marked by the EC number of an enzyme (such as 1.1.1.2), or the name of a homologous gene (such as ITGA) (in a few cases, the name of a single gene (such as ITGA1)). The green background represents human genes. When you hover over the Gene box, you can see the Gene Symbol and Entrez ID of all the genes corresponding to this position. Click on the gene box to link to the corresponding page on KEGG's official website. The corresponding candidate genes are marked with red boxes to facilitate the understanding of gene position and upstream and downstream in the pathway.
                                </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Description: GO term/KEGG pathway description
                                </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) GeneRatio: proportion of enriched genes, that is, the proportion of candidate genes enriched in the GO term or KEGG pathway to all candidate genes corresponding to the GO term or KEGG pathway database </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) BgRatio: background gene ratio, that is, the ratio of the number of human genes contained in the GO term or KEGG pathway to the number of human genes corresponding to all the GO term or KEGG pathway databases </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) pvalue: P-value obtained by enrichment analysis </p> 
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) padjust: p-value after correction </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) qvalue: q-value obtained by enrichment analysis </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) geneID: list of genes enriched in corresponding GO term or KEGG pathway </p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) Count: the number of genes enriched in the corresponding GO term or KEGG pathway </p>
                                </div>
                                
                                
                                <button onclick="hiddenQA(this)">Show Q&A </button>
                                <div style="display: none;">
                                <table style="font-size:12px;font-family: 'Arial'">
                                    <tr><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q1: What is the difference between pvalue, padjust and qvalue in GO and KEGG analysis? Which shall prevail in the analysis?</td></tr>
                                <tr><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A1: p.value reflects the significance level of enrichment analysis, and P-value &lt;0.05 is generally considered significant. p.adjust is the corrected p.value, which is corrected by using FDR in multiple tests. The calculation formula is: p.adjust(i)= P-value (i)*length(p)/rank(p). length(p) refers to the number of enrichment results, and rank(p) refers to the ranking in the enrichment results. qvalue is the qvalue for statistical test of P-value, that is, Q-value (i)=pFDR(T>=i). It is recommended that priority be paid to p.adjust, that is, the corrected p.value. When the values of p.adjust are all 1, it may be caused by the small sample size, which can be referred to p.value. </td></tr>
                                </table>
                                </div>
                                </div>
                                <p class="paragraph">Result: <a href="../../../result/Pro/Pathway" title="Click to open" target="_blank">/result/Pro/Pathway</a></p>
                                
                                <!--------------------------------------------------------------- phenolyzer ------------------------------------------------------------------------------>
                                
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                    <div id="page"></div>
                                <div style="width:100%; margin:0 auto; padding:0;">
                                <a name="Gene-disease phenotypic association analysis"></a>
                                <br />
                                
                                <h5>4.5.4.2 Gene-disease phenotypic association analysis</h5>
                                
                                <a name="DisGeNet database annotation result"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.4.2.1 DisGeNet database annotation result</p>
                                
                                <div>
                                <p class="paragraph">The DisGeNet database (Pinero <i>et al.</i>, 2017) is a database that focuses on gene-disease associations and mutation-disease associations. The current version of the database is V5.0. It contains 561,119 gene-disease association records (GDAs) and 135,588 mutation-disease association records (VDAs). The associated data in this database is used to obtain relevant gene and variation information by disease for subsequent analysis. </p>
                                
                                <p class="center">
                                <p class="name_fig">Table 4.5.21 The Result of gene-disease phenotypic association analysis (only the first few lines of information are taken for display)</p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">geneId</th><th class="tb2">associationType</th><th class="tb2">pmid</th><th class="tb2">score</th><th class="tb2">originalSource</th><th class="tb2">diseaseName</th><th class="tb2">diseaseType</th><th class="tb2">geneSymbol</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">4047</td><td class="tb2">GermlineCausalMutation</td><td class="tb2">30401459</td><td class="tb2">0.32</td><td class="tb2">ORPHANET</td><td class="tb2">Hypotrichosis simplex</td><td class="tb2">disease</td><td class="tb2">LSS</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">4047</td><td class="tb2">GeneticVariation</td><td class="tb2">30401459</td><td class="tb2">0.32</td><td class="tb2">BEFREE</td><td class="tb2">Hypotrichosis simplex</td><td class="tb2">disease</td><td class="tb2">LSS</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">4047</td><td class="tb2">Biomarker</td><td class="tb2">30723320</td><td class="tb2">0.32</td><td class="tb2">BEFREE</td><td class="tb2">Hypotrichosis simplex</td><td class="tb2">disease</td><td class="tb2">LSS</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">2316</td><td class="tb2">Biomarker</td><td class="tb2">nan</td><td class="tb2">0.1</td><td class="tb2">HPO</td><td class="tb2">Hypotrichosis</td><td class="tb2">disease</td><td class="tb2">FLNA</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">4047</td><td class="tb2">Biomarker</td><td class="tb2">nan</td><td class="tb2">0.1</td><td class="tb2">HPO</td><td class="tb2">Hypotrichosis</td><td class="tb2">disease</td><td class="tb2">LSS</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                    <p class="tremark"> (1) geneId: Entrez Gene Identifier, from NCBI</p>
                                    <p class="tremark"> (2) associationType: Association types, including Altered Expression, Biomarker, Chromosomal Rearrangement, Fusion Gene, Genetic Variation, Germline Causal Mutation, Germline Modifying Mutation, Modifying Mutation, PostTranslational Modification, Somatic Modifying Mutation, germline modifying mutation, modifying mutation, posttranslational modification, somatic modifying mutation, Susceptibility Mutation, Therapeutic</p>
                                    <p class="tremark"> (3) pmid: PubMed Unique Identifier, which is the number of literature in the fields of life science and medicine included in the PubMed search engine </p>
                                    <p class="tremark"> (4) score: DisGENET GDA Score (DisGENET score for the Gene Disease association). The score ranges from 0 to 1. A larger value indicates a higher correlation </p>
                                    <p class="tremark"> (5) originalSource: The original reported source for this gene-disease association can be BEFREE, CGI, CLINGEN, CLINVAR, CTD_human, CTD_mouse, CTD_rat, GENOMICS_ENGLAND, GWASCAT, GWASDB, HPO, LHGDN, MGD, or genomics_England. ORPHANET, PSYGENET, RGD, UNIPROT and their combinations
                                    <p class="tremark"> (6) diseaseName: name of disease, from UMLS</p>
                                    <p class="tremark"> (7) diseaseType: disease, phenotype and group </p>
                                    <p class="tremark"> (8) geneSymbol: Official Gene Symbol from NCBI</p>
                                    </div>
                                
                                
                                <a name="Phenolyzer analysis result"></a>
                                <br />
                                
                                <p style="font-size: 15px; font-weight: bold;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.5.4.2.2 Phenolyzer analysis result</p>
                                
                                <div class="hide_in_pdf">
                                    <p style="text-indent: 2em;font-size:14px;line-height:25px;">In disease research, it is very important to determine the association between candidate genes and diseases. According to the diseases/phenotypes provided by the client, genes were screened and ranked by precise algorithms, combined with sequencing results and multiple databases, and the correlation diagram between genes and disease phenotypes was constructed (Yang et al., 2015).</p>
                                
                                
                                                                        <div style="margin:auto;width:900px;height:75;border:0px solid #000000"
                                                                            id="network_control" class="panel panel-primary">
                                                                            <div id="cy" style="width:100%;height:600px;margin:auto"></div>
                                                                            <div style="margin:auto;width:15%;float:left;">
                                                                                <select id="show_edges" class="form-control" title="Association type">
                                                                                    <option selected value="" disabled="true">Association type</option>
                                                                                    <option value="all">all</option>
                                                                                    <option value="HPRD">Protein Interaction</option>
                                                                                    <option value="GeneFamily">In the same GeneFamily </option>
                                                                                    <option value="Biosystem">In the same Biosystem</option>
                                                                                    <option value="TranscriptionInteraction">Transcription Interaction
                                                                                    </option>
                                                                                </select>
                                                                            </div>
                                                                            <div style="margin:auto;width:15%;float:left;">
                                                                                <select id="adjust_layout" class="form-control" title="Arrangement mode">
                                                                                    <option selected value="" disabled="true">Arrangement mode</option>
                                                                                    <option value="force">Random</option>
                                                                                    <option value="grid">matrix/option>
                                                                                </select>
                                                                            </div>
                                                                            <div style="width:69%;float:right">
                                                                                Reset image
                                                                                <button id="undo"
                                                                                    class="button button-raised button-primary button-circle "><i
                                                                                        class="icon-undo"></i></button>
                                                                                &nbsp&nbsp disease
                                                                                <input id="disease_on" type="checkbox" style="margin:auto" checked />
                                                                                &nbsp&nbsp gene
                                                                                <input id="gene_on" type="checkbox" style="margin:auto" checked />
                                                                                <button type="button" style="float:right"
                                                                                    class="button button-raised button-primary button-pill"
                                                                                            id="save_photo">Save image</button>
                                                                            </div>
                                                                        </div>
                                                                        <div style="height:20px;clear:both"></div>
                                                                        <p class="name_fig">Figure 4.5.8 Gene-disease phenotypic association graph</p>
                                                                        <div style="width:100%;text-align:left; ">
                                                                            <p class='tremark'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Click on a point in the diagram to display the disease/gene associated with that point, click on the blank to restore;</p>
                                                                            <p class='tremark'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The pink word represents the target disease/phenotypic keywords, and the blue rounded rectangle represents the related diseases searched according to the keywords;</p>
                                                                            <p class='tremark'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The green dots indicate genes that have been reported or linked to the disease in the database;</p>
                                                        <p class='tremark'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The orange dots represent genes that are thought to be related to the green gene based on multiple associations.</p>
                                                                        </div>
                                                                   
                                                                   </div>
                                                               </div>
                                                        </div>
                                
                                
                                <p class="paragraph">According to the correlation between genes and diseases, the ranking results are as follows:</p>
                                
                                <div id="BArplot">
                                    <div style="height:810px"></div>
                                </div>
                                
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Note:</p>
                                <p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Rankings are for reference only. The correlation score is maximum at 1 and sorted according to the built-in rules, with too many genes showing only the top 20. </p>
                                
                                <p class="name_fig">Figure 4.5.9 Candidate gene ranking graph</p>
                                
                                </div>
                                <p class="paragraph">Result: <a href="../../../result/Pro/Network" title="Click to open" target="_blank">/result/Pro/Network</a></p>
                                
                                
                                <!------------------------------------------------------------- Genemania protein-protein interaction  -------------------------------------------------------->
                                
                                <div id="page">
                                    <p class="head">
                                        <a href="#home" title="返回首页"></a>
                                        <hr />
                                    </p>
                                <a name="Protein functional interaction analysis"></a>
                                <br />
                                
                                <h5>4.5.4.3 Protein functional interaction analysis (PPI analysis)</h5>
                                
                                
                                <p class="paragraph"><a href="http://genemania.org/" target=_blank>GeneMania</a>(Warde-Farley <i>et al.</i>, 2010) was used to analyze the protein functional interaction network of candidate genes, including protein-protein, protein-DNA-genetic interactions, pathways, reactions,gene-protein expression data, protein domains-phenotypic screening profiles. The results of protein interaction analysis are shown in the table below:</p>
                                <p class="center">
                                <p class="name_fig">Table 4.5.22 The result of protein functional interaction analysis ((only the first few lines of information are shown)) 
                                    </p>
                                    <div id="tb">
                                        <table class="gy tb2">
                                            
                                            
                                            <tr class="tb2"><th class="tb2">Gene 1</th><th class="tb2">Gene 2</th><th class="tb2">Weight</th><th class="tb2">Type</th><th class="tb2">Source</th></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">B2M</td><td class="tb2">HLA-A</td><td class="tb2">0.01784945151831996</td><td class="tb2">Co-expression</td><td class="tb2">Agnelli-Neri-2007</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">FHOD3</td><td class="tb2">SVIL</td><td class="tb2">0.008197933916129782</td><td class="tb2">Co-expression</td><td class="tb2">Agnelli-Neri-2007</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">GOLGA8B</td><td class="tb2">GOLGA8A</td><td class="tb2">0.025062193065460737</td><td class="tb2">Co-expression</td><td class="tb2">Agnelli-Neri-2007</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">HLA-C</td><td class="tb2">B2M</td><td class="tb2">0.017749064931532256</td><td class="tb2">Co-expression</td><td class="tb2">Agnelli-Neri-2007</td></tr>
                                            
                                            
                                            
                                            <tr class="tb2"><td class="tb2">HLA-C</td><td class="tb2">HLA-A</td><td class="tb2">0.023532263497408164</td><td class="tb2">Co-expression</td><td class="tb2">Agnelli-Neri-2007</td></tr>
                                            
                                            
                                        </table>
                                    </div>
                                    </p>
                                    <button onclick="hiddenShow(this)">Show Annotation</button>
                                <div style="display: none;">
                                        <p class="tremark">The annotation is the same as the content of section "Variant Annotation" above.</p>
                                    </div>
                                    
                                </div>
                                <p class="paragraph">Result: <a href="../../../result/Pro/PPI" title="Click to open" target="_blank">/result/Pro/PPI</a></p>
                                

                                
																<div id="page">
																  <p class="head"><a href="#home" title="返回首页"></a><a name="HLA"></a><hr /></p><br />
																  <h4> 4.5.5 HLA Typing</h4>
																  <p class="paragraph">
																    <p>HLA is one of the most polymorphic regions in the human genome, characterized by high GC content, making direct alignment with reference genes less effective. The International Immunogenetics Information System (IMGT) is a comprehensive database focusing on vertebrate immunoglobulins, T-cell receptors (TCR), and the major histocompatibility complex (MHC). Using reference genome maps constructed from the IMGT database enables more precise alignment and analysis of target sequences.</p>
																    <p>HLA-HD is employed for HLA gene typing, typically achieving four-digit precision, with a maximum of eight-digit high-resolution typing. Based on the level of differentiation, HLA typing can be classified into four categories: two-digit, four-digit, six-digit, and eight-digit. Two digits represent allele groups, which generally correspond to homologous serological antigens. Four digits can distinguish different subtypes within the same group, including distinct coding proteins caused by mutations. Six digits further differentiate sequence changes from nonsynonymous mutations, while eight digits rely on information outside coding regions, identifying distinct sequences in non-coding regions. Generally, for studying the relationship between antigens and tumors, four-digit resolution is required. Due to technical limitations, achieving eight-digit resolution is often difficult.</p>

																  <p class="center"><img class="middle" src="src/images/HLA_principle_hd.png" style="height: 250px;"></p>  
																  <p class="name_fig">Figure 4.5.10  HLA Naming Convention</p>
																  <!--- <p class="name_fig">Table 4.23 HLA Typing Results</p> -->
																  <p class="name_fig">Table HLA Typing Results</p> 
																  <div id="tb">
																    <table class="tb2">
																      <tr class="tb2">
																        <th class="tb2">Sample</th>
																        <th class="tb2">Gene</th>
																        <th class="tb2">Allele1</th>
																        <th class="tb2">Allele2</th>
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">A</td>
																        
																        <td class="tb2">HLA-A*03:01:01</td>
																        
																        <td class="tb2">HLA-A*24:02:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">B</td>
																        
																        <td class="tb2">HLA-B*07:02:01</td>
																        
																        <td class="tb2">HLA-B*15:01:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">C</td>
																        
																        <td class="tb2">HLA-C*07:02:01</td>
																        
																        <td class="tb2">HLA-C*08:01:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB1</td>
																        
																        <td class="tb2">HLA-DRB1*15:01:01</td>
																        
																        <td class="tb2">HLA-DRB1*08:03:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DQA1</td>
																        
																        <td class="tb2">HLA-DQA1*01:02:01</td>
																        
																        <td class="tb2">HLA-DQA1*01:03:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DQB1</td>
																        
																        <td class="tb2">HLA-DQB1*06:02:01</td>
																        
																        <td class="tb2">HLA-DQB1*06:01:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DPA1</td>
																        
																        <td class="tb2">HLA-DPA1*01:03:01</td>
																        
																        <td class="tb2">HLA-DPA1*02:02:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DPB1</td>
																        
																        <td class="tb2">HLA-DPB1*04:01:01</td>
																        
																        <td class="tb2">HLA-DPB1*02:01:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DMA</td>
																        
																        <td class="tb2">HLA-DMA*01:01:01</td>
																        
																        <td class="tb2">HLA-DMA*01:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DMB</td>
																        
																        <td class="tb2">HLA-DMB*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DOA</td>
																        
																        <td class="tb2">HLA-DOA*01:01:04</td>
																        
																        <td class="tb2">HLA-DOA*01:01:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DOB</td>
																        
																        <td class="tb2">HLA-DOB*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRA</td>
																        
																        <td class="tb2">HLA-DRA*01:02:03</td>
																        
																        <td class="tb2">HLA-DRA*01:02:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB2</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB3</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB4</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB5</td>
																        
																        <td class="tb2">HLA-DRB5*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB6</td>
																        
																        <td class="tb2">HLA-DRB6*02:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB7</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB8</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DRB9</td>
																        
																        <td class="tb2">HLA-DRB9*01:02:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">DPA2</td>
																        
																        <td class="tb2">HLA-DPA2*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">E</td>
																        
																        <td class="tb2">HLA-E*01:03:02</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">F</td>
																        
																        <td class="tb2">HLA-F*01:01:01</td>
																        
																        <td class="tb2">HLA-F*01:03:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">G</td>
																        
																        <td class="tb2">HLA-G*01:04:01</td>
																        
																        <td class="tb2">HLA-G*01:01:01</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">H</td>
																        
																        <td class="tb2">HLA-H*02:04</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">J</td>
																        
																        <td class="tb2">HLA-J*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">K</td>
																        
																        <td class="tb2">HLA-K*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">L</td>
																        
																        <td class="tb2">HLA-L*01:01:01</td>
																        
																        <td class="tb2">HLA-L*01:02</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">T</td>
																        
																        <td class="tb2">HLA-T*03:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">V</td>
																        
																        <td class="tb2">HLA-V*01:01:01</td>
																        
																        <td class="tb2">-</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">W</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																      <tr class="tb2">
																        
																        <td class="tb2">YXHP</td>
																        
																        <td class="tb2">Y</td>
																        
																        <td class="tb2">Not typed</td>
																        
																        <td class="tb2">Not typed</td>
																        
																      </tr>
																      
																    </table>
																  </div>
																</div>
																
						

																<button onclick="hiddenShow(this)">Show Comments</button>
																<div style="display: none;">
																<div class="comment">
																  <ol style="font-size: 12px;font-family: 'SimHei';line-height: 20px">
																    <li>Sample: Sample name</li>
																    <li>Gene: HLA gene locus</li>
																    <li>Allele1/Allele2: Allele typing results, including gene family, classification fields, and mutation information (e.g., HLA-A*02:01:01 represents a specific allele of the HLA-A gene)</li>
																    <li>HLA: Prefix</li>
																    <li>"-": Hyphen separating the HLA prefix and gene name</li>
																    <li>A: Gene locus</li>
																    <li>"*": Delimiter</li>
																    <li>":": Field separator</li>
																  </ol>
																</div>
															  </div>

																<p class="paragraph">Results directory: <a href="../../../result/Pro/HLA" title="Click to open" target="_blank">/result/Pro/HLA</a></p>


                                
                                <!-- ----------------------------------- 软件列表 ------------------------------------------- -->
                                <div id="page">
                                <p class="head"><a href="#home" title = "返回首页"></a>
                                <a name="Appendix"></a>
                                <hr />
                                </p><br />
                                
                                
                                        <h2>5 Appendix</h2>
                                        <h3>5.1 Software list</h3>
                                
                                <p class="paragraph">The software used in analysis are listed below: </p>
                                
                                <p class="center">
                                
                                <p class="name_fig">Main software used in the analysis </p>
                                
                                <div id="tb">
                                <table class="gy">
                                
                                <tr><th>Analytical content</th><th>Software</th><th>Comments</th><th>Version</th></tr>
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;Quality Control</td><td>fastp</td><td>Quality control for the raw data and obtain high-quality data for subsequent analysis</td><td>v0.23.1</td></tr>
                                
                                
                                <tr><td rowspan=3 style="text-align:left">&nbsp;&nbsp;Alignment</td><td>BWA</td><td>Map the sequencing reads to the reference genome, and output the alignment file in the bam format</td><td>v0.7.17</td></tr>
                                <tr><td>Sambamba</td><td>Sort the bam file</td><td>v0.6.7</td></tr>
                                <tr><td>Picard</td><td>Merge all bam files from the same sample and mark the duplicated reads</td><td>v2.18.9</td></tr>
                                
                                <!--<tr><td>GATK</td><td>local realignment of reads around indels; base quality score recalibration</td><td>v3.1</td></tr>-->
                                
                                
                                
                                        
                                        <tr><td style="text-align:left">&nbsp;&nbsp;SNP/InDel detection</td><td>GATK</td><td>Detect and filter SNPs/InDels</td><td>v4.3.0</td></tr>
                                        
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;Annotation</td><td>ANNOVAR</td><td>Annotate variants</td><td>2015Dec14</td></tr>
                                
                                
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;IBD analysis</td><td>king</td><td>Analyze the genetic relationship among the samples</td><td>v2.2.9</td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Variant filtering according to known databases</td><td>in house</td><td>Screen mutations according to the relevant filtering conditions</td><td> </td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Variant filtering based on Deleterious Classification</td><td>in house</td><td>Screen deleterious mutations according to the relevant filtering conditions</td><td> </td></tr>
                                
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;Variant filtering according to dominant/recessive inheritance patterns</td><td>in house</td><td>Screen mutations according to dominant/recessive inheritance patterns for family samples</td><td> </td></tr>
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;Linkage analysis</td><td>Merlin</td><td>Family linkage analysis</td><td>v1.1.2</td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;De novo mutation analysis</td><td>SAMTools/Triodenovo</td><td>To study denovo mutations</td><td>v1.6/v0.06</td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Candidate gene set pathway enrichment analysis</td><td>in house</td><td>To to identify the most important biochemical metabolic pathways and signal transduction pathways involved in mutated genes</td><td> </td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Gene-disease phenotypic association analysis</td><td>Phenolyzer</td><td>To study the relationship between diseases and genes</td><td>v0.4.0</td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Protein functional interaction analysis (PPI analysis)</td><td>GeneMania</td><td>To study functional interaction of proteins</td><td>2014-08-12</td></tr>
                                <tr><td style="text-align:left">&nbsp;&nbsp;Runs of Homozygosity(ROH analysis)</td><td>H3M2</td><td>Identification of Continuous Runs of Homozygosity</td><td>2014-10-20</td></tr>
                                
                                <tr><td style="text-align:left">&nbsp;&nbsp;HLA</td><td>HLAHD</td><td>HLA Typing</td><td>1.2.0.1</td></tr>
                                </table>
                                </div>
                                </p>


                                
                                        <h3>5.2 Method</h3>
                                

                                <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
                                
                                <p class="paragraph">File catalog: <a href="src/method/WES_method.pdf" title="Click to open"
                                                target="_blank">Method</a></p><br />
                                


                                
                                        <h3>5.3 Result File Decompression Method and Format Description</h3>
                                

                                <p class="center">
                                <div id="tb">
                                        <table class="gy">

                                                <tr>
                                                        <th>File Type</th>
                                                        <th>File Description</th>
                                                        <th>Way to Open</th>
                                                </tr>
                                                <tr class="tb2">
                                                        <td class="tb2" rowspan=2>*.tar files</td>
                                                        <td class="tb2">Unix/Linux/Mac</td>
                                                        <td class="tb2">cmd: tar -xvf *.tar</td>
                                                </tr>
                                                <tr>
                                                        <td class="tb2">Windows</td>
                                                        <td class="tb2">Decompress software: WinRAR, 7-Zip</td>
                                                </tr>
                                                <tr class="tb2">
                                                        <td class="tb2" rowspan=2>*.gz files</td>
                                                        <td class="tb2">Unix/Linux/Mac</td>
                                                        <td class="tb2">cmd: gzip -d *.gz</td>
                                                </tr>
                                                <tr>
                                                        <td class="tb2">Windows</td>
                                                        <td class="tb2">Decompress software:WinRAR, 7-Zip</td>
                                                </tr>
                                                <tr class="tb2">
                                                        <td class="tb2" rowspan=2>*.zip files</td>
                                                        <td class="tb2">Unix/Linux/Mac</td>
                                                        <td class="tb2">cmd: unzip *.zip</td>
                                                </tr>
                                                <tr>
                                                        <td class="tb2">Windows</td>
                                                        <td class="tb2">Decompress software:WinRAR, 7-Zip</td>
                                                </tr>
                                                <tr class="tb2">
                                                        <td class="tb2" rowspan=2>file.fq/fastq</td>
                                                        <td class="tb2">Unix/Linux/Mac</td>
                                                        <td class="tb2">cmd: less *.fq.gz or more</td>
                                                </tr>
                                                <tr>
                                                        <td class="tb2">Windows</td>
                                                        <td class="tb2">Software:Editplus/Notepad++ <I>et al.</I></td>
                                                </tr>
                                                <tr class="tb2">
                                                        <td class="tb2" rowspan=2>file.bam</td>
                                                        <td class="tb2">Unix/Linux/Mac</td>
                                                        <td class="tb2">cmd: samtools view/tview *.bam or more</td>
                                                </tr>
                                        </table>
                                </div>
                        </div>


                                
                                <!-- ----------------------------------- 参考文献 ------------------------------------------- -->
                                <div id="page">
                                <p class="head"><a href="#home" title = "返回首页"></a>
                                <a name="References"></a>
                                <hr />
                                </p><br />
                                <h2>6 References</h2>
                                
                                <p style="font-size: 19px; font-weight: bold;">Standard analysis references</p>
                                <p class="ref">[1] Chen S, Zhou Y, Chen Y, et al. fastp: an ultra-fast all-in-one FASTQ preprocessor[J]. Bioinformatics, 2018, 34(17):i884-i890. (Fastp)</br></br>
                                
                                [2] Li H, Durbin R. Fast and accurate short read alignment with Burrows–Wheeler transform[J]. Bioinformatics, 2009, 25(14): 1754-1760. (BWA_MEM)</br></br>
                                
                                [3]  Artem T, Vilella A J, Edwin C, et al. Sambamba: fast processing of NGS alignment formats[J]. Bioinformatics, 2015(12):2032-2034. (Sambamba)</br></br>
                                
                                [4] Picard: http://sourceforge.net/projects/picard/ . (Picard)</br></br>
                                
                                
                                
                                [5]  DePristo M A, Banks E, Poplin R, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data[J]. Nature genetics, 2011, 43(5): 491-498. (GATK)</br></br>
                                
                                [6]  Li H, Handsaker B, Wysoker A, et al. The sequence alignment/map format and SAMtools[J]. Bioinformatics, 2009, 25(16): 2078-2079. (Samtools)
                                
                                
                                
                                [7] Wang K, Li M, Hakonarson H. ANNOVAR: Functional annotation of genetic variants from high-throughput sequencing data[J]. Nucleic acids research, 2010, 38(16): e164. (ANNOVAR)</br> </br>
                                
                                [8]  Kent W J, Sugnet C W, Furey T S, et al. The human genome browser at UCSC[J]. Genome research, 2002, 12(6): 996-1006. (UCSC)</br></br>
                                
                                [9] 1000 Genomes Project Consortium. An integrated map of genetic variation from 1,092 human genomes[J]. Nature, 2012, 491(7422): 56-65. (1000g)</br></br>
                                
                                [10] Sherry S T, Ward M H, Kholodov M, et al. dbSNP: The NCBI database of genetic variation[J]. Nucleic acids research, 2001, 29(1): 308-311. (dbSNP)</br></br>
                                
                                [11] Hamosh A, Scott A F, Amberger J S, et al. Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders[J]. Nucleic acids research, 2005, 33(suppl 1): D514-D517. (OMIM)</br></br>
                                
                                [12] Gene Ontology Consortium. The Gene Ontology (GO) database and informatics resource[J]. Nucleic acids research, 2004, 32(suppl 1): D258-D261. (GO)</br></br>
                                
                                [13] Kanehisa M, Goto S. KEGG: Kyoto encyclopedia of genes and genomes[J]. Nucleic acids research, 2000, 28(1): 27-30. (KEGG PATHWAY)</br></br>
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                
                                </p>
                                
                                <p style="font-size: 19px; font-weight: bold;">Advance analysis references</p>
                                <p class="ref">
                                  
                                  Conrad DF, Keebler JE, DePristo MA, et al. Variation in genome-wide mutation rates within and between human families. Nat Genet. 2011;43(7):712-714. Published 2011 Jun 12. doi:10.1038/ng.862</br></br>
                                  Kong A, Frigge ML, Masson G, et al. Rate of de novo mutations and the importance of father's age to disease risk. Nature. 2012;488(7412):471-475. doi:10.1038/nature11396</br></br>
                                  Muona M, Berkovic SF, Dibbens LM, et al. A recurrent de novo mutation in KCNC1 causes progressive myoclonus epilepsy. Nat Genet. 2015;47(1):39-46. doi:10.1038/ng.3144</br></br>
                                  Sanders SJ, Murtha MT, Gupta AR, et al. De novo mutations revealed by whole-exome sequencing are strongly associated with autism. Nature. 2012;485(7397):237-241. Published 2012 Apr 4. doi:10.1038/nature10945</br></br>
                                  Zaidi S, Choi M, Wakimoto H, et al. De novo mutations in histone-modifying genes in congenital heart disease. Nature. 2013;498(7453):220-223. doi:10.1038/nature12141</br></br>
                                  
                                  
                                  Joshi PK, Esko T, Mattsson H, et al. Directional dominance on stature and cognition in diverse human populations. Nature. 2015;523(7561):459-462. doi:10.1038/nature14618</br></br>
                                  Keller MC, Simonson MA, Ripke S, et al. Runs of homozygosity implicate autozygosity as a schizophrenia risk factor. PLoS Genet. 2012;8(4):e1002656. doi:10.1371/journal.pgen.1002656</br></br>
                                  Magi A, Tattini L, Palombo F, et al. H3M2: detection of runs of homozygosity from whole-exome sequencing data. Bioinformatics. 2014;30(20):2852-2859. doi:10.1093/bioinformatics/btu401</br></br>
                                  
                                  
                                  Piñero J, Bravo À, Queralt-Rosinach N, et al. DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants. Nucleic Acids Res. 2017;45(D1):D833-D839. doi:10.1093/nar/gkw943</br></br>
                                  Yang H, Robinson PN, Wang K. Phenolyzer: phenotype-based prioritization of candidate genes for human diseases. Nat Methods. 2015;12(9):841-843. doi:10.1038/nmeth.3484</br></br>
                                  
                                </p>
                                
                                
                                </div>                                
                                
                                
</div>
</body>
</html>


