<!-- django temple created by chenshuangzhao in 2020.04.01-->

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>
<head>

<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="x-ua-compatible" content="IE=11,edge" />




<title> XXXXXX项目生物信息信息分析结题报告 </title>



<!--<meta> 标签提供了 HTML 文档的元数据。元数据不会显示在客户端，但是会被浏览器解析。META元素通常用于指定网页的描述，关键词. 元数据通常以 名称/值 对出现-->
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<link href="css/buttons.css" rel="stylesheet" />
<link href="css/font-awesome.min.css" type="text/css" rel="stylesheet" />
<link rel="stylesheet" type="text/css" href="css/base.css">
<link href="css/base_new.css" type="text/css" rel="stylesheet">
<link href="css/cloth.css" rel="stylesheet" type="text/css" media="screen" />
<link rel="stylesheet" type="text/css" href="js/fancybox/jquery.fancybox-1.3.4.css" media="screen" />
<link rel="stylesheet" href="css/bootstrap-4.3.1.min.css" />
<link rel="stylesheet" href="css/gallery.css" />
<link rel="stylesheet" href="css/jquery-ui.css" />
<link rel="stylesheet" href="css/jquery.tocify.css" />
<link rel="stylesheet" href="css/style.css" />
<script src="js/jquery-3.3.1.min.js" type="text/javascript"></script>
<script src="js/jquery-ui.min.js" type="text/javascript"></script>
<script src="js/jquery.tocify.min.js" type="text/javascript"></script>
<script src="js/bootstrap-4.3.1.min.js" type="text/javascript"></script>
<script src="js/network.js"></script>
<script src="js/echarts.min.js"></script>
<script src="js/test.js"></script>
<script src="js/scrolltop.js" type="text/javascript"></script>
<script src="js/common.js" type="text/javascript"></script>
<script src="js/jquery.albumSlider.min.js" type="text/javascript"></script>
<script type="text/javascript" src="js/tablecloth.js"></script>
<script type="text/javascript" src="js/fancybox/jquery.mousewheel-3.0.4.pack.js"></script>
<script type="text/javascript" src="js/fancybox/jquery.fancybox-1.3.4.pack.js"></script>
<script src="js/esl.js" type="text/javascript"></script>
<script src="js/scrollTop.js" type="text/javascript"></script>
<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
</script>

  <script type="text/javascript">
    /* show or hide element*/
    function hiddenQA(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Q&A";
      };
      return 0;
    }
  </script>

<script type="text/javascript">
    $(function () {

      var options = {

        selectors: "h2,h3,h4,h5",

        theme: "bootstrap",

        scrollTo: "60"

      };

      var toc = $("#toc").tocify(options).data("toc-tocify");

    });

  </script>





  <script type="text/javascript">

    function trans(img_div_id) {

      img_div = $("#" + img_div_id)

      img = $("#" + img_div_id + " img").first();

      if (img_div.hasClass("fullscreen-div")) {

        img_div.removeClass("fullscreen-div");

        img.removeClass("fullscreen-img");

      }

      else {

        img_div.addClass("fullscreen-div");

        img.addClass("fullscreen-img");

      }

    }



    function filter_display() {

      return $(this).css("display") != "none";

    }



    function show_default() {

      var list_group_id = "#" + $(this).attr("id");

      if ($(list_group_id + " a.active").css("display") == "none") {

        var list_tab = $(list_group_id + " a.active");

        var pan_id = list_tab.attr("href");

        $(pan_id).removeClass("active show");

        list_tab.removeClass("active");

      }

      if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

        $(list_group_id + " a").filter(filter_display).first().tab("show");

      }

    }

    $.fn.show_default = show_default



    function next(botton_id) {

      var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

      while (present_tab.next().length == 1) {

        if (present_tab.next().css("display") != "none") {

          present_tab.next().tab("show");

          break;

        }

        else {

          present_tab = present_tab.next();

        }

      }



    }

    function prev(botton_id) {

      var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

      while (present_tab.prev().length == 1) {

        if (present_tab.prev().css("display") != "none") {

          present_tab.prev().tab("show");

          break;

        }

        else {

          present_tab = present_tab.prev();

        }

      }





    }



    $(document).ready(function () {

      $(".form-control").on("keyup", function () {

        var value = $(this).val().toLowerCase();

        var input_box_id = "#" + $(this).attr("id");

        var list_group_id = input_box_id.replace("_InputList", "_list-tab");

        $(list_group_id + " a").filter(function () {

          $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

        });

        $(list_group_id).show_default();

      });

      $(".list-group").each(show_default);

    });



  </script>

	
<style media="print">
	.noprint {DISPLAY: none;}
	#main{width:95%;right:30px;}
	.printmenu{display:block;}
</style>
<!-- class 寻找 style 定义文档样式,如需链接外部样式表，请使用 <link> 标签,如果没有使用 "scoped" 属性，则每一个 <style> 标签必须位于 head 头部区域, type 唯一可能的值是 "text/css" 规定样式表的 MIME 类型-->
<!-- type 寻找js/css--> 

	<div class="noprint">
		<div style="display: block;" id="goTopBtn">
			<a class="backtotop" title="回顶部"><img src="images/goTop.jpg" width="30" height="30" class="back-tip"/></a>
		</div>
	</div>
	
	<script type="text/javascript">
		var Tree = new Array;
		// nodeId | parentNodeId | nodeName | nodeUrl
	<{jtree}>
	</script> 

  <link href="css/new/base_new.css" type="text/css" rel="stylesheet">

    
  <style>
    .countt {
      counter-reset: slideNum;
    }
    .countt tr:before {
      counter-increment: slideNum;
      content: counter(slideNum)".";
    }
  </style>

<script>
var obj = document.getElementById("div_id");
function hideOrShowEvent(){
var obj= document.getElementById("div_id");
if(obj.style.display=="block"){ //如果是显示则隐藏
obj.style.display="none";
}else{ //如果是隐藏则显示
obj.style.display="block";
}
}
</script>

<script>
var obj = document.getElementById("div_id1");
function hideOrShowEvent1(){
var obj= document.getElementById("div_id1");
if(obj.style.display=="block"){ //如果是显示则隐藏
obj.style.display="none";
}else{ //如果是隐藏则显示
obj.style.display="block";
}
}
</script>  
  
</head>	
<body>

<!-- --------------------------------------------------------------- Menu bar----------------------------------------------------------------------- -->
<!--  <div id="contentLeft" class="linear" style="overflow-y:auto">
    <img style="margin-top:10px;margin-left:5%;width:90%"/>
  <ul id="leftNavigation" class="nav">

<li><a href="#lncRNA-seq Workflow">lncRNA-seq Workflow</a></li>


<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#建库测序流程">实验流程</a>
<ul>

<li><a href="#RNA样品检测"> RNA样品检测</a></li>


<li><a href="#文库构建"><i class="icon-angle-right"></i> 文库构建</a></li>

<li><a href="#库检"><i class="icon-angle-right"></i> 库检</a></li>
<li><a href="#上机测序"><i class="icon-angle-right"></i> 上机测序</a></li>
</ul>
</li> 

<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#Bioinformatics Workflow">Bioinformatics Workflow</a>
<ul>
<li><a href="#Bioinformatics Workflow"><i class="icon-angle-right"></i>Analysis Workflow</a></li>
<li><a href="#Project Results"><i class="icon-angle-right"></i>Project Results</a></li>
</ul>
</li>


<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#Data Quality Control">Data Quality Control</a>
<ul>
<li><a href="#raw data"><i class="icon-angle-right"></i> raw data</a></li>
<li><a href="#Data Filtering"><i class="icon-angle-right"></i> Data Filtering</a></li>
<li><a href="#Error Rate"><i class="icon-angle-right"></i> Error Rate</a></li>
<li><a href="#GC-content"><i class="icon-angle-right"></i> GC-content</a></li>
<li><a href="#Data Quality Control Summary"><i class="icon-angle-right"></i>Data Quality Control Summary</a></li>
</ul>
</li>



<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#Mapping To Reference Genome">Mapping and Assembly</a>
<ul>
<li><a href="#Overview of Mapping Status"><i class="icon-angle-right"></i> Overview of Mapping Status</a></li>
<li><a href="#Reads Distribution in Reference Genome"><i class="icon-angle-right"></i> Reads Distribution in Reference Genome</a></li>
<li><a href="#Visualization of Mapping Results"><i class="icon-angle-right"></i> Visualization of Mapping Results</a></li>



<li><a href="#Transcript Assembly"><i class="icon-angle-right"></i>Transcript Assembly</a></li>



<li><a href="#Transcript Identification"><i class="icon-angle-right"></i>Transcript Identification</a></li>
<li><a href="#Feature comparison of lncRNA and mRNA"><i class="icon-angle-right"></i>Feature comparison of lncRNA and mRNA</a></li>




</ul>
</li>
<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#Quantification">Quantification and Difference</a>
<ul>
<li><a href="#Quantification Results"><i class="icon-angle-right"></i>Quantification Results</a></li>
<li><a href="#Distribution of Expression Levels"><i class="icon-angle-right"></i> Distribution of Expression Levels</a></li>



<li><a href="#Differential Gene List"><i class="icon-angle-right"></i>Differential Gene List</a></li></li>
<li><a href="#Volcano plot of differential genes"><i class="icon-angle-right"></i>Volcano plot of differential genes</a></li></li>
<li><a href="#Clustering of differential genes"><i class="icon-angle-right"></i> Clustering of differential genes</a></li>




</ul>
</li>
<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#Functional analysis">Functional analysis</a>
<ul>
<li><a href="#GO enrichment analysis"><i class="icon-angle-right"></i>GO enrichment analysis</a></li>
<li><a href="#KEGG Enrichment Analysis"><i class="icon-angle-right"></i>KEGG Enrichment Analysis</a></li>








<li><a href="#lncRNA Target Prediction"><i class="icon-angle-right"></i>lncRNA Target Prediction</a></li>
<li><a href="#Differential lncRNA analysis"><i class="icon-angle-right"></i>Differential lncRNA analysis</a></li>
<li><a href="#Enrichment analysis of differential lncRNA target genes"><i class="icon-angle-right"></i>Enrichment analysis of differential lncRNA target genes</a></li>



<li><a href="#Alternative Splicing Analysis"><i class="icon-angle-right"></i>Alternative Splicing Analysis</a></li>



<li><a href="#Mutation"><i class="icon-angle-right"></i>Mutation</a></li>















</ul>
</li>


<li><a href="#Appendix">Appendix</a></li>
<li><a href="#References">References</a></li>
< </ul> 
 -->
  <div id="toc" class="tocify" >
  </div>
<div id="main">

<!-- --------------------------------------项目背景及样品信息---------------------------------------  -->
<!-- --------------------------------------项目背景及样品信息---------------------------------------  -->

 <div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

                <table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

                        style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

                        <tr height="86%">

                                <td width='1%'></td>

                                <td><img src="images/logo2.png"></td>

                                <td>

                                        <div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>

                                </td>

                                <td width="20%"></td>

                                <td width="15%"></td>

                                <td width="25%"

                                        style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">

                                        

                                        <br><a href="mailto:op-rna@novogene.com" target="black"

                                                style="text-decoration:none;color:black;font-size:14px;font-weight:50;">

                                                <a></td>

                                <td width="1%"></td>

                        </tr>

                </table>

        </div>
<!--
<p class="head">
<hr/>
</p>
-->
<div id="page">
    
    
    <h1 align="center"><b class="bold">lncRNA-seq Standard Analysis Report</b></h1>
    
    </br>
    
    

    <table class="gy" style="text-align:left">
      <tbody>
        <tr>
          <td>Contract ID</td>
          <td style="text-align:left">HXXXXX</td>
        </tr>
        <tr>
          <td>Contract Name</td>
          <td style="text-align:left">XXXXXX</td>
        </tr>
        <tr>
          <td>Batch ID</td>
          <td style="text-align:left">PXXXXXXXX</td>
        </tr>
        <tr>
          <td>Reference Genome and Version</td>
          <td style="text-align:left">Homo_sapiens_Ensemble_94</td>
        </tr>
        <tr>
          <td>Report Time</td>
          <td style="text-align:left">20250117135745</td>
        </tr>
        <tr>
          <td>Reminder</td>
          <td style="text-align:left">Partial results are presented in this report, while full results will be delivered in data release.<br/>
          Hyperlink of results in this report will be only valid in data release, after statement confirmation. </td>
        </tr>
      </tbody>
    </table>

	</br>
</div>

<!----------------------------------------结果汇总----------------------------------------->
<div id="page">
	<p class="head">
	<hr/>
	</p>
	
    <h2>Summary of important results</h2>
    
    <p class="paragraph">1.The number of known lncRNA and predicted new lncRNA identified by the project:</p>
    <p class="center">
    <div style="overflow-x: auto">
    	<table class="gy">
        	
          	
          		<tr><th>Types</th><th>Total</th></tr>
          	
       
          	
          		<tr><td>known</td><td>103347</td></tr>
          	
       
          	
          		<tr><td>novel</td><td>2368</td></tr>
          	
       
        </table>
     </div>
     </p>
  


  
      <p class="paragraph">2.Differentially expressed lncRNA results:</p>
      <p class="center">
      <div style="overflow-x: auto">
      	 <table class="gy">
      	   	
      	   	
      	   		<tr><th>compare</th><th>all</th><th>up</th><th>down</th><th>threshold</th></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>1657</td><td>539</td><td>1118</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group2vssample1</td><td>1974</td><td>218</td><td>1756</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>sample1vssample5</td><td>10093</td><td>5016</td><td>5077</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>1657</td><td>539</td><td>1118</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>1657</td><td>539</td><td>1118</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>1657</td><td>539</td><td>1118</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	 </table>
      </div>
      </p>

      <p class="paragraph">3.Differentially expressed mRNA gene level results:</p>
      <p class="center">
      <div style="overflow-x: auto">
      	 <table class="gy">
      	   	
      	   	
      	   		<tr><th>compare</th><th>all</th><th>up</th><th>down</th><th>threshold</th></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>2364</td><td>1008</td><td>1356</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group2vssample1</td><td>5970</td><td>3516</td><td>2454</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>sample1vssample5</td><td>7171</td><td>2692</td><td>4479</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>2364</td><td>1008</td><td>1356</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>2364</td><td>1008</td><td>1356</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	   	
      	   		<tr><td>group1vsgroup2</td><td>2364</td><td>1008</td><td>1356</td><td>edgeR pvalue<=0.05 |log2FoldChange|>=0.0</td></tr>
      	   	
      	 
      	 </table>
      </div>
      </p>
      
	  
  


  
  
	</br>
</div>

<div id="page">
	<p class="head">
	<hr/>
	</p>
	<h2 id="goto_sampleinfo1">1 Introduction</h2>
    
	<p class="paragraph">Next Generation Sequencing (NGS) has become increasingly popular on gene expression studies, by quantifying the transcript read density in the whole transcriptome. Long non-coding RNA (lncRNA) is defined as transcripts longer than 200 nucleotides, with non-coding-protein sequences.</p>
    <p class="paragraph">RNA sequencing via Illumina platforms, based on mechanism of SBS (sequencing by synthesis), has a large range of benefits on high throughput and high accuracy out of low sample requirements. This technical method can be a powerful tool for functional roles in diverse biological processes and human diseases.</p>
    <p class="paragraph">lncRNA sequencing projects are carried out as follows:</p>
    
    
    <center>
      <img class="Project_workflow" src="images/Project_workflow.png" />
      <p class="paragraph">Figure 1 Project workflow</p>
    </center>	
	</br>

</div>

<!-- --------------------------------------实验流程--------------------------------------- -->
<div id="page">

	<p class="head">
	<hr/>
	</p>
    <h2 id="goto_experiment1">2 Library Construction and Sequencing</h2>
    <h3 id="background">2.1 Sample Quality Control</h3>
    <p class="paragraph">Please refer to Novogene’s sample QC report. </p>
    </br>
    <h3 id="background">2.2 Library Construction, Quality Control and Sequencing</h3>
	
  
  <p class="paragraph">Total RNA was used as input material for the RNA sample preparations. Sequencing libraries were generated using RNA Library Prep Kit for Illumina following manufacturer’s recommendations and index codes were added to attribute sequences to each sample. Briefly, mRNA was purified from total RNA using probes to remove rRNA. Fragmentation was carried out using divalent cations under elevated temperature in First Strand Synthesis Reaction Buffer(5X). First strand cDNA was synthesized using random hexamer primer and M-MuLV Reverse Transcriptase (RNaseH). Second strand cDNA synthesis was subsequently performed using DNA Polymerase I and RNase H. Remaining overhangs were converted into blunt ends via exonuclease/polymerase activities. After adenylation of 3’ ends of DNA fragments, Adaptor with hairpin loop structure were ligated to prepare for hybridization. In order to select cDNA fragments of preferentially 370~420 bp in length, the library fragments were purified with AMPure XP system (Beverly, USA). Then 3 µL USER Enzyme was used with size-selected, adaptor-ligated cDNA at 37°C for 15 min followed by 5 min at 95 °C before PCR. Then PCR was performed with Phusion High-Fidelity DNA polymerase, Universal PCR primers and Index (X) Primer. At last, PCR products were purified (AMPure XP system) library quality was assessed on the Agilent 5400 system（Agilent，USA） and quantified by QPCR (1.5 nM).</p>
  <p class="paragraph">The Qualified libraries were pooled and sequenced on Illumina platforms with PE150 strategy, according to effective library concentration and data amount required.</p>
  <p class="center"><img class="normal_library_lncRNA" style="width:17%;height:50vw;text-align:center" src="images/pipline_2.png" /></p>

	
   
    <center> 
    <p class="paragraph">Figure 2 Library construction workflow</p>
    </center>
</div>


<!-- --------------------------------------实验流程---------------------------------------
<div id="page">

	<p class="head">
	<hr/>
	</p>

	<h2 id="goto_experiment1">二、建库测序流程</h2>
	<p class="paragraph">从RNA样品提取到最终数据获得，样品检测、建库、测序等每一环节都会直接影响数据的数量和质量，从而影响后续信息分析的结果。为从源头保证测序数据准确可靠，诺禾致源承诺在数据的所有生产环节都严格把关，从根源上确保高质量数据的产出。建库测序的流程图如下：</p>

	
	
	
	
	<p class="center"><img class="normal_library_lncRNA" style="width:20%;height:32vw;text-align:center" src="images/library_lncRNA.png" /></p>
	

	
	<p class="name_fig">图1 建库测序流程图</p>
	<br />


</div> -->

<!-----------------------1 RNA提取与检测------------------------


<div id="page">



	<a name="RNA样品检测"></a>
	<h3 id="experiment1">1 RNA样品检测</h3>
	<p class="paragraph">从组织或细胞中提取RNA，随后对RNA样品进行严格质控，质控标准主要包括以下3个方面：</p>
	<p class="paragraph">(1) 琼脂糖凝胶电泳：分析样品RNA完整性及是否存在DNA污染；</p>
	<p class="paragraph">(2) Nanodrop：进行初步定量，检测RNA浓度及纯度（OD260/280及OD260/230）；</p>
	<p class="paragraph">(3) Agilent 2100 bioanalyzer：精确检测RNA完整性。 </p>


	<br />


</div>-->

<!----------------------2 文库构建------------------------


<div id="page">
        


	<a name="文库构建"></a>
	<h3 id="experiment2">2 文库构建</h3>


	

	
	<p class="paragraph">诺禾致源采用去除核糖体RNA的方法（circRNA建库增加去线性RNA过程）构建链特异性文库(Parkhomchuk D et al., 2009)。
	首先，从总RNA中去除核糖体RNA，随后将RNA打断成250~300 bp的短片段。以片段化的RNA为模板，随机寡核苷酸为引物合成cDNA第一条链，随后以dNTPs（dUTP、dATP、dGTP和dCTP）为原料合成cDNA第二条链。
	
	纯化后的双链cDNA经过末端修复、加A尾并连接测序接头<sup>[注]</sup>，用AMPure XP beads筛选350~400 bp左右的cDNA。使用USER酶降解含U的cDNA第二链，最后进行PCR扩增并获得文库。链特异性文库具有诸多优势，如相同数据量下可获取更多有效信息；能获得更精准的基因定量、定位与注释信息等。建库原理如下图：</p>
	
	
	<p class="center"><img class="normal_strand_specific" src="images/strand_specific.png" /></p>
	
	
	<p class="name_fig">图2 文库构建流程</p>
	<p class="tremark">注：接头（Adapter），包括P5/P7、index和Rd1 SP/Rd2 SP。P5/P7，用于测序仪进行桥式PCR的过程；Index，标签序列，用来区分不同文库信息；Rd1 SP/Rd2 SP（sequence primer），测序引物结合区域，测序过程理论上由Rd1/Rd2向后开始进行。</p>
	
	
	<p class="paragraph">我们在建库过程中消化了核糖体RNA，之后将片段随机打断为250-300bp，通过片段长度选择，最终得到除了核糖体RNA和小片段RNA（microRNA、siRNA等）以外的所有RNA，包括lncRNA、mRNA、circRNA等。</p>
	

	
	
	

</br>

</div>-->
	
	
<!----------------------3 库检-------------------------
	
<div id="page">
       
	<a name="库检"></a>
	<h3 id="experiment3">3 库检</h3>

	<p class="paragraph">先用 Qubit 初步定量，稀释文库至 1ng/ul；再用 Agilent 2100 bioanalyzer 检测对文库的插入片段长度（insert size）进行检测，insert size 大约分布在 250~300 bp 左右符合预期。Insert size符合预期后，使用 qPCR方法对文库的有效浓度进行准确定量，文库的有效浓度为>2nM,保证了样本文库的质量</p>


	<br />
</div>-->


<!-----------------------4 上机测序------------------------
<div id="page">
	<a name="上机测序"></a>
	<h3 id="experiment4">4 上机测序</h3>

	<p class="paragraph">库检合格后，根据文库的有效浓度及数据产出需求pooling后进行Illumina PE150测序。PE150（Pair end 150 bp）指高通量双端测序，每端各测150 bp。在构建的小片段文库中，Insert cDNA，即插入片段是直接测序的单位。双端测序是将每条插入片段的两端进行测序的方法，由于插入片段的长度分布已知，双端测序时不仅可以获得片段两端的序列，也可获得这两段序列之间的长度，从而便于后续组装和比对。</p>
	<p class="paragraph">测序的基本原理是边合成边测序（Sequencing by Synthesis）。在测序的flow cell中加入四种荧光标记的dNTP、DNA聚合酶以及接头引物进行扩增，在每一个测序簇延伸互补链时，每加入一个被荧光标记的dNTP就能释放出相对应的荧光，测序仪通过捕获荧光信号，并通过计算机软件将光信号转化为测序峰，从而获得待测序片段的序列信息。</p>
	<br />-->

<!-- --------------------------------- 信息分析流程 -------------------------------- -->
<!-- --------------------------------- 信息分析流程 -------------------------------- -->

<div id="page">
<hr />
	<h2 id="pipeline1">3 Bioinformatics Analysis Pipeline</h2>
	
		
   <p class="paragraph">This is the workflow for lncRNA sequencing data of standard bioinformatic analysis with a reference genome, as follows:</p>

	<p class="center">
        
        <img class="normal_pipline_lncRNA" src="images/pipline_lncRNA_hw.png" /></p>
        
    <center>
    <p class="paragraph">Figure 3 Bioinformatic analysis workflow</p>
    </center>
    <p class="paragraph"><strong>Method details involved in the project are available at <a href="LncRNA.method_hw.pdf" title="点击打开" >methods</a></strong></p>
	


	
	</br>
</div>

<!------------------------------四 标准分析结果-------------------------->

<!---------------------------四 标准分析结果---------------------->
	
<div id="page">
	<p class="head">
	<hr/>
	</p>

	<h2 id="resultall">4 Analysis Results</h2>
	
	
	
		
	<!-- <p class="paragraph">RNA-seq标准分析过程主要包括质控，比对，筛选鉴定，定量，差异显著性分析，来源基因分析和功能富集等环节。RNA-seq分析的核心是基因表达差异的显著性，使用统计学方法比较两个或多个条件下的基因表达差异（例如，处理与对照），从中找出与条件相关的差异基因，然后进一步分析这些差异基因的生物学意义。</p> -->


</br>	

<!----------------------------1.1 原始数据说明------------------------>	
<div id="page">
	<h3 id="31">4.1 Data Quality Control</h3>

	<h4 id="result11">4.1.1 Raw Data</h4>
	<p class="paragraph">Original image data file from high-throughput sequencing (like Illumina) is transformed into sequenced reads (called Raw Data or Raw Reads) by CASAVA base recognition (Base Calling). Raw data are stored in FASTQ(fq) format files, which contain sequences of reads and corresponding base quality. Each read has four descriptive lines in as follow:</p>

	<p class="center">
	<img class="normal" src="images/rawdata.png" /></p>
    <p class="step">(1) Line 1: the at sign (@) followed by sequence identifiers and optional description info (such as
      FASTA title line).<br />
      (2) Line 2: base sequences (raw read, A, G, C, and T).<br />
      (3) Line 3: the plus sign (+) optionally followed by the same Illumina sequence identifiers and description
      information as Line 1.<br />
      (4) Line 4: the quality values for each base, corresponding to the data in Line 2.</p>


    <p class="paragraph">Illumina Sequence Identifier:</p>
    <p class="center">
    <table class="gy">
    <tr>
        <th>ID Symbol</th>
        <th>meaning</th>
      </tr>
      <tr>
        <td>HWI-ST1276</td>
        <td>Instrument – unique identifier of the sequencer </td>
      </tr>
      <tr>
        <td>71</td>
        <td>run number – Run number on instrument</td>
      </tr>
      <tr>
        <td>C1162ACXX</td>
        <td>FlowCell ID – ID of flowcell</td>
      </tr>
      <tr>
        <td>1</td>
        <td>LaneNumber – positive integer</td>
      </tr>
      <tr>
        <td>1101</td>
        <td>TileNumber – positive integer</td>
      </tr>
      <tr>
        <td>1208</td>
        <td>X – x coordinate of the spot. Integer which can be negative</td>
      </tr>
      <tr>
        <td>2458</td>
        <td>Y – y coordinate of the spot. Integer which can be negative</td>
      </tr>
      <tr>
        <td>1</td>
        <td>ReadNumber - 1 for single reads; 1 or 2 for paired ends</td>
      </tr>
      <tr>
        <td>N</td>
        <td>whether it is filtered - NB：Y if the read is filtered out, not in the delivered fastq file, N otherwise</td>
      </tr>
      <tr>
        <td>0</td>
        <td>control number - 0 when none of the control bits are on, otherwise it is an even number</td>
      </tr>
      <tr>
        <td>CGATGT</td>
        <td>Illumina index sequences</td>
      </tr>
    </table>
    </p>
    
    <p class="paragraph">The details of Sequencing identifier of Illumina are as follows:</p>
    <p class="step">(1) HWI-ST1276:71 HWI-ST1276, Instrument - unique identifier of the sequencer; 71, run number - Run
      number on instrument.<br />
      (2) C1162ACXX:1:1101:1208:2458 means the coordinate of read on C1162ACXX (Flowcell ID) flowcell, line 1, 1101 tile
      is(x=1208, y=2458).<br />
      (3) 1:N:0:CGATGT the first number is 1 or 2, 1 means single reads or the first read of paired ends, 2 means the
      second of paired ends; the second letter means whether reads is adjusted(Y means yes, N means no); the third
      number represent the number of Control Bits in sequence; six bases on the fourth place is Illumina index sequence.
    </p>


	<br />
	<br />


<!------------------------------1.2 Data Filtering------------------------------>
<p class="head">
<hr/>
</p>
    
	<h4 id="result12">4.1.2 Data Filtering</h4>
	
	
	<p class="paragraph">The Sequenced Reads/raw reads often contain low quality reads or reads with adaptors. As we can see in Figure 4.1, which will affect the further analysis results. In order to avoid this, it's necessary to filter the raw reads under conditions to get the clean reads. Raw reads filtering conditions are as follows:</p>
	<p class="paragraph">(1) Remove reads containing adaptors;</p>
	<p class="paragraph">(2) Remove reads containing N > 10% (N represents base that could not be determined);</p>
	<p class="paragraph">(3) The Qscore (Quality value) of over 50% bases of the read is <= 5.</p>
</p>

<!--<p class="paragraph">Click to open the result directory, and the corresponding file can be viewed directly when the report and the result files are stored in the same folder.<b>Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/1.QC/3.Filter" title="点击打开" >QC_Filter</a> </b></p>-->

</br>

   <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_classfi_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_classfi_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample1" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample1.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample2" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample2.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample3" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample3" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample3.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample4" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample4" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample4.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample5" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample5.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample6" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample6" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample6.classfication.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_classfi_Content_pane_sample7" role="tabpanel"
            aria-labelledby="gallery_figure_classfi_Content_list_sample7" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Classfication/sample7.classfication.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_classfi_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_classfi_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_classfi_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample1"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample1" role="tab"
            aria-controls="sample1">sample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample2"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample2" role="tab"
            aria-controls="sample2">sample2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample3"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample3" role="tab"
            aria-controls="sample3">sample3</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample4"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample4" role="tab"
            aria-controls="sample4">sample4</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample5"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample5" role="tab"
            aria-controls="sample5">sample5</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample6"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample6" role="tab"
            aria-controls="sample6">sample6</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_classfi_Content_list_sample7"
            data-toggle="list" href="#gallery_figure_classfi_Content_pane_sample7" role="tab"
            aria-controls="sample7">sample7</a>
          
        </div>
      </div>
    </div>
    </p>

	<p class="name_fig">Figure 4.1 Composition of raw data</p>
    <p class="premark">Different color for different components</p>
    <p class="premark">(1)Adaptor related: (reads containing adapter) / (total raw reads).</p>
    <p class="premark">(2)Containing N: (reads with more than 10% N) / (total raw reads).</p>
    <p class="premark">(3)Low quality: (reads of low quality) / (total raw reads).</p>
    <p class="premark">(4)Clean reads: (clean reads) / (total raw reads).</p>
    <p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/1.QC/3.Filter" title="点击打开" >QC_Filter</a> </p>
	<br />
	<br />

	
<!---------------------------1.3 测序错误率分布-------------------------->

    
	<h4 id="result13">4.1.3 Error Rate</h4>
	
	
     <p class="paragraph">
      The sequencing error rate of base is calculated from Phred score (Q<sub>phred</sub>) based on Equation 1 (equation 1: Q<sub>phred</sub> = -10log<sub>10</sub><sup>e</sup>), while the Phred score is predicted from a probabilitymodel during base calling. Their relations are show below:</p>
    <p class="paragraph">Relationship of base calling and Phred scores in Illumina Casava.</p>
    <table class="gy">
      <tr>
        <th>Phred score</th>
        <th>Error rate</th>
        <th>Accuracy</th>
        <th>Q-sorce</th>
      </tr>
      <tr>
        <td>10</td>
        <td>1/10</td>
        <td>90%</td>
        <td>Q10</td>
      </tr>
      <tr>
        <td>20</td>
        <td>1/100</td>
        <td>99%</td>
        <td>Q20</td>
      </tr>
      <tr>
        <td>30</td>
        <td>1/1000</td>
        <td>99.9%</td>
        <td>Q30</td>
      </tr>
      <tr>
        <td>40</td>
        <td>1/10000</td>
        <td>99.99%</td>
        <td>Q40</td>
      </tr>
    </table>
    <p class="paragraph">Sequencing error rate is related to the base quality and is affected by many factors such as the sequencer, the sequencing reagents and the samples. For RNA-Seq technology, the distribution of sequencing error rate has two features: </p>
    <p class="step">(1) The error rate will increase when the length of sequenced reads increases. This is due to the consumption of chemical reagents during sequencing, which is common for all Illumina high-throughput sequencing platforms(Hardwick SA et al., 2016).</p>
    <p class="step">(2) The first six bases will suffer from higher sequencing error rate, and this is also the length of random primers which are required for reverse transcription during RNA-seq library construction. Therefore, it is speculated that the higher error rate in the first six bases is due to the incomplete bindings between random primers and RNA templates (Hardwick SA et al., 2016).</p>
	
    

 <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_error_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_error_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample1" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample1.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample2" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample2.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample3" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample3" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample3.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample4" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample4" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample4.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample5" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample5.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample6" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample6" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample6.Error.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_error_Content_pane_sample7" role="tabpanel"
            aria-labelledby="gallery_figure_error_Content_list_sample7" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/Errorrate/sample7.Error.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_error_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_error_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_error_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample1"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample1" role="tab"
            aria-controls="sample1">sample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample2"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample2" role="tab"
            aria-controls="sample2">sample2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample3"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample3" role="tab"
            aria-controls="sample3">sample3</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample4"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample4" role="tab"
            aria-controls="sample4">sample4</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample5"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample5" role="tab"
            aria-controls="sample5">sample5</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample6"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample6" role="tab"
            aria-controls="sample6">sample6</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_error_Content_list_sample7"
            data-toggle="list" href="#gallery_figure_error_Content_pane_sample7" role="tab"
            aria-controls="sample7">sample7</a>
          
        </div>
      </div>
    </div>
    </p>

	<p class="name_fig">Figure 4.2 Distribution of sequencing error rate</p>
	<p class="cenmark">The horizontal axis represents the position of reads, and the vertical axis represents the error rate. </p>
    <p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/1.QC/1.Error" title="点击打开" >QC_Error</a> </p>
	<br />

<!---------------------------1.4 GC含量分布------------------------------>	
	
	
    <h4 id="result14">4.1.4 GC-content</h4>
    
    

	
		
	<p class="paragraph">For RNA-seq, in view of the random interrupted and the principle of G/C, A/T content are respectively equal, G and C, A and T should be equal and content is stable on the whole through the whole sequencing process for non-stranded library (If the library is strand-specific, it may occur AT separation or GC separation). A large variation of sequencing error in the first 6-7 bases is allowed considering the usage of random primer in library construction. It's normal that the first few bases have certain preference in existing high-throughput sequencing technology, for the usage of 6bp random primer in library construction. </p>
	
		
	<!-- <p class="paragraph">核苷酸序列中鸟嘌呤（G）和胞嘧啶（C）所占的比例称为GC含量。GC含量在物种间存在一定特异性，但由于反转录过程中所使用的6bp随机引物，会引起前几位碱基在核苷酸组成上有一定偏好性，产生正常波动，随后则趋于稳定。对于链特异性建库而言，由于只保留了单链信息，可能会出现AT分离或GC分离现象。如图1.3所示：</p> -->

</p>
    
    

</br>
	
    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_GC_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_GC_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample1" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample1.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample2" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample2.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample3" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample3" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample3.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample4" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample4" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample4.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample5" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample5.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample6" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample6" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample6.GC.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_GC_Content_pane_sample7" role="tabpanel"
            aria-labelledby="gallery_figure_GC_Content_list_sample7" onclick="trans(this.id)"><img
              class="img-gallery" src="files/QC/GC/sample7.GC.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_GC_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_GC_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_GC_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample1"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample1" role="tab"
            aria-controls="sample1">sample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample2"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample2" role="tab"
            aria-controls="sample2">sample2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample3"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample3" role="tab"
            aria-controls="sample3">sample3</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample4"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample4" role="tab"
            aria-controls="sample4">sample4</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample5"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample5" role="tab"
            aria-controls="sample5">sample5</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample6"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample6" role="tab"
            aria-controls="sample6">sample6</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_sample7"
            data-toggle="list" href="#gallery_figure_GC_Content_pane_sample7" role="tab"
            aria-controls="sample7">sample7</a>
          
        </div>
      </div>
    </div>
    </p>
    <p class="name_fig">Figure 4.3 GC-content</p>
    <p class="cenmark">Horizontal axis for reads position, vertical axis for single base percentage. Different color for different base type</p>
    <p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/1.QC/2.GC" title="点击打开" >QC_GC</a></p>
    <br />
    <br />
<!-----------------------1.5 数据质量汇总---------------------->

	
	
	<h4 id="result15">4.1.5 Data Quality Control Summary</h4>
	
	
	<p class="paragraph">Clean reads for follow-up analysis were obtained after raw data filtering, sequencing error rate checking and GC content distribution checking, and the data were summarized as shown in table 4.1.</p>
	
</p>
    
	
</br>

	<p class="name_fig">Table 4.1　Summary of data production</p>

	<p class="center">
	<!--<div id="tb" style="width:1100px;height:270px;overflow-y:scroll;overflow-x:hidden;" >-->
	<div id="tb" style="overflow-y:scroll;overflow-x:hidden;" >
		<table class="tb1">
			
			
				<tr><th>sample</th><th>library</th><th>raw_reads</th><th>raw_bases</th><th>clean_reads</th><th>clean_bases</th><th>error_rate</th><th>Q20</th><th>Q30</th><th>GC_pct</th></tr>
			
		  
			
				<tr><td>sample1</td><td>BRRL220000060-1A</td><td>1e+07</td><td>1.5G</td><td>9735746</td><td>1.46G</td><td>0.02</td><td>97.97</td><td>94.31</td><td>49.84</td></tr>
			
		  
			
				<tr><td>sample2</td><td>BRRL220000061-1A</td><td>1e+07</td><td>1.5G</td><td>9695364</td><td>1.45G</td><td>0.03</td><td>97.85</td><td>94.06</td><td>46.58</td></tr>
			
		  
			
				<tr><td>sample3</td><td>BRRL220000062-1A</td><td>1e+07</td><td>1.5G</td><td>9428286</td><td>1.41G</td><td>0.02</td><td>98.46</td><td>95.45</td><td>53.83</td></tr>
			
		  
			
				<tr><td>sample4</td><td>BRRL220000063-1A</td><td>1e+07</td><td>1.5G</td><td>9800400</td><td>1.47G</td><td>0.02</td><td>98.19</td><td>94.8</td><td>49.14</td></tr>
			
		  
			
				<tr><td>sample5</td><td>BRRL220000064-1A</td><td>1e+07</td><td>1.5G</td><td>9772582</td><td>1.47G</td><td>0.02</td><td>98.2</td><td>94.99</td><td>51.36</td></tr>
			
		  
			
				<tr><td>sample6</td><td>BRRL220000065-1A</td><td>1e+07</td><td>1.5G</td><td>9793882</td><td>1.47G</td><td>0.02</td><td>98.2</td><td>94.85</td><td>50.43</td></tr>
			
		  
			
				<tr><td>sample7</td><td>BRRL220000066-1A</td><td>1e+07</td><td>1.5G</td><td>9808088</td><td>1.47G</td><td>0.02</td><td>98.19</td><td>94.94</td><td>50.35</td></tr>
			
		  
		</table>
	</div>		
	</p>
	<!--<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">-->
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1) sample: SampleID.<br />
    (2) library: The number of libraries.<br />
	(3) raw_reads: Reads count from the raw data, four rows as a unit, with statistics of reads count for every sequencing.<br />
    (4) raw_bases: Base number of raw data. (number of raw reads) * (sequence length), converting unit to G.<br />
	(5) clean_reads: Base number of raw data after filtering. (number of clean reads) * (sequence length), converting unit to G.<br />
	(6) clean_bases: (clean base=clean reads*150bp) number multiply read length, saved in G unit.<br />

        (7) error_rate：error_rate: Average sequencing error rate, which is calculated by Qphred=-10log10(e).<br />
        (8) Q20: The percentage of the bases whose Q Phred values is greater than 20. (Number of bases with Q Phred value > 20) / (Number of total bases) *100.<br />
        (9) Q30: The percentage of the bases whose Q Phred values is greater than 30. (Number of bases with Q Phred value > 30) / (Number of total bases) *100.<br />
        (10) GC_pct: The percentage of G&C base numbers of total bases.(G&C base number) / (Total base number)*100.<br />

</ul>
</div>
  <p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/1.QC/4.Stat" title="点击打开" target="_blank">QC_Stat</a></p>
  <br />
<!-------------------------------Data quality assessment Q&A: ------------------------------>
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:The error rate will increase with the higher length of sequencing, what is the acceptable range of error rate?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A: Novogene set a high standard of sequencing quality. Generally, the error rate of a single base should be lower than 1%. In some special cases, the maximum error rate of a single base should not be greater than 6%.</td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:What is the difference between the align software of RNA data and of DNA data?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:Due to the existence of splicing during transcription, a large portion of the measured reads spans different exons, and all RNA data alignment software will support the splice alignment of spliced reads.</td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:Does the full length of reads will be used for mapping, or does the head or tail of reads will be processed before mapping?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:In our experiment, we use the standard RNA-seq kit, whose indexes are in the middle of the adapters and the sequencing will be executed for the sequences with indexes. In this way, the sequences of read 1 and read 2 are from the samples and there is no need to process the sequences in mapping. Even if the adapters are in sequences
  or the quality is low, the read will be removed.</td></tr>

</table>
</div>

	<br />	

<!-------------------------------2 比对------------------------------>



<div id="page" >

	<h3 id="result2">4.2 Mapping To Reference Genome</h3>
    <p class="paragraph">Perform alignments with HISAT2 to the reference. HISAT2 uses a graph-based alignment and has succeeded HISAT and TOPHAT2. HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads. Besides one global GFM index, HISAT2 also includes a large set of small GFM indexes that collectively cover the whole genome. These small indexes (local indexes), combined with multiple alignment strategies, enabled effective alignment of RNA-seq reads, particularly, reads spanning multiple exons(Kim D et al., 2015).</p> 

    <p class="center"><img class="normal_pipline_lncRNA" src="images/hisat2.png" /></p>

</div>


<!------------------------------2.1 比对结果说明-------------------------->

<!--------------------------2.1 比对情况统计-------------------------->
	<h4 id="result21">4.2.1 Overview of Mapping Status</h4>
	

	<p class="center">
	
</br>	
	
	<p class="name_fig">Table 4.2 Summary of reads mapped to reference genome</p>

<p class="center">
<div id="tb" style="overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1" >
			
			
				<tr><th>sample</th><th>total_reads</th><th>total_map</th><th>unique_map</th><th>multi_map</th><th>read1_map</th><th>read2_map</th><th>positive_map</th><th>negative_map</th><th>splice_map</th><th>unsplice_map</th><th>proper_map</th></tr>
			
		  
			
				<tr><td>sample1</td><td>9735746</td><td>9157887(94.06%)</td><td>8408887(86.37%)</td><td>749000(7.69%)</td><td>4222834(43.37%)</td><td>4186053(43.0%)</td><td>4201575(43.16%)</td><td>4207312(43.22%)</td><td>1433921(14.73%)</td><td>6974966(71.64%)</td><td>8104006(83.24%)</td></tr>
			
		  
			
				<tr><td>sample2</td><td>9695364</td><td>9192411(94.81%)</td><td>8604046(88.74%)</td><td>588365(6.07%)</td><td>4317309(44.53%)</td><td>4286737(44.21%)</td><td>4298208(44.33%)</td><td>4305838(44.41%)</td><td>1296335(13.37%)</td><td>7307711(75.37%)</td><td>8324436(85.86%)</td></tr>
			
		  
			
				<tr><td>sample3</td><td>9428286</td><td>8567131(90.87%)</td><td>6473692(68.66%)</td><td>2093439(22.2%)</td><td>3247570(34.44%)</td><td>3226122(34.22%)</td><td>3239659(34.36%)</td><td>3234033(34.3%)</td><td>2687330(28.5%)</td><td>3786362(40.16%)</td><td>6295086(66.77%)</td></tr>
			
		  
			
				<tr><td>sample4</td><td>9800400</td><td>9416677(96.08%)</td><td>8160808(83.27%)</td><td>1255869(12.81%)</td><td>4093768(41.77%)</td><td>4067040(41.5%)</td><td>4083953(41.67%)</td><td>4076855(41.6%)</td><td>1671599(17.06%)</td><td>6489209(66.21%)</td><td>7896094(80.57%)</td></tr>
			
		  
			
				<tr><td>sample5</td><td>9772582</td><td>9276212(94.92%)</td><td>7629244(78.07%)</td><td>1646968(16.85%)</td><td>3825719(39.15%)</td><td>3803525(38.92%)</td><td>3813653(39.02%)</td><td>3815591(39.04%)</td><td>2248465(23.01%)</td><td>5380779(55.06%)</td><td>7428742(76.02%)</td></tr>
			
		  
			
				<tr><td>sample6</td><td>9793882</td><td>9342305(95.39%)</td><td>7746488(79.1%)</td><td>1595817(16.29%)</td><td>3886566(39.68%)</td><td>3859922(39.41%)</td><td>3874855(39.56%)</td><td>3871633(39.53%)</td><td>2252137(23.0%)</td><td>5494351(56.1%)</td><td>7525714(76.84%)</td></tr>
			
		  
			
				<tr><td>sample7</td><td>9808088</td><td>9338290(95.21%)</td><td>7837068(79.9%)</td><td>1501222(15.31%)</td><td>3929093(40.06%)</td><td>3907975(39.84%)</td><td>3918117(39.95%)</td><td>3918951(39.96%)</td><td>2093990(21.35%)</td><td>5743078(58.55%)</td><td>7615658(77.65%)</td></tr>
			
		   
		</table>
	</div>	
</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1) sample: SampleID.<br />
	(2) total_reads: Total clean reads used for analysis.<br />
	(3) total_map: Number and percentage of reads aligned to the genome, the ratio should higher than 70%, total mapping rate: (mapped reads)/(total reads)*100.<br />
	(4) unique_map: Number and percentage of reads aligned to the unique position of the reference genome (for subsequent quantitative data analysis), unique mapping rate: (uniquely mapped reads)/(total reads)*100.<br />
	(5) Multi_map: number and percentage of reads aligned to multiple locations in the reference genome, multiple mapping rate: (multiple mapped reads)/(total reads)*100.<br />
	(6) read1_map: Number and percentage of read1 aligned to the reference genome.<br />
	(7) read2_map: Number and percentage of read2 aligned to the reference genome.<br />
	(8) positive_map: Number and percentage of reads aligned to the positive chain of the reference genome.<br />
	(9) negative_map: Number and percentage of reads aligned to the negative chain of the reference genome.<br />
	(10) splice_map: Number of spliced reads on the genome and its percentage.splice_map: Number of spliced reads on the genome and its percentage.  <br />
	(11) unsplice_map: Number of complete reads aligned to genome and its percentage.<br />
	(12) proper_map: Number of paired read1 and read2 aligned to the genome and its percentage.<br />

</ul>
</div>
    <p class="paragraph">Result Directory:<a href="../../Result_PXXXXXXXX_Homo_sapiens/2.Mapping/2.Stat" title="点击打开" >Mapping_Stat</a></p>
	<br />

<!------------------------------2.2 基因组区域分布------------------>



<div id="page">

	<h4 id="result22">4.2.2 Reads Distribution in Reference Genome</h4>

    <p class="paragraph">Mapped regions can be classified as exons, introns, or intergenic regions. Exon-mapped reads should be the most abundant type of reads when the reference genome is well-annotated. Intron-reads may be derived from pre-mRNA contamination or intron-retention from alternative splicing. Reads mapped to intergenic regions may be derived from ncRNA or a small fraction of DNA contamination (Breschi A et al., 2017). The graphical representations are shown as below: 

<!-------20200328添加, 除样本名修改外, 其余框架内容不做修改-->
</p>
	
    

</br>	
 <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_map_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_map_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample1" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample1.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample2" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample2.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample3" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample3" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample3.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample4" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample4" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample4.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample5" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample5.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample6" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample6" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample6.region.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_map_Content_pane_sample7" role="tabpanel"
            aria-labelledby="gallery_figure_map_Content_list_sample7" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Mapping/Mapregion/sample7.region.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_map_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_map_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_map_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample1"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample1" role="tab"
            aria-controls="sample1">sample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample2"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample2" role="tab"
            aria-controls="sample2">sample2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample3"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample3" role="tab"
            aria-controls="sample3">sample3</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample4"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample4" role="tab"
            aria-controls="sample4">sample4</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample5"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample5" role="tab"
            aria-controls="sample5">sample5</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample6"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample6" role="tab"
            aria-controls="sample6">sample6</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_map_Content_list_sample7"
            data-toggle="list" href="#gallery_figure_map_Content_pane_sample7" role="tab"
            aria-controls="sample7">sample7</a>
          
        </div>
      </div>
    </div>
    </p>

	<p class="name_fig">Figure 4.4 Sequencing reads in the genomic region </p>
    <p class="premark">(1) exon: The number of reads aligned to exon regions of the genome and its proportion in clean reads.</p>
    <p class="premark">(2) Intron: The number of reads aligned to intron regions of the genome and its proportion in clean reads.</p>
    <p class="premark">(3) Intergenic: The number of reads aligned to intergenic regions of the genome and its proportion in clean reads.</p>
    <p class="paragraph">Result Directory:<a href="../../Result_PXXXXXXXX_Homo_sapiens/2.Mapping/1.Region" title="点击打开" >Mapping_Region</a></p>
	<br />
</div>

<!------------------------------2.3 比对可视化------------------>



<div id="page">

    <h4 id="result23">4.2.3 Visualization of Mapping Results</h4>
    <p class="paragraph">Files provided in BAM format--a standard file format that contains mapping results–indicating the information of the corresponding referenced genome and gene annotations for some species. The Integrative Genomics Viewer (IGV) is a recommended software for visualizing data from BAM files.</p>
    <p class="step">(1) It displays the positions of single or multiple reads in the reference genome, as well as read
      distribution between annotated exons, introns or intergenic regions, both in adjustable scale; <br />
      (2) It displays the read abundance of different regions to demonstrate their expression levels, in adjustable
      scale;
      <br />
      (3) It provides annotation information for both genes and splicing isoforms; <br />
      (4) It provides other related annotation information; <br />
      (5) It displays annotations downloaded from remote servers and/or imported from local machines;<br />
      (6) IGV browser usage can refer to our provided documentation <a href="IGVQuickStart.pdf" title="点击打开" >IGVQuickStart</a></p>

<p class="center"><img class="igv" src='images/igv.png'></p>
<p class="name_fig">Figure 4.5  IGV Browser Comparison Visualization of Results </p>

</div>

<!-------------------------------2.4  比对视频链接------------------------------>


<!---------------------------------------转录本拼接（lncRNA）---------------------------------------------->

<!---------------------------------------转录本拼接（Stringtie）---------------------------------------------->



<div id="page">
	
	<h3 id="result6">4.3 Transcript Assembly</h3>
	
    <!-- <h3 id="result6"><a name="转录本拼接">3 转录本拼接</a></h3> -->
    
	<h4 id="result61">4.3.1 Transcript Assembly</h4>
     <p class="paragraph">Stringtie has specific parameters for different libraries, allows accurate  assembling of transcripts and quantifies transcripts (Ghosh S et al., 2016) . So we used the Stringtie software to assemble reads into transcripts and quantify them based on comparisons to genomes (Pertea M et al., 2015).The workflow of StringTie assembly are shown below:</p>
      <p class="center"><img src="images/stingtie.png" width=500 /></p>
<!----------------------
    <p class="center"> <img class="normal_cufflinks-algorithm" src="images/cufflinks-algorithm.png" />
    <p class="name_fig">图3.2 Cufflinks 拼接示意图</p>
    </p>
--------------------------->
</p>
	
	<p class="center">
	
</br>	

    <p class="name_fig">Table 4.3 Result of stringtie assembly(part)
    </p>
    <p class="center">
    <div id="tb" style="overflow-y:scroll;overflow-x:scroll;">
    <table class="tb1">
    <tr class="tb1"><th class="tb1">Seqname</th><th class="tb1">Source</th><th class="tb1">Feature</th><th class="tb1">Start</th><th class="tb1">End</th><th class="tb1">Score</th><th class="tb1">Strand</th><th class="tb1">Frame</th><th class="tb1">Attributes</th></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">946635</td><td  class="tb1" style="text-align:center">946858</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.1"; transcript_id "STRG.1.1"; cov "2.678571"; FPKM "1.960246"; TPM "3.456521";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">946635</td><td  class="tb1" style="text-align:center">946858</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.1"; transcript_id "STRG.1.1"; exon_number "1"; cov "2.678571";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">954549</td><td  class="tb1" style="text-align:center">954911</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.2"; transcript_id "STRG.2.1"; cov "4.118457"; FPKM "3.013992"; TPM "5.314600";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">954549</td><td  class="tb1" style="text-align:center">954911</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.2"; transcript_id "STRG.2.1"; exon_number "1"; cov "4.118457";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">989527</td><td  class="tb1" style="text-align:center">989901</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.3"; transcript_id "STRG.3.1"; cov "8.800000"; FPKM "6.440064"; TPM "11.355825";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">989527</td><td  class="tb1" style="text-align:center">989901</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.3"; transcript_id "STRG.3.1"; exon_number "1"; cov "8.800000";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">990675</td><td  class="tb1" style="text-align:center">991753</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.4"; transcript_id "STRG.4.1"; cov "3.155700"; FPKM "2.309422"; TPM "4.072225";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">990675</td><td  class="tb1" style="text-align:center">991753</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.4"; transcript_id "STRG.4.1"; exon_number "1"; cov "3.155700";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">991859</td><td  class="tb1" style="text-align:center">992170</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.5"; transcript_id "STRG.5.1"; cov "2.878205"; FPKM "2.106344"; TPM "3.714136";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">991859</td><td  class="tb1" style="text-align:center">992170</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.5"; transcript_id "STRG.5.1"; exon_number "1"; cov "2.878205";</td></tr>
    
    </table>
    </div>
    </p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
    (1) Seqname: Chromosome ID.<br />
    (2) Source: The source of the annotation, stringTie here means that the transcript is assembled by stringTie software.<br />
    (3) Feature: The structure type of the annotation information, such as gene, transcript, exon, etc.<br />
    (4) Start: The starting coordinate of the annotation area on the chromosome.<br />
    (5) End: The end coordinate of the annotation area on the chromosome.<br />
    (6) Score: The score of the annotation information which is a description of the possibility of annotation, and the dot is indicated as empty.<br />
    (7) Strand: Annotation region positive and negative chain information on the chromosome.<br />
    (8) Frame: Valid only for annotation types of CDS, valid values are 0/1/2.<br />
    (9) Attributes:  Contains a list of attributes, mainly gene number, transcript number, etc.<br />
</ul>
  <p class="paragraph">Result directory of each sample of transcript assembly GTF:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/3.Assemble" title="点击打开" >Assemble_sample</a></p>
</div>
</br>




<!---------------------------筛选-------------------------->

<!-- 
<option>sample1</option>

<option>sample2</option>

<option>sample3</option>

<option>sample4</option>

<option>sample5</option>

<option>sample6</option>

<option>sample7</option>
 -->
<div id="page">
	<h4 id="result62">4.3.2 Transcript Identification</h4>
    <p class="paragraph">We use the Stringtie software merge parameter to merge the transcripts obtained from the assemblied samples, and remove the ones with uncertain chain direction and the length of the transcripts not exceeding 200nt, then we use gffcompare software to compare with the known database, filter out the known transcripts in the database, and finally predict the coding potential of the new transcripts, and finally get Novel_lncRNA and Novel_mRNA.</p>
	<a name="基本筛选"></a>
    <p class="paragraph">We process the file by a series of filter steps:</p> 
	<p class="tremark">
	Step1: Exon Number Filter: Filter out transcrips with low expression level and single exon transcripts with low confidence. Select transcripts with exon> = 2 (Single exon transcripts will be kept for plant data);<br />
	Step2: Transcript Length Filter: Select transcripts with length> 200bp;<br />
	Step3: Known Transcript Filter: Filter out transcripts which overlap with annotated exon region in database using gffcompare. Add these annotations to lncRNA analysis;<br />
    Step4: Encoding potential screening: whether there is encoding potential is the key to determine whether the transcript is lncRNA. In view of the transcript obtained in the previous step, we synthesized the current mainstream coding potential analysis method(CPC2(Kang YJ et al., 2017)/Pfam(Mistry J et al., 2007)/CNCI(Sun L et al., 2013)) to predict the coding potential, the intersection of transcripts with no coding potential in these software results is taken as the candidate Novel_lncRNA data set predicted by this analysis, the intersection of the transcripts with coding potential in these software results is taken as the Novel_mRNA data set.<br />
    step5: According to the naming guidelines of HGNC(The HUGO Gene Nomenclature Committee) (Wright MW, 2014) on long non-coding RNA, the Novel_lncRNA candidate was finally screened and named according to its position relationship with coding genes, and the Novel_lncRNA analyzed in this study was obtained.<br />
	</p>
	</p>

<!---------------------------转录本筛选之编码潜能筛选维恩图-------------------------->
    <h4 id="result62">4.3.2.1 Coding Potential Filtering</h4>
    <p class="paragraph">Coding potential is essential to determine if a transcript is a lncRNA. Several popular softwares are used for potential analysis, including CPC, CNCI(vertebrate), Pfam Analysis. lncRNAs were predicted by cross-comparing among mentioned methods.</p>
    <p class="center"> <img class="normal" src="files/lncRNAFilter/Coding_potential_filter.result.png" /> </p>
    <p class="name_fig">Figure 4.6 The venn diagram of predicted lncRNAs from different software. </p>
    <p class="cenmark">The venn diagram of predicted lncRNAs from different software. Number in each circle and overlap represent the respective total and shared number of noncoding transcripts predicted by the software.</p>
	<p class="paragraph">Result directory of coding potential screening venn diagram:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/3.Assemble/lncRNAFilter/FilterStat" title="点击打开" >Coding_potential_filter.result</a></p>


<!---------------------------转录本筛选之Novel_lncRNA类型统计-------------------------->
    <h4 id="result62">4.3.2.2 Classification of Novel_lncRNA</h4>
    <p class="paragraph">Pie chart is drawn to show different types of final lncRNA obtained after each filtering step (lincRNA, anti-sense_lncRNA, intronic_lncRNA). Pie chart as follows:</p>
    <p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_classification.png" /> </p>
    <p class="name_fig">Figure 4.7 lncRNA Distribution of different types lncRNA</p>
    <p class="premark">According to the position relationship between HGNC and known mRNA, the newly screened lncrnas were divided into 4 types. (1) Antisense: overlap with one or more exons of protein-coding genes with opposite chains, or there is published evidence of reverse regulation of coding region genes; (2) lincRNA: non-coding RNA located in intergene region; (3) Sense Overlapping: Overlapped by one or more exons of the same protein coding gene; (4) Sense intronic: the intronic region of a coding gene that does not intersect with any exons on the same chain (Mullan LJ et al., 2002).</p>
	<p class="paragraph">Result directory of distribution of different types lncRNA:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/3.Assemble/lncRNAFilter/FilterStat" title="点击打开" >lncRNA_classification</a></p>

</div>
<!---------------------------特征比较-------------------------->	
</br>
<!---------------------------特征比较-------------------------->
<div id="page">

	<h4 id="result63">4.3.3 Feature comparison of lncRNA and mRNA</h4>
	<p class="paragraph">Comparing the length of transcript, the number of exon and the length of ORF of lncRNA and mRNA, we can get the difference between lncRNA and mRNA.</p>
</p>
	

	</br>
	<h4 id="351">4.3.3.1 Comparative analysis of lncRNA and mRNA length</h4>
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_length_density.png" /> </p>
	<p class="name_fig">Figure 4.8 Density distribution diagram of lncRNA and mRNA length comparison</p>
	<p class="premark">The horizontal coordinates are the length of the transcript, the vertical coordinates are the density, the purple areas are known lncRNA (if a genome contains known lncRNA annotations) , and the red areas are predicted novel lncRNA, this chart verifies that the predicted transcript length of novel lncRNA is consistent with the general characteristics (transcripts longer than 200nt) . The Cyan region is known mRNA, and the orange region is predicted novel mRNA for comparison with lncRNA.</p>
	
	
	<br />
	
	<h4 id="351">4.3.3.2 Comparative analysis of exon numbers of lncRNA and mRNA</h4>
	
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_exon_density.png" /> </p>
	
	<p class="name_fig">Figure 4.9 Exon number density distribution of lncRNA and mRNA</p>
	<p class="premark">The horizontal coordinates are the exon number of transcripts, the vertical coordinates are the density, the purple areas are known lncRNA (if a genome contains known lncRNA annotations) , and the red areas are predicted novel lncRNA, by comparing the two, we can verify whether the predicted number of exon of novel lncRNA conforms to the general characteristics (the number of exon is greater than or equal to 2) . The Cyan region is known mRNA, and the orange region is predicted novel mRNA for comparison with lncRNA.</p>
	
	<br />
	
	<h4 id="351">4.3.3.3 Comparative analysis of ORF length of lncRNA and mRNA</h4>
    <p class="paragraph">reading Open reading frame(ORF)is a normal nucleotide sequence of a structural gene that encodes a complete polypeptide chain from the Start codon to the stop codon reading frame, during which there is no stop codon that interrupts translation. When the orfs of lncRNA and mRNA were compared, the sequences of annotated mRNA and annotated lncRNA were extracted by known gene structure annotation, and the ORF sequences of novel lncRNA and novel mRNA were predicted by EMBOSS:getorf(Garber M et al. , 2011). The resulting ORF sequence is transformed into a protein sequence, and the length distribution is as follows:</p>
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_orf_density.png" /> </p>
	<p class="name_fig">Figure 4.10 ORF length density distribution of lncRNA and mRNA</p>
    <p class="premark">The horizontal coordinates predict the length of the ORF transformed into a protein sequence, the vertical coordinates predict the density, the purple areas are known lncRNA(if a genome contains a known lncRNA annotation), and the red areas are predicted novel lncRNA, by comparing the two, we can verify whether the predicted ORF length of novel lncRNA conforms to the general characteristics. The Cyan region is known mRNA, and the orange region is predicted novel mRNA for comparison with lncRNA.</p>
	
	
	<br />
	</div>
	


<!-------------------------------3.4 LncRNA Identification Q&A------------------------------>
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">

<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:How to assemble transcripts for multiple samples?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:The stringtie software is used for each sample assembly. And stringtie-merge will merge transcripts which come from each sample before filtering.</td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:How to define a transcript as lncRNA or TUCP</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:A coding potential prediction will be conducted on the new transcript after initial screening using multiple software. The transcript will be classified as TUCP (transcripts of uncertain coding potential) as long as one of the software considers the transcript having coding potential. Otherwise, the transcript will be classified as lncRNA if all software considers that the transcript does not have coding potential.</td></tr>
</table>
</div>
	<p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/3.Assemble/lncRNAFilter/FeatureSignature" title="点击打开" >FeatureSignature</a></p>

<!------------------------------3 定量------------------------------>

<div id="page">

	<h3 id="result3">4.4 Quantification</h3>

    <p class="paragraph">Quantification is the basis of significance analysis of gene expression differences, which can be divided into gene level quantification and transcription level quantification. Gene level quantification is robust and reliable, and easy to implement in algorithm, but it cannot be accurate to different transcripts of each gene. Transcription level quantitative algorithm is difficult to implement, and its accuracy is not as good as gene level quantitative algorithm. However, in recent years, new algorithms have been developed and their accuracy has been significantly improved (Bray NL et al., 2016). we provides quantitative results at both the gene and transcription levels.</p>

	

	
	

	
	
	</p>
    <p class="paragraph">StringTie uses network flow algorithm to splicing and quantifying transcripts and genes. It classifies reads into different classes, then generates a splicing graph for each class to identify transcripts, and then generates a maximum flow algorithm of flow neural network for each transcript to evaluate expression level. The analysis process are shown below:</p>
	<p class="center">
		<img class="normal" src="images/Stringtie.png" />
	</p>
	
	<p class="paragraph">
	<br />
	<br />


</div>

<!--------------------------3.1 定量结果说明---------------------->


	<h4 id="result31">4.4.1 Quantification Results</h4>
	
    <p class="paragraph">We quantified the expression levels of all the known transcripts and predicted Novel_lncRNA, Novel_mRNA and Unclassified transcripts after the above mapping, assembly and Identification.</p>
	
	
    <p class="paragraph">In RNA-seq, FPKM (the expected number of Fragments Per Kilobase of gene/transcript sequence per Millions base-pairs sequenced) is the mostly used estimate of gene expression levels. FPKM takes into account the effects from both sequencing depth and gene length on counting fragments. mRNA gene expression level quantity results are shown below:</p>

</p>
	
    

</br>

	<p class="name_fig">Table 4.4 Gene FPKM information of each sample</p>

	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>gene_id</th><th>gene_name</th><th>gene_locus</th><th>sample1</th><th>sample2</th><th>sample3</th><th>sample4</th><th>sample5</th><th>sample6</th><th>sample7</th></tr>
			
		  
			
				<tr><td>ENSG00000230368</td><td>FAM41C</td><td>1:868071-876903</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000234711</td><td>TUBB8P11</td><td>1:873292-874349</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000283040</td><td>AL669831.6</td><td>1:874529-877234</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000187961</td><td>KLHL17</td><td>1:960584-965719</td><td>0.149956</td><td>1.747221</td><td>0.542480</td><td>0.000000</td><td>0.138708</td><td>0.273694</td><td>1.956710</td></tr>
			
		  
			
				<tr><td>ENSG00000187634</td><td>SAMD11</td><td>1:923928-944581</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.245914</td><td>0.000000</td></tr>
			
		  
			
				<tr><td>ENSG00000188976</td><td>NOC2L</td><td>1:944203-959309</td><td>1.274181</td><td>5.670410</td><td>0.597106</td><td>2.340009</td><td>5.956353</td><td>4.225698</td><td>5.408510</td></tr>
			
		  
			
				<tr><td>ENSG00000187583</td><td>PLEKHN1</td><td>1:966497-975865</td><td>0.083042</td><td>0.0</td><td>0.000000</td><td>0.893553</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000187642</td><td>PERM1</td><td>1:975204-982093</td><td>0.244146</td><td>0.0</td><td>0.778349</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>XLOC_007951</td><td>XLOC_007951</td><td>1:992563-993021</td><td>10.090649</td><td>0.0</td><td>5.403792</td><td>0.0</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000272512</td><td>AL645608.7</td><td>1:995966-998051</td><td>12.563572</td><td>2.192926</td><td>5.093130</td><td>0.0</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		   
		</table>
	</div>		
	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1) The first column is ID of gene, ENSG, G refers to gene; The first three letters, "ENS," mean "ensemble."<br />
    (2) gene_name: gene name, If not, use "--".<br />
    (3) locus: The position of a gene on a chromosome.<br />
	(4) FPKM value corresponding to gene ID of each sample for each subsequent column.<br />
</ul>	
</div>
  <p class="paragraph">Result directory of mRNA gene expression level quantity results:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/4.Quant/1.Count" title="点击打开" >mRNA_gene.FPKM</a></p>

</br>
<!---------------------------3.2 表达水平分布-------------------------->
<div id="page">


	<h4 id="result32">4.4.2 Distribution of Expression Levels</h4>

		
	
		
    <p class="paragraph">After the expression values (FPKM) of all genes or transcripts in each sample were calculated, the distribution of expression levels of genes in different samples was shown by boxplots.</p>

</p>
	

</br>

	<p class="center">
		<img class="normal1" src="files/Quantification/boxplot.png"/>
	</p>
	
	<p class="name_fig">Figure 4.11 Sample gene expression distribution box plot </p>
	
	
    <p class="cenmark">X axis represents the name of the sample. Y axis indicates the log<sub>10</sub>(FPKM+1). Parameters of box plots are indicated, including maximum, upper quartile, mid-value, lower quartile and minimum.</p> 
	<br />
	<br />
	<p class="paragraph">Result directory of boxplot of mRNA gene expression levels of different samples: <a href="../../Result_PXXXXXXXX_Homo_sapiens/4.Quant/3.Distribution" title="点击打开" >boxplot</a></p> 


</div>




<!---------------------------4 差异分析-------------------------->


<div id="page">



<h3 id="result4">4.5 Differential Expression Analysis</h3>


<!---------------------------4.1 差异基因列表------------------------------>

	
	

		
    <p class="paragraph">After the quantitative analysis, the expression matrix of all samples was obtained, and then the significance analysis of expression difference at gene or transcript level could be carried out to find functional genes or transcripts related to the treatment group. we used edgeR(Robinson MD et al., 2010) software to analyze the significance of expression differences. Significance level was determined by pvalue or corrected pvalue (padj). Padj is the value corrected by multiple test of pvalue using BH method when the false positive rate is high. The smaller the corrected pvalue is, the more significant it is. Padj <= 0.05 was used as the difference significance standard (if padj <= 0.05 was too small, pvalue <= 0.05 was used for difference screening, and the specific difference screening conditions were shown in volcano diagram).</p>
    <p class="paragraph">The differential analysis is mainly divided into three steps:</p>
    <p class="paragraph">(1) the raw readcount is normalized, mainly to correct the sequencing depth;</p>
    <p class="paragraph">(2) the statistical model is used to calculate the hypothesis test's probability (pvalue);</p>
    <p class="paragraph">(3) multiple hypothesis test corrections are used to obtain FDR values (false discovery rate).</p>
	
	
	</br>
	<h4 id="result41">4.5.1 Differential Gene List</h4>



</br>	


	<p class="name_fig">Table 4.5 Differential gene list</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>gene_id</th><th>gene_name</th><th>gene_locus</th><th>group1_FPKM</th><th>group2_FPKM</th><th>log2FoldChange</th><th>pvalue</th><th>padj</th></tr>
			
		  
			
				<tr><td>ENSG00000123689</td><td>G0S2</td><td>1:209675412-209676390</td><td>113.69630375</td><td>0.717038666666667</td><td>7.71505645130916</td><td>3.85428029984956e-11</td><td>3.70382472736685e-07</td></tr>
			
		  
			
				<tr><td>ENSG00000196126</td><td>HLA-DRB1</td><td>6:32578769-32589848</td><td>0.193821</td><td>62.0900873333333</td><td>-7.86453469597298</td><td>1.46687362421868e-10</td><td>7.68788466453008e-07</td></tr>
			
		  
			
				<tr><td>ENSG00000276293</td><td>PIP4K2B</td><td>17:38765689-38800126</td><td>0</td><td>3.722116</td><td>-9.22931376008883</td><td>8.88838729116198e-10</td><td>3.882003149415e-06</td></tr>
			
		  
			
				<tr><td>ENSG00000125740</td><td>FOSB</td><td>19:45467995-45475179</td><td>36.4045835</td><td>0.558543333333333</td><td>6.26371132063224</td><td>4.12337385018474e-09</td><td>1.35066264680114e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000112486</td><td>CCR6</td><td>6:167111807-167139696</td><td>0</td><td>4.481611</td><td>-8.91637270913355</td><td>4.76044581074459e-09</td><td>1.3860831385618e-05</td></tr>
			
		  
			
				<tr><td>XLOC_165816</td><td>XLOC_165816</td><td>CHR_HSCHR6_MHC_DBB_CTG1:31260956-31264273</td><td>198.17379125</td><td>0.478476</td><td>9.2393121544114</td><td>6.2465470190817e-09</td><td>1.63690764635036e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000135424</td><td>ITGA7</td><td>12:55684568-55716043</td><td>16.71872225</td><td>0.028137</td><td>8.7691186137305</td><td>9.24278614834938e-09</td><td>2.20188373652269e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000198502</td><td>HLA-DRB5</td><td>6:32517353-32530287</td><td>0</td><td>41.6268273333333</td><td>-10.8534545180771</td><td>2.30352389471382e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000205710</td><td>C17orf107</td><td>17:4899418-4902934</td><td>20.91248775</td><td>0.135322333333333</td><td>7.12690204181498</td><td>2.67060728521762e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000181847</td><td>TIGIT</td><td>3:114276913-114310288</td><td>0.0847365</td><td>8.63292066666667</td><td>-5.99647519952058</td><td>2.8577533610694e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
		</table>
	</div>				
	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1) gene_id: Gene id<br />
	(2) gene_name: gene name, If not, use "--".<br />
	(3) gene_description: gene description<br />
	(4) gene_locus: The position of a gene on a chromosome.<br />
	(5) sample: The readcount values of each sample after normalization<br />
	(6) group: The standardized average of readcount for each group<br />
	(7) log<sub>2</sub>FoldChange: The ratio of gene expression level between the treatment group and the control group of a comparison combination was processed by the shrinkage model of the differential analysis software, and finally the logarithm was taken with 2 as the base.<br />
	(8) pvalue: Pvalue in hypergenometric test of compare group.<br />
	(9) padj: The corrected pvalue of multiple hypothesis test.<br />
</ul>
</div>

	<br />
	<br />

  <p class="paragraph">Result directory of differential expression of mRNA gene level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/5.Differential/mRNA/1.deglist" title="点击打开" >mRNA.gene_level.deg</a> </p>
</div>

<!-------------------------------4.2 差异基因火山图---------------------------------->

<div id="page">

	<h4 id="result42">4.5.2 Volcano plot of differential genes</h4>
	<p class="paragraph">The differentially expressed genes can be visualized by volcano plot. The threshold is qvalue <= 0.05. (If there is no or little significant differencially expression genes, the threshold will be pvalue <= 0.05. )</p>



</br>

   <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGvolcano_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGvolcano_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGvolcano_Content_pane_group1vsgroup2" role="tabpanel"
            aria-labelledby="gallery_figure_DEGvolcano_Content_list_group1vsgroup2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/Volcano/group1vsgroup2.diffgene.Volcanoplot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGvolcano_Content_pane_group2vssample1" role="tabpanel"
            aria-labelledby="gallery_figure_DEGvolcano_Content_list_group2vssample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/Volcano/group2vssample1.diffgene.Volcanoplot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGvolcano_Content_pane_sample1vssample5" role="tabpanel"
            aria-labelledby="gallery_figure_DEGvolcano_Content_list_sample1vssample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/Volcano/sample1vssample5.diffgene.Volcanoplot.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGvolcano_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGvolcano_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGvolcano_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGvolcano_Content_list_group1vsgroup2"
            data-toggle="list" href="#gallery_figure_DEGvolcano_Content_pane_group1vsgroup2" role="tab"
            aria-controls="group1vsgroup2">group1vsgroup2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGvolcano_Content_list_group2vssample1"
            data-toggle="list" href="#gallery_figure_DEGvolcano_Content_pane_group2vssample1" role="tab"
            aria-controls="group2vssample1">group2vssample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGvolcano_Content_list_sample1vssample5"
            data-toggle="list" href="#gallery_figure_DEGvolcano_Content_pane_sample1vssample5" role="tab"
            aria-controls="sample1vssample5">sample1vssample5</a>
          
        </div>
      </div>
    </div>
    </p>

	
</div>

<p class="name_fig">Figure 4.12 Differential Gene Volcano plot</p>
    <p class="premark">The x-axis shows the fold change in gene expression between different samples, and the y-axis shows the statistical significance of the differences. Significantly up and down regulated genes are highlighted in red and green, respectively. Genes did not express differently between treatment group and control group are in blue. </p>


	<br />
	<br />
<p class="paragraph">Result directory of volcano plot of differential expression of mRNA gene level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/5.Differential/mRNA/1.deglist" title="点击打开" >mRNA.gene_level.diffgene.volcano</a></p>

</div>


<!---------------------------4.3 差异基因聚类-------------------------->

<div id="page">

        <h4 id="result43">4.5.3 Clustering of differential genes</h4>

        <p class="paragraph">All the differentially expressed genes in the comparison group were pooled as the differential gene set. For more than two groups of experiments, cluster analysis can be carried out on different gene sets and genes with similar expression patterns can be clustered together. We used the mainstream hierarchical clustering to cluster the fpkm values of genes, and homogenized the row (Z-score). The genes or samples with similar expression patterns in the heat map will be gathered together. The color in each grid reflects not the gene expression value, but the value obtained after homogenizing the expression data rows (generally between -2 and 2). Therefore, the colors in the heat map can only be compared horizontally (the expression of the same gene in different samples), but not vertically (the same sample). There are not only inter group clustering, but also inter sample clustering. The final report shows the clustering among samples.</p>
</p>
	
    

</br>

        <p class="center">
        <img class="normal1" src="files/DiffEnrich/Cluster/Hcluster_heatmap.png"/>
        </p>

<p class="name_fig">Figure 4.13 Differential expression gene clustering heatmap </p>
		<p class="cenmark">The overall results of FPKM cluster analysis, clustered using the log2(FPKM+1) value. Red color indicates genes with high expression levels, and green color indicates genes with low expression levels. The color ranging from red to blue indicates that log2(FPKM+1) values wherefrom large to small. The chromosome to which each gene belongs, gene's length, and the biological type of the gene are also added to the heatmap </p>

        <br />
        <br />


</div>


<!-------------------------------4.4 Differential Expression Analysis Q&A------------------------------>
<div id="page">
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">

<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:Can I use FPKM for differential analysis?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:Most of the differential analysis software (DESeq, DESeq2 and edgeR) use the raw read counts as the input file and the negative binomial distribution model to estimate the probability of gene differential expression between samples. The software itself does some corrections on the read count (mainly the depth of sequencing), while FPKM is the corrected expression value, so it is unreasonable to use FPKM for the differential analysis to make double corrections. </td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:Why we use padj, can we use pvalue?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:Corrected p value (padj/qvalue), is value after applying multiple statistical hypothesis examination on
        p-value, which is higher than original p-value and more efficient in control false positive.</td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:What is the method for clustering analysis?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:Clustering uses a build-in R package, pheatmap. We focus on the data (mRNA_Diff_gene_heatmap.xls/lncRNA_Diff_transcript_heatmap.xls), which is the union gene/transcript set of all comparison groups. And we use relative gene/transcript expression level, log2(ratios), to do clustering. The clustering will calculate the distance between each genes/transcripts, and evaluate the relative gene/transcript distance through iteration. Finally, it will divide the gene/transcripts into several subgroups according to gene/transcript distance. H-cluster is the main clustering method used by us.</td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:What is clustering for?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A:Clustering can be used to determine the different gene/transcript expression pattern through gathering the gene/transcript with similar expression. These genes/transcripts with similar expression may have similar function or participate in same pathway and metabolic process. It is possible for us to identify unknown gene/transcript function through clustering. Thus, the clustering is useful in expanding biological understanding of gene/transcript expression.</td></tr>

</table>
</div>
	<br />
	<p class="paragraph">Result directory of clustering  of differential expression of mRNA gene level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/5.Differential/mRNA/3.cluster" title="点击打开" >mRNA_Diff_gene_heatmap</a> </p>
</div>

<!---------------------------5 富集分析---------------------------------->


<div id="page">


	<h3  id="result5">4.6 Enrichment Analysis</h3>
	
	
	
    <p class="paragraph">ClusterProfiler(Yu G, 2012) software was used for GO functional enrichment analysis and KEGG pathway enrichment analysis of differential gene sets. Enrichment analysis is based on hypergeometric distribution principle. The result of enrichment analysis was that all differential gene sets, up-regulated differential gene sets and down-regulated differential gene sets of each differential comparison combination were enriched. The table shown in this report shows the enrichment analysis results of a selected comparison combination, and the picture shows the enrichment analysis results of all combinations.</p>
	
	
	

	<p class="center">
		<img class="middle1" src="images/enrichment.png" />
	</p>

	<p class="name_fig">Figure 4.14 Gene Enrichment Analysis Schematic</p>
    <p class="premark">Here, N is the number of all genes with a GO or KEGG annotation, n is the number of DEGs in N, M is the number of all genes annotated to specific terms, and m is number of DEGs in M</p>

	<br />
	<br />

</div>

<!--------------------------5.1 GO富集分析-------------------------->
<div id="page">

<!-- go富集方法需要优化完善 -->

	<h4 id="result51">4.6.1 GO enrichment analysis</h4>
    <p class="paragraph">GO is the abbreviation of Gene Ontology (http://www.geneontology.org/), which is a major bioinformatics classification system to unify the presentation of gene properties across all species. It includes three main branches: cellular component, molecular function and biological process. GO terms with padj < 0.05 are significant enrichment.</p>

</p>
	
    
</br>

	<p class="name_fig">Table 4.6 Partially displayed results of GO enrichment analysis of differential genes</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>Category</th><th>GOID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0002521</td><td>leukocyte differentiation</td><td>126/2199</td><td>455/13522</td><td>2.68112374545469e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000177606/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000173334/ENSG00000170345/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000161405/ENSG00000112715/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000171223/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000155090/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000010610/ENSG00000182578/ENSG00000073861/ENSG00000198286/ENSG00000204577/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000108840/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000100985/ENSG00000116251/ENSG00000100105/ENSG00000142185/ENSG00000110719/ENSG00000115232/ENSG00000132170/ENSG00000172216/ENSG00000134321/ENSG00000111203/ENSG00000110651/ENSG00000134256/ENSG00000122224/ENSG00000135905/ENSG00000068323/ENSG00000131188/ENSG00000172575/ENSG00000135898/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000128590/ENSG00000138795/ENSG00000012223/ENSG00000120899/ENSG00000137154/ENSG00000126353/ENSG00000185507/ENSG00000104856/ENSG00000182885/ENSG00000135426/ENSG00000185338/ENSG00000118260/ENSG00000168421/ENSG00000071553/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000154229/ENSG00000160654/ENSG00000029993/ENSG00000168264/ENSG00000149311/ENSG00000169508/ENSG00000143799/ENSG00000125686/ENSG00000143507/ENSG00000113916/ENSG00000128342/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000081189/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000141968/ENSG00000067082/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000123360/ENSG00000072274/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000112062/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000169710/ENSG00000136997/ENSG00000163374/ENSG00000002330</td><td>CCR6/JUN/CD83/NFATC2/NFKBID/TRIB1/FOS/THEMIS/EGR1/IL18R1/IKZF3/VEGFA/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/JUNB/CD2/CD40LG/SPN/CD8A/KLF10/EGR3/CSF1/SLAMF8/CD4/CSF1R/TBX21/CARD11/LILRB3/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/HDAC5/GATA3/FCRL3/PRKDC/LCK/TCF7/MMP9/RPL22/PATZ1/TRPM2/TCIRG1/ITGA4/PPARG/CEBPB/RSAD2/ITFG2/CD81/CD101/LY9/DOCK10/TFE3/PRR7/RASGRP1/GPR55/FZD5/TNFSF4/BCL3/DNAJB9/LEF1/LTF/PTK2B/RPS6/CCR7/IRF7/RELB/ADGRG3/TESPA1/SOCS1/CREB1/RHOH/ATP6AP1/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/FNIP1/PRKCA/CD3G/HMGB3/IRF2BP2/ATM/GPR183/PARP1/MED1/DUSP10/BCL6/LIF/IL7R/CLEC4D/VSIR/CD3D/CCR9/MEF2C/CAMK4/ITM2A/BCL2/VAV1/KLF6/WNT1/CD79B/SLAMF6/PDE1B/TFRC/CD79A/SOX13/CD74/MAPK14/IKZF1/LOXL3/RAG1/FASN/MYC/YY1AP1/BAD</td><td>126</td><td>54</td><td>ENSG00000177606/ENSG00000112149/ENSG00000167604/ENSG00000173334/ENSG00000170345/ENSG00000120738/ENSG00000115604/ENSG00000112715/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000171223/ENSG00000155090/ENSG00000179388/ENSG00000184371/ENSG00000204577/ENSG00000158869/ENSG00000112299/ENSG00000108840/ENSG00000100985/ENSG00000142185/ENSG00000110719/ENSG00000132170/ENSG00000172216/ENSG00000134321/ENSG00000068323/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000128590/ENSG00000012223/ENSG00000120899/ENSG00000185507/ENSG00000104856/ENSG00000182885/ENSG00000185338/ENSG00000071553/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000029993/ENSG00000168264/ENSG00000143507/ENSG00000113916/ENSG00000128342/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000067082/ENSG00000123360/ENSG00000072274/ENSG00000112062/ENSG00000163374/ENSG00000002330</td><td>72</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000161405/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000158714/ENSG00000010610/ENSG00000182578/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000115232/ENSG00000111203/ENSG00000110651/ENSG00000134256/ENSG00000122224/ENSG00000135905/ENSG00000172575/ENSG00000135898/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000118260/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000154229/ENSG00000160654/ENSG00000149311/ENSG00000169508/ENSG00000143799/ENSG00000125686/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000081189/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000169710/ENSG00000136997</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0030098</td><td>lymphocyte differentiation</td><td>95/2199</td><td>317/13522</td><td>5.34192600043356e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000161405/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000158714/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000108840/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110719/ENSG00000115232/ENSG00000134321/ENSG00000111203/ENSG00000122224/ENSG00000135905/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000128590/ENSG00000138795/ENSG00000120899/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000182885/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000160654/ENSG00000029993/ENSG00000168264/ENSG00000149311/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000141968/ENSG00000067082/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000163374/ENSG00000002330</td><td>CCR6/CD83/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/IKZF3/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/CD2/CD40LG/SPN/CD8A/EGR3/SLAMF8/CD4/TBX21/CARD11/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/HDAC5/GATA3/FCRL3/PRKDC/LCK/TCF7/RPL22/PATZ1/TCIRG1/ITGA4/RSAD2/ITFG2/LY9/DOCK10/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/DNAJB9/LEF1/PTK2B/RPS6/CCR7/RELB/ADGRG3/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/FNIP1/CD3G/HMGB3/IRF2BP2/ATM/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/CCR9/CAMK4/ITM2A/BCL2/VAV1/KLF6/WNT1/CD79B/SLAMF6/CD79A/SOX13/CD74/IKZF1/LOXL3/RAG1/YY1AP1/BAD</td><td>95</td><td>34</td><td>ENSG00000112149/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000158869/ENSG00000112299/ENSG00000108840/ENSG00000110719/ENSG00000134321/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000128590/ENSG00000120899/ENSG00000104856/ENSG00000182885/ENSG00000185338/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000029993/ENSG00000168264/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000067082/ENSG00000163374/ENSG00000002330</td><td>61</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000161405/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000158714/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000115232/ENSG00000111203/ENSG00000122224/ENSG00000135905/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000160654/ENSG00000149311/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0030217</td><td>T cell differentiation</td><td>74/2199</td><td>226/13522</td><td>5.97468406956998e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000116824/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000107485/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110719/ENSG00000134321/ENSG00000122224/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000160654/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000171791/ENSG00000141968/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349/ENSG00000002330</td><td>CCR6/CD83/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/CD2/SPN/CD8A/EGR3/CD4/TBX21/CARD11/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/GATA3/PRKDC/LCK/TCF7/RPL22/PATZ1/TCIRG1/RSAD2/LY9/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/LEF1/RPS6/CCR7/RELB/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/CD3G/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/CCR9/CAMK4/BCL2/VAV1/WNT1/SLAMF6/SOX13/CD74/LOXL3/RAG1/BAD</td><td>74</td><td>25</td><td>ENSG00000112149/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000158869/ENSG00000112299/ENSG00000110719/ENSG00000134321/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000104856/ENSG00000185338/ENSG00000168036/ENSG00000104312/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000002330</td><td>49</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000197471/ENSG00000153563/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000122224/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000160654/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000171791/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0006935</td><td>chemotaxis</td><td>126/2199</td><td>477/13522</td><td>6.51313530286691e-09</td><td>9.26819153597961e-06</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000169429/ENSG00000183691/ENSG00000122861/ENSG00000113070/ENSG00000160791/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000127152/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000120129/ENSG00000182578/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000168329/ENSG00000170962/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000163735/ENSG00000107485/ENSG00000012171/ENSG00000183134/ENSG00000121797/ENSG00000168758/ENSG00000170989/ENSG00000128512/ENSG00000149294/ENSG00000137285/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000162415/ENSG00000156299/ENSG00000167680/ENSG00000114554/ENSG00000010810/ENSG00000182197/ENSG00000020181/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000120899/ENSG00000117525/ENSG00000116106/ENSG00000092969/ENSG00000163220/ENSG00000126353/ENSG00000143185/ENSG00000001617/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000206047/ENSG00000141622/ENSG00000163739/ENSG00000258947/ENSG00000100784/ENSG00000276085/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000169508/ENSG00000163932/ENSG00000105372/ENSG00000154188/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000132003/ENSG00000173585/ENSG00000090776/ENSG00000119630/ENSG00000121966/ENSG00000186810/ENSG00000141968/ENSG00000131437/ENSG00000138448/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000164171/ENSG00000173531/ENSG00000168546/ENSG00000019582/ENSG00000112062/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000106366/ENSG00000006451/ENSG00000198211</td><td>CCR6/KLRK1/TMEM102/CXCL8/NOG/PLAU/HBEGF/CCR5/VEGFA/FOSL1/NR4A3/BCL11B/EDN1/PLAUR/S100A12/CMKLR1/SPN/AMOT/EGR3/CSF1/SLAMF8/DUSP1/CSF1R/GRB10/IL1RN/IRS2/CX3CR1/PDGFD/PDLIM7/CXCL16/FCER1G/CXCL2/RAP1GAP/CXCL5/GATA3/SEMA3B/PTGDR2/CCRL2/SEMA4C/S1PR1/DOCK4/NCAM1/TUBB2B/TRPM2/S100A8/NTNG2/ZSWIM5/TIAM1/SEMA6B/PLXNA1/FYN/EXT1/ADGRA2/DRAXIN/DOK2/CXCR5/PLXND1/EVL/FPR3/MTUS1/SEMA4F/NEO1/LGR6/CXCR6/MEGF8/LEF1/PTCH1/PLCG1/PTK2B/F3/EPHA4/TGFB2/S100A9/CCR7/XCL2/SEMA3F/CREB1/OPHN1/ERBB2/DEFA1/RNF165/CXCL1/TUBB3/RPS6KA5/CCL3L1/PLXNB2/PRKCA/CCL28/NRP1/B3GNT2/C3AR1/SPTBN4/GPR183/PRKCD/RPS19/ANGPT1/MAPK8IP3/CREB3/CSF3R/LPAR1/NRCAM/CNR2/ZSWIM4/CCR9/EFNB1/PGF/CXCR4/CXCR3/VAV1/KIF3A/ITGAV/TNFSF14/PLP2/GRB2/CYP19A1/ITGA2/MST1/GFRA2/CD74/MAPK14/CYFIP1/MYCBP2/SWAP70/SERPINE1/RALA/AC092143.1</td><td>126</td><td>56</td><td>ENSG00000169429/ENSG00000122861/ENSG00000113070/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000179388/ENSG00000184371/ENSG00000120129/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000012171/ENSG00000121797/ENSG00000128512/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000167680/ENSG00000182197/ENSG00000020181/ENSG00000120899/ENSG00000117525/ENSG00000163220/ENSG00000001617/ENSG00000206047/ENSG00000163739/ENSG00000258947/ENSG00000276085/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000163932/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000132003/ENSG00000119630/ENSG00000121966/ENSG00000141968/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000173531/ENSG00000112062/ENSG00000106366/ENSG00000198211</td><td>70</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000183691/ENSG00000160791/ENSG00000127152/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000158714/ENSG00000182578/ENSG00000168329/ENSG00000170962/ENSG00000163735/ENSG00000107485/ENSG00000183134/ENSG00000168758/ENSG00000170989/ENSG00000149294/ENSG00000137285/ENSG00000162415/ENSG00000156299/ENSG00000114554/ENSG00000010810/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000116106/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000141622/ENSG00000100784/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000169508/ENSG00000105372/ENSG00000154188/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000173585/ENSG00000090776/ENSG00000186810/ENSG00000131437/ENSG00000138448/ENSG00000164171/ENSG00000168546/ENSG00000019582/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000006451</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0042330</td><td>taxis</td><td>126/2199</td><td>479/13522</td><td>8.54263111304989e-09</td><td>9.72493125909599e-06</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000169429/ENSG00000183691/ENSG00000122861/ENSG00000113070/ENSG00000160791/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000127152/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000120129/ENSG00000182578/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000168329/ENSG00000170962/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000163735/ENSG00000107485/ENSG00000012171/ENSG00000183134/ENSG00000121797/ENSG00000168758/ENSG00000170989/ENSG00000128512/ENSG00000149294/ENSG00000137285/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000162415/ENSG00000156299/ENSG00000167680/ENSG00000114554/ENSG00000010810/ENSG00000182197/ENSG00000020181/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000120899/ENSG00000117525/ENSG00000116106/ENSG00000092969/ENSG00000163220/ENSG00000126353/ENSG00000143185/ENSG00000001617/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000206047/ENSG00000141622/ENSG00000163739/ENSG00000258947/ENSG00000100784/ENSG00000276085/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000169508/ENSG00000163932/ENSG00000105372/ENSG00000154188/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000132003/ENSG00000173585/ENSG00000090776/ENSG00000119630/ENSG00000121966/ENSG00000186810/ENSG00000141968/ENSG00000131437/ENSG00000138448/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000164171/ENSG00000173531/ENSG00000168546/ENSG00000019582/ENSG00000112062/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000106366/ENSG00000006451/ENSG00000198211</td><td>CCR6/KLRK1/TMEM102/CXCL8/NOG/PLAU/HBEGF/CCR5/VEGFA/FOSL1/NR4A3/BCL11B/EDN1/PLAUR/S100A12/CMKLR1/SPN/AMOT/EGR3/CSF1/SLAMF8/DUSP1/CSF1R/GRB10/IL1RN/IRS2/CX3CR1/PDGFD/PDLIM7/CXCL16/FCER1G/CXCL2/RAP1GAP/CXCL5/GATA3/SEMA3B/PTGDR2/CCRL2/SEMA4C/S1PR1/DOCK4/NCAM1/TUBB2B/TRPM2/S100A8/NTNG2/ZSWIM5/TIAM1/SEMA6B/PLXNA1/FYN/EXT1/ADGRA2/DRAXIN/DOK2/CXCR5/PLXND1/EVL/FPR3/MTUS1/SEMA4F/NEO1/LGR6/CXCR6/MEGF8/LEF1/PTCH1/PLCG1/PTK2B/F3/EPHA4/TGFB2/S100A9/CCR7/XCL2/SEMA3F/CREB1/OPHN1/ERBB2/DEFA1/RNF165/CXCL1/TUBB3/RPS6KA5/CCL3L1/PLXNB2/PRKCA/CCL28/NRP1/B3GNT2/C3AR1/SPTBN4/GPR183/PRKCD/RPS19/ANGPT1/MAPK8IP3/CREB3/CSF3R/LPAR1/NRCAM/CNR2/ZSWIM4/CCR9/EFNB1/PGF/CXCR4/CXCR3/VAV1/KIF3A/ITGAV/TNFSF14/PLP2/GRB2/CYP19A1/ITGA2/MST1/GFRA2/CD74/MAPK14/CYFIP1/MYCBP2/SWAP70/SERPINE1/RALA/AC092143.1</td><td>126</td><td>56</td><td>ENSG00000169429/ENSG00000122861/ENSG00000113070/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000179388/ENSG00000184371/ENSG00000120129/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000012171/ENSG00000121797/ENSG00000128512/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000167680/ENSG00000182197/ENSG00000020181/ENSG00000120899/ENSG00000117525/ENSG00000163220/ENSG00000001617/ENSG00000206047/ENSG00000163739/ENSG00000258947/ENSG00000276085/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000163932/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000132003/ENSG00000119630/ENSG00000121966/ENSG00000141968/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000173531/ENSG00000112062/ENSG00000106366/ENSG00000198211</td><td>70</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000183691/ENSG00000160791/ENSG00000127152/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000158714/ENSG00000182578/ENSG00000168329/ENSG00000170962/ENSG00000163735/ENSG00000107485/ENSG00000183134/ENSG00000168758/ENSG00000170989/ENSG00000149294/ENSG00000137285/ENSG00000162415/ENSG00000156299/ENSG00000114554/ENSG00000010810/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000116106/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000141622/ENSG00000100784/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000169508/ENSG00000105372/ENSG00000154188/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000173585/ENSG00000090776/ENSG00000186810/ENSG00000131437/ENSG00000138448/ENSG00000164171/ENSG00000168546/ENSG00000019582/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000006451</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0019080</td><td>viral gene expression</td><td>60/2199</td><td>187/13522</td><td>5.69661156368897e-08</td><td>5.4041855034196e-05</td><td>ENSG00000177606/ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000128016/ENSG00000080561/ENSG00000153922/ENSG00000174444/ENSG00000089009/ENSG00000175390/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000142089/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000100129/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000139496/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000140564/ENSG00000144231/ENSG00000181222/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000130811/ENSG00000132376/ENSG00000113569/ENSG00000110713/ENSG00000213741</td><td>JUN/RPS3A/RPL3/TRIM32/RPL22/RPS4X/ZFP36/MID2/CHD1/RPL4/AC004086.1/EIF3F/RPS23/RPS27A/RRP1B/RPSA/IFITM3/MDFIC/LEF1/RPS6/RPL10/RPL11/EIF3L/RPL23A/NUCKS1/RPS20/NUP210/RPS2/RPS21/RPS28/RPL10A/RPS18/NUP58/RPL35/NUP160/RPL13/RPL7A/RPS19/RPL19/RPL13A/RPS8/NUP188/RPL15/RPL26/NUP43/RPL23/CTDP1/SUPT5H/RANBP2/FURIN/POLR2D/POLR2A/RPL5/TFAP4/RPS26/EIF3G/INPP5K/NUP155/NUP98/RPS29</td><td>60</td><td>12</td><td>ENSG00000177606/ENSG00000128016/ENSG00000153922/ENSG00000142089/ENSG00000139496/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000140564/ENSG00000181222/ENSG00000132376/ENSG00000110713</td><td>48</td><td>ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000080561/ENSG00000174444/ENSG00000089009/ENSG00000175390/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000100129/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000144231/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000130811/ENSG00000113569/ENSG00000213741</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0019083</td><td>viral transcription</td><td>56/2199</td><td>173/13522</td><td>1.11882349662366e-07</td><td>8.85642947043187e-05</td><td>ENSG00000177606/ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000128016/ENSG00000080561/ENSG00000153922/ENSG00000174444/ENSG00000089009/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000142089/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000139496/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000144231/ENSG00000181222/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000132376/ENSG00000113569/ENSG00000110713/ENSG00000213741</td><td>JUN/RPS3A/RPL3/TRIM32/RPL22/RPS4X/ZFP36/MID2/CHD1/RPL4/AC004086.1/RPS23/RPS27A/RRP1B/RPSA/IFITM3/MDFIC/LEF1/RPS6/RPL10/RPL11/RPL23A/NUCKS1/RPS20/NUP210/RPS2/RPS21/RPS28/RPL10A/RPS18/NUP58/RPL35/NUP160/RPL13/RPL7A/RPS19/RPL19/RPL13A/RPS8/NUP188/RPL15/RPL26/NUP43/RPL23/CTDP1/SUPT5H/RANBP2/POLR2D/POLR2A/RPL5/TFAP4/RPS26/INPP5K/NUP155/NUP98/RPS29</td><td>56</td><td>11</td><td>ENSG00000177606/ENSG00000128016/ENSG00000153922/ENSG00000142089/ENSG00000139496/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000181222/ENSG00000132376/ENSG00000110713</td><td>45</td><td>ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000080561/ENSG00000174444/ENSG00000089009/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000144231/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000113569/ENSG00000213741</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0042110</td><td>T cell activation</td><td>109/2199</td><td>417/13522</td><td>1.24475466906983e-07</td><td>8.85642947043187e-05</td><td>ENSG00000196126/ENSG00000112486/ENSG00000181847/ENSG00000112149/ENSG00000213809/ENSG00000090339/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000158481/ENSG00000117281/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000013725/ENSG00000010610/ENSG00000110448/ENSG00000073861/ENSG00000198286/ENSG00000128271/ENSG00000158869/ENSG00000115085/ENSG00000167984/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000107485/ENSG00000172116/ENSG00000253729/ENSG00000118520/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000196352/ENSG00000100105/ENSG00000110719/ENSG00000018280/ENSG00000172216/ENSG00000134321/ENSG00000107968/ENSG00000110651/ENSG00000122224/ENSG00000173762/ENSG00000010810/ENSG00000105205/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000197635/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000138735/ENSG00000097007/ENSG00000168036/ENSG00000102034/ENSG00000104312/ENSG00000160654/ENSG00000159958/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000151208/ENSG00000100351/ENSG00000173585/ENSG00000152495/ENSG00000090776/ENSG00000171791/ENSG00000141968/ENSG00000125084/ENSG00000162739/ENSG00000197329/ENSG00000125735/ENSG00000072274/ENSG00000177885/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000137507/ENSG00000166349/ENSG00000168071/ENSG00000213658/ENSG00000002330</td><td>HLA-DRB1/CCR6/TIGIT/CD83/KLRK1/ICAM1/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/CD1C/CD160/EOMES/CD2/CD40LG/SPN/CD8A/EGR3/CD6/CD4/CD5/TBX21/CARD11/ADORA2A/FCER1G/ZAP70/NLRC3/CD3E/TP53/ITK/RORC/VNN1/GATA3/CD8B/PRKDC/ARG1/LCK/TCF7/RPL22/CD55/PATZ1/TCIRG1/SLC11A1/CEBPB/RSAD2/MAP3K8/CD81/LY9/CD7/FYN/CLC/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/LEF1/RPS6/CCR7/RELB/DPP4/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/PDE5A/ABL1/CTNNB1/ELF4/RIPK2/CD3G/TNFRSF13C/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/DLG5/GRAP2/CCR9/CAMK4/EFNB1/BCL2/VAV1/WNT1/SLAMF6/PELI1/TNFSF14/TFRC/GRB2/SOX13/CD74/LOXL3/LRRC32/RAG1/CCDC88B/LAT/BAD</td><td>109</td><td>39</td><td>ENSG00000112149/ENSG00000090339/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000128271/ENSG00000158869/ENSG00000112299/ENSG00000118520/ENSG00000196352/ENSG00000110719/ENSG00000018280/ENSG00000172216/ENSG00000134321/ENSG00000107968/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000104856/ENSG00000185338/ENSG00000168036/ENSG00000102034/ENSG00000104312/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000197329/ENSG00000125735/ENSG00000072274/ENSG00000177885/ENSG00000137507/ENSG00000168071/ENSG00000002330</td><td>70</td><td>ENSG00000196126/ENSG00000112486/ENSG00000181847/ENSG00000213809/ENSG00000101096/ENSG00000172673/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000158481/ENSG00000117281/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000013725/ENSG00000010610/ENSG00000110448/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000167984/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000172116/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110651/ENSG00000122224/ENSG00000173762/ENSG00000010810/ENSG00000105205/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000197635/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000138735/ENSG00000097007/ENSG00000160654/ENSG00000159958/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000151208/ENSG00000100351/ENSG00000173585/ENSG00000152495/ENSG00000090776/ENSG00000171791/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0032496</td><td>response to lipopolysaccharide</td><td>77/2199</td><td>272/13522</td><td>3.15399048039311e-07</td><td>0.000199472375715529</td><td>ENSG00000177606/ENSG00000100906/ENSG00000213809/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000160791/ENSG00000163874/ENSG00000078401/ENSG00000187838/ENSG00000171223/ENSG00000165029/ENSG00000013725/ENSG00000168615/ENSG00000181104/ENSG00000131979/ENSG00000066468/ENSG00000136689/ENSG00000168329/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000163735/ENSG00000108840/ENSG00000077150/ENSG00000111961/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000025434/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000117586/ENSG00000127666/ENSG00000239839/ENSG00000012223/ENSG00000126353/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000097007/ENSG00000104312/ENSG00000154229/ENSG00000162551/ENSG00000081985/ENSG00000185187/ENSG00000162889/ENSG00000197894/ENSG00000135334/ENSG00000159674/ENSG00000005381/ENSG00000162892/ENSG00000173039/ENSG00000143507/ENSG00000188822/ENSG00000081189/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000117560/ENSG00000204397/ENSG00000112062/ENSG00000162676/ENSG00000106366/ENSG00000147465</td><td>JUN/NFKBIA/KLRK1/ADM/ICAM1/CXCL8/TNFAIP3/PTGES/TRIB1/FOS/IRAK2/CCR5/ZC3H12A/EDN1/PLSCR3/JUNB/ABCA1/CD6/ADAM9/F2R/GCH1/FGFR2/IL1RN/CX3CR1/JUND/IRAK3/CXCL2/CXCL5/HDAC5/NFKB2/SASH1/LOXL1/ARG1/GJB6/NR1H3/IRAK1/CD55/S100A8/SLC11A1/CEBPB/CMPK2/ZFP36/THBD/TNFSF4/TICAM1/DEFA3/LTF/CCR7/GGT1/DEFA1/CXCL1/ABL1/RIPK2/PRKCA/ALPL/IL12RB2/SIGIRR/MAPKAPK2/ADH5/AKIRIN2/SPON2/MPO/IL24/RELA/DUSP10/CNR2/MEF2C/PLAA/DEFA4/PPARD/PELI1/FASLG/CARD16/MAPK14/GFI1/SERPINE1/STAR</td><td>77</td><td>53</td><td>ENSG00000177606/ENSG00000100906/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000163874/ENSG00000078401/ENSG00000171223/ENSG00000165029/ENSG00000168615/ENSG00000131979/ENSG00000136689/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000108840/ENSG00000077150/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000127666/ENSG00000239839/ENSG00000012223/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000104312/ENSG00000162551/ENSG00000162889/ENSG00000135334/ENSG00000005381/ENSG00000173039/ENSG00000143507/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000204397/ENSG00000112062/ENSG00000106366</td><td>24</td><td>ENSG00000213809/ENSG00000160791/ENSG00000187838/ENSG00000013725/ENSG00000181104/ENSG00000066468/ENSG00000168329/ENSG00000163735/ENSG00000111961/ENSG00000025434/ENSG00000117586/ENSG00000126353/ENSG00000097007/ENSG00000154229/ENSG00000081985/ENSG00000185187/ENSG00000197894/ENSG00000159674/ENSG00000162892/ENSG00000188822/ENSG00000081189/ENSG00000117560/ENSG00000162676/ENSG00000147465</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0002237</td><td>response to molecule of bacterial origin</td><td>79/2199</td><td>284/13522</td><td>4.86204628739224e-07</td><td>0.000276747674678366</td><td>ENSG00000177606/ENSG00000100906/ENSG00000213809/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000160791/ENSG00000163874/ENSG00000078401/ENSG00000187838/ENSG00000171223/ENSG00000165029/ENSG00000013725/ENSG00000168615/ENSG00000181104/ENSG00000131979/ENSG00000066468/ENSG00000136689/ENSG00000168329/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000163735/ENSG00000108840/ENSG00000077150/ENSG00000111961/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000025434/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000163251/ENSG00000117586/ENSG00000127666/ENSG00000239839/ENSG00000128604/ENSG00000012223/ENSG00000126353/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000097007/ENSG00000104312/ENSG00000154229/ENSG00000162551/ENSG00000081985/ENSG00000185187/ENSG00000162889/ENSG00000197894/ENSG00000135334/ENSG00000159674/ENSG00000005381/ENSG00000162892/ENSG00000173039/ENSG00000143507/ENSG00000188822/ENSG00000081189/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000117560/ENSG00000204397/ENSG00000112062/ENSG00000162676/ENSG00000106366/ENSG00000147465</td><td>JUN/NFKBIA/KLRK1/ADM/ICAM1/CXCL8/TNFAIP3/PTGES/TRIB1/FOS/IRAK2/CCR5/ZC3H12A/EDN1/PLSCR3/JUNB/ABCA1/CD6/ADAM9/F2R/GCH1/FGFR2/IL1RN/CX3CR1/JUND/IRAK3/CXCL2/CXCL5/HDAC5/NFKB2/SASH1/LOXL1/ARG1/GJB6/NR1H3/IRAK1/CD55/S100A8/SLC11A1/CEBPB/CMPK2/ZFP36/THBD/FZD5/TNFSF4/TICAM1/DEFA3/IRF5/LTF/CCR7/GGT1/DEFA1/CXCL1/ABL1/RIPK2/PRKCA/ALPL/IL12RB2/SIGIRR/MAPKAPK2/ADH5/AKIRIN2/SPON2/MPO/IL24/RELA/DUSP10/CNR2/MEF2C/PLAA/DEFA4/PPARD/PELI1/FASLG/CARD16/MAPK14/GFI1/SERPINE1/STAR</td><td>79</td><td>55</td><td>ENSG00000177606/ENSG00000100906/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000163874/ENSG00000078401/ENSG00000171223/ENSG00000165029/ENSG00000168615/ENSG00000131979/ENSG00000136689/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000108840/ENSG00000077150/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000163251/ENSG00000127666/ENSG00000239839/ENSG00000128604/ENSG00000012223/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000104312/ENSG00000162551/ENSG00000162889/ENSG00000135334/ENSG00000005381/ENSG00000173039/ENSG00000143507/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000204397/ENSG00000112062/ENSG00000106366</td><td>24</td><td>ENSG00000213809/ENSG00000160791/ENSG00000187838/ENSG00000013725/ENSG00000181104/ENSG00000066468/ENSG00000168329/ENSG00000163735/ENSG00000111961/ENSG00000025434/ENSG00000117586/ENSG00000126353/ENSG00000097007/ENSG00000154229/ENSG00000081985/ENSG00000185187/ENSG00000197894/ENSG00000159674/ENSG00000162892/ENSG00000188822/ENSG00000081189/ENSG00000117560/ENSG00000162676/ENSG00000147465</td></tr>
			
		  
		</table>
	</div>				
	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
		
        (1) Category: Classification of GO databases, including biological processes(BP), cellular components(CC), molecular function(MF).<br />
        (2) ID: Unique identification id of Gene Ontology database.<br />
        (3) Description: Function description corresponding to the GO number.<br />
        (4) GeneRatio: Ratio between the number of differentially expressed genes in each GO term and all differentially expressed genes that can be found in GO database.<br />
        (5) BgRatio: In background GO database, the ratio of all genes concerning this GO term to all genes.<br />
        (6) pvalue: Statistics category term; abbreviation for probability value.<br />
        (7) padj: Adjusted p-value. Generally, GO Terms with Corrected_pValue < 0.05 are significant enrichment.<br />
        (8) geneID: Differentially expressed genes in this term.<br />
        (9) geneName: Differentially expressed genes in this term.<br />
        (10) Count: The number of difference gene annotated to GO number.<br />
        (11) Up: Number of up expressed genes concerning this GO term.<br />
        (12) Up_Gene_id: Up expressed gene id concerning this GO term.<br />
        (13) Down: Number of down expressed genes concerning this GO term.<br />
        (14) Down_Gene_id: Down expressed gene id concerning this GO term.<br /> 

		

		
</ul>
</div>
	<br />


</div>

<!------------------------------5.1 GO富集分析------------------------------>
<div id="page">
	
    <p class="paragraph">In the results of the GO enrichment analysis, the most significant 30 Terms were selected for display. If the results are less than 30, all terms would be drawn, as shown in the following figure. In this figure, the abscissa is GO Term. The ordinate is the significance level of GO Term enrichment. Higher values correspond to higher significance. The different colors represent the three GO subclasses of BP, CC,MF.</p>
    
    
		
    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGGObar_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGGObar_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGGObar_Content_pane_group1vsgroup2" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGObar_Content_list_group1vsgroup2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group1vsgroup2.bar.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGObar_Content_pane_group2vssample1" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGObar_Content_list_group2vssample1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group2vssample1.bar.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGObar_Content_pane_sample1vssample5" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGObar_Content_list_sample1vssample5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/sample1vssample5.bar.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGGObar_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGGObar_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGGObar_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGObar_Content_list_group1vsgroup2"
            data-toggle="list" href="#gallery_figure_DEGGObar_Content_pane_group1vsgroup2" role="tab"
            aria-controls="group1vsgroup2">group1vsgroup2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGObar_Content_list_group2vssample1"
            data-toggle="list" href="#gallery_figure_DEGGObar_Content_pane_group2vssample1" role="tab"
            aria-controls="group2vssample1">group2vssample1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGObar_Content_list_sample1vssample5"
            data-toggle="list" href="#gallery_figure_DEGGObar_Content_pane_sample1vssample5" role="tab"
            aria-controls="sample1vssample5">sample1vssample5</a>
          
        </div>
      </div>
    </div>
    </p>

	<p class="name_fig">Figure 4.15 GO enrichment analysis histogram </p>
        
	
	
	<!--<p class="premark">横坐标为-log<sub>10</sub>（padj），纵坐标为GO条目，n表示富集到该条目的基因个数。</p>-->

	<p class="cenmark">The abscissa in the figure is GO Term, and the ordinate is GO Term's level of significance of enrichment, expressed as -log10(padj). Different colors represent different functional categories</p>

	
	
	<br />

	
    <p class="paragraph">From the GO enrichment analysis results, the most significant 30 GO Terms were selected for display. If the results are less than 30, all Terms would be drawn. In the figure, the abscissa is the ratio of the number of differential genes linked with the GO Term to the total number of differential genes, and the ordinate is GO Term. The size of a point represents the number of genes annotated to a specific GO Term, and the color from red to purple represents the significant level of the enrichment.</p>
    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGGOdot_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGGOdot_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGGOdot_Content_pane_group1vsgroup2_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGOdot_Content_list_group1vsgroup2_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group1vsgroup2.dot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGOdot_Content_pane_group2vssample1_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGOdot_Content_list_group2vssample1_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group2vssample1.dot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGOdot_Content_pane_sample1vssample5_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGOdot_Content_list_sample1vssample5_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/sample1vssample5.dot.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGGOdot_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGGOdot_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGGOdot_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGOdot_Content_list_group1vsgroup2_all"
            data-toggle="list" href="#gallery_figure_DEGGOdot_Content_pane_group1vsgroup2_all" role="tab"
            aria-controls="group1vsgroup2_all">group1vsgroup2_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGOdot_Content_list_group2vssample1_all"
            data-toggle="list" href="#gallery_figure_DEGGOdot_Content_pane_group2vssample1_all" role="tab"
            aria-controls="group2vssample1_all">group2vssample1_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGOdot_Content_list_sample1vssample5_all"
            data-toggle="list" href="#gallery_figure_DEGGOdot_Content_pane_sample1vssample5_all" role="tab"
            aria-controls="sample1vssample5_all">sample1vssample5_all</a>
          
        </div>
      </div>
    </div>
    </p>

	<p class="name_fig">Figure 4.16 GO Enrichment Analysis Scatter Plot </p>
    <p class="premark">The abscissa in the graph is the ratio of the differential gene number to the total number of differential genes on the GO Term, and the ordinate is GO Term.</p>
        <br />
    <p class="paragraph">(Directed Acyclic Graph，DAG) could visualize the enriched GO Term of differential expression genes and its hierarchy. In this graph, branch means the hierarchical relation, and the function ranges become increasingly specific from the top to bottom. In general, the top 5 results of GO enrichment analysis are chosen as the main nodes (shown by box) in directed acyclic graph, and related GO Term are shown together by hierarchical connections. The enrichment degree is illustrated by color shades, the darker the shades are, the higher the enrichment degree is. In this project, DAG of biological process, molecular function and cellular component were drawn sequentially.</p>
        <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGGODAG_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGGODAG_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GObp" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GObp" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group1vsgroup2.all.GObp_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GOcc" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GOcc" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group1vsgroup2.all.GOcc_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GOmf" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GOmf" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group1vsgroup2.all.GOmf_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GObp" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GObp" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group2vssample1.all.GObp_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GOcc" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GOcc" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group2vssample1.all.GOcc_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GOmf" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GOmf" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/group2vssample1.all.GOmf_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GObp" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GObp" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/sample1vssample5.all.GObp_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GOcc" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GOcc" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/sample1vssample5.all.GOcc_DAG.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GOmf" role="tabpanel"
            aria-labelledby="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GOmf" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/GO/sample1vssample5.all.GOmf_DAG.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGGODAG_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGGODAG_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGGODAG_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GObp"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GObp" role="tab"
            aria-controls="group1vsgroup2_all_GObp">group1vsgroup2_all_GObp</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GOcc"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GOcc" role="tab"
            aria-controls="group1vsgroup2_all_GOcc">group1vsgroup2_all_GOcc</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group1vsgroup2_all_GOmf"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group1vsgroup2_all_GOmf" role="tab"
            aria-controls="group1vsgroup2_all_GOmf">group1vsgroup2_all_GOmf</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GObp"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GObp" role="tab"
            aria-controls="group2vssample1_all_GObp">group2vssample1_all_GObp</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GOcc"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GOcc" role="tab"
            aria-controls="group2vssample1_all_GOcc">group2vssample1_all_GOcc</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_group2vssample1_all_GOmf"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_group2vssample1_all_GOmf" role="tab"
            aria-controls="group2vssample1_all_GOmf">group2vssample1_all_GOmf</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GObp"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GObp" role="tab"
            aria-controls="sample1vssample5_all_GObp">sample1vssample5_all_GObp</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GOcc"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GOcc" role="tab"
            aria-controls="sample1vssample5_all_GOcc">sample1vssample5_all_GOcc</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGGODAG_Content_list_sample1vssample5_all_GOmf"
            data-toggle="list" href="#gallery_figure_DEGGODAG_Content_pane_sample1vssample5_all_GOmf" role="tab"
            aria-controls="sample1vssample5_all_GOmf">sample1vssample5_all_GOmf</a>
          
        </div>
      </div>
    </div>
    </p>
   <p class="name_fig">Figure 4.17 GO Enrichment Analysis DAG Chart</p>
   <p class="premark">Each node represents a GO term, and the box represents the enrichment level of TOP5 GOTerms. The depth of the color represents the degree of enrichment, the darker the color is, the higherthe enrichment degree is. Each node shows the name of the term and the padj of enrichment analysis</p> 
	<br />
	<p class="paragraph">Result directory of GO enrichment analysis of mRNA gene expression level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/6.DiffEnrich_mRNA/1.GO" title="点击打开" >GO</a></p>
	
</div>

<!----------------------------------5.2 KEGG富集分析---------------------------------->
	<h4 id="result52">4.6.2 KEGG Enrichment Analysis</h4>
    <p class="paragraph">The interactions of multiple genes may be involved in certain biological functions. KEGG (Kyoto Encyclopedia of Genes and Genomes) is a collection of manually curated databases dealing with genomes, biological pathways, diseases, drugs, and chemical substances. KEGG is utilized for bioinformatics research and education, including data analysis in genomics, metagenomics, metabolomics and other omics studies. Pathway enrichment analysis identifies significantly enriched metabolic pathways or signal transduction pathways associated with differentially expressed genes compared with the whole genome background.</p>
</p>
	
   

</br>
	<p class="name_fig">Table 4.7 Partially displayed results of KEGG enrichment analysis of differential genes</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>KEGGID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>keggID</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th></tr>
			
		  
			
				<tr><td>hsa04640</td><td>Hematopoietic cell lineage</td><td>32/1065</td><td>79/5923</td><td>2.02716750270934e-06</td><td>0.000630449093342605</td><td>ENSG00000198502/ENSG00000158481/ENSG00000116824/ENSG00000153563/ENSG00000184371/ENSG00000010610/ENSG00000182578/ENSG00000110448/ENSG00000115594/ENSG00000198851/ENSG00000115590/ENSG00000172116/ENSG00000196352/ENSG00000115232/ENSG00000173762/XLOC_165892/ENSG00000185291/ENSG00000091409/ENSG00000137070/ENSG00000160654/ENSG00000091181/ENSG00000119535/ENSG00000168685/ENSG00000104921/ENSG00000167286/ENSG00000158477/XLOC_165944/ENSG00000198223/ENSG00000166825/ENSG00000072274/ENSG00000158488/ENSG00000164171</td><td>HLA-DRB5/CD1C/CD2/CD8A/CSF1/CD4/CSF1R/CD5/IL1R1/CD3E/IL1R2/CD8B/CD55/ITGA4/CD7/XLOC_165892/IL3RA/ITGA6/IL11RA/CD3G/IL5RA/CSF3R/IL7R/FCER2/CD3D/CD1A/XLOC_165944/CSF2RA/ANPEP/TFRC/CD1E/ITGA2</td><td>hsa:3127/hsa:911/hsa:914/hsa:925/hsa:1435/hsa:920/hsa:1436/hsa:921/hsa:3554/hsa:916/hsa:7850/hsa:926/hsa:1604/hsa:3676/hsa:924/hsa:3119/hsa:3563/hsa:3655/hsa:3590/hsa:917/hsa:3568/hsa:1441/hsa:3575/hsa:2208/hsa:915/hsa:909/hsa:3125/hsa:1438/hsa:290/hsa:7037/hsa:913/hsa:3673</td><td>32</td><td>9</td><td>ENSG00000184371/ENSG00000115594/ENSG00000115590/ENSG00000196352/ENSG00000185291/ENSG00000119535/ENSG00000198223/ENSG00000166825/ENSG00000072274</td><td>23</td><td>ENSG00000198502/ENSG00000158481/ENSG00000116824/ENSG00000153563/ENSG00000010610/ENSG00000182578/ENSG00000110448/ENSG00000198851/ENSG00000172116/ENSG00000115232/ENSG00000173762/XLOC_165892/ENSG00000091409/ENSG00000137070/ENSG00000160654/ENSG00000091181/ENSG00000168685/ENSG00000104921/ENSG00000167286/ENSG00000158477/XLOC_165944/ENSG00000158488/ENSG00000164171</td></tr>
			
		  
			
				<tr><td>hsa04660</td><td>T cell receptor signaling pathway</td><td>35/1065</td><td>96/5923</td><td>1.15100349179265e-05</td><td>0.00171731840888396</td><td>ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000170345/ENSG00000178562/ENSG00000146232/ENSG00000102245/ENSG00000153563/ENSG00000010610/ENSG00000198286/ENSG00000115085/ENSG00000114423/ENSG00000198851/ENSG00000113263/ENSG00000172116/ENSG00000182866/ENSG00000198821/ENSG00000107968/ENSG00000160293/ENSG00000010810/ENSG00000172575/ENSG00000105647/ENSG00000124181/ENSG00000006062/ENSG00000160654/XLOC_164842/ENSG00000173039/ENSG00000167286/ENSG00000100351/ENSG00000141968/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>JUN/NFKBIA/NFATC2/FOS/CD28/NFKBIE/CD40LG/CD8A/CD4/CARD11/ZAP70/CBLB/CD3E/ITK/CD8B/LCK/CD247/MAP3K8/VAV2/FYN/RASGRP1/PIK3R2/PLCG1/MAP3K14/CD3G/XLOC_164842/RELA/CD3D/GRAP2/VAV1/LCP2/GRB2/MAPK13/MAPK14/LAT</td><td>hsa:3725/hsa:4792/hsa:4773/hsa:2353/hsa:940/hsa:4794/hsa:959/hsa:925/hsa:920/hsa:84433/hsa:7535/hsa:868/hsa:916/hsa:3702/hsa:926/hsa:3932/hsa:919/hsa:1326/hsa:7410/hsa:2534/hsa:10125/hsa:5296/hsa:5335/hsa:9020/hsa:917/hsa:4793/hsa:5970/hsa:915/hsa:9402/hsa:7409/hsa:3937/hsa:2885/hsa:5603/hsa:1432/hsa:27040</td><td>35</td><td>12</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/ENSG00000107968/XLOC_164842/ENSG00000173039/ENSG00000141968/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>23</td><td>ENSG00000101096/ENSG00000178562/ENSG00000102245/ENSG00000153563/ENSG00000010610/ENSG00000198286/ENSG00000115085/ENSG00000114423/ENSG00000198851/ENSG00000113263/ENSG00000172116/ENSG00000182866/ENSG00000198821/ENSG00000160293/ENSG00000010810/ENSG00000172575/ENSG00000105647/ENSG00000124181/ENSG00000006062/ENSG00000160654/ENSG00000167286/ENSG00000100351/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04658</td><td>Th1 and Th2 cell differentiation</td><td>30/1065</td><td>79/5923</td><td>1.93854624034982e-05</td><td>0.00171731840888396</td><td>ENSG00000198502/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000100385/ENSG00000170345/ENSG00000146232/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000107485/ENSG00000182866/ENSG00000198821/XLOC_165570/ENSG00000178573/XLOC_165892/ENSG00000124181/ENSG00000020633/ENSG00000160654/ENSG00000081985/XLOC_164842/ENSG00000173039/ENSG00000167286/XLOC_165944/ENSG00000101384/ENSG00000105397/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>HLA-DRB5/JUN/NFKBIA/NFATC2/IL2RB/FOS/NFKBIE/CD4/TBX21/ZAP70/CD3E/GATA3/LCK/CD247/XLOC_165570/MAF/XLOC_165892/PLCG1/RUNX3/CD3G/IL12RB2/XLOC_164842/RELA/CD3D/XLOC_165944/JAG1/TYK2/MAPK13/MAPK14/LAT</td><td>hsa:3127/hsa:3725/hsa:4792/hsa:4773/hsa:3560/hsa:2353/hsa:4794/hsa:920/hsa:30009/hsa:7535/hsa:916/hsa:2625/hsa:3932/hsa:919/hsa:3460/hsa:4094/hsa:3119/hsa:5335/hsa:864/hsa:917/hsa:3595/hsa:4793/hsa:5970/hsa:915/hsa:3125/hsa:182/hsa:7297/hsa:5603/hsa:1432/hsa:27040</td><td>30</td><td>11</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/XLOC_165570/XLOC_164842/ENSG00000173039/ENSG00000101384/ENSG00000105397/ENSG00000156711/ENSG00000112062</td><td>19</td><td>ENSG00000198502/ENSG00000101096/ENSG00000100385/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000107485/ENSG00000182866/ENSG00000198821/ENSG00000178573/XLOC_165892/ENSG00000124181/ENSG00000020633/ENSG00000160654/ENSG00000081985/ENSG00000167286/XLOC_165944/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04064</td><td>NF-kappa B signaling pathway</td><td>32/1065</td><td>87/5923</td><td>2.20876965772857e-05</td><td>0.00171731840888396</td><td>ENSG00000100906/ENSG00000090339/ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000118503/ENSG00000140379/ENSG00000102245/ENSG00000131323/ENSG00000198286/ENSG00000081041/ENSG00000135960/ENSG00000115594/ENSG00000115085/ENSG00000077150/ENSG00000082512/ENSG00000182866/ENSG00000184216/ENSG00000186197/ENSG00000127666/ENSG00000124181/ENSG00000104856/ENSG00000006062/ENSG00000141527/ENSG00000101017/ENSG00000159958/ENSG00000149311/ENSG00000143799/ENSG00000173039/ENSG00000171791/ENSG00000125735/ENSG00000213658</td><td>NFKBIA/ICAM1/CXCL8/GADD45B/PLAU/TNFAIP3/BCL2A1/CD40LG/TRAF3/CARD11/CXCL2/EDAR/IL1R1/ZAP70/NFKB2/TRAF5/LCK/IRAK1/EDARADD/TICAM1/PLCG1/RELB/MAP3K14/CARD14/CD40/TNFRSF13C/ATM/PARP1/RELA/BCL2/TNFSF14/LAT</td><td>hsa:4792/hsa:3383/hsa:3576/hsa:4616/hsa:5328/hsa:7128/hsa:597/hsa:959/hsa:7187/hsa:84433/hsa:2920/hsa:10913/hsa:3554/hsa:7535/hsa:4791/hsa:7188/hsa:3932/hsa:3654/hsa:128178/hsa:148022/hsa:5335/hsa:5971/hsa:9020/hsa:79092/hsa:958/hsa:115650/hsa:472/hsa:142/hsa:5970/hsa:596/hsa:8740/hsa:27040</td><td>32</td><td>17</td><td>ENSG00000100906/ENSG00000090339/ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000118503/ENSG00000140379/ENSG00000131323/ENSG00000081041/ENSG00000115594/ENSG00000077150/ENSG00000184216/ENSG00000127666/ENSG00000104856/ENSG00000141527/ENSG00000173039/ENSG00000125735</td><td>15</td><td>ENSG00000102245/ENSG00000198286/ENSG00000135960/ENSG00000115085/ENSG00000082512/ENSG00000182866/ENSG00000186197/ENSG00000124181/ENSG00000006062/ENSG00000101017/ENSG00000159958/ENSG00000149311/ENSG00000143799/ENSG00000171791/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04659</td><td>Th17 cell differentiation</td><td>32/1065</td><td>89/5923</td><td>3.74261880689674e-05</td><td>0.00232790889788977</td><td>ENSG00000198502/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000100385/ENSG00000170345/ENSG00000069667/ENSG00000146232/ENSG00000010610/ENSG00000100644/ENSG00000073861/ENSG00000115594/ENSG00000115085/ENSG00000198851/ENSG00000143365/ENSG00000107485/ENSG00000182866/ENSG00000198821/XLOC_165570/ENSG00000103522/XLOC_165892/ENSG00000124181/ENSG00000160654/XLOC_164842/ENSG00000173039/ENSG00000167286/XLOC_165944/ENSG00000104998/ENSG00000105397/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>HLA-DRB5/JUN/NFKBIA/NFATC2/IL2RB/FOS/RORA/NFKBIE/CD4/HIF1A/TBX21/IL1R1/ZAP70/CD3E/RORC/GATA3/LCK/CD247/XLOC_165570/IL21R/XLOC_165892/PLCG1/CD3G/XLOC_164842/RELA/CD3D/XLOC_165944/IL27RA/TYK2/MAPK13/MAPK14/LAT</td><td>hsa:3127/hsa:3725/hsa:4792/hsa:4773/hsa:3560/hsa:2353/hsa:6095/hsa:4794/hsa:920/hsa:3091/hsa:30009/hsa:3554/hsa:7535/hsa:916/hsa:6097/hsa:2625/hsa:3932/hsa:919/hsa:3460/hsa:50615/hsa:3119/hsa:5335/hsa:917/hsa:4793/hsa:5970/hsa:915/hsa:3125/hsa:9466/hsa:7297/hsa:5603/hsa:1432/hsa:27040</td><td>32</td><td>12</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/ENSG00000100644/ENSG00000115594/XLOC_165570/XLOC_164842/ENSG00000173039/ENSG00000105397/ENSG00000156711/ENSG00000112062</td><td>20</td><td>ENSG00000198502/ENSG00000101096/ENSG00000100385/ENSG00000069667/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000143365/ENSG00000107485/ENSG00000182866/ENSG00000198821/ENSG00000103522/XLOC_165892/ENSG00000124181/ENSG00000160654/ENSG00000167286/XLOC_165944/ENSG00000104998/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04060</td><td>Cytokine-cytokine receptor interaction</td><td>54/1065</td><td>185/5923</td><td>0.000100987302324078</td><td>0.0051199393317285</td><td>ENSG00000112486/ENSG00000169429/ENSG00000100385/ENSG00000099985/ENSG00000115604/ENSG00000160791/ENSG00000102245/ENSG00000184371/ENSG00000010610/ENSG00000182578/ENSG00000136689/ENSG00000168329/ENSG00000161921/ENSG00000115607/ENSG00000081041/ENSG00000135960/ENSG00000115594/ENSG00000163735/ENSG00000115590/XLOC_165570/ENSG00000103522/ENSG00000114739/ENSG00000160683/ENSG00000117586/ENSG00000054967/ENSG00000135414/ENSG00000172215/ENSG00000008517/ENSG00000185291/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000101017/ENSG00000130513/ENSG00000163739/ENSG00000137070/ENSG00000151882/ENSG00000081985/ENSG00000159958/ENSG00000162892/ENSG00000091181/ENSG00000119535/ENSG00000006327/ENSG00000128342/ENSG00000168685/ENSG00000173585/ENSG00000104998/ENSG00000121966/ENSG00000186810/ENSG00000107779/ENSG00000125735/ENSG00000117560/ENSG00000198223/ENSG00000123612</td><td>CCR6/CXCL8/IL2RB/OSM/IL18R1/CCR5/CD40LG/CSF1/CD4/CSF1R/IL1RN/CX3CR1/CXCL16/IL18RAP/CXCL2/EDAR/IL1R1/CXCL5/IL1R2/XLOC_165570/IL21R/ACVR2B/CXCR5/TNFSF4/RELT/GDF11/CXCR6/IL32/IL3RA/TGFB2/CCR7/XCL2/CD40/GDF15/CXCL1/IL11RA/CCL28/IL12RB2/TNFRSF13C/IL24/IL5RA/CSF3R/TNFRSF12A/LIF/IL7R/CCR9/IL27RA/CXCR4/CXCR3/BMPR1A/TNFSF14/FASLG/CSF2RA/ACVR1C</td><td>hsa:1235/hsa:3576/hsa:3560/hsa:5008/hsa:8809/hsa:1234/hsa:959/hsa:1435/hsa:920/hsa:1436/hsa:3557/hsa:1524/hsa:58191/hsa:8807/hsa:2920/hsa:10913/hsa:3554/hsa:6374/hsa:7850/hsa:3460/hsa:50615/hsa:93/hsa:643/hsa:7292/hsa:84957/hsa:10220/hsa:10663/hsa:9235/hsa:3563/hsa:7042/hsa:1236/hsa:6846/hsa:958/hsa:9518/hsa:2919/hsa:3590/hsa:56477/hsa:3595/hsa:115650/hsa:11009/hsa:3568/hsa:1441/hsa:51330/hsa:3976/hsa:3575/hsa:10803/hsa:9466/hsa:7852/hsa:2833/hsa:657/hsa:8740/hsa:356/hsa:1438/hsa:130399</td><td>54</td><td>21</td><td>ENSG00000169429/ENSG00000099985/ENSG00000115604/ENSG00000184371/ENSG00000136689/ENSG00000161921/ENSG00000115607/ENSG00000081041/ENSG00000115594/ENSG00000115590/XLOC_165570/ENSG00000054967/ENSG00000185291/ENSG00000130513/ENSG00000163739/ENSG00000119535/ENSG00000006327/ENSG00000128342/ENSG00000121966/ENSG00000125735/ENSG00000198223</td><td>33</td><td>ENSG00000112486/ENSG00000100385/ENSG00000160791/ENSG00000102245/ENSG00000010610/ENSG00000182578/ENSG00000168329/ENSG00000135960/ENSG00000163735/ENSG00000103522/ENSG00000114739/ENSG00000160683/ENSG00000117586/ENSG00000135414/ENSG00000172215/ENSG00000008517/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000101017/ENSG00000137070/ENSG00000151882/ENSG00000081985/ENSG00000159958/ENSG00000162892/ENSG00000091181/ENSG00000168685/ENSG00000173585/ENSG00000104998/ENSG00000186810/ENSG00000107779/ENSG00000117560/ENSG00000123612</td></tr>
			
		  
			
				<tr><td>hsa04380</td><td>Osteoclast differentiation</td><td>39/1065</td><td>122/5923</td><td>0.000115239792032474</td><td>0.0051199393317285</td><td>ENSG00000125740/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000170345/ENSG00000175592/ENSG00000171223/ENSG00000184371/ENSG00000182578/ENSG00000184557/ENSG00000075426/ENSG00000130522/ENSG00000204577/ENSG00000115594/XLOC_165453/ENSG00000077150/ENSG00000182866/XLOC_165520/XLOC_165570/ENSG00000132170/ENSG00000010810/ENSG00000187116/ENSG00000105647/ENSG00000092969/ENSG00000104856/ENSG00000185338/ENSG00000118260/ENSG00000161011/ENSG00000006062/XLOC_164866/ENSG00000158517/ENSG00000173039/ENSG00000152495/ENSG00000105397/ENSG00000108984/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>FOSB/JUN/NFKBIA/NFATC2/FOS/FOSL1/JUNB/CSF1/CSF1R/SOCS3/FOSL2/JUND/LILRB3/IL1R1/XLOC_165453/NFKB2/LCK/XLOC_165520/XLOC_165570/PPARG/FYN/LILRA5/PIK3R2/TGFB2/RELB/SOCS1/CREB1/SQSTM1/MAP3K14/XLOC_164866/NCF1/RELA/CAMK4/TYK2/MAP2K6/LCP2/GRB2/MAPK13/MAPK14</td><td>hsa:2354/hsa:3725/hsa:4792/hsa:4773/hsa:2353/hsa:8061/hsa:3726/hsa:1435/hsa:1436/hsa:9021/hsa:2355/hsa:3727/hsa:102725035/hsa:3554/hsa:11027/hsa:4791/hsa:3932/hsa:353514/hsa:3460/hsa:5468/hsa:2534/hsa:353514/hsa:5296/hsa:7042/hsa:5971/hsa:8651/hsa:1385/hsa:8878/hsa:9020/hsa:8878/hsa:653361/hsa:5970/hsa:814/hsa:7297/hsa:5608/hsa:3937/hsa:2885/hsa:5603/hsa:1432</td><td>39</td><td>30</td><td>ENSG00000125740/ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000175592/ENSG00000171223/ENSG00000184371/ENSG00000184557/ENSG00000075426/ENSG00000130522/ENSG00000204577/ENSG00000115594/XLOC_165453/ENSG00000077150/XLOC_165520/XLOC_165570/ENSG00000132170/ENSG00000187116/ENSG00000104856/ENSG00000185338/ENSG00000161011/XLOC_164866/ENSG00000158517/ENSG00000173039/ENSG00000105397/ENSG00000108984/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>9</td><td>ENSG00000101096/ENSG00000182578/ENSG00000182866/ENSG00000010810/ENSG00000105647/ENSG00000092969/ENSG00000118260/ENSG00000006062/ENSG00000152495</td></tr>
			
		  
			
				<tr><td>hsa04668</td><td>TNF signaling pathway</td><td>33/1065</td><td>100/5923</td><td>0.000195014405066666</td><td>0.00758118499696665</td><td>ENSG00000177606/ENSG00000100906/ENSG00000090339/ENSG00000118503/ENSG00000170345/ENSG00000115604/ENSG00000078401/ENSG00000171223/ENSG00000131323/ENSG00000184371/ENSG00000184557/ENSG00000081041/ENSG00000163735/ENSG00000082512/ENSG00000100985/ENSG00000172216/ENSG00000107968/ENSG00000105647/ENSG00000069399/ENSG00000128272/ENSG00000118260/ENSG00000006062/ENSG00000163739/ENSG00000100784/ENSG00000107175/ENSG00000173039/ENSG00000128342/ENSG00000247077/ENSG00000101384/ENSG00000108984/ENSG00000156711/ENSG00000112062/ENSG00000034152</td><td>JUN/NFKBIA/ICAM1/TNFAIP3/FOS/IL18R1/EDN1/JUNB/TRAF3/CSF1/SOCS3/CXCL2/CXCL5/TRAF5/MMP9/CEBPB/MAP3K8/PIK3R2/BCL3/ATF4/CREB1/MAP3K14/CXCL1/RPS6KA5/CREB3/RELA/LIF/PGAM5/JAG1/MAP2K6/MAPK13/MAPK14/MAP2K3</td><td>hsa:3725/hsa:4792/hsa:3383/hsa:7128/hsa:2353/hsa:8809/hsa:1906/hsa:3726/hsa:7187/hsa:1435/hsa:9021/hsa:2920/hsa:6374/hsa:7188/hsa:4318/hsa:1051/hsa:1326/hsa:5296/hsa:602/hsa:468/hsa:1385/hsa:9020/hsa:2919/hsa:9252/hsa:10488/hsa:5970/hsa:3976/hsa:192111/hsa:182/hsa:5608/hsa:5603/hsa:1432/hsa:5606</td><td>33</td><td>26</td><td>ENSG00000177606/ENSG00000100906/ENSG00000090339/ENSG00000118503/ENSG00000170345/ENSG00000115604/ENSG00000078401/ENSG00000171223/ENSG00000131323/ENSG00000184371/ENSG00000184557/ENSG00000081041/ENSG00000100985/ENSG00000172216/ENSG00000107968/ENSG00000069399/ENSG00000128272/ENSG00000163739/ENSG00000107175/ENSG00000173039/ENSG00000128342/ENSG00000101384/ENSG00000108984/ENSG00000156711/ENSG00000112062/ENSG00000034152</td><td>7</td><td>ENSG00000163735/ENSG00000082512/ENSG00000105647/ENSG00000118260/ENSG00000006062/ENSG00000100784/ENSG00000247077</td></tr>
			
		  
			
				<tr><td>hsa05202</td><td>Transcriptional misregulation in cancer</td><td>46/1065</td><td>159/5923</td><td>0.000404866397152834</td><td>0.0139903832793924</td><td>ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000100385/ENSG00000175197/ENSG00000119508/ENSG00000127152/ENSG00000140379/ENSG00000144802/ENSG00000244405/ENSG00000058091/ENSG00000182578/ENSG00000133895/ENSG00000116717/ENSG00000196092/ENSG00000130222/ENSG00000141510/ENSG00000115590/ENSG00000100985/ENSG00000178573/ENSG00000132170/ENSG00000172216/ENSG00000068323/ENSG00000158711/ENSG00000239839/ENSG00000275714/ENSG00000101017/ENSG00000118971/ENSG00000149311/ENSG00000203852/ENSG00000203811/ENSG00000005381/ENSG00000104885/ENSG00000173039/ENSG00000171843/ENSG00000116809/ENSG00000123080/ENSG00000169696/ENSG00000113916/ENSG00000118058/ENSG00000171988/ENSG00000081189/ENSG00000146674/ENSG00000007516/ENSG00000139626/ENSG00000136997</td><td>CXCL8/GADD45B/PLAU/IL2RB/DDIT3/NR4A3/BCL11B/BCL2A1/NFKBIZ/ETV5/CDK14/CSF1R/MEN1/GADD45A/PAX5/GADD45G/TP53/IL1R2/MMP9/MAF/PPARG/CEBPB/TFE3/ELK4/DEFA3/HIST1H3A/CD40/CCND2/ATM/HIST2H3A/HIST2H3C/MPO/DOT1L/RELA/MLLT3/ZBTB17/CDKN2C/ASPSCR1/BCL6/KMT2A/JMJD1C/MEF2C/IGFBP3/BAIAP3/ITGB7/MYC</td><td>hsa:3576/hsa:4616/hsa:5328/hsa:3560/hsa:1649/hsa:8013/hsa:64919/hsa:597/hsa:64332/hsa:2119/hsa:5218/hsa:1436/hsa:4221/hsa:1647/hsa:5079/hsa:10912/hsa:7157/hsa:7850/hsa:4318/hsa:4094/hsa:5468/hsa:1051/hsa:7030/hsa:2005/hsa:1668/hsa:8350/hsa:958/hsa:894/hsa:472/hsa:333932/hsa:126961/hsa:4353/hsa:84444/hsa:5970/hsa:4300/hsa:7709/hsa:1031/hsa:79058/hsa:604/hsa:4297/hsa:221037/hsa:4208/hsa:3486/hsa:8938/hsa:3695/hsa:4609</td><td>46</td><td>28</td><td>ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000175197/ENSG00000119508/ENSG00000140379/ENSG00000144802/ENSG00000244405/ENSG00000058091/ENSG00000116717/ENSG00000130222/ENSG00000115590/ENSG00000100985/ENSG00000132170/ENSG00000172216/ENSG00000068323/ENSG00000239839/ENSG00000275714/ENSG00000203852/ENSG00000203811/ENSG00000005381/ENSG00000104885/ENSG00000173039/ENSG00000116809/ENSG00000123080/ENSG00000113916/ENSG00000171988/ENSG00000007516</td><td>18</td><td>ENSG00000100385/ENSG00000127152/ENSG00000182578/ENSG00000133895/ENSG00000196092/ENSG00000141510/ENSG00000178573/ENSG00000158711/ENSG00000101017/ENSG00000118971/ENSG00000149311/ENSG00000171843/ENSG00000169696/ENSG00000118058/ENSG00000081189/ENSG00000146674/ENSG00000139626/ENSG00000136997</td></tr>
			
		  
			
				<tr><td>hsa04514</td><td>Cell adhesion molecules (CAMs)</td><td>35/1065</td><td>114/5923</td><td>0.000597510384625838</td><td>0.0185825729618635</td><td>XLOC_165816/ENSG00000198502/ENSG00000181847/XLOC_165828/ENSG00000090339/ENSG00000178562/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000130202/ENSG00000013725/ENSG00000010610/XLOC_165954/ENSG00000172116/ENSG00000149294/ENSG00000196338/ENSG00000115232/ENSG00000196358/XLOC_165971/XLOC_165892/ENSG00000067141/ENSG00000090863/ENSG00000091409/ENSG00000101017/XLOC_165898/ENSG00000088827/ENSG00000091129/ENSG00000107738/XLOC_165944/ENSG00000138448/ENSG00000170558/ENSG00000115884/ENSG00000139626/ENSG00000177707</td><td>XLOC_165816/HLA-DRB5/TIGIT/XLOC_165828/ICAM1/CD28/CD2/CD40LG/SPN/CD8A/NECTIN2/CD6/CD4/XLOC_165954/CD8B/NCAM1/NLGN3/ITGA4/NTNG2/XLOC_165971/XLOC_165892/NEO1/GLG1/ITGA6/CD40/XLOC_165898/SIGLEC1/NRCAM/VSIR/XLOC_165944/ITGAV/CDH2/SDC1/ITGB7/NECTIN3</td><td>hsa:3107/hsa:3127/hsa:201633/hsa:3105/hsa:3383/hsa:940/hsa:914/hsa:959/hsa:6693/hsa:925/hsa:5819/hsa:923/hsa:920/hsa:3105/hsa:926/hsa:4684/hsa:54413/hsa:3676/hsa:84628/hsa:3106/hsa:3119/hsa:4756/hsa:2734/hsa:3655/hsa:958/hsa:3105/hsa:6614/hsa:4897/hsa:64115/hsa:3125/hsa:3685/hsa:1000/hsa:6382/hsa:3695/hsa:25945</td><td>35</td><td>12</td><td>XLOC_165816/XLOC_165828/ENSG00000090339/ENSG00000130202/XLOC_165954/ENSG00000196338/ENSG00000196358/XLOC_165971/XLOC_165898/ENSG00000088827/ENSG00000107738/ENSG00000115884</td><td>23</td><td>ENSG00000198502/ENSG00000181847/ENSG00000178562/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000013725/ENSG00000010610/ENSG00000172116/ENSG00000149294/ENSG00000115232/XLOC_165892/ENSG00000067141/ENSG00000090863/ENSG00000091409/ENSG00000101017/ENSG00000091129/XLOC_165944/ENSG00000138448/ENSG00000170558/ENSG00000139626/ENSG00000177707</td></tr>
			
		  
		</table>
	</div>			
        </p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px"> 
		
		(1) KEGGID: Unique identification id of KEGG database.<br />
		(2) Description: Function description of this pathway.<br />
		(3) GeneRatio: Ratio between the number of differentially expressed genes in each pathway and all differentially expressed genes that can be found in KEGG database.<br />
		(5) BgRatio: In background KEGG database, the ratio of all genes concerning this KEGG pathway to all genes included.<br />
		(6) pvalue: Statistics category term; abbreviation for probability value.<br />
		(7) padj: Adjusted p-value. Generally, KEGG pathways with Corrected_pValue < 0.05 are significant enrichment.<br />
        (8) geneID: Differentially expressed genes in this pathway.<br />
        (9) geneName: Differentially expressed genes in this pathway.<br />
        (10) keggID: Pathway gene id.<br />
        (11) Count: Number of differentially expressed genes concerning this pathway.<br />
        (12) Up: Number of up expressed genes concerning this KEGG pathway.<br />
        (13) Up_Gene_id: Up expressed gene id concerning this KEGG pathway.<br />
        (14) Down: Number of down expressed genes concerning this KEGG pathway.<br />
        (15) Down_Gene_id: Down expressed gene id concerning this KEGG pathway.<br /> 
		

		
        </p>
</ul>
</div>

	<br />
	<br />
<!---------------------------5.2 KEGG富集分析---------------------->

<div id="page">

    <p class="paragraph">In the KEGG enrichment results, the most significant 20 KEGG pathways were selected for display. If the results are less than 20, all pathways would be drawn, as shown in the following figure. In this figure, the abscissa is the KEGG pathway, and the ordinate is the significance level of the pathway enrichment. Higher values correspond to higher significance.</p> 
    <br />
   <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGKEGGbar_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGKEGGbar_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGbar_Content_pane_group1vsgroup2_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGbar_Content_list_group1vsgroup2_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2.bar.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGbar_Content_pane_group2vssample1_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGbar_Content_list_group2vssample1_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1.bar.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGbar_Content_pane_sample1vssample5_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGbar_Content_list_sample1vssample5_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5.bar.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGKEGGbar_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGKEGGbar_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGKEGGbar_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGbar_Content_list_group1vsgroup2_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGbar_Content_pane_group1vsgroup2_all" role="tab"
            aria-controls="group1vsgroup2_all">group1vsgroup2_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGbar_Content_list_group2vssample1_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGbar_Content_pane_group2vssample1_all" role="tab"
            aria-controls="group2vssample1_all">group2vssample1_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGbar_Content_list_sample1vssample5_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGbar_Content_pane_sample1vssample5_all" role="tab"
            aria-controls="sample1vssample5_all">sample1vssample5_all</a>
          
        </div>
      </div>
    </div>
    </p>	
	<p class="name_fig">Figure 4.18 KEGG enrichment analysis histogram</p>
	<p class="cenmark">The abscissa is the KEGG pathway, and the ordinate is the significance level of pathway enrichment </p>

<p class="paragraph">From the KEGG enrichment results, the most significant 20 KEGG pathways were selected for display. If the results are less than 20, all pathways would be drawn. In this figure, the abscissa is the ratio of the number of differential genes linked with the KEGG pathway to the total number of differential genes. The ordinate is KEGG Pathway. The size of a point represents the number of genes annotated to a specific KEGG pathway. The color from red to purple represents the significant level of the enrichment.</p>
    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGKEGGdot_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGKEGGdot_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGdot_Content_pane_group1vsgroup2_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGdot_Content_list_group1vsgroup2_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2.dot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGdot_Content_pane_group2vssample1_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGdot_Content_list_group2vssample1_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1.dot.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGdot_Content_pane_sample1vssample5_all" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGdot_Content_list_sample1vssample5_all" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5.dot.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGKEGGdot_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGKEGGdot_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGKEGGdot_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGdot_Content_list_group1vsgroup2_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGdot_Content_pane_group1vsgroup2_all" role="tab"
            aria-controls="group1vsgroup2_all">group1vsgroup2_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGdot_Content_list_group2vssample1_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGdot_Content_pane_group2vssample1_all" role="tab"
            aria-controls="group2vssample1_all">group2vssample1_all</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGdot_Content_list_sample1vssample5_all"
            data-toggle="list" href="#gallery_figure_DEGKEGGdot_Content_pane_sample1vssample5_all" role="tab"
            aria-controls="sample1vssample5_all">sample1vssample5_all</a>
          
        </div>
      </div>
    </div>
    </p>
	
	<p class="name_fig">Figure 4.19 KEGG enrichment scatter plot</p>
	
    <p class="cenmark">The abscissa in the graph is the ratio of the number of differential genes on the KEGG pathway to the total number of differential genes, and the ordinate is KEGG pathway</p>

</div>	
	
	
<div id="page">

    <p class="paragraph">KEGG enrichment pathway map: the results of KEGG enrichment analysis can be visualized via the signal pathway map, as shown below:</p>

   <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_DEGKEGGpath_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_DEGKEGGpath_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa00514" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa00514" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2_hsa00514.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa04662" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa04662" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2_hsa04662.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa04520" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa04520" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2_hsa04520.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa01523" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa01523" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2_hsa01523.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa05320" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa05320" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group1vsgroup2_hsa05320.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa04662" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa04662" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1_hsa04662.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa01523" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa01523" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1_hsa01523.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa05320" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa05320" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1_hsa05320.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa04520" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa04520" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1_hsa04520.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa00514" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa00514" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/group2vssample1_hsa00514.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa05320" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa05320" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5_hsa05320.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa04662" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa04662" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5_hsa04662.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa00514" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa00514" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5_hsa00514.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa04520" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa04520" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5_hsa04520.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa01523" role="tabpanel"
            aria-labelledby="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa01523" onclick="trans(this.id)"><img
              class="img-gallery" src="files/DiffEnrich/KEGG/sample1vssample5_hsa01523.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_DEGKEGGpath_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_DEGKEGGpath_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_DEGKEGGpath_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa00514"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa00514" role="tab"
            aria-controls="group1vsgroup2_hsa00514">group1vsgroup2_hsa00514</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa04662"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa04662" role="tab"
            aria-controls="group1vsgroup2_hsa04662">group1vsgroup2_hsa04662</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa04520"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa04520" role="tab"
            aria-controls="group1vsgroup2_hsa04520">group1vsgroup2_hsa04520</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa01523"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa01523" role="tab"
            aria-controls="group1vsgroup2_hsa01523">group1vsgroup2_hsa01523</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group1vsgroup2_hsa05320"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group1vsgroup2_hsa05320" role="tab"
            aria-controls="group1vsgroup2_hsa05320">group1vsgroup2_hsa05320</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa04662"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa04662" role="tab"
            aria-controls="group2vssample1_hsa04662">group2vssample1_hsa04662</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa01523"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa01523" role="tab"
            aria-controls="group2vssample1_hsa01523">group2vssample1_hsa01523</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa05320"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa05320" role="tab"
            aria-controls="group2vssample1_hsa05320">group2vssample1_hsa05320</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa04520"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa04520" role="tab"
            aria-controls="group2vssample1_hsa04520">group2vssample1_hsa04520</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_group2vssample1_hsa00514"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_group2vssample1_hsa00514" role="tab"
            aria-controls="group2vssample1_hsa00514">group2vssample1_hsa00514</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa05320"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa05320" role="tab"
            aria-controls="sample1vssample5_hsa05320">sample1vssample5_hsa05320</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa04662"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa04662" role="tab"
            aria-controls="sample1vssample5_hsa04662">sample1vssample5_hsa04662</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa00514"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa00514" role="tab"
            aria-controls="sample1vssample5_hsa00514">sample1vssample5_hsa00514</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa04520"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa04520" role="tab"
            aria-controls="sample1vssample5_hsa04520">sample1vssample5_hsa04520</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_DEGKEGGpath_Content_list_sample1vssample5_hsa01523"
            data-toggle="list" href="#gallery_figure_DEGKEGGpath_Content_pane_sample1vssample5_hsa01523" role="tab"
            aria-controls="sample1vssample5_hsa01523">sample1vssample5_hsa01523</a>
          
        </div>
      </div>
    </div>
    </p>	

	
	<p class="name_fig">Figure 4.20 KEGG enrichment pathway</p>
	
    <p class="paragraph">Result directory of KEGG enrichment analysis of mRNA gene expression level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/6.DiffEnrich_mRNA/2.KEGG" title="点击打开" >KEGG</a></p>








    
    <p class="paragraph">Quantitative differential enrichment is the core content of transcriptome analysis, and it is very important to explore the mechanism of transcriptional regulation. LncRNA and mRNA can be obtained through chain specific library construction, and the difference analysis results of lncRNA and mRNA can be output respectively by default.</p>
    

<!---------------------------lncRNA靶基因预测-------------------------->


<div id="page">

	<h3 id="result7">4.7 Prediction and functional analysis of lncRNA target genes</h3>
    <p class="paragraph">LncRNA has rich biological functions and is widely involved in various important physiological processes of organisms. It can regulate the expression of target genes at the transcriptional level and post-transcriptional level, and identify the differential functional pathways generated by different treatments (Schmitt AM et al., 2016). LncRNA regulates target genes through various mechanisms, and co-location (also known as Cis) and co-expression (also known as Trans) are the most common ways to regulate target genes. Co-location means that lncRNA may have a regulatory effect on adjacent protein-coding genes. The analysis was conducted by searching for genes within 100KB upstream and downstream of lncRNA (Bao Z et al., 2019). Co-expression refers to the transcriptional action of lncRNA on distant target genes, which is analyzed by expression correlation between multiple samples and generally performed when the sample number is greater than 5 (Kopp F et al., 2018).</p>
    <p class="paragraph">we source will conduct target gene prediction for all lncrnas, that is, target genes of lncrnas can be predicted through co-location and co-expression between lncrnas and protein-coding genes. Then, functional enrichment analysis (GO/KEGG) was performed on target genes of differential lncrnas to predict the main functions of lncrnas. Only the prediction results of co-location are shown in the report.</p>
	
	</br>
	<h4 id="result71">4.7.1 lncRNA Target Prediction</h4>
	
    <p class="paragraph" >When the number of samples exceeded 5, co-location and co-expression were analyzed simultaneously. If the sample number is less than 5, only co-location analysis is performed. For the co-location regulation mechanism, we set the threshold of co-location as 100KB upstream and downstream of lncRNA, and the results are shown in the following table:</p>

</p>

</br>
	
	<p class="name_fig">Table 4.11　lncRNA co-location protein_gene</p>
	<p class="center">
	<div id="tb" style="overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1">
			
			
				<tr class="tb1"><th class="tb1">Chromosome</th><th class="tb1">lncRNA_ID</th><th class="tb1">lncRNA_Gene_ID</th><th class="tb1">lncRNA_Gene_Symbol</th><th class="tb1">lncRNA_Status</th><th class="tb1">lncRNA_Start</th><th class="tb1">lncRNA_End</th><th class="tb1">lncRNA_Strand</th><th class="tb1">Gene_Id</th><th class="tb1">Gene_Symbol</th><th class="tb1">Gene_Start</th><th class="tb1">Gene_End</th><th class="tb1">Gene_Strand</th><th class="tb1">Distance</th><th class="tb1">Location</th></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000455207</td><td  class="tb1" style="text-align:center">ENSG00000237094</td><td  class="tb1" style="text-align:center">AL732372.2</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">373182</td><td  class="tb1" style="text-align:center">485208</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284733</td><td  class="tb1" style="text-align:center">OR4F29</td><td  class="tb1" style="text-align:center">450702</td><td  class="tb1" style="text-align:center">451697</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000440200</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">601436</td><td  class="tb1" style="text-align:center">720200</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000419394</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">586945</td><td  class="tb1" style="text-align:center">720194</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000648019</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594458</td><td  class="tb1" style="text-align:center">733064</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000634337</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827796</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000634337</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827796</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">96131</td><td  class="tb1" style="text-align:center">upstream</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000635509</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827769</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000635509</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827769</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">96158</td><td  class="tb1" style="text-align:center">upstream</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000478729</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">940346</td><td  class="tb1" style="text-align:center">942173</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">(1828)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000478729</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">940346</td><td  class="tb1" style="text-align:center">942173</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">ENSG00000188976</td><td  class="tb1" style="text-align:center">NOC2L</td><td  class="tb1" style="text-align:center">944202</td><td  class="tb1" style="text-align:center">959256</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">2029</td><td  class="tb1" style="text-align:center">downstream</td></tr>
			
		  
		</table>
	</div>		
	</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
		(1) Chromosome: Chromosome name.<br />
		(2) lncRNA ID: lncRNA transcript ID.<br />
		(3) lncRNA Gene ID: lncRNA soure gene ID.<br />
		(4) lncRNA Gene Symbol: Source of lncRNA gene name, if there is no lncRNA origin gene name, we use "--" to represent.<br />
		(5) lncRNA Status: The lncRNA attribute can be Annotated_lncRNA or Novel_lncRNA.<br />
		(6) lncRNA_Start: Start position of lncRNA.<br />
		(7) lncRNA_End: End position of lncRNA.<br />
		(8) lncRNA_Strand: LncRNA chain properties.<br />
		(9) Gene ID: The gene ID corresponding to the target mRNA transcript or the target gene ID that predicts the encoding protein.<br />
		(10) Gene Symbol: Target gene name, if there is no target gene name, we use "-" to indicate.<br />
		(11) Gene_Start: Start location of target gene.<br />
		(12) Gene_End: End location of target gene.<br />
		(13) Gene_Strand: Target gene chain attributes.<br />
		(14) Distance: The distance between lncRNA and target gene or the length of overlap of both. The length of overlap is represented by a number with parentheses, and the unit is bp.<br />
		(15) Location: Location relationships of lncRNA to target genes: upstream, downstream, sense, antisense.<br />
</ul>
</div>
    <p class="paragraph">Result directory of LncRNA co-location targeting genes:<a href="../../Result_PXXXXXXXX_Homo_sapiens/7.Diff_lncRNATarget_Enrich" title="点击打开" >lncRNA_mRNA_co_location_results</a></p>
</br>

<!---------------------------7.2 差异lncRNA列表------------------------------>

<div id="page">
	<h4 id="result72">4.7.2 Differential lncRNA analysis</h4>

</p>
	
    
    

</br>

    <p class="paragraph">Differential lncRNA analysis, as mentioned above, results of differential lncRNA analysis are shown in Table 4.12</p>

	<p class="name_fig">Table 4.12 List of different lncRNA</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
				
					<tr class="tb1"><th class="tb1">transcript_id</th><th class="tb1">transcript_name</th><th class="tb1">gene_id</th><th class="tb1">gene_name</th><th class="tb1">transcript_locus</th><th class="tb1">group1_FPKM</th><th class="tb1">group2_FPKM</th><th class="tb1">log2FoldChange</th><th class="tb1">pvalue</th><th class="tb1">padj</th></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">TCONS_00025553</td><td  class="tb1" style="text-align:center">CDC42SE1-OT1</td><td  class="tb1" style="text-align:center">ENSG00000197622</td><td  class="tb1" style="text-align:center">CDC42SE1</td><td  class="tb1" style="text-align:center">1:151052114-151059574</td><td  class="tb1" style="text-align:center">29.2730395</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">11.0460016381348</td><td  class="tb1" style="text-align:center">2.05315763663687e-06</td><td  class="tb1" style="text-align:center">0.00799672754991148</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000400385</td><td  class="tb1" style="text-align:center">AATBC-201</td><td  class="tb1" style="text-align:center">ENSG00000215458</td><td  class="tb1" style="text-align:center">AATBC</td><td  class="tb1" style="text-align:center">21:43805758-43812567</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.673713</td><td  class="tb1" style="text-align:center">-9.44634991810653</td><td  class="tb1" style="text-align:center">5.36113400264938e-06</td><td  class="tb1" style="text-align:center">0.0131525499781809</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000651507</td><td  class="tb1" style="text-align:center">AC015911.11-201</td><td  class="tb1" style="text-align:center">ENSG00000286065</td><td  class="tb1" style="text-align:center">AC015911.11</td><td  class="tb1" style="text-align:center">17:35474904-35537861</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">5.31679533333333</td><td  class="tb1" style="text-align:center">-9.26754621460737</td><td  class="tb1" style="text-align:center">5.48984812223397e-06</td><td  class="tb1" style="text-align:center">0.0131525499781809</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000602755</td><td  class="tb1" style="text-align:center">AL356488.2-201</td><td  class="tb1" style="text-align:center">ENSG00000270066</td><td  class="tb1" style="text-align:center">AL356488.2</td><td  class="tb1" style="text-align:center">1:109100193-109100619</td><td  class="tb1" style="text-align:center">140.51347825</td><td  class="tb1" style="text-align:center">0.0486683333333333</td><td  class="tb1" style="text-align:center">8.10036737373499</td><td  class="tb1" style="text-align:center">6.91494520541465e-06</td><td  class="tb1" style="text-align:center">0.0153118316734938</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000429829</td><td  class="tb1" style="text-align:center">XIST-204</td><td  class="tb1" style="text-align:center">ENSG00000229807</td><td  class="tb1" style="text-align:center">XIST</td><td  class="tb1" style="text-align:center">X:73820656-73852723</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">4.49299233333333</td><td  class="tb1" style="text-align:center">-11.8583655220883</td><td  class="tb1" style="text-align:center">2.20689517599673e-05</td><td  class="tb1" style="text-align:center">0.030324037891828</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000668224</td><td  class="tb1" style="text-align:center">FTX-267</td><td  class="tb1" style="text-align:center">ENSG00000230590</td><td  class="tb1" style="text-align:center">FTX</td><td  class="tb1" style="text-align:center">X:74278483-74292373</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.61152633333333</td><td  class="tb1" style="text-align:center">-8.58891143746329</td><td  class="tb1" style="text-align:center">3.02987362742685e-05</td><td  class="tb1" style="text-align:center">0.0362947784938789</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000476784</td><td  class="tb1" style="text-align:center">TTC3-215</td><td  class="tb1" style="text-align:center">ENSG00000182670</td><td  class="tb1" style="text-align:center">TTC3</td><td  class="tb1" style="text-align:center">21:37107738-37203112</td><td  class="tb1" style="text-align:center">0.01152375</td><td  class="tb1" style="text-align:center">3.332379</td><td  class="tb1" style="text-align:center">-7.68698371716345</td><td  class="tb1" style="text-align:center">3.27769314462907e-05</td><td  class="tb1" style="text-align:center">0.0378724012285485</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000557026</td><td  class="tb1" style="text-align:center">NEK9-210</td><td  class="tb1" style="text-align:center">ENSG00000119638</td><td  class="tb1" style="text-align:center">NEK9</td><td  class="tb1" style="text-align:center">14:75082121-75117245</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.67877833333333</td><td  class="tb1" style="text-align:center">-9.63285669766275</td><td  class="tb1" style="text-align:center">3.28317190324118e-05</td><td  class="tb1" style="text-align:center">0.0378724012285485</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000558234</td><td  class="tb1" style="text-align:center">RORA-207</td><td  class="tb1" style="text-align:center">ENSG00000069667</td><td  class="tb1" style="text-align:center">RORA</td><td  class="tb1" style="text-align:center">15:60528073-60592517</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">2.17551833333333</td><td  class="tb1" style="text-align:center">-8.4824976317958</td><td  class="tb1" style="text-align:center">3.9799893275888e-05</td><td  class="tb1" style="text-align:center">0.0442707479538794</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">TCONS_00203167</td><td  class="tb1" style="text-align:center">MAVS-OT1</td><td  class="tb1" style="text-align:center">ENSG00000088888</td><td  class="tb1" style="text-align:center">MAVS</td><td  class="tb1" style="text-align:center">20:3823260-3876075</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.18659733333333</td><td  class="tb1" style="text-align:center">-9.81062885555481</td><td  class="tb1" style="text-align:center">5.39784567180132e-05</td><td  class="tb1" style="text-align:center">0.0512067524960217</td></tr>
				
		 			 	 
		</table>
	</div>		
	</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
    (1) transcript_id: Transcript id<br />
    (2) transcript_name: Transcript name<br />
    (3) gene_id: Gene id<br />
    (4) gene_name: Gene name, If not, use "--".<br />
    (5) locus: The position of a transcript on a chromosome.<br />
    (6) compare_treat: The average FPKM value of treatment group in compare group<br />
    (7) compare_control: The average FPKM value of control group in compare group<br />
    (8) log<sub>2</sub>FoldChange: The ratio of transcript expression level between the treatment group and the control group of a comparison combination was processed by the shrinkage model of the differential analysis software, and finally the logarithm was taken with 2 as the base.<br />
    (9) pvalue: Pvalue in hypergenometric test of compare group.<br />
    (10) padj: The corrected pvalue of multiple hypothesis test.<br />

</ul>
</div>
    

    <p class="paragraph">Difference of lncRNA can through the volcano's figure and clustering figure to display, as mentioned above, here no longer expatiatory.</p>
    <p class="paragraph">Result directory of differential expression of lncRNA transcript level:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/5.Differential/lncRNA/1.deglist" title="点击打开" >lncRNA.transcript_level.deg</a></p>

	<br />

</div>

<!-------------------------------差异lncRNA火山图---------------------------------->

<!---------------------------lncRNA靶基因富集分析---------------------------------->

<!-- 这部分新版报告没有，是否要保留？ -->

<div id="page">

	<a name="Enrichment analysis of differential lncRNA target genes"><h4  id="result73">4.7.3 Enrichment analysis of differential lncRNA target genes</h4></a>
     <p class="paragraph">GO and KEGG enrichment analysis was performed for co-location and co-expression target genes of differential lncRNA, respectively, to predict the function of lncRNA. If the number of samples was less than 5, only co-location target gene enrichment analysis was performed.</p>
    <p class="paragraph">Go and KEGG enrichment analysis are described above and will not be repeated here.</p>
</p>
	
    <p class="paragraph">Result directory of lncRNA transcription target gene enrichment analysis results:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/7.Diff_lncRNATarget_Enrich" title="点击打开" >Diff_lncRNATarget_Enrich</a></p>

</br>

</div>

<!-------------------------------GO/KEGG 富集分析视频链接------------------------------>

 
 

<!---------------------------------------可变剪接分析---------------------------------->

<div id="page">
	<h3  id="result10">4.8 Alternative Splicing Analysis</h3>
	
	<h4 id="result101">4.8.1 Detection of Differentially Expressed Isoforms</h4>
	
    <p class="paragraph">Alternative splicing means that the same precursor mRNA has multiple splicing procedures to form different mRNA, which is an important mechanism to regulate gene expression and generate protein diversity. For example, various forms of variable splicing are widely present in human cancer progression, and many cancer-related genes are regulated by variable splicing. Abnormal splicing and changes in the splicing process may have an important impact on tumor genesis and development, and cancer-specific splicing variants have potential diagnostic value, so variable shear analysis has gradually become a hot topic in cancer research in recent years (Scotti MM et al., 2016).</p>

	<p class="center">
		<img class="normal" src="images/AS.png">
	</p>

	<p class="name_fig">Figure 4.27 Diagram of alternative splicing</p>

    <p class="paragraph">Alternative splicing difference analysis includes quantitative analysis of alternative splicing events and significance analysis of expression difference. We use rMATS to  alternative splicing analysis (http://rnaseq-mats.sourceforge.net/index.html), the default use rMATS (Shen S et al., 2014).</p>

    <p class="paragraph">rMATS is a suitable RNA-Seq data alternative splicing analysis software [16], which not only can classify variable splicing events, but also perform differential analysis of alternative splicing events across different samples. Alternative splicing events as shown below:</p>

	<p class="center">
		<img class="normala2" src="images/rMATS.png">
	</p>

	<p class="name_fig">Figure 4.28 Variable splicing event classification</p>
    <p class="step">
    (1) SE: Skipped exon<br />
    (2) A5SS: Alternative 5' splice site<br />
    (3) A3SS: Alternative 3' splice site<br />
    (4) MXE: Mutually exclusive exon<br />
    (5) RI: Retained intron</p>
	<br />


</div>


<!---------------------------------------可变剪接事件分析---------------------------------->


    <p class="paragraph">The principle of rMATS detection of alternative splicing is roughly introduced as follows:</p>
	
    <p class="paragraph">Every alternative splicing event corresponds to 2 isoforms, which are referred to as Exon Inclusion Isoform and Exon Skipping Isoform.</p>

	<p class="center">
		<img class="normala3" src="images/SE.png">
	</p>

	<p class="name_fig">Figure 4.29 SE type of alternative splicing type</p>

    <p class="paragraph">Statistics for expression quantity is conducted by dividing the effective length of expression when calculating the ratio of Exon Inclusion Isoform and the 2 Isoform whole expression quantity and differential significant analysis. The threshold of differential AS analysis is set as FDR < 0.05(Only SE event results are displayed), The results are shown in Table 4.13:</p>

</p>
	

	
</br>
	
	<p class="name_fig">Table 4.13 SE Type Differential Alternative Splicing Analysis Partial Results </p>

	<p class="center">
	<div id="tb" style="overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1" >
			
			
				<tr class="tb1"><th class="tb1">GeneID</th><th class="tb1">geneSymbol</th><th class="tb1">chr</th><th class="tb1">strand</th><th class="tb1">exonStart_0base</th><th class="tb1">exonEnd</th><th class="tb1">upstreamES</th><th class="tb1">upstreamEE</th><th class="tb1">downstreamES</th><th class="tb1">downstreamEE</th><th class="tb1">IJC_SAMPLE_1</th><th class="tb1">SJC_SAMPLE_1</th><th class="tb1">IJC_SAMPLE_2</th><th class="tb1">SJC_SAMPLE_2</th><th class="tb1">IncFormLen</th><th class="tb1">SkipFormLen</th><th class="tb1">PValue</th><th class="tb1">FDR</th><th class="tb1">IncLevel1</th><th class="tb1">IncLevel2</th><th class="tb1">IncLevelDifference</th></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160310</td><td  class="tb1" style="text-align:center">PRMT2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">46636438</td><td  class="tb1" style="text-align:center">46636547</td><td  class="tb1" style="text-align:center">46635673</td><td  class="tb1" style="text-align:center">46635763</td><td  class="tb1" style="text-align:center">46636895</td><td  class="tb1" style="text-align:center">46636990</td><td  class="tb1" style="text-align:center">0,0,0,3</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">14,2,5</td><td  class="tb1" style="text-align:center">1,1,1</td><td  class="tb1" style="text-align:center">258</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,NA,NA,1.0</td><td  class="tb1" style="text-align:center">0.89,0.536,0.743</td><td  class="tb1" style="text-align:center">0.277</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160310</td><td  class="tb1" style="text-align:center">PRMT2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">46661799</td><td  class="tb1" style="text-align:center">46661936</td><td  class="tb1" style="text-align:center">46649574</td><td  class="tb1" style="text-align:center">46649739</td><td  class="tb1" style="text-align:center">46663382</td><td  class="tb1" style="text-align:center">46663554</td><td  class="tb1" style="text-align:center">2,1,0,3</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">5,6,6</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">286</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,0.343,NA,1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.219</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160294</td><td  class="tb1" style="text-align:center">MCM3AP</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">46254775</td><td  class="tb1" style="text-align:center">46254844</td><td  class="tb1" style="text-align:center">46254391</td><td  class="tb1" style="text-align:center">46254526</td><td  class="tb1" style="text-align:center">46256788</td><td  class="tb1" style="text-align:center">46256986</td><td  class="tb1" style="text-align:center">1,0,2,2</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">2,13,11</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">218</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,NA,1.0,0.578</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.141</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000182871</td><td  class="tb1" style="text-align:center">COL18A1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45504415</td><td  class="tb1" style="text-align:center">45504556</td><td  class="tb1" style="text-align:center">45504010</td><td  class="tb1" style="text-align:center">45504054</td><td  class="tb1" style="text-align:center">45505133</td><td  class="tb1" style="text-align:center">45505278</td><td  class="tb1" style="text-align:center">9,12,0,5</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">12,6,1</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">290</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,0.86,NA,1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,0.339</td><td  class="tb1" style="text-align:center">0.174</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45134714</td><td  class="tb1" style="text-align:center">45134832</td><td  class="tb1" style="text-align:center">45128401</td><td  class="tb1" style="text-align:center">45128573</td><td  class="tb1" style="text-align:center">45171609</td><td  class="tb1" style="text-align:center">45171684</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">0,0,4</td><td  class="tb1" style="text-align:center">0,1,1</td><td  class="tb1" style="text-align:center">267</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,1.0,NA,NA</td><td  class="tb1" style="text-align:center">NA,0.0,0.691</td><td  class="tb1" style="text-align:center">0.655</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45184473</td><td  class="tb1" style="text-align:center">45184593</td><td  class="tb1" style="text-align:center">45183361</td><td  class="tb1" style="text-align:center">45183510</td><td  class="tb1" style="text-align:center">45184922</td><td  class="tb1" style="text-align:center">45185091</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">0,0,2</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">269</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,NA,NA,0.0</td><td  class="tb1" style="text-align:center">NA,NA,0.526</td><td  class="tb1" style="text-align:center">-0.526</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45204554</td><td  class="tb1" style="text-align:center">45204736</td><td  class="tb1" style="text-align:center">45184922</td><td  class="tb1" style="text-align:center">45185091</td><td  class="tb1" style="text-align:center">45220835</td><td  class="tb1" style="text-align:center">45221014</td><td  class="tb1" style="text-align:center">0,2,0,0</td><td  class="tb1" style="text-align:center">0,2,0,0</td><td  class="tb1" style="text-align:center">2,4,5</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">331</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,0.31,NA,NA</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.69</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000142185</td><td  class="tb1" style="text-align:center">TRPM2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">44423644</td><td  class="tb1" style="text-align:center">44423732</td><td  class="tb1" style="text-align:center">44418422</td><td  class="tb1" style="text-align:center">44418555</td><td  class="tb1" style="text-align:center">44424851</td><td  class="tb1" style="text-align:center">44424939</td><td  class="tb1" style="text-align:center">4,6,13,0</td><td  class="tb1" style="text-align:center">1,0,0,0</td><td  class="tb1" style="text-align:center">1,0,0</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">237</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">0.715,1.0,1.0,NA</td><td  class="tb1" style="text-align:center">1.0,NA,NA</td><td  class="tb1" style="text-align:center">-0.095</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160226</td><td  class="tb1" style="text-align:center">CFAP410</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">44334060</td><td  class="tb1" style="text-align:center">44335804</td><td  class="tb1" style="text-align:center">44333032</td><td  class="tb1" style="text-align:center">44333262</td><td  class="tb1" style="text-align:center">44337648</td><td  class="tb1" style="text-align:center">44337667</td><td  class="tb1" style="text-align:center">4,2,4,0</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">6,6,3</td><td  class="tb1" style="text-align:center">1,1,0</td><td  class="tb1" style="text-align:center">1893</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0,NA</td><td  class="tb1" style="text-align:center">0.321,0.321,1.0</td><td  class="tb1" style="text-align:center">0.453</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160221</td><td  class="tb1" style="text-align:center">GATD3A</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">44134101</td><td  class="tb1" style="text-align:center">44134154</td><td  class="tb1" style="text-align:center">44133609</td><td  class="tb1" style="text-align:center">44133838</td><td  class="tb1" style="text-align:center">44136059</td><td  class="tb1" style="text-align:center">44136173</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">202</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,0.0,NA,1.0</td><td  class="tb1" style="text-align:center">NA,NA,1.0</td><td  class="tb1" style="text-align:center">-0.5</td></tr>
			
		  
		</table>
	</div>		
	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1): GeneID: Gene id of the AS event.<br />
	(2): geneSymbol: Gene name of the AS event.<br />
	(3): chr: The chromosome where the AS event is located.<br />
	(4): strand: The direction of the chromosome chain where the AS event is located.<br />
	(5): exonStart_0base: The start position of AS event (the position that 1 nt upstream of the 5' end of the skipped exon)<br />
	(6): exonEnd: The end position of AS event (the 3' end position of skipped exon).<br />
	(7): upstreamES: The upstream exon start position of the AS event.<br />
	(8): upstreamEE: The upstream exon end position of the AS event.<br />
	(9): downstreamES: The downstream exon start position of the AS event.<br />
	(10): downstreamEE: The downstream exon end position of the AS event.<br />
	(11): IJC_SAMPLE_1: Inclusion junction counts for SAMPLE_1, with only the reads span splicing junctions taken into account. Biological replicates are seperated by comma.<br />
	(12): SJC_SAMPLE_1: Exclusion junction counts for SAMPLE_1, with only the reads span splicing junctions taken into account.<br />
	(13): IJC_SAMPLE_2: Inclusion junction counts for SAMPLE_2, with only the reads span splicing junctions taken into account. Biological replicates are seperated by comma.<br />
	(14): SJC_SAMPLE_2: Exclusion junction counts for SAMPLE_2, with only the reads span splicing junctions taken into account.<br />
	(15): IncFormLen: Length of inclusion form, used for normalization.<br />
	(16): SkipFormLen: Length of skipping form, used for normalization.<br />
	(17): PValue: Pvalue.<br />
	(18): FDR: Adjusted p-value.<br />
	(19): IncLevel1: Inclusion level for SAMPLE_1 replicates (comma separated) calculated from normalized counts.<br />
	(20): IncLevel2: Inclusion level for SAMPLE_2 replicates (comma separated) calculated from normalized counts.<br />
	(21): IncLevelDifference: average(IncLevel1) - average(IncLevel2).<br />
</ul>
	</p>
	<p class="paragraph">Result directory of alternative splicing:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/8.AS/1.daslist" title="点击打开" >AS_list</a></p>
</div>
<!-----------------------------------差异剪接可视化---------------------------------->
<h4 id="result101">4.8.2 Visualization of AS Events</h4>
<p class="paragraph">Characteristics of Sashimi_plot as follows:</p>
<p class="paragraph">(1) The RNA-Seq read densities along exons are shown as histograms, color-coded by the sample. The RNA-Seq densities are aligned to the isoforms drawn at the bottom of the plot, which are automatically read from the GFF annotation of the events given as input.</p>
<p class="paragraph">(2) Junction reads are visualized as arcs connecting the pair of exons that the junction borders. The thickness of the arc is in proportion to the number of junction reads present in the sample, but the actual number of junction reads can be optionally plotted too (as in the main example.)</p>
<p class="paragraph">(3) Vertical coordinate indicates a modified RPKM value</p>




    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_asplot_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_asplot_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_asplot_Content_pane_SE1" role="tabpanel"
            aria-labelledby="gallery_figure_asplot_Content_list_SE1" onclick="trans(this.id)"><img
              class="img-gallery" src="files/AS/ASevent/SE1.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_asplot_Content_pane_SE2" role="tabpanel"
            aria-labelledby="gallery_figure_asplot_Content_list_SE2" onclick="trans(this.id)"><img
              class="img-gallery" src="files/AS/ASevent/SE2.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_asplot_Content_pane_SE3" role="tabpanel"
            aria-labelledby="gallery_figure_asplot_Content_list_SE3" onclick="trans(this.id)"><img
              class="img-gallery" src="files/AS/ASevent/SE3.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_asplot_Content_pane_SE4" role="tabpanel"
            aria-labelledby="gallery_figure_asplot_Content_list_SE4" onclick="trans(this.id)"><img
              class="img-gallery" src="files/AS/ASevent/SE4.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_asplot_Content_pane_SE5" role="tabpanel"
            aria-labelledby="gallery_figure_asplot_Content_list_SE5" onclick="trans(this.id)"><img
              class="img-gallery" src="files/AS/ASevent/SE5.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_asplot_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_asplot_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_asplot_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_asplot_Content_list_SE1"
            data-toggle="list" href="#gallery_figure_asplot_Content_pane_SE1" role="tab"
            aria-controls="SE1">SE1</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_asplot_Content_list_SE2"
            data-toggle="list" href="#gallery_figure_asplot_Content_pane_SE2" role="tab"
            aria-controls="SE2">SE2</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_asplot_Content_list_SE3"
            data-toggle="list" href="#gallery_figure_asplot_Content_pane_SE3" role="tab"
            aria-controls="SE3">SE3</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_asplot_Content_list_SE4"
            data-toggle="list" href="#gallery_figure_asplot_Content_pane_SE4" role="tab"
            aria-controls="SE4">SE4</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_asplot_Content_list_SE5"
            data-toggle="list" href="#gallery_figure_asplot_Content_pane_SE5" role="tab"
            aria-controls="SE5">SE5</a>
          
        </div>
      </div>
    </div>
    </p>

<p class="name_fig">Figure 4.30 SE Type Alternative Splicing Visualization </p>
<p class="premark">The red and orange colors in the figure represent different samples, and the inclusion level value represents the ratio of the total expression of exon inclusion isoform in the two isoforms, which can directly show the expression difference of alternative splicing events in different samples.</p>
<p class="paragraph">Result Directory:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/8.AS/2.dasplot" title="点击打开" >AS_plot</a> </p>
</br>



<!-------------------------------结构性变异分析视频链接------------------------------>

</br>


<!---------------------------------------变异位点分析---------------------------------->

<div id="page">
	<h3  id="result10">4.9 Mutation</h3>
    <p class="paragraph">A single nucleotide polymorphism (SNP) is a substitution of nucleotide sequences occurring commonly within a population (e.g. 1%), in which a single nucleotide in the genome, or other shared sequence exchanges between members of a biological species or paired chromosomes. Two types of variation occur with SNPs, namely transitions and transversions, occur with a probability ratio of 1:2. The high occurrence region of SNPs is often the CG sequences, resulting in C to T transitions, which are associated with the tendency of C to be methylated. In contrast to SNPs, InDeL is a molecular biology term for an insertion or deletion of bases in the genome of an organism, measuring from 1 to 10 000 base pairs in length, which is classified among small genetic variations.</p>
    <p class="paragraph">Mutation site analysis is an important part of RNA-seq structural analysis, including detection of inborn mutation sites and acquired somatic cell mutation sites, which is important for tumor research. We used GATK software(McKenna A et al., 2010) to perform mutation site analysis on sample data and used Snpeff software to annotate the variant sites.</p>

</div>


<!---------------------------------------变异位点检测---------------------------------->

    <h4 id="result101">4.9.1 Variant Site Detection</h4>
    The variant sites are divided into SNP and INDEL. The following table shows the SNP results,genotypes and annotations of each site. 
    
</br>

	<p class="name_fig">Table 4.14 variant site type and its annotation </p>

	<p class="center">
	<div id="tb" style="overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1" >
			
			
				<tr class="tb1"><th class="tb1">CHROM</th><th class="tb1">POS</th><th class="tb1">ID</th><th class="tb1">REF</th><th class="tb1">ALT</th><th class="tb1">QUAL</th><th class="tb1">DP</th><th class="tb1">AD</th><th class="tb1">GT</th><th class="tb1">GeneID</th><th class="tb1">GeneName</th><th class="tb1">FeatureID</th><th class="tb1">Biotype</th><th class="tb1">HGVS_C</th><th class="tb1">HGVS_P</th><th class="tb1">EFFECT</th><th class="tb1">IMPACT</th></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">133129</td><td  class="tb1" style="text-align:center">rs367730352</td><td  class="tb1" style="text-align:center">G</td><td  class="tb1" style="text-align:center">A</td><td  class="tb1" style="text-align:center">394.6</td><td  class="tb1" style="text-align:center">18</td><td  class="tb1" style="text-align:center">3,15</td><td  class="tb1" style="text-align:center">0/1</td><td  class="tb1" style="text-align:center">ENSG00000238009</td><td  class="tb1" style="text-align:center">AL627309.1</td><td  class="tb1" style="text-align:center">ENST00000477740.5</td><td  class="tb1" style="text-align:center">pseudogene</td><td  class="tb1" style="text-align:center">n.-3912C>T</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">upstream_gene_variant</td><td  class="tb1" style="text-align:center">MODIFIER</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">133129</td><td  class="tb1" style="text-align:center">rs367730352</td><td  class="tb1" style="text-align:center">G</td><td  class="tb1" style="text-align:center">A</td><td  class="tb1" style="text-align:center">394.6</td><td  class="tb1" style="text-align:center">18</td><td  class="tb1" style="text-align:center">3,15</td><td  class="tb1" style="text-align:center">0/1</td><td  class="tb1" style="text-align:center">ENSG00000238009</td><td  class="tb1" style="text-align:center">AL627309.1</td><td  class="tb1" style="text-align:center">ENST00000471248.1</td><td  class="tb1" style="text-align:center">pseudogene</td><td  class="tb1" style="text-align:center">n.-3956C>T</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">upstream_gene_variant</td><td  class="tb1" style="text-align:center">MODIFIER</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">133129</td><td  class="tb1" style="text-align:center">rs367730352</td><td  class="tb1" style="text-align:center">G</td><td  class="tb1" style="text-align:center">A</td><td  class="tb1" style="text-align:center">394.6</td><td  class="tb1" style="text-align:center">18</td><td  class="tb1" style="text-align:center">3,15</td><td  class="tb1" style="text-align:center">0/1</td><td  class="tb1" style="text-align:center">ENSG00000268903</td><td  class="tb1" style="text-align:center">AL627309.6</td><td  class="tb1" style="text-align:center">ENST00000494149.2</td><td  class="tb1" style="text-align:center">processed_pseudogene</td><td  class="tb1" style="text-align:center">n.*2012C>T</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">downstream_gene_variant</td><td  class="tb1" style="text-align:center">MODIFIER</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">133129</td><td  class="tb1" style="text-align:center">rs367730352</td><td  class="tb1" style="text-align:center">G</td><td  class="tb1" style="text-align:center">A</td><td  class="tb1" style="text-align:center">394.6</td><td  class="tb1" style="text-align:center">18</td><td  class="tb1" style="text-align:center">3,15</td><td  class="tb1" style="text-align:center">0/1</td><td  class="tb1" style="text-align:center">ENSG00000269981</td><td  class="tb1" style="text-align:center">AL627309.7</td><td  class="tb1" style="text-align:center">ENST00000595919.1</td><td  class="tb1" style="text-align:center">processed_pseudogene</td><td  class="tb1" style="text-align:center">n.*4553C>T</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">downstream_gene_variant</td><td  class="tb1" style="text-align:center">MODIFIER</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">133129</td><td  class="tb1" style="text-align:center">rs367730352</td><td  class="tb1" style="text-align:center">G</td><td  class="tb1" style="text-align:center">A</td><td  class="tb1" style="text-align:center">394.6</td><td  class="tb1" style="text-align:center">18</td><td  class="tb1" style="text-align:center">3,15</td><td  class="tb1" style="text-align:center">0/1</td><td  class="tb1" style="text-align:center">ENSG00000238009</td><td  class="tb1" style="text-align:center">AL627309.1</td><td  class="tb1" style="text-align:center">ENST00000453576.2</td><td  class="tb1" style="text-align:center">pseudogene</td><td  class="tb1" style="text-align:center">n.193+245C>T</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">intron_variant</td><td  class="tb1" style="text-align:center">MODIFIER</td></tr>
			
		  
		</table>
	</div>
	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	(1) CHROM: Chromosome ID of SNPs.<br />
	(2) POS: The postion of variation on chromosome.<br />
	(3) ID: The ID of this variation in dbSNP.<br />
	(4) REF: Bases of variation sites in reference genome.<br />
	(5) ALT: The actual base of the mutation site in the sample.<br />
	(6) QUAL: The quality of mutation sites, the value of phred scheme. The higher the value, the higher the reliability of the genotype.<br />
	(7) DP: The sequencing depth of each sample at the variant site (not all the number of reads, but the number of reads that are greater than a certain quality value.)<br />
	(8) AD: Allele Depth.The number of reads supporting either the reference genotype or SNP genotype.<br />
	(9) GT: The genotype of the locus.<br />
	(10) GeneID: Gene id where the variant site is located.<br />
	(11) GeneName: Gene name where the variant site is located.<br />
	(12) FeatureID: Transcript id where the variant site is located.<br />
	(13) Biotype: Transcript type where the variant site is located.<br />
	(14) HGVS_C: HGVS annotation of variant sites at the DNA level.<br />
	(15) HGVS_P: If variant is coding, this field describes the variant using HGVS notation (Protein level). Since transcript ID is already mentioned in ‘feature ID’, it may be omitted here.<br />
	(16) EFFECT: Detailed description of the effect predicted by SnpEff.<br />
	(17) IMPACT: Effects are categorized by ‘impact’: {High, Moderate, Low, Modifier}. This are pre-defined categories to help users find more significant variants.<br />
</ul>
<p class="paragraph">Result directory of variant site detection:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/9.SNP/2.snpeff" title="点击打开" >snpeff</a></p>
	</p>
</div>

<!--------------------------------------变异位点统计---------------------------------->
<h4 id="result101">4.9.2 Variant Site Statistics</h4>
<p class="paragraph">After applying GATK for the detection of variant sites, we counted each variant site according to the SnpEff annotation information mainly including the function statistics of the variant site, the statistics of the variant site and the influence of the variant site. The variant site function (SNP_function) is statistically plotted from three aspects: synonymous mutation, missense mutation, and nonsense mutation. The variant site region (SNP_region) is statistically plotted from gene structure regions such as EXON, INTRON and INTERGENIC. Variant site influence (SNP_impact) is statistically plotted from four levels of HIGH (high), MODERATE (middle), LOW (low), MODIFIER (self-phenotype effect, and other variant sites should have an impact at the same time). The figure below shows the SNP results.</p>


    <p class="center">
    <div class="row">
      <div class="col-1">
        <div id="gallery_figure_snpplot_Content_button_prev" class="button_prev" onclick="prev(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_prev">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-6">
        <div class="tab-content" id="gallery_figure_snpplot_Content_nav-tabContent">
          
          <div class="tab-pane fade" id="gallery_figure_snpplot_Content_pane_SNP_function" role="tabpanel"
            aria-labelledby="gallery_figure_snpplot_Content_list_SNP_function" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Variant/VariantSite/SNP_function.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_snpplot_Content_pane_SNP_impact" role="tabpanel"
            aria-labelledby="gallery_figure_snpplot_Content_list_SNP_impact" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Variant/VariantSite/SNP_impact.png" /></div>
          
          <div class="tab-pane fade" id="gallery_figure_snpplot_Content_pane_SNP_region" role="tabpanel"
            aria-labelledby="gallery_figure_snpplot_Content_list_SNP_region" onclick="trans(this.id)"><img
              class="img-gallery" src="files/Variant/VariantSite/SNP_region.png" /></div>
          
        </div>
      </div>
      <div class="col-1">
      <div id="gallery_figure_snpplot_Content_button_next" class="button_next" onclick="next(this.id)">
          <svg viewbox="0 0 16000 16000" class="svg_next">
            <circle class="c" cx="8000" cy="8000" r="6080"></circle>
            <polygon class="a"
              points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
            </polygon>
          </svg></div>
      </div>
      <div class="col-4">
        <input class="form-control" id="gallery_figure_snpplot_Content_InputList" type="text" placeholder="Search..">
        <div class="list-group" id="gallery_figure_snpplot_Content_list-tab" role="tablist">
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_snpplot_Content_list_SNP_function"
            data-toggle="list" href="#gallery_figure_snpplot_Content_pane_SNP_function" role="tab"
            aria-controls="SNP_function">SNP_function</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_snpplot_Content_list_SNP_impact"
            data-toggle="list" href="#gallery_figure_snpplot_Content_pane_SNP_impact" role="tab"
            aria-controls="SNP_impact">SNP_impact</a>
          
          <a class="list-group-item list-group-item-action" id="gallery_figure_snpplot_Content_list_SNP_region"
            data-toggle="list" href="#gallery_figure_snpplot_Content_pane_SNP_region" role="tab"
            aria-controls="SNP_region">SNP_region</a>
          
        </div>
      </div>
    </div>
    </p>

<p class="name_fig">Figure 4.31 Variability site statistics </p>
<p class="premark">The abscissa in the figure represents the sample names, and the ordinate represents the number of variants of different genomic regions (EXON, INTRON, INTERGENIC)</p>
<p class="paragraph">Result directory of variant site statistics:  <a href="../../Result_PXXXXXXXX_Homo_sapiens/9.SNP/3.stat" title="点击打开" >SNP_stat</a> </p>
</br>




<!---------------------------------------融合基因分析---------------------------------->


<!---------------------------CircRNA 鉴定（circRNA）--------------------------->



<!---------------------------12  circRNA 表达水平分布-------------------------->


<!---------------------------13  circRNA 差异分析-------------------------->



<!-----------------------------------14 差异 circRNA 来源基因富集分析---------------------------------->



<!---------------------------15 CircRNA 靶向 miRNA 位点分析---------------------------------->




<!-------------------------- CircRNA 编码潜能预测------------------------------->


<!-------------------------------结题报告结构介绍及辅助注释结果文件解读------------------------------>

<!-------------------------------novomagic视频链接------------------------------>
<!-------------------------------------------备注------------------------------------------>


<div id="page">

	<p class="head">
	<hr />
	</p>

	<h2>5 Appendix</h2>

        <!--<h3 id="note_2">Result Directory</h3>
        <p class="paragraph">Click to open the result directory (html), and the corresponding file can be viewed
          directly when the report and the result files are stored in the same folder.</p>
        <p class="paragraph">Result Directory: <a href="../../Result_PXXXXXXXX_Homo_sapiens/result_tree.html" title="点击打开" >html</a>
        </p>

        <table width=80%  id="result_note" cellspacing="0" cellpadding="5" class="gy2">

          <tr>
            <th colspan="3">1.Explanation on result file:</th>
          </tr>

          <tr>
            <td rowspan="2">compressed files in the fomat of *.tar.gz:</td>
            <td>Unix/Linux/Mac user</td>
            <td>use tar -zxvf *.tar.gz command</td>
          </tr>

          <tr>
            <td>Windows user</td>
            <td>use uncompressed software such as WinRAR, 7-Zip et al</td>
          </tr>

          <tr>
            <td rowspan="2">compressed files in the fomat of *.gz:</td>
            <td>Unix/Linux/Mac user</td>
            <td>use gzip –d *.gz command</td>
          </tr>

          <tr>
            <td>Windows user</td>
            <td>use uncompressed software such as WinRAR, 7-Zip et al</td>
          </tr>

          <tr>
            <td rowspan="2">compressed files in the fomat of *.zip:</td>
            <td>Unix/Linux/Mac user</td>
            <td>use unzip *.zip command</td>
          </tr>

          <tr>
            <td>Windows user</td>
            <td>use uncompressed software such as WinRAR, 7-Zip et al</td>
          </tr>

        </table>
        </p>

        <table width=80% id="result_note" cellspacing="0" cellpadding="5" class="gy2">

          <tr>
            <th colspan="3">2. Explanation on how to operating file:</th>

          </tr>

          <tr>
            <td rowspan="2">*.fasta</td>
            <td rowspan="2">sequence file, in the fomat of fasta. in genaral, because sequence of gene or genome is
              large, we provide customer with sampled*.fasta file(partial sequences of *.fasta). It will be more
              convenient to check file fomat.</td>
            <td>unix/Linux/Mac user use less or more command can view sequences in the fomat of *.fasta.</td>

          </tr>

          <tr>
            <td>windows user use editor Editplus/Notepad++ et al</td>

          </tr>

          <tr>
            <td rowspan="2">*.fq/fastq</td>
            <td rowspan="2">reads sequence file, in the fomat of fasta. it is no easy to open because of big file.</td>
            <td>unix/Linux/Mac user use less or more command;</td>

          </tr>

          <tr>
            <td>windows user use editor Editplus/Notepad++ et al</td>

          </tr>

          <tr>
            <td rowspan="2">*.xls,*.txt</td>
            <td rowspan="2">table result file; files are separated by(Tab)</td>
            <td>unix/Linux/Mac user use less or more command</td>

          </tr>

          <tr>
            <td>windows user use editor Editplus/Notepad++ et al, also can use Microsoft Excel to open.</td>

          </tr>

          <tr>
            <td rowspan="2">*.png</td>
            <td rowspan="2">figure result files; bitmap</td>
            <td>unix/Linux/Mac user use display command to open.</td>

          </tr>

          <tr>
            <td>windows user use picture viewer to view, for example photoshop et al</td>

          </tr>

          <tr>
            <td>*.pdf</td>
            <td>figure result files; vectorgraph, can be zoom in or out. it is very convenient for customer to view or
              edit. using Adobe Illustrator to edit,the picture can be used in paper for publishing.</td>
            <td>windows/Mac user use Adobe Reader/Foxit reader/web browser to open, unix/Linux user use evince command
              to open.</td>

          </tr>
        </table>
        </p>-->

        <h3 id="note_2">5.1 Software</h3>
        </a>
        
        
        
        <table class="gy2">
          <tr>
            <th>Analysis</th>
            <th>Software</th>
            <th>Version</th>
            <th>Parameter</th>
            <th>Remarks</th>
          </tr>

          <tr>
            <td rowspan="4">Mapping</td>
            <td>bowtie2</td>
            <td>v2.5.4</td>
            <td>--end-to-end --sensitive --phred33 --no-mixed -X 600</td>
            <td>Mapping to rRNA</td>
          </tr>

          <tr>
            <td>Hisat2</td>
            <td>v2.2.1</td>
            <td>--phred33 --rna-strandness RF --dta-cufflinks --un-conc-gz</td>
            <td>Mapping to reference</td>
          </tr>

          <tr>
            <td>samtools</td>
            <td>v1.20</td>
            <td>--</td>
            <td>--</td>
          </tr>

          <tr>
            <td>picard</td>
            <td>v3.3.0</td>
            <td>--</td>
            <td>--</td>
          </tr>

          <tr>
            <td rowspan="4">Encoding potential screening</td>
            <td>CNCI</td>
            <td>v2.0</td>
            <td>-m ve -p 1</td>
            <td>Coding-Potential Analysis</td>
          </tr>

          <tr>
            <td>CPC2</td>
            <td>v3.2.0</td>
            <td>--</td>
            <td>Coding-Potential Analysis</td>
          </tr>

          <tr>
            <td>Pfam-scan</td>
            <td>v1.6</td>
            <td>-pfamB</td>
            <td>Detection of Known Protein Domain</td>
          </tr>

          <tr>
            <td>getorf</td>
            <td>emboss-6.6</td>
            <td>--</td>
            <td>--</td>
          </tr>


          <tr>
            <td rowspan="3">Quantification</td>
            <td>stringtie</td>
            <td>v2.2.3</td>
            <td>--rf -e</td>
            <td>--</td>
          </tr>

          <tr>
            <td>String-merge</td>
            <td>v2.2.3</td>
            <td>-merge</td>
            <td>--</td>
          </tr>

          <tr>
            <td>gffcompare</td>
            <td>v0.12.6</td>
            <td>-R</td>
            <td>--</td>
          </tr>


          <tr>
            <td rowspan="2">Differential Analysis</td>
            <td>edgeR</td>
            <td>v4.0.16</td>
            <td>--foldchange 1 --padj 0.05</td>
            <td>For sample with bio-replicate</td>
          </tr>

          <tr>
            <td>edgeR</td>
            <td>v4.0.16</td>
            <td>--foldchange 1 --padj 0.05</td>
            <td>For sample without bio-replicate</td>
          </tr>

          <tr>
            <td>Enrichment Analysis</td>
            <td>clusterProfiler</td>
            <td>4.8.1</td>
            <td>For GO,KEGG enrichment</td>
            <td></td>
          </tr>

         <tr>
            <td>Alternative splicing</td>
            <td>rMATS</td>
            <td>v4.3.0</td>
            <td>dEFAULT</td>
            <td>--</td>
          </tr>

         <tr>
            <td>Variant Site Statistics</td>
            <td>GATK</td>
            <td>v4.6.0.0</td>
            <td>MQ < 40.0 and QD <2.0 and FS >30.0 and DP < 10 and QUAL < 20</td>
            <td>--</td>
          </tr>

         <tr>
            <td>Fusion Gene Analysis</td>
            <td>STAR-Fusion</td>
            <td>v1.12.0</td>
            <td>--examine_coding_effect --extract_fusion_reads --FusionInspector inspect --min_junction_reads 2 --min_sum_frags 2</td>
            <td>--</td>
          </tr>



        </table>
              
        </p>

        <h3 id="note_3">5.2 Methods</h3>
        </a>
        <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
        
        <p class="paragraph">File catalog:<a href="LncRNA.methods_hw.pdf" title="点击打开" >methods</a></p>
        
        
	<br />

        <h3 id="note_2">5.3 Result File Decompression Method and Format Description</h3>
        <table class="gy2">
          <tr>
            <th>Compressed File Format</th>
            <th>Operating System</th>
            <th>File Decompression</th>
          </tr>

          <tr>
            <td rowspan="2">*.gz</td>
            <td>Unix/Linux/Mac</td>
            <td>Command line: gzip –d *.gz </td>
          </tr>

          <tr>
            <td>Windows</td>
            <td>Software: such as WinRAR, 7-Zip et al</td>
          </tr>
        
          <tr>
            <td rowspan="2">*.zip</td>
            <td>Unix/Linux/Macr</td>
            <td>Command line: unzip *.zip</td>
          </tr>

          <tr>
            <td>Windows</td>
            <td>Software: such as WinRAR, 7-Zip et al</td>
          </tr>
        </table>

        </br>

        <table class="gy2">
          <tr>
            <th>File Type</th>
            <th>File Description</th>
            <th>Way to Open</th>
          </tr>

          <tr>
            <td rowspan="2">file.readme.pdf</td>
            <td rowspan="2">Results' construction files that help customers to undersdand all the results files better.</td>
            <td>Windows/Mac user use Adobe Reader/Foxit reader/web browser to open.</td>
          </tr>

          <tr>
            <td>Unix/Linux user use evince command to open.</td>
          </tr>

          <tr>
            <td rowspan="2">file.fa/.fasta</td>
            <td rowspan="2">Fasta format is a text-based format for representing sequences.</td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td rowspan="2">file.fq/.fastq</td>
            <td rowspan="2">Text file that contains sequence data delivered from a sequencer. </td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td>file.vcf</td>
            <td>VCF is a flexible and extendable line-oriented text format developed by the 1000 Genomes Project for releases of SNVs, INDELs, copy number variants and structural variants.</td>
            <td>Unix/Linux/Mac user use less or more command;</td>
          </tr>

          <tr>
            <td>file.bam</td>
            <td>BAM is compressed format for SAM file which contains sequence alignment information.</td>
            <td>Unix/Linux/Mac user use samtools software.</td>
          </tr>


          <tr>
            <td rowspan="2">file.pdf/.svg</td>
            <td rowspan="2">Figure result files; vectorgraph, can be zoomed in or out. It is very convenient for customer to view or edit with Adobe Illustrator. The figure can be used in paper for publish.</td>
            <td>unix/Linux user use evince command to open.</td>
          </tr>

          <tr>
            <td>Windows/Mac user use Adobe Reader/Foxit reader/web browser to open.</td>
          </tr>


          <tr>
            <td rowspan="2">file.txt/xls</td>
            <td rowspan="2">Table result file; files are tab delimited.</td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al, also can use Microsoft Excel to open.</td>
          </tr>

          <tr>
            <td rowspan="2">file.gff/gtf</td>
            <td rowspan="2">GFF/GTF is general feature format.</td>
            <td>Unix/Linux/Mac user use less or more command;</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td rowspan="2">file.png</td>
            <td rowspan="2">Image file, bitmap with loseless compression.</td>
            <td>Unix/Linux/Mac user use display command;</td>
          </tr>

          <tr>
            <td>Windows user use Image browser，such as photoshop et al.</td>
          </tr>

          </table>

</div>



<!-------------------------------参考文献---------------------------------->


<div id="page">

        <p class="head">
        <hr />
        </p>

        <h2 id="reference1">6 References</h2>

        <p class="ref">

        
     Bao Z, Yang Z, Huang Z, Zhou Y, Cui Q, Dong D. LncRNADisease 2.0: an updated database of long non-coding RNA-associated diseases. Nucleic Acids Res. 2019;47(D1):D1034-D1037. doi:10.1093/nar/gky905. <br /><br />
    

    
     Bray NL, Pimentel H, Melsted P, Pachter L. Near-optimal probabilistic RNA-seq quantification [published correction appears in Nat Biotechnol. 2016 Aug 9;34(8):888]. Nat Biotechnol. 2016;34(5):525-527. doi:10.1038/nbt.3519. <br /><br />
    

    
     Breschi A, Gingeras TR, Guigó R. Comparative transcriptomics in human and mouse. Nat Rev Genet. 2017;18(7):425-440. doi:10.1038/nrg.2017.19. <br /><br />
    

        

    
     Garber M, Grabherr MG, Guttman M, Trapnell C. Computational methods for transcriptome annotation and quantification using RNA-seq [published correction appears in Nat Methods. 2011 Jun;8(6):following 477]. Nat Methods. 2011;8(6):469-477. doi:10.1038/nmeth.1613.<br /><br />
    

    
     Ghosh S, Chan CK. Analysis of RNA-Seq Data Using TopHat and Cufflinks. Methods Mol Biol. 2016;1374:339-361. doi:10.1007/978-1-4939-3167-5_18.<br /><br />
    


    

        

    
     Hardwick SA, Chen WY, Wong T, et al. Spliced synthetic genes as internal controls in RNA sequencing experiments. Nat Methods. 2016;13(9):792-798. doi:10.1038/nmeth.3958.<br /><br />
    

    
     Kang YJ, Yang DC, Kong L, et al. CPC2: a fast and accurate coding potential calculator based on sequence intrinsic features. Nucleic Acids Res. 2017;45(W1):W12-W16. doi:10.1093/nar/gkx428.<br /><br />
    

    
     Kim D, Langmead B, Salzberg SL. HISAT: a fast spliced aligner with low memory requirements. Nat Methods. 2015;12(4):357-360. doi:10.1038/nmeth.3317.<br /><br />
    

    
     Kopp F, Mendell JT. Functional Classification and Experimental Dissection of Long Noncoding RNAs. Cell. 2018;172(3):393-407. doi:10.1016/j.cell.2018.01.011.<br /><br />
    

        

    
     McKenna A, Hanna M, Banks E, et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 2010;20(9):1297-1303. doi:10.1101/gr.107524.110.<br /><br />
    

        

    
    Mistry J, Bateman A, Finn RD. Predicting active site residue annotations in the Pfam database. BMC Bioinformatics. 2007;8:298. Published 2007 Aug 9. doi:10.1186/1471-2105-8-298.<br /><br />
    

    
     Mullan LJ, Bleasby AJ. Short EMBOSS User Guide. European Molecular Biology Open Software Suite. Brief Bioinform. 2002;3(1):92-94. doi:10.1093/bib/3.1.92.<br /><br />
    

    
     Pertea M, Pertea GM, Antonescu CM, Chang TC, Mendell JT, Salzberg SL. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotechnol. 2015;33(3):290-295. doi:10.1038/nbt.3122.<br /><br />
    

    
     Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26(1):139-140. doi:10.1093/bioinformatics/btp616.<br /><br />
    

    

    
     Schmitt AM, Chang HY. Long Noncoding RNAs in Cancer Pathways. Cancer Cell. 2016;29(4):452-463. doi:10.1016/j.ccell.2016.03.010.<br /><br />
    

    
     Shen S, Park JW, Lu ZX, et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Proc Natl Acad Sci U S A. 2014;111(51):E5593-E5601. doi:10.1073/pnas.1419161111.<br /><br />
    

    
     Sun L, Luo H, Bu D, et al. Utilizing sequence intrinsic composition to classify protein-coding and long non-coding transcripts. Nucleic Acids Res. 2013;41(17):e166. doi:10.1093/nar/gkt646.<br /><br />
     Wright MW. A short guide to long non-coding RNA gene nomenclature. Hum Genomics. 2014;8(1):7. Published 2014 Apr 9. doi:10.1186/1479-7364-8-7.<br /><br />
    

    
     Yu G, Wang LG, Han Y, He QY. clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS. 2012;16(5):284-287. doi:10.1089/omi.2011.0118.<br /><br />
    

    
     Yousefi H, Maheronnaghsh M, Molaei F, et al. Long noncoding RNAs and exosomal lncRNAs: classification, and mechanisms in breast cancer metastasis and drug resistance. Oncogene. 2020;39(5):953-974. doi:10.1038/s41388-019-1040-y.<br /><br />
    

        

        

        </p>


        <br />
        <br />


</div>



<!----------------------------------End---------------------------------->
</div>

</div>
  <script>
    $(document).ready(function () {

      // 初始化materialScrollTop
      $('body').materialScrollTop();

      $('.flexslider').flexslider({
        animation: "slide",
        slideshow: false,
        controlNav: false
      });


      $('#leftNavigation .step').click(function (event) {
        $(this).children('ul').slideToggle();
      });

      // 显示隐藏按钮切换
      $('.hide_or_show').click(function (e) {
        $(this).parent().next().fadeToggle();
      });

    });

    function displaySubMenu(li) {
      var subMenu = li.getElementsByTagName("ul")[0];
      subMenu.style.display = "block";
    }

    function hideSubMenu(li) {
      var subMenu = li.getElementsByTagName("ul")[0];
      subMenu.style.display = "none";
    }
  </script>

</body>


</html>

