<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >

<html>

<head>


<TITLE>Reads-based Shotgun Metagenome Sequencing Analysis Report (MetaPhlAn/HUMAnN)</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />



<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/styles.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/base_new.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>
<script src="src/js/bootstrap-table.js" type="text/javascript"></script>



<script type="text/javascript">

	$(function () {

		var options = {

			selectors: "h2,h3,h4,h5",

			theme: "bootstrap",

			scrollTo: "60"

		};

		var toc = $("#toc").tocify(options).data("toc-tocify");

	});

</script>


<style>

table {
border-collapse:collapse; /* 关键属性：合并表格内外边框(其实表格边框有2px，外面1px，里面还有1px哦) */
border:solid #D1D0CE; /* 设置边框属性；样式(solid=实线)、颜色(#999=灰) */
border-width:0px 0 0px 0px; /* 设置边框状粗细：上 右 下 左 = 对应：1px 0 0 1px */
}

.left{
    border:solid #D1D0CE;border-width:0px 0px 0px 0px;padding:2px;
}

.mid{
    border:solid #D1D0CE;border-width:2px 0px 2px 0px;padding:2px;
}
</style>




<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");

if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");

break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});

$(".list-group").each(show_default);

});



</script>







</head>



<body>



<div id="toc" class="tocify">

</div>





	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

			<tr height="86%">

				<td width='1%'></td>

                                <td><img src="src/pdf_images/logo.png" align="left" height=400 width=400></td>


				<td width="20%"></td>

				<td width="15%"></td>

				<td width="25%"

					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">


					<br><a href="mailto:op-rna@novogene.com" target="black"

						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">

						<a></td>

				<td width="1%"></td>

			</tr>

		</table>

	</div>


</div>



<!--------------------------标题---------------------------------->

<div id="main">

</br><h1>Reads-based Shotgun Metagenome Sequencing Analysis Report (MetaPhlAn/HUMAnN)</h1></br>

<table class="gy" style="text-align:left">

<tbody>



<tr>



<td style="text-align:left">Contract ID</td>



<td style="text-align:left">HXXXSCXXXXXX</td>



</tr>



<tr>



<td style="text-align:left">Contract Name</td>



<td style="text-align:left">Metagenomic Readsmapping Analysis</td>



</tr>



<tr>



<td style="text-align:left">Batch ID</td>



<td style="text-align:left">X204SCXXXXXXXX-XXX-XXXX</td>



</tr>



<tr>



<td style="text-align:left">Report Time</td>



<td style="text-align:left">2024-10-30</td>



</tr>



</tbody></table>

<br /><br />
<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Annotation";
      };
      return 0;
    }
    function hiddenShow1(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Q&A";
      };
      return 0;
    }
</script>

<!----------------------------------------WOBI----------------------------------------->
<br />
<h2>1 Introduction </h2>

<p class="paragraph">Diverse communities of microbes thrive in environments ranging from the human gut (Walter et al., 2011), to the mud in deep sea, to conventionally inhospitable habitats such as acid mine runoff and geothermal hot springs. Environmental DNA sequencing has revealed the expansive biodiversity of microorganisms and clarified the relationship between host-associated microbial communities and host phenotype. Studies of cultured microbes reveal that they are critical components of these environments and provide essential ecosystem services (van der Heijden et al., 2008). Unfortunately, most microbes are notoriously difficult to culture in the laboratory. Therefore, vast amounts of organismal and functional novelty remain to be explored.</p>

<p class="paragraph">Metagenomics, or the study of microbial communities, takes advantage of advances in sequencing technology and analysis methods to comprehensively examine microbial communities directly from their natural habitats, which potentially may reveal novel content (Handelsman et al., 1998). Metagenomic sequencing provides insights into species biodiversity, functions, and pathways present in various microbiomes.</p>

<p class="paragraph">Compared with traditional assembly-based metagenomic processes, the read-based metagenomic workflow directly aligns quality-controlled reads to species (MetaPhlAn) and functional (HUMAnN) databases, independent of assembled sequences. This method rapidly provides abundance information for species and functions, enabling large-scale microbial community studies.</p>

<p class="paragraph">The workflow is shown as follows:</p>

<p class = "center"><img class="center" src="src/pdf_images/project_process_en.png" height=550 width=700/>

<p class="name_fig">Figure 1 Project Workflow</p>

<br />
<br/>

<h2> 2 Library Construction and Sequencing </h2>

<p class="paragraph">The genomic DNA was randomly sheared into shorter fragments. The obtained fragments were then end-repaired, A-tailed, and further ligated with Illumina adapters. The resulting fragments with adapters were size selected, PCR amplified unless otherwise specified as PCR-free, before proceeding for purification. </p>

<p class="paragraph">The experimental procedures of DNA library preparation are shown as follows :</p>

<p class = "center"><img class="center" src="src/pdf_images/experiment_pipeline_en.png" height=550 width=700/>

<p class="name_fig">Figure 2 Workflow of Library Construction</p>

</p>

<p class="paragraph">The library was quantified through Qubit and qPCR, and size distribution detected with fragment analyzer. Quantified libraries were pooled and sequenced on Illumina platforms according to the effective library concentration and required data amount.</p>

<p class="paragraph">The methods involved in the research are as follows:<a href="src/pdf_images/Sequencing_Methods_en.pdf" title="Click to open" >methods</a>.</p>


<h2> 3 Bioinformatics Analysis Pipeline </h2>

<p class="paragraph">The raw data obtained through sequencing contains a certain proportion of low-quality data. Quality control and host filtration were performed on the raw data to obtain clean data, which ensures that the results from subsequent analysis will be accurate and reliable.</p>

<p class="paragraph">Clean data obtained after quality contol was inputted into the HUMAnN software to calculate the abundance of Uniref gene families. HUMAnN invoked the MetaPhlAn software and the ChocoPhlAn database to retrieve the protein database of nucleic acid translations. The abundance of Uniref was normalized using RPK (reads per kilobase) to account for differences in gene length. This process includes two steps: nucleic acid library alignment and protein library alignment. First, species annotation was conducted through MetaPhlAn, and the coding sequences (CDS) of all strains' genes for these species (including corresponding Uniref gene family information) were extracted from the Chocophlan database to construct an index. Next, to provide information on the remaining reads-Uniref alignments (which do not contain species information), Diamond was used to align any reads that did not align with the Uniref protein database.</p>

<p class="paragraph">HUMAnN software obtained functional annotation results based on the correspondence between the UniRef gene IDs and other functional databases such as MetaCyc, KEGG, GO, eggNOG, and Pfam.</p>

<p class="paragraph">Through the species abundance and functional annotations, analyses including species and function bar plots and clustering heatmaps were performed. If there are more than two samples, PCA, PCoA, and NMDS were conducted. For grouped data, Anosim, MetaGenomeSeq, and LEfSe were used to explore differences in species composition and functional composition among samples.</p>

<p class="paragraph">Information analysis workflow is as shown in the figure:</p>

<p class = "center"><img class="center" src="src/pdf_images/information_analysis_en.png" height=550 width=700/>

<p class="name_fig">Figure 3 Information Analysis Workflow Diagram </p>


<p class="paragraph">Notes: When the number of samples is less than 3, PCA, PCoA, NMDSs, clustering analysis and abundance heatmap analysis cannot be carried out. When the number of biological repeats in the group is less than 3, statistical analysis such as Anosim, MetaGenomSeq, and LEfSe are statistically meaningless and the analysis cannot be conducted. </p>

<h2> 4 Analysis Results </h2>
<h3> 4.1 Sequencing Data Processing </h3>

<p class="paragraph">The protocols of data pre-processing are as follows: </p>

<p class="paragraph">1. when either one read contains adapter contamination,the paired read will be discarded.</p>
<p class="paragraph">2. when either one read contains more than 10 percent uncertain nucleotides,the paired read will be discarded.</p>
<p class="paragraph">3. when either one read contains more than 50 percent low quality nucleotides (base quality less than 5),the paired read will be discarded.</p>

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px">Table 4.1 QC statistics</p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>SampleID</th><th>Raw_Base(G)</th><th>Clean_Base(G)</th><th>Clean_Q20(%)</th><th>Clean_Q30(%)</th><th>Clean_GC(%)</th><th>Effective(%)</th><th>NonHost_Base(G)</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>6.59</td><td>6.53</td><td>98.08</td><td>94.18</td><td>62.85</td><td>99.09</td><td>6.53</td></tr>
            
        
            
        <tr><td>A2</td><td>5.91</td><td>5.86</td><td>98.45</td><td>95.26</td><td>62.70</td><td>99.22</td><td>5.86</td></tr>
            
        
            
        <tr><td>A3</td><td>6.19</td><td>6.13</td><td>98.38</td><td>95.03</td><td>61.86</td><td>99.05</td><td>6.13</td></tr>
            
        
            
        <tr><td>A4</td><td>6.64</td><td>6.57</td><td>98.11</td><td>94.25</td><td>62.81</td><td>98.91</td><td>6.57</td></tr>
            
        
            
        <tr><td>A5</td><td>6.11</td><td>6.04</td><td>98.26</td><td>94.72</td><td>61.62</td><td>98.89</td><td>6.04</td></tr>
            
        
            
        <tr><td>A6</td><td>7.49</td><td>7.42</td><td>98.46</td><td>95.33</td><td>62.86</td><td>99.13</td><td>7.42</td></tr>
            
        
            
        <tr><td>B1</td><td>7.98</td><td>7.93</td><td>98.48</td><td>95.39</td><td>61.51</td><td>99.28</td><td>7.93</td></tr>
            
        
            
        <tr><td>B2</td><td>9.31</td><td>9.26</td><td>98.45</td><td>95.23</td><td>62.37</td><td>99.37</td><td>9.26</td></tr>
            
        
            
        <tr><td>B3</td><td>8.91</td><td>8.85</td><td>98.60</td><td>95.70</td><td>62.48</td><td>99.37</td><td>8.85</td></tr>
            
        
            
        <tr><td>B4</td><td>8.79</td><td>8.72</td><td>98.36</td><td>94.99</td><td>62.11</td><td>99.23</td><td>8.72</td></tr>
            
        
            
        <tr><td>B5</td><td>7.46</td><td>7.39</td><td>98.32</td><td>94.95</td><td>60.61</td><td>99.14</td><td>7.39</td></tr>
            
        
            
        <tr><td>B6</td><td>7.28</td><td>7.22</td><td>98.33</td><td>94.89</td><td>62.71</td><td>99.22</td><td>7.22</td></tr>
            
        
            
        <tr><td>C1</td><td>7.32</td><td>7.28</td><td>98.50</td><td>95.47</td><td>66.53</td><td>99.39</td><td>7.28</td></tr>
            
        
            
        <tr><td>C2</td><td>7.30</td><td>7.22</td><td>97.93</td><td>93.83</td><td>66.41</td><td>98.88</td><td>7.22</td></tr>
            
        
            
        <tr><td>C3</td><td>7.29</td><td>7.20</td><td>97.82</td><td>93.59</td><td>67.36</td><td>98.75</td><td>7.20</td></tr>
            
        
            
        <tr><td>C4</td><td>7.31</td><td>7.23</td><td>97.93</td><td>93.79</td><td>67.07</td><td>98.94</td><td>7.23</td></tr>
            
        
            
        <tr><td>C5</td><td>7.02</td><td>6.97</td><td>98.29</td><td>94.80</td><td>66.70</td><td>99.28</td><td>6.97</td></tr>
            
        
            
        <tr><td>C6</td><td>8.61</td><td>8.51</td><td>97.81</td><td>93.55</td><td>66.83</td><td>98.82</td><td>8.51</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>Raw_Base(G)：</strong>Raw sequenced data.</li>
<li><strong>Clean_Base(G)：</strong>Clean data obtained after data filtering.</li>
<li><strong>Clean_Q20(%)：</strong>Percentage of bases with a quality score over 20, indicating an error rate less than 1%.</li>
<li><strong>Clean_Q30(%)：</strong>Percentage of bases with a quality score over 30, corresponding to an error rate less than 0.1%.</li>
<li><strong>Clean_GC(%)：</strong>Calculated an the total count of guanine (G) and cytosine (C) bases devided by the total count of clean bases, expressed as a percentage.</li>
<li><strong>Effective(%)：</strong>Represents the ratio of Clean Data (post-quality control data) to Raw Data, expressed as a percentage.</li>
<li><strong>NonHost_Base(G)：</strong>The total count of clean bases remaining after the removal of host-specific sequences, measured in gigabases (G).</li>
</ul>
</div>

<br />

<p class="paragraph">Results directory: QC_result: result/01.QC/QC_summary.xls</p>
<!------------------------------------- Taxonomic and function Annotation ---------------------------------->

<br/>

    <h3> 4.2 Taxonomy and Function Annotation </h3>

    <p class="paragraph">HUMAnN (HMP Unified Metabolic Analysis Network) is  an integrated software (Franzosa EA et al., 2018) for functional  profiling of metagenomic samples through reads mapping. It allows for  taxonomy analysis, function analysis, and identification of various  pathways present in the samples. The current workflow analysis includes species annotation through MetaPhlAn, metabolic pathway annotation from  MetaCyc, functional annotation from KEGG (KEGG Pathway, Module, KO, EC),  gene ontology annotation from GO, orthologous group  annotation from eggNOG, and protein families annotation from Pfam.</p>

<br/>
<!--MetaPhlAn注释字段识别开始-->



    <h4> 4.2.1 MetaPhlAn Taxonomic Annotation </h4>

    <p class="paragraph">MetaPhlAn (https://huttenhower.sph.harvard.edu/metaphlan/) is a computational tool for profiling the composition of microbial communities (Bacteria, Archaea, and Eukaryotes) from metagenomic shotgun sequencing data (i.e. not 16S) with species-level. With StrainPhlAn, it is possible to perform accurate strain-level microbial profiling.</p>

    <p class="paragraph">With its built-in StrainPhlAn tool, MetaPhlAn not only provides hierarchical classification from kingdom to species but also microbial analyses down to the SGB level. StrainPhlAn identifies microbial genome sequence polymorphisms within a given species, which can be used to differentiate between different genome assemblies within the same species. Although the results of these analyses are often described as "strain-level", these are closer to the SGB level as they represent clusters of genomes within species. While StrainPhlAn helps to reveal genomic diversity within microbial species, the distinction from the traditional strain definition should be noted.</p>

    <p class="paragraph">MetaPhlAn (Blanco-Míguez A et al.,2023) relies on ~5.1M unique clade-specific marker genes identified from ~1M microbial genomes (~236,600 references and 771,500 metagenomic assembled genomes) spanning 26,970 species-level genome bins (SGBs,http://segatalab.cibio.unitn.it/data/Pasolli_et_al.html), 4,992 of them taxonomically unidentified at the species level.</p>




<!--MetaPhlAn注释字段识别结束-->
<br />

<h5> 4.2.1.1 Relative Abundance of Taxonomic Annotation</h5>

<p class="paragraph"> Based on the abundance table of taxonomic level, the top 10 taxa were picked out. Bar charts show the relative taxonomic abundance of each sample (or group). </p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure4_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure4_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure4_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure4_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/p10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_class" role="tabpanel" aria-labelledby="gallery_figure4_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/c10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_order" role="tabpanel" aria-labelledby="gallery_figure4_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/o10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_family" role="tabpanel" aria-labelledby="gallery_figure4_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/f10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_genus" role="tabpanel" aria-labelledby="gallery_figure4_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/g10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_species" role="tabpanel" aria-labelledby="gallery_figure4_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/s10.group.dis.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_strain" role="tabpanel" aria-labelledby="gallery_figure4_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Top/t10.group.dis.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure4_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure4_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure4_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_phylum" data-toggle="list" href="#gallery_figure4_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_class" data-toggle="list" href="#gallery_figure4_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_order" data-toggle="list" href="#gallery_figure4_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_family" data-toggle="list" href="#gallery_figure4_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_genus" data-toggle="list" href="#gallery_figure4_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_species" data-toggle="list" href="#gallery_figure4_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_strain" data-toggle="list" href="#gallery_figure4_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.1 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a taxonomic level as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> Taxonomic annotation visualization: result/02.Annotation/Taxonomic/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.1.2 MetaPhlAn Taxonomic Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant taxonomic level of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure5_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure5_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure5_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure5_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.p.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_class" role="tabpanel" aria-labelledby="gallery_figure5_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.c.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_order" role="tabpanel" aria-labelledby="gallery_figure5_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.o.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_family" role="tabpanel" aria-labelledby="gallery_figure5_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.f.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_genus" role="tabpanel" aria-labelledby="gallery_figure5_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.g.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_species" role="tabpanel" aria-labelledby="gallery_figure5_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.s.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_strain" role="tabpanel" aria-labelledby="gallery_figure5_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_heatmap/heatmap.group1.t.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure5_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure5_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure5_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_phylum" data-toggle="list" href="#gallery_figure5_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_class" data-toggle="list" href="#gallery_figure5_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_order" data-toggle="list" href="#gallery_figure5_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_family" data-toggle="list" href="#gallery_figure5_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_genus" data-toggle="list" href="#gallery_figure5_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_species" data-toggle="list" href="#gallery_figure5_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_strain" data-toggle="list" href="#gallery_figure5_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.2 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates taxonomic level information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!--MetaCyc注释字段识别开始-->



    <h4> 4.2.2 MetaCyc Function Annotation </h4>

    <p class="paragraph"> MetaCyc (Karp PD et al.,2002) is a curated database that contains experimentally established metabolic pathways from all domains of life. These pathways within MetaCyc include primary and secondary metabolism, along with associated metabolites, reactions, enzymes, and genes. The objective of MetaCyc is to catalog the sphere of metabolism by storing a representative sample of each experimentally elucidated pathway.</p>

    <p class="paragraph"> Microbial functional analysis was conducted by using HUMAnN to obtain gene IDs from UniRef90. The correspondence between UniRef90 gene IDs and MetaCyc pathways in the MetaCyc database was used to derive MetaCyc functional annotation results. </p>



<!--MetaCyc注释字段识别结束-->
<br />

<h5> 4.2.2.1 Relative Abundance of MetaCyc Annotation</h5>

<p class="paragraph"> Based on the abundance table of metabolic pathway, the top 10 metabolic pathways were picked out. Bar charts show the relative metabolic pathway abundance of each sample (or group). </p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_Top/top10.png" height=520 width=680/>



<p class="name_fig"> Figure 4.3 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a metabolic pathway as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> MetaCyc annotation visualization: result/02.Annotation/MetaCyc/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.2.2 MetaCyc Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant metabolic pathways of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_heatmap/heatmap.png" height=200 width=300/>



<p class="name_fig"> Figure 4.4 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates metabolic pathway information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!--KEGG注释字段识别开始-->



    <h4> 4.2.3 KEGG Function Annotation </h4>

    <p class="paragraph"> KEGG (Kanehisa M et al., 2000) is a database resource for understanding high-level functions and utilities of the biological system, from the cell to the organism and the ecosystem. Functional studies are accomplished through molecular-level information, especialy through large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies.</p>

    <p class="paragraph"> The KO (KEGG Orthology) database is a database of molecular functions represented in terms of functional orthologs. </p>

    <p class="paragraph"> The EC (KEGG ENZYME) is an implementation of the Enzyme Nomenclature (EC number system) produced by the IUBMB/IUPAC Biochemical Nomenclature  Committee. KEGG ENZYME is based on the ExplorEnz database  at Trinity College Dublin and is maintained in the KEGG  relational database with additional annotation of reaction hierarchy and  sequence data links. </p>

    <p class="paragraph"> The Pathway (KEGG PATHWAY) is a collection of manually drawn pathway maps representing our knowledge of molecular interaction, reaction, and relation networks. </p>

    <p class="paragraph"> The KEGG MODULE database consists of both KEGG modules identified by M numbers and KEGG reaction modules identified by RM numbers, which are manually defined functional units of gene sets and reaction sets, respectively. KEGG modules can be further divided into pathway modules and signature modules. For example, pathway modules are functional units of gene sets within metabolic pathways and include molecular complexes. Whereas, signature modules are functional units of gene sets that characterize phenotypic features. Last but not least, reaction modules are functional units of successive reaction steps within metabolic pathways. </p>

    <p class="paragraph">Microbial functional analysis was conducted using HUMAnN to obtain gene IDs from UniRef90. The correspondence between UniRef90 IDs and KO in the KEGG database was used to derive KEGG functional annotation results.</p>




<!--KEGG注释字段识别结束-->
<br />

<h5> 4.2.3.1 Relative Abundance of KEGG Annotation</h5>

<p class="paragraph"> Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the top 10 KEGG orthologys were picked out. Bar charts show the relative KEGG orthology abundance of each sample (or group). </p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure4_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure4_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure4_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure4_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_Top//top10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure4_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_Top//top10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_EC" role="tabpanel" aria-labelledby="gallery_figure4_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_Top//top10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure4_pane_KO" role="tabpanel" aria-labelledby="gallery_figure4_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_Top//top10.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure4_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure4_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure4_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_KeggModule" data-toggle="list" href="#gallery_figure4_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_KeggPath" data-toggle="list" href="#gallery_figure4_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_EC" data-toggle="list" href="#gallery_figure4_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure4_list_KO" data-toggle="list" href="#gallery_figure4_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.5 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module) as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> KEGG annotation visualization: result/02.Annotation/KEGG/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.3.2 KEGG Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module) of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure5_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure5_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure5_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure5_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_heatmap//heatmap.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure5_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_heatmap//heatmap.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_EC" role="tabpanel" aria-labelledby="gallery_figure5_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_heatmap//heatmap.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure5_pane_KO" role="tabpanel" aria-labelledby="gallery_figure5_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_heatmap//heatmap.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure5_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure5_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure5_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_KeggModule" data-toggle="list" href="#gallery_figure5_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_KeggPath" data-toggle="list" href="#gallery_figure5_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_EC" data-toggle="list" href="#gallery_figure5_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure5_list_KO" data-toggle="list" href="#gallery_figure5_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.6 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module) information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!--GO注释字段识别开始-->



    <h4> 4.2.4 GO Functional Annotation </h4>

    <p class="paragraph"> GO (Gene Ontology) (Harris MA et al.,2004) is a database (https://geneontology.org/docs/ontology-documentation/) established by the Gene Ontology Consortium, aiming to provide a comprehensive resource for the functional annotation of genes and proteins across various species. It serves to define and describe gene and protein functions and is continuously updated as research progresses. </p>

    <p class="paragraph"> The fundamental concept in GO is the "GO IDs," which are used to describe characteristics of genes and gene products. In other words, the GO database assigns labels or tags to each gene to enable researchers to quickly locate their target genes based on these labels. Additionally, these terms are categorized into three different types, referred to as the three main branches of GO: Cellular Component (CC), Biological Process (BP), and Molecular Function (MF). </p>

    <p class="paragraph">Microbial functional analysis was conducted using HUMAnN to obtain gene IDs from UniRef90. The correspondence between UniRef90 IDs and GO IDs in the GO database was utilized to generate GO functional annotation results. </p>



<!--GO注释字段识别结束-->
<br />

<h5> 4.2.4.1 Relative Abundance of GO Annotation</h5>

<p class="paragraph"> Based on the abundance table of gene ontology, the top 10 gene ontologys were picked out. Bar charts show the relative gene ontology abundance of each sample (or group). </p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_Top/top10.png" height=520 width=680/>



<p class="name_fig"> Figure 4.7 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a gene ontology as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> GO annotation visualization: result/02.Annotation/GO/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.4.2 GO Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant gene ontologys of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_heatmap/heatmap.png" height=200 width=300/>



<p class="name_fig"> Figure 4.8 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates gene ontologys information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!--eggNOG注释字段识别开始-->



    <h4> 4.2.5 eggNOG Functional Annotation </h4>

    <p class="paragraph"> The eggNOG (Huerta-Cepas J et al., 2019) is a public database (http://eggnog5.embl.de/#/app/home) of orthology relationships, gene evolutionary histories and functional annotations. </p>

    <p class="paragraph"> An Orthologous Group (OG) is defined as a cluster of three or more homologous sequences that diverge from the same speciation event. </p>

    <p class="paragraph">Microbial functional analysis was conducted using HUMAnN to obtain gene IDs from UniRef90. The correspondence between UniRef90 IDs and eggNOG database Orthologous Groups (OG) was utilized to derive eggNOG functional annotation results. </p>



<!--eggNOG注释字段识别结束-->
<br />

<h5> 4.2.5.1 Relative Abundance of eggNOG Annotation</h5>

<p class="paragraph"> Based on the abundance table of orthologous group, the top 10 orthologous groups were picked out. Bar charts show the relative orthologous group abundance of each sample (or group). </p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_Top/top10.png" height=520 width=680/>



<p class="name_fig"> Figure 4.9 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a orthologous group as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> eggNOG annotation visualization: result/02.Annotation/eggNOG/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.5.2 eggNOG Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant orthologous groups of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_heatmap/heatmap.png" height=200 width=300/>



<p class="name_fig"> Figure 4.10 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates orthologous groups information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!--Pfam注释字段识别开始-->



    <h4> 4.2.6 Pfam Functional Annotation </h4>

    <p class="paragraph">Pfam (Mistry J et al.,2021) is a protein family database (http://pfam.xfam.org) where each protein family is represented by multiple sequence alignments and hidden Markov models. Proteins are generally composed of one or more functional regions, often referred to as domains. Different protein domains determine different protein functions, making the identification of protein domains crucial for protein function analysis. </p>

    <p class="paragraph"> Pfam (protein families) ID are categorized into six types: Family (a collection of related protein regions), Domain (a structural unit), Repeat (short, unstable units that form stable structures when present in multiple copies), Motifs (short units outside the globular domain), Coiled-Coil (regions with 2-7 α-helices forming a coiled-coil structure), and Disordered (conserved regions with compositional bias and/or intrinsic disorder). All protein families are further classified into different clan categories. </p>

    <p class="paragraph">Microbial functional analysis was conducted using HUMAnN to obtain gene IDs from UniRef90. The correspondence between UniRef90 gene IDs and Pfam database Pfam IDs was utilized to derive Pfam functional annotation results. </p>



<!--Pfam注释字段识别结束-->
<br />

<h5> 4.2.6.1 Relative Abundance of Pfam Annotation</h5>

<p class="paragraph"> Based on the abundance table of protein families, the top 10 protein families were picked out. Bar charts show the relative protein families abundance of each sample (or group). </p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_Top/top10.png" height=520 width=680/>



<p class="name_fig"> Figure 4.11 The Barplot of Relative Abundance </p>

<br/>

<p class="paragraph"> Notes: The bar chart of relative abundance represents sample (or group) names on the horizontal axis and the proportional relative abundance on the vertical axis. Each color block corresponds to a protein families as shown in the legend.</p>

<p class="paragraph"> Results directory:</p>
<p class="paragraph"> Pfam annotation visualization: result/02.Annotation/Pfam/Top_{sample, group1}/</p>

<br/>
<br />

<h5> 4.2.6.2 Pfam Abundance Heatmap </h5>

<p class="paragraph"> The abundance distribution of the top 35 dominant protein families level of all samples was displayed in the abundance heatmap. </p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_heatmap/heatmap.png" height=200 width=300/>



<p class="name_fig"> Figure 4.12 Abundance Clustering Heatmap </p>

<br/>

<p class="paragraph"> Notes: The X-axis denotes the sample (or group) names. The Y-axis illustrates protein families level information. The clustering tree is situated on the left-hand side of the chart.  The values depicted on the heatmap represent the Z-score, which is derived after normalizing the relative abundance of annotation results for each line.  The color legend positioned on the right-hand side of the figure delineates the correspondence between heat map colors and Z-values.  Notably, when the raw score falls below the mean value, "Z" assumes a negative value, whereas it is positive when surpassing the mean.</p>

<p class="paragraph"> Results directory：Heatmap of Relative abundance：result/02.Annotation/*/Heatmap*</p>

<br/>
<!------------------------------------- Alpha ---------------------------------->

<br/>

    <h3> 4.3 Alpha Diversity Analysis </h3>

<br/>

<p class="paragraph">Alpha diversity is mainly used to study the diversity of communities within a specific habitat (or sample), which can be assessed by evaluating a series of Alpha diversity indices to obtain information on the richness, diversity, and other characteristics of species and functions in the environmental community.</p>

<p class="paragraph">Through the analysis of Alpha diversity indices, information on species richness, diversity, and coverage in the community can be obtained. By using inter-group difference testing methods, one can detect whether there are significant differences in Alpha diversity index values between every two or more groups.</p>

<br />
<br />

<h4> 4.3.1 MetaPhlAn Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.1.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of taxonomic level, calculate the richness and diversity of each sample (or group).The table illustrates an example with only species level.</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.2 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>C6</td><td>114</td><td>114</td><td>3.54637286542176</td><td>0.947725012116946</td><td>114</td><td>1</td></tr>
            
        
            
        <tr><td>A1</td><td>82</td><td>82</td><td>2.96542613070899</td><td>0.908753903853257</td><td>82</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>107</td><td>107</td><td>3.41786651308298</td><td>0.936239726419583</td><td>107</td><td>1</td></tr>
            
        
            
        <tr><td>A3</td><td>83</td><td>83</td><td>3.17034047355537</td><td>0.921405217840038</td><td>83</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>79</td><td>79</td><td>2.69202423904363</td><td>0.87628654803588</td><td>79</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>92</td><td>92</td><td>2.93444733294299</td><td>0.891235423320962</td><td>92</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>43</td><td>43</td><td>2.38763917878512</td><td>0.841094372165735</td><td>43</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>28</td><td>28</td><td>1.76730689059981</td><td>0.652990471366301</td><td>28</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>35</td><td>35</td><td>2.11565259401053</td><td>0.775893262678412</td><td>35</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>71</td><td>71</td><td>2.58459716401383</td><td>0.844598397665021</td><td>71</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>27</td><td>27</td><td>2.726493813046</td><td>0.916610520011333</td><td>27</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>37</td><td>37</td><td>2.69283462784883</td><td>0.901508430934703</td><td>37</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>97</td><td>97</td><td>3.35627330435924</td><td>0.938420620416443</td><td>97</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>89</td><td>89</td><td>3.3594194426426</td><td>0.94176575179526</td><td>89</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>133</td><td>133</td><td>3.61432695737325</td><td>0.957062442645864</td><td>133</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>86</td><td>86</td><td>3.26132329317424</td><td>0.941788386353709</td><td>86</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>150</td><td>150</td><td>3.63775218190772</td><td>0.949756331661495</td><td>150</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>137</td><td>137</td><td>3.74528891578023</td><td>0.960385700873733</td><td>137</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of species in the community. </li>
<li><strong>chao1: </strong> The total number of species contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the species, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of species distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed species. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.1.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of taxonomic level, the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure6_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure6_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure6_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure6_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/phylum_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_class" role="tabpanel" aria-labelledby="gallery_figure6_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/class_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_order" role="tabpanel" aria-labelledby="gallery_figure6_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/order_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_family" role="tabpanel" aria-labelledby="gallery_figure6_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/family_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_genus" role="tabpanel" aria-labelledby="gallery_figure6_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/genus_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_species" role="tabpanel" aria-labelledby="gallery_figure6_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/species_simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_strain" role="tabpanel" aria-labelledby="gallery_figure6_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_Alpha_box/strain_simpson.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure6_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure6_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure6_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_phylum" data-toggle="list" href="#gallery_figure6_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_class" data-toggle="list" href="#gallery_figure6_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_order" data-toggle="list" href="#gallery_figure6_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_family" data-toggle="list" href="#gallery_figure6_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_genus" data-toggle="list" href="#gallery_figure6_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_species" data-toggle="list" href="#gallery_figure6_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_strain" data-toggle="list" href="#gallery_figure6_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.13 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />
</br>

<h5>4.3.1.3 Biodiversity curves </h5>

<br />

<p class="paragraph">Rarefaction Curves is widely used for indicating the biodiversity of the samples. A rarefaction curve is created by selecting randomly certain amount of sequencing data from the samples, then counting the number of the species they represent. The Rarefaction curves can directly reflect the rationality of the sequencing data volume and indirectly reflect the richness of microbial community in the samples. If the curve is steep, it means that a lot of the species remain to be discovered. If the curve becomes flatter, a credible number of samples have been taken, which means only the scarce species remain to be sampled. </p>

<p class = "center"><img class="center" src="src/files/Alpha_index_rare/richness_species.png" height=550 width=700/>

<p class="name_fig">Figure 4.14 Rarefaction Curves</p>

<br />

<p class="paragraph">Notes: in the Rarefaction curves, each curve represents a sample, and can be colored and shaped by each sample name supplied in the mapping file. The sequence number is on the X-axis and the richness is on the Y-axis.</p>

<p class="paragraph">Results directory : result/03.Alphadiversity/Taxonomy/Alpha_index_rare/richness_species.{png,pdf}</p>

<br />
<br />

<h4> 4.3.2 MetaCyc Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.2.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of metabolic pathway, calculate the richness and diversity of each sample (or group).</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.3 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>381</td><td>381</td><td>5.21325941464509</td><td>0.992301603831776</td><td>381</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>360</td><td>360</td><td>5.20164680046239</td><td>0.992082168365555</td><td>360</td><td>1</td></tr>
            
        
            
        <tr><td>A3</td><td>378</td><td>378</td><td>5.25460050411038</td><td>0.992611741170867</td><td>378</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>371</td><td>371</td><td>5.20460834312578</td><td>0.992132172546921</td><td>371</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>338</td><td>338</td><td>5.19033345910585</td><td>0.992146727283258</td><td>338</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>385</td><td>385</td><td>5.22219359813133</td><td>0.992164027916316</td><td>385</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>359</td><td>359</td><td>5.20981301389762</td><td>0.992782759406051</td><td>359</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>354</td><td>354</td><td>5.18749540754898</td><td>0.992624326329508</td><td>354</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>336</td><td>336</td><td>5.13369469173469</td><td>0.991880880112287</td><td>336</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>393</td><td>393</td><td>5.2658332342079</td><td>0.993375620350809</td><td>393</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>316</td><td>316</td><td>5.11146736004897</td><td>0.991790206836193</td><td>316</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>308</td><td>308</td><td>5.12444164352447</td><td>0.992072087575639</td><td>308</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>394</td><td>394</td><td>5.27088907914187</td><td>0.993117879719896</td><td>394</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>390</td><td>390</td><td>5.26071574510773</td><td>0.993124433919884</td><td>390</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>417</td><td>417</td><td>5.36997455539065</td><td>0.994050423798085</td><td>417</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>389</td><td>389</td><td>5.28854967399156</td><td>0.993352258643642</td><td>389</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>422</td><td>422</td><td>5.31892584939843</td><td>0.99375243266557</td><td>422</td><td>1</td></tr>
            
        
            
        <tr><td>C6</td><td>431</td><td>431</td><td>5.32713503492032</td><td>0.993591660168761</td><td>431</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of function in the community. </li>
<li><strong>chao1: </strong> The total number of function contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the function, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of function distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed function. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.2.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of metabolic pathway, the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_Alpha_box/simpson.png" height=300 width=450/>



<p class="name_fig">Figure 4.15 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />
<br />

<h4> 4.3.3 KEGG Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.3.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), calculate the richness and diversity of each sample (or group).The table illustrates an example with only KEGG orthology.</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.4 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>3339.94010416667</td><td>3340.07142857143</td><td>7.25857285347435</td><td>0.998735696520699</td><td>3339</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>3092.41345463251</td><td>3093.76923076923</td><td>7.25669134605268</td><td>0.998752438135718</td><td>3091</td><td>1</td></tr>
            
        
            
        <tr><td>A3</td><td>3600.07603848862</td><td>3600.16666666667</td><td>7.39575635097555</td><td>0.998930892438479</td><td>3599</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>3322.92010309278</td><td>3323.36363636364</td><td>7.30492841747991</td><td>0.998833136270311</td><td>3322</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>3382.24033762772</td><td>3383.54545454545</td><td>7.34992029985647</td><td>0.998876473249352</td><td>3381</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>3648.21274321146</td><td>3648.75</td><td>7.38423808472039</td><td>0.998912883903736</td><td>3647</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>3078.9200758602</td><td>3078.71428571429</td><td>7.15180343822506</td><td>0.998618732889923</td><td>3078</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>2955.12433955618</td><td>2955.75</td><td>7.14729920569224</td><td>0.998624259572125</td><td>2954</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>2810.77546296296</td><td>2810.55555555556</td><td>7.04742346221421</td><td>0.998502169633803</td><td>2810</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>3638.44672241705</td><td>3638.2</td><td>7.30344979028989</td><td>0.998871521866215</td><td>3638</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>2676.92235399418</td><td>2677.36363636364</td><td>6.93960900905256</td><td>0.998109264171032</td><td>2676</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>2615.25630593979</td><td>2615.4</td><td>6.96969046912782</td><td>0.998370611567213</td><td>2614</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>3714.60481099656</td><td>3714.6</td><td>7.4079689437426</td><td>0.998973752297532</td><td>3714</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>3739.28972214729</td><td>3739.25</td><td>7.40338259128568</td><td>0.998946516837517</td><td>3739</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>3827.7522608934</td><td>3828</td><td>7.38714519202806</td><td>0.998964268521538</td><td>3827</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>3691.15063887021</td><td>3691</td><td>7.40648423903622</td><td>0.99897013310241</td><td>3691</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>3881.5994269341</td><td>3882</td><td>7.3192943070288</td><td>0.998780462776011</td><td>3881</td><td>1</td></tr>
            
        
            
        <tr><td>C6</td><td>4019.60301507538</td><td>4019.54545454545</td><td>7.46461049343356</td><td>0.999030659268487</td><td>4019</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of function in the community. </li>
<li><strong>chao1: </strong> The total number of function contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the function, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of function distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed function. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.3.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure6_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure6_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure6_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure6_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_Alpha_box//simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure6_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_Alpha_box//simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_EC" role="tabpanel" aria-labelledby="gallery_figure6_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_Alpha_box//simpson.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure6_pane_KO" role="tabpanel" aria-labelledby="gallery_figure6_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_Alpha_box//simpson.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure6_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure6_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure6_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_KeggModule" data-toggle="list" href="#gallery_figure6_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_KeggPath" data-toggle="list" href="#gallery_figure6_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_EC" data-toggle="list" href="#gallery_figure6_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure6_list_KO" data-toggle="list" href="#gallery_figure6_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.16 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />
<br />

<h4> 4.3.4 GO Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.4.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of taxonomic level, calculate the richness and diversity of each sample (or group).</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.5 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>3385.97062474142</td><td>3385.78947368421</td><td>5.7308665030792</td><td>0.986264925346629</td><td>3385</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>3166.60714285714</td><td>3170.625</td><td>5.72366727418345</td><td>0.986568396826991</td><td>3165</td><td>1</td></tr>
            
        
            
        <tr><td>A3</td><td>3524.3093887105</td><td>3527.78378378378</td><td>5.727522059491</td><td>0.986134420703771</td><td>3519</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>3319.82113821138</td><td>3319.34482758621</td><td>5.71666288658158</td><td>0.986311523623111</td><td>3319</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>3405.81496686568</td><td>3428</td><td>5.71394783784194</td><td>0.986197622994675</td><td>3401</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>3436.95529038987</td><td>3451.07142857143</td><td>5.72526956468562</td><td>0.986279809803904</td><td>3433</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>3531.02483711555</td><td>3583.06666666667</td><td>5.78584931212155</td><td>0.987021239904071</td><td>3520</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>3633.17386978451</td><td>3652.44117647059</td><td>5.76521551443637</td><td>0.986855240540021</td><td>3622</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>3395.66182346445</td><td>3439</td><td>5.74428727862545</td><td>0.987129799744085</td><td>3388</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>3868.04457742397</td><td>3883.18867924528</td><td>5.80474211047436</td><td>0.98633557660411</td><td>3852</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>4371.82723917027</td><td>4419.64705882353</td><td>5.77163811222362</td><td>0.98734613598722</td><td>4364</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>3298.62986861308</td><td>3300.45833333333</td><td>5.76030048035766</td><td>0.987255490915319</td><td>3292</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>3513.54664484452</td><td>3519</td><td>5.74362633011115</td><td>0.986197086894927</td><td>3511</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>3480.93356643357</td><td>3481.15384615385</td><td>5.73578305059674</td><td>0.986021441499288</td><td>3480</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>3595</td><td>3595</td><td>5.74423360774007</td><td>0.985066506960719</td><td>3595</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>3441</td><td>3441</td><td>5.7376021625515</td><td>0.986088922419711</td><td>3441</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>3604.31926452152</td><td>3604.06666666667</td><td>5.71349458463031</td><td>0.983918773442263</td><td>3604</td><td>1</td></tr>
            
        
            
        <tr><td>C6</td><td>3691.31527961251</td><td>3691.1</td><td>5.74911095852095</td><td>0.985841123111256</td><td>3691</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of function in the community. </li>
<li><strong>chao1: </strong> The total number of function contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the function, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of function distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed function. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.4.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of gene ontology, the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_Alpha_box/simpson.png" height=300 width=450/>



<p class="name_fig">Figure 4.17 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />
<br />

<h4> 4.3.5 eggNOG Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.5.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of taxonomic level, calculate the richness and diversity of each sample (or group).</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.6 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>10011.2523332313</td><td>10013.453125</td><td>8.27588310284458</td><td>0.999424449106845</td><td>9999</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>9536.92899836986</td><td>9548.26086956522</td><td>8.39215462324625</td><td>0.999522052676338</td><td>9523</td><td>0.999993817588244</td></tr>
            
        
            
        <tr><td>A3</td><td>11732.859430605</td><td>11740.8770949721</td><td>8.51722994496517</td><td>0.999568210746639</td><td>11718</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>11144.8996172146</td><td>11149.3696969697</td><td>8.46385363672812</td><td>0.999529499349396</td><td>11133</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>9783.74301575931</td><td>9792.17159763314</td><td>8.45008870367403</td><td>0.999535958523133</td><td>9769</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>11712.6234056352</td><td>11716.5310734463</td><td>8.50337286606701</td><td>0.999549919849512</td><td>11700</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>11242.3504575961</td><td>11242.8993288591</td><td>8.21972677881495</td><td>0.999240459220964</td><td>11234</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>13690.1491416309</td><td>13691.6456692913</td><td>8.50436920957521</td><td>0.999306009490088</td><td>13682</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>9945.82016084304</td><td>9947.25984251968</td><td>8.06279188184709</td><td>0.999271441770805</td><td>9938</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>13638.7331061897</td><td>13642.32</td><td>8.38310811729261</td><td>0.999442367248999</td><td>13630</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>10106.5414781297</td><td>10108.0967741935</td><td>7.92530061605609</td><td>0.998228753875991</td><td>10099</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>8371.46163514826</td><td>8371.91608391608</td><td>8.00672536088917</td><td>0.999272990788623</td><td>8363</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>14361.1941356533</td><td>14365.2848101266</td><td>8.55257925446766</td><td>0.999580860699574</td><td>14350</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>14755.5878097874</td><td>14758.6066666667</td><td>8.58210337340928</td><td>0.999581018485746</td><td>14746</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>15453.080741048</td><td>15454.0821917808</td><td>8.56037864303804</td><td>0.999586791753566</td><td>15445</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>14549.9862530048</td><td>14555.5</td><td>8.60927813986978</td><td>0.999593655085905</td><td>14541</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>14824.9788734929</td><td>14829.288590604</td><td>8.28379290356722</td><td>0.999330657509177</td><td>14814</td><td>1</td></tr>
            
        
            
        <tr><td>C6</td><td>17569.632195122</td><td>17571.0653594771</td><td>8.70482139878576</td><td>0.999629605526893</td><td>17561</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of function in the community. </li>
<li><strong>chao1: </strong> The total number of function contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the function, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of function distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed function. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.5.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of orthologous groups, the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_Alpha_box/simpson.png" height=300 width=450/>



<p class="name_fig">Figure 4.18 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />
<br />

<h4> 4.3.6 Pfam Alpha Diversity Analysis </h4>

<br />

<h5> 4.3.6.1 Alpha diversity Indices </h5>

<p class="paragraph"> Starting from the annotated Reads Number table of taxonomic level, calculate the richness and diversity of each sample (or group).</p>

<br />

<p class="paragraph">                 </p>

<p style="text-align: center;font-size:14px"> Table 4.7 Statistical Results of Alpha Diversity Analysis </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Sample</th><th>ACE</th><th>chao1</th><th>shannon</th><th>simpson</th><th>observed_species</th><th>goods_coverage</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A1</td><td>5521</td><td>5521</td><td>7.23290710335576</td><td>0.998064540931389</td><td>5521</td><td>1</td></tr>
            
        
            
        <tr><td>A2</td><td>5231.15931423793</td><td>5231</td><td>7.18548260625045</td><td>0.998061752643888</td><td>5231</td><td>1</td></tr>
            
        
            
        <tr><td>A3</td><td>5653</td><td>5653</td><td>7.23444506730387</td><td>0.998065435604959</td><td>5653</td><td>1</td></tr>
            
        
            
        <tr><td>A4</td><td>5441.30427422768</td><td>5441.125</td><td>7.20438542218318</td><td>0.998044811962816</td><td>5441</td><td>1</td></tr>
            
        
            
        <tr><td>A5</td><td>5394</td><td>5394</td><td>7.20578765581081</td><td>0.998045766683574</td><td>5394</td><td>1</td></tr>
            
        
            
        <tr><td>A6</td><td>5589</td><td>5589</td><td>7.21619010206416</td><td>0.998082097142626</td><td>5589</td><td>1</td></tr>
            
        
            
        <tr><td>B1</td><td>5154.15876127009</td><td>5154</td><td>7.29051693478162</td><td>0.998435303824313</td><td>5154</td><td>1</td></tr>
            
        
            
        <tr><td>B2</td><td>5241</td><td>5241</td><td>7.28956425315807</td><td>0.998411897527381</td><td>5241</td><td>1</td></tr>
            
        
            
        <tr><td>B3</td><td>5015.62607662608</td><td>5016.2</td><td>7.19130725178</td><td>0.998298948514662</td><td>5015</td><td>1</td></tr>
            
        
            
        <tr><td>B4</td><td>5497</td><td>5497</td><td>7.34725663633755</td><td>0.998480389592123</td><td>5497</td><td>1</td></tr>
            
        
            
        <tr><td>B5</td><td>6449</td><td>6449</td><td>7.23847653623154</td><td>0.998321456750847</td><td>6449</td><td>1</td></tr>
            
        
            
        <tr><td>B6</td><td>4830.15481832543</td><td>4830</td><td>7.21397241791524</td><td>0.998348403356461</td><td>4830</td><td>1</td></tr>
            
        
            
        <tr><td>C1</td><td>5094</td><td>5094</td><td>7.18323479427253</td><td>0.997981576902558</td><td>5094</td><td>1</td></tr>
            
        
            
        <tr><td>C2</td><td>5116.30454684162</td><td>5116.14285714286</td><td>7.19224412676681</td><td>0.998000180538928</td><td>5116</td><td>1</td></tr>
            
        
            
        <tr><td>C3</td><td>5398</td><td>5398</td><td>7.2644866222914</td><td>0.998116651395681</td><td>5398</td><td>1</td></tr>
            
        
            
        <tr><td>C4</td><td>5052</td><td>5052</td><td>7.18999806466036</td><td>0.997965670799459</td><td>5052</td><td>1</td></tr>
            
        
            
        <tr><td>C5</td><td>5485.31287758347</td><td>5485.33333333333</td><td>7.30087077767014</td><td>0.998185630163822</td><td>5485</td><td>1</td></tr>
            
        
            
        <tr><td>C6</td><td>5435</td><td>5435</td><td>7.22167875893908</td><td>0.998010979375251</td><td>5435</td><td>1</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<button onclick="hiddenShow(this)">Show Annotation</button>

<div class="comment">
<ul style="font-size:14px">
<li><strong>ACE: </strong> Estimating number of function in the community. </li>
<li><strong>chao1: </strong> The total number of function contained in the community samples was estimated. </li>
<li><strong>shannon: </strong> The total number of function in the samples and their percentage. The higher the community diversity and the more evenly distributed the function, the greater the shannon index. </li>
<li><strong>simpson: </strong> Characterizes the diversity and evenness of function distribution within a community. </li>
<li><strong>observed: </strong> Number of directly observed function. </li>
<li><strong>goods_coverage: </strong> Sequencing depth index coverage, the higher the sequencing coverage, the larger the index. </li>
</ul>
</div>

<br />

<p class="paragraph"> Results directory: Alpha Diversity indices statistics: result/03.Alphadiversity/*/Alpha_index_table/*_Alphaindex.*.xls </p>

<br />
    <br />

<h5> 4.3.6.2 Alpha Diversity Indices Boxplot </h5>

<p class="paragraph"> Based on the Alpha Diversity  table of protein families, the box of the alpha diversity was displayed.The figure illustrates an example with only one index.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_Alpha_box/simpson.png" height=300 width=450/>



<p class="name_fig">Figure 4.19 Boxplot of Alpha Diversity Indices</p>

<br />

<p class="paragraph">Notes: The box plot is a graphical representation of data distribution that displays the median, quartiles, and outliers, providing a visual understanding of the data spread and extreme values. The X-axis indicates the group of samples, while the Y-axis represents the alpha diversity.</p>

<p class="paragraph"> Results directory: Boxplot of Alpha Diversity Indices: result/03.Alphadiversity/*/Alpha_box_group1/*/*png </p>

<br />

<br/>

    <h3> 4.4 Beta Diversity Analysis </h3>

<br/>

<p class="paragraph">Beta diversity is a measure of biodiversity used to describe the extent of variation between biological communities in different groups. It quantifies the differences in species composition between different groups.Beta diversity analysis includes clustering tree, PCoA, PCA and NMDS. </p>

<h4> 4.4.1 Sample Clustering with Taxonomic and Function Abundance </h4>

<p class="paragraph">Sample clustering analysis based on the Bray-Curtis distance can evaluate the similarity of samples. The distance was calculated according to relative taxonomic abundance. The final results were exhibited by combining the clustering result and the relative abundance of different samples at the taxonomic level or function level. </p>

<br/>
<br/>

<h5> 4.4.1.1 MetaPhlAn cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, The result of clustering was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure7_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure7_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure7_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure7_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.p10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_class" role="tabpanel" aria-labelledby="gallery_figure7_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.c10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_order" role="tabpanel" aria-labelledby="gallery_figure7_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.o10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_family" role="tabpanel" aria-labelledby="gallery_figure7_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.f10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_genus" role="tabpanel" aria-labelledby="gallery_figure7_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.g10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_species" role="tabpanel" aria-labelledby="gallery_figure7_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.s10.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_strain" role="tabpanel" aria-labelledby="gallery_figure7_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_ClusterTree/Bar.tree.t10.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure7_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure7_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure7_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_phylum" data-toggle="list" href="#gallery_figure7_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_class" data-toggle="list" href="#gallery_figure7_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_order" data-toggle="list" href="#gallery_figure7_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_family" data-toggle="list" href="#gallery_figure7_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_genus" data-toggle="list" href="#gallery_figure7_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_species" data-toggle="list" href="#gallery_figure7_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_strain" data-toggle="list" href="#gallery_figure7_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.20 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic level. </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















<br/>

<h5> 4.4.1.2 MetaCyc cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of metabolic pathway level, The result of clustering was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_ClusterTree/MetaCyc_ClusterTree_group1.png" height=440 width=590/>



<p class="name_fig"> Figure 4.21 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the metabolic pathway level. </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















<br/>

<h5> 4.4.1.3 KEGG cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), The result of clustering was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure7_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure7_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure7_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure7_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_ClusterTree/KeggModule_ClusterTree_group1.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure7_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_ClusterTree/KeggPath_ClusterTree_group1.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_EC" role="tabpanel" aria-labelledby="gallery_figure7_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_ClusterTree/EC_ClusterTree_group1.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure7_pane_KO" role="tabpanel" aria-labelledby="gallery_figure7_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_ClusterTree/KO_ClusterTree_group1.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure7_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure7_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure7_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_KeggModule" data-toggle="list" href="#gallery_figure7_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_KeggPath" data-toggle="list" href="#gallery_figure7_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_EC" data-toggle="list" href="#gallery_figure7_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure7_list_KO" data-toggle="list" href="#gallery_figure7_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig"> Figure 4.22 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module). </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















<br/>

<h5> 4.4.1.4 GO cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of gene ontology, The result of clustering was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_ClusterTree/GO_ClusterTree_group1.png" height=440 width=590/>



<p class="name_fig"> Figure 4.23 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the gene ontology. </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















<br/>

<h5> 4.4.1.5 eggNOG cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of orthologous group, The result of clustering was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_ClusterTree/eggNOG_ClusterTree_group1.png" height=440 width=590/>



<p class="name_fig"> Figure 4.24 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the orthologous group. </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















<br/>

<h5> 4.4.1.6 Pfam cluster analysis  </h5>

<p class="paragraph">Based on the abundance table of protein families, The result of clustering was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_ClusterTree/Pfam_ClusterTree_group1.png" height=440 width=590/>



<p class="name_fig"> Figure 4.25 The Clustering Analysis Plot of Relative Abundance </p>

<br />

<p class="paragraph"> Notes: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the protein families. </p>

<p class="paragraph"> Result directory: Clustering tree based on Bray-Curtis distance: result/04.Betadiversity/*/ClusterTree_group1/*/Bar.tree.*10.{svg,png} </p>

<br />





















</br>

<h4> 4.4.2 PCA Analysis </h4>

</br>

<p class="paragraph"> Principal Component Analysis (PCA) Analysis is a method used for dimension reduction analysis. PCA uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. The PCA transformation ensures that the horizontal axis PC1 accounts for the most variation while the vertical axis PC2 accounts for the second most (Avershina et al., 2013). For the samples' community composition, the closer the sample points within the PCA figure indicates more similarity among the samples. </p>
</br>

<h5>4.4.2.1 MetaPhlAn PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, the result of PCA was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure8_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure8_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure8_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure8_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/phylum_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_class" role="tabpanel" aria-labelledby="gallery_figure8_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/class_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_order" role="tabpanel" aria-labelledby="gallery_figure8_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/order_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_family" role="tabpanel" aria-labelledby="gallery_figure8_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/family_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_genus" role="tabpanel" aria-labelledby="gallery_figure8_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/genus_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_species" role="tabpanel" aria-labelledby="gallery_figure8_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/species_PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_strain" role="tabpanel" aria-labelledby="gallery_figure8_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCA/strain_PCA.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure8_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure8_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure8_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_phylum" data-toggle="list" href="#gallery_figure8_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_class" data-toggle="list" href="#gallery_figure8_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_order" data-toggle="list" href="#gallery_figure8_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_family" data-toggle="list" href="#gallery_figure8_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_genus" data-toggle="list" href="#gallery_figure8_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_species" data-toggle="list" href="#gallery_figure8_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_strain" data-toggle="list" href="#gallery_figure8_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.26 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h5>4.4.2.2 MetaCyc PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of metabolic pathway, the result of PCA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_PCA/PCA.png" height=440 width=590/>



<p class="name_fig">Figure 4.27 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h5>4.4.2.3 KEGG PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the result of PCA was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure8_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure8_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure8_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure8_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_PCA//PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure8_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_PCA//PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_EC" role="tabpanel" aria-labelledby="gallery_figure8_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_PCA//PCA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure8_pane_KO" role="tabpanel" aria-labelledby="gallery_figure8_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_PCA//PCA.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure8_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure8_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure8_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_KeggModule" data-toggle="list" href="#gallery_figure8_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_KeggPath" data-toggle="list" href="#gallery_figure8_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_EC" data-toggle="list" href="#gallery_figure8_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure8_list_KO" data-toggle="list" href="#gallery_figure8_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.28 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h5>4.4.2.4 GO PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of gene ontology, the result of PCA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_PCA/PCA.png" height=440 width=590/>



<p class="name_fig">Figure 4.29 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h5>4.4.2.5 eggNOG PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of orthologous groups, the result of PCA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_PCA/PCA.png" height=440 width=590/>



<p class="name_fig">Figure 4.30 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h5>4.4.2.6 Pfam PCA Analysis </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of PCA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_PCA/PCA.png" height=440 width=590/>



<p class="name_fig">Figure 4.31 PCA Analysis</p>

<br />

<p class="paragraph">Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>

<p class="paragraph">Result directory: PCA figure : result/04.Betadiversity/*/PCA_*/*/PCA_*.{png,pdf}</p>

<br /></br>

<h4> 4.4.3 PCoA Analysis </h4>

</br>

<p class="paragraph">Principal Coordinate Analysis (PCoA) is an ordination technique that picks up the primary elements and structure from reduced multi-dimensional data series of eigenvalues and eigenvectors. This technique has the advantage over PCA in that each ecological distance can be investigated. Aggregated samples represent higher species composition similarity than those that are further apart. PCoA result based on Bray-Curtis distance are shown below. </p>
</br>

<h5>4.4.3.1 MetaPhlAn PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, the result of PCoA was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure9_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure9_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure9_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure9_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/phylum_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_class" role="tabpanel" aria-labelledby="gallery_figure9_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/class_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_order" role="tabpanel" aria-labelledby="gallery_figure9_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/order_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_family" role="tabpanel" aria-labelledby="gallery_figure9_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/family_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_genus" role="tabpanel" aria-labelledby="gallery_figure9_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/genus_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_species" role="tabpanel" aria-labelledby="gallery_figure9_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/species_PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_strain" role="tabpanel" aria-labelledby="gallery_figure9_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_PCoA/strain_PCoA.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure9_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure9_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure9_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_phylum" data-toggle="list" href="#gallery_figure9_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_class" data-toggle="list" href="#gallery_figure9_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_order" data-toggle="list" href="#gallery_figure9_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_family" data-toggle="list" href="#gallery_figure9_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_genus" data-toggle="list" href="#gallery_figure9_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_species" data-toggle="list" href="#gallery_figure9_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_strain" data-toggle="list" href="#gallery_figure9_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.32 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h5>4.4.3.2 MetaCyc PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of metabolic pathway, the result of PCoA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_PCoA/PCoA.png" height=440 width=590/>



<p class="name_fig">Figure 4.33 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h5>4.4.3.3 KEGG PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the result of PCoA was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure9_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure9_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure9_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure9_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_PCoA//PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure9_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_PCoA//PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_EC" role="tabpanel" aria-labelledby="gallery_figure9_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_PCoA//PCoA.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure9_pane_KO" role="tabpanel" aria-labelledby="gallery_figure9_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_PCoA//PCoA.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure9_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure9_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure9_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_KeggModule" data-toggle="list" href="#gallery_figure9_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_KeggPath" data-toggle="list" href="#gallery_figure9_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_EC" data-toggle="list" href="#gallery_figure9_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure9_list_KO" data-toggle="list" href="#gallery_figure9_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.34 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h5>4.4.3.4 GO PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of gene ontology, the result of PCoA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_PCoA/PCoA.png" height=440 width=590/>



<p class="name_fig">Figure 4.35 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h5>4.4.3.5 eggNOG PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of orthologous groups, the result of PCoA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_PCoA/PCoA.png" height=440 width=590/>



<p class="name_fig">Figure 4.36 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h5>4.4.3.6 Pfam PCoA Analysis </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of PCoA was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_PCoA/PCoA.png" height=440 width=590/>



<p class="name_fig">Figure 4.37 PCoA Analysis</p>

<p class="paragraph">Note: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color. </p>
<p class="paragraph">Results directory: PCoA figure: result/04.Betadiversity/*/PCoA_group1/*/PCoA_*.{png,pdf} </p>

</br>

<h4> 4.4.4 NMDS Analysis </h4>

</br>

<p class="paragraph">Non-metric Multidimensional Scaling (NMDS) is a ranking method applicable to ecological research. It is a non-linear model designed for a better representation of non-linear biological data structure. NMDS can retain the original relationship among samples while reducing the research targets from a multidimensional space to a low-dimensional space for analysis, positioning, and categorization. This allows NMDS analysis to address the similarity or dissimilarity between targets as a monotonic function of the distance between the points.</p>
</br>

<h5>4.4.4.1 MetaPhlAn NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, the result of NMDS was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure10_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure10_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure10_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure10_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/phylum_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_class" role="tabpanel" aria-labelledby="gallery_figure10_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/class_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_order" role="tabpanel" aria-labelledby="gallery_figure10_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/order_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_family" role="tabpanel" aria-labelledby="gallery_figure10_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/family_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_genus" role="tabpanel" aria-labelledby="gallery_figure10_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/genus_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_species" role="tabpanel" aria-labelledby="gallery_figure10_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/species_NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_strain" role="tabpanel" aria-labelledby="gallery_figure10_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_NMDS/strain_NMDS.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure10_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure10_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure10_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_phylum" data-toggle="list" href="#gallery_figure10_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_class" data-toggle="list" href="#gallery_figure10_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_order" data-toggle="list" href="#gallery_figure10_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_family" data-toggle="list" href="#gallery_figure10_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_genus" data-toggle="list" href="#gallery_figure10_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_species" data-toggle="list" href="#gallery_figure10_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_strain" data-toggle="list" href="#gallery_figure10_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.38 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h5>4.4.4.2 MetaCyc NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of NMDS was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_NMDS/NMDS.png" height=440 width=590/>



<p class="name_fig">Figure 4.39 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h5>4.4.4.3 KEGG NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the result of NMDS was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure10_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure10_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure10_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure10_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_NMDS//NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure10_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_NMDS//NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_EC" role="tabpanel" aria-labelledby="gallery_figure10_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_NMDS//NMDS.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure10_pane_KO" role="tabpanel" aria-labelledby="gallery_figure10_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_NMDS//NMDS.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure10_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure10_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure10_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_KeggModule" data-toggle="list" href="#gallery_figure10_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_KeggPath" data-toggle="list" href="#gallery_figure10_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_EC" data-toggle="list" href="#gallery_figure10_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure10_list_KO" data-toggle="list" href="#gallery_figure10_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.40 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h5>4.4.4.4 GO NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of gene ontology, the result of NMDS was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_NMDS/NMDS.png" height=440 width=590/>



<p class="name_fig">Figure 4.41 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h5>4.4.4.5 eggNOG NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of orthologous groups, the result of NMDS was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_NMDS/NMDS.png" height=440 width=590/>



<p class="name_fig">Figure 4.42 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h5>4.4.4.6 Pfam NMDS Analysis </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of NMDS was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_NMDS/NMDS.png" height=440 width=590/>



<p class="name_fig">Figure 4.43 NMDS Analysis</p>

<br />

<p class="paragraph">Notes: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent. </p>

<p class="paragraph">Result directory: NMDS figure : result/04.Betadiversity/*/NMDS_*/*/NMDS_*.{png,pdf}</p>

<br />
</br>

<h3>4.5 Analysis of Statistical Tests for Taxonomy and Function Differences Between Groups </h3>

</br>

    <h4>4.5.1 Anosim </h4>

    <p class="paragraph"> Anosim (Chapman M G et al.,1999) (https://www.rdocumentation.org/packages/vegan/versions/2.3-5/topics/anosim) is a nonparametric test helps evaluate the reasonability of the division of groups by evaluating whether variation among groups is significantly greater than variation within group. </p>
    
</br>

<h5> 4.5.1.1 MetaPhlAn Anosim </h5>

<p class="paragraph"> Based on the abundance table of taxonomic level, the Anosim analysis was displayed.The table illustrates an example with only species level.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.8 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>1</td><td>0.004</td></tr>
            
        
            
        <tr><td>A-C</td><td>1</td><td>0.005</td></tr>
            
        
            
        <tr><td>B-C</td><td>1</td><td>0.004</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

<h5> 4.5.1.2 MetaCyc Anosim </h5>

<p class="paragraph"> Based on the abundance table of metabolic pathway, the Anosim analysis was displayed.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.9 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>0.45926</td><td>0.004</td></tr>
            
        
            
        <tr><td>A-C</td><td>0.68148</td><td>0.004</td></tr>
            
        
            
        <tr><td>B-C</td><td>0.45926</td><td>0.008</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

<h5> 4.5.1.3 KEGG Anosim </h5>

<p class="paragraph"> Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the Anosim analysis was displayed.The table illustrates an example with only KEGG orthology.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.10 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>0.84074</td><td>0.001</td></tr>
            
        
            
        <tr><td>A-C</td><td>0.89444</td><td>0.001</td></tr>
            
        
            
        <tr><td>B-C</td><td>0.91667</td><td>0.003</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

<h5> 4.5.1.4 GO Anosim </h5>

<p class="paragraph"> Based on the abundance table of gene ontolog, the Anosim analysis was displayed.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.11 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>0.81111</td><td>0.001</td></tr>
            
        
            
        <tr><td>A-C</td><td>0.82593</td><td>0.001</td></tr>
            
        
            
        <tr><td>B-C</td><td>0.85556</td><td>0.005</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

<h5> 4.5.1.5 eggNOG Anosim </h5>

<p class="paragraph"> Based on the abundance table of orthologous groups, the Anosim analysis was displayed.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.12 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>1</td><td>0.003</td></tr>
            
        
            
        <tr><td>A-C</td><td>0.93148</td><td>0.003</td></tr>
            
        
            
        <tr><td>B-C</td><td>0.9963</td><td>0.001</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

<h5> 4.5.1.6 Pfam Anosim </h5>

<p class="paragraph"> Based on the abundance table of protein families, the Anosim analysis was displayed.</p>

</br>

<p style="text-align: center;font-size:14px"> Table 4.13 Anosim analysis </p>

<p class="paragraph">                 </p>

<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Group</th><th>R-value</th><th>P-value</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>A-B</td><td>0.86111</td><td>0.004</td></tr>
            
        
            
        <tr><td>A-C</td><td>0.82222</td><td>0.003</td></tr>
            
        
            
        <tr><td>B-C</td><td>0.91296</td><td>0.002</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<p class="paragraph"> Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance. </p>

<p class="paragraph"> Results directory: Anosim result：result/05.StatisticalTest/*/Anosim_group1 </p>

<br /></br>

    <h4>4.5.2 MetaGenomeSeq </h4>

</br>

    <p class="paragraph"> MetaGenomeSeq (Paulson JN et al., 2013) is developed in the context of metagenomic data comprising multiple sample, and relies on a non-parametric t-test for detecting differentially abundant features in metagenomic studies. The p-value is further corrected as a q-value to discover taxa with significant variation. </p>
    
</br>

<h5>4.5.2.1 MetaPhlAn MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure11_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure11_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure11_pane_phylum" role="tabpanel" aria-labelledby="gallery_figure11_list_phylum" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/phylum_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_class" role="tabpanel" aria-labelledby="gallery_figure11_list_class" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/class_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_order" role="tabpanel" aria-labelledby="gallery_figure11_list_order" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/order_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_family" role="tabpanel" aria-labelledby="gallery_figure11_list_family" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/family_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_genus" role="tabpanel" aria-labelledby="gallery_figure11_list_genus" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/genus_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_species" role="tabpanel" aria-labelledby="gallery_figure11_list_species" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/species_top.12.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_strain" role="tabpanel" aria-labelledby="gallery_figure11_list_strain" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/Taxonomy_MetaGenomeSeq/strain_top.12.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure11_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure11_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure11_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_phylum" data-toggle="list" href="#gallery_figure11_pane_phylum" role="tab" aria-controls="phylum">phylum</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_class" data-toggle="list" href="#gallery_figure11_pane_class" role="tab" aria-controls="class">class</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_order" data-toggle="list" href="#gallery_figure11_pane_order" role="tab" aria-controls="order">order</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_family" data-toggle="list" href="#gallery_figure11_pane_family" role="tab" aria-controls="family">family</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_genus" data-toggle="list" href="#gallery_figure11_pane_genus" role="tab" aria-controls="genus">genus</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_species" data-toggle="list" href="#gallery_figure11_pane_species" role="tab" aria-controls="species">species</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_strain" data-toggle="list" href="#gallery_figure11_pane_strain" role="tab" aria-controls="strain">strain</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.44 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

<h5>4.5.2.2 MetaCyc MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of metabolic pathway, the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_MetaGenomeSeq/MetaGenomeSeq.png" height=440 width=590/>



<p class="name_fig">Figure 4.45 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

<h5>4.5.2.3 KEGG MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure11_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure11_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure11_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure11_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_MetaGenomeSeq//MetaGenomeSeq.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure11_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_MetaGenomeSeq//MetaGenomeSeq.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_EC" role="tabpanel" aria-labelledby="gallery_figure11_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_MetaGenomeSeq//MetaGenomeSeq.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure11_pane_KO" role="tabpanel" aria-labelledby="gallery_figure11_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_MetaGenomeSeq//MetaGenomeSeq.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure11_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure11_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure11_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_KeggModule" data-toggle="list" href="#gallery_figure11_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_KeggPath" data-toggle="list" href="#gallery_figure11_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_EC" data-toggle="list" href="#gallery_figure11_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure11_list_KO" data-toggle="list" href="#gallery_figure11_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.46 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

<h5>4.5.2.4 GO MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of gene ontology, the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_MetaGenomeSeq/MetaGenomeSeq.png" height=440 width=590/>



<p class="name_fig">Figure 4.47 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

<h5>4.5.2.5 eggNOG MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of orthologous groups, the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_MetaGenomeSeq/MetaGenomeSeq.png" height=440 width=590/>



<p class="name_fig">Figure 4.48 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

<h5>4.5.2.6 Pfam MetaGenomeSeq </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of MetaGenomeSeq was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_MetaGenomeSeq/MetaGenomeSeq.png" height=440 width=590/>



<p class="name_fig">Figure 4.49 Boxplot of Significant Changes in Relative Abundance between Groups</p>

<br />

<p class="paragraph">Notes: The X-axis indicates the group of samples; the Y-axis indicates the abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01).</p>

<p class="paragraph">Result Directory: MetaGenomeSeq analysis result: result/05.StatisticalTest/*/MetaGenomeSeq_group1</p>
</br>

    <h4>4.5.3 LEfSe </h4>

</br>

    <p class="paragraph"> To screen for significant Biomarkers with inter-group differences, different species or functions between groups are first detected using the rank-sum test method, and dimensional reduction is achieved through Linear Discriminant Analysis (LDA) to assess the magnitude of the impact of different species or functions (Segata N et al., 2011), resulting in LDA scores. The LEfSe analysis results for inter-group differences include three parts: the distribution bar plot of LDA values, the cladogram (showing the unique phylogenetic distribution of species), and abundance comparison plots of Biomarkers with statistically significant differences between groups. </p>
    









<br/>

<h5> 4.5.3.1 MetaPhlAn LEfSe </h5>

<p class="paragraph">Based on the abundance table of taxonomic level, the result of LEfSe was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Taxonomy_LEfSe/LDA_bar.png" height=440 width=590/>



<p class="name_fig">Figure 4.50 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (4 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class = "center"><img class="center" src="src/files/Taxonomy_LEfSe/LDA_tree.png" height=600 width=600/>

    <p class="name_fig">Figure 4.51 Tree of LDA Scores</p>

    <br />

    <p class="paragraph">Notes : In the Cladogram, circles radiating from the inner side to the outer side represent taxonomic rank from phylum to genus (species). Each circle stands for a distinct taxon at a corresponding taxonomic rank. The diameter of each circle represents proportionally the relative abundance of each taxon. Yellow stands for taxons with non-significant differences. Taxon (biomarkers) with significant differences are colored according to the corresponding group's color. Red nodes mean this microbiota contributes a lot in the group covered by red color, and so do the green nodes. Letters above the circles and corresponding species are annotated on the right side.</p>



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><br/>

<h5> 4.5.3.2 MetaCyc LEfSe </h5>

<p class="paragraph">Based on the abundance table of metabolic pathway, the result of LEfSe was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/MetaCyc_LEfSe/LDA_bar.png" height=440 width=590/>



<p class="name_fig">Figure 4.52 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (2 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><br/>

<h5> 4.5.3.3 KEGG LEfSe </h5>

<p class="paragraph">Based on the abundance table of KEGG orthology (KEGG enzyme/KEGG pathway/KEGG module), the result of LEfSe was shown below.</p>

<br />



    <p class="center">

    <div class="row gallery-row">

        <div class="col-1">

            <div id="gallery_figure12_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-6">

            <div class="tab-content" id="gallery_figure12_nav-tabContent">

                

                    

                        <div class="tab-pane fade active show" id="gallery_figure12_pane_KeggModule" role="tabpanel" aria-labelledby="gallery_figure12_list_KeggModule" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggModule_LEfSe//LDA_bar.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure12_pane_KeggPath" role="tabpanel" aria-labelledby="gallery_figure12_list_KeggPath" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KeggPath_LEfSe//LDA_bar.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure12_pane_EC" role="tabpanel" aria-labelledby="gallery_figure12_list_EC" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/EC_LEfSe//LDA_bar.png" /></div>

                    

                

                    

                        <div class="tab-pane show" id="gallery_figure12_pane_KO" role="tabpanel" aria-labelledby="gallery_figure12_list_KO" onclick="trans(this.id)"><img class="img-gallery" src="./src/files/KO_LEfSe//LDA_bar.png" /></div>

                    

                

            </div>

        </div>

        <div class="col-1">

            <div id="gallery_figure12_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

        </div>

        <div class="col-4">

            <input class="form-control" id="gallery_figure12_InputList" type="text" placeholder="Search..">

            <div class="list-group" id="gallery_figure12_list-tab" role="tablist">

                

                <a class="list-group-item list-group-item-action" id="gallery_figure12_list_KeggModule" data-toggle="list" href="#gallery_figure12_pane_KeggModule" role="tab" aria-controls="KeggModule">KeggModule</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure12_list_KeggPath" data-toggle="list" href="#gallery_figure12_pane_KeggPath" role="tab" aria-controls="KeggPath">KeggPath</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure12_list_EC" data-toggle="list" href="#gallery_figure12_pane_EC" role="tab" aria-controls="EC">EC</a>

                

                <a class="list-group-item list-group-item-action" id="gallery_figure12_list_KO" data-toggle="list" href="#gallery_figure12_pane_KO" role="tab" aria-controls="KO">KO</a>

                

            </div>

        </div>

    </div>



<p class="name_fig">Figure 4.53 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (2 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><br/>

<h5> 4.5.3.4 GO LEfSe </h5>

<p class="paragraph">Based on the abundance table of gene ontology, the result of LEfSe was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/GO_LEfSe/LDA_bar.png" height=440 width=590/>



<p class="name_fig">Figure 4.54 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (2 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><br/>

<h5> 4.5.3.5 eggNOG LEfSe </h5>

<p class="paragraph">Based on the abundance table of orthologous groups, the result of LEfSe was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/eggNOG_LEfSe/LDA_bar.png" height=440 width=590/>



<p class="name_fig">Figure 4.55 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (2 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><br/>

<h5> 4.5.3.6 Pfam LEfSe </h5>

<p class="paragraph">Based on the abundance table of protein families, the result of LEfSe was shown below.</p>

<br />



    <p class = "center"><img class="center" src="src/files/Pfam_LEfSe/LDA_bar.png" height=440 width=590/>



<p class="name_fig">Figure 4.56 Histogram of LDA Scores</p>

<br />

<p class="paragraph"> Notes: The histogram of the LDA scores presents biomarker whose LDA scores are greater than the set threshold (2 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

<br />



    <p class="paragraph"> Result Directory: LEfSe analysis result: result/05.StatisticalTest/*/Lefse_group1 </p><h2>5.Appendix</h2>

    <h3>5.1 List of Analysis Software </h3>
    
    <p class="paragraph">                 </p>

    <div class="table-container">
        <table class="gy" data-pagination="true" data-toggle="table" data-page-size="30">
        
            <thead>
            
                <tr>
                
                    <th>Analysis</th><th>Software</th><th>Version</th><th>Parameter</th><th>Remarks</th>
                
                </tr>
                
            </thead>
            
            <tbody>
                
                
                
                <tr><td>Data quality</td><td>fastp</td><td>0.23.1</td><td>-g -q 5 -u 50 -n 10 -l 150 --min_trim_length 10 --overlap_diff_limit 1</td><td>-</td></tr>
                
                
                
                <tr><td>Host trimming</td><td>bowtie2</td><td>2.5.4</td><td>  --end-to-end --sensitive --no-hd --no-sq -I 200 -X 400 --threads 8</td><td>-</td></tr>
                
                
                
                <tr><td>Taxonomy annotation</td><td>MetaPHlAn4</td><td>4.1.1</td><td>--input_type fastq --nproc 5</td><td>ChocoPhlAn(Version: 2024)</td></tr>
                
                
                
                <tr><td>MetaCyc annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>KEGG(Version: 2023)</td></tr>
                
                
                
                <tr><td>KEGG PATHWAY annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>KEGG(Version: 2023)</td></tr>
                
                
                
                <tr><td>KEGG MODULE annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>KEGG(Version: 2023)</td></tr>
                
                
                
                <tr><td>KEGG Orthology annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>KEGG(Version: 2023)</td></tr>
                
                
                
                <tr><td>KEGG ENZYME annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>KEGG(Version: 2023)</td></tr>
                
                
                
                <tr><td>GO annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>GO(Version: 2023.03)</td></tr>
                
                
                
                <tr><td>eggNOG annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>eggNOG(Version: 2023.03)</td></tr>
                
                
                
                <tr><td>Pfam annotation</td><td>HUMAnN3</td><td>3.9</td><td>--minpath off --gap-fill on --threads 5 --protein-database</td><td>Pfam(Version: 2023.03)</td></tr>
                
                
                
                <tr><td>Barplot</td><td>perl</td><td>5.18.2</td><td>use SVG</td><td>-</td></tr>
                
                
                
                <tr><td>Cluster heatmap</td><td>R</td><td>3.1.0</td><td>R package = pheatmap</td><td>-</td></tr>
                
                
                
                <tr><td>Alpha diversity</td><td>R</td><td>2.15.3</td><td>R package = vegan</td><td>-</td></tr>
                
                
                
                <tr><td>Cluster tree</td><td>perl</td><td>5.18.2</td><td>use SVG</td><td>-</td></tr>
                
                
                
                <tr><td>PCA</td><td>R</td><td>3.0.3</td><td>R package = FactoMineR， ggplot2，grid</td><td>-</td></tr>
                
                
                
                <tr><td>NMDS</td><td>R</td><td>2.15.3</td><td>R package = vegan,permute,lattice</td><td>-</td></tr>
                
                
                
                <tr><td>PCoA</td><td>R</td><td>2.15.3</td><td>R package = extrafont,ggplot2,grid</td><td>-</td></tr>
                
                
                
                <tr><td>Anosim</td><td>R</td><td>2.15.3</td><td>R package = vegan,permute,lattice</td><td>-</td></tr>
                
                
                
                <tr><td>MetaGenomeSeq</td><td>R</td><td>2.15.3</td><td>q value (Benjamini and Hochberg FDR) &lt; 0.05</td><td>-</td></tr>
                
                
                
                <tr><td>MetaGenomeSeq heatmap</td><td>R</td><td>3.1.0</td><td>R package = pheatmap</td><td>-</td></tr>
                
                
                
                <tr><td>MetaGenomeSeq combined</td><td>R</td><td>2.15.3</td><td>R package = ggplot2</td><td>-</td></tr>
                
                
                
                <tr><td>LEfSe</td><td>LEfSe</td><td>1</td><td>MetaPHlAn LDA Score&gt;4，function LDA Score&gt;2</td><td>-</td></tr>
                
                
            
            </tbody>
            
        </table>
        
    </div>
    
    <style>
        .table-container {
            width: 100%;  /* 可以根据需要调整 */
            max-height: 800px; /* 设置容器的最大高度 */
            overflow: auto; /* 启用滚动条 */
            border: 1px solid #ccc; /* 可选的边框 */
        }
    
        table {
            width: 100%; /* 让表格宽度填满容器 */
            border-collapse: collapse; /* 合并边框 */
        }
    
        th, td {
            border: 1px solid #ddd; /* 单元格边框 */
            padding: 8px; /* 单元格内边距 */
            text-align: left; /* 文本左对齐 */
         }
    
        th {
            background-color: #f2f2f2; /* 表头背景色 */
         }
    </style>

<br />

    <h3>5.2 Methods</h3>

    <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.<a href="src/pdf_images/Sequencing_Methods_en.pdf" title="Click to open" >methods</a></p>

    <h3>5.3 Result File Decompression Tool and Format Description </h3>
    
    <p class="paragraph">                 </p>


<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">
    
    <thead>
        
        <tr>
            
            <th>Compressed.format</th><th>Customer.type</th><th>Uncompressed.method</th>
            
        </tr>
            
    </thead>
        
    <tbody>
            
        
            
        <tr><td>compressed files in the fomat of *.tar:</td><td>Unix/Linux/Mac user</td><td>use tar -xvf *.tar command</td></tr>
            
        
            
        <tr><td>compressed files in the fomat of *.tar:</td><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>
            
        
            
        <tr><td>compressed files in the format of *.gz:</td><td>Unix/Linux/Mac user</td><td>use gzip -d *.gz command</td></tr>
            
        
            
        <tr><td>compressed files in the format of *.gz:</td><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>
            
        
            
        <tr><td>compressed files in the format of *.zip:</td><td>Unix/Linux/Mac user</td><td>use unzip *.zip command</td></tr>
            
        
            
        <tr><td>compressed files in the format of *.zip:</td><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>
            
        
        
    </tbody>
        
</table>

<br />

<h2>6 References</h2>

<p class="ref">

Avershina, E., Frisli, T., & Rudi, K. (2013). De novo semi-alignment of 16S rRNA gene sequences for deep phylogenetic characterization of next generation sequencing data. Microbes and Environments, 28(2), 211–216. https://doi.org/10.1264/jsme2.me12157.</br></br>

Blanco-Míguez, A., Beghini, F., Cumbo, F., et al. (2023). Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. Nature Biotechnology, 41(11), 1633–1644. https://doi.org/10.1038/s41587-023-01688-w.</br></br>

Chapman, M., & Underwood, A. (1999). Ecological patterns in multivariate assemblages: information and interpretation of negative values in ANOSIM tests. Marine Ecology Progress Series, 180, 257–265. https://doi.org/10.3354/meps180257.</br></br>

Franzosa, E. A., McIver, L. J., Rahnavard, G., et al. (2018). Species-level functional profiling of metagenomes and metatranscriptomes. Nature Methods, 15(11), 962–968. https://doi.org/10.1038/s41592-018-0176-y.</br></br>

Handelsman, J., Rondon, M. R., Brady, S. F., et al. (1998). Molecular biological access to the chemistry of unknown soil microbes: a new frontier for natural products. Chemistry & Biology, 5(10), R245-R249. https://doi.org/10.1016/s1074-5521(98)90108-9.</br></br>

Harris, M. A., Clark, J., Ireland, A., et al. (2004). The Gene Ontology (GO) database and informatics resource. Nucleic Acids Research, 32(Database issue), D258-D261. https://doi.org/10.1093/nar/gkh036.</br></br>

Huerta-Cepas, J., Szklarczyk, D., Heller, D., et al. (2019). eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Research, 47(D1), D309-D314. https://doi.org/10.1093/nar/gky1085.</br></br>

Kanehisa, M., & Goto, S. (2000). KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Research, 28(1), 27–30. https://doi.org/10.1093/nar/28.1.27.</br></br>

Karp, P. D., Riley, M., Paley, S. M., & Pellegrini-Toole, A. (2002). The MetaCyc Database. Nucleic Acids Research, 30(1), 59–61. https://doi.org/10.1093/nar/30.1.59.</br></br>

Mistry, J., Chuguransky, S., Williams, L., et al. (2021). Pfam: The protein families database in 2021. Nucleic Acids Research, 49(D1), D412-D419. https://doi.org/10.1093/nar/gkaa913.</br></br>

Paulson, J. N., Stine, O. C., Bravo, H. C., & Pop, M. (2013). Differential abundance analysis for microbial marker-gene surveys. Nature Methods, 10(12), 1200–1202. https://doi.org/10.1038/nmeth.2658.</br></br>

Segata, N., Izard, J., Waldron, L., et al. (2011). Metagenomic biomarker discovery and explanation. Genome Biology, 12(6), R60. https://doi.org/10.1186/gb-2011-12-6-r60.</br></br>

Truong, D. T., Tett, A., Pasolli, E., Huttenhower, C., & Segata, N. (2017). Microbial strain-level population structure and genetic diversity from metagenomes. Genome Research, 27(4), 626–638. https://doi.org/10.1101/gr.216242.116.</br></br>

van der Heijden, M. G., Bardgett, R. D., & van Straalen, N. M. (2008). The unseen majority: soil microbes as drivers of plant diversity and productivity in terrestrial ecosystems. Ecology Letters, 11(3), 296–310. https://doi.org/10.1111/j.1461-0248.2007.01139.x.</br></br>

Walter, J., & Ley, R. (2011). The human gut microbiome: ecology and recent evolutionary changes. Annual Review of Microbiology, 65, 411–429. https://doi.org/10.1146/annurev-micro-090110-102830.</br></br>

</p>
