<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >

<html>

<head>

<TITLE>Chromatin Immunoprecipitation Sequencing Analysis Report</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />



<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>

<script type="text/javascript">

        $(function () {

                var options = {

                        selectors: "h2,h3,h4,h5",

                        theme: "bootstrap",

                        scrollTo: "60"

                };

                var toc = $("#toc").tocify(options).data("toc-tocify");

        });

</script>





<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");
if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");


break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});
$(".list-group").each(show_default);

});


</script>

    <script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
  </script>

  <script type="text/javascript">
    /* show or hide element*/
    function hiddenQA(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Q&A";
      };
      return 0;
    }
  </script>


</head>



<body>



<div id="toc" class="tocify">

</div>





	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

			<tr height="86%">

				<td width='1%'></td>

				<td><img src="src/images/logo_slogan.png"></td>

				<td>

					<div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>

				</td>
				<td width="20%"></td>

				<td width="15%"></td>

				<td width="25%"

					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">

					<br><a href="mailto:op-rna@novogene.com" target="black"

						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">

						<a></td>

				<td width="1%"></td>

			</tr>

		</table>

	</div>

<div id="main">
<div id="page">
</br><h1>Chromatin Immunoprecipitation Sequencing Analysis Report</h1></br>
<table class= 'gy' style="text-align:left">
<tbody>
<tr><td>Contract ID</td><td style="text-align:left">HXXXXXXXXXXXXXX</td></tr>
<tr><td>Contract Name</td><td style="text-align:left">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX合同</td></tr>
<tr><td>Batch ID</td><td style="text-align:left">XXXXXXXXXXXXXX-ZXX-JXXX</td></tr>
<tr><td>Reference Genome and Version</td><td style="text-align:left">ensembl_94_mus_musculus_grcm38_primary</td></tr>
<tr><td>Report Time</td><td style="text-align:left">2024-12-09</td></tr>
<tr><td>Warm Tips</td><td style="text-align:left">Partial results are presented in this report, while full results will be delivered in data release.<br>Hyperlink of results in this report will be only valid in data release, after statement confirmation.</td></tr>
</tbody></table>
</p></div>

<!-------------------------------------------Introduction------------------------------------------->
<div id="page0">
</p><br />
<h2><a id="Introduction">1 Introduction</a></h2>
<p class="paragraph" >
Chromatin Immunoprecipitation Sequencing (ChIP-seq) provides genome-wide profiling of DNA targets for histone modification, transcription factors, and other DNA-associated proteins. It combines the selectivity of chromatin immunoprecipitation (ChIP) for recovering specific protein-DNA complexes with the power of Next Generation Sequencing (NGS) for high-throughput sequencing of the recovered DNA. Applications include studies on gene regulation, transcription complex assembly, histone modification, developmental mechanisms, and disease processes. Because the protein-DNA complexes are recovered from living cells, binding sites can be compared in different cell types, tissues or under different conditions. 
</p>
<p class="paragraph" >
At Novogene, we can provide you with high quality sequencing and comprehensive bioinformatics analysis for your ChIP-Seq project. From DNA samples to final data report, each step of sample QC, library preparation and sequencing may have an impact on the quality and quantity of data output, and the quality of data will directly affect the results of subsequent information analysis. Therefore, obtaining high-quality data is the premise to ensure that bioinformatics analysis is correct, comprehensive and credible. In order to ensure the accuracy and reliability of sequencing data from the beginning, Novogene strictly checks every experimental step and fundamentally ensures the output of high-quality data. The workflow is shown as follows:
</p>
<br/><br/>
<p class="center">
<img src="src/images/ChIP-Seq_project_workflow.png"  width=700 height=400 />
</p>
<p align="center"><b>Figure 1 Project workflow</b></p>
</div>



<!-------------------------------Library Construction and Sequencing---------------------------------->
<div id="page1">
</p><br />
<h2 id="id_order_pipeline"><a name="Analysis process">2 Library Construction and Sequencing</a></h2>
<h3 id="id_order_DNA_sample_test" ><a name="DNA sample test">2.1 Sample Quality Control</a></h3>
<p class="paragraph" > Please refer to Novogene’s QC report for methods of sample quality control.</p>

<h3 id="id_order_Library_Preparation_for_Sequencing" ><a name="Library Preparation for Sequencing">2.2 Library Construction, Quality Control and Sequencing</a></h3>
<p class="paragraph" >The DNA fragments were repaired, A-tailed and further ligated with Illumina adapter. The final DNA library was obtained by size selection and PCR amplification. The experimental procedures of DNA library preparation are shown in Figure 2:</p>
<p class="center">
<img src="src/images/ChIP-seq_library_construction_workflow.png" width=150 height=300 />
</p>
<p align="center"><b>Figure 2 Library construction workflow</b></p>
<p class="paragraph" >The library was checked with Qubit and real-time PCR for quantification and bioanalyzer for size distribution detection. Quantified libraries will be pooled and sequenced on Illumina platforms, according to effective library concentration and data amount required</p>
</div>

<!-------------------------------Bioinformatics Analysis Pipeline---------------------------------->
<div id="page1">
</p><br />
<h2 id="id_order_pipeline"><a name="Analysis process">3 Bioinformatics Analysis Pipeline</a></h2>
<p class="paragraph" >The bioinformatic analysis workflow of raw sequencing data is as follows:</p>
<p class="center">
<img src="src/images/ChIP-Seq_analysis_pipeline.png"  width=800 height=500 />
</p>
<p align="center"><b>Figure 3 Bioinformatic analysis workflow</b></p>
</div>



<!----------------------------------------Analysis Results------------------------------------------------>
<!----------------------------------------- FastQC of raw reads-------------------------------------->
<!-------------------------------------------- Trimming --------------------------------------->

</p>
<h2><a id="Analysis Results">4 Analysis Results</a></h2>
<h3 id="id_order_trimming" ><a name="Trimming">4.1 Data Quality Control</a></h3>
<h4>4.1.1 Raw Data</h4>
<p class="paragraph" >
The original raw data from Illumina platform are transformed to Sequenced Reads, known as Raw Data or RAW Reads, by base calling of CASAVA. Raw data are recorded in a FASTQ file, which contains sequencing reads and corresponding sequencing quality. Every read in FASTQ format is stored in four lines, as indicated below:
</p>
<p class="seq">
@HWI-ST1276:71:C1162ACXX:1:1101:1208:2458 1:N:0:CGATGT&nbsp;&nbsp;&nbsp;<br />
NAAGAACACGTTCGGTCACCTCAGCACACTTGTGAATGTCATGGGATCCAT<br />
+<br />
#55???BBBBB?BA@DEEFFCFFHHFFCFFHHHHHHHFAE0ECFFD/AEHH<br />
</p>
<table class="gy">
	<tr>

			<th>Identifier</th>

			<th>Meaning</th>


</tr>
	<tr>

			<td>HWI-ST1276</td>

			<td>Instrument – unique identifier of the sequencer</td>
</tr>
	<tr>
<td>71</td>
<td>Run number – Run number on instrument</td>
</tr>
	<tr>
<td>C1162ACXX</td>
<td>Flowcell ID - ID of flowcell</td>
</tr>
	<tr>
<td>1</td>
<td>LaneNumber - positive integer</td>
</tr>
	<tr>
<td>1101</td>
<td>TileNumber - positive integer</td>
</tr>
	<tr>
<td>1208</td>
<td>X - x coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>2458</td>
<td>Y - y coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>1</td>
<td>ReadNumber - 1 for single reads, 1 or 2 for paired ends</td>
</tr>
	<tr>
<td>N</td>
<td>whether it is filtered - NB: Y if the read is filtered out, not in the delivered fastq files, otherwise N</td>
</tr>
	<tr>
<td>0</td>
<td>Control number - 0 when none of the control bits are on. Otherwise it is an even number</td>
</tr>
	<tr>
<td>CGATGT</td>
<td>Illumina index sequences</td>
</tr>
</table>
<p class="paragraph">Line 1 begins with a '@' character and is followed by the Illumina Sequence Identifiers and an optional description. 
</p>
<p class="paragraph">
Line 2 is the raw sequence read.</p>
<p class="paragraph">
Line 3 begins with a '+' character and is optionally followed by the same sequence identifiers and descriptions as in Line 1.</p>
<p class="paragraph">
Line 4 encodes the quality values for the sequence in Line 2 and must contain the same number of characters as the bases in the sequence.</p>
<p class="paragraph">Sequencing process itself has the possibility of machine errors. Sequencing error rate distribution can reflect the quality of sequencing data. The quality value of each base in sequence information is stored in the FASTQ file. Quality value of the read represented by Qphred and the “e” represents the sequence error rate, Qphred=-10log<sub>10</sub><sup>(e)</sup>. The relationship between sequencing error rate (e) and sequencing base quality value (Qphred) is as following(Base Quality and Phred score relationship with the Illumina CASAVA v1.8 software).</p>
<table class="gy"  >
<tr><th>Phred Score</th><th>Error Rate</th><th>Correct Rate</th><th>Q-score</th></tr>
<tr><td>10</td><td>1/10</td><td>90%</td><td>Q10</td></tr>
<tr><td>20</td><td>1/100</td><td>99%</td><td>Q20</td></tr>
<tr><td>30</td><td>1/1000</td><td>99.9%</td><td>Q30</td></tr>
<tr><td>40</td><td>1/10000</td><td>99.99%</td><td>Q40</td></tr>
</table>
<p class="paragraph">(1) Due to the gradual consumption of reagents during sequencing, the sequencing error rate increases with the length of read, which is a common feature of the Illumina high-throughput sequencing platform (Erlich et al., 2008; Jiang et al., 2011.).</p>
<p class="paragraph">(2) The first six bases have a relatively high error rate due to the incomplete binding of random hexamers used in priming cDNA synthesis. In general, a single base error rate should be lower than 1%.</p>

<h4>4.1.2 FastQC of Raw_reads</h4>
<p class="paragraph" >
As the high-throughput sequencing technology getting mature, we can obtain a mass amount of data easily. The key is how to extract the information that we need from the data. The first step is to check the raw data quality before doing follow-up data analysis. The popular tool for the quality control is software FastQC (Andrews S., 2010). Summary of the raw data quality control is in the following:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure1_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_1_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_raw_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_raw_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_raw_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_1_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure1_1_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_raw_fastqc.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure1_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_St_b1"
data-toggle="list" href="#gallery_figure1_1_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_St_b2"
data-toggle="list" href="#gallery_figure1_1_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_St_b3"
data-toggle="list" href="#gallery_figure1_1_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_St_in"
data-toggle="list" href="#gallery_figure1_1_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.1 Raw reads of FastQC</b></p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Top-left: horizontal axis represents position in read (bp), vertical axis represents quality scores. Different colors represent different quality range;</br>
(2) Top-middle: horizontal axis represents position in read (bp), vertical axis represents base percentage; </br>
(3) Top-right: the distribution of read length;</br>
(4) Bottom-left: GC distribution over all sequences. Red line is the actual GC content distribution, blue line is the theoretical distribution; </br>
(5) Bottom-middle: N-ratio across all bases; </br>
(6) Bottom-right: x-axis is the sequence duplication level. y-axis is the percent of the deduplicated reads.
</br>
</ul>
</div>



</p>
<h4>4.1.3 Raw Data Trimming </h4>
<p class="paragraph" >
For the raw data which pass the quality control, primer mismatch may result in nucleotidic composition bias at first several positions of the reads, which could lead to wrong bases insertion during sequencing. Because of the small fragment size of ChIP-Seq, there are often some adapter-appended reads. Trimming off the adapter sequences and low-quality bases is necessary. The fastp is used to filter raw data and all the follow-up analysis are based on these clean data.
</p>
<p class="paragraph" >The procedure for data trimming is in the following:</p>
<p class="paragraph" >(1) Discard the reads with low quality (proportion of low quality bases larger than 40%);</p>
<p class="paragraph" >(2) Discard the reads with N number (unsure base) larger than 6;</p>
<p class="paragraph" >(3) Discard the reads shorter than 50bp after removing the adapter;</p>


<p class="name_table" align="center">Table 4.1 Summary of raw data quality control</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>Sample</th>
		<th>Raw_reads</th>
		<th>Raw Base(G)</th>
		<th>Clean_reads</th>
		<th>Clean_bases(G)</th>
		<th>Clean_ratio</th>
		<th>GC_content</th>
		<th>Q20_raw</th>
		<th>Q30_raw</th>
		<th>Q20_clean</th>
		<th>Q30_clean</th>
	</tr>
	
	<tr>
		<td>IP_1</td>
		<td>21693716</td>
		<td>6.51</td>
		<td>20452567</td>
		<td>6.08</td>
		<td>93.39%</td>
		<td>53.26%</td>
		<td>96.45%</td>
		<td>91.50%</td>
		<td>98.03%</td>
		<td>94.20%</td>
	</tr>
	
	<tr>
		<td>IP_2</td>
		<td>36030697</td>
		<td>10.81</td>
		<td>35169557</td>
		<td>10.47</td>
		<td>96.85%</td>
		<td>50.97%</td>
		<td>97.47%</td>
		<td>93.32%</td>
		<td>98.29%</td>
		<td>94.73%</td>
	</tr>
	
	<tr>
		<td>IP_3</td>
		<td>26198140</td>
		<td>7.86</td>
		<td>25739187</td>
		<td>7.69</td>
		<td>97.84%</td>
		<td>50.62%</td>
		<td>98.27%</td>
		<td>95.39%</td>
		<td>98.80%</td>
		<td>96.34%</td>
	</tr>
	
	<tr>
		<td>Input</td>
		<td>21442459</td>
		<td>6.43</td>
		<td>20999751</td>
		<td>6.26</td>
		<td>97.36%</td>
		<td>54.13%</td>
		<td>97.81%</td>
		<td>94.24%</td>
		<td>98.48%</td>
		<td>95.39%</td>
	</tr>
	
</table>
</div>


<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment" display="none">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Sample: sample name;<br/>
(2) Raw_reads: the read counts of raw data;<br/>
(3) Raw_bases(G): read counts of raw data multiplied by read length, saved in G unit;<br/>
(4) Clean_reads: the read counts after trimming filtering;<br/>
(5) Clean_bases(G): clean read counts multiplied by read length, saved in G unit;<br/>
(6) Clean_ratio: the proportion of clean bases in raw bases;<br/>
(7) GC_content: the proportion of G and C in all the bases;<br/>
(8) Q20_raw: the proportion of bases with quality score more than 20 of raw data;<br/>
(9) Q30_raw: the proportion of bases with quality score more than 30 of raw data;<br/>
(10) Q20_clean: the proportion of bases with quality score more than 20 of clean data;<br/>
(11) Q30_clean: the proportion of bases with quality score more than 30 of clean data;<br/>
</ul>
</div>
<p class="paragraph" >
Raw data filtering details:</br>
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure_trimming_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure_trimming_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure_trimming_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure_trimming_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1.Classification_of_Raw_Reads.png" /></div>

<div class="tab-pane fade" id="gallery_figure_trimming_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure_trimming_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2.Classification_of_Raw_Reads.png" /></div>

<div class="tab-pane fade" id="gallery_figure_trimming_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure_trimming_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3.Classification_of_Raw_Reads.png" /></div>

<div class="tab-pane fade" id="gallery_figure_trimming_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure_trimming_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in.Classification_of_Raw_Reads.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure_trimming_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure_trimming_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure_trimming_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure_trimming_list_St_b1"
data-toggle="list" href="#gallery_figure_trimming_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_trimming_list_St_b2"
data-toggle="list" href="#gallery_figure_trimming_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_trimming_list_St_b3"
data-toggle="list" href="#gallery_figure_trimming_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_trimming_list_St_in"
data-toggle="list" href="#gallery_figure_trimming_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.2 Raw data filtering details</b></p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Low quality: The number of filtered reads and their proportion to the total raw reads due to low quality;</br>
(2) Too_Many_N: The number of filtered reads and their proportion to the total raw reads due to N content exceeding 10%;</br>
(3) Too_Short: The proportion of raw reads with a length shorter than 50bp after trimming;</br>
(4) Trimmed_With_Adapter: The proportion of raw reads with a length shorter than 50bp, after trimming and removing adapter;</br>
(5) Non_Trimmed: The proportion of raw reads that have not been trimmed</br>
</ul>
</div>

<!--------------------------------------- FastQC of clean reads-------------------------------------->
<div id="page4">
</p><br />

<h4>4.1.4 FastQC of Clean_reads</h4>
<p class="paragraph" >
Quality control of the clean reads after trimming is in the following:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure1_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_2_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_clean_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_clean_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_clean_fastqc.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure1_2_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_clean_fastqc.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure1_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_St_b1"
data-toggle="list" href="#gallery_figure1_2_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_St_b2"
data-toggle="list" href="#gallery_figure1_2_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_St_b3"
data-toggle="list" href="#gallery_figure1_2_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_St_in"
data-toggle="list" href="#gallery_figure1_2_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.3 FastQC of Clean_reads</b></p>

 <button onclick="hiddenShow(this)">Show Annotation</button>
 <div class="comment">
 <ul style="font-size: 12px;font-family:line-height: 20px">
 (1) Top-left: horizontal axis represents position in read (bp), vertical axis represents quality scores, different colors represent different quality range.</br>
 (2) Top-middle: horizontal axis represents position in read (bp), vertical axis represents base percentage.</br>
 (3) Top-right: the distribution of read length.</br>
 (4) Bottom-left: mean GC content (%). Red line is the actual GC content distribution, blue line is the theoretical distribution.</br>
 (5) Bottom-middle: N-ratio across all bases.</br>
 (6) Bottom-right: x-axis is the sequence duplication level. y-axis is the percent of the deduplicated reads.</br>
 </ul></div>
</div>



<!------------------------------- 参考基因组比对----------------------------------->
<div id="page5">
</p><br />
<h3 id="id_order_Mapping" ><a name="Mapping">4.2 Mapping</a></h3>
<h4>4.2.1 Summary of mapping</h4>

<p class="paragraph">

The common tools for mapping are Bowtie, BWA, MAQ, TopHat, etc. We choose proper software and parameters according to different genome characters to do the genome mapping analysis for the filtered reads. Considering the small fragment size of ChIP-Seq, and the percentage of the unique sequence in the total sequence is the most important information, thus, we can map the reads to the reference genome using BWA much more accurately (Li et al., 2009). The summary of mapping is in the following table:

</p>
<p class="name_table" align="center">Table 4.2 Summary of mapping</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>Sample</th>
		<th>Clean_reads</th>
		<th>Mapped</th>
		<th>Unique_mapped</th>
		<th>Dup_Unique_mapped</th>
		<th>Unique_mapped_Deduplicated</th>
	</tr>
	
	<tr>
		<td>IP_1</td>
		<td>20452567</td>
		<td>11019827(53.88%)</td>
		<td>6934034(62.92%)</td>
		<td>3168913</td>
		<td>3765121</td>
	</tr>
	
	<tr>
		<td>IP_2</td>
		<td>35169557</td>
		<td>19395671(55.15%)</td>
		<td>14043699(72.41%)</td>
		<td>7175334</td>
		<td>6868365</td>
	</tr>
	
	<tr>
		<td>IP_3</td>
		<td>25739187</td>
		<td>10628166(41.29%)</td>
		<td>8004261(75.31%)</td>
		<td>4395022</td>
		<td>3609239</td>
	</tr>
	
	<tr>
		<td>Input</td>
		<td>20999751</td>
		<td>12176481(57.98%)</td>
		<td>8612673(70.73%)</td>
		<td>3888772</td>
		<td>4723901</td>
	</tr>
	
</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Sample: sample name; </br>
(2) Clean_reads: the read counts after trimming filtering; </br>
(3) Mapped: the reads mapped to the genome (the precentage is relatively to Clean_reads); </br>
(4) Unique_mapped: the uniquely mapped reads (the precentage is relatively to Clean_reads); </br>
(5) Dup_Unique_mapped: the reads mapped to the exact same position of the genome (the precentage is relatively to Unique_mapped_reads)</br>
(6) Unique_mapped_Deduplicated: The number of reads after removing Dup_Unique_mapped reads from Unique_mapped reads.</br>
</ul>
</div>
</br></br>

<h4>4.2.2 MAPQ</h4>
<p class="paragraph">
The most important thing during ChIP-Seq analysis is the percentage of the unique sequence in the total sequence number. Duplicates are labeled using SAMBLASTER (Faust et al., 2014) and mapping quality value is calculated by MAPQ. Proper quality value is chosen as the only threshold for mapping. Here we choose 13 as the threshold, which refer to that the mapping chance of the accordingly non-unique region is only 0.05. Only one read pair is kept for the duplicates in the followed peak calling.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_1_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_MAPQ_stat.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_MAPQ_stat.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_MAPQ_stat.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure2_1_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_MAPQ_stat.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_St_b1"
data-toggle="list" href="#gallery_figure2_1_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_St_b2"
data-toggle="list" href="#gallery_figure2_1_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_St_b3"
data-toggle="list" href="#gallery_figure2_1_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_St_in"
data-toggle="list" href="#gallery_figure2_1_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.4 Distribution of MAPQ</b></p>
<p><small>Horizontal axis is MAPQ, vertical axis is the reads count.</small></p>
</div>

<div id="page6">
<br />
<h4>4.2.3 Genome-wide distribution of the mapped reads</h4>
<p class="paragraph">

Summary of the density of total mapped reads in different chromosomes (plus and minus) is shown in the followed figure. With 5k slide window size, the medium of the number of reads mapped to each base within window is calculated, and converted to log<sub>2</sub>. The longer the whole chromosome is, the more reads are mapped (Marquez et al. 2012). From the correlation of the number of the mapped reads and the length of the chromosome in the figure, we can see the correlation of the total number of the reads and the length of the chromosome much easier.

</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_2_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_Reads_on_chromosome_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_Reads_on_chromosome_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_Reads_on_chromosome_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_2_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure2_2_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_Reads_on_chromosome_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_St_b1"
data-toggle="list" href="#gallery_figure2_2_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_St_b2"
data-toggle="list" href="#gallery_figure2_2_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_St_b3"
data-toggle="list" href="#gallery_figure2_2_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_2_list_St_in"
data-toggle="list" href="#gallery_figure2_2_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.5 Genomewide distribution of the mapped reads</b></p>
<p><small>
Horizontal axis represents the postition of the chromosome, vertical axis represents the number of the reads mapped to 1000nt window size. Here is the unique mapping and deduplication results.
</small></p>



<h4>4.2.4 Distribution of Reads relative to gene body</h4>
<p class="paragraph">
The computeMatrix module of DeepTools software is used to statistically analyze the signals of gene body and upstream and downstream 3000 bp regions. The specific process is as follows. Divide the entire region into small windows (bins) with a window size of 50bp, and then calculate the average signal strength in each bin. The average signal distribution on the 3000 bp region upstream and downstream of each sample's gene body is shown in Figure 4.6:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure2_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_3_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_body_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_body_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_body_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_body_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_allSample_body_Profile" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_allSample_body_Profile"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/allSample_body.Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_3_pane_allSample_body_heatmap" 
role="tabpanel" aria-labelledby="gallery_figure2_3_list_allSample_body_heatmap"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/allSample_body.heatmap.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure2_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_St_b1"
data-toggle="list" href="#gallery_figure2_3_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_St_b2"
data-toggle="list" href="#gallery_figure2_3_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_St_b3"
data-toggle="list" href="#gallery_figure2_3_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_St_in"
data-toggle="list" href="#gallery_figure2_3_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_allSample_body_Profile"
data-toggle="list" href="#gallery_figure2_3_pane_allSample_body_Profile" role="tab" aria-controls="allSample_body_Profile">allSample_body_Profile</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_3_list_allSample_body_heatmap"
data-toggle="list" href="#gallery_figure2_3_pane_allSample_body_heatmap" role="tab" aria-controls="allSample_body_heatmap">allSample_body_heatmap</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.6 Distribution of Reads relative to gene body</b></p>
<p><small>
Horizontal axis: the position of reads relative to the gene body. Vertical axis: the density of reads
</small>
</p>
<h4>4.2.5 Distribution of Reads relative to TSS</h4>
<p class="paragraph">
The computeMatrix module of DeepTools software is used to statistically analyze the signals of TSS and upstream and downstream 3000 bp regions. The specific process is as follows. Divide the entire region into small windows (bins) with a window size of 50bp, and then calculate the average signal strength in each bin. The average signal distribution on the 3000 bp region upstream and downstream of each sample's TSS is shown in Figure 4.7:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure_readstss_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure_readstss_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure_readstss_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_TSS_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure_readstss_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_TSS_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure_readstss_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_TSS_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure_readstss_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_TSS_Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure_readstss_pane_allSample_TSS_Profile" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_allSample_TSS_Profile"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/allSample_TSS.Profile.png" /></div>

<div class="tab-pane fade" id="gallery_figure_readstss_pane_allSample_TSS_heatmap" 
role="tabpanel" aria-labelledby="gallery_figure_readstss_list_allSample_TSS_heatmap"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/allSample_TSS.heatmap.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure_readstss_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure_readstss_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure_readstss_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_St_b1"
data-toggle="list" href="#gallery_figure_readstss_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_St_b2"
data-toggle="list" href="#gallery_figure_readstss_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_St_b3"
data-toggle="list" href="#gallery_figure_readstss_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_St_in"
data-toggle="list" href="#gallery_figure_readstss_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_allSample_TSS_Profile"
data-toggle="list" href="#gallery_figure_readstss_pane_allSample_TSS_Profile" role="tab" aria-controls="allSample_TSS_Profile">allSample_TSS_Profile</a>

<a class="list-group-item list-group-item-action" id="gallery_figure_readstss_list_allSample_TSS_heatmap"
data-toggle="list" href="#gallery_figure_readstss_pane_allSample_TSS_heatmap" role="tab" aria-controls="allSample_TSS_heatmap">allSample_TSS_heatmap</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.7 Distribution of Reads relative to TSS</b></p>
<p><small>
Horizontal axis: the position of reads relative to the TSS. Vertical axis: the density of reads
</small>
</p>
</div>



<div id="page7">
<br />
<h4>4.2.6 Sample correlation detection</h4>
<p class="paragraph">

Biological replicate has two main uses: One is to prove that the biological experiment can be replicated and there is no large variance. The other one is to make sure that following differential gene analysis can get reliable results. The correlation among samples is an important index to see whether the experiment design is reliable and whether the sampling is right. The correlation coefficient is much closer to 1, the similarity of the expression pattern among samples is much higher.

</p>
<p align="center">
<img class="small"  src="src/images/pearson_correlation_of_bams.png" />
</p>
<p class="name_fig"><b>Figure 4.8  Pearson test among samples</b></p>
<p><small>
Heat-map of sample correlation test, correlation coefficient among samples (Pearson correlation coefficient). The darkness of the color represents how large is the correlation coefficient.
</small></p>

<br />
<h4>4.2.7 Visualization of pileup signal</h4>
<p class="paragraph">

We provide the visualization results of genome wide reads mapping in bam format. IGV (Integrative Genomics Viewer) browser is recommended to view the bam file (Nicol et al., 2009, Thorvaldsdóttir et al., 2013). IGV browser has the following characters: (1) can reveal single or multiple mapping positions in the genome in different scales, including the distribution of the reads in different chromosomes, and the distribution of  annotated exons, introns, splicing junctions and inter-gene region; (2) can reveal reads abundance in different region under different scales; (3) can reveal the annotation information of the gene and alternative splicing isoforms; (4) can reveal other annotation information; (5) can download annotation information from remote and local server. Please check the IGV manual for detail instruction (IGVQuickStart.pdf). The BAM file, BigWig file can be visualized in IGV browser (Kent et al., 2010).

</p>
<p align="center">
	<img height=400 width=900 src="src/images/igv_snapshot.png" />
</p>
<p class="name_fig"><b>Figure 4.9 Visualization of pileup signal by IGV(demo)</b></p>
<p class="paragraph">Result: <a href="../Result_XXXXXXXXXXXXXX-ZXX-JXXX/Mapping" title="点击打开" target="_blank">Mapping</a></p>
</div>



<!----------------------------------- frag_size预测------------------------------>
<div id="page8">
</p><br />
<h3 id="id_order_frag" ><a name="FRAG">4.3 Fragment size prediction</a></h3>
<h4>4.3.1 Summary of fragment size</h4>
<p class="paragraph">
For a specific binding site, there is a significant reads enrichment in the binding site. For double-end sequencing, we predict the insertion frag_sizes based on the data in column 9 (estimated fragment length) of the bam file. The predicted frag_sizes are used for later peak calling.
</p>
<p class="name_table" align="center">Table 4.3 Summary of Fragment size</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>Sample</th>
		<th>frag sizes length</th>
	</tr>
	
	<tr>
		<td>IP_1</td>
		<td>284</td>
	</tr>
	
	<tr>
		<td>IP_2</td>
		<td>325</td>
	</tr>
	
	<tr>
		<td>IP_3</td>
		<td>269</td>
	</tr>
	
	<tr>
		<td>Input</td>
		<td>283</td>
	</tr>
	
</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Sample: sample name</br>
(2) frag sizes length: frag_sizes length</br>
</ul>
</div>

<br /><br />
<h4>4.3.2 K distribution of fragment size</h4>
<p class="paragraph">
Length distribution is analyzed by kernel density method and shown below:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure3_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure3_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure3_1_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure3_1_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_frag_size.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_1_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure3_1_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_frag_size.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_1_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure3_1_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_frag_size.png" /></div>

<div class="tab-pane fade" id="gallery_figure3_1_pane_St_in" 
role="tabpanel" aria-labelledby="gallery_figure3_1_list_St_in"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_in_frag_size.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure3_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure3_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure3_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure3_1_list_St_b1"
data-toggle="list" href="#gallery_figure3_1_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_1_list_St_b2"
data-toggle="list" href="#gallery_figure3_1_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_1_list_St_b3"
data-toggle="list" href="#gallery_figure3_1_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure3_1_list_St_in"
data-toggle="list" href="#gallery_figure3_1_pane_St_in" role="tab" aria-controls="St_in">St_in</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.10 frag_size distribution Figure</b></p>
<p><small>Double end: horizontal axis is the length of the predicted length of frag_sizes, vertical axis is the value of kernel density.</small></p>
<br/>
</div>





<!---------------------------------------Peak Calling----------------------------------->
<div id="page11">
</p><br />
<h3 id="id_order_peak" ><a name="Peak Calling">4.4 Peak Calling</a></h3>
<p class="paragraph">
The annotation of transcription factor binding sites, histone binding sites is the important information for understanding the regulation mechanism and function. Recent developed next generation sequencing technology sequence DNA after Chromatin Immunoprecipitation directly. By mapping to the reference sequence can obtain the information of protein-DNA binding sites directly. By making use of MACS2 software (Zhang et al., 2008) (threshold q value = 0.05) to finish the peak calling, we can calculate the number of peaks, the peak width and its distribution, and find the peak related genes. The results are in the following:
</p>
<h4>4.4.1 Summary of peak calling</h4>

<p class="name_table" align="center">Table 4.4 Summary of peak calling</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>Experiment</th>
		<th>IP</th>
		<th>Input/Mock</th>
		<th>Fragment_length</th>
		<th>Count_of_narrow_peak</th>
		<th>FRiP</th>
		<th>Count_of_summits</th>
		<th>Threshold</th>
	</tr>
	
	<tr>
		<td>IP_1</td>
		<td>IP_1</td>
		<td>Input</td>
		<td>284(preset)</td>
		<td>2698</td>
		<td>27.6401%</td>
		<td>2936</td>
		<td>q:0.05</td>
	</tr>
	
	<tr>
		<td>IP_2</td>
		<td>IP_2</td>
		<td>Input</td>
		<td>325(preset)</td>
		<td>96</td>
		<td>0.2785%</td>
		<td>99</td>
		<td>q:0.05</td>
	</tr>
	
	<tr>
		<td>IP_3</td>
		<td>IP_3</td>
		<td>Input</td>
		<td>269(preset)</td>
		<td>808</td>
		<td>10.2899%</td>
		<td>908</td>
		<td>q:0.05</td>
	</tr>
	
</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Experiment: experimental group name(one ChIP Experiment includes an IP and a control, eg. Input or Mock or no control);</br>
(2) IP:experiment name after chip handling;</br>
(3) Input/Mock: control group;</br>
(4) Fragment_length: frag sizes length</br>
(5) Count_of peak: the number of peak (narrow). If would like to test broad peak, need annotate in the information collection form;</br>
(6) FRiP: the ratio of the numer of the reads in the peak to the total reads, which can test the effect of IP experiment;</br>
(7) Count_of_summits: number of summits. Some peaks can have multiple summits due to close position.</br>
(8) Threshold: The threshold used while peak calling.</br>
</ul>
</div>

<p class="name_table" align="center">Table 4.5 Result of peak calling</p>
<div style="overflow-x: scroll">
<table class="gy">
        <tr>
                <th>Chrom</th>
                <th>Start</th>
                <th>End</th>
                <th>Name</th>
                <th>Score</th>
                <th>Strand</th>
                <th>SignalValue</th>
                <th>-log10Pvalue</th>
                <th>-log10qvalue</th>
                <th>peakSummit</th>
        </tr>
        
        <tr>
                <td>chr01</td>
                <td>38418</td>
                <td>38742</td>
                <td>IP_1_peak_2</td>
                <td>79</td>
                <td>.</td>
                <td>6.17715</td>
                <td>11.1274</td>
                <td>7.98748</td>
                <td>186</td>
        </tr>
        
        <tr>
                <td>chr01</td>
                <td>38951</td>
                <td>39384</td>
                <td>IP_1_peak_3</td>
                <td>144</td>
                <td>.</td>
                <td>4.4444</td>
                <td>17.9879</td>
                <td>14.4489</td>
                <td>214</td>
        </tr>
        
        <tr>
                <td>chr01</td>
                <td>40997</td>
                <td>41298</td>
                <td>IP_1_peak_4</td>
                <td>37</td>
                <td>.</td>
                <td>2.42779</td>
                <td>6.57672</td>
                <td>3.77909</td>
                <td>253</td>
        </tr>
        
        <tr>
                <td>chr01</td>
                <td>65013</td>
                <td>65320</td>
                <td>IP_1_peak_5</td>
                <td>77</td>
                <td>.</td>
                <td>6.74086</td>
                <td>10.9255</td>
                <td>7.79821</td>
                <td>152</td>
        </tr>
        
</table>
</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Chrom: The name of the chromosome where the peak is located</br>
(2) Start: The start position of the peak</br>
(3) End: The end position of the peak</br>
(4) Name: The name of the peak. When the same peak corresponds to different summits, add a, b, c, etc. to indicate the difference</br>
(5) Score: Identifies the brightness level of the peak displayed in the browser, with values ranging from 0 to 1000 based on the fold enrichment. The higher the fold enrichment, the larger the numerical value, and the darker the display color</br>
(6) Strand: Chain information, "." indicates that chains are not distinguished.</br>
(7) SignalValue: The fold enrichment of the peak</br>
(8) -log10pvalue: The significance level of the peak</br>
(9) -log10qvalue: The significance level of the peak</br>
(10) PeakSummit: The position of the summit relative to the start position of the peak</br>
</ul>
</div>


<div id="page12">
</p><br />
<h4>4.4.2 Genome wide distribution of peaks</h4>
<p class="paragraph">
The summary of genome wide distribution of peaks is shown in the figure below. From the number of the peaks mapping to the chromosome and its distribution can reflect the distribution of the protein binding sites. When the number of the chromosomes with peaks is larger than 30, only 15 chromosomes are shown.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_1_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_1_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_on_chromosome_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_1_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_1_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_on_chromosome_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_1_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_1_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_on_chromosome_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_1_list_St_b1"
data-toggle="list" href="#gallery_figure5_1_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_1_list_St_b2"
data-toggle="list" href="#gallery_figure5_1_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_1_list_St_b3"
data-toggle="list" href="#gallery_figure5_1_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.11 Genome wide distribution of peaks</b></p>
<p><small>
Horizontal axis is the coordinate of the peaks in the chromosome. Vertical is the chromosomes. Every blue bar represents a peak. 
</small></p>

<br />
<h4>4.4.3 Distribution of peak width</h4>
<p class="paragraph">
The peak width represents the length of the DNA that is bound by the studied protein. The number of peaks of different peak width is calculated, as shown below. 
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_2_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_2_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_width_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_2_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_2_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_width_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_2_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_2_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_width_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_2_list_St_b1"
data-toggle="list" href="#gallery_figure5_2_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_2_list_St_b2"
data-toggle="list" href="#gallery_figure5_2_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_2_list_St_b3"
data-toggle="list" href="#gallery_figure5_2_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.12 Distribution of peak width</b></p>
<p><small>
Horizontal axis is the width of the peak (nt), vertical axis is the number of the corresponding peaks.
</small></p>
</div>

<div id="page13">
</p><br />
<h4>4.4.4 Distribution of fold enrichment</h4>
<p class="paragraph">
The fold enrichment value is the digital display of the peak signal during peak calling, which can be called signal value. The larger the value, the more reads are enriched to this peak. The peak number of different fold changes is calculated. The distribution of fold enrichment is shown below:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_3_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_3_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_fold_enrichment_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_3_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_3_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_fold_enrichment_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_3_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_3_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_fold_enrichment_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_St_b1"
data-toggle="list" href="#gallery_figure5_3_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_St_b2"
data-toggle="list" href="#gallery_figure5_3_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_St_b3"
data-toggle="list" href="#gallery_figure5_3_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.13 Distribution of fold enrichment</b></p>
<p><small>
Horizontal axis represents the enrichment fold change of the peak. Vertical axis represents the number of the peaks.
</small>
</p>


<br />
<h4>4.4.5 Distribution of q values</h4>
<p class="paragraph">
The significance of the peak is the indication of the credibility of the peak, so the q value for eah peak is calculated. The peak number of differnet q values is calculated and the distribution is shown below:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_4_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_4_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_qvalue_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_4_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_4_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_qvalue_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_4_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_4_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_qvalue_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_4_list_St_b1"
data-toggle="list" href="#gallery_figure5_4_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_4_list_St_b2"
data-toggle="list" href="#gallery_figure5_4_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_4_list_St_b3"
data-toggle="list" href="#gallery_figure5_4_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.14 Distribution of q values</b></p>
<p><small>Horizontal axis represents -log<sub>10</sub> q value of the peak; Vertical axis represents the number of the peaks.</small></p>
</div>

<div id="page14">
</p><br />
<h4>4.4.6 Count of summits in peaks</h4>
<p class="paragraph">
The number of summits in each peak is analysized, which infers the peak type in the IP experiment.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_5_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_5_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_5_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_5_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_summits_count_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_5_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_5_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_summits_count_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_5_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_5_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_summits_count_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_5_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_5_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_5_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_5_list_St_b1"
data-toggle="list" href="#gallery_figure5_5_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_5_list_St_b2"
data-toggle="list" href="#gallery_figure5_5_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_5_list_St_b3"
data-toggle="list" href="#gallery_figure5_5_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.15 Count of summits in peaks</b></p>
<p><small>
Horizontal axis represents the number of summit in each peak. Vertical axis is the corresponding number of the peak.
</small></p>
<b></b>

<br />
<h4>4.4.7 Summits distribution</h4>
<p class="paragraph">
Each peak is divided to 100bp windows and the summits in each window of all the peaks are counted as shown below:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure5_6_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_6_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_6_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure5_6_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_summits_in_peak_position_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_6_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure5_6_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_summits_in_peak_position_distribution.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_6_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure5_6_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_summits_in_peak_position_distribution.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure5_6_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_6_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_6_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_6_list_St_b1"
data-toggle="list" href="#gallery_figure5_6_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_6_list_St_b2"
data-toggle="list" href="#gallery_figure5_6_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_6_list_St_b3"
data-toggle="list" href="#gallery_figure5_6_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.16 Percentile position of summits in peaks</b></p>
<p><small>
Horizontal axis is the position of sumits, vertical axis is the count of the summits.
</small>
</p> 
</div>


<!---------------------------------------------------->



<!-----------------------------------------------宽峰部分结束---------------------------------->



<!---------------------------------------Motif search----------------------------------->

<div id="page17">
</p><br />
<h3 id="id_order_motif" ><a name="Motif">4.5 Motif analysis</a></h3>
<p class="paragraph">
The binding of protein such as transcription factor, histone etc. and DANN is not random, instead, has sequence preference. Motif analysis can not only detect protein specific binding sites but can also obtain the annotated motif and it’s binding site, motif sequence information etc. We used HOMER to perform motif analysis on peak sequences. There are newly recognized motifs and known motif results in the motif result file. Please refer to Figure 4.17 for details.
</p>

<h4>4.5.1 Motif detection</h4>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure6_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure6_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b1_motifs8_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b1_motifs8_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_motifs8_all.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b1_motifs10_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b1_motifs10_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_motifs10_all.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b2_motifs8_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b2_motifs8_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_motifs8_all.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b2_motifs10_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b2_motifs10_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_motifs10_all.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b3_motifs8_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b3_motifs8_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_motifs8_all.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_1_pane_St_b3_motifs10_all"
role="tabpanel" aria-labelledby="gallery_figure6_1_list_St_b3_motifs10_all"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_motifs10_all.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure6_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure6_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure6_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b1_motifs8_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b1_motifs8_all" role="tab" aria-controls="St_b1_motifs8_all">St_b1_motifs8_all</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b1_motifs10_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b1_motifs10_all" role="tab" aria-controls="St_b1_motifs10_all">St_b1_motifs10_all</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b2_motifs8_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b2_motifs8_all" role="tab" aria-controls="St_b2_motifs8_all">St_b2_motifs8_all</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b2_motifs10_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b2_motifs10_all" role="tab" aria-controls="St_b2_motifs10_all">St_b2_motifs10_all</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b3_motifs8_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b3_motifs8_all" role="tab" aria-controls="St_b3_motifs8_all">St_b3_motifs8_all</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_1_list_St_b3_motifs10_all"
data-toggle="list" href="#gallery_figure6_1_pane_St_b3_motifs10_all" role="tab" aria-controls="St_b3_motifs10_all">St_b3_motifs10_all</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.17 Motif sequence</b></p>
<p>
HOMER is a set of tools used for Motif search and second-generation data analysis. It can not only detect known motifs, but also predict potential unknown motifs. This novel algorithm for motif prediction is designed to analyze regulatory elements in Genomics applications (elements here refer to DNA, not proteins). Denovo predicts the motif by first removing redundancy from the generated motif, and then converting the motif into vector values to calculate the Pearson correlation coefficient. The motif results generated by HOMER are displayed using a web page named homerResults.html. And the directory homerResults/ stores all the files and images that the web page depends on. Here is a simple output example:
</p>
</div>

<div id="page18">
<br />
<div style="overflow-x: scroll">
<table class="gy">
<tr><th>Rank</th><th>Motif</th><th>P-value</th><th>log P-pvalue</th><th>% of Targets</th><th>% of Background</th><th>TD(Bg STD</th><th>Best Match/Details</th><th>Motif File</th></tr>
	<tr><td>1
</td><td><svg width="355" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.003,0,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.106,2,48.85)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.159,1,43.55)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.732,0,35.6)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.008,26,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.036,27,48.6)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.105,25,46.8)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.851,25,41.55)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.045,52,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.100,51,46.75)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.392,50,41.75)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.464,50,22.15)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.054,77,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.077,76,46.3)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.415,75,42.45)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.454,75,21.7)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.060,100,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.208,102,46)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.329,101,35.6)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.404,100,19.15)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.040,126,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.067,127,47)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.118,125,43.65)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.775,125,37.75)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.092,152,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.101,151,44.4)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.231,150,39.35)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.576,150,27.8)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.019,177,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.091,176,48.05)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.213,175,43.5)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.677,175,32.85)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.026,201,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.050,200,47.7)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.098,202,45.2)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.826,200,40.3)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.060,226,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.082,227,46)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.094,225,41.9)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.764,225,37.2)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.041,251,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.068,252,46.95)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.236,250,43.55)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.655,250,31.75)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.108,276,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.124,277,43.6)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.183,275,37.4)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.585,275,28.25)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.006,302,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.009,300,48.7)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.062,301,48.25)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.923,300,45.15)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.157,326,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.159,325,41.15)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.207,327,33.2)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.477,325,22.85)">G</text>
  
 </g>
</svg>
</td><td>1e-501</td><td>-1.155e+03</td><td>30.04%</td><td>3.89%</td><td>45.1bp (228.6bp)</td><td>Mad/dmmpmm(Bigfoot)/fly(0.646)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif1.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif1.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif1.motif">motif file (matrix)</A></td></tr>
<tr><td>2
</td><td><svg width="355" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.040,0,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.111,1,47)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.119,2,41.45)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.730,0,35.5)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.002,26,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.112,27,48.9)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.392,25,43.3)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.494,25,23.7)">G</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.031,50,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.076,52,47.45)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.146,51,43.65)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.747,50,36.35)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.048,77,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.162,76,46.6)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.323,75,38.5)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.467,75,22.35)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.113,101,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.167,100,43.35)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.245,102,35)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.475,100,22.75)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.007,127,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.208,126,48.65)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.253,125,38.25)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.532,125,25.6)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.019,152,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.288,150,48.05)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.335,150,33.65)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.358,151,16.9)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,177,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.078,175,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.321,176,45.05)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.600,175,29)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.053,202,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.092,200,46.35)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.162,200,41.75)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.693,201,33.65)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,225,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,226,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.016,227,48.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.982,225,48.1)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.005,251,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.013,252,48.75)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.080,250,48.1)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.902,250,44.1)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.013,277,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.031,276,48.35)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.292,275,46.8)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.664,275,32.2)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.017,301,49)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.026,302,48.15)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.177,300,46.85)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.780,300,38)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,327,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.012,326,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.182,325,48.35)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.805,325,39.25)">C</text>
  
 </g>
</svg>
</td><td>1e-467</td><td>-1.076e+03</td><td>41.86%</td><td>9.46%</td><td>78.3bp (238.5bp)</td><td>E2FA/MA1414.1/Jaspar(0.752)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif2.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif2.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif2.motif">motif file (matrix)</A></td></tr>
<tr><td>3
</td><td><svg width="355" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,0,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,1,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.005,0,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.993,2,48.65)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.013,25,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.090,25,48.35)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.393,26,43.85)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.504,27,24.2)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,50,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,51,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.013,52,48.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.985,50,48.25)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,77,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,75,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.002,75,48.9)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.996,76,48.8)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.003,100,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.013,101,48.85)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.295,100,48.2)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.689,102,33.45)">A</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.026,126,49)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.184,125,47.7)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.307,127,38.5)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.484,125,23.15)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.003,150,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.005,152,48.85)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.483,151,48.6)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.509,150,24.45)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,177,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,175,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,175,48.9)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.997,176,48.85)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.003,202,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.003,200,48.85)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.003,200,48.7)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.991,201,48.55)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,225,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,226,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.066,225,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.932,227,45.6)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,250,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.003,251,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.028,252,48.8)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.968,250,47.4)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,275,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,275,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.002,276,48.9)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.996,277,48.8)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.003,300,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.006,301,48.85)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.009,302,48.55)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.982,300,48.1)">G</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,325,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.003,327,48.95)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.003,325,48.8)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.993,326,48.65)">T</text>
  
 </g>
</svg>
</td><td>1e-460</td><td>-1.059e+03</td><td>20.33%</td><td>1.51%</td><td>163.2bp (236.8bp)</td><td>PH0112.1_Nkx2-3/Jaspar(0.599)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif3.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif3.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif3.motif">motif file (matrix)</A></td></tr>
<tr><td>4
</td><td><svg width="305" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.013,0,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.042,1,48.35)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.205,2,46.25)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.740,0,36)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.002,25,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.066,27,48.9)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.127,26,45.6)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.805,25,39.25)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.030,50,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.109,51,47.5)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.296,52,42.05)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.565,50,27.25)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.002,77,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.007,75,48.9)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.075,75,48.55)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.916,76,44.8)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.002,100,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.013,102,48.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.013,100,48.25)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.972,101,47.6)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.007,125,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.136,125,48.65)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.214,127,41.85)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.643,126,31.15)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,150,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.089,150,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.117,151,44.5)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.793,152,38.65)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.007,175,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.024,176,48.65)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.277,177,47.45)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.692,175,33.6)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.002,200,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.104,201,48.9)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.134,202,43.7)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.760,200,37)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.006,225,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.077,227,48.7)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.200,226,44.85)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.717,225,34.85)">G</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.027,250,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.095,250,47.65)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.131,252,42.9)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.747,251,36.35)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,275,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.005,275,48.95)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.038,277,48.7)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.956,276,46.8)">T</text>
  
 </g>
</svg>
</td><td>1e-384</td><td>-8.852e+02</td><td>26.36%</td><td>4.03%</td><td>150.8bp (286.1bp)</td><td>AT4G12670(MYBrelated)/col-AT4G12670-DAP-Seq(GSE60143)/Homer(0.876)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif4.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif4.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif4.motif">motif file (matrix)</A></td></tr>
<tr><td>5
</td><td><svg width="355" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,0,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.082,1,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.397,2,44.85)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.520,0,25)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,26,49)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.077,25,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.106,25,45.1)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.816,27,39.8)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,50,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.224,52,48.95)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.358,51,37.75)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.417,50,19.85)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,77,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,75,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.025,76,48.9)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.973,75,47.65)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,100,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.003,100,48.95)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.211,102,48.8)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.785,101,38.25)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,125,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.047,125,48.95)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.416,126,46.6)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.536,127,25.8)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,150,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.030,151,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.244,152,47.45)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.725,150,35.25)">G</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,175,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,176,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.126,175,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.872,177,42.6)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,200,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,201,48.95)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.146,200,48.9)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.852,202,41.6)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.040,225,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.111,225,47)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.393,227,41.45)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.456,226,21.8)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,250,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.139,252,48.95)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.333,250,42)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.527,251,25.35)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,277,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,275,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.003,276,48.9)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.995,275,48.75)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,300,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,301,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.003,300,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.995,302,48.75)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,325,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.045,325,48.95)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.080,327,46.7)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.874,326,42.7)">T</text>
  
 </g>
</svg>
</td><td>1e-376</td><td>-8.678e+02</td><td>17.88%</td><td>1.50%</td><td>188.7bp (213.7bp)</td><td>PH0037.1_Hdx/Jaspar(0.631)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif5.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif5.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif5.motif">motif file (matrix)</A></td></tr>
<tr><td>6
</td><td><svg width="305" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,0,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.016,1,48.95)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.073,0,48.15)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.910,2,44.5)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,25,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.012,26,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.458,25,48.35)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.529,27,25.45)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.004,50,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.080,52,48.8)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.423,51,44.8)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.493,50,23.65)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.016,76,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.186,75,48.2)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.241,77,38.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.557,75,26.85)">G</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.013,100,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.067,102,48.35)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.434,101,45)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.486,100,23.3)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.047,125,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.099,126,46.65)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.103,127,41.7)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.751,125,36.55)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,152,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.024,150,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.150,151,47.75)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.825,150,40.25)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,177,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,175,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.032,176,48.9)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.966,175,47.3)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.040,200,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.093,201,47)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.152,202,42.35)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.715,200,34.75)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.025,225,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.061,226,47.75)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.221,227,44.7)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.693,225,33.65)">C</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.004,251,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.014,250,48.8)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.243,250,48.1)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.739,252,35.95)">A</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.007,276,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.029,275,48.65)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.397,275,47.2)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.567,277,27.35)">A</text>
  
 </g>
</svg>
</td><td>1e-358</td><td>-8.256e+02</td><td>30.65%</td><td>6.22%</td><td>102.9bp (228.1bp)</td><td>POL006.1_BREu/Jaspar(0.670)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif6.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif6.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif6.motif">motif file (matrix)</A></td></tr>
<tr><td>7
</td><td><svg width="255" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,2,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.016,0,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.248,1,48.15)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.735,0,35.75)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,25,49)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.039,27,48.95)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.115,26,47)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.845,25,41.25)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.020,50,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.027,50,48)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.443,52,46.65)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.510,51,24.5)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.007,75,49)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.043,75,48.65)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.192,76,46.5)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.758,77,36.9)">A</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,100,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,100,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,101,48.9)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.997,102,48.85)">A</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,126,49)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.057,125,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.079,125,46.1)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.863,127,42.15)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,150,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,151,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.490,152,48.9)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.508,150,24.4)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,175,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.246,177,48.95)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.305,175,36.65)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.448,176,21.4)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,202,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,200,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,200,48.9)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.997,201,48.85)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,225,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,226,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.072,227,48.9)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.926,225,45.3)">C</text>
  
 </g>
</svg>
</td><td>1e-327</td><td>-7.544e+02</td><td>38.39%</td><td>10.91%</td><td>144.0bp (223.0bp)</td><td>ORB2(RRM)/Drosophila_melanogaster-RNCMPT00126-PBM/HughesRNA(0.661)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif7.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif7.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif7.motif">motif file (matrix)</A></td></tr>
<tr><td>8
</td><td><svg width="305" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,2,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,0,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.082,0,48.9)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.916,1,44.8)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,25,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.083,27,48.95)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.084,26,44.8)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.832,25,40.6)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,52,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,50,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.032,50,48.9)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.966,51,47.3)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,75,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,76,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.023,75,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.975,77,47.75)">A</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,100,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,101,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.193,102,48.9)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.805,100,39.25)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,125,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.249,126,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.371,127,36.5)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.379,125,17.95)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,152,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,150,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.140,150,48.9)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.858,151,41.9)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,175,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,176,48.95)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.331,175,48.9)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.667,177,32.35)">A</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,202,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.137,200,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.248,200,42.1)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.614,201,29.7)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,225,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,226,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.254,227,48.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.744,225,36.2)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,252,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,250,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,250,48.9)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.997,251,48.85)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,277,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,275,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.062,275,48.9)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.936,276,45.8)">T</text>
  
 </g>
</svg>
</td><td>1e-299</td><td>-6.901e+02</td><td>16.14%</td><td>1.66%</td><td>156.7bp (214.1bp)</td><td>SPL9(SBP)/colamp-SPL9-DAP-Seq(GSE60143)/Homer(0.694)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif8.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif8.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif8.motif">motif file (matrix)</A></td></tr>
<tr><td>9
</td><td><svg width="255" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,0,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.053,2,48.95)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.084,1,46.3)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.862,0,42.1)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,27,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,25,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,26,48.9)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.997,25,48.85)">C</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.004,50,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.050,51,48.8)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.098,52,46.3)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.848,50,41.4)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,75,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.266,75,48.95)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.285,77,35.65)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.448,76,21.4)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,100,49)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,101,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.201,102,48.9)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.797,100,38.85)">G</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,127,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.052,125,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.375,125,46.35)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.572,126,27.6)">T</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.181,150,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.231,150,39.95)">G</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.280,152,28.4)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.308,151,14.4)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,175,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,176,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.234,177,48.9)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.764,175,37.2)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,200,49)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.262,202,48.95)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.302,201,35.85)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.435,200,20.75)">G</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,225,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.050,226,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.095,227,46.45)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.854,225,41.7)">C</text>
  
 </g>
</svg>
</td><td>1e-296</td><td>-6.830e+02</td><td>27.72%</td><td>6.08%</td><td>119.9bp (233.5bp)</td><td>Vts1p(SAM)/Saccharomyces_cerevisiae-RNCMPT00111-PBM/HughesRNA(0.665)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif9.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif9.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif9.motif">motif file (matrix)</A></td></tr>
<tr><td>10
</td><td><svg width="205" height="50">
 <g font-family="Arial" font-weight="bold" font-size="66.5">
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.046,2,49)">A</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.099,1,46.7)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.395,0,41.75)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.460,0,22)">C</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,27,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.040,25,48.95)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.070,25,46.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.889,26,43.45)">T</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.021,52,49)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.128,50,47.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.274,50,41.55)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.577,51,27.85)">T</text>
  
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,75,49)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.225,76,48.95)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.234,77,37.7)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.540,75,26)">C</text>
  
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,100,49)">C</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.001,100,48.95)">G</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.041,101,48.9)">T</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.957,102,46.85)">A</text>
  
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.001,127,49)">A</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.001,125,48.95)">C</text>
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.233,126,48.9)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.765,125,37.25)">G</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,151,49)">T</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.045,150,48.95)">G</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.050,150,46.7)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.904,152,44.2)">A</text>
  
  <text fill="#DD0000" x="0" y="0"  transform="matrix(0.65,0,0,0.001,176,49)">T</text>
  <text fill="#0000EE" x="0" y="0"  transform="matrix(0.59,0,0,0.028,175,48.95)">C</text>
  <text fill="#00BB00" x="0" y="0"  transform="matrix(0.53,0,0,0.064,177,47.55)">A</text>
  <text fill="#F9A500" x="0" y="0"  transform="matrix(0.55,0,0,0.907,175,44.35)">G</text>
  
 </g>
</svg>
</td><td>1e-282</td><td>-6.516e+02</td><td>38.25%</td><td>12.17%</td><td>128.7bp (182.9bp)</td><td>Nkx2.2(Homeobox)/NPC-Nkx2.2-ChIP-Seq(GSE61673)/Homer(0.715)<BR/><A target="_blank" HREF="src/St_b1/homerResults/motif10.info.html">More Information</A> | <A target="blank" HREF="src/St_b1/homerResults/motif10.similar.html">Similar Motifs Found</A></td><td><A target="_blank" HREF="src/St_b1/homerResults/motif10.motif">motif file (matrix)</A></td></tr>

</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Rank: Serial number<br/>
(2) Motif: Motif sequence visualization<br/>
(3) P-value: Confidence calculation result<br/>
(4) Log P-pvalue: Log P-pvalue value<br/>
(5) % of Targets: The percentage of target sequences in the total denovo sequences<br/>
(6) % of Background: The percentage of background sequences in the total denovo sequences<br/>
(7) STD (Bg STD): The sequence set of the target and background has a standard deviation of 200 bp from the sequence center<br/>
(8) Best Match/Details: The result of the best match<br/>
(9) Motif File: Motif File Result<br/>
</ul>
</div>
<br/>


<!---------------------------------------peak annotion ----------------------------------->

<div id="page20">
</p><br />
<h3 id="id_order_anno" ><a name="Peak annotion">4.6 Peak annotation</a></h3>
<h4>4.6.1 Distribution of peaks in gene functional regions</h4>
<p class="paragraph">
Use ChIPseeker software to calculate the distribution of peaks in various functional areas. The correspondence between peaks and various functional areas is based on the priority order of promoter, 5'UTR, 3'UTR, Exon, Intron, Downstream, and Intergenic. In other words, if the location of a peak is both the Exon of one gene and the Intron of another gene, according to the above priority, it will be considered that the peak is annotated to the Exon rather than the Intron. The annotation results of ChIPseeker are detailed in Table 4.6 and Figure 4.18:
</p>
<p class="name_table" align="center">Table 4.6 List of distribution of peaks in gene functional regions</p>
<div style="overflow-x: scroll">
<table class="gy">
<tr><th>Chrom</th><th>Start</th><th>End</th><th>Width</th><th>Name</th><th>Score</th><th>signalValue</th><th>-log10pvalue</th><th>-log10qvalue</th><th>summit.position</th><th>annotation</th><th>geneStart</th><th>geneEnd</th><th>geneLength</th><th>geneStrand</th><th>geneId</th><th>transcriptId</th><th>distanceToTSS</th><th>geneName</th><tr>

        <tr>
        <td>chr01</td><td>11361</td><td>11644</td><td>284</td><td>IP_1_peak_1</td><td>53</td><td>4.40675</td><td>8.26414</td><td>5.32684</td><td>184</td><td>Distal Intergenic</td><td>432461</td><td>449161</td><td>16701</td><td>-</td><td>Soltu.DM.01G000010</td><td>Soltu.DM.01G000010.1</td><td>437517</td><td>Soltu.DM.01G000010</td>
        </tr>

        <tr>
        <td>chr01</td><td>38419</td><td>38742</td><td>324</td><td>IP_1_peak_2</td><td>79</td><td>6.17715</td><td>11.1274</td><td>7.98748</td><td>186</td><td>Distal Intergenic</td><td>432461</td><td>449161</td><td>16701</td><td>-</td><td>Soltu.DM.01G000010</td><td>Soltu.DM.01G000010.1</td><td>410419</td><td>Soltu.DM.01G000010</td>
        </tr>

        <tr>
        <td>chr01</td><td>38952</td><td>39384</td><td>433</td><td>IP_1_peak_3</td><td>144</td><td>4.4444</td><td>17.9879</td><td>14.4489</td><td>214</td><td>Distal Intergenic</td><td>432461</td><td>449161</td><td>16701</td><td>-</td><td>Soltu.DM.01G000010</td><td>Soltu.DM.01G000010.1</td><td>409777</td><td>Soltu.DM.01G000010</td>
        </tr>

        <tr>
        <td>chr01</td><td>40998</td><td>41298</td><td>301</td><td>IP_1_peak_4</td><td>37</td><td>2.42779</td><td>6.57672</td><td>3.77909</td><td>253</td><td>Distal Intergenic</td><td>432461</td><td>449161</td><td>16701</td><td>-</td><td>Soltu.DM.01G000010</td><td>Soltu.DM.01G000010.1</td><td>407863</td><td>Soltu.DM.01G000010</td>
        </tr>

</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Chrom: The name of a chromosome<br/>
(2) Start: The start position of the peak<br/>
(3) End: The end position of the peak<br/>
(4) Width: Peak width<br/>
(5) Name: The name of the peak. When the same peak corresponds to different peaks, add a, b, c, etc. to indicate the difference<br/>
(6) Score: Indicates the darkness displayed on the peak, with a value ranging from 0 to 1000 based on the enrichment factor. The higher the enrichment factor, the larger the numerical value, and the darker the display color<br/>
(7) SignalValue: The FoldEnrich of the peak<br/>
(8) -log10pvalue: The significance level of the peak<br/>
(9) -log10qvalue: The significance level of the peak<br/>
(10) Summit.position: The position of the peak summit<br/>
(11) Annotation: The functional area where the peak is located<br/>
(12) GeneStart: The start position of the gene closest to the TSS<br/>
(13) GeneEnd: The end position of the gene closest to the TSS<br/>
</ul>
</div>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure6_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_1_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure7_1_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_narrow_peaks_plotAnnoPie.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_1_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure7_1_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_narrow_peaks_plotAnnoPie.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_1_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure7_1_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_narrow_peaks_plotAnnoPie.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_St_b1"
data-toggle="list" href="#gallery_figure7_1_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_St_b2"
data-toggle="list" href="#gallery_figure7_1_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_St_b3"
data-toggle="list" href="#gallery_figure7_1_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.18 Distribution of peaks in gene functional regions</b></p>
</p>


<br />
<h4>4.6.2 The distribution of relative gene positions at the peak summit</h4>
<p class="paragraph">The distribution of peak summits relative to TSS can predict the binding sites of proteins on genes. On the one hand, the effect of IP can be determined based on the binding sites of proteins; On the other hand, protein binding characteristics can be used to predict the regulatory mode or function of proteins. Use ChIPSeeker software to obtain the relevant genes for each peak, and calculate the number of peaks based on their distance from TSS. Analyze the distance distribution between peak and TSS. Please refer to Figure 4.19 for details</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure7_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_Distance_peaks_and_TSS.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b1enlarged" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b1enlarged"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_Distance_peaks_and_TSS_enlarged.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_Distance_peaks_and_TSS.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b2enlarged" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b2enlarged"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_Distance_peaks_and_TSS_enlarged.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_Distance_peaks_and_TSS.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_2_pane_St_b3enlarged" 
role="tabpanel" aria-labelledby="gallery_figure7_2_list_St_b3enlarged"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_Distance_peaks_and_TSS_enlarged.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b1"
data-toggle="list" href="#gallery_figure7_2_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b1enlarged"
data-toggle="list" href="#gallery_figure7_2_pane_St_b1enlarged" role="tab" aria-controls="St_b1enlarged">St_b1enlarged</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b2"
data-toggle="list" href="#gallery_figure7_2_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b2enlarged"
data-toggle="list" href="#gallery_figure7_2_pane_St_b2enlarged" role="tab" aria-controls="St_b2enlarged">St_b2enlarged</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b3"
data-toggle="list" href="#gallery_figure7_2_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_2_list_St_b3enlarged"
data-toggle="list" href="#gallery_figure7_2_pane_St_b3enlarged" role="tab" aria-controls="St_b3enlarged">St_b3enlarged</a>

</div></div></div>
</p>
<p align="center"><b>Figure 4.19 Distribution of Peak Summit Relative TSS</b></p>
<p><small>Horizontal axis: distance from peak to TSS; Vertical axis: The number of peaks.</small></p>

</div>


<div id="page21">
<br />
<h4>4.6.3 GO enrichment analysis</h4>
<p class="paragraph">
Gene Ontology (GO, http://www.geneontology.org/) is international standard classification system for gene function attributes(Ashburner et al., 2000). As the database built by Gene Ontology Consortium, it aims to build up an updating language vocabulary standard to describe gene and protein function for all species(Young et al., 2010). GO covers three domains: Molecular Function, Biological Process and Cellular Component.
</p>
<p class="paragraph">
Gene or protein can find the corresponding GO accession number by ID access or sequence annotation. And every GO number can find its corresponding Term, which is the function classification or cellular localization.
</p>
<p class="paragraph">
Any gene whose position is overlapped with peak is peak related gene. Results of GO enrichment is in the following:
</p>
<p class="name_table" align="center">Table 4.7 peak related gene GO enrichment</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>GO</th>
		<th>Description</th>
		<th>Term_type</th>
		<th>Overrepresented_pValue</th>
		<th>Corrected_pValue</th>
		<th>Gene_item</th>
		<th>Gene_list</th>
		<th>Bg_item</th>
		<th>Bg_list</th>
		<th>genes</th>
	</tr>
	
	<tr>
		<td>GO:0009767</td>
		<td>photosynthetic electron transport chain</td>
		<td>biological_process</td>
		<td>0.00012587</td>
		<td>0.29692</td>
		<td>6</td>
		<td>769</td>
		<td>13</td>
		<td>16988</td>
		<td></td>
	</tr>
	
	<tr>
		<td>GO:0015979</td>
		<td>photosynthesis</td>
		<td>biological_process</td>
		<td>0.00024344</td>
		<td>0.29692</td>
		<td>13</td>
		<td>769</td>
		<td>68</td>
		<td>16988</td>
		<td></td>
	</tr>
	
	<tr>
		<td>GO:0022900</td>
		<td>electron transport chain</td>
		<td>biological_process</td>
		<td>0.00039415</td>
		<td>0.29692</td>
		<td>7</td>
		<td>769</td>
		<td>21</td>
		<td>16988</td>
		<td></td>
	</tr>
	
	<tr>
		<td>GO:0019684</td>
		<td>photosynthesis, light reaction</td>
		<td>biological_process</td>
		<td>0.00055454</td>
		<td>0.29692</td>
		<td>6</td>
		<td>769</td>
		<td>18</td>
		<td>16988</td>
		<td></td>
	</tr>
	
</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) GO: the unique GO ID;</br>
(2) Description: Function description;</br>
(3) Term_type: Including cellular_component,  biological_process and molecular_function;</br>
(4) Over_represented_pValue: significance of enrichment analysis;</br>
(5) Corrected_pValue: P-Value after correction, normally, with padj< 0.05 are enriched;</br>
(6) Gene_item: the number of peak related genes with this GO term;</br>
(7) Gene_list: the number of peak related gene;</br>
(8) Bg_item: the number of background gene with this GO term;</br>
(9) Bg_list: the number of background genes;</br>
(10) Genes: Annotated peak related gene ID (not show in this table, please refer to the results file for details);</br>
(11) Choose the first five results for each experiment.</br>
</ul>
</div>
</div>

<div id="page22">
</p>
<h4>4.6.4 GO enrichment</h4>
<p class="paragraph"> 
Peak overlapping gene GO enrichment bar, which directly reflect the distribution of the number of the peak overlapping genes enrichment on the biological process, cellular component and molecular function.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure7_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_GO.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b1_mf" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b1_mf"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1.Enriched_GO_mf_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b1_cc" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b1_cc"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1.Enriched_GO_cc_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b1_bp" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b1_bp"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1.Enriched_GO_bp_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_GO.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b2_mf" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b2_mf"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2.Enriched_GO_mf_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b2_cc" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b2_cc"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2.Enriched_GO_cc_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b2_bp" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b2_bp"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2.Enriched_GO_bp_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_GO.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b3_mf" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b3_mf"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3.Enriched_GO_mf_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b3_cc" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b3_cc"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3.Enriched_GO_cc_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_3_pane_St_b3_bp" 
role="tabpanel" aria-labelledby="gallery_figure7_3_list_St_b3_bp"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3.Enriched_GO_bp_DAG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b1"
data-toggle="list" href="#gallery_figure7_3_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b1_mf"
data-toggle="list" href="#gallery_figure7_3_pane_St_b1_mf" role="tab" aria-controls="St_b1_mf">St_b1_mf</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b1_cc"
data-toggle="list" href="#gallery_figure7_3_pane_St_b1_cc" role="tab" aria-controls="St_b1_cc">St_b1_cc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b1_bp"
data-toggle="list" href="#gallery_figure7_3_pane_St_b1_bp" role="tab" aria-controls="St_b1_bp">St_b1_bp</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b2"
data-toggle="list" href="#gallery_figure7_3_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b2_mf"
data-toggle="list" href="#gallery_figure7_3_pane_St_b2_mf" role="tab" aria-controls="St_b2_mf">St_b2_mf</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b2_cc"
data-toggle="list" href="#gallery_figure7_3_pane_St_b2_cc" role="tab" aria-controls="St_b2_cc">St_b2_cc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b2_bp"
data-toggle="list" href="#gallery_figure7_3_pane_St_b2_bp" role="tab" aria-controls="St_b2_bp">St_b2_bp</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b3"
data-toggle="list" href="#gallery_figure7_3_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b3_mf"
data-toggle="list" href="#gallery_figure7_3_pane_St_b3_mf" role="tab" aria-controls="St_b3_mf">St_b3_mf</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b3_cc"
data-toggle="list" href="#gallery_figure7_3_pane_St_b3_cc" role="tab" aria-controls="St_b3_cc">St_b3_cc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_3_list_St_b3_bp"
data-toggle="list" href="#gallery_figure7_3_pane_St_b3_bp" role="tab" aria-controls="St_b3_bp">St_b3_bp</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.20 GO enrichment bar chart and DAG figure</b></p>
<p><small>Bar: horizontal axis is the number of the peak overlapping gene. Different color is used to differentiate biological process, cellular component and molecular function.</small></p>
<p><small>DAG figure: every note represents a GO project. Rectangular represents top 10 enrichment GO. Darkness represents the enrichment level. The darker of the color, the higher of the enrichment.</small></p>
<p><small>The name of the project and the significance (padj) are shown in every note.</small></p>
</div>

<p class="center">
<div class="row">
        <div class="col-1">
                <div id="gallery_figure_enrichgodot_button_prev" class="button_prev" onclick="prev(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_prev">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-6">
                <div class="tab-content" id="gallery_figure_enrichgodot_nav-tabContent">
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichgodot_pane_St_b1" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichgodot_list_St_b1"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_GO_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichgodot_pane_St_b2" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichgodot_list_St_b2"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_GO_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichgodot_pane_St_b3" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichgodot_list_St_b3"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_GO_dot.png" />
                        </div>
                        
                </div>
        </div>
        <div class="col-1">
                <div id="gallery_figure_enrichgodot_button_next" class="button_next" onclick="next(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_next">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-4">
                <input class="form-control" id="gallery_figure_enrichgodot_InputList" type="text" placeholder="Search..">
                <div class="list-group" id="gallery_figure_enrichgodot_list-tab" role="tablist">
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichgodot_list_St_b1"
                        data-toggle="list" href="#gallery_figure_enrichgodot_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichgodot_list_St_b2"
                        data-toggle="list" href="#gallery_figure_enrichgodot_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichgodot_list_St_b3"
                        data-toggle="list" href="#gallery_figure_enrichgodot_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>
                        
                </div>
        </div>
</div>		
</p>
<p align="center"><b> Figure 4.21 GO enrichment scatter plot</b></p>
<p><small>The horizontal axis in the figure represents the ratio of the number of differentially expressed genes annotated on GO Term to the total number of differentially expressed genes, while the vertical axis represents GO Term</small></p>

<div id="page23">
</p>
<h4>4.6.5 KEGG enrichment analysis</h4>
<p class="paragraph">
Different genes coordinate with each other to generate the function inside the organism. Pathway enrichment can help identify the main biochemical metabolic pathway and signalling pathway of differentially expressed genes. KEGG (Kyoto Encyclopedia of Genes and Genomes) is a database that can analyze gene function and genome information. It helps researchers study the genes and their expression as a whole network. As the main public database related to pathways (Kanehisa,2008), KEGG provides an excellent pathway search, including the metabolism of carbohydrates, nucleotides, amino acids, and biodegradation. These pathways not only provide the possible metabolism pathway but give a comprehensive annotation of the enzyme in each catalytic reaction, including amino acid sequence, PDB link, etc. It is a powerful tool to study organism metabolism and do network analysis. Pathway enrichment makes use of pathways in the KEGG database as the unit, with hypergeometric tests to find the significantly enriched pathway in the differentially expressed genes with the whole genome as the background.
</p>
<p class="name_table" align="center">Table 4.8 KEGG enrichment list</p>
<div style="overflow-x: scroll">
<table class="gy">
	<tr>
		<th>Term</th>
		<th>Database</th>
		<th>ID</th>
		<th>Input number</th>
		<th>Background number</th>
		<th>Pvalue</th>
		<th>Corrected pvalue</th>
	</tr>
	
	<tr>
		<td>Phenylpropanoid biosynthesis</td>
		<td>KEGG PATHWAY</td>
		<td>sot00940</td>
		<td>22</td>
		<td>250</td>
		<td>0.0053568781463</td>
		<td>0.0607112856581</td>
	</tr>
	
	<tr>
		<td>Propanoate metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00640</td>
		<td>8</td>
		<td>56</td>
		<td>0.00732390514786</td>
		<td>0.0747038325082</td>
	</tr>
	
	<tr>
		<td>Sesquiterpenoid and triterpenoid biosynthesis</td>
		<td>KEGG PATHWAY</td>
		<td>sot00909</td>
		<td>5</td>
		<td>26</td>
		<td>0.0112407468652</td>
		<td>0.104232380023</td>
	</tr>
	
	<tr>
		<td>Arginine and proline metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00330</td>
		<td>8</td>
		<td>68</td>
		<td>0.0196246088039</td>
		<td>0.156245731516</td>
	</tr>
	
	<tr>
		<td>Tyrosine metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00350</td>
		<td>7</td>
		<td>55</td>
		<td>0.0199136716638</td>
		<td>0.156245731516</td>
	</tr>
	
	<tr>
		<td>beta-Alanine metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00410</td>
		<td>8</td>
		<td>71</td>
		<td>0.024197485158</td>
		<td>0.176295963294</td>
	</tr>
	
	<tr>
		<td>Starch and sucrose metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00500</td>
		<td>13</td>
		<td>152</td>
		<td>0.0334390978523</td>
		<td>0.213798423058</td>
	</tr>
	
	<tr>
		<td>Fatty acid degradation</td>
		<td>KEGG PATHWAY</td>
		<td>sot00071</td>
		<td>7</td>
		<td>62</td>
		<td>0.0335370075386</td>
		<td>0.213798423058</td>
	</tr>
	
	<tr>
		<td>Purine metabolism</td>
		<td>KEGG PATHWAY</td>
		<td>sot00230</td>
		<td>9</td>
		<td>101</td>
		<td>0.0569579016865</td>
		<td>0.320918500261</td>
	</tr>
	
</table>
</div>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
(1) Term: Description information of KEGG pathway.</br>
(2) Database: KEGG PATHWAY.</br>
(3) ID: The only pathway identifier in the KEGG database.</br>
(4) Input number: the number of peak overlapping gene in the corresponding pathway.</br>
(5) Background number: The number of the genes in the corresponding pathway.</br>
(6) Pvalue: statistical significant level of enrichment analysis.</br>
(7) Corrected pvalue: statistical significant level after correction. Normally, if the corrected P-value < 0.05 this pathway is enriched.</br>
</ul>
</div>

<br /><br />
<h4>4.6.6 Plot of KEGG enrichment</h4>
<p class="paragraph">
From the KEGG enrichment results, select the 20 most significant KEGG pathways and draw a bar chart for display(Figure 4.22). If there are less than 20 pathways, draw all pathways as shown in the following figure. The horizontal axis in the figure represents the KEGG pathway, and the vertical axis represents the significance level of pathway enrichment. The higher the value, the more significant it is. The value on the column represents the number of differentially expressed genes enriched in this pathway. All analysis results can be found in the result folder.
</p>
<p class="center">
<div class="row">
        <div class="col-1">
                <div id="gallery_figure_enrichkeggbar_button_prev" class="button_prev" onclick="prev(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_prev">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-6">
                <div class="tab-content" id="gallery_figure_enrichkeggbar_nav-tabContent">
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichkeggbar_pane_St_b1" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichkeggbar_list_St_b1"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_KEGG_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichkeggbar_pane_St_b2" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichkeggbar_list_St_b2"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_KEGG_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_enrichkeggbar_pane_St_b3" 
                                role="tabpanel" aria-labelledby="gallery_figure_enrichkeggbar_list_St_b3"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_KEGG_bar.png" />
                        </div>
                        
                </div>
        </div>
        <div class="col-1">
                <div id="gallery_figure_enrichkeggbar_button_next" class="button_next" onclick="next(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_next">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-4">
                <input class="form-control" id="gallery_figure_enrichkeggbar_InputList" type="text" placeholder="Search..">
                <div class="list-group" id="gallery_figure_enrichkeggbar_list-tab" role="tablist">
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichkeggbar_list_St_b1"
                        data-toggle="list" href="#gallery_figure_enrichkeggbar_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichkeggbar_list_St_b2"
                        data-toggle="list" href="#gallery_figure_enrichkeggbar_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_enrichkeggbar_list_St_b3"
                        data-toggle="list" href="#gallery_figure_enrichkeggbar_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>
                        
                </div>
        </div>
</div>		
</p>
<p align="center"><b> Figure 4.22 KEGG enrichment bar chart</b></p>
<p><small>
The horizontal axis in the figure represents the KEGG pathway, and the vertical axis represents the significance level of pathway enrichment
</small>
</p>

<p class="paragraph">
Peak overlapping gene KEGG enrichment scatter (Figure 4.23) is the visualization of KEGG enrichment analysis. In this figure, the extent of KEGG enrichment is measured by Rich factor, q value and the number of the genes that are enriched in this pathway. Rich factor is the ratio of the number of the genes in the corresponding pathway of the peak overlapping gene to the total number of all annotated genes in this pathway. Q value is the p value after multiple hypothesis testing correction. The range of q value is from 0 to 1, with more close to 0, representing more significant for the enrichment. Here only shows the top 20 significant enriched pathways. If the enriched pathways are less than 20, then all of them are shown.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure7_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_4_pane_St_b1" 
role="tabpanel" aria-labelledby="gallery_figure7_4_list_St_b1"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_peak_KEGG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_4_pane_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure7_4_list_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_peak_KEGG.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_4_pane_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure7_4_list_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b3_peak_KEGG.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure7_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_4_list_St_b1"
data-toggle="list" href="#gallery_figure7_4_pane_St_b1" role="tab" aria-controls="St_b1">St_b1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_4_list_St_b2"
data-toggle="list" href="#gallery_figure7_4_pane_St_b2" role="tab" aria-controls="St_b2">St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_4_list_St_b3"
data-toggle="list" href="#gallery_figure7_4_pane_St_b3" role="tab" aria-controls="St_b3">St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.23 KEGG enrichment scatter</b></p>
<p><small>
Vertical axis represents pathway name, horizontal axis represents Rich factor, the size of the dot represents the numbers of the overlapping peak genes in the pathway, and the color of the dots corresponds to different qvalue range. The size of the dots represents the number of the genes whose peaks are overlapping in this pathway.
</small>
</p>
</div>




<!---------------------------------------peak compare----------------------------------->
<div id="page24">
</p><br />
<h3 id="id_order_compare" ><a name="Diff Compare">4.7 Differential analysis</a></h3>
<p class="paragraph">Only when the number of groups is not less than two can the difference analysis be performed between groups.</p>
<p class="paragraph">
For data with biological duplication, DiffBind software is used for differential peak analysis. For data without biological duplication, use edgeR software for differential peak analysis. Use p-value or corrected p-value (padj) to determine the significance level. Padj is the value obtained by using BH's method to perform multiple tests on p-value, and the smaller the corrected P-value, the more significant it is. By default, padj<0.05 and | log2FoldChange |>1 are used as the difference significance criteria.
</p>
<h4>4.7.1 Venn diagram of the peak</h4>
<p class="paragraph">
Draw a Venn diagram between the peaks of the treatment group and the control group. If the overlap between two peak regions has at least 1 bp, it is considered that the two peaks intersect. For data with biological duplication, the peak areas of biological duplication experiments within the group are first merged (the merging rule is: if two peaks have at least 1 bp overlap, these two peak areas are merged), and then draw the venn diagram. The venn diagram is detailed in 4.22:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_1_1_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_1_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_1_1_pane_St_b1_vs_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure8_1_1_list_St_b1_vs_St_b2"
onclick="trans(this.id)"><img class="cover" src="src/images/St_b1_vs_St_b2_venn.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_1_pane_St_b1_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_1_1_list_St_b1_vs_St_b3"
onclick="trans(this.id)"><img class="cover" src="src/images/St_b1_vs_St_b3_venn.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_1_pane_St_b2_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_1_1_list_St_b2_vs_St_b3"
onclick="trans(this.id)"><img class="cover" src="src/images/St_b2_vs_St_b3_venn.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_1_1_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_1_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_1_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_1_list_St_b1_vs_St_b2"
data-toggle="list" href="#gallery_figure8_1_1_pane_St_b1_vs_St_b2" role="tab" aria-controls="St_b1_vs_St_b2">St_b1_vs_St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_1_list_St_b1_vs_St_b3"
data-toggle="list" href="#gallery_figure8_1_1_pane_St_b1_vs_St_b3" role="tab" aria-controls="St_b1_vs_St_b3">St_b1_vs_St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_1_list_St_b2_vs_St_b3"
data-toggle="list" href="#gallery_figure8_1_1_pane_St_b2_vs_St_b3" role="tab" aria-controls="St_b2_vs_St_b3">St_b2_vs_St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.24 Venn diagram of the peak</b></p>
</div>




<div id="page25">
</p><br />
<h4>4.7.2 Enrichment level analysis among different samples</h4>
<p class="paragraph">
RPM value of the peak in different samples (the ratio of 1M reads that enriched to the peak in a single sample) is used to do clustering analysis to determine the enrichment pattern of the same peak in different samples or the enrichment difference of different peaks in the same sample. At the same time, Enrichment comparison between IP and Input in the group can show the peak enrichment in IP experiment. Hierarchical clustering is done based on RPM value in different samples. Different colors represent different clustering information. The more close between the IP sample enrichment, the more similar function or biological process they have.
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_1_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_1_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_1_2_pane_St_b1_vs_St_b2"
role="tabpanel" aria-labelledby="gallery_figure8_1_2_list_St_b1_vs_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_heatmap_of_RPM_in_peak.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_2_pane_St_b1_vs_St_b3"
role="tabpanel" aria-labelledby="gallery_figure8_1_2_list_St_b1_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_heatmap_of_RPM_in_peak.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_1_2_pane_St_b2_vs_St_b3"
role="tabpanel" aria-labelledby="gallery_figure8_1_2_list_St_b2_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_heatmap_of_RPM_in_peak.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_1_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_1_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_1_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_2_list_St_b1_vs_St_b2"
data-toggle="list" href="#gallery_figure8_1_2_pane_St_b1_vs_St_b2" role="tab" aria-controls="St_b1_vs_St_b2">St_b1_vs_St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_2_list_St_b1_vs_St_b3"
data-toggle="list" href="#gallery_figure8_1_2_pane_St_b1_vs_St_b3" role="tab" aria-controls="St_b1_vs_St_b3">St_b1_vs_St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_1_2_list_St_b2_vs_St_b3"
data-toggle="list" href="#gallery_figure8_1_2_pane_St_b2_vs_St_b3" role="tab" aria-controls="St_b2_vs_St_b3">St_b2_vs_St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.25 Enrichment analysis among samples</b></p>
<p><small>
The picture is the clustering among different samples in comparable groups.</small></p>

<br />



<div id="page25">
</p><br />
<h4>4.7.3 Differential volcano diagram</h4>
<p class="paragraph">
For each comparison group, display the overall distribution of Fold Change and FDR in the different peak areas in the form of a volcano diagram, as shown in 4.23:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_2_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_2_pane_St_b1_vs_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure8_2_list_St_b1_vs_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_volcano.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_2_pane_St_b1_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_2_list_St_b1_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_volcano.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_2_pane_St_b2_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_2_list_St_b2_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_volcano.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_2_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_2_list_St_b1_vs_St_b2"
data-toggle="list" href="#gallery_figure8_2_pane_St_b1_vs_St_b2" role="tab" aria-controls="St_b1_vs_St_b2">St_b1_vs_St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_2_list_St_b1_vs_St_b3"
data-toggle="list" href="#gallery_figure8_2_pane_St_b1_vs_St_b3" role="tab" aria-controls="St_b1_vs_St_b3">St_b1_vs_St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_2_list_St_b2_vs_St_b3"
data-toggle="list" href="#gallery_figure8_2_pane_St_b2_vs_St_b3" role="tab" aria-controls="St_b2_vs_St_b3">St_b2_vs_St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.26 Differential volcano diagram</b></p>
<p><small>
The upward peak is represented by a red dot, the downward peak is represented by a blue dot, and the non differential peak is represented by a green dot</small></p>
<p><small>
The horizontal axis is the fold changes in signal intensity of the peak in two experiments; The vertical axis represents the statistical significance of the degree of difference between the two groups of experiments</small></p>

<br />
<h4>4.7.4 Difference violin diagram</h4>
<p class="paragraph">
For each comparison group, display the standardized read count distribution of the difference peak area in the form of a violin diagram, as shown in 4.24:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_3_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_3_pane_St_b1_vs_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure8_3_list_St_b1_vs_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_log2readcount.violin.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_3_pane_St_b1_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_3_list_St_b1_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_log2readcount.violin.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_3_pane_St_b2_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_3_list_St_b2_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_log2readcount.violin.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_3_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_3_list_St_b1_vs_St_b2"
data-toggle="list" href="#gallery_figure8_3_pane_St_b1_vs_St_b2" role="tab" aria-controls="St_b1_vs_St_b2">St_b1_vs_St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_3_list_St_b1_vs_St_b3"
data-toggle="list" href="#gallery_figure8_3_pane_St_b1_vs_St_b3" role="tab" aria-controls="St_b1_vs_St_b3">St_b1_vs_St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_3_list_St_b2_vs_St_b3"
data-toggle="list" href="#gallery_figure8_3_pane_St_b2_vs_St_b3" role="tab" aria-controls="St_b2_vs_St_b3">St_b2_vs_St_b3</a>

</div></div></div>
</p>
<p class="name_fig"><b>Figure 4.27 Difference violin diagram</b></p>
</div>

<div id="page26">
</p><br />
<h4>4.7.5 Annotation of differential peaks</h4>
<p class="paragraph">
Similar to peak annotation analysis, ChIPseeker software is used to annotate all differential peaks, up-regulated peaks, and down-regulated peaks in each functional area, as shown in 4.25:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_4_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_4_pane_St_b1_vs_St_b2" 
role="tabpanel" aria-labelledby="gallery_figure8_4_list_St_b1_vs_St_b2"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_alldiff_plotAnnoPie.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_4_pane_St_b1_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_4_list_St_b1_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_alldiff_plotAnnoPie.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_4_pane_St_b2_vs_St_b3" 
role="tabpanel" aria-labelledby="gallery_figure8_4_list_St_b2_vs_St_b3"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_alldiff_plotAnnoPie.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_4_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_4_list_St_b1_vs_St_b2"
data-toggle="list" href="#gallery_figure8_4_pane_St_b1_vs_St_b2" role="tab" aria-controls="St_b1_vs_St_b2">St_b1_vs_St_b2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_4_list_St_b1_vs_St_b3"
data-toggle="list" href="#gallery_figure8_4_pane_St_b1_vs_St_b3" role="tab" aria-controls="St_b1_vs_St_b3">St_b1_vs_St_b3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_4_list_St_b2_vs_St_b3"
data-toggle="list" href="#gallery_figure8_4_pane_St_b2_vs_St_b3" role="tab" aria-controls="St_b2_vs_St_b3">St_b2_vs_St_b3</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.28 Distribution of gene functional regions with differential peaks</b></p>
<br />

<div id="page27">
<br />
<h4>4.7.6 GO enrichment analysis of differential peak related genes</h4>
<p class="paragraph">
GO enrichment analysis for the differential peak related genes is shown below:
</p>
<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_6_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_6_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b2alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b2alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_alldiff_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b2down" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b2down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_down_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b2up" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b2up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_up_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b3alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b3alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_alldiff_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b3down" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b3down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_down_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b1_vs_St_b3up" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b1_vs_St_b3up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_up_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b2_vs_St_b3alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b2_vs_St_b3alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_alldiff_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b2_vs_St_b3down" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b2_vs_St_b3down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_down_go.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_6_pane_St_b2_vs_St_b3up" 
role="tabpanel" aria-labelledby="gallery_figure8_6_list_St_b2_vs_St_b3up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_up_go.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_6_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_6_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_6_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b2alldiff"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b2alldiff" role="tab" aria-controls="St_b1_vs_St_b2alldiff">St_b1_vs_St_b2alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b2down"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b2down" role="tab" aria-controls="St_b1_vs_St_b2down">St_b1_vs_St_b2down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b2up"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b2up" role="tab" aria-controls="St_b1_vs_St_b2up">St_b1_vs_St_b2up</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b3alldiff"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b3alldiff" role="tab" aria-controls="St_b1_vs_St_b3alldiff">St_b1_vs_St_b3alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b3down"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b3down" role="tab" aria-controls="St_b1_vs_St_b3down">St_b1_vs_St_b3down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b1_vs_St_b3up"
data-toggle="list" href="#gallery_figure8_6_pane_St_b1_vs_St_b3up" role="tab" aria-controls="St_b1_vs_St_b3up">St_b1_vs_St_b3up</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b2_vs_St_b3alldiff"
data-toggle="list" href="#gallery_figure8_6_pane_St_b2_vs_St_b3alldiff" role="tab" aria-controls="St_b2_vs_St_b3alldiff">St_b2_vs_St_b3alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b2_vs_St_b3down"
data-toggle="list" href="#gallery_figure8_6_pane_St_b2_vs_St_b3down" role="tab" aria-controls="St_b2_vs_St_b3down">St_b2_vs_St_b3down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_6_list_St_b2_vs_St_b3up"
data-toggle="list" href="#gallery_figure8_6_pane_St_b2_vs_St_b3up" role="tab" aria-controls="St_b2_vs_St_b3up">St_b2_vs_St_b3up</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.29 GO enrichment bar chart for the differential peak genes</b></p>

<p class="center">
<div class="row">
        <div class="col-1">
                <div id="gallery_figure_diffgodot_button_prev" class="button_prev" onclick="prev(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_prev">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-6">
                <div class="tab-content" id="gallery_figure_diffgodot_nav-tabContent">
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b2alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b2alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_alldiff_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b2down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b2down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_down_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b2up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b2up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_up_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b3alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b3alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_alldiff_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b3down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b3down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_down_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b1_vs_St_b3up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b1_vs_St_b3up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_up_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b2_vs_St_b3alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b2_vs_St_b3alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_alldiff_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b2_vs_St_b3down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b2_vs_St_b3down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_down_go_dot.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffgodot_pane_St_b2_vs_St_b3up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffgodot_list_St_b2_vs_St_b3up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_up_go_dot.png" />
                        </div>
                        
                </div>
        </div>
        <div class="col-1">
                <div id="gallery_figure_diffgodot_button_next" class="button_next" onclick="next(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_next">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-4">
                <input class="form-control" id="gallery_figure_diffgodot_InputList" type="text" placeholder="Search..">
                <div class="list-group" id="gallery_figure_diffgodot_list-tab" role="tablist">
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b2alldiff"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b2alldiff" role="tab" aria-controls="St_b1_vs_St_b2alldiff">St_b1_vs_St_b2alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b2down"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b2down" role="tab" aria-controls="St_b1_vs_St_b2down">St_b1_vs_St_b2down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b2up"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b2up" role="tab" aria-controls="St_b1_vs_St_b2up">St_b1_vs_St_b2up</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b3alldiff"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b3alldiff" role="tab" aria-controls="St_b1_vs_St_b3alldiff">St_b1_vs_St_b3alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b3down"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b3down" role="tab" aria-controls="St_b1_vs_St_b3down">St_b1_vs_St_b3down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b1_vs_St_b3up"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b1_vs_St_b3up" role="tab" aria-controls="St_b1_vs_St_b3up">St_b1_vs_St_b3up</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b2_vs_St_b3alldiff"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b2_vs_St_b3alldiff" role="tab" aria-controls="St_b2_vs_St_b3alldiff">St_b2_vs_St_b3alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b2_vs_St_b3down"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b2_vs_St_b3down" role="tab" aria-controls="St_b2_vs_St_b3down">St_b2_vs_St_b3down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffgodot_list_St_b2_vs_St_b3up"
                        data-toggle="list" href="#gallery_figure_diffgodot_pane_St_b2_vs_St_b3up" role="tab" aria-controls="St_b2_vs_St_b3up">St_b2_vs_St_b3up</a>
                        
                </div>
        </div>
</div>		
</p>
<p align="center"><b> Figure 4.30 GO enrichment scatter plot for the differential peak genes</b></p>



<h4>4.7.7 KEGG enrichment analysis of differential peak related genes</h4>
<p class="paragraph">
KEGG enrichment analysis for the differential peak related genes is shown below:
</p>

<p class="center">
<div class="row">
        <div class="col-1">
                <div id="gallery_figure_diffkeggbar_button_prev" class="button_prev" onclick="prev(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_prev">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-6">
                <div class="tab-content" id="gallery_figure_diffkeggbar_nav-tabContent">
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_alldiff_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_down_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_up_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_alldiff_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_down_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_up_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3alldiff" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3alldiff"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_alldiff_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3down" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3down"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_down_kegg_bar.png" />
                        </div>
                        
                        <div class="tab-pane fade" id="gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3up" 
                                role="tabpanel" aria-labelledby="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3up"
                                onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_up_kegg_bar.png" />
                        </div>
                        
                </div>
        </div>
        <div class="col-1">
                <div id="gallery_figure_diffkeggbar_button_next" class="button_next" onclick="next(this.id)">
                        <svg viewbox="0 0 16000 16000" class="svg_next">
                        <circle class="c" cx="8000" cy="8000" r="6080"></circle>
                        <polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
                        </polygon></svg>
                </div>
        </div>
        <div class="col-4">
                <input class="form-control" id="gallery_figure_diffkeggbar_InputList" type="text" placeholder="Search..">
                <div class="list-group" id="gallery_figure_diffkeggbar_list-tab" role="tablist">
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2alldiff"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2alldiff" role="tab" aria-controls="St_b1_vs_St_b2alldiff">St_b1_vs_St_b2alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2down"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2down" role="tab" aria-controls="St_b1_vs_St_b2down">St_b1_vs_St_b2down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b2up"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b2up" role="tab" aria-controls="St_b1_vs_St_b2up">St_b1_vs_St_b2up</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3alldiff"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3alldiff" role="tab" aria-controls="St_b1_vs_St_b3alldiff">St_b1_vs_St_b3alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3down"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3down" role="tab" aria-controls="St_b1_vs_St_b3down">St_b1_vs_St_b3down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b1_vs_St_b3up"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b1_vs_St_b3up" role="tab" aria-controls="St_b1_vs_St_b3up">St_b1_vs_St_b3up</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3alldiff"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3alldiff" role="tab" aria-controls="St_b2_vs_St_b3alldiff">St_b2_vs_St_b3alldiff</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3down"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3down" role="tab" aria-controls="St_b2_vs_St_b3down">St_b2_vs_St_b3down</a>
                        
                        <a class="list-group-item list-group-item-action" id="gallery_figure_diffkeggbar_list_St_b2_vs_St_b3up"
                        data-toggle="list" href="#gallery_figure_diffkeggbar_pane_St_b2_vs_St_b3up" role="tab" aria-controls="St_b2_vs_St_b3up">St_b2_vs_St_b3up</a>
                        
                </div>
        </div>
</div>		
</p>
<p align="center"><b> Figure 4.31 KEGG enrichment bar chart for differential peak genes</b></p>

<p class="center">
<div class="row">
<div class="col-1">
<div id="gallery_figure8_7_button_prev" class="button_prev" onclick="prev(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_prev">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
</polygon></svg></div></div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_7_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b2alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b2alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_alldiff_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b2down" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b2down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_down_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b2up" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b2up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b2_up_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b3alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b3alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_alldiff_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b3down" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b3down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_down_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b1_vs_St_b3up" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b1_vs_St_b3up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b1_vs_St_b3_up_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b2_vs_St_b3alldiff" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b2_vs_St_b3alldiff"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_alldiff_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b2_vs_St_b3down" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b2_vs_St_b3down"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_down_kegg.png" /></div>

<div class="tab-pane fade" id="gallery_figure8_7_pane_St_b2_vs_St_b3up" 
role="tabpanel" aria-labelledby="gallery_figure8_7_list_St_b2_vs_St_b3up"
onclick="trans(this.id)"><img class="img-gallery" src="src/images/St_b2_vs_St_b3_up_kegg.png" /></div>

</div></div>
<div class="col-1">
<div id="gallery_figure8_7_button_next" class="button_next" onclick="next(this.id)">
<svg viewbox="0 0 16000 16000" class="svg_next">
<circle class="c" cx="8000" cy="8000" r="6080"></circle>
<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
</polygon></svg></div></div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_7_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_7_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b2alldiff"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b2alldiff" role="tab" aria-controls="St_b1_vs_St_b2alldiff">St_b1_vs_St_b2alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b2down"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b2down" role="tab" aria-controls="St_b1_vs_St_b2down">St_b1_vs_St_b2down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b2up"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b2up" role="tab" aria-controls="St_b1_vs_St_b2up">St_b1_vs_St_b2up</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b3alldiff"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b3alldiff" role="tab" aria-controls="St_b1_vs_St_b3alldiff">St_b1_vs_St_b3alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b3down"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b3down" role="tab" aria-controls="St_b1_vs_St_b3down">St_b1_vs_St_b3down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b1_vs_St_b3up"
data-toggle="list" href="#gallery_figure8_7_pane_St_b1_vs_St_b3up" role="tab" aria-controls="St_b1_vs_St_b3up">St_b1_vs_St_b3up</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b2_vs_St_b3alldiff"
data-toggle="list" href="#gallery_figure8_7_pane_St_b2_vs_St_b3alldiff" role="tab" aria-controls="St_b2_vs_St_b3alldiff">St_b2_vs_St_b3alldiff</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b2_vs_St_b3down"
data-toggle="list" href="#gallery_figure8_7_pane_St_b2_vs_St_b3down" role="tab" aria-controls="St_b2_vs_St_b3down">St_b2_vs_St_b3down</a>

<a class="list-group-item list-group-item-action" id="gallery_figure8_7_list_St_b2_vs_St_b3up"
data-toggle="list" href="#gallery_figure8_7_pane_St_b2_vs_St_b3up" role="tab" aria-controls="St_b2_vs_St_b3up">St_b2_vs_St_b3up</a>

</div></div></div>
</p>
<p align="center"><b> Figure 4.32 KEGG enrichment scatter plot for differential peak genes</b></p>
</div>


<!------------------------------------------- Appendix------------------------------------------->
<div id="page28">
</p><br />
<h2><a id="build_library">5 Appendix</a></h2>
<h3 id="id_order_original" ><a name="original_data">5.1 Software</a></h3>
<table class= 'gy' style="text-align:left">
                <tr><th>Analysis</th><th>Software</th><th>Version</th><th>Remarks</th></tr>
                
                <tr><td>Trimming</td><td>fastp</td><td>0.20.0</td><td>length_required=50;n_base_limit=6;poly_x_min_len=10</td></tr>
                <tr><td>QC</td><td>Fastqc</td><td>v0.11.9</td><td></td></tr>
                
                
                <tr><td>Mapping</td><td>BWA</td><td>0.7.12-r1039</td><td>T=30;k=32</td>
                <tr><td>Correlation</td><td>deepTools</td><td>3.5.1</td><td></td></tr>
                
                
                <tr><td>Peak calling</td><td>MACS2</td><td>2.2.7.1</td><td>-q 0.05</td></tr>
                
                
                <tr><td>motif</td><td>homer findMotifsGenome.pl</td><td>4.11</td><td>-gc;-len 8,10,12,14</td></tr>
                
                
                <tr><td>GO</td><td>goseq</td><td>1.38.0</td><td>corrected pvalue<0.05</td></tr>
                <tr><td>KEGG</td><td>KOBAS</td><td>3.0</td><td>corrected pvalue<0.05</td></tr>
                
                
                <tr><td>differential analysis</td><td>diffbind(with replication)</td><td>2.14.0</td><td>FDR<0.05 & |log2FC|>1</td></tr>
                <tr><td>differential analysis</td><td>edgeR(no replication)</td><td>3.28.0</td><td>FDR<0.05 & |log2FC|>1</td></tr>
                
</table>
</br>
<h3 id="id_order_original" ><a name="original_data">5.2 Result File Decompression Method and Format Description</a></h3>
<table class= 'gy' style="text-align:left">
<tr><th>Compressed File Format</th><th>Operating System</th><th>File Decompression</th></tr>
<tr><td rowspan="2">*.gz</td><td>Unix/Linux/Mac</td><td>Command line: gzip –d *.gz</td></tr>
<tr><td>Windows</td><td>Software: such as WinRAR, 7-Zip et al.</td></tr>
<tr><td rowspan="2">*.zip</td><td>Unix/Linux/Mac</td><td>Command line: unzip *.zip</td></tr>
<tr><td>Windows</td><td>Software: such as WinRAR, 7-Zip et al.</td></tr>
</table>
</br>
<table class= 'gy' style="text-align:left">
<tr><th>File Type</th><th>File Description</th><th>Way to Open</th></tr>
<tr><td rowspan="2">file.fa</td><td rowspan="2">Fasta format is a text-based format for representing sequences</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td rowspan="2">file.gtf</td><td rowspan="2">GFF/GTF is general feature format.</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td rowspan="2">file.fq</td><td rowspan="2">Text file that contains sequence data delivered from a sequencer</td><td>Unix/Linux/Mac user use less or more command</td></tr>
<tr><td>Windows user use editor Editplus/Notepad++ et al.</td></tr>
<tr><td>file.bw</td><td>BigWig file. Indexed binary format which is useful for dense, continuous<br>data that will be dispalyed in the Genome Browser as a graph.</td><td>It can be visualized with the IGV observation or upload to UCSC browser.</td></tr>
<tr><td>file.bam</td><td>BAM is compressed format for SAM file which contains sequence<br>alignment information.</td><td>Unix/Linux/Mac user use samtools software.</td></tr>
<tr><td>file.bam.bai</td><td>Index file for bam</td><td>No need to open, put it in the same directory with bam file when<br>manipulate bam.</td></tr>
</table>
</br>
</div>


<!------------------------------------------- reference ------------------------------------------>
<div id="page0">
</p><br />
<h2><a id="Introduction">6 References</a></h2>
<p class="ref">Andrews S. (2010). FastQC: a quality control tool for high throughput sequence data. Available online at: http://www.bioinformatics.babraham.ac.uk/projects/fastqc</p>
<p class="ref">Ashburner M, Ball CA, Blake JA, et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet. 2000;25(1):25-29. doi:10.1038/75556</p>
<p class="ref">Erlich Y, Mitra P P, Delabastide M, et al. Alta-Cyclic: a self-optimizing base caller for next-generation sequencing[J]. Nature methods, 2008, 5(8): 679-682.</p>
<p class="ref">Faust GG, Hall IM. SAMBLASTER: fast duplicate marking and structural variant read extraction. Bioinformatics. 2014;30(17):2503-2505. doi:10.1093/bioinformatics/btu314</p>
<p class="ref">Jiang L, Schlesinger F, Davis C A, et al. Synthetic spike-in standards for RNA-seq experiments[J]. Genome research, 2011, 21(9): 1543-1551.</p>
<p class="ref">Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27-30. doi:10.1093/nar/28.1.27</p>
<p class="ref">Kanehisa M, Araki M, Goto S, et al. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 2008;36(Database issue):D480-D484. doi:10.1093/nar/gkm882</p>
<p class="ref">Kent WJ, Zweig AS, Barber G, Hinrichs AS, Karolchik D. BigWig and BigBed: enabling browsing of large distributed datasets. Bioinformatics. 2010;26(17):2204-2207. doi:10.1093/bioinformatics/btq351</p>
<p class="ref">Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25(14):1754-1760. doi:10.1093/bioinformatics/btp324</p>
<p class="ref">Nicol JW, Helt GA, Blanchard SG Jr, Raja A, Loraine AE. The Integrated Genome Browser: free software for distribution and exploration of genome-scale datasets. Bioinformatics. 2009;25(20):2730-2731. doi:10.1093/bioinformatics/btp472</p>
<p class="ref">Thorvaldsdóttir H, Robinson JT, Mesirov JP. Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 2013;14(2):178-192. doi:10.1093/bib/bbs017</p>
<p class="ref">Young MD, Wakefield MJ, Smyth GK, Oshlack A. Gene ontology analysis for RNA-seq: accounting for selection bias. Genome Biol. 2010;11(2):R14. doi:10.1186/gb-2010-11-2-r14</p>
<p class="ref">Zhang Y, Liu T, Meyer CA, et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 2008;9(9):R137. doi:10.1186/gb-2008-9-9-r137</p>
</div>
</body>
</html>
