<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >

<html>

<head>

<TITLE> RNA-seq Analysis Report</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />



<link rel="stylesheet" href="src/css/jquery-ui.css" />

<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>
<link rel="stylesheet" href="src/css/styles.css"/>

<link rel="stylesheet" href="src/css/base.css"/>

<link rel="stylesheet" href="src/css/gallery.css"/>

<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>

<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>
<script src="src/js/bootstrap-table.js" type="text/javascript"></script>



<script type="text/javascript">

	$(function () {

		var options = {

			selectors: "h2,h3,h4,h5",

			theme: "bootstrap",

			scrollTo: "60"

		};

		var toc = $("#toc").tocify(options).data("toc-tocify");

	});

</script>


<style>

table {
border-collapse:collapse; /* 关键属性：合并表格内外边框(其实表格边框有2px，外面1px，里面还有1px哦) */
border:solid #D1D0CE; /* 设置边框属性；样式(solid=实线)、颜色(#999=灰) */
border-width:0px 0 0px 0px; /* 设置边框状粗细：上 右 下 左 = 对应：1px 0 0 1px */
}

.left{
    border:solid #D1D0CE;border-width:0px 0px 0px 0px;padding:2px;
}

.mid{
    border:solid #D1D0CE;border-width:2px 0px 2px 0px;padding:2px;
}
</style>




<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");

if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");

break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});

$(".list-group").each(show_default);

});



</script>







</head>



<body>



<div id="toc" class="tocify">

</div>





	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"

			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">

			<tr height="86%">

				<td width='1%'></td>

                                <td><img src="src/pdf_images/logo1.png" ></td>

				<td>

					<div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>

				</td>

				<td width="20%"></td>

				<td width="15%"></td>

				<td width="25%"

					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">


					<br><a href="mailto:op-rna@novogene.com" target="black"

						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">

						<a></td>

				<td width="1%"></td>

			</tr>

		</table>

	</div>


</div>



<!--------------------------标题---------------------------------->

<div id="main">

</br><h1>RNAseq analysis report</h1></br>

<table class="gy" style="text-align:left">

<tbody>



<tr>



<td style="text-align:left">Contract ID</td>



<td style="text-align:left">H101SC24071907</td>



</tr>



<tr>



<td style="text-align:left">Contract Name</td>



<td style="text-align:left">天津肿瘤医院6例转录组测序分析技术服务（委托）合同-侯春宇</td>



</tr>



<tr>



<td style="text-align:left">Batch ID</td>



<td style="text-align:left">X101SC2407XXXX-Z01-F001</td>



</tr>



<tr>



<td style="text-align:left">Reference Genome and Version</td>



<td style="text-align:left">Mus_musculus_Ensemble_94</td>



</tr>



<tr>



<td style="text-align:left">Report Time</td>



<td style="text-align:left">2025-01-19</td>



</tr>



<tr>



<td style="text-align:left">Reminder</td>



<td style="text-align:left">This report displays partial results, only showing 5 groups for comparative analysis, while full results will be delivered in data release.<br> Hyperlink of results in this report will be only valid in data release, after statement confirmation.</td>



</tr>



</tbody></table>

<br /><br />
﻿<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show comment";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden comment";
      };
      return 0;
    }
    function hiddenShow1(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Q&A";
      };
      return 0;
    }
	
</script>
<!----------------------------------------建库测序流程----------------------------------------->


<br />

<h2> 1 Introduction </h2>

<p class="paragraph">A transcriptome is a set of all the transcripts in one cell or one population of cells at certain status. Transcriptome analysis assists to study the identification of genes that are differentially expressed in distinct cell populations. Researchers can also gain a deeper insight into gene boundary identification, variable cleavage and transcript variation(Wang et al., 2009).</p>
<p class="paragraph">RNA sequencing via Illumina platforms, based on the mechanism of SBS (sequencing by synthesis), offers a wide range of benefits on high throughput and high accuracy out of low sample requirements. This technical method can be a powerful tool for researching RNA transcriptional activity.</p>
<p class="paragraph">Workflow is shown as follows：</p></br>

<p class="center">





	<p class = "center"><img class="center" src="src/pdf_images/sequence_pipeline.png" height=550 width=650/>
<p class="name_fig">Figure 1 Project workflow </p>



</p>

<!----------------------------------------建库测序流程----------------------------------------->


<br />

<h2> 2 Library Construction and Sequencing </h2>
	<h3>2.1 Sample Quality Control </h3>

<p class="paragraph">Please refer to  QC report for methods of sample quality control.</p>
	<h3>2.2 Library Construction, Quality Control and Sequencing </h3>
<p class="paragraph">Messenger RNA was purified from total RNA using poly-T oligo-attached magnetic beads. After fragmentation, the first strand cDNA was synthesized using random hexamer primers, followed by the second strand cDNA synthesis using either dTTP for non strand specific library or dUTP for strand specific library.</p>
<p class="paragraph">For the non strand specific library, it was ready after end repair, A-tailing, adapter ligation, size selection, amplification, and purification(Figure 2.A).</p>
<p class="paragraph">For the strand specific library, it was ready after end repair, A-tailing, adapter ligation, size selection, amplification, and purification(Figure 2.B).</p>
<p class="center">





	<p class = "center"><img class="center" src="src/pdf_images/Figure_2.A_Workflow_of_non-directional_library_construction.png" height=140 width=220/>
<p class="name_fig">Figure 2.A Workflow of non-directional library construction </p>
<p class = "center"><img class="center" src="src/pdf_images/Figure_2.B_Workflow_of_directional_library_construction.png" height=140 width=220/>
<p class="name_fig">Figure 2.B Workflow of directional library construction</p>
	<p class="paragraph">The library was checked with Qubit and real-time PCR for quantification and bioanalyzer for size distribution detection. Quantified libraries will be pooled and sequenced on Illumina platforms, according to effective library concentration and data amount.</p>


</p>

Method details involved in the project are available at <a href="src/pdf_images/methods_med.pdf" title="点击打开" target="_blank">methods</a>

<br/>



<!----------------------------------------生物信息分析流程----------------------------------------->

<br />

<h2>3 Bioinformatics Analysis Pipeline</h2>

<p class="paragraph">This is the workflow for medical species (human/mouse) mRNA sequencing data of standard bioinformatic analysis with a well-annotated reference genome, as follows:</p>

<p class="center">



	<img class="center" height="550" width="650" src="src/pdf_images/pipline_med.png" /></p>
<p class="name_fig">Figure 3 RNA-seq information analysis technology flow</p>


</p>
<p class="paragraph">For the analysis content of the above figure, if it exists in the content of the contract information analysis, the analysis is performed; if it does not exist, it is not performed.</p>
<br/>

Method details involved in the project are available at <a href="src/pdf_images/methods_med.pdf" title="点击打开" target="_blank">methods</a>

<br/>
<br/>



<!-------------------------------------------- 原始序列数据及质量评估 ----------------------------->




<br />

<h2>4 Analysis Results</h2>



	<h3>4.1 Data Quality Control </h3>
		
		
		<h4>4.1.1 Raw Data</h4>




<p class="paragraph">Original image data file from high-throughput sequencing platforms (like Illumina) is transformed to sequenced 
	reads (called Raw Data or Raw Reads) by CASAVA base recognition (Base Calling). Raw data are stored in FASTQ(fq) format files, 
	which contain sequences of reads and corresponding base quality. Each read has four descriptive lines, as indicated below:</p>

<p class="paragraph">Each read has four descriptive lines, as indicated below:</p>
 
 <div class="block">

<p style="font-size:14px">
<span class="citation"><span class="citation">
 
	@A01426:11:H73WJDSX2:1:1101:2329:1016</span></span>1:N:0:GTGAGATC+GAACTAGC<br />

	TCGCACGCGCTGCCGTATGTGACGCCGTCGCTGCCGCACACAGGATCGTAGAGGCTCGC<br />

	+<br />

	FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,FFFFF:FFFFFFFFFFF<br />

</p>
</div > 
<ol style="list-style-type: decimal">
<li>Line 1: Begins with the at sign (@) followed by sequence identifiers and optional description (such as FASTA header).</li>
<li>Line 2: Base sequences (raw read, A, G, C, and T).</li>
<li>Line 3: Begins with the plus sign (+) optionally followed by the same Illumina sequence identifiers and description information as in Line 1.</li>
<li>Line 4: The quality values for each base, corresponding to the data in Line 2.</li>
</ol>
<p style="font-weight: bold;">Illumina Sequence Identifier:</p>

<table class="gy">
	<tr>

			<th>Identifier</th>

			<th>Meaning</th>


</tr>
	<tr>

			<td>A01426</td>

			<td>Instrument – unique identifier of the sequencer</td>
</tr>
	<tr>
<td>11</td>
<td>run number – Run number on instrument</td>
</tr>
	<tr>
<td>H73WJDSX2</td>
<td>Flowcell ID - ID of flowcell</td>
</tr>
	<tr>
<td>1</td>
<td>LaneNumber - positive integer</td>
</tr>
	<tr>
<td>1101</td>
<td>TileNumber - positive integer</td>
</tr>
	<tr>
<td>2329</td>
<td>X - x coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>1016</td>
<td>Y - y coordinate of the spot. Integer which can be negative </td>
</tr>
	<tr>
<td>1</td>
<td>Read Number - 1 can be single read or Read 2 of paired-end</td>
</tr>
	<tr>
<td>N</td>
<td>Y if the read is filtered out (did not pass), N otherwise</td>
</tr>
	<tr>
<td>0</td>
<td>control number - 0 when none of the control bits are on, otherwise it is an even number</td>
</tr>
	<tr>
<td>GTGAGATC+GAACTAGC</td>
<td>Illumina index sequences</td>
</tr>
</table>
<br/> 
The details of Illumina Sequencing identifier are as follows: 
<ol style="list-style-type: decimal">
<li>A01426:11 A01426, Instrument - unique identifier of the sequencer; 11, Run number - Run number on instrument</li>
<li>H73WJDSX2:1:1101:2329:1016 means the coordinate of read on H73WJDSX2 (Flowcell ID) flowcell, line 1, 1101 tile is(x=2329, y=1016)</li>
<li>1:N:0:GTGAGATC+GAACTAGC the first number is 1 or 2, 1 refers to single reads or the first read of paired ends, 2 refers to the second of paired ends; the second letter means whether reads is adjusted(Y means yes, N means no); the third number represent the number of Control Bits in sequence; six bases on the fourth place is Illumina index sequence.</li>
</ol>	



	<h4>4.1.2 Examination of Sequencing Error Rate</h4>

	<p class="paragraph">The error rate for each base can be transformed by the Phred score as in equation 1. "e" represents sequencing error rate, 
	"Qphred" represents base quality values of Illumina platforms (equation 1: Qphred = -10log10(e)).</p>
	<p class="name center">Table 4.1 Illumina Casava 1.8 version base recognition and Phred score Concise correspondence between them</p>

<table class="gy">
<thead>
<tr>
<th style="text-align:center;width: 8em;">
Phred.score
</th>
<th style="text-align:center;width: 8em;">
Base.Calling.error.rate
</th>
<th style="text-align:center;width: 8em;">
Base.Calling.correct.rate
</th>
<th style="text-align:center;width: 8em;">
Q.sorce
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;width: 8em;">
10
</td>
<td style="text-align:center;">
1/10
</td>
<td style="text-align:center;">
90%
</td>
<td style="text-align:center;">
Q10
</td>
</tr>
<tr>
<td style="text-align:center;width: 8em;">
20
</td>
<td style="text-align:center;">
1/100
</td>
<td style="text-align:center;">
99%
</td>
<td style="text-align:center;">
Q20
</td>
</tr>
<tr>
<td style="text-align:center;width: 8em;">
30
</td>
<td style="text-align:center;">
1/1000
</td>
<td style="text-align:center;">
99.9%
</td>
<td style="text-align:center;">
Q30
</td>
</tr>
<tr>
<td style="text-align:center;width: 8em;">
40
</td>
<td style="text-align:center;">
1/10000
</td>
<td style="text-align:center;">
99.99%
</td>
<td style="text-align:center;">
Q40
</td>
</tr>
</tbody>
</table>


	<p class="paragraph">Sequencing error rate and base quality varies depending on sequencers, reagent residues, and different sample types. For RNA-seq technology, 
	sequencing error rate distribution can be featured as below:</strong></p>

<ul style="font-size: 14px"> 
	<li>Error rate increases with the sequencing reads for consumption of sequencing reagent. 
	It is common in the Illumina high-throughput sequencing platform (Erlich Y, Mitra PP et al.2008; Jiang L, Schlesinger F et al.2011.)</li>


	<li> The first six bases have a relatively high error rate due to the incomplete binding of random hexamers used in priming cDNA synthesis (Jiang et al.). 
		In general, a single base error rate should be lower than 1%. </li>
</ul>



	<br />

	<p class="center">

<div class="row gallery-row">

<div class="col-1">

<div id="gallery_figure1_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>

</div>

<div class="col-6">

<div class="tab-content" id="gallery_figure1_1_nav-tabContent">



<div class="tab-pane fade" id="gallery_figure1_1_pane_nc" role="tabpanel" aria-labelledby="gallery_figure1_1_list_nc" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Errorrate/nc_Error.png" /></div>



<div class="tab-pane fade" id="gallery_figure1_1_pane_H2_inhit" role="tabpanel" aria-labelledby="gallery_figure1_1_list_H2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Errorrate/H2_inhit_Error.png" /></div>



</div>

</div>

<div class="col-1">

<div id="gallery_figure1_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>

</div>

<div class="col-4">

<input class="form-control" id="gallery_figure1_1_InputList" type="text" placeholder="Search..">

<div class="list-group" id="gallery_figure1_1_list-tab" role="tablist">



<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_nc" data-toggle="list" href="#gallery_figure1_1_pane_nc" role="tab" aria-controls="nc">nc</a>



<a class="list-group-item list-group-item-action" id="gallery_figure1_1_list_H2_inhit" data-toggle="list" href="#gallery_figure1_1_pane_H2_inhit" role="tab" aria-controls="H2_inhit">H2_inhit</a>



</div>

</div>

</div>



	</p>

	<p class="name_fig">Figure 4.1 Sequencing data error rate distribution</p>

	<p class="premark center ">The x-axis shows the base position along each sequencing read and the y-axis shows the base error rate.</p>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/1.QC/1.Error" title="点击打开" >QC/1.Error</a> </p>









<br />

	<h4 >4.1.3 GC Content Distribution</h4>

	

	<p class="paragraph"> GC content distribution detects potential AT/GC separation, 
	which affects subsequent gene expression quantification. In view of random fragmentation and biological law 
	of G/C-A/T content, G and C, A and T should be respectively equal, and the content should be stable throughout 
	the entire sequencing process for non-stranded library(Parkhomchuk et al., 2009) (If the library is strand-specific, AT separation or GC separation may occur). 
	A large variation of sequencing error in the first 6-7 bases is allowed considering the use of random primer in library construction, 
	in which it is normal that the first few bases have certain preference in existing high-throughput sequencing technology.
    </p>



	<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure1_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_2_pane_nc" role="tabpanel" aria-labelledby="gallery_figure1_2_list_nc" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/GC/nc_GC.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_2_pane_H2_inhit" role="tabpanel" aria-labelledby="gallery_figure1_2_list_H2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/GC/H2_inhit_GC.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure1_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_nc" data-toggle="list" href="#gallery_figure1_2_pane_nc" role="tab" aria-controls="nc">nc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_2_list_H2_inhit" data-toggle="list" href="#gallery_figure1_2_pane_H2_inhit" role="tab" aria-controls="H2_inhit">H2_inhit</a>

</div>
</div>
</div>

	</p>

	<p class="name_fig">Figure 4.2 GC content distribution</p>

	<p class="premark center ">The x-axis shows each base position within a read, and the y-axis shows the percentage of each base, 
	with each base represented by a different color. </p>

	<p class="premark center">The left side of the vertical dashed line is the GC-content of read 1, the right side is the GC-content of read 2.</p>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/1.QC/2.GC" title="点击打开" >QC/2.GC</a> </p>




<br />

	<h4 >4.1.4 Sequencing Data Filtering</h4>



	<p class="paragraph"> The sequencing reads/raw reads often contain low quality reads or reads with adapters, 
	which will affect the quality of downstream analysis. To avoid this, it is necessary to filter the raw reads and obtain the clean reads.</p>
Raw reads filtering is as follows:
<p class="paragraph">



	<img class="center" height="550" alt="" width="680" src="src/pdf_images/Figure_4.3_Workflow_of_sequencing_data_filtration.png" style="margin: 10px  120px;"/></p>
	<p class="name_fig">Figure 4.3 Workflow of sequencing data filtration</p>
<ul style="font-size: 14px">
<li>Remove reads with adapter contamination.</li>
<li>Remove reads when uncertain nucleotides constitute more than 10 percent of either read (N > 10%).</li>
<li> Remove reads when low quality nucleotides (Base Quality less than 5) constitute more than 50 percent of the read.(Yan et al, 2011)</li></p>
</ul>
	
<p style="font-weight: bold;">Adapter Sequences:</p>

 <div class="block">

<p style="font-size:14px">
<span class="citation"><span class="citation">
    P5 adapter：<br />
    P5→P7’(5’→3 ’)<br />
     AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCT<br />
    <br />
    P7 adapter：<br />
    P5→P7’(5’→3’)<br />
     GATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG<br />
	
</p>
</div > 
<br/>
<br/>
<br/>

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure1_3_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure1_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure1_3_pane_nc" role="tabpanel" aria-labelledby="gallery_figure1_3_list_nc" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Rawstat/nc.rawstat.png" /></div>

<div class="tab-pane fade" id="gallery_figure1_3_pane_H2_inhit" role="tabpanel" aria-labelledby="gallery_figure1_3_list_H2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Rawstat/H2_inhit.rawstat.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure1_3_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure1_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure1_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure1_3_list_nc" data-toggle="list" href="#gallery_figure1_3_pane_nc" role="tab" aria-controls="nc">nc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure1_3_list_H2_inhit" data-toggle="list" href="#gallery_figure1_3_pane_H2_inhit" role="tab" aria-controls="H2_inhit">H2_inhit</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.4 Sample Sequencing Data Filtering</p>
    <p class="premark center ">Note: The proportions of the different colors in the graph represent the proportion of different components.</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
	    <strong> Adapter related</strong>: (reads containing adapter) / (total raw reads).<br/>

        <strong> Containing N</strong>: (reads with more than 10% N) / (total raw reads).<br/>

        <strong> Low quality</strong>: (reads of low quality) / (total raw reads).<br/>

        <strong> Clean reads</strong>: (clean reads) / (total raw reads).<br/>
        
</ul>
</div>



<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/1.QC/3.Filter" title="点击打开" >QC/3.Filter</a> </p>







<br />

	<h4 >4.1.5 Statistics of Sequencing Quality</h4>
	<p class="paragraph">Data are summarized in the table below.</p> 

	<p class="paragraph"></p>

	<p class="name center">Table 4.2 Sample Sampling Data Quality Summary</p>

	<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">

		<thead>

			<tr>

				<th>sample</th><th>library</th><th>raw_reads</th><th>raw_bases</th><th>clean_reads</th><th>clean_bases</th><th>error_rate</th><th>Q20</th><th>Q30</th><th>GC_pct</th>

			</tr>

		</thead>

		<tbody>

			

			<tr><td>nc</td><td>FRAS240206073-1r</td><td>7289920</td><td>1.09G</td><td>7082722</td><td>1.06G</td><td>0.02</td><td>95.69</td><td>89.35</td><td>42.78</td></tr>

			

			<tr><td>H2_inhit</td><td>FRAS240206076-1r</td><td>21839712</td><td>3.28G</td><td>19700916</td><td>2.96G</td><td>0.02</td><td>94.77</td><td>88.1</td><td>40.71</td></tr>

			

		</tbody>

	</table>

<br/>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>sample</strong>: SampleID.</li>
<li><strong>library</strong>: Library ID.</li>
<li><strong>raw_reads</strong>: Reads count from the raw data, four rows as a unit, with statistics of reads count for every sequencing.</li>
<li><strong>raw_bases</strong>: Base number of raw data. (number of raw reads) * (sequence length), converting unit to G.</li>
<li><strong>clean_reads</strong>: Base number of raw data after filtering. (number of clean reads) * (sequence length), converting unit to G.</li>
<li><strong>clean_bases</strong>: (clean base=clean reads*150bp) number multiply read length, saved in G unit.</li>
<li><strong>error_rate</strong>: Average sequencing error rate, which is calculated by Qphred=-10log10(e).</li>
<li><strong>Q20</strong>: The percentage of the bases whose Q Phred values is greater than 20. (Number of bases with Q Phred value &gt; 20) / (Number of total bases) *100.</li>
<li><strong>Q30</strong>: The percentage of the bases whose Q Phred values is greater than 30. (Number of bases with Q Phred value &gt; 30) / (Number of total bases) *100.</li>
<li><strong>GC_pct</strong>: The percentage of G&amp;C base numbers of total bases.(G&amp;C base number) / (Total base number)*100.</li>

</ul>
</div>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/1.QC/4.Stat/data_table.xls" title="点击打开" >QC/4.Stat/data_table.xls</a> </p>
<br/>



<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Interpretation of Relevant Nouns</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><strong>Q20</strong>:The percentage of the total bases have Phred greater than 20. Phred=-10log10(e)<br/><strong>Q30</strong>:The percentage of the total bases have Phred greater than 30. Phred=-10log10(e)</td></tr>

<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">The sequencing error rate increases as the sequence's length increases. What is the acceptable error rate?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">We set a high standard for sequencing quality. Generally, the error rate of a single base should be lower than 1%. In some special cases, the maximum error rate of a single base should not be greater than 6%.</td></tr>
</table>
</div>
<br/>
<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show comment";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden comment";
      };
      return 0;
    }
    function hiddenShow1(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Q&A";
      };
      return 0;
    }
	
</script>

<!----------------------------------------- 参考序列比对 ---------------------------------------->
<br/>


	<h3 >4.2 Alignment</h3>




<p class="paragraph">Perform alignments with <a href="http://ccb.jhu.edu/software/hisat2/faq.shtml">HISAT2</a> to the reference. HISAT2(Mortazavi et al., 2008) uses a graph-based alignment and has succeeded HISAT and TOPHAT2. HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads. Besides one global GFM index, HISAT2 also includes a large set of small GFM indexes that collectively cover the whole genome. These small indexes (local indexes), combined with multiple alignment strategies, enabled effective alignment of RNA-seq reads, particularly, reads spanning multiple exons. The following figure shows the algorithm of split reads comparison by HISAT2:</p>

<p class="center">

	<img class="small" src="src/pdf_images/hisat.jpg" /></p>

</p>

<strong>The algorithm of hisat is divided into three parts:</strong>
<ol style="list-style-type: decimal">
<li>The whole sequence was aligned to a single exon.</li>
<li>The sequencing sequence was piecewise aligned to two exons of the genome.</li>
<li>The sequencing sequence was segmented and aligned to more than three (including three) exons of the genome.</li>
</ol>






<br />

<h4>4.2.1 Mapping Result</h4>
<p class="paragraph">The data are summarized in the table below.</p>
<p class="name center">Table 4.3 Comparison of sample and reference genomes</p>

<div style="overflow-x: auto">

	<table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">

		<thead>

			<tr>

				<th>sample</th><th>total_reads</th><th>total_map</th><th>unique_map</th><th>multi_map</th><th>read1_map</th><th>read2_map</th><th>positive_map</th><th>negative_map</th><th>splice_map</th><th>unsplice_map</th><th>proper_map</th>

			</tr>

		</thead>

		<tbody>

			

			<tr><td>nc</td><td>7082722</td><td>3102556(43.8%)</td><td>2779855(39.25%)</td><td>322701(4.56%)</td><td>1448697(20.45%)</td><td>1331158(18.79%)</td><td>1362350(19.23%)</td><td>1417505(20.01%)</td><td>254034(3.59%)</td><td>2525821(35.66%)</td><td>2237836(31.6%)</td></tr>

			

			<tr><td>H2_inhit</td><td>19700916</td><td>14029837(71.21%)</td><td>13487805(68.46%)</td><td>542032(2.75%)</td><td>6933425(35.19%)</td><td>6554380(33.27%)</td><td>6658589(33.8%)</td><td>6829216(34.66%)</td><td>407535(2.07%)</td><td>13080270(66.39%)</td><td>11399478(57.86%)</td></tr>

			

		</tbody>

	</table>

</div>



<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>sample</strong>: SampleID.</li>
<li><strong>total_reads</strong>: Total clean reads used for analysis.</li>
<li><strong>total_map</strong>: Number and percentage of reads aligned to the genome, the ratio should higher than 70%, total mapping rate: (mapped reads)/(total reads)*100.</li>
<li><strong>unique_map</strong>: Number and percentage of reads aligned to the unique position of the reference genome (for subsequent quantitative data analysis), unique mapping rate: (uniquely mapped reads)/(total reads)*100.</li>
<li><strong>Multi_map</strong>: number and percentage of reads aligned to multiple locations in the reference genome, multiple mapping rate: (multiple mapped reads)/(total reads)*100.</li>
<li><strong>read1_map</strong>: Number and percentage of read1 aligned to the reference genome.</li>
<li><strong>read2_map</strong>: Number and percentage of read2 aligned to the reference genome.</li>
<li><strong>positive_map</strong>: Number and percentage of reads aligned to the positive chain of the reference genome.</li>
<li><strong>negative_map</strong>: Number and percentage of reads aligned to the negative chain of the reference genome.</li>
<li><strong>splice_map</strong>: Number of spliced reads on the genome and its percentage.</li>
<li><strong>unsplice_map</strong>: Number of complete reads aligned to genome and its percentage.</li>
<li><strong>proper_map</strong>: Number of paired read1 and read2 aligned to the genome and its percentage.</li>

</ul>
</div>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/2.Mapping/2.Stat/align_pct.xls" title="点击打开" >Mapping/2.Stat/align_pct.xls</a> </p>


<br />

<h4>4.2.2 Reads Distribution in Reference Genome</h4>



<p class="paragraph">Mapped regions can be classified as exons, introns, or intergenic regions. 
Exon-mapped reads should be the most abundant type of read when the reference genome is well-annotated. 
Intron-reads may be derived from pre-mRNA contamination or intron-retention from alternative splicing. 
Reads mapped to intergenic regions are mainly attributed to weak annotation of the reference genome.
 The distribution of sequencing reads of all samples in the genomic region is shown in the figure below. </p>

<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure2_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure2_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure2_1_pane_nc" role="tabpanel" aria-labelledby="gallery_figure2_1_list_nc" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Mapregion/nc.region.png" /></div>

<div class="tab-pane fade" id="gallery_figure2_1_pane_H2_inhit" role="tabpanel" aria-labelledby="gallery_figure2_1_list_H2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/QC/Mapregion/H2_inhit.region.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure2_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure2_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure2_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_nc" data-toggle="list" href="#gallery_figure2_1_pane_nc" role="tab" aria-controls="nc">nc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure2_1_list_H2_inhit" data-toggle="list" href="#gallery_figure2_1_pane_H2_inhit" role="tab" aria-controls="H2_inhit">H2_inhit</a>

</div>
</div>
</div>
</p>

<p class="name_fig">Figure 4.5 Sequencing reads in the genomic region </p>
<p class="paragraph">Note: The ratios of the different colors in the figure represent the ratio of reads to different regions</p>
<p class="premark">&nbsp</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<strong>exon</strong>: The number of bases aligned to exon regions of the genome and its proportion to the number of bases aligned to the genome.<br/>
<strong>Intron</strong>: The number of bases aligned to intron regions of the genome and its proportion to the number of bases aligned to the genome.<br/>
<strong>Intergenic</strong>: The number of bases aligned to intergenic regions of the genome and its proportion to the number of bases aligned to the genome.<br/>

</ul>
</div>




<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/2.Mapping/1.Region" title="点击打开" >Mapping/1.Region</a> </p>



<br />

<h4>4.2.3 Visualization of Mapping Results</h4>

<p class="paragraph">Files provided in BAM format--a standard file format that contains mapping results–indicate the information of the corresponding referenced genome and gene annotations for some species. 
The Integrative Genomics Viewer (IGV) is a recommended software for visualizing data from BAM files. </p>
<p style="font-weight: bold;">IGV can be featured as below:</p>
<ol style="list-style-type: decimal">
<li>Displaying the positions of single or multiple reads in the reference genome, and read distribution between annotated exons, introns or intergenic regions, both in adjustable scale respectively;</li>
<li>Displaying the read abundance of different regions to demonstrate their expression levels, in adjustable scale;</li>
<li>Providing annotation information for both genes and splicing isoforms;</li>
<li>Providing other related annotation information;</li>
<li>Displaying annotations downloaded from remote servers and/or imported local machines.</li>
<li>IGV browser usage can refer to our provided documentation <a href="src/pdf_images/IGVQuickStart.pdf">IGVQuickStart</a></li>
</ol>
<p class="center">

	<img class="small" src="src/pdf_images/igv.png" />

</p>

<p class="name_fig">Figure 4.6 IGV Browser Comparison Visualization of Results </p>

<br/>

<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
  <table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the difference between the RNA data comparison software and the DNA data comparison software?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Due to the existence of alternative splicing during transcription, a large portion of the measured reads spans different exons, and all RNA data comparison software supports reads' splice comparison.</td></tr>

<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What are the reasons for the lower rate of mapping?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The assembly of reference genome is not satisfying<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The relative relation between the tested species and the reference genome is far<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Special treatments or exogenous contaminations of samples</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: Does the mapping use full length of reads, or does the mapping use reads that are processed at the beginning and the end?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: In our standard procedure, we use the standard RNA-seq kit whose indexes are in the middle of the adapters and the sequencing will be executed for the sequences with indexes. In this way, the sequences of read 1 and read 2 are from the samples and there is no need to process the sequences in mapping. Even if the adapters are in sequences or the quality is low, the read will be removed.</td></tr>
</table>
</div>
<br/>




﻿<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
    function hiddenShow1(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Q&A";
      };
      return 0;
    }
	
</script>
<!----------------------------------------- 基因表达分析 --------------------------------------------------------------->

<br />

<!--  -->

	<h3 >4.3 <span style="text-transform: capitalize;">gene</span> Expression Level Analysis</h3>



<p class="paragraph"><span style="text-transform: capitalize;">gene</span> expression level analysis is the core task in the RNA-seq experiment. <span style="text-transform: capitalize;">gene</span> expression level is calculated by the number of mapped reads.</p>

<h4> 4.3.1 <span style="text-transform: capitalize;">gene</span> Expression Quantification </h4>



<p class="paragraph">The abundance of transcripts reflects gene expression level directly. In RNA-seq experiments, gene expression level is estimated by the abundance of transcripts (count of sequencing) that mapped to genome or exon(Goldstein et al., 2016). Read counts is proportional to gene expression level, gene length and sequencing depth(Liao et al., 2014). FPKM (short for the expected number of Fragments Per Kilobase of transcript sequence per Millions base pairs sequenced)(Trapnell, Cole, et al., 2010) is the most common method of estimating gene expression levels, which takes the effects into consideration of both sequencing depth and gene length on counting of fragments (Mortazavi et al., 2008). The data are summarized in the table below. </p>i

<p class="name center">Table 4.4 Quantitative results of gene expression</p>

<p class="center">



	<div id="tb" style="overflow-x :auto">

		<table class="gy">

			<thead>

				<tr>

					<th>gene_id</th><th>nc</th><th>H2_inhit</th><th>gene_name</th><th>gene_chr</th><th>gene_start</th><th>gene_end</th><th>gene_strand</th><th>gene_length</th><th>gene_biotype</th><th>gene_description</th><th>tf_family</th>

				</tr>

			</thead>

			<tbody>

				

				<tr><td>ENSMUSG00000064351</td><td>19002</td><td>24333</td><td>mt-Co1</td><td>MT</td><td>5328</td><td>6872</td><td>+</td><td>1545</td><td>protein_coding</td><td>mitochondrially encoded cytochrome c oxidase I [Source:MGI Symbol;Acc:MGI:102504]</td><td>-</td></tr>

				

				<tr><td>ENSMUSG00000064339</td><td>8266</td><td>7394</td><td>mt-Rnr2</td><td>MT</td><td>1094</td><td>2675</td><td>+</td><td>1582</td><td>Mt_rRNA</td><td>mitochondrially encoded 16S rRNA [Source:MGI Symbol;Acc:MGI:102492]</td><td>-</td></tr>

				

				<tr><td>ENSMUSG00000064341</td><td>5273</td><td>5613</td><td>mt-Nd1</td><td>MT</td><td>2751</td><td>3707</td><td>+</td><td>957</td><td>protein_coding</td><td>mitochondrially encoded NADH dehydrogenase 1 [Source:MGI Symbol;Acc:MGI:101787]</td><td>-</td></tr>

				

				<tr><td>ENSMUSG00000037443</td><td>5025</td><td>12730</td><td>Cep85</td><td>4</td><td>134129858</td><td>134187112</td><td>-</td><td>5269</td><td>protein_coding</td><td>centrosomal protein 85 [Source:MGI Symbol;Acc:MGI:1917262]</td><td>-</td></tr>

				

			</tbody>

		</table>

	</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>gene_id</strong>: <span style="text-transform: capitalize;">gene</span> number.</li>
<li><strong>sample</strong>: Raw read count values of each sample.</li>

<li><strong>gene_name</strong>: Gene name.</li>

<li><strong>gene_chr</strong>: The name of the chromosome where the gene is located.</li>
<li><strong>gene_start</strong>: The starting position of the chromosome where the gene is located.</li>
<li><strong>gene_end</strong>: The end position of the chromosome where the gene is located.</li>
<li><strong>gene_strand</strong>: The positive and negative strand information of the chromosome where the gene is located.</li>
<li><strong>gene_length</strong>: <span style="text-transform: capitalize;">gene</span> length, the sum of genes from the beginning to the end of all non-overlapping exon regions in chromosomes.</li>
<li><strong>gene_biotype</strong>: <span style="text-transform: capitalize;">gene</span> type,such as coding protein genes,long non-coding genes,etc.</li>
<li><strong>gene_description</strong>: <span style="text-transform: capitalize;">gene</span> description.</li>

<li><strong>tf_family</strong>: The tf_family annotation of novel gene.</li>

</ul>
</div>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/3.Quant/1.Count/gene_count.xls" title="点击打开" >Quant/Count/gene_count.xls</a> </p>


<br />

	<h4>4.3.2 Distribution of <span style="text-transform: capitalize;">gene</span> Expression Levels</h4>


	<p class="paragraph">To compare gene expression levels under different conditions, the distribution of gene expression levels and FPKM(Bray el at., 2014) among different samples are displayed by boxplots. For biological replicates, the final FPKM will be the mean value.  
</p>
	<p class="center">

		<img class="normal1" src="src/files/Quant/Distribution/boxplot.png" />

	</p>



	<p class="name_fig">Figure 4.7 Sample gene expression distribution box plot</p>

	<p class="premark center">X axis represents the name of the sample, Y axis indicates the log2(FPKM+1),

		parameters of box plots are indicated, including maximum, upper quartile, mid-value, lower quartile and minimum.

	</p>
    <p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/3.Quant/3.Distribution/boxplot.svg" title="点击打开" >Quant/3.Distribution/boxplot.svg</a> </p>


	<br />




<br />

	<h4>4.3.3 Correlation Analysis </h4>
	<p class="paragraph">Biological replicates are necessary for any biological experiment including RNA-seq technology. Correlation of the gene expression levels between samples plays an important role in verifying reliability and sample selection, which can not only demonstrate the repeatability of the experiment but estimate the differential gene expression analysis.</p>
	<p class="paragraph">The closer the correlation coefficient is to 1, the higher similarity the samples have. Encode suggests that the square of the Pearson correlation coefficient should be greater than 0.92 (under ideal experiment conditions)(ENCODE Project Consortium, 2004) and the R2 should be greater than 0.8.</p>

	<p class="paragraph">According to all gene expression levels (RPKM or FPKM) of each sample, the correlation coefficient of samples between groups is calculated and drawn as heat maps. It is intuitive to show sample differences and repeat cases between groups. The higher the correlation coefficient of the sample is, the closer the expression pattern is. The correlation coefficient matrix is shown in the following figure. </p>
	<br/>
	

	<p class="center">

		<img class="normal1" src="src/files/Quant/Correlation/correlation.png" />

	</p>



	<p class="name_fig">Figure 4.8 Inter-sample correlation heat map</p>

	<p class="premark center">R<sup>2</sup>: Square of Pearson correlation coefficient(R)</p>

</p>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/3.Quant/2.Correlation/correlation.svg" title="点击打开" >Quant/2.Correlation/correlation.svg</a> </p>

<br />




<br />

<h4>4.3.5 Coexpression Venn Diagram</h4>

<p class="paragraph">The coexpression Venn diagram presents the number of genes that are uniquely expressed within each group/sample, with the overlapping regions showing the number of genes that are co-expressed in two or more groups/samples.</p> 

<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure3_4_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure3_4_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure3_4_pane_Venn1" role="tabpanel" aria-labelledby="gallery_figure3_4_list_Venn1" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Quant/coExpression_venn/ncvsH2_inhit.venn.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure3_4_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure3_4_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure3_4_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure3_4_list_Venn1" data-toggle="list" href="#gallery_figure3_4_pane_Venn1" role="tab" aria-controls="Venn1">Venn1</a>

</div>
</div>
</div>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/3.Quant/4.coExpression_venn" title="点击打开" >Quant/4.coExpression_venn</a> </p>

<br />

<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: How do you deal with reads (reads multi mapped) mapped to multiple locations in the genome during the quantification analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Reads mapped to multiple locations in the genome are not possible to be determined their corresponding genes. Therefore, these reads are directly filtered out during the quantification analysis.</td></tr>

<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: For two genes with overlapping regions, how are the reads in the overlapping region assigned during the quantification analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Same as the previous answer.</td></tr>


<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: How is the information of family annotation of a gene obtained in the quantification analysis table?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: On the one hand, it is annotated by the tf database (AnimalTFDB/PlantTFDB) (for the species already included in the database). On the other hand, it is predicted by protein domain databases such as Pfam/SUPERFAMILY.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: How is FPKM calculated?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: The calculation of FPKM(expected number of Fragments Per Kilobase of transcript sequence per Millions base pairs sequenced) takes into account the influence of sequencing depth and gene length on the count of fragments, and it is a commonly used method for estimating gene expression levels (Trapnell, Cole, et al., 2010).
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the threshold for gene expression levels? Why is this threshold set?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Gene expression is generally considered to be greater than 1 for FPKM, a threshold recommended by mainstream journals.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the significance of the correlation between samples? How is it calculated?
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: The sample correlation represents the similarity among samples. The sample correlation could help us examine the similarity at the gene expression level. Higher correlation means higher similarity and less number of differential express genes. Generally, the correlation between biological replicates should be higher than the correlation of samples with different sources. There are three different calculation methods: A. Pearson correlation; B. Spearman rank correlation; C. Kendall's τ. We uses R language to calculate the Pearson correlation coefficient.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the pca?</td></tr>

<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Principal Component Analysis (PCA) is a multivariate dimensionality reduction analysis method. The key of PCA is to reduce the dimension of the data under the premise of keeping the data as much as possible. Briefly, This method will ignore the less relevant variables to describe the relations between different samples. A dataset is a group of points in a multidimensional space and PCA method will move all the points to a new coordinate system without changing their relative spatial position, making their projections have the largest variances in the new coordinate. In this new coordinate system, the axis with the largest variance of projection is PC1 and the second largest one is PC2.</td></tr>
</table>
</div>
<br/>







<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
    function hiddenShow1(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hidden Q&A";
      };
      return 0;
    }
	
</script>

<!----------------------------------------- 差异表达基因分析 ----------------------------------------->




    <h3>4.4 Differential <span style="text-transform: capitalize;">gene</span> Expression Analysis</h3>




<p class="paragraph">After the gene expression is quantified, statistical analysis of the expression data is required to screen the genes whose expression levels are significantly different in different conditions. The differential analysis is mainly divided into three steps.

</p>
<ul style="font-size: 14px">
<li>First, the raw readcount is normalized, mainly to correct the sequencing depth;</li>
<li>Next, the statistical model is used to calculate the hypothesis test's probability (pvalue);</li>
<li>Finally, multiple hypothesis test corrections are used to obtain FDR values (false discovery rate)(Anders et al., 2010).</li>
</ul>
<p class="paragraph">For different experimental conditions, we selected appropriate software for gene expression differential analysis, as shown in the following table.</p>
<p class="name center">Table 4.5 Software for Differential Analysis and Differential <span style="text-transform: capitalize;">gene</span> Screening Criteria</p>
<table >
<colgroup>
<col width="12%"></col>
<col width="19%"></col>
<col width="15%"></col>
<col width="15%"></col>
<col width="16%"></col>
<col width="20%"></col>
</colgroup>
<thead>
<tr class="header">
<th>Type</th>
<th class="left" align="center">Software</th>
<th class="left" align="center">Normalized Method</th>
<th class="left" align="center">pvalue Calculation Model</th>
<th class="left" align="center">FDR Calculation Method</th>
<th class="left" align="left">Differential <span style="text-transform: capitalize;">gene</span> Screening Threshold</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td class="mid">Biological Replicates</td>
<td class="mid" align="center">DESeq2(Anders et al., 2014)</td>
<td class="mid" align="center">DESeq</td>
<td class="mid" align="center">The Negative Binomial Distribution</td>
<td class="mid" align="center">BH</td>
<td class="mid" align="left">|log2(FoldChange)| &gt;= 1 &amp; padj &lt;= 0.05</td>
</tr>
<tr class="even">
<td class="left">No Biological Replicates</td>
<td class="left" align="center">edgeR(Robinson et al., 2010)</td>
<td class="left" align="center">TMM</td>
<td class="left" align="center">The Negative Binomial Distribution</td>
<td class="left" align="center">BH</td>
<td class="left" align="left">|log2(FoldChange)| &gt;= 1 &amp; padj &lt;= 0.05</td>
</tr>
</tbody>
</table>
<br/>
<p class="paragraph">If the number of different genes screened according to the above threshold is too small (less than 100), there will likely be no significant results in the subsequent functional enrichment analysis. Therefore, we will appropriately reduce the threshold for screening different genes according to the specific conditions of the project. If the project experiment only focuses on the expression of a few genes (such as gene knockout), please ignore the enrichment results, and filter the genes of interest from the differential analysis table below.</p>
<p class="paragraph">In general, if a gene differs more than twice as much in expression in both sets of samples, we believe that such genes are differentially expressed. In order to judge whether the difference in expression between two samples is due to various errors or essential differences, we need to make a hypothesis test on the expression data of all genes in these two samples. The transcriptome analysis is performed on thousands of genes, which leads to the accumulation of false positives. The more the number of genes, the higher the cumulative degree of false positives in the hypothesis test, so the introduction of padj to the hypothesis test P-value is calibrated to control the proportion of false positives(Young et al., 2010).</p>
<p class="paragraph">The screening criteria for differential genes are very important. The standard we give |log2(FoldChange)| >= 1 & padj<= 0.05 is a commonly used empirical value in actual projects. It can be flexibly selected according to the situation. For example, the difference multiple can be 1.5 times, or 3 times, padj include 0.01, 0.05, 0.1, etc. If the number of differential genes screened according to the above criteria is too small, it is likely that there will be no significant results in the subsequent functional enrichment analysis. If the project experiment only focuses on the expression of a few genes (such as gene knockout), please ignore the enrichment results, and filter the genes of interest from the differential analysis table below. Conversely, if the number of differential genes obtained is too large, which is not conducive to the screening of subsequent target genes, screening can be performed using stricter threshold criteria at this time, and screening can be performed using stricter threshold criteria.</p>

<h4> 4.4.1 Result of Differential Expression Analysis</h4>


<p class="paragraph">The differential significance analysis for each compare group is shown in the table below. The table shows the first 5 rows of the differential significance results for all the first compare group.</p>

<p class="name center">Table 4.6 Differential <span style="text-transform: capitalize;">gene</span> List Partial Results Presentation</p>

<p class="center">

    <div id="tb" style="overflow-x: auto">

        <table class="gy">

            <thead>

                <tr>

                    <th>gene_id</th><th>nc</th><th>H2_inhit</th><th>log2FoldChange</th><th>pvalue</th><th>padj</th><th>gene_name</th><th>gene_chr</th><th>gene_start</th><th>gene_end</th><th>gene_strand</th><th>gene_length</th><th>gene_biotype</th><th>gene_description</th><th>tf_family</th>

                </tr>

            </thead>

            <tbody>

                

                <tr><td>ENSMUSG00000024653</td><td>551.042005486399</td><td>1.38777878078145e-17</td><td>12.2206858376484</td><td>1.68386501138593e-30</td><td>2.67313570557516e-26</td><td>Scgb1a1</td><td>19</td><td>9083636</td><td>9087958</td><td>-</td><td>476</td><td>protein_coding</td><td>secretoglobin, family 1A, member 1 (uteroglobin) [Source:MGI Symbol;Acc:MGI:98919]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000024887</td><td>1.38777878078145e-17</td><td>267.790272164895</td><td>-11.1802059749767</td><td>2.43403973118543e-20</td><td>1.93201903662843e-16</td><td>Asah2</td><td>19</td><td>31984654</td><td>32061469</td><td>-</td><td>4682</td><td>protein_coding</td><td>N-acylsphingosine amidohydrolase 2 [Source:MGI Symbol;Acc:MGI:1859310]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000076609</td><td>166.239664138629</td><td>0.637279318344459</td><td>7.73005366792263</td><td>9.84882990336634e-19</td><td>4.17589988919585e-15</td><td>Igkc</td><td>6</td><td>70726435</td><td>70726966</td><td>+</td><td>532</td><td>IG_C_gene</td><td>immunoglobulin kappa constant [Source:MGI Symbol;Acc:MGI:96495]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000031595</td><td>1.38777878078145e-17</td><td>203.006255836852</td><td>-10.7808734271615</td><td>1.05219524767139e-18</td><td>4.17589988919585e-15</td><td>Pdgfrl</td><td>8</td><td>40926212</td><td>40990785</td><td>+</td><td>3126</td><td>protein_coding</td><td>platelet-derived growth factor receptor-like [Source:MGI Symbol;Acc:MGI:1916047]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000021408</td><td>1.38777878078145e-17</td><td>181.634202018198</td><td>-10.6205052210886</td><td>9.69011044845892e-18</td><td>3.07661006738571e-14</td><td>Ripk1</td><td>13</td><td>34002363</td><td>34037147</td><td>+</td><td>8344</td><td>protein_coding</td><td>receptor (TNFRSF)-interacting serine-threonine kinase 1 [Source:MGI Symbol;Acc:MGI:108212]</td><td>-</td></tr>

                

            </tbody>

        </table>

    </div>

</p>


<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>gene_id</strong>: <span style="text-transform: capitalize;">gene</span> id.</li>
<li><strong>sample</strong>: The readcount values of each sample after normalization.</li>

<li><strong>group</strong>: The standardized average of readcount for each group of samples.</li>

<li><strong>log2FoldChange</strong>: The ratio of geneexpression level between the treatment group and the control group was processed by the shrinkage model of the differential analysis software, and finally the logarithm was taken with 2 as the base.</li>
<li><strong>pvalue</strong>: Pvalue in WaldTest.</li>
<li><strong>padj</strong>: The corrected pvalue of multiple hypothesis test.</li>

<li><strong>gene_name</strong>: Gene name.</li>

<li><strong>gene_chr</strong>: The name of the chromosome where the gene is located.</li>
<li><strong>gene_start</strong>: The starting position of the chromosome where the gene is located.</li>
<li><strong>gene_end</strong>: The end position of the chromosome where the gene is located.</li>
<li><strong>gene_strand</strong>: The positive and negative strand information of the chromosome where the gene is located.</li>
<li><strong>gene_length</strong>: <span style="text-transform: capitalize;">gene</span> length, the sum of genes from the beginning to the end of all non-overlapping exon regions in chromosomes.</li>
<li><strong>gene_biotype</strong>: <span style="text-transform: capitalize;">gene</span> type,such as coding protein genes,long non-coding genes,etc.</li>
<li><strong>gene_description</strong>: <span style="text-transform: capitalize;">gene</span> description.</li>

<li><strong>tf_family</strong>: The tf_family annotation of novel gene.</li>

</ul>
</div>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/4.Differential/1.deglist" title="点击打开" >Differential/1.deglist</a> </p>


<p class="paragraph">The significance analysis result of each gene in all compare groups was shown in the following table.</p>

<p class="name center">Table 4.7 Differences in the significance of each gene in all comparison combinations Show</p>

<p class="center">

    <div id="tb" style="overflow-x: auto">

        <table class="gy">

            <thead>

                <tr>

                    <th>gene_id</th><th>nc_count</th><th>H2_inhit_count</th><th>nc_fpkm</th><th>H2_inhit_fpkm</th><th>ncvsH2_inhit_nc</th><th>ncvsH2_inhit_H2_inhit</th><th>ncvsH2_inhit_log2FoldChange</th><th>ncvsH2_inhit_pvalue</th><th>ncvsH2_inhit_padj</th><th>gene_name</th><th>gene_chr</th><th>gene_start</th><th>gene_end</th><th>gene_strand</th><th>gene_length</th><th>gene_biotype</th><th>gene_description</th><th>tf_family</th>

                </tr>

            </thead>

            <tbody>

                

                <tr><td>ENSMUSG00000028180</td><td>0</td><td>74</td><td>0</td><td>16.2156042126839</td><td>1.38777878078145e-17</td><td>49.3942730304474</td><td>-8.7449456466636</td><td>7.10124216492149e-09</td><td>1.91721461510423e-07</td><td>Zranb2</td><td>3</td><td>157534160</td><td>157548410</td><td>+</td><td>6759</td><td>protein_coding</td><td>zinc finger, RAN-binding domain containing 2 [Source:MGI Symbol;Acc:MGI:1858211]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000028182</td><td>0</td><td>0</td><td>0</td><td>0</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>Lrriq3</td><td>3</td><td>155093434</td><td>155194280</td><td>+</td><td>4961</td><td>protein_coding</td><td>leucine-rich repeats and IQ motif containing 3 [Source:MGI Symbol;Acc:MGI:1921685]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000028185</td><td>0</td><td>0</td><td>0</td><td>0</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>Dnase2b</td><td>3</td><td>146580985</td><td>146615596</td><td>-</td><td>2790</td><td>protein_coding</td><td>deoxyribonuclease II beta [Source:MGI Symbol;Acc:MGI:1913283]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000028184</td><td>9</td><td>24</td><td>2.11242704024643</td><td>2.50361723288466</td><td>13.5148668880671</td><td>15.9996858788961</td><td>-0.248387758598897</td><td>0.886004738639446</td><td>0.942905088023919</td><td>Adgrl2</td><td>3</td><td>148815583</td><td>148990555</td><td>-</td><td>14198</td><td>protein_coding</td><td>adhesion G protein-coupled receptor L2 [Source:MGI Symbol;Acc:MGI:2139714]</td><td>-</td></tr>

                

                <tr><td>ENSMUSG00000028187</td><td>10</td><td>29</td><td>8.05139167201385</td><td>10.3773492661512</td><td>15.0122186577637</td><td>19.3392157335789</td><td>-0.368849264617921</td><td>0.605806649571441</td><td>0.670654153552763</td><td>Rpf1</td><td>3</td><td>146505956</td><td>146521429</td><td>-</td><td>4139</td><td>protein_coding</td><td>ribosome production factor 1 homolog [Source:MGI Symbol;Acc:MGI:1917535]</td><td>-</td></tr>

                

            </tbody>

        </table>

    </div>

</p>


<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>gene_id</strong>: <span style="text-transform: capitalize;">gene</span> id.</li>
<li><strong>sample_count</strong>: Raw readcount value of each sample.</li>
<li><strong>sample_fpkm</strong>: FPKM value of each sample.</li>
<li><strong>compare_treat</strong>: The average readcount value of treatment group in compare group after normalized.</li>
<li><strong>compare_control</strong>: The average readcount value of control group in compare group after normalized.</li>
<li><strong>compare_log2FoldChange</strong>: The ratio of gene expression level between the treatment group and the control group of a comparison combination was processed by the shrinkage model of the differential analysis software, and finally the logarithm was taken with 2 as the base.</li>
<li><strong>compare_pvalue</strong>: Pvalue in WaldTest of compare group.</li>
<li><strong>compare_padj</strong>: The corrected pvalue of multiple hypothesis test of compare group.</li>

<li><strong>gene_name</strong>: Gene name.</li>

<li><strong>gene_chr</strong>: The name of the chromosome where the gene is located.</li>
<li><strong>gene_start</strong>: The starting position of the chromosome where the gene is located.</li>
<li><strong>gene_end</strong>: The end position of the chromosome where the gene is located.</li>
<li><strong>gene_strand</strong>: The positive and negative strand information of the chromosome where the gene is located.</li>
<li><strong>gene_length</strong>: <span style="text-transform: capitalize;">gene</span> length, the sum of genes from the beginning to the end of all non-overlapping exon regions in chromosomes.</li>
<li><strong>gene_biotype</strong>: <span style="text-transform: capitalize;">gene</span> type,such as coding protein genes,long non-coding genes,etc.</li>
<li><strong>gene_description</strong>: <span style="text-transform: capitalize;">gene</span> description.</li>

<li><strong>tf_family</strong>: The tf_family annotation of novel gene.</li>

</ul>
</div>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/0.SupFile/all_compare.xls" title="点击打开" >0.SupFile/all_compare.xls</a> </p>


<br />

<h4 > 4.4.2 Differential <span style="text-transform: capitalize;">gene</span> Statistics</h4>

<p class="paragraph">The statistics of the number of differential genes (including up-regulation and down-regulation) for each compare group and the threshold for screening are shown in the table below.</p>
<br/>
<p class="name center">Table 4.8 Differential <span style="text-transform: capitalize;">gene</span> Statistics Results</p>

<p class="center">

    <div id="tb" style="overflow-x: auto">

        <table class="gy" data-pagination="true" data-toggle="table" data-page-size="10">

            <thead>

                <tr>

                    <th>compare</th><th>all</th><th>up</th><th>down</th><th>threshold</th>

                </tr>

            </thead>

            <tbody>

                

                <tr><td>ncvsH2_inhit</td><td>9185</td><td>2153</td><td>7032</td><td>edgeR padj&lt;=0.05 |log2FoldChange|&gt;=1.0</td></tr>

                

            </tbody>

        </table>

    </div>

</p>


<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>compare</strong>: Compare group name.</li>
<li><strong>all</strong>: The total number of differential genes in the compare group.</li>
<li><strong>up</strong>:The up-regulation number of differential genes in the compare group.</li>
<li><strong>down</strong>: The down-regulation number of differential genes in the compare group.</li>
<li><strong>threshold</strong>: The compare group software and thresholds for differential gene screening.</li>
</ul>
</div>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/4.Differential/1.deglist/diff_stat.xls" title="点击打开" >Differential/1.deglist/diff_stat.xls</a> </p>


<p class="paragraph">The number of differential genes (including up-regulation and down-regulation) for each comparison combination is shown in a histogram:</p>

<p class="center">

                <img class="normal1" src="src/files/Differential/DEGlist/diff_stat.png" />

        </p>

<p class="name_fig">Figure 4.9 Difference Comparison Combine Differential <span style="text-transform: capitalize;">gene</span> Number Statistics Histogram</p>
<p class="premark center">
Note: Blue and gray represent the differential genes for up-regulation and down-regulation, respectively, and the numbers on the columns indicate the number of differential genes</p>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/4.Differential/1.deglist" title="点击打开" >Differential/1.deglist/</a> </p>

<p class="paragraph">Volcano plots can be used to infer the overall distribution of differentially expressed genes. In the figure, The x-axis shows the fold change in gene expression between different samples, and the y-axis shows the statistical significance of the differences. Red dots represent up-regulation genes and green dots represent down-regulation genes.</p>
<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure4_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure4_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure4_1_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure4_1_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Differential/Volcano/ncvsH2_inhit.volcano.png" /></div>

</p>
</div>
</div>
<div class="col-1">
<div id="gallery_figure4_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure4_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure4_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure4_1_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure4_1_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>

<p class="name_fig">Figure 4.10 Differential <span style="text-transform: capitalize;">gene</span> Volcano Map</p>

<p class="premark center">The abscissa in the figure is log2FoldChange, and the ordinate is -log10padj or -log10pvalue, the blue dashed line indicates the threshold line for differential gene screening criteria</p>
</p><br />
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/4.Differential/1.deglist" title="点击打开" >Differential/1.deglist/{comparative combination}/_volcano.png</a> </p>


<br />

<h4> 4.4.3 Cluster Analysis</h4>



<p class="paragraph">All the differentially expressed genes in the comparison group were pooled as the differential gene set. For more than two groups of experiments, cluster analysis can be carried out on different gene sets and genes with similar expression patterns can be clustered together. We used the mainstream hierarchical clustering to cluster the fpkm values of genes, and homogenized the row (Z-score). The genes or samples with similar expression patterns in the heat map will be gathered together. The color in each grid reflects not the gene expression value, but the value obtained after homogenizing the expression data rows (generally between - 2 and 2). Therefore, the colors in the heat map can only be compared horizontally (the expression of the same gene in different samples), but not vertically (the same sample). There are not only inter group clustering, but also inter sample clustering. The final report shows the clustering among samples. 
</p>

<br />


<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure4_3_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure4_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure4_3_pane_heatmap" role="tabpanel" aria-labelledby="gallery_figure4_3_list_heatmap" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Differential/Cluster/heatCluster.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure4_3_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure4_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure4_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure4_3_list_heatmap" data-toggle="list" href="#gallery_figure4_3_pane_heatmap" role="tab" aria-controls="heatmap">heatmap</a>

</div>
</div>
</div>
    </p>

    <p class="name_fig">Figure 4.11 differential expression gene clustering heatmap</p>

    <p class="premark center">The overall results of FPKM cluster analysis, clustered using the log2(FPKM+1) value. Red color indicates genes with high expression levels, and blue color indicates genes with low expression levels. The color ranging from red to blue indicates that log2(FPKM+1) values where from large to small. 

    </p>
    
    <br/>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/4.Differential/2.cluster/heatmap.png" title="点击打开" >Differential/2.cluster/heatmap.png</a> </p>
    <br />


<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: Can I use FPKM for differential analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Most of the differential analysis software (DESeq, DESeq2 and edgeR) use the raw read counts as the input file and the negative binomial distribution model to estimate the probability of gene differential expression between samples. The software itself does some corrections on the read count (mainly the depth of sequencing), while FPKM is the corrected expression value, so it is unreasonable to use FPKM for the differential analysis to make double corrections.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: Why do multiple hypothesis tests calculate padj values instead of using pvalues directly to screen for differential genes?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: There is no problem with the usage of pvalue for a single hypothesis test, but in the process of differential analysis, we have to perform a hypothesis test on each gene. There are often tens of thousands of genes in a species, and tens of thousands of hypothesis tests are performed, which leads to greatly increasing false positives. Assuming a pvalue of 0.05 (only five of the one hundred differential genes are false positives), this accuracy is sufficient for a gene that is hypothesized, but for the entire tens of thousands of genes, such accuracy is far from good. For example, for every 10,000 genes tested, around 500 genes are false positive. In order to properly control false positive rates, it is necessary to introduce a stricter indicator which is the corrected pvalue. Of course, if there are too few differential genes, you can also use pvalues instead. As long as it has biological significance and can be verified by experiments.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the maximum threshold that can be set for differential gene screening? Is it appropriate to lower the threshold properly?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: In general, the presetting screening threshold in higher-level article is more stringent, and in some articles, the differential gene screening thresholds are appropriately lower. As in some articles without biological replicates, only padj is used as a differential gene screening threshold, regardless of log2foldchange. While some other articles use pvalue as a screening threshold for differential genes.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What are the steps in cluster analysis?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Logarithmic transformation of gene expression values (FPKM expression matrices) of all samples is performed to approximate the data to an ideal normal distribution (clustering analysis rely on an early assumption that the data is subject to an idealized statistical distribution, usually Normal Distribution), then the software calculates the distance (Euclidean distance) of all the transformed data points, and finally classifies N objects into k groups by the complete clustering method in hierarchical clustering. The objects in each group are mutually similar. The hierarchical clustering algorithm is mainly divided into three steps, as shown below.<br/>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;At the initial moment, all points are themselves a cluster.<br/>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Find the nearest two clusters to form a cluster. The distance between the two clusters refers to the distance between the two nearest points in the cluster.<br/>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Repeat the second step until all points are clustered into individual cluster.<br/></td></tr>

<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: How is normalization calculated in cluster analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: The values of each row of data in the expression matrix are subtracted from the mean of each row of data, and divided by the standard deviation of each row of data. The range of values after normalizationare generally in the interval [-2,2].
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: How is the transcription factor predicted?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Identification of animal transcription factors uses the animal transcription factor database(animalTFDB 3.0), and identification of plant transcription factors uses the plant transcription factor database(PlantTFDB PlantTFDB 4.0). For the species included in the database, if Ensembl geneid is used, the transcription factor is directly screened; the Ensembl geneid is subjected to SUPERFAMILY and Pfam annotation by interproscan software, and the ID of SUPERFAMILY and Pfam of each gene are obtained. Then, the annotations information of the transcription factor family corresponding to each SUPERFAMILY and Pfam in the DBD (Transcription factor prediction database) are used to predict transcription factors. The fungal transcription factor is predicted by the SUPERFAMILY and Pfam annotations by interproscan software, and the ID of SUPERFAMILY and Pfam of each gene are obtained, and then each of the already annotated databases in the DBD(Transcription factor prediction database) are utilized. The information of the transcription factor family corresponding to SUPERFAMILY and Pfam is predicted.
</td></tr>
</table>
</div>
<br/>
﻿<!----------------------------------------- 差异表达基因功能分析 ----------------------------------------->



	<h3> 4.5 Functional Analysis</h3>





<p class="paragraph">

	Through the enrichment analysis of the differential expressed genes, we can find out which biological functions or pathways are significantly associated with differentially expressed genes. We uses the clusterProfiler (Yu G, 2012) software for enrichment analysis, including GO Enrichment, DO Enrichment, KEGG and Reactome database Enrichment etc,.

</p>

<p class="center"><img src="src/pdf_images/enrichment.png" width="400" height="300" ></p>

<p class="name_fig">Figure 4.12 Gene Enrichment Analysis Schematic </p>




<!---------------------------GO富集分析---------------------->



<h4> 4.5.1 GO Enrichment Analysis </h4>



<p class="paragraph"></p>

<p class="paragraph">GO is the abbreviation of Gene Ontology (<a href="http://www.geneontology.org/" class="uri">http://www.geneontology.org/</a>), which is a major bioinformatics classification system to unify the presentation of gene properties across all species. It includes three main branches: cellular component, molecular function and biological process. GO terms with padj &lt 0.05 are significant enrichment.  </b>  </p>
 <br/>
 
 <p class="name center">Table 4.9 Partially displayed results of GO enrichment analysis of differential genes</p>

<p class="center">

	<div id="tb" style="overflow-x: auto">

		<table class="gy">

			<thead>

				<tr>

					<th>Category</th><th>GOID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>Count</th><th>Up</th><th>Up_Gene_names</th><th>Down</th><th>Down_Gene_names</th>

				</tr>

			</thead>

			<tbody>

				

				<tr><td>BP</td><td>GO:0021987</td><td>cerebral cortex development</td><td>59/6541</td><td>78/11172</td><td>0.00117606267779106</td><td>0.999989437129736</td><td>ENSMUSG00000025810/ENSMUSG00000030852/ENSMUSG00000013236/ENSMUSG00000030314/ENSMUSG00000069171/ENSMUSG00000030201/ENSMUSG00000042453/ENSMUSG00000048799/ENSMUSG00000003873/ENSMUSG00000020122/ENSMUSG00000037313/ENSMUSG00000018736/ENSMUSG00000035764/ENSMUSG00000048895/ENSMUSG00000028883/ENSMUSG00000028519/ENSMUSG00000047921/ENSMUSG00000031565/ENSMUSG00000055254/ENSMUSG00000063659/ENSMUSG00000022883/ENSMUSG00000020458/ENSMUSG00000014030/ENSMUSG00000032030/ENSMUSG00000031785/ENSMUSG00000000247/ENSMUSG00000095139/ENSMUSG00000021318/ENSMUSG00000041498/ENSMUSG00000051234/ENSMUSG00000026812/ENSMUSG00000006464/ENSMUSG00000022812/ENSMUSG00000052336/ENSMUSG00000031558/ENSMUSG00000024304/ENSMUSG00000038119/ENSMUSG00000019803/ENSMUSG00000002900/ENSMUSG00000045515/ENSMUSG00000068036/ENSMUSG00000021109/ENSMUSG00000005069/ENSMUSG00000042524/ENSMUSG00000025969/ENSMUSG00000026043/ENSMUSG00000028640/ENSMUSG00000019969/ENSMUSG00000001761/ENSMUSG00000073043/ENSMUSG00000021546/ENSMUSG00000038485/ENSMUSG00000026883/ENSMUSG00000027239/ENSMUSG00000046743/ENSMUSG00000047143/ENSMUSG00000063450/ENSMUSG00000022678/ENSMUSG00000020849</td><td>Nrp1/Tacc2/Ptprs/Atg7/Nr2f1/Lrp6/Reln/Cep120/Bax/Egfr/Tacc3/Ndel1/Fbxo45/Cdk5r1/Sema3a/Dab1/Trappc9/Fgfr1/Ntrk2/Zbtb18/Robo1/Rtn4/Pax5/Cul5/Adgrg1/Lhx2/Pou3f2/Gli3/Kif14/Rnf7/Tsc1/Bbs1/Gsk3b/Cx3cr1/Slit2/Cdh2/Cdon/Nr2e1/Lamb1/Pou3f3/Afdn/Hif1a/Pex5/Sun2/Nrp2/Col3a1/Tfap2c/Psen1/Smo/Atoh1/Hnrnpk/Socs7/Dab2ip/Mdk/Fat4/Dmrta2/Syne2/Nde1/Ywhae</td><td>59</td><td>10</td><td>ENSMUSG00000063659/ENSMUSG00000014030/ENSMUSG00000031785/ENSMUSG00000006464/ENSMUSG00000052336/ENSMUSG00000005069/ENSMUSG00000042524/ENSMUSG00000073043/ENSMUSG00000022678/ENSMUSG00000020849</td><td>49</td><td>ENSMUSG00000025810/ENSMUSG00000030852/ENSMUSG00000013236/ENSMUSG00000030314/ENSMUSG00000069171/ENSMUSG00000030201/ENSMUSG00000042453/ENSMUSG00000048799/ENSMUSG00000003873/ENSMUSG00000020122/ENSMUSG00000037313/ENSMUSG00000018736/ENSMUSG00000035764/ENSMUSG00000048895/ENSMUSG00000028883/ENSMUSG00000028519/ENSMUSG00000047921/ENSMUSG00000031565/ENSMUSG00000055254/ENSMUSG00000022883/ENSMUSG00000020458/ENSMUSG00000032030/ENSMUSG00000000247/ENSMUSG00000095139/ENSMUSG00000021318/ENSMUSG00000041498/ENSMUSG00000051234/ENSMUSG00000026812/ENSMUSG00000022812/ENSMUSG00000031558/ENSMUSG00000024304/ENSMUSG00000038119/ENSMUSG00000019803/ENSMUSG00000002900/ENSMUSG00000045515/ENSMUSG00000068036/ENSMUSG00000021109/ENSMUSG00000025969/ENSMUSG00000026043/ENSMUSG00000028640/ENSMUSG00000019969/ENSMUSG00000001761/ENSMUSG00000021546/ENSMUSG00000038485/ENSMUSG00000026883/ENSMUSG00000027239/ENSMUSG00000046743/ENSMUSG00000047143/ENSMUSG00000063450
</td></tr>

				

				<tr><td>BP</td><td>GO:0030811</td><td>regulation of nucleotide catabolic process</td><td>21/6541</td><td>24/11172</td><td>0.00228241833300589</td><td>0.999989437129736</td><td>ENSMUSG00000004040/ENSMUSG00000031987/ENSMUSG00000006457/ENSMUSG00000011752/ENSMUSG00000028944/ENSMUSG00000022383/ENSMUSG00000059552/ENSMUSG00000033307/ENSMUSG00000021236/ENSMUSG00000020053/ENSMUSG00000034748/ENSMUSG00000020108/ENSMUSG00000005534/ENSMUSG00000020372/ENSMUSG00000034160/ENSMUSG00000021109/ENSMUSG00000018501/ENSMUSG00000003541/ENSMUSG00000006362/ENSMUSG00000027613/ENSMUSG00000027282</td><td>Stat3/Egln1/Actn3/Pgam1/Prkag2/Ppara/Trp53/Mif/Entpd5/Igf1/Sirt6/Ddit4/Insr/Rack1/Ogt/Hif1a/Ncor1/Ier3/Cbfa2t3/Eif6/Mtch2</td><td>21</td><td>9</td><td>ENSMUSG00000031987/ENSMUSG00000011752/ENSMUSG00000021236/ENSMUSG00000005534/ENSMUSG00000020372/ENSMUSG00000034160/ENSMUSG00000003541/ENSMUSG00000006362/ENSMUSG00000027282</td><td>12</td><td>ENSMUSG00000004040/ENSMUSG00000006457/ENSMUSG00000028944/ENSMUSG00000022383/ENSMUSG00000059552/ENSMUSG00000033307/ENSMUSG00000020053/ENSMUSG00000034748/ENSMUSG00000020108/ENSMUSG00000021109/ENSMUSG00000018501/ENSMUSG00000027613
</td></tr>

				

				<tr><td>BP</td><td>GO:0007029</td><td>endoplasmic reticulum organization</td><td>31/6541</td><td>39/11172</td><td>0.00490422823801347</td><td>0.999989437129736</td><td>ENSMUSG00000021484/ENSMUSG00000021669/ENSMUSG00000019787/ENSMUSG00000030126/ENSMUSG00000031668/ENSMUSG00000026924/ENSMUSG00000015290/ENSMUSG00000062234/ENSMUSG00000035325/ENSMUSG00000024759/ENSMUSG00000073639/ENSMUSG00000020458/ENSMUSG00000002395/ENSMUSG00000030082/ENSMUSG00000021066/ENSMUSG00000041779/ENSMUSG00000036104/ENSMUSG00000059811/ENSMUSG00000029125/ENSMUSG00000030654/ENSMUSG00000018171/ENSMUSG00000005069/ENSMUSG00000007122/ENSMUSG00000001143/ENSMUSG00000009207/ENSMUSG00000018820/ENSMUSG00000024758/ENSMUSG00000024392/ENSMUSG00000037720/ENSMUSG00000054455/ENSMUSG00000051256</td><td>Lman2/Col4a3bp/Trdn/Tmcc1/Eif2ak3/Sec16a/Ubl4a/Gak/Sec31a/Atl3/Rab18/Rtn4/Use1/Sec61a1/Atl1/Tram2/Rab3gap1/Atl2/Stx18/Arl6ip1/Vmp1/Pex5/Casq1/Lman2l/Lnpk/Zfyve27/Rtn3/Bag6/Tmem33/Vapb/Jagn1</td><td>31</td><td>11</td><td>ENSMUSG00000021484/ENSMUSG00000026924/ENSMUSG00000015290/ENSMUSG00000035325/ENSMUSG00000021066/ENSMUSG00000059811/ENSMUSG00000030654/ENSMUSG00000005069/ENSMUSG00000018820/ENSMUSG00000024392/ENSMUSG00000054455</td><td>20</td><td>ENSMUSG00000021669/ENSMUSG00000019787/ENSMUSG00000030126/ENSMUSG00000031668/ENSMUSG00000062234/ENSMUSG00000024759/ENSMUSG00000073639/ENSMUSG00000020458/ENSMUSG00000002395/ENSMUSG00000030082/ENSMUSG00000041779/ENSMUSG00000036104/ENSMUSG00000029125/ENSMUSG00000018171/ENSMUSG00000007122/ENSMUSG00000001143/ENSMUSG00000009207/ENSMUSG00000024758/ENSMUSG00000037720/ENSMUSG00000051256
</td></tr>

				

				<tr><td>BP</td><td>GO:0006110</td><td>regulation of glycolytic process</td><td>19/6541</td><td>22/11172</td><td>0.0051783503672955</td><td>0.999989437129736</td><td>ENSMUSG00000004040/ENSMUSG00000006457/ENSMUSG00000011752/ENSMUSG00000028944/ENSMUSG00000022383/ENSMUSG00000059552/ENSMUSG00000033307/ENSMUSG00000021236/ENSMUSG00000020053/ENSMUSG00000034748/ENSMUSG00000020108/ENSMUSG00000005534/ENSMUSG00000034160/ENSMUSG00000021109/ENSMUSG00000018501/ENSMUSG00000003541/ENSMUSG00000006362/ENSMUSG00000027613/ENSMUSG00000027282</td><td>Stat3/Actn3/Pgam1/Prkag2/Ppara/Trp53/Mif/Entpd5/Igf1/Sirt6/Ddit4/Insr/Ogt/Hif1a/Ncor1/Ier3/Cbfa2t3/Eif6/Mtch2</td><td>19</td><td>7</td><td>ENSMUSG00000011752/ENSMUSG00000021236/ENSMUSG00000005534/ENSMUSG00000034160/ENSMUSG00000003541/ENSMUSG00000006362/ENSMUSG00000027282</td><td>12</td><td>ENSMUSG00000004040/ENSMUSG00000006457/ENSMUSG00000028944/ENSMUSG00000022383/ENSMUSG00000059552/ENSMUSG00000033307/ENSMUSG00000020053/ENSMUSG00000034748/ENSMUSG00000020108/ENSMUSG00000021109/ENSMUSG00000018501/ENSMUSG00000027613
</td></tr>

				

				<tr><td>BP</td><td>GO:2000177</td><td>regulation of neural precursor cell proliferation</td><td>49/6541</td><td>66/11172</td><td>0.00575573859351744</td><td>0.999989437129736</td><td>ENSMUSG00000048756/ENSMUSG00000037815/ENSMUSG00000031328/ENSMUSG00000059552/ENSMUSG00000039834/ENSMUSG00000062234/ENSMUSG00000002603/ENSMUSG00000015053/ENSMUSG00000027398/ENSMUSG00000028949/ENSMUSG00000022322/ENSMUSG00000028134/ENSMUSG00000021936/ENSMUSG00000027070/ENSMUSG00000020053/ENSMUSG00000044167/ENSMUSG00000018698/ENSMUSG00000042228/ENSMUSG00000031785/ENSMUSG00000021994/ENSMUSG00000032554/ENSMUSG00000079259/ENSMUSG00000068748/ENSMUSG00000021318/ENSMUSG00000036940/ENSMUSG00000052336/ENSMUSG00000007989/ENSMUSG00000031099/ENSMUSG00000039844/ENSMUSG00000024304/ENSMUSG00000009073/ENSMUSG00000038119/ENSMUSG00000015149/ENSMUSG00000019803/ENSMUSG00000068036/ENSMUSG00000024256/ENSMUSG00000021109/ENSMUSG00000033006/ENSMUSG00000033031/ENSMUSG00000035529/ENSMUSG00000074227/ENSMUSG00000001761/ENSMUSG00000068523/ENSMUSG00000048482/ENSMUSG00000022129/ENSMUSG00000047143/ENSMUSG00000028017/ENSMUSG00000008855/ENSMUSG00000020644</td><td>Foxo3/Ctnna1/Flna/Trp53/Zfp335/Gak/Tgfb1/Gata2/Il1b/Smarcd3/Shcbp1/Ptbp2/Mapk8/Lrp2/Igf1/Foxo1/Lhx1/Lyn/Adgrg1/Wnt5a/Trf/Trim71/Ptprz1/Gli3/Kdm1a/Cx3cr1/Fzd3/Smarca1/Rapgef1/Cdh2/Nf2/Cdon/Sirt2/Nr2e1/Afdn/Adcyap1/Hif1a/Sox10/Cip2a/Prdm4/Spint2/Smo/Gng5/Bdnf/Dct/Dmrta2/Egf/Hdac5/Id2</td><td>49</td><td>14</td><td>ENSMUSG00000048756/ENSMUSG00000037815/ENSMUSG00000021936/ENSMUSG00000031785/ENSMUSG00000036940/ENSMUSG00000052336/ENSMUSG00000031099/ENSMUSG00000009073/ENSMUSG00000035529/ENSMUSG00000074227/ENSMUSG00000068523/ENSMUSG00000048482/ENSMUSG00000022129/ENSMUSG00000008855</td><td>35</td><td>ENSMUSG00000031328/ENSMUSG00000059552/ENSMUSG00000039834/ENSMUSG00000062234/ENSMUSG00000002603/ENSMUSG00000015053/ENSMUSG00000027398/ENSMUSG00000028949/ENSMUSG00000022322/ENSMUSG00000028134/ENSMUSG00000027070/ENSMUSG00000020053/ENSMUSG00000044167/ENSMUSG00000018698/ENSMUSG00000042228/ENSMUSG00000021994/ENSMUSG00000032554/ENSMUSG00000079259/ENSMUSG00000068748/ENSMUSG00000021318/ENSMUSG00000007989/ENSMUSG00000039844/ENSMUSG00000024304/ENSMUSG00000038119/ENSMUSG00000015149/ENSMUSG00000019803/ENSMUSG00000068036/ENSMUSG00000024256/ENSMUSG00000021109/ENSMUSG00000033006/ENSMUSG00000033031/ENSMUSG00000001761/ENSMUSG00000047143/ENSMUSG00000028017/ENSMUSG00000020644
</td></tr>

				

			</tbody>

		</table>

	</div>

</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<li><strong>Category</strong>: Classification of GO databases, including biological processes(BP), cellular components(CC), molecular function(MF).</li>
<li><strong>GOID</strong>: Unique identification id of Gene Ontology database.</li>
<li><strong>Description</strong>: Function description corresponding to the GO number.</li>
<li><strong>GeneRatio</strong>: Ratio between the number of differentially expressed genes in each GO term and all differentially expressed genes that can be found in GO database.</li>
<li><strong>BgRatio</strong>: In background GO database, the ratio of all genes concerning this GO term to all genes.</li>
<li><strong>pvalue</strong>: Statistics category term; abbreviation for probability value.</li>
<li><strong>padj</strong>: Adjusted p-value. Generally, GO Terms with Corrected_pValue &lt; 0.05 are significant enrichment.</li>
<li><strong>geneID</strong>: Differentially expressed genes in this term.</li>
<li><strong>geneName</strong>: Differentially expressed genes in this term.</li>
<li><strong>Count</strong>: The number of difference gene annotated to GO number.</li>
<li><strong>Up</strong>: Number of up expressed genes concerning this GO term.</li>
<li><strong>Up_Gene_names</strong>: Up expressed gene id concerning this GO term.</li>
<li><strong>Down</strong>: Number of down expressed genes concerning this GO term.</li>
<li><strong>Down_Gene_names</strong>: down expressed gene id concerning this GO term.</li>
</ul>
</div>




<p class="paragraph">In the results of the GO enrichment analysis, the most significant 30 Terms were selected for display. If the results are less than 30, all terms would be drawn, as shown in the following figure. In this figure, the abscissa is GO Term. The ordinate is the significance level of GO Term enrichment. Higher values correspond to higher significance. The different colors represent the three GO subclasses of BP,CC,MF. 

</p>


<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_1_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_1_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.bar.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_1_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_1_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
</p>

<p class="name_fig">Figure 4.13 GO enrichment analysis histogram </p>
<p class="premark center">
Note: The abscissa in the figure is GO Term, and the ordinate is GO Term's level of significance of enrichment, expressed as -log10(padj). Different colors represent different functional categories
</p>

<p class="paragraph">From the GO enrichment analysis result, the most significant 30 Terms were selected for display. If the results are less than 30, all Terms would be drawn according to major categories of biological processes, cell components, molecular functions and categories of up and down expressed genes.
</p>

<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_2_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_2_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.updown.bar.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_2_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_2_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
</p>

<p class="name_fig">The abscissa in the figure is GO Term, and the ordinate is GO Term The level of significance of the set </p>



<p class="paragraph">From the GO enrichment analysis results, the most significant 30 GO Terms were selected for display. If the results are less than 30, all Terms would be drawn. In the figure, the abscissa is the ratio of the number of differential genes linked with the GO Term to the total number of differential genes, and the ordinate is GO Term. The size of a point represents the number of genes annotated to a specific GO Term, and the color from red to purple represents the significant level of the enrichment.</p>

<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_3_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_3_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_3_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.dot.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_3_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_3_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_3_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_3_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
</p>

<p class="name_fig">Figure 4.14 GO Enrichment Analysis Scatter Plot </p>
<p class="premark center">
The abscissa in the graph is the ratio of the differential gene number to the total number of differential genes on the GO Term, and the ordinate is GO Term</p>


<p class="paragraph">(Directed Acyclic Graph，DAG) could visualize the enriched GO Term of differential expression genes and its hierarchy. In this graph, branch means the hierarchical relation, and the function ranges become increasingly specific from the top to bottom. In general, the top 5 results of GO enrichment analysis are chosen as the main nodes (shown by box) in directed acyclic graph, and related GO Term are shown together by hierarchical connections. The enrichment degree is illustrated by color shades, the darker the shades are, the higher the enrichment degree is. In this project, DAG of biological process, molecular function and cellular component were drawn sequentially.
</p>


<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_3_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_3_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_3_2_pane_ncvsH2_inhit_bp" role="tabpanel" aria-labelledby="gallery_figure5_3_2_list_ncvsH2_inhit_bp" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.GObp_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_3_2_pane_ncvsH2_inhit_cc" role="tabpanel" aria-labelledby="gallery_figure5_3_2_list_ncvsH2_inhit_cc" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.GOcc_DAG.png" /></div>

<div class="tab-pane fade" id="gallery_figure5_3_2_pane_ncvsH2_inhit_mf" role="tabpanel" aria-labelledby="gallery_figure5_3_2_list_ncvsH2_inhit_mf" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/GO/ncvsH2_inhit.GOmf_DAG.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_3_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_3_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_3_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_2_list_ncvsH2_inhit_bp" data-toggle="list" href="#gallery_figure5_3_2_pane_ncvsH2_inhit_bp" role="tab" aria-controls="ncvsH2_inhit_bp">ncvsH2_inhit_bp</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_2_list_ncvsH2_inhit_cc" data-toggle="list" href="#gallery_figure5_3_2_pane_ncvsH2_inhit_cc" role="tab" aria-controls="ncvsH2_inhit_cc">ncvsH2_inhit_cc</a>

<a class="list-group-item list-group-item-action" id="gallery_figure5_3_2_list_ncvsH2_inhit_mf" data-toggle="list" href="#gallery_figure5_3_2_pane_ncvsH2_inhit_mf" role="tab" aria-controls="ncvsH2_inhit_mf">ncvsH2_inhit_mf</a>

</div>
</div>
</div>
</p>

<p class="name_fig">Figure 4.15 GO Enrichment Analysis DAG Chart </p>
<p class="premark center">
Each node represents a GO term, and the box represents the enrichment level of TOP5 GOTerms. The depth of the color represents the degree of enrichment, the darker the color is, the higherthe enrichment degree is. Each node shows the name of the term and the Padj of enrichment analysis </p>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/5.Enrichment/1.GO" title="点击打开" >Enrichment/GO</a> </p>


<!---------------------------KEGG富集分析---------------------->



<h4> 4.5.2 KEGG Enrichment Analysis</h4>



<p class="center">



	<p class="paragraph"> The interactions of multiple genes may be involved in certain biological functions. KEGG(Kanehisa et al., 2000) (Kyoto Encyclopedia of Genes and Genomes) is a collection of manually curated databases containing resources on genomic, biological-pathway and disease information (Kanehisa et al.,2008). Pathway enrichment analysis identifies significantly enriched metabolic pathways or signal transduction pathways associated with differentially expressed genes, comparing the whole genome background. KEGG pathways with padj < 0.05 are significant enrichment. </p>




	
	<p class="name center">Table 4.10 KEGG Enrichment Analysis Partial Results</p>

	<p class="center">

		<div id="tb" style="overflow-x: auto">

			<table class="gy">

				<thead>

					<tr>

						<th>KEGGID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>keggID</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th>

					</tr>

				</thead>

				<tbody>

					

					<tr><td>mmu00100</td><td>Steroid biosynthesis</td><td>13/3533</td><td>13/6183</td><td>0.000685740843157746</td><td>0.22218003318311</td><td>ENSMUSG00000038567/ENSMUSG00000026675/ENSMUSG00000021273/ENSMUSG00000026600/ENSMUSG00000031604/ENSMUSG00000034926/ENSMUSG00000032018/ENSMUSG00000026818/ENSMUSG00000023045/ENSMUSG00000031349/ENSMUSG00000004880/ENSMUSG00000022351/ENSMUSG00000058454</td><td>Cyp24a1/Hsd17b7/Fdft1/Soat1/Msmo1/Dhcr24/Sc5d/Cel/Soat2/Nsdhl/Lbr/Sqle/Dhcr7</td><td>mmu:13081/mmu:15490/mmu:14137/mmu:20652/mmu:66234/mmu:74754/mmu:235293/mmu:12613/mmu:223920/mmu:18194/mmu:98386/mmu:20775/mmu:13360</td><td>13</td><td>3</td><td>ENSMUSG00000026600/ENSMUSG00000026818/ENSMUSG00000031349</td><td>10</td><td>ENSMUSG00000038567/ENSMUSG00000026675/ENSMUSG00000021273/ENSMUSG00000031604/ENSMUSG00000034926/ENSMUSG00000032018/ENSMUSG00000023045/ENSMUSG00000004880/ENSMUSG00000022351/ENSMUSG00000058454
</td></tr>

					

					<tr><td>mmu04392</td><td>Hippo signaling pathway - multiple species</td><td>17/3533</td><td>19/6183</td><td>0.00265388752154988</td><td>0.42992977849108</td><td>ENSMUSG00000027339/ENSMUSG00000006262/ENSMUSG00000022329/ENSMUSG00000055320/ENSMUSG00000042129/ENSMUSG00000010067/ENSMUSG00000022433/ENSMUSG00000025239/ENSMUSG00000030774/ENSMUSG00000040021/ENSMUSG00000036459/ENSMUSG00000009073/ENSMUSG00000053110/ENSMUSG00000021959/ENSMUSG00000046743/ENSMUSG00000043131/ENSMUSG00000027803</td><td>Rassf2/Mob1b/Stk3/Tead1/Rassf4/Rassf1/Csnk1e/Limd1/Pak1/Lats1/Wtip/Nf2/Yap1/Lats2/Fat4/Mob1a/Wwtr1</td><td>mmu:215653/mmu:68473/mmu:56274/mmu:21676/mmu:213391/mmu:56289/mmu:27373/mmu:29806/mmu:18479/mmu:16798/mmu:101543/mmu:18016/mmu:22601/mmu:50523/mmu:329628/mmu:232157/mmu:97064</td><td>17</td><td>9</td><td>ENSMUSG00000027339/ENSMUSG00000042129/ENSMUSG00000010067/ENSMUSG00000022433/ENSMUSG00000025239/ENSMUSG00000030774/ENSMUSG00000036459/ENSMUSG00000009073/ENSMUSG00000043131</td><td>8</td><td>ENSMUSG00000006262/ENSMUSG00000022329/ENSMUSG00000055320/ENSMUSG00000040021/ENSMUSG00000053110/ENSMUSG00000021959/ENSMUSG00000046743/ENSMUSG00000027803
</td></tr>

					

					<tr><td>mmu00062</td><td>Fatty acid elongation</td><td>19/3533</td><td>23/6183</td><td>0.00942716584856191</td><td>0.999359919641366</td><td>ENSMUSG00000059447/ENSMUSG00000033629/ENSMUSG00000038754/ENSMUSG00000052392/ENSMUSG00000025745/ENSMUSG00000063684/ENSMUSG00000027984/ENSMUSG00000032262/ENSMUSG00000072949/ENSMUSG00000028657/ENSMUSG00000028497/ENSMUSG00000021228/ENSMUSG00000032349/ENSMUSG00000021226/ENSMUSG00000031708/ENSMUSG00000006390/ENSMUSG00000028145/ENSMUSG00000083567/ENSMUSG00000027195</td><td>Hadhb/Hacd3/Elovl3/Acot4/Hadha/Gm13910/Hadh/Elovl4/Acot1/Ppt1/Hacd4/Acot3/Elovl5/Acot2/Tecr/Elovl1/Them4/Gm11451/Hsd17b12</td><td>mmu:231086/mmu:57874/mmu:12686/mmu:171282/mmu:97212/mmu:231086/mmu:15107/mmu:83603/mmu:26897/mmu:19063/mmu:66775/mmu:171281/mmu:68801/mmu:171210/mmu:106529/mmu:54325/mmu:75778/mmu:30963/mmu:56348</td><td>19</td><td>3</td><td>ENSMUSG00000028657/ENSMUSG00000028497/ENSMUSG00000083567</td><td>16</td><td>ENSMUSG00000059447/ENSMUSG00000033629/ENSMUSG00000038754/ENSMUSG00000052392/ENSMUSG00000025745/ENSMUSG00000063684/ENSMUSG00000027984/ENSMUSG00000032262/ENSMUSG00000072949/ENSMUSG00000021228/ENSMUSG00000032349/ENSMUSG00000021226/ENSMUSG00000031708/ENSMUSG00000006390/ENSMUSG00000028145/ENSMUSG00000027195
</td></tr>

					

					<tr><td>mmu05165</td><td>Human papillomavirus infection</td><td>163/3533</td><td>255/6183</td><td>0.0145079853264076</td><td>0.999359919641366</td><td>ENSMUSG00000053007/ENSMUSG00000025812/ENSMUSG00000021277/ENSMUSG00000029561/ENSMUSG00000024182/ENSMUSG00000022105/ENSMUSG00000034218/ENSMUSG00000022521/ENSMUSG00000042453/ENSMUSG00000006728/ENSMUSG00000031628/ENSMUSG00000022996/ENSMUSG00000031925/ENSMUSG00000003873/ENSMUSG00000028238/ENSMUSG00000020122/ENSMUSG00000003031/ENSMUSG00000001930/ENSMUSG00000025809/ENSMUSG00000020516/ENSMUSG00000056116/ENSMUSG00000049791/ENSMUSG00000001750/ENSMUSG00000003233/ENSMUSG00000027276/ENSMUSG00000066036/ENSMUSG00000067235/ENSMUSG00000031887/ENSMUSG00000032409/ENSMUSG00000052459/ENSMUSG00000029306/ENSMUSG00000033379/ENSMUSG00000020758/ENSMUSG00000020184/ENSMUSG00000022997/ENSMUSG00000024985/ENSMUSG00000061859/ENSMUSG00000059552/ENSMUSG00000053835/ENSMUSG00000106447/ENSMUSG00000022770/ENSMUSG00000025958/ENSMUSG00000004285/ENSMUSG00000052572/ENSMUSG00000047123/ENSMUSG00000000555/ENSMUSG00000029661/ENSMUSG00000002015/ENSMUSG00000055024/ENSMUSG00000028047/ENSMUSG00000013160/ENSMUSG00000004936/ENSMUSG00000024121/ENSMUSG00000031490/ENSMUSG00000002325/ENSMUSG00000019899/ENSMUSG00000021178/ENSMUSG00000027490/ENSMUSG00000073409/ENSMUSG00000079491/ENSMUSG00000061232/ENSMUSG00000100713/ENSMUSG00000033327/ENSMUSG00000044167/ENSMUSG00000040033/ENSMUSG00000070348/ENSMUSG00000005871/ENSMUSG00000024620/ENSMUSG00000042284/ENSMUSG00000028364/ENSMUSG00000032796/ENSMUSG00000028423/ENSMUSG00000031502/ENSMUSG00000024241/ENSMUSG00000117269/ENSMUSG00000026626/ENSMUSG00000019210/ENSMUSG00000000125/ENSMUSG00000021051/ENSMUSG00000041417/ENSMUSG00000021114/ENSMUSG00000021994/ENSMUSG00000022483/ENSMUSG00000006273/ENSMUSG00000024576/ENSMUSG00000042349/ENSMUSG00000003184/ENSMUSG00000025199/ENSMUSG00000027230/ENSMUSG00000015575/ENSMUSG00000026812/ENSMUSG00000022812/ENSMUSG00000019302/ENSMUSG00000035929/ENSMUSG00000027641/ENSMUSG00000000881/ENSMUSG00000100857/ENSMUSG00000032243/ENSMUSG00000022607/ENSMUSG00000007989/ENSMUSG00000028163/ENSMUSG00000014767/ENSMUSG00000020689/ENSMUSG00000001506/ENSMUSG00000027523/ENSMUSG00000025499/ENSMUSG00000028530/ENSMUSG00000026478/ENSMUSG00000071359/ENSMUSG00000002900/ENSMUSG00000026147/ENSMUSG00000061143/ENSMUSG00000085667/ENSMUSG00000028945/ENSMUSG00000019846/ENSMUSG00000117253/ENSMUSG00000029528/ENSMUSG00000034664/ENSMUSG00000024079/ENSMUSG00000027793/ENSMUSG00000055799/ENSMUSG00000016283/ENSMUSG00000033793/ENSMUSG00000029071/ENSMUSG00000031077/ENSMUSG00000114087/ENSMUSG00000055413/ENSMUSG00000031834/ENSMUSG00000027111/ENSMUSG00000081683/ENSMUSG00000021022/ENSMUSG00000019969/ENSMUSG00000059409/ENSMUSG00000032744/ENSMUSG00000029304/ENSMUSG00000048126/ENSMUSG00000032487/ENSMUSG00000026479/ENSMUSG00000004056/ENSMUSG00000022817/ENSMUSG00000038648/ENSMUSG00000001507/ENSMUSG00000025158/ENSMUSG00000021112/ENSMUSG00000035198/ENSMUSG00000026768/ENSMUSG00000026840/ENSMUSG00000027009/ENSMUSG00000028991/ENSMUSG00000028017/ENSMUSG00000111796/ENSMUSG00000024500/ENSMUSG00000005469/ENSMUSG00000036961/ENSMUSG00000023067/ENSMUSG00000031666/ENSMUSG00000043154/ENSMUSG00000021611/ENSMUSG00000061062/ENSMUSG00000000142/ENSMUSG00000054128/ENSMUSG00000017843/ENSMUSG00000063065</td><td>Creb5/Pard3/Traf3/Oasl2/Axin1/Rb1/Atm/Crebbp/Reln/Cdk4/Casp3/Wnt10b/Maml2/Bax/Atp6v0d2/Egfr/Cdkn1b/Vwf/Itgb1/Rps6kb1/H2-T22/Fzd4/Tcirg1/Dvl3/Jag1/Ubr4/H2-Q10/Tradd/Atr/Atp6v1a/Ibsp/Atp6v0b/Itgb4/Mdm2/Wnt1/Tcf7l2/Patj/Trp53/H2-T24/Gm42957/Dlg1/Creb1/Atp6v1f/Dlg2/Ticam1/Itga5/Col1a2/Bcap31/Ep300/Thbs3/Atp6v0d1/Map2k1/Atp6v0c/Eif4ebp1/Irf9/Lama2/Psmc1/E2f1/H2-Q6/H2-T10/H2-K1/Ifna7/Tnxb/Foxo1/Stat2/Ccnd1/Apc/Pdgfrb/Itga1/Tnc/Lama1/Nfx1/Col4a1/Sos1/AC151275.1/Ppp2r5a/Atp6v1e1/Wnt3/Ppp2r5e/Pik3r1/Atp6v1d/Wnt5a/Col2a1/Atp6v1b2/Csnk1a1/Ikbke/Irf3/Chuk/Creb3l1/Atp6v0e/Tsc1/Gsk3b/Atp6v0a1/H2-Q4/Rbl1/Dlg3/1700041M19Rik/Itga11/Ptk2/Fzd3/Nfkb1/Tbp/Itgb3/Col1a1/Gnas/Hras/Jak1/Lamc1/Tbpl1/Lamb1/Col9a1/Maml3/Gm12992/Rheb/Lama4/CT025671.3/Pxn/Itga2b/Eif2ak2/Ccna1/Tcf7l1/H2-M2/Atp6v1h/Dvl1/Fadd/Gm7046/H2-Q5/Pik3r2/Itga6/Fzd10/Ppp2r3c/Psen1/Ppp2r5d/Heyl/Spp1/Col6a3/Ptgs2/Lamc2/Akt2/Itgb5/Creb3l2/Itga3/Rfng/Mpp5/Tubg1/Itga8/Lamc3/Itga4/Mtor/Egf/Gm47575/Ppp2r2b/Prkaca/Wnt8b/Cdkn1a/Rbl2/Ppp2r3a/Tert/Gm10093/Axin2/H2-T3/Ppp2r5c/Mapk3</td><td>mmu:231991/mmu:93742/mmu:22031/mmu:23962/mmu:12005/mmu:19645/mmu:11920/mmu:12914/mmu:19699/mmu:12567/mmu:12367/mmu:22410/mmu:270118/mmu:12028/mmu:242341/mmu:13649/mmu:12576/mmu:22371/mmu:16412/mmu:72508/mmu:15039/mmu:14366/mmu:27060/mmu:13544/mmu:16449/mmu:69116/mmu:15007/mmu:71609/mmu:245000/mmu:11964/mmu:15891/mmu:114143/mmu:192897/mmu:17246/mmu:22408/mmu:21416/mmu:12695/mmu:22059/mmu:15042/mmu:213119/mmu:13383/mmu:12912/mmu:66144/mmu:23859/mmu:106759/mmu:16402/mmu:12843/mmu:27061/mmu:328572/mmu:21827/mmu:11972/mmu:26395/mmu:11984/mmu:13685/mmu:16391/mmu:16773/mmu:19179/mmu:13555/mmu:110557/mmu:15024/mmu:14972/mmu:15970/mmu:81877/mmu:56458/mmu:20847/mmu:12443/mmu:11789/mmu:18596/mmu:109700/mmu:21923/mmu:16772/mmu:74164/mmu:12826/mmu:20662/mmu:100038882/mmu:226849/mmu:11973/mmu:22415/mmu:26932/mmu:18708/mmu:73834/mmu:22418/mmu:12824/mmu:11966/mmu:93687/mmu:56489/mmu:54131/mmu:12675/mmu:26427/mmu:11974/mmu:64930/mmu:56637/mmu:11975/mmu:15015/mmu:19650/mmu:53310/mmu:66144/mmu:319480/mmu:14083/mmu:14365/mmu:18033/mmu:21374/mmu:16416/mmu:12842/mmu:14683/mmu:15461/mmu:16451/mmu:226519/mmu:237336/mmu:16777/mmu:12839/mmu:433586/mmu:74769/mmu:19744/mmu:16775/mmu:15182/mmu:19303/mmu:16399/mmu:19106/mmu:12427/mmu:21415/mmu:14990/mmu:108664/mmu:13542/mmu:14082/mmu:18746/mmu:15018/mmu:18709/mmu:16403/mmu:93897/mmu:59032/mmu:19164/mmu:21770/mmu:56198/mmu:20750/mmu:12835/mmu:19225/mmu:16782/mmu:11652/mmu:16419/mmu:208647/mmu:16400/mmu:19719/mmu:56217/mmu:103733/mmu:241226/mmu:23928/mmu:16401/mmu:56717/mmu:13645/mmu:68553/mmu:72930/mmu:18747/mmu:22423/mmu:12575/mmu:19651/mmu:235542/mmu:21752/mmu:433759/mmu:12006/mmu:15043/mmu:26931/mmu:26417</td><td>163</td><td>47</td><td>ENSMUSG00000025812/ENSMUSG00000021277/ENSMUSG00000029561/ENSMUSG00000056116/ENSMUSG00000049791/ENSMUSG00000001750/ENSMUSG00000027276/ENSMUSG00000029306/ENSMUSG00000033379/ENSMUSG00000020758/ENSMUSG00000022997/ENSMUSG00000053835/ENSMUSG00000047123/ENSMUSG00000029661/ENSMUSG00000004936/ENSMUSG00000024121/ENSMUSG00000031490/ENSMUSG00000021178/ENSMUSG00000027490/ENSMUSG00000073409/ENSMUSG00000079491/ENSMUSG00000061232/ENSMUSG00000070348/ENSMUSG00000005871/ENSMUSG00000024620/ENSMUSG00000028364/ENSMUSG00000028423/ENSMUSG00000024241/ENSMUSG00000026626/ENSMUSG00000019210/ENSMUSG00000041417/ENSMUSG00000015575/ENSMUSG00000035929/ENSMUSG00000032243/ENSMUSG00000028163/ENSMUSG00000001506/ENSMUSG00000025499/ENSMUSG00000029528/ENSMUSG00000031834/ENSMUSG00000032744/ENSMUSG00000004056/ENSMUSG00000035198/ENSMUSG00000005469/ENSMUSG00000000142/ENSMUSG00000054128/ENSMUSG00000017843/ENSMUSG00000063065</td><td>116</td><td>ENSMUSG00000053007/ENSMUSG00000024182/ENSMUSG00000022105/ENSMUSG00000034218/ENSMUSG00000022521/ENSMUSG00000042453/ENSMUSG00000006728/ENSMUSG00000031628/ENSMUSG00000022996/ENSMUSG00000031925/ENSMUSG00000003873/ENSMUSG00000028238/ENSMUSG00000020122/ENSMUSG00000003031/ENSMUSG00000001930/ENSMUSG00000025809/ENSMUSG00000020516/ENSMUSG00000003233/ENSMUSG00000066036/ENSMUSG00000067235/ENSMUSG00000031887/ENSMUSG00000032409/ENSMUSG00000052459/ENSMUSG00000020184/ENSMUSG00000024985/ENSMUSG00000061859/ENSMUSG00000059552/ENSMUSG00000106447/ENSMUSG00000022770/ENSMUSG00000025958/ENSMUSG00000004285/ENSMUSG00000052572/ENSMUSG00000000555/ENSMUSG00000002015/ENSMUSG00000055024/ENSMUSG00000028047/ENSMUSG00000013160/ENSMUSG00000002325/ENSMUSG00000019899/ENSMUSG00000100713/ENSMUSG00000033327/ENSMUSG00000044167/ENSMUSG00000040033/ENSMUSG00000042284/ENSMUSG00000032796/ENSMUSG00000031502/ENSMUSG00000117269/ENSMUSG00000000125/ENSMUSG00000021051/ENSMUSG00000021114/ENSMUSG00000021994/ENSMUSG00000022483/ENSMUSG00000006273/ENSMUSG00000024576/ENSMUSG00000042349/ENSMUSG00000003184/ENSMUSG00000025199/ENSMUSG00000027230/ENSMUSG00000026812/ENSMUSG00000022812/ENSMUSG00000019302/ENSMUSG00000027641/ENSMUSG00000000881/ENSMUSG00000100857/ENSMUSG00000022607/ENSMUSG00000007989/ENSMUSG00000014767/ENSMUSG00000020689/ENSMUSG00000027523/ENSMUSG00000028530/ENSMUSG00000026478/ENSMUSG00000071359/ENSMUSG00000002900/ENSMUSG00000026147/ENSMUSG00000061143/ENSMUSG00000085667/ENSMUSG00000028945/ENSMUSG00000019846/ENSMUSG00000117253/ENSMUSG00000034664/ENSMUSG00000024079/ENSMUSG00000027793/ENSMUSG00000055799/ENSMUSG00000016283/ENSMUSG00000033793/ENSMUSG00000029071/ENSMUSG00000031077/ENSMUSG00000114087/ENSMUSG00000055413/ENSMUSG00000027111/ENSMUSG00000081683/ENSMUSG00000021022/ENSMUSG00000019969/ENSMUSG00000059409/ENSMUSG00000029304/ENSMUSG00000048126/ENSMUSG00000032487/ENSMUSG00000026479/ENSMUSG00000022817/ENSMUSG00000038648/ENSMUSG00000001507/ENSMUSG00000025158/ENSMUSG00000021112/ENSMUSG00000026768/ENSMUSG00000026840/ENSMUSG00000027009/ENSMUSG00000028991/ENSMUSG00000028017/ENSMUSG00000111796/ENSMUSG00000024500/ENSMUSG00000036961/ENSMUSG00000023067/ENSMUSG00000031666/ENSMUSG00000043154/ENSMUSG00000021611/ENSMUSG00000061062
</td></tr>

					

					<tr><td>mmu00230</td><td>Purine metabolism</td><td>65/3533</td><td>96/6183</td><td>0.0214218103755913</td><td>0.999359919641366</td><td>ENSMUSG00000030978/ENSMUSG00000026239/ENSMUSG00000091415/ENSMUSG00000021699/ENSMUSG00000064330/ENSMUSG00000025584/ENSMUSG00000027889/ENSMUSG00000020444/ENSMUSG00000030671/ENSMUSG00000020910/ENSMUSG00000028792/ENSMUSG00000025817/ENSMUSG00000019990/ENSMUSG00000028527/ENSMUSG00000023868/ENSMUSG00000022292/ENSMUSG00000048120/ENSMUSG00000021236/ENSMUSG00000000295/ENSMUSG00000002326/ENSMUSG00000000253/ENSMUSG00000074797/ENSMUSG00000029310/ENSMUSG00000025630/ENSMUSG00000026575/ENSMUSG00000025742/ENSMUSG00000108110/ENSMUSG00000024575/ENSMUSG00000024066/ENSMUSG00000033068/ENSMUSG00000011148/ENSMUSG00000025041/ENSMUSG00000021536/ENSMUSG00000015711/ENSMUSG00000062867/ENSMUSG00000029171/ENSMUSG00000020431/ENSMUSG00000028525/ENSMUSG00000019989/ENSMUSG00000031842/ENSMUSG00000026567/ENSMUSG00000041624/ENSMUSG00000083935/ENSMUSG00000024177/ENSMUSG00000078941/ENSMUSG00000083789/ENSMUSG00000069094/ENSMUSG00000039058/ENSMUSG00000023961/ENSMUSG00000031659/ENSMUSG00000110545/ENSMUSG00000075270/ENSMUSG00000003500/ENSMUSG00000026817/ENSMUSG00000014554/ENSMUSG00000041741/ENSMUSG00000032565/ENSMUSG00000020654/ENSMUSG00000020736/ENSMUSG00000025791/ENSMUSG00000022489/ENSMUSG00000042638/ENSMUSG00000029247/ENSMUSG00000106035/ENSMUSG00000032420</td><td>Rrm1/Pde6d/Ak9/Pde4d/Pde6h/Pde8a/Ampd2/Guk1/Pde3b/Adprm/Ak2/Nudt5/Pde7b/Ak4/Pde10a/Rrm2b/Entpd1/Entpd5/Hddc2/Gmpr2/Gmpr/Itpa/Nudt9/Hprt/Nme7/Prps2/Gm18012/Pde6a/Xdh/Entpd6/Adssl1/Nt5c2/Adcy2/Prune1/Impdh2/Pgm1/Adcy1/Pde4b/Enpp3/Pde4c/Adcy10/Gucy1a2/Gm12294/Nme4/Ak6/Gm6092/Pde7a/Ak5/Enpp4/Adcy7/Gm7730/Pde11a/Impdh1/Ak1/Dguok/Pde3a/Nudt16/Adcy3/Nt5c/Pgm2/Pde1b/Gucy2c/Paics/Gm43545/Nt5e</td><td>mmu:20133/mmu:18582/mmu:633979/mmu:238871/mmu:78600/mmu:18584/mmu:109674/mmu:14923/mmu:18576/mmu:66358/mmu:11637/mmu:53893/mmu:29863/mmu:11639/mmu:23984/mmu:382985/mmu:12495/mmu:12499/mmu:69692/mmu:105446/mmu:66355/mmu:16434/mmu:74167/mmu:15452/mmu:171567/mmu:110639/mmu:13178/mmu:225600/mmu:22436/mmu:12497/mmu:11565/mmu:76952/mmu:210044/mmu:229589/mmu:23918/mmu:66681/mmu:432530/mmu:18578/mmu:209558/mmu:110385/mmu:271639/mmu:234889/mmu:102216272/mmu:56520/mmu:102216272/mmu:11636/mmu:18583/mmu:229949/mmu:224794/mmu:11513/mmu:18103/mmu:241489/mmu:23917/mmu:11636/mmu:27369/mmu:54611/mmu:75686/mmu:104111/mmu:50773/mmu:72157/mmu:18574/mmu:14917/mmu:67054/mmu:74167/mmu:23959</td><td>65</td><td>16</td><td>ENSMUSG00000030978/ENSMUSG00000025584/ENSMUSG00000027889/ENSMUSG00000020444/ENSMUSG00000020910/ENSMUSG00000028792/ENSMUSG00000021236/ENSMUSG00000074797/ENSMUSG00000029310/ENSMUSG00000025630/ENSMUSG00000015711/ENSMUSG00000029171/ENSMUSG00000023961/ENSMUSG00000075270/ENSMUSG00000020736/ENSMUSG00000042638</td><td>49</td><td>ENSMUSG00000026239/ENSMUSG00000091415/ENSMUSG00000021699/ENSMUSG00000064330/ENSMUSG00000030671/ENSMUSG00000025817/ENSMUSG00000019990/ENSMUSG00000028527/ENSMUSG00000023868/ENSMUSG00000022292/ENSMUSG00000048120/ENSMUSG00000000295/ENSMUSG00000002326/ENSMUSG00000000253/ENSMUSG00000026575/ENSMUSG00000025742/ENSMUSG00000108110/ENSMUSG00000024575/ENSMUSG00000024066/ENSMUSG00000033068/ENSMUSG00000011148/ENSMUSG00000025041/ENSMUSG00000021536/ENSMUSG00000062867/ENSMUSG00000020431/ENSMUSG00000028525/ENSMUSG00000019989/ENSMUSG00000031842/ENSMUSG00000026567/ENSMUSG00000041624/ENSMUSG00000083935/ENSMUSG00000024177/ENSMUSG00000078941/ENSMUSG00000083789/ENSMUSG00000069094/ENSMUSG00000039058/ENSMUSG00000031659/ENSMUSG00000110545/ENSMUSG00000003500/ENSMUSG00000026817/ENSMUSG00000014554/ENSMUSG00000041741/ENSMUSG00000032565/ENSMUSG00000020654/ENSMUSG00000025791/ENSMUSG00000022489/ENSMUSG00000029247/ENSMUSG00000106035/ENSMUSG00000032420
</td></tr>

					

				</tbody>

			</table>

		</div>

	</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<strong>KEGGID</strong>: Unique identification id of KEGG database.<br/>

<strong>Description</strong>: Function description of this pathway.<br/>

<strong>GeneRatio</strong>: Ratio between the number of differentially expressed genes in each pathway and all

		differentially expressed genes that can be found in KEGG database.<br/>

<strong>BgRatio</strong>: In background KEGG database, the ratio of all genes concerning this KEGG term to all genes.<br/>

<strong>pvalue</strong>: Statistics category term; abbreviation for probability value.<br/>

<strong>padj</strong>: Adjusted p-value. Generally, KEGG pathway with Corrected_pValue

		< 0.05 are significant enrichment.<br/>

<strong>geneID</strong>: Differentially expressed genes related to this pathway.<br/>

<strong>geneName</strong>: Differentially expressed genes in this pathway.<br/>

<strong>keggID</strong>: Pathway gene id.<br/>

<strong>Count</strong>:Number of differentially expressed genes concerning this pathway.<br/>
	
<strong>Up</strong>:Number of up expressed genes concerning this KEGG pathway.<br/>

<strong>Up_Gene_id</strong>:Up expressed gene id concerning this KEGG pathway.<br/>

<strong>Down</strong>:Number of down expressed genes concerning this KEGG pathway.<br/>

<strong>Down_Gene_id</strong>: Down expressed gene id concerning this KEGG pathway.<br/>

</ul>
</div>



	<p class="paragraph">In the KEGG enrichment results, the most significant 20 KEGG pathways were selected for display. If the results are less than 20, all pathways would be drawn, as shown in the following figure. In this figure, the abscissa is the KEGG pathway, and the ordinate is the significance level of the pathway enrichment. Higher values correspond to higher significance.
	</p>
	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_5_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_5_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_5_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_5_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/KEGG/ncvsH2_inhit.bar.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_5_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_5_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_5_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_5_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_5_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.16 KEGG enrichment analysis histogram </p>
    <p class="premark center">
The abscissa is the KEGG pathway, and the ordinate is the significance level of pathway enrichment</p>


	<p class="paragraph">From the KEGG enrichment results, the most significant 20 KEGG pathways were selected for display. If the results are less than 20, all pathways would be drawn. In this figure, the abscissa is the ratio of the number of differential genes linked with the KEGG pathway to the total number of differential genes. The ordinate is KEGG Pathway. The size of a point represents the number of genes annotated to a specific KEGG pathway. The color from red to purple represents the significant level of the enrichment. 	</p>

	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_6_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_6_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_6_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_6_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/KEGG/ncvsH2_inhit.dot.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_6_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_6_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_6_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_6_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_6_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.17 KEGG enrichment scatter plot </p>
        <p class="premark center">
The abscissa in the graph is the ratio of the number of differential genes on the KEGG pathway to the total number of differential genes, and the ordinate is KEGG pathway</p>


	<p class="paragraph"> The html file can interactively help you view the significantly enriched KEGG pathway map by clicking corresponding links. In the map, the KEGG node including up-regulated genes is marked red, and the KEGG node including down-regulated genes is marked green. While yellow means the node contains both types of genes. Hovering over the marked KEGG node will show details of differentially expressed genes, with the same color as above, and the number inside brackets is log2(Foldchange). The above steps can be implemented offline. If you are connected to the Internet, click on each node to get the specific information of each KEGG node in the KEGG database. </p>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/5.Enrichment/2.KEGG" title="点击打开" >Enrichment/2.KEGG</a> </p>
<br/>
    
<html>
<select ><option value="src/files/ncvsH2_inhit.all_KEGGenrich.html">ncvsH2_inhit.all_KEGGenrich.html</option></select><br/>
<iframe src="src/files/ncvsH2_inhit.all_KEGGenrich.html" frameborder="0" height="500" width="80%" style="margin:100px "></iframe>
</html>

<br/>



	<!---------------------------Reactome富集分析---------------------->
    

	<h4> 4.5.3 Reactome Enrichment Analysis</h4>

	<p class="paragraph">The Reactome database brings together the various reactions and biological pathways of human model species. Reactome pathway enrichment with padj less than 0.05 as the threshold for significant enrichment, the enrichment results are shown in the following table.
	</p>
	<p class="name center">Table 4.11 Reactome Enrichment Analysis Partial Results </p>

	<p class="center">

		<div id="tb" style="overflow-x: auto">

			<table class="gy">

				<thead>

					<tr>

						<th>ReactomeID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>EntrezID</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th>

					</tr>

				</thead>

				<tbody>

					

					<tr><td>R-MMU-1660514</td><td>Synthesis of PIPs at the Golgi membrane</td><td>12/3349</td><td>13/5653</td><td>0.0109416772632744</td><td>0.999832181977551</td><td>ENSMUSG00000031481/ENSMUSG00000051853/ENSMUSG00000041720/ENSMUSG00000030660/ENSMUSG00000033628/ENSMUSG00000038861/ENSMUSG00000025949/ENSMUSG00000025178/ENSMUSG00000032571/ENSMUSG00000048076/ENSMUSG00000001173/ENSMUSG00000025240</td><td>Tpte/Arf3/Pi4ka/Pik3c2a/Pik3c3/Pi4kb/Pikfyve/Pi4k2a/Pik3r4/Arf1/Ocrl/Sacm1l</td><td>234129/11842/224020/18704/225326/107650/18711/84095/75669/11840/320634/83493</td><td>12</td><td>3</td><td>ENSMUSG00000051853/ENSMUSG00000025178/ENSMUSG00000025240</td><td>9</td><td>ENSMUSG00000031481/ENSMUSG00000041720/ENSMUSG00000030660/ENSMUSG00000033628/ENSMUSG00000038861/ENSMUSG00000025949/ENSMUSG00000032571/ENSMUSG00000048076/ENSMUSG00000001173
</td></tr>

					

					<tr><td>R-MMU-140877</td><td>Formation of Fibrin Clot (Clotting Cascade)</td><td>19/3349</td><td>23/5653</td><td>0.0157387852013083</td><td>0.999832181977551</td><td>ENSMUSG00000001930/ENSMUSG00000030054/ENSMUSG00000026249/ENSMUSG00000109764/ENSMUSG00000052212/ENSMUSG00000024386/ENSMUSG00000060459/ENSMUSG00000018446/ENSMUSG00000022912/ENSMUSG00000057729/ENSMUSG00000026579/ENSMUSG00000061119/ENSMUSG00000074743/ENSMUSG00000028128/ENSMUSG00000031444/ENSMUSG00000039109/ENSMUSG00000023224/ENSMUSG00000027082/ENSMUSG00000022875</td><td>Vwf/Gp9/Serpine2/Klkb1/Cd177/Proc/Kng2/C1qbp/Pros1/Prtn3/F5/Prcp/Thbd/F3/F10/F13a1/Serping1/Tfpi/Kng1</td><td>22371/54368/20720/16621/68891/19123/385643/12261/19128/19152/14067/72461/21824/14066/14058/74145/12258/21788/16644</td><td>19</td><td>8</td><td>ENSMUSG00000109764/ENSMUSG00000052212/ENSMUSG00000060459/ENSMUSG00000018446/ENSMUSG00000057729/ENSMUSG00000031444/ENSMUSG00000039109/ENSMUSG00000023224</td><td>11</td><td>ENSMUSG00000001930/ENSMUSG00000030054/ENSMUSG00000026249/ENSMUSG00000024386/ENSMUSG00000022912/ENSMUSG00000026579/ENSMUSG00000061119/ENSMUSG00000074743/ENSMUSG00000028128/ENSMUSG00000027082/ENSMUSG00000022875
</td></tr>

					

					<tr><td>R-MMU-3000178</td><td>ECM proteoglycans</td><td>25/3349</td><td>32/5653</td><td>0.0200056760188272</td><td>0.999832181977551</td><td>ENSMUSG00000025809/ENSMUSG00000022892/ENSMUSG00000004098/ENSMUSG00000029307/ENSMUSG00000016995/ENSMUSG00000029661/ENSMUSG00000031375/ENSMUSG00000033327/ENSMUSG00000028364/ENSMUSG00000002341/ENSMUSG00000021614/ENSMUSG00000026837/ENSMUSG00000022483/ENSMUSG00000026042/ENSMUSG00000030789/ENSMUSG00000020689/ENSMUSG00000001506/ENSMUSG00000019929/ENSMUSG00000026147/ENSMUSG00000034664/ENSMUSG00000026043/ENSMUSG00000048126/ENSMUSG00000022817/ENSMUSG00000026768/ENSMUSG00000039952</td><td>Itgb1/App/Col5a3/Dmp1/Matn4/Col1a2/Bgn/Tnxb/Tnc/Ncan/Vcan/Col5a1/Col2a1/Col5a2/Itgax/Itgb3/Col1a1/Dcn/Col9a1/Itga2b/Col3a1/Col6a3/Itgb5/Itga8/Dag1</td><td>16412/11820/53867/13406/17183/12843/12111/81877/21923/13004/13003/12831/12824/12832/16411/16416/12842/13179/12839/16399/12825/12835/16419/241226/13138</td><td>25</td><td>4</td><td>ENSMUSG00000029661/ENSMUSG00000031375/ENSMUSG00000028364/ENSMUSG00000001506</td><td>21</td><td>ENSMUSG00000025809/ENSMUSG00000022892/ENSMUSG00000004098/ENSMUSG00000029307/ENSMUSG00000016995/ENSMUSG00000033327/ENSMUSG00000002341/ENSMUSG00000021614/ENSMUSG00000026837/ENSMUSG00000022483/ENSMUSG00000026042/ENSMUSG00000030789/ENSMUSG00000020689/ENSMUSG00000019929/ENSMUSG00000026147/ENSMUSG00000034664/ENSMUSG00000026043/ENSMUSG00000048126/ENSMUSG00000022817/ENSMUSG00000026768/ENSMUSG00000039952
</td></tr>

					

					<tr><td>R-MMU-191273</td><td>Cholesterol biosynthesis</td><td>13/3349</td><td>15/5653</td><td>0.0235740646537652</td><td>0.999832181977551</td><td>ENSMUSG00000021302/ENSMUSG00000026675/ENSMUSG00000021273/ENSMUSG00000031604/ENSMUSG00000034926/ENSMUSG00000032018/ENSMUSG00000059743/ENSMUSG00000031349/ENSMUSG00000004880/ENSMUSG00000093930/ENSMUSG00000022351/ENSMUSG00000023832/ENSMUSG00000058454</td><td>Ggps1/Hsd17b7/Fdft1/Msmo1/Dhcr24/Sc5d/Fdps/Nsdhl/Lbr/Hmgcs1/Sqle/Acat2/Dhcr7</td><td>14593/15490/14137/66234/74754/235293/110196/18194/98386/208715/20775/110460/13360</td><td>13</td><td>1</td><td>ENSMUSG00000031349</td><td>12</td><td>ENSMUSG00000021302/ENSMUSG00000026675/ENSMUSG00000021273/ENSMUSG00000031604/ENSMUSG00000034926/ENSMUSG00000032018/ENSMUSG00000059743/ENSMUSG00000004880/ENSMUSG00000093930/ENSMUSG00000022351/ENSMUSG00000023832/ENSMUSG00000058454
</td></tr>

					

					<tr><td>R-MMU-2028269</td><td>Signaling by Hippo</td><td>13/3349</td><td>15/5653</td><td>0.0235740646537652</td><td>0.999832181977551</td><td>ENSMUSG00000006262/ENSMUSG00000022329/ENSMUSG00000031628/ENSMUSG00000032531/ENSMUSG00000030516/ENSMUSG00000013076/ENSMUSG00000040021/ENSMUSG00000018209/ENSMUSG00000053110/ENSMUSG00000021959/ENSMUSG00000043131/ENSMUSG00000027803/ENSMUSG00000020849</td><td>Mob1b/Stk3/Casp3/Amotl2/Tjp1/Amotl1/Lats1/Stk4/Yap1/Lats2/Mob1a/Wwtr1/Ywhae</td><td>68473/56274/12367/56332/21872/75723/16798/58231/22601/50523/232157/97064/22627</td><td>13</td><td>3</td><td>ENSMUSG00000013076/ENSMUSG00000043131/ENSMUSG00000020849</td><td>10</td><td>ENSMUSG00000006262/ENSMUSG00000022329/ENSMUSG00000031628/ENSMUSG00000032531/ENSMUSG00000030516/ENSMUSG00000040021/ENSMUSG00000018209/ENSMUSG00000053110/ENSMUSG00000021959/ENSMUSG00000027803
</td></tr>

					

				</tbody>

			</table>

		</div>

	</p>



<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<li><strong>ReactomeID</strong>: Unique identification id of Reactome database.</li>
<li><strong>Description</strong>: Function description of this pathway.</li>
<li><strong>GeneRatio</strong>: Ratio between the number of differentially expressed genes in each Reactome ID and all differentially expressed genes that can be found in Reactome database.</li>
<li><strong>BgRatio</strong>: In background Reactome database, the ratio of all genes annotated on this Reactome pathway and all genes.</li>
<li><strong>pvalue</strong>: Statistics category term; abbreviation for probability value.</li>
<li><strong>padj</strong>: Adjusted p-value. Generally, Reactome pathways with Corrected_pValue &lt; 0.05 are significant enrichment.</li>
<li><strong>geneID</strong>: Differentially expressed genes related to this pathway.</li>
<li><strong>geneName</strong>: Differentially expressed genes in this pathway.</li>
<li><strong>EntrezID</strong>: Entrez gene ID of differentially expressed genes.</li>
<li><strong>Count</strong>: Number of differentially expressed genes concerning this pathway.</li>
<li><strong>Up</strong>: Number of up expressed genes concerning this Reactome pathway.</li>
<li><strong>Up_Gene_id</strong>: Up expressed gene id concerning this Reactome pathway.</li>
<li><strong>Down</strong>: Number of down expressed genes concerning this Reactome pathway.</li>
<li><strong>Down_Gene_id</strong>: Down expressed gene id concerning this Reactome term.</li>
</ul>
</div>


<br />

	<p class="paragraph">In the Reactome enrichment results, the most significant 20 Reactome pathways were selected for display. If the results are less than 20, all pathways would be drawn, as shown in the following figure. In the figure, the abscissa is the Reactome pathway, and the ordinate is the significance level of the pathway enrichment. Higher values correspond to high significance. 
</p>
	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_8_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_8_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_8_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_8_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/Reactome/ncvsH2_inhit.bar.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_8_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_8_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_8_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_8_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_8_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.18 Reactome Enrichment Analysis Histogram </p>
            <p class="premark center">
The abscissa is the Reactome pathway, and the ordinate is the significance level of pathway enrichment</p>



<p class="paragraph">From the Reactome enrichment analysis results, the most significant 20 Reactome pathways were selected for display. If the results are less than 20, all pathways would be drawn. In the figure, the abscissa is the ratio of the number of differential genes linked with the Reactome pathway to the total number of differential genes, and the ordinate is Reactome Pathway. The size of a point represents the number of genes annotated to a specific Reactome pathway, and the color from red to purple represents the significant size of the enrichment.
</p>

	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure5_9_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure5_9_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure5_9_pane_ncvsH2_inhit" role="tabpanel" aria-labelledby="gallery_figure5_9_list_ncvsH2_inhit" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Enrichment/Reactome/ncvsH2_inhit.dot.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure5_9_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure5_9_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure5_9_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure5_9_list_ncvsH2_inhit" data-toggle="list" href="#gallery_figure5_9_pane_ncvsH2_inhit" role="tab" aria-controls="ncvsH2_inhit">ncvsH2_inhit</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.19 Reactome enrichment analysis scatter plot </p>
    <p class="premark center">
The abscissa in the graph is the ratio of the number of differential genes on the Refectome pathway to the total number of differential genes in the Reactome pathway. The ordinate is Reactome pathway.</p>


<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/5.Enrichment/3.Reactome" title="点击打开" >Enrichment/Reactome</a> </p>





	<h4> 4.5.4 Protein-Protein Interaction Network Analysis</h4>

	<p class="paragraph">The protein-protein interaction network is constructed for differential expression gene by searching STRING protein interaction database <a href="http://string-db.org/">STRING</a>.</p>

	<p class="paragraph">Protein-protein interaction is provided as a network file that can be imported into Cytoscape software for visualization and edition. The central organizing metaphor of Cytoscape is a network graph, with molecular species represent as nodes and intermolecular interactions represent as edges.</p>
<ol style="list-style-type: decimal">
<li>Customized network data display using powerful visual styles.</li>
<li>View superposition of gene expression ratios and p-values on the network. Expression data can be mapped to node color, label, border thickness, or border color, etc. according to user-configured colors and visualization schemes.</li>
<li>Layout networks in two dimensions. A variety of layout algorithms are available, including cyclic and spring-embedded layouts.</li>
<li>Zoom in/out and pan for browsing the network.</li>
<li>Use the network manager to easily organize multiple networks. And this structure can be saved in a session file.</li>
<li>Use the bird's eye view to easily navigate large networks.</li>
<li>Easily navigate large networks (100,000+ nodes and edges) by an efficient rendering engine.</li>
</ol>
<br/>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/5.Enrichment/" title="点击打开" >Enrichment/PPI</a> </p>


	

<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the meaning of n, N, i, M in the enrichment analysis schematic?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: In the enrichment analysis results, the column of GeneRatio represents the differential gene set information. The number to the left of the slash represents the number of differential genes annotated to a pathway that corresponds to i of the enrichment schematic. The number to the right of the slash represents the differential gene annotated to all pathways that corresponds to n of the enrichment analysis schematic. The column of BgRatio represents the background gene set information. The number to the left of the slash is the number of background genes annotated to the pathway, corresponding to the M of the enrichment analysis schematic, and to the right of the slash is annotated background genes in all pathways, corresponding to the N of the enrichment analysis schematic.</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: Does the enrichment analysis result only look at the channels of significant enrichment?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: The interpretation of the results of enrichment analysis should be based on biological significance. pvalue and padj only provide a reference. As long as the results can be explained and have biological significance, those pathways that are not significantly enriched are worthy of further study.</td></tr>
</table>
</div>
<br/>
﻿<!------------------------------------- AS分析 ---------------------------------->





<p class="paragraph"></p>



    <h3> 4.6 Alternative Splicing Analysis</h3>



<p class="paragraph">Alternative Splicing (AS) is a universal gene regulation process in most eukaryotes. Eukaryotic split genes consist of intronic and exonic sequences. During RNA processing, the introns are spliced by spliceosomes while the exons are continuously edited and retained in the mature mRNAs. Some pre-mRNAs may have different splicing patterns in different conditions and yield different protein isoforms, which increases biological complexity and adaptability of eukaryotes.</p>

<p class="center">

    <img src="src/pdf_images/AS.png" width="600" height="500" />

</p>


<p class="paragraph"><a href="http://rnaseq-mats.sourceforge.net/index.html">rMATS</a> is a suitable RNA-Seq data alternative splicing analysis software(Shen et al., 2014), which not only can classify variable splicing events, but also perform differential analysis of alternative splicing events across different samples. Alternative splicing events as shown below:</p>

<p class="center">

    <img src="src/pdf_images/rMATS.png" width="600" height="500" />

</p>
<p class="name_fig">Figure 4.20 Variable splicing event classification </p>



<ul style="font-size: 14px;font-family:line-height: 20px">
<strong>Skipped exon</strong>: Skipped exon <br/>
<strong>MXE</strong>: Mutually exclusive exon <br/>
<strong>A5SS</strong>: Alternative 5' splice site <br/>
<strong>A3SS</strong>: Alternative 3' splice site <br/>
<strong>RI</strong>: Retained intron<br/> </p>

</ul>


<h4> 4.6.1 Detection of Differentially Expressed Isoforms </h4></a>

<p class="paragraph">Every alternative splicing event corresponds to 2 isoforms, which are referred to as Exon Inclusion Isoform and Exon Skipping Isoform. Statistics for expression quantity is conducted by dividing the effective length of expression when calculating the ratio of Exon Inclusion Isoform and the 2 Isoform whole expression quantity and differential significant analysis. The threshold of differential AS analysis is set as FDR < 0.05. The AS events in the below table are evaluated only with reads spanning splicing junctions. 
	
</p>

        <p class="center">

        <img class="center" height="500" alt="" width="600" src="src/pdf_images/SE.png" /></p>

</p>

<p class="name_fig">Figure 4.21 SE type of alternative splicing type </p>
<p class="name_fig">Note: The gray box in the figure indicates the exon that has a variable cut, and the black one is the constitutive exon. </p>





<p class="name center">Table 4.12 SE Type Differential Alternative Splicing Analysis Partial Results </p>

<p class="center">

    <div style="overflow-x: auto">

        <table class="gy">

            <thead>

                <tr>

                    <th>GeneID</th><th>geneSymbol</th><th>chr</th><th>strand</th><th>exonStart_0base</th><th>exonEnd</th><th>upstreamES</th><th>upstreamEE</th><th>downstreamES</th><th>downstreamEE</th><th>IJC_SAMPLE_1</th><th>SJC_SAMPLE_1</th><th>IJC_SAMPLE_2</th><th>SJC_SAMPLE_2</th><th>IncFormLen</th><th>SkipFormLen</th><th>PValue</th><th>FDR</th><th>IncLevel1</th><th>IncLevel2</th><th>IncLevelDifference</th>

                </tr>

            </thead>

            <tbody>

                

                <tr><td>ENSMUSG00000024044</td><td>Epb41l3</td><td>17</td><td>+</td><td>69281032</td><td>69281107</td><td>69274703</td><td>69274895</td><td>69284538</td><td>69284916</td><td>0</td><td>13</td><td>12</td><td>0</td><td>224</td><td>149</td><td>3.80941944655e-11</td><td>1.285644923853e-10</td><td>0.0</td><td>1.0</td><td>-1.0</td></tr>

                

                <tr><td>ENSMUSG00000055447</td><td>Cd47</td><td>16</td><td>+</td><td>49908053</td><td>49908078</td><td>49906780</td><td>49906812</td><td>49910868</td><td>49911081</td><td>11</td><td>0</td><td>0</td><td>38</td><td>174</td><td>149</td><td>8.97060203897e-14</td><td>4.8441251010438e-13</td><td>1.0</td><td>0.0</td><td>1.0</td></tr>

                

                <tr><td>ENSMUSG00000022450</td><td>Ndufa6</td><td>15</td><td>-</td><td>82350978</td><td>82351094</td><td>82350139</td><td>82350371</td><td>82354080</td><td>82354322</td><td>12</td><td>0</td><td>44</td><td>13</td><td>265</td><td>149</td><td>0.000689964846775</td><td>0.00093145254314625</td><td>1.0</td><td>0.656</td><td>0.344</td></tr>

                

                <tr><td>ENSMUSG00000068220</td><td>Lgals1</td><td>15</td><td>+</td><td>78928564</td><td>78928667</td><td>78927969</td><td>78928049</td><td>78929632</td><td>78929804</td><td>30</td><td>46</td><td>0</td><td>43</td><td>252</td><td>149</td><td>4.4362949092e-07</td><td>7.98533083656e-07</td><td>0.278</td><td>0.0</td><td>0.278</td></tr>

                

                <tr><td>ENSMUSG00000064326</td><td>Siva1</td><td>12</td><td>+</td><td>112646837</td><td>112647032</td><td>112644678</td><td>112645084</td><td>112647824</td><td>112647981</td><td>9</td><td>0</td><td>0</td><td>15</td><td>344</td><td>149</td><td>7.82305109581e-10</td><td>1.9202034507897274e-09</td><td>1.0</td><td>0.0</td><td>1.0</td></tr>

                

            </tbody>

        </table>

    </div>
<br/>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<li><strong>GeneID</strong>: Gene id of the AS event.</li>
<li><strong>geneSymbol</strong>: Gene name of the AS event.</li>
<li><strong>chr</strong>: The chromosome where the AS event is located.</li>
<li><strong>strand</strong>: The direction of the chromosome chain where the AS event is located.</li>
<li><strong>exonStart_0base</strong>: The start position of AS event (the position that 1 nt upstream of the 5' end of the skipped exon)</li>
<li><strong>exonEnd</strong>: The end position of AS event (the 3' end position of skipped exon).</li>
<li><strong>upstreamES</strong>: The upstream exon start position of the AS event.</li>
<li><strong>upstreamEE</strong>: The upstream exon end position of the AS event.</li>
<li><strong>downstreamES</strong>: The downstream exon start position of the AS event.</li>
<li><strong>downstreamEE</strong>: The downstream exon end position of the AS event.</li>
<li><strong>IJC_SAMPLE_1</strong>: Inclusion junction counts for SAMPLE_1, with only the reads span splicing junctions taken into account. Biological replicates are seperated by comma.</li>
<li><strong>SJC_SAMPLE_1</strong>: Exclusion junction counts for SAMPLE_1, with only the reads span splicing junctions taken into account.</li>
<li><strong>IJC_SAMPLE_2</strong>: Inclusion junction counts for SAMPLE_2, with only the reads span splicing junctions taken into account. Biological replicates are seperated by comma.</li>
<li><strong>SJC_SAMPLE_2</strong>: Exclusion junction counts for SAMPLE_2, with only the reads span splicing junctions taken into account.</li>
<li><strong>IncFormLen</strong>: Length of inclusion form, used for normalization.</li>
<li><strong>SkipFormLen</strong>: Length of skipping form, used for normalization.</li>
<li><strong>PValue</strong>: Pvalue.</li>
<li><strong>FDR</strong>: Adjusted p-value.</li>
<li><strong>IncLevel1</strong>: Inclusion level for SAMPLE_1 replicates (comma separated) calculated from normalized counts.</li>
<li><strong>IncLevel2</strong>: Inclusion level for SAMPLE_2 replicates (comma separated) calculated from normalized counts.</li>
<li><strong>IncLevelDifference</strong>: average(IncLevel1) - average(IncLevel2).</li>
</ul>
</div>

    </p>




 <p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/6.AS/1.daslist" title="点击打开" >Result/AS/1.daslist</a> </p>
 
 
    <h4> 4.6.2 Visualization of AS Events</h4>
	
	<p style="font-weight: bold;"> Characteristics of Sashimi_plot as follows.</p> 
	
	

<ol style="list-style-type: decimal">
<li>The RNA-Seq read densities along exons are shown as histograms, color-coded by the sample. The RNA-Seq densities are aligned to the isoforms drawn at the bottom of the plot, which are automatically read from the GFF annotation of the events given as input.</li>
<li>Junction reads are visualized as arcs connecting the pair of exons that the junction borders. The thickness of the arc is in proportion to the number of junction reads present in the sample, but the actual number of junction reads can be optionally plotted too (as in the main example.)</li>
<li>Vertical coordinate indicates a modified RPKM value</li>
</ol>

	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure6_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure6_2_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure6_2_pane_SE2" role="tabpanel" aria-labelledby="gallery_figure6_2_list_SE2" onclick="trans(this.id)"><img class="img-gallery" src="src/files/AS/ASplot/SE2.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_2_pane_SE3" role="tabpanel" aria-labelledby="gallery_figure6_2_list_SE3" onclick="trans(this.id)"><img class="img-gallery" src="src/files/AS/ASplot/SE3.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_2_pane_SE1" role="tabpanel" aria-labelledby="gallery_figure6_2_list_SE1" onclick="trans(this.id)"><img class="img-gallery" src="src/files/AS/ASplot/SE1.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_2_pane_SE5" role="tabpanel" aria-labelledby="gallery_figure6_2_list_SE5" onclick="trans(this.id)"><img class="img-gallery" src="src/files/AS/ASplot/SE5.png" /></div>

<div class="tab-pane fade" id="gallery_figure6_2_pane_SE4" role="tabpanel" aria-labelledby="gallery_figure6_2_list_SE4" onclick="trans(this.id)"><img class="img-gallery" src="src/files/AS/ASplot/SE4.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure6_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure6_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure6_2_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure6_2_list_SE2" data-toggle="list" href="#gallery_figure6_2_pane_SE2" role="tab" aria-controls="SE2">SE2</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_2_list_SE3" data-toggle="list" href="#gallery_figure6_2_pane_SE3" role="tab" aria-controls="SE3">SE3</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_2_list_SE1" data-toggle="list" href="#gallery_figure6_2_pane_SE1" role="tab" aria-controls="SE1">SE1</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_2_list_SE5" data-toggle="list" href="#gallery_figure6_2_pane_SE5" role="tab" aria-controls="SE5">SE5</a>

<a class="list-group-item list-group-item-action" id="gallery_figure6_2_list_SE4" data-toggle="list" href="#gallery_figure6_2_pane_SE4" role="tab" aria-controls="SE4">SE4</a>

</div>
</div>
</div>
	</p>

    <p class="name_fig">Figure 4.22 SE Type Alternative Splicing Visualization</p>
	<p class="name_fig">Note: The red and orange colors in the figure represent different samples, and the inclusion level value represents the ratio of the total expression of exon inclusion isoform in the two isoforms, which can directly show the expression difference of alternative splicing events in different samples</p>

 <p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/6.AS/2.dasplot" title="点击打开" >Result/AS/2.dasplot</a> </p>
﻿<!-------------------------------------SNP分析--------------------------------------->

<h3> 4.7 SNP&InDel analysis</h3>


<p class="paragraph">

	A single nucleotide polymorphism (SNP) is a substitution of nucleotide sequences occurring commonly within a population (e.g. 1%), 
	
	in which a single nucleotide in the genome, or other shared sequence exchanges between members of a biological species or paired chromosomes. 
	
	Two types of variation occur with SNPs, namely transitions and transversions, occur with a probability ratio of 1:2. 
	
	The high occurrence region of SNPs is often the CG sequences, resulting in C to T transitions, 
	
	which are associated with the tendency of C to be methylated. In contrast to SNPs, 
	
	InDeL is a molecular biology term for an insertion or deletion of bases in the genome of an organism, 
	
	measuring from 1 to 10 000 base pairs in length, which is classified among small genetic variations.

</p>



<p class="paragraph">

	Mutation site analysis is an important part of RNA-seq structural analysis, including detection of inborn mutation sites and acquired somatic cell mutation sites, which is important for tumor research. We used GATK software to perform mutation site analysis on sample data and used Snpeff software to annotate the variant sites(McKenna A et al., 2010).
</p>



	<h4> 4.7.1 Variant Site Detection </h4>

<p class="paragraph">

	The variant sites are divided into SNP and INDEL. The following table shows the SNP results,genotypes and annotations of each site.
</p>

	<p class="name center">Table 4.13 variant site type and its annotation </p>

	<p class="center">

		<div style="overflow-x: auto">

			<table class="gy">

				<thead>

					<tr>

						<th>CHROM</th><th>POS</th><th>ID</th><th>REF</th><th>ALT</th><th>QUAL</th><th>DP</th><th>AD</th><th>GT</th><th>GeneID</th><th>GeneName</th><th>FeatureID</th><th>Biotype</th><th>HGVS_C</th><th>HGVS_P</th><th>EFFECT</th><th>IMPACT</th>

					</tr>

				</thead>

				<tbody>

					

					<tr><td>1</td><td>37737058</td><td>.</td><td>C</td><td>A</td><td>133.6</td><td>13</td><td>9,4</td><td>0/1</td><td>ENSMUSG00000026090-ENSMUSG00000060771</td><td>2010300C02Rik-Tsga10</td><td>ENSMUSG00000026090-ENSMUSG00000060771</td><td>.</td><td>n.37737058C&gt;A</td><td>.</td><td>intergenic_region</td><td>MODIFIER</td></tr>

					

					<tr><td>1</td><td>51953292</td><td>rs586147063</td><td>A</td><td>G</td><td>90.6</td><td>34</td><td>14,5</td><td>0/1</td><td>ENSMUSG00000072324-ENSMUSG00000062939</td><td>Gm8420-Stat4</td><td>ENSMUSG00000072324-ENSMUSG00000062939</td><td>.</td><td>n.51953292A&gt;G</td><td>.</td><td>intergenic_region</td><td>MODIFIER</td></tr>

					

					<tr><td>1</td><td>51953332</td><td>rs582946247</td><td>A</td><td>G</td><td>109.86</td><td>14</td><td>3,5</td><td>0/1</td><td>ENSMUSG00000072324-ENSMUSG00000062939</td><td>Gm8420-Stat4</td><td>ENSMUSG00000072324-ENSMUSG00000062939</td><td>.</td><td>n.51953332A&gt;G</td><td>.</td><td>intergenic_region</td><td>MODIFIER</td></tr>

					

					<tr><td>1</td><td>67090995</td><td>.</td><td>C</td><td>T</td><td>47.6</td><td>10</td><td>6,2</td><td>0/1</td><td>ENSMUSG00000026000-ENSMUSG00000025991</td><td>Lancl1-Cps1</td><td>ENSMUSG00000026000-ENSMUSG00000025991</td><td>.</td><td>n.67090995C&gt;T</td><td>.</td><td>intergenic_region</td><td>MODIFIER</td></tr>

					

					<tr><td>1</td><td>89631726</td><td>.</td><td>C</td><td>T</td><td>79.6</td><td>13</td><td>6,3</td><td>0/1</td><td>ENSMUSG00000055013</td><td>Agap1</td><td>ENSMUST00000212721.1</td><td>protein_coding</td><td>c.*3880C&gt;T</td><td>.</td><td>downstream_gene_variant</td><td>MODIFIER</td></tr>

					

				</tbody>

			</table>


		</div>

	</p>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 14px;font-family:line-height: 20px">
<li><strong>CHROM</strong>: Chromosome ID of SNPs.</li>
<li><strong>POS</strong>: The postion of variation on chromosome.</li>
<li><strong>ID</strong>: The ID of this variation in dbSNP.</li>
<li><strong>REF</strong>: Bases of variation sites in reference genome.</li>
<li><strong>ALT</strong>: The actual base of the mutation site in the sample.</li>
<li><strong>QUAL</strong>: The quality of mutation sites, the value of phred scheme. The higher the value, the higher the reliability of the genotype.</li>
<li><strong>DP</strong>: The sequencing depth of each sample at the variant site (not all the number of reads, but the number of reads that are greater than a certain quality value.)</li>
<li><strong>AD</strong>: Allele Depth.The number of reads supporting either the reference genotype or SNP genotype.</li>
<li><strong>GT</strong>: The genotype of the locus.</li>
<li><strong>GeneID</strong>: Gene id where the variant site is located.</li>
<li><strong>GeneName</strong>: Gene name where the variant site is located.</li>
<li><strong>FeatureID</strong>: Transcript id where the variant site is located.</li>
<li><strong>Biotype</strong>: Transcript type where the variant site is located.</li>
<li><strong>HGVS_C</strong>: HGVS annotation of variant sites at the DNA level.</li>
<li><strong>HGVS_P</strong>: If variant is coding, this field describes the variant using HGVS notation (Protein level). Since transcript ID is already mentioned in ‘feature ID’, it may be omitted here.</li>
<li><strong>EFFECT</strong>: Detailed description of the effect predicted by SnpEff.</li>
<li><strong>IMPACT</strong>: Effects are categorized by ‘impact’: {High, Moderate, Low, Modifier}. This are pre-defined categories to help users find more significant variants.</li>
</ul>
</div>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/7.SNP/2.snpeff" title="点击打开" >SNP/2.snpeff</a> </p>


		<h4> 4.7.2 Variant Site Statistics</h4>

		<p class="paragraph">

			After applying GATK for the detection of variant sites, we counted each variant site according to the SnpEff annotation information mainly including the function statistics of the variant site, the statistics of the variant site and the influence of the variant site. The variant site function (SNP_function) is statistically plotted from three aspects: synonymous mutation, missense mutation, and nonsense mutation. The variant site region (SNP_region) is statistically plotted from gene structure regions such as EXON, INTRON and INTERGENIC. Variant site influence (SNP_impact) is statistically plotted from four levels of HIGH (high), MODERATE (middle), LOW (low), MODIFIER (self-phenotype effect, and other variant sites should have an impact at the same time). The figure below shows the SNP results. 
		</p>

		
	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure7_1_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure7_1_nav-tabContent">

<div class="tab-pane fade" id="gallery_figure7_1_pane_SNP_region" role="tabpanel" aria-labelledby="gallery_figure7_1_list_SNP_region" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Variant/plot/SNP_region.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_1_pane_SNP_function" role="tabpanel" aria-labelledby="gallery_figure7_1_list_SNP_function" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Variant/plot/SNP_function.png" /></div>

<div class="tab-pane fade" id="gallery_figure7_1_pane_SNP_impact" role="tabpanel" aria-labelledby="gallery_figure7_1_list_SNP_impact" onclick="trans(this.id)"><img class="img-gallery" src="src/files/Variant/plot/SNP_impact.png" /></div>

</div>
</div>
<div class="col-1">
<div id="gallery_figure7_1_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure7_1_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure7_1_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_SNP_region" data-toggle="list" href="#gallery_figure7_1_pane_SNP_region" role="tab" aria-controls="SNP_region">SNP_region</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_SNP_function" data-toggle="list" href="#gallery_figure7_1_pane_SNP_function" role="tab" aria-controls="SNP_function">SNP_function</a>

<a class="list-group-item list-group-item-action" id="gallery_figure7_1_list_SNP_impact" data-toggle="list" href="#gallery_figure7_1_pane_SNP_impact" role="tab" aria-controls="SNP_impact">SNP_impact</a>

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.23 variability site statistics</p>
	<p class="name_fig">Note: The abscissa in the figure represents the sample names, and the ordinate represents the number of variants of different genomic regions (EXON, INTRON, INTERGENIC)</p>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/7.SNP/3.stat" title="点击打开" >SNP/3.stat</a> </p>


<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the reference sequence for SNP analysis, and what is REF?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: The reference sequence(REF) is the selected reference genomic sequence, and the SNP analysis used SNP calling by aligning the reads to the reference genome.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: The two numbers in a column in the SNP represent the number of reads that support both REF and ALT bases. Why are some of them zero?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: Taking 0,12 as an example, there are 0 reads that support REF bases, while there are 12 reads support ALT bases.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What does the SNP specifically mean? What is the difference between it and InDel?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: In general, SNP refers to a single nucleotide variation with a frequency of variation greater than 1%. InDel(insertion-deletion) is an insert or deletion.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the meaning of the GT column?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: GT indicates the genotype of a sample. Supposed that one genome of a diploid species has two copies, the same position of the two copies is an allele, and the GT value indicates the two alleles carried by the sample at a certain point. The point is mainly described by three values 0, 1, 2. 0 means the point is consistent with REF. 1 means it is consistent with the first ALT. 2 means it is consistent with the second ALT. 0/0 indicates homozygous. The two copies of the sample have the same alleles as REF. 0/1 indicates heterozygosity, and one copy of the sample is consistent with REF, and the other copy is consistent with ALT. 0/2 indicates heterozygosity. One copy of the sample is consistent with REF, and the other copy is consistent with the second ALT. 1/1 indicates homozygous and the alleles of the two copies of the sample are identical to the ALT. 1/2 indicates heterozygosity, and one copy of the sample is consistent with the first ALT, and the other copy is consistent with the second ALT. 2/2 means homozygous, and two copies of the sample are identical to the second ALT. According to genetic theory, the probability of two ALT appearing in one sample is very low.
</td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q.: What is the HGVS annotation?
</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">A.: HGVS is an abbreviation of Human Genome Variation Society. The gene variation sites are generally annotated with the command rules recommended by HGVS. For specific rules and examples, refer to <a href="http://www.hgvs.org/mutnomen/examplesDNA.html">DNA</a> and <a href="http://www.hgvs.org/mutnomen/examplesAA.html">protein</a>
</td></tr>
</table>
</div>
<br />
﻿<!----------------------------------------- 融合基因检测 -------------------------------------------------------->

	<h3> 4.8 Fusion Gene Analysis </h3>





<p class="paragraph">
		A fusion gene is a chimeric gene in which all or part of two genes are fused together and is generally caused by chromosomal translocations, deletions and so on. The fusion gene was first discovered in malignant tumors of the blood system. Among them, the gene fusion of BCR-ABL in chronic myeloid leukemia is the most classical. The target of Imatinib/Gleevec, which treats chronic myeloid leukemia, is the fusion gene. High-throughput RNA sequencing technology has greatly accelerated the study of fusion genes due to its high throughput, low cost, high detection accuracy and wide detection range. We uses <a href="https://github.com/STAR-Fusion/STAR-Fusion/wiki">STAR-Fusion</a>(Haas et,al., 2019)which is a software package that uses the fusion output of STAR alignment to detect the fusion transcript. The STAR-Fusion analysis process is shown in the figure below. It is divided into SATR alignment, STAR-Fusion.predict，STAR-Fusion.filter, the specific principle can refer to the appendix.

</p>

<h4> 4.8.1 Fusionlist</h4>

	<p class="paragraph">The fusionlist is shown below. JunctionReads and SpanningFrags are the main indicators for detecting fusion genes; in pair-end sequencing, read1 and read2 can be regarded as the ends of a fragment (250bp-300bp). If read1 or read2 crosses the breakpoint of the fusion event, the read is called JunctionRead. The gap area between read1 and read2 spans the breakpoint of the fusion event. Such a fragment is called SpanningFrag.</P>
	</p>
    

<p class="center"><img  src="src/pdf_images/starfuse_model.png" width="700" height="400" ></p>

<p class="name_fig">Figure 4.24 JunctionReads and SpanningFrags Schematic </p>


<p class="name center">Table 4.14 Fusion Gene List</p>

<p class="center">

	<div style="overflow-x: auto">

		<table class="gy">

			<thead>

				<tr>

					<th>FusionName</th><th>JunctionReadCount</th><th>SpanningFragCount</th><th>LeftGene</th><th>LeftLocalBreakpoint</th><th>LeftBreakpoint</th><th>RightGene</th><th>RightLocalBreakpoint</th><th>RightBreakpoint</th><th>SpliceType</th><th>annots</th><th>CDS_LEFT_ID</th><th>CDS_LEFT_RANGE</th><th>CDS_RIGHT_ID</th><th>CDS_RIGHT_RANGE</th><th>PROT_FUSION_TYPE</th><th>FUSION_MODEL</th><th>FUSION_CDS</th><th>FUSION_TRANSL</th><th>PFAM_LEFT</th><th>PFAM_RIGHT</th>

				</tr>

			</thead>

			<tbody>

				

			</tbody>

		</table>

	</div>

<br />
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<li><strong>FusionName</strong>: Fusion event name.</li>
<li><strong>JunctionReadCount</strong>: The number of junction reads that support fusion events.</li>
<li><strong>SpanningFragCount</strong>: The number of Spanning fragments that support fusion events.</li>
<li><strong>LeftGene</strong>: The upstream gene of fusion event, including the gene ID and the gene name, wherein the gene ID is the GENCODE database ID, and the decimal point and the following numbers are removed to be consistent with the previous ones.</li>
<li><strong>LeftLocalBreakpoint</strong>: The breakpoint position of the upstream gene on the gene sequence (the breakpoint position starts from the 5' end of each gene sequence).</li>
<li><strong>LeftBreakpoint</strong>: The genomic coordinates of the upstream gene breakpoint and the positive and negative strand information of the genome of the upstream gene.</li>
<li><strong>RightGene</strong>: The downstream gene of fusion event.</li>
<li><strong>RightLocalBreakpoint</strong>: The breakpoint position of the downstream gene on the gene sequence (the breakpoint position starts from the 5' end of each gene sequence).</li>
<li><strong>RightBreakpoint</strong>: The genomic coordinates of the downstream gene breakpoint and the positive and negative strand information of the genome of the upstream gene.</li>
<li><strong>SpliceType</strong>: Whether the identified fusion breakpoint is located in an exon splice site of a known gene structure.</li>
<li><strong>annots</strong>: Functional annotation information for upstream and downstream genes of fusion events.</li>
<li><strong>CDS_LEFT_ID</strong>: Transcript ID with CDS region in the upstream gene.</li>
<li><strong>CDS_LEFT_RANGE</strong>: The base range(bp) of transcript with CDS regions in the upstream gene.</li>
<li><strong>CDS_RIGHT_ID</strong>: Transcript ID with CDS region in the downstream gene.</li>
<li><strong>CDS_RIGHT_RANGE</strong>: The base range(bp) of transcript with CDS regions in the downstream gene.</li>
<li><strong>PROT_FUSION_TYPE</strong>: The fusion type of the coding gene, FRAMESHIFT (shifting of the reading frame), INFRAME (the frame does not have a frameshift).</li>
<li><strong>FUSION_CDS</strong>: The CDS sequence of the fusion gene, the lower case base is the CDS sequence of the upstream fusion gene, and the upper case is the CDS sequence of the downstream fusion gene.</li>
<li><strong>FUSION_TRANSL</strong>: The amino acid sequence of the translated fusion gene.</li>
<li><strong>PFAM_LEFT</strong>: The PFAM annotation of the upstream gene, such as DUF1515|572-611|0.073, where DUF1515 is the predicted protein domain of PFAM, 572-611 is the base range of the predicted domain, and 0.073 is the similarity between prediction and DUF15.</li>
<li><strong>PFAM_RIGHT</strong>: The PFAM annotation of the downstream gene.</li>
</ul>
</div>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/8.Fusion/1.fusionlist" title="点击打开" >Fusion/1.fusionlist</a> </p>

	<h4> 2.8.2  The Distribution of Fusion Gene on Genome</h4>

	<p class="paragraph">The distribution of fusion gene events on the chromosome is shown by a circos diagram. The loop in the figure consists of multiple chromosomes. Each curve in the loop represents a fusion event. The two ends of the curve represent the breakpoint position of the fusion event. The red indicates that the upstream and downstream genes of the fusion gene are derived from the same chromosome. While blue indicates that the upstream and downstream genes of the fusion gene are derived from different chromosomes. The thickness of the curve represents the expression level of the fusion gene.
	</p>

	<p class="center">

<div class="row gallery-row">
<div class="col-1">
<div id="gallery_figure8_2_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="gallery_figure8_2_nav-tabContent">

</div>
</div>
<div class="col-1">
<div id="gallery_figure8_2_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="gallery_figure8_2_InputList" type="text" placeholder="Search..">
<div class="list-group" id="gallery_figure8_2_list-tab" role="tablist">

</div>
</div>
</div>
	</p>

	<p class="name_fig">Figure 4.25 Fusion Gene Chromosome Distribution</p>

<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/8.Fusion/2.circos" title="点击打开" >Fusion/2.circos</a> </p>



	<h4> 4.8.3 Visualization of Fusion Gene</h4>

	<p class="paragraph"> Fusion gene events can be visualized by IGV software as shown below. </p>

<p class="center"><img  src="src/pdf_images/IGV_FusionInspector_view.png" width="600" height="500" ></p>

<p class="name_fig">Figure 4.26 Fusion Gene Chromosome Distribution </p>






<p class="paragraph"> Import The fusion gene sequence file of each sample in the fusion gene folder in the result file, the gene structure annotation file (providing annotation files in both bed and gtf formats, one of which can be used) and the aligned bam files are respectively into IGV software. The specific file description is as follows.</p>
<div class="block">
<ul style="font-size: 12px;font-family:line-height: 20px">
<li>
<strong>sample.fa</strong>: Fusion gene sequence file, in fasta format.
</li>
<li>
<strong>sample.bed</strong>: Fusion gene structure annotation file, in bed format.
</li>
<li>
<strong>sample.gtf</strong>: Fusion gene structure annotation file, in gtf format.
</li>
<li>
<strong>sample.junction_reads.bam</strong>: Bam files for JunctionReads that support fusion events.
</li>
<li>
<strong>sample.spanning_reads.bam</strong>: Bam files for SpanningFrags that support fusion events.
</li>
</ul>
</div>
<p class="paragraph">Result Directory:  <a href="../Result_X101SC2407XXXX-Z01-F001_Mus_Musculus/8.Fusion/3.IGV" title="点击打开" >Fusion/3.IGV</a> </p>

<button onclick="hiddenShow1(this)">Show Q&A</button>
<div class="comment">
    <div class="warning">
    <p>Which fusion gene should be selected for fusion gene validation?</p></div>
    <div class="block"><p>Highly expressed fusion genes are preferentially selected for validation. Highly expressed fusion gene events have lower false positives and are more easily experimentally verified. However, low expression fusion gene events also have research values. For example, many tumor suppressor genes are lowly expressed and the fusion of these genes may result in the loss of important functions.</p></div>
</div>
<br/>
﻿<!-------------------------------------------- 备注 ------------------------------------------->


<br/>
<!--  --><h2 id="4"><a name="part4">5 Appendix</a></h2>
<br/>


<a name="software catalog" ><h3 id="21">5.1 Software</h3></a>
<br/>





<table class="tf2">
	<tr>
                <th style="text-align:center;">Analysis</th>

                <th style="text-align:center;">Software</th>

                <th style="text-align:center;">Version</th>

                <th style="text-align:center;">Parameter</th>

                <th style="text-align:center;">Remarks</th>

	</tr>
	<tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">Mapping</td>
                <td style="text-align:center;">hisat2</td>
                <td style="text-align:center;">2.2.1</td><td style="text-align:center;">Default</td>
                <td style="text-align:center;">Mapping to a reference</td>
	</tr>
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">Quantification</td>
                <td style="text-align:center;">featureCounts</td>
                <td style="text-align:center;">2.0.6</td>
                <td style="text-align:center;">Default</td>
                <td style="text-align:center;"></td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;vertical-align: middle !important;" rowspan="2">Differential Analysis</td>
                <td style="text-align:center;">DESeq2</td>
                <td style="text-align:center;">1.42.0</td>
                <td style="text-align:center;">|log2(FoldChange)| &gt;= 1 &amp; padj &lt;= 0.05</td>
                <td style="text-align:center;">For sample with bio-replicate</td>
        </tr>
        
        <tr>
                <td style="text-align:center;">edgeR</td>
                <td style="text-align:center;">4.0.16</td>
                <td style="text-align:center;">|log2(FoldChange)| &gt;= 1 &amp; padj &lt;= 0.05</td>
                <td style="text-align:center;">For sample without bio-replicate</td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">Enrichment Analysis</td>
                <td style="text-align:center;">clusterProfiler</td>
                <td style="text-align:center;">4.8.1</td>
                <td style="text-align:center;">padj &lt; 0.05</td>
                <td style="text-align:center;">For GO, KEGG enrichment analysis</td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">GSEA Analysis</td>
                <td style="text-align:center;">gsea</td>
                <td style="text-align:center;">v4.3.2</td>
                <td style="text-align:center;">Default</td>
                <td style="text-align:center;"></td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">Protein-Protein Interaction Analysis</td>
                <td style="text-align:center;">diamond</td>
                <td style="text-align:center;">0.9.14</td>
                <td style="text-align:center;">e-value = 1e-10</td>
                <td style="text-align:center;">Using blast, String database.</td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;">Alternative splicing</td>
                <td style="text-align:center;">rMATS</td>
                <td style="text-align:center;">v4.3.0</td>
                <td style="text-align:center;">Default</td>
                <td style="text-align:center;"></td>
        </tr>
        
        <tr>
                <td style="text-align:center;width: 8em; font-weight: bold;vertical-align: middle !important;" rowspan="2">SNP/InDel Analysis</td>
                <td style="text-align:center;">GATK</td>
                <td style="text-align:center;">4.6.0.0</td>
                <td style="text-align:center;">MQ &lt; 40.0 and QD &lt; 2.0 and FS &gt; 30.0 and DP &lt; 10 and QUAL &lt; 20</td>
                <td style="text-align:center;">SNP/InDel calling</td>
        </tr>
        
        <tr>
                <td style="text-align:center;">snpEff</td>
                <td style="text-align:center;">5.2</td>
                <td style="text-align:center;">Default</td>
                <td style="text-align:center;">SNP/InDel Annotation</td>
        </tr>

        <tr>
            <td style="text-align:center;width: 8em; font-weight: bold;">Fusion gene</td>

            <td style="text-align:center;">Star-fusion</td>

            <td style="text-align:center;">1.12.0</td>

            <td style="text-align:center;">Default</td>

            <td style="text-align:center;">-</td>
	</tr>
		
	</table>

<br/>
<a name="Methods in english" ><h3 id="22">5.2 Methods</h3></a>

<p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>

<p class="paragraph">File catalog: <a href="src/pdf_images/methods_med.pdf" title="点击打开" target="_blank">methods</a> </p>


<a name="Format Description" ><h3 id="22">5.3 Result File Decompression Method and Format Description</h3></a>
<br />
<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>

        <tr><th>Compressed.format</th><th>Customer.type</th><th>Uncompressed.method</th></tr>

        <tr><td rowspan="2">compressed files in the fomat of *.tar:</td><td>Unix/Linux/Mac user</td><td>use tar -xvf *.tar command</td></tr>

        <tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

        <tr><td rowspan="2">compressed files in the format of *.gz:</td><td>Unix/Linux/Mac user</td><td>use gzip –d *.gz command</td></tr>

        <tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

        <tr><td rowspan="2">compressed files in the format of *.zip:</td><td>Unix/Linux/Mac user</td><td>use unzip *.zip command</td></tr>

        <tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>

</table>
<br />
<br />

<table class="tf2">

    <tr>
        <th style="text-align:center;">File.type</th>
        <th style="text-align:center;">Document.description</th>
        <th style="text-align:center;">Open.mode</th>
    </tr>


       <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.readme.pdf</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">Results' construction files that help customers to undersdand all the results files better.</td>
            <td style="text-align:center;">Windows/Mac user use Adobe Reader/Foxit reader/web browser to open.</td>
        </tr>
        <tr>
            <td style="text-align:center;">Unix/Linux user use evince command to open.</td>
        </tr>

       <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.fa/fasta/fna</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">Sequence file, in the fomat of fasta. In genaral, because sequence of gene or genome is large, we provide customer with sampled*.fasta file(partial sequences of *.fasta). It will be more convenient to check file fomat.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use less or more command can view sequences in the fomat of *.fasta.</td>
        </tr>
        <tr>
            <td style="text-align:center;">Windows user use editor Editplus/Notepad++ et al.</td>
        </tr>

        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.fq/fastq</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">Reads sequence file, in the fomat of fastq. In genaral, because sequence of gene or genome is large, we provide customer with sampled*.fastq file ( partial sequences of *.fastq ). It will be more convenient to check file fomat.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use less or more command.</td>
        </tr>
        <tr>
            <td style="text-align:center;">windows users use editor Editplus/Notepad++ et al</td>
       </tr>

        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.txt/xls</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">Table result file; files are separated by(Tab).</td>
            <td style="text-align:center;">unix/Linux/Mac users use less or more commands</td>
        </tr>
       <tr>
            <td style="text-align:center;">windows users use editor Editplus/Notepad++ et al, also can use Microsoft Excel to open.</td>
        </tr>

        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="1">file.pdf/svg</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="1">Figure result files; vectorgraph, can be zoom in or out. It is very convenientfor customer to view or edit. Using Adobe Illustrator to edit,the picture can be used in paper for publishing.</td>
            <td style="text-align:center;">Windows/Mac user use Adobe Reader/Foxit reader/web browser to open；unix/Linux user use evince command to open.</td>
        </tr>
        
        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="1">file.vcf.gz</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="1">This file contains variants in bgzip-compressed VCF (Variant Call Format) format. VCF is a flexible and extendable line-oriented text format developed by the 1000 Genomes Project for releases of SNVs, INDELs, copy number variants and structural variants.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use less or more command</td>
        </tr>

        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.png</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">Image file, bitmap, loseless compression.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use display command.</td>
        </tr>
        <tr>
            <td style="text-align:center;">Windows user use Image browser，such as photoshop et al.</td>
        </tr>
        
        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="1">file.bam</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="1">BAM is compressed format for SAM file which contains sequence alignment information.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use samtools software.</td>
        </tr>
        
        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="1">file.bai/csi</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="1">Index file for bam.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use samtools software.</td>
        </tr>
        
        <tr>
            <td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="2">file.gff/gtf</td>
            <td style="text-align:center;vertical-align: middle !important;" rowspan="2">GFF/GTF is general feature format.</td>
            <td style="text-align:center;">Unix/Linux/Mac user use less or more command;</td>
        </tr>
        <tr>
            <td style="text-align:center;">Windows user use editor Editplus/Notepad++ et al.</td>
        </tr>
</table>








<!-------------------------------------------- 参考文献 ------------------------------------------->

<br/>
<br/>

<h2 id="5"><a id="part5">6 Reference</a></h2>
<br/>


<p class="ref">
Anders S, Huber W. Differential expression analysis for sequence count data[J]. Genome biol, 2010, 11(10): R106.<br /><br/>
Bray N, Pimentel H, Melsted P, et al. Near-optimal RNA-Seq quantification[J]. arXiv preprint arXiv:1505.02710, 2015.<br /><br/>
ENCODE Project Consortium. The ENCODE (ENCyclopedia Of DNA Elements) Project. Science. 2004;306(5696):636-640.<br /><br/>
Garber M, Grabherr M G, Guttman M, et al. Computational methods for transcriptome annotation and quantification using RNA-seq[J]. Nature methods, 2011, 8(6): 469-4.Order_AS.<br /><br/>
Goldstein L D , Cao Y , Pau G , et al. Prediction and Quantification of Splice Events from RNA-Seq Data.[J]. Plos One, 2016, 11(5):e0156132.<br /><br/>
Haas, Brian J.; Dobin, Alexander; Li, Bo; Stransky, Nicolas; Pochet, Nathalie; Regev, Aviv. Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods[J]. Genome Biology, 2019.(STAR-Fusion)<br /><br/>
Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes[J]. Nucleic acids research, 2000, 28(1): 27-30.(KEGG)<br /><br/>
Katz Y, Wang E T, Airoldi E M, et al. Analysis and design of RNA sequencing experiments for identifying isoform regulation[J]. Nature methods, 2010, 7(12): 1009-1015.<br /><br/>
Liao Y1, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomicfeatures.Bioinformatics.2014 ,30(7):923-30.(featureCounts)<br /><br/>
Love M I, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2[J]. Genome biology, 2014, 15(12): 1-21.(DESeq2)<br /><br/>
Mortazavi A, Williams B A, McCue K, et al. Mapping and quantifying mammalian transcriptomes by RNA-Seq[J]. Nature methods, 2008, 5(7): 621-628.<br /><br/>
Patro R, Mount S M, Kingsford C. Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms[J]. Nature biotechnology, 2014, 32(5): 462-464.<br /><br/>
McKenna A, Hanna M, Banks E, et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data[J]. Genome research, 2010, 20(9): 1297-1303.(GATK)<br /><br/>
Parkhomchuk D, Borodina T, Amstislavskiy V, et al. Transcriptome analysis by strand-specific sequencing of complementary DNA[J]. Nucleic acids research, 2009, 37(18): e123-e123.<br /><br/>
Robinson M D, McCarthy D J, Smyth G K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data[J]. Bioinformatics, 2010, 26(1): 139-140.(edgeR)<br /><br/>
Shen S., Park JW., Lu ZX., Lin L., Henry MD., Wu YN., Zhou Q., Xing Y. rMATS: Robust and Flexible Detection of Differential Alternative Splicing from Replicate RNA-Seq Data.(rMATS).<br /><br/>
Trapnell C, Williams BA, Pertea G, et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28(5):511-515.</br></br>
Wang Z, Gerstein M, Snyder M. RNA-Seq: a revolutionary tool for transcriptomics[J]. Nature Reviews Genetics, 2009, 10(1): 57-63.<br /><br/>
Yan L, Yang M, Guo H, et al. Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells. Nat Struct Mol Biol. 2013;20(9):1131-1139.<br /><br/>
Young M D, Wakefield M J, Smyth G K, et al. Method Gene ontology analysis for RNA-seq: accounting for selection bias[J]. Genome Biol, 2010, 11: R14.<br /><br/>
</p>



<!----------------------------------------------- End -------------------------------------------->

</div>

</body>

</html>
