
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >
<html>   

<head>
<TITLE>Shotgun metagenomic short read report</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="icon" href="src/images/bitbug_favicon.ico" type="image/x-icons">

<link rel="stylesheet" href="src/css/jquery-ui.css" />
<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />
<link rel="stylesheet" href="src/css/jquery.tocify.css" />
<link rel="stylesheet" href="src/css/style.css"/>
<link rel="stylesheet" href="src/css/base.css"/>
<link rel="stylesheet" href="src/css/gallery.css"/>
<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>
<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>
<script src="src/js/scrollTop.js" type="text/javascript"></script>
<script type="text/javascript">
	$(function () {
		var options = {
			selectors: "h2,h3,h4,h5",
			theme: "bootstrap",
			scrollTo: "60"
		};
		var toc = $("#toc").tocify(options).data("toc-tocify");
	});
</script>



<script type="text/javascript">
function trans(img_div_id) {
img_div = $("#" + img_div_id)
img = $("#" + img_div_id + " img").first();
if (img_div.hasClass("fullscreen-div")) {
img_div.removeClass("fullscreen-div");
img.removeClass("fullscreen-img");
}
else {
img_div.addClass("fullscreen-div");
img.addClass("fullscreen-img");
}
}
function filter_display() {
return $(this).css("display") != "none";
}
function show_default() {
var list_group_id = "#" + $(this).attr("id");
if ($(list_group_id + " a.active").css("display") == "none") {
var list_tab = $(list_group_id + " a.active");
var pan_id = list_tab.attr("href");
$(pan_id).removeClass("active show");
list_tab.removeClass("active");
}
if ($(list_group_id + " a.active").filter(filter_display).length == 0) {
$(list_group_id + " a").filter(filter_display).first().tab("show");
}
}
$.fn.show_default = show_default
function next(botton_id) {
var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();
while (present_tab.next().length == 1) {
if (present_tab.next().css("display") != "none") {
present_tab.next().tab("show");
break;
}
else {
present_tab = present_tab.next();
}
}
}
function prev(botton_id) {
var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();
while (present_tab.prev().length == 1) {
if (present_tab.prev().css("display") != "none") {
present_tab.prev().tab("show");
break;
}
else {
present_tab = present_tab.prev();
}
}
}
$(document).ready(function () {
$(".form-control").on("keyup", function () {
var value = $(this).val().toLowerCase();
var input_box_id = "#" + $(this).attr("id");
var list_group_id = input_box_id.replace("_InputList", "_list-tab");
$(list_group_id + " a").filter(function () {
$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);
});
$(list_group_id).show_default();
});
$(".list-group").each(show_default);
});
</script>

<script type="text/javascript">
            var pageSize1 = 15;    //每页显示的记录条数
            var curPage1=0;        //当前页
            var lastPage1;        //最后页
            var direct1=0;        //方向
            var len1;            //总行数
            var page1;           //总页数
            var begin1;
            var end1;

            $(document).ready(function display1(){
                len1 =$("#mytable1 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page1=len1 % pageSize1==0 ? len1/pageSize1 : Math.floor(len1/pageSize1)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage1=1;    // 设置当前为第一页
                displayPage1(1);//显示第一页
                document.getElementById("btn01").innerHTML="Current: " + curPage1 + "/" + page1 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl1").innerHTML="Total items: " + len1 + "";        // 显示数据量

                $("#btn11").click(function firstPage1(){    // 首页
                    curPage1=1;
                    direct1 = 0;
                    displayPage1();
                });
                $("#btn21").click(function frontPage1(){    // 上一页
                    direct1=-1;
                    displayPage1();
                });
                $("#btn31").click(function nextPage1(){    // 下一页
                    direct1=1;
                    displayPage1();
                });
                $("#btn41").click(function lastPage1(){    // 尾页
                    curPage1=page1;
                    direct1 = 0;
                    displayPage1();
                });
                $("#btn51").click(function changePage1(){    // 转页
                    curPage1=document.getElementById("changePage1").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage1)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage1 > page1) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct1 = 0;
                    displayPage1();
                });


                $("#pageSizeSet1").click(function setPageSize1(){    // 设置每页显示多少条记录
                    pageSize1 = document.getElementById("pageSize1").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len1 = $("#mytable1 tr").length - 1;
                    page1 = len1 % pageSize1==0 ? len1/pageSize1 : Math.floor(len1/pageSize1)+1;//根据记录条数，计算页数
                    curPage1=1;        //当前页
                    direct1=0;        //方向
                    firstPage1();
                });
            });

            function displayPage1(){
                if(curPage1 <=1 && direct1==-1){
                    direct1=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage1 >= page1 && direct1==1) {
                    direct1=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage1 = curPage1;

                // 修复当len=1时，curPage计算得0的bug
                if (len1 > pageSize1) {
                    curPage1 = ((curPage1 + direct1 + len1) % len1);
                } else {
                    curPage1 = 1;
                }
                document.getElementById("btn01").innerHTML="Current: " + curPage1 + "/" + page1 + " page  | ";        // 显示当前多少页

                begin1=(curPage1-1)*pageSize1 + 1;// 起始记录号
                end1 = begin1 + 1*pageSize1 - 1;    // 末尾记录号


                if(end1 > len1 ) end1=len1;
                $("#mytable1 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable1 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin1 && i<=end1) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize2 = 15;    //每页显示的记录条数
            var curPage2=0;        //当前页
            var lastPage2;        //最后页
            var direct2=0;        //方向
            var len2;            //总行数
            var page2;           //总页数
            var begin2;
            var end2;

            $(document).ready(function display2(){
                len2 =$("#mytable2 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page2=len2 % pageSize2==0 ? len2/pageSize2 : Math.floor(len2/pageSize2)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage2=1;    // 设置当前为第一页
                displayPage2(1);//显示第一页
                document.getElementById("btn02").innerHTML="Current: " + curPage2 + "/" + page2 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl2").innerHTML="Total items: " + len2 + "";        // 显示数据量

                $("#btn12").click(function firstPage2(){    // 首页
                    curPage2=1;
                    direct2 = 0;
                    displayPage2();
                });
                $("#btn22").click(function frontPage2(){    // 上一页
                    direct2=-1;
                    displayPage2();
                });
                $("#btn32").click(function nextPage2(){    // 下一页
                    direct2=1;
                    displayPage2();
                });
                $("#btn42").click(function lastPage2(){    // 尾页
                    curPage2=page2;
                    direct2 = 0;
                    displayPage2();
                });
                $("#btn52").click(function changePage2(){    // 转页
                    curPage2=document.getElementById("changePage2").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage2)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage2 > page2) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct2 = 0;
                    displayPage2();
                });


                $("#pageSizeSet2").click(function setPageSize2(){    // 设置每页显示多少条记录
                    pageSize2 = document.getElementById("pageSize2").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len2 = $("#mytable2 tr").length - 1;
                    page2 = len2 % pageSize2==0 ? len2/pageSize2 : Math.floor(len2/pageSize2)+1;//根据记录条数，计算页数
                    curPage2=1;        //当前页
                    direct2=0;        //方向
                    firstPage2();
                });
            });

            function displayPage2(){
                if(curPage2 <=1 && direct2==-1){
                    direct2=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage2 >= page2 && direct2==1) {
                    direct2=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage2 = curPage2;

                // 修复当len=1时，curPage计算得0的bug
                if (len2 > pageSize2) {
                    curPage2 = ((curPage2 + direct2 + len2) % len2);
                } else {
                    curPage2 = 1;
                }
                document.getElementById("btn02").innerHTML="Current: " + curPage2 + "/" + page2 + " page  | ";        // 显示当前多少页

                begin2=(curPage2-1)*pageSize2 + 1;// 起始记录号
                end2 = begin2 + 1*pageSize2 - 1;    // 末尾记录号


                if(end2 > len2 ) end2=len2;
                $("#mytable2 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable2 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin2 && i<=end2) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize3 = 15;    //每页显示的记录条数
            var curPage3=0;        //当前页
            var lastPage3;        //最后页
            var direct3=0;        //方向
            var len3;            //总行数
            var page3;           //总页数
            var begin3;
            var end3;

            $(document).ready(function display3(){
                len3 =$("#mytable3 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page3=len3 % pageSize3==0 ? len3/pageSize3 : Math.floor(len3/pageSize3)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage3=1;    // 设置当前为第一页
                displayPage3(1);//显示第一页
                document.getElementById("btn03").innerHTML="Current: " + curPage3 + "/" + page3 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl3").innerHTML="Total items: " + len3 + "";        // 显示数据量

                $("#btn13").click(function firstPage3(){    // 首页
                    curPage3=1;
                    direct3 = 0;
                    displayPage3();
                });
                $("#btn23").click(function frontPage3(){    // 上一页
                    direct3=-1;
                    displayPage3();
                });
                $("#btn33").click(function nextPage3(){    // 下一页
                    direct3=1;
                    displayPage3();
                });
                $("#btn43").click(function lastPage3(){    // 尾页
                    curPage3=page3;
                    direct3 = 0;
                    displayPage3();
                });
                $("#btn53").click(function changePage3(){    // 转页
                    curPage3=document.getElementById("changePage3").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage3)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage3 > page3) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct3 = 0;
                    displayPage3();
                });


                $("#pageSizeSet3").click(function setPageSize3(){    // 设置每页显示多少条记录
                    pageSize3 = document.getElementById("pageSize3").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len3 = $("#mytable3 tr").length - 1;
                    page3 = len3 % pageSize3==0 ? len3/pageSize3 : Math.floor(len3/pageSize3)+1;//根据记录条数，计算页数
                    curPage3=1;        //当前页
                    direct3=0;        //方向
                    firstPage3();
                });
            });

            function displayPage3(){
                if(curPage3 <=1 && direct3==-1){
                    direct3=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage3 >= page3 && direct3==1) {
                    direct3=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage3 = curPage3;

                // 修复当len=1时，curPage计算得0的bug
                if (len3 > pageSize3) {
                    curPage3 = ((curPage3 + direct3 + len3) % len3);
                } else {
                    curPage3 = 1;
                }
                document.getElementById("btn03").innerHTML="Current: " + curPage3 + "/" + page3 + " page  | ";        // 显示当前多少页

                begin3=(curPage3-1)*pageSize3 + 1;// 起始记录号
                end3 = begin3 + 1*pageSize3 - 1;    // 末尾记录号


                if(end3 > len3 ) end3=len3;
                $("#mytable3 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable3 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin3 && i<=end3) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize4 = 15;    //每页显示的记录条数
            var curPage4=0;        //当前页
            var lastPage4;        //最后页
            var direct4=0;        //方向
            var len4;            //总行数
            var page4;           //总页数
            var begin4;
            var end4;

            $(document).ready(function display4(){
                len4 =$("#mytable4 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page4=len4 % pageSize4==0 ? len4/pageSize4 : Math.floor(len4/pageSize4)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage4=1;    // 设置当前为第一页
                displayPage4(1);//显示第一页
                document.getElementById("btn04").innerHTML="Current: " + curPage4 + "/" + page4 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl4").innerHTML="Total items: " + len4 + "";        // 显示数据量

                $("#btn14").click(function firstPage4(){    // 首页
                    curPage4=1;
                    direct4 = 0;
                    displayPage4();
                });
                $("#btn24").click(function frontPage4(){    // 上一页
                    direct4=-1;
                    displayPage4();
                });
                $("#btn34").click(function nextPage4(){    // 下一页
                    direct4=1;
                    displayPage4();
                });
                $("#btn44").click(function lastPage4(){    // 尾页
                    curPage4=page4;
                    direct4 = 0;
                    displayPage4();
                });
                $("#btn54").click(function changePage4(){    // 转页
                    curPage4=document.getElementById("changePage4").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage4)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage4 > page4) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct4 = 0;
                    displayPage4();
                });


                $("#pageSizeSet4").click(function setPageSize4(){    // 设置每页显示多少条记录
                    pageSize4 = document.getElementById("pageSize4").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len4 = $("#mytable4 tr").length - 1;
                    page4 = len4 % pageSize4==0 ? len4/pageSize4 : Math.floor(len4/pageSize4)+1;//根据记录条数，计算页数
                    curPage4=1;        //当前页
                    direct4=0;        //方向
                    firstPage4();
                });
            });

            function displayPage4(){
                if(curPage4 <=1 && direct4==-1){
                    direct4=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage4 >= page4 && direct4==1) {
                    direct4=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage4 = curPage4;

                // 修复当len=1时，curPage计算得0的bug
                if (len4 > pageSize4) {
                    curPage4 = ((curPage4 + direct4 + len4) % len4);
                } else {
                    curPage4 = 1;
                }
                document.getElementById("btn04").innerHTML="Current: " + curPage4 + "/" + page4 + " page  | ";        // 显示当前多少页

                begin4=(curPage4-1)*pageSize4 + 1;// 起始记录号
                end4 = begin4 + 1*pageSize4 - 1;    // 末尾记录号


                if(end4 > len4 ) end4=len4;
                $("#mytable4 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable4 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin4 && i<=end4) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize5 = 15;    //每页显示的记录条数
            var curPage5=0;        //当前页
            var lastPage5;        //最后页
            var direct5=0;        //方向
            var len5;            //总行数
            var page5;           //总页数
            var begin5;
            var end5;

            $(document).ready(function display5(){
                len5 =$("#mytable5 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page5=len5 % pageSize5==0 ? len5/pageSize5 : Math.floor(len5/pageSize5)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage5=1;    // 设置当前为第一页
                displayPage5(1);//显示第一页
                document.getElementById("btn05").innerHTML="Current: " + curPage5 + "/" + page5 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl5").innerHTML="Total items: " + len5 + "";        // 显示数据量

                $("#btn15").click(function firstPage5(){    // 首页
                    curPage5=1;
                    direct5 = 0;
                    displayPage5();
                });
                $("#btn25").click(function frontPage5(){    // 上一页
                    direct5=-1;
                    displayPage5();
                });
                $("#btn35").click(function nextPage5(){    // 下一页
                    direct5=1;
                    displayPage5();
                });
                $("#btn45").click(function lastPage5(){    // 尾页
                    curPage5=page5;
                    direct5 = 0;
                    displayPage5();
                });
                $("#btn55").click(function changePage5(){    // 转页
                    curPage5=document.getElementById("changePage5").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage5)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage5 > page5) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct5 = 0;
                    displayPage5();
                });


                $("#pageSizeSet5").click(function setPageSize5(){    // 设置每页显示多少条记录
                    pageSize5 = document.getElementById("pageSize5").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len5 = $("#mytable5 tr").length - 1;
                    page5 = len5 % pageSize5==0 ? len5/pageSize5 : Math.floor(len5/pageSize5)+1;//根据记录条数，计算页数
                    curPage5=1;        //当前页
                    direct5=0;        //方向
                    firstPage5();
                });
            });

            function displayPage5(){
                if(curPage5 <=1 && direct5==-1){
                    direct5=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage5 >= page5 && direct5==1) {
                    direct5=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage5 = curPage5;

                // 修复当len=1时，curPage计算得0的bug
                if (len5 > pageSize5) {
                    curPage5 = ((curPage5 + direct5 + len5) % len5);
                } else {
                    curPage5 = 1;
                }
                document.getElementById("btn05").innerHTML="Current: " + curPage5 + "/" + page5 + " page  | ";        // 显示当前多少页

                begin5=(curPage5-1)*pageSize5 + 1;// 起始记录号
                end5 = begin5 + 1*pageSize5 - 1;    // 末尾记录号


                if(end5 > len5 ) end5=len5;
                $("#mytable5 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable5 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin5 && i<=end5) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize6 = 15;    //每页显示的记录条数
            var curPage6=0;        //当前页
            var lastPage6;        //最后页
            var direct6=0;        //方向
            var len6;            //总行数
            var page6;           //总页数
            var begin6;
            var end6;

            $(document).ready(function display6(){
                len6 =$("#mytable6 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page6=len6 % pageSize6==0 ? len6/pageSize6 : Math.floor(len6/pageSize6)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage6=1;    // 设置当前为第一页
                displayPage6(1);//显示第一页
                document.getElementById("btn06").innerHTML="Current: " + curPage6 + "/" + page6 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl6").innerHTML="Total items: " + len6 + "";        // 显示数据量

                $("#btn16").click(function firstPage6(){    // 首页
                    curPage6=1;
                    direct6 = 0;
                    displayPage6();
                });
                $("#btn26").click(function frontPage6(){    // 上一页
                    direct6=-1;
                    displayPage6();
                });
                $("#btn36").click(function nextPage6(){    // 下一页
                    direct6=1;
                    displayPage6();
                });
                $("#btn46").click(function lastPage6(){    // 尾页
                    curPage6=page6;
                    direct6 = 0;
                    displayPage6();
                });
                $("#btn56").click(function changePage6(){    // 转页
                    curPage6=document.getElementById("changePage6").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage6)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage6 > page6) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct6 = 0;
                    displayPage6();
                });


                $("#pageSizeSet6").click(function setPageSize6(){    // 设置每页显示多少条记录
                    pageSize6 = document.getElementById("pageSize6").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len6 = $("#mytable6 tr").length - 1;
                    page6 = len6 % pageSize6==0 ? len6/pageSize6 : Math.floor(len6/pageSize6)+1;//根据记录条数，计算页数
                    curPage6=1;        //当前页
                    direct6=0;        //方向
                    firstPage6();
                });
            });

            function displayPage6(){
                if(curPage6 <=1 && direct6==-1){
                    direct6=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage6 >= page6 && direct6==1) {
                    direct6=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage6 = curPage6;

                // 修复当len=1时，curPage计算得0的bug
                if (len6 > pageSize6) {
                    curPage6 = ((curPage6 + direct6 + len6) % len6);
                } else {
                    curPage6 = 1;
                }
                document.getElementById("btn06").innerHTML="Current: " + curPage6 + "/" + page6 + " page  | ";        // 显示当前多少页

                begin6=(curPage6-1)*pageSize6 + 1;// 起始记录号
                end6 = begin6 + 1*pageSize6 - 1;    // 末尾记录号


                if(end6 > len6 ) end6=len6;
                $("#mytable6 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable6 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin6 && i<=end6) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize7 = 15;    //每页显示的记录条数
            var curPage7=0;        //当前页
            var lastPage7;        //最后页
            var direct7=0;        //方向
            var len7;            //总行数
            var page7;           //总页数
            var begin7;
            var end7;

            $(document).ready(function display7(){
                len7 =$("#mytable7 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page7=len7 % pageSize7==0 ? len7/pageSize7 : Math.floor(len7/pageSize7)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage7=1;    // 设置当前为第一页
                displayPage7(1);//显示第一页
                document.getElementById("btn07").innerHTML="Current: " + curPage7 + "/" + page7 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl7").innerHTML="Total items: " + len7 + "";        // 显示数据量

                $("#btn17").click(function firstPage7(){    // 首页
                    curPage7=1;
                    direct7 = 0;
                    displayPage7();
                });
                $("#btn27").click(function frontPage7(){    // 上一页
                    direct7=-1;
                    displayPage7();
                });
                $("#btn37").click(function nextPage7(){    // 下一页
                    direct7=1;
                    displayPage7();
                });
                $("#btn47").click(function lastPage7(){    // 尾页
                    curPage7=page7;
                    direct7 = 0;
                    displayPage7();
                });
                $("#btn57").click(function changePage7(){    // 转页
                    curPage7=document.getElementById("changePage7").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage7)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage7 > page7) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct7 = 0;
                    displayPage7();
                });


                $("#pageSizeSet7").click(function setPageSize7(){    // 设置每页显示多少条记录
                    pageSize7 = document.getElementById("pageSize7").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len7 = $("#mytable7 tr").length - 1;
                    page7 = len7 % pageSize7==0 ? len7/pageSize7 : Math.floor(len7/pageSize7)+1;//根据记录条数，计算页数
                    curPage7=1;        //当前页
                    direct7=0;        //方向
                    firstPage7();
                });
            });

            function displayPage7(){
                if(curPage7 <=1 && direct7==-1){
                    direct7=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage7 >= page7 && direct7==1) {
                    direct7=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage7 = curPage7;

                // 修复当len=1时，curPage计算得0的bug
                if (len7 > pageSize7) {
                    curPage7 = ((curPage7 + direct7 + len7) % len7);
                } else {
                    curPage7 = 1;
                }
                document.getElementById("btn07").innerHTML="Current: " + curPage7 + "/" + page7 + " page  | ";        // 显示当前多少页

                begin7=(curPage7-1)*pageSize7 + 1;// 起始记录号
                end7 = begin7 + 1*pageSize7 - 1;    // 末尾记录号


                if(end7 > len7 ) end7=len7;
                $("#mytable7 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable7 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin7 && i<=end7) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize8 = 15;    //每页显示的记录条数
            var curPage8=0;        //当前页
            var lastPage8;        //最后页
            var direct8=0;        //方向
            var len8;            //总行数
            var page8;           //总页数
            var begin8;
            var end8;

            $(document).ready(function display8(){
                len8 =$("#mytable8 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page8=len8 % pageSize8==0 ? len8/pageSize8 : Math.floor(len8/pageSize8)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage8=1;    // 设置当前为第一页
                displayPage8(1);//显示第一页
                document.getElementById("btn08").innerHTML="Current: " + curPage8 + "/" + page8 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl8").innerHTML="Total items: " + len8 + "";        // 显示数据量

                $("#btn18").click(function firstPage8(){    // 首页
                    curPage8=1;
                    direct8 = 0;
                    displayPage8();
                });
                $("#btn28").click(function frontPage8(){    // 上一页
                    direct8=-1;
                    displayPage8();
                });
                $("#btn38").click(function nextPage8(){    // 下一页
                    direct8=1;
                    displayPage8();
                });
                $("#btn48").click(function lastPage8(){    // 尾页
                    curPage8=page8;
                    direct8 = 0;
                    displayPage8();
                });
                $("#btn58").click(function changePage8(){    // 转页
                    curPage8=document.getElementById("changePage8").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage8)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage8 > page8) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct8 = 0;
                    displayPage8();
                });


                $("#pageSizeSet8").click(function setPageSize8(){    // 设置每页显示多少条记录
                    pageSize8 = document.getElementById("pageSize8").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len8 = $("#mytable8 tr").length - 1;
                    page8 = len8 % pageSize8==0 ? len8/pageSize8 : Math.floor(len8/pageSize8)+1;//根据记录条数，计算页数
                    curPage8=1;        //当前页
                    direct8=0;        //方向
                    firstPage8();
                });
            });

            function displayPage8(){
                if(curPage8 <=1 && direct8==-1){
                    direct8=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage8 >= page8 && direct8==1) {
                    direct8=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage8 = curPage8;

                // 修复当len=1时，curPage计算得0的bug
                if (len8 > pageSize8) {
                    curPage8 = ((curPage8 + direct8 + len8) % len8);
                } else {
                    curPage8 = 1;
                }
                document.getElementById("btn08").innerHTML="Current: " + curPage8 + "/" + page8 + " page  | ";        // 显示当前多少页

                begin8=(curPage8-1)*pageSize8 + 1;// 起始记录号
                end8 = begin8 + 1*pageSize8 - 1;    // 末尾记录号


                if(end8 > len8 ) end8=len8;
                $("#mytable8 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable8 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin8 && i<=end8) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize9 = 15;    //每页显示的记录条数
            var curPage9=0;        //当前页
            var lastPage9;        //最后页
            var direct9=0;        //方向
            var len9;            //总行数
            var page9;           //总页数
            var begin9;
            var end9;

            $(document).ready(function display9(){
                len9 =$("#mytable9 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page9=len9 % pageSize9==0 ? len9/pageSize9 : Math.floor(len9/pageSize9)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage9=1;    // 设置当前为第一页
                displayPage9(1);//显示第一页
                document.getElementById("btn09").innerHTML="Current: " + curPage9 + "/" + page9 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl9").innerHTML="Total items: " + len9 + "";        // 显示数据量

                $("#btn19").click(function firstPage9(){    // 首页
                    curPage9=1;
                    direct9 = 0;
                    displayPage9();
                });
                $("#btn29").click(function frontPage9(){    // 上一页
                    direct9=-1;
                    displayPage9();
                });
                $("#btn39").click(function nextPage9(){    // 下一页
                    direct9=1;
                    displayPage9();
                });
                $("#btn49").click(function lastPage9(){    // 尾页
                    curPage9=page9;
                    direct9 = 0;
                    displayPage9();
                });
                $("#btn59").click(function changePage9(){    // 转页
                    curPage9=document.getElementById("changePage9").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage9)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage9 > page9) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct9 = 0;
                    displayPage9();
                });


                $("#pageSizeSet9").click(function setPageSize9(){    // 设置每页显示多少条记录
                    pageSize9 = document.getElementById("pageSize9").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len9 = $("#mytable9 tr").length - 1;
                    page9 = len9 % pageSize9==0 ? len9/pageSize9 : Math.floor(len9/pageSize9)+1;//根据记录条数，计算页数
                    curPage9=1;        //当前页
                    direct9=0;        //方向
                    firstPage9();
                });
            });

            function displayPage9(){
                if(curPage9 <=1 && direct9==-1){
                    direct9=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage9 >= page9 && direct9==1) {
                    direct9=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage9 = curPage9;

                // 修复当len=1时，curPage计算得0的bug
                if (len9 > pageSize9) {
                    curPage9 = ((curPage9 + direct9 + len9) % len9);
                } else {
                    curPage9 = 1;
                }
                document.getElementById("btn09").innerHTML="Current: " + curPage9 + "/" + page9 + " page  | ";        // 显示当前多少页

                begin9=(curPage9-1)*pageSize9 + 1;// 起始记录号
                end9 = begin9 + 1*pageSize9 - 1;    // 末尾记录号


                if(end9 > len9 ) end9=len9;
                $("#mytable9 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable9 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin9 && i<=end9) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize10 = 15;    //每页显示的记录条数
            var curPage10=0;        //当前页
            var lastPage10;        //最后页
            var direct10=0;        //方向
            var len10;            //总行数
            var page10;           //总页数
            var begin10;
            var end10;

            $(document).ready(function display10(){
                len10 =$("#mytable10 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page10=len10 % pageSize10==0 ? len10/pageSize10 : Math.floor(len10/pageSize10)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage10=1;    // 设置当前为第一页
                displayPage10(1);//显示第一页
                document.getElementById("btn010").innerHTML="Current: " + curPage10 + "/" + page10 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl10").innerHTML="Total items: " + len10 + "";        // 显示数据量

                $("#btn110").click(function firstPage10(){    // 首页
                    curPage10=1;
                    direct10 = 0;
                    displayPage10();
                });
                $("#btn210").click(function frontPage10(){    // 上一页
                    direct10=-1;
                    displayPage10();
                });
                $("#btn310").click(function nextPage10(){    // 下一页
                    direct10=1;
                    displayPage10();
                });
                $("#btn410").click(function lastPage10(){    // 尾页
                    curPage10=page10;
                    direct10 = 0;
                    displayPage10();
                });
                $("#btn510").click(function changePage10(){    // 转页
                    curPage10=document.getElementById("changePage10").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage10)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage10 > page10) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct10 = 0;
                    displayPage10();
                });


                $("#pageSizeSet10").click(function setPageSize10(){    // 设置每页显示多少条记录
                    pageSize10 = document.getElementById("pageSize10").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len10 = $("#mytable10 tr").length - 1;
                    page10 = len10 % pageSize10==0 ? len10/pageSize10 : Math.floor(len10/pageSize10)+1;//根据记录条数，计算页数
                    curPage10=1;        //当前页
                    direct10=0;        //方向
                    firstPage10();
                });
            });

            function displayPage10(){
                if(curPage10 <=1 && direct10==-1){
                    direct10=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage10 >= page10 && direct10==1) {
                    direct10=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage10 = curPage10;

                // 修复当len=1时，curPage计算得0的bug
                if (len10 > pageSize10) {
                    curPage10 = ((curPage10 + direct10 + len10) % len10);
                } else {
                    curPage10 = 1;
                }
                document.getElementById("btn010").innerHTML="Current: " + curPage10 + "/" + page10 + " page  | ";        // 显示当前多少页

                begin10=(curPage10-1)*pageSize10 + 1;// 起始记录号
                end10 = begin10 + 1*pageSize10 - 1;    // 末尾记录号


                if(end10 > len10 ) end10=len10;
                $("#mytable10 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable10 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin10 && i<=end10) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize11 = 15;    //每页显示的记录条数
            var curPage11=0;        //当前页
            var lastPage11;        //最后页
            var direct11=0;        //方向
            var len11;            //总行数
            var page11;           //总页数
            var begin11;
            var end11;

            $(document).ready(function display11(){
                len11 =$("#mytable11 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page11=len11 % pageSize11==0 ? len11/pageSize11 : Math.floor(len11/pageSize11)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage11=1;    // 设置当前为第一页
                displayPage11(1);//显示第一页
                document.getElementById("btn011").innerHTML="Current: " + curPage11 + "/" + page11 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl11").innerHTML="Total items: " + len11 + "";        // 显示数据量

                $("#btn111").click(function firstPage11(){    // 首页
                    curPage11=1;
                    direct11 = 0;
                    displayPage11();
                });
                $("#btn211").click(function frontPage11(){    // 上一页
                    direct11=-1;
                    displayPage11();
                });
                $("#btn311").click(function nextPage11(){    // 下一页
                    direct11=1;
                    displayPage11();
                });
                $("#btn411").click(function lastPage11(){    // 尾页
                    curPage11=page11;
                    direct11 = 0;
                    displayPage11();
                });
                $("#btn511").click(function changePage11(){    // 转页
                    curPage11=document.getElementById("changePage11").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage11)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage11 > page11) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct11 = 0;
                    displayPage11();
                });


                $("#pageSizeSet11").click(function setPageSize11(){    // 设置每页显示多少条记录
                    pageSize11 = document.getElementById("pageSize11").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len11 = $("#mytable11 tr").length - 1;
                    page11 = len11 % pageSize11==0 ? len11/pageSize11 : Math.floor(len11/pageSize11)+1;//根据记录条数，计算页数
                    curPage11=1;        //当前页
                    direct11=0;        //方向
                    firstPage11();
                });
            });

            function displayPage11(){
                if(curPage11 <=1 && direct11==-1){
                    direct11=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage11 >= page11 && direct11==1) {
                    direct11=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage11 = curPage11;

                // 修复当len=1时，curPage计算得0的bug
                if (len11 > pageSize11) {
                    curPage11 = ((curPage11 + direct11 + len11) % len11);
                } else {
                    curPage11 = 1;
                }
                document.getElementById("btn011").innerHTML="Current: " + curPage11 + "/" + page11 + " page  | ";        // 显示当前多少页

                begin11=(curPage11-1)*pageSize11 + 1;// 起始记录号
                end11 = begin11 + 1*pageSize11 - 1;    // 末尾记录号


                if(end11 > len11 ) end11=len11;
                $("#mytable11 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable11 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin11 && i<=end11) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize12 = 15;    //每页显示的记录条数
            var curPage12=0;        //当前页
            var lastPage12;        //最后页
            var direct12=0;        //方向
            var len12;            //总行数
            var page12;           //总页数
            var begin12;
            var end12;

            $(document).ready(function display12(){
                len12 =$("#mytable12 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page12=len12 % pageSize12==0 ? len12/pageSize12 : Math.floor(len12/pageSize12)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage12=1;    // 设置当前为第一页
                displayPage12(1);//显示第一页
                document.getElementById("btn012").innerHTML="Current: " + curPage12 + "/" + page12 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl12").innerHTML="Total items: " + len12 + "";        // 显示数据量

                $("#btn112").click(function firstPage12(){    // 首页
                    curPage12=1;
                    direct12 = 0;
                    displayPage12();
                });
                $("#btn212").click(function frontPage12(){    // 上一页
                    direct12=-1;
                    displayPage12();
                });
                $("#btn312").click(function nextPage12(){    // 下一页
                    direct12=1;
                    displayPage12();
                });
                $("#btn412").click(function lastPage12(){    // 尾页
                    curPage12=page12;
                    direct12 = 0;
                    displayPage12();
                });
                $("#btn512").click(function changePage12(){    // 转页
                    curPage12=document.getElementById("changePage12").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage12)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage12 > page12) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct12 = 0;
                    displayPage12();
                });


                $("#pageSizeSet12").click(function setPageSize12(){    // 设置每页显示多少条记录
                    pageSize12 = document.getElementById("pageSize12").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len12 = $("#mytable12 tr").length - 1;
                    page12 = len12 % pageSize12==0 ? len12/pageSize12 : Math.floor(len12/pageSize12)+1;//根据记录条数，计算页数
                    curPage12=1;        //当前页
                    direct12=0;        //方向
                    firstPage12();
                });
            });

            function displayPage12(){
                if(curPage12 <=1 && direct12==-1){
                    direct12=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage12 >= page12 && direct12==1) {
                    direct12=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage12 = curPage12;

                // 修复当len=1时，curPage计算得0的bug
                if (len12 > pageSize12) {
                    curPage12 = ((curPage12 + direct12 + len12) % len12);
                } else {
                    curPage12 = 1;
                }
                document.getElementById("btn012").innerHTML="Current: " + curPage12 + "/" + page12 + " page  | ";        // 显示当前多少页

                begin12=(curPage12-1)*pageSize12 + 1;// 起始记录号
                end12 = begin12 + 1*pageSize12 - 1;    // 末尾记录号


                if(end12 > len12 ) end12=len12;
                $("#mytable12 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable12 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin12 && i<=end12) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize13 = 15;    //每页显示的记录条数
            var curPage13=0;        //当前页
            var lastPage13;        //最后页
            var direct13=0;        //方向
            var len13;            //总行数
            var page13;           //总页数
            var begin13;
            var end13;

            $(document).ready(function display13(){
                len13 =$("#mytable13 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page13=len13 % pageSize13==0 ? len13/pageSize13 : Math.floor(len13/pageSize13)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage13=1;    // 设置当前为第一页
                displayPage13(1);//显示第一页
                document.getElementById("btn013").innerHTML="Current: " + curPage13 + "/" + page13 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl13").innerHTML="Total items: " + len13 + "";        // 显示数据量

                $("#btn113").click(function firstPage13(){    // 首页
                    curPage13=1;
                    direct13 = 0;
                    displayPage13();
                });
                $("#btn213").click(function frontPage13(){    // 上一页
                    direct13=-1;
                    displayPage13();
                });
                $("#btn313").click(function nextPage13(){    // 下一页
                    direct13=1;
                    displayPage13();
                });
                $("#btn413").click(function lastPage13(){    // 尾页
                    curPage13=page13;
                    direct13 = 0;
                    displayPage13();
                });
                $("#btn513").click(function changePage13(){    // 转页
                    curPage13=document.getElementById("changePage13").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage13)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage13 > page13) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct13 = 0;
                    displayPage13();
                });


                $("#pageSizeSet13").click(function setPageSize13(){    // 设置每页显示多少条记录
                    pageSize13 = document.getElementById("pageSize13").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len13 = $("#mytable13 tr").length - 1;
                    page13 = len13 % pageSize13==0 ? len13/pageSize13 : Math.floor(len13/pageSize13)+1;//根据记录条数，计算页数
                    curPage13=1;        //当前页
                    direct13=0;        //方向
                    firstPage13();
                });
            });

            function displayPage13(){
                if(curPage13 <=1 && direct13==-1){
                    direct13=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage13 >= page13 && direct13==1) {
                    direct13=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage13 = curPage13;

                // 修复当len=1时，curPage计算得0的bug
                if (len13 > pageSize13) {
                    curPage13 = ((curPage13 + direct13 + len13) % len13);
                } else {
                    curPage13 = 1;
                }
                document.getElementById("btn013").innerHTML="Current: " + curPage13 + "/" + page13 + " page  | ";        // 显示当前多少页

                begin13=(curPage13-1)*pageSize13 + 1;// 起始记录号
                end13 = begin13 + 1*pageSize13 - 1;    // 末尾记录号


                if(end13 > len13 ) end13=len13;
                $("#mytable13 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable13 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin13 && i<=end13) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize14 = 15;    //每页显示的记录条数
            var curPage14=0;        //当前页
            var lastPage14;        //最后页
            var direct14=0;        //方向
            var len14;            //总行数
            var page14;           //总页数
            var begin14;
            var end14;

            $(document).ready(function display14(){
                len14 =$("#mytable14 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page14=len14 % pageSize14==0 ? len14/pageSize14 : Math.floor(len14/pageSize14)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage14=1;    // 设置当前为第一页
                displayPage14(1);//显示第一页
                document.getElementById("btn014").innerHTML="Current: " + curPage14 + "/" + page14 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl14").innerHTML="Total items: " + len14 + "";        // 显示数据量

                $("#btn114").click(function firstPage14(){    // 首页
                    curPage14=1;
                    direct14 = 0;
                    displayPage14();
                });
                $("#btn214").click(function frontPage14(){    // 上一页
                    direct14=-1;
                    displayPage14();
                });
                $("#btn314").click(function nextPage14(){    // 下一页
                    direct14=1;
                    displayPage14();
                });
                $("#btn414").click(function lastPage14(){    // 尾页
                    curPage14=page14;
                    direct14 = 0;
                    displayPage14();
                });
                $("#btn514").click(function changePage14(){    // 转页
                    curPage14=document.getElementById("changePage14").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage14)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage14 > page14) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct14 = 0;
                    displayPage14();
                });


                $("#pageSizeSet14").click(function setPageSize14(){    // 设置每页显示多少条记录
                    pageSize14 = document.getElementById("pageSize14").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len14 = $("#mytable14 tr").length - 1;
                    page14 = len14 % pageSize14==0 ? len14/pageSize14 : Math.floor(len14/pageSize14)+1;//根据记录条数，计算页数
                    curPage14=1;        //当前页
                    direct14=0;        //方向
                    firstPage14();
                });
            });

            function displayPage14(){
                if(curPage14 <=1 && direct14==-1){
                    direct14=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage14 >= page14 && direct14==1) {
                    direct14=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage14 = curPage14;

                // 修复当len=1时，curPage计算得0的bug
                if (len14 > pageSize14) {
                    curPage14 = ((curPage14 + direct14 + len14) % len14);
                } else {
                    curPage14 = 1;
                }
                document.getElementById("btn014").innerHTML="Current: " + curPage14 + "/" + page14 + " page  | ";        // 显示当前多少页

                begin14=(curPage14-1)*pageSize14 + 1;// 起始记录号
                end14 = begin14 + 1*pageSize14 - 1;    // 末尾记录号


                if(end14 > len14 ) end14=len14;
                $("#mytable14 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable14 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin14 && i<=end14) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>



</head>

<body>
<div id="toc" class="tocify">
</div>
	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">
		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"
			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
			<tr height="86%">
				<td width='1%'></td>
				<td><img src="src/images/logo1.png"></td>
				<td>
					<div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>
				</td>
				<td width="20%"></td>
				<td width="15%"></td>
				<td width="25%"
					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">
					<br><a href="mailto:op-rna@novogene.com" target="black"
						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">
						<a></td>
				<td width="1%"></td>
			</tr>
		</table>
	</div>
</div>  
    
<!--------------------------标题---------------------------------->
<div id="main">
<br><h1>Shotgun metagenomic short read report</h1></br>
<table class="gy">
<tr>
<td>Contract ID</td>
<td style="text-align :left">HXXXXXXXX</td>
</tr>
<tr>
<td>Contract Name</td>
<td style="text-align :left">XXXXXXXXXXXXXXXXXXXXXXXX</td>
</tr>
<tr>
<td>Batch ID</td>
<td style="text-align :left">X101SCXXXXXXXX-XXX-XXXX</td>
</tr>
<tr>
<td>Report Time</td>
<td style="text-align :left">2024-10-30</td>
</tr>
<tr>
<td>Tel</td>
<td style="text-align :left">400-658-1585</td>
</tr>
<tr>
<td>Mail</td>
<td style="text-align :left">service-meta@novogene.com</td>
</tr>
<tr>
</table>
<br /><br />
<hr />
</p><br />
<h2>1 Introduction</h2>


    <p class="paragraph">Diverse communities of microbes thrive in environments ranging from the human gut (Walter et al., 2011), to the mud in deep sea, to conventionally inhospitable habitats such as acid mine runoff and geothermal hot springs. Environmental DNA sequencing has revealed the expansive biodiversity of microorganisms and clarified the relationship between host-associated microbial communities and host phenotype. Studies of cultured microbes reveal that they are critical components of these environments and provide essential ecosystem services (van der Heijden et al., 2008). Unfortunately, most microbes are notoriously difficult to culture in the laboratory. Therefore, vast amounts of organismal and functional novelty remain to be explored.</p>

    <p class="paragraph">Metagenomics, or the study of microbial communities, takes advantage of advances in sequencing technology and analysis methods to comprehensively examine microbial communities directly from their natural habitats, which potentially may reveal novel content (Handelsman et al., 1998). Metagenomic sequencing provides insights into species biodiversity, functions, and pathways present in variant microbiomes.</p>

    <p class="paragraph">With the rapid development of sequencing and informatics technologies, metagenomic studies with Next Generation Sequencing (NGS) is a fundamental strategy to study community diversity and their characteristics. Metagenomic sequencing with NGS is also notable for the ability to get tremendous data and abundant information (Tringe et al., 2005; Raes et al., 2007). NGS has been utilized in more and more far-reaching projects, like the Human Microbiome Project (HMP) and Earth Microbiome Project (EMP). The workflow is shown as follows:</p>

<p class="center">
<img src="src/images/experimentWorkflow_en.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 1 Project workflow</strong></p>  
</p>

        <br />

        <br />

<hr />
</p><br />
<h2>2 Library Construction and Sequencing</h2>


    <a name="T4"></a>
    <h4>2.1 Sample Quality Control</h4>    

    <p class="paragraph">Please refer to QC report for methods of sample quality control.</p>

    <a name="T4"></a>
    <h4>2.2 Library Construction, Quality Control, and Sequencing</h4>    

    <p class="paragraph">The genomic DNA was randomly sheared into shorter fragments. The obtained fragments were then end-repaired, A-tailed, and further ligated with sequencing adapters. The resulting fragments with adapters were size selected and PCR amplified, unless otherwise specified as PCR-free, before proceeding for purification.</p>

    <p class="paragraph">The experimental procedures of DNA library preparation are shown in Figure 2.</p>

<p class="center">
<img src="src/images/experiment_pipeline_en.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 2 Workflow of library construction</strong></p>  
</p>

    <p class="paragraph">The library was quantified using Qubit and qPCR, and the size distribution was detected with a fragment analyzer. Quantified libraries were pooled and sequenced according to the effective library concentration and required data amount.</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="files/Sequencing_Methods_en.pdf" target="_blank"><b>Method</b></a></p>
        

        <br />

        <br />

<hr />
</p><br />
<h2>3 Bioinformatics Analysis Pipeline</h2>


    <p class="paragraph">Data Quality Control: Post-sequencing, a portion of the raw data was identified as low-quality. Through quality control and host filtering, effective (clean) data was obtained to ensure accuracy in subsequent analyses.</p>

    <p class="paragraph">Metagenome Assembly: The clean data from each sample, after quality control, was used for metagenome assembly.</p>

    <p class="paragraph">Gene Prediction: Using MetaGeneMark, gene prediction was performed on the assembled scaffolds. The predicted genes were pooled and dereplicated to create a gene catalogue, enabling the assessment of gene abundance for each sample.</p>

    <p class="paragraph">Taxonomy Annotation: The metagenomic reads were compared with the microNR database for taxonomy annotation, resulting in abundance tables at various taxonomic ranks.</p>

    <p class="paragraph">Function Annotation: The functions of coding sequences were inferred by comparing them with databases like KEGG, eggNOG, and CAZy, providing a functional profile of the metagenome.</p>

    <p class="paragraph">Statistical and Comparative Analysis: Utilizing the abundance tables, analyses including clustering, Anosim, PCA, PCoA, and NMDS were conducted. For grouped data, MetaGenomeSeq and LEfSe were used for multivariate analysis and pathway comparison.</p>

    <p class="paragraph">ARGs Analysis: Antibiotic resistant genes were annotated using the CARD database, revealing their abundance and species distribution.</p>

    <p class="paragraph">MGE Analysis: Unigenes were compared against databases for insertion sequences, integrons, and plasmids to determine their abundance.</p>

<p class="center">
<img src="src/images/bioinfoWorkflow_en.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 3 Analysis Workflow </strong></p>  
</p>
<p class="premark" >Notes: When the number of samples is less than 3, PCA, PCoA, NMDS, clustering analysis and abundance heatmap analysis cannot be carried out. When the number of biological repeats in the group is less than 3, statistical analysis such as Anosim, MetaGenomeSeq, and LEfSe are statistically meaningless and the analysis cannot be conducted. </p>

        <br />

        <br />

<hr />
</p><br />
<h2>4 Analysis Results</h2>


        <br />

        <br />

<h3>4.1 Sequencing data processing</h3>
        
    <p class="paragraph">The protocols of data pre-processing are as follows:</p>

    <p class="paragraph">We will discard the paired reads in the following situation: when either one read contains adapter contamination; when either one read contains more than 10 percent uncertain nucleotides; when either one read contains more than 50 percent low quality nucleotides (base quality less than 5).</p>

<p class="name_table"><p class="center"><strong>Table 4.1 QC statistics</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable1" >
        <thead><tr>
<a id="btn01"></a>
<a id="sjzl1"></a>
<a  href="javascript:void(0)" id="btn11"> | First | </a>
<a  href="javascript:void(0)" id="btn21"> Previous | </a>
<a  href="javascript:void(0)" id="btn31"> Next | </a>
<a  href="javascript:void(0)" id="btn41"> Last | </a>
<a>Go to </a>
<input id="changePage1" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn51"> Jump </a>
                
<th style="text-align :center">SampleID</th><th style="text-align :center">Raw_Base(G)</th><th style="text-align :center">Clean_Base(G)</th><th style="text-align :center">Clean_Q20(%)</th><th style="text-align :center">Clean_Q30(%)</th><th style="text-align :center">Clean_GC(%)</th><th style="text-align :center">Effective(%)</th><th style="text-align :center">NonHost_Base(G)
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">6.07</td><td style="text-align :center">6.03</td><td style="text-align :center">97.10</td><td style="text-align :center">91.62</td><td style="text-align :center">50.34</td><td style="text-align :center">99.22</td><td style="text-align :center">6.01
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">6.52</td><td style="text-align :center">6.47</td><td style="text-align :center">97.84</td><td style="text-align :center">93.69</td><td style="text-align :center">51.00</td><td style="text-align :center">99.32</td><td style="text-align :center">6.20
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">6.32</td><td style="text-align :center">6.29</td><td style="text-align :center">97.56</td><td style="text-align :center">92.87</td><td style="text-align :center">50.01</td><td style="text-align :center">99.50</td><td style="text-align :center">6.25
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">6.19</td><td style="text-align :center">6.16</td><td style="text-align :center">97.66</td><td style="text-align :center">93.12</td><td style="text-align :center">50.64</td><td style="text-align :center">99.43</td><td style="text-align :center">6.13
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">6.12</td><td style="text-align :center">6.06</td><td style="text-align :center">97.58</td><td style="text-align :center">92.93</td><td style="text-align :center">49.01</td><td style="text-align :center">99.05</td><td style="text-align :center">5.47
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">7.04</td><td style="text-align :center">7.00</td><td style="text-align :center">98.22</td><td style="text-align :center">94.71</td><td style="text-align :center">50.63</td><td style="text-align :center">99.41</td><td style="text-align :center">6.72
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">11.67</td><td style="text-align :center">11.46</td><td style="text-align :center">97.83</td><td style="text-align :center">93.90</td><td style="text-align :center">47.04</td><td style="text-align :center">98.21</td><td style="text-align :center">6.43
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">9.68</td><td style="text-align :center">9.50</td><td style="text-align :center">97.67</td><td style="text-align :center">93.46</td><td style="text-align :center">48.33</td><td style="text-align :center">98.12</td><td style="text-align :center">5.93
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">6.10</td><td style="text-align :center">6.02</td><td style="text-align :center">96.96</td><td style="text-align :center">91.34</td><td style="text-align :center">51.11</td><td style="text-align :center">98.70</td><td style="text-align :center">5.99
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">6.24</td><td style="text-align :center">6.18</td><td style="text-align :center">97.42</td><td style="text-align :center">92.56</td><td style="text-align :center">48.49</td><td style="text-align :center">99.08</td><td style="text-align :center">5.38
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">6.10</td><td style="text-align :center">6.03</td><td style="text-align :center">96.94</td><td style="text-align :center">91.25</td><td style="text-align :center">51.13</td><td style="text-align :center">98.92</td><td style="text-align :center">5.92
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">7.52</td><td style="text-align :center">7.41</td><td style="text-align :center">97.24</td><td style="text-align :center">92.23</td><td style="text-align :center">48.42</td><td style="text-align :center">98.65</td><td style="text-align :center">5.78
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">6.47</td><td style="text-align :center">6.43</td><td style="text-align :center">97.24</td><td style="text-align :center">92.09</td><td style="text-align :center">50.59</td><td style="text-align :center">99.31</td><td style="text-align :center">6.04
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">7.98</td><td style="text-align :center">7.90</td><td style="text-align :center">97.67</td><td style="text-align :center">93.28</td><td style="text-align :center">47.86</td><td style="text-align :center">98.95</td><td style="text-align :center">6.27
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">8.29</td><td style="text-align :center">8.18</td><td style="text-align :center">97.69</td><td style="text-align :center">93.38</td><td style="text-align :center">47.00</td><td style="text-align :center">98.70</td><td style="text-align :center">6.06
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">6.17</td><td style="text-align :center">6.12</td><td style="text-align :center">97.66</td><td style="text-align :center">93.20</td><td style="text-align :center">45.85</td><td style="text-align :center">99.20</td><td style="text-align :center">5.75
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">8.90</td><td style="text-align :center">8.75</td><td style="text-align :center">97.57</td><td style="text-align :center">93.19</td><td style="text-align :center">46.48</td><td style="text-align :center">98.33</td><td style="text-align :center">5.19
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">7.93</td><td style="text-align :center">7.85</td><td style="text-align :center">97.78</td><td style="text-align :center">93.62</td><td style="text-align :center">48.02</td><td style="text-align :center">98.96</td><td style="text-align :center">6.62
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Raw_Base(G): Raw sequenced data.</p>
<p class="premark" >Clean_Base(G): Clean data obtained after data filtering.</p>
<p class="premark" >Clean_Q20: Percentage of bases with a quality score over 20, indicating an error rate less than 1%.</p>
<p class="premark" >Clean_Q30: Percentage of bases with a quality score over 30, corresponding to an error rate less than 0.1%.</p>
<p class="premark" >Clean_GC(%): Calculated as the total count of guanine (G) and cytosine (C) bases divided by the total count of clean bases, expressed as a percentage.</p>
<p class="premark" >Effective(%): Represents the ratio of Clean Data (post-quality control data) to Raw Data, expressed as a percentage.</p>
<p class="premark" >NonHost_Base(G): The total count of clean bases remaining after the removal of host-specific sequences, measured in gigabases (G).</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">QC_result: result/01.QC/QC_summary.xls</p>

        <br />

        <br />

<h3>4.2 Metagenome Assembly</h3>
        
    <p class="paragraph">Samples successfully passing Quality Control (QC) underwent initial assembly using MEGAHIT, configured with the meta-large preset (for more details, see: https://www.metagenomics.wiki/tools/assembly/megahit). Scaftigs from each sample shorter than 500 base pairs (bp) were discard, following methodologies established in previous studies (Qin et al., 2014; Zeller et al., 2014; Sunagawa et al., 2015; Li et al., 2014). The resulting effective scaftigs were then utilized for further analysis and gene prediction. </p>

<p class="name_table"><p class="center"><strong>Table 4.2 Statistics of scaftigs (>=500 bp) </strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable2" >
        <thead><tr>
<a id="btn02"></a>
<a id="sjzl2"></a>
<a  href="javascript:void(0)" id="btn12"> | First | </a>
<a  href="javascript:void(0)" id="btn22"> Previous | </a>
<a  href="javascript:void(0)" id="btn32"> Next | </a>
<a  href="javascript:void(0)" id="btn42"> Last | </a>
<a>Go to </a>
<input id="changePage2" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn52"> Jump </a>
                
<th style="text-align :center">SampleID</th><th style="text-align :center">Total len(bp)</th><th style="text-align :center">Scaftigs num</th><th style="text-align :center">Average len(bp)</th><th style="text-align :center">N50 len(bp)</th><th style="text-align :center">N90 len(bp)</th><th style="text-align :center">Max len(bp)
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">165,447,841</td><td style="text-align :center">69,857</td><td style="text-align :center">2,368.38</td><td style="text-align :center">10,965</td><td style="text-align :center">708</td><td style="text-align :center">478,642
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">213,570,341</td><td style="text-align :center">111,489</td><td style="text-align :center">1,915.62</td><td style="text-align :center">4,092</td><td style="text-align :center">656</td><td style="text-align :center">646,982
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">235,682,388</td><td style="text-align :center">121,901</td><td style="text-align :center">1,933.39</td><td style="text-align :center">3,883</td><td style="text-align :center">666</td><td style="text-align :center">646,981
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">144,363,110</td><td style="text-align :center">50,624</td><td style="text-align :center">2,851.67</td><td style="text-align :center">15,391</td><td style="text-align :center">785</td><td style="text-align :center">478,624
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">183,960,238</td><td style="text-align :center">86,142</td><td style="text-align :center">2,135.55</td><td style="text-align :center">9,028</td><td style="text-align :center">657</td><td style="text-align :center">412,322
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">249,341,826</td><td style="text-align :center">138,844</td><td style="text-align :center">1,795.84</td><td style="text-align :center">3,263</td><td style="text-align :center">642</td><td style="text-align :center">646,982
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">319,519,425</td><td style="text-align :center">166,683</td><td style="text-align :center">1,916.93</td><td style="text-align :center">3,518</td><td style="text-align :center">672</td><td style="text-align :center">481,015
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">132,072,777</td><td style="text-align :center">89,750</td><td style="text-align :center">1,471.56</td><td style="text-align :center">1,905</td><td style="text-align :center">596</td><td style="text-align :center">818,692
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">121,835,665</td><td style="text-align :center">76,525</td><td style="text-align :center">1,592.10</td><td style="text-align :center">2,583</td><td style="text-align :center">610</td><td style="text-align :center">819,022
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">247,135,836</td><td style="text-align :center">148,448</td><td style="text-align :center">1,664.80</td><td style="text-align :center">2,709</td><td style="text-align :center">626</td><td style="text-align :center">488,277
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">92,541,796</td><td style="text-align :center">54,664</td><td style="text-align :center">1,692.92</td><td style="text-align :center">2,594</td><td style="text-align :center">638</td><td style="text-align :center">819,154
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">185,528,748</td><td style="text-align :center">117,961</td><td style="text-align :center">1,572.80</td><td style="text-align :center">2,107</td><td style="text-align :center">630</td><td style="text-align :center">498,778
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">200,981,089</td><td style="text-align :center">105,572</td><td style="text-align :center">1,903.73</td><td style="text-align :center">4,001</td><td style="text-align :center">648</td><td style="text-align :center">818,507
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">265,057,699</td><td style="text-align :center">148,610</td><td style="text-align :center">1,783.58</td><td style="text-align :center">3,019</td><td style="text-align :center">654</td><td style="text-align :center">460,710
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">309,520,243</td><td style="text-align :center">169,297</td><td style="text-align :center">1,828.27</td><td style="text-align :center">3,159</td><td style="text-align :center">664</td><td style="text-align :center">437,747
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">170,647,887</td><td style="text-align :center">87,284</td><td style="text-align :center">1,955.09</td><td style="text-align :center">4,858</td><td style="text-align :center">655</td><td style="text-align :center">488,694
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">225,060,591</td><td style="text-align :center">140,455</td><td style="text-align :center">1,602.37</td><td style="text-align :center">2,417</td><td style="text-align :center">620</td><td style="text-align :center">646,981
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">317,578,445</td><td style="text-align :center">168,282</td><td style="text-align :center">1,887.18</td><td style="text-align :center">3,369</td><td style="text-align :center">673</td><td style="text-align :center">479,734
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Total len(bp): stands for length of all the Scaftigs.</p>
<p class="premark" >Scaftigs num: stands for the total number of Scaftigs.</p>
<p class="premark" >Average len(bp): stands for the average length of all the Scaftigs.</p>
<p class="premark" >N50 len(bp): defined as the shortest sequence length at 50% of the genome.</p>
<p class="premark" >N90 len(bp): defined as the shortest sequence length at 90% of the genome.</p>
<p class="premark" >Max len(bp): means the max length of Scaftigs.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Scaffold statistics table: result/02.Assembly/total.scafSeq.stat.info.xls</p>

    <p class="paragraph">Scaftigs with length longer than 500 bp: result/02.Assembly/total.scaftigs.stat.info.xls</p>

    <p class="paragraph">Assembly results for all samples: result/02.Assembly/Sample_Name</p>

    <p class="paragraph">Length distribution of scaftigs: result/02.Assembly/Sample_Name/*.{svg,png}</p>

        <br />

        <br />

<h3>4.3 Gene Prediction and Abundance Analysis</h3>
        
    <a name="T4"></a>
    <h4>4.3.1 Introduction of gene prediction and abundance analysis</h4>    

    <p class="paragraph">Scaftigs measuring 500 base pairs (bp) or more were selected for Open Reading Frame (ORF) prediction using MetaGeneMark, a method validated in several studies (Zhu et al., 2010; Karlsson et al., 2012; Karlsson et al., 2013; Oh et al., 2014). ORFs smaller than 100 nucleotide acids were excluded (Qin et al., 2010).</p>

    <p class="paragraph">These ORFs underwent dereplication using CD-HIT (Li et al., 2006; Fu et al., 2012) to create non-redundant gene catalogues, with a focus on continuous gene sequences encoding nucleic acids (Sunagawa et al., 2015). The dereplication settings were: identity at 95% and coverage at 90%, selecting the longest sequence as the representative gene (unigene)(Li et al., 2014; Zeller et al., 2014).</p>

    <p class="paragraph">Clean data were then mapped to the gene catalogue using Bowtie2 to quantify gene abundance. This abundance was calculated based on the number of mapped reads and gene length, using a computational formula referenced in multiple studies (Karlsson et al., 2012; Cotillard et al., 2013; Le Chatelier et al., 2013; Oh et al., 2014; Zeller et al., 2014; Villar et al., 2015). Subsequent analyses were based on these gene catalogue abundance.</p>

<p class="center">
<img src="src/images/formula.png" width="700" height="350" />
<p class="name_fig"><strong></strong></p>  
</p>
<p class="premark" >Note: r stands for number of mapping reads. L stands for the length of gene.</p>

    <p class="paragraph">Downstream analyses were performed based on the abundance of gene catalogues.</p>

    <a name="T4"></a>
    <h4>4.3.2 Gene catalogue statistics</h4>    

<p class="name_table"><p class="center"><strong>Table 4.3 The Statistic of gene catalogues </strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy">
        <thead><tr>
<th style="text-align :center">ORFs_NO.</th><th style="text-align :center">1,319,089
</th></tr></thead>
<tr>
<td style="text-align :center">Integrity_start</td><td style="text-align :center">293,325(22.24%)
</td></tr>
<tr>
<td style="text-align :center">Integrity_end</td><td style="text-align :center">251,360(19.06%)
</td></tr>
<tr>
<td style="text-align :center">Integrity_none</td><td style="text-align :center">127,814(9.69%)
</td></tr>
<tr>
<td style="text-align :center">Integrity_all</td><td style="text-align :center">646,590(49.02%)
</td></tr>
<tr>
<td style="text-align :center">Total_Len.(Mbp)</td><td style="text-align :center">897.37
</td></tr>
<tr>
<td style="text-align :center">Average_Len.(bp)</td><td style="text-align :center">680.3
</td></tr>
<tr>
<td style="text-align :center">GC_Percent</td><td style="text-align :center">50.60
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >ORFs_NO.: number of genes in gene catalogue.</p>
<p class="premark" >Integrity_start: amount and percentage of genes only containing start codon.</p>
<p class="premark" >Integrity_end: amount and percentage of genes only containing stop codon.</p>
<p class="premark" >Integrity_none: amount and percentage of genes not containing start or stop codon.</p>
<p class="premark" >Integrity_all: amount and percentage of genes containing both start and stop codon.</p>
<p class="premark" >Total_Len. (Mbp): the total length of gene catalogue (million).</p>
<p class="premark" >Average_Len. (bp): the average length of genes in gene catalogue.</p>
<p class="premark" >GC_Percent: the percentage of GC content of genes in gene catalogue.</p>

    <p class="paragraph"></details></p>

<p class="center">
<img src="files/pictures/03.GeneComp/Unigenes.CDS.cdhit.fa.len.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.1 ORFs length distribution</strong></p>  
</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Gene catalogue length distribution: result/03.GenePredict/GenePredict/UniGenes/*.{svg,png}</p>

    <a name="T4"></a>
    <h4>4.3.3 Core-pan genome analysis</h4>    

    <p class="paragraph">Based on the gene abundance table, rarefaction curves were plotted with the number of core-genes and pan-genes separately by randomly drawing sampling. The figures were shown as follows:</p>

<p class="center">
<img src="files/pictures/03.GeneComp/combine.gene.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.2 Core-pan gene rarefaction curve </strong></p>  
</p>

    <p class="paragraph">Notes: a) Pan genome rarefaction curve. b) Core genome rarefaction curve. In the figure, X-axis means the number of samples that sampled. In the figure, Y-axis means the gene number.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Rarefaction curve: result/03.GenePredict/GeneStat/core_pan/*.{png,pdf}</p>

    <a name="T4"></a>
    <h4>4.3.4 Correlation analysis of samples</h4>    

    <p class="paragraph">The spearman's correlation coefficient between samples is a critical index which reflects the reliability of experiment and the reasonability of the chosen samples.</p>

<p class="center">
<img src="files/pictures/03.GeneComp/correlation.heatmap.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.3 Correlation between samples</strong></p>  
</p>

    <p class="paragraph">Notes: Blue bubbles mean positive correlation. Red bubbles mean negative correlation. The darker the color, the greater the association. </p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Correlation matrix for the relation: result/03.GenePredict/GeneStat/correlation/*.{png,pdf}</p>

    <a name="T4"></a>
    <h4>4.3.5 Gene number analysis</h4>    

    <p class="paragraph">To investigate the difference of gene number among groups, the gene numbers of different groups were shown as follows in a box chart:</p>

<p class="center">
<img src="files/pictures/03.GeneComp/group.genebox.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.4 Gene number of different groups</strong></p>  
</p>

    <p class="paragraph">Notes: X-axis indicates the group information; Y-axis indicates the gene number.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Gene number of different groups figures:  result/03.GenePredict/GeneStat/genebox_group1/*.{png,pdf} </p>

        <br />

        <br />

<h3>4.4 Taxonomy and Function Annotation</h3>
        
    <a name="T4"></a>
    <h4>4.4.1 Analysis of species composition</h4>    

    <p class="paragraph">NR database (https://www.ncbi.nlm.nih.gov/): a non-redundant protein amino acid sequence database containing non-redundant data from SwissProt, PIR (Protein Information Resource), PRF (Protein Research Foundation), PDB (Protein Data Bank) protein databases as well as protein data translated from GenBank and RefSeq CDS data.</p>

    <p class="paragraph">Micro_NR database: Microbial sequences of bacteria, fungi, archaea and viruses extracted from the NR database were used for microbiological analyses.</p>

    <a name="T5"></a>
    <h5>4.4.1.1 Introduction of Taxonomic Annotation</h5>    

    <p class="paragraph">Taxonomic diversity serves as a way of profiling a community and can be used to ascertain the similarity of two or more communities. This part involves identifying those reads that are marker gene homologs to a database of taxonomically informative gene families, using sequence or phylogenetic similarity to the database sequences (Micro_NR database) (Buchfink B et al., 2015) to taxonomically annotate each metagenomic homolog (MEGAN) (Huson DH et al., 2011).</p>

    <a name="T5"></a>
    <h5>4.4.1.2 Krona presentation</h5>    

    <p class="paragraph">Krona allows hierarchical data to be explored with zoomable pie charts, and aims to answer questions regarding the relative abundance of taxa across multiple levels of the hierarchy simultaneously (Ondov et al., 2011). The software supplements existing metagenomic visualizations by creating clearer depictions of abundance estimates and by enabling in-depth understanding of the underlying classifications. An example picture was as follows:</p>

<p class="center">
<img src="src/images/metagenome.krona.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.5 Taxonomy annotation visualization </strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="files/pictures/04.Annotation/taxonomy.krona.html" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: circles from inside to outside stand for different taxa, and the area of sector means respective proportion of different taxa. Please refer to the description on the webpage (see details: https://github.com/marbl/Krona/wiki/).</p>

    <p class="paragraph">Based on the abundance table of each taxonomic level, the top 10 taxa were picked out and the other taxa were set as "Others". Bar charts show the relative taxonomy abundance of each sample in different taxonomic level.  </p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativetaxonomyabundanceofeachsampleindifferenttaxonomiclevel_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativetaxonomyabundanceofeachsampleindifferenttaxonomiclevel_nav-tabContent">

<div class="tab-pane active show" id="toptaxkpng1" role="tabpanel" aria-labelledby="toptaxkpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.k.png" /></div>

<div class="tab-pane fade" id="toptaxppng2" role="tabpanel" aria-labelledby="toptaxppng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.p.png" /></div>

<div class="tab-pane fade" id="toptaxcpng3" role="tabpanel" aria-labelledby="toptaxcpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.c.png" /></div>

<div class="tab-pane fade" id="toptaxopng4" role="tabpanel" aria-labelledby="toptaxopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.o.png" /></div>

<div class="tab-pane fade" id="toptaxfpng5" role="tabpanel" aria-labelledby="toptaxfpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.f.png" /></div>

<div class="tab-pane fade" id="toptaxgpng6" role="tabpanel" aria-labelledby="toptaxgpng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.g.png" /></div>

<div class="tab-pane fade" id="toptaxspng7" role="tabpanel" aria-labelledby="toptaxspng7" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.tax.s.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativetaxonomyabundanceofeachsampleindifferenttaxonomiclevel_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativetaxonomyabundanceofeachsampleindifferenttaxonomiclevel_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativetaxonomyabundanceofeachsampleindifferenttaxonomiclevel_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="toptaxkpng1" data-toggle="list" href="#toptaxkpng1" role="tab" aria-controls="toptaxkpng1">Kingdom</a>

<a class="list-group-item list-group-item-action" id="toptaxppng2" data-toggle="list" href="#toptaxppng2" role="tab" aria-controls="toptaxppng2">Phylum</a>

<a class="list-group-item list-group-item-action" id="toptaxcpng3" data-toggle="list" href="#toptaxcpng3" role="tab" aria-controls="toptaxcpng3">Class</a>

<a class="list-group-item list-group-item-action" id="toptaxopng4" data-toggle="list" href="#toptaxopng4" role="tab" aria-controls="toptaxopng4">Order</a>

<a class="list-group-item list-group-item-action" id="toptaxfpng5" data-toggle="list" href="#toptaxfpng5" role="tab" aria-controls="toptaxfpng5">Family</a>

<a class="list-group-item list-group-item-action" id="toptaxgpng6" data-toggle="list" href="#toptaxgpng6" role="tab" aria-controls="toptaxgpng6">Genus</a>

<a class="list-group-item list-group-item-action" id="toptaxspng7" data-toggle="list" href="#toptaxspng7" role="tab" aria-controls="toptaxspng7">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.6 Relative taxonomy abundance of each sample in different taxonomic level</strong></p>  

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Taxonomy annotation visualization: result/04.TaxAnnotation/MicroNR/Krona/taxonomy.krona.html</p>

    <p class="paragraph">Top 10 species analysis: result/04.Annotation/MicroNR/{Top_sample,Top_group1}, including Kindom, Phylum, Class, Order, Family, Genus, and Species</p>

    <a name="T5"></a>
    <h5>4.4.1.3 Taxonomic abundance heatmap</h5>    

    <p class="paragraph">The abundance distribution of dominant 35 genera of all samples was displayed in the species abundance heatmap.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapingenuslevel_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapingenuslevel_nav-tabContent">

<div class="tab-pane active show" id="Taxclusterkpng1" role="tabpanel" aria-labelledby="Taxclusterkpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.k.png" /></div>

<div class="tab-pane fade" id="Taxclusterppng2" role="tabpanel" aria-labelledby="Taxclusterppng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.p.png" /></div>

<div class="tab-pane fade" id="Taxclustercpng3" role="tabpanel" aria-labelledby="Taxclustercpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.c.png" /></div>

<div class="tab-pane fade" id="Taxclusteropng4" role="tabpanel" aria-labelledby="Taxclusteropng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.o.png" /></div>

<div class="tab-pane fade" id="Taxclusterfpng5" role="tabpanel" aria-labelledby="Taxclusterfpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.f.png" /></div>

<div class="tab-pane fade" id="Taxclustergpng6" role="tabpanel" aria-labelledby="Taxclustergpng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.g.png" /></div>

<div class="tab-pane fade" id="Taxclusterspng7" role="tabpanel" aria-labelledby="Taxclusterspng7" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/Tax.cluster.s.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapingenuslevel_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapingenuslevel_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapingenuslevel_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Taxclusterkpng1" data-toggle="list" href="#Taxclusterkpng1" role="tab" aria-controls="Taxclusterkpng1">Kingdom</a>

<a class="list-group-item list-group-item-action" id="Taxclusterppng2" data-toggle="list" href="#Taxclusterppng2" role="tab" aria-controls="Taxclusterppng2">Phylum</a>

<a class="list-group-item list-group-item-action" id="Taxclustercpng3" data-toggle="list" href="#Taxclustercpng3" role="tab" aria-controls="Taxclustercpng3">Class</a>

<a class="list-group-item list-group-item-action" id="Taxclusteropng4" data-toggle="list" href="#Taxclusteropng4" role="tab" aria-controls="Taxclusteropng4">Order</a>

<a class="list-group-item list-group-item-action" id="Taxclusterfpng5" data-toggle="list" href="#Taxclusterfpng5" role="tab" aria-controls="Taxclusterfpng5">Family</a>

<a class="list-group-item list-group-item-action" id="Taxclustergpng6" data-toggle="list" href="#Taxclustergpng6" role="tab" aria-controls="Taxclustergpng6">Genus</a>

<a class="list-group-item list-group-item-action" id="Taxclusterspng7" data-toggle="list" href="#Taxclusterspng7" role="tab" aria-controls="Taxclusterspng7">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.7 Abundance clustering heatmap in genus level</strong></p>  
<p class="premark" >Notes: X-axis indicates sample name. Y-axis indicates taxonomic information. The clustering tree is at the left side of this chart. The absolute value of "Z" represents the distance between the raw score and the population mean in units of the standard deviation. "Z" is negative when the raw score is less than the mean value, positive when more.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureGenenumberclusteringheatmapingenuslevel_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureGenenumberclusteringheatmapingenuslevel_nav-tabContent">

<div class="tab-pane active show" id="heatmapgntaxkingdompng1" role="tabpanel" aria-labelledby="heatmapgntaxkingdompng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.kingdom.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxphylumpng2" role="tabpanel" aria-labelledby="heatmapgntaxphylumpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.phylum.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxclasspng3" role="tabpanel" aria-labelledby="heatmapgntaxclasspng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.class.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxorderpng4" role="tabpanel" aria-labelledby="heatmapgntaxorderpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.order.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxfamilypng5" role="tabpanel" aria-labelledby="heatmapgntaxfamilypng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.family.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxgenuspng6" role="tabpanel" aria-labelledby="heatmapgntaxgenuspng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.genus.png" /></div>

<div class="tab-pane fade" id="heatmapgntaxspeciespng7" role="tabpanel" aria-labelledby="heatmapgntaxspeciespng7" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/heatmap_gn.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureGenenumberclusteringheatmapingenuslevel_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureGenenumberclusteringheatmapingenuslevel_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureGenenumberclusteringheatmapingenuslevel_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="heatmapgntaxkingdompng1" data-toggle="list" href="#heatmapgntaxkingdompng1" role="tab" aria-controls="heatmapgntaxkingdompng1">Kingdom</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxphylumpng2" data-toggle="list" href="#heatmapgntaxphylumpng2" role="tab" aria-controls="heatmapgntaxphylumpng2">Phylum</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxclasspng3" data-toggle="list" href="#heatmapgntaxclasspng3" role="tab" aria-controls="heatmapgntaxclasspng3">Class</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxorderpng4" data-toggle="list" href="#heatmapgntaxorderpng4" role="tab" aria-controls="heatmapgntaxorderpng4">Order</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxfamilypng5" data-toggle="list" href="#heatmapgntaxfamilypng5" role="tab" aria-controls="heatmapgntaxfamilypng5">Family</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxgenuspng6" data-toggle="list" href="#heatmapgntaxgenuspng6" role="tab" aria-controls="heatmapgntaxgenuspng6">Genus</a>

<a class="list-group-item list-group-item-action" id="heatmapgntaxspeciespng7" data-toggle="list" href="#heatmapgntaxspeciespng7" role="tab" aria-controls="heatmapgntaxspeciespng7">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.8 Gene number clustering heatmap in genus level</strong></p>  
<p class="premark" >Notes: X-axis indicates sample name. Y-axis indicates taxonomic information. Different colors represent different unigene number.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Heatmap of Relative abundance: result/04.Annotation/MicroNR/Heatmap_sample/, including Kingdom, Phylum, Class, Order, Family, Genus, and Species</p>

    <p class="paragraph">Unigenes annotation heatmap: result/04.TaxAnnotation/MicroNR/Heatmap_genenums_sample/, including Kingdom, Phylum, Class, Order, Family, Genus, and Species</p>

    <a name="T4"></a>
    <h4>4.4.2 KEGG function annotation</h4>    

    <p class="paragraph">Kyoto Encyclopedia of Genes and Genomes (KEGG).</p>

    <p class="paragraph">KEGG is an integrated database resource consisting of eighteen databases (including computationally generated SSDB), in which KEGG PATHWAY and KEGG ORTHOLOGY (KO) are the core databases. The KO database is a database of molecular functions represented in terms of functional orthologs. KEGG PATHWAY is a reference database for Pathway Mapping and each pathway can be divided into three levels. Besides, KEGG ENZYME (ec) is also used in function annotation.</p>

    <a name="T5"></a>
    <h5>4.4.2.1 Basic annotation methods of KEGG</h5>    

    <p class="paragraph">Comparison of Unigenes to functional databases using DIAMOND software (blastp, eval <= 1e-5).</p>

    <p class="paragraph">Result filtering: For each sequence, the result with the highest score (one HSP > 60 bits) is selected for subsequent analysis.</p>

    <p class="paragraph">From the comparison results, the relative abundance of different functional tiers (the relative abundance of each functional tier is equal to the sum of the relative abundance of the genes annotated to that functional tier) was counted.</p>

    <p class="paragraph">From the results of the functional annotation and the gene abundance table, a table of the number of genes in each sample at each taxonomic level was obtained, and for a given function the number of genes in a given sample was equal to the number of genes whose abundance was not 0 among the genes annotated for that function.</p>

    <a name="T5"></a>
    <h5>4.4.2.2 Functional abundance of KEGG</h5>    

    <p class="paragraph">Based on Unique genes annotation results, cartogram of annotated gene number of KEGG database was drawn. The results were showed as follows:level1, level2, level3, KO, ec, module.</p>

<p class="center">
<img src="files/pictures/04.Annotation/kegg.unigenes.num.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.9 Summarized chart for the gene number annotated by KEGG database (Level 1)</strong></p>  
</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plot for annotation results of KEGG database: result/04.Annotation/KEGG/KEGG_Anno/KEGG.unigenes.num.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.4.2.3 Relative abundance of KEGG</h5>    

    <p class="paragraph">According to the relative abundance in level 1 of KEGG database, cartogram of the functional abundance of each sample was drawn.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofKEGG_nav-tabContent">

<div class="tab-pane active show" id="topkegglevelpng1" role="tabpanel" aria-labelledby="topkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.level1.png" /></div>

<div class="tab-pane fade" id="topkegglevelpng2" role="tabpanel" aria-labelledby="topkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.level2.png" /></div>

<div class="tab-pane fade" id="topkegglevelpng3" role="tabpanel" aria-labelledby="topkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.level3.png" /></div>

<div class="tab-pane fade" id="topkeggkopng4" role="tabpanel" aria-labelledby="topkeggkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.ko.png" /></div>

<div class="tab-pane fade" id="topkeggecpng5" role="tabpanel" aria-labelledby="topkeggecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.ec.png" /></div>

<div class="tab-pane fade" id="topkeggmodulepng6" role="tabpanel" aria-labelledby="topkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topkegglevelpng1" data-toggle="list" href="#topkegglevelpng1" role="tab" aria-controls="topkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="topkegglevelpng2" data-toggle="list" href="#topkegglevelpng2" role="tab" aria-controls="topkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="topkegglevelpng3" data-toggle="list" href="#topkegglevelpng3" role="tab" aria-controls="topkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="topkeggkopng4" data-toggle="list" href="#topkeggkopng4" role="tab" aria-controls="topkeggkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="topkeggecpng5" data-toggle="list" href="#topkeggecpng5" role="tab" aria-controls="topkeggecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="topkeggmodulepng6" data-toggle="list" href="#topkeggmodulepng6" role="tab" aria-controls="topkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.10 Relative abundance of KEGG</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plots: result/04.Annotation/KEGG/Top_{sample,group1}/*/*.top10.{svg,png}</p>

    <a name="T5"></a>
    <h5>4.4.2.4 Relative abundance cluster analysis of KEGG</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the KEGG database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofKEGG_nav-tabContent">

<div class="tab-pane active show" id="KEGGclusterlevelpng1" role="tabpanel" aria-labelledby="KEGGclusterlevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.level1.png" /></div>

<div class="tab-pane fade" id="KEGGclusterlevelpng2" role="tabpanel" aria-labelledby="KEGGclusterlevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.level2.png" /></div>

<div class="tab-pane fade" id="KEGGclusterlevelpng3" role="tabpanel" aria-labelledby="KEGGclusterlevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.level3.png" /></div>

<div class="tab-pane fade" id="KEGGclusterkopng4" role="tabpanel" aria-labelledby="KEGGclusterkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.ko.png" /></div>

<div class="tab-pane fade" id="KEGGclusterecpng5" role="tabpanel" aria-labelledby="KEGGclusterecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.ec.png" /></div>

<div class="tab-pane fade" id="KEGGclustermodulepng6" role="tabpanel" aria-labelledby="KEGGclustermodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/KEGG.cluster.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="KEGGclusterlevelpng1" data-toggle="list" href="#KEGGclusterlevelpng1" role="tab" aria-controls="KEGGclusterlevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="KEGGclusterlevelpng2" data-toggle="list" href="#KEGGclusterlevelpng2" role="tab" aria-controls="KEGGclusterlevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="KEGGclusterlevelpng3" data-toggle="list" href="#KEGGclusterlevelpng3" role="tab" aria-controls="KEGGclusterlevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="KEGGclusterkopng4" data-toggle="list" href="#KEGGclusterkopng4" role="tab" aria-controls="KEGGclusterkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="KEGGclusterecpng5" data-toggle="list" href="#KEGGclusterecpng5" role="tab" aria-controls="KEGGclusterecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="KEGGclustermodulepng6" data-toggle="list" href="#KEGGclustermodulepng6" role="tab" aria-controls="KEGGclustermodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.11 Abundance clustering heatmap of KEGG</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">KEGG cluster heatmap:result/04.Annotation/KEGG/Heatmap_sample/*/heatmap.sample.*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.4.3 eggNOG function annotation</h4>    

    <p class="paragraph">Evolutionary genealogy of genes: Non-supervised Orthologous Groups (eggNOG).</p>

    <p class="paragraph">EggNOG, the database of orthologous groups and functional annotation, can be divided into 24 categories in the first level, and each category is represented by a capital letter.</p>

    <a name="T5"></a>
    <h5>4.4.3.1 Basic annotation methods of eggNOG</h5>    

    <p class="paragraph">Comparison of Unigenes to functional databases using DIAMOND software (blastp, eval <= 1e-5).</p>

    <p class="paragraph">Result filtering: For each sequence, the result with the highest score (one HSP > 60 bits) is selected for subsequent analysis.</p>

    <p class="paragraph">From the comparison results, the relative abundance of different functional tiers (the relative abundance of each functional tier is equal to the sum of the relative abundance of the genes annotated to that functional tier) was counted.</p>

    <p class="paragraph">From the results of the functional annotation and the gene abundance table, a table of the number of genes in each sample at each taxonomic level was obtained, and for a given function the number of genes in a given sample was equal to the number of genes whose abundance was not 0 among the genes annotated for that function.</p>

    <a name="T5"></a>
    <h5>4.4.3.2 Functional abundance of eggNOG</h5>    

    <p class="paragraph">Based on Unique genes annotation results, cartogram of annotated gene number of eggNOG database was drawn. The results were showed as follows:level1, level2, og.</p>

<p class="center">
<img src="files/pictures/04.Annotation/eggnog.unigenes.num.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.12 Summarized chart for the gene number annotated by eggNOG database (Level 1)</strong></p>  
</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plot for annotation results of eggNOG database: result/04.Annotation/eggNOG/eggNOG_Anno/eggNOG.unigenes.num.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.4.3.3 Relative abundance of eggNOG</h5>    

    <p class="paragraph">According to the relative abundance in level 1 of eggNOG database, cartogram of the functional abundance of each sample was drawn.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofeggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofeggNOG_nav-tabContent">

<div class="tab-pane active show" id="topeggnoglevelpng1" role="tabpanel" aria-labelledby="topeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="topeggnoglevelpng2" role="tabpanel" aria-labelledby="topeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.eggnog.level2.png" /></div>

<div class="tab-pane fade" id="topeggnogogpng3" role="tabpanel" aria-labelledby="topeggnogogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.eggnog.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofeggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofeggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofeggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topeggnoglevelpng1" data-toggle="list" href="#topeggnoglevelpng1" role="tab" aria-controls="topeggnoglevelpng1">levle1</a>

<a class="list-group-item list-group-item-action" id="topeggnoglevelpng2" data-toggle="list" href="#topeggnoglevelpng2" role="tab" aria-controls="topeggnoglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="topeggnogogpng3" data-toggle="list" href="#topeggnogogpng3" role="tab" aria-controls="topeggnogogpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.13 Relative abundance of eggNOG</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plots: result/04.Annotation/eggNOG/Top_{sample,group1}/*/*.top10.{svg,png}</p>

    <a name="T5"></a>
    <h5>4.4.3.4 Relative abundance cluster analysis of eggNOG</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the eggNOG database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofeggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofeggNOG_nav-tabContent">

<div class="tab-pane active show" id="eggNOGclusterlevelpng1" role="tabpanel" aria-labelledby="eggNOGclusterlevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/eggNOG.cluster.level1.png" /></div>

<div class="tab-pane fade" id="eggNOGclusterlevelpng2" role="tabpanel" aria-labelledby="eggNOGclusterlevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/eggNOG.cluster.level2.png" /></div>

<div class="tab-pane fade" id="eggNOGclusterogpng3" role="tabpanel" aria-labelledby="eggNOGclusterogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/eggNOG.cluster.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofeggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofeggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofeggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="eggNOGclusterlevelpng1" data-toggle="list" href="#eggNOGclusterlevelpng1" role="tab" aria-controls="eggNOGclusterlevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="eggNOGclusterlevelpng2" data-toggle="list" href="#eggNOGclusterlevelpng2" role="tab" aria-controls="eggNOGclusterlevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="eggNOGclusterogpng3" data-toggle="list" href="#eggNOGclusterogpng3" role="tab" aria-controls="eggNOGclusterogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.14 Abundance clustering heatmap of eggNOG</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">eggNOG cluster heatmap:result/04.Annotation/eggNOG/Heatmap_sample/*/heatmap.sample.*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.4.4 CAZy function annotation</h4>    

    <p class="paragraph">CAZy is a database dedicated to the display and analysis of genomic, structural and biochemical information on Carbohydrate-Active Enzymes. The database mainly contains six functional classes in the first level: Glycoside Hydrolase, GlycosylTransferase, Polysaccharide Lyase, Carbohydrate Esterases, Auxiliary Activities and Carbohydrate-Binding Modules.</p>

    <a name="T5"></a>
    <h5>4.4.4.1 Basic annotation methods</h5>    

    <p class="paragraph">Comparison of Unigenes to functional databases using DIAMOND software (blastp, eval <= 1e-5).</p>

    <p class="paragraph">Result filtering: For each sequence, the result with the highest score (one HSP > 60 bits) is selected for subsequent analysis.</p>

    <p class="paragraph">From the comparison results, the relative abundance of different functional tiers (the relative abundance of each functional tier is equal to the sum of the relative abundance of the genes annotated to that functional tier) was counted.</p>

    <p class="paragraph">From the results of the functional annotation and the gene abundance table, a table of the number of genes in each sample at each taxonomic level was obtained, and for a given function the number of genes in a given sample was equal to the number of genes whose abundance was not 0 among the genes annotated for that function.</p>

    <a name="T5"></a>
    <h5>4.4.4.2 Functional abundance of CAZy</h5>    

    <p class="paragraph">Based on Unique genes annotation results, cartogram of annotated gene number of CAZy database was drawn. The results were showed as follows:level1, level2, ec</p>

<p class="center">
<img src="files/pictures/04.Annotation/cazy.unigenes.num.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.15 Summarized chart for the gene number annotated by CAZy database (Level 1)</strong></p>  
</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plot for annotation results of CAZy database: result/04.Annotation/CAZy/Anno/CAZy.unigenes.num.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.4.4.3 Relative abundance of CAZy</h5>    

    <p class="paragraph">According to the relative abundance in level 1 of CAZy database, cartogram of the functional abundance of each sample was drawn.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofCAZy_nav-tabContent">

<div class="tab-pane active show" id="topcazylevelpng1" role="tabpanel" aria-labelledby="topcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.cazy.level1.png" /></div>

<div class="tab-pane fade" id="topcazylevelpng2" role="tabpanel" aria-labelledby="topcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.cazy.level2.png" /></div>

<div class="tab-pane fade" id="topcazyecpng3" role="tabpanel" aria-labelledby="topcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topcazylevelpng1" data-toggle="list" href="#topcazylevelpng1" role="tab" aria-controls="topcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="topcazylevelpng2" data-toggle="list" href="#topcazylevelpng2" role="tab" aria-controls="topcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="topcazyecpng3" data-toggle="list" href="#topcazyecpng3" role="tab" aria-controls="topcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.16 Relative abundance of CAZy</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plots at level 1: result/04.Annotation/CAZy/Top_{sample,group1}/level1/level1.top10.{svg,png}</p>

    <a name="T5"></a>
    <h5>4.4.4.4 Relative abundance cluster analysis of CAZy</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the CAZy database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofCAZy_nav-tabContent">

<div class="tab-pane active show" id="CAZyclusterlevelpng1" role="tabpanel" aria-labelledby="CAZyclusterlevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/CAZy.cluster.level1.png" /></div>

<div class="tab-pane fade" id="CAZyclusterlevelpng2" role="tabpanel" aria-labelledby="CAZyclusterlevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/CAZy.cluster.level2.png" /></div>

<div class="tab-pane fade" id="CAZyclusterecpng3" role="tabpanel" aria-labelledby="CAZyclusterecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/CAZy.cluster.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CAZyclusterlevelpng1" data-toggle="list" href="#CAZyclusterlevelpng1" role="tab" aria-controls="CAZyclusterlevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="CAZyclusterlevelpng2" data-toggle="list" href="#CAZyclusterlevelpng2" role="tab" aria-controls="CAZyclusterlevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="CAZyclusterecpng3" data-toggle="list" href="#CAZyclusterecpng3" role="tab" aria-controls="CAZyclusterecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.17 Abundance clustering heatmap of CAZy</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">CAZy cluster heatmap:result/04.Annotation/CAZy/Heatmap_sample/*/heatmap.sample.*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.4.5 VFDB function annotation</h4>    

    <p class="paragraph">VFDB database, or Virulence Factors of Pathogenic Bacteria database, is a database of virulence factors from pathogenic bacteria, chlamydia and mycoplasma. This database not only contains species information with general characterization of virulence genes, but also provides detailed description of virulence gene functions and pathogenic mechanisms. It contains 26 species with 469 virulence factors, 24 virulence islands, and 2,505 genes related to virulence factors.</p>

    <a name="T5"></a>
    <h5>4.4.5.1 Basic annotation methods of VFDB</h5>    

    <p class="paragraph">Comparison of Unigenes to functional databases using DIAMOND software (blastp, eval <= 1e-5).</p>

    <p class="paragraph">Result filtering: For each sequence, the result with the highest score (one HSP > 60 bits) is selected for subsequent analysis.</p>

    <p class="paragraph">From the comparison results, the relative abundance of different functional tiers (the relative abundance of each functional tier is equal to the sum of the relative abundance of the genes annotated to that functional tier) was counted, in which the VFDB database was divided into 2 tiers.</p>

    <p class="paragraph">From the results of the functional annotation and the gene abundance table, a table of the number of genes in each sample at each taxonomic level was obtained, and for a given function the number of genes in a given sample was equal to the number of genes whose abundance was not 0 among the genes annotated for that function.</p>

    <p class="paragraph">Bar plot for annotation results of VFDB database: result/04.Annotation/VFDB/Anno/VFDB.unigenes.num.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.4.5.2 Relative abundance of VFDB</h5>    

    <p class="paragraph">According to the relative abundance in level 1 of VFDB database, cartogram of the functional abundance of each sample was drawn.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofVFDB_nav-tabContent">

<div class="tab-pane active show" id="topvfdblevelpng1" role="tabpanel" aria-labelledby="topvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="topvfdblevelpng2" role="tabpanel" aria-labelledby="topvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topvfdblevelpng1" data-toggle="list" href="#topvfdblevelpng1" role="tab" aria-controls="topvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="topvfdblevelpng2" data-toggle="list" href="#topvfdblevelpng2" role="tab" aria-controls="topvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.18 Relative abundance of VFDB</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plots at level 1: result/04.Annotation/VFDB/Top_{sample,group1}/level1/level1.top10.{svg,png}</p>

    <a name="T5"></a>
    <h5>4.4.5.3 Relative abundance cluster analysis of VFDB</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the VFDB database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofVFDB_nav-tabContent">

<div class="tab-pane active show" id="VFDBclusterlevelpng1" role="tabpanel" aria-labelledby="VFDBclusterlevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/VFDB.cluster.level1.png" /></div>

<div class="tab-pane fade" id="VFDBclusterlevelpng2" role="tabpanel" aria-labelledby="VFDBclusterlevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/VFDB.cluster.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="VFDBclusterlevelpng1" data-toggle="list" href="#VFDBclusterlevelpng1" role="tab" aria-controls="VFDBclusterlevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="VFDBclusterlevelpng2" data-toggle="list" href="#VFDBclusterlevelpng2" role="tab" aria-controls="VFDBclusterlevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.19 Abundance clustering heatmap of VFDB</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">VFDB cluster heatmap:result/04.Annotation/VFDB/Heatmap_sample/*/heatmap.sample.*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.4.6 PHI function annotation</h4>    

    <p class="paragraph">PHI database, or Pathogen-Host Interaction database, is a database focused on pathogen-host interactions. It collects, organizes and provides information on the interactions between various pathogens (e.g., bacteria, viruses, fungi, and parasites) and their hosts, including humans and other plants and animals.</p>

    <a name="T5"></a>
    <h5>4.4.6.1 Basic annotation methods of PHI</h5>    

    <p class="paragraph">Comparison of Unigenes to functional databases using DIAMOND software (blastp, eval <= 1e-5).</p>

    <p class="paragraph">Result filtering: For each sequence, the result with the highest score (one HSP > 60 bits) is selected for subsequent analysis.</p>

    <p class="paragraph">From the comparison results, the relative abundance of different functional tiers (the relative abundance of each functional tier is equal to the sum of the relative abundance of the genes annotated to that functional tier) was counted, in which the PHI database was divided into 3 tiers.</p>

    <p class="paragraph">From the results of the functional annotation and the gene abundance table, a table of the number of genes in each sample at each taxonomic level was obtained, and for a given function the number of genes in a given sample was equal to the number of genes whose abundance was not 0 among the genes annotated for that function.</p>

    <a name="T5"></a>
    <h5>4.4.6.2 Relative abundance of PHI</h5>    

    <p class="paragraph">According to the relative abundance in level 1 of PHI database, cartogram of the functional abundance of each sample was drawn.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofPHI_nav-tabContent">

<div class="tab-pane active show" id="topphilevelpng1" role="tabpanel" aria-labelledby="topphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.phi.level1.png" /></div>

<div class="tab-pane fade" id="topphilevelpng2" role="tabpanel" aria-labelledby="topphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.phi.level2.png" /></div>

<div class="tab-pane fade" id="topphilevelpng3" role="tabpanel" aria-labelledby="topphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/top10.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topphilevelpng1" data-toggle="list" href="#topphilevelpng1" role="tab" aria-controls="topphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="topphilevelpng2" data-toggle="list" href="#topphilevelpng2" role="tab" aria-controls="topphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="topphilevelpng3" data-toggle="list" href="#topphilevelpng3" role="tab" aria-controls="topphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.20 Relative abundance of PHI</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Bar plots at level 1: result/04.Annotation/PHI/Top_{sample,group1}/level1/level1.top10.{svg,png}</p>

    <a name="T5"></a>
    <h5>4.4.6.3 Relative abundance cluster analysis of PHI</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the PHI database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofPHI_nav-tabContent">

<div class="tab-pane active show" id="PHIclusterlevelpng1" role="tabpanel" aria-labelledby="PHIclusterlevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/PHI.cluster.level1.png" /></div>

<div class="tab-pane fade" id="PHIclusterlevelpng2" role="tabpanel" aria-labelledby="PHIclusterlevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/PHI.cluster.level2.png" /></div>

<div class="tab-pane fade" id="PHIclusterlevelpng3" role="tabpanel" aria-labelledby="PHIclusterlevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/04.Annotation/PHI.cluster.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PHIclusterlevelpng1" data-toggle="list" href="#PHIclusterlevelpng1" role="tab" aria-controls="PHIclusterlevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PHIclusterlevelpng2" data-toggle="list" href="#PHIclusterlevelpng2" role="tab" aria-controls="PHIclusterlevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PHIclusterlevelpng3" data-toggle="list" href="#PHIclusterlevelpng3" role="tab" aria-controls="PHIclusterlevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.21 Abundance clustering heatmap of PHI</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">PHI cluster heatmap:result/04.Annotation/PHI/Heatmap_sample/*/heatmap.sample.*.{pdf,png}</p>

        <br />

        <br />

<h3>4.5 Taxonomy and Function Diversity</h3>
        
    <a name="T4"></a>
    <h4>4.5.1 Alpha Diversity Analysis</h4>    

    <a name="T5"></a>
    <h5>4.5.1.1 Micro_NR</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of taxonomic level, calculate the richness and diversity of each sample (or group).The table illustrates an example with only species level.</p>

<p class="name_table"><p class="center"><strong>Table 4.4 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable3" >
        <thead><tr>
<a id="btn03"></a>
<a id="sjzl3"></a>
<a  href="javascript:void(0)" id="btn13"> | First | </a>
<a  href="javascript:void(0)" id="btn23"> Previous | </a>
<a  href="javascript:void(0)" id="btn33"> Next | </a>
<a  href="javascript:void(0)" id="btn43"> Last | </a>
<a>Go to </a>
<input id="changePage3" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn53"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">4153.16211857023</td><td style="text-align :center">4192.97540983607</td><td style="text-align :center">4.23652508774237</td><td style="text-align :center">0.958209284387269</td><td style="text-align :center">3989</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">4602.73575329709</td><td style="text-align :center">4635.43961352657</td><td style="text-align :center">4.50987271592601</td><td style="text-align :center">0.959378874314839</td><td style="text-align :center">4469</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">4788.9436452936</td><td style="text-align :center">4853.04918032787</td><td style="text-align :center">4.64437045044106</td><td style="text-align :center">0.969292478414934</td><td style="text-align :center">4659</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">3423.94758375198</td><td style="text-align :center">3485.5303030303</td><td style="text-align :center">3.96051157512467</td><td style="text-align :center">0.940471347191081</td><td style="text-align :center">3268</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">4632.57530473899</td><td style="text-align :center">4642.5</td><td style="text-align :center">4.56171871530976</td><td style="text-align :center">0.967498440202328</td><td style="text-align :center">4521</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">4918.94735285027</td><td style="text-align :center">4962.80597014925</td><td style="text-align :center">4.44206779235049</td><td style="text-align :center">0.951241490383073</td><td style="text-align :center">4794</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">5518.4225119167</td><td style="text-align :center">5563.82258064516</td><td style="text-align :center">4.50076093446646</td><td style="text-align :center">0.935731650056453</td><td style="text-align :center">5416</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">4764.9617400887</td><td style="text-align :center">4805.17374517375</td><td style="text-align :center">2.65115165076833</td><td style="text-align :center">0.68023414806216</td><td style="text-align :center">4625</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">4784.89214043213</td><td style="text-align :center">4829.84435797665</td><td style="text-align :center">2.90484187183182</td><td style="text-align :center">0.77064545616734</td><td style="text-align :center">4630</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">5544.13245194484</td><td style="text-align :center">5570.5</td><td style="text-align :center">4.39565960090137</td><td style="text-align :center">0.951783196735385</td><td style="text-align :center">5469</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">4196.11300437445</td><td style="text-align :center">4298.1013215859</td><td style="text-align :center">2.1159518202566</td><td style="text-align :center">0.609310388100981</td><td style="text-align :center">3999</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">5043.2503030825</td><td style="text-align :center">5090.16901408451</td><td style="text-align :center">3.48569723474056</td><td style="text-align :center">0.838059772573625</td><td style="text-align :center">4912</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">5168.35162106894</td><td style="text-align :center">5190.424</td><td style="text-align :center">4.49203323759425</td><td style="text-align :center">0.96721798846755</td><td style="text-align :center">5043</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">5270.91753977194</td><td style="text-align :center">5308.20873786408</td><td style="text-align :center">4.4582227679025</td><td style="text-align :center">0.960715860494338</td><td style="text-align :center">5151</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">5576.1807877487</td><td style="text-align :center">5606.85534591195</td><td style="text-align :center">4.73414115765239</td><td style="text-align :center">0.972438698767873</td><td style="text-align :center">5510</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">4851.77395793166</td><td style="text-align :center">4884.05882352941</td><td style="text-align :center">3.73204558562936</td><td style="text-align :center">0.921457130317677</td><td style="text-align :center">4714</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">5317.7939557869</td><td style="text-align :center">5331.68181818182</td><td style="text-align :center">4.6063846836177</td><td style="text-align :center">0.968427081913359</td><td style="text-align :center">5241</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">5614.77681166886</td><td style="text-align :center">5663.54022988506</td><td style="text-align :center">4.7139141440765</td><td style="text-align :center">0.972334502038189</td><td style="text-align :center">5520</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Micro_NR Alpha Diversity indices statistics: result/05.Diversity/MicroNR/AlphaDiversity/Alpha_index_table/tax_Alphaindex.*.xls</p>

    <a name="T5"></a>
    <h5>4.5.1.2 KEGG</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of KEGG, calculate the richness and diversity of each sample (or group).The table illustrates an example with only level3.</p>

<p class="name_table"><p class="center"><strong>Table 4.5 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable4" >
        <thead><tr>
<a id="btn04"></a>
<a id="sjzl4"></a>
<a  href="javascript:void(0)" id="btn14"> | First | </a>
<a  href="javascript:void(0)" id="btn24"> Previous | </a>
<a  href="javascript:void(0)" id="btn34"> Next | </a>
<a  href="javascript:void(0)" id="btn44"> Last | </a>
<a>Go to </a>
<input id="changePage4" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn54"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">273.707444598338</td><td style="text-align :center">273</td><td style="text-align :center">4.77411934855941</td><td style="text-align :center">0.987928684299046</td><td style="text-align :center">272</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">285</td><td style="text-align :center">285</td><td style="text-align :center">4.76457945770708</td><td style="text-align :center">0.987648971413218</td><td style="text-align :center">285</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">270</td><td style="text-align :center">270</td><td style="text-align :center">4.77166244174991</td><td style="text-align :center">0.987768810460186</td><td style="text-align :center">270</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">269</td><td style="text-align :center">269</td><td style="text-align :center">4.76710073185288</td><td style="text-align :center">0.987827943762566</td><td style="text-align :center">269</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">286.622402962375</td><td style="text-align :center">285.5</td><td style="text-align :center">4.77100544765721</td><td style="text-align :center">0.987880727814654</td><td style="text-align :center">283</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">283.714825816905</td><td style="text-align :center">285.5</td><td style="text-align :center">4.75105915789526</td><td style="text-align :center">0.98720532799813</td><td style="text-align :center">278</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">303.147321428571</td><td style="text-align :center">303</td><td style="text-align :center">4.72245567149193</td><td style="text-align :center">0.98655069095923</td><td style="text-align :center">303</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">305.441840277778</td><td style="text-align :center">312</td><td style="text-align :center">4.63637994828289</td><td style="text-align :center">0.984859130862253</td><td style="text-align :center">302</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">287</td><td style="text-align :center">287</td><td style="text-align :center">4.63189666241211</td><td style="text-align :center">0.984668674272447</td><td style="text-align :center">287</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">306.306818181818</td><td style="text-align :center">304.333333333333</td><td style="text-align :center">4.65682146718501</td><td style="text-align :center">0.984927662405129</td><td style="text-align :center">301</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">295.360248447205</td><td style="text-align :center">295.5</td><td style="text-align :center">4.60057688981784</td><td style="text-align :center">0.984162491412753</td><td style="text-align :center">295</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">297</td><td style="text-align :center">297</td><td style="text-align :center">4.66394302418658</td><td style="text-align :center">0.985165454799683</td><td style="text-align :center">297</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">415.929402573907</td><td style="text-align :center">415.333333333333</td><td style="text-align :center">4.67284280123368</td><td style="text-align :center">0.985395128698675</td><td style="text-align :center">415</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">415.53825</td><td style="text-align :center">415</td><td style="text-align :center">4.67590954874842</td><td style="text-align :center">0.985014290363385</td><td style="text-align :center">415</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">415</td><td style="text-align :center">415</td><td style="text-align :center">4.761726454503</td><td style="text-align :center">0.985823169202734</td><td style="text-align :center">415</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">412</td><td style="text-align :center">412</td><td style="text-align :center">4.68908642661952</td><td style="text-align :center">0.984705702675436</td><td style="text-align :center">412</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">414.302900128103</td><td style="text-align :center">414</td><td style="text-align :center">4.67650949443659</td><td style="text-align :center">0.98521938619125</td><td style="text-align :center">414</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">416</td><td style="text-align :center">416</td><td style="text-align :center">4.71185254502252</td><td style="text-align :center">0.98526606576296</td><td style="text-align :center">416</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">KEGG Alpha Diversity indices statistics: result/05.Diversity/KEGG/AlphaDiversity/Alpha_index_table/KEGG_Alphaindex.*.xls</p>

    <a name="T5"></a>
    <h5>4.5.1.3 eggNOG</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of eggNOG, calculate the richness and diversity of each sample (or group).The table illustrates an example with only og.</p>

<p class="name_table"><p class="center"><strong>Table 4.6 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable5" >
        <thead><tr>
<a id="btn05"></a>
<a id="sjzl5"></a>
<a  href="javascript:void(0)" id="btn15"> | First | </a>
<a  href="javascript:void(0)" id="btn25"> Previous | </a>
<a  href="javascript:void(0)" id="btn35"> Next | </a>
<a  href="javascript:void(0)" id="btn45"> Last | </a>
<a>Go to </a>
<input id="changePage5" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn55"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">152504.599479114</td><td style="text-align :center">153133.947639359</td><td style="text-align :center">10.0308450107352</td><td style="text-align :center">0.999855253951031</td><td style="text-align :center">149285</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">162663.085761941</td><td style="text-align :center">163204.680626546</td><td style="text-align :center">10.2334392392994</td><td style="text-align :center">0.999876439458422</td><td style="text-align :center">159904</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">169126.640172966</td><td style="text-align :center">170039.179750711</td><td style="text-align :center">10.3798407715437</td><td style="text-align :center">0.999881247443049</td><td style="text-align :center">166868</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">132799.509769068</td><td style="text-align :center">133354.936900056</td><td style="text-align :center">9.81051100719566</td><td style="text-align :center">0.999829571396549</td><td style="text-align :center">129080</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">163229.388556906</td><td style="text-align :center">163546.916968647</td><td style="text-align :center">10.1513746017508</td><td style="text-align :center">0.999861500677522</td><td style="text-align :center">160807</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">174509.249480247</td><td style="text-align :center">175136.448888889</td><td style="text-align :center">10.4439069602209</td><td style="text-align :center">0.99988742958355</td><td style="text-align :center">172227</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">216442.692036418</td><td style="text-align :center">217381.022929936</td><td style="text-align :center">10.6687509218202</td><td style="text-align :center">0.99989923521811</td><td style="text-align :center">214258</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">188301.105426006</td><td style="text-align :center">189445.893065693</td><td style="text-align :center">9.97157599261154</td><td style="text-align :center">0.999834118948884</td><td style="text-align :center">184526</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">186280.372550211</td><td style="text-align :center">187579.126540881</td><td style="text-align :center">10.1160966387582</td><td style="text-align :center">0.999861134540014</td><td style="text-align :center">182517</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">211885.200124527</td><td style="text-align :center">212416.459049545</td><td style="text-align :center">10.5720278220846</td><td style="text-align :center">0.999890405077804</td><td style="text-align :center">209938</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">167895.847107994</td><td style="text-align :center">169689.026962539</td><td style="text-align :center">9.73775863769876</td><td style="text-align :center">0.999810109884451</td><td style="text-align :center">162519</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">198189.338070971</td><td style="text-align :center">199220.965997468</td><td style="text-align :center">10.3916857730234</td><td style="text-align :center">0.999882522163175</td><td style="text-align :center">195954</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">207489.893751353</td><td style="text-align :center">208037.286532422</td><td style="text-align :center">10.4787461728404</td><td style="text-align :center">0.999896225564693</td><td style="text-align :center">202754</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">216463.903421993</td><td style="text-align :center">217943.891014691</td><td style="text-align :center">10.4786726680676</td><td style="text-align :center">0.999888625691143</td><td style="text-align :center">214092</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">221932.523445026</td><td style="text-align :center">222731.370864886</td><td style="text-align :center">10.6369735109214</td><td style="text-align :center">0.999890338389475</td><td style="text-align :center">219781</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">207213.636732081</td><td style="text-align :center">207630.462481275</td><td style="text-align :center">10.219197904323</td><td style="text-align :center">0.999880367165942</td><td style="text-align :center">204181</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">210383.156206416</td><td style="text-align :center">210910.465192021</td><td style="text-align :center">10.5243461558896</td><td style="text-align :center">0.999889771877566</td><td style="text-align :center">208377</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">220135.56193467</td><td style="text-align :center">221209.377305463</td><td style="text-align :center">10.5607272186908</td><td style="text-align :center">0.999881922546549</td><td style="text-align :center">217390</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">eggNOG Alpha Diversity indices statistics: result/05.Diversity/eggNOG/AlphaDiversity/Alpha_index_table/eggNOG_Alphaindex.*.xls</p>

    <a name="T5"></a>
    <h5>4.5.1.4 CAZy</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of CAZy, calculate the richness and diversity of each sample (or group).The table illustrates an example with only level2.</p>

<p class="name_table"><p class="center"><strong>Table 4.7 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable6" >
        <thead><tr>
<a id="btn06"></a>
<a id="sjzl6"></a>
<a  href="javascript:void(0)" id="btn16"> | First | </a>
<a  href="javascript:void(0)" id="btn26"> Previous | </a>
<a  href="javascript:void(0)" id="btn36"> Next | </a>
<a  href="javascript:void(0)" id="btn46"> Last | </a>
<a>Go to </a>
<input id="changePage6" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn56"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">315.312536254797</td><td style="text-align :center">314.6</td><td style="text-align :center">4.22572314998574</td><td style="text-align :center">0.964337753356556</td><td style="text-align :center">314</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">318.214953271028</td><td style="text-align :center">318</td><td style="text-align :center">4.2356347474775</td><td style="text-align :center">0.96510466630556</td><td style="text-align :center">318</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">326.336526077098</td><td style="text-align :center">326</td><td style="text-align :center">4.21433113268013</td><td style="text-align :center">0.964255266260201</td><td style="text-align :center">326</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">304.545796308954</td><td style="text-align :center">304.125</td><td style="text-align :center">4.21598324541129</td><td style="text-align :center">0.965094789977119</td><td style="text-align :center">304</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">323.943910989179</td><td style="text-align :center">323.5</td><td style="text-align :center">4.25715557512568</td><td style="text-align :center">0.96563683295147</td><td style="text-align :center">322</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">328.075435519603</td><td style="text-align :center">327.6</td><td style="text-align :center">4.21043990155504</td><td style="text-align :center">0.963683989142133</td><td style="text-align :center">327</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">331.376680391569</td><td style="text-align :center">330.5</td><td style="text-align :center">4.1759323901759</td><td style="text-align :center">0.961842108822012</td><td style="text-align :center">330</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">312.680309442895</td><td style="text-align :center">313</td><td style="text-align :center">3.92760303601137</td><td style="text-align :center">0.956142341984569</td><td style="text-align :center">310</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">316.413154031694</td><td style="text-align :center">315.5</td><td style="text-align :center">4.02230554411756</td><td style="text-align :center">0.95913785458032</td><td style="text-align :center">314</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">324.754409165509</td><td style="text-align :center">325.5</td><td style="text-align :center">4.08575025422213</td><td style="text-align :center">0.959941700907112</td><td style="text-align :center">324</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">301.304899930616</td><td style="text-align :center">302.090909090909</td><td style="text-align :center">3.93406985825386</td><td style="text-align :center">0.957900540309229</td><td style="text-align :center">295</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">314.568775686428</td><td style="text-align :center">314.2</td><td style="text-align :center">3.99236952338246</td><td style="text-align :center">0.957977119677532</td><td style="text-align :center">314</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">345.295207490014</td><td style="text-align :center">345.142857142857</td><td style="text-align :center">4.04328177272977</td><td style="text-align :center">0.957837136968416</td><td style="text-align :center">343</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">341.827769557028</td><td style="text-align :center">342</td><td style="text-align :center">4.07812283334349</td><td style="text-align :center">0.95964309287744</td><td style="text-align :center">341</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">355.328571128285</td><td style="text-align :center">355.75</td><td style="text-align :center">4.10944296584069</td><td style="text-align :center">0.957804510804491</td><td style="text-align :center">352</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">343.423950089238</td><td style="text-align :center">342.428571428571</td><td style="text-align :center">4.08995668833175</td><td style="text-align :center">0.963238807332684</td><td style="text-align :center">341</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">341.181461030023</td><td style="text-align :center">341</td><td style="text-align :center">4.06111769030604</td><td style="text-align :center">0.956837981854216</td><td style="text-align :center">341</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">353.499705739011</td><td style="text-align :center">352.75</td><td style="text-align :center">4.06764003512731</td><td style="text-align :center">0.956146606838414</td><td style="text-align :center">352</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">CAZy Alpha Diversity indices statistics: result/05.Diversity/CAZy/AlphaDiversity/Alpha_index_table/CAZy_Alphaindex.*.xls</p>

    <a name="T5"></a>
    <h5>4.5.1.5 VFDB</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of VFDB, calculate the richness and diversity of each sample (or group).The table illustrates an example with only level2.</p>

<p class="name_table"><p class="center"><strong>Table 4.8 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable7" >
        <thead><tr>
<a id="btn07"></a>
<a id="sjzl7"></a>
<a  href="javascript:void(0)" id="btn17"> | First | </a>
<a  href="javascript:void(0)" id="btn27"> Previous | </a>
<a  href="javascript:void(0)" id="btn37"> Next | </a>
<a  href="javascript:void(0)" id="btn47"> Last | </a>
<a>Go to </a>
<input id="changePage7" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn57"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">388.964525555637</td><td style="text-align :center">388.75</td><td style="text-align :center">4.60681106128574</td><td style="text-align :center">0.98470464011846</td><td style="text-align :center">385</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">397.24041730913</td><td style="text-align :center">397</td><td style="text-align :center">4.72389561029957</td><td style="text-align :center">0.98644835192534</td><td style="text-align :center">397</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">403.03142081448</td><td style="text-align :center">403.5</td><td style="text-align :center">4.75642218434215</td><td style="text-align :center">0.986779414541669</td><td style="text-align :center">401</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">359.65264472372</td><td style="text-align :center">358.105263157895</td><td style="text-align :center">4.50371181939169</td><td style="text-align :center">0.983493528006494</td><td style="text-align :center">357</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">400.070456412606</td><td style="text-align :center">399.666666666667</td><td style="text-align :center">4.66003241837505</td><td style="text-align :center">0.985297833356714</td><td style="text-align :center">399</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">428.721668468319</td><td style="text-align :center">427.4</td><td style="text-align :center">4.79496079870919</td><td style="text-align :center">0.987081171202975</td><td style="text-align :center">425</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">477.46007927133</td><td style="text-align :center">479.111111111111</td><td style="text-align :center">4.8583725112885</td><td style="text-align :center">0.986956227260229</td><td style="text-align :center">473</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">453.105265940573</td><td style="text-align :center">452.769230769231</td><td style="text-align :center">4.65794775277879</td><td style="text-align :center">0.982700147697841</td><td style="text-align :center">450</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">439.798688328006</td><td style="text-align :center">439.666666666667</td><td style="text-align :center">4.72728292467256</td><td style="text-align :center">0.984065139323195</td><td style="text-align :center">438</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">474.228619058982</td><td style="text-align :center">473.75</td><td style="text-align :center">4.90421579051127</td><td style="text-align :center">0.986858924579309</td><td style="text-align :center">473</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">422.860236418291</td><td style="text-align :center">425.1</td><td style="text-align :center">4.55318367644758</td><td style="text-align :center">0.980740144991998</td><td style="text-align :center">416</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">462.366228159147</td><td style="text-align :center">464.333333333333</td><td style="text-align :center">4.87039302374642</td><td style="text-align :center">0.986420684981125</td><td style="text-align :center">461</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">422.015365013354</td><td style="text-align :center">426.375</td><td style="text-align :center">4.7970003134788</td><td style="text-align :center">0.98630623148142</td><td style="text-align :center">415</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">442.005573189256</td><td style="text-align :center">442.142857142857</td><td style="text-align :center">4.8146976230964</td><td style="text-align :center">0.985876990923083</td><td style="text-align :center">440</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">438.405496200752</td><td style="text-align :center">442.2</td><td style="text-align :center">4.80690610314774</td><td style="text-align :center">0.986319274165337</td><td style="text-align :center">435</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">420.532976209875</td><td style="text-align :center">421</td><td style="text-align :center">4.71480188994615</td><td style="text-align :center">0.983807665962192</td><td style="text-align :center">418</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">415.950879765396</td><td style="text-align :center">419</td><td style="text-align :center">4.8090694266594</td><td style="text-align :center">0.986351806659103</td><td style="text-align :center">414</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">440.323104610956</td><td style="text-align :center">443.75</td><td style="text-align :center">4.78512278327314</td><td style="text-align :center">0.986134841169294</td><td style="text-align :center">434</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">VFDB Alpha Diversity indices statistics: result/05.Diversity/VFDB/AlphaDiversity/Alpha_index_table/VFDB_Alphaindex.*.xls</p>

    <a name="T5"></a>
    <h5>4.5.1.6 PHI</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of PHI, calculate the richness and diversity of each sample (or group).The table illustrates an example with only level1.</p>

<p class="name_table"><p class="center"><strong>Table 4.9 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable8" >
        <thead><tr>
<a id="btn08"></a>
<a id="sjzl8"></a>
<a  href="javascript:void(0)" id="btn18"> | First | </a>
<a  href="javascript:void(0)" id="btn28"> Previous | </a>
<a  href="javascript:void(0)" id="btn38"> Next | </a>
<a  href="javascript:void(0)" id="btn48"> Last | </a>
<a>Go to </a>
<input id="changePage8" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn58"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">201</td><td style="text-align :center">201</td><td style="text-align :center">4.01759895340563</td><td style="text-align :center">0.968251945852084</td><td style="text-align :center">201</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">213.217614301625</td><td style="text-align :center">212.75</td><td style="text-align :center">4.0455743772716</td><td style="text-align :center">0.969478833047895</td><td style="text-align :center">212</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">210</td><td style="text-align :center">210</td><td style="text-align :center">4.06607335419074</td><td style="text-align :center">0.96984182576026</td><td style="text-align :center">210</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">200.206607729125</td><td style="text-align :center">199.75</td><td style="text-align :center">3.9989296105317</td><td style="text-align :center">0.968881328985398</td><td style="text-align :center">199</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">212</td><td style="text-align :center">212</td><td style="text-align :center">4.03725953724112</td><td style="text-align :center">0.96873147064498</td><td style="text-align :center">212</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">214</td><td style="text-align :center">214</td><td style="text-align :center">4.08325482974162</td><td style="text-align :center">0.970873227472076</td><td style="text-align :center">214</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">217.570881226054</td><td style="text-align :center">216</td><td style="text-align :center">4.08974464276702</td><td style="text-align :center">0.969045577602434</td><td style="text-align :center">215</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">219.744663846709</td><td style="text-align :center">218.5</td><td style="text-align :center">3.97011291985018</td><td style="text-align :center">0.967050525570614</td><td style="text-align :center">217</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">220.409962871287</td><td style="text-align :center">222</td><td style="text-align :center">3.96627329603286</td><td style="text-align :center">0.965778952140164</td><td style="text-align :center">217</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">225.79859541436</td><td style="text-align :center">225.25</td><td style="text-align :center">4.06400505287462</td><td style="text-align :center">0.967360764709619</td><td style="text-align :center">225</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">204.641517318228</td><td style="text-align :center">205</td><td style="text-align :center">3.91842811777064</td><td style="text-align :center">0.964926787797584</td><td style="text-align :center">204</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">219.624441848297</td><td style="text-align :center">219.333333333333</td><td style="text-align :center">3.99160103073352</td><td style="text-align :center">0.964622868561648</td><td style="text-align :center">219</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">236.365472580093</td><td style="text-align :center">235.25</td><td style="text-align :center">4.04084851612471</td><td style="text-align :center">0.967825529822778</td><td style="text-align :center">234</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">241.801979530165</td><td style="text-align :center">242.5</td><td style="text-align :center">4.03696408799607</td><td style="text-align :center">0.967121068825541</td><td style="text-align :center">241</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">241.406060606061</td><td style="text-align :center">240.25</td><td style="text-align :center">4.12384770413627</td><td style="text-align :center">0.970215262684218</td><td style="text-align :center">240</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">238.327971359609</td><td style="text-align :center">238</td><td style="text-align :center">3.94969923613933</td><td style="text-align :center">0.962657889600896</td><td style="text-align :center">238</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">238.340425531915</td><td style="text-align :center">239</td><td style="text-align :center">4.07136359697583</td><td style="text-align :center">0.968899322601918</td><td style="text-align :center">238</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">244.066582407443</td><td style="text-align :center">246</td><td style="text-align :center">4.11760993118336</td><td style="text-align :center">0.970468601671707</td><td style="text-align :center">243</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PHI Alpha Diversity indices statistics: result/05.Diversity/PHI/AlphaDiversity/Alpha_index/PHI_Alphaindex.*.xls</p>

    <a name="T4"></a>
    <h4>4.5.2 Sample clustering with taxonomic and function abundance</h4>    

    <p class="paragraph">Sample clustering analysis based on Bray-Curtis distance can evaluate the similarity of samples. The distance was calculated according to relative taxonomic abundance. And the final results were exhibited by combining the clustering result and relative abundance of different samples in phylum level or function level.</p>

    <a name="T5"></a>
    <h5>4.5.2.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceinMicroNR_nav-tabContent">

<div class="tab-pane active show" id="Bartreektaxpng1" role="tabpanel" aria-labelledby="Bartreektaxpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.k.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreeptaxpng2" role="tabpanel" aria-labelledby="Bartreeptaxpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.p.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreectaxpng3" role="tabpanel" aria-labelledby="Bartreectaxpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.c.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreeotaxpng4" role="tabpanel" aria-labelledby="Bartreeotaxpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.o.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreeftaxpng5" role="tabpanel" aria-labelledby="Bartreeftaxpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.f.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreegtaxpng6" role="tabpanel" aria-labelledby="Bartreegtaxpng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.g.tax.10.png" /></div>

<div class="tab-pane fade" id="Bartreestaxpng7" role="tabpanel" aria-labelledby="Bartreestaxpng7" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.s.tax.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceinMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceinMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreektaxpng1" data-toggle="list" href="#Bartreektaxpng1" role="tab" aria-controls="Bartreektaxpng1">Kingdom</a>

<a class="list-group-item list-group-item-action" id="Bartreeptaxpng2" data-toggle="list" href="#Bartreeptaxpng2" role="tab" aria-controls="Bartreeptaxpng2">Phylum</a>

<a class="list-group-item list-group-item-action" id="Bartreectaxpng3" data-toggle="list" href="#Bartreectaxpng3" role="tab" aria-controls="Bartreectaxpng3">Class</a>

<a class="list-group-item list-group-item-action" id="Bartreeotaxpng4" data-toggle="list" href="#Bartreeotaxpng4" role="tab" aria-controls="Bartreeotaxpng4">Order</a>

<a class="list-group-item list-group-item-action" id="Bartreeftaxpng5" data-toggle="list" href="#Bartreeftaxpng5" role="tab" aria-controls="Bartreeftaxpng5">Family</a>

<a class="list-group-item list-group-item-action" id="Bartreegtaxpng6" data-toggle="list" href="#Bartreegtaxpng6" role="tab" aria-controls="Bartreegtaxpng6">Genus</a>

<a class="list-group-item list-group-item-action" id="Bartreestaxpng7" data-toggle="list" href="#Bartreestaxpng7" role="tab" aria-controls="Bartreestaxpng7">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.22 Clustering tree based on Bray-Curtis distance in Micro_NR</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/MicroNR/ClusterTree_group1</p>

    <a name="T5"></a>
    <h5>4.5.2.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceinKEGG_nav-tabContent">

<div class="tab-pane active show" id="Bartreelevelkeggpng1" role="tabpanel" aria-labelledby="Bartreelevelkeggpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level1.kegg.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelkeggpng2" role="tabpanel" aria-labelledby="Bartreelevelkeggpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level2.kegg.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelkeggpng3" role="tabpanel" aria-labelledby="Bartreelevelkeggpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level3.kegg.10.png" /></div>

<div class="tab-pane fade" id="Bartreekokeggpng4" role="tabpanel" aria-labelledby="Bartreekokeggpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.ko.kegg.10.png" /></div>

<div class="tab-pane fade" id="Bartreeeckeggpng5" role="tabpanel" aria-labelledby="Bartreeeckeggpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.ec.kegg.10.png" /></div>

<div class="tab-pane fade" id="Bartreemodulekeggpng6" role="tabpanel" aria-labelledby="Bartreemodulekeggpng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.module.kegg.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceinKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceinKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreelevelkeggpng1" data-toggle="list" href="#Bartreelevelkeggpng1" role="tab" aria-controls="Bartreelevelkeggpng1">level1</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelkeggpng2" data-toggle="list" href="#Bartreelevelkeggpng2" role="tab" aria-controls="Bartreelevelkeggpng2">level2</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelkeggpng3" data-toggle="list" href="#Bartreelevelkeggpng3" role="tab" aria-controls="Bartreelevelkeggpng3">level3</a>

<a class="list-group-item list-group-item-action" id="Bartreekokeggpng4" data-toggle="list" href="#Bartreekokeggpng4" role="tab" aria-controls="Bartreekokeggpng4">KO</a>

<a class="list-group-item list-group-item-action" id="Bartreeeckeggpng5" data-toggle="list" href="#Bartreeeckeggpng5" role="tab" aria-controls="Bartreeeckeggpng5">ec</a>

<a class="list-group-item list-group-item-action" id="Bartreemodulekeggpng6" data-toggle="list" href="#Bartreemodulekeggpng6" role="tab" aria-controls="Bartreemodulekeggpng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.23 Clustering tree based on Bray-Curtis distance in KEGG</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/KEGG/ClusterTree_group1</p>

    <a name="T5"></a>
    <h5>4.5.2.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceineggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceineggNOG_nav-tabContent">

<div class="tab-pane active show" id="Bartreeleveleggnogpng1" role="tabpanel" aria-labelledby="Bartreeleveleggnogpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level1.eggnog.10.png" /></div>

<div class="tab-pane fade" id="Bartreeleveleggnogpng2" role="tabpanel" aria-labelledby="Bartreeleveleggnogpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level2.eggnog.10.png" /></div>

<div class="tab-pane fade" id="Bartreeogeggnogpng3" role="tabpanel" aria-labelledby="Bartreeogeggnogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.og.eggnog.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceineggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceineggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceineggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreeleveleggnogpng1" data-toggle="list" href="#Bartreeleveleggnogpng1" role="tab" aria-controls="Bartreeleveleggnogpng1">level1</a>

<a class="list-group-item list-group-item-action" id="Bartreeleveleggnogpng2" data-toggle="list" href="#Bartreeleveleggnogpng2" role="tab" aria-controls="Bartreeleveleggnogpng2">level2</a>

<a class="list-group-item list-group-item-action" id="Bartreeogeggnogpng3" data-toggle="list" href="#Bartreeogeggnogpng3" role="tab" aria-controls="Bartreeogeggnogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.24 Clustering tree based on Bray-Curtis distance in eggNOG</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/eggNOG/ClusterTree_group1</p>

    <a name="T5"></a>
    <h5>4.5.2.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceinCAZy_nav-tabContent">

<div class="tab-pane active show" id="Bartreelevelcazypng1" role="tabpanel" aria-labelledby="Bartreelevelcazypng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level1.cazy.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelcazypng2" role="tabpanel" aria-labelledby="Bartreelevelcazypng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level2.cazy.10.png" /></div>

<div class="tab-pane fade" id="Bartreeeccazypng3" role="tabpanel" aria-labelledby="Bartreeeccazypng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.ec.cazy.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceinCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceinCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreelevelcazypng1" data-toggle="list" href="#Bartreelevelcazypng1" role="tab" aria-controls="Bartreelevelcazypng1">level1</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelcazypng2" data-toggle="list" href="#Bartreelevelcazypng2" role="tab" aria-controls="Bartreelevelcazypng2">level2</a>

<a class="list-group-item list-group-item-action" id="Bartreeeccazypng3" data-toggle="list" href="#Bartreeeccazypng3" role="tab" aria-controls="Bartreeeccazypng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.25 Clustering tree based on Bray-Curtis distance in CAZy</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/CAZy/ClusterTree_group1</p>

    <a name="T5"></a>
    <h5>4.5.2.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceinVFDB_nav-tabContent">

<div class="tab-pane active show" id="Bartreelevelvfdbpng1" role="tabpanel" aria-labelledby="Bartreelevelvfdbpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level1.vfdb.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelvfdbpng2" role="tabpanel" aria-labelledby="Bartreelevelvfdbpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level2.vfdb.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceinVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceinVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreelevelvfdbpng1" data-toggle="list" href="#Bartreelevelvfdbpng1" role="tab" aria-controls="Bartreelevelvfdbpng1">level1</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelvfdbpng2" data-toggle="list" href="#Bartreelevelvfdbpng2" role="tab" aria-controls="Bartreelevelvfdbpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.26 Clustering tree based on Bray-Curtis distance in VFDB</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/VFDB/ClusterTree_group1</p>

    <a name="T5"></a>
    <h5>4.5.2.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureClusteringtreebasedonBrayCurtisdistanceinPHI_nav-tabContent">

<div class="tab-pane active show" id="Bartreelevelphipng1" role="tabpanel" aria-labelledby="Bartreelevelphipng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level1.phi.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelphipng2" role="tabpanel" aria-labelledby="Bartreelevelphipng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level2.phi.10.png" /></div>

<div class="tab-pane fade" id="Bartreelevelphipng3" role="tabpanel" aria-labelledby="Bartreelevelphipng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/Bar.tree.level3.phi.10.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureClusteringtreebasedonBrayCurtisdistanceinPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureClusteringtreebasedonBrayCurtisdistanceinPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureClusteringtreebasedonBrayCurtisdistanceinPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="Bartreelevelphipng1" data-toggle="list" href="#Bartreelevelphipng1" role="tab" aria-controls="Bartreelevelphipng1">level1</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelphipng2" data-toggle="list" href="#Bartreelevelphipng2" role="tab" aria-controls="Bartreelevelphipng2">level2</a>

<a class="list-group-item list-group-item-action" id="Bartreelevelphipng3" data-toggle="list" href="#Bartreelevelphipng3" role="tab" aria-controls="Bartreelevelphipng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.27 Clustering tree based on Bray-Curtis distance in PHI</strong></p>  
<p class="premark" >Note: In the figure, the left side is the structure of the Bray-Curtis distance clustering tree; the right side is the relative abundance distribution of each sample at the taxonomic or functional level.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Clustering tree based on Bray-Curtis distance: result/05.Diversity/PHI/ClusterTree_group1</p>

    <a name="T4"></a>
    <h4>4.5.3 PCA Analysis</h4>    

    <p class="paragraph">Principal Component Analysis (PCA) Analysis is a method used for dimension reduction analysis. PCA uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. The PCA transformation ensures that the horizontal axis PC1 accounts for the most variation while the vertical axis PC2 accounts for the second most (Avershina et al., 2013). For the samples' community composition, the closer the sample points within the PCA figure indicates more similarity among the samples.</p>

    <a name="T5"></a>
    <h5>4.5.3.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinMicroNR_nav-tabContent">

<div class="tab-pane active show" id="PCAtaxphylumpng1" role="tabpanel" aria-labelledby="PCAtaxphylumpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.phylum.png" /></div>

<div class="tab-pane fade" id="PCAtaxclasspng2" role="tabpanel" aria-labelledby="PCAtaxclasspng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.class.png" /></div>

<div class="tab-pane fade" id="PCAtaxorderpng3" role="tabpanel" aria-labelledby="PCAtaxorderpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.order.png" /></div>

<div class="tab-pane fade" id="PCAtaxfamilypng4" role="tabpanel" aria-labelledby="PCAtaxfamilypng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.family.png" /></div>

<div class="tab-pane fade" id="PCAtaxgenuspng5" role="tabpanel" aria-labelledby="PCAtaxgenuspng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.genus.png" /></div>

<div class="tab-pane fade" id="PCAtaxspeciespng6" role="tabpanel" aria-labelledby="PCAtaxspeciespng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAtaxphylumpng1" data-toggle="list" href="#PCAtaxphylumpng1" role="tab" aria-controls="PCAtaxphylumpng1">Phylum</a>

<a class="list-group-item list-group-item-action" id="PCAtaxclasspng2" data-toggle="list" href="#PCAtaxclasspng2" role="tab" aria-controls="PCAtaxclasspng2">Class</a>

<a class="list-group-item list-group-item-action" id="PCAtaxorderpng3" data-toggle="list" href="#PCAtaxorderpng3" role="tab" aria-controls="PCAtaxorderpng3">Order</a>

<a class="list-group-item list-group-item-action" id="PCAtaxfamilypng4" data-toggle="list" href="#PCAtaxfamilypng4" role="tab" aria-controls="PCAtaxfamilypng4">Family</a>

<a class="list-group-item list-group-item-action" id="PCAtaxgenuspng5" data-toggle="list" href="#PCAtaxgenuspng5" role="tab" aria-controls="PCAtaxgenuspng5">Genus</a>

<a class="list-group-item list-group-item-action" id="PCAtaxspeciespng6" data-toggle="list" href="#PCAtaxspeciespng6" role="tab" aria-controls="PCAtaxspeciespng6">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.28 PCA analysis in Micro_NR</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/MicroNR/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/MicroNR/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/MicroNR/PCA_group1/*/PCA.xls</p>

    <a name="T5"></a>
    <h5>4.5.3.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinKEGG_nav-tabContent">

<div class="tab-pane active show" id="PCAkegglevelpng1" role="tabpanel" aria-labelledby="PCAkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.level1.png" /></div>

<div class="tab-pane fade" id="PCAkegglevelpng2" role="tabpanel" aria-labelledby="PCAkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.level2.png" /></div>

<div class="tab-pane fade" id="PCAkegglevelpng3" role="tabpanel" aria-labelledby="PCAkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.level3.png" /></div>

<div class="tab-pane fade" id="PCAkeggkopng4" role="tabpanel" aria-labelledby="PCAkeggkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.ko.png" /></div>

<div class="tab-pane fade" id="PCAkeggecpng5" role="tabpanel" aria-labelledby="PCAkeggecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.ec.png" /></div>

<div class="tab-pane fade" id="PCAkeggmodulepng6" role="tabpanel" aria-labelledby="PCAkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAkegglevelpng1" data-toggle="list" href="#PCAkegglevelpng1" role="tab" aria-controls="PCAkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCAkegglevelpng2" data-toggle="list" href="#PCAkegglevelpng2" role="tab" aria-controls="PCAkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCAkegglevelpng3" data-toggle="list" href="#PCAkegglevelpng3" role="tab" aria-controls="PCAkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="PCAkeggkopng4" data-toggle="list" href="#PCAkeggkopng4" role="tab" aria-controls="PCAkeggkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="PCAkeggecpng5" data-toggle="list" href="#PCAkeggecpng5" role="tab" aria-controls="PCAkeggecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="PCAkeggmodulepng6" data-toggle="list" href="#PCAkeggmodulepng6" role="tab" aria-controls="PCAkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.29 PCA analysis in KEGG</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/KEGG/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/KEGG/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/KEGG/PCA_group1/*/PCA.xls</p>

    <a name="T5"></a>
    <h5>4.5.3.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisineggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisineggNOG_nav-tabContent">

<div class="tab-pane active show" id="PCAeggnoglevelpng1" role="tabpanel" aria-labelledby="PCAeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="PCAeggnoglevelpng2" role="tabpanel" aria-labelledby="PCAeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.eggnog.level2.png" /></div>

<div class="tab-pane fade" id="PCAeggnogogpng3" role="tabpanel" aria-labelledby="PCAeggnogogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.eggnog.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisineggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisineggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisineggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAeggnoglevelpng1" data-toggle="list" href="#PCAeggnoglevelpng1" role="tab" aria-controls="PCAeggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCAeggnoglevelpng2" data-toggle="list" href="#PCAeggnoglevelpng2" role="tab" aria-controls="PCAeggnoglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCAeggnogogpng3" data-toggle="list" href="#PCAeggnogogpng3" role="tab" aria-controls="PCAeggnogogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.30 PCA analysis in eggNOG</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/eggNOG/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/eggNOG/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/eggNOG/PCA_group1/*/PCA.xls</p>

    <a name="T5"></a>
    <h5>4.5.3.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinCAZy_nav-tabContent">

<div class="tab-pane active show" id="PCAcazylevelpng1" role="tabpanel" aria-labelledby="PCAcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.cazy.level1.png" /></div>

<div class="tab-pane fade" id="PCAcazylevelpng2" role="tabpanel" aria-labelledby="PCAcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.cazy.level2.png" /></div>

<div class="tab-pane fade" id="PCAcazyecpng3" role="tabpanel" aria-labelledby="PCAcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAcazylevelpng1" data-toggle="list" href="#PCAcazylevelpng1" role="tab" aria-controls="PCAcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCAcazylevelpng2" data-toggle="list" href="#PCAcazylevelpng2" role="tab" aria-controls="PCAcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCAcazyecpng3" data-toggle="list" href="#PCAcazyecpng3" role="tab" aria-controls="PCAcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.31 PCA analysis in CAZy</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/CAZy/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/CAZy/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/CAZy/PCA_group1/*/PCA.xls</p>

    <a name="T5"></a>
    <h5>4.5.3.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinVFDB_nav-tabContent">

<div class="tab-pane active show" id="PCAvfdblevelpng1" role="tabpanel" aria-labelledby="PCAvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="PCAvfdblevelpng2" role="tabpanel" aria-labelledby="PCAvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAvfdblevelpng1" data-toggle="list" href="#PCAvfdblevelpng1" role="tab" aria-controls="PCAvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCAvfdblevelpng2" data-toggle="list" href="#PCAvfdblevelpng2" role="tab" aria-controls="PCAvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.32 PCA analysis in VFDB</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/VFDB/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/VFDB/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/VFDB/PCA_group1/*/PCA.xls</p>

    <a name="T5"></a>
    <h5>4.5.3.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinPHI_nav-tabContent">

<div class="tab-pane active show" id="PCAphilevelpng1" role="tabpanel" aria-labelledby="PCAphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.phi.level1.png" /></div>

<div class="tab-pane fade" id="PCAphilevelpng2" role="tabpanel" aria-labelledby="PCAphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.phi.level2.png" /></div>

<div class="tab-pane fade" id="PCAphilevelpng3" role="tabpanel" aria-labelledby="PCAphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCA.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCAphilevelpng1" data-toggle="list" href="#PCAphilevelpng1" role="tab" aria-controls="PCAphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCAphilevelpng2" data-toggle="list" href="#PCAphilevelpng2" role="tab" aria-controls="PCAphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCAphilevelpng3" data-toggle="list" href="#PCAphilevelpng3" role="tab" aria-controls="PCAphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.33 PCA analysis in PHI</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/05.Diversity/PHI/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/05.Diversity/PHI/PCA_group1/*/PCA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/PHI/PCA_group1/*/PCA.xls</p>

    <a name="T4"></a>
    <h4>4.5.4 PCoA Analysis</h4>    

    <p class="paragraph">Principal Coordinate Analysis (PCoA) is an ordination technique which picks up the primary elements and structure from reduced multi-dimensional data series of eigenvalues and eigenvectors. This technique has the advantage over PCA in that each ecological distance can be investigated. Aggregated samples represent higher species composition similarity than those that are further apart. PCoA result based on Bray-Curtis distance are shown below.</p>

    <a name="T5"></a>
    <h5>4.5.4.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinMicroNR_nav-tabContent">

<div class="tab-pane active show" id="PCoAtaxphylumpng1" role="tabpanel" aria-labelledby="PCoAtaxphylumpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.phylum.png" /></div>

<div class="tab-pane fade" id="PCoAtaxclasspng2" role="tabpanel" aria-labelledby="PCoAtaxclasspng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.class.png" /></div>

<div class="tab-pane fade" id="PCoAtaxorderpng3" role="tabpanel" aria-labelledby="PCoAtaxorderpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.order.png" /></div>

<div class="tab-pane fade" id="PCoAtaxfamilypng4" role="tabpanel" aria-labelledby="PCoAtaxfamilypng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.family.png" /></div>

<div class="tab-pane fade" id="PCoAtaxgenuspng5" role="tabpanel" aria-labelledby="PCoAtaxgenuspng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.genus.png" /></div>

<div class="tab-pane fade" id="PCoAtaxspeciespng6" role="tabpanel" aria-labelledby="PCoAtaxspeciespng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAtaxphylumpng1" data-toggle="list" href="#PCoAtaxphylumpng1" role="tab" aria-controls="PCoAtaxphylumpng1">Phylum</a>

<a class="list-group-item list-group-item-action" id="PCoAtaxclasspng2" data-toggle="list" href="#PCoAtaxclasspng2" role="tab" aria-controls="PCoAtaxclasspng2">Class</a>

<a class="list-group-item list-group-item-action" id="PCoAtaxorderpng3" data-toggle="list" href="#PCoAtaxorderpng3" role="tab" aria-controls="PCoAtaxorderpng3">Order</a>

<a class="list-group-item list-group-item-action" id="PCoAtaxfamilypng4" data-toggle="list" href="#PCoAtaxfamilypng4" role="tab" aria-controls="PCoAtaxfamilypng4">Family</a>

<a class="list-group-item list-group-item-action" id="PCoAtaxgenuspng5" data-toggle="list" href="#PCoAtaxgenuspng5" role="tab" aria-controls="PCoAtaxgenuspng5">Genus</a>

<a class="list-group-item list-group-item-action" id="PCoAtaxspeciespng6" data-toggle="list" href="#PCoAtaxspeciespng6" role="tab" aria-controls="PCoAtaxspeciespng6">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.34 PCoA analysis in Micro_NR</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/MicroNR/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/MicroNR/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/MicroNR/PCoA_group1/*/PCoA.xls</p>

    <a name="T5"></a>
    <h5>4.5.4.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinKEGG_nav-tabContent">

<div class="tab-pane active show" id="PCoAkegglevelpng1" role="tabpanel" aria-labelledby="PCoAkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.level1.png" /></div>

<div class="tab-pane fade" id="PCoAkegglevelpng2" role="tabpanel" aria-labelledby="PCoAkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.level2.png" /></div>

<div class="tab-pane fade" id="PCoAkegglevelpng3" role="tabpanel" aria-labelledby="PCoAkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.level3.png" /></div>

<div class="tab-pane fade" id="PCoAkeggkopng4" role="tabpanel" aria-labelledby="PCoAkeggkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.ko.png" /></div>

<div class="tab-pane fade" id="PCoAkeggecpng5" role="tabpanel" aria-labelledby="PCoAkeggecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.ec.png" /></div>

<div class="tab-pane fade" id="PCoAkeggmodulepng6" role="tabpanel" aria-labelledby="PCoAkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAkegglevelpng1" data-toggle="list" href="#PCoAkegglevelpng1" role="tab" aria-controls="PCoAkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCoAkegglevelpng2" data-toggle="list" href="#PCoAkegglevelpng2" role="tab" aria-controls="PCoAkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCoAkegglevelpng3" data-toggle="list" href="#PCoAkegglevelpng3" role="tab" aria-controls="PCoAkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="PCoAkeggkopng4" data-toggle="list" href="#PCoAkeggkopng4" role="tab" aria-controls="PCoAkeggkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="PCoAkeggecpng5" data-toggle="list" href="#PCoAkeggecpng5" role="tab" aria-controls="PCoAkeggecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="PCoAkeggmodulepng6" data-toggle="list" href="#PCoAkeggmodulepng6" role="tab" aria-controls="PCoAkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.35 PCoA analysis in KEGG</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/KEGG/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/KEGG/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/KEGG/PCoA_group1/*/PCoA.xls</p>

    <a name="T5"></a>
    <h5>4.5.4.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisineggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisineggNOG_nav-tabContent">

<div class="tab-pane active show" id="PCoAeggnoglevelpng1" role="tabpanel" aria-labelledby="PCoAeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="PCoAeggnoglevelpng2" role="tabpanel" aria-labelledby="PCoAeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.eggnog.level2.png" /></div>

<div class="tab-pane fade" id="PCoAeggnogogpng3" role="tabpanel" aria-labelledby="PCoAeggnogogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.eggnog.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisineggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisineggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisineggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAeggnoglevelpng1" data-toggle="list" href="#PCoAeggnoglevelpng1" role="tab" aria-controls="PCoAeggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCoAeggnoglevelpng2" data-toggle="list" href="#PCoAeggnoglevelpng2" role="tab" aria-controls="PCoAeggnoglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCoAeggnogogpng3" data-toggle="list" href="#PCoAeggnogogpng3" role="tab" aria-controls="PCoAeggnogogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.36 PCoA analysis in eggNOG</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/eggNOG/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/eggNOG/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/eggNOG/PCoA_group1/*/PCoA.xls</p>

    <a name="T5"></a>
    <h5>4.5.4.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinCAZy_nav-tabContent">

<div class="tab-pane active show" id="PCoAcazylevelpng1" role="tabpanel" aria-labelledby="PCoAcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.cazy.level1.png" /></div>

<div class="tab-pane fade" id="PCoAcazylevelpng2" role="tabpanel" aria-labelledby="PCoAcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.cazy.level2.png" /></div>

<div class="tab-pane fade" id="PCoAcazyecpng3" role="tabpanel" aria-labelledby="PCoAcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAcazylevelpng1" data-toggle="list" href="#PCoAcazylevelpng1" role="tab" aria-controls="PCoAcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCoAcazylevelpng2" data-toggle="list" href="#PCoAcazylevelpng2" role="tab" aria-controls="PCoAcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCoAcazyecpng3" data-toggle="list" href="#PCoAcazyecpng3" role="tab" aria-controls="PCoAcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.37 PCoA analysis in CAZy</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/CAZy/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/CAZy/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/CAZy/PCoA_group1/*/PCoA.xls</p>

    <a name="T5"></a>
    <h5>4.5.4.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinVFDB_nav-tabContent">

<div class="tab-pane active show" id="PCoAvfdblevelpng1" role="tabpanel" aria-labelledby="PCoAvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="PCoAvfdblevelpng2" role="tabpanel" aria-labelledby="PCoAvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAvfdblevelpng1" data-toggle="list" href="#PCoAvfdblevelpng1" role="tab" aria-controls="PCoAvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCoAvfdblevelpng2" data-toggle="list" href="#PCoAvfdblevelpng2" role="tab" aria-controls="PCoAvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.38 PCoA analysis in VFDB</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/VFDB/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/VFDB/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/VFDB/PCoA_group1/*/PCoA.xls</p>

    <a name="T5"></a>
    <h5>4.5.4.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinPHI_nav-tabContent">

<div class="tab-pane active show" id="PCoAphilevelpng1" role="tabpanel" aria-labelledby="PCoAphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.phi.level1.png" /></div>

<div class="tab-pane fade" id="PCoAphilevelpng2" role="tabpanel" aria-labelledby="PCoAphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.phi.level2.png" /></div>

<div class="tab-pane fade" id="PCoAphilevelpng3" role="tabpanel" aria-labelledby="PCoAphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/PCoA.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="PCoAphilevelpng1" data-toggle="list" href="#PCoAphilevelpng1" role="tab" aria-controls="PCoAphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="PCoAphilevelpng2" data-toggle="list" href="#PCoAphilevelpng2" role="tab" aria-controls="PCoAphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="PCoAphilevelpng3" data-toggle="list" href="#PCoAphilevelpng3" role="tab" aria-controls="PCoAphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.39 PCoA analysis in PHI</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/05.Diversity/PHI/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/05.Diversity/PHI/PCoA_group1/*/PCoA.{png,pdf}</p>

    <p class="paragraph">Result of principal components: result/05.Diversity/PHI/PCoA_group1/*/PCoA.xls</p>

    <a name="T4"></a>
    <h4>4.5.5 NMDS Analysis</h4>    

    <p class="paragraph">Non-metric Multidimensional Scaling (NMDS) is a ranking method applicable to ecological researches. It is a non-linear model designed for a better representation of non-linear biological data structure. NMDS can retain the original relationship among samples while reducing the research targets from multidimensional space to a low-dimensional space for analysis, positioning and categorization. This allows NMDS analysis to address the similarity or dissimilarity between targets as a monotonic function of the distance between the points. NMDS analysis results are shown as follows:</p>

    <a name="T5"></a>
    <h5>4.5.5.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinMicroNR_nav-tabContent">

<div class="tab-pane active show" id="NMDStaxphylumpng1" role="tabpanel" aria-labelledby="NMDStaxphylumpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.phylum.png" /></div>

<div class="tab-pane fade" id="NMDStaxclasspng2" role="tabpanel" aria-labelledby="NMDStaxclasspng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.class.png" /></div>

<div class="tab-pane fade" id="NMDStaxorderpng3" role="tabpanel" aria-labelledby="NMDStaxorderpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.order.png" /></div>

<div class="tab-pane fade" id="NMDStaxfamilypng4" role="tabpanel" aria-labelledby="NMDStaxfamilypng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.family.png" /></div>

<div class="tab-pane fade" id="NMDStaxgenuspng5" role="tabpanel" aria-labelledby="NMDStaxgenuspng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.genus.png" /></div>

<div class="tab-pane fade" id="NMDStaxspeciespng6" role="tabpanel" aria-labelledby="NMDStaxspeciespng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDStaxphylumpng1" data-toggle="list" href="#NMDStaxphylumpng1" role="tab" aria-controls="NMDStaxphylumpng1">Phylum</a>

<a class="list-group-item list-group-item-action" id="NMDStaxclasspng2" data-toggle="list" href="#NMDStaxclasspng2" role="tab" aria-controls="NMDStaxclasspng2">Class</a>

<a class="list-group-item list-group-item-action" id="NMDStaxorderpng3" data-toggle="list" href="#NMDStaxorderpng3" role="tab" aria-controls="NMDStaxorderpng3">Order</a>

<a class="list-group-item list-group-item-action" id="NMDStaxfamilypng4" data-toggle="list" href="#NMDStaxfamilypng4" role="tab" aria-controls="NMDStaxfamilypng4">Family</a>

<a class="list-group-item list-group-item-action" id="NMDStaxgenuspng5" data-toggle="list" href="#NMDStaxgenuspng5" role="tab" aria-controls="NMDStaxgenuspng5">Genus</a>

<a class="list-group-item list-group-item-action" id="NMDStaxspeciespng6" data-toggle="list" href="#NMDStaxspeciespng6" role="tab" aria-controls="NMDStaxspeciespng6">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.40 NMDS analysis in Micro_NR</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/MicroNR/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/MicroNR/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/MicroNR/NMDS_group1/*/NMDS_scores.xls</p>

    <a name="T5"></a>
    <h5>4.5.5.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinKEGG_nav-tabContent">

<div class="tab-pane active show" id="NMDSkegglevelpng1" role="tabpanel" aria-labelledby="NMDSkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.level1.png" /></div>

<div class="tab-pane fade" id="NMDSkegglevelpng2" role="tabpanel" aria-labelledby="NMDSkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.level2.png" /></div>

<div class="tab-pane fade" id="NMDSkegglevelpng3" role="tabpanel" aria-labelledby="NMDSkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.level3.png" /></div>

<div class="tab-pane fade" id="NMDSkeggkopng4" role="tabpanel" aria-labelledby="NMDSkeggkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.ko.png" /></div>

<div class="tab-pane fade" id="NMDSkeggecpng5" role="tabpanel" aria-labelledby="NMDSkeggecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.ec.png" /></div>

<div class="tab-pane fade" id="NMDSkeggmodulepng6" role="tabpanel" aria-labelledby="NMDSkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDSkegglevelpng1" data-toggle="list" href="#NMDSkegglevelpng1" role="tab" aria-controls="NMDSkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="NMDSkegglevelpng2" data-toggle="list" href="#NMDSkegglevelpng2" role="tab" aria-controls="NMDSkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="NMDSkegglevelpng3" data-toggle="list" href="#NMDSkegglevelpng3" role="tab" aria-controls="NMDSkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="NMDSkeggkopng4" data-toggle="list" href="#NMDSkeggkopng4" role="tab" aria-controls="NMDSkeggkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="NMDSkeggecpng5" data-toggle="list" href="#NMDSkeggecpng5" role="tab" aria-controls="NMDSkeggecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="NMDSkeggmodulepng6" data-toggle="list" href="#NMDSkeggmodulepng6" role="tab" aria-controls="NMDSkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.41 NMDS analysis in KEGG</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/KEGG/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/KEGG/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/KEGG/NMDS_group1/*/NMDS_scores.xls</p>

    <a name="T5"></a>
    <h5>4.5.5.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisineggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisineggNOG_nav-tabContent">

<div class="tab-pane active show" id="NMDSeggnoglevelpng1" role="tabpanel" aria-labelledby="NMDSeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="NMDSeggnoglevelpng2" role="tabpanel" aria-labelledby="NMDSeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.eggnog.level2.png" /></div>

<div class="tab-pane fade" id="NMDSeggnogogpng3" role="tabpanel" aria-labelledby="NMDSeggnogogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.eggnog.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisineggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisineggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisineggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDSeggnoglevelpng1" data-toggle="list" href="#NMDSeggnoglevelpng1" role="tab" aria-controls="NMDSeggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="NMDSeggnoglevelpng2" data-toggle="list" href="#NMDSeggnoglevelpng2" role="tab" aria-controls="NMDSeggnoglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="NMDSeggnogogpng3" data-toggle="list" href="#NMDSeggnogogpng3" role="tab" aria-controls="NMDSeggnogogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.42 NMDS analysis in eggNOG</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/eggNOG/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/eggNOG/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/eggNOG/NMDS_group1/*/NMDS_scores.xls</p>

    <a name="T5"></a>
    <h5>4.5.5.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinCAZy_nav-tabContent">

<div class="tab-pane active show" id="NMDScazylevelpng1" role="tabpanel" aria-labelledby="NMDScazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.cazy.level1.png" /></div>

<div class="tab-pane fade" id="NMDScazylevelpng2" role="tabpanel" aria-labelledby="NMDScazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.cazy.level2.png" /></div>

<div class="tab-pane fade" id="NMDScazyecpng3" role="tabpanel" aria-labelledby="NMDScazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDScazylevelpng1" data-toggle="list" href="#NMDScazylevelpng1" role="tab" aria-controls="NMDScazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="NMDScazylevelpng2" data-toggle="list" href="#NMDScazylevelpng2" role="tab" aria-controls="NMDScazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="NMDScazyecpng3" data-toggle="list" href="#NMDScazyecpng3" role="tab" aria-controls="NMDScazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.43 NMDS analysis in CAZy</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/CAZy/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/CAZy/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/CAZy/NMDS_group1/*/NMDS_scores.xls</p>

    <a name="T5"></a>
    <h5>4.5.5.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinVFDB_nav-tabContent">

<div class="tab-pane active show" id="NMDSvfdblevelpng1" role="tabpanel" aria-labelledby="NMDSvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="NMDSvfdblevelpng2" role="tabpanel" aria-labelledby="NMDSvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDSvfdblevelpng1" data-toggle="list" href="#NMDSvfdblevelpng1" role="tab" aria-controls="NMDSvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="NMDSvfdblevelpng2" data-toggle="list" href="#NMDSvfdblevelpng2" role="tab" aria-controls="NMDSvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.44 NMDS analysis in VFDB</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/VFDB/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/VFDB/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/VFDB/NMDS_group1/*/NMDS_scores.xls</p>

    <a name="T5"></a>
    <h5>4.5.5.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinPHI_nav-tabContent">

<div class="tab-pane active show" id="NMDSphilevelpng1" role="tabpanel" aria-labelledby="NMDSphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.phi.level1.png" /></div>

<div class="tab-pane fade" id="NMDSphilevelpng2" role="tabpanel" aria-labelledby="NMDSphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.phi.level2.png" /></div>

<div class="tab-pane fade" id="NMDSphilevelpng3" role="tabpanel" aria-labelledby="NMDSphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/05.Diversity/NMDS.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="NMDSphilevelpng1" data-toggle="list" href="#NMDSphilevelpng1" role="tab" aria-controls="NMDSphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="NMDSphilevelpng2" data-toggle="list" href="#NMDSphilevelpng2" role="tab" aria-controls="NMDSphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="NMDSphilevelpng3" data-toggle="list" href="#NMDSphilevelpng3" role="tab" aria-controls="NMDSphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.45 NMDS analysis in PHI</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/05.Diversity/PHI/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/05.Diversity/PHI/NMDS_group1/*/NMDS.{png,pdf}</p>

    <p class="paragraph">Results of the Stress value analysis are shown in: result/05.Diversity/PHI/NMDS_group1/*/NMDS_scores.xls</p>

        <br />

        <br />

<h3>4.6 Analysis of statistical tests for Taxonomy and Function differences between groups</h3>
        
    <a name="T4"></a>
    <h4>4.6.1 Anosim</h4>    

    <p class="paragraph">Anosim is a nonparametric test helps evaluate the reasonability of the division of groups by evaluating whether variation among groups is significantly greater than variation within group. Click Anosim(https://www.rdocumentation.org/packages/vegan/versions/2.3-5/topics/anosim) for the detailed calculation steps.</p>

    <a name="T5"></a>
    <h5>4.6.1.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatMicroNR_nav-tabContent">

<div class="tab-pane active show" id="anosimtaxkingdompng1" role="tabpanel" aria-labelledby="anosimtaxkingdompng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.kingdom.png" /></div>

<div class="tab-pane fade" id="anosimtaxphylumpng2" role="tabpanel" aria-labelledby="anosimtaxphylumpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.phylum.png" /></div>

<div class="tab-pane fade" id="anosimtaxclasspng3" role="tabpanel" aria-labelledby="anosimtaxclasspng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.class.png" /></div>

<div class="tab-pane fade" id="anosimtaxorderpng4" role="tabpanel" aria-labelledby="anosimtaxorderpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.order.png" /></div>

<div class="tab-pane fade" id="anosimtaxfamilypng5" role="tabpanel" aria-labelledby="anosimtaxfamilypng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.family.png" /></div>

<div class="tab-pane fade" id="anosimtaxgenuspng6" role="tabpanel" aria-labelledby="anosimtaxgenuspng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.genus.png" /></div>

<div class="tab-pane fade" id="anosimtaxspeciespng7" role="tabpanel" aria-labelledby="anosimtaxspeciespng7" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimtaxkingdompng1" data-toggle="list" href="#anosimtaxkingdompng1" role="tab" aria-controls="anosimtaxkingdompng1">Kingdom</a>

<a class="list-group-item list-group-item-action" id="anosimtaxphylumpng2" data-toggle="list" href="#anosimtaxphylumpng2" role="tab" aria-controls="anosimtaxphylumpng2">Phylum</a>

<a class="list-group-item list-group-item-action" id="anosimtaxclasspng3" data-toggle="list" href="#anosimtaxclasspng3" role="tab" aria-controls="anosimtaxclasspng3">Class</a>

<a class="list-group-item list-group-item-action" id="anosimtaxorderpng4" data-toggle="list" href="#anosimtaxorderpng4" role="tab" aria-controls="anosimtaxorderpng4">Order</a>

<a class="list-group-item list-group-item-action" id="anosimtaxfamilypng5" data-toggle="list" href="#anosimtaxfamilypng5" role="tab" aria-controls="anosimtaxfamilypng5">Family</a>

<a class="list-group-item list-group-item-action" id="anosimtaxgenuspng6" data-toggle="list" href="#anosimtaxgenuspng6" role="tab" aria-controls="anosimtaxgenuspng6">Genus</a>

<a class="list-group-item list-group-item-action" id="anosimtaxspeciespng7" data-toggle="list" href="#anosimtaxspeciespng7" role="tab" aria-controls="anosimtaxspeciespng7">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.46 Anosim at Micro_NR</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/MicroNR/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.1.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatKEGG_nav-tabContent">

<div class="tab-pane active show" id="anosimkegglevelpng1" role="tabpanel" aria-labelledby="anosimkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.level1.png" /></div>

<div class="tab-pane fade" id="anosimkegglevelpng2" role="tabpanel" aria-labelledby="anosimkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.level2.png" /></div>

<div class="tab-pane fade" id="anosimkegglevelpng3" role="tabpanel" aria-labelledby="anosimkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.level3.png" /></div>

<div class="tab-pane fade" id="anosimkeggkopng4" role="tabpanel" aria-labelledby="anosimkeggkopng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.ko.png" /></div>

<div class="tab-pane fade" id="anosimkeggecpng5" role="tabpanel" aria-labelledby="anosimkeggecpng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.ec.png" /></div>

<div class="tab-pane fade" id="anosimkeggmodulepng6" role="tabpanel" aria-labelledby="anosimkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimkegglevelpng1" data-toggle="list" href="#anosimkegglevelpng1" role="tab" aria-controls="anosimkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="anosimkegglevelpng2" data-toggle="list" href="#anosimkegglevelpng2" role="tab" aria-controls="anosimkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="anosimkegglevelpng3" data-toggle="list" href="#anosimkegglevelpng3" role="tab" aria-controls="anosimkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="anosimkeggkopng4" data-toggle="list" href="#anosimkeggkopng4" role="tab" aria-controls="anosimkeggkopng4">KO</a>

<a class="list-group-item list-group-item-action" id="anosimkeggecpng5" data-toggle="list" href="#anosimkeggecpng5" role="tab" aria-controls="anosimkeggecpng5">ec</a>

<a class="list-group-item list-group-item-action" id="anosimkeggmodulepng6" data-toggle="list" href="#anosimkeggmodulepng6" role="tab" aria-controls="anosimkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.47 Anosim at KEGG</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/KEGG/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.1.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimateggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimateggNOG_nav-tabContent">

<div class="tab-pane active show" id="anosimeggnoglevelpng1" role="tabpanel" aria-labelledby="anosimeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="anosimeggnoglevelpng2" role="tabpanel" aria-labelledby="anosimeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.eggnog.level2.png" /></div>

<div class="tab-pane fade" id="anosimeggnogogpng3" role="tabpanel" aria-labelledby="anosimeggnogogpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.eggnog.og.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimateggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimateggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimateggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimeggnoglevelpng1" data-toggle="list" href="#anosimeggnoglevelpng1" role="tab" aria-controls="anosimeggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="anosimeggnoglevelpng2" data-toggle="list" href="#anosimeggnoglevelpng2" role="tab" aria-controls="anosimeggnoglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="anosimeggnogogpng3" data-toggle="list" href="#anosimeggnogogpng3" role="tab" aria-controls="anosimeggnogogpng3">og</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.48 Anosim at eggNOG</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/eggNOG/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.1.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatCAZy_nav-tabContent">

<div class="tab-pane active show" id="anosimcazylevelpng1" role="tabpanel" aria-labelledby="anosimcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.cazy.level1.png" /></div>

<div class="tab-pane fade" id="anosimcazylevelpng2" role="tabpanel" aria-labelledby="anosimcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.cazy.level2.png" /></div>

<div class="tab-pane fade" id="anosimcazyecpng3" role="tabpanel" aria-labelledby="anosimcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimcazylevelpng1" data-toggle="list" href="#anosimcazylevelpng1" role="tab" aria-controls="anosimcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="anosimcazylevelpng2" data-toggle="list" href="#anosimcazylevelpng2" role="tab" aria-controls="anosimcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="anosimcazyecpng3" data-toggle="list" href="#anosimcazyecpng3" role="tab" aria-controls="anosimcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.49 Anosim at CAZy</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/CAZy/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.1.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatVFDB_nav-tabContent">

<div class="tab-pane active show" id="anosimvfdblevelpng1" role="tabpanel" aria-labelledby="anosimvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="anosimvfdblevelpng2" role="tabpanel" aria-labelledby="anosimvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimvfdblevelpng1" data-toggle="list" href="#anosimvfdblevelpng1" role="tab" aria-controls="anosimvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="anosimvfdblevelpng2" data-toggle="list" href="#anosimvfdblevelpng2" role="tab" aria-controls="anosimvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.50 Anosim at VFDB</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/VFDB/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.1.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatPHI_nav-tabContent">

<div class="tab-pane active show" id="anosimphilevelpng1" role="tabpanel" aria-labelledby="anosimphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.phi.level1.png" /></div>

<div class="tab-pane fade" id="anosimphilevelpng2" role="tabpanel" aria-labelledby="anosimphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.phi.level2.png" /></div>

<div class="tab-pane fade" id="anosimphilevelpng3" role="tabpanel" aria-labelledby="anosimphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/anosim.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimphilevelpng1" data-toggle="list" href="#anosimphilevelpng1" role="tab" aria-controls="anosimphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="anosimphilevelpng2" data-toggle="list" href="#anosimphilevelpng2" role="tab" aria-controls="anosimphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="anosimphilevelpng3" data-toggle="list" href="#anosimphilevelpng3" role="tab" aria-controls="anosimphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.51 Anosim at PHI</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/PHI/Anosim_group1/*/*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.6.2 MetaGenomeSeq analysis</h4>    

    <p class="paragraph">MetaGenomeSeq is developed in the context of metagenomic data comprising multiple samples, and relies on a non-parametric t-test for detecting differentially abundant features in metagenomic studies. The p-value is further corrected as q-value to discover taxa with significant variation.</p>

    <a name="T5"></a>
    <h5>4.6.2.1 Micro_NR</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinMicroNR_nav-tabContent">

<div class="tab-pane active show" id="metastattaxphylumpng1" role="tabpanel" aria-labelledby="metastattaxphylumpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.phylum.png" /></div>

<div class="tab-pane fade" id="metastattaxclasspng2" role="tabpanel" aria-labelledby="metastattaxclasspng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.class.png" /></div>

<div class="tab-pane fade" id="metastattaxorderpng3" role="tabpanel" aria-labelledby="metastattaxorderpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.order.png" /></div>

<div class="tab-pane fade" id="metastattaxfamilypng4" role="tabpanel" aria-labelledby="metastattaxfamilypng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.family.png" /></div>

<div class="tab-pane fade" id="metastattaxgenuspng5" role="tabpanel" aria-labelledby="metastattaxgenuspng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.genus.png" /></div>

<div class="tab-pane fade" id="metastattaxspeciespng6" role="tabpanel" aria-labelledby="metastattaxspeciespng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.species.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastattaxphylumpng1" data-toggle="list" href="#metastattaxphylumpng1" role="tab" aria-controls="metastattaxphylumpng1">Phylum</a>

<a class="list-group-item list-group-item-action" id="metastattaxclasspng2" data-toggle="list" href="#metastattaxclasspng2" role="tab" aria-controls="metastattaxclasspng2">Class</a>

<a class="list-group-item list-group-item-action" id="metastattaxorderpng3" data-toggle="list" href="#metastattaxorderpng3" role="tab" aria-controls="metastattaxorderpng3">Order</a>

<a class="list-group-item list-group-item-action" id="metastattaxfamilypng4" data-toggle="list" href="#metastattaxfamilypng4" role="tab" aria-controls="metastattaxfamilypng4">Family</a>

<a class="list-group-item list-group-item-action" id="metastattaxgenuspng5" data-toggle="list" href="#metastattaxgenuspng5" role="tab" aria-controls="metastattaxgenuspng5">Genus</a>

<a class="list-group-item list-group-item-action" id="metastattaxspeciespng6" data-toggle="list" href="#metastattaxspeciespng6" role="tab" aria-controls="metastattaxspeciespng6">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.52 Box chart for species with significantly variant relative abundance between groups in Micro_NR</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonMicroNR_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonMicroNR_nav-tabContent">

<div class="tab-pane active show" id="metastattaxphylumclusterpcapng1" role="tabpanel" aria-labelledby="metastattaxphylumclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.phylum.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastattaxclassclusterpcapng2" role="tabpanel" aria-labelledby="metastattaxclassclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.class.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastattaxorderclusterpcapng3" role="tabpanel" aria-labelledby="metastattaxorderclusterpcapng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.order.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastattaxfamilyclusterpcapng4" role="tabpanel" aria-labelledby="metastattaxfamilyclusterpcapng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.family.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastattaxgenusclusterpcapng5" role="tabpanel" aria-labelledby="metastattaxgenusclusterpcapng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.genus.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastattaxspeciesclusterpcapng6" role="tabpanel" aria-labelledby="metastattaxspeciesclusterpcapng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.tax.species.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonMicroNR_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonMicroNR_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonMicroNR_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastattaxphylumclusterpcapng1" data-toggle="list" href="#metastattaxphylumclusterpcapng1" role="tab" aria-controls="metastattaxphylumclusterpcapng1">Phylum</a>

<a class="list-group-item list-group-item-action" id="metastattaxclassclusterpcapng2" data-toggle="list" href="#metastattaxclassclusterpcapng2" role="tab" aria-controls="metastattaxclassclusterpcapng2">Class</a>

<a class="list-group-item list-group-item-action" id="metastattaxorderclusterpcapng3" data-toggle="list" href="#metastattaxorderclusterpcapng3" role="tab" aria-controls="metastattaxorderclusterpcapng3">Order</a>

<a class="list-group-item list-group-item-action" id="metastattaxfamilyclusterpcapng4" data-toggle="list" href="#metastattaxfamilyclusterpcapng4" role="tab" aria-controls="metastattaxfamilyclusterpcapng4">Family</a>

<a class="list-group-item list-group-item-action" id="metastattaxgenusclusterpcapng5" data-toggle="list" href="#metastattaxgenusclusterpcapng5" role="tab" aria-controls="metastattaxgenusclusterpcapng5">Genus</a>

<a class="list-group-item list-group-item-action" id="metastattaxspeciesclusterpcapng6" data-toggle="list" href="#metastattaxspeciesclusterpcapng6" role="tab" aria-controls="metastattaxspeciesclusterpcapng6">Species</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.53 PCA and abundance heatmap based on Micro_NR</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/MicroNR/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.6.2.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinKEGG_nav-tabContent">

<div class="tab-pane active show" id="metastatkegglevelpng1" role="tabpanel" aria-labelledby="metastatkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level1.png" /></div>

<div class="tab-pane fade" id="metastatkegglevelpng2" role="tabpanel" aria-labelledby="metastatkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level2.png" /></div>

<div class="tab-pane fade" id="metastatkegglevelpng3" role="tabpanel" aria-labelledby="metastatkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level3.png" /></div>

<div class="tab-pane fade" id="metastatkeggecpng4" role="tabpanel" aria-labelledby="metastatkeggecpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.ec.png" /></div>

<div class="tab-pane fade" id="metastatkeggkopng5" role="tabpanel" aria-labelledby="metastatkeggkopng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.ko.png" /></div>

<div class="tab-pane fade" id="metastatkeggmodulepng6" role="tabpanel" aria-labelledby="metastatkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatkegglevelpng1" data-toggle="list" href="#metastatkegglevelpng1" role="tab" aria-controls="metastatkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatkegglevelpng2" data-toggle="list" href="#metastatkegglevelpng2" role="tab" aria-controls="metastatkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatkegglevelpng3" data-toggle="list" href="#metastatkegglevelpng3" role="tab" aria-controls="metastatkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="metastatkeggecpng4" data-toggle="list" href="#metastatkeggecpng4" role="tab" aria-controls="metastatkeggecpng4">ec</a>

<a class="list-group-item list-group-item-action" id="metastatkeggkopng5" data-toggle="list" href="#metastatkeggkopng5" role="tab" aria-controls="metastatkeggkopng5">KO</a>

<a class="list-group-item list-group-item-action" id="metastatkeggmodulepng6" data-toggle="list" href="#metastatkeggmodulepng6" role="tab" aria-controls="metastatkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.54 Box chart for species with significantly variant relative abundance between groups in KEGG</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonKEGG_nav-tabContent">

<div class="tab-pane active show" id="metastatkegglevelclusterpcapng1" role="tabpanel" aria-labelledby="metastatkegglevelclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level1.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatkegglevelclusterpcapng2" role="tabpanel" aria-labelledby="metastatkegglevelclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level2.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatkegglevelclusterpcapng3" role="tabpanel" aria-labelledby="metastatkegglevelclusterpcapng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.level3.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatkeggecclusterpcapng4" role="tabpanel" aria-labelledby="metastatkeggecclusterpcapng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.ec.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatkeggkoclusterpcapng5" role="tabpanel" aria-labelledby="metastatkeggkoclusterpcapng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.ko.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatkeggmoduleclusterpcapng6" role="tabpanel" aria-labelledby="metastatkeggmoduleclusterpcapng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.kegg.module.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatkegglevelclusterpcapng1" data-toggle="list" href="#metastatkegglevelclusterpcapng1" role="tab" aria-controls="metastatkegglevelclusterpcapng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatkegglevelclusterpcapng2" data-toggle="list" href="#metastatkegglevelclusterpcapng2" role="tab" aria-controls="metastatkegglevelclusterpcapng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatkegglevelclusterpcapng3" data-toggle="list" href="#metastatkegglevelclusterpcapng3" role="tab" aria-controls="metastatkegglevelclusterpcapng3">level3</a>

<a class="list-group-item list-group-item-action" id="metastatkeggecclusterpcapng4" data-toggle="list" href="#metastatkeggecclusterpcapng4" role="tab" aria-controls="metastatkeggecclusterpcapng4">ec</a>

<a class="list-group-item list-group-item-action" id="metastatkeggkoclusterpcapng5" data-toggle="list" href="#metastatkeggkoclusterpcapng5" role="tab" aria-controls="metastatkeggkoclusterpcapng5">KO</a>

<a class="list-group-item list-group-item-action" id="metastatkeggmoduleclusterpcapng6" data-toggle="list" href="#metastatkeggmoduleclusterpcapng6" role="tab" aria-controls="metastatkeggmoduleclusterpcapng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.55 PCA and abundance heatmap based on KEGG</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/KEGG/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.6.2.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsineggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsineggNOG_nav-tabContent">

<div class="tab-pane active show" id="metastateggnoglevelpng1" role="tabpanel" aria-labelledby="metastateggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.eggnog.level1.png" /></div>

<div class="tab-pane fade" id="metastateggnoglevelpng2" role="tabpanel" aria-labelledby="metastateggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.eggnog.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsineggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsineggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsineggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastateggnoglevelpng1" data-toggle="list" href="#metastateggnoglevelpng1" role="tab" aria-controls="metastateggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastateggnoglevelpng2" data-toggle="list" href="#metastateggnoglevelpng2" role="tab" aria-controls="metastateggnoglevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.56 Box chart for species with significantly variant relative abundance between groups in eggNOG</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedoneggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedoneggNOG_nav-tabContent">

<div class="tab-pane active show" id="metastateggnoglevelclusterpcapng1" role="tabpanel" aria-labelledby="metastateggnoglevelclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.eggnog.level1.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastateggnoglevelclusterpcapng2" role="tabpanel" aria-labelledby="metastateggnoglevelclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.eggnog.level2.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedoneggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedoneggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedoneggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastateggnoglevelclusterpcapng1" data-toggle="list" href="#metastateggnoglevelclusterpcapng1" role="tab" aria-controls="metastateggnoglevelclusterpcapng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastateggnoglevelclusterpcapng2" data-toggle="list" href="#metastateggnoglevelclusterpcapng2" role="tab" aria-controls="metastateggnoglevelclusterpcapng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.57 PCA and abundance heatmap based on eggNOG</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/eggNOG/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.6.2.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinCAZy_nav-tabContent">

<div class="tab-pane active show" id="metastatcazylevelpng1" role="tabpanel" aria-labelledby="metastatcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.level1.png" /></div>

<div class="tab-pane fade" id="metastatcazylevelpng2" role="tabpanel" aria-labelledby="metastatcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.level2.png" /></div>

<div class="tab-pane fade" id="metastatcazyecpng3" role="tabpanel" aria-labelledby="metastatcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatcazylevelpng1" data-toggle="list" href="#metastatcazylevelpng1" role="tab" aria-controls="metastatcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatcazylevelpng2" data-toggle="list" href="#metastatcazylevelpng2" role="tab" aria-controls="metastatcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatcazyecpng3" data-toggle="list" href="#metastatcazyecpng3" role="tab" aria-controls="metastatcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.58 Box chart for species with significantly variant relative abundance between groups in CAZy</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonCAZy_nav-tabContent">

<div class="tab-pane active show" id="metastatcazylevelclusterpcapng1" role="tabpanel" aria-labelledby="metastatcazylevelclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.level1.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatcazylevelclusterpcapng2" role="tabpanel" aria-labelledby="metastatcazylevelclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.level2.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatcazyecclusterpcapng3" role="tabpanel" aria-labelledby="metastatcazyecclusterpcapng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.cazy.ec.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatcazylevelclusterpcapng1" data-toggle="list" href="#metastatcazylevelclusterpcapng1" role="tab" aria-controls="metastatcazylevelclusterpcapng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatcazylevelclusterpcapng2" data-toggle="list" href="#metastatcazylevelclusterpcapng2" role="tab" aria-controls="metastatcazylevelclusterpcapng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatcazyecclusterpcapng3" data-toggle="list" href="#metastatcazyecclusterpcapng3" role="tab" aria-controls="metastatcazyecclusterpcapng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.59 PCA and abundance heatmap based on CAZy</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/CAZy/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.6.2.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinVFDB_nav-tabContent">

<div class="tab-pane active show" id="metastatvfdblevelpng1" role="tabpanel" aria-labelledby="metastatvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.vfdb.level1.png" /></div>

<div class="tab-pane fade" id="metastatvfdblevelpng2" role="tabpanel" aria-labelledby="metastatvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.vfdb.level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatvfdblevelpng1" data-toggle="list" href="#metastatvfdblevelpng1" role="tab" aria-controls="metastatvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatvfdblevelpng2" data-toggle="list" href="#metastatvfdblevelpng2" role="tab" aria-controls="metastatvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.60 Box chart for species with significantly variant relative abundance between groups in VFDB</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonVFDB_nav-tabContent">

<div class="tab-pane active show" id="metastatvfdblevelclusterpcapng1" role="tabpanel" aria-labelledby="metastatvfdblevelclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.vfdb.level1.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatvfdblevelclusterpcapng2" role="tabpanel" aria-labelledby="metastatvfdblevelclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.vfdb.level2.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatvfdblevelclusterpcapng1" data-toggle="list" href="#metastatvfdblevelclusterpcapng1" role="tab" aria-controls="metastatvfdblevelclusterpcapng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatvfdblevelclusterpcapng2" data-toggle="list" href="#metastatvfdblevelclusterpcapng2" role="tab" aria-controls="metastatvfdblevelclusterpcapng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.61 PCA and abundance heatmap based on VFDB</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/VFDB/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.6.2.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinPHI_nav-tabContent">

<div class="tab-pane active show" id="metastatphilevelpng1" role="tabpanel" aria-labelledby="metastatphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level1.png" /></div>

<div class="tab-pane fade" id="metastatphilevelpng2" role="tabpanel" aria-labelledby="metastatphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level2.png" /></div>

<div class="tab-pane fade" id="metastatphilevelpng3" role="tabpanel" aria-labelledby="metastatphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforspecieswithsignificantlyvariantrelativeabundancebetweengroupsinPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatphilevelpng1" data-toggle="list" href="#metastatphilevelpng1" role="tab" aria-controls="metastatphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatphilevelpng2" data-toggle="list" href="#metastatphilevelpng2" role="tab" aria-controls="metastatphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatphilevelpng3" data-toggle="list" href="#metastatphilevelpng3" role="tab" aria-controls="metastatphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.62 Box chart for species with significantly variant relative abundance between groups in PHI</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding species. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for species whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonPHI_nav-tabContent">

<div class="tab-pane active show" id="metastatphilevelclusterpcapng1" role="tabpanel" aria-labelledby="metastatphilevelclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level1.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatphilevelclusterpcapng2" role="tabpanel" aria-labelledby="metastatphilevelclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level2.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatphilevelclusterpcapng3" role="tabpanel" aria-labelledby="metastatphilevelclusterpcapng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/metastat.phi.level3.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatphilevelclusterpcapng1" data-toggle="list" href="#metastatphilevelclusterpcapng1" role="tab" aria-controls="metastatphilevelclusterpcapng1">level1</a>

<a class="list-group-item list-group-item-action" id="metastatphilevelclusterpcapng2" data-toggle="list" href="#metastatphilevelclusterpcapng2" role="tab" aria-controls="metastatphilevelclusterpcapng2">level2</a>

<a class="list-group-item list-group-item-action" id="metastatphilevelclusterpcapng3" data-toggle="list" href="#metastatphilevelclusterpcapng3" role="tab" aria-controls="metastatphilevelclusterpcapng3">levle3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.63 PCA and abundance heatmap based on PHI</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents species annotation; the left side of the heatmap is a clustering tree of species. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/06.StatisticalTest/PHI/MetaGenomeSeq_group1/*</p>

    <a name="T4"></a>
    <h4>4.6.3 LEfSe Analysis</h4>    

    <p class="paragraph">LEfSe (linear discriminant analysis (LDA) Effect Size) analysis allows high-dimensional biomarker discovery and explanation. LEfSe identifies genomic features (genes, pathways or taxa) characterizing the differences between two or more biological conditions (or classes). It enables the characterization of microbial taxa specific to an experimental or environmental condition, and the identification of metagenomic biomarkers in different communities.</p>

    <a name="T5"></a>
    <h5>4.6.3.1 Micro_NR</h5>    

<p class="center">
<img src="files/pictures/06.StatisticalTest/LDA.tax_combine.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.64 Histogram of LDA scores on Micro_NR</strong></p>  
</p>
<p class="premark" >Notes: a) The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (4 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). b) In the Cladogram, circles radiating from the inner side to the outer side represent taxonomic rank from phylum to genus (species). Each circle stands for a distinct taxon at a corresponding taxonomic rank. The diameter of each circle represents proportionally the relative abundance of each taxon. Yellow stands for taxons with non-significant differences. Taxon (biomarkers) with significant differences are colored according to the corresponding group's color. Red nodes mean this microbiota contributes a lot in the group covered by red color, and so do the green nodes. Letters above the circles and corresponding species are annotated on the right side.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/MicroNR/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.3.2 KEGG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresonKEGG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresonKEGG_nav-tabContent">

<div class="tab-pane active show" id="LDAkegglevelpng1" role="tabpanel" aria-labelledby="LDAkegglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_level1.png" /></div>

<div class="tab-pane fade" id="LDAkegglevelpng2" role="tabpanel" aria-labelledby="LDAkegglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_level2.png" /></div>

<div class="tab-pane fade" id="LDAkegglevelpng3" role="tabpanel" aria-labelledby="LDAkegglevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_level3.png" /></div>

<div class="tab-pane fade" id="LDAkeggecpng4" role="tabpanel" aria-labelledby="LDAkeggecpng4" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_ec.png" /></div>

<div class="tab-pane fade" id="LDAkeggkopng5" role="tabpanel" aria-labelledby="LDAkeggkopng5" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_ko.png" /></div>

<div class="tab-pane fade" id="LDAkeggmodulepng6" role="tabpanel" aria-labelledby="LDAkeggmodulepng6" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.kegg_module.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresonKEGG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresonKEGG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresonKEGG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAkegglevelpng1" data-toggle="list" href="#LDAkegglevelpng1" role="tab" aria-controls="LDAkegglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="LDAkegglevelpng2" data-toggle="list" href="#LDAkegglevelpng2" role="tab" aria-controls="LDAkegglevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="LDAkegglevelpng3" data-toggle="list" href="#LDAkegglevelpng3" role="tab" aria-controls="LDAkegglevelpng3">level3</a>

<a class="list-group-item list-group-item-action" id="LDAkeggecpng4" data-toggle="list" href="#LDAkeggecpng4" role="tab" aria-controls="LDAkeggecpng4">ec</a>

<a class="list-group-item list-group-item-action" id="LDAkeggkopng5" data-toggle="list" href="#LDAkeggkopng5" role="tab" aria-controls="LDAkeggkopng5">KO</a>

<a class="list-group-item list-group-item-action" id="LDAkeggmodulepng6" data-toggle="list" href="#LDAkeggmodulepng6" role="tab" aria-controls="LDAkeggmodulepng6">module</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.65 Histogram of LDA scores on KEGG</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/KEGG/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.3.3 eggNOG</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresoneggNOG_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresoneggNOG_nav-tabContent">

<div class="tab-pane active show" id="LDAeggnoglevelpng1" role="tabpanel" aria-labelledby="LDAeggnoglevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.eggnog_level1.png" /></div>

<div class="tab-pane fade" id="LDAeggnoglevelpng2" role="tabpanel" aria-labelledby="LDAeggnoglevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.eggnog_level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresoneggNOG_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresoneggNOG_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresoneggNOG_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAeggnoglevelpng1" data-toggle="list" href="#LDAeggnoglevelpng1" role="tab" aria-controls="LDAeggnoglevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="LDAeggnoglevelpng2" data-toggle="list" href="#LDAeggnoglevelpng2" role="tab" aria-controls="LDAeggnoglevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.66 Histogram of LDA scores on eggNOG</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/eggNOG/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.3.4 CAZy</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresonCAZy_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresonCAZy_nav-tabContent">

<div class="tab-pane active show" id="LDAcazylevelpng1" role="tabpanel" aria-labelledby="LDAcazylevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.cazy_level1.png" /></div>

<div class="tab-pane fade" id="LDAcazylevelpng2" role="tabpanel" aria-labelledby="LDAcazylevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.cazy_level2.png" /></div>

<div class="tab-pane fade" id="LDAcazyecpng3" role="tabpanel" aria-labelledby="LDAcazyecpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.cazy_ec.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresonCAZy_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresonCAZy_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresonCAZy_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAcazylevelpng1" data-toggle="list" href="#LDAcazylevelpng1" role="tab" aria-controls="LDAcazylevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="LDAcazylevelpng2" data-toggle="list" href="#LDAcazylevelpng2" role="tab" aria-controls="LDAcazylevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="LDAcazyecpng3" data-toggle="list" href="#LDAcazyecpng3" role="tab" aria-controls="LDAcazyecpng3">ec</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.67 Histogram of LDA scores on CAZy</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/CAZy/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.3.5 VFDB</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresonVFDB_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresonVFDB_nav-tabContent">

<div class="tab-pane active show" id="LDAvfdblevelpng1" role="tabpanel" aria-labelledby="LDAvfdblevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.vfdb_level1.png" /></div>

<div class="tab-pane fade" id="LDAvfdblevelpng2" role="tabpanel" aria-labelledby="LDAvfdblevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.vfdb_level2.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresonVFDB_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresonVFDB_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresonVFDB_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAvfdblevelpng1" data-toggle="list" href="#LDAvfdblevelpng1" role="tab" aria-controls="LDAvfdblevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="LDAvfdblevelpng2" data-toggle="list" href="#LDAvfdblevelpng2" role="tab" aria-controls="LDAvfdblevelpng2">level2</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.68 Histogram of LDA scores on VFDB</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/VFDB/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.6.3.6 PHI</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresonPHI_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresonPHI_nav-tabContent">

<div class="tab-pane active show" id="LDAphilevelpng1" role="tabpanel" aria-labelledby="LDAphilevelpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.phi_level1.png" /></div>

<div class="tab-pane fade" id="LDAphilevelpng2" role="tabpanel" aria-labelledby="LDAphilevelpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.phi_level2.png" /></div>

<div class="tab-pane fade" id="LDAphilevelpng3" role="tabpanel" aria-labelledby="LDAphilevelpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/06.StatisticalTest/LDA.phi_level3.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresonPHI_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresonPHI_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresonPHI_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAphilevelpng1" data-toggle="list" href="#LDAphilevelpng1" role="tab" aria-controls="LDAphilevelpng1">level1</a>

<a class="list-group-item list-group-item-action" id="LDAphilevelpng2" data-toggle="list" href="#LDAphilevelpng2" role="tab" aria-controls="LDAphilevelpng2">level2</a>

<a class="list-group-item list-group-item-action" id="LDAphilevelpng3" data-toggle="list" href="#LDAphilevelpng3" role="tab" aria-controls="LDAphilevelpng3">level3</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.69 Histogram of LDA scores on PHI</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents species (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/06.StatisticalTest/PHI/LEfSe_group1/*/*.{pdf,png}</p>

    <a name="T4"></a>
    <h4>4.6.4 Metabolic pathway</h4>    

    <p class="paragraph">To study the variances of pathway patterns in different groups (samples), the web-version pathway report was created. The whole pathway report has two parts:</p>

    <p class="paragraph">Part 1 (mPath): mPath is a statistical package for comparing metagenomic data-sets in the pathway level (using KEGG pathway information). Metapath relies on a graph-theoretic definition of statistical significance in order to identify pathway motifs that differ between groups or samples from two treatment populations. In this part, the compared pathway overview shows shared and unique pathway information between groups or samples. Nodes represent chemicals, and lines represent enzymatic reactions, of which the shared ones are marked in red, while the unique ones are marked in blue (group A/sample A) or green (group B/sample B);</p>

    <p class="paragraph">Part 2: The annotated metabolic pathway. In this part, nodes represent chemicals. Rectangles represent enzymes and different colors represent different Unigenes numbers (the numbers increase from blue to red); Touching your mouse on the colored rectangles, the Genes ID annotated will be displayed. Rectangles filled in yellow are those significantly variant among groups (if there is no such analysis, the information will not display); Touching your mouse on the enzymes, the box chart for enzymes with significantly variant abundance will be displayed.</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="files/pictures/06.StatisticalTest/pathwaymaps/pathway.html" target="_blank"><b>Click for detail.</b></a></p>
        

<p class="center">
<img src="src/images/mpath.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.70 The compare analysis plot for Metabolic Pathway of multi-samples</strong></p>  
</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">Metabolic Pathway analysis results : result/06.StatisticalTest/KEGG/iPATH/Pathway_group1</p>

        <br />

        <br />

<h3>4.7 Antibiotic Resistance Gene Annotation</h3>
        
    <p class="paragraph">Efforts on identifying new antibiotics were once a top research and development priority among pharmaceutical companies. However, the treatment of infections is increasingly compromised by the ability of bacteria to develop resistance to antibiotics through mutation or acquisition of resistance genes. Antibiotic resistance genes are also potentially used by bio-terrorists in genetically modified organisms. CARD (the Comprehensive Antibiotic Research Database) database is used to facilitate identification and characterization of antibiotic resistance genes. Resistance genes are further categorized to resistance types by their resistance profiles and sequence similarity. Each gene or type is annotated with rich information, including resistance profile and resistance mechanism.</p>

    <a name="T4"></a>
    <h4>4.7.1 Function Annotation</h4>    

    <p class="paragraph">CARD Analysis: Antibiotic resistant genes were annotated using the CARD database, revealing their abundance and function distribution.</p>

    <p class="paragraph">MGEs Analysis: Unigenes were compared against databases for insertion sequences, integrons, and plasmids to determine their abundance.</p>

    <a name="T5"></a>
    <h5>4.7.1.1 CARD</h5>    

    <a name="T5"></a>
    <h5>4.7.1.1.1 Introduction of antibiotic resistance gene annotation</h5>    

    <p class="paragraph">All unique genes were blastp against CARD database (blastp, evalue ≤ 1e-5). Relative abundance of different resistance genes were calculated based on above results.According to the abundance table of resistance genes, various analyses were performed.</p>

    <a name="T5"></a>
    <h5>4.7.1.1.2 Relative abundance of resistance genes</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofantibioticresistancegene_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofantibioticresistancegene_nav-tabContent">

<div class="tab-pane active show" id="cardppmtoppng1" role="tabpanel" aria-labelledby="cardppmtoppng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/card.ppm.top.png" /></div>

<div class="tab-pane fade" id="cardRelativePercenttoppng2" role="tabpanel" aria-labelledby="cardRelativePercenttoppng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/card.RelativePercent.top.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofantibioticresistancegene_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofantibioticresistancegene_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofantibioticresistancegene_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="cardppmtoppng1" data-toggle="list" href="#cardppmtoppng1" role="tab" aria-controls="cardppmtoppng1">ppm</a>

<a class="list-group-item list-group-item-action" id="cardRelativePercenttoppng2" data-toggle="list" href="#cardRelativePercenttoppng2" role="tab" aria-controls="cardRelativePercenttoppng2">relative</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.71 Relative abundance of antibiotic resistance gene</strong></p>  
<p class="premark" > Notes: For the legend, the resistance gene with two or more resistance types are shown using abbreviations. a) The relative abundance in Unit ppm is the result of magnifying 10e6 times of the original abundance data. b) Percentage of resistance genes in each sample.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Bar plots at ARO: result/07.ResistantGene/CARD/Anno/Bar/ARO/Unigenes.*.{png,svg}</p>

    <a name="T5"></a>
    <h5>4.7.1.2 MGEs</h5>    

    <p class="paragraph">Mobile genetic elements (MGEs) are a class of DNA that can transfer within or across different genomes and include integrons, Insertion Sequence (IS), and plasmids. Among them, integrons enhances the adaptability of bacterial survival and leads to the horizontal spread of multi-drug resistance genes in bacteria by capturing and integrating exogenous genes and transforming them into functional gene expression units. The database Integrall, is used for integron annotation. While, Insertion sequences (IS) are the simplest transposable elements, capable of inserting into different genomic locations, potentially causing mutations or altering gene expression. The database isfinder is used for the annotation of insertion sequences. Whereas, plasmids are in a cyclic configuration and can replicate independently. Most plasmids are in bacteria and other microogranisms such as yeast and even in the mitochondria of plants. The database plasmid is used for plasmid abundance annotation.</p>

    <a name="T5"></a>
    <h5>4.7.1.2.1 Introduction to mobile genetic elements annotation</h5>    

    <p class="paragraph">Mobile genetic elements (MGEs), unigenes were compared with isfinder, integrall, and plasmid databases, respectively, to obtain abundance information. The annotated abundance information was further visualized, and the results of abundance histogram and relative abundance heatmap were displayed.</p>

    <p class="paragraph">All unique genes were blastp against MGEs database (blastx, evalue ≤ 1e-5).Relative abundance of different resistance genes were calculated based on above results.According to the abundance table of resistance genes, various analyses were performed.</p>

    <a name="T5"></a>
    <h5>4.7.1.2.2 Relative abundance of mobile genetic elements</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRelativeabundanceofMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRelativeabundanceofMGEs_nav-tabContent">

<div class="tab-pane active show" id="mgesintegralltoppng1" role="tabpanel" aria-labelledby="mgesintegralltoppng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/mges.integrall.top.png" /></div>

<div class="tab-pane fade" id="mgesisfindertoppng2" role="tabpanel" aria-labelledby="mgesisfindertoppng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/mges.isfinder.top.png" /></div>

<div class="tab-pane fade" id="mgesplasmidtoppng3" role="tabpanel" aria-labelledby="mgesplasmidtoppng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/mges.plasmid.top.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRelativeabundanceofMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRelativeabundanceofMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRelativeabundanceofMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="mgesintegralltoppng1" data-toggle="list" href="#mgesintegralltoppng1" role="tab" aria-controls="mgesintegralltoppng1">integrall</a>

<a class="list-group-item list-group-item-action" id="mgesisfindertoppng2" data-toggle="list" href="#mgesisfindertoppng2" role="tab" aria-controls="mgesisfindertoppng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="mgesplasmidtoppng3" data-toggle="list" href="#mgesplasmidtoppng3" role="tab" aria-controls="mgesplasmidtoppng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.72 Relative abundance of MGEs</strong></p>  
<p class="premark" > Notes: For the legend, the resistance gene with two or more MGE types are shown using abbreviations. a) The relative abundance in Unit ppm is the result of magnifying 10e6 times of the original abundance data. b) Percentage of resistance genes in each sample.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Bar plots at MGEs: result/07.ResistantGene/MGES/*/Anno/Bar/*/Unigenes.*.{png,svg}</p>

    <a name="T4"></a>
    <h4>4.7.2 Function Diversity Analysis</h4>    

    <a name="T5"></a>
    <h5>4.7.2.1 Alpha Diversity Analysis</h5>    

    <a name="T5"></a>
    <h5>4.7.2.1.1 CARD</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of CARD, calculate the richness and diversity of each sample (or group).</p>

<p class="name_table"><p class="center"><strong>Table 4.10 Statistical Results of Alpha Diversity Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable9" >
        <thead><tr>
<a id="btn09"></a>
<a id="sjzl9"></a>
<a  href="javascript:void(0)" id="btn19"> | First | </a>
<a  href="javascript:void(0)" id="btn29"> Previous | </a>
<a  href="javascript:void(0)" id="btn39"> Next | </a>
<a  href="javascript:void(0)" id="btn49"> Last | </a>
<a>Go to </a>
<input id="changePage9" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn59"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">43.4293228139382</td><td style="text-align :center">42.25</td><td style="text-align :center">0.0108799927823185</td><td style="text-align :center">0.00213200249121237</td><td style="text-align :center">42</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">42</td><td style="text-align :center">42</td><td style="text-align :center">0.0108125588490909</td><td style="text-align :center">0.00205967057114076</td><td style="text-align :center">42</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">47.6708382526564</td><td style="text-align :center">49</td><td style="text-align :center">0.0102213839331502</td><td style="text-align :center">0.00194342009360282</td><td style="text-align :center">46</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">36.3895184341492</td><td style="text-align :center">36</td><td style="text-align :center">0.0102539067824851</td><td style="text-align :center">0.00201037859773734</td><td style="text-align :center">36</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">48.2867741935484</td><td style="text-align :center">50</td><td style="text-align :center">0.0114341090936032</td><td style="text-align :center">0.0022065765958339</td><td style="text-align :center">47</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">48.5463059313215</td><td style="text-align :center">47.3333333333333</td><td style="text-align :center">0.010932664138539</td><td style="text-align :center">0.00205451641063437</td><td style="text-align :center">47</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">63.3033204569738</td><td style="text-align :center">63</td><td style="text-align :center">0.0150913616056208</td><td style="text-align :center">0.00284167796574919</td><td style="text-align :center">63</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">59</td><td style="text-align :center">59</td><td style="text-align :center">0.0152080666931286</td><td style="text-align :center">0.00301811527226215</td><td style="text-align :center">59</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">59.3798638730575</td><td style="text-align :center">59</td><td style="text-align :center">0.0143670337710835</td><td style="text-align :center">0.00281104427513068</td><td style="text-align :center">59</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">62.471360039362</td><td style="text-align :center">62</td><td style="text-align :center">0.0148030807153351</td><td style="text-align :center">0.00279805836584013</td><td style="text-align :center">62</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">57.1100242199074</td><td style="text-align :center">56</td><td style="text-align :center">0.0160782550731268</td><td style="text-align :center">0.0032536383384687</td><td style="text-align :center">54</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">63.256372325899</td><td style="text-align :center">63</td><td style="text-align :center">0.0159735301416334</td><td style="text-align :center">0.00307908127553369</td><td style="text-align :center">63</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">51.037037037037</td><td style="text-align :center">50</td><td style="text-align :center">0.0123106958597068</td><td style="text-align :center">0.00234614892723195</td><td style="text-align :center">49</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">52</td><td style="text-align :center">52</td><td style="text-align :center">0.0131942103245942</td><td style="text-align :center">0.0025078619319544</td><td style="text-align :center">52</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">52</td><td style="text-align :center">52</td><td style="text-align :center">0.0143076301804716</td><td style="text-align :center">0.00273236245309016</td><td style="text-align :center">52</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">49.4855586046236</td><td style="text-align :center">49</td><td style="text-align :center">0.0111012746113426</td><td style="text-align :center">0.00212270389053248</td><td style="text-align :center">49</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">54.3632</td><td style="text-align :center">54</td><td style="text-align :center">0.0141608344719529</td><td style="text-align :center">0.00270314480666356</td><td style="text-align :center">54</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">53.0677551020408</td><td style="text-align :center">53</td><td style="text-align :center">0.014489028117648</td><td style="text-align :center">0.00278805913838731</td><td style="text-align :center">52</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">CARD Alpha Diversity indices statistics: result/07.ResistantGene/CARD/Diversity/AlphaDiversity/Alpha_index_table/*.xls</p>

    <a name="T5"></a>
    <h5>4.7.2.1.2 MGEs</h5>    

    <p class="paragraph">Starting from the annotated Absolute table of integrall, calculate the richness and diversity of each sample (or group).</p>

<p class="name_table"><p class="center"><strong>Table 4.11 Statistical Results of integrall</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable10" >
        <thead><tr>
<a id="btn010"></a>
<a id="sjzl10"></a>
<a  href="javascript:void(0)" id="btn110"> | First | </a>
<a  href="javascript:void(0)" id="btn210"> Previous | </a>
<a  href="javascript:void(0)" id="btn310"> Next | </a>
<a  href="javascript:void(0)" id="btn410"> Last | </a>
<a>Go to </a>
<input id="changePage10" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn510"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">253.709411148522</td><td style="text-align :center">254.5</td><td style="text-align :center">3.94261174483602</td><td style="text-align :center">0.961561709975321</td><td style="text-align :center">251</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">258.944668303247</td><td style="text-align :center">258.5</td><td style="text-align :center">3.81769277128211</td><td style="text-align :center">0.95262997851552</td><td style="text-align :center">258</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">273.667203072505</td><td style="text-align :center">277</td><td style="text-align :center">3.92077733899562</td><td style="text-align :center">0.958484400852559</td><td style="text-align :center">266</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">240.606307234303</td><td style="text-align :center">241.2</td><td style="text-align :center">3.67639178067444</td><td style="text-align :center">0.944648198754464</td><td style="text-align :center">237</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">260.284893617021</td><td style="text-align :center">261.142857142857</td><td style="text-align :center">4.07272978948873</td><td style="text-align :center">0.967730262426443</td><td style="text-align :center">256</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">266.599948373989</td><td style="text-align :center">270</td><td style="text-align :center">3.70138572558111</td><td style="text-align :center">0.941327752171184</td><td style="text-align :center">263</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">312.140416396996</td><td style="text-align :center">313</td><td style="text-align :center">4.04029826613588</td><td style="text-align :center">0.965865120097148</td><td style="text-align :center">310</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">282.923252978464</td><td style="text-align :center">283.5</td><td style="text-align :center">3.64546810254977</td><td style="text-align :center">0.949093532827869</td><td style="text-align :center">279</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">290.614629644684</td><td style="text-align :center">290.076923076923</td><td style="text-align :center">3.70616253015464</td><td style="text-align :center">0.952434746680321</td><td style="text-align :center">285</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">304.838701702928</td><td style="text-align :center">304.428571428571</td><td style="text-align :center">4.05330763499738</td><td style="text-align :center">0.967275816998184</td><td style="text-align :center">304</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">273.155567986051</td><td style="text-align :center">277.272727272727</td><td style="text-align :center">3.47350642962142</td><td style="text-align :center">0.93978629078754</td><td style="text-align :center">260</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">295.200402138209</td><td style="text-align :center">294.75</td><td style="text-align :center">3.85156085530933</td><td style="text-align :center">0.957985152596157</td><td style="text-align :center">294</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">273.172961838221</td><td style="text-align :center">274.142857142857</td><td style="text-align :center">3.95012014275983</td><td style="text-align :center">0.964234266177722</td><td style="text-align :center">269</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">275.245611907886</td><td style="text-align :center">275</td><td style="text-align :center">3.9431808248913</td><td style="text-align :center">0.963283717044341</td><td style="text-align :center">272</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">285.001328107919</td><td style="text-align :center">283.625</td><td style="text-align :center">4.08839536849014</td><td style="text-align :center">0.969327640017278</td><td style="text-align :center">281</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">271.535993981054</td><td style="text-align :center">269.111111111111</td><td style="text-align :center">3.83644337281334</td><td style="text-align :center">0.960256883389439</td><td style="text-align :center">263</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">273.129500818331</td><td style="text-align :center">274.2</td><td style="text-align :center">4.02848344354065</td><td style="text-align :center">0.967376132376368</td><td style="text-align :center">270</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">279.926779836336</td><td style="text-align :center">289</td><td style="text-align :center">4.09889366718822</td><td style="text-align :center">0.969974783074737</td><td style="text-align :center">274</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">MGEs-integrall Alpha Diversity indices statistics: result/07.ResistantGene/MGES/integrall/Diversity/AlphaDiversity/Alpha_index_table/*.xls</p>

    <p class="paragraph">Starting from the annotated Absolute table of isfinder, calculate the richness and diversity of each sample (or group).</p>

<p class="name_table"><p class="center"><strong>Table 4.12 Statistical Results of isfinder</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable11" >
        <thead><tr>
<a id="btn011"></a>
<a id="sjzl11"></a>
<a  href="javascript:void(0)" id="btn111"> | First | </a>
<a  href="javascript:void(0)" id="btn211"> Previous | </a>
<a  href="javascript:void(0)" id="btn311"> Next | </a>
<a  href="javascript:void(0)" id="btn411"> Last | </a>
<a>Go to </a>
<input id="changePage11" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn511"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">41.1948951325683</td><td style="text-align :center">46</td><td style="text-align :center">2.21809121135371</td><td style="text-align :center">0.847318863773997</td><td style="text-align :center">36</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">45.2024979303891</td><td style="text-align :center">45</td><td style="text-align :center">2.27945145940268</td><td style="text-align :center">0.843845173973907</td><td style="text-align :center">45</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">46.2274436090226</td><td style="text-align :center">45.25</td><td style="text-align :center">2.59160492321184</td><td style="text-align :center">0.890220685818359</td><td style="text-align :center">45</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">42.4237034663248</td><td style="text-align :center">42.5</td><td style="text-align :center">1.96332334762071</td><td style="text-align :center">0.793209662826052</td><td style="text-align :center">41</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">49.3565750882759</td><td style="text-align :center">49</td><td style="text-align :center">2.46233490700998</td><td style="text-align :center">0.857841391591495</td><td style="text-align :center">49</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">44.5208124065953</td><td style="text-align :center">44</td><td style="text-align :center">2.32037044329062</td><td style="text-align :center">0.831581542475161</td><td style="text-align :center">43</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">89.2641101809344</td><td style="text-align :center">92</td><td style="text-align :center">3.32244133235588</td><td style="text-align :center">0.944073134737333</td><td style="text-align :center">85</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">80.1805989728704</td><td style="text-align :center">79.5</td><td style="text-align :center">3.13138754341404</td><td style="text-align :center">0.917693731796936</td><td style="text-align :center">79</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">74.2762390670554</td><td style="text-align :center">74</td><td style="text-align :center">3.02243566918527</td><td style="text-align :center">0.919285662842829</td><td style="text-align :center">74</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">85.252254798325</td><td style="text-align :center">85</td><td style="text-align :center">3.45114302584521</td><td style="text-align :center">0.953661552781257</td><td style="text-align :center">84</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">70.3033307058211</td><td style="text-align :center">69.4285714285714</td><td style="text-align :center">2.88728534841578</td><td style="text-align :center">0.901008106318343</td><td style="text-align :center">69</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">80.4743500958875</td><td style="text-align :center">79.75</td><td style="text-align :center">3.32171669965773</td><td style="text-align :center">0.940187766581334</td><td style="text-align :center">79</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">65.7386377808263</td><td style="text-align :center">63.5</td><td style="text-align :center">2.72534161535579</td><td style="text-align :center">0.883339848244223</td><td style="text-align :center">61</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">71.3790555555556</td><td style="text-align :center">71</td><td style="text-align :center">2.8276204616678</td><td style="text-align :center">0.893843497317979</td><td style="text-align :center">71</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">64.3314675351109</td><td style="text-align :center">64</td><td style="text-align :center">2.86092409916657</td><td style="text-align :center">0.89072669149166</td><td style="text-align :center">64</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">61.6484375</td><td style="text-align :center">61</td><td style="text-align :center">2.79289058563099</td><td style="text-align :center">0.906213802803415</td><td style="text-align :center">61</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">64</td><td style="text-align :center">64</td><td style="text-align :center">2.79532166041778</td><td style="text-align :center">0.888639077149691</td><td style="text-align :center">64</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">64.354071963272</td><td style="text-align :center">64</td><td style="text-align :center">2.74192039440485</td><td style="text-align :center">0.878250820258648</td><td style="text-align :center">64</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">MGEs-isfinder Alpha Diversity indices statistics: result/07.ResistantGene/MGES/isfinder/Diversity/AlphaDiversity/Alpha_index_table/*.xls</p>

    <p class="paragraph">Starting from the annotated Absolute table of plasmid, calculate the richness and diversity of each sample (or group).</p>

<p class="name_table"><p class="center"><strong>Table 4.13 Statistical Results of plasmid</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable12" >
        <thead><tr>
<a id="btn012"></a>
<a id="sjzl12"></a>
<a  href="javascript:void(0)" id="btn112"> | First | </a>
<a  href="javascript:void(0)" id="btn212"> Previous | </a>
<a  href="javascript:void(0)" id="btn312"> Next | </a>
<a  href="javascript:void(0)" id="btn412"> Last | </a>
<a>Go to </a>
<input id="changePage12" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn512"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">ACE</th><th style="text-align :center">chao1</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson</th><th style="text-align :center">observed_species</th><th style="text-align :center">goods_coverage
</th></tr></thead>
<tr>
<td style="text-align :center">A1</td><td style="text-align :center">1052.91759097362</td><td style="text-align :center">1083.56</td><td style="text-align :center">4.58480391144521</td><td style="text-align :center">0.967182793041138</td><td style="text-align :center">998</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A2</td><td style="text-align :center">1144.50221130055</td><td style="text-align :center">1161.62222222222</td><td style="text-align :center">4.85613872474593</td><td style="text-align :center">0.981143680101542</td><td style="text-align :center">1111</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A3</td><td style="text-align :center">1182.06990673271</td><td style="text-align :center">1201.42</td><td style="text-align :center">4.80865645311165</td><td style="text-align :center">0.977773150563363</td><td style="text-align :center">1135</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A4</td><td style="text-align :center">948.479933794371</td><td style="text-align :center">954.448275862069</td><td style="text-align :center">4.54251592898836</td><td style="text-align :center">0.97264929558409</td><td style="text-align :center">904</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A5</td><td style="text-align :center">1134.56531063942</td><td style="text-align :center">1143.59649122807</td><td style="text-align :center">4.66961284670357</td><td style="text-align :center">0.973095638978488</td><td style="text-align :center">1100</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">A6</td><td style="text-align :center">1209.90212956472</td><td style="text-align :center">1221.875</td><td style="text-align :center">4.74157743395307</td><td style="text-align :center">0.977725113715331</td><td style="text-align :center">1185</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B1</td><td style="text-align :center">1617.30104555384</td><td style="text-align :center">1645.96226415094</td><td style="text-align :center">4.84812107017735</td><td style="text-align :center">0.969697979730342</td><td style="text-align :center">1577</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B2</td><td style="text-align :center">1451.51252779942</td><td style="text-align :center">1469.11428571429</td><td style="text-align :center">4.63036547618701</td><td style="text-align :center">0.975371616879388</td><td style="text-align :center">1408</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B3</td><td style="text-align :center">1419.81688067635</td><td style="text-align :center">1441.33823529412</td><td style="text-align :center">4.85926271522677</td><td style="text-align :center">0.98097149464482</td><td style="text-align :center">1370</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B4</td><td style="text-align :center">1590.55312254093</td><td style="text-align :center">1596.51666666667</td><td style="text-align :center">4.78222670369474</td><td style="text-align :center">0.966610320883688</td><td style="text-align :center">1565</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B5</td><td style="text-align :center">1277.35783322903</td><td style="text-align :center">1293.84375</td><td style="text-align :center">4.60412223606863</td><td style="text-align :center">0.975943227752389</td><td style="text-align :center">1227</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">B6</td><td style="text-align :center">1513.61193568103</td><td style="text-align :center">1524.6</td><td style="text-align :center">4.85114709003732</td><td style="text-align :center">0.974663215617233</td><td style="text-align :center">1488</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C1</td><td style="text-align :center">1310.81171788802</td><td style="text-align :center">1335.05172413793</td><td style="text-align :center">4.86447804451848</td><td style="text-align :center">0.974340015257395</td><td style="text-align :center">1266</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C2</td><td style="text-align :center">1371.60921797556</td><td style="text-align :center">1394.46808510638</td><td style="text-align :center">4.83697773114787</td><td style="text-align :center">0.973513742561464</td><td style="text-align :center">1337</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C3</td><td style="text-align :center">1393.97589641123</td><td style="text-align :center">1441.375</td><td style="text-align :center">4.70304205564494</td><td style="text-align :center">0.959330617751046</td><td style="text-align :center">1350</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C4</td><td style="text-align :center">1337.28261734878</td><td style="text-align :center">1377.10909090909</td><td style="text-align :center">4.89927459134063</td><td style="text-align :center">0.981487928371699</td><td style="text-align :center">1274</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C5</td><td style="text-align :center">1324.31976123196</td><td style="text-align :center">1341.875</td><td style="text-align :center">4.67588232789292</td><td style="text-align :center">0.958257490544209</td><td style="text-align :center">1293</td><td style="text-align :center">1
</td></tr>
<tr>
<td style="text-align :center">C6</td><td style="text-align :center">1372.59878128302</td><td style="text-align :center">1404.26315789474</td><td style="text-align :center">4.5824911567591</td><td style="text-align :center">0.946232923724435</td><td style="text-align :center">1337</td><td style="text-align :center">1
</td></tr>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) ACE: Estimated number of species in the community.</p>
<p class="premark" >(2) chao1: The total number of species contained in the community samples was estimated.</p>
<p class="premark" >(3) shannon: The total number of taxa in the samples and their percentage.The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(4) simpson: characterizes the diversity and evenness of species distribution within a community.</p>
<p class="premark" >(5) observed_species: Number of directly observed species.</p>
<p class="premark" >(6) goods_coverage: Sequencing depth index coverage, the higher the sequencing coverage, the larger the index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">MGEs-plasmid Alpha Diversity indices statistics: result/07.ResistantGene/MGES/plasmid/Diversity/AlphaDiversity/Alpha_index_table/*.xls</p>

    <a name="T5"></a>
    <h5>4.7.2.2 Relative abundance cluster analysis of Function</h5>    

    <a name="T5"></a>
    <h5>4.7.2.2.1 CARD</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the CARD database, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<img src="files/pictures/07.ResistantGene/cluster.card.top.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.73 Abundance clustering heatmap of CARD</strong></p>  
</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">CARD cluster heatmap:result/07.ResistantGene/CARD/Anno/Heatmap_sample/ARO/ARO_heat/heat.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.7.2.2.2 MGEs</h5>    

    <p class="paragraph">Based on the functional annotations and abundance information of all the samples in the three MGEs databases, the top 35 functions in terms of abundance and their abundance information in each sample were selected to draw a heat map and clustered at the level of functional differences.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAbundanceclusteringheatmapofMGEs_nav-tabContent">

<div class="tab-pane active show" id="clusterintegralltoppng1" role="tabpanel" aria-labelledby="clusterintegralltoppng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/cluster.integrall.top.png" /></div>

<div class="tab-pane fade" id="clusterisfindertoppng2" role="tabpanel" aria-labelledby="clusterisfindertoppng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/cluster.isfinder.top.png" /></div>

<div class="tab-pane fade" id="clusterplasmidtoppng3" role="tabpanel" aria-labelledby="clusterplasmidtoppng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/cluster.plasmid.top.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAbundanceclusteringheatmapofMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAbundanceclusteringheatmapofMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAbundanceclusteringheatmapofMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="clusterintegralltoppng1" data-toggle="list" href="#clusterintegralltoppng1" role="tab" aria-controls="clusterintegralltoppng1">integrall</a>

<a class="list-group-item list-group-item-action" id="clusterisfindertoppng2" data-toggle="list" href="#clusterisfindertoppng2" role="tab" aria-controls="clusterisfindertoppng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="clusterplasmidtoppng3" data-toggle="list" href="#clusterplasmidtoppng3" role="tab" aria-controls="clusterplasmidtoppng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.74 Abundance clustering heatmap of MGEs</strong></p>  

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MGEs cluster heatmap:result/07.ResistantGene/MGES/*/Anno/Heatmap_sample/*/*_heat/heat.{pdf,png}</p>

    <a name="T5"></a>
    <h5>4.7.2.3 PCA Analysis</h5>    

    <p class="paragraph">Principal Component Analysis (PCA) Analysis is a method used for dimension reduction analysis. PCA uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. The PCA transformation ensures that the horizontal axis PC1 accounts for the most variation while the vertical axis PC2 accounts for the second most (Avershina et al., 2013). For the samples’ community composition, the closer the sample points within the PCA figure indicates more similarity among the samples. </p>

    <a name="T5"></a>
    <h5>4.7.2.3.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/pca.card.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.75 PCA analysis in CARD</strong></p>  
</p>
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/07.ResistantGene/CARD/Diversity/PCA_group1/ARO/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/07.ResistantGene/CARD/Diversity/PCA_group1/ARO/PCA.{png,pdf}</p>

    <a name="T5"></a>
    <h5>4.7.2.3.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAanalysisinMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAanalysisinMGEs_nav-tabContent">

<div class="tab-pane active show" id="pcaintegrallpng1" role="tabpanel" aria-labelledby="pcaintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pca.integrall.png" /></div>

<div class="tab-pane fade" id="pcaisfinderpng2" role="tabpanel" aria-labelledby="pcaisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pca.isfinder.png" /></div>

<div class="tab-pane fade" id="pcaplasmidpng3" role="tabpanel" aria-labelledby="pcaplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pca.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAanalysisinMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAanalysisinMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAanalysisinMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="pcaintegrallpng1" data-toggle="list" href="#pcaintegrallpng1" role="tab" aria-controls="pcaintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="pcaisfinderpng2" data-toggle="list" href="#pcaisfinderpng2" role="tab" aria-controls="pcaisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="pcaplasmidpng3" data-toggle="list" href="#pcaplasmidpng3" role="tab" aria-controls="pcaplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.76 PCA analysis in MGEs</strong></p>  
<p class="premark" >Notes: PCA: The X-axis is the first principal component; the Y-axis is the second principal component. The percentage is the contribution of the principal components to the sample's variation. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA figure with sample name: result/07.ResistantGene/MGES/*/Diversity/PCA_group1/*/PCA_lable.{png,pdf}</p>

    <p class="paragraph">PCA figure without sample name: result/07.ResistantGene/MGES/*/Diversity/PCA_group1/*/PCA.{png,pdf}</p>

    <a name="T5"></a>
    <h5>4.7.2.4 PCoA Analysis</h5>    

    <p class="paragraph">Principal Coordinate analysis (PCoA) is an ordination technique, which picks up the primary elements and structure from reduced multi-dimensional data series of eigenvalues and eigenvectors. This technique has the advantage over PCA in that each ecological distance can be investigated. Aggregated samples represent higher function composition similarity than those that are further apart. PCoA results based on Bray-Curtis distance are shown below.</p>

    <a name="T5"></a>
    <h5>4.7.2.4.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/pcoa.card.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.77 PCoA analysis in CARD</strong></p>  
</p>
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/07.ResistantGene/CARD/Diversity/PCoA_group1/ARO/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/07.ResistantGene/CARD/Diversity/PCoA_group1/ARO/PCoA.{png,pdf}</p>

    <a name="T5"></a>
    <h5>4.7.2.4.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCoAanalysisinMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCoAanalysisinMGEs_nav-tabContent">

<div class="tab-pane active show" id="pcoaintegrallpng1" role="tabpanel" aria-labelledby="pcoaintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pcoa.integrall.png" /></div>

<div class="tab-pane fade" id="pcoaisfinderpng2" role="tabpanel" aria-labelledby="pcoaisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pcoa.isfinder.png" /></div>

<div class="tab-pane fade" id="pcoaplasmidpng3" role="tabpanel" aria-labelledby="pcoaplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/pcoa.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCoAanalysisinMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCoAanalysisinMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCoAanalysisinMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="pcoaintegrallpng1" data-toggle="list" href="#pcoaintegrallpng1" role="tab" aria-controls="pcoaintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="pcoaisfinderpng2" data-toggle="list" href="#pcoaisfinderpng2" role="tab" aria-controls="pcoaisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="pcoaplasmidpng3" data-toggle="list" href="#pcoaplasmidpng3" role="tab" aria-controls="pcoaplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.78 PCoA analysis in MGEs</strong></p>  
<p class="premark" >Notes: PCoA: The X-axis is the first principal coordinate; the Y-axis is the second principal coordinate. The percentage stands for the contribution of the principal coordinate to the variation in samples. Each point in the graph stands for a sample. Samples belonging to the same group are in the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCoA figure with sample name: result/07.ResistantGene/MGES/*/Diversity/PCoA_group1/*/PCoA_lable.{png,pdf}</p>

    <p class="paragraph">PCoA figure without sample name: result/07.ResistantGene/MGES/*/Diversity/PCoA_group1/*/PCoA.{png,pdf}</p>

    <a name="T5"></a>
    <h5>4.7.2.5 NMDS Analysis</h5>    

    <p class="paragraph">Non-metric Multidimensional Scaling (NMDS) is a ranking method applicable to ecological research. It is a non-linear model designed for a better representation of non-linear biological data structure. NMDS can retain the original relationship among samples while reducing the research targets from a multidimensional space to a low-dimensional space for analysis, positioning, and characterization. This allows NMDS analysis to address the similarity or dissimilarity between targets as a monotonic function of the distance between the points. NMDs analysis results are shown below.</p>

    <a name="T5"></a>
    <h5>4.7.2.5.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/nmds.card.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.79 NMDS analysis in CARD</strong></p>  
</p>
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/07.ResistantGene/CARD/Diversity/NMDS_group1/ARO/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/07.ResistantGene/CARD/Diversity/NMDSgroup1/ARO/NMDS.{png,pdf}</p>

    <a name="T5"></a>
    <h5>4.7.2.5.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureNMDSanalysisinMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureNMDSanalysisinMGEs_nav-tabContent">

<div class="tab-pane active show" id="nmdsintegrallpng1" role="tabpanel" aria-labelledby="nmdsintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/nmds.integrall.png" /></div>

<div class="tab-pane fade" id="nmdsisfinderpng2" role="tabpanel" aria-labelledby="nmdsisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/nmds.isfinder.png" /></div>

<div class="tab-pane fade" id="nmdsplasmidpng3" role="tabpanel" aria-labelledby="nmdsplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/nmds.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureNMDSanalysisinMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureNMDSanalysisinMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureNMDSanalysisinMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="nmdsintegrallpng1" data-toggle="list" href="#nmdsintegrallpng1" role="tab" aria-controls="nmdsintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="nmdsisfinderpng2" data-toggle="list" href="#nmdsisfinderpng2" role="tab" aria-controls="nmdsisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="nmdsplasmidpng3" data-toggle="list" href="#nmdsplasmidpng3" role="tab" aria-controls="nmdsplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.80 NMDS analysis in MGEs</strong></p>  
<p class="premark" >Notes: NMDS: NMDS reflects the pairwise dissimilarity between objects in a low-dimensional space. Samples belonging to the same group are in the same color. When the value of Stress factor is less than 0.2, it is considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS figure with sample name: result/07.ResistantGene/MGES/*/Diversity/NMDS_group1/*/NMDS_lable.{png,pdf}</p>

    <p class="paragraph">NMDS figure without sample name: result/07.ResistantGene/MGES/*/Diversity/NMDS_group1/*/NMDS.{png,pdf}</p>

    <a name="T4"></a>
    <h4>4.7.3 Statistical analysis for ARG</h4>    

    <a name="T5"></a>
    <h5>4.7.3.1 Anosim analysis</h5>    

    <p class="paragraph">Anosim is a nonparametric test to evaluate whether variation among groups is significantly greater than variation within groups, which helps to evaluate the reasonability of the division of groups.</p>

    <a name="T5"></a>
    <h5>4.7.3.1.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/anosim.card.card.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.81 Anosim at CARD</strong></p>  
</p>
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/07.ResistantGene/CARD/StatisticalTest/Anosim_group1</p>

    <a name="T5"></a>
    <h5>4.7.3.1.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureAnosimatMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureAnosimatMGEs_nav-tabContent">

<div class="tab-pane active show" id="anosimmgesintegrallpng1" role="tabpanel" aria-labelledby="anosimmgesintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/anosim.mges.integrall.png" /></div>

<div class="tab-pane fade" id="anosimmgesisfinderpng2" role="tabpanel" aria-labelledby="anosimmgesisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/anosim.mges.isfinder.png" /></div>

<div class="tab-pane fade" id="anosimmgesplasmidpng3" role="tabpanel" aria-labelledby="anosimmgesplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/anosim.mges.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureAnosimatMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureAnosimatMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureAnosimatMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="anosimmgesintegrallpng1" data-toggle="list" href="#anosimmgesintegrallpng1" role="tab" aria-controls="anosimmgesintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="anosimmgesisfinderpng2" data-toggle="list" href="#anosimmgesisfinderpng2" role="tab" aria-controls="anosimmgesisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="anosimmgesplasmidpng3" data-toggle="list" href="#anosimmgesplasmidpng3" role="tab" aria-controls="anosimmgesplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.82 Anosim at MGEs</strong></p>  
<p class="premark" >Notes: Plotted by the rank value on the Y-axis. R-value is a number between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is greater than inter-group. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/07.ResistantGene/MGES/*/StatisticalTest/Anosim_group1</p>

    <a name="T5"></a>
    <h5>4.7.3.2 MetaGenomeSeq analysis</h5>    

    <p class="paragraph">MetaGenomeSeq is developed in the context of metagenomic data comprising multiple samples, and relies on a non-parametric t-test for detecting differentially abundant features in metagenomic studies. The p-value is further corrected as q-value to discover taxa with significant variation.</p>

    <a name="T5"></a>
    <h5>4.7.3.2.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/metastat.card.card.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.83 Box chart for function with significantly variant relative abundance between groups</strong></p>  
</p>
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding function. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for function whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<img src="files/pictures/07.ResistantGene/metastat.card.card.cluster.pca.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.84 PCA and abundance heatmap based on CARD</strong></p>  
</p>
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents function annotation; the left side of the heatmap is a clustering tree of function. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/07.ResistantGene/CARD/StatisticalTest/MetaGenomeSeq_group1/ARO</p>

    <a name="T5"></a>
    <h5>4.7.3.2.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxchartforfunctionwithsignificantlyvariantrelativeabundancebetweengroups_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxchartforfunctionwithsignificantlyvariantrelativeabundancebetweengroups_nav-tabContent">

<div class="tab-pane active show" id="metastatmgesintegrallpng1" role="tabpanel" aria-labelledby="metastatmgesintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.integrall.png" /></div>

<div class="tab-pane fade" id="metastatmgesisfinderpng2" role="tabpanel" aria-labelledby="metastatmgesisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.isfinder.png" /></div>

<div class="tab-pane fade" id="metastatmgesplasmidpng3" role="tabpanel" aria-labelledby="metastatmgesplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxchartforfunctionwithsignificantlyvariantrelativeabundancebetweengroups_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxchartforfunctionwithsignificantlyvariantrelativeabundancebetweengroups_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxchartforfunctionwithsignificantlyvariantrelativeabundancebetweengroups_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatmgesintegrallpng1" data-toggle="list" href="#metastatmgesintegrallpng1" role="tab" aria-controls="metastatmgesintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="metastatmgesisfinderpng2" data-toggle="list" href="#metastatmgesisfinderpng2" role="tab" aria-controls="metastatmgesisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="metastatmgesplasmidpng3" data-toggle="list" href="#metastatmgesplasmidpng3" role="tab" aria-controls="metastatmgesplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.85 Box chart for function with significantly variant relative abundance between groups</strong></p>  
<p class="premark" >Notes: The X-axis indicates the group of samples; the Y-axis indicates the relative abundance of corresponding function. "*" means the variation between the two groups is significant (q <0.05); "**" means the variation between the two groups is extremely significant (q <0.01). </p>

    <p class="paragraph">PCA and abundance heatmap for function whose abundance were significantly different between groups were shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigurePCAandabundanceheatmapbasedonMGEs_nav-tabContent">

<div class="tab-pane active show" id="metastatmgesintegrallclusterpcapng1" role="tabpanel" aria-labelledby="metastatmgesintegrallclusterpcapng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.integrall.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatmgesisfinderclusterpcapng2" role="tabpanel" aria-labelledby="metastatmgesisfinderclusterpcapng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.isfinder.cluster.pca.png" /></div>

<div class="tab-pane fade" id="metastatmgesplasmidclusterpcapng3" role="tabpanel" aria-labelledby="metastatmgesplasmidclusterpcapng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/metastat.mges.plasmid.cluster.pca.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigurePCAandabundanceheatmapbasedonMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigurePCAandabundanceheatmapbasedonMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigurePCAandabundanceheatmapbasedonMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="metastatmgesintegrallclusterpcapng1" data-toggle="list" href="#metastatmgesintegrallclusterpcapng1" role="tab" aria-controls="metastatmgesintegrallclusterpcapng1">integrall</a>

<a class="list-group-item list-group-item-action" id="metastatmgesisfinderclusterpcapng2" data-toggle="list" href="#metastatmgesisfinderclusterpcapng2" role="tab" aria-controls="metastatmgesisfinderclusterpcapng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="metastatmgesplasmidclusterpcapng3" data-toggle="list" href="#metastatmgesplasmidclusterpcapng3" role="tab" aria-controls="metastatmgesplasmidclusterpcapng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.86 PCA and abundance heatmap based on MGEs</strong></p>  
<p class="premark" >Notes: a) The abundance heatmap: The X-axis represents the sample information; the Y-axis represents function annotation; the left side of the heatmap is a clustering tree of function. The value of the heatmap is Z value which is standardized. The absolute value of "Z" has the same meaning as above. b) PCA: each point represents a sample and samples in the same group are represented by the same color.</p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">MetaGenomeSeq analysis result: result/07.ResistantGene/MGES/*/StatisticalTest/MetaGenomeSeq_group1/*</p>

    <a name="T5"></a>
    <h5>4.7.3.3 LEfSe analysis</h5>    

    <p class="paragraph">LEfSe (linear discriminant analysis (LDA) Effect Size) analysis allows high-dimensional biomarker discovery and explanation. LEfSe identifies genomic features (genes, pathways or taxa) characterizing the differences between two or more biological conditions (or classes). It enables the characterization of microbial taxa specific to an experimental or environmental condition, and the identification of metagenomic biomarkers in different communities.</p>

    <a name="T5"></a>
    <h5>4.7.3.3.1 CARD</h5>    

<p class="center">
<img src="files/pictures/07.ResistantGene/LDA.ARO.png" width="700" height="350" />
<p class="name_fig"><strong>Figure 4.87 Histogram of LDA scores on CARD</strong></p>  
</p>
<p class="premark" >Notes: The histogram of the LDA scores presents function (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/07.ResistantGene/CARD/StatisticalTest/LEfSe_group1/ARO</p>

    <a name="T5"></a>
    <h5>4.7.3.3.2 MGEs</h5>    

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscoresonMGEs_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscoresonMGEs_nav-tabContent">

<div class="tab-pane active show" id="LDAintegrallpng1" role="tabpanel" aria-labelledby="LDAintegrallpng1" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/LDA.integrall.png" /></div>

<div class="tab-pane fade" id="LDAisfinderpng2" role="tabpanel" aria-labelledby="LDAisfinderpng2" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/LDA.isfinder.png" /></div>

<div class="tab-pane fade" id="LDAplasmidpng3" role="tabpanel" aria-labelledby="LDAplasmidpng3" onclick="trans(this.id)"><img class="img-gallery" src="files/pictures/07.ResistantGene/LDA.plasmid.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscoresonMGEs_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscoresonMGEs_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscoresonMGEs_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="LDAintegrallpng1" data-toggle="list" href="#LDAintegrallpng1" role="tab" aria-controls="LDAintegrallpng1">integrall</a>

<a class="list-group-item list-group-item-action" id="LDAisfinderpng2" data-toggle="list" href="#LDAisfinderpng2" role="tab" aria-controls="LDAisfinderpng2">isfinder</a>

<a class="list-group-item list-group-item-action" id="LDAplasmidpng3" data-toggle="list" href="#LDAplasmidpng3" role="tab" aria-controls="LDAplasmidpng3">plasmid</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.88 Histogram of LDA scores on MGEs</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores presents function (biomarker) whose LDA scores are larger than the set threshold (3 set by default). The LDA score represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups). </p>

    <p class="paragraph">Result Directory:</p>

    <p class="paragraph">LEfSe analysis result: result/07.ResistantGene/MGES/*/StatisticalTest/LEfSe_group1/*</p>

        <br />

        <br />

<hr />
</p><br />
<h2>5 Appendix</h2>


        <br />

        <br />

<h3>5.1 List of analysis software</h3>
        
<p class="name_table"><p class="center"><strong>Table 5.1 Main softwore used in the analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable13" >
        <thead><tr>
<a id="btn013"></a>
<a id="sjzl13"></a>
<a  href="javascript:void(0)" id="btn113"> | First | </a>
<a  href="javascript:void(0)" id="btn213"> Previous | </a>
<a  href="javascript:void(0)" id="btn313"> Next | </a>
<a  href="javascript:void(0)" id="btn413"> Last | </a>
<a>Go to </a>
<input id="changePage13" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn513"> Jump </a>
                
<th style="text-align :center">Analytical content</th><th style="text-align :center">Software</th><th style="text-align :center">Version</th><th style="text-align :center">Parameter</th><th style="text-align :center">Remarks
</th></tr></thead>
<tr>
<td style="text-align :center">Data quality</td><td style="text-align :center">fastp</td><td style="text-align :center">0.23.1</td><td style="text-align :center">-g -q 5 -u 50 -n 15 -l 150 --overlap_diff_limit 1 --overlap_diff_percent_limit 10</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Host trimming</td><td style="text-align :center">bowtie2</td><td style="text-align :center">2.5.4</td><td style="text-align :center">  --end-to-end --sensitive --no-hd --no-sq -I 200 -X 400 --threads 8 </td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Assembly</td><td style="text-align :center">megahit</td><td style="text-align :center">1.2.9</td><td style="text-align :center">--presets meta-large -m 180000000000 -t 2</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Gene prediction</td><td style="text-align :center">GeneMark.hmm</td><td style="text-align :center">2.1</td><td style="text-align :center"> -a  -d -f G -p 1  </td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">ORFs dereplication</td><td style="text-align :center">CD-HIT</td><td style="text-align :center">4.5.8</td><td style="text-align :center">  -T 6 -G 0 -aS 0.9 -g 1 -d 0  -c 0.95 -n 5 -M 8000</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Reads mapping</td><td style="text-align :center">bowtie2</td><td style="text-align :center">2.5.4</td><td style="text-align :center">  --end-to-end --sensitive --no-hd --no-sq -I 200 -X 400 --threads 8 </td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Correlation heatmap</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = corrplot</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Corepan</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = ggplot2</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">vennplot </td><td style="text-align :center">R</td><td style="text-align :center">v3.0.3</td><td style="text-align :center">R package = VennDiagram</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Differential analysis of gene numbers</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = ggplot2</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Correlation analysis based on gene nubmers</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = corrplot</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">MicroNR</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">MicroNR(Version: 2024.03)
</td></tr>
<tr>
<td style="text-align :center">KEGG</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">KEGG(Version: 2024.03)
</td></tr>
<tr>
<td style="text-align :center">eggNOG</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">eggNOG(Version: 5.0)
</td></tr>
<tr>
<td style="text-align :center">CAZy</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">CAZy(Version: 2024.03)
</td></tr>
<tr>
<td style="text-align :center">VFDB</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">VFDB(Version: 2024.06)
</td></tr>
<tr>
<td style="text-align :center">PHI</td><td style="text-align :center">diamond</td><td style="text-align :center">2.1.9</td><td style="text-align :center">-p 4 -e 1e-5 -k 50 --id 30 --sensitive</td><td style="text-align :center">PHI(Version: 2023.03)
</td></tr>
<tr>
<td style="text-align :center">CARD</td><td style="text-align :center">RGI</td><td style="text-align :center">6.0.2</td><td style="text-align :center">-d wgs --clean</td><td style="text-align :center">CARD(Version:3.2.6)
</td></tr>
<tr>
<td style="text-align :center">MGEs</td><td style="text-align :center">blastn</td><td style="text-align :center">2.12.0</td><td style="text-align :center">-num_threads 6 -outfmt 6 -evalue 1e-5</td><td style="text-align :center">Integrall(Version: 2021.09),Isfinder(Version: 2023.10),Plasmid(Version: 2021.06)
</td></tr>
<tr>
<td style="text-align :center">Alpha index analysis</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = vegan</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">PCA analysis</td><td style="text-align :center">R</td><td style="text-align :center">3.0.3</td><td style="text-align :center">R package = FactoMineR， ggplot2，grid</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">NMDS analysis</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = vegan,permute,lattice</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Anosim</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = vegan,permute,lattice</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">PCoA</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = extrafont,ggplot2,grid</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Clustering heatmap</td><td style="text-align :center">R</td><td style="text-align :center">3.1.0</td><td style="text-align :center">R package = pheatmap</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">MetaGenomeSeq</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">q value (Benjamini and Hochberg FDR) < 0.05</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">MetaGenomeSeq heatmap</td><td style="text-align :center">R</td><td style="text-align :center">3.1.0</td><td style="text-align :center">R package = pheatmap</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">MetaGenomeSeq combined</td><td style="text-align :center">R</td><td style="text-align :center">2.15.3</td><td style="text-align :center">R package = ggplot2</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">LEfSe</td><td style="text-align :center">LEfSe</td><td style="text-align :center">1.0</td><td style="text-align :center">LDA Score>4</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Krona</td><td style="text-align :center">perl</td><td style="text-align :center">5.18.2</td><td style="text-align :center">use KronaTools</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Bar plot of relative abundance</td><td style="text-align :center">perl</td><td style="text-align :center">5.18.2</td><td style="text-align :center">use SVG</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Clustering Tree</td><td style="text-align :center">perl</td><td style="text-align :center">5.18.2</td><td style="text-align :center">use SVG</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Circos plot</td><td style="text-align :center">circos</td><td style="text-align :center">v0.64</td><td style="text-align :center">　</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Bar plot of functional relative abundance</td><td style="text-align :center">perl</td><td style="text-align :center">5.18.2</td><td style="text-align :center">use SVG</td><td style="text-align :center">
</td></tr>
<tr>
<td style="text-align :center">Comparison of pathways</td><td style="text-align :center">perl</td><td style="text-align :center">5.18.2</td><td style="text-align :center">use SVG</td><td style="text-align :center">
</td></tr>

</table>
</div>
</p>

        <br />

        <br />

<h3>5.2 Methods</h3>
        
    <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="files/Sequencing_Methods_en.pdf" target="_blank"><b>Method</b></a></p>
        

        <br />

        <br />

<h3>5.3 Notes</h3>
        
    <p class="paragraph">It is strongly advised to scan this report on Firefox (http://www.firefox.com.cn/download/), for several browsers such as IE may disable for table display. More vectograms and statistic data are provided in delivery directory.</p>

    <p class="paragraph">The result file is recommended to use a professional text editor such as Excel or EditPlus to open.</p>

        <br />

        <br />

<h3>5.4 Result File Decompression Tool and Format Description</h3>
        
<p class="name_table"><p class="center"><strong>Table 5.2 Result File Decompression Tool and Format Description</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable14" >
        <thead><tr>
<a id="btn014"></a>
<a id="sjzl14"></a>
<a  href="javascript:void(0)" id="btn114"> | First | </a>
<a  href="javascript:void(0)" id="btn214"> Previous | </a>
<a  href="javascript:void(0)" id="btn314"> Next | </a>
<a  href="javascript:void(0)" id="btn414"> Last | </a>
<a>Go to </a>
<input id="changePage14" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn514"> Jump </a>
                
<th style="text-align :center">Compressed format</th><th style="text-align :center">Customer type</th><th style="text-align :center">Decompression method
</th></tr></thead>
<tr>
<td style="text-align :center">*.gz format</td><td style="text-align :center">Unix/Linux/Mac</td><td style="text-align :center">Open with gzip -d *.gz command
</td></tr>
<tr>
<td style="text-align :center">*.gz format</td><td style="text-align :center">Windows</td><td style="text-align :center">Open with decompression software such as WinRAR, 7-Zip, etc
</td></tr>
<tr>
<td style="text-align :center">*.zip format</td><td style="text-align :center">Unix/Linux/Mac</td><td style="text-align :center">Open with unzip *.zip command
</td></tr>
<tr>
<td style="text-align :center">*.zip format</td><td style="text-align :center">Windows</td><td style="text-align :center">Open with decompression software such as WinRAR, 7-Zip, etc
</td></tr>
<tr>
<td style="text-align :center">*.tar format</td><td style="text-align :center">Unix/Linux/Mac</td><td style="text-align :center">Open with tar -xvf *.tar command
</td></tr>
<tr>
<td style="text-align :center">*.tar format</td><td style="text-align :center">Windows</td><td style="text-align :center">Open with decompression software such as WinRAR, 7-Zip, etc
</td></tr>

</table>
</div>
</p>

        <br />

        <br />

<hr />
</p><br />
<h2>6 References</h2>


    <p class="paragraph">Avershina, E., Frisli, T., & Rudi, K. (2013). De novo semi-alignment of 16S rRNA gene sequences for deep phylogenetic characterization of next generation sequencing data. Microbes and environments, 28(2), 211–216. https://doi.org/10.1264/jsme2.me12157</p>

    <p class="paragraph">Bäckhed, F., Roswall, J., Peng, Y., Feng, Q., Jia, H., Kovatcheva-Datchary, P., Li, Y., Xia, Y., Xie, H., Zhong, H., Khan, M. T., Zhang, J., Li, J., Xiao, L., Al-Aama, J., Zhang, D., Lee, Y. S., Kotowska, D., Colding, C., Tremaroli, V., … Wang, J. (2015). Dynamics and Stabilization of the Human Gut Microbiome during the First Year of Life. Cell host & microbe, 17(6), 852. https://doi.org/10.1016/j.chom.2015.05.012</p>

    <p class="paragraph">Buchfink, B., Xie, C., & Huson, D. H. (2015). Fast and sensitive protein alignment using DIAMOND. Nature methods, 12(1), 59–60. https://doi.org/10.1038/nmeth.3176</p>

    <p class="paragraph">Cantarel, B. L., Coutinho, P. M., Rancurel, C., Bernard, T., Lombard, V., & Henrissat, B. (2009). The Carbohydrate-Active EnZymes database (CAZy): an expert resource for Glycogenomics. Nucleic acids research, 37(Database issue), D233–D238. https://doi.org/10.1093/nar/gkn663</p>

    <p class="paragraph">Cotillard, A., Kennedy, S. P., Kong, L. C., Prifti, E., Pons, N., Le Chatelier, E., Almeida, M., Quinquis, B., Levenez, F., Galleron, N., Gougis, S., Rizkalla, S., Batto, J. M., Renault, P., ANR MicroObes consortium, Doré, J., Zucker, J. D., Clément, K., & Ehrlich, S. D. (2013). Dietary intervention impact on gut microbial gene richness. Nature, 500(7464), 585–588. https://doi.org/10.1038/nature12480</p>

    <p class="paragraph">Feng, Q., Liang, S., Jia, H., Stadlmayr, A., Tang, L., Lan, Z., Zhang, D., Xia, H., Xu, X., Jie, Z., Su, L., Li, X., Li, X., Li, J., Xiao, L., Huber-Schönauer, U., Niederseer, D., Xu, X., Al-Aama, J. Y., Yang, H., … Wang, J. (2015). Gut microbiome development along the colorectal adenoma-carcinoma sequence. Nature communications, 6, 6528. https://doi.org/10.1038/ncomms7528</p>

    <p class="paragraph">Forsberg, K. J., Patel, S., Gibson, M. K., Lauber, C. L., Knight, R., Fierer, N., & Dantas, G. (2014). Bacterial phylogeny structures soil resistomes across habitats. Nature, 509(7502), 612–616. https://doi.org/10.1038/nature13377</p>

    <p class="paragraph">Fu, L., Niu, B., Zhu, Z., Wu, S., & Li, W. (2012). CD-HIT: accelerated for clustering the next-generation sequencing data. Bioinformatics (Oxford, England), 28(23), 3150–3152. https://doi.org/10.1093/bioinformatics/bts565</p>

    <p class="paragraph">Huson, D. H., Mitra, S., Ruscheweyh, H. J., Weber, N., & Schuster, S. C. (2011). Integrative analysis of environmental sequences using MEGAN4. Genome research, 21(9), 1552–1560. https://doi.org/10.1101/gr.120618.111</p>

    <p class="paragraph">Kanehisa, M., Goto, S., Hattori, M., Aoki-Kinoshita, K. F., Itoh, M., Kawashima, S., Katayama, T., Araki, M., & Hirakawa, M. (2006). From genomics to chemical genomics: new developments in KEGG. Nucleic acids research, 34(Database issue), D354–D357. https://doi.org/10.1093/nar/gkj102</p>

    <p class="paragraph">Kanehisa, M., Goto, S., Sato, Y., Kawashima, M., Furumichi, M., & Tanabe, M. (2014). Data, information, knowledge and principle: back to metabolism in KEGG. Nucleic acids research, 42(Database issue), D199–D205. https://doi.org/10.1093/nar/gkt1076</p>

    <p class="paragraph">Karlsson, F. H., Fåk, F., Nookaew, I., Tremaroli, V., Fagerberg, B., Petranovic, D., Bäckhed, F., & Nielsen, J. (2012). Symptomatic atherosclerosis is associated with an altered gut metagenome. Nature communications, 3, 1245. https://doi.org/10.1038/ncomms2266</p>

    <p class="paragraph">Karlsson, F. H., Tremaroli, V., Nookaew, I., Bergström, G., Behre, C. J., Fagerberg, B., Nielsen, J., & Bäckhed, F. (2013). Gut metagenome in European women with normal, impaired and diabetic glucose control. Nature, 498(7452), 99–103. https://doi.org/10.1038/nature12198</p>

    <p class="paragraph">Le Chatelier, E., Nielsen, T., Qin, J., Prifti, E., Hildebrand, F., Falony, G., Almeida, M., Arumugam, M., Batto, J. M., Kennedy, S., Leonard, P., Li, J., Burgdorf, K., Grarup, N., Jørgensen, T., Brandslund, I., Nielsen, H. B., Juncker, A. S., Bertalan, M., Levenez, F., … Pedersen, O. (2013). Richness of human gut microbiome correlates with metabolic markers. Nature, 500(7464), 541–546. https://doi.org/10.1038/nature12506</p>

    <p class="paragraph">Li, J., Jia, H., Cai, X., Zhong, H., Feng, Q., Sunagawa, S., Arumugam, M., Kultima, J. R., Prifti, E., Nielsen, T., Juncker, A. S., Manichanh, C., Chen, B., Zhang, W., Levenez, F., Wang, J., Xu, X., Xiao, L., Liang, S., Zhang, D., … MetaHIT Consortium (2014). An integrated catalog of reference genes in the human gut microbiome. Nature biotechnology, 32(8), 834–841. https://doi.org/10.1038/nbt.2942</p>

    <p class="paragraph">Li, W., & Godzik, A. (2006). Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. Bioinformatics (Oxford, England), 22(13), 1658–1659. https://doi.org/10.1093/bioinformatics/btl158</p>

    <p class="paragraph">Liu, B., & Pop, M. (2009). ARDB--Antibiotic Resistance Genes Database. Nucleic acids research, 37(Database issue), D443–D447. https://doi.org/10.1093/nar/gkn656</p>

    <p class="paragraph">Martínez, J. L., Coque, T. M., & Baquero, F. (2015). What is a resistance gene? Ranking risk in resistomes. Nature reviews. Microbiology, 13(2), 116–123. https://doi.org/10.1038/nrmicro3399</p>

    <p class="paragraph">Mende, D. R., Waller, A. S., Sunagawa, S., Järvelin, A. I., Chan, M. M., Arumugam, M., Raes, J., & Bork, P. (2012). Assessment of metagenomic assembly using simulated next generation sequencing data. PloS one, 7(2), e31386. https://doi.org/10.1371/journal.pone.0031386</p>

    <p class="paragraph">Nielsen, H. B., Almeida, M., Juncker, A. S., Rasmussen, S., Li, J., Sunagawa, S., Plichta, D. R., Gautier, L., Pedersen, A. G., Le Chatelier, E., Pelletier, E., Bonde, I., Nielsen, T., Manichanh, C., Arumugam, M., Batto, J. M., Quintanilha Dos Santos, M. B., Blom, N., Borruel, N., Burgdorf, K. S., … MetaHIT Consortium (2014). Identification and assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes. Nature biotechnology, 32(8), 822–828. https://doi.org/10.1038/nbt.2939</p>

    <p class="paragraph">Noval Rivas, M., Burton, O. T., Wise, P., Zhang, Y. Q., Hobson, S. A., Garcia Lloret, M., Chehoud, C., Kuczynski, J., DeSantis, T., Warrington, J., Hyde, E. R., Petrosino, J. F., Gerber, G. K., Bry, L., Oettgen, H. C., Mazmanian, S. K., & Chatila, T. A. (2013). A microbiota signature associated with experimental food allergy promotes allergic sensitization and anaphylaxis. The Journal of allergy and clinical immunology, 131(1), 201–212. https://doi.org/10.1016/j.jaci.2012.10.026</p>

    <p class="paragraph">Oh, J., Byrd, A. L., Deming, C., Conlan, S., NISC Comparative Sequencing Program, Kong, H. H., & Segre, J. A. (2014). Biogeography and individuality shape function in the human skin metagenome. Nature, 514(7520), 59–64. https://doi.org/10.1038/nature13786</p>

    <p class="paragraph">Ondov, B. D., Bergman, N. H., & Phillippy, A. M. (2011). Interactive metagenomic visualization in a Web browser. BMC bioinformatics, 12, 385. https://doi.org/10.1186/1471-2105-12-385</p>

    <p class="paragraph">Powell, S., Forslund, K., Szklarczyk, D., Trachana, K., Roth, A., Huerta-Cepas, J., Gabaldón, T., Rattei, T., Creevey, C., Kuhn, M., Jensen, L. J., von Mering, C., & Bork, P. (2014). eggNOG v4.0: nested orthology inference across 3686 organisms. Nucleic acids research, 42(Database issue), D231–D239. https://doi.org/10.1093/nar/gkt1253</p>

    <p class="paragraph">Qin, J., Li, R., Raes, J., Arumugam, M., Burgdorf, K. S., Manichanh, C., Nielsen, T., Pons, N., Levenez, F., Yamada, T., Mende, D. R., Li, J., Xu, J., Li, S., Li, D., Cao, J., Wang, B., Liang, H., Zheng, H., Xie, Y., … Wang, J. (2010). A human gut microbial gene catalogue established by metagenomic sequencing. Nature, 464(7285), 59–65. https://doi.org/10.1038/nature08821</p>

    <p class="paragraph">Qin, J., Li, Y., Cai, Z., Li, S., Zhu, J., Zhang, F., Liang, S., Zhang, W., Guan, Y., Shen, D., Peng, Y., Zhang, D., Jie, Z., Wu, W., Qin, Y., Xue, W., Li, J., Han, L., Lu, D., Wu, P., … Wang, J. (2012). A metagenome-wide association study of gut microbiota in type 2 diabetes. Nature, 490(7418), 55–60. https://doi.org/10.1038/nature11450</p>

    <p class="paragraph">Qin, N., Yang, F., Li, A., Prifti, E., Chen, Y., Shao, L., Guo, J., Le Chatelier, E., Yao, J., Wu, L., Zhou, J., Ni, S., Liu, L., Pons, N., Batto, J. M., Kennedy, S. P., Leonard, P., Yuan, C., Ding, W., Chen, Y., … Li, L. (2014). Alterations of the human gut microbiome in liver cirrhosis. Nature, 513(7516), 59–64. https://doi.org/10.1038/nature13568</p>

    <p class="paragraph">Raes, J., Foerstner, K. U., & Bork, P. (2007). Get the most out of your metagenome: computational analysis of environmental sequence data. Current opinion in microbiology, 10(5), 490–498. https://doi.org/10.1016/j.mib.2007.09.001</p>

    <p class="paragraph">Segata, N., Izard, J., Waldron, L., Gevers, D., Miropolsky, L., Garrett, W. S., & Huttenhower, C. (2011). Metagenomic biomarker discovery and explanation. Genome biology, 12(6), R60. https://doi.org/10.1186/gb-2011-12-6-r60</p>

    <p class="paragraph">Sunagawa, S., Coelho, L. P., Chaffron, S., Kultima, J. R., Labadie, K., Salazar, G., Djahanschiri, B., Zeller, G., Mende, D. R., Alberti, A., Cornejo-Castillo, F. M., Costea, P. I., Cruaud, C., d'Ovidio, F., Engelen, S., Ferrera, I., Gasol, J. M., Guidi, L., Hildebrand, F., Kokoszka, F., … Bork, P. (2015). Ocean plankton. Structure and function of the global ocean microbiome. Science (New York, N.Y.), 348(6237), 1261359. https://doi.org/10.1126/science.1261359</p>

    <p class="paragraph">Tringe, S. G., von Mering, C., Kobayashi, A., Salamov, A. A., Chen, K., Chang, H. W., Podar, M., Short, J. M., Mathur, E. J., Detter, J. C., Bork, P., Hugenholtz, P., & Rubin, E. M. (2005). Comparative metagenomics of microbial communities. Science (New York, N.Y.), 308(5721), 554–557. https://doi.org/10.1126/science.1107851</p>

    <p class="paragraph">van der Heijden, M. G., Bardgett, R. D., & van Straalen, N. M. (2008). The unseen majority: soil microbes as drivers of plant diversity and productivity in terrestrial ecosystems. Ecology letters, 11(3), 296–310. https://doi.org/10.1111/j.1461-0248.2007.01139.x</p>

    <p class="paragraph">Villar, E., Farrant, G. K., Follows, M., Garczarek, L., Speich, S., Audic, S., Bittner, L., Blanke, B., Brum, J. R., Brunet, C., Casotti, R., Chase, A., Dolan, J. R., d'Ortenzio, F., Gattuso, J. P., Grima, N., Guidi, L., Hill, C. N., Jahn, O., Jamet, J. L., … Iudicone, D. (2015). Ocean plankton. Environmental characteristics of Agulhas rings affect interocean plankton transport. Science (New York, N.Y.), 348(6237), 1261447. https://doi.org/10.1126/science.1261447</p>

    <p class="paragraph">Walter, J., & Ley, R. (2011). The human gut microbiome: ecology and recent evolutionary changes. Annual review of microbiology, 65, 411–429. https://doi.org/10.1146/annurev-micro-090110-102830 </p>

    <p class="paragraph">Zeller, G., Tap, J., Voigt, A. Y., Sunagawa, S., Kultima, J. R., Costea, P. I., Amiot, A., Böhm, J., Brunetti, F., Habermann, N., Hercog, R., Koch, M., Luciani, A., Mende, D. R., Schneider, M. A., Schrotz-King, P., Tournigand, C., Tran Van Nhieu, J., Yamada, T., Zimmermann, J., … Bork, P. (2014). Potential of fecal microbiota for early-stage detection of colorectal cancer. Molecular systems biology, 10(11), 766. https://doi.org/10.15252/msb.20145645</p>

    <p class="paragraph">Zhu, W., Lomsadze, A., & Borodovsky, M. (2010). Ab initio gene identification in metagenomic sequences. Nucleic acids research, 38(12), e132. https://doi.org/10.1093/nar/gkq275</p>

        <br />

        <br />


</body> 
    
</html>    
    
