
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>

<head>



<title> TESTXXXXXXXXXX Project Standard Analysis Report </title>



	


	<!-- add 20200312 -->
	<link rel="stylesheet" href="src/css/jquery.tocify.css" />
	<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
	<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

	<!--<script src="src/js/jquery-1.4.2.min.js"></script>-->
	<link rel="stylesheet" type="text/css" href="src/css/base_new.css" />
	<link rel="StyleSheet" href="/src/" type="text/css">
	<link rel="stylesheet" type="text/css" href="src/js/fancybox/jquery.fancybox-1.3.4.css" media="screen" />
	<link rel="stylesheet" href="src/css/style.css" />
	<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />
	<link href="src/css/cloth.css" rel="stylesheet" type="text/css" media="screen" />
	<script type="text/javascript" src="src/js/tablecloth.js"></script>
	<script src="src/js/bootstrap.js" type="text/javascript"></script>
	<script src="src/js/echarts.min.js" type="text/javascript"></script>
	<script src="src/js/scrollTop.js"></script>
	<script src="src/js/common.js"></script>
	<script src="src/js/jquery.albumSlider.min.js"></script>
	<script src="src/js/tree/tree.js"></script>
	<script src="src/js/fancybox/jquery.mousewheel-3.0.4.pack.js"></script>
	<script src="src/js/fancybox/jquery.fancybox-1.3.4.pack.js"></script>
	<link rel="stylesheet" href="src/css/jquery-ui.css" />
<link rel="stylesheet" href="src/css/gallery.css"/>

	<link rel="stylesheet" type="text/css" href="src/css/font-awesome.min.css" />
	<link rel="stylesheet" type="text/css" href="src/css/buttons.css" />
	<link rel="stylesheet" href="src/css/memu_style.css" media="screen" type="text/css" />

<link rel="stylesheet" href="src/css/jquery.tocify.css" />

<link rel="stylesheet" href="src/css/style.css"/>

<link rel="stylesheet" href="src/css/base.css"/>


<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>

<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>

<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>

<script src="src/js/scrollTop.js" type="text/javascript"></script>



	<style>
		.longtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		.sublongtext {
			line-height: 16px !important;
			padding-top: 6px;
			padding-bottom: 7px;
		}

		#menu_list li {
			height: 45px;
			line-height: 45px;
		}
	</style>


	<script>
		$(document).ready(function () {
			/*
			*   Examples - images
			*/

			$("a#example1").fancybox();

			$("a#example2").fancybox({
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'elastic'
			});
			$("a#example3").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("a#example4").fancybox({
				'opacity': true,
				'overlayShow': false,
				'transitionIn': 'elastic',
				'transitionOut': 'none'
			});

			$("a#example5").fancybox();

			$("a#example6").fancybox({
				'titlePosition': 'outside',
				'overlayColor': '#000',
				'overlayOpacity': 0.9
			});

			$("a#example7").fancybox({
				'titlePosition': 'inside'
			});

			$("a#example8").fancybox({
				'titlePosition': 'over'
			});

			$("a[rel=example_group]").fancybox({
				'transitionIn': 'none',
				'transitionOut': 'none',
				'titlePosition': 'over',
				'titleFormat': function (title, currentArray, currentIndex, currentOpts) {
					return '<span id="fancybox-title-over">Image ' + (currentIndex + 1) + ' / ' + currentArray.length + (title.length ? ' &nbsp; ' + title : '') + '</span>';
				}
			});

			/*
			*   Examples - various
			*/

			$("#various1").fancybox({
				'titlePosition': 'inside',
				'transitionIn': 'none',
				'transitionOut': 'none'
			});

			$("#various2").fancybox();

			$("#various3").fancybox({
				'width': '75%',
				'height': '75%',
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none',
				'type': 'iframe'
			});

			$("#various4").fancybox({
				'padding': 0,
				'autoScale': false,
				'transitionIn': 'none',
				'transitionOut': 'none'
			});
		});
	</script>

	<script type="text/javascript">

		function trans (img_div_id) {

			img_div = $("#" + img_div_id)

			img = $("#" + img_div_id + " img").first();

			if (img_div.hasClass("fullscreen-div")) {

				img_div.removeClass("fullscreen-div");

				img.removeClass("fullscreen-img");

			}

			else {

				img_div.addClass("fullscreen-div");

				img.addClass("fullscreen-img");

			}

		}



		function filter_display () {

			return $(this).css("display") != "none";

		}



		function show_default () {

			var list_group_id = "#" + $(this).attr("id");
			if ($(list_group_id + " a.active").css("display") == "none") {

				var list_tab = $(list_group_id + " a.active");

				var pan_id = list_tab.attr("href");

				$(pan_id).removeClass("active show");

				list_tab.removeClass("active");

			}

			if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

				$(list_group_id + " a").filter(filter_display).first().tab("show");

			}

		}

		$.fn.show_default = show_default



		function next (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

			while (present_tab.next().length == 1) {

				if (present_tab.next().css("display") != "none") {

					present_tab.next().tab("show");


					break;

				}

				else {

					present_tab = present_tab.next();

				}

			}



		}

		function prev (botton_id) {

			var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

			while (present_tab.prev().length == 1) {

				if (present_tab.prev().css("display") != "none") {

					present_tab.prev().tab("show");

					break;

				}

				else {

					present_tab = present_tab.prev();

				}

			}





		}



		$(document).ready(function () {

			$(".form-control").on("keyup", function () {

				var value = $(this).val().toLowerCase();

				var input_box_id = "#" + $(this).attr("id");

				var list_group_id = input_box_id.replace("_InputList", "_list-tab");

				$(list_group_id + " a").filter(function () {

					$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

				});

				$(list_group_id).show_default();

			});
			$(".list-group").each(show_default);

		});
	</script>

	<style media="print">
		.noprint {
			DISPLAY: none;
		}
	</style>

	<div class="noprint">
		<div style="display: block;" id="goTopBtn">
			<a class="backtotop" title="回顶部"><img src="src/images/goTop.jpg" width="30" height="30" class="back-tip" /></a>
		</div>
	</div>

	<script>
		function displaySubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "block";
		}
		function hideSubMenu (li) {
			var subMenu = li.getElementsByTagName("ul")[0];
			subMenu.style.display = "none";
		} 
	</script>

	</script>
	<script type="text/javascript">
		/* show or hide element*/
		function hiddenShow (btnObj) {
			ctObj = btnObj.nextElementSibling;
			if (ctObj.style.display == 'inline') {
				ctObj.style.display = 'none';
				btnObj.innerHTML = "Show Annotation";
			} else {
				ctObj.style.display = 'inline';
				btnObj.innerHTML = "Hide Annotation";
			};
			return 0;
		}
	</script>

	<script>
		var Tree = new Array;
	// nodeId | parentNodeId | nodeName | nodeUrl
< {jtree} >
	</script>

	<script>
			$(function () {
				//纵向,默认,移动间隔2
				$('div.albumSlider').albumSlider();
				//横向设置
				$('div.albumSlider-h').albumSlider({ direction: 'h', step: 3 });
			});
	</script>


	<!-- 20200314 -->
	<script type="text/javascript">
		$(function () {
			var options = {
				selectors: "h2,h3,h4,h5",
				theme: "bootstrap",
				scrollTo: "60"
			};
			var toc = $("#toc").tocify(options).data("toc-tocify");
		});
	</script>

	<!------------------------------------>



</head>

<body id="body">

	<!------------------------------------顶部图片logo---------------------------------------->
	<div id="toc" class="tocify">
	</div>


	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">

		<table width="100%" height="60px" border="0" cellpadding="5" cellspacing="0"
			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
            
			<tr height="0%">

				<td width='0%'></td>

				<td><img height="60px" src="src/images/logo_new.png"></td>

				<td width="0%"></td>

				<td width="40%"></td>

				<td width="20%" style="text-align:right;padding:0px;margin:0px;color:black;font-size:8px;font-weight:50;">

					<a>
				</td>

				<td width="1%"></td>

			</tr>
            
		</table>
	</div>



	<!-- 不可删，删后是满页，不靠右  -->
	<div id="main">
		<!---------------------------------------menu overdd-------------------------------------------->


		<!---------------------------------------- 报告标题 ---------------------------------------------->


		<div id="page">
			<p><br /><br />

		
				<h1> <font size="6"> Human WES Standard Analysis Report </font></h1>
		



			<span style="display: block;text-align: center;font-size: 4em;font-family: Arial;color: #003f76;"></span>
			<br /><br /></p>
			<table class="gy">
			
				<tr>
					
						<td>Contract ID</td>
					
						<td>H20XXXXX</td>
					
				</tr>
			
				<tr>
					
						<td>Contract Name</td>
					
						<td>TESTXXXXXXXXXX</td>
					
				</tr>
			
				<tr>
					
						<td>Batch ID</td>
					
						<td>X202XXXXXX-Z01-F001</td>
					
				</tr>
			
				<tr>
					
						<td>Reference Genome and Version</td>
					
						<td>hg38</td>
					
				</tr>
			
				<tr>
					
						<td>Report Time</td>
					
						<td>20250106</td>
					
				</tr>
			
				<tr>
					
						<td>Reminder</td>
					
						<td>Partial results are presented in this report, while full results will be delivered in data release.Hyperlink of results in this report will be only valid in data release, after statement confirmation.</td>
					
				</tr>
			
				<br /><br /></p>
			</table>
		</div>


		<div id="page">
			<br />
			<h2>1 Introduction</h2>
			<h3>1.1 Background</h3>
			
			<p class="paragraph">Exome sequencing provides a cost-effective alternative to whole genome sequencing as it
				targets only coding region of the human genome responsible for a majority of known diseases and their
				related variants. Whether you are conducting studies in rare Mendelian disorders, complex diseases, cancer
				research, or human population studies, our company's comprehensive human whole exome sequencing provides a
				high-quality, affordable and convenient solution.</p>
			<p class="paragraph">Our company's bioinformatics analysis includes data QC, mapping with reference genome,
				SNP/InDel, somatic SNP/InDel/CNV Calling, statistics and annotation. Our company utilizes prevalent software in
				bioinformatics analysis, e.g. BWA, Sambamba, GATK, etc.</p>
			<p class="paragraph">In particular, the bioinformatics pipeline applies the exome aggregation consortium
				(ExAC). ExAC dataset spans 60,706 unrelated individuals representing various specific diseases, for
				tremendous genetic studies across human population. The scalable database greatly facilitates the research
				of disease pathogenesis.</p>
			<br />
			<p class="paragraph">Workflow is shown as follows:</p>
			<p class="center"> <img class="middle" src="src/images/hWES_Project_Workflow.png" /> </p>
			<br>
			
			<p class="name_fig">Figure 1.1 Project workflow </p>
			
			

			
			<br>	



		<!----------------------------------------样品信息----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Sample information"></a>
				<hr />
			</p><br />
			<h3>1.2 Sample Information</h3>
			<p class="center">
			<p class="name_fig">Table 1 Sample information</p>
			<div id="tb">
				<table class="gy">
				
					
					<tr>
						
							<th>PatientID</th>
						
							<th>SampleID</th>
						
							<th>ID</th>
						
							<th>Type</th>
						</tr>
					
				
					
					<tr>
						
							<td>P2</td>
						
							<td>P2_N</td>
						
							<td>FKDO230288860-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P3</td>
						
							<td>P3_N</td>
						
							<td>FKDO230288864-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P4</td>
						
							<td>P4_T</td>
						
							<td>FKDO230288867-1A</td>
						
							<td>T</td>
						</tr>
					
				
					
					<tr>
						
							<td>P3</td>
						
							<td>P3_T</td>
						
							<td>FKDO230288863-1A</td>
						
							<td>T</td>
						</tr>
					
				
					
					<tr>
						
							<td>P4</td>
						
							<td>P4_N</td>
						
							<td>FKDO230288868-1A</td>
						
							<td>N</td>
						</tr>
					
				
					
					<tr>
						
							<td>P2</td>
						
							<td>P2_T</td>
						
							<td>FKDO230288859-1A</td>
						
							<td>T</td>
						</tr>
					
				
				</table>
			</div>
			</p>
		

			<p class="tremark" align="left">Type: sample type (N: normal; T: tumor; P: patient; U: unknown)</p>

			
			<p class="paragraph">We performed cluster analysis among the samples based on the detected SNPs. The result can help to determine whether two paired samples were from the same patient. </p>
			<p class="center">
			<img height="250px" src="src/pictures/SNP/Samples.cluster.png" />
			<p class="name_fig">Figure 1.2 Cluster analysis among the samples</p>
			<p class="premark">&nbsp</p>
			</p>
			
		</div>
	</div>



		<!----------------------------------------实验流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Experimental procedure"></a>
				<hr />
			</p><br />
			<h2>2 Library Construction and Sequencing</h2>
			<a name="样品检测"></a>
			<br />
			<h3>2.1 Sample Quality Control</h3>
			<p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;Methods of sample quality control refer to QC report.</p>
			<br />

			<a name="建库捕获"></a>

			<h3>2.2 Library Construction, Quality Control and Sequencing</h3>



			
			<p class="paragraph">The genomic DNA was randomly sheared into short fragments with the size of 180-280 bp. The
				obtained fragments were end repaired, A-tailed, and further ligated with Illumina adapters. The fragments
				with adapters were PCR amplified, size selected, and purified. Hybridization capture of libraries was
				proceeded according to the following procedures. Briefly, the prepped libraries were hybridized in the
				buffer with biotin-labeled probes, and magnetic beads with streptavidin were used to capture the exons of
				genes. Subsequently, non-hybridized fragments were washed out and probes were digested. The captured
				libraries were enriched by PCR amplification.</p>
			<p class="paragraph">The experimental procedures of DNA library preparation are shown in Figure 2.</p>
			<br>
			<p class="center">
				<img style="width: 200px;" src="src/images/WES_library_construction.png" />
			</p>
			<p class="name_fig">Figure 2 Workflow of library construction</p>
			<p class="paragraph">The library was checked with Qubit and real-time PCR for quantification and bioanalyzer
				for size distribution detection. Quantified libraries were pooled and sequenced on Illumina platforms
				with PE150 strategy, according to the effective library concentration and data amount required.</p>
			

			

			

			


			
			<p class="paragraph">Method details involved in the project are available at <a	href="src/method/WES_method.pdf" title="Click to open" target="_blank">method.</a></p>
			



			<a name="上机测序"></a>
		</div>

		<!----------------------------------------生物信息分析流程----------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<br />
				<a name="Bioinformatics analysis pipeline"></a>
				<hr />
			</p><br />

			<h2>3 Bioinformatics Analysis Pipeline</h2>

			<p class="paragraph">Bioinformatics analysis includes:</p>
			<p class="paragraph">1) Data quality control: filtering out reads containing adapters or with low quality;</p>
			<p class="paragraph">2) Alignment with reference genome, statistics of sequencing depth and coverage;</p>

			
			<p class="paragraph">3) SNP/InDel calling, annotation and statistics;</p>
			<p class="paragraph">4) Somatic SNP/InDel/CNV calling, annotation and statistics (tumor-normal paired
				samples).</p>
			
			
			

			
			<p class="paragraph">The flowchart below depicts the pipeline of data QC and bioinformatics analysis we
				used. Somatic analysis will be performed only when tumor-normal paired samples are provided. </p>
			<p class="center">
				<img style="width: 500px;" src="src/images/primary_workflow_for_WES_demo.png" />
			</p>
			
			
			



			<p class="name_fig">Figure 3 Bioinformatics analysis pipeline</p>

		
			
			<p class="paragraph">Method details involved in the project are available at <a href="Method/WES_method.pdf"
				title="Click to open" target="_blank">method.</a></p>
			

		</div>
		<!----------------------------------------------- 原始序列数据 --------------------------------------->
		<div id="page">
			<p class="head"><a href="#home" title="返回首页"></a>
				<a name="Analysis Result"></a>
				<hr />
			</p><br />

			<h2>4 Analysis Result</h2>
			<a name="Raw data"></a>
			<br />
			<h3>4.1 Raw Data</h3>

			<p class="paragraph">The original fluorescence image files obtained from sequencing platform are transformed to
				short reads (Raw data) by base calling. These short reads are recorded in FASTQ (.fq) format files, which
				contain sequence information (reads) and corresponding base quality.</p>

			<p class="paragraph">Each read in FASTQ format is stored in four lines as follows: </p>
			<p class="seq">
				
				@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG <br />
				GCTCTTTGCCCTTCTCGTCGAAAATTGTCTCCTCATTCGAAACTTCTCTGT <br />
				+ <br />
				@@CFFFDEHHHHFIJJJ@FHGIIIEHIIJBHHHIJJEGIIJJIGHIGHCCF </p>
				<p class="paragraph">Line 1 begins with a '@' character which is followed by a sequence identifier and an
				optional description. </p>
				<p class="paragraph">Line 2 shows the sequenced bases. </p>
				<p class="paragraph">Line 3 begins with a '+' character and is optionally followed by the same sequence
				identifier. </p>
				<p class="paragraph">Line 4 encodes the sequencing quality for each base in line 2, and contains the same number
				of characters as bases in line 2. </p>
				<p class="paragraph">Illumina sequence identifier details: </p>
				<div id="tb">
					<p class="center">
					<table class="gy">
						<tr>
							<td>EAS139</td>
							<td>Unique instrument name</td>
						</tr>
						<tr>
							<td>136</td>
							<td>Run ID</td>
						</tr>
						<tr>
							<td>FC706VJ</td>
							<td>Flowcell ID</td>
						</tr>
						<tr>
							<td>2</td>
							<td>Flowcell lane</td>
						</tr>
						<tr>
							<td>2104</td>
							<td>Tile number within the flowcell lane</td>
						</tr>
						<tr>
							<td>15343</td>
							<td>'x'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>197393</td>
							<td>'y'-coordinate of the cluster within the tile</td>
						</tr>
						<tr>
							<td>1</td>
							<td>Member of a pair, 1 or 2 (paired-end or mate-pair reads only)</td>
						</tr>
						<tr>
							<td>Y</td>
							<td>Y if the read fails filter (read is bad), otherwise N</td>
						</tr>
						<tr>
							<td>18</td>
							<td>0 when none of the control bits are on, Otherwise it is an even number</td>
						</tr>
						<tr>
							<td>ATCACG</td>
							<td>Index sequence</td>
						</tr>
					</table>
					</p>
				</div>
				

			<p class="paragraph">The ASCII value of each character at the fourth line minus 33 equals to the Phred-scaled
				quality value of the corresponding sequenced base in the second line. The relationship between sequencing
				error rate (e) and base quality value (Qphred) can be expressed by the following equation: </p>
			<p class="center">Qphred = -10log10(e)</p>
			<p class="paragraph">The table below shows examples of corresponding values among sequencing error rate (e),
				base quality value (Qphred) and characters. </p>
			<div id="tb">
				<p class="center">
				<table class="gy">
					<tr>
						<th>Sequencing error rate</th>
						<th>Sequencing quality value</th>
						<th>Corresponding character</th>
					</tr>
					<tr>
						<td>5%</td>
						<td>13</td>
						<td>.</td>
					</tr>
					<tr>
						<td>1%</td>
						<td>20</td>
						<td>5</td>
					</tr>
					<tr>
						<td>0.1%</td>
						<td>30</td>
						<td>?</td>
					</tr>
					<tr>
						<td>0.01%</td>
						<td>40</td>
						<td>I</td>
					</tr>
				</table>
				</p>
			</div>

			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Quality control"></a>
					<hr />
				</p><br />
				<h3>4.2 Quality Control</h3>
				<a name="Sequencing data filtration"></a>
				<br />
				<h4>4.2.1 Sequencing Data Filtration</h4>
				<p class="paragraph">Sequencing artifacts, including reads containing adapter contamination, low-quality
					nucleotides and unrecognizable nucleotides (N), apparently brings ambiguity for the subsequent
					bioinformatics analysis. Hence quality control is an essential step and applied to guarantee the
					meaningful analysis. All of downstream analysis will be based on clean reads that pass quality control.
				</p>
				<p class="paragraph">Our company performed quality control according to following procedures: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1) Discard paired-end reads if either one
					read contains adapter contamination (>10 nucleotides aligned to the adapter, allowing ≤ 10% mismatches);
					<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2) Discard paired-end reads if more than 10% of bases are
					uncertain (read as N) in either one read; <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3) Discard paired-end reads if the proportion of
					low-quality (Phred quality &lt; 5) bases is over 50% in either one read. </p>
				<p class="paragraph">DNA-Seq Adapter (Adapter, Oligonucleotide sequences for DNA Sample Prep Kits)
					information: </p>
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCT<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P7 adapter: <br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;P5→P7’(5’→3’)<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_class_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_class_nav-tabContent">
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3.pie3d.png" /></div>
							
							<div class="tab-pane fade" id="gallery_figure_class_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_class_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Pie/P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3.pie3d.png" /></div>
							
						</div>

					</div>

					<div class="col-1">
						<div id="gallery_figure_class_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_class_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_class_list-tab" role="tablist">

								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3">P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3">P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3">P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3">P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3">P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3</a>
								
							<a class="list-group-item list-group-item-action"
								id="gallery_figure_class_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_class_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3">P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3</a>
							
							


						</div>
					</div>
				</div>
			   </p>


				<p class="name_fig">Figure 4.1 Raw data filtration result</p>
				<br />

				</script>
				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
					<p class="tremark">
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Clean Reads: read pairs that passed quality
						control; <br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Containing N: read pairs in either one read of
						which more than 10% of bases are uncertain;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Low Quality: read pairs in either one read of
						which the proportion of low-quality bases is over 50%;<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Adapter Related: read pairs that contain adapter
						contamination in either one read.</p>
				</div>
				<br />
				<br />

				<a name="Sequencing error rate examination"></a>
				<br />


				<h4>4.2.2 Examination of Sequencing Error Rate </h4>
				<p class="paragraph">Sequencing error rate and base quality can be affected by various factors such as
					sequencing platform, chemical reagent and sample quality. Due to the consumption of chemical reagents,
					error rate is increasing with the read extension, which is a common feature of the high throughput
					sequencing platform.</p>

				<p class="center">
				<div class="row gallery-row">
					<div class="col-1">
						<div id="gallery_figure_error_button_prev" class="button_prev" onclick="prev(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_prev">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-6">
						<div class="tab-content" id="gallery_figure_error_nav-tabContent">

							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3.Error.png" /></div>
							

							<div class="tab-pane fade" id="gallery_figure_error_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
								role="tabpanel"
								aria-labelledby="gallery_figure_error_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
								onclick="trans(this.id)"><img class="img-gallery"
									src="src/pictures/Error/P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3.Error.png" /></div>
							


						</div>
					</div>
					<div class="col-1">
						<div id="gallery_figure_error_button_next" class="button_next" onclick="next(this.id)"><svg
								viewbox="0 0 16000 16000" class="svg_next">
								<circle class="c" cx="8000" cy="8000" r="6080"></circle>
								<polygon class="a"
									points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
								</polygon>
							</svg></div>
					</div>
					<div class="col-4">
						<input class="form-control" id="gallery_figure_error_InputList" type="text" placeholder="Search..">
						<div class="list-group" id="gallery_figure_error_list-tab" role="tablist">
							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3">P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3">P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3">P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3">P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3" role="tab"
								aria-controls="P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3">P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3</a>

							

							<a class="list-group-item list-group-item-action"
								id="gallery_figure_error_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3" data-toggle="list"
								href="#gallery_figure_error_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3" role="tab"
								aria-controls="P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3">P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3</a>

							

						</div>
					</div>
				</div>
			   </p>

				<p class="name_fig">Figure 4.2 Sequencing error rate distribution</p>
				<p class="premark">The x-axis represents position in reads, and the y-axis represents the average error rate
					of bases of all reads at a position. </p>


				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="GC content distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.3 GC Content Distribution</h4>

					<p class="paragraph">Evaluation of GC content distribution aims to check the potential of AT/GC
						separation phenomenon, which maybe resulted from sample contamination, sequencing bias or errors in
						library preparation. </p>
					<p class="paragraph">In theory, GC or AT content should be constant along read positions. But it is
						common to see that the first 6 to 7 bases in both reads (read1 & read2) fluctuate in GC content, due
						to primer amplification bias and some other reasons. </p>

					<p class="center">
					<div class="row gallery-row">
						<div class="col-1">
							<div id="gallery_figure_gc_button_prev" class="button_prev" onclick="prev(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_prev">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-6">
							<div class="tab-content" id="gallery_figure_gc_nav-tabContent">

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3.GC.png" /></div>

								

								<div class="tab-pane fade"
									id="gallery_figure_gc_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
									role="tabpanel"
									aria-labelledby="gallery_figure_gc_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
									onclick="trans(this.id)"><img class="img-gallery"
										src="src/pictures/GC/P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3.GC.png" /></div>

								


							</div>
						</div>
						<div class="col-1">
							<div id="gallery_figure_gc_button_next" class="button_next" onclick="next(this.id)"><svg
									viewbox="0 0 16000 16000" class="svg_next">
									<circle class="c" cx="8000" cy="8000" r="6080"></circle>
									<polygon class="a"
										points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
									</polygon>
								</svg></div>
						</div>
						<div class="col-4">
							<input class="form-control" id="gallery_figure_gc_InputList" type="text" placeholder="Search..">
							<div class="list-group" id="gallery_figure_gc_list-tab" role="tablist">

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
									role="tab"
									aria-controls="P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3">P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
									role="tab"
									aria-controls="P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3">P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
									role="tab"
									aria-controls="P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3">P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
									role="tab"
									aria-controls="P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3">P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
									role="tab"
									aria-controls="P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3">P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3</a>

								

								<a class="list-group-item list-group-item-action"
									id="gallery_figure_gc_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
									data-toggle="list"
									href="#gallery_figure_gc_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
									role="tab"
									aria-controls="P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3">P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3</a>

								

							</div>
						</div>
					</div>
				   </p>
					<p class="name_fig">Figure 4.3 GC content distribution</p>

					<p class="premark">The x-axis represents the position in reads, and the y-axis represents the percentage
						of each type of bases (A, T, G, C); different bases can be distinguished by different colors.</p>
				</div>

				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequencing quality distribution"></a>
						<hr />
					</p><br />

					<h4>4.2.4 Sequencing Quality Distribution</h4>

					<p class="paragraph">The Phred-scaled quality scores of most bases should be greater than 20, it is also
						required for downstream analysis. Base quality decreases along reads is common, which is an inherent
						circumstance of next generation sequencing. </p>
					<p class="center">
					

						<div class="row gallery-row">
							<div class="col-1">
								<div id="gallery_figure_qm_button_prev" class="button_prev" onclick="prev(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_prev">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-6">
								<div class="tab-content" id="gallery_figure_qm_nav-tabContent">
	
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3.QM.png"></div>
									
									
	
									<div class="tab-pane fade"
										id="gallery_figure_qm_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
										role="tabpanel"
										aria-labelledby="gallery_figure_qm_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
										onclick="trans(this.id)"><img class="img-gallery"
											src="src/pictures/QM/P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3.QM.png"></div>
									
									
									
								</div>
							</div>
							<div class="col-1">
								<div id="gallery_figure_qm_button_next" class="button_next" onclick="next(this.id)"><svg
										viewbox="0 0 16000 16000" class="svg_next">
										<circle class="c" cx="8000" cy="8000" r="6080"></circle>
										<polygon class="a"
											points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
										</polygon>
									</svg></div>
							</div>
							<div class="col-4">
								<input class="form-control" id="gallery_figure_qm_InputList" type="text" placeholder="Search..">
								<div class="list-group" id="gallery_figure_qm_list-tab" role="tablist">
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3"
										role="tab"
										aria-controls="P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3">P2_N_FKDO230288860-1A_BHV35HDSX5-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3"
										role="tab"
										aria-controls="P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3">P3_N_FKDO230288864-1A_AHV5J2DSX5-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3"
										role="tab"
										aria-controls="P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3">P4_T_FKDO230288867-1A_AHV5J2DSX5-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3"
										role="tab"
										aria-controls="P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3">P3_T_FKDO230288863-1A_BHV35HDSX5-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3"
										role="tab"
										aria-controls="P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3">P4_N_FKDO230288868-1A_BHV35HDSX5-new_L3</a>
	
									
	
									<a class="list-group-item list-group-item-action"
										id="gallery_figure_qm_list_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
										data-toggle="list"
										href="#gallery_figure_qm_pane_P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3"
										role="tab"
										aria-controls="P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3">P2_T_FKDO230288859-1A_AHV5J2DSX5-new_L3</a>
	
									
	
						
								</div>
							</div>
						</div>
					   </p>	
				</div>

				<p class="name_fig">Figure 4.4 Sequencing quality distribution</p>
				<p class="premark">The x-axis represents the position in reads, and the y-axis represents the average
					quality score of bases of all reads at a position.</p>

				<p class="premark">&nbsp</p>
				</p>


				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Statistics of sequencing quality"></a>
						<hr />
					</p><br />
					<h4>4.2.5 Statistics of Sequencing Quality</h4>

					<P class="paragraph">According to the sequencing feature of sequencing platforms, our company requires the
						value of Q30 (the percentage of bases with Phred-scaled quality scores is greater than 30) should be
						above 85% for paired-end sequencing data.</p>

					<p class="name_fig">Table 4.1 Overview of data production quality</p>
					<div id="tb">
						<table class="gy">
							
							<tr>
								<th>Sample name</th>
								<th>NovoID</th>
								<th>Flowcell/Lane</th>
								<th>Raw reads</th>
								<th>Raw data(G)</th>
								<th>Effective(%)</th>
								<th>Error(%)</th>
								<th>Q20(%)</th>
								<th>Q30(%)</th>
								<th>GC(%)</th>
							</tr>
							

						
							
								
									<tr>
										<td style="text-align:left">P2_N</td>
										<td>FKDO230288860-1A</td>
										<td>BHV35HDSX5-new_L3</td>
										<td>33398791</td>
										<td rowspan=1>10.0</td>
										<td>98.43</td>
										<td>0.02</td>
										<td>98.36</td>
										<td>95.31</td>
										<td>52.80</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">P3_N</td>
										<td>FKDO230288864-1A</td>
										<td>AHV5J2DSX5-new_L3</td>
										<td>42861712</td>
										<td rowspan=1>12.9</td>
										<td>98.49</td>
										<td>0.03</td>
										<td>97.48</td>
										<td>93.22</td>
										<td>52.18</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">P4_T</td>
										<td>FKDO230288867-1A</td>
										<td>AHV5J2DSX5-new_L3</td>
										<td>33383218</td>
										<td rowspan=1>10.0</td>
										<td>98.77</td>
										<td>0.02</td>
										<td>98.03</td>
										<td>94.39</td>
										<td>52.97</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">P3_T</td>
										<td>FKDO230288863-1A</td>
										<td>BHV35HDSX5-new_L3</td>
										<td>34602676</td>
										<td rowspan=1>10.4</td>
										<td>98.62</td>
										<td>0.02</td>
										<td>98.47</td>
										<td>95.56</td>
										<td>51.63</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">P4_N</td>
										<td>FKDO230288868-1A</td>
										<td>BHV35HDSX5-new_L3</td>
										<td>35509415</td>
										<td rowspan=1>10.7</td>
										<td>98.53</td>
										<td>0.02</td>
										<td>98.53</td>
										<td>95.68</td>
										<td>51.67</td>
									</tr>
								
							
						
							
								
									<tr>
										<td style="text-align:left">P2_T</td>
										<td>FKDO230288859-1A</td>
										<td>AHV5J2DSX5-new_L3</td>
										<td>31898098</td>
										<td rowspan=1>9.6</td>
										<td>98.61</td>
										<td>0.03</td>
										<td>97.53</td>
										<td>93.28</td>
										<td>52.66</td>
									</tr>
								
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">

							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) NovoID: NovoID for the sample.<br />
							
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Flowcell/Lane: The flowcell ID and lane
							number.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Raw reads: The number of sequencing reads
							pairs; four lines would be considered as one unit according to the format of FASTQ.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Raw data: The original sequencing
							data.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) Effective: The percentage of clean reads
							in all raw reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) Error: The average error rate of all
							bases on read1 and read2; the error rate of a base is obtained from equation 1.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) Q20: The percent of bases with
							phred-scaled quality scores greater than 20.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) Q30: The percent of bases with
							phred-scaled quality scores greater than 30.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) GC content: The percentage of G and C in
							the all bases.<br /></p>
					</div>
				
				</div>
				<!--
	</div>
	</div>
	-->


				<!----------------------------------------- 测序深度,覆盖度统计 -------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Sequence alignment"></a>
						<hr />
					</p><br />
					<h3>4.3 Sequence Alignment</h3>
					<P class="paragraph">Burrows-Wheeler Aligner (BWA) (Li <I><I>et al.</I></I>, 2018) was utilized to map the paired-end
						clean reads to the human reference genome. The original mapping result can be obtained in BAM format.
						Sambamba (Tarasov <I>et al.</I>, 2015) is used for sorting the BAM files, and Picard (The Quick Start is
						available online: https://broadinstitute.github.io/picard/) is utilized to mark duplicate reads.</p>
					<P class="paragraph">Final BAM files can be obtained after these steps. Then the coverage and depth will
						be computed based on BAM.</p>
					<br />
					<h4>4.3.1 Sequencing Depth & Coverage Distribution</h4>
					<a name="测序深度,覆盖度分布"></a>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/depth.png" />
					<p class="name_fig">Figure 4.5 Sequencing depth</p>
					<!-- <p class="name_fig">Figure 3.1 Sequencing depth</p>-->
					<p class="premark">The left figure shows sequencing depth distribution of all bases in each sample. The
						x-axis represents the sequencing depth, and the y-axis represents the fraction of bases with the given
						sequencing depth. <br />
						The right figure shows accumulative sequencing depth distribution of all bases in each sample. The
						x-axis represents the accumulative sequencing depth, and the y-axis represents the fraction of bases
						above the given sequencing depth.
					</p>
					<p class="center">
						<img class="middle" src="src/pictures/Mapping/covByChr0.png" />

					<p class="name_fig">Figure 4.6 Average sequencing depth (bar plot) and coverage (dot-line plot) in each
						chromosome</p>
					<!--<p class="name_fig">Figure 3.2　Average sequencing depth (bar plot) and coverage (dot-line plot) in each chromosome</p>-->
					<p class="premark">The x-axis represents the chromosome; the left y-axis represents the average depth;
						the right y-axis represents the coverage (proportion of covered bases). </p>
					<p class="premark">&nbsp</p>
				</div>

				<div id="page">
					<h4>4.3.2 Statistics of Mapping, Coverage and Depth</h4>
					<a name="覆盖度统计结果"></a>
					<p class="center">
					<p class="name_fig">Table 4.2 Statistics of mapping, coverage and depth in each sample</p>
					<!--<p class="name_fig">Table 3　Statistics of mapping, coverage and depth in each sample</p>-->
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
									
									<th>Sample name</th>
									
								
									
									<th>P2_N</th>
									
								
									
									<th>P3_N</th>
									
								
									
									<th>P4_T</th>
									
								
									
									<th>P3_T</th>
									
								
									
									<th>P4_N</th>
									
								
									
									<th>P2_T</th>
									
								
							</tr>
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Total</td>
										
									
										
										<td>65745584 (100%)</td>
										
									
										
										<td>84429494 (100%)</td>
										
									
										
										<td>65944442 (100%)</td>
										
									
										
										<td>68247462 (100%)</td>
										
									
										
										<td>69976942 (100%)</td>
										
									
										
										<td>62910928 (100%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Duplicate</td>
										
									
										
										<td>23239670 (35.37%)</td>
										
									
										
										<td>17926252 (21.25%)</td>
										
									
										
										<td>13604562 (20.64%)</td>
										
									
										
										<td>25261566 (37.04%)</td>
										
									
										
										<td>24441364 (34.95%)</td>
										
									
										
										<td>12435327 (19.79%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Mapped</td>
										
									
										
										<td>65697155 (99.93%)</td>
										
									
										
										<td>84354286 (99.91%)</td>
										
									
										
										<td>65904593 (99.94%)</td>
										
									
										
										<td>68205373 (99.94%)</td>
										
									
										
										<td>69938639 (99.95%)</td>
										
									
										
										<td>62850272 (99.90%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Properly mapped</td>
										
									
										
										<td>65364828 (99.42%)</td>
										
									
										
										<td>83831112 (99.29%)</td>
										
									
										
										<td>65558622 (99.41%)</td>
										
									
										
										<td>67775654 (99.31%)</td>
										
									
										
										<td>69538836 (99.37%)</td>
										
									
										
										<td>62510136 (99.36%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">PE mapped</td>
										
									
										
										<td>65658432 (99.87%)</td>
										
									
										
										<td>84291048 (99.84%)</td>
										
									
										
										<td>65871528 (99.89%)</td>
										
									
										
										<td>68170004 (99.89%)</td>
										
									
										
										<td>69907894 (99.90%)</td>
										
									
										
										<td>62797152 (99.82%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">SE mapped</td>
										
									
										
										<td>77446 (0.12%)</td>
										
									
										
										<td>126476 (0.15%)</td>
										
									
										
										<td>66130 (0.10%)</td>
										
									
										
										<td>70738 (0.10%)</td>
										
									
										
										<td>61490 (0.09%)</td>
										
									
										
										<td>106240 (0.17%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr</td>
										
									
										
										<td>203774 (0.31%)</td>
										
									
										
										<td>311286 (0.37%)</td>
										
									
										
										<td>204952 (0.31%)</td>
										
									
										
										<td>270672 (0.40%)</td>
										
									
										
										<td>262570 (0.38%)</td>
										
									
										
										<td>196994 (0.31%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">With mate mapped to a different chr ((mapQ>=5))</td>
										
									
										
										<td>163228 (0.25%)</td>
										
									
										
										<td>262704 (0.31%)</td>
										
									
										
										<td>169139 (0.26%)</td>
										
									
										
										<td>217000 (0.32%)</td>
										
									
										
										<td>216163 (0.31%)</td>
										
									
										
										<td>157239 (0.25%)</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Initial_bases_on_target</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
										
										<td>60507855</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Initial_bases_on_or_near_target</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
										
										<td>136466827</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Total_effective_yield(Mb)</td>
										
									
										
										<td>9816.43</td>
										
									
										
										<td>12602.88</td>
										
									
										
										<td>9850.42</td>
										
									
										
										<td>10193.05</td>
										
									
										
										<td>10453.05</td>
										
									
										
										<td>9389.86</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Effective_yield_on_target(Mb)</td>
										
									
										
										<td>7061.65</td>
										
									
										
										<td>9020.20</td>
										
									
										
										<td>7001.25</td>
										
									
										
										<td>7190.65</td>
										
									
										
										<td>7356.97</td>
										
									
										
										<td>6699.40</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_effective_bases_on_target</td>
										
									
										
										<td>71.94%</td>
										
									
										
										<td>71.57%</td>
										
									
										
										<td>71.08%</td>
										
									
										
										<td>70.54%</td>
										
									
										
										<td>70.38%</td>
										
									
										
										<td>71.35%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_effective_bases_on_or_near_target</td>
										
									
										
										<td>91.65%</td>
										
									
										
										<td>91.60%</td>
										
									
										
										<td>91.54%</td>
										
									
										
										<td>90.87%</td>
										
									
										
										<td>90.33%</td>
										
									
										
										<td>91.84%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Average_sequencing_depth_on_target</td>
										
									
										
										<td>116.71</td>
										
									
										
										<td>149.07</td>
										
									
										
										<td>115.71</td>
										
									
										
										<td>118.84</td>
										
									
										
										<td>121.59</td>
										
									
										
										<td>110.72</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Bases_covered_on_target</td>
										
									
										
										<td>60075376</td>
										
									
										
										<td>60337580</td>
										
									
										
										<td>60120802</td>
										
									
										
										<td>60234928</td>
										
									
										
										<td>60116091</td>
										
									
										
										<td>60138501</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Coverage_of_target_region</td>
										
									
										
										<td>99.29%</td>
										
									
										
										<td>99.72%</td>
										
									
										
										<td>99.36%</td>
										
									
										
										<td>99.55%</td>
										
									
										
										<td>99.35%</td>
										
									
										
										<td>99.39%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_100x</td>
										
									
										
										<td>43.96%</td>
										
									
										
										<td>57.25%</td>
										
									
										
										<td>43.39%</td>
										
									
										
										<td>46.48%</td>
										
									
										
										<td>48.08%</td>
										
									
										
										<td>42.42%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_50x</td>
										
									
										
										<td>72.22%</td>
										
									
										
										<td>81.45%</td>
										
									
										
										<td>71.84%</td>
										
									
										
										<td>74.63%</td>
										
									
										
										<td>76.02%</td>
										
									
										
										<td>72.47%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_20x</td>
										
									
										
										<td>91.01%</td>
										
									
										
										<td>94.49%</td>
										
									
										
										<td>90.99%</td>
										
									
										
										<td>92.02%</td>
										
									
										
										<td>92.52%</td>
										
									
										
										<td>91.61%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_10x</td>
										
									
										
										<td>96.03%</td>
										
									
										
										<td>97.72%</td>
										
									
										
										<td>96.12%</td>
										
									
										
										<td>96.59%</td>
										
									
										
										<td>96.73%</td>
										
									
										
										<td>96.43%</td>
										
									
									</tr>
								
							
						
							
								
									<tr>
									
										
										<td style="text-align:left">Fraction_of_target_covered_with_at_least_4x</td>
										
									
										
										<td>98.40%</td>
										
									
										
										<td>99.20%</td>
										
									
										
										<td>98.49%</td>
										
									
										
										<td>98.75%</td>
										
									
										
										<td>98.66%</td>
										
									
										
										<td>98.63%</td>
										
									
									</tr>
								
							
						



						</table>
					</div>
					</p>

					
					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of clean reads.
							The below indexes are calculated based on the clean reads.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Duplicate: The number of duplicated reads
							(percentage: duplicated reads/clean reads).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Mapped: The number of reads that mapped
							to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) Properly mapped: The number of reads that
							mapped to the reference genome and within the expected insert size (percentage). <br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) PE mapped: The number of pair-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) SE mapped: The number of single-end reads
							that mapped to the reference genome (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) With mate mapped to a different chr: The
							number of reads with mate reads mapped to different chromosomes (percentage).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) With mate mapped to a different chr (mapQ
							>= 5): The number of reads with mate reads mapped to different chromosomes and the MAQ > 5.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) Initial_bases_on_target: The number of
							bases in the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) Initial_bases_on_or_near_target: The
							number of bases in the target region or flanking region of the target.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) Total_effective_yield (Mb) : The data
							size of the effective reads mapped to the reference genome (MB as a unit).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Effective_yield_on_target (Mb) : The
							data size of the reads mapped to the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(14) Fraction_of_effective_bases_on_target:
							The percentage of bases mapped to the target region in all bases mapped to the reference genome
							<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Effective_sequences_on_target/Total_effective_yield).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(15)
							Fraction_of_effective_bases_on_or_near_target: The percentage of bases mapped to the target or
							flanking region in all bases mapped to the reference genome.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(16) Average_sequencing_depth_on_target: The
							average sequencing depth in the target region (Effective_sequences_on_target * 1
							million/Initial_bases_on_target).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(17) Bases_covered_on_target: The number of
							the bases covered in the target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(18) Coverage_of_target_region: The coverage
							percentage of target regions (Base_covered_on_target/Initial_bases_on_target).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(19)
							Fraction_of_target_covered_with_at_least_100x: The percentage of bases with depth >100X in target
							regions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(20)
							Fraction_of_target_covered_with_at_least_50x: The percentage of bases with depth >50X in target
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(21)
							Fraction_of_target_covered_with_at_least_20x: The percentage of bases with depth > 20X in the
							target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(22)
							Fraction_of_target_covered_with_at_least_10x: The percentage of bases with depth > 10X in the
							target region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(23)
							Fraction_of_target_covered_with_at_least_4x: The percentage of bases with depth > 4X in the target
							region.<br />
						</p>
					</div>
					
					<p class="paragraph"><a href="../../../02.Bam" title="Click to open" target="_blank">Result: 02.Bam</a></p>
				</div>
				
				<!---------------------------------------------- snp/InDel结果 ------------------------------------------------------------->
				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="Variant detection"></a>
						<hr />
					</p><br />
					<h3>4.4 Germline Mutation Detection</h3>
					<br />
					<h4>4.4.1 SNP Detection Result</h4>
					<p class="paragraph"> Single nucleotide polymorphisms (SNPs), also known as single nucleotide variants
						(SNVs), constitute the largest class of genetic variants in genome. A typical human genome has about
						3.6 million SNPs (DePristo <I>et al.</I>, 2011). Statistics of detected SNPs are shown below.</p>
					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.3 The number of SNPs in various genomic regions </p>
						<!--<p class="name_fig">Table 4.1 The number of SNPs in various genomic regions </p>-->
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>P2_N</th>
									
								
									
										<th>P3_N</th>
									
								
									
										<th>P4_T</th>
									
								
									
										<th>P3_T</th>
									
								
									
										<th>P4_N</th>
									
								
									
										<th>P2_T</th>
									
								
							</tr>

							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>22926</td>
										
									
										
											<td>22804</td>
										
									
										
											<td>22917</td>
										
									
										
											<td>22620</td>
										
									
										
											<td>22908</td>
										
									
										
											<td>23077</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;synonymous_SNP</td>
										
									
										
											<td>11701</td>
										
									
										
											<td>11657</td>
										
									
										
											<td>11720</td>
										
									
										
											<td>11541</td>
										
									
										
											<td>11717</td>
										
									
										
											<td>11760</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;missense_SNP</td>
										
									
										
											<td>10811</td>
										
									
										
											<td>10781</td>
										
									
										
											<td>10894</td>
										
									
										
											<td>10704</td>
										
									
										
											<td>10889</td>
										
									
										
											<td>10899</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>91</td>
										
									
										
											<td>90</td>
										
									
										
											<td>93</td>
										
									
										
											<td>95</td>
										
									
										
											<td>92</td>
										
									
										
											<td>96</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>13</td>
										
									
										
											<td>11</td>
										
									
										
											<td>11</td>
										
									
										
											<td>10</td>
										
									
										
											<td>11</td>
										
									
										
											<td>10</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>26</td>
										
									
										
											<td>27</td>
										
									
										
											<td>23</td>
										
									
										
											<td>27</td>
										
									
										
											<td>23</td>
										
									
										
											<td>27</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>270</td>
										
									
										
											<td>228</td>
										
									
										
											<td>163</td>
										
									
										
											<td>236</td>
										
									
										
											<td>167</td>
										
									
										
											<td>276</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>111613</td>
										
									
										
											<td>140098</td>
										
									
										
											<td>124621</td>
										
									
										
											<td>115713</td>
										
									
										
											<td>125027</td>
										
									
										
											<td>119477</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>5820</td>
										
									
										
											<td>6928</td>
										
									
										
											<td>6433</td>
										
									
										
											<td>5872</td>
										
									
										
											<td>6263</td>
										
									
										
											<td>6186</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>3021</td>
										
									
										
											<td>3340</td>
										
									
										
											<td>3296</td>
										
									
										
											<td>2919</td>
										
									
										
											<td>3055</td>
										
									
										
											<td>3222</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>554</td>
										
									
										
											<td>560</td>
										
									
										
											<td>562</td>
										
									
										
											<td>542</td>
										
									
										
											<td>564</td>
										
									
										
											<td>560</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>3516</td>
										
									
										
											<td>4014</td>
										
									
										
											<td>3849</td>
										
									
										
											<td>3543</td>
										
									
										
											<td>3814</td>
										
									
										
											<td>3657</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>11318</td>
										
									
										
											<td>15229</td>
										
									
										
											<td>13014</td>
										
									
										
											<td>11820</td>
										
									
										
											<td>13302</td>
										
									
										
											<td>11949</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>38</td>
										
									
										
											<td>51</td>
										
									
										
											<td>42</td>
										
									
										
											<td>46</td>
										
									
										
											<td>45</td>
										
									
										
											<td>44</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>4001</td>
										
									
										
											<td>5036</td>
										
									
										
											<td>4756</td>
										
									
										
											<td>4047</td>
										
									
										
											<td>4232</td>
										
									
										
											<td>4565</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>2105</td>
										
									
										
											<td>2851</td>
										
									
										
											<td>2489</td>
										
									
										
											<td>2151</td>
										
									
										
											<td>2300</td>
										
									
										
											<td>2316</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>63952</td>
										
									
										
											<td>94006</td>
										
									
										
											<td>74178</td>
										
									
										
											<td>69686</td>
										
									
										
											<td>81013</td>
										
									
										
											<td>67240</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>228597</td>
										
									
										
											<td>294583</td>
										
									
										
											<td>255863</td>
										
									
										
											<td>238705</td>
										
									
										
											<td>262260</td>
										
									
										
											<td>242004</td>
										
									
								</tr>
								
							
						
	
						</table>
						</p>
					</div>


					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of SNPs in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							synonymous_SNP: A single nucleotide change that does not cause an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							missense_SNP: A single nucleotide change that causes an amino acid change.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stopgain: A nonsynonymous SNP that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
							stoploss: A nonsynonymous SNP that leads to the elimination of stop codon at the variant
							site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of SNPs in intronic
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of SNPs in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of SNPs in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of SNPs within 2-bp
							of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of SNPs in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of SNPs in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of SNPs within
							2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of SNPs within 1kb
							away from the transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of SNPs within
							1kb away from the transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of SNPs in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of SNPs.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src='' /></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/SNP/P2_N.snp.png'><img
												src='src/pictures/SNP/P2_N.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/P3_N.snp.png'><img
												src='src/pictures/SNP/P3_N.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/P4_T.snp.png'><img
												src='src/pictures/SNP/P4_T.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/P3_T.snp.png'><img
												src='src/pictures/SNP/P3_T.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/P4_N.snp.png'><img
												src='src/pictures/SNP/P4_N.snp.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/SNP/P2_T.snp.png'><img
												src='src/pictures/SNP/P2_T.snp.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>
					<p class="name_fig">Figure 4.7 Number of SNPs in various genomic regions (left pie plot); number of
						different types of SNPs in coding region (right pie plot)</p>
					<p class="premark">&nbsp</p>

					<p class="center">
					<p class="name_fig">Table 4.4 Feature of SNPs</p>
					<div id="tb">
						<table class="gy">
						
							
							<tr>
								
											
										<th>Sample name</th>
									
								
									
										<th>P2_N</th>
									
								
									
										<th>P3_N</th>
									
								
									
										<th>P4_T</th>
									
								
									
										<th>P3_T</th>
									
								
									
										<th>P4_N</th>
									
								
									
										<th>P2_T</th>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>228597</td>
									
								
									
										<td>294583</td>
									
								
									
										<td>255863</td>
									
								
									
										<td>238705</td>
									
								
									
										<td>262260</td>
									
								
									
										<td>242004</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>74855</td>
									
								
									
										<td>89488</td>
									
								
									
										<td>83353</td>
									
								
									
										<td>72684</td>
									
								
									
										<td>78591</td>
									
								
									
										<td>80904</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>153742</td>
									
								
									
										<td>205095</td>
									
								
									
										<td>172510</td>
									
								
									
										<td>166021</td>
									
								
									
										<td>183669</td>
									
								
									
										<td>161100</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts</td>
									
								
									
										<td>157725</td>
									
								
									
										<td>203091</td>
									
								
									
										<td>176737</td>
									
								
									
										<td>164453</td>
									
								
									
										<td>180347</td>
									
								
									
										<td>166909</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">tv</td>
									
								
									
										<td>70872</td>
									
								
									
										<td>91492</td>
									
								
									
										<td>79126</td>
									
								
									
										<td>74252</td>
									
								
									
										<td>81913</td>
									
								
									
										<td>75095</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">ts/tv</td>
									
								
									
										<td>2.23</td>
									
								
									
										<td>2.22</td>
									
								
									
										<td>2.23</td>
									
								
									
										<td>2.21</td>
									
								
									
										<td>2.20</td>
									
								
									
										<td>2.22</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>215705(94.36%)</td>
									
								
									
										<td>277473(94.19%)</td>
									
								
									
										<td>242403(94.74%)</td>
									
								
									
										<td>225177(94.33%)</td>
									
								
									
										<td>246855(94.13%)</td>
									
								
									
										<td>229322(94.76%)</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>12892</td>
									
								
									
										<td>17110</td>
									
								
									
										<td>13460</td>
									
								
									
										<td>13528</td>
									
								
									
										<td>15405</td>
									
								
									
										<td>12682</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts</td>
									
								
									
										<td>8029</td>
									
								
									
										<td>10662</td>
									
								
									
										<td>8632</td>
									
								
									
										<td>8478</td>
									
								
									
										<td>9617</td>
									
								
									
										<td>7892</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_tv</td>
									
								
									
										<td>4863</td>
									
								
									
										<td>6448</td>
									
								
									
										<td>4828</td>
									
								
									
										<td>5050</td>
									
								
									
										<td>5788</td>
									
								
									
										<td>4790</td>
									
								
							</tr>
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel_ts/tv</td>
									
								
									
										<td>1.65</td>
									
								
									
										<td>1.65</td>
									
								
									
										<td>1.79</td>
									
								
									
										<td>1.68</td>
									
								
									
										<td>1.66</td>
									
								
									
										<td>1.65</td>
									
								
							</tr>
							
						

						</table>
					</div>
					</p>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) transition (ts) : The number of
							transitions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) transversion (tv) : The number of
							transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ts/tv: The number of transitions divided
							by the number of transversions.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) dbSNP percentage: The number of SNPs that
							have been reported in dbSNP database divided by the total number of SNPs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) novel: The number of SNPs that have not been reported in
							dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) novel ts: The number of transition SNPs
							that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) novel tv: The number of transversion
							SNPs that have not been reported in dbSNP database.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) novel ts/tv: Novel ts divided by novel
							tv.</p>
						<p class="tremark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/SNP" title="Click to open" target="_blank">Result: 03.Result_Homo_sapiens/result/Mutation/SNP</a></p>
				</div>

				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="InDel检测结果"></a>
						<hr />
					</p><br />
					<h4>4.4.2 InDel Detection Result</h4>
					<p class="paragraph">Small insertions and deletions (InDels) that are less than 50 bp in length
						constitute another class of genetic variants in human genome. </p>
					<p class="paragraph">The InDels occurred in coding region or splicing sites may cause changes in
						transcripts and proteins. If the number of inserted or deleted nucleotides is not in frame of three or
						multiples of three, the entire reading frame would be altered. The statistics of InDels (DePristo <I>et al.</I>, 2011) called in the
						samples are listed below: </p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.5 The number of InDels in various genomic regions</p>
						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>P2_N</th>
									
								
									
										<th>P3_N</th>
									
								
									
										<th>P4_T</th>
									
								
									
										<th>P3_T</th>
									
								
									
										<th>P4_N</th>
									
								
									
										<th>P2_T</th>
									
								
							</tr>
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">CDS</td>
										
									
										
											<td>672</td>
										
									
										
											<td>694</td>
										
									
										
											<td>690</td>
										
									
										
											<td>651</td>
										
									
										
											<td>687</td>
										
									
										
											<td>673</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift deletion</td>
										
									
										
											<td>132</td>
										
									
										
											<td>140</td>
										
									
										
											<td>140</td>
										
									
										
											<td>140</td>
										
									
										
											<td>136</td>
										
									
										
											<td>136</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">frameshift insertion</td>
										
									
										
											<td>86</td>
										
									
										
											<td>101</td>
										
									
										
											<td>90</td>
										
									
										
											<td>88</td>
										
									
										
											<td>93</td>
										
									
										
											<td>84</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift deletion</td>
										
									
										
											<td>229</td>
										
									
										
											<td>234</td>
										
									
										
											<td>243</td>
										
									
										
											<td>210</td>
										
									
										
											<td>245</td>
										
									
										
											<td>228</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">nonframeshift insertion</td>
										
									
										
											<td>177</td>
										
									
										
											<td>182</td>
										
									
										
											<td>188</td>
										
									
										
											<td>167</td>
										
									
										
											<td>184</td>
										
									
										
											<td>179</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
										
									
										
											<td>17</td>
										
									
										
											<td>16</td>
										
									
										
											<td>15</td>
										
									
										
											<td>15</td>
										
									
										
											<td>15</td>
										
									
										
											<td>18</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
										
									
										
											<td>3</td>
										
									
										
											<td>2</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">startloss</td>
										
									
										
											<td>3</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
										
											<td>2</td>
										
									
										
											<td>3</td>
										
									
										
											<td>3</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
										
									
										
											<td>22</td>
										
									
										
											<td>15</td>
										
									
										
											<td>9</td>
										
									
										
											<td>24</td>
										
									
										
											<td>8</td>
										
									
										
											<td>19</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intronic</td>
										
									
										
											<td>20673</td>
										
									
										
											<td>25814</td>
										
									
										
											<td>23039</td>
										
									
										
											<td>21937</td>
										
									
										
											<td>23405</td>
										
									
										
											<td>22018</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR3</td>
										
									
										
											<td>1140</td>
										
									
										
											<td>1318</td>
										
									
										
											<td>1254</td>
										
									
										
											<td>1154</td>
										
									
										
											<td>1202</td>
										
									
										
											<td>1185</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">UTR5</td>
										
									
										
											<td>462</td>
										
									
										
											<td>577</td>
										
									
										
											<td>535</td>
										
									
										
											<td>456</td>
										
									
										
											<td>513</td>
										
									
										
											<td>497</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">splicing</td>
										
									
										
											<td>211</td>
										
									
										
											<td>246</td>
										
									
										
											<td>254</td>
										
									
										
											<td>226</td>
										
									
										
											<td>254</td>
										
									
										
											<td>223</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_exonic</td>
										
									
										
											<td>417</td>
										
									
										
											<td>467</td>
										
									
										
											<td>456</td>
										
									
										
											<td>383</td>
										
									
										
											<td>442</td>
										
									
										
											<td>452</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_intronic</td>
										
									
										
											<td>1762</td>
										
									
										
											<td>2350</td>
										
									
										
											<td>2026</td>
										
									
										
											<td>1977</td>
										
									
										
											<td>2119</td>
										
									
										
											<td>1796</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">ncRNA_splicing</td>
										
									
										
											<td>11</td>
										
									
										
											<td>8</td>
										
									
										
											<td>6</td>
										
									
										
											<td>5</td>
										
									
										
											<td>7</td>
										
									
										
											<td>13</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">upstream</td>
										
									
										
											<td>648</td>
										
									
										
											<td>826</td>
										
									
										
											<td>784</td>
										
									
										
											<td>632</td>
										
									
										
											<td>672</td>
										
									
										
											<td>710</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">downstream</td>
										
									
										
											<td>347</td>
										
									
										
											<td>413</td>
										
									
										
											<td>408</td>
										
									
										
											<td>340</td>
										
									
										
											<td>387</td>
										
									
										
											<td>386</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">intergenic</td>
										
									
										
											<td>7847</td>
										
									
										
											<td>11326</td>
										
									
										
											<td>8932</td>
										
									
										
											<td>8789</td>
										
									
										
											<td>10449</td>
										
									
										
											<td>7961</td>
										
									
								</tr>
								
							
						
							
								
								<tr>
									
										
											<td style="text-align:left">Total</td>
										
									
										
											<td>34130</td>
										
									
										
											<td>43982</td>
										
									
										
											<td>38335</td>
										
									
										
											<td>36502</td>
										
									
										
											<td>40085</td>
										
									
										
											<td>35854</td>
										
									
								</tr>
								
							
						

						</table>
						</p>
					</div>

					<script type="text/javascript">
						$(function () {
							$('table#Error tr').find('th:eq(1)').hide();
							$('table#Error tr').find('td:eq(1)').hide();
						})
					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of InDels in coding
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_deletion:
							A deletion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_insertion:
							An insertion of one or more nucleotides that cause frameshift changes in protein coding
							sequence.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_deletion:
							A deletion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_insertion:
							An insertion that does not cause frameshift changes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stopgain: An
							insertion or a deletion that leads to the creation of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stoploss: An
							insertion or a deletion that leads to the elimination of stop codon at the variant site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;unknown:
							Unknown function (due to various errors in the gene structure definition in the database
							file).<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of InDels in
							intronic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of InDels in 3'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of InDels in 5'UTR
							region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of InDels within
							2-bp of a splicing junction.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of InDels in
							exonic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of InDels in
							intronic region of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of InDels
							within 2-bp of a splicing junction of non-coding RNAs.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of InDels within
							1kb away from transcription start site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of InDels within
							1kb away from transcription end site.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of InDels in
							intergenic region.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of InDels.</p>
						<p class="premark">&nbsp</p>
					</div>

					<p class="center">
					<div class="albumSlider1 albumSlider">
						<div class="fullview"><img src=/></div>
						<div class="slider">
							<div class="button movebackward" title="向上滚动"></div>
							<div class="imglistwrap">
								<ul class="imglist">
								

									<li><a id="example2" href='src/pictures/INDEL/P2_N.indel.png'><img
												src='src/pictures/INDEL/P2_N.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/P3_N.indel.png'><img
												src='src/pictures/INDEL/P3_N.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/P4_T.indel.png'><img
												src='src/pictures/INDEL/P4_T.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/P3_T.indel.png'><img
												src='src/pictures/INDEL/P3_T.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/P4_N.indel.png'><img
												src='src/pictures/INDEL/P4_N.indel.JPEG' /></a></li>
								
								

									<li><a id="example2" href='src/pictures/INDEL/P2_T.indel.png'><img
												src='src/pictures/INDEL/P2_T.indel.JPEG' /></a></li>
								
								

								</ul>
							</div>
							<div class="button moveforward" title="向下滚动"></div>
						</div>
					</div>

					<p class="name_fig">Figure 4.8 Number of InDels in various genomic regions (left); number ofdifferent
						types of InDels in coding region (right)</p>
					<p class="premark">&nbsp</p>
					</p>

					<div id="tb">
						<p class="center">
						<p class="name_fig">Table 4.6 Feature of InDels in genome</p>

						<table class="gy">
						
							
							<tr>
								
									
										<th>Sample name</th>
									
								
									
										<th>P2_N</th>
									
								
									
										<th>P3_N</th>
									
								
									
										<th>P4_T</th>
									
								
									
										<th>P3_T</th>
									
								
									
										<th>P4_N</th>
									
								
									
										<th>P2_T</th>
									
								
							</tr>

							
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>34130</td>
									
								
									
										<td>43982</td>
									
								
									
										<td>38335</td>
									
								
									
										<td>36502</td>
									
								
									
										<td>40085</td>
									
								
									
										<td>35854</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Het</td>
									
								
									
										<td>13939</td>
									
								
									
										<td>17795</td>
									
								
									
										<td>15973</td>
									
								
									
										<td>14217</td>
									
								
									
										<td>15144</td>
									
								
									
										<td>15090</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">Hom</td>
									
								
									
										<td>20191</td>
									
								
									
										<td>26187</td>
									
								
									
										<td>22362</td>
									
								
									
										<td>22285</td>
									
								
									
										<td>24941</td>
									
								
									
										<td>20764</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">dbSNP percentage</td>
									
								
									
										<td>29079(85.20%)</td>
									
								
									
										<td>37341(84.90%)</td>
									
								
									
										<td>32679(85.25%)</td>
									
								
									
										<td>31012(84.96%)</td>
									
								
									
										<td>34043(84.93%)</td>
									
								
									
										<td>30788(85.87%)</td>
									
								
							</tr>			
							
						
							
							<tr>
								
									
										<td style="text-align:left">novel</td>
									
								
									
										<td>5051</td>
									
								
									
										<td>6641</td>
									
								
									
										<td>5656</td>
									
								
									
										<td>5490</td>
									
								
									
										<td>6042</td>
									
								
									
										<td>5066</td>
									
								
							</tr>			
							
						

						</table>
						</p>
					</div>

					</script>
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="tremark">
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) Total: The total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) Het: The number of heterozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) Hom: The number of homozygotes.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) dbSNP percentage: The number of InDels
							that have been reported in dbSNP database divided by the total number of InDels.<br />
							&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) novel: The number of InDels that have not
							been reported in dbSNP database.<br />
						<p class="premark">&nbsp</p>
					</div>

					<p class="paragraph"><a href="../../result/Mutation/INDEL" title="Click to open" target="_blank">Result: 03.Result_Homo_sapiens/result/Mutation/INDEL</a></p>
				</div>

				

				
			
				
				


				
				<div id="page">
					<p class="head"><a href="#home" title="返回首页"></a>
						<a name="SNP/InDel结果注释"></a>
						<hr />
					</p><br />
					<h4>4.4.3 Variant Annotation </h4>
					<p class="paragraph">Following genomic variant detection, annotation of variants with the tool ANNOVAR (Wang et
						al., 2010) in multiple aspects are performed, including protein coding changes, genomic regions affected by
						the variants, allele frequency, deleteriousness prediction, etc. The main databases used are as
						follows:<br />
					<ul>
						<li style="font-size: 0.6em">RefSeq (O'Leary <I>et al.</I>, 2016) and Gencode (Frankish <I>et al.</I>, 2021) databases
							were used to find genomic regions affected by the variant and possible changes in protein. </li>
						<li style="font-size: 0.6em">We annotated the features of the genomic regions affected by the variants, such
							as cytoband, small RNA, conserved mammalian microRNA regulatory target sites, conservative regions of
							vertebrates, transcription factor binding sites, repeats, etc. </li>
						<li style="font-size: 0.6em">SIFT (Ng <I>et al.</I>, 2003), PolyPhen (Adzhubei <I>et al.</I>, 2013), MutationAssessor
							(Reva <I>et al.</I>, 2011), LRT (Chun <I>et al.</I>, 2009) and CADD (Rentzsch <I>et al.</I>, 2019) scores were used to predict
							the deleteriousness of mutations. GERP++ (Huber <I>et al.</I>, 2020) scores were used to access the conservation
							of mutations.</li>
						<li style="font-size: 0.6em">The established databases, such as 1000 Human Genome (Abecasis <I>et al.</I>, 2012),
							Exome Aggregation Consortium (ExAC) (Kobayashi <I>et al.</I>, 2017), Genome Aggregation Database (gnomAD) (Pio
							<I>et al.</I>, 2021) and exome sequencing project (ESP), were used to find alternative allele frequencies in
							populations that are reported.</li>
						<li style="font-size: 0.6em">Databases dbSNP (Sherry <I>et al.</I>, 2001), COSMIC (Tate <I>et al.</I>, 2019), OMIM (Hamosh
							<I>et al.</I>, 2005), GWAS Catalog (Buniello <I>et al.</I>, 2019) and HGMD (Stenson <I>et al.</I>, 2020) were used to find
							reported information of the variant, such as top SNPs in GWAS and cancer/disease associations. </li>
						<li style="font-size: 0.6em">Databases including Gene Ontology (Lee <I>et al.</I>, 2004), KEGG (Kanehisa <I>et al.</I>,
							2000), Reactome (Jassal <I>et al.</I>, 2020), Biocarta and PID (Schaefer <I>et al.</I>, 2009) were applied to provide
							functional or pathway annotation. </li>
					</ul>
					</p>
		
					<p class="center">
					
					<p class="name_fig">Table 4.7 Example of annotation result</p>
					
					<div id="tb">
						<table class="gy">
							
								
								<tr>
									
										<th>Priority</th>
									
										<th>CHROM</th>
									
										<th>POS</th>
									
										<th>ID</th>
									
										<th>REF</th>
									
										<th>ALT</th>
									
										<th>QUAL</th>
									
										<th>FILTER</th>
									
										<th>GeneName</th>
									
										<th>Description</th>
									
										<th>Func</th>
									
										<th>Gene</th>
									
										<th>GeneDetail</th>
									
										<th>ExonicFunc</th>
									
										<th>AAChange</th>
									
										<th>Gencode</th>
									
										<th>cpgIslandExt</th>
									
										<th>cytoBand</th>
									
										<th>wgRna</th>
									
										<th>genomicSuperDups</th>
									
										<th>Repeat</th>
									
										<th>avsnp</th>
									
										<th>CLNALLELEID</th>
									
										<th>CLNDN</th>
									
										<th>CLNDISDB</th>
									
										<th>CLNREVSTAT</th>
									
										<th>CLNSIG</th>
									
										<th>cosmic</th>
									
										<th>gwasCatalog</th>
									
										<th>1000g_SAS</th>
									
										<th>1000g_EUR</th>
									
										<th>1000g_AFR</th>
									
										<th>1000g_AMR</th>
									
										<th>1000g_EAS</th>
									
										<th>1000g_ALL</th>
									
										<th>esp6500siv2_all</th>
									
										<th>gnomad_exome_AF</th>
									
										<th>gnomad_exome_AF_raw</th>
									
										<th>gnomad_exome_AF_afr</th>
									
										<th>gnomad_exome_AF_sas</th>
									
										<th>gnomad_exome_AF_amr</th>
									
										<th>gnomad_exome_AF_eas</th>
									
										<th>gnomad_exome_AF_nfe</th>
									
										<th>gnomad_exome_AF_fin</th>
									
										<th>gnomad_exome_AF_asj</th>
									
										<th>gnomad_exome_AF_oth</th>
									
										<th>gnomad_genome_AF</th>
									
										<th>gnomad_genome_AF_raw</th>
									
										<th>gnomad_genome_AF_afr</th>
									
										<th>gnomad_genome_AF_sas</th>
									
										<th>gnomad_genome_AF_amr</th>
									
										<th>gnomad_genome_AF_eas</th>
									
										<th>gnomad_genome_AF_nfe</th>
									
										<th>gnomad_genome_AF_fin</th>
									
										<th>gnomad_genome_AF_asj</th>
									
										<th>gnomad_genome_AF_oth</th>
									
										<th>ExAC_ALL</th>
									
										<th>ExAC_AFR</th>
									
										<th>ExAC_AMR</th>
									
										<th>ExAC_EAS</th>
									
										<th>ExAC_FIN</th>
									
										<th>ExAC_NFE</th>
									
										<th>ExAC_OTH</th>
									
										<th>ExAC_SAS</th>
									
										<th>dbscSNV_ADA_SCORE</th>
									
										<th>dbscSNV_RF_SCORE</th>
									
										<th>Interpro_domain</th>
									
										<th>SIFT</th>
									
										<th>Polyphen2_HDIV</th>
									
										<th>Polyphen2_HVAR</th>
									
										<th>LRT</th>
									
										<th>MutationTaster</th>
									
										<th>MutationAssessor</th>
									
										<th>FATHMM</th>
									
										<th>phyloP100way_vertebrate</th>
									
										<th>phyloP30way_mammalian</th>
									
										<th>SiPhy_29way_logOdds</th>
									
										<th>CADD</th>
									
										<th>gerp++gt2</th>
									
										<th>REVEL</th>
									
										<th>INFO</th>
									
										<th>FORMAT</th>
									
										<th>P2_T</th>
									
										<th>Ori_REF</th>
									
										<th>Ori_ALT</th>
									
										<th>shared_hom</th>
									
										<th>shared_het</th>
									
										<th>OMIM</th>
									
										<th>GWAS_Pubmed_pValue</th>
									
										<th>HGMD_Disease_ID</th>
									
										<th>HGMD_Mutation_URL</th>
									
										<th>PubMedID</th>
									
										<th>GO_BP</th>
									
										<th>GO_CC</th>
									
										<th>GO_MF</th>
									
										<th>KEGG_PATHWAY</th>
									
										<th>PID_PATHWAY</th>
									
										<th>BIOCARTA_PATHWAY</th>
									
										<th>REACTOME_PATHWAY</th>
									
										<th>Expression_summary</th>
									
										<th>RNA tissue specificity</th>
									
										<th>RNA tissue distribution</th>
									
										<th>Subcellular_Location</th>
									
										<th>RNA tissue specific nTPM</th>
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>17365</td>
										
									
										
											<td>rs369606208</td>
										
									
										
											<td>C</td>
										
									
										
											<td>G</td>
										
									
										
											<td>39.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>WASH7P</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_exonic</td>
										
									
										
											<td>NR_024540</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000488147.1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.993729;Name=chr9:10843</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs369606208</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ID=COSN28855450;OCCURENCE=1(prostate),1(large_intestine),4(haematopoietic_and_lymphoid_tissue)</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.0524</td>
										
									
										
											<td>0.0848</td>
										
									
										
											<td>0.0150</td>
										
									
										
											<td>0.0239</td>
										
									
										
											<td>0.0054</td>
										
									
										
											<td>0.0717</td>
										
									
										
											<td>0.0595</td>
										
									
										
											<td>0.0387</td>
										
									
										
											<td>0.0148</td>
										
									
										
											<td>0.0417</td>
										
									
										
											<td>0.0213</td>
										
									
										
											<td>0.0867</td>
										
									
										
											<td>0.0101</td>
										
									
										
											<td>0.0425</td>
										
									
										
											<td>0.0212</td>
										
									
										
											<td>0.0962</td>
										
									
										
											<td>0.0219</td>
										
									
										
											<td>0.0318</td>
										
									
										
											<td>0.0105</td>
										
									
										
											<td>0.0209</td>
										
									
										
											<td>0.2553</td>
										
									
										
											<td>0.1603</td>
										
									
										
											<td>0.2210</td>
										
									
										
											<td>0.3841</td>
										
									
										
											<td>0.2245</td>
										
									
										
											<td>0.2715</td>
										
									
										
											<td>0.2581</td>
										
									
										
											<td>0.2883</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1.301158,14.40</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=0.829;DP=13;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=44.86;MQRankSum=0.256;QD=3.3;ReadPosRankSum=-0.215;SOR=0.307</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:10,2:12:47:47,0,348</td>
										
									
										
											<td>C</td>
										
									
										
											<td>G</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>17614</td>
										
									
										
											<td>rs201057270</td>
										
									
										
											<td>G</td>
										
									
										
											<td>A</td>
										
									
										
											<td>170.64</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>WASH7P</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ncRNA_exonic</td>
										
									
										
											<td>NR_024540</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000488147.1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.993729;Name=chr9:10843</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs201057270</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ID=COSN28540625;OCCURENCE=3(prostate),2(haematopoietic_and_lymphoid_tissue)</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.0562</td>
										
									
										
											<td>0.0933</td>
										
									
										
											<td>0.0084</td>
										
									
										
											<td>0.0920</td>
										
									
										
											<td>0.0969</td>
										
									
										
											<td>0.2443</td>
										
									
										
											<td>0.0490</td>
										
									
										
											<td>0.0584</td>
										
									
										
											<td>0.0242</td>
										
									
										
											<td>0.0566</td>
										
									
										
											<td>0.0497</td>
										
									
										
											<td>0.0925</td>
										
									
										
											<td>0.0126</td>
										
									
										
											<td>0.0994</td>
										
									
										
											<td>0.0658</td>
										
									
										
											<td>0.2334</td>
										
									
										
											<td>0.0566</td>
										
									
										
											<td>0.0609</td>
										
									
										
											<td>0.0262</td>
										
									
										
											<td>0.0615</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1.134517,13.09</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=1;AF=0.5;AN=2;BaseQRankSum=1.193;DP=36;ExcessHet=0;FS=1.64;MLEAC=1;MLEAF=0.5;MQ=43.55;MQRankSum=1.054;QD=4.88;ReadPosRankSum=-1.495;SOR=1.11</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>0/1:27,8:35:99:178,0,726</td>
										
									
										
											<td>G</td>
										
									
										
											<td>A</td>
										
									
										
											<td>0</td>
										
									
										
											<td>1</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>69511</td>
										
									
										
											<td>rs2691305</td>
										
									
										
											<td>A</td>
										
									
										
											<td>G</td>
										
									
										
											<td>5324.06</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>OR4F5</td>
										
									
										
											<td>.</td>
										
									
										
											<td>exonic</td>
										
									
										
											<td>NM_001005484</td>
										
									
										
											<td>.</td>
										
									
										
											<td>missense SNV</td>
										
									
										
											<td>OR4F5:NM_001005484:exon1:c.A421G:p.T141A</td>
										
									
										
											<td>ENST00000641515.2</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.994828;Name=chr19:60000</td>
										
									
										
											<td>.</td>
										
									
										
											<td>rs2691305</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ID=COSM4144171;OCCURENCE=1(thyroid)</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.7598</td>
										
									
										
											<td>0.9616</td>
										
									
										
											<td>0.7768</td>
										
									
										
											<td>0.6019</td>
										
									
										
											<td>0.9847</td>
										
									
										
											<td>0.9484</td>
										
									
										
											<td>0.9999</td>
										
									
										
											<td>0.9707</td>
										
									
										
											<td>0.9900</td>
										
									
										
											<td>0.9766</td>
										
									
										
											<td>0.9418</td>
										
									
										
											<td>0.8457</td>
										
									
										
											<td>0.8612</td>
										
									
										
											<td>0.5948</td>
										
									
										
											<td>0.9776</td>
										
									
										
											<td>0.8953</td>
										
									
										
											<td>0.9998</td>
										
									
										
											<td>0.9674</td>
										
									
										
											<td>0.9907</td>
										
									
										
											<td>0.9784</td>
										
									
										
											<td>0.8655</td>
										
									
										
											<td>0.9394</td>
										
									
										
											<td>0.5942</td>
										
									
										
											<td>0.9507</td>
										
									
										
											<td>0.9994</td>
										
									
										
											<td>0.9907</td>
										
									
										
											<td>0.9716</td>
										
									
										
											<td>0.9597</td>
										
									
										
											<td>0.9832</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GPCR, rhodopsin-like, 7TM;GPCR, rhodopsin-like, 7TM</td>
										
									
										
											<td>0.652,T</td>
										
									
										
											<td>0.0,B</td>
										
									
										
											<td>0.0,B</td>
										
									
										
											<td>0.000,N</td>
										
									
										
											<td>1.000,P</td>
										
									
										
											<td>-0.855,N</td>
										
									
										
											<td>1.26,T</td>
										
									
										
											<td>-0.684</td>
										
									
										
											<td>-0.327</td>
										
									
										
											<td>4.198</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.053</td>
										
									
										
											<td>AC=2;AF=1;AN=2;DP=177;ExcessHet=0;FS=0;MLEAC=2;MLEAF=1;MQ=44.49;QD=31.32;SOR=0.925</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>1/1:0,170:170:99:5338,510,0</td>
										
									
										
											<td>A</td>
										
									
										
											<td>G</td>
										
									
										
											<td>1</td>
										
									
										
											<td>0</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GOBP_G_PROTEIN_COUPLED_RECEPTOR_SIGNALING_PATHWAY;GOBP_SENSORY_PERCEPTION;GOBP_SENSORY_PERCEPTION_OF_CHEMICAL_STIMULUS;GOBP_SENSORY_PERCEPTION_OF_SMELL;GOBP_DETECTION_OF_CHEMICAL_STIMULUS;GOBP_NERVOUS_SYSTEM_PROCESS;GOBP_DETECTION_OF_STIMULUS_INVOLVED_IN_SENSORY_PERCEPTION;GOBP_DETECTION_OF_STIMULUS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>GOMF_G_PROTEIN_COUPLED_RECEPTOR_ACTIVITY;GOMF_MOLECULAR_TRANSDUCER_ACTIVITY</td>
										
									
										
											<td>KEGG_OLFACTORY_TRANSDUCTION</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>REACTOME_OLFACTORY_SIGNALING_PATHWAY;REACTOME_SENSORY_PERCEPTION;REACTOME_GPCR_DOWNSTREAM_SIGNALING</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
								
								<tr>
									
										
											<td style="text-align:left">L</td>
										
									
										
											<td>chr1</td>
										
									
										
											<td>190729</td>
										
									
										
											<td>.</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>37.32</td>
										
									
										
											<td>PASS</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>intergenic</td>
										
									
										
											<td>NR_106918,NR_026823</td>
										
									
										
											<td>dist=2771;dist=14400</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>ENST00000623083.4</td>
										
									
										
											<td>.</td>
										
									
										
											<td>1p36.33</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Score=0.989229;Name=chr19:60000</td>
										
									
										
											<td>Score=989;Name="336:L3(LINE)"</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>0.3136</td>
										
									
										
											<td>0.3414</td>
										
									
										
											<td>0.2090</td>
										
									
										
											<td>0.4588</td>
										
									
										
											<td>0.3680</td>
										
									
										
											<td>0.4945</td>
										
									
										
											<td>0.3426</td>
										
									
										
											<td>0.3370</td>
										
									
										
											<td>0.2948</td>
										
									
										
											<td>0.3216</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>AC=2;AF=1;AN=2;DP=2;ExcessHet=0;FS=0;MLEAC=1;MLEAF=0.5;MQ=48.09;QD=18.66;SOR=0.693</td>
										
									
										
											<td>GT:AD:DP:GQ:PL</td>
										
									
										
											<td>1/1:0,2:2:6:49,6,0</td>
										
									
										
											<td>C</td>
										
									
										
											<td>T</td>
										
									
										
											<td>1</td>
										
									
										
											<td>0</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>Not detected</td>
										
									
										
											<td>.</td>
										
									
										
											<td>.</td>
										
									
								</tr>
								
							
						</table>
					</div>
					<br />
		
					<button onclick="hiddenShow(this)">Show Annotation</button>
					<div style="display: none;">
						<p class="premark">&nbsp;</p>
                        

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>Note: </b>Annotation information includes seven parts: Priority information (1), Chromosome regions and gene structures related to this variation (2-21), Database (frequency) annotation (22-64), Conservation (Deleterious) prediction (65-80), Basic information on the variation (81-87), Gene function and pathway annotation (88-99), and Gene tissue-specific expression annotation (100-104).<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The first item is Priority information.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(1) Priority: </b>H: High: indicates that the mutation is not in the genomic repeat region but in the exonic or splicing region (that is, there is no information for genomicSuperDups and Repeat annotation). The frequency of it in the 1000 genome database is less than 0.01. At least one of SIFT, Polyphen, MutationTaster and CADD predicted that the site is deleterious. M: Medium: indicates that this site is not in the genomic repeat region but in the exonic or splicing region (that is, the mutation cannot be annotated by genomicSuperDups and Repeat). The frequency of it in the 1000-genome database is less than 0.01. L: Low, indicating that the site does not meet the H or M condition of all other mutations. The International Mouse Phenotyping Consortium (IMPC) performs large-scale gene knockout and phenotypic analysis on mice. The symbols (*, +, -) after the priority (H, M, L) mean: *: the gene is lethal (complete lethal gene) in the analysis of large-scale gene knockout phenotype of mice; +: subviable gene (possible lethal gene: < 12.5% embryo survival) in the analysis of large-scale gene knockout phenotype of mice; -: In a large-scale gene knockout phenotype analysis of mice, the gene was viable (viable gene).<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The second item is chrosomal regions and gene structures related to this variation--The Gene where the variation sites located may be associated with certain diseases. We annotates chrosomal regions and gene structures related to this variation, to help teachers understand gene structure and areas information.</b><br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(2) CHROM: </b>Chromosome ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(3) POS: </b>The position of the variant on chromosomes. The value refers to the position of the first base in the REF sting.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(4) ID: </b>The rs number of the variant in dbSNP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(5) REF: </b>Reference base(s).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(6) ALT: </b>Alternate base(s). Comma separated list of alternate non-reference alleles detected in at least one of the samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(7) QUAL: </b>Quality value for the variant. Phred-scaled quality score for the assertion made in ALT. i.e. -10log10 prob (call in ALT is wrong).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(8) FILTER: </b>Filter status, PASS if the position has passed all filters.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(9) GeneName: </b>Names of genes in which this variant is located according to the refGene annotations.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(10) Description: </b>Gene description.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(11) Func: </b>This field tells whether the variant hits exons or hits intergenic regions, or hits introns, or hits non-coding RNA genes. The value of this field takes the following precedence: exonic = splicing > ncRNA > UTR5/UTR3 > intronic > upstream/downstream > intergenic. Notes: 1. When a variant hits different genes or transcripts, the variant may fit multiple functional categories, and then the precedence mentioned above is used to decide what function to print out; 2. The 'exonic' here refers only to coding exonic portion, but not UTR portion, as there are two keywords (UTR5, UTR3) that are specifically reserved for UTR annotations; 3. If a variant is located in both 5'UTR and 3'UTR region (possibly for two different genes), then the 'UTR5,UTR3' will be printed as the output; 4. 'splicing' in ANNOVAR is defined as variant that is within 2bp away from an exon/intron boundary by default; 5. 'splicing' in ANNOVAR only refers to the 2bp in the intron that is close to an exon; 6. The term 'upstream' and 'downstream' is defined as 1kb away from transcription start site or transcription end site, respectively, taking in account of the strand of the mRNA. If a variant is located in both downstream and upstream region (possibly for 2 different genes), then the 'upstream, downstream' will be printed as the output.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(12) Gene: </b>The transcript name(s). If a variant has 'intergenic' in 'Func' field, this field will give two neighboring transcripts. If a variant hits multiple transcripts with different functional categories, only transcript names in accordance with the value of 'Func' field will be output. For example, rs333970 hits the exonic, splicing, intronic, exonic of the four transcripts of gene CSF1, the 'Func' value will be 'exonic; splicing' and the 'Gene' value will be 'NM_000757, NM_172210, NM_172212' (NM_172211 will be ignored).<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(13) GeneDetail: </b>Description of the sequence change in UTR, splicing, ncRNA_splicing or intergenic region. If 'Func' is 'exonic; splicing' or 'splicing', this field gives the sequence change in splicing region(s); for example, NM_172210:exon6:c.1090+5C>A, NM_172210 is the transcript identifier; exon6:c.1090+5C>A is the sequence change and means that this C>A substitution is at the fifth base downstream from the 6th exon (1090 is the end position of the 6th exon of the cDNA). If 'Func' is 'intergenic', this field gives the distance to the neighboring transcripts, such as 'dist=1366; dist=22344'. If 'Func' is 'UTR*', this field gives the sequence change in UTR; for example, NM_198576:c.*19C>T means that this C>T substitution is at the 19th base downstream from stop codon on NM_198576.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(14) ExonicFunc: </b>This field tells the functional consequences of the variant (possible values included: missense SNV, synonymous SNV, frameshift insertion, frameshift deletion, nonframeshift insertion, nonframeshift deletion, frameshift block substitution, nonframshift block substitution, stopgain, stoploss, unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(15) AAChange: </b>This field tells the amino acid changes as a result of the exonic variant. Only exonic variants have information in this field, i.e. when 'Func' is 'exonic' or 'exonic; splicing', this field gives the amino acid change in each related transcript. For example, AIM1L:NM_001039775:exon2:c.C2768T:p.P923L, AIM1L is gene name; NM_001039775 is the transcript identifier; exon2 means this variant is on the second exon of NM_001039775; c.C2768T is the sequence change and means that this C>T substitution is at the 2,768 position on the cDNA; p.P923L is the amino acid change and means that the 923 amino acid on protein is changed from Pro to Leu due to this variant. Another example, NADK:NM_001198995:exon10:c.1240_1241insAGG:p.G414delinsEG, c.1240_1241insAGG is the sequence change and means that there is a 3bp insertion between position 1,240 and 1,241 on the cDNA; p.G414delinsEG is the amono acid change and means that Gly at the 414th amino acid on protein is changed to Glu-Gly.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(16) Gencode: </b>The transcript name(s) in which this variant is located according to Gencode gene definitions.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(17) cpgIslandExt: </b>CpG island prediction. CpG:116 means that there are 116 CG dinucleotides in this CpG island.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(18) cytoband: </b>This field gives the Giemsa-stained chromosomes bands. When a variant spans multiple bands, they will be connected by a dash (for example, 1q21.1-q23.3).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(19) wgRna: </b>Gene names of snoRNAs and microRNAs based on the miRBase Release and snoRNABase.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(20) genomicSuperDups: </b>This field tells whether the variant hits segmental duplications in reference genome. Variants that mapped to segmental duplications are most likely sequence alignment errors and should be treated with extreme caution. The 'Score' field in output is the sequence identity ranging from 0 to 1 between two genomic segments. The 'Name' field represents the other "matching" segments in genome. For example, 'Score=0.994828;Name=chr19:60000' means that the fragment at the position of chr19:60000 is homologous to the fragment containing this variant, and the sequence identity is 0.994828. Note, for a region to be included in the segmental duplications, at least 1 Kb of the total sequence (containing at least 500bp of non-RepeatMasked sequence) had to align and a sequence identity of at least 90% was required.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(21) Repeat: </b>This field tells whether the variant hits interspersed repeats and low complexity DNA sequences output by RepeatMasker program, such as SINE, LINE and Simple repeats. For example, 'Score=180;Name=1385:(CACCC) n(Simple_repeat)', 180 is the score of the repeat, (CACCC) n is the name of the repeat, 'Simple_repeat' is type of repeat. Note, variants mapped to repeats are likely to be false and should be treated with extreme caution.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The third item is database annotation—There are a great number of common polymorphism sites in human population, while many deleterious variants are rare or low-frequency. This part gives the allele frequency and clinical information for each variant.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(22) avsnp: </b>The RS number of the variant in dbSNP database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(23) CLNALLELEID: </b>The ClinVar Allele ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(24) CLNDN: </b>ClinVar's preferred disease name for the concept specified by disease identifiers in CLNDISDB.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(25) CLNDISDB: </b>Tag-value pairs of disease database name and identifier, e.g. OMIM: NNNNNN.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(26) CLNREVSTAT: </b>ClinVar review status for the Variation ID.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(27) CLNSIG: </b>Clinical significance for this single variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(28) cosmic: </b>The Cosmic id in Catalogue Of Somatic Mutations In Cancer (COSMIC) database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(29) gwasCatalog: </b>This field tells whether this variant was previously reported to be associated with diseases or traits in genome-wide association studies. It lists the disease names related to this variation. "." means this variation has not been reported by published GWAS study.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(30) 1000g_SAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(31) 1000g_EUR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(32) 1000g_AFR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(33) 1000g_AMR: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(34) 1000g_EAS: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(35) 1000g_ALL: </b>This field gives the allele frequency for the allele in ALT in 1000 Genomes Project (released in August, 2015) in ALL population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(36) esp6500siv2_all: </b>The ESP is a NHLBI funded exome sequencing project aiming to identify genetic variants in exonic regions from over 6000 individuals, including healthy ones as well as subjects with different diseases. This field gives alternative allele frequency for the variant in ESP.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(37) gnomAD_exome_AF: </b>The Genome Aggregation Database (gnomAD), is a coalition of investigators seeking to aggregate and harmonize exome and genome sequencing data from a variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. This release comprises two callsets: exome sequence data from 123,136 individuals and whole genome sequencing from 15,496 individuals. GenomAD 2.1.1 was used for annotation analysis of the mutation sites. gnomAD_exome_AF gives allele frequency for the variation in all exome  samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(38) gnomAD_exome_AF_raw: </b>The allele frequency for the variation in gnomAD_exome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(39) gnomAD_exome_AF_afr: </b>The allele frequency for the variation in gnomAD_exome for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(40) gnomAD_exome_AF_sas: </b>The allele frequency for the variation in gnomAD_exome for South Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(41) gnomAD_exome_AF_amr: </b>The allele frequency for the variation in gnomAD_exome for Latino/Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(42) gnomAD_exome_AF_eas: </b>The allele frequency for the variation in gnomAD_exome for East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(43) gnomAD_exome_AF_nfe: </b>The allele frequency for the variation in gnomAD_exome for Non-Finnish European population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(44) gnomAD_exome_AF_fin: </b>The allele frequency for the variation in gnomAD_exome for  Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(45) gnomAD_exome_AF_asj: </b>The allele frequency for the variation in gnomAD_exome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(46) gnomAD_exome_AF_oth: </b>The allele frequency for the variation in gnomAD_exome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(47) gnomAD_genome_AF: </b>gnomAD_genome_AF gives allele frequency for the variation in all genome samples of gnomAD. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(48) gnomAD_genome_AF_raw: </b>The allele frequency for the variation in gnomAD_genome that is unfiltered.</br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(49) gnomAD_genome_AF_afr: </b>The allele frequency for the variation in gnomAD_genome for African population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(50) gnomAD_genome_AF_sas: </b>The allele frequency for the variation in gnomAD_genome for  South Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(51) gnomAD_genome_AF_amr: </b>The allele frequency for the variation in gnomAD_genome for  Latino/Admixed American population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(52) gnomAD_genome_AF_eas: </b>The allele frequency for the variation in gnomAD_genome for  East Asian population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(53) gnomAD_genome_AF_nfe: </b>The allele frequency for the variation in gnomAD_genome for  Non-Finnish European population.  <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(54) gnomAD_genome_AF_fin: </b>The allele frequency for the variation in gnomAD_genome for  Finnish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(55) gnomAD_genome_AF_asj: </b>The allele frequency for the variation in gnomAD_genome for  Ashkenazi Jewish population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(56) gnomAD_genome_AF_oth: </b>The allele frequency for the variation in gnomAD_genome for  Other (population not assigned) population. <br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(57) ExAC_ALL: </b>ExAC is short for Exome Aggregation Consortium. The data set spans 60,706 unrelated individuals and should serve as a useful reference set of allele frequencies for severe disease studies. Currently supported population groups include ALL, AFR (African), AMR (Admixed American), EAS (East Asian), FIN (Finnish), NFE (Non-finnish European), OTH (other) and SAS (South Asian). ExAC_ALL gives alternative allele frequency for the variation in ALL ExAC samples.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(58) ExAC_AFR: </b>The alternative allele frequency for the variation in ExAC for African population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(59) ExAC_AMR: </b>The alternative allele frequency for the variation in ExAC for Admixed American population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(60) ExAC_EAS: </b>The alternative allele frequency for the variation in ExAC for East Asian population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(61) ExAC_FIN: </b>The alternative allele frequency for the variation in ExAC for Finnish population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(62) ExAC_NFE: </b>The alternative allele frequency for the variation in ExAC for Non-Finnish European population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(63) ExAC_OTH: </b>The alternative allele frequency for the variation in ExAC for other population.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(64) ExAC_SAS: </b>The alternative allele frequency for the variation in ExAC for South Asian population.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fourth item is functional prediction—These annotations can help to evaluate deleteriousness of a variation. Note 1: SIFT, Polyphen2, MutationTaster, LRT, MutationAssessor and FATHMM are similar and all predict whether an amino acid substitution affects protein function; only coding variants have these annotations. Note 2: phyloP, SiPhy, gerp++ and CADD are similar and predict the conservation level of the site; these types of 'conservation scores' only consider conservation level at the current base, and they do not care about the actual nucleotide identity, so synonymous and non-synonymous variants at the same site will be scored as the same; these scores are used for finding functionally important sites, so variants that confer increased susceptibility may be scored well.</b><br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(65) dbscSNV_ADA_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on adaptive boosting. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(66) dbscSNV_RF_SCORE: </b>an annotated database for predicting whether mutations affect splicing based on Random Forest. dbscSNV is based on a variety of softwares to predict whether mutations affect splicing, and two machine learning methods, Ada and RF, are used to get two integrated prediction scores. It is recommended to use 0.6 as the decision threshold, that is, if at least one of the two dbscSNV scores is greater than 0.6, it is considered that abrupt change may affect splicing.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(67) Interpro_domain: </b>The domain of the protein where the mutation is annotated in the Interpro database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(68) SIFT: </b>SIFT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction; the scores and predictions are separated by comma. There are two possible predictions: D (Deleterious, score&lt=0.05)); T (Tolerated, score&gt0.05).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(69) Polyphen2_HDIV: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanDiv database. This annotation should be used when evaluating rare alleles at loci potentially involved in complex phenotypes, dense mapping of regions identified by genome-wide association studies, and analysis of natural selection from sequence data. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score&gt=0.957), P (possibly damaging, 0.453&lt=score&lt=0.956), B (benign, score&lt=0.452).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(70) Polyphen2_HVAR: </b>PolyPhen 2 (dbNSFP version 3.3a) annotation based on HumanVar database. This annotation should be used for diagnostics of Mendelian diseases. The annotation consists of score and categorical prediction. There are three possible predictions: D (Porobably damaging, score>=0.909), P (possibly damaging, 0.447&lt=score&lt=0.909), B (benign, score&lt=0.446).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(71) LRT: </b>LRT annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are three possible predictions: D (Deleterious), N (Neutral), U (Unknown).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(72) MutationTaster: </b>MutationTaster annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: 'A' (Disease_causing_automatic), 'D (Disease_causing), 'N' (Polymorphism), 'P' (Polymorphism_automatic'). D and N are categorized by only score, while A and P are categorized by score and other information (if nonsynonymous SNV leads to stop-gain, the variation will be predicted an 'A'; if all three genotypes of nonsynonymous SNV has frequency information in HapMap, the variation will be predicted a 'P'). So, both A and D should be considered deleterious.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(73) MutationAssessor: </b>MutationAssessor annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are four possible predictions: H (high), M (medium), L (low), N (neutral). H/M means functional and L/N means non-functional.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(74) FATHMM: </b>FATHMM annotation (dbNSFP version 3.3a). The annotation consists of score and categorical prediction. There are two possible predictions: D (Deleterious, score&lt=-1.5)); T (Tolerated, score&gt-1.5).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(75) phyloP100way_vertebrate: </b>PhyloP score (dbNSFP version 3.3a) based on the whole genome alignment of 100 vertebrates. Generally, the higher the score, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(76) phyloP30way_mammalian: </b>PhyloP score (dbNSFP) based on the whole genome alignment of 30 mammals.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(77) SiPhy_29way_logOdds: </b>SiPhy score (dbNSFP version 3.3a) based on the whole genome alignment of 29 mammals genomes. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(78) CADD: </b>CADD (Combined Annotation Dependent Depletion) is a score that is based on SVM on multiple other scores. It assigns a score to each possible mutation in the human genome including non-coding and coding variants. In the output, the comma-delimited values are raw scores and phred-scaled scores. "." stands for CADD score&lt10. For phred-scaled scores, 10 means 10% percentile highest scores, 20 means 1% percentile highest scores, and 30% means 0.1% percentile highest scores. CADD official website suggests 15 as a cutoff; in published studies, 10 or 15 is used as a cutoff.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(79) gerp++gt2: </b>GERP++ scores for all mutations with GERP++>2 in human genome, as this threshold is typically regarded as evolutionarily conserved and potentially functional. Variants with '.' in this field should be considered not conserved. The larger the score is, the more conserved the site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(80) REVEL: </b>A method for predicting the pathogenicity of rare missense mutations by integrating multiple software prediction scores. The score of REVEL ranges from 0 to 1, and the larger the score is, the more deleterious it will be. In the literature, when the score is 0.5, the sensitivity and specificity of detecting harmful mutations are 0.754 and 0.891 respectively.<br />


<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The fifth item is basic information on the variation  —This part shows the detail information of variation sites, including allelic depths , types of DNA base before and after mutation, and genotype information, et al. these information play a critical role in pedigree analysis.</b> <br />

<p class="premark">&nbsp;</p>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(81) INFO: </b>Information about this variation from variant calling software. INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: &ltkey&gt=&ltdata&gt[,data].<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(82) FORMAT: </b>The FORMAT field specifies the data types and order (colon-separated alphanumeric String). This is followed by one field per sample, with the colon-separated data corresponding to the types specified in the FORMAT.<br />


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GT: genotype, encoded as allele values separated by either of / or |. The allele values are 0 for the reference allele (what is in the Ori_REF field), 1 for the first allele listed in Ori_ALT, 2 for the second allele list in Ori_ALT and so on. 0/0 and 1/1 represent homozygous. 0/1 represents heterozygous. ‘.’ means that a call cannot be made for a sample at a given locus.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AD: Allelic depths for the ref and alt alleles in the order listed (Allelic depths).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;DP: Approximate read depth (reads with MQ=255 or with bad mates are filtered).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GQ: Genotype Quality.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PL: Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(83) SampleID: </b>The colon-separated data in this sample corresponding to the types specified in the FORMAT.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(84) Ori_REF: </b>The reference allele (what is in the REF field) in VCF file. For InDel, the allele in "REF" field in this file may be different from (usually shorter than) the "REF" in VCF file.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(85) Ori_ALT: </b>The alternative allele(s) (what is in the ALT field) in VCF file. In this file, the allele in "ALT" field corresponds to one alelle in "Ori_ALT" field; for InDel, the allele in "ALT" field may be different from (usually shorter than) the corresponding allele in the "Ori_ALT" field.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(86) shared_hom: </b>The number of samples with homozygous mutations at the current site.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(87) shared_het: </b>The number of samples with a heterozygous mutation at the current site.<br />

<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The sixth item is gene function and pathway annotation: These annotations are for genes containing this variation.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(88) OMIM: </b>Annotation from Online Mendelian Inheritance in Man (OMIM).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(89) GWAS_Pubmed_pValue: </b>Annotation from the NHGRI-EBI GWAS Catalog. The value is like 'pubmedID(p-value);pubmedID(p-value)'. 'pubmedID' is PubMed ID of publication of the study which reported the association between the variation and disease. 'p-value' is the corresponding p-value in the publication.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(90) HGMD_Disease_ID: </b>Annotation from the Human Gene Mutation Database (HGMD®). The value is like 'ID_HGMD(Disease_name);ID_HGMD(Disease_name)'. ID_HGMD is HGMD internal identifier. Disease_name is the name for the disease or condition associated with the mutation.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(91) HGMD_mutation_URL: </b> Annotation from the Human Gene Mutation Database (HGMD®). The value is infomation about this variant.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(92) PubMedID: </b>Given PubMed documents associated with the mutations or gene number, which from the HGMD and OMIM database.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(93-95) GO_BP, GO_CC, GO_MF:</b>Annotation from Gene Ontology. BP is Biological Process; CC is cellular component; MF is molecular function.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(96) KEGG_PATHWAY: </b>Annotation from KEGG PATHWAY Database.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(97) PID_PATHWAY: </b>Annotation from PID (Pathway Interaction Database).<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(98) BIOCARTA_PATHWAY: </b>Annotation from BioCarta.<br />

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(99) REACTOME_PATHWAY: </b>Annotation from Reactome Pathway Database.<br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>The seventh item is gene tissue-specific expression. This section presents the Human Protein Atlas (HPA) database recorded protein expression of protein-encoding genes in 44 tissues of normal human and detection results of RNA transcription levels in 32 tissues, as well as links to corresponding visual results on HPA website.</b> <br />
<p class="premark">&nbsp;</p>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(100) Expression_summary: </b>An overview of protein expression, combining immunohistochemical results, RNA results and subcellular distribution results, to describe the distribution of gene expression in different tissues and at the subcellular level.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(101) RNA tissue specificity: </b>Elevated expression includes three subcategory types of elevated expression:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enriched: At least four-fold higher mRNA level in a particular tissue compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Group enriched: At least four-fold higher average mRNA level in a group of 2-5 tissues compared to any other tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tissue enhanced: At least four-fold higher mRNA level in a particular tissue compared to the average level in all other tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In addition, there is another form of low tissue specificity: Low expression in various tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(102) RNA tissue distribution: </b>Visualization of gene expression in tissues. All elevated genes are categorized as:<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in single: Detected in a single tissue;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in some: Detected in more than one but less than one third of tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in many: Detected in at least a third but not all tissues;<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Detected in all: Detected in all tissues.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(103) Subcellular_Location: </b>subcellular distribution positions of genes, such as "nucleus", "mitochondria", etc.<br />
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>(104) RNA tissue specific nTPM: </b>Consensus transcript expression levels summarized per gene in 50 tissues based on transcriptomics data from HPA and GTEx. The consensus normalized expression ("nTPM") value is calculated as the maximum nTPM value for each gene in the two data sources. If the "RNA tissue specificity" is "Low tissue specificity", this column is empty.<br />



		
						</ul>
					</div>
				</div>
		</div>
	


			<p class="paragraph"><a href="../../result/Mutation/SNP/Annotation" title="Click to open" target="_blank">Result:03.Result_Homo_sapiens/result/Mutation/SNP/Annotation</a></p>
				

			
			<!-------------------------------------------- somaticSNP/InDel检测------------------------------------------->
			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Somatic Mutation Detection"></a>
					<hr />
				</p><br />

				<h3>4.5  Somatic Mutation Detection</h3>
				<p class='paragraph'>Somatic mutations refer to those variants that occur in somatic cells instead of germ cells. They neither inherit from parents nor pass on to offspring, but could cause changes in the genetic structure of some contemporary cells. The driver mutations that included in somatic mutations play important roles in tumorigenesis and further evolution. Therefore, focusing on the somatic mutations is the key point of tumor research, and it is the feature that different from disease research.</p>
				
				<h4>4.5.1 Somatic SNP Detection Result</h4>
					
						<p class='paragraph'>We used the tool MuTect(Cibulskis et al., 2013) to detect somatic SNPs, and the tool Strelka (Saunders et al., 2012)to detect somatic InDels. The statistics of detected somatic SNPs in the tumor samples are listed below: </p>
					
				<div id="tb">
					<p class="center">
					
					<p class="name_fig">Table 4.8 The number of somatic SNPs in various genomic regions</p>
					
					<table class="gy">
					
						
						<tr>
							
								
									<th>Sample name</th>
								
							
								
									<th>P2_T</th>
								
							
								
									<th>P3_T</th>
								
							
								
									<th>P4_T</th>
								
							
						</tr>
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">CDS</td>
									
								
									
										<td>30</td>
									
								
									
										<td>139</td>
									
								
									
										<td>81</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;synonymous_SNP</td>
									
								
									
										<td>8</td>
									
								
									
										<td>33</td>
									
								
									
										<td>34</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;missense_SNP</td>
									
								
									
										<td>21</td>
									
								
									
										<td>97</td>
									
								
									
										<td>43</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
									
								
									
										<td>0</td>
									
								
									
										<td>8</td>
									
								
									
										<td>3</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
									
								
									
										<td>1</td>
									
								
									
										<td>1</td>
									
								
									
										<td>1</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">intronic</td>
									
								
									
										<td>38</td>
									
								
									
										<td>178</td>
									
								
									
										<td>49</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">UTR3</td>
									
								
									
										<td>0</td>
									
								
									
										<td>7</td>
									
								
									
										<td>3</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">UTR5</td>
									
								
									
										<td>2</td>
									
								
									
										<td>12</td>
									
								
									
										<td>4</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">splicing</td>
									
								
									
										<td>3</td>
									
								
									
										<td>4</td>
									
								
									
										<td>3</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_exonic</td>
									
								
									
										<td>4</td>
									
								
									
										<td>6</td>
									
								
									
										<td>6</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_intronic</td>
									
								
									
										<td>5</td>
									
								
									
										<td>10</td>
									
								
									
										<td>4</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_splicing</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">upstream</td>
									
								
									
										<td>0</td>
									
								
									
										<td>6</td>
									
								
									
										<td>2</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">downstream</td>
									
								
									
										<td>0</td>
									
								
									
										<td>1</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">intergenic</td>
									
								
									
										<td>8</td>
									
								
									
										<td>29</td>
									
								
									
										<td>14</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>90</td>
									
								
									
										<td>393</td>
									
								
									
										<td>166</td>
									
								
							</tr>
							
						
					
					</table>	
					</p>
				</div>

				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
					<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of somatic SNPs in coding region.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  synonymous_SNP: A single nucleotide change that does not cause an amino acid change.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  missense_SNP: A single nucleotide change that causes an amino acid change.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  stopgain: A nonsynonymous SNP that leads to the creation of stop codon at the variant site.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  stoploss: A nonsynonymous SNP that leads to the elimination of stop codon at the variant site.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  unknown: Unknown function (due to various errors in the gene structure definition in the database file).<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of somatic SNPs in intronic region.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of somatic SNPs in 3'UTR region.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of somatic SNPs in 5'UTR region.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of somatic SNPs within 2-bp of a splicing junction.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of somatic SNPs in exonic region of non-coding RNAs.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of somatic SNPs in intronic region of non-coding RNAs.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of somatic SNPs within 2-bp of a splicing junction of non-coding RNAs.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of somatic SNPs within 1kb away from the transcription start site.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of somatic SNPs within 1kb away from the transcription end site.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of somatic SNPs in intergenic region.<br />
						&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of somatic SNPs.</p>
						<p class="premark">&nbsp</p>
				</div>
				<p class="paragraph"><a href="../../result/Somatic/Somatic_SNP" title="Click to open" target="_blank">Result:03.Result_Homo_sapiens/result/Somatic/Somatic_SNP</a></p>
				<br />
				
				<h4>4.5.2 Somatic InDel Detection Result</h4>
				<div id="tb">
					<p class="center">
					
						<p class="name_fig">Table 4.9 The number of somatic InDel in various genomic regions</p>
					
					<table class="gy">
					
						
						<tr>
							
								
									<th>Sample name</th>
								
							
								
									<th>P2_T</th>
								
							
								
									<th>P3_T</th>
								
							
								
									<th>P4_T</th>
								
							
						</tr>
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">CDS</td>
									
								
									
										<td>1</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;frameshift_deletion</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;frameshift_insertion</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;nonframeshift_deletion</td>
									
								
									
										<td>1</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;nonframeshift_insertion</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stopgain</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;stoploss</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">&nbsp;&nbsp;&nbsp;&nbsp;unknown</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">intronic</td>
									
								
									
										<td>23</td>
									
								
									
										<td>15</td>
									
								
									
										<td>34</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">UTR3</td>
									
								
									
										<td>3</td>
									
								
									
										<td>2</td>
									
								
									
										<td>1</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">UTR5</td>
									
								
									
										<td>3</td>
									
								
									
										<td>0</td>
									
								
									
										<td>1</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">splicing</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_exonic</td>
									
								
									
										<td>2</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_intronic</td>
									
								
									
										<td>0</td>
									
								
									
										<td>6</td>
									
								
									
										<td>2</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">ncRNA_splicing</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">upstream</td>
									
								
									
										<td>1</td>
									
								
									
										<td>2</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">downstream</td>
									
								
									
										<td>0</td>
									
								
									
										<td>1</td>
									
								
									
										<td>0</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">intergenic</td>
									
								
									
										<td>14</td>
									
								
									
										<td>13</td>
									
								
									
										<td>11</td>
									
								
							</tr>
							
						
					
						
							
							<tr>
								
									
										<td style="text-align:left">Total</td>
									
								
									
										<td>47</td>
									
								
									
										<td>39</td>
									
								
									
										<td>49</td>
									
								
							</tr>
							
						
					
					</table>
				</p>
				</div>

				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
					<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) CDS: The number of somatic InDels in coding region.<br />
					&nbsp;		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_deletion: A deletion of one or more nucleotides that cause frameshift changes in protein coding sequence.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frameshift_insertion: An insertion of one or more nucleotides that cause frameshift changes in protein coding sequence.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_deletion: A deletion that does not cause frameshift changes.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;non-frameshift_insertion: An insertion that does not cause frameshift changes.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stopgain: An insertion or a deletion that leads to the creation of stop codon at the variant site.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stoploss: An insertion or a deletion that leads to the elimination of stop codon at the variant site.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;unknown: Unknown function (due to various errors in the gene structure definition in the database file).<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) intronic: The number of somatic InDels in intronic region.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) UTR3: The number of somatic InDels in 3'UTR region.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) UTR5: The number of somatic InDels in 5'UTR region.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) splicing: The number of somatic InDels within 2-bp of a splicing junction.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) ncRNA_exonic: The number of somatic InDels in exonic region of non-coding RNAs.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(8) ncRNA_intronic: The number of somatic InDels in intronic region of non-coding RNAs.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(9) ncRNA_splicing: The number of somatic InDels within 2-bp of a splicing junction of non-coding RNAs.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(10) upstream: The number of somatic InDels within 1kb away from transcription start site.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(11) downstream: The number of somatic InDels within 1kb away from transcription end site.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(12) intergenic: The number of somatic InDels in intergenic region.<br />
					&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(13) Total: The total number of somatic InDels.</p>
     			</div>
				 <p class="paragraph"><a href="../../result/Somatic/somatic_INDEL" title="Click to open" target="_blank">Result:03.Result_Homo_sapiens/result/Somatic/Somatic_INDEL</a></p>
			</div>
			

			

			
			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Somatic CNV Detection Result"></a>
					<hr />
				</p><br />
				<h4>4.5.3 Somatic CNV Detection Result</h4>
				<p class="paragraph">Statistics of detected somatic CNVs(Boeva et al., 2012) are listed below:</p>
				<p class="center">
				
					<p class="name_fig">Table 4.10 Statistics of detected somatic CNVs</p>
				

				<div id="tb">
					<table class="gy">
					<tr>
						<th>Sample name</th>
						<th>gain_count</th>
						<th>gain_size</th>
						<th>loss_count</th>
						<th>loss_size</th>
						<th>total_count</th>
						<th>total_size</th>
					</tr>
					
						
							<tr>
								
									<td>P2_T</td>
								
									<td>15</td>
								
									<td>10377592</td>
								
									<td>6</td>
								
									<td>468344</td>
								
									<td>21</td>
								
									<td>10845936</td>
								
							</tr>
						
					
						
							<tr>
								
									<td>P3_T</td>
								
									<td>133</td>
								
									<td>372592363</td>
								
									<td>36</td>
								
									<td>382289839</td>
								
									<td>169</td>
								
									<td>754882202</td>
								
							</tr>
						
					
						
							<tr>
								
									<td>P4_T</td>
								
									<td>81</td>
								
									<td>2264952</td>
								
									<td>13</td>
								
									<td>831713</td>
								
									<td>94</td>
								
									<td>3096665</td>
								
							</tr>
						
					
					</table>
				</div>
				<br />

				</script>
				<button onclick="hiddenShow(this)">Show Annotation</button>
				<div style="display: none;">
				<p class="tremark">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Note: <br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1) Sample name: Sample ID.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2) gain_count: The number of gains.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3) gain_size: The total size of gains.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4) loss_count: The number of losses.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5) loss_size: The total size of losses.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(6) total_count: The total number of CNVs.<br />
				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(7) total_size: The total size of CNVs.<br />
				<p class="premark">&nbsp</p>
				</div><br/><br/>
				<p class="center">
					<img height=500px; src="src/pictures/SV/somatic_cnv_type_stat.png" />
					
					
						<p class="name_fig">Figure 4.9 The size of genomic regions affected by somatic CNVs in each sample</p>
					
					<!--<p class="name_fig">Figure 5.　The size of genomic regions affected by somatic CNVs in each sample</p>-->
					<p class="premark">The x-axis represents the samples, and the y-axis represents the total size of genomic regions affected by gains or losses (Mb).</p>
					<p class="premark">&nbsp</p>
					</p>
					<br />
					<p class="paragraph"><a href="../../result/Somatic/Somatic_CNV" title="Click to open" target="_blank">Result:03.Result_Homo_sapiens/result/Somatic/Somatic_CNV</a></p>
			
			
			
			
			</div>



<!-------------------------------------------- 癌症高级分析------------------------------------------->

			<div id="page">
			<!--<div id="page">-->
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="Advanced analysis"></a>
					<hr />
				</p><br />

				<h3>4.6  Advanced analysis</h3>
<!-- ------------------------------------------ 易感基因 -------------------------------------  -->

<div id="page">
  <a name="Screening for Predisposing Genes"></a>
  <h4 id="goto_resultadv1"><a id="Screening for Predisposing Genes" name="Screening of oncogenes, tumor suppressor genes and predisposing genes">4.6.1 Screening for Predisposing Genes</a></h4>
  <p class="paragraph">
    Predisposing genes are mutated genes that are associated with increased susceptibility to cancer. In this analysis, the called SNPs/InDels in normal samples were screened for possible predisposing genes based on the following filtering criteria:
  </p>
  <p class="tremark">
    <ul style="font-size: 14px;font-family: '黑体-简'">
	
      <li>Filter out variants with sequencing depth lower than 10X</li>
      <li>Filter out variants affecting genes not included in the list of genes reported by Cancer Gene Census (CGC);</li>
      <li>Filter out synonymous mutations and variants within intergenic, noncoding and intronic regions;</li>
      <li>Filter out variants located in segmental duplication or repetitive regions;</li>
      <li>Filter out variants based on deleteriousness predictions with four types of scores, including SIFT, Polyphen2 HVAR, Polyphen2 HDIV and MutationTaster scores (predicted as 'damaging' by at least one type of scores, or 'possibly damaging' by ≥ 2 types of scores)</li>
    </ul>
  </p>
  
  <div style="height: 600px; margin: auto; text-align: center;">
    <img src='src/pictures/PredisposeGenes/Gene.Mutation.png' style="max-height: 600px;" />
  </div>
  <p class="name_fig" style="font-size: 12px" style="text-align: center;">Figure 4.3 predisposing genes landscape map</p>
  <p class="premark">The abscissa is the sample, the ordinate is the gene, the top is the number of mutations in susceptible genes in each sample, and the right is the number of mutations in each gene in these samples. Using oncoprint mapping, take the first 30 susceptibility gene mutations to display normal samples. If the samples do not have the 30 gene mutations, they will not be displayed by default.</p><br/>
  <p class="paragraph">
    Through the CGC（Cancer Gene Census, http://cancer.sanger.ac.uk/cancergenome/ projects/census/）、FACD（Familial Cancer Database, http://www.familialcancerdatabase.nl/）、intOGen（Martínez-Jiménez <i>et al.</i>，2020） and the reported genes summarized in nature literature by our company, annotate the above screening results, so as to further interpret the results according to personalized research needs.
  </p>
  <p class="name_fig" style="font-size: 12px">Table 4.6 Predisposing gene screening results</p>
  <iframe id="predispos_table_iframe" style="width:95%; height: 440px; text-align: center; margin: auto;" src="src/tables/predispose_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe><br/>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Hugo_Symbol: gene name</li>
      <li>Chromosome: chromosome ID</li>
      <li>Position: coordinate</li>
      <li>Ref_allele: reference allele</li>
      <li>Alt_allele: alternative allele</li>
      <li>Variant_Classification: classification of the variant</li>
      <li>AAChange: information about animo acid changes</li>
      <li>CGC_Cancers: the corresponding cancer type in CGC database</li>
    </ol>
  </div>
  </div><br/><br/>
  
  <p class="paragraph"><a href="../../../result/Pro/PredisposeGenes" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/PredisposeGenes</a></p>
</div>


<!-- ------------------------------------------ 突变频谱和突变特征 -------------------------------------  -->
<!-- ------------------------------------------ 突变频谱 -----------------------------------------------  -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="Mutational Spectrum & Mutational Signature"></a><hr /></p><br />
  <h4 id="goto_resultadv2">4.6.2 Mutational Spectrum & Mutational Signature Analysis</h4>
  <p class="paragraph">Mutational spectrum and mutational signature analyses were performed based on somatic SNVs. From the results, we can know the proportion and pattern of each type of substitutions (e.g. C>A/G>T) across tumor samples, which may help us reveal the mutational processes underlying the development of cancer.</p>
    <h5><a id="Mutational Spectrum">4.6.2.1 Mutational Spectrum</a></h5>
      <p class="paragraph">Six substitution types of point mutations were considered in this analysis, including C>A/G>T, C>G/G>C, C>T/G>A, T>A/A>T, T>C/A>G and T>G/A>C. The figures below show the results of mutational spectrum analysis:</p>
      <p class="center">
      <div class="albumSlider">
        <div class="fullview">
          <img src='src/images/mutation_spectrum.barplot.png'/>
        </div>
        <div class="slider">
          <div class="button movebackward" title="向上滚动">
          </div>
          <div class="imglistwrap">
            <ul class="imglist">
              <li><a id="example2" href='src/pictures/mutation_spectrum.barplot.png' ><img src='src/pictures/mutation_spectrum.barplot.jpeg'/></a></li>
              <li><a id="example2" href='src/pictures/mutation_spectrum.heatmap.png' ><img src='src/pictures/mutation_spectrum.heatmap.jpeg'/></a></li>
            </ul>
          </div>
          <div class="button moveforward" title="向下滚动">
          </div>
        </div>
      </div>
      </p>
      <p class="name_fig" style="font-size: 12px">Figure 4.4 The bar plot and heatmap of mutational spectrum</p>
      <p class="premark"> In the bar plot of mutational spectrum, the horizontal axis represents samples, and the vertical axis represents the proportion of each substitution type. Different types are distinguished by different colors.</p>
      <p class="premark"> In the heatmap of mutational spectrum, each column represents one sample, and each row represents one type of substitutions. The darker the color is, the higher the proportion of one substitution type is.
      </p>
</div>
<!-- ------------------------------------------ 突变特征 -------------------------------------  -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="Mutational signature"></a><hr /></p><br />
  <h5><a id="Mutational signature">4.6.2.2 Mutational signature</a></h5>
  <p class="paragraph">Somatic mutations are present in all cells of the human body and occur throughout life. They are the consequence of multiple mutational processes, including the intrinsic slight infidelity of the DNA replication machinery, exogenous or endogenous mutagen exposures, enzymatic modification of DNA and defective DNA repair. Different mutational processes often generate different combinations of mutation types, termed ‘mutational signatures’. The profile of each signature is displayed using the six substitution types: C>A, C>G, C>T, T>A, T>C, and T>G (all substitutions are referred to by the pyrimidine of the mutated Watson–Crick base pair). Further, each of the substitutions is examined by incorporating information on the bases immediately 5’ and 3’ to each mutated base generating 96 possible mutation types (6 types of substitution * 4 types of 5’ base * 4 types of 3’ base). In this analysis, mutational signatures were obtained by an approach named Nonnegtive Matrix Factorization (NMF).</p>
  <p class="center">
  <div class="albumSlider">
    <div class="fullview">
      <img src='src/images/mutation_signature.png'/>
    </div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动">
      </div>
      <div class="imglistwrap">
        <ul class="imglist">
          <li><a id="example2" href='src/pictures/mutation_signature.png' ><img src='src/pictures/mutation_signature.jpeg'/></a></li>
          <li><a id="example2" href='src/pictures/mutation_signature.contributions.png' ><img src='src/pictures/mutation_signature.contributions.jpeg'/></a></li>
          <li><a id="example2" href='src/pictures/Cosine_similarity.png' ><img src='src/pictures/Cosine_similarity.jpeg'/></a></li>
        </ul>
      </div>
      <div class="button moveforward" title="向下滚动">
      </div>
    </div>
  </div>
  </p>
  <p class="name_fig" style="font-size: 12px">Figure 4.5 Identified mutational signature</p>
  <p class="premark">The first figure shows mutational signatures. The mutation types are on the horizontal axes, whereas vertical axes depict the percentage of mutations attributed to a specific mutation type.</p>
  <p class="premark">The second figure shows the proportion of each type of mutational signatures in each sample.</p>
  <p class="premark">The third figure shows hierarchical clustering of cosine similarities between the identified mutational signatures in samples (Signature A/B/C) and 30 known mutational signatures. The darker the color is, the closer the cosine similarity value is to 1 and the more likely the two compared signatures are actually the same.</p>
  <p class="center">
    <p class="name_fig" style="font-size: 12px">Table 4.7 Mutational signature annotation</p>
    <iframe style="text-align: center; width: 95%; height: 650px; margin:auto;" src="src/tables/signature_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Our signature: the identified mutational signature in samples</li>
      <li>Near reference signature: the known mutational signature that is the most similar one to the identified signature in samples</li>
      <li>Cosine similarity: the cosine similarity value</li>
      <li>Filter with cosine similarity > 0.9：whether the cosine similarity is over 0.9, which is the cutoff to determine whether the two compared signatures are the same</li>
      <li>Cancer types: the cancer types associated with the known signature</li>
      <li>Proposed aetiology：the proposed aetiology of the known signature</li>
      <li>Additional mutational features：additional mutational features of the known signature</li>
      <li>Comments：other information about the known mutational signature</li>
    </ol>
  </div>
  </div><br/><br/>
  <p class="paragraph"><a href="../../../result/Pro/Signatures" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Signatures</a></p>
</div>

<!-- ------------------------------------------ 驱动基因检测 -------------------------------------  -->

<!-- ------------------------------------------ 已知驱动基因 -------------------------------------  -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="Identification of Known Driver Genes"></a><hr /></p><br />
  <h4 id="goto_resultadv3"><a>4.6.3 Identification of Known Driver Genes</a></h4>
  <p class="paragraph">
    Cancer is essentially a 'disease of the genome', which evolves with accumulations of tens to thousands of mutations. However, only a few of them 'drive' tumorigenesis by affecting genes, i.e. driver genes, which upon alteration confer selective growth advantage to tumor cells. Hundreds of driver genes have been reported by previous researches. Identification of known driver genes in our tumor samples may help us pinpoint the key genes critical for cancer initiation and progressions.<br/>
    <ul style="font-size: 14px;font-family: '黑体-简'">
        <li>Filter out variants with sequencing depth lower than 10X</li>
        <li>Filter out variants affecting genes not included in the above list of known driver genes,Filter out variants recorded in dbSNP database (dbSNP144), but keep those that have deposited in COSMIC database (comsic70) or with low (<0.01) alternative allele frequency reported by 1000 Genome project (1000g2015aug_all)</li>
        <li>Filter out synonymous mutations and variants within intergenic, noncoding and intronic regions;</li>
        <li>Filter out variants located in segmental duplication or repetitive regions; </li>
        <li>Filter out variants based on deleteriousness predictions with four types of scores, including SIFT, Polyphen2 HVAR, Polyphen2 HDIV and MutationTaster scores (predicted as 'damaging' by at least one type of scores, or 'possibly damaging' by ≥ 2 types of socores)</li>
    </ul>
  </p>
  <p class="paragraph">
    Then the somatic mutation obtained by screening was compared with the known driver genes in the database to screen out the known driver genes in the tumor sample.
  </p>
  <p class="paragraph">
    The sources of known driver genes:：
    <ul style="font-size: 14px;font-family: '黑体-简'">
      <li><a href= "http://cancer.sanger.ac.uk/cancergenome/projects/census/" target='_blank' >CGC513</a>: Driver genes listed in Cancer Gene Census(CGC) database;</li>
      <li><a href= "http://www.sciencemag.org/content/339/6127/1546.full" target='_blank' >Bert Vogelstein125</a>: 125 mut-driver genes reported by Vogelstein et al (2013, Science 339: 1546-1558)</li>
      <li><a href= "http://www.nature.com/nature/journal/v502/n7471/abs/nature12634.html" target='_blank' >SMG127</a>:  127 significantly mutated genes reported by Kandoth et al (2013, Nature 502: 333-339)</li>
      <li><a href= "http://www.nature.com/srep/2013/131002/srep02650/full/srep02650.html" target='_blank' >Comprehensive435</a>: 435 driver genes (291 high-confidence drivers and 144 candidate drivers) reported by Tamborero et al (2013, Sci Rep 3: 2650).</li>
    </ul>
  </p>
  
<!-- ------------------------------- 表3.2 -----------------------------------------  -->
  <div style="margin: auto; height: 600px; text-align: center;">
    <img style="max-height: 600px;" src='src/pictures/DriverGenes/Gene.Mutation.png'/>
    <p class="name_fig" style="font-size: 12px">Figure 4.6 Landscape map of known driver genes</p>
    <p class="premark">Note：The abscissa is the sample, the ordinate is the gene, the top is the number of mutations of each sample in the driver gene, and the right is the number of mutations of each gene in these samples. Using oncoprint mapping, take the top 30 driver gene mutations to display tumor samples. If the samples do not have the 30 gene mutations, they will not be displayed by default.</p>
  </div><br/><br/>
  <p class="center">
    <p class="name_fig" style="font-size: 12px"><br/><br/>Table 4.8 Known driver gene analysis results</p>
    <iframe style="text-align: center; width: 95%; height: 528.0px;margin: auto;" src="src/tables/driver_gene_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Hugo_Symbol: Hugo_Symbol: gene name</li>
      <li>Chromosome: Chromosome: chromosome ID</li>
      <li>Position: coordinate</li>
      <li>Ref_allele：reference allele</li>
      <li>Alt_allele：alternative allele</li>
      <li>Variant_Classification：classification of the variant</li>
      <li>AAChange: amino acid changes</li>
      <li>CGC: corresponding cancer names in CGC database</li>
      <li>BertVogelstein125: the driver gene has been reported by Vogelstein et al (2013, Science 339: 1546-1558); TSG: Tumor suppressor gene</li>
      <li>Comprehensive435: the driver gene has been reported by Tamborero et al (2013, Sci Rep 3: 2650)</li>
      <li>SMG127:the driver gene has been reported by Kandoth et al (2013, Nature 502: 333-339); values denote mutation frequency</li>
    </ol>
  </div>
  </div><br/><br/>
  
  <p class="paragraph"><a href="../../../result/Pro/KnownDriverGenes" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/KnownDriverGenes</a></p>
</div>


<!-- ------------------------------------------ 肿瘤高频突变分析 -------------------------------------  -->
<!-- ------------------------------------------ 高频突变基因统计 -------------------------------------  -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="肿瘤高频突变分析"></a><hr /></p><br />
  <h4 id="goto_resultadv4"><a id="肿瘤高频突变分析" name="肿瘤高频突变分析">4.6.4 Significantly mutated genes</a></h4>

  <p class="paragraph">Significantly mutated genes（Significantly mutated genes, SMG）refer to those that show a significantly higher mutation rate than the background mutation rate (BMR), which indicates positive selection during tumorigenesis.Cellular pathways with significant accretions of somatic mutations are known as significantly mutated pathways (SMPs). Analyses on SMGs and SMPs help us pinpoint the key genes and pathways critical for cancer initiation and progression. We performed the identification of SMGs and SMPs with both SNVs and InDels considered. All analyses were performed with modules of the software <a href="http://search.cpan.org/dist/Genome-Model-Tools-Music/" target=_blank>MuSic</a>.</p>


    <h5><a id="高频突变基因统计">4.6.4.1 A summary of the identified SMGs</a></h5>
    
  <p class="paragraph">No relevant significantly mutated genes were detected</p><br/><br/>
  

</div>

<!-- ------------------------------------------ 高频突变热图 -------------------------------------  -->




<!-- ------------------------------------------ 高频突变基因富集分析 -------------------------------------  -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="高频突变基因富集分析"></a><hr /></p><br />

  
  <h5><a id="高频通路分析">4.6.4.2 Significantly mutated pathways</a></h5>
  


  <p class="paragraph">Databases used include KEGG, Biocarta, Protein Interaction Database and Reactome</p>

  <p class="center">
    <p class="name_fig" style="font-size: 12px">Table 4.9 Significantly mutated pathways </p>
    <iframe style="text-align: center; width: 95%; height: 440px;margin: auto;" src="src/tables/pathway_enrichment_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      
      <li>Pathway： pathway ID</li>
      <li>Name：pathway name</li>
      <li>Class：database name</li>
      <li>Samples_Affected：number of samples with mutations in the pathway</li>
      <li>Total_Variations：total number of mutations</li>
      <li>p-value：P-value</li>
      <li>FDR：false discovery rate</li>
      
    </ol>
  </div>
  </div><br/><br/>
  <p class="paragraph"><a href="../../../result/Pro/SMG" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/SMG</a></p>
</div>



<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="肿瘤突变负荷"></a><hr /></p><br />
  <h4 id="goto_resultadv16">4.6.5 tumor mutational burden</h4>
  <p class="paragraph">
Tumor mutational burden（TMB）is a measure of the number of somatic protein coding base mutations that occur in tumor samples. It generally refers to the number of somatic nonsynonymous mutations (snvs and indels) per megabase pair (MB) in a specific genomic region, which can indirectly reflect the ability and degree of neoantigens produced by tumors（Schumacher <i>et al.</i>, 2015; Chalmers <i>et al.</i>, 2017）。High tumor mutational burden (TMB), an emerging biomarker sensitive to immune checkpoint inhibitors, has been demonstrated to be more significantly associated with PD-1 or PD-L1 immunotherapy response than PD-1 or PD-L1 expression(Chalmers <i>et al.</i>, 2017)High TMB can assist in predicting the efficacy of immunotherapy in patients with non-small cell lung cancer, small cell lung cancer, urothelial cancer, endometrial cancer, breast cancer, colorectal cancer and melanoma（Rizvi ec al. 2015; Rosenberg <i>et al.</i>, 2016; Chan <i>et al.</i>, 2019）。
The non synonymous mutations of somatic mutation (SNV, indel) in the patient's tumor tissue were used to calculate the TMB. The specific analysis strategy was as follows:<br />

1）Filter out variants with depth lower than 20×<br />


2）Keep variants with VAF ≥ 5% ;<br />
3）Keep variants with 1000 genome、GnomAD（Pio <i>et al.</i>, 2021）、Exac database （AF<1%）；<br />

4）The TMB value was calculated according to the CDs length of the exon capture interval；<br />


</p>
  <p class="name_fig" style="font-size: 12px">Table 4.10 Tumor mutational burden(SNP&InDel)</p>
  <div id="tb">
    <table class="tb2">
      
        
        <tr class="tb2"><th class="tb2">Sample</th><th class="tb2">Num_mutations</th><th class="tb2">Num_nonsy</th><th class="tb2">TMB_nonsy</th></tr>
        
      
        
        <tr class="tb2"><td class="tb2">P4_T</td><td class="tb2">80</td><td class="tb2">20</td><td class="tb2">0.6257</td></tr>
        
      
        
        <tr class="tb2"><td class="tb2">P3_T</td><td class="tb2">138</td><td class="tb2">66</td><td class="tb2">2.0648</td></tr>
        
      
        
        <tr class="tb2"><td class="tb2">P2_T</td><td class="tb2">30</td><td class="tb2">9</td><td class="tb2">0.2816</td></tr>
        
      
    </table>
  </div>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Sample: Name of tumor sample</li>
      <li>Num_mutations: Total number of snvs and indel mutations in CDs region</li>
      <li>Num_nonsy: Number of snvs and indel nonsynonymous mutations</li>
      <li>TMB_nonsy: TMB</li>
    </ol>
  </div>
  </div><br/><br/>
  <p class="paragraph"><a href="../../../result/Pro/TMB" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/TMB</a></p>
</div>





<!-- ------------------------------------------ HLA---------------------------------------- -->

<div id="page">
  <p class="head"><a href="#home" title="返回首页"></a><a name="HLA"></a><hr /></p><br />
  <h4 id="goto_resultadv200">4.6.6 HLA Typing</h4>
  <p class="paragraph">
    <p>HLA is one of the most polymorphic regions in the human genome, characterized by high GC content, making direct alignment with reference genes less effective. The International Immunogenetics Information System (IMGT) is a comprehensive database focusing on vertebrate immunoglobulins, T-cell receptors (TCR), and the major histocompatibility complex (MHC). Using reference genome maps constructed from the IMGT database enables more precise alignment and analysis of target sequences.</p>
    <p>HLA-HD is employed for HLA gene typing, typically achieving four-digit precision, with a maximum of eight-digit high-resolution typing. As shown in the figure, based on the level of differentiation, HLA typing can be classified into four categories: two-digit, four-digit, six-digit, and eight-digit. Two digits represent allele groups, which generally correspond to homologous serological antigens. Four digits can distinguish different subtypes within the same group, including distinct coding proteins caused by mutations. Six digits further differentiate sequence changes from nonsynonymous mutations, while eight digits rely on information outside coding regions, identifying distinct sequences in non-coding regions. Generally, for studying the relationship between antigens and tumors, four-digit resolution is required. Due to technical limitations, achieving eight-digit resolution is often difficult.</p>




  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
      <ol style="font-size: 12px;font-family: 'SimHei';line-height: 20px">
        <li>HLA: Prefix</li>
        <li>"-": Hyphen separating the HLA prefix and gene name</li>
        <li>A: Gene locus</li>
        <li>"*": Delimiter</li>
        <li>Field1(02): Allele group</li>
        <li>":": Field separator</li>
        <li>Field2(101): Specifically expressed HLA protein</li>
        <li>Field3(01): Indicates synonymous substitutions in coding regions</li>
        <li>Field4(02): Represents mutations in non-coding regions</li>
        <li>N: Suffix indicating expression status</li>
        <li>L: Low expression compared to normal levels;</li>
        <li>S: Encodes soluble secreted proteins;</li>
        <li>C: Encodes cytoplasmic products;</li>
        <li>A: Indicates abnormal expression of the allele;</li>
        <li>Q: Questionable allele expression, where mutations affect the normal expression of other alleles.</li>
      </ol>
   </div>
   </div>
  </p>
  
  <p class="name_fig" style="font-size: 12px">Table 4.11 HLA Typing Results</p>
  <div id="tb">
    <table class="tb2">
      <tr class="tb2">
        <th class="tb2">Sample</th>
        <th class="tb2">Gene</th>
        <th class="tb2">Allele1</th>
        <th class="tb2">Allele2</th>
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">A</td>
        
        <td class="tb2">HLA-A*11:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">B</td>
        
        <td class="tb2">HLA-B*15:02:01</td>
        
        <td class="tb2">HLA-B*13:01:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">C</td>
        
        <td class="tb2">HLA-C*03:04:01</td>
        
        <td class="tb2">HLA-C*08:01:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB1</td>
        
        <td class="tb2">HLA-DRB1*15:01:01</td>
        
        <td class="tb2">HLA-DRB1*12:02:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DQA1</td>
        
        <td class="tb2">HLA-DQA1*06:01:01</td>
        
        <td class="tb2">HLA-DQA1*01:02:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DQB1</td>
        
        <td class="tb2">HLA-DQB1*06:01:01</td>
        
        <td class="tb2">HLA-DQB1*03:01:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DPA1</td>
        
        <td class="tb2">HLA-DPA1*01:03:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DPB1</td>
        
        <td class="tb2">HLA-DPB1*02:01:02</td>
        
        <td class="tb2">HLA-DPB1*21:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DMA</td>
        
        <td class="tb2">HLA-DMA*01:02</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DMB</td>
        
        <td class="tb2">HLA-DMB*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DOA</td>
        
        <td class="tb2">HLA-DOA*01:01:02</td>
        
        <td class="tb2">HLA-DOA*01:01:04</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DOB</td>
        
        <td class="tb2">HLA-DOB*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRA</td>
        
        <td class="tb2">HLA-DRA*01:02:02</td>
        
        <td class="tb2">HLA-DRA*01:01:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB2</td>
        
        <td class="tb2">HLA-DRB2*01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB3</td>
        
        <td class="tb2">HLA-DRB3*03:01:03</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB4</td>
        
        <td class="tb2">Not typed</td>
        
        <td class="tb2">Not typed</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB5</td>
        
        <td class="tb2">HLA-DRB5*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB6</td>
        
        <td class="tb2">HLA-DRB6*02:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB7</td>
        
        <td class="tb2">Not typed</td>
        
        <td class="tb2">Not typed</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB8</td>
        
        <td class="tb2">Not typed</td>
        
        <td class="tb2">Not typed</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DRB9</td>
        
        <td class="tb2">HLA-DRB9*01:02:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">DPA2</td>
        
        <td class="tb2">HLA-DPA2*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">E</td>
        
        <td class="tb2">HLA-E*01:01:01</td>
        
        <td class="tb2">HLA-E*01:03:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">F</td>
        
        <td class="tb2">HLA-F*01:01:02</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">G</td>
        
        <td class="tb2">HLA-G*01:01:03</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">H</td>
        
        <td class="tb2">HLA-H*02:01:01</td>
        
        <td class="tb2">HLA-H*02:04</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">J</td>
        
        <td class="tb2">HLA-J*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">K</td>
        
        <td class="tb2">Not typed</td>
        
        <td class="tb2">Not typed</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">L</td>
        
        <td class="tb2">HLA-L*01:01:01</td>
        
        <td class="tb2">HLA-L*01:02</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">T</td>
        
        <td class="tb2">HLA-T*03:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">V</td>
        
        <td class="tb2">HLA-V*01:01:01</td>
        
        <td class="tb2">-</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">W</td>
        
        <td class="tb2">HLA-W*03:01:01</td>
        
        <td class="tb2">HLA-W*01:01:01</td>
        
      </tr>
      
      <tr class="tb2">
        
        <td class="tb2">P2_N</td>
        
        <td class="tb2">Y</td>
        
        <td class="tb2">Not typed</td>
        
        <td class="tb2">Not typed</td>
        
      </tr>
      
    </table>
  </div>
 </div>
  


  <button onclick="hiddenShow(this)">Show Annotation</button>
     <div style="display: none;">
     <div class="comment">
       <ol style="font-size: 12px;font-family: 'SimHei';line-height: 20px">
         <li>Sample: Sample name</li>
         <li>Gene: HLA gene locus</li>
         <li>Allele1/Allele2: Allele typing results, including gene family, classification fields, and mutation information (e.g., HLA-A*02:01:01 represents a specific allele of the HLA-A gene)</li>
       </ol>
     </div>
     </div>
  
   <p class="paragraph">Results directory: <a href="../../../result/Pro/HLA" title="Click to open" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/HLA</a></p>










<!-- ------------------------------------------ 基因组变异Circos展示 ------------------------------------- -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"></a><a name="基因组变异Circos展示"></a><hr /></p><br />
  <h4 id="goto_resultadv10"><a id="基因组变异Circos展示" name="基因组变异Circos展示">4.6.7 Genomic variation Circos</a></h4>
  <p class="paragraph">We use <a href="http://circos.ca/" target=_blank>Circos</a>to show the somatic variants that occurred in the tumor samples.</p>
  <div style="text-align: center;">
      <img src="src/pictures/circos/P3_T.circos.png" style="height: 400px;" id="Circos_plot">
  </div>
  <form style="text-align: center;">
    <label id='lblSelect'>
    <select id="selectPointOfCircos" style="margin: 0; padding: 1%" onchange="circosPlot(this.options[this.options.selectedIndex].innerHTML);">

    
        <option>P2_T</option>
    
        <option>P4_T</option>
    
        <option>P3_T</option>
    


    </select>
    </label>&nbsp;&nbsp;
    <button type="button" id="pageUp" class="button button-normal button-border button-circle">
    <i class="icon-circle-arrow-up"></i>
    </button>
    <button type="button" id="pageDown" class="button button-normal button-border button-circle">
    <i class="icon-circle-arrow-down"></i>
    </button>
  </form>
    <p class="name_fig" style="font-size: 12px">Figure 4.8 Circos </p>
      <button onclick="hiddenShow(this)">Show Annotation</button>
      <div style="display: none;">
      <div class="comment">
       <ul style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
	 
	 
	 <li>Circos diagram will be made only when CNV analysis is available. The Circos diagram has four rings from outside to inside</li>
	 
	 
         <li>The first ring is the chromosome information, the number is the chromosome number, the corresponding chromosome length and karyotype</li>
         <li>In the second ring, the sequencing coverage map (blue), with each 0.5mbp as a unit, was used to calculate its average coverage</li>
         <li>In the third ring, green dots indicate the density of snpindel, and high and low indicate the density, with each 0.5mbp as a unit. Calculation method: number of snpindel within 0.5m BP /1 MBP</li>
	 
	 
         <li>In ring 4, CNV results, red indicates gain, blue indicates loss, and green indicates copy number norma</li>
	 
	 
       </ul>
      </div>
      </div>
  
  <script type="text/javascript">
    function circosPlot(title) {
      dd = document.getElementById("Circos_plot");
      dd.src = "src/pictures/circos/"+title+".circos.png";
      return 0
    }
    $(function() {
        $("#pageUp").click(function(){
            let downNode = this.nextElementSibling;
            let selectNode = this.parentNode.firstElementChild.firstElementChild;
            if (selectNode.selectedIndex == 0){
                this.style.visibility = 'hidden';
            }else{
                selectNode.selectedIndex = selectNode.selectedIndex - 1;
                circosPlot(selectNode.options[selectNode.selectedIndex].innerHTML);
                downNode.style.visibility = 'visible';
            };
        });
        $("#pageDown").click(function(){
            let upNode = this.previousElementSibling;
            let selectNode = this.parentNode.firstElementChild.firstElementChild;
            if ( selectNode.selectedIndex == selectNode.options.length - 1){
                this.style.visibility = 'hidden';
            }else{
                selectNode.selectedIndex = selectNode.selectedIndex + 1;
                circosPlot(selectNode.options[selectNode.selectedIndex].innerHTML);
                upNode.style.visibility = 'visible';
            }
        });
        $("#selectPointOfCircos").change(function(){
            let upNode = this.parentNode.nextElementSibling;
            let downNode = upNode.nextElementSibling;
            upNode.style.visibility = 'visible';
            downNode.style.visibility = 'visible';
        });
    });
  </script>
  <p class="paragraph"><a href="../../../result/Pro/Circos" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Circos</a></p>
<!--</div>-->

  
  
				<!-- ------------------------------------------ 高频CNV分析 -------------------------------------  -->
				<!-- ------------------------------------------ CNV分布 -------------------------------------  -->
				
				<h4 id="goto_resultadv5"><a id="High frequency CNV analysis" name="High frequency CNV analysis">4.6.8 High frequency CNV analysis</a></h4>
				<p class='paragraph'>Somatic mutations refer to those variants that occur in somatic cells instead of germ cells. They neither inherit from parents nor pass on to offspring, but could cause changes in the genetic structure of some contemporary cells. The driver mutations that included in somatic mutations play important roles in tumorigenesis and further evolution. Therefore, focusing on the somatic mutations is the key point of tumor research, and it is the feature that different from disease research.</p>
				<h5><a id="Distribution of CNV">4.6.8.1 Distribution of CNV in tumor samples</a></h5>
				<p class="paragraph"> We used Control-FREEC for the detection of somatic CNV and analyzed the sub allele frequency (BAF) at the SNP site. The following figure shows the distribution of CNV and BAF in the genome.</p>
				<div style="margin: auto; width: 800px; height: 280px; text-align: center;">
				<iframe src="src/adv_src/gistic_cnv_show.html" style="width: 100%; height: 100%; margin: auto;" frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no" ></iframe>
				</div>
				<p class="name_fig" style="font-size: 12px">Figure 4.9 The distribution of CNV and BAF in the genome</p>
				<p class="premark">The above figure shows the distribution of CNV throughout the entire genome. The horizontal coordinates are chromosomes 1-22, with red indicating an increase in copy number, blue indicating a decrease in copy number, and green indicating no change in copy number. The following figure shows the distribution of BAF, with orange indicating consistent distribution of AB alleles, blue indicating a preference for AB allele distribution, and BAF separation of 0,1 indicating a loss of heterozygosity (LOH).。</p>
				<p class="center"><img class="wid2" src="src/pictures/CNV.heatmap.png" alt='CNV.heatmap.png'/></p>
				<p class="name_fig" style="font-size: 12px">Figure 4.10 The distribution of CNV in tumor samples</p>
				<p class="premark">The horizontal axis represents chromosomes 1-22, and the vertical axis represents different samples. The red and blue parts in the figure represent the somatic CNVs in the individual. The red represents an increase in copy number, while the blue represents a decrease in copy number. The darker the color, the greater the change in copy count.</p>
<!--</div>-->	
<!-- ------------------------------------------ CNV重现性分析 -------------------------------------  -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="Enrichment analysis of high-frequency mutated genes"></a><hr /></p><br />
  <h5><a id="CNV reproducibility analysis">4.6.8.2 High frequency CNV analysis</a></h5>
  
  <p class="paragraph">No relevant high-frequency CNVs were screened.</p><br/><br/>
  
  <p class="paragraph"><a href="../../../result/Pro/GISTIC" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/GISTIC</a></p>
<!--</div>-->




<!-- ------------------------------------------ 肿瘤纯度和倍性分析 ------------------------------------- -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="肿瘤纯度和倍性分析"></a><hr /></p><br />
  <h4 id="goto_resultadv7">4.6.9 Purity & Ploidy Analyses</h4>
  <p class="paragraph">When DNA is extracted from mixed samples that contain both tumor and normal tissues, the proportion of tumor cells (purity) and the absolute copy number of the tumor genome (ploidy) are unknown to us. To estimate the purity and ploidy of samples, we used the software ABSOLUTE that makes use of the information about the allele frequency of somatic variants.</p>
  <p class="center">
    <p class="name_fig" style="font-size: 12px">Table 4.12 Purity and ploidy analysis results of Absolute</p>
    <iframe style="text-align: center; width: 95%; height: 224.0px;margin: auto;" src="src/tables/absolute_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Sample：sample name</li>
      <li>Call status：call status (‘call’ denotes that purity and ploidy information of the sample has been estimated, while ‘non-aneuploid’ not)</li>
      <li>Purity：the estimated purity of the sample , i.e., the proportion of tumor cells in all cells</li>
      <li>Ploidy：the estimated ploidy of the sample, i.e., the absolute copy number of the tumor genome</li>
      <li>Cancer DNA fraction：the proportion of tumor DNA</li>
      <li>Subclonal genome fraction：the fraction of subclonal genome</li>
    </ol>
  </div>
  </div><br/><br/>
  <p class="paragraph"><a href="../../../result/Pro/Absolute" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Absolute</a></p>
<!--</div>-->


<!-- ------------------------------------------ 克隆结构分析 ------------------------------------- -->

<!-- ------------------------------------------ Pyclone分析 ------------------------------------- -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="克隆结构分析"></a><hr /></p><br />
  <h4 id="goto_resultadv8">4.6.10 Intra-tumor Heterogeneity Analysis</h4>
  <p class="paragraph">
    Cancer is a disease largely driven by accumulated somatic mutations. Many of these are clonal mutations and occur in the founding cell to initiate cancer. They become uniformly present in the tumor. Others are subclonal mutations, which occur in an existing neoplastic cell and then passed on only to the subpopulation of cells. Subclonal populations thus contain all of the founding clonal mutations, as well as subclonal mutations. Intra-tumor heterogeneity refers to the heterogeneous composition of tumor cells. Deciphering the intra-tumor heterogeneity and clonal architecture may contribute to understanding of therapy resistance.
  </p>
  <p class="paragraph">
    The software<a href="https://bitbucket.org/aroth85/pyclone/wiki/browse/" target=_blank>PyClone</a>was used to perform this analysis
  </p>
  <h5><a id="Pyclone分析">4.6.10.1 Somatic mutation CCF calculations</a></h5>
  <p class="paragraph">
    CCF (cancer cell fraction), also known as cancer cell fraction of mutation, refers to the proportion of tumor cells carrying a certain mutation in all tumor cells. CCF value is the core basis for pyclone to study the clonal structure. It mainly uses the purity of tumor cells, genomic copy number changes of tumor samples and allele frequency of somatic mutations to accurately quantify the proportion of mutant tumor cells, and then analyzes the clonal structure of tumor tissues. The CCF of each mutation site in tumor cells is shown in the table below.
  </p>
  <p class="center">
    <p class="name_fig" style="font-size: 12px">Table 4.13 Somatic Mutation CCF Analysis Results</p>
    <iframe style="text-align: center; width: 95%; height: 352.0px;margin: auto;" src="src/tables/ccf_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>mutation_id: a specific site on each gene</li>
      <li>sample_id: sample name</li>
      <li>cluster_id: The clone cluster to which each point in column mutation_id belongs after clustering</li>
      <li>celluar_prevalence: the CCF value corresponding to each point in column mutation_id</li>
      <li>celluar_prevalence_std: the standard deviation of the CCF values corresponding to each point in column mutation_id</li>
      <li>variant_allele_frequency: allele mutation frequency</li>
    </ol>
  </div>
  </div>
  <h5><a id="Pyclone分析">4.6.10.2 Single sample clone structure analysis</a></h5>
  <p class="paragraph">
    The heterogeneity of cancer can be divided into inter individual heterogeneity, intra individual tumor heterogeneity, and intra individual tumor heterogeneity, as shown in the following figure. According to the selection of samples, corresponding clone evolution analysis can be conducted, and the single sample clone structure mainly studies the heterogeneity within tumor samples.
  </p>
  <p class="center">
    <img src="src/images/clone_example.png" style="max-height: 500px; max-width: 800px;" />
    <p class="name_fig" style="font-size: 12px">Figuer 4.11 Schematic diagram of tumor heterogeneity hierarchy</p>
  </p>
  <p class="paragraph">
    The existence of tumor heterogeneity may lead to drugs selectively killing only a portion of tumor cells, while surviving tumor cells may gradually evolve from subclones to primary clones. The CCF value reflects the proportion of tumor cells with this mutation, which is closer to 1, indicating that it is an early mutation that is common to all tumor cells, that is, a dominant clonal mutation; The smaller the CCF value, the more subclonal mutations are present in only some tumor cells. This analysis is implemented by default using Cyclone software, and the clone structure analysis results of the sample are shown in the following figure。
  </p>
  <div class="albumSlider">
    <div class="fullview">
      <img src='src/pictures/pyclone/P2_T.cellular_frequencies.random50.png'/>
    </div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动">
      </div>
      <div class="imglistwrap">
        <ul class="imglist">
          
          <li><a id="example2" href='src/pictures/pyclone/P2_T.cellular_frequencies.random50.png' ><img src='src/pictures/pyclone/P2_T.cellular_frequencies.random50.jpeg'/></a></li>
          
          <li><a id="example2" href='src/pictures/pyclone/P4_T.cellular_frequencies.random50.png' ><img src='src/pictures/pyclone/P4_T.cellular_frequencies.random50.jpeg'/></a></li>
          
          <li><a id="example2" href='src/pictures/pyclone/P3_T.cellular_frequencies.random50.png' ><img src='src/pictures/pyclone/P3_T.cellular_frequencies.random50.jpeg'/></a></li>
          
        </ul>
      </div>
      <div class="button moveforward" title="向下滚动">
      </div>
    </div>
  </div>
  <p class="name_fig" style="font-size: 12px">Figure 4.12 clonal analysis（Pyclone）</p>
  <p class="premark">The horizontal axis in the figure represents the frequency of mutated tumor cells (CCF), and the vertical axis represents the gene where the mutation site is located. In the figure, the frequency of mutated cells is represented in red. The closer the cell frequency of the mutation is to 1, it indicates that it is a master clone mutation, and the smaller the value of cell frequency, it indicates that it is a subclone mutation.</p><br/>
  <p class="paragraph"><a href="../../../result/Pro/Clone/pyclone" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Clone/pyclone</a></p>
<!--</div>-->










<!-- ------------------------------------------ MRT高频突变基因相关性分析 ------------------------------------- -->

<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="MRT高频突变基因相关性分析"></a><hr /></p><br />
  <h4 id="goto_resultadv11">4.6.11 Mutation Relation Test of Significantly Mutated Genes</h4>
  <p class="paragraph">Mutation relation test (MRT) attempts to reveal correlations and mutual exclusion relationships among significantly mutated genes in a pairwise fashion. Positive correlations suggest that the two genes are mutated concurrently and putatively function synergistically to promote carcinogenesis. Negative correlations imply that the two genes are mutated exclusively and mutations of the two genes may be driver events acting independently for carcinogenesis.
In this analysis, we tried to find any relationships among mutated genes by using the MRT module of the toolkit MuSic (Dees N D et al. 2012). Normally, the identified significantly mutated genes (SMGs) by the SMG module of MuSic are considered for this test. However, in the case that the number of identified SMGs is less than 2, the 50 genes most frequently affected by somatic mutations of all samples are considered for this test.</p>
  
  <p class="center">
    <p class="name_fig" style="font-size: 12px">Table 4.14 Result of MRT analysis</p>
    <iframe style="text-align: center; width: 95%; height: 352.0px;margin: auto;" src="src/tables/mrt_music_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Gene1：the gene identifier of one gene</li>
      <li>Gene2：the gene identifier of another gene</li>
      <li>CntGene1：the number of samples in which Gene1 is mutated</li>
      <li>CntGene2：the number of samples in which Gene2 is mutated</li>
      <li>AndCnt：the number of samples in which both Gene1 and Gene2 are mutated</li>
      <li>XorCnt：the number of samples in which either Gene1 or Gene2 is mutated</li>
      <li>Pvalue_And：the P-value for concurrent mutation events (high confidence level when Pvalue_And is less than 0.05)</li>
      <li>Pvalue_Xor：the P-value for mutually exclusive mutation events (high confidence level when Pvalue_Xor is less than 0.05)</li>
    </ol>
  </div>
  </div><br/><br/>
  				
<!--</div>-->
<!-- ------------------------------------------ 高频突变基因协同作用热图展示 -------------------------------------  -->
  <p class="paragraph">The figure is based on the values of "Pvalue_And" shown in the above table.</p>
  <p class="center"><img class="wid" src="src/pictures/Pvalue_And.png" alt='Pvalue_And.png' style="max-height: 600px;"/></p>
  <p class="name_fig" style="font-size: 12px">Figure 4.13 A heatmap showing concurrent relationship among significantly mutated genes</p>
  <p class="premark"> Both the x-axis and y-axis represent significantly mutated genes. The darker the color is, the more reliable the relationship is</p>
<!-- ------------------------------------------ 高频突变基因互斥作用热图展示 -------------------------------------  -->
  <p class="paragraph">The figure is based on the values of "Pvalue_Xor" shown in the above table.</p>
  <p class="center"><img class="wid" src="src/pictures/Pvalue_Xor.png" alt='Pvalue_Xor.png' style="max-height: 600px;"/></p>
  <p class="name_fig" style="font-size: 12px">图4.14 A heatmap showing mutual exclusion relationship among significantly mutated genes<</p>
  <p class="premark"> Both the x-axis and y-axis represent significantly mutated genes. The darker the color is, the more reliable the relationship is.</p>
  <p class="paragraph"><a href="../../../result/Pro/Mrt" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Mrt</a></p>
				
				
<!-- ------------------------------------------ OncodriveCLUST驱动基因预测 ------------------------------------- -->
				
<!-- ------------------------------------------ Oncodrive驱动基因预测 ------------------------------------- -->

<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="Oncodrive驱动基因预测"></a><hr /></p><br />
  <h4 id="goto_resultadv12.2">4.6.12 Oncodrive driven gene prediction</h4>
  <p class="paragraph">
    Identifying driving genes from a large number of Somatic Mutations remains one of the challenges in cancer genome research. Although some known driving genes have been obtained through databases and reported scientific literature, there are still many unknown driving genes that need to be predicted and studied. In order to comprehensively predict the driving genes related to cancer, we will analyze them from three different levels: Significantly Mutated Genes (SMG), Mutation Clustering Bias (CUST bias), and Functional Impact (FI), and integrate the results.
    <ol style="font-size:14px;font-family: '黑体-简'">
      <li>Based on SMG analysis, it refers to mining high-frequency mutated genes (RMG) from the background mutation rate, which shows the results of basic analysis of A+version of high-frequency mutated gene Music.</li>
      <li>Analysis based on mutation clustering bias refers to the tendency to cluster mutations in specific regions of a protein to form mutation clusters (CFUST bias).</li>
      <li>FI based analysis refers to the preference for functional mutation accumulation (FM bias), which focuses on the protein harmfulness caused by mutations.</li>
    </ol>
  </p>
  <h5>4.6.12.1 OncodriveCLUST</h5>
  <p class="paragraph">
    Due to the lack of selection pressure, synonymous mutations are randomly and evenly distributed throughout the entire gene, while gain of function mutations often aggregate to form mutation clusters in specific regions of the protein. The differences in distribution characteristics indicate that gain of function mutations give cancer cells a selective growth advantage, Therefore, it is under positive selection pressure during the process of tumor occurrence and development. The OncodriveCLUST analysis method (Tamborro D et al. 2013) considers the preference of driving mutations to form mutation clusters in the site distribution, and constructs a background mutation rate model using the unbiased distribution of synonymous mutations to search for possible driving mutations.
  </p>
  <h5>4.6.12.2 OncodriveFM</h5>
  <p class="paragraph">
    OncodriveFM uses SIFT, PolyPhen2 (PPH2), and MutationAssessor (MA) to score the protein toxicity of mutated genes to obtain a matrix of three FI scores. Furthermore, the average FI values for each gene (the same gene may have multiple mutation sites) corresponding to the three methods are calculated, and compared with the average FI values of the background (all samples of Somatic mutations) to calculate the P-value of FM bias for each gene. Finally, the P-values obtained from the three methods were integrated using Fisher's combined probability test to calculate their statistical significance. It is believed that genes with high FI mutation accumulation have positive selection for cancer evolution and are considered as candidate driving genes.
  </p>
  <h5>4.6.12.3 result</h5>
  <p class="paragraph">
    At present, different methods for predicting cancer driving genes have their own advantages and limitations, so combining multiple methods can provide a more comprehensive comparison and evaluation of cancer driving genes. Therefore, we will integrate the predicted results of MuSiC, OncodriveCLUST, and OncodriveFM software to comprehensively predict driving genes, in order to facilitate further research on cancer driving mechanisms.
  </p>
  
  <p class="center"><a href="src/pictures/common_driver_gene.png" target="_blank"><img src="src/pictures/common_driver_gene.png" style="max-height: 400px;" /></a></p>
  <p class="name_fig" style="font-size: 12px">Figuer 4.15 Scatter plot of shared and unique driving genes distribution</p>
  <p class="premark">Among them, the x-axis (y=0) indicates that the gene has not been detected by OncodriveFM, and the higher the x-value, the higher the reliability of the gene in OncodriveCluster software; On the y-axis (x=0), it indicates that the gene has not been detected by OncodriveCluster. The higher the y-value, the higher the reliability of the gene in OncodriveCluster software;, Yellow indicates that the gene has not been detected by Music SMG, while the redder the color, the higher the reliability of the gene in Music SMG. In addition to these three dimensions, the size of the point is directly proportional to the number of samples with mutations in the gene.</p><br/>
  <p class="name_fig" style="font-size: 12px">Table 4.15 Summary of Oncodrive Predictive Driver Gene Results</p>
  <iframe style="text-align: center; width: 95%; height: 160.0px;margin: auto;" src="src/tables/oncodrive_table.html"  frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no"></iframe><br/>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Gene：gene name</li>
      <li>OncodriveClust_pvalue：The p-value corresponding to this gene in OncodriveCluster software, where '-' indicates that the gene has not been detected by the software</li>
      <li>OncodriveClust_qvalue：The q value corresponding to this gene in OncodriveCluster software, where '-' indicates that the gene has not been detected by the software</li>
      <li>OncodriveFM_pvalue：The p-value corresponding to this gene in OncodriveFM software, where '-' indicates that the gene has not been detected by the software</li>
      <li>OncodriveFM_qvalue：The q value corresponding to this gene in OncodriveFM software, where '-' indicates that the gene has not been detected by the software</li>
      <li>SMG_pvalue：The p-value corresponding to this gene in Music SMG software, where '-' indicates that the gene has not been detected by the software</li>
      <li>SMG_qvalue：The q value corresponding to this gene in Music SMG software, where '-' indicates that the gene has not been detected by the software</li>
    </ol>
  </div>
  </div><br/><br/>
  
  <p class="paragraph"><a href="../../../result/Pro/Oncodrive" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Oncodrive</a></p>
<!--</div>-->
			

<!-- ------------------------------------------ NovoDrug靶向用药预测 ------------------------------------- -->






<!-- ------------------------------------------ 杂合性缺失分析 ------------------------------------- -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="杂合性缺失分析"></a><hr /></p><br />
  <h4 id="goto_resultadv16">4.6.13 Loss of heterozygosity</h4>
  <p class="paragraph">Loss of heterozygosity（Loss of heterozygosity, LOH） refers to the heterozygous state of the site in normal tissue, while in tumor tissue, some sites that were originally heterozygous are converted to homozygous state due to mechanisms such as changes in chromosome copy number, gene conversion, somatic recombination, and non segregation during mitosis. The result of transitioning from a heterozygous state to a homozygous state is called loss of heterozygosity. Loss of heterozygosity is generally associated with tumor suppressor genes (such as TP53), and when both alleles are present, it inhibits the occurrence of malignant tumors. When one allele is significantly abnormal, the other allele is also in an inactive state due to deletion or other reasons, and will no longer have inhibitory effects, leading to the transformation of cells into cancer cells. We use Control-FREEC software to detect the LOH region in tumors, further obtain SNV mutation sites in the LOH region through the In house process, and annotate tumor suppressor gene information.</p>
  
  <p class="center">
  <p class="name_fig" style="font-size: 12px">Table 4.16 Loss of heterozygosity analysis results</p>
  <div id="tb">
    <table class="tb2">
      
        
        <tr class="tb2"><th class="tb2">Chrom</th><th class="tb2">Start</th><th class="tb2">End</th><th class="tb2">Copynumber</th></tr>
        
      
        
        <tr class="tb2"><td class="tb2">chr19</td><td class="tb2">7,916,263</td><td class="tb2">7,928,186</td><td class="tb2">6</td></tr>
        
      
    </table>
  </div>
  </p>
  <button onclick="hiddenShow(this)">Show Annotation</button>
  <div style="display: none;">
  <div class="comment">
    <ol style="font-size: 12px;font-family: '黑体-简';line-height: 20px">
      <li>Chrom: chromosome </li>
      <li>Start:start position</li>
      <li>End: end position</li>
      <li>Copynumber: Copynumber</li>
    </ol>
  </div>
  </div><br/><br/>
  
  <p class="paragraph"><a href="../../../result/Pro/LOH" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/LOH</a></p>
<!--</div>-->





<!-- ------------------------------------------ 突变展示 -------------------------------------  -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="突变展示"></a><hr /></p><br />
  <h4 id="goto_resultadv17">4.6.14 Mutation display</h4>
  <h5><a id="高频突变基因突变位点展示">4.6.14.1 Display of high-frequency mutated gene mutation sites</a></h5>
  
  <p class="paragraph">Visually display high-frequency mutated genes, displaying the transcripts of the mutation sites annotated by ANNOVAR, including functional domains, mutation sites, and mutation types.</p>
  <div style="margin: auto; max-width: 800px; height: 420px; text-align: center;">
    <iframe src="src/adv_src/smg_mut_show.html" style="width: 100%; height: 100%; margin: auto;" frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no" ></iframe>
  </div><br/>
  <p class="name_fig" style="font-size: 12px">Figuer 4.16 High frequency mutation gene mutation display</p>
  <p class="premark">The left legend displays the mutation types and protein functional domains. Different colored blocks in the figure represent different functional domains, the numbers below the figure represent length (in amino acid units), and the vertical line represents information about point mutations</p>
<!--</div>-->
  


<!-- ------------------------------------------ 突变展示 -------------------------------------  -->
<!--<div id="page">-->
<p class="head"><a href="#home" title = "返回首页"></a><a name="驱动基因突变位点展示"></a><hr /></p><br />
  <p class="paragraph">
    Usually, mutations in oncogenes are more likely to accumulate in certain structural domains and mutate into clusters, such as KRAS. On the other hand, tumor suppressor genes tend to undergo sporadic mutations, such as TP53. To visually display the mutation characteristics and distribution of genes, we used In house software to display the distribution of mutation sites in protein functional domains. The following figure shows the specific mutation types and corresponding protein functional domains.
  </p>
  <h5><a id="驱动基因突变位点展示">4.6.14.2 Display of driving gene mutation sites</a></h5>
  
  <p class="paragraph">Visually display the detected driving genes, displaying the transcripts of the mutation sites annotated by ANNOVAR, including functional domains, mutation sites, and mutation types.</p>
  <div style="margin: auto; max-width: 800px; height: 420px; text-align: center;">
    <iframe src="src/adv_src/od_mut_show.html" style="width: 100%; height: 100%; margin: auto;" frameborder="no" border="0" marginwidth="0" marginheight="0" scrolling="no" ></iframe>
  </div><br/>
  <p class="name_fig" style="font-size: 12px">Figure 4.17 Display of driving gene mutations</p>
  <p class="premark">The left legend displays the mutation types and protein functional domains. Different colored blocks in the figure represent different functional domains, the numbers below the figure represent length (in amino acid units), and the vertical line represents information about point mutations</p>
<!--</div>-->
  


<p class="paragraph"><a href="../../../result/Pro/Mut_Onshow" title="点击打开" target="_blank">03.Result_X202XXXXXX-Z01-F001_Cancer/result/Pro/Mut_Onshow</a></p>





			<!-------------------------------------------- 参考文献 ------------------------------------------->

			<div id="page">
				<p class="head"><a href="#home" title="返回首页"></a>
					<a name="References"></a>
					<hr />
				</p><br />

				<h2>5 Appendix</h2>

				<h3>5.1 Softwares</h3>

				<p class="center">
				<p class="name_fig">Main softwares used in the analysis </p>

				<div id="tb">
					<table class="gy">

						<tr class="tb3"><th class="tb3">Analytical content</th><th class="tb3">Software</th><th class="tb3">Version</th><th class="tb3">Parameter</th><th class="tb3">Remarks</th></tr>
  
  <tr class="tb3"><td class="tb3" rowspan=1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Quality Control</td><td class="tb3">fastp</td><td class="tb3">0.23.1</td><td class="tb3">-n 15 -q 5 -u 50 -l 150</td><td class="tb3">Quality Control</td></tr>
  
  <tr class="tb3"><td class="tb3" rowspan=2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Alignment</td><td class="tb3">BWA</td><td class="tb3">0.7.17</td><td class="tb3">-t 5 -M</td><td class="tb3">Reads alignment</td></tr>
  
  <tr class="tb3"><td class="tb3">Sambamba</td><td class="tb3">v1.0.0</td><td class="tb3">-t 4</td><td class="tb3">Sort the bam file and mark the duplicated reads</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">&nbsp;&nbsp;SNPs/InDels detection</td><td class="tb3"><a href="https://www.broadinstitute.org/gatk/" target=_blank>GATK</a></td><td class="tb3">v4.3.0</td><td class="tb3">-stand-call-conf 30</td><td class="tb3">Detect and filter SNPs/InDels</td></tr>
  
  
  
  
  
  
  
  
  
  
  
  
  
  <tr class="tb3"><td class="tb3">&nbsp;&nbsp;Somatic SNPs/InDels</td><td class="tb3">MuTect、Strelka</td><td class="tb3">muTect：v2.2-25-g2a68eab，Strelka：v2.9.10</td><td class="tb3">-T MuTect -rf BadCigar</td><td class="tb3">Detect Somatic SNPs/Indels</td></tr>
  
  
  
  
  
  
  
  
  
  <tr class="tb3"><td class="tb3">&nbsp;&nbsp;Somatic CNVs detection</td><td class="tb3"><a href="http://bioinfo-out.curie.fr/projects/freec/"  target=_blank>Control-FREEC</a></td><td class="tb3">v11.4</td><td class="tb3">default</td><td class="tb3">Detect Somatic CNVs</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">&nbsp;&nbsp;Variant Annotation</td><td class="tb3"><a href="http://www.openbioinformatics.org/annovar/"  target=_blank>ANNOVAR</a></td><td class="tb3">2017June8</td><td class="tb3">default</td><td class="tb3">Structural and functional annotations of detected variations</td></tr>
  
  
  <tr class="tb3"><td class="tb3">Screening for Predisposing Genes</td><td class="tb3">in-house</td><td class="tb3"></td><td class="tb3"></td><td class="tb3">Screening for pred     isposing genes based on Cancer Gene Census database<a href="http://cancer.sanger.ac.uk/census" target=_blank>Cancer Gene Census</a>database</tr>
  
  
  <tr class="tb3"><td class="tb3">Mutational Spectrum & Mutational Signature</td><td class="tb3"><a href="http://renozao.github.io/NMF/master/index.html" target=_blank>NMF</a></td><td class="tb3">0.22</td><td class="tb3">default</td><td class="tb3">The mutation signatures of somatic mututions were obtained by the approach named Nonnegtive Matrix Fact     orization</td></tr>
  
  
  <tr class="tb3"><td class="tb3">Identification of Known Driver Genes</td><td class="tb3">in-house</td><td class="tb3"></td><td class="tb3"></td><td class="tb3">The sources of      known driver genes include<a href="http://cancer.sanger.ac.uk/cancergenome/projects/census/" target=_blank >CGC、<a href= "http://www.sciencemag.org/content/339/6127/1546.full" target=_blank >Bert Vogelstein、<a href= "http://www.nature.com/nature/journal/v502/n7471/abs/nature12634.html"target=_blank > SMG127、<a href= "http://www.nature.com/srep/2013/131002/srep02650/full/srep02650.html" target=_blank >comprehensive</a>database</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">Significantly Mutated Gene</td><td class="tb3"><a href="http://search.cpan.org/dist/Genome-Model-Tools-Music/" target=_blank>MuSiC</a></td><td class="tb3">Genome-Model-Tools-Music-0.04</td><td class="tb3">default</td><td class="tb3">Significantly Mutated Genes (SMGs) identification and pathway enrichment analysis on SMGs</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">Tumor Mutational burden</td><td class="tb3">in-house</td><td class="tb3"></td><td class="tb3"></td><td class="tb3">Analysis of tumor mutational burden</td></tr>
  


  
  <tr class="tb3"><td class="tb3">&nbsp;&nbsp;HLA</td><td class="tb3">HLAHD</td><td class="tb3">1.2.0.1</td><td class="tb3">-t 7 -m 100 -c 1.0</td><td class="tb3">HLA type</td></tr>
  



  
  <tr class="tb3"><td class="tb3"> Identification of Driver Somatic CNVs</td><td class="tb3"><a href="http://www.broadinstitute.org/cancer/cga/gistic" target=_blank>GISTIC</a></td><td class="tb3">GISTIC2.0</td><td class="tb3">default</td><td class="tb3">Driver genes targeted by CNVs</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">Tumor Purity & Ploidy Estimation</td><td class="tb3"><a href="http://www.broadinstitute.org/cancer/cga/absolute" target=_blank>ABSOLUTE</a></td><td class="tb3">1.0.6</td><td class="tb3">default</td><td class="tb3">Purity and ploidy analyses of tumor samples</td></tr>
  
  
  <tr class="tb3"><td class="tb3">Intra-tumor Heterogeneity Analysis</td><td class="tb3"><a href="https://bitbucket.org/aroth85/pyclone/wiki/browse/" target=_blank>PyClone</a></td><td class="tb3">0.12.7</td><td class="tb3">default</td><td class="tb3">Clone analysis of tumor samples</td></tr>
  
  
  
  
  
  <tr class="tb3"><td class="tb3">Circos diagram displaying</td><td class="tb3"><a href="http://circos.ca/" target=_blank>circos</a></td><td class="tb3">0.64</td><td class="tb3">default</td><td class="tb3">Circos diagram displaying</td></tr>
  
  
  <tr class="tb3"><td class="tb3">Mutation Relation Test of Significantly Mutated Genes</td><td class="tb3">MuSiC</td><td class="tb3">Genome-Model-Tools-Music-0.04</td><td class="tb3">smg  --max-fdr 0.200000<td class="tb3">Identification of relationships among significantly mutated genes</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">Identification of Driver Genes </td><td class="tb3">OncodriveClust, OncodriveFM, Music smg</td><td class="tb3"></td><td class="tb3">default</td><td class="tb3">Synthesize multiple methods to predict driver genes more comprehensively</td></tr>
  
  
  
  <tr class="tb3"><td class="tb3">Loss of heterozygosity analysis</td><td class="tb3">in-house</td><td class="tb3"></td><td class="tb3"></td><td class="tb3">Loss of heterozygosity analysis</td></tr>
  
  
  <tr class="tb3"><td class="tb3">Mutation Site Displaying</td><td class="tb3">in-house</td><td class="tb3"></td><td class="tb3"></td><td class="tb3">SMG mutation and driver mutation Site Displaying</td></tr>
  
  

					</table>
				</div>
				</p><br />


				<h3>5.2 Method</h3>

				<p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
				
				<p class="paragraph">File catalog: <a href="src/method/WES_method.pdf" title="Click to open"
						target="_blank">Method</a></p><br />
				

				<h3>5.3 Result File Decompression Method and Format Description</h3>

				<p class="center">
				<div id="tb">
					<table class="gy">

						<tr>
							<th>File Type</th>
							<th>File Description</th>
							<th>Way to Open</th>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.tar files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: tar -xvf *.tar</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software: WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.gz files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: gzip -d *.gz</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software:WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>*.zip files</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: unzip *.zip</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Decompress software:WinRAR, 7-Zip</td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>file.fq/fastq</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: less *.fq.gz or more</td>
						</tr>
						<tr>
							<td class="tb2">Windows</td>
							<td class="tb2">Software:Editplus/Notepad++ <I>et al.</I></td>
						</tr>
						<tr class="tb2">
							<td class="tb2" rowspan=2>file.bam</td>
							<td class="tb2">Unix/Linux/Mac</td>
							<td class="tb2">cmd: samtools view/tview *.bam or more</td>
						</tr>
					</table>
				</div>
			</div>

			<div id="page">
			<br />
			<br />

				<h2>6 References</h2>


				<p class="ref">Abecasis GR, Auton A, <I>et al.</I> An integrated map of genetic variation from 1,092 human genomes.
					Nature. 2012;491(7422):56-65. doi:10.1038/nature11632 (1000 Genomes)</br></br>

					Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using
					PolyPhen-2. Curr Protoc Hum Genet. 2013;Chapter 7:Unit7.20. doi:10.1002/0471142905.hg0720s76
					(PolyPhen)</br></br>

					Boeva V, Popova T, Bleakley K, <I>et al.</I> Control-FREEC: a tool for assessing copy number and allelic content
					using next-generation sequencing data. Bioinformatics. 2012;28(3):423-425.
					doi:10.1093/bioinformatics/btr670 (Control-FREEC)</br></br>

					Buniello A, MacArthur JAL, Cerezo M, <I>et al.</I> The NHGRI-EBI GWAS Catalog of published genome-wide
					association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res.
					2019;47(D1):D1005-D1012. doi:10.1093/nar/gky1120 (GWAS Catalog)</br></br>

					Chun S, Fay JC. Identification of deleterious mutations within three human genomes. Genome Res.
					2009;19(9):1553-1561. doi:10.1101/gr.092619.109 (LRT)</br></br>

					Cibulskis K, Lawrence MS, Carter SL, <I>et al.</I> Sensitive detection of somatic point mutations in impure and
					heterogeneous cancer samples. Nat Biotechnol. 2013;31(3):213-219. doi:10.1038/nbt.2514 (muTect)</br></br>


					DePristo MA, Banks E, Poplin R, <I>et al.</I> A framework for variation discovery and genotyping using
					next-generation DNA sequencing data. Nat Genet. 2011;43(5):491-498. doi:10.1038/ng.806 (GATK)</br></br>

					Frankish A, Diekhans M, Jungreis I, <I>et al.</I> GENCODE 2021. Nucleic Acids Res. 2021;49(D1):D916-D923.
					doi:10.1093/nar/gkaa1087 (GENCODE)</br></br>

					Garber M, Guttman M, Clamp M, Zody MC, Friedman N, Xie X. Identifying novel constrained elements by
					exploiting biased substitution patterns. Bioinformatics. 2009;25(12):i54-i62.
					doi:10.1093/bioinformatics/btp190 (SiPhy)</br> </br>

					Hamosh A, Scott AF, Amberger JS, Bocchini CA, McKusick VA. Online Mendelian Inheritance in Man (OMIM), a
					knowledgebase of human genes and genetic disorders. Nucleic Acids Res. 2005;33(Database issue):D514-D517.
					doi:10.1093/nar/gki033 (OMIM)</br></br>

					Harris MA, Clark J, Ireland A, <I>et al.</I> The Gene Ontology (GO) database and informatics resource. Nucleic
					Acids Res. 2004;32(Database issue):D258-D261. doi:10.1093/nar/gkh036 (GO)</br></br>

					Huber CD, Kim BY, Lohmueller KE. Population genetic models of GERP scores suggest pervasive turnover of
					constrained sites across mammalian evolution. PLoS Genet. 2020;16(5):e1008827. Published 2020 May 29.
					doi:10.1371/journal.pgen.1008827 (GERP)</br></br>

					Jassal B, Matthews L, Viteri G, <I>et al.</I> The reactome pathway knowledgebase. Nucleic Acids Res.
					2020;48(D1):D498-D503. doi:10.1093/nar/gkz1031 (Reactome)</br></br>

					Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27-30.
					doi:10.1093/nar/28.1.27 (KEGG PATHWAY)</br></br>

					Kent WJ, Sugnet CW, Furey TS, <I>et al.</I> The human genome browser at UCSC. Genome Res. 2002;12(6):996-1006.
					doi:10.1101/gr.229102 (UCSC)</br></br>

					Kobayashi Y, Yang S, Nykamp K, Garcia J, Lincoln SE, Topper SE. Pathogenic variant burden in the ExAC
					database: an empirical approach to evaluating population data for clinical variant interpretation. Genome
					Med. 2017;9(1):13. Published 2017 Feb 6. doi:10.1186/s13073-017-0403-7 (ExAc)</br></br>

					Li H, Durbin R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics.
					2010;26(5):589-595. doi:10.1093/bioinformatics/btp698 (BWA_MEM)</br></br>

					Ng PC, Henikoff S. SIFT: Predicting amino acid changes that affect protein function. Nucleic Acids Res.
					2003;31(13):3812-3814. doi:10.1093/nar/gkg509 (SIFT)</br></br>

					O'Leary NA, Wright MW, Brister JR, <I>et al.</I> Reference sequence (RefSeq) database at NCBI: current status,
					taxonomic expansion, and functional annotation. Nucleic Acids Res. 2016;44(D1):D733-D745.
					doi:10.1093/nar/gkv1189 (RefSeq)</br></br>

					Pio MG, Siffo S, Scheps KG, <I>et al.</I> Curating the gnomAD database: Report of novel variants in the
					thyrogobulin gene using in silico bioinformatics algorithms. Mol Cell Endocrinol. 2021;534:111359.
					doi:10.1016/j.mce.2021.111359 (gnomAD)</br></br>

					Pollard KS, Hubisz MJ, Rosenbloom KR, Siepel A. Detection of nonneutral substitution rates on mammalian
					phylogenies. Genome Res. 2010;20(1):110-121. doi:10.1101/gr.097857.109 (phyloP)</br></br>

					Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. CADD: predicting the deleteriousness of variants
					throughout the human genome. Nucleic Acids Res. 2019;47(D1):D886-D894. doi:10.1093/nar/gky1016
					(CADD)</br></br>

					Reva B, Antipin Y, Sander C. Predicting the functional impact of protein mutations: application to cancer
					genomics. Nucleic Acids Res. 2011;39(17):e118. doi:10.1093/nar/gkr407 (MutationAssessor)</br></br>

					Schaefer CF, Anthony K, Krupa S, <I>et al.</I> PID: the Pathway Interaction Database. Nucleic Acids Res.
					2009;37(Database issue):D674-D679. doi:10.1093/nar/gkn653 (PID)</br></br>

					Rausch T, Zichner T, Schlattl A, Stutz AM, Benes V, Korbel JO. DELLY: structural variant discovery by
					integrated paired-end and split-read analysis. Bioinformatics. 2012;28(18):i333-i339.
					doi:10.1093/bioinformatics/bts378 (DELLY)</br></br>

					Saunders CT, Wong WS, Swamy S, Becq J, Murray LJ, Cheetham RK. Strelka: accurate somatic small-variant
					calling from sequenced tumor-normal sample pairs. Bioinformatics. 2012;28(14):1811-1817.
					doi:10.1093/bioinformatics/bts271 (Strelka)</br></br>

					Sherry ST, Ward MH, Kholodov M, <I>et al.</I> dbSNP: the NCBI database of genetic variation. Nucleic Acids Res.
					2001;29(1):308-311. doi:10.1093/nar/29.1.308 (dbSNP)</br></br>

					Shihab HA, Gough J, Cooper DN, <I>et al.</I> Predicting the functional, molecular, and phenotypic consequences
					of amino acid substitutions using hidden Markov models. Hum Mutat. 2013;34(1):57-65.
					doi:10.1002/humu.22225 (FATHMM)</br></br>

					Steinhaus R, Proft S, Schuelke M, Cooper DN, Schwarz JM, Seelow D. MutationTaster2021. Nucleic Acids Res.
					2021;49(W1):W446-W451. doi:10.1093/nar/gkab266 (MutationTaster)</br></br>

					Stenson PD, Mort M, Ball EV, <I>et al.</I> The Human Gene Mutation Database (HGMD): optimizing its use in a
					clinical diagnostic or research setting. Hum Genet. 2020;139(10):1197-1207.
					doi:10.1007/s00439-020-02199-3 (HGMD)</br></br>

					Tarasov A, Vilella AJ, Cuppen E, Nijman IJ, Prins P. Sambamba: fast processing of NGS alignment formats.
					Bioinformatics. 2015;31(12):2032-2034. doi:10.1093/bioinformatics/btv098 (Sambamba)</br></br>

					Tate JG, Bamford S, Jubb HC, <I>et al.</I> COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids
					Res. 2019;47(D1):D941-D947. doi:10.1093/nar/gky1015 (COSMIC)</br></br>

					Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput
					sequencing data. Nucleic Acids Res. 2010;38(16):e164. doi:10.1093/nar/gkq603 (ANNOVAR)</br></br>
					
					<p style="font-size: 19px; font-weight: bold;">Advanced Analysis References</p>
  
  <p class="ref">
  
  Alexandrov LB, Nik-Zainal S, Wedge DC, Campbell PJ, Stratton MR. Deciphering signatures of mutational processes operative in human cancer. Cell Rep. 2013;3(1):246-259. doi:10.1016/j.celrep.2012.12.008 (NMF)</br></br>
  Alexandrov LB, Nik-Zainal S, Wedge DC, et al. Signatures of mutational processes in human cancer [published correction appears in Nature. 2013 Oct 10;502(7470):258. Imielinsk, Marcin [corrected to Imielinski, Marcin]]. Nature. 2013;500(7463):415-421. doi:10.1038/nature12477 (Signature)</br></br>
  
  
  Carter SL, Cibulskis K, Helman E, et al. Absolute quantification of somatic DNA alterations in human cancer. Nat Biotechnol. 2012;30(5):413-421. doi:10.1038/nbt.2203 (Absolute)</br></br>
  
  
  Chalmers ZR, Connelly CF, Fabrizio D, et al. Analysis of 100,000 human cancer genomes reveals the landscape of tumor mutational burden. Genome Med. 2017;9(1):34. Published 2017 Apr 19. doi:10.1186/s13073-017-0424-2 (TMB)</br></br>
  Chan TA, Yarchoan M, Jaffee E, et al. Development of tumor mutation burden as an immunotherapy biomarker: utility for the oncology clinic. Ann Oncol. 2019;30(1):44-56. doi:10.1093/annonc/mdy495 (TMB)</br></br>
  
  
  Dees ND, Zhang Q, Kandoth C, et al. MuSiC: identifying mutational significance in cancer genomes. Genome Res. 2012;22(8):1589-1598. doi:10.1101/gr.134635.111 (MuSiC)</br></br>
  
  
  Karczewski KJ, Weisburd B, Thomas B, et al. The ExAC browser: displaying reference data information from over 60 000 exomes. Nucleic Acids Res. 2017;45(D1):D840-D845. doi:10.1093/nar/gkw971 (ExAc)</br></br>
  Martínez-Jiménez F, Muiños F, Sentís I, et al. A compendium of mutational cancer driver genes. Nat Rev Cancer. 2020;20(10):555-572. doi:10.1038/s41568-020-0290-x (intOGen)</br></br>
  
  
  Mermel CH, Schumacher SE, Hill B, Meyerson ML, Beroukhim R, Getz G. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 2011;12(4):R41. doi:10.1186/gb-2011-12-4-r41 (GISTIC)</br></br>
  
  
  Pio MG, Siffo S, Scheps KG, et al. Curating the gnomAD database: Report of novel variants in the thyrogobulin gene using in silico bioinformatics algorithms. Mol Cell Endocrinol. 2021;534:111359. doi:10.1016/j.mce.2021.111359 (gnomAD)</br></br>
  
  
  
  Rizvi NA, Hellmann MD, Snyder A, et al. Cancer immunology. Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer. Science. 2015;348(6230):124-128. doi:10.1126/science.aaa1348</br></br>
  Rosenberg JE, Hoffman-Censits J, Powles T, et al. Atezolizumab in patients with locally advanced and metastatic urothelial carcinoma who have progressed following treatment with platinum-based chemotherapy: a single-arm, multicentre, phase 2 trial. Lancet. 2016;387(10031):1909-1920. doi:10.1016/S0140-6736(16)00561-4</br></br>
  
  
  Roth A, Khattra J, Yap D, et al. PyClone: statistical inference of clonal population structure in cancer. Nat Methods. 2014;11(4):396-398. doi:10.1038/nmeth.2883 (PyClone)</br></br>
  
  
  Schaefer CF, Anthony K, Krupa S, et al. PID: the Pathway Interaction Database. Nucleic Acids Res. 2009;37(Database issue):D674-D679. doi:10.1093/nar/gkn653 (PID)</br></br>
  
  
  Schumacher TN, Schreiber RD. Neoantigens in cancer immunotherapy. Science. 2015;348(6230):69-74. doi:10.1126/science.aaa4971</br></br>
  
  
  
  Tate JG, Bamford S, Jubb HC, et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 2019;47(D1):D941-D947. doi:10.1093/nar/gky1015 (COSMIC)</br></br>
  
  
  Wendl MC, Wallis JW, Lin L, et al. PathScan: a tool for discerning mutational significance in groups of putative cancer genes. Bioinformatics. 2011;27(12):1595-1602. doi:10.1093/bioinformatics/btr193 (PathScan)</br></br>
  
  </p>
  
  


				</p>
			</div>

			<!-- shan diao 3 ge div-->
		</div>
</body>

</html>





