
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" >
<html>   

<head>
<TITLE>Novogene 16S Amplicon QIIME2 Analysis Report</TITLE>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="icon" href="src/images/bitbug_favicon.ico" type="image/x-icons">

<link rel="stylesheet" href="src/css/jquery-ui.css" />
<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />
<link rel="stylesheet" href="src/css/jquery.tocify.css" />
<link rel="stylesheet" href="src/css/style.css"/>
<link rel="stylesheet" href="src/css/base.css"/>
<link rel="stylesheet" href="src/css/gallery.css"/>
<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>
<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>
<script src="src/js/scrollTop.js" type="text/javascript"></script>
<script type="text/javascript">
	$(function () {
		var options = {
			selectors: "h2,h3,h4,h5",
			theme: "bootstrap",
			scrollTo: "60"
		};
		var toc = $("#toc").tocify(options).data("toc-tocify");
	});
</script>



<script type="text/javascript">
function trans(img_div_id) {
img_div = $("#" + img_div_id)
img = $("#" + img_div_id + " img").first();
if (img_div.hasClass("fullscreen-div")) {
img_div.removeClass("fullscreen-div");
img.removeClass("fullscreen-img");
}
else {
img_div.addClass("fullscreen-div");
img.addClass("fullscreen-img");
}
}
function filter_display() {
return $(this).css("display") != "none";
}
function show_default() {
var list_group_id = "#" + $(this).attr("id");
if ($(list_group_id + " a.active").css("display") == "none") {
var list_tab = $(list_group_id + " a.active");
var pan_id = list_tab.attr("href");
$(pan_id).removeClass("active show");
list_tab.removeClass("active");
}
if ($(list_group_id + " a.active").filter(filter_display).length == 0) {
$(list_group_id + " a").filter(filter_display).first().tab("show");
}
}
$.fn.show_default = show_default
function next(botton_id) {
var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();
while (present_tab.next().length == 1) {
if (present_tab.next().css("display") != "none") {
present_tab.next().tab("show");
break;
}
else {
present_tab = present_tab.next();
}
}
}
function prev(botton_id) {
var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();
while (present_tab.prev().length == 1) {
if (present_tab.prev().css("display") != "none") {
present_tab.prev().tab("show");
break;
}
else {
present_tab = present_tab.prev();
}
}
}
$(document).ready(function () {
$(".form-control").on("keyup", function () {
var value = $(this).val().toLowerCase();
var input_box_id = "#" + $(this).attr("id");
var list_group_id = input_box_id.replace("_InputList", "_list-tab");
$(list_group_id + " a").filter(function () {
$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);
});
$(list_group_id).show_default();
});
$(".list-group").each(show_default);
});
</script>

<script type="text/javascript">
            var pageSize1 = 15;    //每页显示的记录条数
            var curPage1=0;        //当前页
            var lastPage1;        //最后页
            var direct1=0;        //方向
            var len1;            //总行数
            var page1;           //总页数
            var begin1;
            var end1;

            $(document).ready(function display1(){
                len1 =$("#mytable1 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page1=len1 % pageSize1==0 ? len1/pageSize1 : Math.floor(len1/pageSize1)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage1=1;    // 设置当前为第一页
                displayPage1(1);//显示第一页
                document.getElementById("btn01").innerHTML="Current: " + curPage1 + "/" + page1 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl1").innerHTML="Total items: " + len1 + "";        // 显示数据量

                $("#btn11").click(function firstPage1(){    // 首页
                    curPage1=1;
                    direct1 = 0;
                    displayPage1();
                });
                $("#btn21").click(function frontPage1(){    // 上一页
                    direct1=-1;
                    displayPage1();
                });
                $("#btn31").click(function nextPage1(){    // 下一页
                    direct1=1;
                    displayPage1();
                });
                $("#btn41").click(function lastPage1(){    // 尾页
                    curPage1=page1;
                    direct1 = 0;
                    displayPage1();
                });
                $("#btn51").click(function changePage1(){    // 转页
                    curPage1=document.getElementById("changePage1").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage1)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage1 > page1) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct1 = 0;
                    displayPage1();
                });


                $("#pageSizeSet1").click(function setPageSize1(){    // 设置每页显示多少条记录
                    pageSize1 = document.getElementById("pageSize1").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len1 = $("#mytable1 tr").length - 1;
                    page1 = len1 % pageSize1==0 ? len1/pageSize1 : Math.floor(len1/pageSize1)+1;//根据记录条数，计算页数
                    curPage1=1;        //当前页
                    direct1=0;        //方向
                    firstPage1();
                });
            });

            function displayPage1(){
                if(curPage1 <=1 && direct1==-1){
                    direct1=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage1 >= page1 && direct1==1) {
                    direct1=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage1 = curPage1;

                // 修复当len=1时，curPage计算得0的bug
                if (len1 > pageSize1) {
                    curPage1 = ((curPage1 + direct1 + len1) % len1);
                } else {
                    curPage1 = 1;
                }
                document.getElementById("btn01").innerHTML="Current: " + curPage1 + "/" + page1 + " page  | ";        // 显示当前多少页

                begin1=(curPage1-1)*pageSize1 + 1;// 起始记录号
                end1 = begin1 + 1*pageSize1 - 1;    // 末尾记录号


                if(end1 > len1 ) end1=len1;
                $("#mytable1 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable1 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin1 && i<=end1) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize2 = 15;    //每页显示的记录条数
            var curPage2=0;        //当前页
            var lastPage2;        //最后页
            var direct2=0;        //方向
            var len2;            //总行数
            var page2;           //总页数
            var begin2;
            var end2;

            $(document).ready(function display2(){
                len2 =$("#mytable2 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page2=len2 % pageSize2==0 ? len2/pageSize2 : Math.floor(len2/pageSize2)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage2=1;    // 设置当前为第一页
                displayPage2(1);//显示第一页
                document.getElementById("btn02").innerHTML="Current: " + curPage2 + "/" + page2 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl2").innerHTML="Total items: " + len2 + "";        // 显示数据量

                $("#btn12").click(function firstPage2(){    // 首页
                    curPage2=1;
                    direct2 = 0;
                    displayPage2();
                });
                $("#btn22").click(function frontPage2(){    // 上一页
                    direct2=-1;
                    displayPage2();
                });
                $("#btn32").click(function nextPage2(){    // 下一页
                    direct2=1;
                    displayPage2();
                });
                $("#btn42").click(function lastPage2(){    // 尾页
                    curPage2=page2;
                    direct2 = 0;
                    displayPage2();
                });
                $("#btn52").click(function changePage2(){    // 转页
                    curPage2=document.getElementById("changePage2").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage2)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage2 > page2) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct2 = 0;
                    displayPage2();
                });


                $("#pageSizeSet2").click(function setPageSize2(){    // 设置每页显示多少条记录
                    pageSize2 = document.getElementById("pageSize2").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len2 = $("#mytable2 tr").length - 1;
                    page2 = len2 % pageSize2==0 ? len2/pageSize2 : Math.floor(len2/pageSize2)+1;//根据记录条数，计算页数
                    curPage2=1;        //当前页
                    direct2=0;        //方向
                    firstPage2();
                });
            });

            function displayPage2(){
                if(curPage2 <=1 && direct2==-1){
                    direct2=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage2 >= page2 && direct2==1) {
                    direct2=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage2 = curPage2;

                // 修复当len=1时，curPage计算得0的bug
                if (len2 > pageSize2) {
                    curPage2 = ((curPage2 + direct2 + len2) % len2);
                } else {
                    curPage2 = 1;
                }
                document.getElementById("btn02").innerHTML="Current: " + curPage2 + "/" + page2 + " page  | ";        // 显示当前多少页

                begin2=(curPage2-1)*pageSize2 + 1;// 起始记录号
                end2 = begin2 + 1*pageSize2 - 1;    // 末尾记录号


                if(end2 > len2 ) end2=len2;
                $("#mytable2 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable2 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin2 && i<=end2) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize3 = 15;    //每页显示的记录条数
            var curPage3=0;        //当前页
            var lastPage3;        //最后页
            var direct3=0;        //方向
            var len3;            //总行数
            var page3;           //总页数
            var begin3;
            var end3;

            $(document).ready(function display3(){
                len3 =$("#mytable3 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page3=len3 % pageSize3==0 ? len3/pageSize3 : Math.floor(len3/pageSize3)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage3=1;    // 设置当前为第一页
                displayPage3(1);//显示第一页
                document.getElementById("btn03").innerHTML="Current: " + curPage3 + "/" + page3 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl3").innerHTML="Total items: " + len3 + "";        // 显示数据量

                $("#btn13").click(function firstPage3(){    // 首页
                    curPage3=1;
                    direct3 = 0;
                    displayPage3();
                });
                $("#btn23").click(function frontPage3(){    // 上一页
                    direct3=-1;
                    displayPage3();
                });
                $("#btn33").click(function nextPage3(){    // 下一页
                    direct3=1;
                    displayPage3();
                });
                $("#btn43").click(function lastPage3(){    // 尾页
                    curPage3=page3;
                    direct3 = 0;
                    displayPage3();
                });
                $("#btn53").click(function changePage3(){    // 转页
                    curPage3=document.getElementById("changePage3").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage3)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage3 > page3) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct3 = 0;
                    displayPage3();
                });


                $("#pageSizeSet3").click(function setPageSize3(){    // 设置每页显示多少条记录
                    pageSize3 = document.getElementById("pageSize3").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len3 = $("#mytable3 tr").length - 1;
                    page3 = len3 % pageSize3==0 ? len3/pageSize3 : Math.floor(len3/pageSize3)+1;//根据记录条数，计算页数
                    curPage3=1;        //当前页
                    direct3=0;        //方向
                    firstPage3();
                });
            });

            function displayPage3(){
                if(curPage3 <=1 && direct3==-1){
                    direct3=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage3 >= page3 && direct3==1) {
                    direct3=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage3 = curPage3;

                // 修复当len=1时，curPage计算得0的bug
                if (len3 > pageSize3) {
                    curPage3 = ((curPage3 + direct3 + len3) % len3);
                } else {
                    curPage3 = 1;
                }
                document.getElementById("btn03").innerHTML="Current: " + curPage3 + "/" + page3 + " page  | ";        // 显示当前多少页

                begin3=(curPage3-1)*pageSize3 + 1;// 起始记录号
                end3 = begin3 + 1*pageSize3 - 1;    // 末尾记录号


                if(end3 > len3 ) end3=len3;
                $("#mytable3 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable3 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin3 && i<=end3) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize4 = 15;    //每页显示的记录条数
            var curPage4=0;        //当前页
            var lastPage4;        //最后页
            var direct4=0;        //方向
            var len4;            //总行数
            var page4;           //总页数
            var begin4;
            var end4;

            $(document).ready(function display4(){
                len4 =$("#mytable4 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page4=len4 % pageSize4==0 ? len4/pageSize4 : Math.floor(len4/pageSize4)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage4=1;    // 设置当前为第一页
                displayPage4(1);//显示第一页
                document.getElementById("btn04").innerHTML="Current: " + curPage4 + "/" + page4 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl4").innerHTML="Total items: " + len4 + "";        // 显示数据量

                $("#btn14").click(function firstPage4(){    // 首页
                    curPage4=1;
                    direct4 = 0;
                    displayPage4();
                });
                $("#btn24").click(function frontPage4(){    // 上一页
                    direct4=-1;
                    displayPage4();
                });
                $("#btn34").click(function nextPage4(){    // 下一页
                    direct4=1;
                    displayPage4();
                });
                $("#btn44").click(function lastPage4(){    // 尾页
                    curPage4=page4;
                    direct4 = 0;
                    displayPage4();
                });
                $("#btn54").click(function changePage4(){    // 转页
                    curPage4=document.getElementById("changePage4").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage4)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage4 > page4) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct4 = 0;
                    displayPage4();
                });


                $("#pageSizeSet4").click(function setPageSize4(){    // 设置每页显示多少条记录
                    pageSize4 = document.getElementById("pageSize4").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len4 = $("#mytable4 tr").length - 1;
                    page4 = len4 % pageSize4==0 ? len4/pageSize4 : Math.floor(len4/pageSize4)+1;//根据记录条数，计算页数
                    curPage4=1;        //当前页
                    direct4=0;        //方向
                    firstPage4();
                });
            });

            function displayPage4(){
                if(curPage4 <=1 && direct4==-1){
                    direct4=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage4 >= page4 && direct4==1) {
                    direct4=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage4 = curPage4;

                // 修复当len=1时，curPage计算得0的bug
                if (len4 > pageSize4) {
                    curPage4 = ((curPage4 + direct4 + len4) % len4);
                } else {
                    curPage4 = 1;
                }
                document.getElementById("btn04").innerHTML="Current: " + curPage4 + "/" + page4 + " page  | ";        // 显示当前多少页

                begin4=(curPage4-1)*pageSize4 + 1;// 起始记录号
                end4 = begin4 + 1*pageSize4 - 1;    // 末尾记录号


                if(end4 > len4 ) end4=len4;
                $("#mytable4 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable4 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin4 && i<=end4) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize5 = 15;    //每页显示的记录条数
            var curPage5=0;        //当前页
            var lastPage5;        //最后页
            var direct5=0;        //方向
            var len5;            //总行数
            var page5;           //总页数
            var begin5;
            var end5;

            $(document).ready(function display5(){
                len5 =$("#mytable5 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page5=len5 % pageSize5==0 ? len5/pageSize5 : Math.floor(len5/pageSize5)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage5=1;    // 设置当前为第一页
                displayPage5(1);//显示第一页
                document.getElementById("btn05").innerHTML="Current: " + curPage5 + "/" + page5 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl5").innerHTML="Total items: " + len5 + "";        // 显示数据量

                $("#btn15").click(function firstPage5(){    // 首页
                    curPage5=1;
                    direct5 = 0;
                    displayPage5();
                });
                $("#btn25").click(function frontPage5(){    // 上一页
                    direct5=-1;
                    displayPage5();
                });
                $("#btn35").click(function nextPage5(){    // 下一页
                    direct5=1;
                    displayPage5();
                });
                $("#btn45").click(function lastPage5(){    // 尾页
                    curPage5=page5;
                    direct5 = 0;
                    displayPage5();
                });
                $("#btn55").click(function changePage5(){    // 转页
                    curPage5=document.getElementById("changePage5").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage5)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage5 > page5) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct5 = 0;
                    displayPage5();
                });


                $("#pageSizeSet5").click(function setPageSize5(){    // 设置每页显示多少条记录
                    pageSize5 = document.getElementById("pageSize5").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len5 = $("#mytable5 tr").length - 1;
                    page5 = len5 % pageSize5==0 ? len5/pageSize5 : Math.floor(len5/pageSize5)+1;//根据记录条数，计算页数
                    curPage5=1;        //当前页
                    direct5=0;        //方向
                    firstPage5();
                });
            });

            function displayPage5(){
                if(curPage5 <=1 && direct5==-1){
                    direct5=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage5 >= page5 && direct5==1) {
                    direct5=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage5 = curPage5;

                // 修复当len=1时，curPage计算得0的bug
                if (len5 > pageSize5) {
                    curPage5 = ((curPage5 + direct5 + len5) % len5);
                } else {
                    curPage5 = 1;
                }
                document.getElementById("btn05").innerHTML="Current: " + curPage5 + "/" + page5 + " page  | ";        // 显示当前多少页

                begin5=(curPage5-1)*pageSize5 + 1;// 起始记录号
                end5 = begin5 + 1*pageSize5 - 1;    // 末尾记录号


                if(end5 > len5 ) end5=len5;
                $("#mytable5 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable5 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin5 && i<=end5) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize6 = 15;    //每页显示的记录条数
            var curPage6=0;        //当前页
            var lastPage6;        //最后页
            var direct6=0;        //方向
            var len6;            //总行数
            var page6;           //总页数
            var begin6;
            var end6;

            $(document).ready(function display6(){
                len6 =$("#mytable6 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page6=len6 % pageSize6==0 ? len6/pageSize6 : Math.floor(len6/pageSize6)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage6=1;    // 设置当前为第一页
                displayPage6(1);//显示第一页
                document.getElementById("btn06").innerHTML="Current: " + curPage6 + "/" + page6 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl6").innerHTML="Total items: " + len6 + "";        // 显示数据量

                $("#btn16").click(function firstPage6(){    // 首页
                    curPage6=1;
                    direct6 = 0;
                    displayPage6();
                });
                $("#btn26").click(function frontPage6(){    // 上一页
                    direct6=-1;
                    displayPage6();
                });
                $("#btn36").click(function nextPage6(){    // 下一页
                    direct6=1;
                    displayPage6();
                });
                $("#btn46").click(function lastPage6(){    // 尾页
                    curPage6=page6;
                    direct6 = 0;
                    displayPage6();
                });
                $("#btn56").click(function changePage6(){    // 转页
                    curPage6=document.getElementById("changePage6").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage6)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage6 > page6) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct6 = 0;
                    displayPage6();
                });


                $("#pageSizeSet6").click(function setPageSize6(){    // 设置每页显示多少条记录
                    pageSize6 = document.getElementById("pageSize6").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len6 = $("#mytable6 tr").length - 1;
                    page6 = len6 % pageSize6==0 ? len6/pageSize6 : Math.floor(len6/pageSize6)+1;//根据记录条数，计算页数
                    curPage6=1;        //当前页
                    direct6=0;        //方向
                    firstPage6();
                });
            });

            function displayPage6(){
                if(curPage6 <=1 && direct6==-1){
                    direct6=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage6 >= page6 && direct6==1) {
                    direct6=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage6 = curPage6;

                // 修复当len=1时，curPage计算得0的bug
                if (len6 > pageSize6) {
                    curPage6 = ((curPage6 + direct6 + len6) % len6);
                } else {
                    curPage6 = 1;
                }
                document.getElementById("btn06").innerHTML="Current: " + curPage6 + "/" + page6 + " page  | ";        // 显示当前多少页

                begin6=(curPage6-1)*pageSize6 + 1;// 起始记录号
                end6 = begin6 + 1*pageSize6 - 1;    // 末尾记录号


                if(end6 > len6 ) end6=len6;
                $("#mytable6 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable6 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin6 && i<=end6) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize7 = 15;    //每页显示的记录条数
            var curPage7=0;        //当前页
            var lastPage7;        //最后页
            var direct7=0;        //方向
            var len7;            //总行数
            var page7;           //总页数
            var begin7;
            var end7;

            $(document).ready(function display7(){
                len7 =$("#mytable7 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page7=len7 % pageSize7==0 ? len7/pageSize7 : Math.floor(len7/pageSize7)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage7=1;    // 设置当前为第一页
                displayPage7(1);//显示第一页
                document.getElementById("btn07").innerHTML="Current: " + curPage7 + "/" + page7 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl7").innerHTML="Total items: " + len7 + "";        // 显示数据量

                $("#btn17").click(function firstPage7(){    // 首页
                    curPage7=1;
                    direct7 = 0;
                    displayPage7();
                });
                $("#btn27").click(function frontPage7(){    // 上一页
                    direct7=-1;
                    displayPage7();
                });
                $("#btn37").click(function nextPage7(){    // 下一页
                    direct7=1;
                    displayPage7();
                });
                $("#btn47").click(function lastPage7(){    // 尾页
                    curPage7=page7;
                    direct7 = 0;
                    displayPage7();
                });
                $("#btn57").click(function changePage7(){    // 转页
                    curPage7=document.getElementById("changePage7").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage7)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage7 > page7) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct7 = 0;
                    displayPage7();
                });


                $("#pageSizeSet7").click(function setPageSize7(){    // 设置每页显示多少条记录
                    pageSize7 = document.getElementById("pageSize7").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len7 = $("#mytable7 tr").length - 1;
                    page7 = len7 % pageSize7==0 ? len7/pageSize7 : Math.floor(len7/pageSize7)+1;//根据记录条数，计算页数
                    curPage7=1;        //当前页
                    direct7=0;        //方向
                    firstPage7();
                });
            });

            function displayPage7(){
                if(curPage7 <=1 && direct7==-1){
                    direct7=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage7 >= page7 && direct7==1) {
                    direct7=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage7 = curPage7;

                // 修复当len=1时，curPage计算得0的bug
                if (len7 > pageSize7) {
                    curPage7 = ((curPage7 + direct7 + len7) % len7);
                } else {
                    curPage7 = 1;
                }
                document.getElementById("btn07").innerHTML="Current: " + curPage7 + "/" + page7 + " page  | ";        // 显示当前多少页

                begin7=(curPage7-1)*pageSize7 + 1;// 起始记录号
                end7 = begin7 + 1*pageSize7 - 1;    // 末尾记录号


                if(end7 > len7 ) end7=len7;
                $("#mytable7 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable7 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin7 && i<=end7) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize8 = 15;    //每页显示的记录条数
            var curPage8=0;        //当前页
            var lastPage8;        //最后页
            var direct8=0;        //方向
            var len8;            //总行数
            var page8;           //总页数
            var begin8;
            var end8;

            $(document).ready(function display8(){
                len8 =$("#mytable8 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page8=len8 % pageSize8==0 ? len8/pageSize8 : Math.floor(len8/pageSize8)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage8=1;    // 设置当前为第一页
                displayPage8(1);//显示第一页
                document.getElementById("btn08").innerHTML="Current: " + curPage8 + "/" + page8 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl8").innerHTML="Total items: " + len8 + "";        // 显示数据量

                $("#btn18").click(function firstPage8(){    // 首页
                    curPage8=1;
                    direct8 = 0;
                    displayPage8();
                });
                $("#btn28").click(function frontPage8(){    // 上一页
                    direct8=-1;
                    displayPage8();
                });
                $("#btn38").click(function nextPage8(){    // 下一页
                    direct8=1;
                    displayPage8();
                });
                $("#btn48").click(function lastPage8(){    // 尾页
                    curPage8=page8;
                    direct8 = 0;
                    displayPage8();
                });
                $("#btn58").click(function changePage8(){    // 转页
                    curPage8=document.getElementById("changePage8").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage8)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage8 > page8) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct8 = 0;
                    displayPage8();
                });


                $("#pageSizeSet8").click(function setPageSize8(){    // 设置每页显示多少条记录
                    pageSize8 = document.getElementById("pageSize8").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len8 = $("#mytable8 tr").length - 1;
                    page8 = len8 % pageSize8==0 ? len8/pageSize8 : Math.floor(len8/pageSize8)+1;//根据记录条数，计算页数
                    curPage8=1;        //当前页
                    direct8=0;        //方向
                    firstPage8();
                });
            });

            function displayPage8(){
                if(curPage8 <=1 && direct8==-1){
                    direct8=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage8 >= page8 && direct8==1) {
                    direct8=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage8 = curPage8;

                // 修复当len=1时，curPage计算得0的bug
                if (len8 > pageSize8) {
                    curPage8 = ((curPage8 + direct8 + len8) % len8);
                } else {
                    curPage8 = 1;
                }
                document.getElementById("btn08").innerHTML="Current: " + curPage8 + "/" + page8 + " page  | ";        // 显示当前多少页

                begin8=(curPage8-1)*pageSize8 + 1;// 起始记录号
                end8 = begin8 + 1*pageSize8 - 1;    // 末尾记录号


                if(end8 > len8 ) end8=len8;
                $("#mytable8 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable8 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin8 && i<=end8) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize9 = 15;    //每页显示的记录条数
            var curPage9=0;        //当前页
            var lastPage9;        //最后页
            var direct9=0;        //方向
            var len9;            //总行数
            var page9;           //总页数
            var begin9;
            var end9;

            $(document).ready(function display9(){
                len9 =$("#mytable9 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page9=len9 % pageSize9==0 ? len9/pageSize9 : Math.floor(len9/pageSize9)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage9=1;    // 设置当前为第一页
                displayPage9(1);//显示第一页
                document.getElementById("btn09").innerHTML="Current: " + curPage9 + "/" + page9 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl9").innerHTML="Total items: " + len9 + "";        // 显示数据量

                $("#btn19").click(function firstPage9(){    // 首页
                    curPage9=1;
                    direct9 = 0;
                    displayPage9();
                });
                $("#btn29").click(function frontPage9(){    // 上一页
                    direct9=-1;
                    displayPage9();
                });
                $("#btn39").click(function nextPage9(){    // 下一页
                    direct9=1;
                    displayPage9();
                });
                $("#btn49").click(function lastPage9(){    // 尾页
                    curPage9=page9;
                    direct9 = 0;
                    displayPage9();
                });
                $("#btn59").click(function changePage9(){    // 转页
                    curPage9=document.getElementById("changePage9").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage9)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage9 > page9) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct9 = 0;
                    displayPage9();
                });


                $("#pageSizeSet9").click(function setPageSize9(){    // 设置每页显示多少条记录
                    pageSize9 = document.getElementById("pageSize9").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len9 = $("#mytable9 tr").length - 1;
                    page9 = len9 % pageSize9==0 ? len9/pageSize9 : Math.floor(len9/pageSize9)+1;//根据记录条数，计算页数
                    curPage9=1;        //当前页
                    direct9=0;        //方向
                    firstPage9();
                });
            });

            function displayPage9(){
                if(curPage9 <=1 && direct9==-1){
                    direct9=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage9 >= page9 && direct9==1) {
                    direct9=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage9 = curPage9;

                // 修复当len=1时，curPage计算得0的bug
                if (len9 > pageSize9) {
                    curPage9 = ((curPage9 + direct9 + len9) % len9);
                } else {
                    curPage9 = 1;
                }
                document.getElementById("btn09").innerHTML="Current: " + curPage9 + "/" + page9 + " page  | ";        // 显示当前多少页

                begin9=(curPage9-1)*pageSize9 + 1;// 起始记录号
                end9 = begin9 + 1*pageSize9 - 1;    // 末尾记录号


                if(end9 > len9 ) end9=len9;
                $("#mytable9 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable9 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin9 && i<=end9) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>


<script type="text/javascript">
            var pageSize10 = 15;    //每页显示的记录条数
            var curPage10=0;        //当前页
            var lastPage10;        //最后页
            var direct10=0;        //方向
            var len10;            //总行数
            var page10;           //总页数
            var begin10;
            var end10;

            $(document).ready(function display10(){
                len10 =$("#mytable10 tr").length - 1;    // 求这个表的总行数，剔除第一行介绍
                page10=len10 % pageSize10==0 ? len10/pageSize10 : Math.floor(len10/pageSize10)+1;//根据记录条数，计算页数
                // alert("page==="+page);
                curPage10=1;    // 设置当前为第一页
                displayPage10(1);//显示第一页
                document.getElementById("btn010").innerHTML="Current: " + curPage10 + "/" + page10 + " page | ";    // 显示当前多少页
                document.getElementById("sjzl10").innerHTML="Total items: " + len10 + "";        // 显示数据量

                $("#btn110").click(function firstPage10(){    // 首页
                    curPage10=1;
                    direct10 = 0;
                    displayPage10();
                });
                $("#btn210").click(function frontPage10(){    // 上一页
                    direct10=-1;
                    displayPage10();
                });
                $("#btn310").click(function nextPage10(){    // 下一页
                    direct10=1;
                    displayPage10();
                });
                $("#btn410").click(function lastPage10(){    // 尾页
                    curPage10=page10;
                    direct10 = 0;
                    displayPage10();
                });
                $("#btn510").click(function changePage10(){    // 转页
                    curPage10=document.getElementById("changePage10").value * 1;
                    if (!/^[1-9]\d*$/.test(curPage10)) {
                        alert("请输入正整数");
                        return ;
                    }
                    if (curPage10 > page10) {
                        alert("超出数据页面");
                        return ;
                    }
                    direct10 = 0;
                    displayPage10();
                });


                $("#pageSizeSet10").click(function setPageSize10(){    // 设置每页显示多少条记录
                    pageSize10 = document.getElementById("pageSize10").value;    //每页显示的记录条数
                    if (!/^[1-9]\d*$/.test(pageSize)) {
                        alert("请输入正整数");
                        return ;
                    }
                    len10 = $("#mytable10 tr").length - 1;
                    page10 = len10 % pageSize10==0 ? len10/pageSize10 : Math.floor(len10/pageSize10)+1;//根据记录条数，计算页数
                    curPage10=1;        //当前页
                    direct10=0;        //方向
                    firstPage10();
                });
            });

            function displayPage10(){
                if(curPage10 <=1 && direct10==-1){
                    direct10=0;
                    alert("Already the first page !!!");
                    return;
                } else if (curPage10 >= page10 && direct10==1) {
                    direct10=0;
                    alert("Already the last page !!!");
                    return ;
                }
                lastPage10 = curPage10;

                // 修复当len=1时，curPage计算得0的bug
                if (len10 > pageSize10) {
                    curPage10 = ((curPage10 + direct10 + len10) % len10);
                } else {
                    curPage10 = 1;
                }
                document.getElementById("btn010").innerHTML="Current: " + curPage10 + "/" + page10 + " page  | ";        // 显示当前多少页

                begin10=(curPage10-1)*pageSize10 + 1;// 起始记录号
                end10 = begin10 + 1*pageSize10 - 1;    // 末尾记录号


                if(end10 > len10 ) end10=len10;
                $("#mytable10 tr").hide();    // 首先，设置这行为隐藏
                $("#mytable10 tr").each(function(i){    // 然后，通过条件判断决定本行是否恢复显示
                    if((i>=begin10 && i<=end10) || i==0 )//显示begin<=x<=end的记录
                        $(this).show();
                });

             }
</script>



</head>

<body>
<div id="toc" class="tocify">
</div>
	<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">
		<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"
			style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
			<tr height="86%">
				<td width='1%'></td>
				<td><img src="src/images/logo1_EN.png"></td>
				<td>
					<div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div>
				</td>
				<td width="20%"></td>
				<td width="15%"></td>
				<td width="25%"
					style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">
					<br><a href="mailto:op-rna@novogene.com" target="black"
						style="text-decoration:none;color:black;font-size:14px;font-weight:50;">
						<a></td>
				<td width="1%"></td>
			</tr>
		</table>
	</div>
</div>  
    
<!--------------------------标题---------------------------------->
<div id="main">
<br><h1>Novogene 16S Amplicon QIIME2 Analysis Report</h1></br>
<table class="gy">
<tr>
<td>Contract ID</td>
<td style="text-align :left">X204SC00000000-Z01</td>
</tr>
<tr>
<td>Contract Name</td>
<td style="text-align :left">amplicon-WBI</td>
</tr>
<tr>
<td>Batch ID</td>
<td style="text-align :left">X204SC00000000-Z01-F00X</td>
</tr>
<tr>
<td>Amplified Region</td>
<td style="text-align :left">16SV4</td>
</tr>
<tr>
<td>Primer Seq</td>
<td style="text-align :left">GTGCCAGCMGCCGCGGTAA,GGACTACHVGGGTWTCTAAT</td>
</tr>
<tr>
<td>Report Time</td>
<td style="text-align :left">2024-09-11</td>
</tr>
<tr>
<td>Reminder</td>
<td style="text-align :left">Partial results are presented in this report, while full results will be delivered in data release. <br>Hyperlink of results in this report will be only valid in data release, after statement confirmation. <br>If there are any doubts regarding the presentation of the final report's results or the file's location, please review the <a href= "#miao">ReadMe.html</a> file.</td>
</tr>
</table>
<br /><br />
<hr />
</p><br />

<h2>1 Introduction</h2>


    <p class="paragraph">16S ribosomal RNA (rRNA) is a component of the 30S small subunit of the ribosome in prokaryotic cells, containing 9 hypervariable regions (V1-V9), ranging from about 30-100 base pairs long that are involved in the secondary structure of the small ribosomal subunit. The degree of conservation varies widely between hypervariable regions, with more conserved regions correlating to higher-rank taxonomy and less conserved regions to lower ranks, such as genus and species. Therefore, 16S rDNA can be used in reconstructing bacterial phylogenies and classifications.</p>

    <p class="paragraph">Sequence variation in the amplicon gene is widely used to characterize diverse microbial communities. With the continuous development of high-throughput sequencing platforms, the illumina sequencing platform could be used for taxonomic classification. Amplicon sequencing technology gradually becomes an important method to study the composition and structure of the microbial communities.</p>

    <p class="paragraph">Additionally, conservative regions enable the design of universal primers. Since 16S rDNA amplicon sequencing typically selects one or several hypervariable regions of interest, the universal primer used during PCR amplification is designed from the conservative regions that encompass the hypervariable regions of interest, which will allow for the identification of microorganisms from the hypervariable regions through subsequent analysis (Caporaso et al., 2011; Youssef et al., 2009; Hess et al., 2011).</p>

    <p class="paragraph">In the process from DNA samples to final data, each step such as sample test, PCR, purification, library preparation, and sequencing will affect the quality and quantity of data, while the data quality will directly affect the subsequent information analysis results. In order to keep the accuracy and reliability of sequencing data, quality control (QC) is performed at each step of the procedure.</p>

    <p class="paragraph">The workflow is shown as follows:</p>

<p class="center">
<img src="src/pictures/project_pipeline_en.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 1.1 Project workflow</strong></p>  
</p>

        <br />

        <br />

<hr />
</p><br />

<h2>2 Library Construction and Sequencing</h2>


    <a name="T4"></a>
    <h4>2.1 Sample Quality Control</h4>    

    <p class="paragraph">Methods of sample quality control refer to QC report.</p>

    <a name="T4"></a>
    <h4>2.2 Library Construction, Quality Control, and Sequencing</h4>    

    <p class="paragraph">PCR amplification of targeted regions was performed by using specific primers connecting with barcodes. The PCR products of proper size were selected through agarose gel electrophoresis. The same amount of PCR products from each sample was pooled, end-repaired, A-tailed, and further ligated with illumina adapters. Libraries were sequenced on a paired-end Illumina platform. The experimental procedures of DNA library preparation are shown in Figure 2.1.</p>

<p class="center">
<img src="src/pictures/wobi_pipeline_en.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 2.1 Workflow of library construction</strong></p>  
</p>

    <p class="paragraph">The library was checked with Qubit and real-time PCR for quantification, while a bioanalyzer was used for size distribution detection. Quantified libraries were pooled and sequenced on Illumina platforms according to the effective library concentration and data amount required.</p>

    <p class="paragraph">Method details involved in the project are available at <a href="#miao">method</a>.</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Sample preparation</p>
<p class="asked" >Q: Why can the sample be amplified without bands in gel electrophoresis ?</p>
<p class="questions" >The concentration of the sample is too low to detect the band. PCR is positive because the starting amount of PCR template needs to be very low.</p>
<p class="asked" >Q: How to detect the concentration of the sample ?</p>
<p class="questions" >Qubit is used, which can accurately quantify the DNA concentration.</p>
<p class="asked" >Q: Why does the sample degrade ?</p>
<p class="questions" >Degradation is caused by too long pyrolysis time and too slow operation. Poor preservation of nucleic acid leads to degradation.</p>
<p class="asked" >Q: DNA sample purification (RNA digestion) conditions ?</p>
<p class="questions" >The digestion condition of the experiment is to take stock liquid 3 μL + 6 μL ddH2O + 1 μL RNase A (original concentration 10mg/mL), 37 ℃, 15 min. Then take 2 μL products for electrophoresis.</p>
<p class="asked" >Q: Impact of too low or too high GC on PCR?</p>
<p class="questions" >The AT content of DNA is high and the thermodynamic stability is low, which will lead to the inability of polymerase replication or the introduction of errors. High GC content brings high thermal stability. In the process of PCR amplification, high GC templates are difficult to be completely denatured. Even if denatured, when the temperature of the annealing step decreases, these chains may also form strong intramolecular hydrogen bonds, which makes it difficult for the polymerase to play its role.</p>
<p class="asked" >Q: Which pair of ITS primers is better for distiller's yeast and fermentation samples ?</p>
<p class="questions" >If the sample type is distiller's yeast or there may be yeast in fermentation, it is recommended to do ITS2. For conventional ITS1, the yeast will be greater than 500 bp; If you do ITS2, double bands may appear on the electrophoretic detection map, and the size of ITS2 fragment is about 430 bp.</p>
<p class="asked" >Q: Why use 2% gel for PCR product electrophoresis ?</p>
<p class="questions" >The concentration of agarose gel used in experiments is usually 0.5~2%. The low concentration is used for electrophoresis of large pieces of nucleic acid, and the high concentration is used for small pieces of analysis. What we do is PCR amplification products, and the bands generally do not exceed 500 bp, so we choose high concentration gel for electrophoresis.</p>
<p class="asked" >Q: The primer sequences used in each region are as follows:</p>

<p class="name_table"><p class="center"><strong>Table 2.1 The primer sequences</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable1" >
        <thead><tr>
<a id="btn01"></a>
<a id="sjzl1"></a>
<a  href="javascript:void(0)" id="btn11"> | First | </a>
<a  href="javascript:void(0)" id="btn21"> Previous | </a>
<a  href="javascript:void(0)" id="btn31"> Next | </a>
<a  href="javascript:void(0)" id="btn41"> Last | </a>
<a>Go to </a>
<input id="changePage1" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn51"> Jump </a>
                
<th style="text-align :center">Species Type</th><th style="text-align :center">Amplified region</th><th style="text-align :center">Primer
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">Bacteria</td><td style="text-align :center">16Sv4</td><td style="text-align :center">GTGCCAGCMGCCGCGGTAA,GGACTACHVGGGTWTCTAAT
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Bacteria</td><td style="text-align :center">16Sv34</td><td style="text-align :center">CCTAYGGGRBGCASCAG,GGACTACNNGGGTATCTAAT
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Bacteria</td><td style="text-align :center">16Sv45</td><td style="text-align :center">GTGCCAGCMGCCGCGGTAA,CCGTCAATTCCTTTGAGTTT
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Bacteria</td><td style="text-align :center">16Sv57</td><td style="text-align :center">AACMGGATTAGATACCCKG,ACGTCATCCCCACCTTCC
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Eukaryote</td><td style="text-align :center">18Sv4</td><td style="text-align :center">GCGGTAATTCCAGCTCCAA,AATCCRAGAATTTCACCTCT
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Fungi</td><td style="text-align :center">ITS1-1F</td><td style="text-align :center">CTTGGTCATTTAGAGGAAGTAA,GCTGCGTTCTTCATCGATGC
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Fungi</td><td style="text-align :center">ITS1-5F</td><td style="text-align :center">GGAAGTAAAAGTCGTAACAAGG,GCTGCGTTCTTCATCGATGC
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Fungi</td><td style="text-align :center">ITS2</td><td style="text-align :center">GCATCGATGAAGAACGCAGC,TCCTCCGCTTATTGATATGC
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Archaea</td><td style="text-align :center">Novel-Archaea-V4</td><td style="text-align :center">CAGCCGCCGCGGTAA,GTGCTCCCCCGCCAATTCCT
</td></tr></tbody>

</table>
</div>
</p>
<p class="asked" >Q: The purpose of degenerate bases in primers.</p>
<p class="questions" >Degenerate base: the nucleotide at a certain position of the designed primer sequence can be two or more different bases. The synthesized primer is a mixture of different sequences at this position, which is intended to increase the amplification efficiency, because research shows that the base site of this segment in different samples is not a fixed conservative sequence.</p>
<p class="asked" >Negative control</p>
<p class="asked" >Q: Why there is no negative control set for library building and sequencing of amplicon.</p>
<p class="questions" >During the construction and sequencing of the amplicon library, the amplification process is all based on the PCR reaction site with a negative control, that is, ddH2O is used as a negative reagent for subsequent PCR operations. Only when the negative control does not have a band, subsequent experiments will be conducted. The reagents used in our primer dilution are all prepared with sterilized ddH2O. Therefore, the use of ddH2O as a negative control in the control can explain the situation, so we are not in the process of negative sample library building, sequencing and other follow-up work. In addition, the experiment uses ddH2O as the negative control. Even if there is slight pollution, the data output rate will be extremely low (that is, CK has no stripes). The number of reads is very small (generally not more than 1000). According to the current information analysis experience, the data of this level is not enough to cause such a high abundance difference in the statistical level in our article results. The high abundance difference in the results should not be caused by reagent contamination.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<hr />
</p><br />

<h2>3 Bioinformatics Analysis Pipeline</h2>


    <p class="paragraph">The raw data obtained by sequencing has a certain proportion of dirty data. In order to make the results of bioinformatic analysis more accurate and reliable, we do splicing and filtering of raw data to obtain clean data firstly; Based on clean data, we use DADA2 or deblur to reduce noise (DADA2 is used by default) (Li et al., 2020). For the obtained ASVs, on the one hand, the representative sequence of each ASV is annotated to obtain the corresponding species information and the abundance distribution based on the species (Callahan et al., 2017). Then, analyze the abundance, Alpha diversity calculation, Venn graph, and Flower graph of ASVs to obtain the species richness and uniformity information of the samples, and the common and unique ASVs information among different samples or groups. On the other hand, multiple sequence alignment of ASVs can be performed and phylogenetic trees constructed. Through dimensionality reduction analysis such as PCoA, PCA, NMDS, and sample clustering tree display, the differences in community structure between different samples or groups can be explored. In order to further explore the differences in community structure in different groups, statistical analysis methods such as T-test, MetagenomeSeq, LEfSe, etc. are used to test the significance of differences in species composition and community structure of groups. The annotation results of the amplicons can also be associated with the corresponding functional databases, and the PICRUSt2 software can be used to predict and analyze the function of the microbial community in the ecological sample. The bioinformatic analysis workflow after obtaining the sequencing data is shown as follows:</p>

<p class="center">
<img src="src/pictures/information_analysis_flow_qiime2_en.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 3.1 Analysis Workflow</strong></p>  
</p>

    <p class="paragraph">Notes:</p>

    <p class="paragraph">In the analysis workflow chart, the blue content is the standard analysis results, and the green content is the advanced analysis results. The result of advanced analysis is not provided by default. If you want to analyze advanced analysis, please fill in the corresponding part in the Bioinformatics Form.</p>

    <p class="paragraph">If number of samples is less than 3, β-diversity cannot be carried out.</p>

    <p class="paragraph">If less than 2 groups and less than 3 biological repetitions in each group, Difference of alpha diversity indices between groups, Community differences analysis(Species variance statistice, Community variance statistice) cannot be carried out.</p>

    <p class="paragraph">Environmental factor correlation analysis requires the customer to provide environmental factor data.</p>

    <p class="paragraph">Method details involved in the project are available at <a href="#miao">method</a>.</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q: What documents are included in the analysis deliverables?</p>
<p class="questions" >The analysis results will be determined according to the actual analysis content of the customer. In general, it includes original offline data (result/00. Rawdata), filtered data (result/01. cleardata), species annotation data (result/02. OTUanalysis), sample complexity data (result/03. AlphaDiversity), community structure diversity data (result/04. BetaDiversity), interactive webpage display results (result/05. WebShow), functional analysis (result/06. Function_prediction) The final report (result/report X101SC ************) and read me. Please check ReadMe.html in the result file for details.</p>
<p class="asked" >Q: What is the check size in the result?</p>
<p class="questions" >The check size does not need to be processed. It is a file used to check whether the result download is complete. The value in the check size is the size of the result compressed package. The value unit of the check size is bytes.</p>
<p class="asked" >Q: Interpretation of fastq files in clean data</p>
<p class="questions" >The fastq file is data with base quality value. @ starts with id, the second line is sequence, the third line is+, and the fourth line is quality value.</p>
<p class="asked" >Q: Is clear and complete for the icon annotation data in the result, and can it be further adjusted?</p>
<p class="questions" >At present, the results displayed in the analysis report provided by us will be in the form of vector graph (pdf format or svg format), which conforms to the resolution, and can be modified with the help of an additional editor (color, font, column shape, size, etc.).</p>
<p class="asked" >Q: The final report of the webpage version cannot view the table, only the header, no content.</p>
<p class="questions" >It may be the compatibility/support problem of the browser. The tables in the report are controlled by JS language. The support of different browsers is different. Firefox browser is recommended (<a href"=http://www.firefox.com.cn/">http://www.firefox.com.cn/</a>) check the web version of the final report we delivered. If it is still not displayed, enter about:config in the Firefox address bar and search for privacy.file_unique_origin, change the value from true to false.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<hr />
</p><br />

<h2>4 Analysis Results</h2>


        <br />

        <br />

<h3>4.1 Sequencing data processing</h3>
        
    <p class="paragraph">The amplicon was sequenced on illumina platform to generate paired-end raw reads (Raw PE), and then merged and pre-treated to obtain Clean Tags. The chimeric sequences in Clean Tags were detected and removed to obtain the Effective Tags which can be used for subsequent analysis. The summarizations obtained in each step of data processing are shown in the table as follow:</p>

<p class="name_table"><p class="center"><strong>Table 4.1 QC statistics</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable2" >
        <thead><tr>
<a id="btn02"></a>
<a id="sjzl2"></a>
<a  href="javascript:void(0)" id="btn12"> | First | </a>
<a  href="javascript:void(0)" id="btn22"> Previous | </a>
<a  href="javascript:void(0)" id="btn32"> Next | </a>
<a  href="javascript:void(0)" id="btn42"> Last | </a>
<a>Go to </a>
<input id="changePage2" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn52"> Jump </a>
                
<th style="text-align :center">Sample</th><th style="text-align :center">RawPE</th><th style="text-align :center">Combined</th><th style="text-align :center">Qualified</th><th style="text-align :center">Nochime</th><th style="text-align :center">Base(nt)</th><th style="text-align :center">Avglen(nt)</th><th style="text-align :center">GC</th><th style="text-align :center">Q20</th><th style="text-align :center">Q30
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">A1</td><td style="text-align :center">164413</td><td style="text-align :center">162636</td><td style="text-align :center">162226</td><td style="text-align :center">150049</td><td style="text-align :center">37944563</td><td style="text-align :center">252.88</td><td style="text-align :center">54.65%</td><td style="text-align :center">99.48%</td><td style="text-align :center">97.91%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A2</td><td style="text-align :center">103727</td><td style="text-align :center">102231</td><td style="text-align :center">101801</td><td style="text-align :center">83809</td><td style="text-align :center">21193801</td><td style="text-align :center">252.88</td><td style="text-align :center">52.59%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.72%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A3</td><td style="text-align :center">105695</td><td style="text-align :center">104106</td><td style="text-align :center">103653</td><td style="text-align :center">86929</td><td style="text-align :center">21985665</td><td style="text-align :center">252.92</td><td style="text-align :center">52.18%</td><td style="text-align :center">99.41%</td><td style="text-align :center">97.68%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A4</td><td style="text-align :center">113512</td><td style="text-align :center">112027</td><td style="text-align :center">111539</td><td style="text-align :center">88067</td><td style="text-align :center">22273787</td><td style="text-align :center">252.92</td><td style="text-align :center">52.51%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.75%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A5</td><td style="text-align :center">105763</td><td style="text-align :center">104532</td><td style="text-align :center">104125</td><td style="text-align :center">84932</td><td style="text-align :center">21478385</td><td style="text-align :center">252.89</td><td style="text-align :center">52.81%</td><td style="text-align :center">99.46%</td><td style="text-align :center">97.86%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A6</td><td style="text-align :center">106916</td><td style="text-align :center">105598</td><td style="text-align :center">105236</td><td style="text-align :center">75480</td><td style="text-align :center">19086764</td><td style="text-align :center">252.87</td><td style="text-align :center">53.03%</td><td style="text-align :center">99.46%</td><td style="text-align :center">97.82%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A7</td><td style="text-align :center">106101</td><td style="text-align :center">104699</td><td style="text-align :center">104341</td><td style="text-align :center">83993</td><td style="text-align :center">21235171</td><td style="text-align :center">252.82</td><td style="text-align :center">52.56%</td><td style="text-align :center">99.46%</td><td style="text-align :center">97.82%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A8</td><td style="text-align :center">119941</td><td style="text-align :center">118544</td><td style="text-align :center">118152</td><td style="text-align :center">94454</td><td style="text-align :center">23879302</td><td style="text-align :center">252.81</td><td style="text-align :center">52.21%</td><td style="text-align :center">99.46%</td><td style="text-align :center">97.85%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A9</td><td style="text-align :center">105237</td><td style="text-align :center">103603</td><td style="text-align :center">103221</td><td style="text-align :center">88742</td><td style="text-align :center">22418005</td><td style="text-align :center">252.62</td><td style="text-align :center">53.15%</td><td style="text-align :center">99.41%</td><td style="text-align :center">97.64%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A10</td><td style="text-align :center">102153</td><td style="text-align :center">100661</td><td style="text-align :center">100287</td><td style="text-align :center">85484</td><td style="text-align :center">21619313</td><td style="text-align :center">252.90</td><td style="text-align :center">52.39%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.72%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A11</td><td style="text-align :center">103816</td><td style="text-align :center">102460</td><td style="text-align :center">102049</td><td style="text-align :center">85058</td><td style="text-align :center">21511550</td><td style="text-align :center">252.90</td><td style="text-align :center">50.66%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.80%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A12</td><td style="text-align :center">109090</td><td style="text-align :center">107492</td><td style="text-align :center">107122</td><td style="text-align :center">92291</td><td style="text-align :center">23316203</td><td style="text-align :center">252.64</td><td style="text-align :center">52.27%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.73%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A13</td><td style="text-align :center">103528</td><td style="text-align :center">102219</td><td style="text-align :center">101883</td><td style="text-align :center">82489</td><td style="text-align :center">20860684</td><td style="text-align :center">252.89</td><td style="text-align :center">51.92%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.79%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A14</td><td style="text-align :center">105276</td><td style="text-align :center">103853</td><td style="text-align :center">103506</td><td style="text-align :center">85151</td><td style="text-align :center">21523210</td><td style="text-align :center">252.77</td><td style="text-align :center">52.56%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.76%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A15</td><td style="text-align :center">105586</td><td style="text-align :center">103884</td><td style="text-align :center">103486</td><td style="text-align :center">86045</td><td style="text-align :center">21758676</td><td style="text-align :center">252.88</td><td style="text-align :center">51.24%</td><td style="text-align :center">99.40%</td><td style="text-align :center">97.62%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A16</td><td style="text-align :center">105092</td><td style="text-align :center">103533</td><td style="text-align :center">103148</td><td style="text-align :center">81867</td><td style="text-align :center">20706413</td><td style="text-align :center">252.93</td><td style="text-align :center">51.10%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.73%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A17</td><td style="text-align :center">104020</td><td style="text-align :center">102857</td><td style="text-align :center">102560</td><td style="text-align :center">92470</td><td style="text-align :center">23342888</td><td style="text-align :center">252.44</td><td style="text-align :center">54.10%</td><td style="text-align :center">99.39%</td><td style="text-align :center">97.48%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A18</td><td style="text-align :center">102560</td><td style="text-align :center">101538</td><td style="text-align :center">101123</td><td style="text-align :center">81717</td><td style="text-align :center">20653107</td><td style="text-align :center">252.74</td><td style="text-align :center">53.43%</td><td style="text-align :center">99.42%</td><td style="text-align :center">97.75%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A19</td><td style="text-align :center">113741</td><td style="text-align :center">112725</td><td style="text-align :center">112323</td><td style="text-align :center">98432</td><td style="text-align :center">24849852</td><td style="text-align :center">252.46</td><td style="text-align :center">53.84%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.84%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A20</td><td style="text-align :center">105973</td><td style="text-align :center">104973</td><td style="text-align :center">104550</td><td style="text-align :center">84431</td><td style="text-align :center">21342166</td><td style="text-align :center">252.78</td><td style="text-align :center">52.84%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.78%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A21</td><td style="text-align :center">105461</td><td style="text-align :center">104368</td><td style="text-align :center">103936</td><td style="text-align :center">91101</td><td style="text-align :center">23005951</td><td style="text-align :center">252.53</td><td style="text-align :center">52.72%</td><td style="text-align :center">99.40%</td><td style="text-align :center">97.70%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A22</td><td style="text-align :center">112183</td><td style="text-align :center">110882</td><td style="text-align :center">110476</td><td style="text-align :center">93871</td><td style="text-align :center">23738613</td><td style="text-align :center">252.89</td><td style="text-align :center">51.28%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.77%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A23</td><td style="text-align :center">116535</td><td style="text-align :center">116083</td><td style="text-align :center">115498</td><td style="text-align :center">95401</td><td style="text-align :center">24132616</td><td style="text-align :center">252.96</td><td style="text-align :center">50.32%</td><td style="text-align :center">99.28%</td><td style="text-align :center">97.42%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A24</td><td style="text-align :center">102841</td><td style="text-align :center">102427</td><td style="text-align :center">101908</td><td style="text-align :center">88087</td><td style="text-align :center">22282772</td><td style="text-align :center">252.96</td><td style="text-align :center">49.78%</td><td style="text-align :center">99.24%</td><td style="text-align :center">97.34%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A25</td><td style="text-align :center">104236</td><td style="text-align :center">103816</td><td style="text-align :center">103152</td><td style="text-align :center">93851</td><td style="text-align :center">23733652</td><td style="text-align :center">252.89</td><td style="text-align :center">52.01%</td><td style="text-align :center">99.21%</td><td style="text-align :center">97.24%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A26</td><td style="text-align :center">113770</td><td style="text-align :center">113348</td><td style="text-align :center">112805</td><td style="text-align :center">103185</td><td style="text-align :center">26097009</td><td style="text-align :center">252.91</td><td style="text-align :center">51.34%</td><td style="text-align :center">99.34%</td><td style="text-align :center">97.55%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A27</td><td style="text-align :center">106091</td><td style="text-align :center">104388</td><td style="text-align :center">104034</td><td style="text-align :center">97163</td><td style="text-align :center">24563189</td><td style="text-align :center">252.80</td><td style="text-align :center">53.23%</td><td style="text-align :center">99.28%</td><td style="text-align :center">97.16%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A28</td><td style="text-align :center">105861</td><td style="text-align :center">104543</td><td style="text-align :center">104206</td><td style="text-align :center">100417</td><td style="text-align :center">25395690</td><td style="text-align :center">252.90</td><td style="text-align :center">53.94%</td><td style="text-align :center">99.34%</td><td style="text-align :center">97.43%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A29</td><td style="text-align :center">102722</td><td style="text-align :center">101251</td><td style="text-align :center">100896</td><td style="text-align :center">97064</td><td style="text-align :center">24552164</td><td style="text-align :center">252.95</td><td style="text-align :center">53.46%</td><td style="text-align :center">99.34%</td><td style="text-align :center">97.33%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A30</td><td style="text-align :center">114582</td><td style="text-align :center">111678</td><td style="text-align :center">111197</td><td style="text-align :center">105370</td><td style="text-align :center">26632579</td><td style="text-align :center">252.75</td><td style="text-align :center">52.31%</td><td style="text-align :center">99.13%</td><td style="text-align :center">96.58%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A31</td><td style="text-align :center">103500</td><td style="text-align :center">102123</td><td style="text-align :center">101798</td><td style="text-align :center">96418</td><td style="text-align :center">24371606</td><td style="text-align :center">252.77</td><td style="text-align :center">52.43%</td><td style="text-align :center">99.30%</td><td style="text-align :center">97.29%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A32</td><td style="text-align :center">103028</td><td style="text-align :center">101676</td><td style="text-align :center">101370</td><td style="text-align :center">96477</td><td style="text-align :center">24401489</td><td style="text-align :center">252.93</td><td style="text-align :center">53.39%</td><td style="text-align :center">99.34%</td><td style="text-align :center">97.36%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A33</td><td style="text-align :center">116998</td><td style="text-align :center">115326</td><td style="text-align :center">114915</td><td style="text-align :center">109771</td><td style="text-align :center">27769444</td><td style="text-align :center">252.98</td><td style="text-align :center">52.34%</td><td style="text-align :center">99.30%</td><td style="text-align :center">97.26%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A34</td><td style="text-align :center">105000</td><td style="text-align :center">103661</td><td style="text-align :center">103242</td><td style="text-align :center">92687</td><td style="text-align :center">23441270</td><td style="text-align :center">252.91</td><td style="text-align :center">52.65%</td><td style="text-align :center">99.30%</td><td style="text-align :center">97.27%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A35</td><td style="text-align :center">103342</td><td style="text-align :center">101950</td><td style="text-align :center">101570</td><td style="text-align :center">95006</td><td style="text-align :center">24020334</td><td style="text-align :center">252.83</td><td style="text-align :center">52.57%</td><td style="text-align :center">99.30%</td><td style="text-align :center">97.25%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A36</td><td style="text-align :center">103776</td><td style="text-align :center">102254</td><td style="text-align :center">101798</td><td style="text-align :center">96280</td><td style="text-align :center">24342240</td><td style="text-align :center">252.83</td><td style="text-align :center">51.68%</td><td style="text-align :center">99.23%</td><td style="text-align :center">97.04%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B1</td><td style="text-align :center">71189</td><td style="text-align :center">70962</td><td style="text-align :center">70771</td><td style="text-align :center">69689</td><td style="text-align :center">17624039</td><td style="text-align :center">252.90</td><td style="text-align :center">54.11%</td><td style="text-align :center">99.52%</td><td style="text-align :center">98.15%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B2</td><td style="text-align :center">51115</td><td style="text-align :center">50921</td><td style="text-align :center">50816</td><td style="text-align :center">47212</td><td style="text-align :center">11939295</td><td style="text-align :center">252.89</td><td style="text-align :center">53.29%</td><td style="text-align :center">99.58%</td><td style="text-align :center">98.33%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B3</td><td style="text-align :center">73283</td><td style="text-align :center">72960</td><td style="text-align :center">72834</td><td style="text-align :center">67205</td><td style="text-align :center">16994870</td><td style="text-align :center">252.88</td><td style="text-align :center">53.18%</td><td style="text-align :center">99.58%</td><td style="text-align :center">98.33%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B4</td><td style="text-align :center">69835</td><td style="text-align :center">68798</td><td style="text-align :center">68610</td><td style="text-align :center">60727</td><td style="text-align :center">15357038</td><td style="text-align :center">252.89</td><td style="text-align :center">53.35%</td><td style="text-align :center">99.40%</td><td style="text-align :center">97.68%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B5</td><td style="text-align :center">117381</td><td style="text-align :center">116960</td><td style="text-align :center">116738</td><td style="text-align :center">111158</td><td style="text-align :center">28132782</td><td style="text-align :center">253.09</td><td style="text-align :center">54.28%</td><td style="text-align :center">99.61%</td><td style="text-align :center">98.42%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B6</td><td style="text-align :center">117208</td><td style="text-align :center">116852</td><td style="text-align :center">116618</td><td style="text-align :center">111523</td><td style="text-align :center">28206626</td><td style="text-align :center">252.92</td><td style="text-align :center">52.72%</td><td style="text-align :center">99.57%</td><td style="text-align :center">98.31%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B7</td><td style="text-align :center">89923</td><td style="text-align :center">89615</td><td style="text-align :center">89404</td><td style="text-align :center">86120</td><td style="text-align :center">21782025</td><td style="text-align :center">252.93</td><td style="text-align :center">53.22%</td><td style="text-align :center">99.57%</td><td style="text-align :center">98.28%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B8</td><td style="text-align :center">134708</td><td style="text-align :center">134267</td><td style="text-align :center">133907</td><td style="text-align :center">128228</td><td style="text-align :center">32435973</td><td style="text-align :center">252.96</td><td style="text-align :center">53.38%</td><td style="text-align :center">99.52%</td><td style="text-align :center">98.11%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B9</td><td style="text-align :center">177392</td><td style="text-align :center">175425</td><td style="text-align :center">174905</td><td style="text-align :center">162729</td><td style="text-align :center">41119782</td><td style="text-align :center">252.69</td><td style="text-align :center">54.17%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.82%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B10</td><td style="text-align :center">55866</td><td style="text-align :center">55611</td><td style="text-align :center">55498</td><td style="text-align :center">53859</td><td style="text-align :center">13619720</td><td style="text-align :center">252.88</td><td style="text-align :center">53.94%</td><td style="text-align :center">99.63%</td><td style="text-align :center">98.49%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B11</td><td style="text-align :center">180466</td><td style="text-align :center">178786</td><td style="text-align :center">178338</td><td style="text-align :center">163443</td><td style="text-align :center">41328370</td><td style="text-align :center">252.86</td><td style="text-align :center">54.39%</td><td style="text-align :center">99.47%</td><td style="text-align :center">97.90%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B12</td><td style="text-align :center">106516</td><td style="text-align :center">105710</td><td style="text-align :center">105489</td><td style="text-align :center">101730</td><td style="text-align :center">25713410</td><td style="text-align :center">252.76</td><td style="text-align :center">52.22%</td><td style="text-align :center">99.48%</td><td style="text-align :center">97.92%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B13</td><td style="text-align :center">103526</td><td style="text-align :center">102811</td><td style="text-align :center">102512</td><td style="text-align :center">99239</td><td style="text-align :center">25099620</td><td style="text-align :center">252.92</td><td style="text-align :center">50.14%</td><td style="text-align :center">99.42%</td><td style="text-align :center">97.85%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B14</td><td style="text-align :center">110377</td><td style="text-align :center">109263</td><td style="text-align :center">108903</td><td style="text-align :center">87968</td><td style="text-align :center">22251115</td><td style="text-align :center">252.95</td><td style="text-align :center">52.98%</td><td style="text-align :center">99.49%</td><td style="text-align :center">97.96%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B15</td><td style="text-align :center">101999</td><td style="text-align :center">101073</td><td style="text-align :center">100646</td><td style="text-align :center">87457</td><td style="text-align :center">22120211</td><td style="text-align :center">252.93</td><td style="text-align :center">52.20%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.86%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B16</td><td style="text-align :center">112850</td><td style="text-align :center">111917</td><td style="text-align :center">111557</td><td style="text-align :center">88185</td><td style="text-align :center">22307650</td><td style="text-align :center">252.96</td><td style="text-align :center">51.76%</td><td style="text-align :center">99.50%</td><td style="text-align :center">98.00%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B17</td><td style="text-align :center">103418</td><td style="text-align :center">102112</td><td style="text-align :center">101658</td><td style="text-align :center">85391</td><td style="text-align :center">21586512</td><td style="text-align :center">252.80</td><td style="text-align :center">53.59%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.81%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B18</td><td style="text-align :center">102047</td><td style="text-align :center">100805</td><td style="text-align :center">100408</td><td style="text-align :center">83518</td><td style="text-align :center">21123118</td><td style="text-align :center">252.92</td><td style="text-align :center">52.80%</td><td style="text-align :center">99.44%</td><td style="text-align :center">97.75%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B19</td><td style="text-align :center">106778</td><td style="text-align :center">105390</td><td style="text-align :center">104948</td><td style="text-align :center">85565</td><td style="text-align :center">21627288</td><td style="text-align :center">252.76</td><td style="text-align :center">52.84%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.82%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B20</td><td style="text-align :center">103976</td><td style="text-align :center">102434</td><td style="text-align :center">102031</td><td style="text-align :center">87159</td><td style="text-align :center">22028727</td><td style="text-align :center">252.74</td><td style="text-align :center">51.48%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.70%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C1</td><td style="text-align :center">89793</td><td style="text-align :center">89523</td><td style="text-align :center">89324</td><td style="text-align :center">87755</td><td style="text-align :center">22198868</td><td style="text-align :center">252.96</td><td style="text-align :center">53.75%</td><td style="text-align :center">99.53%</td><td style="text-align :center">98.18%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C2</td><td style="text-align :center">135887</td><td style="text-align :center">135440</td><td style="text-align :center">135118</td><td style="text-align :center">131753</td><td style="text-align :center">33315843</td><td style="text-align :center">252.87</td><td style="text-align :center">53.90%</td><td style="text-align :center">99.51%</td><td style="text-align :center">98.14%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C3</td><td style="text-align :center">132856</td><td style="text-align :center">132232</td><td style="text-align :center">131941</td><td style="text-align :center">121156</td><td style="text-align :center">30644970</td><td style="text-align :center">252.94</td><td style="text-align :center">53.27%</td><td style="text-align :center">99.57%</td><td style="text-align :center">98.33%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C4</td><td style="text-align :center">133203</td><td style="text-align :center">132722</td><td style="text-align :center">132411</td><td style="text-align :center">125234</td><td style="text-align :center">31668353</td><td style="text-align :center">252.87</td><td style="text-align :center">53.18%</td><td style="text-align :center">99.59%</td><td style="text-align :center">98.37%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C5</td><td style="text-align :center">141871</td><td style="text-align :center">136987</td><td style="text-align :center">136546</td><td style="text-align :center">129529</td><td style="text-align :center">32740391</td><td style="text-align :center">252.76</td><td style="text-align :center">54.27%</td><td style="text-align :center">99.23%</td><td style="text-align :center">96.91%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C6</td><td style="text-align :center">132032</td><td style="text-align :center">131641</td><td style="text-align :center">131357</td><td style="text-align :center">128179</td><td style="text-align :center">32412091</td><td style="text-align :center">252.87</td><td style="text-align :center">53.53%</td><td style="text-align :center">99.56%</td><td style="text-align :center">98.26%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C7</td><td style="text-align :center">146941</td><td style="text-align :center">146282</td><td style="text-align :center">145981</td><td style="text-align :center">135384</td><td style="text-align :center">34309214</td><td style="text-align :center">253.42</td><td style="text-align :center">53.02%</td><td style="text-align :center">99.57%</td><td style="text-align :center">98.29%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C8</td><td style="text-align :center">133242</td><td style="text-align :center">132858</td><td style="text-align :center">132579</td><td style="text-align :center">126655</td><td style="text-align :center">32042551</td><td style="text-align :center">252.99</td><td style="text-align :center">53.46%</td><td style="text-align :center">99.60%</td><td style="text-align :center">98.35%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C9</td><td style="text-align :center">133890</td><td style="text-align :center">133324</td><td style="text-align :center">133014</td><td style="text-align :center">123028</td><td style="text-align :center">31122331</td><td style="text-align :center">252.97</td><td style="text-align :center">52.51%</td><td style="text-align :center">99.49%</td><td style="text-align :center">98.06%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C10</td><td style="text-align :center">123000</td><td style="text-align :center">122612</td><td style="text-align :center">122288</td><td style="text-align :center">118667</td><td style="text-align :center">29999635</td><td style="text-align :center">252.81</td><td style="text-align :center">53.30%</td><td style="text-align :center">99.51%</td><td style="text-align :center">98.10%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C11</td><td style="text-align :center">136654</td><td style="text-align :center">136227</td><td style="text-align :center">135907</td><td style="text-align :center">131296</td><td style="text-align :center">33187594</td><td style="text-align :center">252.77</td><td style="text-align :center">53.69%</td><td style="text-align :center">99.49%</td><td style="text-align :center">98.05%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C12</td><td style="text-align :center">54816</td><td style="text-align :center">54506</td><td style="text-align :center">54423</td><td style="text-align :center">53191</td><td style="text-align :center">13448822</td><td style="text-align :center">252.84</td><td style="text-align :center">53.36%</td><td style="text-align :center">99.60%</td><td style="text-align :center">98.39%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C13</td><td style="text-align :center">177076</td><td style="text-align :center">175463</td><td style="text-align :center">175010</td><td style="text-align :center">165866</td><td style="text-align :center">41931341</td><td style="text-align :center">252.80</td><td style="text-align :center">53.03%</td><td style="text-align :center">99.48%</td><td style="text-align :center">97.91%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C14</td><td style="text-align :center">118136</td><td style="text-align :center">117325</td><td style="text-align :center">117018</td><td style="text-align :center">111324</td><td style="text-align :center">28157367</td><td style="text-align :center">252.93</td><td style="text-align :center">52.04%</td><td style="text-align :center">99.33%</td><td style="text-align :center">97.56%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C15</td><td style="text-align :center">119943</td><td style="text-align :center">118546</td><td style="text-align :center">118043</td><td style="text-align :center">89185</td><td style="text-align :center">22546978</td><td style="text-align :center">252.81</td><td style="text-align :center">53.33%</td><td style="text-align :center">99.45%</td><td style="text-align :center">97.82%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C16</td><td style="text-align :center">105633</td><td style="text-align :center">104314</td><td style="text-align :center">103900</td><td style="text-align :center">84447</td><td style="text-align :center">21355655</td><td style="text-align :center">252.89</td><td style="text-align :center">52.05%</td><td style="text-align :center">99.41%</td><td style="text-align :center">97.70%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C17</td><td style="text-align :center">105803</td><td style="text-align :center">104329</td><td style="text-align :center">103927</td><td style="text-align :center">84488</td><td style="text-align :center">21357515</td><td style="text-align :center">252.79</td><td style="text-align :center">51.64%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.74%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C18</td><td style="text-align :center">103782</td><td style="text-align :center">102321</td><td style="text-align :center">101912</td><td style="text-align :center">88480</td><td style="text-align :center">22370710</td><td style="text-align :center">252.83</td><td style="text-align :center">52.93%</td><td style="text-align :center">99.40%</td><td style="text-align :center">97.64%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C19</td><td style="text-align :center">110783</td><td style="text-align :center">109507</td><td style="text-align :center">109078</td><td style="text-align :center">89301</td><td style="text-align :center">22586744</td><td style="text-align :center">252.93</td><td style="text-align :center">51.76%</td><td style="text-align :center">99.43%</td><td style="text-align :center">97.77%
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C20</td><td style="text-align :center">103226</td><td style="text-align :center">101097</td><td style="text-align :center">100677</td><td style="text-align :center">96708</td><td style="text-align :center">24458302</td><td style="text-align :center">252.91</td><td style="text-align :center">53.53%</td><td style="text-align :center">99.25%</td><td style="text-align :center">96.99%
</td></tr></tbody>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes:</p>
<p class="premark" >(1) RawPE represents the original PE reads after sequencing;</p>
<p class="premark" >(2) Combined represents tags merged from PE reads;</p>
<p class="premark" >(3) Qualified represents tags after filtering;</p>
<p class="premark" >(4) Nochime represents tags after filtering chimera and can be finally used for subsequent analysis;</p>
<p class="premark" >(5) Base is the number of bases of the Effective Tags;</p>
<p class="premark" >(6) AvgLen represents average length of Effective Tags;</p>
<p class="premark" >(7) Q20 and Q30 are the percentages of bases whose quality value in Effective Tags is greater than 20 (sequencing error rate is less than 1%) and 30 (sequencing error rate is less than 0.1%);</p>
<p class="premark" >(8) GC (%) represents GC content in Effective Tags;</p>
<p class="premark" >Note: When conducting joint analysis or providing sample data for analysis, the quality control results of this part of the sample will not be displayed in the results.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Removed barcode and primer sequence (rawPE): result/01.RawData/Sample_Name/*_1(2).fastq.gz</p>

    <p class="paragraph">Combined sequence: result/01.RawData/Sample_Name/*.extendedFrags.fastq.gz</p>

    <p class="paragraph">Barcodes and primers information: result/01.RawData/SampleSeq_info.xls</p>

    <p class="paragraph">Removed chimera sequence (Nochime): result/01.CleanData/Sample_Name/*.effective.fastq.gz,*.fna.gz</p>

    <p class="paragraph">Customer provided data is not displayed in the table by default</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Flash software merging </p>
<p class="asked" >Q: What are the conditions for combined reads that have overlap with Flash software? How to deal with those that do not meet the condition? How many bp should be removed from the 3' end and then continue merging?</p>
<p class="questions" >Flash software has two important parameters in merging: the maximum mismatch rate (0.1) and the minimum overlap region (10bp), that is, we have to ensure that the mismatch rate is no more than 0.1 and the minimum overlap of PE reads is no less than 10 bases when splicing. Considering that the sequence quality at the 3' end has a systematic tendency to decrease, we will partially intercept the PEreads at the 3' end based on the length of the fragment to ensure the length of the overlap region of PE reads, which will help to ensure the quality of the bases in the overlap region and improve the merging rate.</p>
<p class="asked" >Q: What is the difference between averaging, homogenization and standardization?</p>
<p class="questions" >Averaging and homogenization both refer to the random sampling of sequences from different samples according to a certain rule (e.g., we go by the minimum number of sequences) so that the number of sequences in different samples is the same. Standardization is generally a Z-value calculation for data that fluctuates over a large range, scaling the data so that it falls into a small specific interval that is easy to display in a graph.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<h3>4.2 OTU/ASV analysis and taxonomic annotation</h3>
        
    <a name="T4"></a>
    <h4>4.2.1 Denoise and Species Annotation</h4>    

    <p class="paragraph">The DADA2 method (Callahan et al., 2016) is mainly used for noise reduction. It no longer uses similarity clustering, but only performs dereplication or equivalent to 100% similarity clustering. Each de-duplicated sequence generated after noise reduction using DADA2 is called ASVs (Amplicon Sequence Variants), or feature sequence (corresponding to the OTU representative sequence), and the abundance table of these sequences in the sample is called the feature table (corresponds to the OTU table). The DADA2 method is more sensitive and specific than the traditional OTU method and can detect the true biological mutations missed by the OTU method while outputting fewer false sequences (Callahan et al., 2019). Compared with OTUs, ASVs improve the accuracy, comprehensiveness, and repeatability of marker gene data analysis (Amir et al., 2017).</p>

    <p class="paragraph">By applying QIIME2's classify-sklearn algorithm (Bokulich et al., 2018; Bolyen et al., 2019), a pre-trained Naive Bayes classifier is used for species annotation of each ASV. Annotation database of the project is Silva 138.1.</p>

    <p class="paragraph">According to the results of ASVs annotations and the feature tables of each sample, the species abundance tables at the level of kingdom, phyla, class, order, family, genus, and species are obtained. These abundance tables with annotation information are the core content of amplicon analysis. According to different experimental purposes, one or several species of key concern can be selected from the species abundance table of each classification level (usually focusing on the phylum and genus level), combined with the species composition and differential analysis of different samples (groups), and cluster analysis to conduct in-depth research.</p>

    <p class="paragraph">Note: Qiime1 analysis only contains OTU analysis results, while Qiime2 analysis only contains ASV analysis results.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Representative sequence: result/02.FeatureAnalysis/feature.fasta</p>

    <p class="paragraph">Species abundance display table: result/02.FeatureAnalysis/tables/(sample,group)/featureTable_Relative/</p>

    <a name="T4"></a>
    <h4>4.2.2 Relative abundance</h4>    

    <p class="paragraph">According to the taxonomic annotation results, the top 10 taxa of each sample or group at each taxonomic rank (Phylum, Class, Order, Family, Genus, Species) were selected to form the distribution histogram of the relative abundance of taxa. This allows the visualization of the taxa with a higher relative abundance and their proportion in different classification levels for each sample or group. The relative abundance of taxa in the phylum is illustrated below:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureTaxarelativeabundanceinphylum_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureTaxarelativeabundanceinphylum_nav-tabContent">

<div class="tab-pane active show" id="topsamplepbarplotpng1" role="tabpanel" aria-labelledby="topsamplepbarplotpng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.p.barplot.png" /></div>

<div class="tab-pane fade" id="topsamplecbarplotpng2" role="tabpanel" aria-labelledby="topsamplecbarplotpng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.c.barplot.png" /></div>

<div class="tab-pane fade" id="topsampleobarplotpng3" role="tabpanel" aria-labelledby="topsampleobarplotpng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.o.barplot.png" /></div>

<div class="tab-pane fade" id="topsamplefbarplotpng4" role="tabpanel" aria-labelledby="topsamplefbarplotpng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.f.barplot.png" /></div>

<div class="tab-pane fade" id="topsamplegbarplotpng5" role="tabpanel" aria-labelledby="topsamplegbarplotpng5" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.g.barplot.png" /></div>

<div class="tab-pane fade" id="topsamplesbarplotpng6" role="tabpanel" aria-labelledby="topsamplesbarplotpng6" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/top10.sample.s.barplot.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureTaxarelativeabundanceinphylum_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureTaxarelativeabundanceinphylum_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureTaxarelativeabundanceinphylum_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="topsamplepbarplotpng1" data-toggle="list" href="#topsamplepbarplotpng1" role="tab" aria-controls="topsamplepbarplotpng1">top10.sample.p.barplot.png</a>

<a class="list-group-item list-group-item-action" id="topsamplecbarplotpng2" data-toggle="list" href="#topsamplecbarplotpng2" role="tab" aria-controls="topsamplecbarplotpng2">top10.sample.c.barplot.png</a>

<a class="list-group-item list-group-item-action" id="topsampleobarplotpng3" data-toggle="list" href="#topsampleobarplotpng3" role="tab" aria-controls="topsampleobarplotpng3">top10.sample.o.barplot.png</a>

<a class="list-group-item list-group-item-action" id="topsamplefbarplotpng4" data-toggle="list" href="#topsamplefbarplotpng4" role="tab" aria-controls="topsamplefbarplotpng4">top10.sample.f.barplot.png</a>

<a class="list-group-item list-group-item-action" id="topsamplegbarplotpng5" data-toggle="list" href="#topsamplegbarplotpng5" role="tab" aria-controls="topsamplegbarplotpng5">top10.sample.g.barplot.png</a>

<a class="list-group-item list-group-item-action" id="topsamplesbarplotpng6" data-toggle="list" href="#topsamplesbarplotpng6" role="tab" aria-controls="topsamplesbarplotpng6">top10.sample.s.barplot.png</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.2 Taxa relative abundance in phylum</strong></p>  

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: Y-axis represents "Relative Abundance" and X-axis represents "Samples Name". "Others" represents a total relative abundance of the rest phyla besides the top 10 phyla.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">The top 10 taxa relative abundance in each taxonomic rank: result/03.TaxonVisual/barplot/(sample,group)/ ; Including Phylum, Class, Order, Family, Genus and Species</p>

    <a name="T4"></a>
    <h4>4.2.3 Taxonomic abundance cluster heatmap</h4>    

    <p class="paragraph">According to the abundance information of top 35 genus of all samples, the heatmap was drawn to check whether the samples with similar processing are clustered or not, while the similarity and differences of samples can also be observed. The result is shown as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureTaxonomicabundanceclusterheatmap_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureTaxonomicabundanceclusterheatmap_nav-tabContent">

<div class="tab-pane active show" id="heatmapsampleppng1" role="tabpanel" aria-labelledby="heatmapsampleppng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.p.png" /></div>

<div class="tab-pane fade" id="heatmapsamplecpng2" role="tabpanel" aria-labelledby="heatmapsamplecpng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.c.png" /></div>

<div class="tab-pane fade" id="heatmapsampleopng3" role="tabpanel" aria-labelledby="heatmapsampleopng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.o.png" /></div>

<div class="tab-pane fade" id="heatmapsamplefpng4" role="tabpanel" aria-labelledby="heatmapsamplefpng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.f.png" /></div>

<div class="tab-pane fade" id="heatmapsamplegpng5" role="tabpanel" aria-labelledby="heatmapsamplegpng5" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.g.png" /></div>

<div class="tab-pane fade" id="heatmapsamplespng6" role="tabpanel" aria-labelledby="heatmapsamplespng6" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/heatmap.sample.s.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureTaxonomicabundanceclusterheatmap_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureTaxonomicabundanceclusterheatmap_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureTaxonomicabundanceclusterheatmap_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="heatmapsampleppng1" data-toggle="list" href="#heatmapsampleppng1" role="tab" aria-controls="heatmapsampleppng1">heatmap.sample.p.png</a>

<a class="list-group-item list-group-item-action" id="heatmapsamplecpng2" data-toggle="list" href="#heatmapsamplecpng2" role="tab" aria-controls="heatmapsamplecpng2">heatmap.sample.c.png</a>

<a class="list-group-item list-group-item-action" id="heatmapsampleopng3" data-toggle="list" href="#heatmapsampleopng3" role="tab" aria-controls="heatmapsampleopng3">heatmap.sample.o.png</a>

<a class="list-group-item list-group-item-action" id="heatmapsamplefpng4" data-toggle="list" href="#heatmapsamplefpng4" role="tab" aria-controls="heatmapsamplefpng4">heatmap.sample.f.png</a>

<a class="list-group-item list-group-item-action" id="heatmapsamplegpng5" data-toggle="list" href="#heatmapsamplegpng5" role="tab" aria-controls="heatmapsamplegpng5">heatmap.sample.g.png</a>

<a class="list-group-item list-group-item-action" id="heatmapsamplespng6" data-toggle="list" href="#heatmapsamplespng6" role="tab" aria-controls="heatmapsamplespng6">heatmap.sample.s.png</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.3 Taxonomic abundance cluster heatmap</strong></p>  

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: plotted by sample name on the X-axis and the Y-axis represents the genus. The absolute value of 'z' represents the distance between the raw score and the mean of the standard deviation. 'Z' is negative when the raw score is below the mean, and vice versa.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Taxonomic abundance heatmap in each taxonomic rank：result/03.TaxonVisual/heatmap/(sample,group)/ ; Including Phylum, Class, Order, Family, Genus and Species</p>

    <a name="T4"></a>
    <h4>4.2.4 Venn and Flower diagram</h4>    

    <p class="paragraph">According to the analysis result of feature seqs and research requirements, we normalized the feature seq table, analyzed both the common and unique information for different samples (groups), and generated the Venn and Flower diagram.</p>

    <p class="paragraph">Venn diagram based on feature seqs:</p>

<p class="center">
<img src="src/pictures/venn_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.4 Venn diagram</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/venn_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: each circle represents one sample or group. Values in overlapping parts represent common feature seqs. The Others are specific feature seqs in each sample.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Venn diagrams：result/03.TaxonVisual/venn/(sample,group)/*</p>

    <p class="paragraph">Venn diagram based on feature seqs:</p>

<p class="center">
<img src="src/pictures/flower_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.5 Flower diagram</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/flower_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: In the flower, each petal represents one (group) sample, different colors represent different samples (groups), the middle core number represents the number of feature sequences shared by all samples, and the number on the petal represents the number of feature sequences unique to this sample (group).</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Venn diagrams：result/03.TaxonVisual/flower/(sample,group)/*</p>

    <a name="T4"></a>
    <h4>4.2.5 The evolutionary tree in genus</h4>    

    <p class="paragraph">To further study the phylogenetic relationship of the genus, the top 100 genera were selected and the evolutionary tree was drawn using the aligned representative sequences. The relative abundance of each genus was also displayed along the genus in the figure as follow：</p>

<p class="center">
<img src="src/pictures/genus_100.tree.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.6 The evolutionary tree in genus</strong></p>  
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: different colors of the branches represent different phyla. Relative abundance of each genus in each group was displayed outside the circle and different colors represent different groups.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">The evolutionary tree in genus by groups: result/03.TaxonVisual/tree100/(sample,group)/*</p>

    <a name="T4"></a>
    <h4>4.2.6 Ternary plot</h4>    

    <p class="paragraph">To find the differences in dominant taxa among the three groups of samples at each taxonomic rank (Phylum, Class, Order, Family, Genus, Species), the top 10 taxa with the average abundance of the three groups of samples at each taxonomic rank were selected to generate ternary plot (Bulgarelli et al., 2015). The result is shown as follows:</p>

<p class="center">
<img src="src/pictures/ternary_display1.jpeg" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.7 Ternary plot</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/ternary_display1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: three vertexes represent three samples or groups. Circles represent dominant taxa and the size of circles are proportional to the relative abundance. The sample of group which the circle close to has higher abundance of this phylum.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Ternary plot: result/03.TaxonVisual/ternary/(sample,group)/*/*ternary.(pdf,jpeg), Including Phylum, Class, Order, Family, Genus and Species.</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q: What are the main concerns of biological information analysis results ?</p>
<p class="questions" >The species abundance of the sample includes absolute abundance and relative abundance. The downstream alpha and beta diversity analysis is also completed based on the homogenized abundance table. Through the species abundance table, we can directly understand the species composition and distribution in the sample, and also select significantly different species or target species for further analysis in combination with our own project background. Taking the Relative Catalog as an example, the catalog contains the relative abundance of each species at the 7 species classification levels (Kingom, Phylum, Class, Order, Family, Genus, Specie), and the relative abundance information of each characteristic sequence in each sample. To view the relative abundance of Actinomycetaceae in each sample, search for "Actinomycetaceae" in the file result/02.FeatureAnalysis/tables/(sample,group)/featureTable_Relative/*relative.xls to query the corresponding relative abundance information.</p>
<p class="asked" >Interpretation of bacteria results</p>
<p class="asked" >Q: What is the relationship between the NT database and the Silva database described in the final report ?</p>
<p class="questions" >The Silva database is a database of general bacterial selection, which has been updated at present. The NT library is an NCBI nucleic acid library, which contains more information and many annotation results, including species other than bacteria.</p>
<p class="asked" >Q: Why is there mitochondria (mitochondria?) and chloroplast (chloroplast?) in the 16S results? and chloroplast?</p>
<p class="questions" >In our annotation results, mitochondria and chloroplast appear. The names in our annotation database are official names, which have their sources and bases. Mitochondria belongs to Proteobacteria (Proteobacteria), Alphaproteobacteria (α  Proteobacteria), Rickettsiales (Rickettsiales), which is indeed a prokaryotic microbe. In addition, literatures: Fitzpatrick D A, Creevey C J, Mcinerney J O. Genome phytogenes indicate a meaningful alpha proterozoic physiology and support a grouping of the mitochondria with the Rickettsiales [J] Molecular Biology & Evolution， 2006， 23(1):74-85。 It is mentioned in this reference that mitochondria is classified as Rickettsiales. For chloroplast, which is common in cyanobacteria, it has been mentioned in many literatures and translated into chloroplast like cyanobacteria, but it is not recommended to translate it like this, and it can be named directly in Latin.</p>
<p class="asked" >Interpretation of archaea results</p>
<p class="asked" >Q: Why there are many bacteria in the results of archaea annotation ?</p>
<p class="questions" >The reason is that the content of archaea in the sample is too low. Because of the high homology between archaea and bacteria, the high abundance of bacterial sequences will be amplified during PCR, which is unavoidable. In fact, this is also normal. Just like the endophyte, when the sample is contaminated with plant DNA, 16S primers will amplify the chloroplast sequence (the chloroplast sequence and bacterial sequence are also highly homologous). Therefore, the distribution of sample species directly determines the results of early PCR and late data analysis.</p>
<p class="asked" >Q: How to understand "relative" in the relative abundance level of bar plot results ?</p>
<p class="questions" >At a certain classification level (for example, Phylum, Class, Order, Family, Genus, Species), the relative abundance of the species is the number of tags corresponding to a certain level in the sample divided by the total number of tags obtained from the total clustering of the sample. At a taxonomic level, the sum of the relative abundance of all species in the same sample is equal to 1.</p>
<p class="asked" >Q: Treatment methods for large differences in biological repeated results</p>
<p class="questions" >We usually recommend more than 5 biological replicates, at least 3. For individual samples with large differences between duplicate samples, it is recommended that:</p>
<p class="questions" >1) Biological replicates selected from samples may be influenced by many other factors in addition to the set grouping conditions, causing differences in analytical results.</p>
<p class="questions" >2) For individual samples with significant outliers, it is speculated that the sample may be the cause (e.g., problems during sampling, preservation, extraction, amplification, etc.) and it is recommended that the sample be excluded from analysis.</p>
<p class="asked" >Q: The union method of "representative sequence" of Venn diagram in groups:</p>
<p class="questions" >When calculating the number of "representative sequences" in each group, the union method used for counting the representative sequences of the group (that is, if there is only one sample in the group with the representative sequence, it is considered that the representative sequence exists in the group. If there is no representative sequence in all the duplicate samples, it is considered that the representative sequence does not exist in the reorganization).</p>
<p class="asked" >Q: What is the reason for the heavy host contamination in the amplicon data, and most of them can only be annotated to the Kingdom level after eliminating the hosts ?</p>
<p class="questions" >The reason is that there are not many sequences left after eliminating the host, and the length of the remaining sequence fragments is short. A short sequence will be annotated to multiple species. When using the lca algorithm to annotate, you will get the annotation results of the previous level, and the results can only be annotated to the limit.</p>
<p class="asked" >Q: What is the meaning of "others" in the bar plot of Taxa relative abundance?</p>
<p class="questions" >The Species ranking 11 and later or those not annotated are classified as "others".</p>
<p class="asked" >Q: What is the reason for some species in the abundance table end with _ or __ , and this species cannot be found in the database ?</p>
<p class="questions" >If there is an underline at the end of a species name, it is because the species has different evolutionary relationships and some visualizations do not allow duplicate annotation names, so we use an underline to distinguish them. Specific annotation information can be found in the abundance table.</p>
<p class="asked" >Why choose Micro_ NT database for species annotation ?</p>
<p class="questions" >Micro_NT is a sub library obtained by extracting archaea, fungi, viruses, and bacteria from the NT. The reason of choosing Micro_NT is to pay more attention to information related to microorganisms.</p>
<p class="asked" >Q: The relative abundance histogram of the Phylum level, the selection criteria of the top 10 Phylum ?</p>
<p class="questions" >The maximum ranking method was used for the barplot of species abundance. Take the level top10 Phylum as an example: first take the maximum relative abundance of each Phylum in each sample, and then sort it. Then take the top ten Phylums for mapping. We can simply obtain the distribution of species with differences between samples in each sample.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<h3>4.3 Alpha Diversity Analysis</h3>
        
    <p class="paragraph">Alpha diversity is applied to the analysis of microbial community diversity within the sample (Li et al., 2013) by analyzing the diversity of a single sample (Alpha diversity) can reflect the richness and diversity of microbial communities in each sample, including species accumulation boxplot, biodiversity curves and a series of statistical analysis.</p>

    <a name="T4"></a>
    <h4>4.3.1 Alpha diversity indices</h4>    

    <p class="paragraph">Generally speaking, the identity sequences of OTUs obtained through clustering or ASVs obtained through noised are considered homologous within species. Through such methods, α The statistical index of diversity is summarized as follows:</p>

<p class="name_table"><p class="center"><strong>Table 4.2 Alpha diversity indices</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable3" >
        <thead><tr>
<a id="btn03"></a>
<a id="sjzl3"></a>
<a  href="javascript:void(0)" id="btn13"> | First | </a>
<a  href="javascript:void(0)" id="btn23"> Previous | </a>
<a  href="javascript:void(0)" id="btn33"> Next | </a>
<a  href="javascript:void(0)" id="btn43"> Last | </a>
<a>Go to </a>
<input id="changePage3" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn53"> Jump </a>
                
<th style="text-align :center">Sample_Name</th><th style="text-align :center">chao1</th><th style="text-align :center">dominance</th><th style="text-align :center">goods_coverage</th><th style="text-align :center">observed_features</th><th style="text-align :center">pielou_e</th><th style="text-align :center">shannon</th><th style="text-align :center">simpson
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">A1</td><td style="text-align :center">1351.968</td><td style="text-align :center">0.120</td><td style="text-align :center">0.995</td><td style="text-align :center">1210</td><td style="text-align :center">0.546</td><td style="text-align :center">5.594</td><td style="text-align :center">0.880
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A2</td><td style="text-align :center">401.738</td><td style="text-align :center">0.033</td><td style="text-align :center">0.999</td><td style="text-align :center">385</td><td style="text-align :center">0.703</td><td style="text-align :center">6.035</td><td style="text-align :center">0.967
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A3</td><td style="text-align :center">412.103</td><td style="text-align :center">0.030</td><td style="text-align :center">0.999</td><td style="text-align :center">395</td><td style="text-align :center">0.714</td><td style="text-align :center">6.162</td><td style="text-align :center">0.970
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A4</td><td style="text-align :center">472.750</td><td style="text-align :center">0.031</td><td style="text-align :center">0.999</td><td style="text-align :center">439</td><td style="text-align :center">0.706</td><td style="text-align :center">6.199</td><td style="text-align :center">0.969
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A5</td><td style="text-align :center">479.091</td><td style="text-align :center">0.045</td><td style="text-align :center">0.999</td><td style="text-align :center">462</td><td style="text-align :center">0.652</td><td style="text-align :center">5.768</td><td style="text-align :center">0.955
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A6</td><td style="text-align :center">474.909</td><td style="text-align :center">0.032</td><td style="text-align :center">0.999</td><td style="text-align :center">467</td><td style="text-align :center">0.699</td><td style="text-align :center">6.201</td><td style="text-align :center">0.968
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A7</td><td style="text-align :center">412.800</td><td style="text-align :center">0.050</td><td style="text-align :center">0.999</td><td style="text-align :center">398</td><td style="text-align :center">0.662</td><td style="text-align :center">5.716</td><td style="text-align :center">0.950
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A8</td><td style="text-align :center">477.920</td><td style="text-align :center">0.030</td><td style="text-align :center">0.999</td><td style="text-align :center">446</td><td style="text-align :center">0.706</td><td style="text-align :center">6.218</td><td style="text-align :center">0.970
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A9</td><td style="text-align :center">380.286</td><td style="text-align :center">0.131</td><td style="text-align :center">0.999</td><td style="text-align :center">358</td><td style="text-align :center">0.573</td><td style="text-align :center">4.861</td><td style="text-align :center">0.869
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A10</td><td style="text-align :center">400.000</td><td style="text-align :center">0.048</td><td style="text-align :center">0.999</td><td style="text-align :center">378</td><td style="text-align :center">0.682</td><td style="text-align :center">5.841</td><td style="text-align :center">0.952
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A11</td><td style="text-align :center">386.292</td><td style="text-align :center">0.061</td><td style="text-align :center">0.999</td><td style="text-align :center">357</td><td style="text-align :center">0.662</td><td style="text-align :center">5.616</td><td style="text-align :center">0.939
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A12</td><td style="text-align :center">613.371</td><td style="text-align :center">0.144</td><td style="text-align :center">0.998</td><td style="text-align :center">573</td><td style="text-align :center">0.518</td><td style="text-align :center">4.744</td><td style="text-align :center">0.856
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A13</td><td style="text-align :center">395.941</td><td style="text-align :center">0.042</td><td style="text-align :center">0.999</td><td style="text-align :center">384</td><td style="text-align :center">0.702</td><td style="text-align :center">6.023</td><td style="text-align :center">0.958
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A14</td><td style="text-align :center">392.000</td><td style="text-align :center">0.041</td><td style="text-align :center">0.999</td><td style="text-align :center">383</td><td style="text-align :center">0.694</td><td style="text-align :center">5.955</td><td style="text-align :center">0.959
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A15</td><td style="text-align :center">393.156</td><td style="text-align :center">0.046</td><td style="text-align :center">0.999</td><td style="text-align :center">383</td><td style="text-align :center">0.674</td><td style="text-align :center">5.781</td><td style="text-align :center">0.954
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A16</td><td style="text-align :center">491.167</td><td style="text-align :center">0.048</td><td style="text-align :center">0.999</td><td style="text-align :center">462</td><td style="text-align :center">0.694</td><td style="text-align :center">6.143</td><td style="text-align :center">0.952
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A17</td><td style="text-align :center">198.143</td><td style="text-align :center">0.063</td><td style="text-align :center">0.999</td><td style="text-align :center">185</td><td style="text-align :center">0.629</td><td style="text-align :center">4.734</td><td style="text-align :center">0.937
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A18</td><td style="text-align :center">454.429</td><td style="text-align :center">0.043</td><td style="text-align :center">0.999</td><td style="text-align :center">436</td><td style="text-align :center">0.674</td><td style="text-align :center">5.911</td><td style="text-align :center">0.957
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A19</td><td style="text-align :center">302.667</td><td style="text-align :center">0.277</td><td style="text-align :center">1.000</td><td style="text-align :center">295</td><td style="text-align :center">0.481</td><td style="text-align :center">3.950</td><td style="text-align :center">0.723
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A20</td><td style="text-align :center">374.636</td><td style="text-align :center">0.039</td><td style="text-align :center">0.999</td><td style="text-align :center">351</td><td style="text-align :center">0.693</td><td style="text-align :center">5.858</td><td style="text-align :center">0.961
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A21</td><td style="text-align :center">361.583</td><td style="text-align :center">0.200</td><td style="text-align :center">0.999</td><td style="text-align :center">346</td><td style="text-align :center">0.503</td><td style="text-align :center">4.241</td><td style="text-align :center">0.800
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A22</td><td style="text-align :center">364.667</td><td style="text-align :center">0.039</td><td style="text-align :center">1.000</td><td style="text-align :center">357</td><td style="text-align :center">0.714</td><td style="text-align :center">6.057</td><td style="text-align :center">0.961
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A23</td><td style="text-align :center">261.240</td><td style="text-align :center">0.072</td><td style="text-align :center">1.000</td><td style="text-align :center">252</td><td style="text-align :center">0.621</td><td style="text-align :center">4.955</td><td style="text-align :center">0.928
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A24</td><td style="text-align :center">355.022</td><td style="text-align :center">0.088</td><td style="text-align :center">0.999</td><td style="text-align :center">331</td><td style="text-align :center">0.568</td><td style="text-align :center">4.755</td><td style="text-align :center">0.912
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A25</td><td style="text-align :center">253.562</td><td style="text-align :center">0.036</td><td style="text-align :center">1.000</td><td style="text-align :center">247</td><td style="text-align :center">0.712</td><td style="text-align :center">5.659</td><td style="text-align :center">0.964
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A26</td><td style="text-align :center">273.333</td><td style="text-align :center">0.047</td><td style="text-align :center">0.999</td><td style="text-align :center">254</td><td style="text-align :center">0.678</td><td style="text-align :center">5.415</td><td style="text-align :center">0.953
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A27</td><td style="text-align :center">320.286</td><td style="text-align :center">0.061</td><td style="text-align :center">1.000</td><td style="text-align :center">316</td><td style="text-align :center">0.634</td><td style="text-align :center">5.267</td><td style="text-align :center">0.939
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A28</td><td style="text-align :center">202.882</td><td style="text-align :center">0.192</td><td style="text-align :center">1.000</td><td style="text-align :center">199</td><td style="text-align :center">0.527</td><td style="text-align :center">4.021</td><td style="text-align :center">0.808
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A29</td><td style="text-align :center">238.105</td><td style="text-align :center">0.139</td><td style="text-align :center">1.000</td><td style="text-align :center">234</td><td style="text-align :center">0.553</td><td style="text-align :center">4.353</td><td style="text-align :center">0.861
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A30</td><td style="text-align :center">406.353</td><td style="text-align :center">0.021</td><td style="text-align :center">1.000</td><td style="text-align :center">394</td><td style="text-align :center">0.762</td><td style="text-align :center">6.572</td><td style="text-align :center">0.979
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A31</td><td style="text-align :center">397.917</td><td style="text-align :center">0.023</td><td style="text-align :center">1.000</td><td style="text-align :center">390</td><td style="text-align :center">0.761</td><td style="text-align :center">6.547</td><td style="text-align :center">0.977
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A32</td><td style="text-align :center">269.636</td><td style="text-align :center">0.087</td><td style="text-align :center">0.999</td><td style="text-align :center">256</td><td style="text-align :center">0.618</td><td style="text-align :center">4.940</td><td style="text-align :center">0.913
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A33</td><td style="text-align :center">278.000</td><td style="text-align :center">0.038</td><td style="text-align :center">0.999</td><td style="text-align :center">265</td><td style="text-align :center">0.694</td><td style="text-align :center">5.583</td><td style="text-align :center">0.962
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A34</td><td style="text-align :center">260.120</td><td style="text-align :center">0.043</td><td style="text-align :center">1.000</td><td style="text-align :center">254</td><td style="text-align :center">0.678</td><td style="text-align :center">5.415</td><td style="text-align :center">0.957
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A35</td><td style="text-align :center">392.667</td><td style="text-align :center">0.025</td><td style="text-align :center">1.000</td><td style="text-align :center">387</td><td style="text-align :center">0.746</td><td style="text-align :center">6.409</td><td style="text-align :center">0.975
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A36</td><td style="text-align :center">393.038</td><td style="text-align :center">0.020</td><td style="text-align :center">1.000</td><td style="text-align :center">389</td><td style="text-align :center">0.770</td><td style="text-align :center">6.624</td><td style="text-align :center">0.980
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B1</td><td style="text-align :center">196.667</td><td style="text-align :center">0.055</td><td style="text-align :center">1.000</td><td style="text-align :center">195</td><td style="text-align :center">0.663</td><td style="text-align :center">5.047</td><td style="text-align :center">0.945
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B2</td><td style="text-align :center">328.667</td><td style="text-align :center">0.028</td><td style="text-align :center">1.000</td><td style="text-align :center">325</td><td style="text-align :center">0.746</td><td style="text-align :center">6.225</td><td style="text-align :center">0.972
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B3</td><td style="text-align :center">409.615</td><td style="text-align :center">0.018</td><td style="text-align :center">1.000</td><td style="text-align :center">405</td><td style="text-align :center">0.779</td><td style="text-align :center">6.749</td><td style="text-align :center">0.982
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B4</td><td style="text-align :center">689.942</td><td style="text-align :center">0.019</td><td style="text-align :center">1.000</td><td style="text-align :center">687</td><td style="text-align :center">0.751</td><td style="text-align :center">7.081</td><td style="text-align :center">0.981
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B5</td><td style="text-align :center">549.603</td><td style="text-align :center">0.135</td><td style="text-align :center">0.999</td><td style="text-align :center">530</td><td style="text-align :center">0.575</td><td style="text-align :center">5.204</td><td style="text-align :center">0.865
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B6</td><td style="text-align :center">344.333</td><td style="text-align :center">0.054</td><td style="text-align :center">0.999</td><td style="text-align :center">325</td><td style="text-align :center">0.679</td><td style="text-align :center">5.667</td><td style="text-align :center">0.946
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B7</td><td style="text-align :center">371.276</td><td style="text-align :center">0.053</td><td style="text-align :center">0.999</td><td style="text-align :center">343</td><td style="text-align :center">0.665</td><td style="text-align :center">5.603</td><td style="text-align :center">0.947
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B8</td><td style="text-align :center">400.923</td><td style="text-align :center">0.061</td><td style="text-align :center">0.999</td><td style="text-align :center">360</td><td style="text-align :center">0.636</td><td style="text-align :center">5.399</td><td style="text-align :center">0.939
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B9</td><td style="text-align :center">1221.665</td><td style="text-align :center">0.070</td><td style="text-align :center">0.996</td><td style="text-align :center">1121</td><td style="text-align :center">0.579</td><td style="text-align :center">5.863</td><td style="text-align :center">0.930
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B10</td><td style="text-align :center">285.714</td><td style="text-align :center">0.106</td><td style="text-align :center">1.000</td><td style="text-align :center">282</td><td style="text-align :center">0.588</td><td style="text-align :center">4.787</td><td style="text-align :center">0.894
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B11</td><td style="text-align :center">1431.557</td><td style="text-align :center">0.084</td><td style="text-align :center">0.994</td><td style="text-align :center">1294</td><td style="text-align :center">0.573</td><td style="text-align :center">5.921</td><td style="text-align :center">0.916
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B12</td><td style="text-align :center">581.000</td><td style="text-align :center">0.067</td><td style="text-align :center">0.998</td><td style="text-align :center">540</td><td style="text-align :center">0.561</td><td style="text-align :center">5.089</td><td style="text-align :center">0.933
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B13</td><td style="text-align :center">410.500</td><td style="text-align :center">0.119</td><td style="text-align :center">0.998</td><td style="text-align :center">375</td><td style="text-align :center">0.511</td><td style="text-align :center">4.368</td><td style="text-align :center">0.881
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B14</td><td style="text-align :center">310.091</td><td style="text-align :center">0.030</td><td style="text-align :center">0.999</td><td style="text-align :center">296</td><td style="text-align :center">0.709</td><td style="text-align :center">5.817</td><td style="text-align :center">0.970
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B15</td><td style="text-align :center">307.913</td><td style="text-align :center">0.110</td><td style="text-align :center">0.999</td><td style="text-align :center">289</td><td style="text-align :center">0.615</td><td style="text-align :center">5.030</td><td style="text-align :center">0.890
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B16</td><td style="text-align :center">346.500</td><td style="text-align :center">0.047</td><td style="text-align :center">0.999</td><td style="text-align :center">330</td><td style="text-align :center">0.683</td><td style="text-align :center">5.716</td><td style="text-align :center">0.953
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B17</td><td style="text-align :center">450.615</td><td style="text-align :center">0.105</td><td style="text-align :center">0.999</td><td style="text-align :center">421</td><td style="text-align :center">0.590</td><td style="text-align :center">5.146</td><td style="text-align :center">0.895
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B18</td><td style="text-align :center">398.216</td><td style="text-align :center">0.060</td><td style="text-align :center">0.999</td><td style="text-align :center">388</td><td style="text-align :center">0.682</td><td style="text-align :center">5.867</td><td style="text-align :center">0.940
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B19</td><td style="text-align :center">361.034</td><td style="text-align :center">0.059</td><td style="text-align :center">0.999</td><td style="text-align :center">345</td><td style="text-align :center">0.643</td><td style="text-align :center">5.422</td><td style="text-align :center">0.941
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B20</td><td style="text-align :center">655.900</td><td style="text-align :center">0.066</td><td style="text-align :center">0.998</td><td style="text-align :center">619</td><td style="text-align :center">0.602</td><td style="text-align :center">5.579</td><td style="text-align :center">0.934
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C1</td><td style="text-align :center">211.550</td><td style="text-align :center">0.120</td><td style="text-align :center">1.000</td><td style="text-align :center">207</td><td style="text-align :center">0.567</td><td style="text-align :center">4.364</td><td style="text-align :center">0.880
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C2</td><td style="text-align :center">357.920</td><td style="text-align :center">0.149</td><td style="text-align :center">0.999</td><td style="text-align :center">326</td><td style="text-align :center">0.497</td><td style="text-align :center">4.149</td><td style="text-align :center">0.851
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C3</td><td style="text-align :center">895.821</td><td style="text-align :center">0.018</td><td style="text-align :center">0.998</td><td style="text-align :center">851</td><td style="text-align :center">0.753</td><td style="text-align :center">7.325</td><td style="text-align :center">0.982
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C4</td><td style="text-align :center">518.808</td><td style="text-align :center">0.046</td><td style="text-align :center">0.999</td><td style="text-align :center">475</td><td style="text-align :center">0.664</td><td style="text-align :center">5.906</td><td style="text-align :center">0.954
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C5</td><td style="text-align :center">460.294</td><td style="text-align :center">0.058</td><td style="text-align :center">0.999</td><td style="text-align :center">415</td><td style="text-align :center">0.609</td><td style="text-align :center">5.299</td><td style="text-align :center">0.942
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C6</td><td style="text-align :center">286.050</td><td style="text-align :center">0.083</td><td style="text-align :center">0.999</td><td style="text-align :center">258</td><td style="text-align :center">0.601</td><td style="text-align :center">4.817</td><td style="text-align :center">0.917
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C7</td><td style="text-align :center">931.432</td><td style="text-align :center">0.026</td><td style="text-align :center">0.998</td><td style="text-align :center">866</td><td style="text-align :center">0.695</td><td style="text-align :center">6.780</td><td style="text-align :center">0.974
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C8</td><td style="text-align :center">585.400</td><td style="text-align :center">0.062</td><td style="text-align :center">0.998</td><td style="text-align :center">527</td><td style="text-align :center">0.623</td><td style="text-align :center">5.631</td><td style="text-align :center">0.938
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C9</td><td style="text-align :center">730.983</td><td style="text-align :center">0.027</td><td style="text-align :center">0.999</td><td style="text-align :center">694</td><td style="text-align :center">0.709</td><td style="text-align :center">6.691</td><td style="text-align :center">0.973
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C10</td><td style="text-align :center">280.857</td><td style="text-align :center">0.053</td><td style="text-align :center">0.999</td><td style="text-align :center">271</td><td style="text-align :center">0.656</td><td style="text-align :center">5.302</td><td style="text-align :center">0.947
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C11</td><td style="text-align :center">352.562</td><td style="text-align :center">0.082</td><td style="text-align :center">0.999</td><td style="text-align :center">323</td><td style="text-align :center">0.606</td><td style="text-align :center">5.049</td><td style="text-align :center">0.918
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C12</td><td style="text-align :center">271.882</td><td style="text-align :center">0.085</td><td style="text-align :center">1.000</td><td style="text-align :center">271</td><td style="text-align :center">0.620</td><td style="text-align :center">5.013</td><td style="text-align :center">0.915
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C13</td><td style="text-align :center">879.084</td><td style="text-align :center">0.088</td><td style="text-align :center">0.996</td><td style="text-align :center">772</td><td style="text-align :center">0.538</td><td style="text-align :center">5.160</td><td style="text-align :center">0.912
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C14</td><td style="text-align :center">740.776</td><td style="text-align :center">0.046</td><td style="text-align :center">0.998</td><td style="text-align :center">710</td><td style="text-align :center">0.618</td><td style="text-align :center">5.851</td><td style="text-align :center">0.954
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C15</td><td style="text-align :center">535.667</td><td style="text-align :center">0.045</td><td style="text-align :center">0.999</td><td style="text-align :center">501</td><td style="text-align :center">0.658</td><td style="text-align :center">5.898</td><td style="text-align :center">0.955
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C16</td><td style="text-align :center">349.714</td><td style="text-align :center">0.028</td><td style="text-align :center">0.999</td><td style="text-align :center">339</td><td style="text-align :center">0.726</td><td style="text-align :center">6.103</td><td style="text-align :center">0.972
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C17</td><td style="text-align :center">430.727</td><td style="text-align :center">0.039</td><td style="text-align :center">0.999</td><td style="text-align :center">404</td><td style="text-align :center">0.687</td><td style="text-align :center">5.951</td><td style="text-align :center">0.961
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C18</td><td style="text-align :center">360.091</td><td style="text-align :center">0.048</td><td style="text-align :center">0.999</td><td style="text-align :center">351</td><td style="text-align :center">0.672</td><td style="text-align :center">5.680</td><td style="text-align :center">0.952
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C19</td><td style="text-align :center">384.500</td><td style="text-align :center">0.024</td><td style="text-align :center">0.999</td><td style="text-align :center">368</td><td style="text-align :center">0.731</td><td style="text-align :center">6.232</td><td style="text-align :center">0.976
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">C20</td><td style="text-align :center">276.240</td><td style="text-align :center">0.105</td><td style="text-align :center">1.000</td><td style="text-align :center">267</td><td style="text-align :center">0.570</td><td style="text-align :center">4.595</td><td style="text-align :center">0.895
</td></tr></tbody>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >The definition of each alpha diversity indices can be found in QA part.</p>
<p class="premark" >(1) chao1: The total number of species contained in the community sample was estimated. The more low-abundance species in the community, the greater the chao1 index.</p>
<p class="premark" >(2) dominance: The smaller the probability of randomly selecting two sequences from the same sample, the better the homogeneity of community species, and the smaller the index.</p>
<p class="premark" >(3) goods_coverage: The higher the sequencing coverage, the greater the index.</p>
<p class="premark" >(4)observed_features: The number of species observed, the higher the index, the more species observed.</p>
<p class="premark" >(5) pielou_e: Uniformity index, the more uniform the species, the greater the pielou_e.</p>
<p class="premark" >(6) shannon: The total number of taxa in the samples and their percentage. The higher the community diversity and the more evenly distributed the species, the greater the shannon index.</p>
<p class="premark" >(7) simpson: The diversity and evenness of species distribution within the community were characterized, and the better the species evenness, the greater the Simpson index.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Alpha Diversity indices statistics: result/04.AlphaDiversity/sample/1_2.alpha_index_table/alpha_diversity_index.sample.txt</p>

    <a name="T4"></a>
    <h4>4.3.2 Biodiversity curves:</h4>    

    <p class="paragraph">Rarefaction curves and Rank abundance curves are widely used for indicating the biodiversity of the samples. A rarefaction curve is created by selecting randomly certain amount of sequencing data from the samples, then counting the number of the species they represent (i.e.,the number of OTUs). The Rarefaction curves can directly reflect the rationality of the sequencing data volume and indirectly reflect the richness of microbial community in the samples. If the curve is steep, it means that a lot of the species remain to be discovered. If the curve becomes flatter, a credible number of samples have been taken, which means only the scarce species remain to be sampled. (number of reads chosen for normalization: cutoff=46733.)</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureRarefactionCurves_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureRarefactionCurves_nav-tabContent">

<div class="tab-pane active show" id="shannonpng1" role="tabpanel" aria-labelledby="shannonpng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/shannon.png" /></div>

<div class="tab-pane fade" id="observedfeaturespng2" role="tabpanel" aria-labelledby="observedfeaturespng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/observed_features.png" /></div>

<div class="tab-pane fade" id="dominancepng3" role="tabpanel" aria-labelledby="dominancepng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/dominance.png" /></div>

<div class="tab-pane fade" id="chaopng4" role="tabpanel" aria-labelledby="chaopng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/chao1.png" /></div>

<div class="tab-pane fade" id="pielouepng5" role="tabpanel" aria-labelledby="pielouepng5" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/pielou_e.png" /></div>

<div class="tab-pane fade" id="goodscoveragepng6" role="tabpanel" aria-labelledby="goodscoveragepng6" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/goods_coverage.png" /></div>

<div class="tab-pane fade" id="simpsonpng7" role="tabpanel" aria-labelledby="simpsonpng7" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/simpson.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureRarefactionCurves_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureRarefactionCurves_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureRarefactionCurves_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="shannonpng1" data-toggle="list" href="#shannonpng1" role="tab" aria-controls="shannonpng1">shannon</a>

<a class="list-group-item list-group-item-action" id="observedfeaturespng2" data-toggle="list" href="#observedfeaturespng2" role="tab" aria-controls="observedfeaturespng2">observed_features</a>

<a class="list-group-item list-group-item-action" id="dominancepng3" data-toggle="list" href="#dominancepng3" role="tab" aria-controls="dominancepng3">dominance</a>

<a class="list-group-item list-group-item-action" id="chaopng4" data-toggle="list" href="#chaopng4" role="tab" aria-controls="chaopng4">chao1</a>

<a class="list-group-item list-group-item-action" id="pielouepng5" data-toggle="list" href="#pielouepng5" role="tab" aria-controls="pielouepng5">pielou_e</a>

<a class="list-group-item list-group-item-action" id="goodscoveragepng6" data-toggle="list" href="#goodscoveragepng6" role="tab" aria-controls="goodscoveragepng6">goods_coverage</a>

<a class="list-group-item list-group-item-action" id="simpsonpng7" data-toggle="list" href="#simpsonpng7" role="tab" aria-controls="simpsonpng7">simpson</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.8 Rarefaction Curves</strong></p>  

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: in the Rarefaction curves, each curve represents a sample, and can be colored and shaped by each sample name supplied in the mapping file. The sequence number is on the X-axis and the observed OTUs number is on the Y-axis.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Species accumulation boxplot is used to show the diversity and also be used to judge the adequacy of sample size and predict the richness of microbial community. The result is shown the in the figure as follow:</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/specaccum_test.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        

<p class="center">
<img src="src/pictures/specaccum_test.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.9 Species accumulation boxplot</strong></p>  
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: plotted by the "Sample num" on the X-axis and "OTU_num" on the Y-axis. With the increase of sample size, the boxplot showing a sharp rise indicates large amount of species found in samples while the boxplot turning to flat represents adequate sample numbers.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Rank abundance curve is used to sort the feature sequences in the sample through the relative abundance (or the number of sequences included) from the largest to the smallest in order to obtain the corresponding sorting number. By setting the OTUs sorting number as the abscissa, and the relative abundance in the OTUs (also the relative percentage of the number of sequences in the OTUs) as the ordinate, the points were connected with polylines to obtain the Rank abundance curve, which can directly reflect the species richness and evenness within  the samples.</p>

<p class="center">
<img src="src/pictures/rank_abundance.sample.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.10 Rank Abundance curves of Alpha</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/rank_abundance.sample.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: in the Rarefaction curves, each curve represents a sample, and can be colored and shaped by each sample name supplied in the mapping file. The sequence number is on the X-axis and the observed OTUs number is on the Y-axis. For the Rank abundance curves, each curve represents a single sample, plotted by OTU relative abundance on the Y-axis and the OTU abundance rank on the X-axis, which can be colored and shaped by each sample name supplied in the mapping file.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">The result of Rarefaction curves: result/04.AlphaDiversity/visual_rarefaction/(sample,group)/*</p>

    <p class="paragraph">The result of Accumulation boxplot: result/04.AlphaDiversity/visual_specaccum/sample/*.(png,pdf)</p>

    <p class="paragraph">The result of Rank abundance curves: result/04.AlphaDiversity/visual_rankAbundance/(sample,group)/*.(png,pdf)</p>

    <a name="T4"></a>
    <h4>4.3.3 Difference of alpha diversity indices between groups</h4>    

    <p class="paragraph">Boxplots were formed to analyze the differences in Alpha Diversity indices between groups. T-test, Wilcox,Kruskal-Wallis and Tukey tests (Wilcox test are for 2 groups while Kruskal-Wallis and Tukey tests are for more than 2 groups) were performed for the significant difference analysis between groups. Boxplots based on Observed_species and Shannon indices are shown as follow, which can reflect the maximum, minimum, median and abnormal value of index from each groups:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBoxplotsfordifferenceofalphadiversityindices_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBoxplotsfordifferenceofalphadiversityindices_nav-tabContent">

<div class="tab-pane active show" id="shannonboxpng1" role="tabpanel" aria-labelledby="shannonboxpng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/shannon_box1.png" /></div>

<div class="tab-pane fade" id="observedfeaturesboxpng2" role="tabpanel" aria-labelledby="observedfeaturesboxpng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/observed_features_box1.png" /></div>

<div class="tab-pane fade" id="dominanceboxpng3" role="tabpanel" aria-labelledby="dominanceboxpng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/dominance_box1.png" /></div>

<div class="tab-pane fade" id="chaoboxpng4" role="tabpanel" aria-labelledby="chaoboxpng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/chao1_box1.png" /></div>

<div class="tab-pane fade" id="pieloueboxpng5" role="tabpanel" aria-labelledby="pieloueboxpng5" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/pielou_e_box1.png" /></div>

<div class="tab-pane fade" id="goodscoverageboxpng6" role="tabpanel" aria-labelledby="goodscoverageboxpng6" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/goods_coverage_box1.png" /></div>

<div class="tab-pane fade" id="simpsonboxpng7" role="tabpanel" aria-labelledby="simpsonboxpng7" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/simpson_box1.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBoxplotsfordifferenceofalphadiversityindices_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBoxplotsfordifferenceofalphadiversityindices_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBoxplotsfordifferenceofalphadiversityindices_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="shannonboxpng1" data-toggle="list" href="#shannonboxpng1" role="tab" aria-controls="shannonboxpng1">shannon</a>

<a class="list-group-item list-group-item-action" id="observedfeaturesboxpng2" data-toggle="list" href="#observedfeaturesboxpng2" role="tab" aria-controls="observedfeaturesboxpng2">observed_features</a>

<a class="list-group-item list-group-item-action" id="dominanceboxpng3" data-toggle="list" href="#dominanceboxpng3" role="tab" aria-controls="dominanceboxpng3">dominance</a>

<a class="list-group-item list-group-item-action" id="chaoboxpng4" data-toggle="list" href="#chaoboxpng4" role="tab" aria-controls="chaoboxpng4">chao1</a>

<a class="list-group-item list-group-item-action" id="pieloueboxpng5" data-toggle="list" href="#pieloueboxpng5" role="tab" aria-controls="pieloueboxpng5">pielou_e</a>

<a class="list-group-item list-group-item-action" id="goodscoverageboxpng6" data-toggle="list" href="#goodscoverageboxpng6" role="tab" aria-controls="goodscoverageboxpng6">goods_coverage</a>

<a class="list-group-item list-group-item-action" id="simpsonboxpng7" data-toggle="list" href="#simpsonboxpng7" role="tab" aria-controls="simpsonboxpng7">simpson</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.11 Boxplots for difference of alpha diversity indices</strong></p>  
<p class="premark" >Note: The horizontal axis of the box plot represent the groups, while the vertical axis represents the corresponding alpha diversity index value.</p>

    <p class="paragraph">Box plots with Alpha Diversity indices：</p>

    <p class="paragraph">Results directory: result/04.AlphaDiversity/visual_alphaDiversity/group/*</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q: What do alpha diversity results look for?</p>
<p class="questions" >The assessment of species richness and homogeneity within the samples is presented in result/03.AlphaDiversity/alpha_diversity_index.txt, which contains the results of seven different Alpha diversity indices (ACE, chao1, goods_coverage, observed_ speciesshannon, simpson, PD_whole_tree) were calculated, and the index values reflect the magnitude of complexity within the sample communities. By looking at the results of the significance test of the alpha diversity index between groupings (result/03.AlphaDiversity/Alpha_div/*.txt), the groupings with significantly higher or lower species diversity can be quickly found for further analysis in conjunction with biological treatments.</p>
<p class="asked" >Q: What is Alpha-diversity and Beta-diversity?</p>
<p class="questions" >Alpha-diversity is mainly concerned with the number of species in a focal homogeneous ecological context, and is therefore also referred to as diversity within an ecological context. In the analysis, several different Alpha-diversity indices of Observed-species, Chao1, Shannon, Simpson, and Good- coverage were selected to characterize the diversity and homogeneity of species distribution in the samples, and to visualize the sequencing depth and data volume.</p>
<p class="questions" >Reference website:</p>
<p class="questions" >Observed-species - observed_otus (<a href="http://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.observed_otus">http://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.observed_otus</a>);</p>
<p class="questions" >Coverage - the Good’s coverage (<a href="http://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.goods_coverage">http://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.goods_coverage</a>);</p>
<p class="questions" >Chao - the Chao1 estimator (<a href="https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.chao1.html">https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.chao1.html</a>);</p>
<p class="questions" >ACE - the ACE estimator (<a href="http://scikit- bio.org/docs/latest/generated/skbio.diversity.alpha.ace.html#skbio.diversity.alpha.ace">http://scikit- bio.org/docs/latest/generated/skbio.diversity.alpha.ace.html#skbio.diversity.alpha.ace</a>);</p>
<p class="questions" >Shannon - the Shannon index (<a href="https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.shannon.html">https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.shannon.html</a>);</p>
<p class="questions" >Simpson - the Simpson index (<a href="https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.simpson.html">https://scikit-bio.org/docs/latest/generated/skbio.diversity.alpha.simpson.html</a>);</p>
<p class="questions" >PD_whole_tree - PD_whole_tree index(<a href="http://scikit- bio.org/docs/latest/generated/skbio.diversity.alpha.faith_pd.html?highlight=pd#skbio.diversity.alpha.faith_pd">http://scikit- bio.org/docs/latest/generated/skbio.diversity.alpha.faith_pd.html?highlight=pd#skbio.diversity.alpha.faith_pd</a>)</p>
<p class="questions" >Beta-diversity refers to the dissimilarity of species composition between different communities along an environmental gradient or the rate of species turnover along an environmental gradient also known as inter-ecological diversity. We mainly use (un)weightedunifrac, i.e., weighted and unweighted unifrac algorithm, and also Bray-curtis algorithm. The unweightedunifrac algorithm calculates the distance matrix between samples without considering the ASV abundance, while the weighted unifrac takes into account the ASV abundance information. The Bray-curtis algorithm incorporates species abundance information, but does not take into account species evolutionary relationships.</p>
<p class="questions" >(un)weighted unifrac：<a href="http://en.wikipedia.org/wiki/UniFrac">http://en.wikipedia.org/wiki/UniFrac</a></p>
<p class="questions" >Bray-curtis：<a href="http://en.wikipedia.org/wiki/Bray-Curtis_dissimilarity">http://en.wikipedia.org/wiki/Bray-Curtis_dissimilarity</a></p>
<p class="asked" >Q: Calculation of Alpha Diversity and the Principle of Sequence Extraction in Rarefaction Curve Plotting</p>
<p class="questions" >The default parameters of Qiime software were used to set the minimum number of extracted sequences as 10, the maximum number of extracted sequences as the minimum number of corresponding sequences among all samples, and the step size as (maximum number of extracted - minimum number of extracted)/10, with 10 repetitions of sampling in each step.</p>
<p class="questions" >For details, please refer to: <a href="http://qiime.org/scripts/alpha_diversity.html">http://qiime.org/scripts/alpha_diversity.html</a>.</p>
<p class="asked" >Q: What is the meaning and difference between several Alpha Diversity Indices?</p>
<p class="questions" >Observed_species index: The number of species (i.e., the number of ASVs) visually observed is counted by randomly sampling a certain amount of sequenced data from the sample. The Rarefaction Curve is a curve constructed from the amount of data extracted and the number of corresponding species. When the curve tends to be flat, it indicates that the amount of sequencing data is progressively reasonable, and more data will only produce a small number of new ASVs.</p>
<p class="questions" >Goods_coverage index: It is a more commonly used index of sequencing depth, which includes the number of ASVs containing only one sequence and the total number of sequences present in the sample in the calculation, so it can reflect the sequencing depth of the sample more realistically.</p>
<p class="questions" >Chao1 index: It is one of the widely used Alpha diversity measures in ecology to estimate the total number of species contained in community samples. It also provides a good representation of the presence of low abundance species in the community because it incorporates information on species with abundance of 1 and 2 in its calculation.</p>
<p class="questions" >ACE index (Abundance Coverage based Estimator): is an index used to estimate the number of ASVs in a community, and is one of the common indices used in ecology to estimate the total number of species. It differs from the Chao1 algorithm in that the number of species with only 1 occurrence, species with 10 or fewer occurrences, and species with more than 10 occurrences are counted separately in its calculation and used as the basis for assessing the number of unmeasured species. ACE takes into account not only the abundance of species but also the probability of species occurrence in the sample, and is therefore a better index of the overall condition of the community anyway.</p>
<p class="questions" >Shannon diversity index: also known as Shannon-Wiener or Shannon-Weaver index. It is calculated by taking into account the total number of taxa in the sample (Richness), and the proportion of each taxon (Abundance). The higher the community diversity and the more evenly distributed the species, the higher the Shannon index.</p>
<p class="questions" >Simpson Index: characterizes the diversity and evenness of species distribution within a community by calculating the probability that two randomly sampled individuals belong to different species. Simpson's index can be described at <a href="http://www.countrysideinfo.co.uk/simpsons.htm">http://www.countrysideinfo.co.uk/simpsons.htm</a></p>
<p class="questions" >PD_whole_tree Index: PD index is calculated based on evolutionary distance, and it responds to the kinship of species within the community. The more complex the kinship, the more distant the evolutionary distance, the larger the PD index.</p>
<p class="questions" >Both shannon and simpson indices are used to describe community diversity (species richness and evenness), while for simpson's index there are three presentations, namely Simpson's Index (D), Simpson's Index of Diversity (1 - D) and Simpson's Reciprocal Index (1 / D). They have similar effects in reflecting community diversity but the results are calculated in different forms. The Simpson's Index of Diversity (1 - D) is the probability that two randomly sampled individuals belong to different species, i.e., the larger the simpson's index, the higher the diversity of the community (the greater the probability that the two sampled individuals belong to different species), so the calculated results and the reflected trend are synchronized with the shannon index. There is also a Simpson's Index (D), which is the probability that two randomly sampled individuals in a community belong to the same species, which is the opposite of the shannon index.</p>
<p class="asked" >Q: Does the unsaturated ASV Rarefaction curve (not reaching plateau) mean that there is not enough data?</p>
<p class="questions" >The Rarefaction Curve, which is a random sampling of a certain amount of sequencing data from a sample, counts the number of species they represent (i.e., the number of ASVs). In a dilution curve, when the curve tends to be flat, it indicates that the amount of sequencing data is progressively reasonable, and more data volume will only generate a small number of new OTUs. However, as the amount of sequencing data continues to increase, the curve may have a tendency to increase under relatively relaxed QC conditions, mainly because of sequencing errors and increasing abundance information of sequencing products, causing the curve to maintain an increasing trend.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<h3>4.4 Beta Diversity Analysis</h3>
        
    <p class="paragraph">Beta Diversity is a comparative analysis of microbial community composition of different samples. First, according to the taxonomy annotation results of all samples and the abundance information of feature sequences, the feature sequence information of the same classification is combined to obtain the species abundance information table (Profiling Table). At the same time, the phylogenetic relationship between feature sequences is used to further calculate the unweighted unifrac distance (Lozupone C et al., 2005; Lozupone C et al., 2011). Unifrac distance is to calculate the distance between samples by using the evolutionary information between microbial sequences in each sample. If there are more than two samples, a distance matrix can be obtained. Then, the weighted unifrac distance (unweighted unifrac) is further constructed using the abundance information of the feature sequence (Lozupone CA et al., 2007). Finally, the differences between different samples (groups) were found through Beta diversity index inter group difference analysis, multivariate statistical methods such as Principal Component Analysis (PCA), Principal coordinates Analysis (PCoA), and Non-Metric Multidimensional Scaling (NMDS).</p>

    <a name="T4"></a>
    <h4>4.4.1 Beta diversity heatmap</h4>    

    <p class="paragraph">Weighted Unifrac distance and Unweighted Unifrac distance were selected to measure the dissimilarity coefficient between pairwise samples, which is a widely used phylogenetic measurement method in microbial community sequencing projects. The heatmap based on the Weighted Unifrac and Unweighted Unifrac distances is plotted in the figure as follow：</p>

<p class="center">
<img src="src/pictures/beta_diversity.heatmap1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.12 Beta diversity heatmap</strong></p>  
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >Notes: each grid represents pairwise dissimilarity coefficient between pairwise samples, in which Weighted Unifrac distance displayed above and Unweighted Unifrac distance conversely.</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory: </p>

    <p class="paragraph">Beta diversity heatmap: result/05.BetaDiversity/beta_unifracHeatmap/(sample,group)*(svg,png)</p>

    <a name="T4"></a>
    <h4>4.4.2 Unweighted Pair-group Method with Arithmetic Mean(UPGMA)</h4>    

    <p class="paragraph">To study the similarity among different samples, clustering analysis was used to construct a cluster tree. The Unweighted Pair-group Method with Arithmetic Mean (UPGMA) is a hierarchical clustering method widely used in ecology for the classification of samples (Backeljau T et al.,1996). The basic ideas of UPGMA are as follow. First, samples with the closest distance are clustered together to form a new node (as a new sample). Its branching point is 1/2 away from the original two samples. Then the average distance between the newly created "sample" and other samples is calculated and the two closest samples could be found again to repeat above steps. A complete clustering tree could be obtained until all samples are clustered together.</p>

    <p class="paragraph">Weighted Unifrac distance matrix was calculated before used for UPGMA cluster analysis. They were displayed with the integration of clustering results and the relative abundance of each sample by phylum in the figure as follow:</p>

<p class="center">
<img src="src/pictures/upgma1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.13 UPGMA cluster tree based on Weighted Unifrac distance</strong></p>  
</p>
<p class="premark" >Notes: plotted with UPGMA tree on the left and the relative abundance in phylum map on the right.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">UPGMA results in (un)weighted_unifrac distance: result/05.BetaDiversity/beat_upgma/(sample,group)/*</p>

    <a name="T4"></a>
    <h4>4.4.3 Principal Coordinates Analysis (PCoA)</h4>    

    <p class="paragraph">Principal Coordinates Analysis (PCoA) is an ordination technique, which picks up the main elements and structures from reduced multi-dimensional data series of eigenvalues and eigenvectors. The technique has the advantage over PCA in that each ecological distance can be investigated. Weighted Unifrac and Unweighted Unifrac are calculated to assist the PCoA analysis. The result is shown as follows:</p>

    <p class="paragraph">The 2D PCoA diagram is presented in the first and second principal coordinates. The display results are as follows:</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/PCoA1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        

<p class="center">
<img src="src/pictures/PCoA1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.14 PCoA based on Weighted Unifrac distance(left) and Unweighted Unifrac distance(right)</strong></p>  
</p>
<p class="premark" >Notes: each point represents a sample, plotted by a principal component on the X-axis and another principal component on the Y-axis, which was colored by group. The percentage on each axis indicates the contribution value to discrepancy among samples.</p>

    <p class="paragraph">And the 3D PCoA map selects three main coordinates for interactive web page display, and the coordinates can be flexibly adjusted. The display results are as follows (real PCoA figures of our sample can be viewed by clicking the link below):</p>

<p class="center">
<img src="src/pictures/3D_pcoa_display_en.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.15 3D PCoA</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/hyperlinks/weighted_unifrac_pcoa3d/index.html" target="_blank"><b>weighted_unifrac 3D PCoA To view full size picture please click here</b></a></p>
        

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/hyperlinks/unweighted_unifrac_pcoa3d/index.html" target="_blank"><b>unweighted_unifrac 3D PCoA To view full size picture please click here</b></a></p>
        

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">result/05.BetaDiversity/beta_pcoa/(sample,group)/*</p>

    <a name="T4"></a>
    <h4>4.4.4 Principal component analysis (PCA)</h4>    

    <p class="paragraph">PCA (Jolliffe I T et al., 1986) is a method based on Euclidean distances to apply variance decomposition to reduce the dimensions of multidimensional data, so as to extract the most important elements and structures in the data (Avershina E et al., 2013). PCA analysis can extract two coordinate axes that reflect the difference between samples to the greatest extent, so as to reflect the difference of multidimensional data on the two-dimensional coordinate map, and then reveal the simple rules under the background of complex data. If the community composition of the samples is more similar, their distances in the PCA map are closer. PCA analysis results based on feature sequence level are shown as follows:</p>

<p class="center">
<img src="src/pictures/PCA1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.16 PCA</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/PCA1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: each point represents a sample, plotted by the second principal component on the Y-axis and the first principal component on the X-axis, which was colored by group.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">PCA result: result/05.BetaDiversity/beta_pca/(sample,group)/*</p>

    <a name="T4"></a>
    <h4>4.4.5 Non-Metric Multidimensional Scaling (NMDS)</h4>    

    <p class="paragraph">Non-metric multidimensional scaling analysis is a ranking method applicable to ecological research (Noval et al., 2013). It’s a non-linear model designed for a better representation of non-linear biological data structure aiming at overcoming the flaws in methods based on a linear model, including PCA and PCoA. The result of NMDS analysis based on OTUs is in the figure as follows:</p>

<p class="center">
<img src="src/pictures/NMDS1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.17 NMDS based on Weighted Unifrac distance(left) and Unweighted Unifrac distance(right)</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/NMDS1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: each data point in the graph stands for a sample. The distance between data points reflects the extent of variation. Samples belongs to the same group are in the same color. When the value of Stress factor is less than 0.2, it’s considered that NMDS is reliable to some extent.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">NMDS result: result/05.BetaDiversity/beta_nmds/(sample,group)/*</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q: What do beta diversity results look for?</p>
<p class="questions" >For beta diversity analysis, that is, comparison of microbial community structure differences among samples, there are many analysis methods.</p>
<p class="questions" >The degree of difference in community structure between two samples can be visualized by the unifrac distance between two samples. The results are detailed in result/05.BetaDiversity/beta_distance/(sample,group)/(un)weighted_unifrac.(sample,group).txt</p>
<p class="questions" >The difference of this community structure can be displayed in two dimensions through PCA, PCoA and NMDS dimension reduction maps. The difference of community structure between groups (or samples) can also be judged by the aggregation and separation of grouped samples (or single samples).</p>
<p class="questions" >The samples can be clustered through the UPGMA clustering tree (result/05.BetaDiversity/beta_upgma), and the similarity clustering between samples can be seen more clearly.</p>
<p class="questions" >Through the analysis results of beta diversity, we can see whether the difference in community structure between samples is consistent with biological grouping, and explain this clustering or difference in combination with biological problems. In addition, UPGMA can interpret the clustering of samples in combination with the distribution of high abundance classification.</p>
<p class="questions" >For grouped projects, in-depth research can be conducted through statistical analysis. Through LEfSe analysis, Biomarker with statistical difference between groups can be found. T-test and MetagenomeSeq analysis can make significant analysis of species difference between groups at 6 levels of phyla, class, order, family, genus and species, and then obtain species with significant difference between two groups at different levels. Quantify the contribution of species to the difference through Simper analysis. Anosim and MRPP analysis can judge whether the difference of community structure between groups is significant, and can also compare the size of intra group difference and inter group difference.</p>
<p class="questions" >In addition, if you want to know the impact of environmental factors on the microbial community structure, or the important environmental driving factors affecting the distribution of samples, you can choose the environmental factor correlation analysis. Combined with the community diversity index, the environmental impact factors that significantly affect the species richness and evenness of the community can be obtained.</p>
<p class="asked" >Q: Interpretation of PCA analysis</p>
<p class="questions" >In multivariate statistical analysis, principal components analysis (PCA) is a technique for analyzing and simplifying data sets. Principal components analysis is often used to reduce the dimensionality of a dataset while maintaining the features in the dataset that contribute the most to each other's variance. This is done by keeping the lower-order principal components and ignoring the higher-order principal components. The mathematical definition of PCA is an orthogonalized linear transformation that transforms the data into a new coordinate system such that the first major variance of any projection of this data is at the first coordinate (called the first principal component), the second major variance is at the second coordinate (the second principal component), and so on. Given a set of points in Euclidean space, the first principal component corresponds to a line that averages the points through the multidimensional space while ensuring that the sum of squares of the distances from each point to this line is minimized. After removing the first principal component, the second principal component is obtained in the same way. The input data for the PCA plot is the sample microbial species abundance, with the first principal component, i.e., the dimension that contributes most to the difference in species abundance between samples, and the second principal component, i.e., the dimension that contributes second most to the difference in species abundance between samples, in the horizontal coordinate, and the second principal component, i.e., the dimension that contributes second most to the difference in species abundance between samples, in the vertical coordinate. Each sample point represents the projection of the sample point in the multidimensional space of species abundance on the plane of the first and second principal components.</p>
<p class="asked" >Q: PCA charting script</p>
<p class="questions" >PCA uses the dudi.pca function from the ade4 R package for Pca</p>
<p class="asked" >Q: What are the input files for PCoA?</p>
<p class="questions" >Input file: sorted_otu_table.Biom (OTU table in biom format), all.mf (grouping file), rep_Set.tre (evolution tree file), beta_params.txt (parameter file, which can not be set) to get the unifrac distance file (un)weighted_unifrac_dm.txt is used as the input to draw a PCoA diagram with R.</p>
<p class="asked" >Q: Interpretation of PCoA Analysis</p>
<p class="questions" >PCoA is a method to extract main variables from complex multidimensional variable data and visualize them. Similar to the main idea of PCA, the purpose of PCoA is to find some main coordinate systems in a matrix. In our 16S information analysis process, the input file of PCoA is the beta diversity distance matrix, which is the difference value matrix composed of the OTU abundance of two samples. After PCoA analysis, the following file is generated: where, is listed as the sample, representing the principal coordinate axis, and the lowest is the percentage of the characteristic value and the difference that can be explained by each coordinate.</p>
<p class="asked" >Q: Differences and Selection of PCoA/UPGMA Maps Plotted by Weighted and Unweighted Distances</p>
<p class="questions" >The difference between the two is that the former takes into account the abundance information of OTUs in the sample when calculating the distance between community samples, while the latter does not take into account the relative abundance information. If the biological problems studied are closely related to abundance information, the weighted results may be more appropriate. If the biological problems studied are not closely related to abundance, or the distinction between groups is more closely related to low abundance OTUs, the unweighted results may be more appropriate. If we do not know whether the biological problems studied are closely related to abundance, we can use this method to see which method is more effective in classification.</p>
<p class="asked" >Q: The analysis principle of (Un) Weighted Unifrac</p>
<p class="questions" >Our Unweighted Unifrac and Weighted Unifrac are obtained by using QIIME software. The process of building the Unifrac distance is as follows: for 16s rDNA, PyNAST can be used to build the phylogenetic relationship between OTUs, and further calculate the Unweighted Unifrac distance. Unifrac distance is to calculate the distance between samples by using the evolutionary information between microbial sequences in each sample. If there are more than two samples, a distance matrix can be obtained. Then, we use the abundance information of OTUs to further construct the Weighted Unifrac distance (Unweighted Unifrac).</p>
<p class="asked" >Q: When using QIIME to draw PCoA, there are two sets of data, discrete vs continuous, what is the difference between them?</p>
<p class="questions" >Discrete vs continuous refers to the method of coloring samples (groups). For example, for continuous factors such as PH, time, temperature, etc., continuous will color according to the color gradient from red to blue according to the grouping. On the contrary, if it is a discrete factor of different groups studied, it is recommended to use discrete. In a word, the difference between the two is that the coloring method is inconsistent when drawing, and the spatial distribution of PCoA is consistent.</p>
<p class="asked" >Q: What does UPGMA Weighted Unifrac mean? Why use it as an indicator?</p>
<p class="questions" >The two figures are UPGMA figures gathered by UniFrac software. Generally, the unweighted method is used when the research purpose is only concerned about the existence of species. If the species abundance needs to be considered for clustering, the weighted method is selected. It refers to the data type used in the calculation of distance matrix, and is a clustering distance widely used in microbial community research. If you are interested, you can refer to the literature in this field: Research on Statistical Methods in Macrogenomic Data Analysis. For a detailed description of UniFrac, see the website: <a href="http://www.tinygene.com/statistic-analysis/unifrac">http://www.tinygene.com/statistic-analysis/unifrac</a>.</p>
<p class="asked" >Q: The column graph in the unweighted clustering tree of UPGMA is inconsistent with that in the relative abundance column graph of top10. Why?</p>
<p class="questions" >The column graph in the UPGMA clustering tree selects the top 10 gate, which is divided into weighted and unweighted. Among them, based on the graph made by weighted distance, two factors were considered in the calculation: the existence of species and the abundance of species, such as the number of occurrences of OTUs corresponding to species A at the phylum level in the sample, and the abundance of each occurrence is a, b, etc. The number of occurrences of species A and the abundance of each occurrence are considered in the calculation. The result of the column graph here is similar to that of the top 10 relative abundance column graph. In the graph based on the unweighted distance, only the existence of species is considered in the calculation, for example, the OTU corresponding to species A at the phylum level is counted as 1 when it appears in the sample once, regardless of the abundance of the OTU corresponding to species A in the sample. Therefore, although the species is still top10, the corresponding column length will change.</p>
<p class="asked" >Q: What about the maximum, minimum, median, outliers and boundary values of the unweighted_unifrac box chart in the beta.div?</p>
<p class="questions" >These specific values are calculated according to the distance matrix and are intermediate files during our analysis, so they are not provided to the teacher in the results.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<h3>4.5 Community differences analysis</h3>
        
    <p class="paragraph">The Adonis and Anosim analysis methods allow the analysis of inter-group community structure variability. Through statistical analysis, species with significant variation in abundance between groups can be targeted and the enrichment of differential species within different subgroups can be obtained.</p>

    <a name="T4"></a>
    <h4>4.5.1 Analysis of Similarity (Anosim)</h4>    

    <p class="paragraph">Anosim analysis is a nonparametric test to evaluate whether variation among groups is significantly larger than variation within groups, which helps to evaluate the reasonability of the division of groups. The results are shown in the table as follow:</p>

<p class="name_table"><p class="center"><strong>Table 4.3 Anosim results</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable4" >
        <thead><tr>
<a id="btn04"></a>
<a id="sjzl4"></a>
<a  href="javascript:void(0)" id="btn14"> | First | </a>
<a  href="javascript:void(0)" id="btn24"> Previous | </a>
<a  href="javascript:void(0)" id="btn34"> Next | </a>
<a  href="javascript:void(0)" id="btn44"> Last | </a>
<a>Go to </a>
<input id="changePage4" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn54"> Jump </a>
                
<th style="text-align :center">Group</th><th style="text-align :center">R-value</th><th style="text-align :center">P-value
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">A-B</td><td style="text-align :center">0.18781</td><td style="text-align :center">0.001
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A-C</td><td style="text-align :center">0.21983</td><td style="text-align :center">0.001
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B-C</td><td style="text-align :center">-0.04244</td><td style="text-align :center">0.964
</td></tr></tbody>

</table>
</div>
</p>
<p class="premark" >Notes: R-value is between -1 and 1. A positive R value means that inter-group variation is considered significant, while a negative R-value suggests that inner-group variation is larger than inter-group variation, therefore, no significant differences. The confidence degree is represented by P-value, whose value less than 0.05 suggests statistical significance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Anosim result: result/06.StatisticalTest/anosim/*</p>

    <a name="T4"></a>
    <h4>4.5.2 Multi-response permutation procedure (MRPP) analysis</h4>    

    <p class="paragraph">MRPP analysis (O'Reilly FJ et al.,1980) similar with Anosim, which aims at determining whether the difference of microbial community structure among groups is significant. It’s usually applied with methods of dimension reduction like PCA, PCoA, and NMDS. The result is shown in the table as follow:</p>

<p class="name_table"><p class="center"><strong>Table 4.4 MRPP result</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable5" >
        <thead><tr>
<a id="btn05"></a>
<a id="sjzl5"></a>
<a  href="javascript:void(0)" id="btn15"> | First | </a>
<a  href="javascript:void(0)" id="btn25"> Previous | </a>
<a  href="javascript:void(0)" id="btn35"> Next | </a>
<a  href="javascript:void(0)" id="btn45"> Last | </a>
<a>Go to </a>
<input id="changePage5" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn55"> Jump </a>
                
<th style="text-align :center">Group</th><th style="text-align :center">A</th><th style="text-align :center">observed-delta</th><th style="text-align :center">expected-delta</th><th style="text-align :center">Significance
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">A-B</td><td style="text-align :center">0.01998</td><td style="text-align :center">0.67457</td><td style="text-align :center">0.68832</td><td style="text-align :center">0.001
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A-C</td><td style="text-align :center">0.02525</td><td style="text-align :center">0.6699</td><td style="text-align :center">0.68725</td><td style="text-align :center">0.002
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B-C</td><td style="text-align :center">-0.00575</td><td style="text-align :center">0.70815</td><td style="text-align :center">0.70411</td><td style="text-align :center">0.919
</td></tr></tbody>

</table>
</div>
</p>
<p class="premark" >A small value in the column titled observe-delta indicates that the inner-group variation is small, while a large one in the column of expected-delta means that the inter-group variation is great. A positive A-value suggests that variation among groups is larger than variation within groups, while a negative one shows the opposite relationship. The difference among groups is significant if the value in the column of Significance is less than 0.05.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">MRPP result: result/06.StatisticalTest/mrpp/*</p>

    <a name="T4"></a>
    <h4>4.5.3 Adonis</h4>    

    <p class="paragraph">ADONIS is also called permutational MANOVA or nonparametric MANOVA, which is a method of nonparametric multivariate variance test according to the Bray-Curtis distance matrix. This method can analyze the explanation of grouping factor on the difference of samples and estimate the significance of grouping by permutation test (Anon, 2001; McArdle et al., 2001; Hamidi et al., 2019; Zapala et al., 2006). The result is shown in the table as follow:</p>

<p class="name_table"><p class="center"><strong>Table 4.5 Adonis result</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable6" >
        <thead><tr>
<a id="btn06"></a>
<a id="sjzl6"></a>
<a  href="javascript:void(0)" id="btn16"> | First | </a>
<a  href="javascript:void(0)" id="btn26"> Previous | </a>
<a  href="javascript:void(0)" id="btn36"> Next | </a>
<a  href="javascript:void(0)" id="btn46"> Last | </a>
<a>Go to </a>
<input id="changePage6" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn56"> Jump </a>
                
<th style="text-align :center">Group</th><th style="text-align :center">Df</th><th style="text-align :center">SumsOfSqs</th><th style="text-align :center">MeanSqs</th><th style="text-align :center">F.Model</th><th style="text-align :center">R2</th><th style="text-align :center">Pr(>F)
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">A-B</td><td style="text-align :center">1(54)</td><td style="text-align :center">0.71433(12.7253)</td><td style="text-align :center">0.71433(0.23565)</td><td style="text-align :center">3.03127</td><td style="text-align :center">0.05315(0.94685)</td><td style="text-align :center">0.001
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">A-C</td><td style="text-align :center">1(54)</td><td style="text-align :center">0.85596(12.58322)</td><td style="text-align :center">0.85596(0.23302)</td><td style="text-align :center">3.67328</td><td style="text-align :center">0.06369(0.93631)</td><td style="text-align :center">0.001
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">B-C</td><td style="text-align :center">1(38)</td><td style="text-align :center">0.15385(9.77787)</td><td style="text-align :center">0.15385(0.25731)</td><td style="text-align :center">0.59789</td><td style="text-align :center">0.01549(0.98451)</td><td style="text-align :center">0.946
</td></tr></tbody>

</table>
</div>
</p>

    <p class="paragraph"><details><summary> Show Annotation </summary></p>
<p class="premark" >(1) Df: the degree of freedom</p>
<p class="premark" >(2) SumsOfSqs: sums of squares of deviations</p>
<p class="premark" >(3) MeanSqs: SumsOfSqs/Df (The parentheses indicate the results of data processing within the group, while the parentheses indicate the results of data processing between groups)</p>
<p class="premark" >(4) F.Model: the F-test value (higher F value, higher differential between group, more significant grouping factor)</p>
<p class="premark" >(5) R2: the explanation of grouping factor on the difference of samples, calculated from the ratio of grouping variance and total variance</p>
<p class="premark" >(6) Pr: P-value. Values in parentheses stand for Residual Error</p>

    <p class="paragraph"></details></p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Adonis result: result/06.StatisticalTest/adonis/*</p>

    <a name="T4"></a>
    <h4>4.5.4 Simper</h4>    

    <p class="paragraph">Simper (Similarity percentage) (Warton D I et al., 2012) is a decomposition of the Bray-Curtis difference index that quantifies the contribution of each species to the difference between the two groups. The results are presented for the top 10 species and their abundance that contribute to the difference between the two groups. The results are presented as follows:</p>

<p class="center">
<img src="src/pictures/simper1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.18 Variance Contribution of Simper</strong></p>  
</p>
<p class="premark" >Note: The top 10 species in terms of contribution were selected for the graph. The vertical axis represents the species, the horizontal axis is the sample, the bubble size represents the relative abundance of the species, and the Contribution is the contribution of the species in the two groups of variability.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Simper result: result/06.StatisticalTest/simper</p>

    <a name="T4"></a>
    <h4>4.5.5 T-test</h4>    

    <p class="paragraph">T-test is performed to determine species with significant variation between groups (p-value < 0.05) at various taxon ranks including phylum, class, order, family, genus, and species. The result is shown in the figure as follow:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureBetweengroupTtestanalysis_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureBetweengroupTtestanalysis_nav-tabContent">

<div class="tab-pane active show" id="CATtestpng1" role="tabpanel" aria-labelledby="CATtestpng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-A_Ttest1.png" /></div>

<div class="tab-pane fade" id="CBTtestpng2" role="tabpanel" aria-labelledby="CBTtestpng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-B_Ttest1.png" /></div>

<div class="tab-pane fade" id="BATtestpng3" role="tabpanel" aria-labelledby="BATtestpng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/B-A_Ttest1.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureBetweengroupTtestanalysis_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureBetweengroupTtestanalysis_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureBetweengroupTtestanalysis_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CATtestpng1" data-toggle="list" href="#CATtestpng1" role="tab" aria-controls="CATtestpng1">C-A</a>

<a class="list-group-item list-group-item-action" id="CBTtestpng2" data-toggle="list" href="#CBTtestpng2" role="tab" aria-controls="CBTtestpng2">C-B</a>

<a class="list-group-item list-group-item-action" id="BATtestpng3" data-toggle="list" href="#BATtestpng3" role="tab" aria-controls="BATtestpng3">B-A</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.19 Between-group T-test analysis</strong></p>  
<p class="premark" >Notes: The left panel is the abundance of species showing significant differences between groups. Each bar represents the mean value of the abundance in each group of the specie showing significant differences between groups. The right panel is the confidential interval of between-group variations. The left-most part of each circle stands for the lower limit of 95% confidential interval, while the right-most part is the upper limit. The center of the circle stands for the difference in the mean value. The color of the circle is in agreement with the group whose mean value is higher. The right-most value is the p-value of the significance test of between-group variation.</p>

    <p class="paragraph">The following is the resulting volcano plot, which by default shows the results at the same classification level as the above figure:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureVolcanoplot_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureVolcanoplot_nav-tabContent">

<div class="tab-pane active show" id="CATtestVolcanopng1" role="tabpanel" aria-labelledby="CATtestVolcanopng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-A_Ttest1.Volcano.png" /></div>

<div class="tab-pane fade" id="CBTtestVolcanopng2" role="tabpanel" aria-labelledby="CBTtestVolcanopng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-B_Ttest1.Volcano.png" /></div>

<div class="tab-pane fade" id="BATtestVolcanopng3" role="tabpanel" aria-labelledby="BATtestVolcanopng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/B-A_Ttest1.Volcano.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureVolcanoplot_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureVolcanoplot_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureVolcanoplot_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CATtestVolcanopng1" data-toggle="list" href="#CATtestVolcanopng1" role="tab" aria-controls="CATtestVolcanopng1">C-A</a>

<a class="list-group-item list-group-item-action" id="CBTtestVolcanopng2" data-toggle="list" href="#CBTtestVolcanopng2" role="tab" aria-controls="CBTtestVolcanopng2">C-B</a>

<a class="list-group-item list-group-item-action" id="BATtestVolcanopng3" data-toggle="list" href="#BATtestVolcanopng3" role="tab" aria-controls="BATtestVolcanopng3">B-A</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.20 Volcano plot</strong></p>  
<p class="premark" >A volcano plot is a presentation of a scatter plot, which usually consists of several parts, including significantly up-regulated difference species and significantly down-regulated difference species. Generally, the horizontal axis is the fold difference of the difference species in the comparison group, while the vertical axis is the p-value of the significant between-group difference test for the difference species. Each point in the graph represents a differential species, where Up represents the higher abundance of that differential species in the first comparison group than in the second comparison group, while Down is the opposite.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/06.StatisticalTest/ttest/*/*/*/*.(xls,png,pdf)</p>

    <p class="paragraph">Significantly different results: result/06.StatisticalTest/ttest/*/*/*/boxplot</p>

    <a name="T4"></a>
    <h4>4.5.6 MetagenomeSeq</h4>    

    <p class="paragraph">The metagenomeSeq method was used to test the hypothesis of species richness data between groups to obtain the p value, and the species with significant differences between groups were screened according to the p value (<0.05), and the abundance distribution box chart of different species among groups was drawn. The default display of results for phylum or genus levels is as follows:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureMetagenomeSeqanalysisbetweengroups_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureMetagenomeSeqanalysisbetweengroups_nav-tabContent">

<div class="tab-pane active show" id="CAmetagenomeSeqpng1" role="tabpanel" aria-labelledby="CAmetagenomeSeqpng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-A_metagenomeSeq1.png" /></div>

<div class="tab-pane fade" id="CBmetagenomeSeqpng2" role="tabpanel" aria-labelledby="CBmetagenomeSeqpng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-B_metagenomeSeq1.png" /></div>

<div class="tab-pane fade" id="BAmetagenomeSeqpng3" role="tabpanel" aria-labelledby="BAmetagenomeSeqpng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/B-A_metagenomeSeq1.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureMetagenomeSeqanalysisbetweengroups_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureMetagenomeSeqanalysisbetweengroups_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureMetagenomeSeqanalysisbetweengroups_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CAmetagenomeSeqpng1" data-toggle="list" href="#CAmetagenomeSeqpng1" role="tab" aria-controls="CAmetagenomeSeqpng1">C-A</a>

<a class="list-group-item list-group-item-action" id="CBmetagenomeSeqpng2" data-toggle="list" href="#CBmetagenomeSeqpng2" role="tab" aria-controls="CBmetagenomeSeqpng2">C-B</a>

<a class="list-group-item list-group-item-action" id="BAmetagenomeSeqpng3" data-toggle="list" href="#BAmetagenomeSeqpng3" role="tab" aria-controls="BAmetagenomeSeqpng3">B-A</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.21 MetagenomeSeq analysis between groups</strong></p>  
<p class="premark" >Notes: Plotted by relative abundance on the Y-axis and the group name on the X-axis. Horizontal line represents the two groups with significant variation. Each point represents a sample, and box charts of different colors represent different groups. There are only a maximum of nine images in the displayed results, and the rest can be found in the result folder.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/06.StatisticalTest/metagenomeSeq/*/*/*/*.(xls,png,pdf)</p>

    <p class="paragraph">Significantly different results: result/06.StatisticalTest/metagenomeSeq/*/*/*/boxplot</p>

    <a name="T4"></a>
    <h4>4.5.7 LEfSe</h4>    

    <p class="paragraph">LEfSe (linear discriminant analysis (LDA) Effect Size) was used to analyze and detect biomarkers (Segata et al., 2011).</p>

    <p class="paragraph">LEfSe is a software designed to discover high-dimensional biomarkers and reveal metagenomic characteristics (including genes, metabolites or taxa), so it can be used to distinguish two or more biological classes. It emphasizes statistical significance, biological consistency, and effect correlation, and allows researchers to identify characteristics of abundance and related classes. The result was composed of a histogram of LDA scores and the Cladogram among groups. The histogram of the LDA scores and Cladogram are shown as the results of LEfSe analysis for evaluating of biomarkers with statistically difference among groups. The results are as follow:</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureHistogramofLDAscores_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureHistogramofLDAscores_nav-tabContent">

<div class="tab-pane active show" id="CALDApng1" role="tabpanel" aria-labelledby="CALDApng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-A_LDA1.png" /></div>

<div class="tab-pane fade" id="ABCLDApng2" role="tabpanel" aria-labelledby="ABCLDApng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/A-B-C_LDA1.png" /></div>

<div class="tab-pane fade" id="CBLDApng3" role="tabpanel" aria-labelledby="CBLDApng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-B_LDA1.png" /></div>

<div class="tab-pane fade" id="BALDApng4" role="tabpanel" aria-labelledby="BALDApng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/B-A_LDA1.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureHistogramofLDAscores_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureHistogramofLDAscores_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureHistogramofLDAscores_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CALDApng1" data-toggle="list" href="#CALDApng1" role="tab" aria-controls="CALDApng1">C-A</a>

<a class="list-group-item list-group-item-action" id="ABCLDApng2" data-toggle="list" href="#ABCLDApng2" role="tab" aria-controls="ABCLDApng2">A-B-C</a>

<a class="list-group-item list-group-item-action" id="CBLDApng3" data-toggle="list" href="#CBLDApng3" role="tab" aria-controls="CBLDApng3">C-B</a>

<a class="list-group-item list-group-item-action" id="BALDApng4" data-toggle="list" href="#BALDApng4" role="tab" aria-controls="BALDApng4">B-A</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.22 Histogram of LDA scores</strong></p>  
<p class="premark" >Notes: The histogram of the LDA scores represents species(biomarker) whose abundance shows significant differences among groups. The selecting criteria is that LDA scores are larger than the set threshold (4 set by default).</p>
<p class="premark" >The length of each bin, namely, the LDA score, represents the effect size (the extent to which a biomarker can explain the differentiating phenotypes among groups).</p>
<p class="premark" >If a group in the figure is missing, it indicates that there is no species with significant difference in this group.</p>
<p class="premark" >The names of species represented by English letters in the figure are shown in the legend on the right.</p>

<p class="center">
<div class="row gallery-row">
<div class="col-1">
<div id="FigureCladogram_button_prev" class="button_prev" onclick="prev(this.id)"><svg viewbox="0 0 16000 16000" class="svg_prev"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 "></polygon></svg></div>
</div>
<div class="col-6">
<div class="tab-content" id="FigureCladogram_nav-tabContent">

<div class="tab-pane active show" id="CALDAtreepng1" role="tabpanel" aria-labelledby="CALDAtreepng1" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-A_LDA1.tree.png" /></div>

<div class="tab-pane fade" id="ABCLDAtreepng2" role="tabpanel" aria-labelledby="ABCLDAtreepng2" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/A-B-C_LDA1.tree.png" /></div>

<div class="tab-pane fade" id="CBLDAtreepng3" role="tabpanel" aria-labelledby="CBLDAtreepng3" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/C-B_LDA1.tree.png" /></div>

<div class="tab-pane fade" id="BALDAtreepng4" role="tabpanel" aria-labelledby="BALDAtreepng4" onclick="trans(this.id)"><img class="img-gallery" src="src/pictures/B-A_LDA1.tree.png" /></div>

</div>
</div>
<div class="col-1">
<div id="FigureCladogram_button_next" class="button_next" onclick="next(this.id)"><svg viewbox="0 0 16000 16000" class="svg_next"><circle class="c" cx="8000" cy="8000" r="6080"></circle><polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 "></polygon></svg></div>
</div>
<div class="col-4">
<input class="form-control" id="FigureCladogram_InputList" type="text" placeholder="Search..">
<div class="list-group" id="FigureCladogram_list-tab" role="tablist">

<a class="list-group-item list-group-item-action" id="CALDAtreepng1" data-toggle="list" href="#CALDAtreepng1" role="tab" aria-controls="CALDAtreepng1">C-A</a>

<a class="list-group-item list-group-item-action" id="ABCLDAtreepng2" data-toggle="list" href="#ABCLDAtreepng2" role="tab" aria-controls="ABCLDAtreepng2">A-B-C</a>

<a class="list-group-item list-group-item-action" id="CBLDAtreepng3" data-toggle="list" href="#CBLDAtreepng3" role="tab" aria-controls="CBLDAtreepng3">C-B</a>

<a class="list-group-item list-group-item-action" id="BALDAtreepng4" data-toggle="list" href="#BALDAtreepng4" role="tab" aria-controls="BALDAtreepng4">B-A</a>

</div>
</div>
</div>
<p class="name_fig"><strong>Figure 4.23 Cladogram</strong></p>  
<p class="premark" >Notes: In Cladogram, circles radiating from inner side to outer side represents taxonomic rank from phylum to genus (species). Each circle stands for a distinct taxon at corresponding taxonomic rank. The diameter of each circle represents proportionally the relative abundance of each taxon. Yellow stands for taxons with non-significant differences." </p>
<p class="premark" >Taxons (biomarkers) with significant differences are colored according to corresponding group’s color. Red nodes mean these microbiota contributes a lot in the group covered by red color, so do the green nodes. Letters above the circles and corresponding species are annotated on the right side.</p>
<p class="premark" >If a group in the figure is missing, it indicates that there is no species with significant difference in this group.</p>
<p class="premark" >The names of species represented by English letters in the figure are shown in the legend on the right.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">LEfSe cladogram: result/06.StatisticalTest/lefse/*/*/*.(res,png,pdf)</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q：Details of significance test of inter-group community structure variability</p>
<p class="questions" >(1) Anosim (Analysis of similarities) is a non parametric test method. It first calculates the relationship (or similarity) between samples through variables, then calculates the relationship ranking, and finally determines whether the difference between groups is significantly different from the difference within groups through ranking for replacement test. This test has two important values, one is the p value, which can judge whether the comparison between groups and within groups is significant. The other is the R value, which can be used to determine the difference between groups and within groups. Anosim is used to test whether the difference between groups is significantly greater than the difference within groups, so as to judge whether grouping is meaningful. The specific calculation process: <a href="https://www.cnblogs.com/ZhengAbel/p/12016362.html">https://www.cnblogs.com/ZhengAbel/p/12016362.html</a>. </p>
<p class="questions" >(2) MRPP analysis is similar to Anosim, but MRPP is based on the Bray-Curtis parametric test for analyzing whether differences in microbial community structure between groups are significant, and is usually used in conjunction with downscaled plots such as PCA, PCoA, and NMDS. The specific calculation process: <a href="https://www.rdocumentation.org/packages/Blossom/versions/1.4/topics/mrpp">https://www.rdocumentation.org/packages/Blossom/versions/1.4/topics/mrpp</a>. </p>
<p class="questions" >(3) ADONIS (permanent MANOVA), also known as nonparametric MANOVA, is a non parametric multivariate ANOVA method based on distance matrix (such as Bray Curtis, Euclidean, etc.). The adonis in the report is Permanova. In addition, Adonis analysis can also help determine whether the community differences between groups are significant.</p>
<p class="asked" >Q: In Anosim and MRPP analysis, all pairwise comparison P values returned from the two kinds of analysis are 0.1, but it can be clearly seen from the PCoA diagram that they are obviously separated. How to explain this phenomenon? </p>
<p class="questions" >For Anosim and MRPP, the sample size should be large (more than 10, the more the better). The failure to reach a significant level may be caused by the following two reasons: 1. Even though the dimension reduction map score is relatively open, it does not reach a significant level. 2. For the case that there are only three samples in each group, because the sample size is too small, its internal algorithm is difficult to find the case with significant difference (p<=0.05)</p>
<p class="asked" >Q: How to explain that the R value is greater than 0 and the p value is greater than 0.05 in Anosim analysis ?</p>
<p class="questions" >Anosim analysis is to get the R value first. If the R value is greater than 0, the difference between the groups is large, but only the R value is not enough to indicate the difference between the two groups. There is also a need for a p value. If the p value is greater than 0.05, the result that the R value is greater than 0 has no credibility.</p>
<p class="asked" >Q: Methods for comparing the differences of community structure composition among groups</p>
<p class="questions" >If you want to get the difference of community structure composition between different treatments, you can use MetagenomeSeq analysis to extract the species information with significant difference between groups. You can also use the distance matrix between groups as input to conduct Anosim analysis. The result is a value similar to p-value, which can be used to judge whether there is significant difference between groups.</p>
<p class="asked" >Q:Interpretation of the results of MetagenomeSeq analysis of significant differences between groups</p>
<p class="questions" >We will provide the analysis results of significant differences between groups at the level of phylum, class, order, family, genus and species.</p>
<p class="questions" >Note: We used MetagenomeSeq (<a href="http://cbcb.umd.edu/software/metagenomeSeq">http://cbcb.umd.edu/software/metagenomeSeq</a>) software to obtain p-values by hypothesis testing of species abundance data between groups, and q-values by correcting for p-values. Finally, the species with significant differences were screened according to p or q values.</p>
<p class="questions" >*.psig.xls is the species selected for significant differences between groups (|logFC|>1 and P <0.05).</p>
<p class="questions" >Table description: Taxo is species classification information.logFC: log-fold-changes between each pair of  groups(the value is calculated internally by the software); se: Standard Error; pvalues: P value; adjPvalues: q value (adjust p-value).</p>
<p class="asked" >Interpretation of the results of LEfSe</p>
<p class="asked" >Q: LEfSe software parameter setting</p>
<p class="questions" >The default setting of the LDA score in the lefse software is 2. The size of the LDA score represents the size of the effect of the difference species, and anything with an LDA score greater than 2 is a plausible difference species, and the larger the value, the greater the effect of the difference species. In the literature, the LDA score is set to 2, 3, or 4. We used the LDA score of 4 because it is more stringent than 2. However, if the difference between the teacher's sample groups is not very large and the number of species found to be different is small, which is not enough to meet the analysis requirements, we can lower the LDA score to do so in order to find more species that are different.</p>
<p class="asked" >Q: Specific content description of lefse.res document</p>
<p class="questions" >There is an LDA.1.res file in the lefse result file, which is the default output format of lefse. The output: *.res format file contains the following two lines.</p>
<p class="questions" >Bacteria.Firmicutes.Clostridia.Clostridiales.Ruminococcaceae 5.0923016841 Low_O2 4.74694106197 2.91304680962e-07Bacteria.Tenericutes.Mollicutes.Mycoplasmatales.Mycoplasmataceae.Mycoplasma2.55257491798</p>
<p class="questions" >There are 5 columns: the first column is the name of the biomarker, the second column is the value of log10 with the highest mean abundance, if the mean abundance is less than 10 it is calculated as 10, the third column is the name of the group of differential genes or species enrichment, the fourth column is the LDA value, the fifth column is the p-value of Kruskal-Wallis rank sum test, and if it is not a biomarker it is indicated by "-" indicates .</p>
<p class="asked" >Q: The grouping colors in LEfSe evolutionary branching graph and LDA column graph are less than the number of mapping groups.</p>
<p class="questions" >In the display results of LEfSe, all display species have significant differences among all groups. The coloring principle of these species is: in which group is enriched (in which group is the highest in abundance), the color of this group is displayed. Therefore, if a specific group does not have the current display diagram, The reason is that the abundance of all species with significant differences between these groups in this group is low or non-existent (that is, these species are not enriched in this group).</p>
<p class="asked" >Q: Interpretation of abundance comparison chart of different species in different groups in LEfSe analysis</p>
<p class="questions" >For example:</p>
<p class="questions" >The abundance distribution of different groups was set to 1 in the sample with the highest abundance, and the abundance of this differential species in other groups was the relative value relative to the group with the highest abundance.</p>
<p class="questions" >Meaning of the solid and dashed lines in the graph: the solid and dashed lines indicate the mean and median values of the relative abundance of each sample in the grouping, respectively.</p>
<p class="questions" >If there is no column in a group, it means that there is no such species of interest in the group.</p>
<p class="asked" >Q: What is the meaning of the classification name at the top of the fan in the Cladogram</p>
<p class="questions" >The classification names in the Cladogram (such as Actinobacteria) represent the consistency of the variation trend of statistical and biological grouping differences. Each leaf node in the figure corresponds to a species, and the diameter of its circle at each leaf node is proportional to the corresponding species abundance. The coloring of differences is based on the grouping of species with high abundance. If the change trend of the leaf node of the lower level classification level is the same as that of the ancestor branch of the upper level in different groups, the corresponding upper level classification name will be marked in the figure to get some representative differential classification information.</p>
<p class="asked" >Q: What information can be obtained from the LDA.Cladogram?</p>
<p class="questions" >In the LDA.Cladogram diagram, the circle radiating from inside to outside represents the classification level from Phylum to Genus (or Species). Each small circle at different classification levels represents a classification at that level. The diameter of the small circle is proportional to the relative abundance. Coloring principle: the species without significant differences are uniformly colored in yellow, and the different species Biomarker follows the group to color. The red node represents the microbial group that plays an important role in the red group, and the green node represents the microbial group that plays an important role in the green group.</p>
<p class="asked" >Q: Why is the number of species in LDA.Cladogram different from that in LDA.Histogram ?</p>
<p class="questions" >The LDA.Cladogram only shows the species of class, order and family, while the LDA.Histogram shows all the species with significant differences.</p>
<p class="asked" >Q：What is the difference between LEfSe (LDA Effect Size) and MetagenomeSeq? How to explain the case of few biomakers?</p>
<p class="questions" >Lefse and MetagenomeSeq are two different presentation forms. Their results are very different because of their different algorithms, and Lefse is a multi group analysis, while MetagenomeSeq is a comparison between two groups.</p>
<p class="questions" >Lefse algorithm: firstly, use nonparametric factorial Kruskal Wallis (KW) sum rank test (t nonparametric factor Kruskal Wallis and rank test) to detect species with significant difference in abundance between different groups, then use the grouped Wilcoxon rank sum test to judge the difference between groups, and finally use linear discriminant analysis (LDA) to reduce the dimension and evaluate the impact of species with significant difference (i.e. LDA Score).</p>
<p class="questions" >MetagenomeSeq algorithm: firstly, hypothesis test is carried out on the species abundance data between groups to obtain the p value, and the corrected q value is obtained by correcting the p value. Finally, the species with significant differences were screened according to p or q values. The significance analysis of species difference among groups was conducted at 6 levels of phylum, class, order, family, genus and species, and the species with significant difference were obtained at different levels.</p>
<p class="questions" >So if you want to see the different species between two groups, you can refer to the MetagenomeSeq results. If you want to see multiple groups together, you can refer to lefse.</p>
<p class="questions" >The reason why Biomakers are rare is that on the one hand, our Lefse score values are strictly set, which is 4 by default and 2 by default for Lefse software. On the other hand, it may be that there is no significant difference among species.</p>
<p class="asked" >Interpretation of the results of MetagenomeSeq analysis</p>
<p class="asked" >Q: In the MetagenomeSeq analysis, q<=0.05 of the family (genus, order or species) is null? Did not do it or did not meet the conditions? If MetagenomeSeq analysis p<=0.05, several biomarkers are selected, but q<=0.05 does not, what does it mean?</p>
<p class="questions" >Biomarker screening method: Perform a difference analysis on the species abundance data between groups, using | logFC |>1 and P value<0.05 as the threshold for significant difference screening, in order to identify microbial species with significant differences between the two groups of samples, obtain the p-value, and correct the p-value to obtain the corrected q value. Finally, select species with significant differences based on the p-value or q value. For the case that q<=0.05 is empty, we did not screen the eligible species according to the conditions.</p>
<p class="questions" >In addition, since P value is the p value of hypothesis test, Q value is the q value corrected by p value. Strictly speaking, the screening condition of q value is more strict than that of p value, but if there are fewer species screened by q value, the analysis can be conducted according to p value.</p>
<p class="asked" >In the statistical analysis of species difference between groups, tow different statistical methods, T-test and LEfSe, were used, with different results.</p>
<p class="questions" >As the tow statistical analysis methods use different statistical test methods, it is normal if the results are different. Among them, T-test uses t-test, while LEfSe uses rank sum test and linear discriminant analysis (LDA), so the screening results of the three statistical analysis methods are reliable, and you can choose the most consistent analysis results according to your own research background.</p>
<p class="asked" >A summary of the problems involved in t-test analysis</p>
<p class="asked" >Q: The t-test is required to meet the normal distribution, but we did not test whether the data meet the normal distribution in the process.</p>
<p class="questions" >The t-test is divided into independent sample t-test (unmatched) and dependent sample t-test (paired), and the results of independent sample t-test are given by default in our process. The analysis result is to use the statistical method of t test to find out which two groups have significant differences in the variables of interest (mean value). Of course, we can verify the normality of the data before doing the t-test. If it conforms to the normal distribution, these verifications in turn enhance our confidence in the results obtained (refer to the book, R in Action Chapter 7 Basic Statistical Analysis and Chapter 9 Analysis of Variance).</p>
<p class="asked" >Q: Tukey and Kruskal-Wallis, the difference between the two analysis methods. Which is more referential in the case of multiple groups ?</p>
<p class="questions" >These two kinds of analysis are both two comparison methods, which can give significance test. Tukey is a parameter test, and the data should conform to normal distribution. Wilcox is a non parametric test, which does not require whether the data conform to the normal distribution, so the latter is more general. The result file should be the data obtained based on these two tests. The subsequent use method for the two groups of data is: if there are only two groups, T-test and wilcox test are selected. If there are more than two groups, Tukey test and the kruskal test of agricolae package are selected.</p>
<p class="asked" >Q: The difference of *psig.xls and *qsig.xls ？</p>
<p class="questions" >One is the result of significant p value, and the other is the result of significant q value by correcting p value.</p>
<p class="asked" >Q: Is the result of the picture in the t-test based on the p value or the q value ?</p>
<p class="questions" >When mapping according to the significant p value, the results that the group mean abundance is less than 0.001 (the default value is 0.001) are filtered out. Therefore, the results shown in the figure may be less than those in psig.xls</p>
<p class="asked" >Q: How to get q value by p value correction?</p>
<p class="questions" >The method is: Benjamini and Hochberg False Discovery Rate. The correction process of this method for p value is as follows:</p>
<p class="questions" >Here is how it works:</p>
<p class="questions" >1) The p-values of each gene are ranked from the smallest to the largest.</p>
<p class="questions" >2) The largest p-value remains as it is.</p>
<p class="questions" >3) The second largest p-value is multiplied by the total number of genes in gene</p>
<p class="questions" >list divided by its rank. If less than 0.05, it is significant.</p>
<p class="questions" >Corrected p-value = p-value*(n/n-1) < 0.05, if so, gene is significant.</p>
<p class="questions" >4) The third p-value is multiplied as in step 3:</p>
<p class="questions" >Corrected p-value = p-value*(n/n-2) < 0.05, if so, gene is significant.</p>
<p class="questions" >And so on.</p>
<p class="asked" >Q: In the result chart on the right, the explanation of the confidence interval.</p>
<p class="questions" >The color of the center circle is the same as that of the group with large mean value. The left side of the vertical dotted line indicates that the mean difference is negative, and the right side of the vertical dotted line indicates that the mean difference is positive. The analysis on the right part of the whole graph is the interval estimation of the difference between the mean values of the two groups of data. The default significance level is α= 0.05, a horizontal dotted line represents that the probability of the mean difference falling within the range of this interval is 95%. The greater the interval change, the greater the possibility of the mean difference fluctuation, and to some extent, the poor repeatability of the data within the group.</p>
<p class="asked" >Q: Why does T test show that species have differences between groups (p<0.05), but it is not shown?</p>
<p class="questions" >Because T test mapping needs to meet two conditions: p_value is less than 0.05, and the average abundance of species among groups is greater than 0.001. If the p value of two species in the sample is less than 0.05, but their abundance in each group is less than 0.001, there is no figure for this part.</p>

    <p class="paragraph"></details></p>

        <br />

        <br />

<h3>4.6 Function Prediction</h3>
        
    <a name="T4"></a>
    <h4>4.6.1 PICRUSt2</h4>    

    <p class="paragraph">PICRUSt2 is a bioinformatics software package for metagenomic function prediction based on marker gene (such as 16S rRNA). For detailed prediction process, please refer to the description on the webpage (see details: <a href="https://github.com/picrust/picrust2/wiki">https://github.com/picrust/picrust2/wik</a> ). At present, function prediction can be carried out according to 16S sequencing data based on KEGG, COG, PFAM, TIGRFAM, MetaCyc databases.</p>

    <a name="T4"></a>
    <h4>4.6.2 PICRUSt2 Clustering of Function Abundance</h4>    

    <p class="paragraph">Based on the annotation results of the database, the top 10 functional information of each sample or subgroup in terms of maximum abundance at each annotation level are selected to generate a functional relative abundance bar stack chart, so as to visually view the functions with higher relative abundance and their proportions at different annotation levels for each sample. The KO database relative abundance bar chart is shown as an example as follows:</p>

<p class="center">
<img src="src/pictures/picrust2_bar.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.24 Bar plot of PICRUSt2 Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust2_bar.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: Y-axis represents "Relative Abundance" and X-axis represents "Samples Name". "Others" represents a total relative abundance of the rest phyla besides the top 10 phyla.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/05.FunctionPrediction/picrust2/barplot</p>

    <a name="T4"></a>
    <h4>4.6.3 PICRUSt2 Relative abundance of function annotation</h4>    

    <p class="paragraph">According to the function annotation and abundance information of samples for different databases, the top 35 abundance of functions and their abundance information in each sample are selected to draw the heatmap, and clustered at different functional levels. Here, only the prediction results of KO database are shown:</p>

<p class="center">
<img src="src/pictures/picrust2.KO.heatmap.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.25 Heatmap of PICRUSt2 Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust2.KO.heatmap.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: X-axis represents the sample name and the Y-axis represents the function annotation. The cluster tree on the left side of the figure is the function cluster tree; the corresponding value of the heatmap is the Z value of function relative abundance after standardization, that is, the Z value of a sample in a certain function is the differences between the relative abundance of the sample in the function and the average relative abundance of all samples in the function divided by the standard deviation of all samples in the function.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Cluster map of function abundance for different databases: result/05.FunctionPrediction/picrust2/heatmap</p>

    <a name="T4"></a>
    <h4>4.6.4 Venn/Flower Diagram of Function Annotation</h4>    

    <p class="paragraph">In order to investigate the distribution of function number among samples (groups), the common and unique functions among different samples (groups) were analyzed and the Venn/Flower diagram was drawn. Here, only the prediction results of KO database are shown:</p>

<p class="center">
<img src="src/pictures/picrust2_venn_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.26 Venn Diagram of PICRUSt2 Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust2_venn_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: each circle in the figure represents a sample (group), the number in the overlap of circles represents the number of common functions between samples (groups), and the number without overlapping parts represents the number of unique functions of the sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis results: result/08.FunctionPrediction/picrust2/venn/*/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/picrust2/venn/*/*/*/*.txt</p>

<p class="center">
<img src="src/pictures/picrust2_flower_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.27 Flower Diagram of PICRUSt2 Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust2_flower_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: In the flower, each petal represents one (group) sample, different colors represent different samples (groups), the middle core number represents the number of feature sequences shared by all samples, and the number on the petal represents the number of feature sequences unique to this sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis results: result/08.FunctionPrediction/picrust2/flower/*/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/picrust2/flower/*/*/*/*.txt</p>

    <a name="T4"></a>
    <h4>4.6.5 T-test of Function Annotation</h4>    

    <p class="paragraph">T-test was used to test the functional differences between different groups. Here, only the prediction results of KO database are shown:</p>

<p class="center">
<img src="src/pictures/picrust2_t_test1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.28 T-test Analysis of Functional Differences between Groups</strong></p>  
</p>
<p class="premark" >Note: the left figure shows the functional abundance differences between groups, and each bar in the figure represents the average abundance of the function in different groups; the right figure shows the confidence of inter-groups differences. The leftmost point of each circle in the figure represents the lower limit of 95% confidence interval, and the rightmost point of the circle represents the upper limit of 95% confidence interval. The center of the circle represents the differences of the mean value, and the color of the circle represents the P value of differences significance test between groups of the corresponding functions.</p>

    <p class="paragraph">The volcano diagram is as follows:</p>

<p class="center">
<img src="src/pictures/picrust2_t_test1_volcano.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.29 Volcano Map</strong></p>  
</p>
<p class="premark" >Note: volcano map is a form of scatter diagram, which is usually composed of several parts: significantly up regulating function and significantly down regulating function. Generally, the X-axis is the multiple differences of the functions with differences between groups, while the Y-axis is the P-value of significance test. Each point in the figure represents a function with differences, where 'Up' represents that the abundance of the function with differences in the first comparison group is higher than that in the second, while 'Down' represents the opposite.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis results: result/08.FunctionPrediction/picrust2/ttest/*/*/*/*/*.(xls,pdf,png)</p>

    <p class="paragraph">Species with significant differences: result/08.FunctionPrediction/picrust2/ttest/*/*/*/*/boxplot</p>
<p class="premark" >Notes: boxplot is based on p<0.05 was used to screen the functions with significant differences, and top50 was selected for drawing according to the order of abundance.</p>

    <a name="T4"></a>
    <h4>4.6.6 Tax4Fun Function Prediction</h4>    

    <p class="paragraph">Tax4Fun is an R program package based on the 16S Silva database for functional prediction of intestinal, soil and other environmental samples. The prediction accuracy is high, and the experimental results show that it is better than PICRUSt function prediction, especially suitable for soil and other complex environmental samples. Tax4Fun Function prediction is realized by the nearest neighbor method based on the minimum 16S rRNA sequence similarity. The specific method is to extract the whole genome 16S rRNA gene sequence of prokaryotes in KEGG database and use BLASTN algorithm to compare it to SILVA SSU Ref NR database (BLAST bitscore>1500) to establish a correlation matrix. The whole genome functional information of KEGG database annotated by UProC and PAUDA methods is mapped to SILVA database to realize functional annotation of SILVA database. Sequencing samples use SILVA database sequence as reference sequence to cluster OTUs, and then obtain functional annotation information.</p>

    <a name="T4"></a>
    <h4>4.6.7 Clustering of Function Abundance</h4>    

    <p class="paragraph">Based on the annotation results of the database, the top 10 functional information of each sample or subgroup in terms of maximum abundance at each annotation level are selected to generate a functional relative abundance bar stack chart, so as to visually view the functions with higher relative abundance and their proportions at different annotation levels for each sample. The relative abundance bar chart of Level1 is shown as an example as follows:</p>

<p class="center">
<img src="src/pictures/tax4fun_bar.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.30 Bar plot of Tax4Fun Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/tax4fun_bar.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: Y-axis represents "Relative Abundance" and X-axis represents "Samples Name". "Others" represents a total relative abundance of the rest phyla besides the top 10 phyla.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/barplot</p>

    <a name="T4"></a>
    <h4>4.6.8 Tax4Fun Relative abundance of function annotation</h4>    

    <p class="paragraph">According to the function annotation and abundance information of samples for different databases, the top 35 abundance of functions and their abundance information in each sample are selected to draw the heatmap, and clustered at different functional levels. Here, only the prediction results of Level1 database are shown:</p>

<p class="center">
<img src="src/pictures/tax4fun_heatmap.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.31 Heatmap of Tax4Fun Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/tax4fun_heatmap.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: X-axis represents the sample name and the Y-axis represents the function annotation. The cluster tree on the left side of the figure is the function cluster tree; the corresponding value of the heatmap is the Z value of function relative abundance after standardization, that is, the Z value of a sample in a certain function is the differences between the relative abundance of the sample in the function and the average relative abundance of all samples in the function divided by the standard deviation of all samples in the function.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/heatmap</p>

    <a name="T4"></a>
    <h4>4.6.9 PCA Analysis of Tax4Fun Function Annotation</h4>    

    <p class="paragraph">Principal component analysis (PCA) dimensionality reduction analysis was conducted based on the abundance statistical results of functional annotation for the database. The more similar the composition of function among the samples are, the closer the distance of their corresponding data points on the PCA graph are.</p>

<p class="center">
<img src="src/pictures/tax4fun_PCA1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.32 PCA Result of Tax4Fun Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/tax4fun_PCA1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: The X-axis represents the first principal component, and the percentage represents the contribution value of the first principal component to the function differences; the Y-axis represents the second principal component, and the percentage represents the contribution value of the second principal component to the function differences; each point in the graph represents a sample, and the samples in the same group are represented by the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/pca/*/*/*.(png,pdf)</p>

    <a name="T4"></a>
    <h4>4.6.10 Venn/Flower Diagram of Tax4Fun Function Annotation</h4>    

    <p class="paragraph">In order to investigate the distribution of function number among samples (groups), the common and unique functions among different samples (groups) were analyzed and the Venn/Flower diagram was drawn. The results are as follow:</p>

<p class="center">
<img src="src/pictures/tax4fun_venn_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.33 Venn Diagram of Tax4Fun Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/tax4fun_venn_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: each circle in the figure represents a sample (group), the number in the overlap of circles represents the number of common functions between samples (groups), and the number without overlapping parts represents the number of unique functions of the sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/venn/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/tax4fun/venn/*/*/*.txt</p>

<p class="center">
<img src="src/pictures/tax4fun_flower_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.34 Flower Diagram of Tax4Fun Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/tax4fun_flower_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: In the flower, each petal represents one (group) sample, different colors represent different samples (groups), the middle core number represents the number of feature sequences shared by all samples, and the number on the petal represents the number of feature sequences unique to this sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/flower/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/tax4fun/flower/*/*/*.txt</p>

    <a name="T4"></a>
    <h4>4.6.11 T-test of Tax4Fun Function Annotation</h4>    

    <p class="paragraph">T-test was used to test the functional differences between different groups. Here, only the prediction results of level3 database are shown:</p>

<p class="center">
<img src="src/pictures/tax4fun_t_test1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.35 T-test Analysis of Functional Differences between Groups</strong></p>  
</p>
<p class="premark" >Notes: The left panel is the abundance of species showing significant differences between groups. Each bar represents the mean value of the abundance in each group of the specie showing significant differences between groups. The right panel is the confidential interval of between-group variations. The left-most part of each circle stands for the lower limit of 95% confidential interval, while the right-most part is the upper limit. The center of the circle stands for the difference in the mean value. The color of the circle is in agreement with the group whose mean value is higher. The right-most value is the p-value of the significance test of between-group variation.</p>

    <p class="paragraph">The volcano diagram is as follows:</p>

<p class="center">
<img src="src/pictures/tax4fun_t_test1_volcano.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.36 Volcano Map</strong></p>  
</p>
<p class="premark" >Note: volcano map is a form of scatter diagram, which is usually composed of several parts: significantly up regulating function and significantly down regulating function. Generally, the X-axis is the multiple differences of the functions with differences between groups, while the Y-axis is the P-value of significance test. Each point in the figure represents a function with differences, where 'Up' represents that the abundance of the function with differences in the first comparison group is higher than that in the second, while 'Down' represents the opposite.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/tax4fun/ttest/*/*/*/*/.(xls,pdf,png)</p>

    <p class="paragraph">Function with significant differences: result/08.FunctionPrediction/tax4fun/ttest/*/*/*/*/boxplot</p>
<p class="premark" >Notes: boxplot is based on p<0.05 was used to screen the functions with significant differences, and top50 was selected for drawing according to the order of abundance.</p>

    <a name="T4"></a>
    <h4>4.6.12 PICRUSt Function Prediction</h4>    

    <p class="paragraph">PICRUSt is a bioinformatics software package for predicting metagenomic functions based on Marker genes (such as 16S rRNA). At present, function prediction based on KEGG database can be conducted according to 16S sequencing data.</p>

    <a name="T4"></a>
    <h4>4.6.13 Clustering of PICRUSt Function Abundance</h4>    

    <p class="paragraph">Based on the annotation results of the database, the top 10 functional information of each sample or subgroup in terms of maximum abundance at each annotation level are selected to generate a functional relative abundance bar stack chart, so as to visually view the functions with higher relative abundance and their proportions at different annotation levels for each sample. The relative abundance bar chart of Level1 is shown as an example as follows:</p>

<p class="center">
<img src="src/pictures/picrust_bar.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.37 Bar plot of PICRUSt Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust_bar.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: Y-axis represents "Relative Abundance" and X-axis represents "Samples Name". "Others" represents a total relative abundance of the rest phyla besides the top 10 phyla.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/barplot</p>

    <a name="T4"></a>
    <h4>4.6.14 PICRUSt Relative abundance of function annotation</h4>    

    <p class="paragraph">According to the function annotation and abundance information of samples for different databases, the top 35 abundance of functions and their abundance information in each sample are selected to draw the heatmap, and clustered at different functional levels. Here, only the prediction results of Level1 database are shown:</p>

<p class="center">
<img src="src/pictures/picrust_heatmap.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.38 Heatmap of PICRUSt Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust_heatmap.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: X-axis represents the sample name and the Y-axis represents the function annotation. The cluster tree on the left side of the figure is the function cluster tree; the corresponding value of the heatmap is the Z value of function relative abundance after standardization, that is, the Z value of a sample in a certain function is the differences between the relative abundance of the sample in the function and the average relative abundance of all samples in the function divided by the standard deviation of all samples in the function.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/heatmap</p>

    <a name="T4"></a>
    <h4>4.6.15 PCA Analysis of PICRUSt Function Annotation</h4>    

    <p class="paragraph">Principal component analysis (PCA) dimensionality reduction analysis was conducted based on the abundance statistical results of functional annotation for the database. If the functional composition of the samples is more similar, the distance between them in the dimension reduction diagram is closer:</p>

<p class="center">
<img src="src/pictures/picrust_PCA1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.39 PCA Result of PICRUSt Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust_PCA1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: The X-axis represents the first principal component, and the percentage represents the contribution value of the first principal component to the function differences; the Y-axis represents the second principal component, and the percentage represents the contribution value of the second principal component to the function differences; each point in the graph represents a sample, and the samples in the same group are represented by the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/pca/*/*/*.(png,pdf)</p>

    <a name="T4"></a>
    <h4>4.6.16 Venn/Flower Diagram of PICRUSt Function Annotation</h4>    

    <p class="paragraph">In order to investigate the distribution of function number among samples (groups), the common and unique functions among different samples (groups) were analyzed and the Venn/Flower diagram was drawn. The results are as follow:</p>

<p class="center">
<img src="src/pictures/picrust_venn_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.40 Venn Diagram of PICRUSt Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust_venn_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: each circle in the figure represents a sample (group), the number in the overlap of circles represents the number of common functions between samples (groups), and the number without overlapping parts represents the number of unique functions of the sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/venn/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/picrust/venn/*/*/*.txt</p>

<p class="center">
<img src="src/pictures/picrust_flower_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.41 Flower Diagram of PICRUSt Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/picrust_flower_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: In the flower, each petal represents one (group) sample, different colors represent different samples (groups), the middle core number represents the number of feature sequences shared by all samples, and the number on the petal represents the number of feature sequences unique to this sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/flower/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/picrust/flower/*/*/*.txt</p>

    <a name="T4"></a>
    <h4>4.6.17 T-test of PICRUSt Function Annotation</h4>    

    <p class="paragraph">T-test was used to test the functional differences between different groups. Here, only the prediction results of Level3 database are shown:</p>

<p class="center">
<img src="src/pictures/picrust_t_test1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.42 T-test Analysis of Functional Differences between Groups</strong></p>  
</p>
<p class="premark" >Notes: The left panel is the abundance of species showing significant differences between groups. Each bar represents the mean value of the abundance in each group of the specie showing significant differences between groups. The right panel is the confidential interval of between-group variations. The left-most part of each circle stands for the lower limit of 95% confidential interval, while the right-most part is the upper limit. The center of the circle stands for the difference in the mean value. The color of the circle is in agreement with the group whose mean value is higher. The right-most value is the p-value of the significance test of between-group variation.</p>

    <p class="paragraph">The volcano diagram is as follows:</p>

<p class="center">
<img src="src/pictures/picrust_t_test1_volcano.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.43 Volcano Map</strong></p>  
</p>
<p class="premark" >Note: volcano map is a form of scatter diagram, which is usually composed of several parts: significantly up regulating function and significantly down regulating function. Generally, the X-axis is the multiple differences of the functions with differences between groups, while the Y-axis is the P-value of significance test. Each point in the figure represents a function with differences, where 'Up' represents that the abundance of the function with differences in the first comparison group is higher than that in the second, while 'Down' represents the opposite.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/picrust/ttest/*/*/*.(xls,pdf,png)</p>

    <p class="paragraph">Function with significant differences: result/08.FunctionPrediction/picrust/ttest/*/*/*/boxplot</p>
<p class="premark" >Notes: boxplot is based on p<0.05 was used to screen the functions with significant differences, and top50 was selected for drawing according to the order of abundance.</p>

    <a name="T4"></a>
    <h4>4.6.18 BugBase Function Prediction</h4>    

    <p class="paragraph">BugBase is a widely used tool for predicting high level phenotypes of 16S microorganisms since 2016. In addition to phenotypic prediction, this tool can also perform inter group difference analysis for different phenotypes, and display statistical charts. At present, microbial communities can be classified according to seven types of phenotypes: Gram Positive, Gram Negative, Biofilm Forming, Pathogenic, Mobile Element Containing, Oxygen Utilization, including Aerobic, Anaerobic, and Cultivably Yanaerobic, and Oxidative Stress Tolerance.</p>

    <a name="T4"></a>
    <h4>4.6.19 Phenotypic abundance display for BugBase</h4>    

    <p class="paragraph">According to the phenotype prediction results, obtain the phenotype prediction results of each taxon. When drawing a stacked column chart of a phenotype, select the taxon whose phenotype prediction result is "positive", and obtain the species annotation results of the selected taxon. Set an abundance value (find out the bacteria phyla with the highest abundance in all samples or groups and divide its abundance by 10). Other gates with abundance less than this value are classified as "Other". Finally, the abundance column between samples and groups was drawn.</p>

<p class="center">
<img src="src/pictures/bugbase.bar.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.44 Bar plot of Aerobic</strong></p>  
</p>

    <p class="paragraph">Notes: The abscissa (Sample Name) is the sample name. The ordinate (Relative Abundance) indicates the relative abundance.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/bugbase/BugBase_result/otu_contributions/; Including (Aerobic、Anaerobic、Contains_Mobile_Elements、Facultatively_Anaerobic、Forms_Biofilms、Gram_Negative、Gram_Positive、Potentially_Pathogenic、Stress_Tolerant) results of 9 phenotypes.</p>

    <a name="T4"></a>
    <h4>4.6.20 Analysis of phenotypic abundance for BugBase</h4>    

    <p class="paragraph">In the analysis of phenotypic abundance difference between groups, the box chart can directly reflect the median, dispersion, maximum, minimum and abnormal values of species diversity within the group (please refer to the box chart for the interpretation of the box chart). In addition to the inter group comparison box chart of each phenotype, the predicted_phenotypes folder also contains the inter group difference test statistical results of each phenotype (* _stats. txt). The difference test includes Pairwise Mann Whitney Wilcoxon Tests between any two groups, p-values corrected by FDR, and Kruskal Wallis Test results between multiple groups. Taking Aerobic and Anaerobic indexes as an example, the box chart of the difference analysis between groups is as follows:</p>

<p class="center">
<img src="src/pictures/bugbase.diff.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.45 Box Chart of Aerobic Index</strong></p>  
</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">The box chart of the differences among the nine phenotypes and the results of the difference significance test: result/08.FunctionPrediction/bugbase/BugBase_result/predicted_phenotypes:</p>

    <a name="T4"></a>
    <h4>4.6.21 FAPROTAX Function Prediction</h4>    

    <p class="paragraph">FAPROTAX software has built a functional classification database based on species information. The current version, based on reference evidence, contains more than 80 functional classifications of carbon, nitrogen, phosphorus, sulfur and other elements, animal and plant pathogens, methane generation, and fermentation, covering more than 4600 different prokaryotic species, and has a good prediction effect on the biochemical cycle process of environmental samples.</p>

    <a name="T4"></a>
    <h4>4.6.22 Clustering of FAPROTAX Function Abundance</h4>    

    <p class="paragraph">According to the database annotation results, the abundance information of each sample is counted, and the results are shown as follows:</p>

<p class="center">
<img src="src/pictures/faprotax_bar.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.46 Bar plot of FAPROTAX Function Annotation</strong></p>  
</p>
<p class="premark" >Notes: Y-axis represents "Relative Abundance" and X-axis represents "Samples Name". "Others" represents a total relative abundance of the rest phyla besides the top 10 phyla.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/faprotax/barplot</p>

    <p class="paragraph">According to the function annotation and abundance information of samples for different databases, the top 35 abundance of functions and their abundance information in each sample are selected to draw the heatmap, and clustered at different functional levels.</p>

<p class="center">
<img src="src/pictures/faprotax.anno.heatmap.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.47 Heatmap of FAPROTAX Function Annotation</strong></p>  
</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Heatmap of function abundance: result/08.FunctionPrediction/faprotax/heatmap</p>

    <a name="T4"></a>
    <h4>4.6.24 PCA Analysis of FAPROTAX Function Annotation</h4>    

    <p class="paragraph">Principal component analysis (PCA) dimensionality reduction analysis was conducted based on the abundance statistical results of functional annotation for the database</p>

<p class="center">
<img src="src/pictures/faprotax_PCA1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.48 PCA Result of FAPROTAX Function Annotation</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/faprotax_PCA1.pdf" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: The X-axis represents the first principal component, and the percentage represents the contribution value of the first principal component to the function differences; the Y-axis represents the second principal component, and the percentage represents the contribution value of the second principal component to the function differences; each point in the graph represents a sample, and the samples in the same group are represented by the same color.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Results directory: result/08.FunctionPrediction/faprotax/pca/*/*/*.(png,pdf)</p>

    <a name="T4"></a>
    <h4>4.6.25 Venn/Flower Diagram of FAPROTAX Function Annotation</h4>    

    <p class="paragraph">In order to investigate the distribution of function number among samples (groups), the common and unique functions among different samples (groups) were analyzed and the Venn/Flower diagram was drawn. The results are as follow:</p>

<p class="center">
<img src="src/pictures/faprotax_venn_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.49 Venn Diagram of FAPROTAX Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/faprotax_venn_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Note: each circle in the figure represents a sample (group), the number in the overlap of circles represents the number of common functions between samples (groups), and the number without overlapping parts represents the number of unique functions of the sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/faprotax/venn/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/faprotax/venn/*/*/*.txt</p>

<p class="center">
<img src="src/pictures/faprotax_flower_display1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.50 Flower Diagram of FAPROTAX Results</strong></p>  
</p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/pictures/faprotax_flower_display1.svg" target="_blank"><b>To view full size picture please click here</b></a></p>
        
<p class="premark" >Notes: In the flower, each petal represents one (group) sample, different colors represent different samples (groups), the middle core number represents the number of feature sequences shared by all samples, and the number on the petal represents the number of feature sequences unique to this sample (group).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/faprotax/flower/*/*/*.(png,svg)</p>

    <p class="paragraph">Plotting data: result/08.FunctionPrediction/faprotax/flower/*/*/*.txt</p>

    <a name="T4"></a>
    <h4>4.6.26 T-test of FAPROTAX Function Annotation</h4>    

    <p class="paragraph">T-test was used to test the functional differences between different groups. Here, only the prediction results of one are shown:</p>

<p class="center">
<img src="src/pictures/faprotax_t_test1.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.51 T-test Analysis of Functional Differences between Groups</strong></p>  
</p>
<p class="premark" >Notes: The left panel is the abundance of species showing significant differences between groups. Each bar represents the mean value of the abundance in each group of the specie showing significant differences between groups. The right panel is the confidential interval of between-group variations. The left-most part of each circle stands for the lower limit of 95% confidential interval, while the right-most part is the upper limit. The center of the circle stands for the difference in the mean value. The color of the circle is in agreement with the group whose mean value is higher. The right-most value is the p-value of the significance test of between-group variation.</p>

    <p class="paragraph">The volcano diagram is as follows:</p>

<p class="center">
<img src="src/pictures/faprotax_t_test1_volcano.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.52 Volcano Map</strong></p>  
</p>
<p class="premark" >Note: volcano map is a form of scatter diagram, which is usually composed of several parts: significantly up regulating function and significantly down regulating function. Generally, the X-axis is the multiple differences of the functions with differences between groups, while the Y-axis is the P-value of significance test. Each point in the figure represents a function with differences, where 'Up' represents that the abundance of the function with differences in the first comparison group is higher than that in the second, while 'Down' represents the opposite.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/08.FunctionPrediction/faprotax/ttest/*/*/*/*/*.(xls,pdf,png)</p>

    <p class="paragraph">Function with significant differences: result/08.FunctionPrediction/faprotax/ttest/*/*/*/*/boxplot</p>
<p class="premark" >Notes: boxplot is based on p<0.05 was used to screen the functions with significant differences, and top50 was selected for drawing according to the order of abundance.</p>

        <br />

        <br />

<h3>4.7 Correlation analysis</h3>
        
    <a name="T4"></a>
    <h4>4.7.1 Network Analysis</h4>    

    <p class="paragraph">The network diagram provides a new perspective for studying of environmental community structure and function of complex microbial environment. The symbiotic relationship of microorganisms varied in different environment. The species network diagram can be used to see the impact of different environmental factors on microbial adaptability, as well as the dominant species and the closely interacting species group in a certain environment. These dominant species and species groups often paly a unique and important role in maintaining the structure and functional stability of the microbial community in the environment (Jiao et al., 2017; Qin et al., 2012).</p>

    <p class="paragraph">Correlation coefficient of all samples were calculated by methods of Spearman Correlation Coefficient (SCC) or Pearson Correlation Coefficient (PCC). After filtering (cutoff＝0.6), the table of correlation coefficient was used to generate Network diagram together with species abundance.</p>

<p class="center">
<img src="src/pictures/Network.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.53 Network diagram</strong></p>  
</p>
<p class="premark" >Note: the node represents the genus, the node size represents the average relative abundance of the genus, the node color of the same phylum is same, the thickness of the line between nodes is positively correlated with the absolute value of the correlation coefficient of the species interaction, and the line color is positively correlated with the positive and negative correlation respectively (red represents positive correlation, blue represents negative correlation).</p>

    <p class="paragraph">Results directory: result/07.CorrelationAnalysis/network/</p>

    <p class="paragraph"><details><summary> Show Q&A </summary></p>
<p class="asked" >Q：Network Drawing process and software</p>
<p class="questions" >Use our script to process and calculate the species correlation, then use dot text language to describe it, and finally use graphviz-2.38.0 software to process it for visual display.</p>
<p class="asked" >Q：There are more than 100 species listed in the list file of the network diagram, but only dozens of species are shown in the picture. Why?</p>
<p class="questions" >The species listed in the table have a correlation coefficient of more than 0.6. The mapping process will filter out those edge, node and species with an average relative abundance of less than 0.005%. The Spearman correlation coefficient used in the network diagram is also filtered with P value, and the filtering condition is p<0.05.</p>

    <p class="paragraph"></details></p>

    <a name="T4"></a>
    <h4>4.7.2 Network3D</h4>    

    <p class="paragraph">Dynamic network diagram can more clearly show the relationship between a certain microbial genus and other microorganisms.</p>

    <p class="paragraph">Spearman correlation coefficient was calculated for Top 100 microbial genera. Then select the connections whose correlation coefficient is greater than 0.6 or less than - 0.6 and significant (p-value is less than 0.05) as effective connections to draw the network diagram. The results of the web version are displayed in the results:</p>

<p class="center">
<img src="src/pictures/Network3D.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.54 Dynamic network diagram display</strong></p>  
</p>
<p class="premark" >Note: This display image is for reference only, and the actual results can be found in the results file. Different nodes represent different genera, node size represents the degree of connectivity of the genus, and the same color represents the same door level (as shown in the figure). The thickness of the connection between nodes is positively correlated with the absolute value of the correlation coefficient of species interaction.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Web display: result/07.CorrelationAnalysis/network3D/sample/netfig/net.html</p>

    <p class="paragraph">Draw network diagram data:</p>

    <p class="paragraph">Nodes file：result/07.CorrelationAnalysis/network3D/sample/node.xls</p>

    <p class="paragraph">Wire file: result/07.CorrelationAnalysis/network3D/sample/edge.xls</p>

        <br />

        <br />

<h3>4.8 Environmental association analysis</h3>
        
    <p class="paragraph">Correlation analysis is a method of statistical evaluation used to study the strength and the direction of a relationship between two, numerically measured, continuous variables.</p>

    <p class="paragraph">For studying the relationship between environmental factors and species or species abundance (alpha diversity), Spearman rank correlation and Mantel test analysis are used to test the association between two ranked variables, or one ranked variable and one measurement variable, and calculate the P-value (significance).</p>

    <a name="T4"></a>
    <h4>4.8.1 Spearman correlation test</h4>    

    <p class="paragraph">Spearman correlation test is usually denoted by rho (Algina et al.,1999). Spearman’s rank correlation coefficient serves as a nonparametric measure of rank linear correlation (the statistical dependence between the ranks of two variables), regardless of the distribution of the variables. Spearman rank correlation has been applied in a wide range.</p>

    <p class="paragraph">Spearman rank correlation test was used to test the correlation between environmental factors and species or species abundance (alpha diversity), to obtain the correlation and significant P-value of the two ranking variables. The result shows below:</p>

<p class="center">
<img src="src/pictures/correlation.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.55 Spearman correlation heatmap</strong></p>  
</p>
<p class="premark" >Notes: </p>
<p class="premark" >Rows: environmental factors; Columns: species; Colored tiles corresponding the coefficient of Spearman rank correlation r value, whose range is from -1 to 1. r<0 negative correlation; r>0 positive correlation; ＊ means significant (p<0.05).</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">The result of Spearman correlation based on Alpha diversity table, Relative abundance tables in each taxonomic rank can be found: result/09.EnvironmentFactor/spearman/*/correlation.png</p>

    <a name="T4"></a>
    <h4>4.8.2 Canonical Correspondence Analysis (CCA)/ Redundancy Analysis (RDA)</h4>    

    <p class="paragraph">CCA/RDA (Sheik et al., 2012)mainly focuses on inter-sample relationships or inter-species correlations, the interpretation of the relationships between samples, species, and environmental variables differs to figure out the major driving environmental forces.</p>

    <p class="paragraph">CCA/RDA can be considered as the ordination approach based on correspondence analysis, in which the correspondence analysis is combined with multivariate regression analysis. They calculated a regression of the ordination axes on the measured environmental variables, also known as multiple direct gradient analysis. CCA is based on unimodal species–environment relationships, while RDA is based on linear models. The results show below:</p>

<p class="center">
<img src="src/pictures/CCA.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.56 CCA</strong></p>  
</p>
<p class="premark" >Notes: in CCA plot, arrows indicate environmental factors. The length of arrows shows the association between community distribution and species distribution. The longer the arrows, the greater the association, and vice versa. The angle between arrow and ordination axes responds to the association between an environmental factor and ordination axes. The smaller the angle, the greater the association, and vice versa. Relationship of two environmental factors is positive when their included angle is acute, and negative correlation when obtuse angle.</p>

    <p class="paragraph">The significance of each environmental factor is tested by envfit function. CCAenvfit table shows the significance analysis results of environmental factors. The results are shown as follows:</p>

<p class="name_table"><p class="center"><strong>Table 4.6 CCAenvfit Analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable7" >
        <thead><tr>
<a id="btn07"></a>
<a id="sjzl7"></a>
<a  href="javascript:void(0)" id="btn17"> | First | </a>
<a  href="javascript:void(0)" id="btn27"> Previous | </a>
<a  href="javascript:void(0)" id="btn37"> Next | </a>
<a  href="javascript:void(0)" id="btn47"> Last | </a>
<a>Go to </a>
<input id="changePage7" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn57"> Jump </a>
                
<th style="text-align :center">""</th><th style="text-align :center">"CCA1"</th><th style="text-align :center">"CCA2"</th><th style="text-align :center">"r2"</th><th style="text-align :center">"Pr(>r)"
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">"age"</td><td style="text-align :center">0.898558027844298</td><td style="text-align :center">0.438854726072952</td><td style="text-align :center">0.471552602079523</td><td style="text-align :center">0.000999500249875062
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">"bmi"</td><td style="text-align :center">0.974076742788525</td><td style="text-align :center">-0.22621781353045</td><td style="text-align :center">0.392647314267272</td><td style="text-align :center">0.000499750124937531
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">"VSS"</td><td style="text-align :center">-0.752412606271381</td><td style="text-align :center">0.658692090376002</td><td style="text-align :center">0.43920225252603</td><td style="text-align :center">0.000499750124937531
</td></tr></tbody>

</table>
</div>
</p>
<p class="premark" >Note: The value corresponding to CCA1 and CCA2 is the cosine of the angle between the environment factor arrow and the sorting axis, which represents the correlation between the environment factor and the sorting axis. The r2 is used to measure the degree of correlation between each typical variable. The larger the value of r2, the stronger the predictive ability between two sets of variables, that is, the stronger the correlation between typical variables. The Pr value is an indicator used to determine whether the typical correlation coefficient is significant.</p>

    <p class="paragraph">Numerous environmental factors have an effect on microflora. Most of them are correlation. Therefore, only major influencing factors are selected before environmental factor analysis.</p>

    <p class="paragraph">VIF (variance inflation factor) analysis (Gross et al., 2003) will calculate the VIF value of each environmental factor to be filtered. Generally, the environmental factor would be kicked out as useless factor when its VIF value is greater than 20. This process would repeat till VIF values of all factors are less than 20.</p>

    <p class="paragraph">BioENV analysis (Massol et al., 2021) calculated relevance value of different environmental factors assemblies with microflora. And then it could find out the most relevant assembly which has the greatest impact on microflora.</p>

    <p class="paragraph">Doing CCA/RDA of VIF-based and BioENV-based selected environmental factors respectively, the results can be found in Results directory.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">All environmental factors CCA/RDA results: result/09.EnvironmentFactor/CCA/CCA_standard*</p>

    <p class="paragraph">Selected environmental factors CCA/RDA results based on VIF: result/09.EnvironmentFactor/CCA/CCA_VIF/*</p>

    <p class="paragraph">Selected environmental factors CCA/RDA results based on BioENV: result/09.EnvironmentFactor/CCA/CCA_BioENV/*</p>

    <p class="paragraph">Selected environmental factors CCA/RDA results based on envfit: result/09.EnvironmentFactor/CCA/CCA_FitENV/*</p>

    <a name="T4"></a>
    <h4>4.8.3 dbRDA Analysis</h4>    

    <p class="paragraph">The dbRDA analysis (Xiao Qian LI et al., 2015) is mainly used to reflect the relationship between the distance based on flora and environmental factors. RDA analysis reflects the Euclidean distance between quadrats on the graph. The community data contains 0 values or other analysis methods based on European distance must be converted to Euclidean distance before RDA analysis can be carried out. The dbRDA analysis is a distance-based redundancy analysis, which is applicable to any distance matrix. For example, the Bray-Curtis distance is the most commonly used indicator to reflect the difference between two communities. The calculation is based on the relationship between the quantitative characteristics of different species in the sample and environmental factors. The results are shown as follows:</p>

<p class="center">
<img src="src/pictures/dbRDA.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.57 dbRDA</strong></p>  
</p>
<p class="premark" >Note: The horizontal ordinate represents the first principal component, that is, the contribution value of the first principal component to the sample difference. The ordinate represents the second principal component, that is, the contribution value of the second principal component to the sample difference. Each point in the figure represents a sample. Environmental factors are generally represented by arrows, and the length of the arrow line represents the degree of correlation between an environmental factor and community distribution and species distribution. The longer the arrows, the greater the correlation, and vice versa.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Analysis result: result/04.BetaDiversity/Environment_factor/multiCCA/CCA*/dbRDA*</p>

    <a name="T4"></a>
    <h4>4.8.4 Mantel test</h4>    

    <p class="paragraph">The Mantel test is a statistical test of the correlation between two matrices (Yang et al., 2007). The test is commonly used in ecology, where the data are usually estimates of the "distance" between objects such as species of organisms. The result is shown below:</p>

<p class="name_table"><p class="center"><strong>Table 4.7 Mantel test</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable8" >
        <thead><tr>
<a id="btn08"></a>
<a id="sjzl8"></a>
<a  href="javascript:void(0)" id="btn18"> | First | </a>
<a  href="javascript:void(0)" id="btn28"> Previous | </a>
<a  href="javascript:void(0)" id="btn38"> Next | </a>
<a  href="javascript:void(0)" id="btn48"> Last | </a>
<a>Go to </a>
<input id="changePage8" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn58"> Jump </a>
                
<th style="text-align :center">Variable</th><th style="text-align :center">r</th><th style="text-align :center">P
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">age-bmi-VSS</td><td style="text-align :center">0.005527</td><td style="text-align :center">0.453
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">age-bmi</td><td style="text-align :center">0.01099</td><td style="text-align :center">0.545
</td></tr></tbody>

</table>
</div>
</p>
<p class="premark" >Notes: variable represents the information of environmental factors. The correlation coefficient is r. P means p-value of significance test. The greater the r-value is, the greater the correlation between environmental factors and species abundance is. P<0.05 means significant.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">Mantel test result: result/09.EnvironmentFactor/mantelTest/*/mantel_test_table.xls</p>

    <a name="T4"></a>
    <h4>4.8.5 Variance Partitioning Canonical Correspondence Analysis (VPA)</h4>    

    <p class="paragraph">VPA (Variance Partitioning Canonical Correspondence Analysis) (Yang et al., 2007) divides the total variance of the community distribution into the respective contributions and covariations of each set of environmental variables.</p>

    <p class="paragraph">The commonly constrained ordinations are RDA and CCA. Constrained ordinations can only demonstrate the variance of the species distribution in response to environmental factors. When the effects of a particular environmental variable on species distribution need to be tested after elimination of possible effects due to other (environmental) variables which are specified as covariables in the constrained analysis, partial ordination may be used (pCCA, pRDA). Furthermore, partial ordination can partition the total variance of the species table into the respective contribution of each set of environmental variables and their covariations using both standard and partially constrained ordinations. The result displayed as follows:</p>

<p class="center">
<img src="src/pictures/vpa.png" width="600" height="450" />
<p class="name_fig"><strong>Figure 4.58 VPA</strong></p>  
</p>
<p class="premark" >Notes: the overlap represents mutual interpretation in terrain factors and soil factors. Outside is invalid interpretation.</p>

    <p class="paragraph">Results directory:</p>

    <p class="paragraph">result/04.BetaDiversity/Environment_factor/VPA/VPA.svg</p>

        <br />

        <br />

<hr />
</p><br />

<h2>5 Appendix</h2>


        <br />

        <br />

<h3>5.1 List of analysis software</h3>
        
<p class="name_table"><p class="center"><strong>Main softwore used in the analysis</strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable9" >
        <thead><tr>
<a id="btn09"></a>
<a id="sjzl9"></a>
<a  href="javascript:void(0)" id="btn19"> | First | </a>
<a  href="javascript:void(0)" id="btn29"> Previous | </a>
<a  href="javascript:void(0)" id="btn39"> Next | </a>
<a  href="javascript:void(0)" id="btn49"> Last | </a>
<a>Go to </a>
<input id="changePage9" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn59"> Jump </a>
                
<th style="text-align :center">Analytical content</th><th style="text-align :center">Software</th><th style="text-align :center">Version</th><th style="text-align :center">Parameters</th><th style="text-align :center">Remarks
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">Reads splited</td><td style="text-align :center">python</td><td style="text-align :center">3.6.13</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Reads filtration</td><td style="text-align :center">cutadapt</td><td style="text-align :center">3.3</td><td style="text-align :center">--minimum-length 20</td><td style="text-align :center">Remove short sequence and intercept over sequence
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Reads merging</td><td style="text-align :center">flash</td><td style="text-align :center">1.2.11</td><td style="text-align :center">--min-overlap 10  --max-mismatch-densit 0.2</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Tags flitration</td><td style="text-align :center">fastp</td><td style="text-align :center">0.23.1</td><td style="text-align :center">-q 19 -u 15</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Chimeras Removal</td><td style="text-align :center">vsearch</td><td style="text-align :center">2.16.0</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Denoise</td><td style="text-align :center">qiime2</td><td style="text-align :center">202202</td><td style="text-align :center">dada2</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">ASV tree</td><td style="text-align :center">qiime2</td><td style="text-align :center">202202</td><td style="text-align :center">phylogeny</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Taxonomy annotation</td><td style="text-align :center">qiime2</td><td style="text-align :center">202202</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Top10 barplot</td><td style="text-align :center">perl</td><td style="text-align :center">5.26.2</td><td style="text-align :center">SVG</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Heatmap</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">pheatmap</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Ternary plot</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vcd</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Venn</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">VennDiagram</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Flower</td><td style="text-align :center">perl</td><td style="text-align :center">5.26.2</td><td style="text-align :center">SVG</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Tree100</td><td style="text-align :center">perl</td><td style="text-align :center">5.26.2</td><td style="text-align :center">SVG</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Alpha diversity indices</td><td style="text-align :center">qiime2</td><td style="text-align :center">202202</td><td style="text-align :center">chao1, dominance, goods_coverage, observed_otus, pielou_e, shannon, simpson</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Specaccum</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, reshape2</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Rank_abundance</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">RColorBrewer</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Rarefaction curve/indexes</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">plyr, reshape2, ggplot2</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Alpha diversity analysis</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">ggplot2, agricolae</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">BetaDistance</td><td style="text-align :center">qiime2</td><td style="text-align :center">202202</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">PCA</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">ade4, cluster, grid, fpc, clusterSim, RColorBrewer, ggplot2, vegan, permute, lattice, stringr, grid, cowplot</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">DCA</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, permute, lattice, methods, ggplot2</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">UnifracHeatmap</td><td style="text-align :center">perl</td><td style="text-align :center">5.26.2</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">PCoA</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">ggplot2, extrafont, grid, ade4</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">NMDS</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">ade4, cluster, grid, fpc, clusterSim, RColorBrewer, ggplot2, vegan, permute, lattice, stringr, grid, cowplot</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">BetaDiv</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">ggplot2, optparse</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Adonis</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, ggplot2, tidyr, dplyr, gridExtra, permute, lattice</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Anosim</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, ggplot3, tidyr, dplyr, gridExtra, permute, lattice</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">MRPP</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, ggplot4, tidyr, dplyr, gridExtra, permute, lattice</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Simper</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">vegan, ggplot2, tidyr, dplyr, gridExtra</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">LEFse</td><td style="text-align :center">lefse</td><td style="text-align :center">1.1.01</td><td style="text-align :center">-l 4</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">T-test</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">metagenomeSeq</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">metagenomeSeq</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Picrust2</td><td style="text-align :center">picrust2</td><td style="text-align :center">2.3.0</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Picrust</td><td style="text-align :center">picrust</td><td style="text-align :center">1.1.4</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Tax4fun</td><td style="text-align :center">tax4fun</td><td style="text-align :center">0.3.1</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">FAPROTAX</td><td style="text-align :center">python</td><td style="text-align :center">2.7.15</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Bugbase</td><td style="text-align :center">R</td><td style="text-align :center">3.4.0</td><td style="text-align :center">reshape2, plyr, gridExtra, ggplot2, digest, labeling, RColorBrewer, Matrix, grid, biom, beeswarm, optparse, RJSONIO</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Funguild</td><td style="text-align :center">python</td><td style="text-align :center">2.7.15</td><td style="text-align :center">-</td><td style="text-align :center">-
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">Environmental association analysis</td><td style="text-align :center">R</td><td style="text-align :center">4.0.3</td><td style="text-align :center">psych, vegan, ggplot2, grid, ggrepel, MASS</td><td style="text-align :center">-
</td></tr></tbody>

</table>
</div>
</p>

        <br />

        <br />

<h3>5.2 Methods</h3>
        
    <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly<a href="#" name="miao">.</a></p>

        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/hyperlinks/Methods_QIIME2_EN.pdf" target="_blank"><b>File catalog: Method</b></a></p>
        

        <br />

        <br />

<h3>5.3 Result File Decompression Tool and Format Description</h3>
        
<p class="name_table"><p class="center"><strong></strong></p>        
<p class="center">
<div style="overflow-x: auto">
<table class="gy" id="mytable10" >
        <thead><tr>
<a id="btn010"></a>
<a id="sjzl10"></a>
<a  href="javascript:void(0)" id="btn110"> | First | </a>
<a  href="javascript:void(0)" id="btn210"> Previous | </a>
<a  href="javascript:void(0)" id="btn310"> Next | </a>
<a  href="javascript:void(0)" id="btn410"> Last | </a>
<a>Go to </a>
<input id="changePage10" type="text" size="1" maxlength="4"/>
<a> page </a>
<a  href="javascript:void(0)" id="btn510"> Jump </a>
                
<th style="text-align :center">Compressed format</th><th style="text-align :center">Customer type</th><th style="text-align :center">Decompression method
</th></tr></thead>
<tbody><tr>
<td style="text-align :center">*.gz format</td><td style="text-align :center">Unix/Linux/Mac</td><td style="text-align :center">Open with gzip -d *.gz command
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">*.gz format</td><td style="text-align :center">Windows</td><td style="text-align :center">Open with decompression software such as WinRAR, 7-Zip, etc
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">*.zip format</td><td style="text-align :center">Unix/Linux/Mac</td><td style="text-align :center">Open with unzip *.zip command
</td></tr></tbody>
<tbody><tr>
<td style="text-align :center">*.zip format</td><td style="text-align :center">Windows</td><td style="text-align :center">Open with decompression software such as WinRAR, 7-Zip, etc
</td></tr></tbody>

</table>
</div>
</p>

        <br />

        <br />

<h3>5.4 Delivery data result directory</h3>
        
        <p class="paragraph">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="src/hyperlinks/ReadMe_qiime2_EN.html" target="_blank"><b>ReadMe.html</b></a></p>
        

        <br />

        <br />

<hr />
</p><br />

<h2>6 References</h2>


    <p class="paragraph">Algina J, Keselman HJ. Comparing squared multiple correlation coefficients: Examination of a confidence interval and a test significance. Psychological Methods. 1999;4(1):76-83. doi:10.1037/1082-989x.4.1.76.</p>

    <p class="paragraph">Amir A, McDonald D, Navas-Molina JA, et al. Deblur Rapidly Resolves Single-Nucleotide Community Sequence Patterns.mSystems. 2017;2(2):e00191-16. Published 2017 Mar 7. doi:10.1128/mSystems.00191-16.</p>

    <p class="paragraph">Anderson MJ. A new method for non-parametric multivariate analysis of variance.Austral Ecology. 2001;26(1):32-46. doi:10.1111/j.1442-9993.2001.01070.pp.x.</p>

    <p class="paragraph">Avershina, Ekaterina, Trine Frisli, and Knut Rudi. De novo Semi-alignment of 16S rRNA Gene Sequences for Deep Phylogenetic Characterizationof Next Generation Sequencing Data. Microbes and Environments 28.2 (2013): 211-216..</p>

    <p class="paragraph">Bachy C, Dolan JR, López-García P, Deschamps P, Moreira D. Accuracy of protist diversity assessments: morphology compared with cloning and direct pyrosequencing of 18S rRNA genes and ITS regions using the conspicuous tintinnid ciliates as a case study. ISME J. 2013;7(2):244-255. doi:10.1038/ismej.2012.106.</p>

    <p class="paragraph">Backeljau T, De Bruyn L, De Wolf H, Jordaens K, Van Dongen S, Winnepennincks B. Multiple UPGMA and Neighbor-joining Trees and the Performance of Some Computer Packages. Molecular Biology and Evolution. 1996;13(2):309-313. doi:10.1093/oxfordjournals.molbev.a025590.</p>

    <p class="paragraph">Bengtsson-Palme J, Ryberg M, Hartmann M, et al. Improved software detection and extraction of ITS1 and ITS2 from ribosomal ITS sequences offungi and other eukaryotes for analysis of environmental sequencing data. Bunce M, ed.Methods in Ecology and Evolution. Published online July 2013:914-919. doi:10.1111/2041-210x.12073.</p>

    <p class="paragraph">Bokulich NA, Kaehler BD, Rideout JR, et al. Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2's q2-feature-classifier plugin.Microbiome. 2018;6(1):90. Published 2018 May 17. doi:10.1186/s40168-018-0470-z.</p>

    <p class="paragraph">Bolyen E, Rideout JR, Dillon MR, et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2 [publishedcorrection appears in Nat Biotechnol. 2019 Sep;37(9):1091].Nat Biotechnol. 2019;37(8):852-857. doi:10.1038/s41587-019-0209-9.</p>

    <p class="paragraph">Breiman L. Random Forests. Machine Learning. 2001;45(1):5-32. doi:10.1023/a:1010933404324.</p>

    <p class="paragraph">Bulgarelli D, Garrido-Oter R, Münch PC, et al. Structure and function of the bacterial root microbiota in wild and domesticated barley.CellHost Microbe. 2015;17(3):392-403. doi:10.1016/j.chom.2015.01.011.</p>

    <p class="paragraph">Callahan BJ, McMurdie PJ, Holmes SP. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. ISME J. 2017;11(12):2639-2643. doi:10.1038/ismej.2017.119.</p>

    <p class="paragraph">Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJ, Holmes SP. DADA2: High-resolution sample inference from Illumina amplicon data. NatMethods. 2016;13(7):581-583. doi:10.1038/nmeth.3869.</p>

    <p class="paragraph">Callahan BJ, Wong J, Heiner C, et al. High-throughput amplicon sequencing of the full-length 16S rRNA gene with single-nucleotide resolution. Nucleic Acids Research. 2019;47(18):e103-e103. doi:10.1093/nar/gkz569.</p>

    <p class="paragraph">Caporaso JG, Lauber CL, Walters WA, et al. Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. Proc Natl Acad Sci U S A. 2011;108 Suppl 1(Suppl 1):4516-4522. doi:10.1073/pnas.1000080107.</p>

    <p class="paragraph">Caporaso JG, Lauber CL, Walters WA, et al. Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq platforms. ISME J. 2012;6(8):1621-1624. doi:10.1038/ismej.2012.8.</p>

    <p class="paragraph">Chapman M, Underwood A. Ecological patterns in multivariate assemblages:information and interpretation of negative values in ANOSIM tests.Marine Ecology Progress Series. 1999;180:257-265. doi:10.3354/meps180257.</p>

    <p class="paragraph">Cheung MK, Au CH, Chu KH, Kwan HS, Wong CK. Composition and genetic diversity of picoeukaryotes in subtropical coastal waters as revealed by 454 pyrosequencing. ISME J. 2010;4(8):1053-1059. doi:10.1038/ismej.2010.26.</p>

    <p class="paragraph">Clarke K, Ainsworth M. A method of linking multivariate community structure to environmental variables.Marine Ecology Progress Series. 1993;92:205-219. doi:10.3354/meps092205.</p>

    <p class="paragraph">Degnan PH, Ochman H. Illumina-based analysis of microbial community diversity. ISME J. 2012;6(1):183-194. doi:10.1038/ismej.2011.74.</p>

    <p class="paragraph">Findley K, Oh J, Yang J, et al. Topographic diversity of fungal and bacterial communities in human skin. Nature. 2013;498(7454):367-370. doi:10.1038/nature12171.</p>

    <p class="paragraph">Groß J. The R Journal: Variance inflation factors. R News. 2003;3(1):13-15. Accessed October 18, 2022. https://journal.r-project.org/articles/RN-2003-004/.</p>

    <p class="paragraph">Hess M, Sczyrba A, Egan R, et al. Metagenomic discovery of biomass-degrading genes and genomes from cow rumen. Science. 2011;331(6016):463-467. doi:10.1126/science.1200387.</p>

    <p class="paragraph">Kruskal JB. Nonmetric multidimensional scaling: A numerical method.Psychometrika. 1964;29(2):115-129. doi:10.1007/bf02289694.</p>

    <p class="paragraph">Li B, Zhang X, Guo F, Wu W, Zhang T. Characterization of tetracycline resistant bacterial community in saline activated sludge using batch stress incubation with high-throughput sequencing analysis. Water Res. 2013;47(13):4207-4216. doi:10.1016/j.watres.2013.04.021.</p>

    <p class="paragraph">Li M, Shao D, Zhou J, et al. Signatures within esophageal microbiota with progression of esophageal squamous cell carcinoma. Chinese Journal of Cancer Research. 2020;32(6):755-767. doi:10.21147/j.issn.1000-9604.2020.06.09.</p>

    <p class="paragraph">Lozupone C, Knight R. UniFrac: a new phylogenetic method for comparing microbial communities. Appl Environ Microbiol. 2005;71(12):8228-8235. doi:10.1128/AEM.71.12.8228-8235.2005.</p>

    <p class="paragraph">Lozupone C, Lladser ME, Knights D, Stombaugh J, Knight R. UniFrac: an effective distance metric for microbial community comparison. ISME J.2011;5(2):169-172. doi:10.1038/ismej.2010.133.</p>

    <p class="paragraph">Lozupone CA, Hamady M, Kelley ST, Knight R. Quantitative and qualitative beta diversity measures lead to different insights into factors that structure microbial communities. Appl Environ Microbiol. 2007;73(5):1576-1585. doi:10.1128/AEM.01996-06.</p>

    <p class="paragraph">Luo C, Tsementzi D, Kyrpides N, Read T, Konstantinidis KT. Direct comparisons of Illumina vs. Roche 454 sequencing technologies on the samemicrobial community DNA sample [published correction appears in PLoS One. 2012;7(3):10.1371/annotation/64ba358f-a483-46c2-b224-eaa5b9a33939]. PLoS One. 2012;7(2):e30087. doi:10.1371/journal.pone.0030087.</p>

    <p class="paragraph">McArdle BH, Anderson MJ. FITTING MULTIVARIATE MODELS TO COMMUNITY DATA: A COMMENT ON DISTANCE-BASED REDUNDANCY ANALYSIS.Ecology. 2001;82(1):290-297. doi:10.1890/0012-9658(2001)082[0290:fmmtcd]2.0.co;2.</p>

    <p class="paragraph">Minchin PR. An evaluation of the relative robustness of techniques for ecological ordination. Vegetatio. 1987;69(1-3):89-107. doi:10.1007/bf00038690.</p>

    <p class="paragraph">Noval Rivas M, Burton OT, Wise P, et al. A microbiota signature associated with experimental food allergy promotes allergic sensitization and anaphylaxis. J Allergy Clin Immunol. 2013;131(1):201-212. doi:10.1016/j.jaci.2012.10.026.</p>

    <p class="paragraph">Oros-Sichler M, Gomes NC, Neuber G, Smalla K. A new semi-nested PCR protocol to amplify large 18S rRNA gene fragments for PCR-DGGE analysisof soil fungal communities. J Microbiol Methods. 2006;65(1):63-75. doi:10.1016/j.mimet.2005.06.014.</p>

    <p class="paragraph">O’Reilly FJ, Mielke PW. Asymptotic normality of MRPP statistics from invariance principles of u-statistics.Communications in Statistics - Theory and Methods. 1980;9(6):629-637. doi:10.1080/03610928008827907.</p>

    <p class="paragraph">Richards LE, Jolliffe IT. Principal Component Analysis.Journal of Marketing Research. 1988;25(4):410. doi:10.2307/3172953.</p>

    <p class="paragraph">Segata N, Izard J, Waldron L, et al. Metagenomic biomarker discovery and explanation. Genome Biol. 2011;12(6):R60. Published 2011 Jun 24. doi:10.1186/gb-2011-12-6-r60.</p>

    <p class="paragraph">Sheik CS, Mitchell TW, Rizvi FZ, et al. Exposure of soil microbial communities to chromium and arsenic alters their diversity and structure.PLoS One. 2012;7(6):e40059. doi:10.1371/journal.pone.0040059.</p>

    <p class="paragraph">Stat M, Pochon X, Franklin EC, et al. The distribution of the thermally tolerant symbiont lineage (Symbiodinium clade D) in corals from Hawaii: correlations with host and the history of ocean thermal stress. Ecol Evol. 2013;3(5):1317-1329. doi:10.1002/ece3.556.</p>

    <p class="paragraph">Stoeck T, Behnke A, Christen R, et al. Massively parallel tag sequencing reveals the complexity of anaerobic marine protistan communities. BMC Biol. 2009;7:72. Published 2009 Nov 3. doi:10.1186/1741-7007-7-72.</p>

    <p class="paragraph">Warton DI, Wright ST, Wang Y. Distance-based multivariate analyses confound location and dispersion effects.Methods in Ecology and Evolution. 2011;3(1):89-101. doi:10.1111/j.2041-210x.2011.00127.x.</p>

    <p class="paragraph">Xiao-Qian, L. I. , Zhu, J. H. , & Song, Y. T. . (2015). Comparison and analysis of statistics methods of mantel test and dbrda in landscapegenetics.Journal of Liaoning University(Natural Sciences Edition)..</p>

    <p class="paragraph">Yang SL, Zhang J, Xu XJ. Influence of the Three Gorges Dam on downstream delivery of sediment and its environmental implications, Yangtze River.Geophysical Research Letters. 2007;34(10). doi:10.1029/2007gl029472.</p>

    <p class="paragraph">Youssef N, Sheik CS, Krumholz LR, Najar FZ, Roe BA, Elshahed MS. Comparison of species richness estimates obtained using nearly complete fragments and simulated pyrosequencing-generated fragments in 16S rRNA gene-based environmental surveys. Appl Environ Microbiol. 2009;75(16):5227-5236. doi:10.1128/AEM.00592-09.</p>

    <p class="paragraph">Zapala MA, Schork NJ. Multivariate regression analysis of distance matrices for testing associations between gene expression patterns and related variables. Proc Natl Acad Sci U S A. 2006;103(51):19430-19435. doi:10.1073/pnas.0609333103.</p>

        <br />

        <br />


</body> 
    
</html>    
    