<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>
<head>
	<title> XXXXSCXXXXXXXX-ZXX-JXXX-BX-16RNA-Seq Reprot </title>
	<META NAME="Author" CONTENT="guoyang@novogene.cn"> 
	<META NAME="Modified" CONTENT="mengfanrui@novogene.cn"> 
	<META NAME="Version" CONTENT="2014820v2.0"> 
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />


<link rel="stylesheet" href="src/css/jquery-ui.css" />
<link rel="stylesheet" href="src/css/bootstrap-4.3.1.min.css" />
<link rel="stylesheet" href="src/css/jquery.tocify.css" />
<link rel="stylesheet" href="src/css/style.css"/>
<link rel="stylesheet" href="src/css/base.css"/>
<link rel="stylesheet" href="src/css/gallery.css"/>
<script src="src/js/jquery-3.3.1.min.js" type="text/javascript"></script>
<script src="src/js/jquery-ui.min.js" type="text/javascript"></script>
<script src="src/js/jquery.tocify.min.js" type="text/javascript"></script>
<script src="src/js/bootstrap-4.3.1.min.js" type="text/javascript"></script>
<script src="src/js/scrollTop.js" type="text/javascript"></script>

<script type="text/javascript">
    /* show or hide element*/
    function hiddenShow(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Annotation";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Annotation";
      };
      return 0;
    }
</script>

<script type="text/javascript">
    /* show or hide element*/
    function hiddenQA(btnObj){
      ctObj = btnObj.nextElementSibling;
      if (ctObj.style.display == 'inline'){
        ctObj.style.display = 'none';
        btnObj.innerHTML = "Show Q&A";
      }else{
        ctObj.style.display = 'inline';
        btnObj.innerHTML = "Hide Q&A";
      };
      return 0;
    }
</script>

<script type="text/javascript">


	$(function () {

		var options = {

			selectors: "h2,h3,h4,h5",

			theme: "bootstrap",

			scrollTo: "60"

		};

		var toc = $("#toc").tocify(options).data("toc-tocify");

	});

</script>



<script type="text/javascript">

function trans(img_div_id) {

img_div = $("#" + img_div_id)

img = $("#" + img_div_id + " img").first();

if (img_div.hasClass("fullscreen-div")) {

img_div.removeClass("fullscreen-div");

img.removeClass("fullscreen-img");

}

else {

img_div.addClass("fullscreen-div");

img.addClass("fullscreen-img");

}

}



function filter_display() {

return $(this).css("display") != "none";

}



function show_default() {

var list_group_id = "#" + $(this).attr("id");

if ($(list_group_id + " a.active").css("display") == "none") {

var list_tab = $(list_group_id + " a.active");

var pan_id = list_tab.attr("href");

$(pan_id).removeClass("active show");

list_tab.removeClass("active");

}

if ($(list_group_id + " a.active").filter(filter_display).length == 0) {

$(list_group_id + " a").filter(filter_display).first().tab("show");

}

}

$.fn.show_default = show_default



function next(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_next", "_list-tab") + " .active").first();

while (present_tab.next().length == 1) {

if (present_tab.next().css("display") != "none") {

present_tab.next().tab("show");

break;

}

else {

present_tab = present_tab.next();

}

}



}

function prev(botton_id) {

var present_tab = $("#" + botton_id.replace("_button_prev", "_list-tab") + " .active").first();

while (present_tab.prev().length == 1) {

if (present_tab.prev().css("display") != "none") {

present_tab.prev().tab("show");

break;

}

else {

present_tab = present_tab.prev();

}

}





}



$(document).ready(function () {

$(".form-control").on("keyup", function () {

var value = $(this).val().toLowerCase();

var input_box_id = "#" + $(this).attr("id");

var list_group_id = input_box_id.replace("_InputList", "_list-tab");

$(list_group_id + " a").filter(function () {

$(this).toggle($(this).text().toLowerCase().indexOf(value) > -1);

});

$(list_group_id).show_default();

});

$(".list-group").each(show_default);

});



</script>

</head>


<body>

<div id="toc" class="tocify">
</div>

<div style="position:fixed; top:0; left:0; width:100%; z-index:1000;">
	<table width="100%" height="50px" border="0" cellpadding="5" cellspacing="0"
		style="background:#F3F3F3; border-bottom:1px solid #eee;border-top: none;background-image:-moz-linear-gradient(top,#EEEEEE,#FFFFFF); background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#EEEEEE), color-stop(1,#FFFFFF));filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#EEEEEE', endColorstr='#FFFFFF', GradientType='0')">
		<tr height="86%">
			<td width='1%'></td>
			<td><img src="src/images/logo1.png"></td>
			<td><div style="margin:5px;border-left:1px solid #DCDCDC;height:30px"></div></td>
			<td width="20%"></td>
			<td width="15%"></td>
			<td width="25%"
				style="text-align:right;padding:0px;margin:0px;color:black;font-size:14px;font-weight:50;">
				<!---Tel: 916-252-0068-383
				<br><a href="mailto:op-rna@novogene.com" target="black"
				style="text-decoration:none;color:black;font-size:14px;font-weight:50;">Email: inquiry_us@novogene.com<a></td>--->
			<td width="1%"></td>
		</tr>
	</table>
</div>



<!--------------------------标题---------------------------------->

<div id="main">
</br><h1> RNA-seq Analysis Report</h1></br>
<table class="gy">
<tbody>
<tr>
<td>Contract ID</td>
<td style="text-align:left">HXXXSCXXXXXXXX</td>
</tr>
<tr>
<td>Contract Name</td>
<td style="text-align:left">XXXXXXXXXXXXXXXXXXXX</td>
</tr>
<tr>
<td>Batch ID</td>
<td style="text-align:left">XXXXSCXXXXXXXX-ZXX-JXXX-BX-16</td>
</tr>
<tr>
<td>Species</td>
<td style="text-align:left">animal</td>
</tr>
<tr>
<td>Report Time</td>
<td style="text-align:left">2025-01-06</td>
</tr>
<tr>
<td>Reminder</td>
<td style="text-align:left">Partial results are presented in this report, while full results will be delivered in data release.<br/>
Hyperlink of results in this report will be only valid in data release, after statement confirmation. </td>
</tr>
</tbody></table>
<br/><br/>

<!----------------------------------------建库测序流程----------------------------------------->

<h2>1 Introduction</h2>
<p class="paragraph">From RNA samples extraction to final data, every step of procedures containing samples quality control, library construction and sequencing would directly affect the quality and quantity of data, which determines the bioinformatic analysis results. Novogene promises to strictly control on all production data with high quality output and ensure the raw data accurate and reliable.</p>

<p class="paragraph">Workflow is shown as follows:</p>

<p class = "center"><img class="center" src="src/images/sequence_pipeline.png" height=550 width=650/></p>
<p class="name_fig">Figure 1 Project workflow</p>

<h2>2 Library Construction and Sequencing</h2>
<h3>2.1 Sample Quality Control</h3>
<p class="paragraph">Please refer to Novogene’s QC report for methods of sample quality control.</p>
<h3>2.2 Library Construction, Quality Control and Sequencing</h3>
<p class="paragraph">Messenger RNA was purified from total RNA using poly-T oligo-attached magnetic beads. After fragmentation, the first strand cDNA was synthesized using random hexamer primers, followed by the second strand cDNA synthesis using either dTTP for non strand specific library or dUTP for strand specific library.</p>
<p class="paragraph">For the non strand specific library, it was ready after end repair, A-tailing, adapter ligation, size selection, amplification, and purification(Figure 2.A).</p>
<p class="paragraph">For the strand specific library, it was ready after end repair, A-tailing, adapter ligation, size selection, amplification, and purification(Figure 2.B).</p>

<p class = "center"><img class="center" src="src/images/Workflow_A.png"  width=20%/>
<p class="name_fig">Figure 2.A Workflow of non-directional library construction</p>

<p class = "center"><img class="center" src="src/images/Workflow_B.png"  width=20%/>
<p class="name_fig">Figure 2.B Workflow of directional library construction</p>

<p class="paragraph">The library was checked with Qubit and real-time PCR for quantification and bioanalyzer for size distribution detection. Quantified libraries will be pooled and sequenced on Illumina platforms, according to effective library concentration and data amount.</p>
	<!----------------------------------------生物信息分析流程----------------------------------------->


<h2 id="pipline">3 Bioinformatics Analysis Pipeline</h2>
<p class="paragraph">The analysis workflow for species without a reference genome is as follows:</p>
<p class="center">

<img class="center" src="src/images/noref_pipeline.png" height=450 width=500/>

</p>
<p class="paragraph"><strong>Method details involved in the project are available at <a href="src/denovo_report_method.pdf" title="点击打开" target="_blank">methods</a></strong></p>

<!-------------------------------------------- 原始序列数据 --------------------------------------->


<h2>4 Analysis Results</h2>
<h3>4.1 Data Quality Control</h3>
<p class="paragraph">Original image data file from high-throughput sequencing (like Illumina) is transformed into sequenced reads (called Raw Data or Raw Reads) by CASAVA base recognition (Base Calling). Raw data are stored in FASTQ(fq)(Cock PJ et al., 2010) format files, which contain sequences of reads and corresponding base quality. Each read has four descriptive lines as follow:</p>
<p style="font-weight: bold;">FASTQ file:</p>
<div class="bs-docs-qa">
<p class="seq">
@A00564:177:HTF77DSXX:2:1101:1416:1031 1:N:0:CCTAATCC+GTACGCAA<br />
TNTAGATCTTAGGCGGTTGGTGTAGGTATCGACACAGGTCTTCATTTTGGCTAATAATGTACCAACGGAAGTCTGACATTCTGACTGTTCTTCTTCCTCTTCTTCCCCATTCTCTGTAGAAAGAGGCAACAACAGCGGCACCATAGATGT<br />
+<br />
F#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF:FFFFF:FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF:FFFFFFFFFFFFFFFFFFFFFFFFFFF:FFFF:FFFFF:FFFFFFFFFFFFFFFFFFFFF,FFFFFFF<br />
</div>
(1) Line 1: the at sign (@) followed by sequence identifiers and optional description info (such as FASTA title line).<br>
(2) Line 2: base sequences (raw read, A, G, C, and T).<br>
(3) Line 3: the plus sign (+) optionally followed by the same Illumina sequence identifiers and description information as Line 1. <br>
(4) Line 4: the quality values for each base, corresponding to the data in Line 2.<br>
<p style="font-weight: bold;">Illumina Sequence Identifier:</p>
<table class="gy">
<tr><th>Identifier</th><th>Meaning</th></tr>
<tr><td>HWI-ST1276</td><td>Instrument – unique identifier of the sequencer </td></tr>
<tr><td>71</td><td>Run number – Run number on instrument</td></tr>
<tr><td>C1162ACXX</td><td>Flow cell ID – ID of flow cell</td></tr>
<tr><td>1</td><td>Lane Number – positive integer</td></tr>
<tr><td>1101</td><td>Tile Number – positive integer</td></tr>
<tr><td>1208</td><td>X – x coordinate of the spot. Integer which can be negative</td></tr>
<tr><td>2458</td><td>Y – y coordinate of the spot. Integer which can be negative</td></tr>
<tr><td>1</td><td>Read Number - 1 can be single read or Read 2 of paired-end</td></tr>
<tr><td>N</td><td>Y if the read is filtered out (did not pass), N otherwise</td></tr>
<tr><td>0</td><td>control number - 0 when none of the control bits are on, otherwise it is an even number</td></tr>
<tr ><td>CGATGT</td><td>Illumina index sequences</td></tr>
</table>
<p class="tremark">The details of Sequencing identifier of Illumina are as follows:</p>
<p class="tremark">(1) HWI-ST1276:71 HWI-ST1276, Instrument - unique identifier of the sequencer; 71, run number - Run number on instrument.</p>
<p class="tremark">(2) C1162ACXX:1:1101:1208:2458 means the coordinate of read on C1162ACXX (Flowcell ID) flowcell, line 1, 1101 tile is(x=1208, y=2458).</p>
<p class="tremark">(3) 1:N:0:CGATGT the first number is 1 or 2, 1 means single reads or the first read of paired ends, 2 means the second of paired ends; the second letter means whether reads is adjusted(Y means yes, N means no); the third number represent the number of Control Bits in sequence; six bases on the fourth place is Illumina index sequence.</p>
<br>
<!----------------------------------------- 测序数据质量评估 -------------------------------------->

<h4>4.1.1 Examination of Sequencing Error Rate</h4>
<p class="paragraph">The error rate for each base can be transformed by the Phred score as in equation 1. "e" represents sequencing error rate, "Qphred" represents base quality values of Illumina platforms (equation 1: Qphred = -10log<sub>10</sub>(e)).</p>

<table class="gy">
<tr><th>Phred score</th><th>ASCII code</th><th>error rate</th><th>correct rate</th><th>Q-sorce</th></tr>
<tr><td>10</td><td>+ (10+33)</td><td>1/10</td><td>90%</td><td>Q10</td></tr>
<tr><td>20</td><td>5 (20+33)</td><td>1/100</td><td>99%</td><td>Q20</td></tr>
<tr><td>30</td><td>? (30+33)</td><td>1/1000</td><td>99.9%</td><td>Q30</td></tr>
<tr><td>40</td><td>I (40+33)</td><td>1/10000</td><td>99.99%</td><td>Q40</td></tr>
</table>
<br>
<p class="paragraph">Sequencing error rate and base quality varies depending on sequencers, reagent residues, and different sample types. For RNA-seq technology, sequencing error rate distribution can be featured:</p>
<p class="paragraph">(1) Error rate increases with the sequencing reads for consumption of sequencing reagent. It is common in the Illumina high-throughput sequencing platform(Erlich et al., 2008; Jiang L et al., 2011.).</p>
<p class="paragraph">(2) The first six bases have a relatively high error rate due to the incomplete binding of random hexamers used in priming cDNA synthesis(Jiang L et al., 2011). In general, a single base error rate should be lower than 1%.</p><br/>
<!---p class="paragraph"><b>Result Directory: <a href="../XXXXSCXXXXXXXX-ZXX-JXXX-BX-16_results/1.QC/ErrorRate" target="_blank">Result/1.QC/ErrorRate</a></b></p--->

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_ErrorRate_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_ErrorRate_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_B1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_B1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_B2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_B2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_B3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_B3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B3.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_G1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_G1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_G2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_G2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_G3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_G3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G3.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EN1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EN2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EN3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN3.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LN1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LN2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LN3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN3.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_EZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_EZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ3.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ1.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ2.error_rate_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_ErrorRate_pane_LZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_ErrorRate_list_LZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ3.error_rate_distribution.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_ErrorRate_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_ErrorRate_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_ErrorRate_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_B1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_B1" role="tab" aria-controls="B1">B1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_B2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_B2" role="tab" aria-controls="B2">B2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_B3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_B3" role="tab" aria-controls="B3">B3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_G1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_G1" role="tab" aria-controls="G1">G1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_G2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_G2" role="tab" aria-controls="G2">G2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_G3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_G3" role="tab" aria-controls="G3">G3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EN1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EN1" role="tab" aria-controls="EN1">EN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EN2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EN2" role="tab" aria-controls="EN2">EN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EN3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EN3" role="tab" aria-controls="EN3">EN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LN1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LN1" role="tab" aria-controls="LN1">LN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LN2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LN2" role="tab" aria-controls="LN2">LN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LN3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LN3" role="tab" aria-controls="LN3">LN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EZ1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EZ1" role="tab" aria-controls="EZ1">EZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EZ2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EZ2" role="tab" aria-controls="EZ2">EZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_EZ3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_EZ3" role="tab" aria-controls="EZ3">EZ3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LZ1"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LZ1" role="tab" aria-controls="LZ1">LZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LZ2"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LZ2" role="tab" aria-controls="LZ2">LZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_ErrorRate_list_LZ3"
	data-toggle="list" href="#gallery_figure_ErrorRate_pane_LZ3" role="tab" aria-controls="LZ3">LZ3</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.1　Error Rate Distribution</p>
<p class="premark center ">The x-axis shows the base position along each sequencing read and the y-axis shows the base error rate.</p><br>

<!--------------------------------------------------------->



<h4>4.1.2 GC Content Distribution</h4>
<p class="paragraph">GC content distribution is to detect potential AT/GC separation, which affects subsequent gene expression quantification. In view of random fragmentation and biological law of G/C-A/T content, G and C, A and T should be respectively equal, and the content should be stable throughout the whole sequencing process for non-stranded library (If the library is strand-specific, AT separation or GC separation may occur). A large variation of sequencing error in the first 6-7 bases is allowed considering the usage of random primer in library construction, which is normal that the first few bases have certain preference in existing high-throughput sequencing technology.<b></b></p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_GC_Content_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_GC_Content_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_B1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_B1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_B2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_B2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_B3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_B3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B3.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_G1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_G1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_G2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_G2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_G3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_G3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G3.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EN1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EN2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EN3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN3.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LN1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LN2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LN3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN3.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_EZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_EZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ3.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ1.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ2.GC_content_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GC_Content_pane_LZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_GC_Content_list_LZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ3.GC_content_distribution.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_GC_Content_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_GC_Content_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_GC_Content_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_B1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_B1" role="tab" aria-controls="B1">B1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_B2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_B2" role="tab" aria-controls="B2">B2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_B3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_B3" role="tab" aria-controls="B3">B3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_G1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_G1" role="tab" aria-controls="G1">G1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_G2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_G2" role="tab" aria-controls="G2">G2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_G3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_G3" role="tab" aria-controls="G3">G3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EN1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EN1" role="tab" aria-controls="EN1">EN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EN2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EN2" role="tab" aria-controls="EN2">EN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EN3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EN3" role="tab" aria-controls="EN3">EN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LN1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LN1" role="tab" aria-controls="LN1">LN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LN2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LN2" role="tab" aria-controls="LN2">LN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LN3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LN3" role="tab" aria-controls="LN3">LN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EZ1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EZ1" role="tab" aria-controls="EZ1">EZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EZ2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EZ2" role="tab" aria-controls="EZ2">EZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_EZ3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_EZ3" role="tab" aria-controls="EZ3">EZ3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LZ1"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LZ1" role="tab" aria-controls="LZ1">LZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LZ2"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LZ2" role="tab" aria-controls="LZ2">LZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GC_Content_list_LZ3"
	data-toggle="list" href="#gallery_figure_GC_Content_pane_LZ3" role="tab" aria-controls="LZ3">LZ3</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.2 GC Content Distribution</p>
<p class="premark center">The x-axis shows each base position within a read, and the y-axis shows the percentage of each base, with each base represented by a different color. The left side of the vertical dashed line is the GC-content of read 1, the right side is the GC-content of read 2.</p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/1.QC/2.GC" target="_blank">Result/1.QC/2.GC</a></p>


<!-------------------------------------------------------->


<h4><a>4.1.3 Sequencing Data Filtering</a></h4>

<p class="paragraph">The sequenced reads/raw reads often contain low quality reads or reads with adapters, which will affect the quality of downstream analysis. To avoid this, it’s necessary to filter the raw reads and get the clean reads.<b></b></p>
<p style="font-weight: bold;">Raw reads filtering is as follows:</p>
<img class="center" height="550" alt="" width="680" src="src/images/data_filter.png" /></p>
<p class="step">(1) Remove reads with adapter contamination.<br />
(2) Remove reads when uncertain nucleotides constitute more than 10 percent of either read (N > 10%).<br />
(3) Remove reads when low quality nucleotides (Base Quality less than 5) constitute more than 50 percent of the read.<br></p>

<p style="font-weight: bold;">Adapter Sequences:</p>

 <div class="block">

<p style="font-size:14px">
<span class="citation"><span class="citation">
    P5 adapter：<br />
    P5→P7’(5’→3’)<br />
     AATGATACGGCGACCACCGAGATCTACAC[i5]ACACTCTTTCCCTACACGACGCTCTTCCGATCT<br />
    <br />
    P7 adapter：<br />
    P5→P7’(5’→3’)<br />
     GATCGGAAGAGCACACGTCTGAACTCCAGTCAC[i7]ATCTCGTATGCCGTCTTCTGCTTG<br />
    <br />
</p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_DataFiltration_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_DataFiltration_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_B1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_B1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_B2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_B2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_B3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_B3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/B3.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_G1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_G1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_G2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_G2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_G3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_G3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/G3.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EN1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EN2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EN3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EN3.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LN1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LN1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LN2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LN2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LN3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LN3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LN3.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_EZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_EZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZ3.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LZ1" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LZ1"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ1.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LZ2" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LZ2"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ2.raw_reads_classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_DataFiltration_pane_LZ3" 
	role="tabpanel" aria-labelledby="gallery_figure_DataFiltration_list_LZ3"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LZ3.raw_reads_classification.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_DataFiltration_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_DataFiltration_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_DataFiltration_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_B1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_B1" role="tab" aria-controls="B1">B1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_B2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_B2" role="tab" aria-controls="B2">B2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_B3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_B3" role="tab" aria-controls="B3">B3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_G1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_G1" role="tab" aria-controls="G1">G1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_G2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_G2" role="tab" aria-controls="G2">G2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_G3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_G3" role="tab" aria-controls="G3">G3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EN1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EN1" role="tab" aria-controls="EN1">EN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EN2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EN2" role="tab" aria-controls="EN2">EN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EN3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EN3" role="tab" aria-controls="EN3">EN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LN1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LN1" role="tab" aria-controls="LN1">LN1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LN2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LN2" role="tab" aria-controls="LN2">LN2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LN3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LN3" role="tab" aria-controls="LN3">LN3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EZ1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EZ1" role="tab" aria-controls="EZ1">EZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EZ2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EZ2" role="tab" aria-controls="EZ2">EZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_EZ3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_EZ3" role="tab" aria-controls="EZ3">EZ3</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LZ1"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LZ1" role="tab" aria-controls="LZ1">LZ1</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LZ2"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LZ2" role="tab" aria-controls="LZ2">LZ2</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_DataFiltration_list_LZ3"
	data-toggle="list" href="#gallery_figure_DataFiltration_pane_LZ3" role="tab" aria-controls="LZ3">LZ3</a>
	
	</div>
	</div>
	</div>
	</p>


<p class="name_fig">Figure 4.3 Composition of Raw Reads</p>

<p class="premark">Results are shown as percentage of total raw reads.
<p class="premark">(1) Adapter related: (reads containing adapter) / (total raw reads).</p>
<p class="premark">(2) Containing N: (reads with more than 10% N) / (total raw reads).</p>
<p class="premark">(3) Low quality: (reads of low quality) / (total raw reads).</p>
<p class="premark">(4) Clean reads: (clean reads) / (total raw reads).</p>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/1.QC/3.Filter" target="_blank">Result/1.QC/3.Filter</a></p>
<br>
<!-------------------------------------------------------->

<h4><a>4.1.4 Data Quality Control Summary</a></h4>
<p class="paragraph">The data is summarized in the table below.<b> </b></p>
<p class="center">Table 4.1 Data Production</p>
<div id="tb">
<table class="gy">

<tr><th>sample</th><th>library</th><th>raw_reads</th><th>raw_bases</th><th>clean_reads</th><th>clean_bases</th><th>error_rate</th><th>Q20</th><th>Q30</th><th>GC_pct</th></tr>

        <tr>
        <td>B1</td><td>FRAS230282358-1r</td><td>23093152</td><td>6.9</td><td>22765127</td><td>6.8</td><td>0.03</td><td>97.79</td><td>93.87</td><td>41.23</td>
        </tr>

        <tr>
        <td>B2</td><td>FRAS230282359-1r</td><td>23479045</td><td>7.0</td><td>23131423</td><td>6.9</td><td>0.03</td><td>97.68</td><td>93.67</td><td>41.31</td>
        </tr>

        <tr>
        <td>B3</td><td>FRAS230282360-1r</td><td>23731316</td><td>7.1</td><td>23409419</td><td>7.0</td><td>0.03</td><td>97.74</td><td>93.69</td><td>40.62</td>
        </tr>

        <tr>
        <td>G1</td><td>FRAS230282361-1r</td><td>23727127</td><td>7.1</td><td>23398301</td><td>7.0</td><td>0.03</td><td>97.61</td><td>93.42</td><td>41.38</td>
        </tr>

        <tr>
        <td>G2</td><td>FRAS230282362-1r</td><td>23602696</td><td>7.1</td><td>23234226</td><td>7.0</td><td>0.03</td><td>97.49</td><td>93.14</td><td>41.17</td>
        </tr>

        <tr>
        <td>G3</td><td>FRAS230282363-1r</td><td>22356895</td><td>6.7</td><td>22002582</td><td>6.6</td><td>0.03</td><td>97.66</td><td>93.63</td><td>41.17</td>
        </tr>

        <tr>
        <td>EN1</td><td>FRAS230282364-1r</td><td>24134110</td><td>7.2</td><td>23566863</td><td>7.1</td><td>0.03</td><td>97.83</td><td>93.92</td><td>42.98</td>
        </tr>

        <tr>
        <td>EN2</td><td>FRAS230282365-1r</td><td>22173390</td><td>6.7</td><td>21778080</td><td>6.5</td><td>0.03</td><td>97.62</td><td>93.52</td><td>40.26</td>
        </tr>

        <tr>
        <td>EN3</td><td>FRAS230282366-1r</td><td>21352356</td><td>6.4</td><td>20888698</td><td>6.3</td><td>0.03</td><td>97.69</td><td>93.64</td><td>41.53</td>
        </tr>

        <tr>
        <td>LN1</td><td>FRAS230282367-1r</td><td>22256122</td><td>6.7</td><td>21741234</td><td>6.5</td><td>0.03</td><td>97.85</td><td>94.02</td><td>41.65</td>
        </tr>

        <tr>
        <td>LN2</td><td>FRAS230282368-1r</td><td>22756420</td><td>6.8</td><td>22064843</td><td>6.6</td><td>0.03</td><td>97.55</td><td>93.51</td><td>40.79</td>
        </tr>

        <tr>
        <td>LN3</td><td>FRAS230282369-1r</td><td>23671762</td><td>7.1</td><td>23182117</td><td>7.0</td><td>0.03</td><td>97.81</td><td>93.94</td><td>41.82</td>
        </tr>

        <tr>
        <td>EZ1</td><td>FRAS230282370-1r</td><td>22241594</td><td>6.7</td><td>21736602</td><td>6.5</td><td>0.03</td><td>96.94</td><td>92.51</td><td>40.89</td>
        </tr>

        <tr>
        <td>EZ2</td><td>FRAS230282371-1r</td><td>23003439</td><td>6.9</td><td>22583325</td><td>6.8</td><td>0.03</td><td>97.54</td><td>93.49</td><td>40.99</td>
        </tr>

        <tr>
        <td>EZ3</td><td>FRAS230282372-1r</td><td>23662854</td><td>7.1</td><td>23169911</td><td>7.0</td><td>0.03</td><td>96.80</td><td>91.64</td><td>40.85</td>
        </tr>

        <tr>
        <td>LZ1</td><td>FRAS230282373-1r</td><td>23278165</td><td>7.0</td><td>22808177</td><td>6.8</td><td>0.03</td><td>97.60</td><td>93.36</td><td>38.05</td>
        </tr>

        <tr>
        <td>LZ2</td><td>FRAS230282374-1r</td><td>22644931</td><td>6.8</td><td>22114204</td><td>6.6</td><td>0.03</td><td>97.74</td><td>93.79</td><td>37.18</td>
        </tr>

        <tr>
        <td>LZ3</td><td>FRAS230282375-1r</td><td>24036316</td><td>7.2</td><td>23500139</td><td>7.1</td><td>0.03</td><td>97.63</td><td>93.55</td><td>39.69</td>
        </tr>

</table>

</table>
</div>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<strong>sample:</strong> SampleID.<br />
	<strong>library:</strong> The number of libraries.<br />
	<strong>raw_reads:</strong> Reads count from the raw data, four rows as a unit, with statistics of reads count for every sequencing.<br />
	<strong>raw_bases:</strong> Base number of raw data. (number of raw reads) * (sequence length), converting unit to G.<br />
	<strong>clean_reads:</strong> Base number of raw data after filtering. (number of clean reads) * (sequence length), converting unit to G.<br />
	<strong>clean_bases:</strong> (clean base=clean reads*150bp) number multiply read length, saved in G unit.<br />
	<strong> error_rate:</strong> Average sequencing error rate, which is calculated by Qphred=-10log10(e).<br />	
	<strong>Q20:</strong> The percentage of the bases whose Q Phred values is greater than 20. (Number of bases with Q Phred value > 20) / (Number of total bases) *100.<br />
	<strong>Q30:</strong> The percentage of the bases whose Q Phred values is greater than 30. (Number of bases with Q Phred value > 30) / (Number of total bases) *100.<br />
	<strong>GC_pct:</strong> The percentage of G&C base numbers of total bases.(G&C base number) / (Total base number)*100.<br />
</ul>
</div>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/1.QC/4.Stat/data_table.xls" target="_blank">Result/1.QC/4.Stat/data_table.xls</a></p>
<!-------------------------------------------------------->
<br>

<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:The error rate will increase with the higher length of sequencing, what is the acceptable range of error rate?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Novogene sets a high requirement for sequencing quality. Generally, error rate of single base should be lower than 1%. For special case, the maximum error rate for single base lower than 6% is acceptable.<br><br></td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q:What is the difference between the align software of RNA data and of DNA data?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>To guarantee the quality of bioinformatic analysis, Novogene sets several criteria for getting clean data:<br>
	<p class="premark">1.Removing the reads with adapter;</p>
	<p class="premark">2.Removing the reads containing more than 10% N (N represents the uncertain base while base calling);</p>
	<p class="premark">3.Removing low quality reads (a read containing more than 50% of bases with low quality score: quality score <= 20).</p></td></tr>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br><strong>adapter:</strong> A necessary part for illumina sequencing. It is introduced while library preparation and could be recognized by oligos fixed in flow cell.<br><br><strong>index:</strong> It enables the probability of sequencing several samples simultaneously in the same lane. It is introduced during library preparation and can be used to distinguish samples while data analysis.<br><br><strong>Q20,Q30:</strong> The sequencing quality score of a given base, Q, is defined by the following equation: Q = -10log10(e), where e is the estimated probability of the base call being wrong. A quality score of 20 (Q20) represents error rate of 1 in 100, with a corresponding call accuracy of 99%. Similarly, Q30 means the call accuracy of 99.9%.<br><br><strong>raw data/raw reads:</strong> Data from sequencing platform before data filtering.<br><br></strong><strong>clean data/clean reads:</strong> The filtered data, which is processed by data quality control. All the further bioinformatic analysis are based on clean reads.<br></td></tr>
</table>
</div>
<br />

<br>
<!-------------------------------------------------------->




<!----------------------------------------- 转录本拼接 -------------------------------------->


<h3 id="id_order_trinity">4.2  Transcript Reconstruction </h3>


<h4>4.2.1 Transcriptome Reconstruction</h4>
<p class="paragraph">For samples in the absence of a reference genome, clean reads need to be assembled to get a reference sequence for the following analysis. Trinity(Grabherr MG et al., 2011) is the software chosen to complete the transcriptome reconstruction process.</p>
<p class="paragraph">Trinity is developed by Broad Institute and Hebrew University of Jerusalem. It is a professional transcriptome assembler software (comprising modules entitled Inchworm, Chrysalis and Butterfly). The workflow of Trinity is as follows:</p>
<p class="paragraph"><b>(1)Inchworm</b>: Constructs a k-mer dictionary from all sequenced reads (in practice, k = 31), selects the most frequent seeding k-mer in the dictionary and extends the seed in each direction to form a contig assembly.</p>
<p class="paragraph"><b>(2)Chrysalis</b>: Chrysalis clusters minimally overlapping Inchworm contigs into sets of connected components, and constructs complete de Bruijn graphs for each component. Each component defines a collection of Inchworm contigs that are likely to be derived from alternative splice forms or closely related paralogs. </p>
<p class="paragraph"><b>(3)Butterfly</b>: Butterfly reconstructs plausible, full-length, linear transcripts by reconciling the individual de Bruijn graphs generated by Chrysalis with the original reads and paired ends. It reconstructs distinct transcripts for splice isoforms and paralogous genes, and resolves ambiguities stemming from errors or from sequences >k bases long that are shared between transcripts. The final assembled result file: TRINITY.fasta.</p>
<p class="center">
<img class="center" src="src/images/trinity.png" height=300 width=450 />
</p>
<p class="center">Figure 4.4  Overview of Trinity</p>
<p class="premark">(a) Inchworm assembles the reads data set (short black lines, top) by greedily searching for paths in a k-mer graph (middle), resulting in a collection of linear contigs (color lines, bottom), with each k-mer present only once in the contigs. </p>
<p class="premark">(b) Chrysalis pools contigs (colored lines) if they share at least one k-1-mer and if reads span the junction between contigs, and then it builds indviduals de Bruijn Graphs from each pool. </p>
<p class="premark">(c) Butterfly takes each de Bruijn Graph from Chrysalis (top), and trims spurious edges and compacts linear paths (middle). It then reconciles the graph with reads (dashed colored arrows, bottom) and pairs (not shown), and outputs one linear sequence for each splice form and/or paralogous transcript represented in the graph (bottom, colored sequence).</p>

<br />
<p class="paragraph">The sequencing data of assembled transcriptome is recorded as FASTA format as follows:</p>
<div class="bs-docs-qa">
<p class='seq'>>TRINITY_DN100_c0_g1_i1 len=347 path=[0:0-346]<br />CTAGGCAGGGTGGTTGAAGGAATCAGACCTACGTAGCAGATTTCGGGAACACTTGATTGCAATACTGTTTCAGGATTGGA<br />GCCCTTTATTTTAAGAGTAATTCTAAAGGTATGAGATAGGAGGGCTACTCTGCAAACCCACTAGTAGTCTTCTCTATTTA<br />AGTTTGTCAAAATATATAGGACAATAATGATAGGTATTAAAATCGTAAGTTTTATTTTATAGATTTTGTTTCTGTGTATG<br />TAGTAGTAGTTCTAAAAACGAAATTTTAAGTGGCTAAACCTAGTGAATGATAATTTCAGGTGAATGAGTATATTGCATGG<br />AGAAATGAAAGGTTAAATGGTAATGTG
<br />>TRINITY_DN105_c0_g1_i1 len=359 path=[0:0-358]<br />TTGCCATAGTATACTCTGTTTTCAGAATGAAACTGTTTTCTTCGTAAGGAACCACTTTGTATTAGTTGTCGTCAGCCTTG<br />GTATCCTCCCTCATAACACACTCGAACTCAGAATCTAACAGCAGGAAGCTGTTGTCTTTCTTGACGAGCAGGTAGTCTCG<br />GATTGGGTACACCGAGAGAGCGCTGATTTCGAGCTGGTTGATCTTCTCCATCCGGTTTTTATTGCTGGCGTCAATCGAAT<br />AAATATTCACTCTGTCTTTGCAAATGGTGAAGATTTAATTTTTTTATGAAATTGTCGCATCGTTGATCGCCCCATTGAGA<br />TCATAGTCTCCCAGTTTCTTGGCGTTGTGGTCATAGATG
<br /></p>
</div>
<p class="premark">Line 1 starts with '>' character and followed by the id number of the transcript; 'len=' shows the length of the transcript, which is the base number of the transcript; 'path' includes the pathway information from de Bruijn Graph subComponet. From line 2 to the end encodes the sequence information of the transcript. More detailed explanation could be found from Trinity's website <a href="http://trinityrnaseq.github.io." target=_blank>http://trinityrnaseq.github.io.</a>.</p>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/2.Assembly/1.AssembledTranscriptome/Trinity.fasta" target="_blank">Result/2.Assembly/1.AssembledTranscriptome/Trinity.fasta</a></p>
<br>
<!-------------------------------------------------------->





<h4>4.2.2  Hierarchical Clustering</h4>
<p class="paragraph">Corset(Davidson NM et al., 2014) works by clustering contigs based on shared reads, and separates contigs when different expression patterns between samples are observed. Corset also uses the read information to filter out contigs with a low number of mapped read (less than 10 reads by default).</p>
<p class="paragraph">The sequencing data of hierarchical clustering transcriptome is recorded as FASTA format as follows:</p>
<div class="bs-docs-qa">
<p class='seq'>>Cluster-23194.0<br/>TCACTTGCTACCCAGCAGAAATATTAACAGCAGGGAAGGTTCATAGAAATAAGCAGAATTTTTATATTAAATTTTCTTAT<br/>TTTTATAGTCATCTAATTTTCATATACATGCCCTCCATCTTAGCTACTAATTTAGTGGCCTCAAATAGGATAGGGAATTA<br/>TTTTGACTTTAAGACTGACGAAGGGATAAGACATGCACAGCAAGTTATTGTTGTTAGGCGATGGTCTAATTATTTTATTG<br/>GAGGCATTACTGTATAAGAAGGAATGAAATCGTGAAATTTCTTGATTTTAAGGAACGTTGACATTCTAAGCTTCCAGAAT<br/>GAGTTTAGAAATAAGGAATTTCTTTTTTCTTTAAGCCAAATTTTAC<br/>>Cluster-29245.103206<br/>GGAAGTCTTTGTAAAAAAATATACAGTATTATATAATCTGTTAGCCTATAATGCAACAAAAGAGTATTTTGAATCCAGAT<br/>CCTATAAAACTTATTTTTTAAAATTTGAATCTTTTTAGTTCCCTAAAAGGAAATATTTCAACCCTAAAAGGAAATATTTC<br/>AAAATAATTGCTTACACTTTCTACAAATTGCTTAAACATAATAATCTTCACTTCATAACTAACTGTACTAGCAATACCAT<br/>TAAAGTGATGAATATAATAATTTTCTGTTGTACTGGTACCACCCTTTAAAGTATTAAAAAAAAAATATTAAAAGAGTACC<br/>CAAAAATAATCTAAAATCAATGCAAGCATAAACCAATTAAATATTACAATGTTCATTTACATCAACAGTGAAAAGTAAAT<br/>ATTTTTTCTTGTCCAACATAAACAGAAATTCTGATCAGCCCCGTCAGCCTTTGTTGTTAACTGTTGGAATATCCACATGT<br/>ATTTGATCTTAATTAAAGCCAAAAGATTCAATAACTTTTGTATTGCCTTTACTATTTCAGTCCAATTCTGAATTTTACTT<br/>TTACTTCTTAAAATGCTCAAGAAACTTCAACATACTGAATAATGAAGATTTGTTCTTTAGAATTTTTTACAAAACTAAAT<br/>ATACTGTAAACTAGCTACTTATAAAAAAAAAAAAAATAGGGAAATCTTGT<br/></p>
</div>
<p class="premark">Header line starts with '>' character and followed by the sequence id, "Clusters-X.Y". "X" means the super cluster ID, each super cluster contains all contigs that share one or more reads with another contig in the same super cluster. "Y" means the cluster number of the super cluster ID. More detailed explanation could be found from Corset's website  <a href="https://github.com/Oshlack/Corset/wiki" target=_blank>https://github.com/Oshlack/Corset/wiki</a>.</p>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/2.Assembly/1.AssembledTranscriptome/unigene.fasta" target="_blank">Result/2.Assembly/1.AssembledTranscriptome/unigene.fasta</a></p>
<br>

<!-------------------------------------------------------->





<h4>4.2.3 Transcript Length Distribution</h4>
<p class="paragraph">Clean reads are de novo assembled by Trinity to get assembly transcriptome. Then Corset will perform Hierarchical Clustering to remove redundence. The longest transcripts of each cluster will be selected as unigenes. Length distribution information of transcripts and unigenes are listed in the following tables:</p>
<p class="center">Table 4.2 Overview of the Number of Transcripts and Unigenes in Different Length Intervals</p>
<table class="gy">
	<tr><th>Transcript Length Interval</th><th>200bp-500bp</th><th>500bp-1kbp</th><th>1k-2kbp</th><th>>2kbp</th><th>Total</th></tr>
	
			<tr>
				<td>Number of transcripts</td><td>143377</td><td>102692</td><td>46507</td><td>33438</td><td>326014</td>
			</tr>
	
			<tr>
				<td>Number of Unigenes</td><td>73224</td><td>43789</td><td>18473</td><td>12129</td><td>147615</td>
			</tr>
	
</table>
<br/>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<br />
	<strong>Length_interval:</strong> Represents different length intervals of Transcript/Unigene<br />
    <strong>Number of transcripts:</strong> Indicates the number of Transcript in the corresponding length interval<br />
    <strong>Number of Unigenes:</strong> Indicates the number of Unigene in the corresponding length interval<br />
</ul>
</div>

<p class="center">Table 4.3　Overview of the Length Distribution of Transcripts and Unigenes</p>
<table class="gy">
	<tr><th>Type</th><th>min length</th><th>mean length</th><th>median length</th><th>max length</th><th>N50</th><th>N90</th><th>total nucleotides</th></tr>
	
			<tr>
			<td>Transcript</td><td>301</td><td>968</td><td>550</td><td>26175</td><td>1418</td><td>405</td><td>315515683</td>
			</tr>
	
			<tr>
			<td>Unigene</td><td>301</td><td>872</td><td>504</td><td>26175</td><td>1183</td><td>377</td><td>128671784</td>
			</tr>
	
</table>

<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<br />
	<strong>Min_length:</strong> the minimum length of Transcript/Unigene<br />
	<strong>Mean_length:</strong> the average length of Transcript/Unigene<br />
	<strong>Median_length:</strong> the middle length of Transcript/Unigene<br />
	<strong>Max_length:</strong> the maximum length of Transcript/Unigene<br />
	<strong>N50:</strong> spliced transcripts are sorted by length from long to short, and the length of spliced transcripts is N50, which can be used to evaluate the splicing effect.<br />
	<strong>N90:</strong> spliced transcripts are sorted by length from long to short, and the length of spliced transcripts is N90, which can be used to evaluate the splicing effect.<br />
	<strong>Total_nucleotides:</strong> the total number of nucleotides splicing Transcript/UniGene. -N50/N90: Ranking all transcripts by its length from long to short. Adding the length one by one from the first. When the sum is greater than 50%/90% of the total length, the last added sequence length is N50/N90.<br />
</ul>
</div>

<br/>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_LengthDistribution_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_LengthDistribution_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_LengthDistribution_pane_Length_distribution" 
	role="tabpanel" aria-labelledby="gallery_figure_LengthDistribution_list_Length_distribution"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Unigene_Transcript.length_distribution.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_LengthDistribution_pane_Transcript_Length_distribution" 
	role="tabpanel" aria-labelledby="gallery_figure_LengthDistribution_list_Transcript_Length_distribution"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Transcript_bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_LengthDistribution_pane_Unigene_Length_distribution" 
	role="tabpanel" aria-labelledby="gallery_figure_LengthDistribution_list_Unigene_Length_distribution"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Unigene_bar.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_LengthDistribution_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_LengthDistribution_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_LengthDistribution_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_LengthDistribution_list_Length_distribution"
	data-toggle="list" href="#gallery_figure_LengthDistribution_pane_Length_distribution" role="tab" aria-controls="Length_distribution">Length_distribution</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_LengthDistribution_list_Transcript_Length_distribution"
	data-toggle="list" href="#gallery_figure_LengthDistribution_pane_Transcript_Length_distribution" role="tab" aria-controls="Transcript_Length_distribution">Transcript_Length_distribution</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_LengthDistribution_list_Unigene_Length_distribution"
	data-toggle="list" href="#gallery_figure_LengthDistribution_pane_Unigene_Length_distribution" role="tab" aria-controls="Unigene_Length_distribution">Unigene_Length_distribution</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.5  Length Distribution of Transcripts and Unigene</p>
<p class="premark center">X-axis indicates length interval of transcript and unigene; Y-axis indicates the frequency of transcript and unigene in each length interval.</p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/2.Assembly/2.AssemblyINFO" target="_blank">Result/2.Assembly/2.AssemblyINFO</a></p>
<br>
<!-------------------------------------------------------->


<h4>4.2.4 Assembly Assesment</h4>
<p class="paragraph">Busco (benchmarking universal single copy originals) evaluation is to evaluate the integrity of transcripts by using single copy ortholog library, tblastn, authorus and Hmmer software. We use busco software to analyze the splicing results Trinity.fasta ， unigene.fa and cluster.fasta Evaluate the splicing quality, and evaluate the accuracy and integrity of the splicing results according to the proportion and integrity of the comparison. For the introduction of busco, please refer to the following link: http://busco.ezlab.org/ .</p>

<p class="center">
<img class="center" src="src/images/Busco_plot.png"  width="50%/>
</p>

<p class="name_fig">Figure 4.6 Busco evaluation results of spliced transcripts (different colors represent different types of spliced transcripts)</p>

<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<br />
	<strong>S:</strong> Complete Single-Copy BUSCOs<br />
	<strong>D:</strong> Complete Duplicated BUSCOs<br />
	<strong>F:</strong> Fragmented BUSCOs<br />
	<strong>M:</strong> Missing BUSCOs<br />
	<strong>n:</strong> Total BUSCO groups searched<br />
</ul>
</div>

<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/2.Assembly/3.AssemblyAssesment" target="_blank">Result/2.Assembly/3.AssemblyAssesment</a></p>
<br>
<!-------------------------------------------------------->

<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the main parameters used in reconstruction?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The main parameters involved in reconstruction:<br>
	<br><strong>–SS_lib_type:</strong> The reconstruction parameter of strand specific library. It is used to distinguish the strand information of reads and enhance the accuracy of reconstruction.<br>
	<br><strong>–min_kmer_cov:</strong> Minimum kmer coverage, and its value is 3 by default in Novogene. This parameter will be used in k-mer filtering. If k-mer’s coverage minor than 3, it will be recognized as a low quality k-mer (Probably, this k-mer is caused by sequencing mistakes) and be discarded in reconstruction. If you focus on genes with low abundance, min_kmer_cov can be set as 2. <br>
	<br><strong>–min_glue:</strong> When contig aggregated to form components, supporting reads was needed: There is n reads spanning the junction area, and expanding (k-1)/2 bases(k is k-mer’s k value, k=31 in TRINITY, which means it expands 12bp in both sides) match. The min_glue is reads number n. In this case, if higher reconstruction quality is desired with reducing the number of produced reconstructed gene, we can modulate this parameter to achieve the goal.<br><br>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: In reconstruction, how to deal with strand specific library?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>For strand specific library, TRINITY distinguish the positive and negative strand by –SS_lib_type:<br>
	<br><strong>Pair-end strand specific library:</strong> A: RF=====Reverse Forward, First strand is Reverse, Second is Forward; B: FR=====Forward Reverse, First strand is Forward, second is Reverse;<br>
	<br><strong>Single-read strand specific library:</strong> R=====Reverse, strand is Reverse; B: F=====Forward, strand is Forward.<br><br>
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Will singlets sequence file and contig file be provided?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>According to TRINITY algorithm, the final reconstructed sequences is in TRINITY.fasta(see <strong>3.2.1 Transcriptome Reconstruction</strong> results), the singlets and contig file are not provided. Details: http://trinityrnaseq.github.io.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: How do we get unigene? What are the rationale?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Assembled transcripts by TRINITY in De novo projects are not true representatives of different isoforms, but arise from artifacts such as sequencing errors, repeats, variation in coverage or genetic variation within a diploid individual or pooled population. Hence, the number of contigs produced by a de novo assembly is typically large. Here Corset works by clustering contigs based on shared reads, but separates contigs when different expression patterns between samples are observed. Genes that share sequences, such as paralogues, are likely to have small distances between each other, resulting in many reads are shared. Further more, in order to get the best representation of a gene, Novogene will choose the longest transcript from Corset filtered contigs, which will be annotated and fed into following analysis. This method is widely used in RNA-seq analysis and also officially recommended by TRINITY.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why the same gene can generate different transcripts?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The transcripts from same gene could be the results of alternative splicing, allele, multiple copy of same gene, homolog, ortholog, etc., which make differences exist among these transcripts.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">
	<br><strong>k-mer:</strong> When you do the continuous splicing on reads with moving size k window, you will get a k length sequence each time, and k-mer are those sequences in length k. For read ATCGTT, k is 3, and you will get following k-mer: ATC,TCG,CGT,CTT. TRINITY k is 31.<br>
	<br><strong>k-mer Dictionary:</strong> It is used to store k-mer and its coverage, where “key” is similar to the index in the real dictionary, “value” is similar to the description in real dictionary. In this case, “key” represents k-mer, and “value” is k-mer’s coverage. “key” is unique, with corresponding unique value.<br>
	<br><strong>contig:</strong> In TRINITY reconstruction, in Inchworm step we will obtain the contig. According to TRINITY algorithm, different contig can not share more than k sequences, therefore, those Inchworm contigs cannot well present different alternative splicings and alleles, which need more process.<br>
	<br><strong>components:</strong> In TRINITY reconstruction, the contigs from Inchworm could be clustered and aggregated by its overlapping relationship, and form the components. They are a set of potential alternative splicing isoform or potential allele representation.<br>
	<br><strong>de Bruijn graph:</strong> In TRINITY reconstruction, in (chrysalis) step, components will be used to form the de Bruijn graph through overlapping relationship, which is used to acquiring the final result sequence and alternative splicing sequence.<br>
	<br><strong>Unigene:</strong> The longest transcript of each gene, was called as unigene, which is a basis for following analysis.<br>
	<br><strong>N50/N90:</strong> Ranking all transcripts by its length from high to low. Adding the length one by one from the first. When the sum is greater than 50%/90% of the total length, the last added sequence length is N50/N90.<br>
		
</table>
</div>
<br />
<br>





<!-------------------------------------------------------->
<h3 id="id_order_annotation">4.3  Gene Functional Annotation</h3>
<h4>4.3.1 Gene Functional Annotation</h4>
<p class="paragraph">To achieve comprehensive gene functional annotation, seven databases are applied by Novogene. The function and characteristics of the seven databases are as follows: <br /></p>
<p class="paragraph">Nr (NCBI non-redundant protein sequences): it is the formal protein sequence databases of NCBI, which includes protein sequence information from GenBank, PDB (Protein Data Bank), SwissProt, PIR (Protein Information Resource), PRF (Protein Research Foundation) etc.<br /></p>
<p class="paragraph">Nt (NCBI nucleotide sequences): it is the formal nucleotide sequence database of NCBI. It includes nucleotide sequence from GenBank, EMBL and DDBJ (but does not contain EST, STS, GSS, WGS, TSA, PAT, HTG).<br /></p>
<p class="paragraph">Pfam (Protein family)(Finn RD et al., 2008): it is the most comprehensive collection of protein domains and families, represented as multiple sequence alignments and as profile hidden Markov models. Many proteins are composed of structural domains while the protein sequence of a specific structural domain possess a certain degree of conservative property. In Pfam database, proteins are classified into different protein families according to their structural domains, and the HMM statistical model of each family's amino acid sequence is established by alignment of the protein sequences. According to the reliability of annotations, PFAM families are classified into two categories, Pfam-A and Pfam-B. Pfam-A family consists of a curated seed alignment containing a small set of representative members of the family, profile hidden Markov models (profile HMMs) built from the seed alignment and an automatically generated full alignment which contains all detectable protein sequences belonging to the family, as defined by profile HMM searches of primary sequence databases. Pfam-B entries are automatically generated from the ProDom database, and are represented by a single alignment. Through HMMER3 program, the established HMM model can be searched to annotate unigenes. More details: <a href='http://pfam.xfam.org' target=_blank>http://pfam.xfam.org</a>.<br /></p>
<p class="paragraph">
KOG/COG: Both COG (Cluster of Orthologous Groups of proteins) and KOG (euKaryotic Orthologous Groups) are based on NCBI's gene orthologous relationships. COG is specific to prokaryotes while KOG is specific to eukaryotes. According to their evolutionary relationships, COG/KOG divides the homologous genes from different species into different Ortholog clusters. The COG collection currently consists of 138,458 proteins, which form 4873 COGs and the current KOG set consists of 4852 clusters of orthologs, which include 59,838 proteins. As genes from the same ortholog own the same function, the functional annotation can be shared to the other members from the same COG/KOG clusters. More details could be found from the following website: <a href='http://www.ncbi.nlm.nih.gov/COG/' target=_blank>http://www.ncbi.nlm.nih.gov/COG/</a>.  <br /></p>
<p class="paragraph">
Swiss-Prot: A manually annotated and reviewed protein sequence database. It's a high quality protein sequence database, which brings together experimental results, computed features and scientific conclusions. More details could be found from the following website:  <a href="http://www.ebi.ac.uk/uniprot/" target=_blank>http://www.ebi.ac.uk/uniprot/</a>. <br /></p>
<p class="paragraph">
KEGG (Kyoto Encyclopedia of Genes and Genome): KEGG is a database resource for understanding high-level functions and utilities of the biological system, such as the cell, the organism and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by genome sequencing and other high-throughput experimental technologies. It contains KEGG PATHWAY, KEGG DRUG, KEGG DISEASE, KEGG MODULE, KEGG GENES, KEGG GENOME etc. And KO system (KEGG ORTHOLOG) combines each KEGG annotation system. KEGG has established a complete KO annotation system which can accomplish the function annotation of the genome/transcriptome of a newly sequenced species. More details could be found from the following website:  <a href="http://www.genome.jp/kegg/" target=_blank>http://www.genome.jp/kegg/ </a>.<br /></p>
<p class="paragraph">GO (Gene Ontology): GO is the established standard for the functional annotation of gene products. GO vocabulary is a controlled vocabulary used to classify the following functional attributes of gene products: Biological Process (BP), Molecular Function (MF) and Cellular Component (CC). GO term is the basic unit of GO system. Each term has a unique identifier. The relationship between the GO term of each ontology can form a Directed Acyclic Topology. More details could be found from the website:  <a href="http://www.geneontology.org/" target=_blank>http://www.geneontology.org/</a>. <br /></p>
<p class="paragraph">The software and parameters used in each database are as follows: </p>
<p class="paragraph">NT: NCBI blast(Altschul SF et al., 1997). The evalue threshold is 1e-5 (Each unigene shows top10 alignment results).</p>
<p class="paragraph">NR, SwissProt, KOG: Diamond(Buchfink B et al., 2015. The evalue threshold is 1e-5 (Each unigene shows top10 alignment results).</p>
<p class="paragraph">PFAM (The prediction of protein structure domain): HMMER package,hmmscan. The evalue threshold is 0.01;</p>
<p class="paragraph">GO (Based on the protein annotation results of NR and Pfam): Blast2GO(Götz et al., 2008) and novogene script. The evalue threshold is 1e-6;</p>
<p class="paragraph">KEGG: KAAS(Moriya Y et al., 2007), KEGG Automatic Annotation Server. The evalue threshold is 1e-5.</p>


<p class="paragraph">The statistics of successfully annotated genes by each database are shown in <strong>Table 4.4 </strong></p>
<p class="center">Table 4.4  The Ratio of Successfully Annotated Genes </p>
<table class="gy">
<tr><th>Statistical Items</th><th>Number of Unigenes</th><th>Percentage (%)</th></tr>

	<tr >
	<td>Annotated in NR</td><td>41119</td><td>27.85</td>
	</tr>

	<tr >
	<td>Annotated in NT</td><td>28694</td><td>19.43</td>
	</tr>

	<tr >
	<td>Annotated in KO</td><td>19440</td><td>13.16</td>
	</tr>

	<tr >
	<td>Annotated in SwissProt</td><td>5016</td><td>3.39</td>
	</tr>

	<tr >
	<td>Annotated in PFAM</td><td>35538</td><td>24.07</td>
	</tr>

	<tr >
	<td>Annotated in GO</td><td>35535</td><td>24.07</td>
	</tr>

	<tr >
	<td>Annotated in KOG</td><td>15366</td><td>10.4</td>
	</tr>

	<tr >
	<td>Annotated in all Databases</td><td>1012</td><td>0.68</td>
	</tr>

	<tr >
	<td>Annotated in at least one Database</td><td>63750</td><td>43.18</td>
	</tr>

	<tr >
	<td>Total Unigenes</td><td>147615</td><td>100</td>
	</tr>

</table>
<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<strong>Statistical Items:</strong> Annotated Database Type<br>
<strong>Number of Unigenes:</strong> The number of genes successfully annotated in each Database.<br>
<strong>Percentage:</strong> The percentage of genes successfully annotated in Database in total unigene number<br>
</ul>
</div>
<br>
<p class="paragraph">The venn diagram is mapped with 5 selected database annotation result from 7 database results:</p>
<p class="center">
<img class="center" src="src/images/Venn_diagram.png" height="500" width="500"/>
</p>
<p class="name_fig">Figure 4.7  The Venn Diagram is Mapped with Database Annotation </p>
<br />
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/3.Annotation/1.GeneFunctionalAnnotation/Annotation.statistics.xls" target="_blank">Result/3.Annotation/1.GeneFunctionalAnnotation/Annotation.statistics.xls</a></p>
<br>
<!-------------------------------------------------------->
<h4><a>4.3.2 Gene Annotation Results through Nr Database</a></h4>
<p class="center">Table 4.5  Part of Gene Annotation Results through Nr Database</p>
<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>Genes</th><th>Gene Length</th><th>NR GI</th><th>NR ID</th><th>NR Score</th><th>NR Evalue</th><th>NR Description</th></tr>
					
			  
					
							<tr><td>Cluster-23194.0</td><td>366</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.103206</td><td>690</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.108386</td><td>313</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.84315</td><td>375</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-28564.0</td><td>303</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
			</table>
	</div>
</p>
<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
<strong>Gene ID:</strong> The gene ID from Corset results.<br>
<strong>Gene Length:</strong> The longest length generated by the gene transcript.<br>
<strong>NR GI:</strong> The GenBank ID of the annotated nucleotide.<br>
<strong>NR ID:</strong> The NR ID of the annotated nucleotide.<br>
<strong>NR Score:</strong> Alignment score based on a specific score matrix.<br>
<strong>NR E-value:</strong> Expected value calculated according to the score, query sequence's length and library size. Evalue essentially represents the false positive rate, the smaller is the better.<br>
<strong>NR Description:</strong> NR function note description.<br>
</ul>
</div>
<p class="paragraph">The species distribution, E-value distribution and similarity distribution plots are as follows: </p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_Distribution_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_Distribution_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_Distribution_pane_Species_Classification" 
	role="tabpanel" aria-labelledby="gallery_figure_Distribution_list_Species_Classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Species_classification.3dpie.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Distribution_pane_E-value_Distribution" 
	role="tabpanel" aria-labelledby="gallery_figure_Distribution_list_E-value_Distribution"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Evalue_Distribution.3dpie.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Distribution_pane_Similarity_Distribution" 
	role="tabpanel" aria-labelledby="gallery_figure_Distribution_list_Similarity_Distribution"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Similarity_Distribution.3dpie.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_Distribution_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_Distribution_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_Distribution_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Distribution_list_Species_Classification"
	data-toggle="list" href="#gallery_figure_Distribution_pane_Species_Classification" role="tab" aria-controls="Species_Classification">Species_Classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Distribution_list_E-value_Distribution"
	data-toggle="list" href="#gallery_figure_Distribution_pane_E-value_Distribution" role="tab" aria-controls="E-value_Distribution">E-value_Distribution</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Distribution_list_Similarity_Distribution"
	data-toggle="list" href="#gallery_figure_Distribution_pane_Similarity_Distribution" role="tab" aria-controls="Similarity_Distribution">Similarity_Distribution</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.8 Nr Distribution</p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/3.Annotation/1.GeneFunctionalAnnotation" target="_blank">Result/3.Annotation/1.GeneFunctionalAnnotation</a></p>
<br>
<!-------------------------------------------------------->
<h4><a>4.3.3 GO Classification</a></h4>
<p class="paragraph">After GO annotation, the successfully annotated genes will be grouped into three main GO domains: Biological Process (BP), Cellular Component (CC), Molecular Function (MF).</p>
<p class="center">
<img class="wid" src="src/images/GO_classification_bar.png" alt='GO_classification_bar.png'/>
</p>
<p class="name_fig">Figure 4.9 GO Classification</p>
<p class="premark center"> From the top to the bottom, we are showing blasting results: 1) species distribution (long scientific name is represented as “…” ); 2) E-value distribution; 3) similarity distribution </p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/3.Annotation/2.GOclassification" target="_blank">Result/3.Annotation/2.GOclassification</a></p>
<br>

<!----------------------------------------- KOG分类 -------------------------------------->

<h4><a>4.3.4 KOG Classification</a></h4>
<p class="paragraph">KOG is divided into groups. Figure 4.10 shows the classifications of the genes successfully annotated in KOG.</p>
<p class="center">
<img class="center" src="src/images/KOG_classification.png" height="550" width="750"/>
</p>
<p class="name_fig">Figure 4.10 KOG Classification</p>
<p class="premark center">The X-axis is the names of KOG groups, and the Y-axis is the percentage of genes annotated under this group in the total annotated genes.</p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/3.Annotation/3.KOGclassification" target="_blank">Result/3.Annotation/3.KOGclassification</a></p>
<br>
<!----------------------------------------- KEGG分类 -------------------------------------->

<h4><a>4.3.5 KEGG Classification</a></h4>
<p class="paragraph">The genes successfully annotated in KEGG can be classified according to the KEGG pathway they joined in.</p>
<p class="center">
<img class="small3" src="src/images/KEGG_classification.png" />
</p>
<p class="name_fig">Figure 4.11 KEGG Classification</p>
<p class="premark center">The Y-axis is the names of KEGG pathways, and the X-axis is the number of the genes annotated in the pathway and the ratio between the number in this pathway and the total number of annotated genes. The KEGG metabolic pathways gene involved in are divided into 5 branches: C: Cellular Processes, E: Environmental Information Processing, G: Genetic Information Processing, M: Metabolism, O: Organismal Systems.2.4.3 Correlation Analysis. In addition, B: Brite Hierarchies, N: Not Included in Pathway or Brite.</p>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/3.Annotation/4.KEGGclassification" target="_blank">Result/3.Annotation/4.KEGGclassification</a></p>


<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
    <tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the main reasons for low annotation rate?</td></tr>
    <tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Firstly, part of transcriptome data are none-coding; Secondly, annotation rate has a relationship with the species and the annotation database used in annotation step, better annotation in the database will bring better annotation result, usally, the annotation rate will be higher. Finally, the annotation result is the result of mapping the total unigene to the sequence in anntation database, which is highly objective, although more annotated genes will benefit following research.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Is the annotation of Gene ontology using Nr top 10 in blast2go? or it has combining it with the data of interpro scan?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The annotation of Gene ontology not only use Nr top10 through blast2go, but also combine the pfam data in interpro scan.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: How is the annotation accurary of 7 databases?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>We use 7 main stream database to do the gene functional annotation. The selection of annotation is controlled (For example, blast Evalue<1e-5). Additionally, all the software of annotation are officially recommended (blast2go,KAAS, etc.). Therefore, the annotation from each database should be reliable. Also, different type annotation have different focus(For example, PFAM focus on protein structure domain, KEGG focus on metabolic pathway, detailed description could be found in readme file of final report).<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the paramters of annotation through different databases?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>see 3.4.1 Gene Functional Annotation.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why do not annotate each and every transcripts?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Different transcripts from same gene could be the results of alternative splicing, allele, multiple copy of same gene, homolog, ortholog and etc, which make differences exist in these transcripts. However, it will be redunct and challenging to annotate all these transcript, usually, the longest transcprit could represent these transcripts. Thus, Novogene choose the longest one in Trinity.Corset.fasta as unigene to do annotation. However, the reference sequence for quantitative analysis is Trinity.Corset.fasta. This method is the main stream method in RNA-seq analysis, also recommended by Trinity. Following differential expression analyis and enrichment analysis are based on gene level, thus the annotation is focusing on gene level.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Species catogory in kegg?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>To be simplified, for the limitation of usage of KEGG and GENES database, the phenomenon that plant gene is divided into immunology or other unreasonale term will occur. In annotation of the unigene, we use KAAS. Basically, it apply blast to unigene and the gene in database(KEGG, GENES) to find the mapping of query seq and reference seq. Accoring to the annotation information in the KEGG GENES database divided the mapped reference seq to several KO group, and scoring each K number in the group. According to the scores, it allocates a K number to each query seq, and used the K number to get the funcional and pathway information. For KEGG, GENES databases do not have enough plant species, the annotation information for plant is much less than animals, which is the main reason for wrong annotation term. If needed, we can filtered unwanted term.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Diamond software</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Diamond is a software which is based on blast and optimizes it (blastx and blastp). Compared with blast, it could keep a same rate of 90%-99% in short sequence alignment, but greatly improves the mapping speed.<br><br>		
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">
	<br><strong>Evalue:</strong>  Evalue represents the possibility that two sequences is unrelated, which is used to evaluate the similarity of two sequences. Lower Evlaue means higher similarity.<br>
	<br><strong>best hit:</strong> One gene that has the best matching rate to a specific gene in the target genome.<br>
	<br><strong>Similarity:</strong> The level of similarity between given sequences.<br>
	<br><strong>Homology:</strong> The sequences evolved from a common ancester. When two protein have a common ancester in evolution, we think they are homologous.<br>
</table>
</div>
<br>
<br>


<!---------------------------------------- CDS 预测 ------------------------------------>


<h3 id="id_order_cds">4.4 CDS Prediction </h3>
<h4><a>4.4.1 CDS Prediction</a></h4>
<p class="paragraph">CDS prediction can be divided into two steps: 1. Using BLAST to align unigene sequences to databases of NR and Swissprot. If matched, CDS shall be extracted from in the unigene sequences and translated into peptide sequences based on the standard codon table (from 5’ to 3’). 2. Unigenes with no hits while blasting would be analyzed by TransDecoder(3.0.1) to predict their coding regions and determine their sequence direction. Part of results are shown as follows:<br>

<p class="paragraph">CDS extracted from BLAST:</p>
<div class="bs-docs-qa">
<p class='seq'>>Cluster-24312.0;orf1 len=396  frame:1  start:1  end:396  probable glutamate receptor [Homarus americanus]<br/>AAAGCCGTGTCATCCTTCGGTAAGGGCAAGCCCATCTCCTTCAGTGAGTCGATGTTCTAC<br/>AGTTGGGCTGTGATGCTGGAAGATCCTCCTATGCAAACCCCTTCCAGTGTCTCTGGCCAG<br/>ATGATGGTCGGATGGTGGCTGGTCGCGAGCCTGTTGATTTCCACCGGCTTCAGGTCGTCT<br/>CTGGTGGCTCACATCACAGTCCCAGGGAAGACGAAACCATTAAACAGCTTCACAGACATG<br/>ATCAAACGACCAAATTACCGCTGGGGCATTGATTCCCGGGTTCATGGGTTTGCCATAAGC<br/>ATTTTTGGCAATAATCCAGACCCAGTCGTCCAGTACGTCGGTAAATCCCTAGAGGTCTCA<br/>AATCTGGAAGACGGATTGAAGAGGACAAGCACCGGA<br/>>Cluster-1212.0;orf1 len=240  frame:2  start:2  end:241  kinesin motor domain protein [Ichthyophthirius multifiliis]||EGR28887.1 kinesin motor domain protein [Ichthyophthirius multifiliis]<br/>CTTGAGCTCTATAATGAAGAAGTCGTCGATCTGCTCTCCGACAACAGAAAAAACAAGCTG<br/>CAGATCCATGAAGATAAAGATAAGGGCGTCTATGTGAAAGATTTGAGTGCTTATCCAGTA<br/>AAAACTGTGGAGGATATGAATAATAAGCTTCGAGAGGGCAGTCAAAACCGGCATGTGGGC<br/>TCCACGGGAATGAACAATGTCTCTTCTCGATCACATTCTATCTTTATGCTGAGAATATAA<br/></p>
</div>
<p class="premark">Header information interpretation:</p>
<p class="premark">>Cluster-X.Y: the sequence ID, the only identifier for the gene;</p>
<p class="premark">orf1: pridicted ORF id, one gene may have two or more pridicted ORFs;</p>
<p class="premark">len: the base length of this ORF; </p>
<p class="premark">frame: the reading frame of this ORF in the original gene, '-' the minus strand; </p>
<p class="premark">start: the start position of this ORF in the original gene; </p>
<p class="premark">end: the end position of this ORF in the original gene; </p>
<p class="premark">gi|***|ref|XP_***|: descriptions of the gene blasted protein.</p>

<p class="paragraph">CDS predicted by TransDecoder </p>
<div class="bs-docs-qa">
<p class='seq'>>Gene.9621::Cluster-10.0::g.9621::m.9621 Gene.9621::Cluster-10.0::g.9621  ORF type:internal len:114 (+) Cluster-10.0:1-339(+)<br/>GTAAACGGTAACGATGATTTGCACCCTGAAGGGCCAGAGGGGAATACTGGGTTCGTTGAT<br/>GCGCTTAAGGAAGTACAAAGTAAATATGGAATTATCAGTGAAGAGGACAATGTGGATGAT<br/>AAGGAGGAAGAGGAACAAGGTACTTCGAATGGAAGAGATGGACTTGGTCATACAGATACC<br/>GAAGACAAAGATGAAGAAGATGAAGACTACTGGGAAGGAAAAGATTATGATTTCTGGAAC<br/>AGAGATAACGATGGCAGTGAAGAATACTGGGCAAGGTGGGACGAAAAGGTTGATACTTTA<br/>TGGGATGATGATTACTCAGAATATGATTACGAAAGCTCT<br/>>Gene.12867::Cluster-1000.0::g.12867::m.12867 Gene.12867::Cluster-1000.0::g.12867  ORF type:internal len:102 (-) Cluster-1000.0:3-305(-)<br/>GACAACGACGTCTTAAATCGCACGATTAGGACTGATAAAATATTCACATTTGAAGTCGGG<br/>CATTGCCGATATCAGAATGTGGAGAATTTTCAAAGGAATAACTGCGATGATGAAGATCTT<br/>CAGGTTAAGATCTCTAATACAATATGTAAGGTTATCGAGGGCAAGCTCGTAAAAGCTACG<br/>GAAATAGTCTTTAGGGATAAGACCATTACGATCGAAGACACCTACAATTTTGTCCCTAGC<br/>AATGGCCCCAGCCTGGTTGAAAACGGCAACGGGTCTTCAAAAGGCTCAAACGATGGCTCT<br/>ATA<br/></p>
</div>
<br>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/4.CDSprediction" target="_blank">Result/4.CDSprediction</a></p>
<br>
<!----------------------------------------- CDS预测Q&A -------------------------------------->
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why does functional annotation use 7 databases, while CDS prediction only use NR and SwissProt?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>NR and Swissprot are protein database. Based on the coding of real protein, coding region in the reconstructed gene can be predict.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: While alignment to the protein database, if the mapped species is far from the target species, how will the result be affected?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>It’s an objective result. If the percentage is not very high, the result wouldn’t be significantly affected.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why some unigenes do not have nucleotide sequence or amino acid sequence?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Due to the limitation of prediction, it cannot guarantee 100% of inputs have corresponding prediction results. In addition, the non-coding may have ployA and can be detected, these non-coding sequences exist in the result.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the parameters for BLAST?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>IFor BLAST, we choose Evalue=1e-5.<br><br>
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why doesn’t the results of transcriptome reconstruction and CDS prediction start with ATG?</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Firstly, RNA-seq could include part of UTR or part of non-coding sequence. Thus, the initiate sequences of transcripts could be others instead of ATG. Secondly, since the transcripts are assembled by reads, it could be part of coding area instead of whole coding area, which is similar to the result of CDS prediction. Thus, it is reasonable that the initiation is not ATG.<br><br>		
	<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
	<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39">
	<br><strong>ORF:</strong> open reading frame, a complete polypeptide could be translated from start codon to end codon in reading frame.<br>
	<br><strong>CDS:</strong> CDS (coding sequence) is used to code protein, which is part of mRNA.<br>
</table>
</div>
<br>
<br>



<!-----------------------------------------转录因子注释------------------------------------>



<h3><a>4.5 Transcription Factor Analysis</a></h3>
<p class="paragraph">iTAK is used to perform the transcription factor analysis of plants. It uses hmmerscan to identify the TF(transcription factor) by predifined TF families and rules in the database. The identify and classify method is published by Perez-Rodriguez et al (2010) and Jin et al (2014). AnimalTFDB database is used to perform the transcription factor analysis of animals.</p>
<p class="center">Table 4.6  Transcription Factor Analysis Results</p>
<table class="gy">
<tr><th>tf_id</th><th>Subfamily</th><th>Family</th></tr>
<!-- <tr><th>Gene ID</th><th>Subfamily</th><th>Family</th></tr> -->

<tr>
<td>Cluster-0.0-0R</td><td>MYB-related</td><td>MYB->MYB-related</td>
</tr>

<tr>
<td>Cluster-10289.10299-1F</td><td>bZIP</td><td>bZIP</td>
</tr>

<tr>
<td>Cluster-10289.10672-0R</td><td>bZIP</td><td>bZIP</td>
</tr>

<tr>
<td>Cluster-10289.10686-2R</td><td>GRAS</td><td>GRAS</td>
</tr>

</table>
<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<strong>geneID:</strong> the ID of genes.<br />
	<strong>Subfamily:</strong> the subfamily of the transcription factors(only for plant).<br />
	<strong>family:</strong> the family name of the transcription factors.<br />
</ul>
</div>
<br/>
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/5.TransFactor" target="_blank">Result/5.TransFactor</a></p>
<br>



<!---------------------------------------- 基因表达水平分析 --------------------------------------->


<h3 id="id_order_rsem">4.6 Gene Expression Analysis</h3>

<h4><a>4.6.1 Reference Alignment</a></h4>
<p class="paragraph">De novo transcriptome filtered by Corset is used as a reference(ref). RSEM(Li B et al., 2011) will map reads back to transcriptome and quantify the expression level. The summary of mapping results are shown as follows: </p>

<p class="center">Table 4.7 Overview of the Alignment Situation</p>
<table class="gy">
<tr><th>Sample Name</th><th>Total Reads</th><th>Total Mapped</th></tr>

	<tr>
	<td>B1</td><td>22765127</td><td>16219376(71.25%)</td>
	</tr>

	<tr>
	<td>B2</td><td>23131423</td><td>16556772(71.58%)</td>
	</tr>

	<tr>
	<td>B3</td><td>23409419</td><td>17490624(74.72%)</td>
	</tr>

	<tr>
	<td>G1</td><td>23398301</td><td>16789867(71.76%)</td>
	</tr>

	<tr>
	<td>G2</td><td>23234226</td><td>16307454(70.19%)</td>
	</tr>

	<tr>
	<td>G3</td><td>22002582</td><td>15557858(70.71%)</td>
	</tr>

	<tr>
	<td>EN1</td><td>23566863</td><td>18013374(76.44%)</td>
	</tr>

	<tr>
	<td>EN2</td><td>21778080</td><td>15552753(71.41%)</td>
	</tr>

	<tr>
	<td>EN3</td><td>20888698</td><td>14529683(69.56%)</td>
	</tr>

	<tr>
	<td>LN1</td><td>21741234</td><td>15500684(71.30%)</td>
	</tr>

	<tr>
	<td>LN2</td><td>22064843</td><td>15892983(72.03%)</td>
	</tr>

	<tr>
	<td>LN3</td><td>23182117</td><td>16780273(72.38%)</td>
	</tr>

	<tr>
	<td>EZ1</td><td>21736602</td><td>15685503(72.16%)</td>
	</tr>

	<tr>
	<td>EZ2</td><td>22583325</td><td>16203271(71.75%)</td>
	</tr>

	<tr>
	<td>EZ3</td><td>23169911</td><td>16467324(71.07%)</td>
	</tr>

	<tr>
	<td>LZ1</td><td>22808177</td><td>17332629(75.99%)</td>
	</tr>

	<tr>
	<td>LZ2</td><td>22114204</td><td>17055843(77.13%)</td>
	</tr>

	<tr>
	<td>LZ3</td><td>23500139</td><td>16235274(69.09%)</td>
	</tr>

</table>
<br>
<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<strong>Sample Name:</strong> Sample name.<br />
	<strong>Total Reads:</strong> Clean reads number.<br />
	<strong>Total Mapped:</strong> Total number of reads that can be mapped to the reference genome.<br />
</ul>
</div>
<br />
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/5.Quant/1.Count" target="_blank">Result/5.Quant/1.Count</a></p>
<br>
<!-------------------------------------------------------->

<h4><a>4.6.2 Summary of Gene Expression Levels</a></h4>
<p class="paragraph">To calculate the gene expression level, RSEM analysed the mapping results of Bowtie, and then got the read count for each gene of each sample. Furthermore, converted them into FPKM value. In RNA-seq, FPKM, short for the expected number of Fragments Per Kilobase of transcript sequence per Millions base pairssequenced, is the most common method of estimating gene expression levels, which takes into account the effects of both sequencing depth and gene length oncounting of fragments(Trapnell C et al., 2010).</p>

<p class="center">Table 4.8 Gene Expression Summary</p>
<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>gene_id</th><th>B1</th><th>B2</th><th>B3</th><th>G1</th><th>G2</th><th>G3</th><th>EN1</th><th>EN2</th><th>EN3</th><th>LN1</th><th>LN2</th><th>LN3</th><th>EZ1</th><th>EZ2</th><th>EZ3</th><th>LZ1</th><th>LZ2</th><th>LZ3</th><th>Gene Length</th><th>NR GI</th><th>NR ID</th><th>NR Score</th><th>NR Evalue</th><th>NR Description</th><th>NT GI</th><th>NT ID</th><th>NT Score</th><th>NT Evalue</th><th>NT Description</th><th>KO ID</th><th>KO Name</th><th>KO Description</th><th>Http Link</th><th>Swissprot ID</th><th>Swissprot Score</th><th>Swissprot Evalue</th><th>Swissprot Description</th><th>PFAM ID</th><th>PFAM description</th><th>Gene Ontology Biological Pathway</th><th>BP Description</th><th>Gene Ontology Molecular Function</th><th>MF Description</th><th>Gene Ontology Cellular Component</th><th>CC Description</th><th>KOG ID</th><th>KOG Description</th></tr>
					
			  
					
							<tr><td>Cluster-23194.0</td><td>0.00</td><td>0.00</td><td>5.00</td><td>2.00</td><td>4.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>3.00</td><td>366</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-22704.0</td><td>0.00</td><td>0.00</td><td>11.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>441</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-28564.0</td><td>2.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>15.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>303</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.108384</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>3.00</td><td>0.00</td><td>7.00</td><td>3.00</td><td>421</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-24312.0</td><td>0.00</td><td>2.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>1.00</td><td>1.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>396</td><td>XP_045588862.1</td><td></td><td>299</td><td>1.41e-31</td><td>uncharacterized protein LOC123750795 [Procambarus clarkii]</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>PF00060</td><td>Ligand-gated ion channel</td><td>GO:0006811</td><td>monoatomic ion transport</td><td>GO:0015276</td><td>ligand-gated monoatomic ion channel activity</td><td>GO:0016020</td><td>membrane</td><td>--</td><td>--</td></tr>
					
			  
			</table>
	</div>
	</p>

	<button onclick="hiddenShow(this)">Show Annotation</button>
	<div class="comment">
	<ul style="font-size: 12px;font-family:line-height: 20px">
		<strong>gene_id:</strong> Gene ID.<br />
		<strong>Sample Name:</strong> The read count value of each sample.<br />
	</ul>	
	</div>
<br />
<br>

<p class="center">Table 4.9  FPKM Result</p>
<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>gene_id</th><th>B1</th><th>B2</th><th>B3</th><th>G1</th><th>G2</th><th>G3</th><th>EN1</th><th>EN2</th><th>EN3</th><th>LN1</th><th>LN2</th><th>LN3</th><th>EZ1</th><th>EZ2</th><th>EZ3</th><th>LZ1</th><th>LZ2</th><th>LZ3</th><th>Gene Length</th><th>NR GI</th><th>NR ID</th><th>NR Score</th><th>NR Evalue</th><th>NR Description</th><th>NT GI</th><th>NT ID</th><th>NT Score</th><th>NT Evalue</th><th>NT Description</th><th>KO ID</th><th>KO Name</th><th>KO Description</th><th>Http Link</th><th>Swissprot ID</th><th>Swissprot Score</th><th>Swissprot Evalue</th><th>Swissprot Description</th><th>PFAM ID</th><th>PFAM description</th><th>Gene Ontology Biological Pathway</th><th>BP Description</th><th>Gene Ontology Molecular Function</th><th>MF Description</th><th>Gene Ontology Cellular Component</th><th>CC Description</th><th>KOG ID</th><th>KOG Description</th></tr>
					
			  
					
							<tr><td>Cluster-23194.0</td><td>0.00</td><td>0.00</td><td>4.59</td><td>1.83</td><td>3.66</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>2.54</td><td>366</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-22704.0</td><td>0.00</td><td>0.00</td><td>4.78</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>441</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-28564.0</td><td>5.53</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>13.78</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>303</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.108384</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>1.55</td><td>0.00</td><td>2.95</td><td>1.49</td><td>421</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-24312.0</td><td>0.00</td><td>1.44</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.64</td><td>0.65</td><td>0.00</td><td>0.00</td><td>0.00</td><td>396</td><td>XP_045588862.1</td><td></td><td>299</td><td>1.41e-31</td><td>uncharacterized protein LOC123750795 [Procambarus clarkii]</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>PF00060</td><td>Ligand-gated ion channel</td><td>GO:0006811</td><td>monoatomic ion transport</td><td>GO:0015276</td><td>ligand-gated monoatomic ion channel activity</td><td>GO:0016020</td><td>membrane</td><td>--</td><td>--</td></tr>
					
			  
			</table>
	</div>
	</p>

	<button onclick="hiddenShow(this)">Show Annotation</button>
	<div class="comment">
	<ul style="font-size: 12px;font-family:line-height: 20px">
		<strong>gene_id:</strong> Gene ID.<br />
		<strong>Sample Name:</strong> The read count value of each sample.<br />
	</ul>	
	</div>
<br />
<br>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/5.Quant/1.Count" target="_blank">Result/5.Quant/1.Count</a></p>
<br>

<!-------------------------------------------------------->

<h4><a>4.6.3 Comparison of Gene Expression Levels</a></h4>
<p class="paragraph">To compare gene expression levels under different conditions, an FPKM distribution diagram and box plot are used. For biological replicates, the final FPKM would be the mean value.</p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_FPKM_Distribution_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_FPKM_Distribution_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_FPKM_Distribution_pane_FPKM_density" 
	role="tabpanel" aria-labelledby="gallery_figure_FPKM_Distribution_list_FPKM_density"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/FPKM_density.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_FPKM_Distribution_pane_FPKM_boxplot" 
	role="tabpanel" aria-labelledby="gallery_figure_FPKM_Distribution_list_FPKM_boxplot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/FPKM_boxplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_FPKM_Distribution_pane_FPKM_violin" 
	role="tabpanel" aria-labelledby="gallery_figure_FPKM_Distribution_list_FPKM_violin"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/FPKM_violin.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_FPKM_Distribution_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_FPKM_Distribution_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_FPKM_Distribution_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_FPKM_Distribution_list_FPKM_density"
	data-toggle="list" href="#gallery_figure_FPKM_Distribution_pane_FPKM_density" role="tab" aria-controls="FPKM_density">FPKM_density</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_FPKM_Distribution_list_FPKM_boxplot"
	data-toggle="list" href="#gallery_figure_FPKM_Distribution_pane_FPKM_boxplot" role="tab" aria-controls="FPKM_boxplot">FPKM_boxplot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_FPKM_Distribution_list_FPKM_violin"
	data-toggle="list" href="#gallery_figure_FPKM_Distribution_pane_FPKM_violin" role="tab" aria-controls="FPKM_violin">FPKM_violin</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.12 Different Gene Expression Levels under Different Experimental Conditions</p>
<br>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/5.Quant/2.Distribution" target="_blank">Result/5.Quant/2.Distribution</a></p>
<br>


<!----------------------------------------RNA-seq相关性--------------------------------------->


<h4 id="id_order_rnaseqqc">4.6.4 Correlation Analysis </h4>
<p class="paragraph">Biological replicates are necessary for any biological experiment, including those involving RNA-seq technology (Hansen et al., 2010). In RNA-seq, replicates have a two-fold purpose. First, they demonstrate whether the experiment is repeatable, and secondly, they can reveal differences in gene expression between samples. The correlation between samples is an important indicator for testing the reliability of the experiment. The closer the correlation coefficient is to 1, the greater the similarity of the samples. ENCODE suggests that the square of the Pearson correlation coefficient should be larger than 0.92, under ideal experimental conditions(ENCODE Project Consortium, 2004). 
</p>

<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_SampleCorrelation_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_SampleCorrelation_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_SampleCorrelation_pane_Pearson_correlation" 
	role="tabpanel" aria-labelledby="gallery_figure_SampleCorrelation_list_Pearson_correlation"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/cor_pearson.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_SampleCorrelation_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_SampleCorrelation_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_SampleCorrelation_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_SampleCorrelation_list_Pearson_correlation"
	data-toggle="list" href="#gallery_figure_SampleCorrelation_pane_Pearson_correlation" role="tab" aria-controls="Pearson_correlation">Pearson_correlation</a>
	
	</div>
	</div>
	</div>
	</p>
	<p class="name_fig">Figure 4.13 Sample Correlation</p>
	<p class="center">If the samples are more than 4 groups, then only present the scatter diagrams between biological replicates The scatter diagrams demonstrate the correlation coefficient between samples; R<sup>2</sup>, the square of the Pearson coefficient. Heat maps of the correlation coefficient between samples are also shown.</p>
<br>





<!----------------------------------------- 基因表达水平分析Q&A -------------------------------------->

<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the differences between FPKM and RPKM?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Both FPKM and RPKM are used in gene expression level normalization. FPKM will consider the fragment difference in the two read of paired end read, while RPKM consider reads: rather than using read counts, approximates the relative abundance of transcripts in terms of fragments observed from an RNA-Seq experiment, which may not be represented by a single read, such as in paired-end RNA-Seq experiments.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the gene expression threshold? what is the rationale?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>In bioinformatic analysis without genome reference, we think the gene with FPKM>0.3 is expressed. This threshold is recommend by many prestigious journals and can well reflect the gene expression to large extent.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: In TRINITY reconstruction result, what is the reason the unmapped case appear and the case that readcount and FPKM are 0?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The main reason are listed as following: <br><br>
<p class="premark">1: Trinity won’t use all the clean reads.</p>
<p class="premark">2: It is related to the mismatch parameter(TRINITY reconstruction allow more mismatch than RSEM. For this reason, the reconstruction result may not be matched in alignment process.</p>
<p class="premark">3: The mapping rate in table means the percentage of two end mapping. If only one end is mapped, the read is not be counted (For PE sequencing).</p></td></tr>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Are the transcripts included in gene expression analysis, besides the unigene?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>We use the software RSEM to do analysis, it is compatible with Trinity and can do the quantification on transcripts as well as unigene. Although Novogene do the quantification on unigene and Transcripts in the same time, the following analysis is based on unigene and the result provides the unigene expression level.<br><br>   
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What dose the sample correlation mean? What is the calculation method?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The sample correlation represents the similarity among samples. The sample correlation could help us examine the similarity at gene expression level. Higher correlation means higher similarity, and less number of differential express genes. Generally, the correlation between biological replicates should be higher than the correlation of samples with different sources. There are three different calculation methods: A. Pearson correlation; B. Spearman rank correlation; C. Kendall’s τ. Novogene uses R language to calculate the Pearson correlation coefficient.<br><br> 
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br><strong>FPKM:</strong> expected number of Fragments Per Kilobase of transcript sequence per Millions base pairs sequenced <br>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br><strong>RPKM:</strong> expected number of reads Per Kilobase of transcript sequence per Millions base pairs sequenced<br></td></tr>
</table>
</div>
<br />
<br>

<!---------------------------------------- 差异表达分析 ------------------------------------->


<h3 id="id_order_diffexp">4.7  Differential Expression Analysis </h3>

<p class="paragraph">The input data for differential gene expression analysis are readcounts from gene expression level analysis. The differential gene expression analysis contains three steps: </p>
<p class="paragraph">1)Readcounts Normalization;</p>
<p class="paragraph">2)Model dependent p-value estimation; </p>
<p class="paragraph">3)FDR value estimation based on multiple hypothesis testing.</p>
<p class="paragraph">Different softwares and parameter sets are applied in different situations. The analysis methods are listed below:</p>
<div style="width:98%">
<table class="gy2">
<tr><th>Type</th><th>Software</th><th>Normalzation method</th><th>p-value estimation model</th><th>FDR estimation method</th><th>Differentially expressed gene screening stardard</th></tr>
<tr><td>With biological duplicates</td><td>DESeq2(Love MI et al., 2014)</td><td>DESeq2</td><td>negative binomial distribution</td><td>BH</td><td>|log2(FoldChange)| >= 1&padj <= 0.05</td></tr>
<tr><td>Without biological duplicate</td><td>edgeR(Robinson MD et al., 2010)</td><td>TMM</td><td>negative binomial distribution</td><td>BH</td><td>|log2(FoldChange)| >= 1& padj <= 0.005</td></tr>
</table>
</div>
<br>
<p class="paragraph">The readcount value of the ith gene in the jth sample is K<sub>ij</sub>, then</p>
<p class="center">Negative binomial distribution: K<sub>ij</sub> ~ NB(μ<sub>ij</sub>,σ<sub>ij</sub><sup>2</sup>)</p>
<p class="center">Poisson distribution: K<sub>ij</sub> ～ P(μ<sub>ij</sub>)</p>

<h4><a>4.7.1 List of Differentially Expressed Genes</a></h4>
<p class="paragraph">The list of differentially expressed genes is as follows:</p>
<p class="center">Table 4.10 Differentially Expressed Genes</p>

<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>gene_id</th><th>B_readcount</th><th>G_readcount</th><th>log2FoldChange</th><th>pvalue</th><th>padj</th><th>Gene Length</th><th>NR GI</th><th>NR ID</th><th>NR Score</th><th>NR Evalue</th><th>NR Description</th><th>NT GI</th><th>NT ID</th><th>NT Score</th><th>NT Evalue</th><th>NT Description</th><th>KO ID</th><th>KO Name</th><th>KO Description</th><th>Http Link</th><th>Swissprot ID</th><th>Swissprot Score</th><th>Swissprot Evalue</th><th>Swissprot Description</th><th>PFAM ID</th><th>PFAM description</th><th>Gene Ontology Biological Pathway</th><th>BP Description</th><th>Gene Ontology Molecular Function</th><th>MF Description</th><th>Gene Ontology Cellular Component</th><th>CC Description</th><th>KOG ID</th><th>KOG Description</th><th>Family</th></tr>
					
			  
					
							<tr><td>Cluster-29245.42578</td><td>3.33306852945386</td><td>1165.43554169564</td><td>-8.4289</td><td>5.9075e-26</td><td>6.4337e-21</td><td>2341</td><td>AFW03837.1</td><td></td><td>3564</td><td>0.0</td><td>prophenoloxidase [Palaemon carinicauda]</td><td>JQ812983.1</td><td>JQ812983</td><td>2305</td><td>0</td><td>Exopalaemon carinicauda prophenoloxidase mRNA, complete cds</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>PF00264</td><td>Common central domain of tyrosinase</td><td>--</td><td>--</td><td>GO:0016491</td><td>oxidoreductase activity</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.62562</td><td>438.530430314354</td><td>10.7636764683147</td><td>5.3482</td><td>1.1498e-20</td><td>6.2611e-16</td><td>1046</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.58846</td><td>235.070369994005</td><td>4.30805910062435</td><td>5.7701</td><td>8.0833e-20</td><td>2.9344e-15</td><td>556</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>PF06427//PF03493</td><td>UDP-glucose:Glycoprotein Glucosyltransferase//Calcium-activated BK potassium channel alpha subunit</td><td>GO:0006813//GO:0006486</td><td>potassium ion transport//protein glycosylation</td><td>GO:0003980</td><td>UDP-glucose:glycoprotein glucosyltransferase activity</td><td>GO:0016020</td><td>membrane</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.65296</td><td>215.044611830766</td><td>3.36383882710639</td><td>5.9936</td><td>8.991e-17</td><td>2.448e-12</td><td>1293</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
					
							<tr><td>Cluster-29245.58106</td><td>1769.26380712884</td><td>59.4122605244778</td><td>4.8962</td><td>5.0444e-16</td><td>1.0987e-11</td><td>704</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td></tr>
					
			  
			</table>
	</div>
	</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<strong>Gene ID:</strong> Gene ID.<br />
	<strong>Sample_readcount:</strong> The read count values of sample.<br />
	<strong>log2FoldChange:</strong> log2(Group1/Group2).<br />
	<strong>pval:</strong> The p-value.<br />
	<strong>padj:</strong> The p-value after normalization. The smaller the p-adjusted value is, the more significant is the difference.<br />
</ul>
</div>
<br>
<br>
<!------------------------------差异基因筛选------------------------------>

<h4><a>4.7.2 Filtering the Differential Expression Gene</a></h4>
<p class="paragraph">The number of differentially expressed genes (including up-regulated and down regulated genes) and the criteria for screening differentially expressed genes are shown in the table below.</p>

<p class="center">Table 4.11 Display of statistical results of differential genes</p>

<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>compare</th><th>all</th><th>up</th><th>down</th><th>threshold</th></tr>
					
			  
					
							<tr><td>BvsG</td><td>804</td><td>424</td><td>380</td><td>DESeq2 padj<=0.05 |log2FoldChange|>=1.0</td></tr>
					
			  
					
							<tr><td>GvsEN</td><td>341</td><td>316</td><td>25</td><td>DESeq2 padj<=0.05 |log2FoldChange|>=1.0</td></tr>
					
			  
					
							<tr><td>ENvsLN</td><td>2701</td><td>1075</td><td>1626</td><td>DESeq2 padj<=0.05 |log2FoldChange|>=1.0</td></tr>
					
			  
					
							<tr><td>LNvsEZ</td><td>4849</td><td>1025</td><td>3824</td><td>DESeq2 padj<=0.05 |log2FoldChange|>=1.0</td></tr>
					
			  
					
							<tr><td>EZvsLZ</td><td>3075</td><td>982</td><td>2093</td><td>DESeq2 padj<=0.05 |log2FoldChange|>=1.0</td></tr>
					
			  
			</table>
	</div>
	</p>

<button onclick="hiddenShow(this)">Show Annotation</button>
<div class="comment">
<ul style="font-size: 12px;font-family:line-height: 20px">
	<strong>compare:</strong> name of compare.<br />
	<strong>all:</strong> Number of all different genes.<br />
	<strong>up:</strong> Number of up-regulated gene.<br />
	<strong>down:</strong> Number of down-regulated gene<br />
	<strong>threshold:</strong> The software and threshold of differential gene screening by this comparison combination.<br />
</ul>
</div>
<br>
<br>
<!------------------------------>
<p class="paragraph">The number of differentially expressed genes (including up-regulated and down regulated genes) in each comparison combination was shown by histogram.</p>

<p class="center">
<img class="center" src="src/images/diff_stat.png" height=450 width=500/>
</p>	
<p class="name_fig">Figure 4.14 statistical histogram of the number of different genes in different comparison combinations</p>
<p class="premark">Note: gray and blue indicate up-regulated and down regulated differential genes respectively, and the number on the column indicates the number of differential genes.</p>
<br>


<p class="paragraph">Volcano plots can be used to infer the overall distribution of differentially expressed genes.</p>

<p class="paragraph">For experiments without biological replicates, the threshold is normally set as: [log<sub>2</sub>(Fold Change)] >= 1 and q-value <= 0.005.</p>



	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_Differential_Expression_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_Differential_Expression_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_BvsG" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_BvsG"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_GvsEN" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_GvsEN"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_ENvsLN" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_ENvsLN"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_LNvsEZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_LNvsEZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_EZvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_EZvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_BvsLN" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_BvsLN"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_BvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_BvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_GvsLN" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_GvsLN"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_GvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_GvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ.Volcanoplot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Differential_Expression_pane_ENvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Differential_Expression_list_ENvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ.Volcanoplot.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_Differential_Expression_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_Differential_Expression_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_Differential_Expression_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_BvsG"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_BvsG" role="tab" aria-controls="BvsG">BvsG</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_GvsEN"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_GvsEN" role="tab" aria-controls="GvsEN">GvsEN</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_ENvsLN"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_ENvsLN" role="tab" aria-controls="ENvsLN">ENvsLN</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_LNvsEZ"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_LNvsEZ" role="tab" aria-controls="LNvsEZ">LNvsEZ</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_EZvsLZ"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_EZvsLZ" role="tab" aria-controls="EZvsLZ">EZvsLZ</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_BvsLN"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_BvsLN" role="tab" aria-controls="BvsLN">BvsLN</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_BvsLZ"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_BvsLZ" role="tab" aria-controls="BvsLZ">BvsLZ</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_GvsLN"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_GvsLN" role="tab" aria-controls="GvsLN">GvsLN</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_GvsLZ"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_GvsLZ" role="tab" aria-controls="GvsLZ">GvsLZ</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Differential_Expression_list_ENvsLZ"
	data-toggle="list" href="#gallery_figure_Differential_Expression_pane_ENvsLZ" role="tab" aria-controls="ENvsLZ">ENvsLZ</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.15 Volcano Plot</p>
<p class="premark">The x-axis shows the fold change in gene expression between different samples, and the y-axis shows the statistical significance of the differences. Statistically significant differences are represented by red dots.</p>
<br>
<p class="premark">The sum of the numbers in each circle is the total number of genes expressed within a group, and the overlap represents the genes expressed in common between groups.</p>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/6.Differetial/1.deglist" target="_blank">Result/6.Differetial/1.deglist</a></p>
<br>

<!---------------Diff Venn--------------->


<h4><a>4.7.3 The Venn Diagram of Differentially Expressed Genes</a></h4>
<p class="paragraph">The Venn diagram presents the number of genes that are uniquely expressed differentially within each group, with the overlapping regions showing the number of genes that are expressed in two or more groups. (The diagram depicts only the results for groups 2, 3 and 4). </p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_Venn_Diagram_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_Venn_Diagram_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_Venn_Diagram_pane_BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Venn_Diagram_list_BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ.Venn_diagram.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_Venn_Diagram_pane_BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ" 
	role="tabpanel" aria-labelledby="gallery_figure_Venn_Diagram_list_BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ.Venn_diagram.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_Venn_Diagram_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_Venn_Diagram_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_Venn_Diagram_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Venn_Diagram_list_BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ"
	data-toggle="list" href="#gallery_figure_Venn_Diagram_pane_BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ" role="tab" aria-controls="BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ">BvsG_GvsEN_ENvsLN_LNvsEZ_EZvsLZ</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Venn_Diagram_list_BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ"
	data-toggle="list" href="#gallery_figure_Venn_Diagram_pane_BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ" role="tab" aria-controls="BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ">BvsLN_BvsLZ_GvsLN_GvsLZ_ENvsLZ</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.16 Venn Diagram of Expression Genes</p>
<p class="premark">The sum of the numbers in each circle is the total number of genes expressed within a group, and the overlap represents the genes expressed in common between groups. Use Fpkm>3.0 as the criterion.</p>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/6.Differetial/3.Venn" target="_blank">Result/6.Differetial/3.Venn</a></p>
<br>




<!--------------------------------------- 差异基因表达水平聚类分析 ------------------------------------>


<h4><a>4.7.4 Cluster Analysis of Gene Expression Differences</a></h4>
<p class="paragraph">Cluster Analysis is used to find genes with similar expression patterns under various experimental conditions. By clustering genes with similar expression patterns, it may be possible to discern unknown functions of previously characterized genes or the function of unknown genes. In hierarchical clustering, areas of different colors denote different groups (clusters) of genes, and genes within each cluster may have similar functions or take part in the same biological process.</p>


	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_Cluster_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_Cluster_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_Cluster_pane_heatmap" 
	role="tabpanel" aria-labelledby="gallery_figure_Cluster_list_heatmap"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/Hcluster_heatmap.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_Cluster_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_Cluster_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_Cluster_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_Cluster_list_heatmap"
	data-toggle="list" href="#gallery_figure_Cluster_pane_heatmap" role="tab" aria-controls="heatmap">heatmap</a>
	
	</div>
	</div>
	</div>
	</p>


<p class="name_fig">Figure 4.17 Cluster Analysis</p>

<p class="premark">The overall results of FPKM cluster analysis, clustered using the log<sub>10</sub>(FPKM+1) value. Red denotes genes with high expression levels, and blue denotes genes with low expression levels. The color range from red to blue represents the log<sub>10</sub>(FPKM+1) value from large to small. </p>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/6.Differential/2.Cluster" target="_blank">Result/6.Differential/2.Cluster</a></p>
<br>
<!----------------------------------------- 差异表达分析Q&A -------------------------------------->

<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: How to define differentially expressed genes? How to define the up-regulated gene and down-regulated gene?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>For sample without biological replicates, TMM is first used to normalize the read count value, and DEGseq is used to do the analysis. Filtering threshold are qvalue < 0.005 & |log2FoldChange|>1 ; For differentially expressed gene, if log2Foldchange>0, we defined it is up-regulated and vice versa.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Can we use FPKM to do gene expression difference analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Novogene use readcount to do gene expression difference analysis, which can be normalized by DESeq or TMM. FPKM is a readcount normalization method as well. However, according to the performance of different method indicated in following figure(Dillies MA et al., 2013), DESeq and TMM are better than FPKM. Therefore, FPKM is not recommended.<br><br>
	<br/><div style="text-align:center"><img src="src/images/normalization_evaluate.png" /></div><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: How to evaluate the gene expression difference among samples?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Significant difference can be detected by corrected pvalue. The Lower the corrected pvalue is, the higher the signifance. In the meantime, the difference can be evaluated through |log2Foldchange|, higher |log2Foldchange| means larger differences in expression level.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is difference between two FPKM density plot in gene expression analysis and differentially expressed gene analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>FPKM density plot in differentially expressed gene analysis is a combination of all samples, in which different color represent different samples. Through the overall expression distribution, we can examine the difference among samples. However, the FPKM density plot in gene expression analysis only reflects the information of single sample.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: One gene has very different expression levels in two samples, why it is not in the significantly differentially expressed gene list?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>The selection of differentially expressed gene based on statistical model, the difference in absolute expression level is not sufficient:<br>
<br>Firstly: The influence of sequencing depth, higher sequencing depth tend to bring more readcount, which should be offseted by normalization. (In project with biological replicates, Using DESeq; Without biological replicates, Using TMM)<br>
<br>Secondly: In differentially expressed gene analysis, we need to estimate the distribution of readcount, empirically, readcount follows negative binomial distribution. In the project with replicates, the quality of replication will affect the result. The worse the replication, some differentially expressed gene will be masked. After estimating the parameter, a statistical examination should be performed.<br>
<br>Finally: After pvalue calculation, we need to apply a correction on pvalue basing on multiple hypotheses examination to reduce the false positive. Generally, this process will make padj greater than pvalue, making part of gene passed pvalue threshold be filtered by padj.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the acceptable threshold range for differentially expressed gene analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>There is no determined conclusion for the choose of threshold. Generally, The requirement will be more strict in better journal. However, in some papers such as PLOS ONE, the requirement on threshold is not that strict. For example, In some papers without biological replicates, the author will only use qvalue without log2foldchange.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: How to calculate the foldchange and pvalue for gene with 0 readcount?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>In DESeq, if one gene have a corrected readcount equals to 0 in one sample and readcount higher than 0 in another sample, foldchange will be INF or -INF; If the gene have 0 readcount in both samples, log2foldchange and pvalue, qvalue will be NA; In DEGseq, if one gene have a 0 readcount in a sample, foldchange, pvalue and qvalue will be NA.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: In differentially expressed gene list, if one gene have 0 readcount in one sample and have positive readcount in another sample, can we believe that this gene is expressed in one sample and does not in another?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>In project without genome reference, gene with rpkm>0.3 are believed to express; In project with genome reference, gene with rpkm>1 are believed to express. It is not recommended to evaluate expression by readcount.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Can we use part of the gene to do differentially expressed gene analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>No, differentially expressed gene analysis is a comprehensive analysis, which is based on the general gene expression. The analysis software also recommend us to use the whole readcount data instead of part gene data, for the whole data contains the information about sequence depth and reads distribution.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the method for clustering analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Clustering uses a build-in R package, pheatmap. We focus on the data (union_for_cluster), which is the overlap of gene set of different samples. And we use relative gene expression level, log2(ratios), to do clustering. The clustering will calculate the distance between each genes, and evaluate the relative gene distance through iteration. Finally, it will divide the genes into several subgroups according to gene distance. H-clusger, K-means and SOM are the main clustering method used by us. Most of them can be implemented in R language and other open source softwares.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is clustering for?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Clustering can be used to determine the different gene expression pattern through gathering the gene with similar expression. These genes with similar expression may have similar function or participate in same pathway and metabolic process. It is possible for us to identify unknown gene function through clustering. Thus, the clustering is useful in expanding biological understanding of gene expression.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Why we use padj, can we use pvalue?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Corrected p value (padj/qvalue), is value after applying multiple statistical hypothesis examination on p-value, which is higher than original p-value and more efficient in control false positive.<br><br>
</table>
</div>
<br />
<br>


<!-------------------------------GO 富集分析-------------------------->


<h3 id="id_order_go"><a name="差异基因GO富集分析">4.8  GO Enrichment Analysis</a></h3>
<h4><a>4.8.1 GO Enrichment Analysis of DEGs</a></h4>
<p class="paragraph">GO is the abbreviation of Gene Ontology (<a href="http://www.geneontology.org/" target=_blank>http://www.geneontology.org/</a>), which is a major bioinformatics classification system to unify the presentation of gene properties across all species. It includes three main branches: cellular compares, molecular function and biological process. GO terms with padj <= 0.05 are significant enrichment.</p>

<p class="center">Table 4.12 Significantly Enriched GO terms in DEGs</p>
<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>Category</th><th>GOID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>Count</th><th>geneID</th></tr>
					
			  
					
							<tr><td>CC</td><td>GO:0005576</td><td>extracellular region</td><td>35|308</td><td>1636|31096</td><td>0.0001938</td><td>0.021706</td><td>35</td><td>Cluster-29245.92091,Cluster-29245.57898,Cluster-29245.53192,Cluster-29245.74437,Cluster-29245.38178,Cluster-29245.42186,Cluster-29245.91168,Cluster-29245.7517,Cluster-29245.28638,Cluster-29245.28055,Cluster-29245.65629,Cluster-29245.47178,Cluster-29245.88147,Cluster-29245.46781,Cluster-29245.38785,Cluster-29245.38004,Cluster-32560.1,Cluster-29245.27519,Cluster-29245.59704,Cluster-29245.92287,Cluster-29245.59603,Cluster-29245.21660,Cluster-29245.45485,Cluster-29245.57117,Cluster-29245.28765,Cluster-29245.26862,Cluster-29245.95793,Cluster-29245.73718,Cluster-29245.61514,Cluster-29245.82671,Cluster-29245.91571,Cluster-29245.23811,Cluster-29245.66439,Cluster-29245.15638,Cluster-28618.0</td></tr>
					
			  
					
							<tr><td>MF</td><td>GO:0030234</td><td>enzyme regulator activity</td><td>20|308</td><td>827|31096</td><td>0.00079132</td><td>0.044314</td><td>20</td><td>Cluster-29245.58546,Cluster-29245.57898,Cluster-29245.28638,Cluster-29245.60273,Cluster-29245.29014,Cluster-32560.1,Cluster-29245.50592,Cluster-29245.53192,Cluster-29245.21660,Cluster-29245.53145,Cluster-31580.0,Cluster-29245.29100,Cluster-29245.42186,Cluster-29245.72043,Cluster-29245.81787,Cluster-29245.59603,Cluster-29245.28801,Cluster-29245.45485,Cluster-29245.91719,Cluster-29245.46781</td></tr>
					
			  
					
							<tr><td>BP</td><td>GO:0007005</td><td>mitochondrion organization</td><td>9|308</td><td>287|31096</td><td>0.0045633</td><td>0.17036</td><td>9</td><td>Cluster-29245.28765,Cluster-29245.57898,Cluster-29245.38004,Cluster-29245.64143,Cluster-29245.52276,Cluster-29245.59948,Cluster-29245.74689,Cluster-29245.63887,Cluster-29245.62221</td></tr>
					
			  
					
							<tr><td>MF</td><td>GO:0003700</td><td>DNA-binding transcription factor activity</td><td>16|308</td><td>732|31096</td><td>0.0088327</td><td>0.24732</td><td>16</td><td>Cluster-29245.51732,Cluster-29245.63268,Cluster-29245.50442,Cluster-29245.29947,Cluster-29245.25264,Cluster-29245.30743,Cluster-29245.54662,Cluster-29245.38725,Cluster-29245.56327,Cluster-29245.67695,Cluster-29245.92931,Cluster-29245.91786,Cluster-29245.61096,Cluster-29245.41983,Cluster-29245.67238,Cluster-29245.59326</td></tr>
					
			  
					
							<tr><td>BP</td><td>GO:0048856</td><td>anatomical structure development</td><td>10|308</td><td>480|31096</td><td>0.044329</td><td>0.87399</td><td>10</td><td>Cluster-29245.7517,Cluster-29245.91682,Cluster-29245.38004,Cluster-29245.23805,Cluster-29245.91571,Cluster-29245.26308,Cluster-29245.73718,Cluster-29245.46218,Cluster-29245.61760,Cluster-29245.48346</td></tr>
					
			  
			</table>
	</div>
	</p>

	<button onclick="hiddenShow(this)">Show Annotation</button>
	<div class="comment">
	<ul style="font-size: 12px;font-family:line-height: 20px">
		<strong>Category:</strong> different class of GO id. CC, BP and MF are the abbreviation of cellular component, biological process and molecular function, respectively.<br />
		<strong>ID:</strong> the unique identification id of Gene Ontology database.<br />
		<strong>Description:</strong> function description of Gene Ontology.<br />
		<strong>GeneRatio:</strong> ratio of differentially expressed genes to all genes for this GO term.<br />
		<strong>BgRatio:</strong> in background GO database, the ratio of all genes concerning this GO term to all genes.<br />
		<strong>pvalue:</strong> statistics category term; abbreviation of probability value.<br />
		<strong>padj:</strong> adjusted p-value. Generally, GO terms with Corrected_pValue <= 0.05 are significant enrichment.<br />
		<strong>Count:</strong> the number of differentially expressed genes concerning this GO term.<br />
		<strong>geneID:</strong> the IDs of differentially expressed genes concerning this GO term.<br />
		<strong>up:</strong> the number of up regulation differentially expressed genes concerning this GO term.<br />
		<strong>Up_Gene_names:</strong> the IDs of up regulation differentially expressed genes concerning this GO term.<br />
		<strong>Down:</strong> the number of down regulation differentially expressed genes concerning this GO term.<br />
		<strong>Down_Gene_names:</strong> the IDs of down regulation differentially expressed genes concerning this GO term. <br /> 

	</ul>	
	</div>
<br />
<br>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_GO_Enrich_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_GO_Enrich_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsG_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsG_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsG_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsG_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsG_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsG_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsLN_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsLN_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsLN_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsLN_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsLN_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsLN_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsLZ_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsLZ_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_BvsLZ_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_BvsLZ_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLN_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLN_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLN_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLN_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLN_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLN_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLZ_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLZ_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLZ_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLZ_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_ENvsLZ_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_ENvsLZ_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_EZvsLZ_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_EZvsLZ_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_EZvsLZ_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_EZvsLZ_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_EZvsLZ_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_EZvsLZ_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsEN_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsEN_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsEN_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsEN_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsEN_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsEN_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLN_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLN_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLN_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLN_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLN_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLN_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLZ_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLZ_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLZ_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLZ_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_GvsLZ_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_GvsLZ_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ.bar_graph_updown.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_LNvsEZ_GO_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_LNvsEZ_GO_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ_GO.bar.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_LNvsEZ_GO_classification" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_LNvsEZ_GO_classification"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ_GO.classification.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Enrich_pane_LNvsEZ_GO_updown_bar" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Enrich_list_LNvsEZ_GO_updown_bar"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ.bar_graph_updown.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_GO_Enrich_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_GO_Enrich_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_GO_Enrich_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsG_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsG_GO_bar" role="tab" aria-controls="BvsG_GO_bar">BvsG_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsG_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsG_GO_classification" role="tab" aria-controls="BvsG_GO_classification">BvsG_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsG_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsG_GO_updown_bar" role="tab" aria-controls="BvsG_GO_updown_bar">BvsG_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsLN_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsLN_GO_bar" role="tab" aria-controls="BvsLN_GO_bar">BvsLN_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsLN_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsLN_GO_classification" role="tab" aria-controls="BvsLN_GO_classification">BvsLN_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsLN_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsLN_GO_updown_bar" role="tab" aria-controls="BvsLN_GO_updown_bar">BvsLN_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsLZ_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsLZ_GO_classification" role="tab" aria-controls="BvsLZ_GO_classification">BvsLZ_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_BvsLZ_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_BvsLZ_GO_updown_bar" role="tab" aria-controls="BvsLZ_GO_updown_bar">BvsLZ_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLN_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLN_GO_bar" role="tab" aria-controls="ENvsLN_GO_bar">ENvsLN_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLN_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLN_GO_classification" role="tab" aria-controls="ENvsLN_GO_classification">ENvsLN_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLN_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLN_GO_updown_bar" role="tab" aria-controls="ENvsLN_GO_updown_bar">ENvsLN_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLZ_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLZ_GO_bar" role="tab" aria-controls="ENvsLZ_GO_bar">ENvsLZ_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLZ_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLZ_GO_classification" role="tab" aria-controls="ENvsLZ_GO_classification">ENvsLZ_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_ENvsLZ_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_ENvsLZ_GO_updown_bar" role="tab" aria-controls="ENvsLZ_GO_updown_bar">ENvsLZ_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_EZvsLZ_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_EZvsLZ_GO_bar" role="tab" aria-controls="EZvsLZ_GO_bar">EZvsLZ_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_EZvsLZ_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_EZvsLZ_GO_classification" role="tab" aria-controls="EZvsLZ_GO_classification">EZvsLZ_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_EZvsLZ_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_EZvsLZ_GO_updown_bar" role="tab" aria-controls="EZvsLZ_GO_updown_bar">EZvsLZ_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsEN_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsEN_GO_bar" role="tab" aria-controls="GvsEN_GO_bar">GvsEN_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsEN_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsEN_GO_classification" role="tab" aria-controls="GvsEN_GO_classification">GvsEN_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsEN_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsEN_GO_updown_bar" role="tab" aria-controls="GvsEN_GO_updown_bar">GvsEN_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLN_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLN_GO_bar" role="tab" aria-controls="GvsLN_GO_bar">GvsLN_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLN_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLN_GO_classification" role="tab" aria-controls="GvsLN_GO_classification">GvsLN_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLN_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLN_GO_updown_bar" role="tab" aria-controls="GvsLN_GO_updown_bar">GvsLN_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLZ_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLZ_GO_bar" role="tab" aria-controls="GvsLZ_GO_bar">GvsLZ_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLZ_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLZ_GO_classification" role="tab" aria-controls="GvsLZ_GO_classification">GvsLZ_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_GvsLZ_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_GvsLZ_GO_updown_bar" role="tab" aria-controls="GvsLZ_GO_updown_bar">GvsLZ_GO_updown_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_LNvsEZ_GO_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_LNvsEZ_GO_bar" role="tab" aria-controls="LNvsEZ_GO_bar">LNvsEZ_GO_bar</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_LNvsEZ_GO_classification"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_LNvsEZ_GO_classification" role="tab" aria-controls="LNvsEZ_GO_classification">LNvsEZ_GO_classification</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Enrich_list_LNvsEZ_GO_updown_bar"
	data-toggle="list" href="#gallery_figure_GO_Enrich_pane_LNvsEZ_GO_updown_bar" role="tab" aria-controls="LNvsEZ_GO_updown_bar">LNvsEZ_GO_updown_bar</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.18 GO enrichment Bar Chart of DEGs</p>
<p class="premark">Note: the abscissa of classification bar chart is the GO term of the next level of the three go categories, and the ordinate is the number of differential genes annotated to the term (including the sub term of the term). The three different classifications represent the three basic classifications of GO term (from left to right are biological processes, cellular components, and molecular functions)</p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_GO_Dot_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_GO_Dot_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_BvsG_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_BvsG_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_BvsLN_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_BvsLN_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_BvsLZ_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_BvsLZ_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_ENvsLN_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_ENvsLN_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_ENvsLZ_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_ENvsLZ_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_EZvsLZ_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_EZvsLZ_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_GvsEN_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_GvsEN_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_GvsLN_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_GvsLN_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_GvsLZ_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_GvsLZ_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ_GO.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_Dot_pane_LNvsEZ_GO_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_Dot_list_LNvsEZ_GO_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ_GO.dot.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_GO_Dot_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_GO_Dot_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_GO_Dot_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_BvsG_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_BvsG_GO_dot" role="tab" aria-controls="BvsG_GO_dot">BvsG_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_BvsLN_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_BvsLN_GO_dot" role="tab" aria-controls="BvsLN_GO_dot">BvsLN_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_BvsLZ_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_BvsLZ_GO_dot" role="tab" aria-controls="BvsLZ_GO_dot">BvsLZ_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_ENvsLN_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_ENvsLN_GO_dot" role="tab" aria-controls="ENvsLN_GO_dot">ENvsLN_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_ENvsLZ_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_ENvsLZ_GO_dot" role="tab" aria-controls="ENvsLZ_GO_dot">ENvsLZ_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_EZvsLZ_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_EZvsLZ_GO_dot" role="tab" aria-controls="EZvsLZ_GO_dot">EZvsLZ_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_GvsEN_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_GvsEN_GO_dot" role="tab" aria-controls="GvsEN_GO_dot">GvsEN_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_GvsLN_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_GvsLN_GO_dot" role="tab" aria-controls="GvsLN_GO_dot">GvsLN_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_GvsLZ_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_GvsLZ_GO_dot" role="tab" aria-controls="GvsLZ_GO_dot">GvsLZ_GO_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_Dot_list_LNvsEZ_GO_dot"
	data-toggle="list" href="#gallery_figure_GO_Dot_pane_LNvsEZ_GO_dot" role="tab" aria-controls="LNvsEZ_GO_dot">LNvsEZ_GO_dot</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.19 GO enrichment Bar Chart of DEGs</p>

<p class="premark">The png format of the statistical histogram of up-regulated genes and down-regulated genes related to GO enrichment(all differential genes)</p>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/7.Enrichment/1.GO" target="_blank">Result/7.Enrichment/1.GO</a></p>
<br>
<!------------------------------topGO有向无环图分析------------------------------->

<h4><a>4.8.2 GO Enrichment DAG Figure</a></h4>
<p class="paragraph">DAG (Directed Acyclic Graph, DAG) can visually display the enriched GO term of differential expression genes and its hierarchy. (<strong>Figure</strong>) illustrates the topGO DAGs. Branch means hierarchical relationship and the function ranges become more and more specific from top to bottom. DAG of biological process, molecular function and cellular component are shown respectively in the report.</p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_GO_DAG_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_GO_DAG_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsG_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsG_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsG_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsG_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsG_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsG_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLN_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLN_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLN_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLN_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLN_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLN_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLZ_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLZ_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLZ_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLZ_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_BvsLZ_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_BvsLZ_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLN_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLN_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLN_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLN_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLN_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLN_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLZ_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLZ_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLZ_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLZ_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_ENvsLZ_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_ENvsLZ_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_EZvsLZ_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_EZvsLZ_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_EZvsLZ_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_EZvsLZ_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_EZvsLZ_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_EZvsLZ_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsEN_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsEN_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsEN_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsEN_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsEN_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsEN_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLN_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLN_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLN_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLN_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLN_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLN_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLZ_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLZ_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLZ_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLZ_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_GvsLZ_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_GvsLZ_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_LNvsEZ_BP" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_LNvsEZ_BP"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ.DEG_Enriched_GO_bp_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_LNvsEZ_CC" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_LNvsEZ_CC"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ.DEG_Enriched_GO_cc_DAG.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_GO_DAG_pane_LNvsEZ_MF" 
	role="tabpanel" aria-labelledby="gallery_figure_GO_DAG_list_LNvsEZ_MF"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ.DEG_Enriched_GO_mf_DAG.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_GO_DAG_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_GO_DAG_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_GO_DAG_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsG_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsG_BP" role="tab" aria-controls="BvsG_BP">BvsG_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsG_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsG_CC" role="tab" aria-controls="BvsG_CC">BvsG_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsG_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsG_MF" role="tab" aria-controls="BvsG_MF">BvsG_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLN_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLN_BP" role="tab" aria-controls="BvsLN_BP">BvsLN_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLN_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLN_CC" role="tab" aria-controls="BvsLN_CC">BvsLN_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLN_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLN_MF" role="tab" aria-controls="BvsLN_MF">BvsLN_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLZ_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLZ_BP" role="tab" aria-controls="BvsLZ_BP">BvsLZ_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLZ_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLZ_CC" role="tab" aria-controls="BvsLZ_CC">BvsLZ_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_BvsLZ_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_BvsLZ_MF" role="tab" aria-controls="BvsLZ_MF">BvsLZ_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLN_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLN_BP" role="tab" aria-controls="ENvsLN_BP">ENvsLN_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLN_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLN_CC" role="tab" aria-controls="ENvsLN_CC">ENvsLN_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLN_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLN_MF" role="tab" aria-controls="ENvsLN_MF">ENvsLN_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLZ_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLZ_BP" role="tab" aria-controls="ENvsLZ_BP">ENvsLZ_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLZ_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLZ_CC" role="tab" aria-controls="ENvsLZ_CC">ENvsLZ_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_ENvsLZ_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_ENvsLZ_MF" role="tab" aria-controls="ENvsLZ_MF">ENvsLZ_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_EZvsLZ_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_EZvsLZ_BP" role="tab" aria-controls="EZvsLZ_BP">EZvsLZ_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_EZvsLZ_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_EZvsLZ_CC" role="tab" aria-controls="EZvsLZ_CC">EZvsLZ_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_EZvsLZ_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_EZvsLZ_MF" role="tab" aria-controls="EZvsLZ_MF">EZvsLZ_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsEN_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsEN_BP" role="tab" aria-controls="GvsEN_BP">GvsEN_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsEN_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsEN_CC" role="tab" aria-controls="GvsEN_CC">GvsEN_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsEN_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsEN_MF" role="tab" aria-controls="GvsEN_MF">GvsEN_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLN_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLN_BP" role="tab" aria-controls="GvsLN_BP">GvsLN_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLN_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLN_CC" role="tab" aria-controls="GvsLN_CC">GvsLN_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLN_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLN_MF" role="tab" aria-controls="GvsLN_MF">GvsLN_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLZ_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLZ_BP" role="tab" aria-controls="GvsLZ_BP">GvsLZ_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLZ_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLZ_CC" role="tab" aria-controls="GvsLZ_CC">GvsLZ_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_GvsLZ_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_GvsLZ_MF" role="tab" aria-controls="GvsLZ_MF">GvsLZ_MF</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_LNvsEZ_BP"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_LNvsEZ_BP" role="tab" aria-controls="LNvsEZ_BP">LNvsEZ_BP</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_LNvsEZ_CC"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_LNvsEZ_CC" role="tab" aria-controls="LNvsEZ_CC">LNvsEZ_CC</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_GO_DAG_list_LNvsEZ_MF"
	data-toggle="list" href="#gallery_figure_GO_DAG_pane_LNvsEZ_MF" role="tab" aria-controls="LNvsEZ_MF">LNvsEZ_MF</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.20 Illustration of topGO DAG</p>

<p class="premark center">Each node represents a GO term, and TOP10 GO terms are boxed. The darker the color is, the higher is the enrichment level of the term. The name and p-value of each term are present on the node.</p>
<br>

<!----------------------------------------- GO富集分析Q&A -------------------------------------->

<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the analysis software used in GO Functional Enrichment Analysis</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>GO Functional Enrichment analysis uses R package. Enrichment process uses GOseq(Young MD et al., 2010); topGO ACG plotting uses topGO.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: Generally, GO Functional Enrichment are divided into 2 level, can we do more?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>GO functional enrichment analysis focus on the statistical examiniation on all the annoatated GO term. It can be any level.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the difference between GO enrichment and GO classification</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>GO classfification correlate the gene with its GO function to get genes annotation information. GO enrichment is examining the enrichment of a group gene with similar GO function, which is used to analyze the gene with similar function.<br><br>		
</table>
</div>
<br />
<br>


<!---------------------------差异基因 KEGG富集分析---------------------------->


<h3 id="id_order_kegg">4.9  KEGG Enrichment Analysis</h3>
<h4><a>4.9.1 KEGG Pathway Enrichment Analysis</a></h4>
<p class="paragraph">The interactions of multiple genes may be involved in certain biological functions. KEGG (Kyoto Encyclopedia of Genes and Genomes) is a collection of manually curated databases dealing with genomes, biological pathways, diseases, drugs, and chemical substances(Kanehisa M et al., 2008). KEGG is utilized for bioinformatics research and education, including data analysis in genomics, metagenomics, metabolomics and other omics studies. Pathway enrichment analysis identifies significantly enriched metabolic pathways or signal transduction pathways associated with differentially expressed genes compared with the whole genome background.</p>
<p class="center">
<img class="center" src="src/images/enrichment.png" />
</p>
<p class="paragraph">Here, N is the number of all genes with a KEGG annotation, n is the number of DEGs in N, M is the number of all genes annotated to specific pathways, and m is number of DEGs in M.</p>
<p class="center">Table 4.13 KEGG Enrichment List</p>
<p class="center">
	<div id="tb">
			<table class="tb1">
					
					
							<tr><th>Category</th><th>KEGGID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>Count</th><th>geneID</th></tr>
					
			  
					
							<tr><td>KEGG</td><td>ko04916</td><td>Melanogenesis</td><td>11|88</td><td>142|12110</td><td>1.39950260298e-07</td><td>2.54709473742e-05</td><td>11</td><td>Cluster-29245.7517|Cluster-29245.78753|Cluster-29245.26697|Cluster-29245.91538|Cluster-29245.82385|Cluster-29245.91044|Cluster-29245.43271|Cluster-29245.93580|Cluster-29245.49786|Cluster-29245.75620|Cluster-29245.43421</td></tr>
					
			  
					
							<tr><td>KEGG</td><td>ko00350</td><td>Tyrosine metabolism</td><td>9|88</td><td>103|12110</td><td>7.58616639568e-07</td><td>6.90341142007e-05</td><td>9</td><td>Cluster-29245.78753|Cluster-29245.91538|Cluster-29245.82385|Cluster-29245.91044|Cluster-29245.43271|Cluster-29245.93580|Cluster-29245.49786|Cluster-29245.75620|Cluster-29245.43421</td></tr>
					
			  
					
							<tr><td>KEGG</td><td>ko04350</td><td>TGF-beta signaling pathway</td><td>6|88</td><td>101|12110</td><td>0.000397639381893</td><td>0.0241234558348</td><td>6</td><td>Cluster-29245.53192|Cluster-29245.91217|Cluster-29245.64605|Cluster-29245.28801|Cluster-29245.63443|Cluster-29245.33360</td></tr>
					
			  
					
							<tr><td>KEGG</td><td>ko04512</td><td>ECM-receptor interaction</td><td>5|88</td><td>70|12110</td><td>0.000553879173338</td><td>0.0252015023869</td><td>5</td><td>Cluster-29245.34516|Cluster-29245.26805|Cluster-29245.58090|Cluster-29245.57634|Cluster-29245.28043</td></tr>
					
			  
					
							<tr><td>KEGG</td><td>ko04514</td><td>Cell adhesion molecules (CAMs)</td><td>4|88</td><td>53|12110</td><td>0.00165891090396</td><td>0.0511509876108</td><td>4</td><td>Cluster-29245.29296|Cluster-29245.76277|Cluster-29245.34516|Cluster-29245.28043</td></tr>
					
			  
			</table>
	</div>
	</p>

	<button onclick="hiddenShow(this)">Show Annotation</button>
	<div class="comment">
	<ul style="font-size: 12px;font-family:line-height: 20px">
		<strong>Category:</strong> KEGG database.<br/>
		<strong>ID:</strong> the unique identification id of KEGG database.<br/>
		<strong>Description:</strong> function description of this pathway.<br/>
		<strong>GeneRatio:</strong> ratio of differentially expressed genes to all genes, which are concerned with this pathway.<br/>
		<strong>BgRatio:</strong> in background KEGG database, the ratio of all genes concerning this KEGG term to all genes.<br/>
		<strong>pvalue:</strong> statistics category term; abbreviation of probability value.<br/>
		<strong>padj:</strong> adjusted p-value. Generally, kegg terms with Corrected_pValue <= 0.05 are significant enrichment.<br/>
		<strong>Count:</strong> the number of differentially expressed genes concerning in this term.<br/>
		<strong>geneID:</strong> the IDs of differentially expressed genes concerning in this term.<br/>
		<strong>up:</strong> the number of up regulation differentially expressed genes concerning in this term.<br/>
		<strong>Up_Gene_names:</strong> the IDs of up regulation differentially expressed genes concerning in this term.<br/>
		<strong>Down:</strong> the number of down regulation differentially expressed genes concerning in this term.<br/>
		<strong>Down_Gene_names:</strong> the IDs of down regulation differentially expressed genes concerning in this term.<br/>
	</ul>	
	</div>
<br />
<br/>
<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/7.Enrichment/2.KEGG" target="_blank">Result/7.Enrichment/2.KEGG</a></p>

<br>
<!--------------------------差异基因KEGG富集散点图------------------------------->

<h4><a>4.9.2 KEGG Enrichment Plot</a></h4>
<p class="paragraph">KEGG enrichment scattered plot shows the DEGs enrichment analysis results in KEGG pathway. The degree of KEGG enrichment is measured by Rich factor, q-value and the number of genes enriched in this pathway. Rich factor refers to the ratio of the DEGs number in the pathway and the number of all genes annotated in the pathway. Q-value is the p-value after normalization and its range is [0,1]. The smaller q-value is, the more significant the enrichment is. The top20 significantly DEGs enriched pathways are displayed in the report. If the enriched pathways are less than 20, all enriched pathways are displayed. 
</p>

	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_KEGG_Enrich_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_KEGG_Enrich_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_BvsG_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_BvsG_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsG_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_BvsLN_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_BvsLN_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLN_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_BvsLZ_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_BvsLZ_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/BvsLZ_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_ENvsLN_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_ENvsLN_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLN_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_ENvsLZ_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_ENvsLZ_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ENvsLZ_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_EZvsLZ_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_EZvsLZ_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/EZvsLZ_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_GvsEN_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_GvsEN_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsEN_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_GvsLN_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_GvsLN_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLN_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_GvsLZ_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_GvsLZ_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/GvsLZ_KEGG.dot.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Enrich_pane_LNvsEZ_KEGG_dot" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Enrich_list_LNvsEZ_KEGG_dot"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/LNvsEZ_KEGG.dot.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_KEGG_Enrich_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_KEGG_Enrich_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_KEGG_Enrich_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_BvsG_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_BvsG_KEGG_dot" role="tab" aria-controls="BvsG_KEGG_dot">BvsG_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_BvsLN_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_BvsLN_KEGG_dot" role="tab" aria-controls="BvsLN_KEGG_dot">BvsLN_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_BvsLZ_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_BvsLZ_KEGG_dot" role="tab" aria-controls="BvsLZ_KEGG_dot">BvsLZ_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_ENvsLN_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_ENvsLN_KEGG_dot" role="tab" aria-controls="ENvsLN_KEGG_dot">ENvsLN_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_ENvsLZ_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_ENvsLZ_KEGG_dot" role="tab" aria-controls="ENvsLZ_KEGG_dot">ENvsLZ_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_EZvsLZ_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_EZvsLZ_KEGG_dot" role="tab" aria-controls="EZvsLZ_KEGG_dot">EZvsLZ_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_GvsEN_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_GvsEN_KEGG_dot" role="tab" aria-controls="GvsEN_KEGG_dot">GvsEN_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_GvsLN_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_GvsLN_KEGG_dot" role="tab" aria-controls="GvsLN_KEGG_dot">GvsLN_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_GvsLZ_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_GvsLZ_KEGG_dot" role="tab" aria-controls="GvsLZ_KEGG_dot">GvsLZ_KEGG_dot</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Enrich_list_LNvsEZ_KEGG_dot"
	data-toggle="list" href="#gallery_figure_KEGG_Enrich_pane_LNvsEZ_KEGG_dot" role="tab" aria-controls="LNvsEZ_KEGG_dot">LNvsEZ_KEGG_dot</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.21 KEGG Enrichment Bar Plot and Dot Plot of DEGs</p>

<!----<p class="premark center">The y-axis represents the name of the pathway and the x-axis represents the Rich factor. Dot size represents the number of different genes and the color indicates the q-value.</p>------>
<br>

<!------------------------------富集KEGG通路图----------------------------->

<h4><a>2.9.3 KEGG Enrichment Pathway</a></h4>
<p class="paragraph">KEGG enrichment pathway shows the DEGs significantly enriched pathways. In the diagram, if this node contains up-regulated genes, the KO node is labeled in red. If the node contains down-regulated genes, the KO node is labeled in green. If the node contains both up and down-regulated genes, the labeled color is yellow.
</p>
	<p class="center">
	<div class="row">
	<div class="col-1">
	<div id="gallery_figure_KEGG_Pathway_button_prev" class="button_prev" onclick="prev(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_prev">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="8789.5,4902.8 8789.5,5846.4 6801.7,8000 8789.5,10253.6 8789.5,11097.2 5930.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-6">
	<div class="tab-content" id="gallery_figure_KEGG_Pathway_nav-tabContent">
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Pathway_pane_ko04014" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Pathway_list_ko04014"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ko04014.png" /></div>
	
	<div class="tab-pane fade" id="gallery_figure_KEGG_Pathway_pane_ko04260" 
	role="tabpanel" aria-labelledby="gallery_figure_KEGG_Pathway_list_ko04260"
	onclick="trans(this.id)"><img class="img-gallery" src="src/images/ko04260.png" /></div>
	
	</div></div>
	<div class="col-1">
	<div id="gallery_figure_KEGG_Pathway_button_next" class="button_next" onclick="next(this.id)">
	<svg viewbox="0 0 16000 16000" class="svg_next">
	<circle class="c" cx="8000" cy="8000" r="6080"></circle>
	<polygon class="a" points="7210.5,4902.8 7210.5,5846.4 9198.3,8000 7210.5,10253.6 7210.5,11097.2 10069.5,8000 ">
	</polygon></svg></div></div>
	<div class="col-4">
	<input class="form-control" id="gallery_figure_KEGG_Pathway_InputList" type="text" placeholder="Search..">
	<div class="list-group" id="gallery_figure_KEGG_Pathway_list-tab" role="tablist">
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Pathway_list_ko04014"
	data-toggle="list" href="#gallery_figure_KEGG_Pathway_pane_ko04014" role="tab" aria-controls="ko04014">ko04014</a>
	
	<a class="list-group-item list-group-item-action" id="gallery_figure_KEGG_Pathway_list_ko04260"
	data-toggle="list" href="#gallery_figure_KEGG_Pathway_pane_ko04260" role="tab" aria-controls="ko04260">ko04260</a>
	
	</div>
	</div>
	</div>
	</p>

<p class="name_fig">Figure 4.22 Diagram Showing Significantly Enriched KEGG Pathway</p>


<!----------------------------------------- KEGG富集分析Q&A -------------------------------------->
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the meaning and advantages of differentially expressed gene enrichment analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Gene enrichment analysis is a method gathering gene into biological function or pathway through bioinformatic database and statistical tools. This method is a powerful tool in understanding gene function, including GO enrichment analysis and KEGG enrichment analysis. The GO enrichment analysis help us understand the gene function of differentially expressed gene, while the KEGG enrichment analysis help us understand the metabolic pathway that genes partcipated. Understanding the function and participated pathway of differentially expressed gene is one approach for us to explain the difference in phenotype and other biological phenomenon.<br><br>
	Gene enrichment analysis has following advantages compared to analysis based on single gene:<br>
	<strong>1</strong> Higher sensitivity and lower rate of false postive.<br>
	<strong>2</strong> Integrating the information of gene interaction, the results are more reliable<br>
	<strong>3</strong> Simutaneously dealing with large gene expression data with uniform classfication information.<br>
	<strong>4</strong> Being reproductive.<br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What are the meaning of color and node in KEGG pathway figure?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>KEGG Pathway have five types as following:<br><br>
	<strong>map:</strong> Reference pathway<br>
	<strong>ko:</strong> Reference pathway (KO)<br>
	<strong>ec:</strong> Reference pathway (EC)<br>
	<strong>rn:</strong> Reference pathway (Reaction)<br>
	<strong>>org:</strong> Organism-specific pathway map<br>
	Nodes and color:<br>
	<strong>1</strong> “map” pathway: Node represent a specfic gene, enzyme coded by the gene and related reaction, click the node to get these informations<br>
	<strong>2</strong> “ko/ec/rn” pathway: In ko pathway, the node only represent a specific gene; In ec pathway, the node represent the related enzyme;In rn pathway, node represent a paticular reaction including the substrates and reaction type. (Background is blue).<br>
	<strong>3</strong> “org” pathway: Species specific pathway figure, same as map pathway. (The background of species related node is green)<br>
	Each pathway has a unique ID number, such as map00010, which is the access number in KEGG database. In project without genome reference, it can only be ko -reference pathway.<br><br>
</table>
</div>
<br />
<br>





<!------------------------------------- 差异基因蛋白互作网络分析 ---------------------------------->


<h3 id="id_order_ppi"><a>4.10  Protein-Protein Interaction Analysis</a></h3>
<div><h4><a>4.10.1 Protein-Protein Interaction Analysis</a></h4></div>
<p class="paragraph">The protein-protein interaction network is constructed for differential expression gene by searching STRING protein interaction database (<a href="http://string-db.org/" target="_blank">http://string-db.org/</a>. Protein-protein interaction is provided as network file which can be imported into Cytoscape(Shannon P et al., 2003) software and visualized and edited. The central organizing metaphor of Cytoscape is a network graph, with molecular species represented as nodes and intermolecular interactions represented as links, that is, edges, between nodes.</p>
<p class="paragraph">(1) Customize network data display using powerful visual styles.</p>
<p class="paragraph">(2) View a superposition of gene expression ratios and p-values on the network. Expression data can be mapped to node color, label, border thickness, or border color, etc. according to user-configurable colors and visualization schemes.</p>
<p class="paragraph">(3) Layout networks in two dimensions. A variety of layout algorithms are available, including cyclic and spring-embedded layouts.</p>
<p class="paragraph">(4) Zoom in/out and pan for browsing the network.</p>
<p class="paragraph">(5) Use the network manager to easily organize multiple networks. And this structure can be saved in a session file.</p>
<p class="paragraph">(6) Use the bird's eye view to easily navigate large networks.</p>
<p class="paragraph">(7) Easily navigate large networks (100,000+ nodes and edges) by efficient rendering engine.</p>

<p class="center">
<img class='wid2' src="src/images/PPI.png" />
</p>
<p class="name_fig">Figure 4.23 Visualization of Cytoscape</p>

<!-------------------------------------------------------->
<button onclick="hiddenQA(this)">Show Q&A</button>
<div class="comment">
<table style="font-size:12px;font-family: 'Times New Roman' 'Montserrat'">
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Q: What is the software used in PPI analysis? What is database used in PPI analysis?</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br>Trough blastx alignment, we can find the interaction relationship of gene in STRING database, then input the interaction data into Cytoscape to visualize the network.<br><br>
<tr style="background-color:#036EB8"><td align="left" style="background-color:#036EB8;color:#FFFFFF">Terminology</td></tr>
<tr style="background-color:#f1f5fb"><td align="left" style="background-color:#f1f5fb;color:#3E3A39"><br><strong>clustering coefficient(clustering coefficient):</strong>  A clustering coefficient is a measure of the degree to which nodes in a graph tend to cluster together.<br>
</table>
</div>
<br />
<br>

<p class="paragraph">Result Directory: <a href="../Result_XXXXSCXXXXXXXX-ZXX-JXXX-BX-16/7.Enrichment/3.PPI" target="_blank">Result/7.Enrichment/3.PPI</a></p>
<br>




<!---------------------------------------- SSR分析 --------------------------------------->




<!---------------------------------------- SNP和InDel分析 --------------------------------------->


<!-------------------------------------------- 备注 ------------------------------------------->


<h2><a id="备注">5 Appendix</a></h2>
<!------------------------------------------软件列表------------------------------------------------------>
<h3  id="文件目录列表">5.1 Software</h3>

	<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class="gy2">
		<tr><th>Analysis</th><th>Software</th><th>Version</th><th>Parameter</th><th>Remark</th></tr>
		<tr><td rowspan="4">Assembly</td><td rowspan="2">Trinity</td><td rowspan="2">2.15.1</td><td>minKmerCov=3</td><td rowspan="2">-</td></tr>
		<tr><td>min_glue=4</td></tr>
		<tr><td>Corset</td><td>1.09</td><td>-f ture, Default, -m 10</td><td>remove redundancy</td></tr>
		<tr><td>BUSCO</td><td>3.0.2</td><td>-m tran</td><td>-</td></tr>
		<tr><td rowspan="5">Gene Functional Annotation</td><td>Diamond</td><td>2.1.6</td><td>e-value = 1e-5</td><td>NR, KOG/COG, Swiss-Prot</td></tr>
		<tr><td>Diamond, KAAS</td><td>2.1.6</td><td>e-value = 1e-5</td><td>KEGG Annotation</td></tr>
		<tr><td>NCBI blast</td><td>2.13.0</td><td>e-value = 1e-5</td><td>NT Annotation</td></tr>
		<tr><td>hmmscan</td><td>HMMER 3.4</td><td>e-value = 0.01</td><td>Pfam Annotation</td></tr>
		<tr><td>blast2go</td><td>b2g4pipe_v2.5</td><td>e-value = 1e-6</td><td>GO Annotation</td></tr>
		<tr><td>Mapping and Quantification</td><td>RSEM</td><td>1.2.28</td><td>--estimate-rspd -mismatch-rate 0.3</td><td>mapping to Corset filtered transcriptome</td></tr>
		<tr><td>Mutation</td><td>BCFtools</td><td>1.21</td><td>bcftools mpileup -q 1 -C 50 -m 2 -F 0.002 -Ou -f</td><td>-</td></tr>
		<tr><td>SSR Analysis</td><td>MISA, primer3</td><td>primer3-2.6.1</td><td>SSR: 1-10 2-6 3-5 4-5 5-5 6-5</td><td>Misa detect SSR, primer3 Primer Design</td></tr>
		<tr><td rowspan="2">Differential Expression Analysis</td><td>DESeq2</td><td>1.42.0</td><td>padj <= 0.05 |log<sub>2</sub>(foldchange)| >= 1</td><td rowspan="2">For sample with bio-replicate using DESeq2, samples without bio-replicate using EdgeR.</td></tr>
		<tr><td>edgeR</td><td>4.0.16</td><td>padj <= 0.005 & |log<sub>2</sub>(foldchange)| >= 1</td></tr>
		<tr><td>GO Enrichment</td><td>GOSeq, topGO</td><td>1.32.0, 2.32.0</td><td>Corrected P-Value <= 0.05 </td><td>-</td></tr>
		<tr><td>KEGG Enrichment</td><td>KOBAS</td><td>v3.0</td><td>Corrected P-Value <= 0.05</td><td>-</td></tr>
		<tr><td>Protein-Protein Interaction Analysis</td><td>diamond blastx</td><td>2.6.1</td><td>e-value = 1e-10</td><td>Using blast, String database.</td></tr>
	</table>
	<br>

<!------------------------------------------------------------------------------------------------>
<h3  id="文件目录列表">5.2 Methods</h3>
        </a>
        <p class="paragraph">Click to open the method(pdf), and the corresponding file can be viewed directly.</p>
        
        <p class="paragraph">File catalog:<a href="src/denovo_report_method.pdf" title="点击打开" target="_blank">methods</a></p>
        
<div>        
	<br />

        <h3 id="note_2">5.3 Result File Decompression Method and Format Description</h3>
        <table class="gy2">
          <tr>
            <th>Compressed.format</th>
            <th>Customer.type</th>
            <th>Uncompressed.method</th>
          </tr>

		  <tr><td rowspan="2">*.tar</td><td>Unix/Linux/Mac user</td><td>use tar -xvf *.tar command</td></tr>
		  <tr><td>Windows user</td><td>use uncompressed software such as WinRAR, 7-Zip et al</td></tr>
		  
          <tr>
            <td rowspan="2">*.gz</td>
            <td>Unix/Linux/Mac</td>
            <td>Command line: gzip –d *.gz </td>
          </tr>

          <tr>
            <td>Windows</td>
            <td>Software: such as WinRAR, 7-Zip et al</td>
          </tr>
       <br> 
          <tr>
            <td rowspan="2">*.zip</td>
            <td>Unix/Linux/Macr</td>
            <td>Command line: unzip *.zip</td>
          </tr>

          <tr>
            <td>Windows</td>
            <td>Software: such as WinRAR, 7-Zip et al</td>
          </tr>
        </table>
<br>
        <table class="gy2">
          <tr>
            <th>File.type</th>
            <th>Document.description</th>
            <th>Open.mode</th>
          </tr>
</br>

          <tr>
            <td rowspan="2">file.readme.pdf</td>
            <td rowspan="2">Results' construction files that help customers to undersdand all the results files better.</td>
            <td>Windows/Mac user use Adobe Reader/Foxit reader/web browser to open.</td>
          </tr>

          <tr>
            <td>Unix/Linux user use evince command to open.</td>
          </tr>

          <tr>
            <td rowspan="2">file.fa/.fasta</td>
            <td rowspan="2">Fasta format is a text-based format for representing sequences.</td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td rowspan="2">file.fq/.fastq</td>
            <td rowspan="2">Text file that contains sequence data delivered from a sequencer. </td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td>file.vcf</td>
            <td>VCF is a flexible and extendable line-oriented text format developed by the 1000 Genomes Project for releases of SNVs, INDELs, copy number variants and structural variants.</td>
            <td>Unix/Linux/Mac user use less or more command;</td>
          </tr>

          <tr>
            <td rowspan="2">file.pdf/.svg</td>
            <td rowspan="2">Figure result files; vectorgraph, can be zoomed in or out. It is very convenient for customer to view or edit with Adobe Illustrator. The figure can be used in paper for publish.</td>
            <td>unix/Linux user use evince command to open.</td>
          </tr>

          <tr>
            <td>Windows/Mac user use Adobe Reader/Foxit reader/web browser to open.</td>
          </tr>


          <tr>
            <td rowspan="2">file.txt/xls</td>
            <td rowspan="2">Table result file; files are tab delimited.</td>
            <td>Unix/Linux/Mac user use less or more command</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al, also can use Microsoft Excel to open.</td>
          </tr>

          <tr>
            <td rowspan="2">file.gff/gtf</td>
            <td rowspan="2">GFF/GTF is general feature format.</td>
            <td>Unix/Linux/Mac user use less or more command;</td>
          </tr>

          <tr>
            <td>Windows user use editor Editplus/Notepad++ et al.</td>
          </tr>

          <tr>
            <td rowspan="2">file.png</td>
            <td rowspan="2">Image file, bitmap with loseless compression.</td>
            <td>Unix/Linux/Mac user use display command;</td>
          </tr>

          <tr>
            <td>Windows user use Image browser，such as photoshop et al.</td>
          </tr>

          </table>

</div>
<br>
<!------/div------>
<!-------------------------------------------- 参考文献 ------------------------------------------->



<div id="page">
	<hr />

	<h2 id="reference1">6 References</h2>

<p class="ref">
	Altschul SF, Madden TL, Schäffer AA, et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 1997;25(17):3389-3402. doi:10.1093/nar/25.17.3389<br><br>
	Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using DIAMOND. Nat Methods. 2015;12(1):59-60. doi:10.1038/nmeth.3176<br><br>
	Chepelev I, Wei G, Tang Q, Zhao K. Detection of single nucleotide variations in expressed exons of the human genome using RNA-Seq. Nucleic Acids Res. 2009;37(16):e106. doi:10.1093/nar/gkp507<br><br>
	Cock PJ, Fields CJ, Goto N, Heuer ML, Rice PM. The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants. Nucleic Acids Res. 2010;38(6):1767-1771. doi:10.1093/nar/gkp1137<br><br>
	Davidson NM, Oshlack A. Corset: enabling differential gene expression analysis for de novo assembled transcriptomes. Genome Biol. 2014;15(7):410. Published 2014 Jul 26. doi:10.1186/s13059-014-0410-6<br><br>
	Dillies MA, Rau A, Aubert J, et al. A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. Brief Bioinform. 2013;14(6):671-683. doi:10.1093/bib/bbs046<br><br>
	ENCODE Project Consortium The ENCODE (ENCyclopedia of DNA elements) project. Science. 2004;306(5696):636-640.
	Finn RD, Tate J, Mistry J, et al. The Pfam protein families database. Nucleic Acids Res. 2008;36(Database issue):D281-D288. doi:10.1093/nar/gkm960<br><br>
	Götz S, García-Gómez JM, Terol J, et al. High-throughput functional annotation and data mining with the Blast2GO suite. Nucleic Acids Res. 2008;36(10):3420-3435. doi:10.1093/nar/gkn176<br><br>
	Grabherr MG, Haas BJ, Yassour M, et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotechnol. 2011;29(7):644-652. Published 2011 May 15. doi:10.1038/nbt.1883<br><br>
	Jiang L, Schlesinger F, Davis CA, et al. Synthetic spike-in standards for RNA-seq experiments. Genome Res. 2011;21(9):1543-1551. doi:10.1101/gr.121095.111<br><br>
	Jin J, Zhang H, Kong L, Gao G, Luo J. PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors. Nucleic Acids Res. 2014;42(Database issue):D1182-D1187. doi:10.1093/nar/gkt1016<br><br>
	Kanehisa M, Araki M, Goto S, et al. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 2008;36(Database issue):D480-D484. doi:10.1093/nar/gkm882<br><br>
	Li B, Dewey CN. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics. 2011;12:323. Published 2011 Aug 4. doi:10.1186/1471-2105-12-323<br><br>
	Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550. doi:10.1186/s13059-014-0550-8<br><br>
	Mao X, Cai T, Olyarchuk JG, Wei L. Automated genome annotation and pathway identification using the KEGG Orthology (KO) as a controlled vocabulary. Bioinformatics. 2005;21(19):3787-3793. doi:10.1093/bioinformatics/bti430<br><br>
	Moriya Y, Itoh M, Okuda S, Yoshizawa AC, Kanehisa M. KAAS: an automatic genome annotation and pathway reconstruction server. Nucleic Acids Res. 2007;35(Web Server issue):W182-W185. doi:10.1093/nar/gkm321<br><br>
	Pérez-Rodríguez P, Riaño-Pachón DM, Corrêa LG, Rensing SA, Kersten B, Mueller-Roeber B. PlnTFDB: updated content and new features of the plant transcription factor database. Nucleic Acids Res. 2010;38(Database issue):D822-D827. doi:10.1093/nar/gkp805<br><br>
	Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26(1):139-140. doi:10.1093/bioinformatics/btp616<br><br>
	Shannon P, Markiel A, Ozier O, et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2003;13(11):2498-2504. doi:10.1101/gr.1239303<br><br>
	Trapnell C, Williams BA, Pertea G, et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28(5):511-515. doi:10.1038/nbt.1621<br><br>
	Young MD, Wakefield MJ, Smyth GK, Oshlack A. Gene ontology analysis for RNA-seq: accounting for selection bias. Genome Biol. 2010;11(2):R14. doi:10.1186/gb-2010-11-2-r14<br><br>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
</p>
</body>
</html>
