
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
  <META NAME="Author" CONTENT="liuruolin@novogene.cn">
    <META NAME="Version" CONTENT="20171224" "Silent Night ^_^">
    <title>Ref3 ReadMe</title>
  <!--tree/start-->
  <script src="https://cdn.bootcss.com/jquery-treetable/3.2.0/jquery.treetable.min.js"></script>
  <link href="https://cdn.bootcss.com/jquery-treetable/3.2.0/css/jquery.treetable.min.css" rel="stylesheet">
  <link href="https://cdn.bootcss.com/jquery-treetable/3.2.0/css/jquery.treetable.theme.default.css" rel="stylesheet">
  <!--tree/end-->
  <style type='text/css'>
  .task-list { list-style-type: none; }
    .task-list-item { position: relative; padding-left: 2em; }
    .task-list-item input { position: absolute; top: 0px; left: 0px; }
   </style>
  </head>
  <body>
    <div id="main">
      <p><h3 align="center"  style="color:#696969" >Ref3 Readme</h3></p>
      <table id="example-advanced">
        <caption>
          <a href="#" onclick="jQuery('#example-advanced').treetable('expandAll'); return false;">Expand all
          <a href="#" onclick="jQuery('#example-advanced').treetable('collapseAll'); return false;">Collapse all
        </caption>
        <thead>
          <tr>
            <th>Result</th>
            <th>Contents</th>


            
      <th>Type</th>
          </tr>
        </thead>
      <tbody>

  <tr data-tt-id='2'><td><span class='folder'><a href='./01.SMRT'>01.SMRT</span></td><td>【Iso-seq result file】</td><td>folder</td></tr>   
  <tr data-tt-id='2-1' data-tt-parent-id='2'><td><span class='file'><a href='./01.SMRT/01.SMRT--ReadMe.pdf'>01.SMRT--ReadMe.pdf</span></td><td>【SMRT Delivery Results Catalog Instructions】</td><td>table</td></tr>
  <tr data-tt-id='2-4' data-tt-parent-id='2'><td><span class='folder'>Sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='2-4-1' data-tt-parent-id='2-4'><td><span class='folder'>ccs</span></td><td>【Circular Consensus Sequence Catalog】</td><td>folder</td></tr> 
  <tr data-tt-id='2-4-1-1' data-tt-parent-id='2-4-1'><td><span class='file'>Sample.ccs.fastq.zip</span></td><td>【Circular consensus sequence, fastq format】</td><td>file</td></tr> 
    <tr data-tt-id='2-4-2' data-tt-parent-id='2-4'><td><span class='folder'>classify</span></td><td>【transcript taxonomy】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-2-1' data-tt-parent-id='2-4-2'><td><span class='file'>Sample.isoseq_flnc.fasta</span></td><td>【Full-length non-chimeric sequence file, fasta format】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-3' data-tt-parent-id='2-4'><td><span class='folder'>cluster</span></td><td>【Transcript Clustering Results Catalog】</td><td>folder</td></tr>
    <tr data-tt-id='2-4-3-1' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_hq.fasta</span></td><td>【Modified high-quality transcript sequences in fasta format】</td><td>file</td></tr>
    <tr data-tt-id='2-4-3-2' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_lq.fast</span></td><td>【Modified low-quality transcript sequences, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='2-4-3-3' data-tt-parent-id='2-4-3'><td><span class='file'>Sample.polished_consensus.fasta</span></td><td>【Modified consensus transcript sequence, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4' data-tt-parent-id='2-4'><td><span class='folder'>stat</span></td><td>【Table of Transcript Statistical Results】</td><td>folder</td></tr>  
  <tr data-tt-id='2-4-4-1' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_distri.xls</span></td><td>【Polymerase_reads length distribution statistical results】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-2' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_distri.{png|pdf}</span></td><td>【Polymerase_reads length distribution graph】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-4' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.polymerase_reads_stat.xls</span></td><td>【polymerase_reads statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-5' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_stat.xls</span></td><td>【subreads statistical results】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-6' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_distri.xls</span></td><td>【Statistical results of subreads length distribution】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-7' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.subreads_distri.{png|pdf}</span></td><td>【subreads length distribution chart】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-9' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_stat.xls</span></td><td>【The statistical results of the length distribution of the circular consensus sequence】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-10' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_distri.xls</span></td><td>【Circular Consensus Sequence Statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-11' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.ccs_distri.{png|pdf}</span></td><td>【Circular Consensus Sequence Length Distribution】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-13' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_stat.xls</span></td><td>【Statistical results of full-length non-chimeric sequences】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-14' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_distri.xls</span></td><td>【Statistical results of length distribution of full-length non-chimeric sequences】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-15' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.flnc_distri.{png|pdf}</span></td><td>【Full-length non-chimeric sequence length distribution map】</td><td>file</td></tr>

  <tr data-tt-id='2-4-4-18' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_stat.xls</span></td><td>【Consensus sequence statistics】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-19' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_distri.xls</span></td><td>【Statistical results of consensus sequence length distribution】</td><td>file</td></tr>
  <tr data-tt-id='2-4-4-20' data-tt-parent-id='2-4-4'><td><span class='file'>Sample.consensus_distri.{png|pdf}</span></td><td>【Consensus sequence length distribution graph】</td><td>file</td></tr>


  <tr data-tt-id='3'><td><span class='folder'><a href='./02.Correct'>02.Correct</span></td><td>【Second-generation data correction catalog】</td><td>folder</td></tr>
  <tr data-tt-id='3-1' data-tt-parent-id='3'><td><span class='file'><a href='./02.Correct/02.Correct_ReadMe.pdf'>02.Correct_ReadMe.pdf</span></td><td>【02.Correct delivery result catalog description】</td><td>table</td></tr>
  <tr data-tt-id='3-2' data-tt-parent-id='3'><td><span class='folder'>Sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='3-2-1' data-tt-parent-id='3-2'><td><span class='file'>sample.corrected.fa</span></td><td>【Data corrected flnc sequence in fasta format】</td><td>file</td></tr>
  <tr data-tt-id='3-2-2' data-tt-parent-id='3-2'><td><span class='file'>sample.correct_stat.xls</span></td><td>【Statistical table of data before and after data correction】</td><td>file</td></tr>

  <tr data-tt-id='4'><td><span class='folder'><a href='./03.GMAP'>03.GMAP</span></td><td>【Transcript mapping reference genome comparison directory】</td><td>folder</td></tr>
  <tr data-tt-id='4-1' data-tt-parent-id='4'><td><span class='file'><a href='./03.GMAP/03.GMAP_ReadMe.pdf'>03.GMAP_ReadMe.pdf</span></td><td>【03. GMAP Delivery Results Catalog Description】</td><td>table</td></tr>
  <tr data-tt-id='4-2' data-tt-parent-id='4'><td><span class='folder'>Sample*</span></td><td>【Sample catalog】</td><td>folder</td></tr>

  <tr data-tt-id='4-2-1' data-tt-parent-id='4-2'><td><span class='file'>sample.mapping_stat.xls</span></td><td>【The result display table after comparing gmap to the reference genome】</td><td>folder</td></tr>

  <tr data-tt-id='4-2-3' data-tt-parent-id='4-2'><td><span class='file'>saturation.{png|pdf}</span></td><td>【Saturation plot for full-length threads and genes】</td><td>folder</td></tr>
  <tr data-tt-id='4-2-4' data-tt-parent-id='4-2'><td><span class='file'>sample.pie.{png|pdf}</span></td><td>【Pie charts of the results of gmap comparison to the reference genome】</td><td>folder</td></tr>


  <tr data-tt-id='4-2-6' data-tt-parent-id='4-2'><td><span class='file'>sample.density.{png|pdf}</span></td><td>【Reads comparison reference genome density distribution map】</td><td>folder</td></tr>
  <tr data-tt-id='4-2-7' data-tt-parent-id='4-2'><td><span class='file'>sample.ReadsVSlength.{png|pdf}</span></td><td>【The relationship between the number of Reads and the length of the chromosome】</td><td>folder</td></tr>
  <tr data-tt-id='4-2-8' data-tt-parent-id='4-2'><td><span class='file'>sample.range.{png|svg}</span></td><td>【The display of coverage and identity results after comparing gmap to the reference genome】</td><td>folder</td></tr>

  <tr data-tt-id='4-2-9' data-tt-parent-id='4-2'><td><span class='file'>sample.aligned.bam</span></td><td>【The alignment result after the consensus sequence is aligned to the reference genome, bam format】</td><td>folder</td></tr>
  <tr data-tt-id='4-2-10' data-tt-parent-id='4-2'><td><span class='file'>sample.junction.pie.{png|pdf}</span></td><td>【gmap splice site classification map】</td><td>folder</td></tr>

  <tr data-tt-id='4-2-11' data-tt-parent-id='4-2'><td><span class='file'>sample.aligned.bam.bai</span></td><td>【The index of the bam file after the consensus sequence is aligned to the reference genome】</td><td>folder</td></tr>

  <tr data-tt-id='5'><td><span class='folder'><a href='./04.Structure'>04.Structure</span></td><td>【Structural Annotation Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-1' data-tt-parent-id='5'><td><span class='file'><a href='./04.Structure/04.Structure_ReadMe.pdf'>04.Structure_ReadMe.pdf</span></td><td>【04. Description of Structure Delivery Results Catalog】</td><td>file</td></tr>
  <tr data-tt-id='5-2' data-tt-parent-id='5'><td><span class='folder'>01.transcripts</span></td><td>【Transcript Identification Analysis Results】</td><td>folder</td></tr>
  <tr data-tt-id='5-2-1' data-tt-parent-id='5-2'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>

<tr data-tt-id='5-2-1-1' data-tt-parent-id='5-2-1'><td><span class='file'>sample.gtf</span></td><td>【Sample reference gene set gtf file】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-2' data-tt-parent-id='5-2-1'><td><span class='file'>sample.fasta</span></td><td>【Transcript sequences corrected for clustering by the tapis pipeline, fasta format】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-3' data-tt-parent-id='5-2-1'><td><span class='file'>sample.id2id.xls</span></td><td>【Correspondence file of transcript id and gene id of the sample】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-4' data-tt-parent-id='5-2-1'><td><span class='file'>sample.novelgene.xls</span></td><td>【New gene outcomes predicted by the tapis pipeline】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-5' data-tt-parent-id='5-2-1'><td><span class='file'>sample.noveliso.xls</span></td><td>【New transcript results predicted by the tapis pipeline】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-6' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isotype.{png|pdf}</span></td><td>【Transcript taxonomy map of samples】</td><td>file</td></tr>

<tr data-tt-id='5-2-1-8' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isoform_stat.xls</span></td><td>【Transcript sequence statistics results after clustering correction by tapis process】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-9' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isoform_length_compare.xls</span></td><td>【Three-generation transcripts and reference transcript length distribution】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-10' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isoform_compare.{png|pdf}</span></td><td>【Transcript length distribution map】</td><td>file</td></tr>

<tr data-tt-id='5-2-1-12' data-tt-parent-id='5-2-1'><td><span class='file'>sample.gene_isoform_compare_ratio.xls</span></td><td>【Gene-to-transcript distribution ratio】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-13' data-tt-parent-id='5-2-1'><td><span class='file'>sample.gene_isoform_compare.{png|pdf}</span></td><td>【Gene-corresponding transcript distribution map】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-15' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isoform_exon_compare_ratio.xls</span></td><td>【Transcript-to-exon distribution ratio】</td><td>file</td></tr>
<tr data-tt-id='5-2-1-16' data-tt-parent-id='5-2-1'><td><span class='file'>sample.isoform_exon_compare.{png|pdf}</span></td><td>【Transcript-corresponding exon distribution map】</td><td>file</td></tr>

<tr data-tt-id='5-2-2' data-tt-parent-id='5-2'><td><span class='folder'>transcript_venn</span></td><td>【Different sample transcript comparison directory, the number of samples is required to be greater than 2】</td><td>folder</td></tr>
  <tr data-tt-id='5-2-2-1' data-tt-parent-id='5-2-2'><td><span class='file'>sample1-yes-sample2-yes.txt</span></td><td>【Transcript ids present in sample1 that also exist in sample2】</td><td>file</td></tr>
  <tr data-tt-id='5-2-2-2' data-tt-parent-id='5-2-2'><td><span class='file'>sample1-yes-sample2-no.txt</span></td><td>【Transcript ids present in sample1 and absent in sample2】</td><td>file</td></tr>
  <tr data-tt-id='5-2-2-3' data-tt-parent-id='5-2-2'><td><span class='file'>sample1-no-sample2-yes.txt</span></td><td>【A transcript id that is also present in sample2 does not exist in sample1】</td><td>file</td></tr>
  <tr data-tt-id='5-2-2-4' data-tt-parent-id='5-2-2'><td><span class='file'>all.txt</span></td><td>【All transcript ids in sample1 and sample2】</td><td>file</td></tr>
  <tr data-tt-id='5-2-2-5' data-tt-parent-id='5-2-2'><td><span class='file'>sample1_sample2.DEG_Venn_diagram.{pdf|png}</span></td><td>【Transcript Venn plots for sample1 and sample2】</td><td>file</td></tr>


  <tr data-tt-id='5-3' data-tt-parent-id='5'><td><span class='folder'>02.AS</span></td><td>【Alternative splicing analysis results】</td><td>folder</td></tr>
  <tr data-tt-id='5-3-1' data-tt-parent-id='5-3'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-3-1-1' data-tt-parent-id='5-3-1'><td><span class='file'>sample_stat_out.xls</span></td><td>【Statistical results file of the ratio of seven alternative splicing modes of the sample】</td><td>file</td></tr>
  <tr data-tt-id='5-3-1-2' data-tt-parent-id='5-3-1'><td><span class='file'>sample_AS_total.xls</span></td><td>【Statistical results after alternative splicing】</td><td>file</td></tr>
  <tr data-tt-id='5-3-1-3' data-tt-parent-id='5-3-1'><td><span class='file'>Sample.pi.{png|pdf}</span></td><td>【Proportion chart of seven alternative splicing modes】</td><td>file</td></tr>

  <tr data-tt-id='5-4' data-tt-parent-id='5'><td><span class='folder'>03.APA</span></td><td>【Variable polyadenylation outcome】</td><td>folder</td></tr>
  <tr data-tt-id='5-4-1' data-tt-parent-id='5-4'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-3-1-1' data-tt-parent-id='5-4-1'><td><span class='file'>polyA_stat.xls</span></td><td>【Statistical results after polyA processing】</td><td>file</td></tr>
  <tr data-tt-id='5-3-1-2' data-tt-parent-id='5-4-1'><td><span class='file'>polyA.{png|svg}</span></td><td>【polyA result chart】</td><td>file</td></tr>
  <tr data-tt-id='5-3-1-3' data-tt-parent-id='5-4-1'><td><span class='file'>polyA_summary.xls</span></td><td>【polyA site results predicted by the tapis pipeline】</td><td>file</td></tr>

  <tr data-tt-id='5-5' data-tt-parent-id='5'><td><span class='folder'>04.Annotation</span></td><td>【database comment】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2' data-tt-parent-id='5-5'><td><span class='folder'>Unmap_anno</span></td><td>【Unmapped Transcript Annotation】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1' data-tt-parent-id='5-5-2'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-1' data-tt-parent-id='5-5-2-1'><td><span class='folder'>NR</span></td><td>【NR database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-1-1' data-tt-parent-id='5-5-2-1-1'><td><span class='file'>sample.NR.filter.m8.xls</span></td><td>【The highest-scoring alignment between the consensus sequence and the NR database】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-1-2' data-tt-parent-id='5-5-2-1-1'><td><span class='file'>sample.NR.filter.m8.anno.xls</span></td><td>【Consensus sequence and NR database comparison result species information annotation】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-1-3' data-tt-parent-id='5-5-2-1-1'><td><span class='file'>sample.nr.species.xls</span></td><td>【NR database comparison annotation results species distribution list】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-1-4' data-tt-parent-id='5-5-2-1-1'><td><span class='file'>sample.nr.anno.{pdf|png}</span></td><td>【Species distribution map of NR database comparison annotation results.pdfTOP20】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-2' data-tt-parent-id='5-5-2-1'><td><span class='folder'>GO</span></td><td>【GO database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-2-1' data-tt-parent-id='5-5-2-1-2'><td><span class='file'>sample.GO_classification_count.txt</span></td><td>【GO database annotation statistical results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-2-2' data-tt-parent-id='5-5-2-1-2'><td><span class='file'>sample.GO.anno.xls</span></td><td>【List of GO annotation information】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-2-3' data-tt-parent-id='5-5-2-1-2'><td><span class='file'>sample.GO_classification_bar.{pdf|png}</span></td><td>【GO Annotated Taxonomy Map】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-3' data-tt-parent-id='5-5-2-1'><td><span class='folder'>KEGG</span></td><td>【KEGG database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-3-1' data-tt-parent-id='5-5-2-1-3'><td><span class='file'>sample.KEGG.filter.m8.xls</span></td><td>【A list of the highest scoring results for consensus sequences compared to the KEGG database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-3-2' data-tt-parent-id='5-5-2-1-3'><td><span class='file'>sample.KEGG.filter.m8.anno.xls</span></td><td>【Consensus sequence and KEGG database alignment annotation results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-3-3' data-tt-parent-id='5-5-2-1-3'><td><span class='file'>sample.kegg.anno.stat.xls</span></td><td>【KEGG database annotation results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-3-4' data-tt-parent-id='5-5-2-1-3'><td><span class='file'>sample.concensus.kegg.num.{pdf|png}</span></td><td>【KEGG pathway annotation gene number result picture pdf|png format】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-4' data-tt-parent-id='5-5-2-1'><td><span class='folder'>PFAM</span></td><td>【PFAM database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-4-1' data-tt-parent-id='5-5-2-1-4'><td><span class='file'>sample.pfam.anno.xls</span></td><td>【Protein annotation results of pfam database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-5' data-tt-parent-id='5-5-2-1'><td><span class='folder'>NT</span></td><td>【NT database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-5-1' data-tt-parent-id='5-5-2-1-5'><td><span class='file'>sample.NT.filter.m8.xls</span></td><td>【The alignment with the highest score for the consensus sequence alignment】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-5-2' data-tt-parent-id='5-5-2-1-5'><td><span class='file'>sample.NT.filter.m8.anno.xls</span></td><td>【Consensus sequence and NT database comparison results Species information annotation results】</td><td>file</td></tr>

  <tr data-tt-id='5-5-2-1-6' data-tt-parent-id='5-5-2-1'><td><span class='folder'>KOG</span></td><td>【KOG database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-6-1' data-tt-parent-id='5-5-2-1-6'><td><span class='file'>sample.KOG.filter.m8.xls</span></td><td>【The highest-scoring alignment between the consensus sequence and the KOG database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-6-2' data-tt-parent-id='5-5-2-1-6'><td><span class='file'>sample.KOG.filter.m8.anno.xls</span></td><td>【Protein classification information annotation of consensus sequence and KOG database comparison results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-6-3' data-tt-parent-id='5-5-2-1-6'><td><span class='file'>sample.KOG.level.xls</span></td><td>【KOG database feature annotation Annotate information by KOG number】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-6-4' data-tt-parent-id='5-5-2-1-6'><td><span class='file'>sample.KOG.class.xls</span></td><td>【KOG database functional annotation protein classification 26 types of annotation information】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-6-5' data-tt-parent-id='5-5-2-1-6'><td><span class='file'>sample.KOG.class.{pdf|png}</span></td><td>【KOG database functional annotation protein classification map.pdf|png】</td><td>file</td></tr>

  <tr data-tt-id='5-5-2-1-7' data-tt-parent-id='5-5-2-1'><td><span class='folder'>Swissprot</span></td><td>【Swissprot database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-2-1-7-1' data-tt-parent-id='5-5-2-1-7'><td><span class='file'>sample.Swissprot.filter.m8.xls</span></td><td>【Consensus sequence alignment with the highest score in the SwissProt database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-7-2' data-tt-parent-id='5-5-2-1-7'><td><span class='file'>sample.Swissprot.filter.m8.anno.xls</span></td><td>【Annotation information of SwissProt database blast comparison results】</td><td>file</td></tr>

  <tr data-tt-id='5-5-2-1-1' data-tt-parent-id='5-5-2-1'><td><span class='file'>sample.anno.xls</span></td><td>【Annotation results of the seven major databases】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-2' data-tt-parent-id='5-5-2-1'><td><span class='file'>sample.anno.stat.xls</span></td><td>【Functional annotation results of each transcript in each database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-2-1-3' data-tt-parent-id='5-5-2-1'><td><span class='file'>sample.anno.{png|svg}</span></td><td>【Statistical chart of annotation results of each database】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1' data-tt-parent-id='5-5'><td><span class='folder'>Novel_anno</span></td><td>【new gene annotation】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1' data-tt-parent-id='5-5-1'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-1' data-tt-parent-id='5-5-1-1'><td><span class='folder'>NR</span></td><td>【NR database annotation results】</td><td>folder</td></tr>


  <tr data-tt-id='5-5-1-1-1-1' data-tt-parent-id='5-5-1-1-1'><td><span class='file'>sample.NR.filter.m8.xls</span></td><td>【The highest-scoring alignment between the consensus sequence and the NR database】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-1-2' data-tt-parent-id='5-5-1-1-1'><td><span class='file'>sample.NR.filter.m8.anno.xls</span></td><td>【Consensus sequence and NR database comparison result species information annotation】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-1-3' data-tt-parent-id='5-5-1-1-1'><td><span class='file'>sample.nr.species.xls</span></td><td>【NR database comparison annotation results species distribution list】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-1-4' data-tt-parent-id='5-5-1-1-1'><td><span class='file'>sample.nr.anno.{pdf|png}</span></td><td>【Species distribution map of NR database comparison annotation results.pdfTOP20】</td><td>folder</td></tr>

  <tr data-tt-id='5-5-1-1-2' data-tt-parent-id='5-5-1-1'><td><span class='folder'>GO</span></td><td>【GO database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-2-1' data-tt-parent-id='5-5-1-1-2'><td><span class='file'>sample.GO_classification_count.txt</span></td><td>【GO database annotation statistical results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-2-2' data-tt-parent-id='5-5-1-1-2'><td><span class='file'>sample.GO.anno.xls</span></td><td>【List of GO annotation information】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-2-3' data-tt-parent-id='5-5-1-1-2'><td><span class='file'>sample.GO_classification_bar.{pdf|png}</span></td><td>【GO Annotated Taxonomy Map】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-3' data-tt-parent-id='5-5-1-1'><td><span class='folder'>KEGG</span></td><td>【KEGG database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-3-1' data-tt-parent-id='5-5-1-1-3'><td><span class='file'>sample.KEGG.filter.m8.xls</span></td><td>【A list of the highest scoring results for consensus sequences compared to the KEGG database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-3-2' data-tt-parent-id='5-5-1-1-3'><td><span class='file'>sample.KEGG.filter.m8.anno.xls</span></td><td>【Consensus sequence and KEGG database alignment annotation results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-3-3' data-tt-parent-id='5-5-1-1-3'><td><span class='file'>sample.kegg.anno.stat.xls</span></td><td>【KEGG database annotation results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-3-4' data-tt-parent-id='5-5-1-1-3'><td><span class='file'>sample.concensus.kegg.num.{pdf|png}</span></td><td>【KEGG pathway annotation gene number result picture pdf|png format】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-4' data-tt-parent-id='5-5-1-1'><td><span class='folder'>PFAM</span></td><td>【PFAM database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-4-1' data-tt-parent-id='5-5-1-1-4'><td><span class='file'>sample.pfam.anno.xls</span></td><td>【Protein annotation results of pfam database】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-5' data-tt-parent-id='5-5-1-1'><td><span class='folder'>NT</span></td><td>【NT database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-5-1' data-tt-parent-id='5-5-1-1-5'><td><span class='file'>sample.NT.filter.m8.xls</span></td><td>【The alignment with the highest score for the consensus sequence alignment】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-5-2' data-tt-parent-id='5-5-1-1-5'><td><span class='file'>sample.NT.filter.m8.anno.xls</span></td><td>【Consensus sequence and NT database comparison results Species information annotation results】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-6' data-tt-parent-id='5-5-1-1'><td><span class='folder'>KOG</span></td><td>【KOG database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-6-1' data-tt-parent-id='5-5-1-1-6'><td><span class='file'>sample.KOG.filter.m8.xls</span></td><td>【The highest-scoring alignment between the consensus sequence and the KOG database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-6-2' data-tt-parent-id='5-5-1-1-6'><td><span class='file'>sample.KOG.filter.m8.anno.xls</span></td><td>【Protein classification information annotation of consensus sequence and KOG database comparison results】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-6-3' data-tt-parent-id='5-5-1-1-6'><td><span class='file'>sample.KOG.level.xls</span></td><td>【KOG database feature annotation Annotate information by KOG number】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-6-4' data-tt-parent-id='5-5-1-1-6'><td><span class='file'>sample.KOG.class.xls</span></td><td>【KOG database functional annotation protein classification 26 types of annotation information】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-6-5' data-tt-parent-id='5-5-1-1-6'><td><span class='file'>sample.KOG.class.{pdf|png}</span></td><td>【KOG database functional annotation protein classification map.pdf|png】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-7' data-tt-parent-id='5-5-1-1'><td><span class='folder'>Swissprot</span></td><td>【Swissprot database annotation results】</td><td>folder</td></tr>
  <tr data-tt-id='5-5-1-1-7-1' data-tt-parent-id='5-5-1-1-7'><td><span class='file'>sample.Swissprot.filter.m8.xls</span></td><td>【Consensus sequence alignment with the highest score in the SwissProt database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-7-2' data-tt-parent-id='5-5-1-1-7'><td><span class='file'>sample.Swissprot.filter.m8.anno.xls</span></td><td>【Annotation information of SwissProt database blast comparison results】</td><td>file</td></tr>

  <tr data-tt-id='5-5-1-1-1' data-tt-parent-id='5-5-1-1'><td><span class='file'>sample.anno.xls</span></td><td>【Annotation results of the seven major databases】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-2' data-tt-parent-id='5-5-1-1'><td><span class='file'>sample.anno.stat.xls</span></td><td>【Functional annotation results of each transcript in each database】</td><td>file</td></tr>
  <tr data-tt-id='5-5-1-1-3' data-tt-parent-id='5-5-1-1'><td><span class='file'>sample.anno.{png|svg}</span></td><td>【Statistical chart of annotation results of each database】</td><td>file</td></tr>



  <tr data-tt-id='5-6' data-tt-parent-id='5'><td><span class='folder'>05.fusion</span></td><td>【Fusion gene identification results】</td><td>folder</td></tr>
  <tr data-tt-id='5-6-1' data-tt-parent-id='5-6'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='5-6-1-1' data-tt-parent-id='5-6-1'><td><span class='file'>sample.fusion.anno.xls</span></td><td>【Statistical fusion gene results】</td><td>file</td></tr>
  <tr data-tt-id='5-6-1-1' data-tt-parent-id='5-6-1'><td><span class='file'>sample.fusion.fa</span></td><td>【Fusion gene sequence, fasta format】</td><td>file</td></tr>
  <tr data-tt-id='5-6-1-1' data-tt-parent-id='5-6-1'><td><span class='file'>sample.circos.{png|svg}</span></td><td>【Fusion gene circos diagram】</td><td>file</td></tr>


  <tr data-tt-id='6'><td><span class='folder'><a href='./05.TF'>05.TF</span></td><td>【Transcription factor prediction results】</td><td>folder</td></tr>
  <tr data-tt-id='6-1' data-tt-parent-id='6'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>
  <tr data-tt-id='6-1-1' data-tt-parent-id='6-1'><td><span class='file'>sample.transcription_factor.txt</span></td><td>【Transcription Factor Results】</td><td>file</td></tr>
  <tr data-tt-id='6-1-3' data-tt-parent-id='6-1'><td><span class='file'>sample.TF.{png|svg}</span></td><td>【Transcription factor frequency statistics】</td><td>file</td></tr>
  <tr data-tt-id='6-1-4' data-tt-parent-id='6-1'><td><span class='file'>sample.tf_stat.xls</span></td><td>【Statistical Table of Transcription Factor Results】</td><td>file</td></tr>
  <tr data-tt-id='6-2' data-tt-parent-id='6'><td><span class='file'><a href='./05.TF/05.TF_ReadMe.pdf'>05.TF_ReadMe.pdf</span></td><td>【05. Description of TF Delivery Results Catalog】</td><td>file</td></tr>

  <tr data-tt-id='7'><td><span class='folder'><a href='./06.LncRNA'>06.LncRNA</span></td><td>【LncRNA Analysis Results Catalog】</td><td>folder</td></tr>
  <tr data-tt-id='7-1' data-tt-parent-id='7'><td><span class='folder'>sample</span></td><td>【Sample catalog】</td><td>folder</td></tr>

  <tr data-tt-id='7-1-1' data-tt-parent-id='7-1'><td><span class='file'>sample.cnci.xls</span></td><td>【CNCI Software Coding Potential Prediction Results】</td><td>file</td></tr>
  <tr data-tt-id='7-1-2' data-tt-parent-id='7-1'><td><span class='file'>sample.cpc.xls</span></td><td>【CPC Software Coding Potential Prediction Results】</td><td>file</td></tr>
<tr data-tt-id='7-1-3' data-tt-parent-id='7-1'><td><span class='file'>sample.pfam.xls</span></td><td>【pfam software coding potential prediction results】</td><td>file</td></tr>
<tr data-tt-id='7-1-4' data-tt-parent-id='7-1'><td><span class='file'>sample.plek.xls</span></td><td>【plek software coding potential prediction results】</td><td>file</td></tr>

<tr data-tt-id='7-1-6' data-tt-parent-id='7-1'><td><span class='file'>sample.venn.{png|pdf}</span></td><td>【The Venn diagram of LncRNA predicted by each coding potential prediction software in pdf|png format】</td><td>file</td></tr>
<tr data-tt-id='7-1-7' data-tt-parent-id='7-1'><td><span class='file'>sample.lnc.fa</span></td><td>【LncRNA file, fasta format】</td><td>file</td></tr>
<tr data-tt-id='7-1-9' data-tt-parent-id='7-1'><td><span class='file'>sample.lncRNA_classification.{png|pdf}</span></td><td>【Pie chart of lncRNA classification in pdf|png format】</td><td>file</td></tr>
<tr data-tt-id='7-1-10' data-tt-parent-id='7-1'><td><span class='file'>sample.lnc.classify.txt</span></td><td>【lncRNA classification and identification results】</td><td>file</td></tr>
<tr data-tt-id='7-1-11' data-tt-parent-id='7-1'><td><span class='file'>sample.lnc.stat.xls</span></td><td>【Statistical Table of LncRNA Classification】</td><td>file</td></tr>
<tr data-tt-id='7-1-12' data-tt-parent-id='7-1'><td><span class='file'>sample.lnc.xls</span></td><td>【Each coding potential prediction software predicts the results of LncRNA】</td><td>file</td></tr>
<tr data-tt-id='7-1-15' data-tt-parent-id='7-1'><td><span class='file'>sample.isoform.stat.xls</span></td><td>【LncRNA Transcript Statistics】</td><td>file</td></tr>
<tr data-tt-id='7-1-16' data-tt-parent-id='7-1'><td><span class='file'>sample.length.density.{png|pdf}</span></td><td>【LncRNA length distribution density map】</td><td>file</td></tr>
<tr data-tt-id='7-1-18' data-tt-parent-id='7-1'><td><span class='file'>sample.isoform_exon.{png|pdf}</span></td><td>【LncRNA exon number distribution histogram】</td><td>file</td></tr>
<tr data-tt-id='7-2' data-tt-parent-id='7'><td><span class='file'><a href='./06.LncRNA/06.LncRNA_ReadMe.pdf'>06.LncRNA_ReadMe.pdf</span></td><td>【06. LncRNA Delivery Results Catalog Description】</td><td>file</td></tr>


     </tbody>
      </table>    
    </div>
  
    <script src="http://www.jq22.com/jquery/jquery-1.10.2.js"></script>
    <script src="https://cdn.bootcss.com/jquery-treetable/3.2.0/jquery.treetable.js"></script>
    <script>
      $("#example-basic").treetable({ expandable: true });

      $("#example-basic-static").treetable();

      $("#example-basic-expandable").treetable({ expandable: true });

      $("#example-advanced").treetable({ expandable: true });

      // Highlight selected row
      $("#example-advanced tbody").on("mousedown", "tr", function() {
        $(".selected").not(this).removeClass("selected");
        $(this).toggleClass("selected");
      });

      // Drag & Drop Example Code
      $("#example-advanced .file, #example-advanced .folder").draggable({
        helper: "clone",
        opacity: .75,
        refreshPositions: true, // Performance?
        revert: "invalid",
        revertDuration: 300,
        scroll: true
      });

      $("#example-advanced .folder").each(function() {
        $(this).parents("#example-advanced tr").droppable({
          accept: ".file, .folder",
          drop: function(e, ui) {
            var droppedEl = ui.draggable.parents("tr");
            $("#example-advanced").treetable("move", droppedEl.data("ttId"), $(this).data("ttId"));
          },
          hoverClass: "accept",
          over: function(e, ui) {
            var droppedEl = ui.draggable.parents("tr");
            if(this != droppedEl[0] && !$(this).is(".expanded")) {
              $("#example-advanced").treetable("expandNode", $(this).data("ttId"));
            }
          }
        });
      });

      $("form#reveal").submit(function() {
        var nodeId = $("#revealNodeId").val()

        try {
          $("#example-advanced").treetable("reveal", nodeId);
        }
        catch(error) {
          alert(error.message);
        }

        return false;
      });
    </script>
  
  <form action="">
  <fieldset>
  <legend><p><button type="button" class="btn btn-default btn-sm">
          <span class="glyphicon glyphicon-exclamation-sign"></span>Tip:</button></p></legend>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Full-length transcripts: CCS sequences containing 5'-primer, 3'-primer and poly-A.</p>
    </li>
  </ul>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Reference genome alignment: The first column of the id2id file is the transcript id measured in the third generation, the second column is the transcript id compared to the reference genome, and the third column is the gene id compared to the reference genome. The new gene is compared to the reference genome not yet reads for the comment area.</p>
    </li>
  </ul>
  <ul class='task-list'>
    <li class='task-list-item task-list-done' >
      <input type='checkbox' disabled='disabled' checked> </input><p>Shared transcripts:When there are less than 2 third-generation samples, it's not possible to do the analysis of shared transcripts and the corresponding venn diagram.</p>
    </li>
  </ul>
 


  
  </fieldset>
  </form>
  <hr style="height:5px;border:none;border-top:5px ridge grey;" />
  <table align="center"  height="100%" style="font-size:12px font-weight:bold color:#F5F5F5" >
  <td "align="center" valign="top">If you have any questions, please consult project sales or operations, we will be happy to serve you.Sincere thanks for your trust in Novogene!</td>
  </body>
</html>

      
