<!-- django temple created by chenshuangzhao in 2020.04.01-->

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN">
<html>
<head>

<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="x-ua-compatible" content="IE=11,edge" />




<title> XXXXXX项目生物信息信息分析结题报告 </title>



<!--<meta> 标签提供了 HTML 文档的元数据。元数据不会显示在客户端，但是会被浏览器解析。META元素通常用于指定网页的描述，关键词. 元数据通常以 名称/值 对出现-->
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<link href="css/buttons.css" rel="stylesheet" />
<link href="css/font-awesome.min.css" type="text/css" rel="stylesheet" />
<link rel="stylesheet" type="text/css" href="css/base.css">
<link href="css/base_new.css" type="text/css" rel="stylesheet">
<link href="css/cloth.css" rel="stylesheet" type="text/css" media="screen" />
<link rel="stylesheet" type="text/css" href="js/fancybox/jquery.fancybox-1.3.4.css" media="screen" />
<script src='js/jquery-1.4.2.min.js'></script>
<script src="js/network.js"></script>
<script src="js/echarts.min.js"></script>
<script src="js/test.js"></script>
<script src="js/scrolltop.js" type="text/javascript"></script>
<script src="js/common.js" type="text/javascript"></script>
<script src="js/jquery.albumSlider.min.js" type="text/javascript"></script>
<script type="text/javascript" src="js/tablecloth.js"></script>
<script type="text/javascript" src="js/fancybox/jquery.mousewheel-3.0.4.pack.js"></script>
<script type="text/javascript" src="js/fancybox/jquery.fancybox-1.3.4.pack.js"></script>
<script src="js/esl.js" type="text/javascript"></script>
<script src="js/bootstrap.js" type="text/javascript"></script>
	
<style media="print">
	.noprint {DISPLAY: none;}
	#main{width:95%;right:30px;}
	.printmenu{display:block;}
</style>
<!-- class 寻找 style 定义文档样式,如需链接外部样式表，请使用 <link> 标签,如果没有使用 "scoped" 属性，则每一个 <style> 标签必须位于 head 头部区域, type 唯一可能的值是 "text/css" 规定样式表的 MIME 类型-->
<!-- type 寻找js/css--> 

	<div class="noprint">
		<div style="display: block;" id="goTopBtn">
			<a class="backtotop" title="回顶部"><img src="images/goTop.jpg" width="30" height="30" class="back-tip"/></a>
		</div>
	</div>
	
	<script type="text/javascript">
		var Tree = new Array;
		// nodeId | parentNodeId | nodeName | nodeUrl
	<{jtree}>
	</script> 

  <link href="css/new/base_new.css" type="text/css" rel="stylesheet">

    
  <style>
    .countt {
      counter-reset: slideNum;
    }
    .countt tr:before {
      counter-increment: slideNum;
      content: counter(slideNum)".";
    }
  </style>

</head>	
<body>

<!-- --------------------------------------------------------------- Menu bar----------------------------------------------------------------------- -->
<!-- 
 <div id="contentLeft" class="linear" style="overflow-y:auto">
    <img style="margin-top:10px;margin-left:5%;width:90%" src="images/logo1.png" />
  <ul id="leftNavigation" class="nav">
  
<li><a href="#项目背景及样本信息">项目背景</a></li>
<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#建库测序流程">实验流程</a>
<ul>

<li><a href="#RNA样品检测"> RNA样品检测</a></li>


<li><a href="#文库构建"><i class="icon-angle-right"></i> 文库构建</a></li>

<li><a href="#库检"><i class="icon-angle-right"></i> 库检</a></li>
<li><a href="#上机测序"><i class="icon-angle-right"></i> 上机测序</a></li>
</ul>
</li>


<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#生物信息分析流程">生物信息流程</a>
<ul>
<li><a href="#生物信息分析流程"><i class="icon-angle-right"></i> 生物信息分析流程</a></li>


<li><a href="#标准分析结果"><i class="icon-angle-right"></i> 标准分析结果</a></li>
</ul>
</li>

<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#数据质量控制">质控</a>
<ul>
<li><a href="#原始序列数据"><i class="icon-angle-right"></i> 原始序列数据</a></li>
<li><a href="#测序数据质量评估"><i class="icon-angle-right"></i> 测序数据质量评估</a></li>
<li><a href="#测序数据过滤"><i class="icon-angle-right"></i> 测序数据过滤</a></li>
<li><a href="#测序错误率分布"><i class="icon-angle-right"></i> 测序错误率分布</a></li>

<li><a href="#GC含量分布"><i class="icon-angle-right"></i> GC含量分布</a></li>

<li><a href="#数据质量汇总"><i class="icon-angle-right"></i> 数据质量汇总</a></li>
</ul>
</li>



<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#比对分析">序列比对与拼接</a>
<ul>
<li><a href="#比对情况统计"><i class="icon-angle-right"></i> 比对情况统计</a></li>
<li><a href="#基因组区域分布"><i class="icon-angle-right"></i> 基因组区域分布</a></li>



<li><a href="#转录本拼接"><i class="icon-angle-right"></i> 转录本拼接</a></li>



<li><a href="#转录本筛选"><i class="icon-angle-right"></i> 转录本筛选</a></li>
<li><a href="#lncRNA与mRNA特征比较"><i class="icon-angle-right"></i> lncRNA与mRNA特征比较</a></li>




</ul>
</li>
<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#定量分析">定量与差异</a>
<ul>
<li><a href="#定量结果说明"><i class="icon-angle-right"></i> 定量结果说明</a></li>
<li><a href="#表达水平分布"><i class="icon-angle-right"></i> 表达水平分布</a></li>



<li><a href="#差异基因列表"><i class="icon-angle-right"></i> 差异基因列表</a></li></li>
<li><a href="#差异基因火山图"><i class="icon-angle-right"></i> 差异基因火山图</a></li></li>
<li><a href="#差异基因聚类"><i class="icon-angle-right"></i> 差异基因聚类</a></li>




</ul>
</li>
<li onmouseover="displaySubMenu(this)" onmouseout="hideSubMenu(this)"><a href="#功能分析">功能分析</a>
<ul>
<li><a href="#GO功能富集"><i class="icon-angle-right"></i> GO功能富集</a></li>
<li><a href="#通路富集结果列表"><i class="icon-angle-right"></i> KEGG通路富集</a></li>



<li><a href="#lncRNA靶基因预测"><i class="icon-angle-right"></i> lncRNA靶基因预测</a></li>
<li><a href="#差异lncRNA分析"><i class="icon-angle-right"></i> 差异lncRNA分析</a></li>
<li><a href="#差异lncRNA靶基因富集分析"><i class="icon-angle-right"></i> 差异lncRNA靶基因富集分析</a></li>



<li><a href="#差异可变剪切分析"><i class="icon-angle-right"></i> 差异可变剪切分析</a></li>













</ul>
</li>


<li><a href="#备注">备注</a></li>
<li><a href="#参考文献">参考文献</a></li>

  </div> -->
<div id="main">

<!-- --------------------------------------项目背景及样品信息---------------------------------------  -->
<!-- --------------------------------------项目背景及样品信息---------------------------------------  -->
<div id="page">

	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 " >项目编号：PXXXXXXXX</font></b></p>
	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 ">项目名称：XXXXXX</font></b></p>
	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 ">结题编号：PXXXXXXXX_Homo_sapiens</font></b></p>
	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 ">结题日期：20250117135745</font></b></p>
	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 ">售后电话：400-658-1585先按3再按2，语音提示为：lncRNA、Small RNA、circRNA产品请按2</font></b></p>
	<p><b class="bold"><font size="4" face="微软雅黑" style="color: #003f76 ">售后邮箱：service-nsRNA@novogene.com</font></b></p>
	</br>
</div>

<!----------------------------------------结果汇总----------------------------------------->
<div id="page">
	<p class="head">
	<a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	<a name="项目背景及样本信息">北京诺禾致源科技股份有限公司</a>
	<hr/>
	</p>
	
    <h2>重要结果汇总</h2>
  
    <p class="paragraph">1.该项目鉴定到的已知lncRNA和预测的新的lncRNA的个数分别为：</p>
    <p class="center">
    <div style="overflow-x: auto">
    	<table class="gy">
        	
          	
          		<tr><th>Types</th><th>Total</th></tr>
          	
       
          	
          		<tr><td>known</td><td>103347</td></tr>
          	
       
          	
          		<tr><td>novel</td><td>2368</td></tr>
          	
       
        </table>
     </div>
     </p>
  


  
      <p class="paragraph">2. 每个差异比较组合差异（DIFF）lncRNA个数，以及上调（UP）、下调（DOWN）的lncRNA个数分别为：</p>
      <p class="center">
      <div style="overflow-x: auto">
      	 <table class="gy">
      	   	
      	 </table>
      </div>
      </p>
      

	    


  
  
	</br>
</div>

<div id="page">
	<p class="head">
	<a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	<a name="项目背景及样本信息">北京诺禾致源科技股份有限公司</a>
	<hr/>
	</p>
	<h2 id="goto_sampleinfo1">一、项目背景及样本信息</h2>
	<a name="项目背景"></a>
	<h3 id="background">1 项目背景：转录组的研究意义</h3>
	<p class="paragraph">LncRNA（long non-coding RNA）即长链非编码RNA，是一类长度大于200nt并且一般不编码蛋白质的RNA分子。长久以来，它们被当作无用的转录物因而被忽视<sup>[1]</sup>。然而近十年来，越来越多的研究证明，其在真核生物尤其是哺乳动物中广泛地存在，并起到重要的生物学作用。</p>
    <p class="paragraph">与mRNA相同的是，lncRNA同样由细胞中的RNA聚合酶II转录得到；不同的是，其转录丰度和稳定性都较低，并且大都不具有poly A尾的结构。从功能上看，lncRNA的主要通过顺式和反式两种调节方式，分别作用于靶基因的转录水平、转录后水平、翻译水平及表观水平等，起到调控靶基因的表达水平的作用<sup>[2]</sup>。在生物体中，lncRNA一般具有较强的组织和细胞特异性，并且在最近的研究中也被发现与物种间差异以及发育有很强的关联性。此外，lncRNA也被发现在癌症、免疫缺陷等众多疾病的致病机制中扮演着重要角色<sup>[1]</sup>。</p>
	<p class="paragraph">环状 RNA（circular RNA，circRNA）长度在 200nt 以上,主要由 pre-mRNA 通过可变剪切加工产生；5’端和 3’端共价结合形成闭合结构，广泛存在与动物和植物中。特殊的环状结构使其能够逃逸 RNA 酶的降解，增强体内稳定性。此外，其具有一定的保守性（低于 mRNA）和细胞或组织类型特异性。</p>
	
	</br>

</div>


<!-- --------------------------------------实验流程--------------------------------------- -->
<!-- --------------------------------------实验流程--------------------------------------- -->
<div id="page">

	<p class="head">
	<a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	<a name="建库测序流程">北京诺禾致源科技股份有限公司</a>
	<hr/>
	</p>

	<h2 id="goto_experiment1">二、建库测序流程</h2>
	<p class="paragraph">从RNA样品提取到最终数据获得，样品检测、建库、测序等每一环节都会直接影响数据的数量和质量，从而影响后续信息分析的结果。为从源头保证测序数据准确可靠，诺禾致源承诺在数据的所有生产环节都严格把关，从根源上确保高质量数据的产出。建库测序的流程图如下：</p>

	
	
	
	
	<p class="center"><img class="normal_library_lncRNA" style="width:20%;height:32vw;text-align:center" src="images/library_lncRNA.png" /></p>
	

	
	<p class="name_fig">图1 建库测序流程图</p>
	<br />


</div>

<!-----------------------1 RNA提取与检测-------------------------->
<div id="page">

	<a name="RNA样品检测"></a>
	<h3 id="experiment1">1 RNA样品检测</h3>
	<p class="paragraph">从组织或细胞中提取RNA，随后对RNA样品进行严格质控，质控标准主要包括以下3个方面：</p>
	<p class="paragraph">(1) 琼脂糖凝胶电泳：分析样品RNA完整性及是否存在DNA污染；</p>
	<p class="paragraph">(2) Nanodrop：进行初步定量，检测RNA浓度及纯度（OD260/280及OD260/230）；</p>
	<p class="paragraph">(3) Agilent 2100 bioanalyzer：精确检测RNA完整性。 </P>


	<br />


</div>

<!----------------------2 文库构建-------------------------->
<div id="page">
        


	<a name="文库构建"></a>
	<h3 id="experiment2">2 文库构建</h3>


	

	
	<p class="paragraph">诺禾致源采用去除核糖体RNA的方法（circRNA建库增加去线性RNA过程）构建链特异性文库<sup>[3]</sup>。
	首先，从总RNA中去除核糖体RNA，随后将RNA打断成250~300 bp的短片段。以片段化的RNA为模板，随机寡核苷酸为引物合成cDNA第一条链，随后以dNTPs（dUTP、dATP、dGTP和dCTP）为原料合成cDNA第二条链。
	
	纯化后的双链cDNA经过末端修复、加A尾并连接测序接头<sup>[注]</sup>，用AMPure XP beads筛选350~400 bp左右的cDNA。使用USER酶降解含U的cDNA第二链，最后进行PCR扩增并获得文库。链特异性文库具有诸多优势，如相同数据量下可获取更多有效信息；能获得更精准的基因定量、定位与注释信息等。建库原理如下图：</p>
		
        
        <p class="center"><img class="normal_strand_specific" src="images/strand_specific.png" /></p>
        
        
	<p class="name_fig">图2 文库构建流程</p>
	<p class="tremark">注：接头（Adapter），包括P5/P7、index和Rd1 SP/Rd2 SP。P5/P7，用于测序仪进行桥式PCR的过程；Index，标签序列，用来区分不同文库信息；Rd1 SP/Rd2 SP（sequence primer），测序引物结合区域，测序过程理论上由Rd1/Rd2向后开始进行。</p>
	
	<p class="paragraph">我们在建库过程中消化了核糖体RNA，之后将片段随机打断为250-300bp，通过片段长度选择，最终得到除了核糖体RNA和小片段RNA（microRNA、siRNA等）以外的所有RNA，包括lncRNA、mRNA、circRNA等。</p>
	
	
	
	
</br>
</div>	
	
	
<!----------------------3 库检-------------------------->
	
<div id="page">
       
	<a name="库检"></a>
	<h3 id="experiment3">3 库检</h3>

	<p class="paragraph">先用 Qubit 初步定量，稀释文库至 1ng/ul；再用 Agilent 2100 bioanalyzer 检测对文库的插入片段长度（insert size）进行检测，insert size 大约分布在 250~300 bp 左右符合预期。Insert size符合预期后，使用 qPCR方法对文库的有效浓度进行准确定量，文库的有效浓度为>2nM,保证了样本文库的质量</p>


	<br />
</div>


<!-----------------------4 上机测序-------------------------->
<div id="page">
	<a name="上机测序"></a>
	<h3 id="experiment4">4 上机测序</h3>

	<p class="paragraph">库检合格后，根据文库的有效浓度及数据产出需求pooling后进行Illumina PE150测序。PE150（Pair end 150 bp）指高通量双端测序，每端各测150 bp。在构建的小片段文库中，Insert cDNA，即插入片段是直接测序的单位。双端测序是将每条插入片段的两端进行测序的方法，由于插入片段的长度分布已知，双端测序时不仅可以获得片段两端的序列，也可获得这两段序列之间的长度，从而便于后续组装和比对。</p>
	<p class="paragraph">测序的基本原理是边合成边测序（Sequencing by Synthesis）。在测序的flow cell中加入四种荧光标记的dNTP、DNA聚合酶以及接头引物进行扩增，在每一个测序簇延伸互补链时，每加入一个被荧光标记的dNTP就能释放出相对应的荧光，测序仪通过捕获荧光信号，并通过计算机软件将光信号转化为测序峰，从而获得待测序片段的序列信息。</p>
	<br />


</div>

<!-- --------------------------------- 信息分析流程 -------------------------------- -->
<!-- --------------------------------- 信息分析流程 -------------------------------- -->
<div id="page">
<p class="head"><a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a><a name="生物信息分析流程">北京诺禾致源科技股份有限公司</a><hr /></p>
	<h2 id="pipeline1">三、生物信息分析流程</h2>
    <p class="paragraph">测序获得原始测序序列（Raw data）后进入信息分析流程，流程分为两个阶段：</p>
	<p class="paragraph">1）测序数据质量评估：主要通过对测序错误率、数据量、比对率等进行统计，评估建库测序是否达到了标准，符合标准则进行后续的分析，否则需重新建库或加测；</p>
   
	
	
	
		
   <p class="paragraph">2）信息挖掘及分析：RNA-seq标准分析过程包括质控、比对、拼接、筛选、定量、差异显著性分析和功能富集等环节，以及与转录本结构变异相关的可变剪切、变异位点等分析内容。同时，对于来自人或小鼠的癌症组织样本,诺禾致源推出适合癌症研究的个性化分析内容，包括融合基因、分子分型和GSEA富集分析等。</p>

	<p class="center">
	<img class="normal_pipline_lncRNA" src="images/pipline_lncRNA.png" /></p>	
	


	
	
	
	<p class="name_fig">图3 信息分析流程图</p>
		
	</br>
</div>
	
<!------------------------------四 标准分析结果-------------------------->

<!---------------------------四 标准分析结果---------------------->
	
<div id="page">
	<p class="head">
	<a href="#home" title="返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	 <a name="标准分析结果">北京诺禾致源科技股份有限公司</a>
	<hr/>
	</p>

	<h2 id="resultall">四、标准分析结果</h2>
	
	
		
	<p class="paragraph">RNA-seq标准分析过程主要包括质控，比对，拼接，筛选，定量，差异显著性分析，功能富集等环节。RNA-seq分析的核心是基因表达差异的显著性，使用统计学方法比较两个或多个条件下的基因表达差异（例如，处理与对照），从中找出与条件相关的差异基因，然后进一步分析这些差异基因的生物学意义。</p>
	<p class="paragraph">长链非编码 RNA（lncRNA）是一类转录本长度超过 200nt 的 RNA 分子，此类 RNA 不编码蛋白，以 RNA的形式在多种层面上（表观遗传调控、转录调控以及转录后调控等）调控基因的表达。lncRNA 起初被认为是RNA 聚合酶 II 转录的副产物，不具有生物学功能。然而，近年来的研究表明， lncRNA 参与了 X 染色体沉默，基因组印记以及染色质修饰，转录激活，转录干扰，核内运输等多种重要的调控过程。</p>
	<p class="paragraph">通过去核糖体链特异性文库得到的RNA主要包括lncRNA、mRNA、circRNA等，针对生物体内转录调控机制研究所需，我们主要分析mRNA、lncRNA及circRNA。</p>
	
	
		
	<!-- <p class="paragraph">RNA-seq标准分析过程主要包括质控，比对，筛选鉴定，定量，差异显著性分析，来源基因分析和功能富集等环节。RNA-seq分析的核心是基因表达差异的显著性，使用统计学方法比较两个或多个条件下的基因表达差异（例如，处理与对照），从中找出与条件相关的差异基因，然后进一步分析这些差异基因的生物学意义。</p> -->


</br>	
</div>

<!----------------------------1.1 原始数据说明------------------------>	
<div id="page">
	<a name="测序数据质量评估"><h3 id="31">1 测序数据质量评估(结果文件夹: Results/QC)</h3></a>
	<p class="paragraph">测序数据的产生经过了RNA提取、建库、测序多个步骤，这些步骤会产生少许低质量或者无效数据，比如建库阶段会存在建库长度的偏差，测序阶段会出现测序错误的情况。所以需要对下机得到的原始数据进行质量控制，保证后续分析的准确性。</p>
	
	
	</br>
	<a name="原始序列数据"><h4 id="result11">1.1 原始序列数据</h4></a>
	<p class="paragraph">高通量测序（如Illumina PE150/SE50）下机得到的原始图像文件经CASAVA碱基识别转化为测序读段（Sequenced Reads），以FASTQ格式存储。FASTQ是一种存储生物序列及相应质量值的常用文本格式，每条测序读段由四行组成，如下图所示：</p>

	<p class="center">
	<img class="normal" src="images/rawdata.png" /></p>

    <p class="paragraph">Illumina测序标识符详细信息如下：</p>
    <p class="center">
    <table class="gy">
    <tr>
        <th>标示符</th>
        <th>含义</th>
      </tr>
      <tr>
        <td>HWI-ST1276</td>
        <td>Instrument – unique identifier of the sequencer </td>
      </tr>
      <tr>
        <td>71</td>
        <td>run number – Run number on instrument</td>
      </tr>
      <tr>
        <td>C1162ACXX</td>
        <td>FlowCell ID – ID of flowcell</td>
      </tr>
      <tr>
        <td>1</td>
        <td>LaneNumber – positive integer</td>
      </tr>
      <tr>
        <td>1101</td>
        <td>TileNumber – positive integer</td>
      </tr>
      <tr>
        <td>1208</td>
        <td>X – x coordinate of the spot. Integer which can be negative</td>
      </tr>
      <tr>
        <td>2458</td>
        <td>Y – y coordinate of the spot. Integer which can be negative</td>
      </tr>
      <tr>
        <td>1</td>
        <td>ReadNumber - 1 for single reads; 1 or 2 for paired ends</td>
      </tr>
      <tr>
        <td>N</td>
        <td>whether it is filtered - NB：Y if the read is filtered out, not in the delivered fastq file, N otherwise</td>
      </tr>
      <tr>
        <td>0</td>
        <td>control number - 0 when none of the control bits are on, otherwise it is an even number</td>
      </tr>
      <tr>
        <td>CGATGT</td>
        <td>Illumina index sequences</td>
      </tr>
    </table>
    </p>


	<br />
	<br />


</div>
		
<!------------------------------1.2 测序数据过滤------------------------------>

<div id="page">


	<a name="测序数据过滤"><h4 id="result12">1.2 测序数据过滤</h4></a>
	<p class="paragraph">测序获得的原始数据中包含少量带有测序接头或测序质量较低的reads，如图1.1所示。为了保证数据分析的质量及可靠性，需要对原始数据进行过滤，过滤内容如下：</p>
	<p class="paragraph">(1) 去除带接头（adapter）的reads；</p>
	<p class="paragraph">(2) 去除N（N表示无法确定碱基信息）的比例大于0.002的reads；</p>
	<p class="paragraph">(3) 当单端read中含有的低质量碱基数超过该条read长度比例的50%时，需要去除此对paired reads。</p>
</p>

<p class="paragraph">各样本的测序数据过滤情况见结果文件：1.QC/Classfication，如下图所示。</p>

	
</br>
<p class="center">
	<div class="albumSlider">
				<div class="fullview">
					<img src=sample1/>
				</div>
				<div class="slider">
					<div class="button movebackward" title="向上滚动">
					</div>
					<div class="imglistwrap">
						<ul class="imglist">
							
							<li><a id="example2" href=sample1 ><img src=files/QC/Classfication/sample1.classfication.png/></a></li>
							
							<li><a id="example2" href=sample2 ><img src=files/QC/Classfication/sample2.classfication.png/></a></li>
							
							<li><a id="example2" href=sample3 ><img src=files/QC/Classfication/sample3.classfication.png/></a></li>
							
							<li><a id="example2" href=sample4 ><img src=files/QC/Classfication/sample4.classfication.png/></a></li>
							
							<li><a id="example2" href=sample5 ><img src=files/QC/Classfication/sample5.classfication.png/></a></li>
							
							<li><a id="example2" href=sample6 ><img src=files/QC/Classfication/sample6.classfication.png/></a></li>
							
							<li><a id="example2" href=sample7 ><img src=files/QC/Classfication/sample7.classfication.png/></a></li>
							
						</ul>
					</div>
					<div class="button moveforward" title="向下滚动">
					</div>
				</div>
			</div>
		</p>

	<p class="name_fig">图1.1 原始数据组成</p>
	<p class="cenmark">不同颜色比例分别代表不同组分的比例，(1) Adapter related：带接头的reads；(2) Containing N：含有未知碱基的reads；(3) Low quality：测序质量低的reads；(4) Clean reads：通过数据过滤的reads。</p>

	<br />
	<br />

</div>
	
<!---------------------------1.3 测序错误率分布-------------------------->
<div id="page">


	<a name="测序错误率分布"><h4 id="result13">1.3 测序错误率分布</h4></a>
	<p class="paragraph">测序过程本身存在发生机器错误的可能性，测序错误率分布检查可以反映测序数据的质量，序列信息中每个碱基的测序质量值保存在FASTQ文件中，若reads的碱基质量值用Q<sub>Phred</sub>表示，则可计算测序错误率 e=10<sup>(-Q<sub>Phred</sub>/10)</sup> 或表示为 Q<sub>Phred</sub>=-10log<sub>10</sub>(e)。Illunima Casava 1.8版本碱基识别与Phred分值之间的简明对应关系如下表所示。</p>
	
	<p class="name_fig">表1.1　Illumina Casava 1.8版本碱基识别与Phred分值之间的简明对应关系</p>
	<table class="tb1">
		<p class="center">
		<tr><th>Phred分值</th><th>不正确的碱基识别</th><th>碱基正确识别率</th><th>Q-sorce</th></tr>
		<tr><td>10</td><td>1/10</td><td>90%</td><td>Q10</td></tr>
		<tr><td>20</td><td>1/100</td><td>99%</td><td>Q20</td></tr>
		<tr><td>30</td><td>1/1000</td><td>99.9%</td><td>Q30</td></tr>
		<tr><td>40</td><td>1/10000</td><td>99.99%</td><td>Q40</td></tr>
		</p>
	</table>

	<br />

	<p class="paragraph">测序错误率与碱基质量有关，受测序仪本身、测序试剂、样品等多个因素共同影响。对于RNA-seq技术，测序错误率分布具有两个特点：</p>
	<p class="paragraph">(1) 测序错误率会随着测序序列（Sequenced Reads）长度的增加而升高，这是由于测序过程中化学试剂的消耗而导致的，并且为Illumina高通量测序平台都具有的特征<sup>[4]</sup>。</p>
	<p class="paragraph">(2) 前几个碱基的位置也会发生较高的测序错误率，是因为在RNA-seq建库过程中反转录需要随机引物，推测前几个碱基测序错误率较高的原因是随机引物碱基和RNA模版的不完全结合<sup>[4]</sup>。</p>

</p>
	
	<p class="paragraph">各样本测序错误率分布见结果文件：1.QC/Errorrate，如下图所示。</p>

	
	<br />

	<p class="center">

		<div class="albumSlider">
					<div class="fullview">
						<img src=sample1/>
					</div>
					<div class="slider">
						<div class="button movebackward" title="向上滚动">
						</div>
						<div class="imglistwrap">
							<ul class="imglist">
								
								<li><a id="example2" href=sample1 ><img src=files/QC/Errorrate/sample1.Error.png/></a></li>
								
								<li><a id="example2" href=sample2 ><img src=files/QC/Errorrate/sample2.Error.png/></a></li>
								
								<li><a id="example2" href=sample3 ><img src=files/QC/Errorrate/sample3.Error.png/></a></li>
								
								<li><a id="example2" href=sample4 ><img src=files/QC/Errorrate/sample4.Error.png/></a></li>
								
								<li><a id="example2" href=sample5 ><img src=files/QC/Errorrate/sample5.Error.png/></a></li>
								
								<li><a id="example2" href=sample6 ><img src=files/QC/Errorrate/sample6.Error.png/></a></li>
								
								<li><a id="example2" href=sample7 ><img src=files/QC/Errorrate/sample7.Error.png/></a></li>
								
							</ul>
						</div>
						<div class="button moveforward" title="向下滚动">
						</div>
					</div>
				</div>	
			
			</p>
		

	<p class="name_fig">图1.2 测序错误率分布图</p>
	<p class="cenmark">测序片段不同碱基位置错误率分布图，其中横坐标为 reads 的碱基位置，纵坐标为单碱基错误率。</p>
	<p class="paragraph">测序错误率分布检查用于检测在测序长度范围内，有无某些位置的碱基存在异常的高错误率，例如若Read1或Read2中间位置的碱基测序错误率显著高于其他位置，则说明可能存在异常碱基。</p>


	<br />
</div>	
	
<!---------------------------1.4 GC含量分布------------------------------>	
<div id="page">
 
	<a name="测序错误率分布"><h4 id="result14">1.4 GC含量分布</h4></a>

	
		
	<p class="paragraph">核苷酸序列中鸟嘌呤（G）和胞嘧啶（C）所占的比例称为GC含量。GC含量在物种间存在一定特异性，但由于反转录过程中所使用的6bp随机引物，会引起前几位碱基在核苷酸组成上有一定偏好性，产生正常波动，随后则趋于稳定。对于链特异性建库而言，由于只保留了单链信息，可能会出现AT分离或GC分离现象。如图1.3所示：</p>
	
		
	<!-- <p class="paragraph">核苷酸序列中鸟嘌呤（G）和胞嘧啶（C）所占的比例称为GC含量。GC含量在物种间存在一定特异性，但由于反转录过程中所使用的6bp随机引物，会引起前几位碱基在核苷酸组成上有一定偏好性，产生正常波动，随后则趋于稳定。对于链特异性建库而言，由于只保留了单链信息，可能会出现AT分离或GC分离现象。如图1.3所示：</p> -->
</p>	
	<p class="paragraph">各样本GC含量分布见结果文件：1.QC/GC，如下图所示。</p>

	
</br>
	
	<p class="center">

		<div class="albumSlider">
					<div class="fullview">
						<img src="files/QC/GC/sample1.GC.png"/>
					</div>
					<div class="slider">
						<div class="button movebackward" title="向上滚动">
						</div>
						<div class="imglistwrap">
							<ul class="imglist">
								
								<li><a id="example2" href="files/QC/GC/sample1.GC.png" ><img src="files/QC/GC/sample1.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample2.GC.png" ><img src="files/QC/GC/sample2.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample3.GC.png" ><img src="files/QC/GC/sample3.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample4.GC.png" ><img src="files/QC/GC/sample4.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample5.GC.png" ><img src="files/QC/GC/sample5.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample6.GC.png" ><img src="files/QC/GC/sample6.GC.png"/></a></li>
								
								<li><a id="example2" href="files/QC/GC/sample7.GC.png" ><img src="files/QC/GC/sample7.GC.png"/></a></li>
								
							</ul>
						</div>
						<div class="button moveforward" title="向下滚动">
						</div>
					</div>
				</div>	
			
			</p>

<p class="name_fig">图1.3 GC含量分布图</p>
	<p class="cenmark">横坐标为reads的碱基位置，纵坐标为单碱基所占的比例，不同颜色代表不同的碱基类型</p>


	<br />
	<br />
</div>		
	
<!-----------------------1.5 数据质量汇总---------------------->

<div id="page">


	<a name="数据质量汇总"><h4 id="result15">1.5 数据质量汇总</h4></a>
	<p class="paragraph">经过原始数据过滤、测序错误率检查及GC含量分布检查，获得后续分析使用的clean reads，数据汇总如表1.2所示。</p>
	
</p>
	<p class="paragraph">各样本数据质量汇总见结果文件：1.QC/Datastat，如下图所示。</p>
</br>
	
</br>

	<p class="name_fig">表1.2　数据质量汇总</p>

	<p class="center">
	<!--<div id="tb" style="width:1100px;height:270px;overflow-y:scroll;overflow-x:hidden;" >-->
	<div id="tb" style="overflow-y:scroll;overflow-x:hidden;" >
		<table class="tb1">
			
			
				<tr><th>sample</th><th>library</th><th>raw_reads</th><th>raw_bases</th><th>clean_reads</th><th>clean_bases</th><th>error_rate</th><th>Q20</th><th>Q30</th><th>GC_pct</th></tr>
			
		  
			
				<tr><td>sample1</td><td>BRRL220000060-1A</td><td>1e+07</td><td>1.5G</td><td>9735746</td><td>1.46G</td><td>0.02</td><td>97.97</td><td>94.31</td><td>49.84</td></tr>
			
		  
			
				<tr><td>sample2</td><td>BRRL220000061-1A</td><td>1e+07</td><td>1.5G</td><td>9695364</td><td>1.45G</td><td>0.03</td><td>97.85</td><td>94.06</td><td>46.58</td></tr>
			
		  
			
				<tr><td>sample3</td><td>BRRL220000062-1A</td><td>1e+07</td><td>1.5G</td><td>9428286</td><td>1.41G</td><td>0.02</td><td>98.46</td><td>95.45</td><td>53.83</td></tr>
			
		  
			
				<tr><td>sample4</td><td>BRRL220000063-1A</td><td>1e+07</td><td>1.5G</td><td>9800400</td><td>1.47G</td><td>0.02</td><td>98.19</td><td>94.8</td><td>49.14</td></tr>
			
		  
			
				<tr><td>sample5</td><td>BRRL220000064-1A</td><td>1e+07</td><td>1.5G</td><td>9772582</td><td>1.47G</td><td>0.02</td><td>98.2</td><td>94.99</td><td>51.36</td></tr>
			
		  
			
				<tr><td>sample6</td><td>BRRL220000065-1A</td><td>1e+07</td><td>1.5G</td><td>9793882</td><td>1.47G</td><td>0.02</td><td>98.2</td><td>94.85</td><td>50.43</td></tr>
			
		  
			
				<tr><td>sample7</td><td>BRRL220000066-1A</td><td>1e+07</td><td>1.5G</td><td>9808088</td><td>1.47G</td><td>0.02</td><td>98.19</td><td>94.94</td><td>50.35</td></tr>
			
		  
		</table>
	</div>		
	</p>
	<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">

	<p class="tremark">
	(1) Sample name：样品名<br />
	(2) Raw reads：原始数据中的reads数<br />
	(3) Clean reads：原始数据过滤后的reads数<br />
	(4) Raw bases：原始数据中的碱基数<br />
	(5) Clean bases：原始数据过滤后的碱基数<br />	
	(6) Error rate：数据整体测序错误率<br />
	(7) Q20：Phred数值大于20的碱基占总碱基的百分比<br />
	(8) Q30：Phred数值大于30的碱基占总碱基的百分比<br />
	(9) GC content：clean reads中G与C占四种碱基的百分比<br />
	</p>
</div>


	<br />
	<br />


<!-------------------------------1.6  QC视频链接------------------------------>
<a name="QC视频讲解"><h4 id="result16">1.6 数据产出及质控讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1mV7hLr0tMsjnEmViCPQCZw  " class="uri">https://pan.baidu.com/s/1mV7hLr0tMsjnEmViCPQCZw </a></font> 提取码: <font face="Arial">9y2t  </font></p>

	<br />
	<br />	
</div>




<!-------------------------------2 比对------------------------------>



<div id="page" >

	<a name="比对分析"><h3 id="result2">2 比对分析(结果文件夹: Results/Mapping)</h3></a>

    <p class="paragraph">将有效测序数据（clean reads）比对到基因组或转录组是后续分析的基础，诺禾致源可采用Tophat2（http://tophat.cbcb.umd.edu）、Hisat2（http://ccb.jhu.edu/software/hisat2）、STAR（http://code.google.com/p/rna-star）软件对RNA-Seq测序数据进行比对分析<sup>[5]</sup>。Hisat2是经典比对软件Tophat2的作者推荐的升级类软件，mapping率高且找到 Junction reads的正确率高，因此我们默认采用Hisat2<sup>[6]</sup>。</p>
    <p class="paragraph">比对输出结果一般为标准的SAM文件，由sanger制定，是一种以TAB为分隔符的文本格式，主要用于测序序列比对到基因组上的结果展示。BAM文件是SAM文件的二进制格式，因其占据存储资源小，运算速度快，很多比对软件均以BAM格式作为标准输出，利用samtools工具可查看BAM文件的具体内容。</p>


</p>

</div>

<!------------------------------2.1 比对结果说明-------------------------->

<!--------------------------2.1 比对情况统计-------------------------->


	<a name="比对情况统计"><h4 id="result21">2.1 比对情况统计</h4></a>
    <p class="paragraph">利用比对软件将测序得到的clean reads比对到到参考基因组上，统计其可以mapping到参考基因组的的有效数据，并进一步进行mapping率等的统计。</p>
	
</p>	
	
	<p class="paragraph">各样本比对情况见结果文件：2.Mapping/Mapstat/MapStat.xls，如下图所示。</p>

	
</br>	

	<p class="name_fig">表2.1 所有样本比对情况一览表</p>

<p class="center">
<div id="tb" style="width:1100px;height:286px;overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1" >
			
			
				<tr><th>sample</th><th>total_reads</th><th>total_map</th><th>unique_map</th><th>multi_map</th><th>read1_map</th><th>read2_map</th><th>positive_map</th><th>negative_map</th><th>splice_map</th><th>unsplice_map</th><th>proper_map</th></tr>
			
		  
			
				<tr><td>sample1</td><td>9735746</td><td>9157887(94.06%)</td><td>8408887(86.37%)</td><td>749000(7.69%)</td><td>4222834(43.37%)</td><td>4186053(43.0%)</td><td>4201575(43.16%)</td><td>4207312(43.22%)</td><td>1433921(14.73%)</td><td>6974966(71.64%)</td><td>8104006(83.24%)</td></tr>
			
		  
			
				<tr><td>sample2</td><td>9695364</td><td>9192411(94.81%)</td><td>8604046(88.74%)</td><td>588365(6.07%)</td><td>4317309(44.53%)</td><td>4286737(44.21%)</td><td>4298208(44.33%)</td><td>4305838(44.41%)</td><td>1296335(13.37%)</td><td>7307711(75.37%)</td><td>8324436(85.86%)</td></tr>
			
		  
			
				<tr><td>sample3</td><td>9428286</td><td>8567131(90.87%)</td><td>6473692(68.66%)</td><td>2093439(22.2%)</td><td>3247570(34.44%)</td><td>3226122(34.22%)</td><td>3239659(34.36%)</td><td>3234033(34.3%)</td><td>2687330(28.5%)</td><td>3786362(40.16%)</td><td>6295086(66.77%)</td></tr>
			
		  
			
				<tr><td>sample4</td><td>9800400</td><td>9416677(96.08%)</td><td>8160808(83.27%)</td><td>1255869(12.81%)</td><td>4093768(41.77%)</td><td>4067040(41.5%)</td><td>4083953(41.67%)</td><td>4076855(41.6%)</td><td>1671599(17.06%)</td><td>6489209(66.21%)</td><td>7896094(80.57%)</td></tr>
			
		  
			
				<tr><td>sample5</td><td>9772582</td><td>9276212(94.92%)</td><td>7629244(78.07%)</td><td>1646968(16.85%)</td><td>3825719(39.15%)</td><td>3803525(38.92%)</td><td>3813653(39.02%)</td><td>3815591(39.04%)</td><td>2248465(23.01%)</td><td>5380779(55.06%)</td><td>7428742(76.02%)</td></tr>
			
		  
			
				<tr><td>sample6</td><td>9793882</td><td>9342305(95.39%)</td><td>7746488(79.1%)</td><td>1595817(16.29%)</td><td>3886566(39.68%)</td><td>3859922(39.41%)</td><td>3874855(39.56%)</td><td>3871633(39.53%)</td><td>2252137(23.0%)</td><td>5494351(56.1%)</td><td>7525714(76.84%)</td></tr>
			
		  
			
				<tr><td>sample7</td><td>9808088</td><td>9338290(95.21%)</td><td>7837068(79.9%)</td><td>1501222(15.31%)</td><td>3929093(40.06%)</td><td>3907975(39.84%)</td><td>3918117(39.95%)</td><td>3918951(39.96%)</td><td>2093990(21.35%)</td><td>5743078(58.55%)</td><td>7615658(77.65%)</td></tr>
			
		   
		</table>
	</div>	
</p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
	<p class="tremark">
	(1) Sample name：样本名称。<br />
	(2) Total reads：用于比对分析的clean reads数。<br />
	(3) Total mapped reads：比对到基因组上的reads数。<br />
	(4) Multiple mapped reads：比对到基因组多个位置的reads数。<br />
	(5) Uniquely mapped reads：比对到基因组单一位置的reads数。<br />
	(6) Read-1：read1中比对到基因组上的reads数。<br />
	(7) Read-2：read2中比对到基因组上的reads数。<br />
	(8) Reads map to '+'：比对到基因组上正链的reads数。<br />
	(9) Reads map to '-'：比对到基因组上负链的reads数。<br />
	(10) Non-splice reads：整段比对到外显子的 reads 的数。<br />
	(11) Splice reads：分段比对到两个外显子上的 reads (也称为Junction reads）的数，Splice reads的百分比取决于测序片段的长度。<br />
	(12) Reads mapped in proper pairs：成对比对到参考基因组上的reads占总clean reads数的百分比。<br />
	(13) Proper-paired reads map to different chrom：比对到基因组上不同染色体的reads占总clean reads数的百分比。<br />
	</p>
</div>

	<br />
	<br />

<!------------------------------2.2 基因组区域分布------------------>
<div id="page">

	<a name="基因组区域分布"><h4 id="result22">2.2 基因组区域分布</h4></a>

	<p class="paragraph">根据比对结果，分别统计reads在基因组外显子区域，内含子区域以及基因间区所占的比例。比对到内含子区域的reads可能来源于前体mRNA或可变剪接事件滞留的内含子。比对到基因间区的reads，可能来源于ncRNA或少许DNA片段污染<sup>[7]</sup>。所有样本的测序reads在基因组区域分布情况如图2.1所示。</p>
	</p>
	
	<p class="paragraph">各样本基因组区域分布图见结果文件：2.Mapping/Mapregion，如下图所示。</p>

	
</br>	
	<p class="center">
		<div class="albumSlider">
					<div class="fullview">
						<img src=sample1/>
					</div>
					<div class="slider">
						<div class="button movebackward" title="向上滚动">
						</div>
						<div class="imglistwrap">
							<ul class="imglist">
								
								<li><a id="example2" href=sample1 ><img src=files/Mapping/Mapregion/sample1.region.png/></a></li>
								
								<li><a id="example2" href=sample2 ><img src=files/Mapping/Mapregion/sample2.region.png/></a></li>
								
								<li><a id="example2" href=sample3 ><img src=files/Mapping/Mapregion/sample3.region.png/></a></li>
								
								<li><a id="example2" href=sample4 ><img src=files/Mapping/Mapregion/sample4.region.png/></a></li>
								
								<li><a id="example2" href=sample5 ><img src=files/Mapping/Mapregion/sample5.region.png/></a></li>
								
								<li><a id="example2" href=sample6 ><img src=files/Mapping/Mapregion/sample6.region.png/></a></li>
								
								<li><a id="example2" href=sample7 ><img src=files/Mapping/Mapregion/sample7.region.png/></a></li>
								
							</ul>
						</div>
						<div class="button moveforward" title="向下滚动">
						</div>
					</div>
				</div>
			</p>

	<p class="name_fig">图2.1 reads在基因组不同区域的分布</p>
	<p class="cenmark">reads在基因组不同区域的分布图，不同颜色代表不同的基因组区域，其中：（1）exonic：reads 在外显子区域的比例；（2）intronic：reads 在内含子区域的比例；（3）intergenic：reads 在基因间区的比例。</p>

	<br />

</div>


<!-------------------------------2.3  比对视频链接------------------------------>
<div id="page">

	<a name="比对视频讲解"><h4 id="result23">2.3 参考基因组比对讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1siCF4YStp7ENGUhS25utqg " class="uri">https://pan.baidu.com/s/1siCF4YStp7ENGUhS25utqg </a></font> 提取码: <font face="Arial">kscp </font></p>

		<br />

</div>

<!---------------------------------------转录本拼接（lncRNA）---------------------------------------------->

<!---------------------------------------转录本拼接（Stringtie）---------------------------------------------->



<div id="page">
	
	<h3 id="result6"><a name="转录本拼接">3 转录本拼接(结果文件夹: Results/Assembly_lncRNAFilter)</a></h3>
	
    <!-- <h3 id="result6"><a name="转录本拼接">3 转录本拼接(结果文件夹: Results/Assembly*)</a></h3> -->
    
	<h4 id="result61"><a name="转录本拼接">3.1 转录本拼接</h4></a>
    <p class="paragraph">Stringtie针对不同的文库有特定的参数，可准确较为精准的拼接转录本以及实现转录本定量<sup>[9]</sup>。因而我们使用Stringtie软件在基于比对到基因组上的结果，将reads拼接成转录本并进行定量<sup>[8]</sup>。</p>
<!----------------------
    <p class="center"> <img class="normal_cufflinks-algorithm" src="images/cufflinks-algorithm.png" />
    <p class="name_fig">图3.2 Cufflinks 拼接示意图</p>
    </p>
--------------------------->
</p>
	
	<p class="paragraph">各样本转录本拼接gtf见结果文件：3.Assembly_lncRNAFilter，如下图所示。</p>

	
</br>	

    <p class="name_fig">表3 Stringtie 拼接结果展示（部分）
    </p>
    <p class="center">
    <div id="tb" style="width:1100px;height:286px;overflow-y:scroll;overflow-x:scroll;">
    <table class="tb1">
    <tr class="tb1"><th class="tb1">Seqname</th><th class="tb1">Source</th><th class="tb1">Feature</th><th class="tb1">Start</th><th class="tb1">End</th><th class="tb1">Score</th><th class="tb1">Strand</th><th class="tb1">Frame</th><th class="tb1">Attributes</th></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">946635</td><td  class="tb1" style="text-align:center">946858</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.1"; transcript_id "STRG.1.1"; cov "2.678571"; FPKM "1.960246"; TPM "3.456521";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">946635</td><td  class="tb1" style="text-align:center">946858</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.1"; transcript_id "STRG.1.1"; exon_number "1"; cov "2.678571";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">954549</td><td  class="tb1" style="text-align:center">954911</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.2"; transcript_id "STRG.2.1"; cov "4.118457"; FPKM "3.013992"; TPM "5.314600";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">954549</td><td  class="tb1" style="text-align:center">954911</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.2"; transcript_id "STRG.2.1"; exon_number "1"; cov "4.118457";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">989527</td><td  class="tb1" style="text-align:center">989901</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.3"; transcript_id "STRG.3.1"; cov "8.800000"; FPKM "6.440064"; TPM "11.355825";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">989527</td><td  class="tb1" style="text-align:center">989901</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.3"; transcript_id "STRG.3.1"; exon_number "1"; cov "8.800000";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">990675</td><td  class="tb1" style="text-align:center">991753</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.4"; transcript_id "STRG.4.1"; cov "3.155700"; FPKM "2.309422"; TPM "4.072225";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">990675</td><td  class="tb1" style="text-align:center">991753</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.4"; transcript_id "STRG.4.1"; exon_number "1"; cov "3.155700";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">transcript</td><td  class="tb1" style="text-align:center">991859</td><td  class="tb1" style="text-align:center">992170</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.5"; transcript_id "STRG.5.1"; cov "2.878205"; FPKM "2.106344"; TPM "3.714136";</td></tr>
    
    <tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">StringTie</td><td  class="tb1" style="text-align:center">exon</td><td  class="tb1" style="text-align:center">991859</td><td  class="tb1" style="text-align:center">992170</td><td  class="tb1" style="text-align:center">1000</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">.</td><td  class="tb1" style="text-align:center">gene_id "STRG.5"; transcript_id "STRG.5.1"; exon_number "1"; cov "2.878205";</td></tr>
    
    </table>
    </div>
    </p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
    <p class="tremark">
    (1) Seqname：chromosome 或 scaffold 的名称;<br />
    (2) Source：数据来源信息，为 "Stringtie"<br />
    (3) Feature：序列类型描述，为 "transcript" 或 "exon"<br />
    (4) Start：起始坐标<br />
    (5) End：终止坐标<br />
    (6) Score：每个基因表达丰度最高的转录本得分为1000，其他表达量较低的转录本（Minor isoforms）根据其表达量与丰度最高转录本表达量的比值进行打分(minor FPKM/major FPKM) <br />
    (7) Strand：序列正负链信息<br />
    (8) Frame：序列编码起始位点的信息，Cufflinks 不进行起始和终止密码子的预测，这>一列为'.'<br />
    (9) Attributes：序列的其他描述信息，如基因 ID，转录本 ID 以及定量信息等</p>
</div>
</div>
</br>




<!---------------------------筛选-------------------------->

<!-- 
<option>sample1</option>

<option>sample2</option>

<option>sample3</option>

<option>sample4</option>

<option>sample5</option>

<option>sample6</option>

<option>sample7</option>
 -->
<div id="page">
	<a name="转录本筛选"><h4 id="result62">3.2 转录本筛选(结果文件夹: Results/Assembly_lncRNAFilter/lncRNAFilter)</h4></a>
	<p class="paragraph">我们使用Cuffmerge软件对各样本拼接得到的转录本进行合并，去掉其中链方向不确定和转录本长度不超过200nt的转录本，接下来利用Cuffcompare软件跟已知数据库进行比较，过滤掉数据库已知的转录本， 最后对筛选到的新转录本进行编码潜能预测等，最终得到Novel_lncRNA和Novel_mRNA。</p>
	<p class="center"> <img class="normal_lncRNA_filter_pipeline" src="images/lncRNA_filter_pipeline.png" /> </p>
	<p class="name_fig">图3.3 Novel lncRNA和Novel mRNA筛选流程图</p>
	<a name="基本筛选"></a>
	<p class="tremark">
	step1. 转录本exon个数筛选：过滤转录本拼接结果中低可信度的单外显子转录本（植物则会保留），选择exon个数≥2的转录本；<br />
	step2. 转录本长度筛选：选择转录本长度大于200nt的转录本；<br />
	step3. 转录本已知注释筛选：通过Cuffcompare软件，筛除与数据库注释exon区域有重叠的转录本，并将数据库中，与本次拼接转录本exon区域有重叠的lncRNA作为数据库注释lncRNA纳入到后续分析；<br />
	step4. 编码潜能筛选：是否具有编码潜能是判断转录本是否为lncRNA的关键条件。针对上一步获得的转录本，我们综合了目前主流的编码潜能分析方法(CPC2/Pfam/CNCI)进行编码潜能预测，取这些软件结果中没有编码潜能的转录本的交集作为本次分析预测得到的候选Novel_lncRNA数据集，取这些软件结果中有编码潜能的转录本的交集作为本次分析预测得到的Novel_mRNA数据集；<br />
	step5. 参考HGNC（The HUGO Gene Nomenclature Committee ）（Wright M W, 2014）关于long non-coding RNA的命名指导标准，对候选的Novel_lncRNA根据其与编码基因的位置关系进行最终筛选及命名，得到本次分析的Novel_lncRNA。<br />
	</p>
</p>
	
	<p class="paragraph">Novel_lncRNA类型分布图见结果文件：3.Assembly_lncRNAFilter/lncRNAFilter/FilterStat/Novel_lncRNA_classification.png，如下图所示。</p>

</br>
</div>
<!---------------------------特征比较-------------------------->	
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_classification.png" /> </p>
	<p class="name_fig">图3.4 lncRNA 类型分布图</p>
	<p class="cenmark">诺禾致源参考HGNC根据和已知的mRNA的位置关系，将新筛选出的lncRNA分为4种类型。（1）antisense：与相反链的蛋白质编码基因的一个或多个外显子相重叠，或者有已发表的证据表明反向调控编码区基因；（2）lincRNA：位于基因间区的非编码RNA；（3）sense overlapping：与相同链的蛋白质编码基因的一个或多个外显子相重叠；（4）sense intronic：在一个编码基因的内含子区且不与同链上的任何外显子相交<sup>[10]</sup>。</p>
					
	

</br>
<div id="page">

	<a name="lncRNA与mRNA特征比较"><h4 id="result63">3.3 lncRNA与mRNA特征比较</h4></a>
	<p class="paragraph">将lncRNA与mRNA进行转录本长度、exon个数以及ORF长度的对比，一方面得到lncRNA与mRNA的差别，另一方面可以验证我们预测得到的novel lncRNA是否符合一般特征。</p>
</p>
	
	<p class="paragraph">lncRNA与mRNA特征比较见结果文件：3.Assembly_lncRNAFilter/lncRNAFilter/FeatureSignature，如下图所示。</p>

</br>		

	<h4 id="351">3.3.1 lncRNA 与 mRNA 的长度比较分析</h4>
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_length_density.png" /> </p>
	<p class="name_fig">图3.5 lncRNA 与 mRNA 的长度比较密度分布图</p>
	<p class="cenmark">图中横坐标为转录本长度，纵坐标为密度，紫色区域为已知的lncRNA（如果有基因组包含已知的lncRNA注释），红色区域为预测得到的新的lncRNA，通过此图可以验证预测的novel lncRNA的转录本长度是否符合一般特征（转录本长度超过200nt）。青色区域为已知的mRNA，橙色区域为预测得到的新的mRNA，用以比较和lncRNA的差别。</p>
	
	
	<br />
	
	<h4 id="351">3.3.2 lncRNA与mRNA的exon数目比较分析</h4>
	
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_exon_density.png" /> </p>
	
	<p class="name_fig">图3.6 lncRNA 与 mRNA 的 exon 数目密度分布图</p>
	<p class="cenmark">图中横坐标为转录本的exon数目，纵坐标为密度，紫色区域为已知的lncRNA（如果有基因组包含已知的lncRNA注释），红色区域为预测得到的新的lncRNA，通过两者对比可以验证预测的novel lncRNA的exon数目是否符合一般特征（exon个数大于等于2的转录本）。青色区域为已知的mRNA，橙色区域为预测得到的新的mRNA，用以比较和lncRNA的差别。</p>
	
	
	<br />
	
	<h4 id="351">3.3.3 lncRNA与mRNA的ORF长度比较分析</h4>
	<p class="paragraph">开放阅读框（open reading frame，ORF）是结构基因的正常核苷酸序列，从起始密码子到终止密码子的阅读框可编码完整的多肽链，期间不存在使翻译中断的终止密码子。我们对lncRNA与mRNA的ORF进行比较分析时，annotated mRNA和annotated lncRNA的序列通过已知基因结构注释进行提取，novel lncRNA和novel mRNA的ORF序列通过EMBOSS：getorf进行预测<sup>[11]</sup>。将得到的ORF序列转化为蛋白序列，得到长度分布如下图：</p>
	<p class="center"> <img class="normal" src="files/lncRNAFilter/lncRNA_orf_density.png" /> </p>
	<p class="name_fig">图3.7 lncRNA 与 mRNA 的 ORF长度密度分布图</p>
	<p class="cenmark">图中横坐标为预测ORF转化为蛋白序列后的长度，纵坐标为密度，紫色区域为已知的lncRNA（如果有基因组包含已知的lncRNA注释），红色区域为预测得到的新的lncRNA，通过两者对比可以验证预测的novel lncRNA的ORF长度是否符合一般特征。青色区域为已知的mRNA，橙色区域为预测得到的新的mRNA，用以比较和lncRNA的差别。</p>
	
	
	<br />
	</div>
	
<!-------------------------------3.4  转录本拼接视频链接------------------------------>
<div id="page">

	<a name="转录本拼接视频讲解"><h4 id="result64">3.4 转录本拼接讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1I9_SqvfOsPRWn4hWO4jeGw " class="uri">https://pan.baidu.com/s/1I9_SqvfOsPRWn4hWO4jeGw </a></font> 提取码: <font face="Arial">4t47 </font></p>

	<br />
</div>


<!------------------------------3 定量------------------------------>

<div id="page">

	<a name="定量分析"><h3 id="result3">4 定量分析(结果文件夹: Results/Quantification)</h3></a>

	<p class="paragraph">定量是表达差异显著性分析的基础，目前定量可分为基因水平和转录本水平定量。基因水平定量稳健可靠，在算法上容易实现，但无法精确到每个基因的不同转录本。转录本水平定量算法实现起来难度大，精准性不及基因水平定量，但近年来不断有新的算法被开发出来，精准性有显著提高<sup>[12]</sup>。诺禾致源同时提供基因水平和转录本水平的定量结果。</p>

	

	
	

	
	
	<p class="paragraph">StringTie应用网络流算法进行转录本和基因的拼接与定量，StringTie先将reads分为不同的类，然后再针对每个类的reads生成一个拼接图来确定转录本，之后每个转录本产生一个流神经网络的最大流算法来评估表达水平；分析流程如图4.1所示：
	</p>
	<p class="center">
		<img class="normal" src="images/Stringtie.png" />
	</p>
	<p class="name_fig">图4.1 Stringtie分析流程</p>
	
	<p class="paragraph">
	<br />
	<br />


</div>

<!--------------------------3.1 定量结果说明---------------------->


	<a name="定量结果说明"><h4 id="result31">4.1 定量结果说明</h4></a>
	
	<p class="paragraph">我们将上述比对、拼接、筛选后的已知转录本和预测得到的Novel_lncRNA、 Novel_mRNA以及Unclassified转录本全部进行表达水平定量。</p>
	
	
	<p class="paragraph">RNA-seq的基因表达水平通常用RPKM或FPKM表示。RPKM用于单端测序，FPKM用于双端测序，在lncRNA-seq中使用FPKM表示。FPKM （Fragments Per Kilobase of transcript sequence per Millions base pairs sequenced） 是每百万 fragments 中来自某一基因/转录本每千碱基长度的 fragments 数目，其同时校正了测序深度和基因长度对 fragments 计数的影响，是目前最为常用的基因表达水平估算方法。定量结果如表4.1所示：</p>
	</p>
	
	<p class="paragraph">mRNA基因水平表达量文件见结果文件：4.Quantification/mRNA_gene.FPKM.xls，如下图所示。</p>

</br>	

	<p class="name_fig">表4.1 各样品基因 FPKM 信息</p>

	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>gene_id</th><th>gene_name</th><th>gene_locus</th><th>sample1</th><th>sample2</th><th>sample3</th><th>sample4</th><th>sample5</th><th>sample6</th><th>sample7</th></tr>
			
		  
			
				<tr><td>ENSG00000230368</td><td>FAM41C</td><td>1:868071-876903</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000234711</td><td>TUBB8P11</td><td>1:873292-874349</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000283040</td><td>AL669831.6</td><td>1:874529-877234</td><td>0.000000</td><td>0.0</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.000000</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000187961</td><td>KLHL17</td><td>1:960584-965719</td><td>0.149956</td><td>1.747221</td><td>0.542480</td><td>0.000000</td><td>0.138708</td><td>0.273694</td><td>1.956710</td></tr>
			
		  
			
				<tr><td>ENSG00000187634</td><td>SAMD11</td><td>1:923928-944581</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.000000</td><td>0.245914</td><td>0.000000</td></tr>
			
		  
			
				<tr><td>ENSG00000188976</td><td>NOC2L</td><td>1:944203-959309</td><td>1.274181</td><td>5.670410</td><td>0.597106</td><td>2.340009</td><td>5.956353</td><td>4.225698</td><td>5.408510</td></tr>
			
		  
			
				<tr><td>ENSG00000187583</td><td>PLEKHN1</td><td>1:966497-975865</td><td>0.083042</td><td>0.0</td><td>0.000000</td><td>0.893553</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000187642</td><td>PERM1</td><td>1:975204-982093</td><td>0.244146</td><td>0.0</td><td>0.778349</td><td>0.000000</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>XLOC_007951</td><td>XLOC_007951</td><td>1:992563-993021</td><td>10.090649</td><td>0.0</td><td>5.403792</td><td>0.0</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		  
			
				<tr><td>ENSG00000272512</td><td>AL645608.7</td><td>1:995966-998051</td><td>12.563572</td><td>2.192926</td><td>5.093130</td><td>0.0</td><td>0.0</td><td>0.0</td><td>0.0</td></tr>
			
		   
		</table>
	</div>		
	</p>

	<p class="tremark">  
	(1) 第一列为基因/转录本ID，ENSG，G指基因；ENST，T指转录本。前三个字母（ENS），意思是“ENSEMBL”<br />
    (2) gene_name：基因名称，若无用"--"表示 <br />
    (3) locus: 基因在染色体上的位置 <br />
	(4) 此后每列为每个样本的基因/转录本ID所对应的FPKM值 <br />
	

</p>


</br>
<!---------------------------3.2 表达水平分布-------------------------->
<div id="page">


	<a name="表达水平分布"><h4 id="result32">4.2 表达水平分布</h4></a>

		
	
		
	<p class="paragraph">计算各样本所有基因或转录本的表达值（RPKM或FPKM）后，通过盒形图展示不同样本基因或转录本表达水平的分布情况，如图4.1所示。</p>

	<p class="center">
		<img class="normal1" src="files/Quantification/boxplot.png"/>
	</p>
	
	<p class="name_fig">图4.1 各样本表达水平分布</p>
	
	
	<p class="cenmark">不同样本表达水平盒形图，其中横坐标为样品名称，纵坐标为log<sub>10</sub>（FPKM+1），每个区域的盒形图对应五个统计量（从上而下分别为最大值、上四分位数、中值、下四分位数和最小值）</p>
	<br />
	<br />


</div>

<!-------------------------------3.3  定量视频链接------------------------------>
<div id="page">

	<a name="转录本定量视频讲解"><h4 id="result33">4.3 定量讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/11Auo6rK76_js3xytpe2q1Q  " class="uri">https://pan.baidu.com/s/11Auo6rK76_js3xytpe2q1Q  </a></font> 提取码: <font face="Arial">v3z8 </font></p>
    
	<br />
	<br />
</div>


<!---------------------------4 差异分析-------------------------->


<div id="page">



	<a name="基因水平差异分析"><h3 id="result4">5 差异分析(结果文件夹: Results/Diff)</h3></a>


<!---------------------------4.1 差异基因列表------------------------------>

	
	

		
	<p class="paragraph">定量分析完成后，得到所有样本的表达矩阵，便可进行基因或转录本水平的表达差异显著性分析，以便寻找与处理组相关的功能基因或转录本。诺禾致源采用edgeR软件进行表达差异显著性分析。用p value或矫正后的p value（padj）判断显著性水平。Padj是当假阳性率较高时利用BH的方法对p-value进行多重检验校正得到的值，得到的校正后的P-value 值越小代表越显著。将padj小于0.05作为差异显著性标准(若padj小于0.05筛选差异过少则使用p value小于0.05进行差异筛选，具体差异筛选条件见火山图)<sup>[13]</sup>。</p>
	
	
	</br>
	<a name="差异基因列表"><h4 id="result41">5.1 差异基因列表</h4></a>
	<p class="paragraph">mRNA基因水平差异表达情况见结果文件：5.Diff/mRNA/Group2_vs_Group1/Group2_vs_Group1.mRNA.diffgene.xls，如下图所示。</p>

</br>	

	<p class="paragraph">差异分析结果如表5.1所示。</p>

	<p class="name_fig">表5.1 差异基因列表</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>gene_id</th><th>gene_name</th><th>gene_locus</th><th>group1_FPKM</th><th>group2_FPKM</th><th>log2FoldChange</th><th>pvalue</th><th>padj</th></tr>
			
		  
			
				<tr><td>ENSG00000123689</td><td>G0S2</td><td>1:209675412-209676390</td><td>113.69630375</td><td>0.717038666666667</td><td>7.71505645130916</td><td>3.85428029984956e-11</td><td>3.70382472736685e-07</td></tr>
			
		  
			
				<tr><td>ENSG00000196126</td><td>HLA-DRB1</td><td>6:32578769-32589848</td><td>0.193821</td><td>62.0900873333333</td><td>-7.86453469597298</td><td>1.46687362421868e-10</td><td>7.68788466453008e-07</td></tr>
			
		  
			
				<tr><td>ENSG00000276293</td><td>PIP4K2B</td><td>17:38765689-38800126</td><td>0</td><td>3.722116</td><td>-9.22931376008883</td><td>8.88838729116198e-10</td><td>3.882003149415e-06</td></tr>
			
		  
			
				<tr><td>ENSG00000125740</td><td>FOSB</td><td>19:45467995-45475179</td><td>36.4045835</td><td>0.558543333333333</td><td>6.26371132063224</td><td>4.12337385018474e-09</td><td>1.35066264680114e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000112486</td><td>CCR6</td><td>6:167111807-167139696</td><td>0</td><td>4.481611</td><td>-8.91637270913355</td><td>4.76044581074459e-09</td><td>1.3860831385618e-05</td></tr>
			
		  
			
				<tr><td>XLOC_165816</td><td>XLOC_165816</td><td>CHR_HSCHR6_MHC_DBB_CTG1:31260956-31264273</td><td>198.17379125</td><td>0.478476</td><td>9.2393121544114</td><td>6.2465470190817e-09</td><td>1.63690764635036e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000135424</td><td>ITGA7</td><td>12:55684568-55716043</td><td>16.71872225</td><td>0.028137</td><td>8.7691186137305</td><td>9.24278614834938e-09</td><td>2.20188373652269e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000198502</td><td>HLA-DRB5</td><td>6:32517353-32530287</td><td>0</td><td>41.6268273333333</td><td>-10.8534545180771</td><td>2.30352389471382e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000205710</td><td>C17orf107</td><td>17:4899418-4902934</td><td>20.91248775</td><td>0.135322333333333</td><td>7.12690204181498</td><td>2.67060728521762e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
			
				<tr><td>ENSG00000181847</td><td>TIGIT</td><td>3:114276913-114310288</td><td>0.0847365</td><td>8.63292066666667</td><td>-5.99647519952058</td><td>2.8577533610694e-08</td><td>4.77263143232671e-05</td></tr>
			
		  
		</table>
	</div>				
	</p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
	<p class="tremark">
	(1) gene_id：基因编号<br />
	(2) gene_name：基因名称，若无用"--"表示<br />
	(3) locus: 基因在染色体上的位置<br />
	(4) group1：比较组合1的FPKM值<br />
	(5) group2：比较组合2的FPKM值<br />
	(6) log<sub>2</sub>FoldChange：两个样本或比较组合的差异倍数值，以log<sub>2</sub>（group1/group2）来表示，根据差异分析软件计算所得<br />
	(7) pvalue：显著性检验的p值<br />
	(8) padj：多重假设检验校正后的p值<br />
	</p>
</div>

	<br />
	<br />


</div>

<!-------------------------------4.2 差异基因火山图---------------------------------->

<div id="page">

	<a name="差异基因火山图"><h4 id="result42">5.2 差异基因火山图</h4></a>
	<p class="paragraph">火山图可直观显示表达差异显著性基因的整体分布情况。</p>
	
	<p class="paragraph">mRNA基因水平差异表达火山图见结果文件：5.Diff/mRNA/Group2_vs_Group1/Group2_vs_Group1.mRNA.diffgene.Volcanoplot.png，如下图所示。</p>

</br>	
	<p class="center">

		<div class="albumSlider">
					<div class="fullview">
						<img src=group1vsgroup2/>
					</div>
					<div class="slider">
						<div class="button movebackward" title="向上滚动">
						</div>
						<div class="imglistwrap">
							<ul class="imglist">
								
								<li><a id="example2" href=group1vsgroup2 ><img src=files/DiffEnrich/Volcano/group1vsgroup2.diffgene.Volcanoplot.png/></a></li>
								
								<li><a id="example2" href=group2vssample1 ><img src=files/DiffEnrich/Volcano/group2vssample1.diffgene.Volcanoplot.png/></a></li>
								
								<li><a id="example2" href=sample1vssample5 ><img src=files/DiffEnrich/Volcano/sample1vssample5.diffgene.Volcanoplot.png/></a></li>
								
							</ul>
						</div>
						<div class="button moveforward" title="向下滚动">
						</div>
					</div>
				</div>	
			
	</p>

<p class="name_fig">图5.1 差异基因火山图</p>
	<p class="cenmark">横坐标表示基因在不同样本或比较组合间的表达倍数变化（log<sub>2</sub>FoldChange），横坐标的绝对值越大表明两个比较组合之间的表达变化倍数越大; 纵坐标表示表
达差异的显著性水平。表达上调基因用红色点表示，下调基因用绿色点表示，蓝色点为未发生显著变化的基因。</p>


	<br />
	<br />


</div>


<!---------------------------4.3 差异基因聚类-------------------------->

<div id="page">

        <a name="差异基因聚类"><h4 id="result43">5.3 差异基因聚类</h4></a>

        <p class="paragraph">聚类图是差异表达基因的另一种展示方式，将表达模式相近的基因聚在一起，这些基因可能具有共同的功能或参与到共同的代谢途径及信号通路中。诺禾致源采用主流的层次聚类法，将log<sub>10</sub>（FPKM+1）值进行转换并进行聚类。聚类结果用热图（heatmap）表示，如图5.2所示。</p>
 
</p>
	
	<p class="paragraph">mRNA基因水平差异表达聚类图见结果文件：5.Diff/mRNA/Cluster/Hcluster_heatmap.png，如下图所示。</p>

</br>

	<p class="center">
		<img class="normal1" src="files/DiffEnrich/Cluster/Hcluster_heatmap.png"/>
		</p>

<p class="name_fig">图5.2 差异基因层次聚类热图</p>
		<p class="cenmark">横坐标为样本，纵坐标为差异基因，左侧根据表达相似程度对基因进行聚类，上方根据表达谱的相似程度对每个样本进行聚类，由蓝至红表达量逐渐上调，数字为均一化后的相对表达量。</p>

        <br />
        <br />


</div>

<!-------------------------------4.4  差异视频链接------------------------------>
<div id="page">

	<a name="差异分析视频讲解"><h4 id="result44">5.4 差异分析讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1SVMsT_2IBFofLokhbSdGdg" class="uri">https://pan.baidu.com/s/1SVMsT_2IBFofLokhbSdGdg</a></font> 提取码: <font face="Arial">ww3i </font></p>
    
	<br />
	<br />
</div>

<!---------------------------5 富集分析---------------------------------->


<div id="page">


	<a name="GO/KEGG 富集分析"><h3  id="result5">6 GO/KEGG 富集分析(结果文件夹: Results/DiffEnrich_mRNA)</h3></a>
	
	
	
	<p class="paragraph">在生物体内，不同基因相互协调行驶其生物学功能。通过pathway显著性富集可以探寻差异表达基因参与的最主要的生化代谢途径和信号转导途径。诺禾致源采用clusterProfiler（http://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html）软件，选取广泛使用的注释基因数据库GO、KEGG，对差异基因进行通路富集分析（Yu G et al., 2012）。GO（Gene Ontology）是描述基因功能的综合性数据库，可分为分子功能（Molecular Function），生物过程（biological process）和细胞组成（cellular component）三个部分。KEGG（Kyoto Encyclopedia of Genes and Genomes）是整合了基因组、化学和系统功能信息的综合性数据库。</p>
	
	
	
	<p class="paragraph">富集分析基于超几何分布原理，如下图所示，其中差异基因集为经分析所得显著差异基因列表（能够注释到ENTREZ数据库中），背景基因集为所有注释到ENTREZ数据库的进行差异显著分析的基因集。</p>

	<p class="center">
		<img class="normala2" src="images/enrichment.png" />
	</p>

	<p class="name_fig">图6.1 富集分析原理</p>


	<br />
	<br />

</div>

<!--------------------------5.1 GO富集分析-------------------------->
<div id="page">

<!-- go富集方法需要优化完善 -->

	<a name="GO功能富集"><h4 id="result51">6.1 GO功能富集</h4></a>
	<p class="paragraph">Gene Ontology（简称 GO，http://www.geneontology.org/）是基因功能国际标准分类体系。GO可分为分子功能（Molecular Function），生物过程（biological process）和细胞组成（cellular component）三个部分。GO 富集分析方法为 GOseq，此方法基于 Wallenius non-central hyper-geometric distribution，相对于普通的 Hyper-geometric distribution，此分布的特点是从某个类别中抽取个体的概率与从某个类别之外抽取一个个体的概率是不同的，这种概率的不同是通过对基因长度的偏好性进行估计得到的，从而能更为准确地计算出 GO term 被来源基因富集的概率。GO富集以padj小于0.05（默认）或pvalue小于0.05为显著富集。富集结果如表6.1所示：</p>

</p>
	
	<p class="paragraph">mRNA基因表达水平GO富集分析见结果文件：6.DiffEnrich_mRNA/Group2_vs_Group1/GO，如下图所示。</p>

</br>

	<p class="name_fig">表6.1 GO富集列表</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>Category</th><th>GOID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0002521</td><td>leukocyte differentiation</td><td>126/2199</td><td>455/13522</td><td>2.68112374545469e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000177606/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000173334/ENSG00000170345/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000161405/ENSG00000112715/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000171223/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000155090/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000010610/ENSG00000182578/ENSG00000073861/ENSG00000198286/ENSG00000204577/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000108840/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000100985/ENSG00000116251/ENSG00000100105/ENSG00000142185/ENSG00000110719/ENSG00000115232/ENSG00000132170/ENSG00000172216/ENSG00000134321/ENSG00000111203/ENSG00000110651/ENSG00000134256/ENSG00000122224/ENSG00000135905/ENSG00000068323/ENSG00000131188/ENSG00000172575/ENSG00000135898/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000128590/ENSG00000138795/ENSG00000012223/ENSG00000120899/ENSG00000137154/ENSG00000126353/ENSG00000185507/ENSG00000104856/ENSG00000182885/ENSG00000135426/ENSG00000185338/ENSG00000118260/ENSG00000168421/ENSG00000071553/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000154229/ENSG00000160654/ENSG00000029993/ENSG00000168264/ENSG00000149311/ENSG00000169508/ENSG00000143799/ENSG00000125686/ENSG00000143507/ENSG00000113916/ENSG00000128342/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000081189/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000141968/ENSG00000067082/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000123360/ENSG00000072274/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000112062/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000169710/ENSG00000136997/ENSG00000163374/ENSG00000002330</td><td>CCR6/JUN/CD83/NFATC2/NFKBID/TRIB1/FOS/THEMIS/EGR1/IL18R1/IKZF3/VEGFA/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/JUNB/CD2/CD40LG/SPN/CD8A/KLF10/EGR3/CSF1/SLAMF8/CD4/CSF1R/TBX21/CARD11/LILRB3/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/HDAC5/GATA3/FCRL3/PRKDC/LCK/TCF7/MMP9/RPL22/PATZ1/TRPM2/TCIRG1/ITGA4/PPARG/CEBPB/RSAD2/ITFG2/CD81/CD101/LY9/DOCK10/TFE3/PRR7/RASGRP1/GPR55/FZD5/TNFSF4/BCL3/DNAJB9/LEF1/LTF/PTK2B/RPS6/CCR7/IRF7/RELB/ADGRG3/TESPA1/SOCS1/CREB1/RHOH/ATP6AP1/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/FNIP1/PRKCA/CD3G/HMGB3/IRF2BP2/ATM/GPR183/PARP1/MED1/DUSP10/BCL6/LIF/IL7R/CLEC4D/VSIR/CD3D/CCR9/MEF2C/CAMK4/ITM2A/BCL2/VAV1/KLF6/WNT1/CD79B/SLAMF6/PDE1B/TFRC/CD79A/SOX13/CD74/MAPK14/IKZF1/LOXL3/RAG1/FASN/MYC/YY1AP1/BAD</td><td>126</td><td>54</td><td>ENSG00000177606/ENSG00000112149/ENSG00000167604/ENSG00000173334/ENSG00000170345/ENSG00000120738/ENSG00000115604/ENSG00000112715/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000171223/ENSG00000155090/ENSG00000179388/ENSG00000184371/ENSG00000204577/ENSG00000158869/ENSG00000112299/ENSG00000108840/ENSG00000100985/ENSG00000142185/ENSG00000110719/ENSG00000132170/ENSG00000172216/ENSG00000134321/ENSG00000068323/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000128590/ENSG00000012223/ENSG00000120899/ENSG00000185507/ENSG00000104856/ENSG00000182885/ENSG00000185338/ENSG00000071553/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000029993/ENSG00000168264/ENSG00000143507/ENSG00000113916/ENSG00000128342/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000067082/ENSG00000123360/ENSG00000072274/ENSG00000112062/ENSG00000163374/ENSG00000002330</td><td>72</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000161405/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000158714/ENSG00000010610/ENSG00000182578/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000115232/ENSG00000111203/ENSG00000110651/ENSG00000134256/ENSG00000122224/ENSG00000135905/ENSG00000172575/ENSG00000135898/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000118260/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000154229/ENSG00000160654/ENSG00000149311/ENSG00000169508/ENSG00000143799/ENSG00000125686/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000081189/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000169710/ENSG00000136997</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0030098</td><td>lymphocyte differentiation</td><td>95/2199</td><td>317/13522</td><td>5.34192600043356e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000161405/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000158714/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000108840/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110719/ENSG00000115232/ENSG00000134321/ENSG00000111203/ENSG00000122224/ENSG00000135905/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000128590/ENSG00000138795/ENSG00000120899/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000182885/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000160654/ENSG00000029993/ENSG00000168264/ENSG00000149311/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000141968/ENSG00000067082/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349/ENSG00000163374/ENSG00000002330</td><td>CCR6/CD83/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/IKZF3/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/CD2/CD40LG/SPN/CD8A/EGR3/SLAMF8/CD4/TBX21/CARD11/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/HDAC5/GATA3/FCRL3/PRKDC/LCK/TCF7/RPL22/PATZ1/TCIRG1/ITGA4/RSAD2/ITFG2/LY9/DOCK10/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/DNAJB9/LEF1/PTK2B/RPS6/CCR7/RELB/ADGRG3/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/FNIP1/CD3G/HMGB3/IRF2BP2/ATM/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/CCR9/CAMK4/ITM2A/BCL2/VAV1/KLF6/WNT1/CD79B/SLAMF6/CD79A/SOX13/CD74/IKZF1/LOXL3/RAG1/YY1AP1/BAD</td><td>95</td><td>34</td><td>ENSG00000112149/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000158869/ENSG00000112299/ENSG00000108840/ENSG00000110719/ENSG00000134321/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000128590/ENSG00000120899/ENSG00000104856/ENSG00000182885/ENSG00000185338/ENSG00000168036/ENSG00000104312/ENSG00000217128/ENSG00000029993/ENSG00000168264/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000067082/ENSG00000163374/ENSG00000002330</td><td>61</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000161405/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000158714/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000160856/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000115232/ENSG00000111203/ENSG00000122224/ENSG00000135905/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000160654/ENSG00000149311/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000078596/ENSG00000171791/ENSG00000125084/ENSG00000007312/ENSG00000162739/ENSG00000105369/ENSG00000143842/ENSG00000019582/ENSG00000185811/ENSG00000115318/ENSG00000166349</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0030217</td><td>T cell differentiation</td><td>74/2199</td><td>226/13522</td><td>5.97468406956998e-10</td><td>1.13359672413308e-06</td><td>ENSG00000112486/ENSG00000112149/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000163508/ENSG00000116824/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000158869/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000107485/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110719/ENSG00000134321/ENSG00000122224/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000168036/ENSG00000104312/ENSG00000160654/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000171791/ENSG00000141968/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349/ENSG00000002330</td><td>CCR6/CD83/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/EOMES/CD2/SPN/CD8A/EGR3/CD4/TBX21/CARD11/FCER1G/ZAP70/CD3E/TP53/ITK/RORC/VNN1/GATA3/PRKDC/LCK/TCF7/RPL22/PATZ1/TCIRG1/RSAD2/LY9/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/LEF1/RPS6/CCR7/RELB/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/ABL1/CTNNB1/RIPK2/CD3G/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/CCR9/CAMK4/BCL2/VAV1/WNT1/SLAMF6/SOX13/CD74/LOXL3/RAG1/BAD</td><td>74</td><td>25</td><td>ENSG00000112149/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000158869/ENSG00000112299/ENSG00000110719/ENSG00000134321/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000104856/ENSG00000185338/ENSG00000168036/ENSG00000104312/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000002330</td><td>49</td><td>ENSG00000112486/ENSG00000101096/ENSG00000172673/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000163508/ENSG00000116824/ENSG00000197471/ENSG00000153563/ENSG00000010610/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000122224/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000097007/ENSG00000160654/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000173585/ENSG00000152495/ENSG00000171791/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0006935</td><td>chemotaxis</td><td>126/2199</td><td>477/13522</td><td>6.51313530286691e-09</td><td>9.26819153597961e-06</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000169429/ENSG00000183691/ENSG00000122861/ENSG00000113070/ENSG00000160791/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000127152/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000120129/ENSG00000182578/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000168329/ENSG00000170962/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000163735/ENSG00000107485/ENSG00000012171/ENSG00000183134/ENSG00000121797/ENSG00000168758/ENSG00000170989/ENSG00000128512/ENSG00000149294/ENSG00000137285/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000162415/ENSG00000156299/ENSG00000167680/ENSG00000114554/ENSG00000010810/ENSG00000182197/ENSG00000020181/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000120899/ENSG00000117525/ENSG00000116106/ENSG00000092969/ENSG00000163220/ENSG00000126353/ENSG00000143185/ENSG00000001617/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000206047/ENSG00000141622/ENSG00000163739/ENSG00000258947/ENSG00000100784/ENSG00000276085/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000169508/ENSG00000163932/ENSG00000105372/ENSG00000154188/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000132003/ENSG00000173585/ENSG00000090776/ENSG00000119630/ENSG00000121966/ENSG00000186810/ENSG00000141968/ENSG00000131437/ENSG00000138448/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000164171/ENSG00000173531/ENSG00000168546/ENSG00000019582/ENSG00000112062/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000106366/ENSG00000006451/ENSG00000198211</td><td>CCR6/KLRK1/TMEM102/CXCL8/NOG/PLAU/HBEGF/CCR5/VEGFA/FOSL1/NR4A3/BCL11B/EDN1/PLAUR/S100A12/CMKLR1/SPN/AMOT/EGR3/CSF1/SLAMF8/DUSP1/CSF1R/GRB10/IL1RN/IRS2/CX3CR1/PDGFD/PDLIM7/CXCL16/FCER1G/CXCL2/RAP1GAP/CXCL5/GATA3/SEMA3B/PTGDR2/CCRL2/SEMA4C/S1PR1/DOCK4/NCAM1/TUBB2B/TRPM2/S100A8/NTNG2/ZSWIM5/TIAM1/SEMA6B/PLXNA1/FYN/EXT1/ADGRA2/DRAXIN/DOK2/CXCR5/PLXND1/EVL/FPR3/MTUS1/SEMA4F/NEO1/LGR6/CXCR6/MEGF8/LEF1/PTCH1/PLCG1/PTK2B/F3/EPHA4/TGFB2/S100A9/CCR7/XCL2/SEMA3F/CREB1/OPHN1/ERBB2/DEFA1/RNF165/CXCL1/TUBB3/RPS6KA5/CCL3L1/PLXNB2/PRKCA/CCL28/NRP1/B3GNT2/C3AR1/SPTBN4/GPR183/PRKCD/RPS19/ANGPT1/MAPK8IP3/CREB3/CSF3R/LPAR1/NRCAM/CNR2/ZSWIM4/CCR9/EFNB1/PGF/CXCR4/CXCR3/VAV1/KIF3A/ITGAV/TNFSF14/PLP2/GRB2/CYP19A1/ITGA2/MST1/GFRA2/CD74/MAPK14/CYFIP1/MYCBP2/SWAP70/SERPINE1/RALA/AC092143.1</td><td>126</td><td>56</td><td>ENSG00000169429/ENSG00000122861/ENSG00000113070/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000179388/ENSG00000184371/ENSG00000120129/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000012171/ENSG00000121797/ENSG00000128512/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000167680/ENSG00000182197/ENSG00000020181/ENSG00000120899/ENSG00000117525/ENSG00000163220/ENSG00000001617/ENSG00000206047/ENSG00000163739/ENSG00000258947/ENSG00000276085/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000163932/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000132003/ENSG00000119630/ENSG00000121966/ENSG00000141968/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000173531/ENSG00000112062/ENSG00000106366/ENSG00000198211</td><td>70</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000183691/ENSG00000160791/ENSG00000127152/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000158714/ENSG00000182578/ENSG00000168329/ENSG00000170962/ENSG00000163735/ENSG00000107485/ENSG00000183134/ENSG00000168758/ENSG00000170989/ENSG00000149294/ENSG00000137285/ENSG00000162415/ENSG00000156299/ENSG00000114554/ENSG00000010810/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000116106/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000141622/ENSG00000100784/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000169508/ENSG00000105372/ENSG00000154188/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000173585/ENSG00000090776/ENSG00000186810/ENSG00000131437/ENSG00000138448/ENSG00000164171/ENSG00000168546/ENSG00000019582/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000006451</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0042330</td><td>taxis</td><td>126/2199</td><td>479/13522</td><td>8.54263111304989e-09</td><td>9.72493125909599e-06</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000169429/ENSG00000183691/ENSG00000122861/ENSG00000113070/ENSG00000160791/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000127152/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000179388/ENSG00000184371/ENSG00000158714/ENSG00000120129/ENSG00000182578/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000168329/ENSG00000170962/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000163735/ENSG00000107485/ENSG00000012171/ENSG00000183134/ENSG00000121797/ENSG00000168758/ENSG00000170989/ENSG00000128512/ENSG00000149294/ENSG00000137285/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000162415/ENSG00000156299/ENSG00000167680/ENSG00000114554/ENSG00000010810/ENSG00000182197/ENSG00000020181/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000120899/ENSG00000117525/ENSG00000116106/ENSG00000092969/ENSG00000163220/ENSG00000126353/ENSG00000143185/ENSG00000001617/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000206047/ENSG00000141622/ENSG00000163739/ENSG00000258947/ENSG00000100784/ENSG00000276085/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000169508/ENSG00000163932/ENSG00000105372/ENSG00000154188/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000132003/ENSG00000173585/ENSG00000090776/ENSG00000119630/ENSG00000121966/ENSG00000186810/ENSG00000141968/ENSG00000131437/ENSG00000138448/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000164171/ENSG00000173531/ENSG00000168546/ENSG00000019582/ENSG00000112062/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000106366/ENSG00000006451/ENSG00000198211</td><td>CCR6/KLRK1/TMEM102/CXCL8/NOG/PLAU/HBEGF/CCR5/VEGFA/FOSL1/NR4A3/BCL11B/EDN1/PLAUR/S100A12/CMKLR1/SPN/AMOT/EGR3/CSF1/SLAMF8/DUSP1/CSF1R/GRB10/IL1RN/IRS2/CX3CR1/PDGFD/PDLIM7/CXCL16/FCER1G/CXCL2/RAP1GAP/CXCL5/GATA3/SEMA3B/PTGDR2/CCRL2/SEMA4C/S1PR1/DOCK4/NCAM1/TUBB2B/TRPM2/S100A8/NTNG2/ZSWIM5/TIAM1/SEMA6B/PLXNA1/FYN/EXT1/ADGRA2/DRAXIN/DOK2/CXCR5/PLXND1/EVL/FPR3/MTUS1/SEMA4F/NEO1/LGR6/CXCR6/MEGF8/LEF1/PTCH1/PLCG1/PTK2B/F3/EPHA4/TGFB2/S100A9/CCR7/XCL2/SEMA3F/CREB1/OPHN1/ERBB2/DEFA1/RNF165/CXCL1/TUBB3/RPS6KA5/CCL3L1/PLXNB2/PRKCA/CCL28/NRP1/B3GNT2/C3AR1/SPTBN4/GPR183/PRKCD/RPS19/ANGPT1/MAPK8IP3/CREB3/CSF3R/LPAR1/NRCAM/CNR2/ZSWIM4/CCR9/EFNB1/PGF/CXCR4/CXCR3/VAV1/KIF3A/ITGAV/TNFSF14/PLP2/GRB2/CYP19A1/ITGA2/MST1/GFRA2/CD74/MAPK14/CYFIP1/MYCBP2/SWAP70/SERPINE1/RALA/AC092143.1</td><td>126</td><td>56</td><td>ENSG00000169429/ENSG00000122861/ENSG00000113070/ENSG00000112715/ENSG00000175592/ENSG00000119508/ENSG00000078401/ENSG00000011422/ENSG00000163221/ENSG00000179388/ENSG00000184371/ENSG00000120129/ENSG00000106070/ENSG00000136689/ENSG00000185950/ENSG00000196923/ENSG00000161921/ENSG00000158869/ENSG00000081041/ENSG00000076864/ENSG00000012171/ENSG00000121797/ENSG00000128512/ENSG00000142185/ENSG00000143546/ENSG00000196358/ENSG00000167680/ENSG00000182197/ENSG00000020181/ENSG00000120899/ENSG00000117525/ENSG00000163220/ENSG00000001617/ENSG00000206047/ENSG00000163739/ENSG00000258947/ENSG00000276085/ENSG00000170340/ENSG00000171860/ENSG00000160460/ENSG00000163932/ENSG00000138834/ENSG00000107175/ENSG00000119535/ENSG00000132003/ENSG00000119630/ENSG00000121966/ENSG00000141968/ENSG00000125735/ENSG00000102007/ENSG00000177885/ENSG00000137869/ENSG00000173531/ENSG00000112062/ENSG00000106366/ENSG00000198211</td><td>70</td><td>ENSG00000112486/ENSG00000213809/ENSG00000181284/ENSG00000183691/ENSG00000160791/ENSG00000127152/ENSG00000174600/ENSG00000197471/ENSG00000126016/ENSG00000158714/ENSG00000182578/ENSG00000168329/ENSG00000170962/ENSG00000163735/ENSG00000107485/ENSG00000183134/ENSG00000168758/ENSG00000170989/ENSG00000149294/ENSG00000137285/ENSG00000162415/ENSG00000156299/ENSG00000114554/ENSG00000010810/ENSG00000162490/ENSG00000147443/ENSG00000160683/ENSG00000004399/ENSG00000196405/ENSG00000187474/ENSG00000129422/ENSG00000135622/ENSG00000067141/ENSG00000133067/ENSG00000172215/ENSG00000105429/ENSG00000138795/ENSG00000185920/ENSG00000124181/ENSG00000116106/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000118260/ENSG00000079482/ENSG00000141736/ENSG00000141622/ENSG00000100784/ENSG00000196576/ENSG00000154229/ENSG00000151882/ENSG00000099250/ENSG00000169508/ENSG00000105372/ENSG00000154188/ENSG00000198121/ENSG00000091129/ENSG00000188822/ENSG00000173585/ENSG00000090776/ENSG00000186810/ENSG00000131437/ENSG00000138448/ENSG00000164171/ENSG00000168546/ENSG00000019582/ENSG00000273749/ENSG00000005810/ENSG00000133789/ENSG00000006451</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0019080</td><td>viral gene expression</td><td>60/2199</td><td>187/13522</td><td>5.69661156368897e-08</td><td>5.4041855034196e-05</td><td>ENSG00000177606/ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000128016/ENSG00000080561/ENSG00000153922/ENSG00000174444/ENSG00000089009/ENSG00000175390/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000142089/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000100129/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000139496/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000140564/ENSG00000144231/ENSG00000181222/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000130811/ENSG00000132376/ENSG00000113569/ENSG00000110713/ENSG00000213741</td><td>JUN/RPS3A/RPL3/TRIM32/RPL22/RPS4X/ZFP36/MID2/CHD1/RPL4/AC004086.1/EIF3F/RPS23/RPS27A/RRP1B/RPSA/IFITM3/MDFIC/LEF1/RPS6/RPL10/RPL11/EIF3L/RPL23A/NUCKS1/RPS20/NUP210/RPS2/RPS21/RPS28/RPL10A/RPS18/NUP58/RPL35/NUP160/RPL13/RPL7A/RPS19/RPL19/RPL13A/RPS8/NUP188/RPL15/RPL26/NUP43/RPL23/CTDP1/SUPT5H/RANBP2/FURIN/POLR2D/POLR2A/RPL5/TFAP4/RPS26/EIF3G/INPP5K/NUP155/NUP98/RPS29</td><td>60</td><td>12</td><td>ENSG00000177606/ENSG00000128016/ENSG00000153922/ENSG00000142089/ENSG00000139496/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000140564/ENSG00000181222/ENSG00000132376/ENSG00000110713</td><td>48</td><td>ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000080561/ENSG00000174444/ENSG00000089009/ENSG00000175390/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000100129/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000144231/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000130811/ENSG00000113569/ENSG00000213741</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0019083</td><td>viral transcription</td><td>56/2199</td><td>173/13522</td><td>1.11882349662366e-07</td><td>8.85642947043187e-05</td><td>ENSG00000177606/ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000128016/ENSG00000080561/ENSG00000153922/ENSG00000174444/ENSG00000089009/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000142089/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000139496/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000144231/ENSG00000181222/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000132376/ENSG00000113569/ENSG00000110713/ENSG00000213741</td><td>JUN/RPS3A/RPL3/TRIM32/RPL22/RPS4X/ZFP36/MID2/CHD1/RPL4/AC004086.1/RPS23/RPS27A/RRP1B/RPSA/IFITM3/MDFIC/LEF1/RPS6/RPL10/RPL11/RPL23A/NUCKS1/RPS20/NUP210/RPS2/RPS21/RPS28/RPL10A/RPS18/NUP58/RPL35/NUP160/RPL13/RPL7A/RPS19/RPL19/RPL13A/RPS8/NUP188/RPL15/RPL26/NUP43/RPL23/CTDP1/SUPT5H/RANBP2/POLR2D/POLR2A/RPL5/TFAP4/RPS26/INPP5K/NUP155/NUP98/RPS29</td><td>56</td><td>11</td><td>ENSG00000177606/ENSG00000128016/ENSG00000153922/ENSG00000142089/ENSG00000139496/ENSG00000060069/ENSG00000196235/ENSG00000153201/ENSG00000181222/ENSG00000132376/ENSG00000110713</td><td>45</td><td>ENSG00000145425/ENSG00000100316/ENSG00000119401/ENSG00000116251/ENSG00000198034/ENSG00000080561/ENSG00000174444/ENSG00000089009/ENSG00000186468/ENSG00000143947/ENSG00000160208/ENSG00000168028/ENSG00000135272/ENSG00000138795/ENSG00000137154/ENSG00000147403/ENSG00000142676/ENSG00000198242/ENSG00000069275/ENSG00000008988/ENSG00000132182/ENSG00000140988/ENSG00000171858/ENSG00000233927/ENSG00000198755/ENSG00000231500/ENSG00000136942/ENSG00000030066/ENSG00000167526/ENSG00000148303/ENSG00000105372/ENSG00000108298/ENSG00000142541/ENSG00000142937/ENSG00000095319/ENSG00000174748/ENSG00000161970/ENSG00000120253/ENSG00000125691/ENSG00000144231/ENSG00000122406/ENSG00000090447/ENSG00000197728/ENSG00000113569/ENSG00000213741</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0042110</td><td>T cell activation</td><td>109/2199</td><td>417/13522</td><td>1.24475466906983e-07</td><td>8.85642947043187e-05</td><td>ENSG00000196126/ENSG00000112486/ENSG00000181847/ENSG00000112149/ENSG00000213809/ENSG00000090339/ENSG00000101096/ENSG00000167604/ENSG00000172673/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000144802/ENSG00000158481/ENSG00000117281/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000179388/ENSG00000013725/ENSG00000010610/ENSG00000110448/ENSG00000073861/ENSG00000198286/ENSG00000128271/ENSG00000158869/ENSG00000115085/ENSG00000167984/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000112299/ENSG00000107485/ENSG00000172116/ENSG00000253729/ENSG00000118520/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000196352/ENSG00000100105/ENSG00000110719/ENSG00000018280/ENSG00000172216/ENSG00000134321/ENSG00000107968/ENSG00000110651/ENSG00000122224/ENSG00000173762/ENSG00000010810/ENSG00000105205/ENSG00000131188/ENSG00000172575/ENSG00000163251/ENSG00000117586/ENSG00000069399/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000104856/ENSG00000197635/ENSG00000135426/ENSG00000185338/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000138735/ENSG00000097007/ENSG00000168036/ENSG00000102034/ENSG00000104312/ENSG00000160654/ENSG00000159958/ENSG00000169508/ENSG00000143507/ENSG00000113916/ENSG00000168685/ENSG00000166527/ENSG00000107738/ENSG00000167286/ENSG00000151208/ENSG00000100351/ENSG00000173585/ENSG00000152495/ENSG00000090776/ENSG00000171791/ENSG00000141968/ENSG00000125084/ENSG00000162739/ENSG00000197329/ENSG00000125735/ENSG00000072274/ENSG00000177885/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000137507/ENSG00000166349/ENSG00000168071/ENSG00000213658/ENSG00000002330</td><td>HLA-DRB1/CCR6/TIGIT/CD83/KLRK1/ICAM1/NFATC2/NFKBID/THEMIS/EGR1/IL18R1/ZC3H12A/JMJD6/RORA/BCL11B/DTX1/CD28/NFKBIZ/CD1C/CD160/EOMES/CD2/CD40LG/SPN/CD8A/EGR3/CD6/CD4/CD5/TBX21/CARD11/ADORA2A/FCER1G/ZAP70/NLRC3/CD3E/TP53/ITK/RORC/VNN1/GATA3/CD8B/PRKDC/ARG1/LCK/TCF7/RPL22/CD55/PATZ1/TCIRG1/SLC11A1/CEBPB/RSAD2/MAP3K8/CD81/LY9/CD7/FYN/CLC/PRR7/RASGRP1/FZD5/TNFSF4/BCL3/LEF1/RPS6/CCR7/RELB/DPP4/TESPA1/SOCS1/RHOH/ERBB2/RUNX3/PDE5A/ABL1/CTNNB1/ELF4/RIPK2/CD3G/TNFRSF13C/GPR183/DUSP10/BCL6/IL7R/CLEC4D/VSIR/CD3D/DLG5/GRAP2/CCR9/CAMK4/EFNB1/BCL2/VAV1/WNT1/SLAMF6/PELI1/TNFSF14/TFRC/GRB2/SOX13/CD74/LOXL3/LRRC32/RAG1/CCDC88B/LAT/BAD</td><td>109</td><td>39</td><td>ENSG00000112149/ENSG00000090339/ENSG00000167604/ENSG00000120738/ENSG00000115604/ENSG00000163874/ENSG00000070495/ENSG00000144802/ENSG00000179388/ENSG00000128271/ENSG00000158869/ENSG00000112299/ENSG00000118520/ENSG00000196352/ENSG00000110719/ENSG00000018280/ENSG00000172216/ENSG00000134321/ENSG00000107968/ENSG00000131188/ENSG00000163251/ENSG00000069399/ENSG00000104856/ENSG00000185338/ENSG00000168036/ENSG00000102034/ENSG00000104312/ENSG00000143507/ENSG00000113916/ENSG00000166527/ENSG00000107738/ENSG00000141968/ENSG00000197329/ENSG00000125735/ENSG00000072274/ENSG00000177885/ENSG00000137507/ENSG00000168071/ENSG00000002330</td><td>70</td><td>ENSG00000196126/ENSG00000112486/ENSG00000181847/ENSG00000213809/ENSG00000101096/ENSG00000172673/ENSG00000069667/ENSG00000127152/ENSG00000135144/ENSG00000178562/ENSG00000158481/ENSG00000117281/ENSG00000163508/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000013725/ENSG00000010610/ENSG00000110448/ENSG00000073861/ENSG00000198286/ENSG00000115085/ENSG00000167984/ENSG00000198851/ENSG00000141510/ENSG00000113263/ENSG00000143365/ENSG00000107485/ENSG00000172116/ENSG00000253729/ENSG00000182866/ENSG00000081059/ENSG00000116251/ENSG00000100105/ENSG00000110651/ENSG00000122224/ENSG00000173762/ENSG00000010810/ENSG00000105205/ENSG00000172575/ENSG00000117586/ENSG00000138795/ENSG00000137154/ENSG00000126353/ENSG00000197635/ENSG00000135426/ENSG00000168421/ENSG00000141736/ENSG00000020633/ENSG00000138735/ENSG00000097007/ENSG00000160654/ENSG00000159958/ENSG00000169508/ENSG00000168685/ENSG00000167286/ENSG00000151208/ENSG00000100351/ENSG00000173585/ENSG00000152495/ENSG00000090776/ENSG00000171791/ENSG00000125084/ENSG00000162739/ENSG00000143842/ENSG00000019582/ENSG00000115318/ENSG00000166349/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0032496</td><td>response to lipopolysaccharide</td><td>77/2199</td><td>272/13522</td><td>3.15399048039311e-07</td><td>0.000199472375715529</td><td>ENSG00000177606/ENSG00000100906/ENSG00000213809/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000160791/ENSG00000163874/ENSG00000078401/ENSG00000187838/ENSG00000171223/ENSG00000165029/ENSG00000013725/ENSG00000168615/ENSG00000181104/ENSG00000131979/ENSG00000066468/ENSG00000136689/ENSG00000168329/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000163735/ENSG00000108840/ENSG00000077150/ENSG00000111961/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000025434/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000117586/ENSG00000127666/ENSG00000239839/ENSG00000012223/ENSG00000126353/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000097007/ENSG00000104312/ENSG00000154229/ENSG00000162551/ENSG00000081985/ENSG00000185187/ENSG00000162889/ENSG00000197894/ENSG00000135334/ENSG00000159674/ENSG00000005381/ENSG00000162892/ENSG00000173039/ENSG00000143507/ENSG00000188822/ENSG00000081189/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000117560/ENSG00000204397/ENSG00000112062/ENSG00000162676/ENSG00000106366/ENSG00000147465</td><td>JUN/NFKBIA/KLRK1/ADM/ICAM1/CXCL8/TNFAIP3/PTGES/TRIB1/FOS/IRAK2/CCR5/ZC3H12A/EDN1/PLSCR3/JUNB/ABCA1/CD6/ADAM9/F2R/GCH1/FGFR2/IL1RN/CX3CR1/JUND/IRAK3/CXCL2/CXCL5/HDAC5/NFKB2/SASH1/LOXL1/ARG1/GJB6/NR1H3/IRAK1/CD55/S100A8/SLC11A1/CEBPB/CMPK2/ZFP36/THBD/TNFSF4/TICAM1/DEFA3/LTF/CCR7/GGT1/DEFA1/CXCL1/ABL1/RIPK2/PRKCA/ALPL/IL12RB2/SIGIRR/MAPKAPK2/ADH5/AKIRIN2/SPON2/MPO/IL24/RELA/DUSP10/CNR2/MEF2C/PLAA/DEFA4/PPARD/PELI1/FASLG/CARD16/MAPK14/GFI1/SERPINE1/STAR</td><td>77</td><td>53</td><td>ENSG00000177606/ENSG00000100906/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000163874/ENSG00000078401/ENSG00000171223/ENSG00000165029/ENSG00000168615/ENSG00000131979/ENSG00000136689/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000108840/ENSG00000077150/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000127666/ENSG00000239839/ENSG00000012223/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000104312/ENSG00000162551/ENSG00000162889/ENSG00000135334/ENSG00000005381/ENSG00000173039/ENSG00000143507/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000204397/ENSG00000112062/ENSG00000106366</td><td>24</td><td>ENSG00000213809/ENSG00000160791/ENSG00000187838/ENSG00000013725/ENSG00000181104/ENSG00000066468/ENSG00000168329/ENSG00000163735/ENSG00000111961/ENSG00000025434/ENSG00000117586/ENSG00000126353/ENSG00000097007/ENSG00000154229/ENSG00000081985/ENSG00000185187/ENSG00000197894/ENSG00000159674/ENSG00000162892/ENSG00000188822/ENSG00000081189/ENSG00000117560/ENSG00000162676/ENSG00000147465</td></tr>
			
		  
			
				<tr><td>BP</td><td>GO:0002237</td><td>response to molecule of bacterial origin</td><td>79/2199</td><td>284/13522</td><td>4.86204628739224e-07</td><td>0.000276747674678366</td><td>ENSG00000177606/ENSG00000100906/ENSG00000213809/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000160791/ENSG00000163874/ENSG00000078401/ENSG00000187838/ENSG00000171223/ENSG00000165029/ENSG00000013725/ENSG00000168615/ENSG00000181104/ENSG00000131979/ENSG00000066468/ENSG00000136689/ENSG00000168329/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000163735/ENSG00000108840/ENSG00000077150/ENSG00000111961/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000025434/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000163251/ENSG00000117586/ENSG00000127666/ENSG00000239839/ENSG00000128604/ENSG00000012223/ENSG00000126353/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000097007/ENSG00000104312/ENSG00000154229/ENSG00000162551/ENSG00000081985/ENSG00000185187/ENSG00000162889/ENSG00000197894/ENSG00000135334/ENSG00000159674/ENSG00000005381/ENSG00000162892/ENSG00000173039/ENSG00000143507/ENSG00000188822/ENSG00000081189/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000117560/ENSG00000204397/ENSG00000112062/ENSG00000162676/ENSG00000106366/ENSG00000147465</td><td>JUN/NFKBIA/KLRK1/ADM/ICAM1/CXCL8/TNFAIP3/PTGES/TRIB1/FOS/IRAK2/CCR5/ZC3H12A/EDN1/PLSCR3/JUNB/ABCA1/CD6/ADAM9/F2R/GCH1/FGFR2/IL1RN/CX3CR1/JUND/IRAK3/CXCL2/CXCL5/HDAC5/NFKB2/SASH1/LOXL1/ARG1/GJB6/NR1H3/IRAK1/CD55/S100A8/SLC11A1/CEBPB/CMPK2/ZFP36/THBD/FZD5/TNFSF4/TICAM1/DEFA3/IRF5/LTF/CCR7/GGT1/DEFA1/CXCL1/ABL1/RIPK2/PRKCA/ALPL/IL12RB2/SIGIRR/MAPKAPK2/ADH5/AKIRIN2/SPON2/MPO/IL24/RELA/DUSP10/CNR2/MEF2C/PLAA/DEFA4/PPARD/PELI1/FASLG/CARD16/MAPK14/GFI1/SERPINE1/STAR</td><td>79</td><td>55</td><td>ENSG00000177606/ENSG00000100906/ENSG00000148926/ENSG00000090339/ENSG00000169429/ENSG00000118503/ENSG00000148344/ENSG00000173334/ENSG00000170345/ENSG00000134070/ENSG00000163874/ENSG00000078401/ENSG00000171223/ENSG00000165029/ENSG00000168615/ENSG00000131979/ENSG00000136689/ENSG00000130522/ENSG00000090376/ENSG00000081041/ENSG00000108840/ENSG00000077150/ENSG00000129038/ENSG00000118520/ENSG00000121742/ENSG00000184216/ENSG00000196352/ENSG00000143546/ENSG00000018280/ENSG00000172216/ENSG00000134326/ENSG00000128016/ENSG00000178726/ENSG00000163251/ENSG00000127666/ENSG00000239839/ENSG00000128604/ENSG00000012223/ENSG00000100031/ENSG00000206047/ENSG00000163739/ENSG00000104312/ENSG00000162551/ENSG00000162889/ENSG00000135334/ENSG00000005381/ENSG00000173039/ENSG00000143507/ENSG00000137055/ENSG00000164821/ENSG00000112033/ENSG00000197329/ENSG00000204397/ENSG00000112062/ENSG00000106366</td><td>24</td><td>ENSG00000213809/ENSG00000160791/ENSG00000187838/ENSG00000013725/ENSG00000181104/ENSG00000066468/ENSG00000168329/ENSG00000163735/ENSG00000111961/ENSG00000025434/ENSG00000117586/ENSG00000126353/ENSG00000097007/ENSG00000154229/ENSG00000081985/ENSG00000185187/ENSG00000197894/ENSG00000159674/ENSG00000162892/ENSG00000188822/ENSG00000081189/ENSG00000117560/ENSG00000162676/ENSG00000147465</td></tr>
			
		  
		</table>
	</div>				
	</p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
        <p class="tremark">

		
        (1) Category：GO数据库子类<br />
        (2) ID：GO编号<br />
        (3) Description：GO编号对应的功能描述<br />
        (4) GeneRatio：注释到GO编号上的差异基因数与差异基因总数的比值<br />
        (5) BgRatio：注释到GO编号上的背景基因数与背景基因总数的比值<br />
        (6) pvalue：显著性检验p值<br />
        (7) padj：校正后的p值<br />
		

		
</div>
        </p>
	<br />


</div>

<!------------------------------5.1 GO富集分析------------------------------>
<div id="page">
	
	<p class="paragraph">根据GO富集分析结果，从BP，CC，MF3类选取前20个Term绘制柱状图进行展示，若不足20个，则绘制所有Term信息，结果如下图所示。</p>
    
    
		
	<p class="center">

<div class="albumSlider">
			<div class="fullview">
				<img src=group1vsgroup2/>
			</div>
			<div class="slider">
				<div class="button movebackward" title="向上滚动">
				</div>
				<div class="imglistwrap">
					<ul class="imglist">
						
						<li><a id="example2" href=group1vsgroup2 ><img src=files/DiffEnrich/GO/group1vsgroup2.bar.png/></a></li>
						
						<li><a id="example2" href=group2vssample1 ><img src=files/DiffEnrich/GO/group2vssample1.bar.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5 ><img src=files/DiffEnrich/GO/sample1vssample5.bar.png/></a></li>
						
					</ul>
				</div>
				<div class="button moveforward" title="向下滚动">
				</div>
			</div>
		</div>	
	
	</p>

	<p class="name_fig">图6.2 GO富集柱状图</p>
    

	
	<p class="cenmark">横坐标为 GO 三个大类的下一层级的 GO term，纵坐标为注释到该 term 下（包括该 term 的子 term）的差异基因个数，及其个数占被注释上的差异基因总数的比例。3 种不同分类表示 GO term 的三种基本分类（从左往右依次为生物学过程，细胞成分，分子功能）。</p>

	
	
	<p class="paragraph">一条GO term上可能会富集到上百个基因，或者是众多基因会富集到众多通路中，影响图片整体美观感，因此本结题报告只展示示例图，该绘图文件会放置在该部分对应的结果文件夹中，您可根据自己感兴趣的GO term和差异基因去magic平台上进行绘图，KEGG富集同理。</p>
	<p class="center">
	
		<img class="norma11" style="width:20%;height:18vw;text-align:center" src="files/DiffEnrich/GO/demo.rich.png" />

	</p>
	<p class="name_fig">图6.3 GO富集圈图</p>
		<p class="cenmark">图片右侧不同颜色代表不同GO term，左侧对应富集在该GO term上的基因，表达上调基因用红色点表示，下调基因用绿色点表示。</p>
		
	<br />

	
	<p class="center">

<div class="albumSlider">
			<div class="fullview">
				<img src=group1vsgroup2_all/>
			</div>
			<div class="slider">
				<div class="button movebackward" title="向上滚动">
				</div>
				<div class="imglistwrap">
					<ul class="imglist">
						
						<li><a id="example2" href=group1vsgroup2_all ><img src=files/DiffEnrich/GO/group1vsgroup2.dot.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_all ><img src=files/DiffEnrich/GO/group2vssample1.dot.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_all ><img src=files/DiffEnrich/GO/sample1vssample5.dot.png/></a></li>
						
					</ul>
				</div>
				<div class="button moveforward" title="向下滚动">
				</div>
			</div>
		</div>	
	
	</p>
	<p class="name_fig">图6.3 GO富集散点图</p>
	<p class="cenmark">横坐标为富集到该条目中基因个数占该条目总基因个数的比例，纵坐标为GO条目，每个点表示该条目的富集程度，颜色越趋近于红色表示富集程度越高。每个点的大小表示富集到该条目的基因个数</p>
        <br />
	<br />
	
</div>

<!----------------------------------5.3 KEGG富集分析---------------------------------->
	<a name="通路富集结果列表"><h4 id="result52">6.2 KEGG通路富集分析</h4></a>
	<p class="paragraph">在生物体内，不同基因相互协调行使其生物学功能，通过Pathway显著性富集能确定候选靶基因参与的最主要生化代谢途径和信号转导途径。KEGG（Kyoto Encyclopedia of Genes and Genomes）是有关Pathway的主要公共数据库。Pathway显著性富集分析以KEGG Pathway为单位，应用超几何检验，找出与整个基因组背景相比，在候选靶基因中显著性富集的Pathway。同样的，KEGG通路富集以p value或padj小于0.05为显著富集。</p>
</p>
	
	<p class="paragraph">mRNA基因表达水平KEGG富集分析见结果文件：6.DiffEnrich_mRNA/Group2_vs_Group1/KEGG，如下图所示。</p>

</br>
	<p class="name_fig">表6.2 KEGG富集列表</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
			
				<tr><th>KEGGID</th><th>Description</th><th>GeneRatio</th><th>BgRatio</th><th>pvalue</th><th>padj</th><th>geneID</th><th>geneName</th><th>keggID</th><th>Count</th><th>Up</th><th>Up_Gene_id</th><th>Down</th><th>Down_Gene_id</th></tr>
			
		  
			
				<tr><td>hsa04640</td><td>Hematopoietic cell lineage</td><td>32/1065</td><td>79/5923</td><td>2.02716750270934e-06</td><td>0.000630449093342605</td><td>ENSG00000198502/ENSG00000158481/ENSG00000116824/ENSG00000153563/ENSG00000184371/ENSG00000010610/ENSG00000182578/ENSG00000110448/ENSG00000115594/ENSG00000198851/ENSG00000115590/ENSG00000172116/ENSG00000196352/ENSG00000115232/ENSG00000173762/XLOC_165892/ENSG00000185291/ENSG00000091409/ENSG00000137070/ENSG00000160654/ENSG00000091181/ENSG00000119535/ENSG00000168685/ENSG00000104921/ENSG00000167286/ENSG00000158477/XLOC_165944/ENSG00000198223/ENSG00000166825/ENSG00000072274/ENSG00000158488/ENSG00000164171</td><td>HLA-DRB5/CD1C/CD2/CD8A/CSF1/CD4/CSF1R/CD5/IL1R1/CD3E/IL1R2/CD8B/CD55/ITGA4/CD7/XLOC_165892/IL3RA/ITGA6/IL11RA/CD3G/IL5RA/CSF3R/IL7R/FCER2/CD3D/CD1A/XLOC_165944/CSF2RA/ANPEP/TFRC/CD1E/ITGA2</td><td>hsa:3127/hsa:911/hsa:914/hsa:925/hsa:1435/hsa:920/hsa:1436/hsa:921/hsa:3554/hsa:916/hsa:7850/hsa:926/hsa:1604/hsa:3676/hsa:924/hsa:3119/hsa:3563/hsa:3655/hsa:3590/hsa:917/hsa:3568/hsa:1441/hsa:3575/hsa:2208/hsa:915/hsa:909/hsa:3125/hsa:1438/hsa:290/hsa:7037/hsa:913/hsa:3673</td><td>32</td><td>9</td><td>ENSG00000184371/ENSG00000115594/ENSG00000115590/ENSG00000196352/ENSG00000185291/ENSG00000119535/ENSG00000198223/ENSG00000166825/ENSG00000072274</td><td>23</td><td>ENSG00000198502/ENSG00000158481/ENSG00000116824/ENSG00000153563/ENSG00000010610/ENSG00000182578/ENSG00000110448/ENSG00000198851/ENSG00000172116/ENSG00000115232/ENSG00000173762/XLOC_165892/ENSG00000091409/ENSG00000137070/ENSG00000160654/ENSG00000091181/ENSG00000168685/ENSG00000104921/ENSG00000167286/ENSG00000158477/XLOC_165944/ENSG00000158488/ENSG00000164171</td></tr>
			
		  
			
				<tr><td>hsa04660</td><td>T cell receptor signaling pathway</td><td>35/1065</td><td>96/5923</td><td>1.15100349179265e-05</td><td>0.00171731840888396</td><td>ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000170345/ENSG00000178562/ENSG00000146232/ENSG00000102245/ENSG00000153563/ENSG00000010610/ENSG00000198286/ENSG00000115085/ENSG00000114423/ENSG00000198851/ENSG00000113263/ENSG00000172116/ENSG00000182866/ENSG00000198821/ENSG00000107968/ENSG00000160293/ENSG00000010810/ENSG00000172575/ENSG00000105647/ENSG00000124181/ENSG00000006062/ENSG00000160654/XLOC_164842/ENSG00000173039/ENSG00000167286/ENSG00000100351/ENSG00000141968/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>JUN/NFKBIA/NFATC2/FOS/CD28/NFKBIE/CD40LG/CD8A/CD4/CARD11/ZAP70/CBLB/CD3E/ITK/CD8B/LCK/CD247/MAP3K8/VAV2/FYN/RASGRP1/PIK3R2/PLCG1/MAP3K14/CD3G/XLOC_164842/RELA/CD3D/GRAP2/VAV1/LCP2/GRB2/MAPK13/MAPK14/LAT</td><td>hsa:3725/hsa:4792/hsa:4773/hsa:2353/hsa:940/hsa:4794/hsa:959/hsa:925/hsa:920/hsa:84433/hsa:7535/hsa:868/hsa:916/hsa:3702/hsa:926/hsa:3932/hsa:919/hsa:1326/hsa:7410/hsa:2534/hsa:10125/hsa:5296/hsa:5335/hsa:9020/hsa:917/hsa:4793/hsa:5970/hsa:915/hsa:9402/hsa:7409/hsa:3937/hsa:2885/hsa:5603/hsa:1432/hsa:27040</td><td>35</td><td>12</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/ENSG00000107968/XLOC_164842/ENSG00000173039/ENSG00000141968/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>23</td><td>ENSG00000101096/ENSG00000178562/ENSG00000102245/ENSG00000153563/ENSG00000010610/ENSG00000198286/ENSG00000115085/ENSG00000114423/ENSG00000198851/ENSG00000113263/ENSG00000172116/ENSG00000182866/ENSG00000198821/ENSG00000160293/ENSG00000010810/ENSG00000172575/ENSG00000105647/ENSG00000124181/ENSG00000006062/ENSG00000160654/ENSG00000167286/ENSG00000100351/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04658</td><td>Th1 and Th2 cell differentiation</td><td>30/1065</td><td>79/5923</td><td>1.93854624034982e-05</td><td>0.00171731840888396</td><td>ENSG00000198502/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000100385/ENSG00000170345/ENSG00000146232/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000107485/ENSG00000182866/ENSG00000198821/XLOC_165570/ENSG00000178573/XLOC_165892/ENSG00000124181/ENSG00000020633/ENSG00000160654/ENSG00000081985/XLOC_164842/ENSG00000173039/ENSG00000167286/XLOC_165944/ENSG00000101384/ENSG00000105397/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>HLA-DRB5/JUN/NFKBIA/NFATC2/IL2RB/FOS/NFKBIE/CD4/TBX21/ZAP70/CD3E/GATA3/LCK/CD247/XLOC_165570/MAF/XLOC_165892/PLCG1/RUNX3/CD3G/IL12RB2/XLOC_164842/RELA/CD3D/XLOC_165944/JAG1/TYK2/MAPK13/MAPK14/LAT</td><td>hsa:3127/hsa:3725/hsa:4792/hsa:4773/hsa:3560/hsa:2353/hsa:4794/hsa:920/hsa:30009/hsa:7535/hsa:916/hsa:2625/hsa:3932/hsa:919/hsa:3460/hsa:4094/hsa:3119/hsa:5335/hsa:864/hsa:917/hsa:3595/hsa:4793/hsa:5970/hsa:915/hsa:3125/hsa:182/hsa:7297/hsa:5603/hsa:1432/hsa:27040</td><td>30</td><td>11</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/XLOC_165570/XLOC_164842/ENSG00000173039/ENSG00000101384/ENSG00000105397/ENSG00000156711/ENSG00000112062</td><td>19</td><td>ENSG00000198502/ENSG00000101096/ENSG00000100385/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000107485/ENSG00000182866/ENSG00000198821/ENSG00000178573/XLOC_165892/ENSG00000124181/ENSG00000020633/ENSG00000160654/ENSG00000081985/ENSG00000167286/XLOC_165944/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04064</td><td>NF-kappa B signaling pathway</td><td>32/1065</td><td>87/5923</td><td>2.20876965772857e-05</td><td>0.00171731840888396</td><td>ENSG00000100906/ENSG00000090339/ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000118503/ENSG00000140379/ENSG00000102245/ENSG00000131323/ENSG00000198286/ENSG00000081041/ENSG00000135960/ENSG00000115594/ENSG00000115085/ENSG00000077150/ENSG00000082512/ENSG00000182866/ENSG00000184216/ENSG00000186197/ENSG00000127666/ENSG00000124181/ENSG00000104856/ENSG00000006062/ENSG00000141527/ENSG00000101017/ENSG00000159958/ENSG00000149311/ENSG00000143799/ENSG00000173039/ENSG00000171791/ENSG00000125735/ENSG00000213658</td><td>NFKBIA/ICAM1/CXCL8/GADD45B/PLAU/TNFAIP3/BCL2A1/CD40LG/TRAF3/CARD11/CXCL2/EDAR/IL1R1/ZAP70/NFKB2/TRAF5/LCK/IRAK1/EDARADD/TICAM1/PLCG1/RELB/MAP3K14/CARD14/CD40/TNFRSF13C/ATM/PARP1/RELA/BCL2/TNFSF14/LAT</td><td>hsa:4792/hsa:3383/hsa:3576/hsa:4616/hsa:5328/hsa:7128/hsa:597/hsa:959/hsa:7187/hsa:84433/hsa:2920/hsa:10913/hsa:3554/hsa:7535/hsa:4791/hsa:7188/hsa:3932/hsa:3654/hsa:128178/hsa:148022/hsa:5335/hsa:5971/hsa:9020/hsa:79092/hsa:958/hsa:115650/hsa:472/hsa:142/hsa:5970/hsa:596/hsa:8740/hsa:27040</td><td>32</td><td>17</td><td>ENSG00000100906/ENSG00000090339/ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000118503/ENSG00000140379/ENSG00000131323/ENSG00000081041/ENSG00000115594/ENSG00000077150/ENSG00000184216/ENSG00000127666/ENSG00000104856/ENSG00000141527/ENSG00000173039/ENSG00000125735</td><td>15</td><td>ENSG00000102245/ENSG00000198286/ENSG00000135960/ENSG00000115085/ENSG00000082512/ENSG00000182866/ENSG00000186197/ENSG00000124181/ENSG00000006062/ENSG00000101017/ENSG00000159958/ENSG00000149311/ENSG00000143799/ENSG00000171791/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04659</td><td>Th17 cell differentiation</td><td>32/1065</td><td>89/5923</td><td>3.74261880689674e-05</td><td>0.00232790889788977</td><td>ENSG00000198502/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000100385/ENSG00000170345/ENSG00000069667/ENSG00000146232/ENSG00000010610/ENSG00000100644/ENSG00000073861/ENSG00000115594/ENSG00000115085/ENSG00000198851/ENSG00000143365/ENSG00000107485/ENSG00000182866/ENSG00000198821/XLOC_165570/ENSG00000103522/XLOC_165892/ENSG00000124181/ENSG00000160654/XLOC_164842/ENSG00000173039/ENSG00000167286/XLOC_165944/ENSG00000104998/ENSG00000105397/ENSG00000156711/ENSG00000112062/ENSG00000213658</td><td>HLA-DRB5/JUN/NFKBIA/NFATC2/IL2RB/FOS/RORA/NFKBIE/CD4/HIF1A/TBX21/IL1R1/ZAP70/CD3E/RORC/GATA3/LCK/CD247/XLOC_165570/IL21R/XLOC_165892/PLCG1/CD3G/XLOC_164842/RELA/CD3D/XLOC_165944/IL27RA/TYK2/MAPK13/MAPK14/LAT</td><td>hsa:3127/hsa:3725/hsa:4792/hsa:4773/hsa:3560/hsa:2353/hsa:6095/hsa:4794/hsa:920/hsa:3091/hsa:30009/hsa:3554/hsa:7535/hsa:916/hsa:6097/hsa:2625/hsa:3932/hsa:919/hsa:3460/hsa:50615/hsa:3119/hsa:5335/hsa:917/hsa:4793/hsa:5970/hsa:915/hsa:3125/hsa:9466/hsa:7297/hsa:5603/hsa:1432/hsa:27040</td><td>32</td><td>12</td><td>ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000146232/ENSG00000100644/ENSG00000115594/XLOC_165570/XLOC_164842/ENSG00000173039/ENSG00000105397/ENSG00000156711/ENSG00000112062</td><td>20</td><td>ENSG00000198502/ENSG00000101096/ENSG00000100385/ENSG00000069667/ENSG00000010610/ENSG00000073861/ENSG00000115085/ENSG00000198851/ENSG00000143365/ENSG00000107485/ENSG00000182866/ENSG00000198821/ENSG00000103522/XLOC_165892/ENSG00000124181/ENSG00000160654/ENSG00000167286/XLOC_165944/ENSG00000104998/ENSG00000213658</td></tr>
			
		  
			
				<tr><td>hsa04060</td><td>Cytokine-cytokine receptor interaction</td><td>54/1065</td><td>185/5923</td><td>0.000100987302324078</td><td>0.0051199393317285</td><td>ENSG00000112486/ENSG00000169429/ENSG00000100385/ENSG00000099985/ENSG00000115604/ENSG00000160791/ENSG00000102245/ENSG00000184371/ENSG00000010610/ENSG00000182578/ENSG00000136689/ENSG00000168329/ENSG00000161921/ENSG00000115607/ENSG00000081041/ENSG00000135960/ENSG00000115594/ENSG00000163735/ENSG00000115590/XLOC_165570/ENSG00000103522/ENSG00000114739/ENSG00000160683/ENSG00000117586/ENSG00000054967/ENSG00000135414/ENSG00000172215/ENSG00000008517/ENSG00000185291/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000101017/ENSG00000130513/ENSG00000163739/ENSG00000137070/ENSG00000151882/ENSG00000081985/ENSG00000159958/ENSG00000162892/ENSG00000091181/ENSG00000119535/ENSG00000006327/ENSG00000128342/ENSG00000168685/ENSG00000173585/ENSG00000104998/ENSG00000121966/ENSG00000186810/ENSG00000107779/ENSG00000125735/ENSG00000117560/ENSG00000198223/ENSG00000123612</td><td>CCR6/CXCL8/IL2RB/OSM/IL18R1/CCR5/CD40LG/CSF1/CD4/CSF1R/IL1RN/CX3CR1/CXCL16/IL18RAP/CXCL2/EDAR/IL1R1/CXCL5/IL1R2/XLOC_165570/IL21R/ACVR2B/CXCR5/TNFSF4/RELT/GDF11/CXCR6/IL32/IL3RA/TGFB2/CCR7/XCL2/CD40/GDF15/CXCL1/IL11RA/CCL28/IL12RB2/TNFRSF13C/IL24/IL5RA/CSF3R/TNFRSF12A/LIF/IL7R/CCR9/IL27RA/CXCR4/CXCR3/BMPR1A/TNFSF14/FASLG/CSF2RA/ACVR1C</td><td>hsa:1235/hsa:3576/hsa:3560/hsa:5008/hsa:8809/hsa:1234/hsa:959/hsa:1435/hsa:920/hsa:1436/hsa:3557/hsa:1524/hsa:58191/hsa:8807/hsa:2920/hsa:10913/hsa:3554/hsa:6374/hsa:7850/hsa:3460/hsa:50615/hsa:93/hsa:643/hsa:7292/hsa:84957/hsa:10220/hsa:10663/hsa:9235/hsa:3563/hsa:7042/hsa:1236/hsa:6846/hsa:958/hsa:9518/hsa:2919/hsa:3590/hsa:56477/hsa:3595/hsa:115650/hsa:11009/hsa:3568/hsa:1441/hsa:51330/hsa:3976/hsa:3575/hsa:10803/hsa:9466/hsa:7852/hsa:2833/hsa:657/hsa:8740/hsa:356/hsa:1438/hsa:130399</td><td>54</td><td>21</td><td>ENSG00000169429/ENSG00000099985/ENSG00000115604/ENSG00000184371/ENSG00000136689/ENSG00000161921/ENSG00000115607/ENSG00000081041/ENSG00000115594/ENSG00000115590/XLOC_165570/ENSG00000054967/ENSG00000185291/ENSG00000130513/ENSG00000163739/ENSG00000119535/ENSG00000006327/ENSG00000128342/ENSG00000121966/ENSG00000125735/ENSG00000198223</td><td>33</td><td>ENSG00000112486/ENSG00000100385/ENSG00000160791/ENSG00000102245/ENSG00000010610/ENSG00000182578/ENSG00000168329/ENSG00000135960/ENSG00000163735/ENSG00000103522/ENSG00000114739/ENSG00000160683/ENSG00000117586/ENSG00000135414/ENSG00000172215/ENSG00000008517/ENSG00000092969/ENSG00000126353/ENSG00000143185/ENSG00000101017/ENSG00000137070/ENSG00000151882/ENSG00000081985/ENSG00000159958/ENSG00000162892/ENSG00000091181/ENSG00000168685/ENSG00000173585/ENSG00000104998/ENSG00000186810/ENSG00000107779/ENSG00000117560/ENSG00000123612</td></tr>
			
		  
			
				<tr><td>hsa04380</td><td>Osteoclast differentiation</td><td>39/1065</td><td>122/5923</td><td>0.000115239792032474</td><td>0.0051199393317285</td><td>ENSG00000125740/ENSG00000177606/ENSG00000100906/ENSG00000101096/ENSG00000170345/ENSG00000175592/ENSG00000171223/ENSG00000184371/ENSG00000182578/ENSG00000184557/ENSG00000075426/ENSG00000130522/ENSG00000204577/ENSG00000115594/XLOC_165453/ENSG00000077150/ENSG00000182866/XLOC_165520/XLOC_165570/ENSG00000132170/ENSG00000010810/ENSG00000187116/ENSG00000105647/ENSG00000092969/ENSG00000104856/ENSG00000185338/ENSG00000118260/ENSG00000161011/ENSG00000006062/XLOC_164866/ENSG00000158517/ENSG00000173039/ENSG00000152495/ENSG00000105397/ENSG00000108984/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>FOSB/JUN/NFKBIA/NFATC2/FOS/FOSL1/JUNB/CSF1/CSF1R/SOCS3/FOSL2/JUND/LILRB3/IL1R1/XLOC_165453/NFKB2/LCK/XLOC_165520/XLOC_165570/PPARG/FYN/LILRA5/PIK3R2/TGFB2/RELB/SOCS1/CREB1/SQSTM1/MAP3K14/XLOC_164866/NCF1/RELA/CAMK4/TYK2/MAP2K6/LCP2/GRB2/MAPK13/MAPK14</td><td>hsa:2354/hsa:3725/hsa:4792/hsa:4773/hsa:2353/hsa:8061/hsa:3726/hsa:1435/hsa:1436/hsa:9021/hsa:2355/hsa:3727/hsa:102725035/hsa:3554/hsa:11027/hsa:4791/hsa:3932/hsa:353514/hsa:3460/hsa:5468/hsa:2534/hsa:353514/hsa:5296/hsa:7042/hsa:5971/hsa:8651/hsa:1385/hsa:8878/hsa:9020/hsa:8878/hsa:653361/hsa:5970/hsa:814/hsa:7297/hsa:5608/hsa:3937/hsa:2885/hsa:5603/hsa:1432</td><td>39</td><td>30</td><td>ENSG00000125740/ENSG00000177606/ENSG00000100906/ENSG00000170345/ENSG00000175592/ENSG00000171223/ENSG00000184371/ENSG00000184557/ENSG00000075426/ENSG00000130522/ENSG00000204577/ENSG00000115594/XLOC_165453/ENSG00000077150/XLOC_165520/XLOC_165570/ENSG00000132170/ENSG00000187116/ENSG00000104856/ENSG00000185338/ENSG00000161011/XLOC_164866/ENSG00000158517/ENSG00000173039/ENSG00000105397/ENSG00000108984/ENSG00000043462/ENSG00000177885/ENSG00000156711/ENSG00000112062</td><td>9</td><td>ENSG00000101096/ENSG00000182578/ENSG00000182866/ENSG00000010810/ENSG00000105647/ENSG00000092969/ENSG00000118260/ENSG00000006062/ENSG00000152495</td></tr>
			
		  
			
				<tr><td>hsa04668</td><td>TNF signaling pathway</td><td>33/1065</td><td>100/5923</td><td>0.000195014405066666</td><td>0.00758118499696665</td><td>ENSG00000177606/ENSG00000100906/ENSG00000090339/ENSG00000118503/ENSG00000170345/ENSG00000115604/ENSG00000078401/ENSG00000171223/ENSG00000131323/ENSG00000184371/ENSG00000184557/ENSG00000081041/ENSG00000163735/ENSG00000082512/ENSG00000100985/ENSG00000172216/ENSG00000107968/ENSG00000105647/ENSG00000069399/ENSG00000128272/ENSG00000118260/ENSG00000006062/ENSG00000163739/ENSG00000100784/ENSG00000107175/ENSG00000173039/ENSG00000128342/ENSG00000247077/ENSG00000101384/ENSG00000108984/ENSG00000156711/ENSG00000112062/ENSG00000034152</td><td>JUN/NFKBIA/ICAM1/TNFAIP3/FOS/IL18R1/EDN1/JUNB/TRAF3/CSF1/SOCS3/CXCL2/CXCL5/TRAF5/MMP9/CEBPB/MAP3K8/PIK3R2/BCL3/ATF4/CREB1/MAP3K14/CXCL1/RPS6KA5/CREB3/RELA/LIF/PGAM5/JAG1/MAP2K6/MAPK13/MAPK14/MAP2K3</td><td>hsa:3725/hsa:4792/hsa:3383/hsa:7128/hsa:2353/hsa:8809/hsa:1906/hsa:3726/hsa:7187/hsa:1435/hsa:9021/hsa:2920/hsa:6374/hsa:7188/hsa:4318/hsa:1051/hsa:1326/hsa:5296/hsa:602/hsa:468/hsa:1385/hsa:9020/hsa:2919/hsa:9252/hsa:10488/hsa:5970/hsa:3976/hsa:192111/hsa:182/hsa:5608/hsa:5603/hsa:1432/hsa:5606</td><td>33</td><td>26</td><td>ENSG00000177606/ENSG00000100906/ENSG00000090339/ENSG00000118503/ENSG00000170345/ENSG00000115604/ENSG00000078401/ENSG00000171223/ENSG00000131323/ENSG00000184371/ENSG00000184557/ENSG00000081041/ENSG00000100985/ENSG00000172216/ENSG00000107968/ENSG00000069399/ENSG00000128272/ENSG00000163739/ENSG00000107175/ENSG00000173039/ENSG00000128342/ENSG00000101384/ENSG00000108984/ENSG00000156711/ENSG00000112062/ENSG00000034152</td><td>7</td><td>ENSG00000163735/ENSG00000082512/ENSG00000105647/ENSG00000118260/ENSG00000006062/ENSG00000100784/ENSG00000247077</td></tr>
			
		  
			
				<tr><td>hsa05202</td><td>Transcriptional misregulation in cancer</td><td>46/1065</td><td>159/5923</td><td>0.000404866397152834</td><td>0.0139903832793924</td><td>ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000100385/ENSG00000175197/ENSG00000119508/ENSG00000127152/ENSG00000140379/ENSG00000144802/ENSG00000244405/ENSG00000058091/ENSG00000182578/ENSG00000133895/ENSG00000116717/ENSG00000196092/ENSG00000130222/ENSG00000141510/ENSG00000115590/ENSG00000100985/ENSG00000178573/ENSG00000132170/ENSG00000172216/ENSG00000068323/ENSG00000158711/ENSG00000239839/ENSG00000275714/ENSG00000101017/ENSG00000118971/ENSG00000149311/ENSG00000203852/ENSG00000203811/ENSG00000005381/ENSG00000104885/ENSG00000173039/ENSG00000171843/ENSG00000116809/ENSG00000123080/ENSG00000169696/ENSG00000113916/ENSG00000118058/ENSG00000171988/ENSG00000081189/ENSG00000146674/ENSG00000007516/ENSG00000139626/ENSG00000136997</td><td>CXCL8/GADD45B/PLAU/IL2RB/DDIT3/NR4A3/BCL11B/BCL2A1/NFKBIZ/ETV5/CDK14/CSF1R/MEN1/GADD45A/PAX5/GADD45G/TP53/IL1R2/MMP9/MAF/PPARG/CEBPB/TFE3/ELK4/DEFA3/HIST1H3A/CD40/CCND2/ATM/HIST2H3A/HIST2H3C/MPO/DOT1L/RELA/MLLT3/ZBTB17/CDKN2C/ASPSCR1/BCL6/KMT2A/JMJD1C/MEF2C/IGFBP3/BAIAP3/ITGB7/MYC</td><td>hsa:3576/hsa:4616/hsa:5328/hsa:3560/hsa:1649/hsa:8013/hsa:64919/hsa:597/hsa:64332/hsa:2119/hsa:5218/hsa:1436/hsa:4221/hsa:1647/hsa:5079/hsa:10912/hsa:7157/hsa:7850/hsa:4318/hsa:4094/hsa:5468/hsa:1051/hsa:7030/hsa:2005/hsa:1668/hsa:8350/hsa:958/hsa:894/hsa:472/hsa:333932/hsa:126961/hsa:4353/hsa:84444/hsa:5970/hsa:4300/hsa:7709/hsa:1031/hsa:79058/hsa:604/hsa:4297/hsa:221037/hsa:4208/hsa:3486/hsa:8938/hsa:3695/hsa:4609</td><td>46</td><td>28</td><td>ENSG00000169429/ENSG00000099860/ENSG00000122861/ENSG00000175197/ENSG00000119508/ENSG00000140379/ENSG00000144802/ENSG00000244405/ENSG00000058091/ENSG00000116717/ENSG00000130222/ENSG00000115590/ENSG00000100985/ENSG00000132170/ENSG00000172216/ENSG00000068323/ENSG00000239839/ENSG00000275714/ENSG00000203852/ENSG00000203811/ENSG00000005381/ENSG00000104885/ENSG00000173039/ENSG00000116809/ENSG00000123080/ENSG00000113916/ENSG00000171988/ENSG00000007516</td><td>18</td><td>ENSG00000100385/ENSG00000127152/ENSG00000182578/ENSG00000133895/ENSG00000196092/ENSG00000141510/ENSG00000178573/ENSG00000158711/ENSG00000101017/ENSG00000118971/ENSG00000149311/ENSG00000171843/ENSG00000169696/ENSG00000118058/ENSG00000081189/ENSG00000146674/ENSG00000139626/ENSG00000136997</td></tr>
			
		  
			
				<tr><td>hsa04514</td><td>Cell adhesion molecules (CAMs)</td><td>35/1065</td><td>114/5923</td><td>0.000597510384625838</td><td>0.0185825729618635</td><td>XLOC_165816/ENSG00000198502/ENSG00000181847/XLOC_165828/ENSG00000090339/ENSG00000178562/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000130202/ENSG00000013725/ENSG00000010610/XLOC_165954/ENSG00000172116/ENSG00000149294/ENSG00000196338/ENSG00000115232/ENSG00000196358/XLOC_165971/XLOC_165892/ENSG00000067141/ENSG00000090863/ENSG00000091409/ENSG00000101017/XLOC_165898/ENSG00000088827/ENSG00000091129/ENSG00000107738/XLOC_165944/ENSG00000138448/ENSG00000170558/ENSG00000115884/ENSG00000139626/ENSG00000177707</td><td>XLOC_165816/HLA-DRB5/TIGIT/XLOC_165828/ICAM1/CD28/CD2/CD40LG/SPN/CD8A/NECTIN2/CD6/CD4/XLOC_165954/CD8B/NCAM1/NLGN3/ITGA4/NTNG2/XLOC_165971/XLOC_165892/NEO1/GLG1/ITGA6/CD40/XLOC_165898/SIGLEC1/NRCAM/VSIR/XLOC_165944/ITGAV/CDH2/SDC1/ITGB7/NECTIN3</td><td>hsa:3107/hsa:3127/hsa:201633/hsa:3105/hsa:3383/hsa:940/hsa:914/hsa:959/hsa:6693/hsa:925/hsa:5819/hsa:923/hsa:920/hsa:3105/hsa:926/hsa:4684/hsa:54413/hsa:3676/hsa:84628/hsa:3106/hsa:3119/hsa:4756/hsa:2734/hsa:3655/hsa:958/hsa:3105/hsa:6614/hsa:4897/hsa:64115/hsa:3125/hsa:3685/hsa:1000/hsa:6382/hsa:3695/hsa:25945</td><td>35</td><td>12</td><td>XLOC_165816/XLOC_165828/ENSG00000090339/ENSG00000130202/XLOC_165954/ENSG00000196338/ENSG00000196358/XLOC_165971/XLOC_165898/ENSG00000088827/ENSG00000107738/ENSG00000115884</td><td>23</td><td>ENSG00000198502/ENSG00000181847/ENSG00000178562/ENSG00000116824/ENSG00000102245/ENSG00000197471/ENSG00000153563/ENSG00000013725/ENSG00000010610/ENSG00000172116/ENSG00000149294/ENSG00000115232/XLOC_165892/ENSG00000067141/ENSG00000090863/ENSG00000091409/ENSG00000101017/ENSG00000091129/XLOC_165944/ENSG00000138448/ENSG00000170558/ENSG00000139626/ENSG00000177707</td></tr>
			
		  
		</table>
	</div>			
        </p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
        <p class="tremark">
        
		
		(1) Category：KEGG数据库<br />
		(2) ID：KEGG通路编号<br />
		(3) Description：KEGG通路编号对应的功能描述<br />
		(4) GeneRatio：注释到KEGG通路编号上的差异基因数与差异基因总数的比值<br />
		(5) BgRatio：注释到KEGG通路编号上的背景基因数与背景基因总数的比值<br />
		(6) pvalue：显著性检验p值<br />
		(7) padj：校正后的p值<br />
		

		
        </p>
</div>

	<br />
	<br />
<!---------------------------5.3 KEGG富集分析---------------------->

<div id="page">

    <p class="paragraph">候选靶基因KEGG富集散点图是KEGG富集分析结果的图形化展示方式。在此图中，KEGG富集程度通过Rich factor、Qvalue和富集到此通路上的基因个数来衡量。其中Rich factor指差异表达的基因中位于该pathway条目的基因数目与所有有注释基因中位于该pathway条目的基因总数的比值。Rich factor越大，表示富集的程度越大。Qvalue是做过多重假设检验校正之后的Pvalue，Qvalue的取值范围为[0,1]，越接近于零，表示富集越显著。我们挑选了富集前20位的pathway条目在该图中进行展示，若富集的pathway条目不足20条，则全部展示。</p>


<p class="center">

	<div class="albumSlider">
				<div class="fullview">
					<img src=group1vsgroup2_all/>
				</div>
				<div class="slider">
					<div class="button movebackward" title="向上滚动">
					</div>
					<div class="imglistwrap">
						<ul class="imglist">
							
							<li><a id="example2" href=group1vsgroup2_all ><img src=files/DiffEnrich/KEGG/group1vsgroup2.bar.png/></a></li>
							
							<li><a id="example2" href=group2vssample1_all ><img src=files/DiffEnrich/KEGG/group2vssample1.bar.png/></a></li>
							
							<li><a id="example2" href=sample1vssample5_all ><img src=files/DiffEnrich/KEGG/sample1vssample5.bar.png/></a></li>
							
						</ul>
					</div>
					<div class="button moveforward" title="向下滚动">
					</div>
				</div>
			</div>	
		
		</p>
	
			<p class="name_fig">图6.4 KEGG富集柱状图</p>
			<p class="cenmark">横坐标为-log<sub>10</sub>(padj)，纵坐标为KEGG条目，括号中n的数量表示富集到该KEGG的基因个数</p>


<p class="center"></p>
<div class="albumSlider">
			<div class="fullview">
				<img src=group1vsgroup2_all/>
			</div>
			<div class="slider">
				<div class="button movebackward" title="向上滚动">
				</div>
				<div class="imglistwrap">
					<ul class="imglist">
						
						<li><a id="example2" href=group1vsgroup2_all ><img src=files/DiffEnrich/KEGG/group1vsgroup2.dot.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_all ><img src=files/DiffEnrich/KEGG/group2vssample1.dot.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_all ><img src=files/DiffEnrich/KEGG/sample1vssample5.dot.png/></a></li>
											
					</ul>
				</div>
				<div class="button moveforward" title="向下滚动">
				</div>
			</div>
		</div>	
	
	</p>

	
	<p class="name_fig">图6.5 KEGG富集散点图</p>
	
	<p class="cenmark">图中纵坐标代表不同的通路，横坐标代表相应通路显著差异表达基因占该通路所有基因的比例。圆圈大小代表富集在相应通路中的基因数目，圆圈越大，代表富集在该通路中的基因越多。颜色代表富集显著性，越接近红色，代表越显著。</p>

</div>	
	
	
<div id="page">

    <p class="paragraph">KEGG富集通路图: KEGG富集分析结果，可通过信号通路图进行直观的展示，如下图所示：</p>
	
	<p class="center">	
<div class="albumSlider">
			<div class="fullview">
				<img src=group1vsgroup2_hsa00514/>
			</div>
			<div class="slider">
				<div class="button movebackward" title="向上滚动">
				</div>
				<div class="imglistwrap">
					<ul class="imglist">
						
						<li><a id="example2" href=group1vsgroup2_hsa00514 ><img src=files/DiffEnrich/KEGG/group1vsgroup2_hsa00514.png/></a></li>
						
						<li><a id="example2" href=group1vsgroup2_hsa04662 ><img src=files/DiffEnrich/KEGG/group1vsgroup2_hsa04662.png/></a></li>
						
						<li><a id="example2" href=group1vsgroup2_hsa04520 ><img src=files/DiffEnrich/KEGG/group1vsgroup2_hsa04520.png/></a></li>
						
						<li><a id="example2" href=group1vsgroup2_hsa01523 ><img src=files/DiffEnrich/KEGG/group1vsgroup2_hsa01523.png/></a></li>
						
						<li><a id="example2" href=group1vsgroup2_hsa05320 ><img src=files/DiffEnrich/KEGG/group1vsgroup2_hsa05320.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_hsa04662 ><img src=files/DiffEnrich/KEGG/group2vssample1_hsa04662.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_hsa01523 ><img src=files/DiffEnrich/KEGG/group2vssample1_hsa01523.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_hsa05320 ><img src=files/DiffEnrich/KEGG/group2vssample1_hsa05320.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_hsa04520 ><img src=files/DiffEnrich/KEGG/group2vssample1_hsa04520.png/></a></li>
						
						<li><a id="example2" href=group2vssample1_hsa00514 ><img src=files/DiffEnrich/KEGG/group2vssample1_hsa00514.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_hsa05320 ><img src=files/DiffEnrich/KEGG/sample1vssample5_hsa05320.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_hsa04662 ><img src=files/DiffEnrich/KEGG/sample1vssample5_hsa04662.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_hsa00514 ><img src=files/DiffEnrich/KEGG/sample1vssample5_hsa00514.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_hsa04520 ><img src=files/DiffEnrich/KEGG/sample1vssample5_hsa04520.png/></a></li>
						
						<li><a id="example2" href=sample1vssample5_hsa01523 ><img src=files/DiffEnrich/KEGG/sample1vssample5_hsa01523.png/></a></li>
							
					</ul>
				</div>
				<div class="button moveforward" title="向下滚动">
				</div>
			</div>
		</div>	
	
	</p>

        
        <p class="name_fig">图6.6 KEGG富集通路图</p>
        <p class="cenmark">图中颜色深浅代表样本的差异倍数，红色代表表达上调，绿色代表表达下调。颜色越深，差异倍数越大。</p>
        

	
    <p class="paragraph">定量差异富集为转录组分析的核心内容，对于生物体转录调控机制的探索非常重要。通过链特异性建库，我们可以获得lncRNA和mRNA，我们默认分别输出lncRNA和mRNA的差异分析结果。</p>
	

</br>
</div>



<!---------------------------lncRNA靶基因预测-------------------------->


<div id="page">

	<h3 id="result7"><a name="lncRNA靶基因预测与功能分析">7 lncRNA靶基因预测与功能分析(结果文件夹: Results/Diff_lncRNATarget_Enrich)</a></h3>
	<p class="paragraph">llncRNA生物学功能丰富，广泛参与生物体各类重要生理过程，可在转录水平和转录后水平上对靶基因的表达进行调控，并找出不同处理产生的差异功能通路<sup>[17]</sup>。lncRNA调控靶基因的机制多种多样，最常见的对靶基因的方式是co-location和co-expression靶基因调控。co-location（共位置）是指lncRNA可能对临近的蛋白编码基因存在调控作用，通过寻找lncRNA上下游100kb内的基因进行分析<sup>[18]</sup>；co-expression（共表达）是指lncRNA转录后作用于位置较远的靶基因的作用方式，通过多个样本间的表达相关性进行分析，一般在样本数大于5时进行此分析<sup>[19]</sup>。</p>
	<p class="paragraph">诺禾致源会对所有的lncRNA进行靶基因预测，即通过lncRNA与蛋白编码基因的位置关系（co-location）和表达相关性（co-expression）预测lncRNA的靶基因。然后，对差异lncRNA的靶基因进行功能富集分析（GO/KEGG）预测lncRNA的主要功能。报告中仅展示co-location的预测结果。</p>
	
	</br>
	<a name="lncRNA靶基因预测"><h4 id="result71">7.1 lncRNA靶基因预测</h4></a>
	
	<p class="paragraph" > 当样本数超过5个时，我们同时进行co-location和co-expression分析；若样本数小于5个则仅进行co-location分析。针对co-location调控机制，我们将co-location的阈值设定为lncRNA上下游100kb，结果如下表所示：</p>
</p>
	
	<p class="paragraph">lncRNA转录本共位置靶向基因结果见结果文件：7.Diff_lncRNATarget_Enrich/lncRNA_mRNA_co_location_results.xls，如下图所示。</p>

</br>	

	<p class="name_fig">表7.1　lncRNA co-location protein_gene列表</p>
	<p class="center">
	<div id="tb" style="width:1100px;height:286px;overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1">
			
			
				<tr class="tb1"><th class="tb1">Chromosome</th><th class="tb1">lncRNA_ID</th><th class="tb1">lncRNA_Gene_ID</th><th class="tb1">lncRNA_Gene_Symbol</th><th class="tb1">lncRNA_Status</th><th class="tb1">lncRNA_Start</th><th class="tb1">lncRNA_End</th><th class="tb1">lncRNA_Strand</th><th class="tb1">Gene_Id</th><th class="tb1">Gene_Symbol</th><th class="tb1">Gene_Start</th><th class="tb1">Gene_End</th><th class="tb1">Gene_Strand</th><th class="tb1">Distance</th><th class="tb1">Location</th></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000455207</td><td  class="tb1" style="text-align:center">ENSG00000237094</td><td  class="tb1" style="text-align:center">AL732372.2</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">373182</td><td  class="tb1" style="text-align:center">485208</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284733</td><td  class="tb1" style="text-align:center">OR4F29</td><td  class="tb1" style="text-align:center">450702</td><td  class="tb1" style="text-align:center">451697</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000440200</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">601436</td><td  class="tb1" style="text-align:center">720200</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000419394</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">586945</td><td  class="tb1" style="text-align:center">720194</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000648019</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594458</td><td  class="tb1" style="text-align:center">733064</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000634337</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827796</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000634337</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827796</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">96131</td><td  class="tb1" style="text-align:center">upstream</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000635509</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827769</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000284662</td><td  class="tb1" style="text-align:center">OR4F16</td><td  class="tb1" style="text-align:center">685678</td><td  class="tb1" style="text-align:center">686673</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">(995)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000635509</td><td  class="tb1" style="text-align:center">ENSG00000230021</td><td  class="tb1" style="text-align:center">AL669831.3</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">594308</td><td  class="tb1" style="text-align:center">827769</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">96158</td><td  class="tb1" style="text-align:center">upstream</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000478729</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">940346</td><td  class="tb1" style="text-align:center">942173</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">923927</td><td  class="tb1" style="text-align:center">944581</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">(1828)</td><td  class="tb1" style="text-align:center">sense</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">ENST00000478729</td><td  class="tb1" style="text-align:center">ENSG00000187634</td><td  class="tb1" style="text-align:center">SAMD11</td><td  class="tb1" style="text-align:center">Annotated_lncRNA</td><td  class="tb1" style="text-align:center">940346</td><td  class="tb1" style="text-align:center">942173</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">ENSG00000188976</td><td  class="tb1" style="text-align:center">NOC2L</td><td  class="tb1" style="text-align:center">944202</td><td  class="tb1" style="text-align:center">959256</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">2029</td><td  class="tb1" style="text-align:center">downstream</td></tr>
			
		  
		</table>
	</div>		
	</p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
	<p class="tremark">
		(1) Chromosome：染色体名称 <br />
		(2) lncRNA ID：ENSEMBL 数据库 ID <br />
		(3) lncRNA Gene ID：lncRNA 来源基因 ID <br />
		(4) lncRNA Gene Symbol：lncRNA 来源基因名称，若无用"--"表示 <br />
		(5) lncRNA Status：lncRNA属性，分为Annotated_lncRNA和Novel_lncRNA<br />
		(6) lncRNA_Start: lncRNA起始位置 <br />
		(7) lncRNA_End: lncRNA终止位置 <br />
		(8) lncRNA_Strand：lncRNA链属性 <br />
		(9) Gene ID：靶向mRNA转录本对应的基因ID or 预测编码蛋白的靶基因ID <br />
		(10) Gene Symbol：靶基因名称，若无用“－”表示 <br />
		(11) Gene_Start：靶基因起始位置 <br />
		(12) Gene_End：靶基因终止位置 <br />
		(13) Gene_Strand：靶基因链属性 <br />
		(14) Distance：lncRNA与靶基因的距离或两者overlap的长度，overlap的长度用带括号的数字表示，单位为 bp<br />
		(15) Location：lncRNA相对靶基因的位置关系：upstream，downstream，sense，antisense<br />

</div>
</br>

<!---------------------------7.2 差异lncRNA列表------------------------------>

<div id="page">
	<a name="差异lncRNA分析"><h4 id="result72">7.2 差异lncRNA分析</h4></a>

	<p class="paragraph">差异lncRNA分析，如前所述，差异lncRNA分析结果如表7.2所示。</p>
</p>
	
	<p class="paragraph">lncRNA转录本水平差异表达情况见结果文件：5.Diff/lncRNA/Group2_vs_Group1/，如下图所示。</p>

</br>

	<p class="name_fig">表7.2 差异lncRNA列表</p>
	<p class="center">
	<div id="tb">
		<table class="tb1">
			
				
					<tr class="tb1"><th class="tb1">transcript_id</th><th class="tb1">transcript_name</th><th class="tb1">gene_id</th><th class="tb1">gene_name</th><th class="tb1">transcript_locus</th><th class="tb1">group1_FPKM</th><th class="tb1">group2_FPKM</th><th class="tb1">log2FoldChange</th><th class="tb1">pvalue</th><th class="tb1">padj</th></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">TCONS_00025553</td><td  class="tb1" style="text-align:center">CDC42SE1-OT1</td><td  class="tb1" style="text-align:center">ENSG00000197622</td><td  class="tb1" style="text-align:center">CDC42SE1</td><td  class="tb1" style="text-align:center">1:151052114-151059574</td><td  class="tb1" style="text-align:center">29.2730395</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">11.0460016381348</td><td  class="tb1" style="text-align:center">2.05315763663687e-06</td><td  class="tb1" style="text-align:center">0.00799672754991148</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000400385</td><td  class="tb1" style="text-align:center">AATBC-201</td><td  class="tb1" style="text-align:center">ENSG00000215458</td><td  class="tb1" style="text-align:center">AATBC</td><td  class="tb1" style="text-align:center">21:43805758-43812567</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.673713</td><td  class="tb1" style="text-align:center">-9.44634991810653</td><td  class="tb1" style="text-align:center">5.36113400264938e-06</td><td  class="tb1" style="text-align:center">0.0131525499781809</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000651507</td><td  class="tb1" style="text-align:center">AC015911.11-201</td><td  class="tb1" style="text-align:center">ENSG00000286065</td><td  class="tb1" style="text-align:center">AC015911.11</td><td  class="tb1" style="text-align:center">17:35474904-35537861</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">5.31679533333333</td><td  class="tb1" style="text-align:center">-9.26754621460737</td><td  class="tb1" style="text-align:center">5.48984812223397e-06</td><td  class="tb1" style="text-align:center">0.0131525499781809</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000602755</td><td  class="tb1" style="text-align:center">AL356488.2-201</td><td  class="tb1" style="text-align:center">ENSG00000270066</td><td  class="tb1" style="text-align:center">AL356488.2</td><td  class="tb1" style="text-align:center">1:109100193-109100619</td><td  class="tb1" style="text-align:center">140.51347825</td><td  class="tb1" style="text-align:center">0.0486683333333333</td><td  class="tb1" style="text-align:center">8.10036737373499</td><td  class="tb1" style="text-align:center">6.91494520541465e-06</td><td  class="tb1" style="text-align:center">0.0153118316734938</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000429829</td><td  class="tb1" style="text-align:center">XIST-204</td><td  class="tb1" style="text-align:center">ENSG00000229807</td><td  class="tb1" style="text-align:center">XIST</td><td  class="tb1" style="text-align:center">X:73820656-73852723</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">4.49299233333333</td><td  class="tb1" style="text-align:center">-11.8583655220883</td><td  class="tb1" style="text-align:center">2.20689517599673e-05</td><td  class="tb1" style="text-align:center">0.030324037891828</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000668224</td><td  class="tb1" style="text-align:center">FTX-267</td><td  class="tb1" style="text-align:center">ENSG00000230590</td><td  class="tb1" style="text-align:center">FTX</td><td  class="tb1" style="text-align:center">X:74278483-74292373</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.61152633333333</td><td  class="tb1" style="text-align:center">-8.58891143746329</td><td  class="tb1" style="text-align:center">3.02987362742685e-05</td><td  class="tb1" style="text-align:center">0.0362947784938789</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000476784</td><td  class="tb1" style="text-align:center">TTC3-215</td><td  class="tb1" style="text-align:center">ENSG00000182670</td><td  class="tb1" style="text-align:center">TTC3</td><td  class="tb1" style="text-align:center">21:37107738-37203112</td><td  class="tb1" style="text-align:center">0.01152375</td><td  class="tb1" style="text-align:center">3.332379</td><td  class="tb1" style="text-align:center">-7.68698371716345</td><td  class="tb1" style="text-align:center">3.27769314462907e-05</td><td  class="tb1" style="text-align:center">0.0378724012285485</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000557026</td><td  class="tb1" style="text-align:center">NEK9-210</td><td  class="tb1" style="text-align:center">ENSG00000119638</td><td  class="tb1" style="text-align:center">NEK9</td><td  class="tb1" style="text-align:center">14:75082121-75117245</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.67877833333333</td><td  class="tb1" style="text-align:center">-9.63285669766275</td><td  class="tb1" style="text-align:center">3.28317190324118e-05</td><td  class="tb1" style="text-align:center">0.0378724012285485</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">ENST00000558234</td><td  class="tb1" style="text-align:center">RORA-207</td><td  class="tb1" style="text-align:center">ENSG00000069667</td><td  class="tb1" style="text-align:center">RORA</td><td  class="tb1" style="text-align:center">15:60528073-60592517</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">2.17551833333333</td><td  class="tb1" style="text-align:center">-8.4824976317958</td><td  class="tb1" style="text-align:center">3.9799893275888e-05</td><td  class="tb1" style="text-align:center">0.0442707479538794</td></tr>
				
		 	
				
					<tr class="tb1"><td  class="tb1" style="text-align:center">TCONS_00203167</td><td  class="tb1" style="text-align:center">MAVS-OT1</td><td  class="tb1" style="text-align:center">ENSG00000088888</td><td  class="tb1" style="text-align:center">MAVS</td><td  class="tb1" style="text-align:center">20:3823260-3876075</td><td  class="tb1" style="text-align:center">0</td><td  class="tb1" style="text-align:center">3.18659733333333</td><td  class="tb1" style="text-align:center">-9.81062885555481</td><td  class="tb1" style="text-align:center">5.39784567180132e-05</td><td  class="tb1" style="text-align:center">0.0512067524960217</td></tr>
				
		 			 	 
		</table>
	</div>		
	</p>
<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
	<p class="tremark">
	(1) transcript_id：转录本id<br />
	(2) transcript_name：转录本名称 <br />
	(3) gene_id：基因编号<br />
	(4) gene_name：基因名称，若无用"--"表示<br />
	(5) locus：该转录本在染色体上的位置<br />
	(6) group1：比较组合1的FPKM值<br />
	(7) group2：比较组合2的FPKM值<br />
	(8) log<sub>2</sub>FoldChange：两个样本或比较组合的差异倍数值，以log<sub>2</sub>（group1/group2）来表示，根据差异分析软件计算所得<br />
	(9) pvalue：显著性检验的p值<br />
	(10) padj：多重假设检验校正后的p值<br />

	</p>
</div>


	<p class="paragraph">差异lncRNA可以通过火山图和聚类图进行展示，如前所述，这里不再详述。</p>


	<br />

</div>

<!-------------------------------差异lncRNA火山图---------------------------------->

<!---------------------------lncRNA靶基因富集分析---------------------------------->

<!-- 这部分新版报告没有，是否要保留？ -->

<div id="page">

	<a name="差异lncRNA靶基因富集分析"><h4  id="result73">7.3 差异lncRNA靶基因富集分析</h4></a>
	<p class="paragraph">分别对差异lncRNA的co-location和co-expression的靶基因进行GO和KEGG富集分析，以预测lncRNA的功能，若样品数少于5个，则仅进行co-location靶基因富集分析。</p>
	<p class="paragraph">GO 和KEGG富集分析如前所述，在此不再赘述。</p>
</p>
	
	<p class="paragraph">lncRNA转录本靶向基因富集分析结果见结果文件：7.Diff_lncRNATarget_Enrich/Group2_vs_Group1/，如下图所示。</p>

</br>
</div>

<!-------------------------------GO/KEGG 富集分析视频链接------------------------------>
<div id="page">
	<a name="GO/KEGG 富集分析视频讲解"><h4 id="result74">7.4 GO/KEGG 富集分析讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1vdG7hIZpnVFWNqnXHROkcw " class="uri">https://pan.baidu.com/s/1vdG7hIZpnVFWNqnXHROkcw </a></font> 提取码: <font face="Arial">i5th </font></p>
    
	<br />

</div>

 
 

<!---------------------------------------可变剪接分析---------------------------------->

<div id="page">
	<a name="结构性变异分析"><h3  id="result10">8 结构性变异分析(结果文件夹: Results/Structural_Variation)</h3></a>
	
	<a name="差异可变剪切分析"><h4 id="result101">8.1 差异可变剪切分析(结果文件夹: Results/Structural_Variation/Altersplice)</h4></a>
	
	<p class="paragraph">可变剪切是指同一种前体mRNA具有多种剪切程序，形成不同的mRNA，是调控基因表达和产生蛋白质多样性的重要机制。例如, 在人类癌症进程中，广泛存在着各种形式的可变剪切，许多癌症相关基因受可变剪切调控。异常剪切、剪切过程改变，都可能对肿瘤发生发展产生重要影响，癌症特异性的剪接变体具有潜在的诊断价值，因此近些年来可变剪切分析在癌症研究中逐渐成为热点<sup>[21]</sup>。</p>

	<p class="center">
		<img class="normal" src="images/AS.png">
	</p>

	<p class="name_fig">图8.1 可变剪切形成示意图</p>

	<p class="paragraph">可变剪切差异分析包括可变剪切事件定量及表达差异显著性分析。诺禾致源可通过rMATS（http://rnaseq-mats.sourceforge.net/index.html）或JuncBASE（http://compbio.berkeley.edu/proj/juncbase/Home.html）两款软件进行可变剪切分析，默认使用rMATS<sup>[21]</sup>。</p>

	<p class="paragraph">rMATS分析五种可变剪切事件，分别为SE（外显子跳跃）、RI（内含子滞留）、MXE（互斥外显子）、A5SS（差异5’剪切位点）、A3SS（差异3’剪切位点），如图8.2所示。</p>

	<p class="center">
		<img class="normala2" src="images/rMATS.png">
	</p>

	<p class="name_fig">图8.2 rMATS分析可变剪切的五种类型</p>
	<br />
	<br />


</div>


<!---------------------------------------可变剪接事件分析---------------------------------->


	<p class="paragraph">rMATS检出可变剪切的原理大致介绍如下：</p>
	
	<p class="paragraph">每个可变剪切事件对应两个剪接体（Isoform），分别为Inclusion Isoform和Skipping Isoform，如图8.3所示（以SE事件为例）。</p>

	<p class="center">
		<img class="normala3" src="images/SE.png">
	</p>

	<p class="name_fig">图8.3 外显子跳跃事件表达量统计示例</p>

	<p class="paragraph">分别对两个Isoform进行表达量统计，并除以其有效长度得到校正后表达量，然后计算Exon Inclusion Isoform在两个Isoform总表达量的比值，最后进行两组间差异显著性分析，即为差异可变剪切分析原理，结果如表8.1所示（仅展示SE事件结果）：</p>

</p>
	
	<p class="paragraph">差异可变剪切结果见结果文件：8.Structural_Variation/Altersplice/Group2_vs_Group1，如下图所示。</p>

</br>
	
	<p class="name_fig">表8.1 差异可变剪接分析结果</p>

	<p class="center">
	<div id="tb" style="width:1100px;height:286px;overflow-y:scroll;overflow-x:scroll;">
		<table class="tb1" >
			
			
				<tr class="tb1"><th class="tb1">GeneID</th><th class="tb1">geneSymbol</th><th class="tb1">chr</th><th class="tb1">strand</th><th class="tb1">exonStart_0base</th><th class="tb1">exonEnd</th><th class="tb1">upstreamES</th><th class="tb1">upstreamEE</th><th class="tb1">downstreamES</th><th class="tb1">downstreamEE</th><th class="tb1">IJC_SAMPLE_1</th><th class="tb1">SJC_SAMPLE_1</th><th class="tb1">IJC_SAMPLE_2</th><th class="tb1">SJC_SAMPLE_2</th><th class="tb1">IncFormLen</th><th class="tb1">SkipFormLen</th><th class="tb1">PValue</th><th class="tb1">FDR</th><th class="tb1">IncLevel1</th><th class="tb1">IncLevel2</th><th class="tb1">IncLevelDifference</th></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160310</td><td  class="tb1" style="text-align:center">PRMT2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">46636438</td><td  class="tb1" style="text-align:center">46636547</td><td  class="tb1" style="text-align:center">46635673</td><td  class="tb1" style="text-align:center">46635763</td><td  class="tb1" style="text-align:center">46636895</td><td  class="tb1" style="text-align:center">46636990</td><td  class="tb1" style="text-align:center">0,0,0,3</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">14,2,5</td><td  class="tb1" style="text-align:center">1,1,1</td><td  class="tb1" style="text-align:center">258</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,NA,NA,1.0</td><td  class="tb1" style="text-align:center">0.89,0.536,0.743</td><td  class="tb1" style="text-align:center">0.277</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160310</td><td  class="tb1" style="text-align:center">PRMT2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">46661799</td><td  class="tb1" style="text-align:center">46661936</td><td  class="tb1" style="text-align:center">46649574</td><td  class="tb1" style="text-align:center">46649739</td><td  class="tb1" style="text-align:center">46663382</td><td  class="tb1" style="text-align:center">46663554</td><td  class="tb1" style="text-align:center">2,1,0,3</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">5,6,6</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">286</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,0.343,NA,1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.219</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160294</td><td  class="tb1" style="text-align:center">MCM3AP</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">46254775</td><td  class="tb1" style="text-align:center">46254844</td><td  class="tb1" style="text-align:center">46254391</td><td  class="tb1" style="text-align:center">46254526</td><td  class="tb1" style="text-align:center">46256788</td><td  class="tb1" style="text-align:center">46256986</td><td  class="tb1" style="text-align:center">1,0,2,2</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">2,13,11</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">218</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,NA,1.0,0.578</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.141</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000182871</td><td  class="tb1" style="text-align:center">COL18A1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45504415</td><td  class="tb1" style="text-align:center">45504556</td><td  class="tb1" style="text-align:center">45504010</td><td  class="tb1" style="text-align:center">45504054</td><td  class="tb1" style="text-align:center">45505133</td><td  class="tb1" style="text-align:center">45505278</td><td  class="tb1" style="text-align:center">9,12,0,5</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">12,6,1</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">290</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,0.86,NA,1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,0.339</td><td  class="tb1" style="text-align:center">0.174</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45134714</td><td  class="tb1" style="text-align:center">45134832</td><td  class="tb1" style="text-align:center">45128401</td><td  class="tb1" style="text-align:center">45128573</td><td  class="tb1" style="text-align:center">45171609</td><td  class="tb1" style="text-align:center">45171684</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">0,0,4</td><td  class="tb1" style="text-align:center">0,1,1</td><td  class="tb1" style="text-align:center">267</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,1.0,NA,NA</td><td  class="tb1" style="text-align:center">NA,0.0,0.691</td><td  class="tb1" style="text-align:center">0.655</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45184473</td><td  class="tb1" style="text-align:center">45184593</td><td  class="tb1" style="text-align:center">45183361</td><td  class="tb1" style="text-align:center">45183510</td><td  class="tb1" style="text-align:center">45184922</td><td  class="tb1" style="text-align:center">45185091</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">0,0,2</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">269</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,NA,NA,0.0</td><td  class="tb1" style="text-align:center">NA,NA,0.526</td><td  class="tb1" style="text-align:center">-0.526</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000197381</td><td  class="tb1" style="text-align:center">ADARB1</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">45204554</td><td  class="tb1" style="text-align:center">45204736</td><td  class="tb1" style="text-align:center">45184922</td><td  class="tb1" style="text-align:center">45185091</td><td  class="tb1" style="text-align:center">45220835</td><td  class="tb1" style="text-align:center">45221014</td><td  class="tb1" style="text-align:center">0,2,0,0</td><td  class="tb1" style="text-align:center">0,2,0,0</td><td  class="tb1" style="text-align:center">2,4,5</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">331</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,0.31,NA,NA</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0</td><td  class="tb1" style="text-align:center">-0.69</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000142185</td><td  class="tb1" style="text-align:center">TRPM2</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">44423644</td><td  class="tb1" style="text-align:center">44423732</td><td  class="tb1" style="text-align:center">44418422</td><td  class="tb1" style="text-align:center">44418555</td><td  class="tb1" style="text-align:center">44424851</td><td  class="tb1" style="text-align:center">44424939</td><td  class="tb1" style="text-align:center">4,6,13,0</td><td  class="tb1" style="text-align:center">1,0,0,0</td><td  class="tb1" style="text-align:center">1,0,0</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">237</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">0.715,1.0,1.0,NA</td><td  class="tb1" style="text-align:center">1.0,NA,NA</td><td  class="tb1" style="text-align:center">-0.095</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160226</td><td  class="tb1" style="text-align:center">CFAP410</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">-</td><td  class="tb1" style="text-align:center">44334060</td><td  class="tb1" style="text-align:center">44335804</td><td  class="tb1" style="text-align:center">44333032</td><td  class="tb1" style="text-align:center">44333262</td><td  class="tb1" style="text-align:center">44337648</td><td  class="tb1" style="text-align:center">44337667</td><td  class="tb1" style="text-align:center">4,2,4,0</td><td  class="tb1" style="text-align:center">0,0,0,0</td><td  class="tb1" style="text-align:center">6,6,3</td><td  class="tb1" style="text-align:center">1,1,0</td><td  class="tb1" style="text-align:center">1893</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">1.0,1.0,1.0,NA</td><td  class="tb1" style="text-align:center">0.321,0.321,1.0</td><td  class="tb1" style="text-align:center">0.453</td></tr>
			
		  
			
				<tr class="tb1"><td  class="tb1" style="text-align:center">ENSG00000160221</td><td  class="tb1" style="text-align:center">GATD3A</td><td  class="tb1" style="text-align:center">21</td><td  class="tb1" style="text-align:center">+</td><td  class="tb1" style="text-align:center">44134101</td><td  class="tb1" style="text-align:center">44134154</td><td  class="tb1" style="text-align:center">44133609</td><td  class="tb1" style="text-align:center">44133838</td><td  class="tb1" style="text-align:center">44136059</td><td  class="tb1" style="text-align:center">44136173</td><td  class="tb1" style="text-align:center">0,0,0,1</td><td  class="tb1" style="text-align:center">0,1,0,0</td><td  class="tb1" style="text-align:center">0,0,1</td><td  class="tb1" style="text-align:center">0,0,0</td><td  class="tb1" style="text-align:center">202</td><td  class="tb1" style="text-align:center">149</td><td  class="tb1" style="text-align:center">1</td><td  class="tb1" style="text-align:center">1.0</td><td  class="tb1" style="text-align:center">NA,0.0,NA,1.0</td><td  class="tb1" style="text-align:center">NA,NA,1.0</td><td  class="tb1" style="text-align:center">-0.5</td></tr>
			
		  
		</table>
	</div>		
	</p>
	<div style="width:1100px;height:115px;overflow-y:scroll;overflow-x:hidden;">
	<p class="tremark">
	(1) ID：可变剪接事件编号<br />
	(2) GeneID：可变剪接事件所在基因编号<br />
	(3) geneSymbol：可变剪接事件所在基因名称<br />
	(4) chr：可变剪接事件所在染色体<br />
	(5) strand：可变剪接事件所在链的方向<br />
	(6) exonStart_0base：发生该可变剪切事件的外显子起始位置<br />
	(7) exonEnd：发生该可变剪切事件的外显子终止位置<br />
	(8) upstreamES：发生该可变剪切事件上游exon起始位置<br />
	(9) upstreamEE：发生该可变剪切事件上游exon终止位置<br />
	(10) downstreamES：发生该可变剪切事件下游exon起始位置<br />
	(11) downstreamEE：发生该可变剪切事件下游exon终止位置<br />
	(12) IC_SAMPLE_1：可变剪接事件Exon Inclusion Isoform在SAMPLE_1（比较组合1）中的表达量<br />
	(13) SC_SAMPLE_1：可变剪接事件Exon Skipping Isoform在SAMPLE_1（比较组合1）中的表达量<br />
	(14) IC_SAMPLE_2：可变剪接事件Exon Inclusion Isoform在SAMPLE_2（比较组合2）中的表达量<br />
	(15) SC_SAMPLE_2：可变剪接事件Exon Skipping Isoform在SAMPLE_2（比较组合2）中的表达量<br />
	(16) IncFormLen：可变剪接事件Exon Inclusion Isoform的有效长度<br />
	(17) SkipFormLen：可变剪接事件Exon Skipping Isoform的有效长度<br />
	(18) PValue：可变剪接事件表达差异显著性p值<br />
	(19) FDR：可变剪接事件表达差异显著性FDR值，即矫正后的p值<br />
	(20) IncLevel1：SAMPLE_1中可变剪切事件的Exon Inclusion Isoform 与两个Isoform总表达量的比值，即比较组合1中Inclusion Isoform的占比<br />
	(21) IncLevel2：SAMPLE_2中可变剪切事件的Exon Inclusion Isoform在两个Isoform总表达量的比值，即比较组合2中Inclusion Isoform的占比<br />
	(22) IncLevelDifference：IncLevel1与IncLevel2的差值，即两比较组合之间Inclusion Isoform占比的差值<br />
	</p>
</div>

</br>

<!-------------------------------结构性变异分析视频链接------------------------------>
<div id="page">

	<a name="结构性变异分析视频讲解"><h4 id="result102">8.2 结构性变异分析讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1anwlg45eL7iPoiYVwe6yGw " class="uri">https://pan.baidu.com/s/1anwlg45eL7iPoiYVwe6yGw </a></font> 提取码: <font face="Arial">brtu  </font></p>
    
</div>

</br>


<!-------------------------- 融合基因分析------------------------------->



<!---------------------------CircRNA 鉴定（circRNA）--------------------------->


<!---------------------------12  circRNA 表达水平分布-------------------------->


<!---------------------------13  circRNA 差异分析-------------------------->


<!-----------------------------------14 差异 circRNA 来源基因富集分析---------------------------------->


<!---------------------------15 CircRNA 靶向 miRNA 位点分析---------------------------------->




<!-------------------------- CircRNA 编码潜能预测------------------------------->


<!-------------------------------结题报告结构介绍及辅助注释结果文件解读------------------------------>
<div id="page">

	<a name="结题报告结构介绍及辅助注释结果文件解读"><h4 id="result17">结题报告结构介绍及辅助注释结果文件讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://pan.baidu.com/s/1S3AC9i7IDVLAt45vfNnW7Q " class="uri">https://pan.baidu.com/s/1S3AC9i7IDVLAt45vfNnW7Q </a></font> 提取码: <font face="Arial">q9r3  </font></p>
	
	<br />
 	<br />
</div>


<!-------------------------------novomagic视频链接------------------------------>
<div id="page">

	<a name="Novomagic讲解视频"><h4 id="result19">Novomagic讲解视频</h4></a>

<p class="paragraph">链接: <font face="Arial"><a href="https://www.bilibili.com/video/BV1ng411G7T1?from=search&seid=10881808897661774313" class="uri">https://www.bilibili.com/video/BV1ng411G7T1?from=search&seid=10881808897661774313</font></p>
	
	<br />
 	<br />
</div>

<!-------------------------------------------备注------------------------------------------>


<div id="page">

	<p class="head">
	<a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	<a name="备注">北京诺禾致源科技股份有限公司</a>
	<hr />
	</p>

	<a name="备注"><h2  id="bei_zhu">五、备注</h2></a>
<a name="1 结果文件解压方法" ><h3 id="20">1 结果文件解压方法</h3></a>
	<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>
<tr><th colspan="3">结果文件解压方法</th></tr>

<tr><td rowspan="2">*.tar.gz形式的压缩文件：</td><td>Unix/Linux/Mac用户</td><td>使用tar -zxvf *.tar.gz命令</td></tr><tr><td>Windows用户</td><td>使用解压缩软件如WinRAR、7-Zip等</td></tr>

<tr><td rowspan="2">*.gz形式的压缩文件：</td><td>Unix/Linux/Mac用户</td><td>使用命令gzip -d *.gz命令</td></tr><tr><td>Windows用户</td><td>使用解压缩软件如WinRAR、7-Zip等</td></tr>

<tr><td rowspan="2">*.zip形式的压缩文件：</td><td>Unix/Linux/Mac用户</td><td>使用命令unzip *.zip命令</td></tr><tr><td>Windows用户</td><td>使用解压缩软件如WinRAR、7-Zip等</td></tr>

	</table>

<a name="2 结果文件格式说明" ><h3 id="20">2 结果文件格式说明</h3></a>
	<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>
<tr><th colspan="3">结果文件格式说明</th></tr>

<tr><td rowspan="2">*.fasta</td><td  style="word-break:break-all" rowspan="2">序列文件，fasta格式，一般为基因序列或者基因组序列。因文件一般较大，打开较为困难，为您提供了sampled*.fasta文件（部分*.fasta中的序列），方便您查看文件格式。</td><td>unix/Linux/Mac用户使用 less 或 more 命令；</td></tr><tr><td>windows使用高级文本编辑器Editplus/Notepad++等。</td></tr>

<tr><td rowspan="2">*.fq/fastq</td><td  style="word-break:break-all" rowspan="2">序列文件，fastq格式，一般为reads序列；因文件一般较大，打开较为困难。</td><td>unix/Linux/Mac用户使用 less 或 more 命令；</td></tr><tr><td>windows用户使用高级文本编辑器Editplus/Notepad++等。</td></tr>

<tr><td rowspan="2">*.xls，*.txt</td><td  style="word-break:break-all" rowspan="2">结果数据表格文件；文件以制表符（Tab）分隔。</td><td>unix/Linux/Mac用户使用 less 或 more 命令；</td></tr><tr><td>windows用户使用高级文本编辑器Editplus/Notepad++ 等，也可以用Microsoft Excel打开。</td></tr>

<tr><td rowspan="2">*.png</td><td  style="word-break:break-all" rowspan="2">结果图像文件；是位图，无损压缩。</td><td>unix/Linux/Mac用户使用display命令打开。</td></tr><tr><td>windows用户可以使用图片浏览器打开，如photoshop等。</td></tr>

<tr><td>*.pdf</td><td style="word-break:break-all">结果图像文件；是矢量图，可以放大和缩小而不失真，方便用户查看和编辑处理，可使用Adobe Illustrator进行图片编辑，用于文章发表等。</td><td>windows/Mac用户可以使用Adobe Reader/福昕阅读器/网页>浏览器等打开，unix/Linux用户使用evince命令打开。</td></tr>
	</table>


<h3 id="20">3 高级个性化分析列表和说明</h3>
	<table width=95% align="center" id="result_note" cellspacing="0" cellpadding="5" class='tf2'>
<tr><th colspan="1.5">分析条目</th><th  style="word-break:break-all"  colspan="2">交付结果（信息）</th><th style="word-break:break-all" colspan="2">分析条件（信息）</th></tr>

<tr><td colspan="1.5">WGCNA分析</td><td style="word-break:break-all"  colspan="2">模块层次聚类图，模块间相关热性图，模块基因聚类热图，模块基因表达图，样品表达模式热图</td><td style="word-break:break-all" colspan="2">建议25个以上样品（8-15也可以，基于皮尔森）</td></tr>

<tr><td colspan="1.5">GSEA 分析</td><td style="word-break:break-all"  colspan="2">GSEA富集结果图、GSEA富集基因热图</td><td style="word-break:break-all"  colspan="2">差异组合超过10个（物种为人小鼠）；根据表达趋势富集；须有表达量的注释，GO和KEGG；针对所有基因，不只是差异基因</td></tr>

<tr><td colspan="1.5">STEM 分析</td><td style="word-break:break-all"  colspan="2">基因表达模块图，模块内基因表达折线图</td><td style="word-break:break-all"  colspan="2">时间点在4-8个之间</td></tr>

<tr><td colspan="1.5">RNA编辑分析</td><td style="word-break:break-all"  colspan="2">编辑位点在染色体上分布，RNA编辑类型分布，RNA编辑类型堆积柱状图，RNA编辑位点在不同功能域的分布，编辑位点在不同功能域的堆积柱状图</td><td style="word-break:break-all"  colspan="2">需要链特异性建库</td></tr>

<tr><td colspan="1.5">个性化热图绘制</td><td style="word-break:break-all"  colspan="2">多样品差异热图分析，展示纵向标签的热图</td><td style="word-break:break-all"  colspan="2">多样品两两比较（多阶段性或者多时期样品比对分析）</td></tr>

<tr><td colspan="1.5">个性化circos图绘制</td><td style="word-break:break-all"  colspan="2">差异基因在染色体上的分布，多信息数据可视化circos图</td><td style="word-break:break-all"  colspan="2">针对差异基因</td></tr>

<tr><td colspan="1.5">维诺图分析</td><td style="word-break:break-all"  colspan="2">维诺图</td><td style="word-break:break-all"  colspan="2">几百个基因；针对于网站中有的物种可做；别的物种有参考基因组的需要提高参考基因组（有格式要求）以及GO和KEGG注释；无参也可准备相应的注释文件</td></tr>



<tr><td colspan="1.5">节律基因分析</td><td style="word-break:break-all"  colspan="2">节律基因注释，节律基因聚类分析，节律基因GO富集分析和KEGG富集分析</td><td style="word-break:break-all"  colspan="2">需要有亲代，子代信息、需要时间在2天内点6个时间节点以上</td></tr>

<tr><td colspan="1.5">等位基因偏向性表达（ASE）分析</td><td style="word-break:break-all"  colspan="2">等位基因偏向性分析表格</td><td style="word-break:break-all"  colspan="2">基于SNP/INDEL，有亲本和子代信息；涉及SNP尽量增加测序深度</td></tr>

<tr><td colspan="1.5">lncRNA Rfam家族分析</td><td style="word-break:break-all"  colspan="2">LncRNA的Rfam注释结果，LncRNA与mRNA同源性关系预测</td><td style="word-break:break-all"  colspan="2"></td></tr>

<tr><td colspan="1.5">内参基因分析 </td><td style="word-break:break-all"  colspan="2">NormFinder筛选出的内参基因</td><td style="word-break:break-all"  colspan="2">2个样品以上</td></tr>

<tr><td colspan="1.5">疾病癌症队列分子分型</td><td style="word-break:break-all"  colspan="2">癌症队列分子分型结果</td><td style="word-break:break-all"  colspan="2">20个以上样品（多样本变化差异）；没经过人为筛选的几千个基因</td></tr>
<tr><td colspan="1.5">疾病癌症样品病毒表达分析</td><td style="word-break:break-all"  colspan="2">病毒表达与癌症分型比较图</td><td style="word-break:break-all"  colspan="2">样品为人</td></tr>

<tr><td colspan="1.5">疾病癌症样品微环境分析 </td><td style="word-break:break-all"  colspan="2">病毒表达与癌症分型比较图</td><td style="word-break:break-all"  colspan="2">样品为人</td></tr>

<tr><td colspan="1.5">病毒整合位点检测</td><td style="word-break:break-all"  colspan="2">检测病毒插入circos展示图</td><td style="word-break:break-all"  colspan="2">样品为人</td></tr>

<tr><td colspan="1.5">疾病癌症融合基因注释</td><td style="word-break:break-all"  colspan="2">检测融合基因的circos展示图</td><td style="word-break:break-all"  colspan="2">样品为人和小鼠</td></tr>


	</table>
        <a name="3 结果文件目录列表" ><h3 id="20">4 结果文件目录列表</h3></a>
        <p class="paragraph">结果文件目录列表：<a href="../../Results/ReadMe_ncRNAseq.html" title="点击打开" target="_blank">HTML</a></p>
		
        
        <a name="5 软件说明列表" ><h3 id="20">5 软件说明列表</h3></a>
        <p class="paragraph">软件说明列表：<a href="nc_softwares.pdf" title="点击打开" target="_blank">PDF</a></p>
        

        
        <a name="6 lncRNA常见QA" ><h3 id="20">5 lncRNA常见 Q&A</h3></a>
        <p class="paragraph">lncRNA常见 Q&A：<a href="LncRNA_qa.pdf" title="点击打开" target="_blank">PDF</a></p>

        <a name="7 Methods英文版" ><h3 id="20">6 LncRNA Methods</h3></a>
        <p class="paragraph">Methods英文版：<a href="Methods.TransRef.pdf" title="点击打开" target="_blank">PDF</a></p>

        <a name="8 Q&A" ><h3 id="20">7 lncRNA验证手段和一般研究思路 </h3></a>
        <p class="paragraph">lncRNA验证手段和一般研究思路：<a href="lncRNA_methods.pdf" title="点击打开" target="_blank">PDF</a></p>
        

        

        <a name="12 项目优惠券" ><h3 id="20">8 项目优惠券 </h3></a>
        <p class="paragraph">项目优惠券申请：<a href="project_coupon.pdf" title="点击打开" target="_blank">PDF</a></p>

	<br />
	<br />


</div>



<!-------------------------------参考文献---------------------------------->



<div id="page">

	<p class="head">
	<a href="#home" title = "返回首页"><img class="logo" align="left" src="images/logo.png" /></a>
	<a name="参考文献">北京诺禾致源科技股份有限公司</a>
	<hr />
	</p>

	<h2 id="reference1"><a name="参考文献">六、参考文献</a></h2>

	<p class="ref">

	
	[1] Yousefi H, Maheronnaghsh M, Molaei F, et al. Long noncoding RNAs and exosomal lncRNAs: classification, and mechanisms in breast cancer metastasis and drug resistance[J].Oncogene.2019.<br /><br />
	[2] Kopp F, Mendell JT. Functional Classification and Experimental Dissection of Long Noncoding RNAs[J]. Cell. 2018, 172(3), 393–407. doi:10.1016/j.cell.2018.01.011.<br /><br />
	[3] Parkhomchuk D, Borodina T, Amstislavskiy V, et al. Transcriptome analysis by strand-specific sequencing of complementary DNA[J]. Nucleic Acids Research. 2009, 37(18), e123–e123. doi:10.1093/nar/gkp596.<br /><br />
	[4] Hardwick SA, Chen WY, Wong T, et al. Spliced synthetic genes as internal controls in RNA sequencing experiments [J]. Nature Methods. 2016, 13(9), 792–798. doi:10.1038/nmeth.3958.<br /><br />
	
	
	
	[5] Vieth B, Parekh S, Ziegenhain C, et al. A systematic evaluation of single cell RNA-seq analysis pipelines [J]. Nature Communications. 2019, 10, 1–11.<br /><br />
	[6] Kim D, Langmead B, & Salzberg SL. HISAT: a fast spliced aligner with low memory requirements [J]. Nature Methods. 12(4), 357–360. doi:10.1038/nmeth.3317. <br /><br />
	[7] Breschi A, Gingeras TR, Guigó R. Comparative transcriptomics in human and mouse. Nature Reviews Genetics. 2017, 18(7), 425–440. doi:10.1038/nrg.2017.19. <br /><br />
	
	
	
	[8] Pertea M, Pertea GM, Antonescu CM, et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads [J]. Nature Biotechnology. 2015, 33(3), 290–295. doi:10.1038/nbt.3122.<br /><br /> 
	[9] Ghosh S, Chan C-K K. Analysis of RNA-Seq Data Using TopHat and Cufflinks [J]. Methods in Molecular Biology. 2016, 339–361. doi:10.1007/978-1-4939-3167-5_18.<br /><br /> 
	[10] Mullan LJ. Short EMBOSS user guide [J]. Briefings in Bioinformatics. 2002, 3(1), 92–94. doi:10.1093/bib/3.1.92.<br /><br /> 
	

	
	[11] Garber M, Grabherr MG, Guttman M, Trapnell C. Computational methods for transcriptome annotation and quantification using RNA-seq [J]. Nature Methods. 2011, 8(6), 469–477. doi:10.1038/nmeth.1613.<br /><br /> 
	[12] Bray NL, Pimentel H, Melsted P, Pachter L. Near-optimal probabilistic RNA-seq quantification [J]. Nature Biotechnology. 2016, 34(5), 525–527. doi:10.1038/nbt.3519. <br /><br />
	

	
	[13] Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data [J]. Bioinformatics. 2019, 26(1), 139–140. doi:10.1093/bioinformatics/btp616.<br /><br />  
	

	
	[14] Young MD, Wakefield MJ, Smyth GK, Oshlack A. Gene ontology analysis for RNA-seq: accounting for selection bias [J]. Genome Biology. 2010, 11(2), R14. doi:10.1186/gb-2010-11-2-r14.<br /><br /> 
	[15] Kanehisa M, Araki M, Goto S, et al. KEGG for linking genomes to life and the environment [J]. Nucleic Acids Research. 2008, 36(Database), D480–D484. doi:10.1093/nar/gkm882.<br /><br /> 
	[16] Kanehisa M, Furumichi M, Tanabe M, et al. KEGG: new perspectives on genomes, pathways, diseases and drugs [J]. Nucleic Acids Research. 2016, 45(D1), D353–D361. doi:10.1093/nar/gkw1092. <br /><br />
	

	
	[17] Schmitt AM, Chang HY. Long Noncoding RNAs in Cancer Pathways [J]. Cancer Cell. 2016, 29(4), 452–463. doi:10.1016/j.ccell.2016.03.010.<br /><br /> 
	[18] Bao Z, Yang Z, Huang Z, et al. LncRNADisease 2.0: an updated database of long non-coding RNA-associated diseases [J]. Nucleic Acids Research. 2018, doi:10.1093/nar/gky905. <br /><br />
	[19] Kopp F, Mendell JT. Functional Classification and Experimental Dissection of Long Noncoding RNAs [J]. Cell. 2018, 172(3), 393–407. doi:10.1016/j.cell.2018.01.011.<br /><br /> 
	

	
	[20] Scotti MM, and Swanson MS. RNA mis-splicing in disease [J]. Nat. Rev. Genet. 2016, 17, 19–32.<br /><br />
	[21] Shen S, Park JW, Lu Z, et al. rMATS: Robust and flexible detection of differential alternative splicing from replicate RNA-Seq data [J]. Proceedings of the National Academy of Sciences. 2014, 111(51), E5593–E5601. doi:10.1073/pnas.1419161111.<br /><br />
	
	
	
	
	
	
	
	
	

	</p>


	<br />
	<br />


</div>


<!----------------------------------End---------------------------------->
</div>

</div>
  <script>
    $(document).ready(function () {

      // 初始化materialScrollTop
      $('body').materialScrollTop();

      $('.flexslider').flexslider({
        animation: "slide",
        slideshow: false,
        controlNav: false
      });


      $('#leftNavigation .step').click(function (event) {
        $(this).children('ul').slideToggle();
      });

      // 显示隐藏按钮切换
      $('.hide_or_show').click(function (e) {
        $(this).parent().next().fadeToggle();
      });

    });

    function displaySubMenu(li) {
      var subMenu = li.getElementsByTagName("ul")[0];
      subMenu.style.display = "block";
    }

    function hideSubMenu(li) {
      var subMenu = li.getElementsByTagName("ul")[0];
      subMenu.style.display = "none";
    }
  </script>

</body>


</html>
